<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/2/183/5419551","@type":"ScholarlyArticle","name":"A siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants","datePublished":"2019-03-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2019-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz002","keywords":["green seaweed genome","UTC clade","nuclear pore protein","TALE homeobox genes","segmental duplication"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/2/183/5419551","author":[{"name":"Arimoto, Asuka","affiliation":"Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person","sameAs":"http://orcid.org/0000-0001-8220-5920"},{"name":"Nishitsuji, Koki","affiliation":"Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person"},{"name":"Higa, Yoshimi","affiliation":"Onna Village Fisheries Cooperative, Onna, Okinawa, Japan","@type":"Person"},{"name":"Arakaki, Nana","affiliation":"DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person"},{"name":"Hisata, Kanako","affiliation":"Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person"},{"name":"Shinzato, Chuya","affiliation":"Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person"},{"name":"Satoh, Noriyuki","affiliation":"Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person"},{"name":"Shoguchi, Eiichi","affiliation":"Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan","@type":"Person"}],"description":"Abstract.  Genome evolution and development of unicellular, multinucleate macroalgae (siphonous algae) are poorly known, although various multicellular organism","pageStart":"183","pageEnd":"192","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636244854&Signature=s~VhPG9zBcD3PP9pUiJJHcm47OU-5SpffenGC~QScrsFxgSm6JmLzUu7ulGksoA649txV~C38-Nqm8QTHWA4QNUrp7CbYCHpiCZKwX4gxbx7VKpvIfZCpmVfQAxOtWOKy9ka72ovj93StBeC1VuX~RmicJoxYfQjumLXp5VsLJbA6ubDuMIdyzGeg1bDP9rpwcnkrb6GpQviHjlJVEB33gkIDGqcszCNxlVOGU9-~-d7GQ~saZCKK7J~rJBo0hu~M5bEH2rj-3HOkUzyd1JqKcSSvwcR2UDFO960czh13XXBg1iU7JgqDBkEyEmxB1TQZlopHBne7xJlPHFqhR38Xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636244854&Signature=s~VhPG9zBcD3PP9pUiJJHcm47OU-5SpffenGC~QScrsFxgSm6JmLzUu7ulGksoA649txV~C38-Nqm8QTHWA4QNUrp7CbYCHpiCZKwX4gxbx7VKpvIfZCpmVfQAxOtWOKy9ka72ovj93StBeC1VuX~RmicJoxYfQjumLXp5VsLJbA6ubDuMIdyzGeg1bDP9rpwcnkrb6GpQviHjlJVEB33gkIDGqcszCNxlVOGU9-~-d7GQ~saZCKK7J~rJBo0hu~M5bEH2rj-3HOkUzyd1JqKcSSvwcR2UDFO960czh13XXBg1iU7JgqDBkEyEmxB1TQZlopHBne7xJlPHFqhR38Xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants">
<meta property="og:description" content="Abstract.  Genome evolution and development of unicellular, multinucleate macroalgae (siphonous algae) are poorly known, although various multicellular organism">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/2/183/5419551">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636244854&amp;Signature=s~VhPG9zBcD3PP9pUiJJHcm47OU-5SpffenGC~QScrsFxgSm6JmLzUu7ulGksoA649txV~C38-Nqm8QTHWA4QNUrp7CbYCHpiCZKwX4gxbx7VKpvIfZCpmVfQAxOtWOKy9ka72ovj93StBeC1VuX~RmicJoxYfQjumLXp5VsLJbA6ubDuMIdyzGeg1bDP9rpwcnkrb6GpQviHjlJVEB33gkIDGqcszCNxlVOGU9-~-d7GQ~saZCKK7J~rJBo0hu~M5bEH2rj-3HOkUzyd1JqKcSSvwcR2UDFO960czh13XXBg1iU7JgqDBkEyEmxB1TQZlopHBne7xJlPHFqhR38Xg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-03-28">
<meta name="citation_author" content="Arimoto, Asuka">
<meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_author" content="Nishitsuji, Koki">
<meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_author" content="Higa, Yoshimi">
<meta name="citation_author_institution" content="Onna Village Fisheries Cooperative, Onna, Okinawa, Japan">
<meta name="citation_author" content="Arakaki, Nana">
<meta name="citation_author_institution" content="DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_author" content="Hisata, Kanako">
<meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_author" content="Shinzato, Chuya">
<meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_author" content="Satoh, Noriyuki">
<meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_author" content="Shoguchi, Eiichi">
<meta name="citation_author_institution" content="Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan">
<meta name="citation_title" content="A siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants">
<meta name="citation_firstpage" content="183">
<meta name="citation_lastpage" content="192">
<meta name="citation_doi" content="10.1093/dnares/dsz002">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="2">
<meta name="citation_publication_date" content="2019/04/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Plant architecture without multicellularity: quandaries over patterning and the soma-germline divide in siphonous algae;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=287">
<meta name="citation_reference" content="citation_title=Rediscovery of naturally occurring seagrape Caulerpa lentillifera from the Gulf of Mannar and its mariculture;  citation_journal_title=Curr. Sci;  citation_year=2009;  citation_volume=97;  citation_pages=1418-20; ">
<meta name="citation_reference" content="citation_title=Loss or gain? Invasive aliens and biodiversity in the Mediterranean Sea;  citation_journal_title=Mar. Pollut. Bull;  citation_year=2007;  citation_volume=55;  citation_pages=314-22; ">
<meta name="citation_reference" content="citation_title=Microbial communities in marine sediments modify success of an invasive macrophyte;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=9845">
<meta name="citation_reference" content="citation_title=Caulerpa consumption, nutritional value and farming in the Indo-Pacific region;  citation_journal_title=J. Appl. Phycol;  citation_year=2017;  citation_volume=29;  citation_pages=2249-66; ">
<meta name="citation_reference" content="citation_title=The evolution of the land plant life cycle;  citation_journal_title=New Phytol;  citation_year=2010;  citation_volume=185;  citation_pages=27-41; ">
<meta name="citation_reference" content="citation_title=The evo-devo of multinucleate cells, tissues, and organisms, and an alternative route to multicellularity;  citation_journal_title=Evol. Dev;  citation_year=2013;  citation_volume=15;  citation_pages=466-74; ">
<meta name="citation_reference" content="citation_publisher=CRC Press, Boca Raton;  citation_title=Unravelling the Algae: The Past, Present, and Future of Algal Systematics;  citation_year=2007;  citation_pages=123-153; ">
<meta name="citation_reference" content="citation_title=Phylogeny and molecular evolution of the green algae;  citation_journal_title=Crit. Rev. Plant Sci;  citation_year=2012;  citation_volume=31;  citation_pages=1-46; ">
<meta name="citation_reference" content="Guiry M. D., Guiry G. M. AlgaeBase. National University of Ireland, Galway. http://www.algaebase.org (26 April 2017, date last accessed). ">
<meta name="citation_reference" content="citation_title=New phylogenetic hypotheses for the core Chlorophyta based on chloroplast sequence data;  citation_journal_title=Front. Ecol. Evol;  citation_year=2014;  citation_volume=2;  citation_pages=63">
<meta name="citation_reference" content="citation_title=Evolution and cytological diversification of the green seaweeds (Ulvophyceae);  citation_journal_title=Mol. Biol. Evol;  citation_year=2010;  citation_volume=27;  citation_pages=2052-61; ">
<meta name="citation_reference" content="citation_title=Understanding “green” multicellularity: do seaweeds hold the key?,;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5;  citation_pages=737">
<meta name="citation_reference" content="citation_title=Insights into the evolution of multicellularity from the sea lettuce genome;  citation_journal_title=Curr. Biol;  citation_year=2018;  citation_volume=28;  citation_pages=2921-33; ">
<meta name="citation_reference" content="citation_title=What ever happened to Acetabularia? Bringing a once-classic model system into the age of molecular genetics;  citation_journal_title=Int. Rev. Cytol;  citation_year=1998;  citation_volume=182;  citation_pages=1-67; ">
<meta name="citation_reference" content="citation_title=Preparation of megabase-sized DNA from a variety of organisms using the nuclei method for advanced genomics research;  citation_journal_title=Nat. Protoc;  citation_year=2012;  citation_volume=7;  citation_pages=467-78; ">
<meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; ">
<meta name="citation_reference" content="citation_title=NextClip: an analysis and read preparation tool for Nextera Long Mate Pair libraries;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=566-8; ">
<meta name="citation_reference" content="citation_title=Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory;  citation_journal_title=BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=238">
<meta name="citation_reference" content="Li H.  2013 , Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM, arXiv preprint arXiv: 1303.3997v2 [q-bio.GN]. ">
<meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; ">
<meta name="citation_reference" content="citation_title=Redundans: an assembly pipeline for highly heterozygous genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=e113">
<meta name="citation_reference" content="citation_title=BIGMAC: breaking inaccurate genomes and merging assembled contigs for long read metagenomic assembly;  citation_journal_title=BMC Bioinformatics;  citation_year=2016;  citation_volume=17;  citation_pages=435">
<meta name="citation_reference" content="citation_title=BinSanity: unsupervised clustering of environmental microbial assemblies using coverage and affinity propagation;  citation_journal_title=PeerJ;  citation_year=2017;  citation_volume=5;  citation_pages=e3035">
<meta name="citation_reference" content="citation_title=Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e47768">
<meta name="citation_reference" content="citation_title=Assembly scaffolding with PE-contaminated mate-pair libraries;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=1925-32; ">
<meta name="citation_reference" content="citation_title=LINKS: scalable, alignment-free scaffolding of draft genomes with long reads;  citation_journal_title=Gigascience;  citation_year=2015;  citation_volume=4;  citation_pages=35">
<meta name="citation_reference" content="citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e112963">
<meta name="citation_reference" content="citation_title=BUSCO applications from quality assessments to gene prediction and phylogenomics;  citation_journal_title=Mol. Biol. Evol;  citation_year=2018;  citation_volume=35;  citation_pages=543-8; ">
<meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; ">
<meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; ">
<meta name="citation_reference" content="citation_title=GenomeScope: fast reference-free genome profiling from short reads;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_pages=2202-4; ">
<meta name="citation_reference" content="citation_title=The variant call format and VCFtools;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; ">
<meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; ">
<meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=15-21; ">
<meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=5654-66; ">
<meta name="citation_reference" content="citation_title=BRAKER1: unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=767-9; ">
<meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison;  citation_journal_title=BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=31">
<meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7">
<meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; ">
<meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; ">
<meta name="citation_reference" content="citation_title=iTAK: a program for genome-wide prediction and classification of plant transcription factors, transcriptional regulators, and protein kinases;  citation_journal_title=Mol. Plant;  citation_year=2016;  citation_volume=9;  citation_pages=1667-70; ">
<meta name="citation_reference" content="citation_title=Strigolactone signaling in Arabidopsis regulates shoot development by targeting D53-Like SMXL repressor proteins for ubiquitination and degradation;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=3128-42; ">
<meta name="citation_reference" content="citation_title=Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=4288-97; ">
<meta name="citation_reference" content="citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=157">
<meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-66; ">
<meta name="citation_reference" content="citation_title=trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1972-3; ">
<meta name="citation_reference" content="citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=268-74; ">
<meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=1572-4; ">
<meta name="citation_reference" content="citation_title=ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees;  citation_title=BMC Bioinformatics;  citation_year=2018;  citation_volume=19;  citation_pages=153">
<meta name="citation_reference" content="citation_title=BUCKy: gene tree/species tree reconciliation with Bayesian concordance analysis;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2910-1; ">
<meta name="citation_reference" content="citation_title=A draft genome of the brown alga, Cladosiphon okamuranus, S-strain: a platform for future studies of ‘mozuku’ biology;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=561-70; ">
<meta name="citation_reference" content="citation_title=UpSetR: an R package for the visualization of intersecting sets and their properties;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_pages=2938-40; ">
<meta name="citation_reference" content="Felsenstein J.  2005 , PHYLIP (Phylogeny Inference Package) version 3.6, University of Washington. ">
<meta name="citation_reference" content="citation_title=The complete chloroplast and mitochondrial genomes of the green macroalga Ulva sp. UNA00071828 (Ulvophyceae, Chlorophyta);  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0121020">
<meta name="citation_reference" content="citation_title=Chloroplast phylogenomic inference of green algae relationships;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=20528">
<meta name="citation_reference" content="citation_title=The GC-rich mitochondrial and plastid genomes of the green alga Coccomyxa give insight into the evolution of organelle DNA nucleotide landscape;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e23624">
<meta name="citation_reference" content="citation_title=The Chlamydomonas genome reveals the evolution of key animal and plant functions;  citation_journal_title=Science;  citation_year=2007;  citation_volume=318;  citation_pages=245-50; ">
<meta name="citation_reference" content="citation_title=Genomic analysis of organismal complexity in the multicellular green alga Volvox carteri;  citation_journal_title=Science;  citation_year=2010;  citation_volume=329;  citation_pages=223-6; ">
<meta name="citation_reference" content="citation_title=The Chlorella variabilis NC64A genome reveals adaptation to photosymbiosis, coevolution with viruses, and cryptic sex;  citation_journal_title=Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=2943-55; ">
<meta name="citation_reference" content="citation_title=The genome of the polar eukaryotic microalga Coccomyxa subellipsoidea reveals traits of cold adaptation;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=R39">
<meta name="citation_reference" content="citation_title=The tiny eukaryote Ostreococcus provides genomic insights into the paradox of plankton speciation;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2007;  citation_volume=104;  citation_pages=7705-10; ">
<meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium;  citation_journal_title=Nat. Genet;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; ">
<meta name="citation_reference" content="citation_title=Evolution in the cycles of life;  citation_journal_title=Annu. Rev. Genet;  citation_year=2016;  citation_volume=50;  citation_pages=133-54; ">
<meta name="citation_reference" content="citation_title=Insights into the origin and evolution of the plant hormone signaling machinery;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=167;  citation_pages=872-86; ">
<meta name="citation_reference" content="citation_title=An intracellular transcriptomic atlas of the giant coenocyte Caulerpa taxifolia;  citation_journal_title=PLoS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1004900">
<meta name="citation_reference" content="citation_title=Genome-wide analysis of homeobox gene family in legumes: identification, gene duplication and expression profiling;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0119198">
<meta name="citation_reference" content="citation_title=Early sexual origins of homeoprotein heterodimerization and evolution of the plant KNOX/BELL family;  citation_journal_title=Cell;  citation_year=2008;  citation_volume=133;  citation_pages=829-40; ">
<meta name="citation_reference" content="citation_title=Antagonistic roles for KNOX1 and KNOX2 genes in patterning the land plant body plan following an ancient gene duplication;  citation_journal_title=PLoS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1004980">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/2/183/28495569/dsz002.pdf">
<meta name="description" content="Abstract.  Genome evolution and development of unicellular, multinucleate macroalgae (siphonous algae) are poorly known, although various multicellular organism">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/2/183/5419551">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/2/183/5419551">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/2/183/5419551");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsz002';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f183%2f5419551" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f183%2f5419551" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115366"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1576706095&amp;Signature=oFEzc0Tkyykx363PcYzttVtLJdTeyRhUyTxqxqjVjVd1xpKxLkEkF7fWJuuuakH9YsIO3mNMHKE4htxi1b-sApZmc4Zhmr4e7o9FtNUdQrUg1yinx2LakGX1oX5Qgk3RCQQAmrku8mrwwfvuOqrLF0aBDWHyN3bOa~xbqejmfBt42rCrIFfnkx9-~1a72UqYqvlDG1mxAUFFdlmKkG5gtHa6vI8wqZPf8acQWVdrmcbRWkpvR3TI4UdmsxJZ-wptx0XZRFacEtiWziAMit54v2ZXJq5usG3eIURms4FsnmPyQMweQ4l3t58qjCfKM6f5VkLCnUfScKMY8CCp4LWg0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/2">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="134009613">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009613">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009615">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009615">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009620">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009620">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009646">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009646">3. Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009668">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009668">4. Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009673">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009673">Data availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009675">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009675">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="134009681">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009681">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/2/171/5304673">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/2/193/5476134">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    A siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asuka Arimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 98 966 8653. Fax. +81 98 966 8622. Email: <a href="mailto:asuka.arimoto@oist.jp" target="_blank">asuka.arimoto@oist.jp</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0001-8220-5920">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0001-8220-5920
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asuka+Arimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arimoto%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Arimoto%20Asuka%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Asuka Arimoto</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koki Nishitsuji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koki+Nishitsuji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishitsuji%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nishitsuji%20Koki%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Koki Nishitsuji</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshimi Higa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Onna Village Fisheries Cooperative, Onna, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshimi+Higa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Higa%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Higa%20Yoshimi%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Yoshimi Higa</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nana Arakaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">DNA Sequencing Section, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nana+Arakaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arakaki%20N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Arakaki%20Nana%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Nana Arakaki</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kanako Hisata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kanako+Hisata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hisata%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hisata%20Kanako%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Kanako Hisata</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chuya Shinzato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>


        <div class="info-author-correspondence">
            Present address: Atmosphere and Ocean Research Institute, The University of Tokyo, Kashiwanoha, Kashiwa 277-8564, Japan.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chuya+Shinzato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shinzato%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shinzato%20Chuya%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Chuya Shinzato</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriyuki Satoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriyuki+Satoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satoh%20N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Satoh%20Noriyuki%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Noriyuki Satoh</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eiichi Shoguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Marine Genomics Unit, Okinawa Institute of Science and Technology Graduate University, Onna, Okinawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eiichi+Shoguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shoguchi%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shoguchi%20Eiichi%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Eiichi Shoguchi</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 183–192, <a href="https://doi.org/10.1093/dnares/dsz002">https://doi.org/10.1093/dnares/dsz002</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">28 March 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">13 December 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">15 February 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">28 March 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5419551" href="/dnaresearch/article-pdf/26/2/183/28495569/dsz002.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Asuka Arimoto, Koki Nishitsuji, Yoshimi Higa, Nana Arakaki, Kanako Hisata, Chuya Shinzato, Noriyuki Satoh, Eiichi Shoguchi,  A siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants, <em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 183–192, <a href="https://doi.org/10.1093/dnares/dsz002">https://doi.org/10.1093/dnares/dsz002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5419551&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20siphonous%20macroalgal%20genome%20suggests%20convergent%20functions%20of%20homeobox%20genes%20in%20algae%20and%20land%20plants&amp;publicationDate=2019-03-28&amp;volumeNum=26&amp;issueNum=2&amp;author=Arimoto%2C%20Asuka%3B%20Nishitsuji%2C%20Koki&amp;startPage=183&amp;endPage=192&amp;contentId=10.1093%2Fdnares%2Fdsz002&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f183%2f5419551" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="134009613" id="134009613" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Genome evolution and development of unicellular, multinucleate macroalgae (siphonous algae) are poorly known, although various multicellular organisms have been studied extensively. To understand macroalgal developmental evolution, we assembled the ∼26 Mb genome of a siphonous green alga, <em>Caulerpa lentillifera</em>, with high contiguity, containing 9,311 protein-coding genes. Molecular phylogeny using 107 nuclear genes indicates that the diversification of the class Ulvophyceae, including <em>C. lentillifera</em>, occurred before the split of the Chlorophyceae and Trebouxiophyceae. Compared with other green algae, the TALE superclass of homeobox genes, which expanded in land plants, shows a series of lineage-specific duplications in this siphonous macroalga. Plant hormone signalling components were also expanded in a lineage-specific manner. Expanded transport regulators, which show spatially different expression, suggest that the structural patterning strategy of a multinucleate cell depends on diversification of nuclear pore proteins. These results not only imply functional convergence of duplicated genes among green plants, but also provide insight into evolutionary roots of green plants. Based on the present results, we propose cellular and molecular mechanisms involved in the structural differentiation in the siphonous alga.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"green seaweed genome"'>green seaweed genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"UTC clade"'>UTC clade</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"nuclear pore protein"'>nuclear pore protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"TALE homeobox genes"'>TALE homeobox genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"segmental duplication"'>segmental duplication</a>
</div>                    <h2 scrollto-destination="134009615" id="134009615" class="section-title">1. Introduction</h2>
<p>The green alga, <em>Caulerpa lentillifera</em>, belongs to the family Caulerpaceae of the order Bryopsidales. Macroscopic morphologies observed in the Bryopsidales are composed of a large, multinucleated, single cell (<a reveal-id="dsz002-F1" data-open="dsz002-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>).<a reveal-id="dsz002-B1" data-open="dsz002-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> This type of body plan is termed siphonous, and multicellularity is not observed in this group. Some siphonous algae, including <em>C. lentillifera</em>, reach meters in size, likely being the largest single cells on earth. <em>C. lentillifera</em> possesses structures analogous to fronds (leaf-like), stolons (stem-like) and rhizoids (root-like) (<a reveal-id="dsz002-F1" data-open="dsz002-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Frond morphologies are diverse in <em>Caulerpa</em>. <em>C</em>. <em>lentillifera</em> resembles clusters of green grapes (<a reveal-id="dsz002-F1" data-open="dsz002-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), and is commonly known as sea grapes, or umi-budo in Japanese. This alga is one of the major edible seaweeds in the subtropical/tropical Asia-Pacific region, especially cultivated for market in Okinawa, Japan.<a reveal-id="dsz002-B2" data-open="dsz002-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a></p>                        <a id="134009617" scrollto-destination="134009617"></a>
<div id="dsz002-F1" data-id="dsz002-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f1.png?Expires=1576485542&amp;Signature=ezHax6Ay75D0VcZX9asGv6cPFALEVi8mq7CYj2EElJgEbi094oUu14gjgo-I0-ciSpFnqeygsSaUTvayyP1eQ7svF9C9e-Ng~eoVoeZ6Ks~wc~mPEcIQC85-UHkFysT8BsC7jALms4kOgZLwGQlLJxQZsV~smXfe83fxW5qi82vxl~Dl2tQwuokavYNvi~UV5GMp~msOa25gKYSJnCRmzbi0wuHcsrKke5tF2SyNR2Iv9OGXOiB8GNBFdtsTST-WrSfP6ixL0LQ0klALl5t~ay~RK~WO-2v8iQwoJ2AtkN4bs3Iy8JGwdbuzxYEthf49gNaGa09st58UD7KBFXTuXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The siphonous alga, Caulerpa lentillifera. Cultivated C. lentillifera (photo by Dr Ken Maeda). The alga consists of many grape-like vesicles connected by stolons, and the entire alga comprises one cell with many nuclei. Scale bar, 10 mm." path-from-xml="dsz002f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009617/dsz002f1.tif" path-from-xml="dsz002f1.tif" target="_blank">Open in new tab</a><a section="134009617" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f1.png?Expires=1576485542&amp;Signature=obdd2rXQ~UnXfq8j8IcR1LisaHDVohisBK8W-wGyRxz6Y8OHyMgQ5eW8T2In66m~xmNdiJARMRYqQwhEB5hxcRG7P7CaH3vafy--8DTS~gS~ThnXj7aLOjPGGyCvArVizTkSlceejcQpoqTMx~njdmH8mCZNoevhH788Tz6sxykoStvkovdyHsJ5Hp-O7o1EHwcfQgM613EYcVYuSkrp9j7kZeEGfdT6DTOUKmECkeR5ASeMWvQDKnHsBXgfX3cvFNM9XPIbkofhQVqwuu4tCPD59dyH15Ef-bx1lfTILvTXthsutUCjeEk0n24tNXlEHYFjRq14f9tVOjFBaRv8-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009617&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The siphonous alga, <em>Caulerpa lentillifera.</em> Cultivated <em>C. lentillifera</em> (photo by Dr Ken Maeda). The alga consists of many grape-like vesicles connected by stolons, and the entire alga comprises one cell with many nuclei. Scale bar, 10 mm.</p></div>
</div>
<div content-id="dsz002-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f1.png?Expires=1576485542&amp;Signature=ezHax6Ay75D0VcZX9asGv6cPFALEVi8mq7CYj2EElJgEbi094oUu14gjgo-I0-ciSpFnqeygsSaUTvayyP1eQ7svF9C9e-Ng~eoVoeZ6Ks~wc~mPEcIQC85-UHkFysT8BsC7jALms4kOgZLwGQlLJxQZsV~smXfe83fxW5qi82vxl~Dl2tQwuokavYNvi~UV5GMp~msOa25gKYSJnCRmzbi0wuHcsrKke5tF2SyNR2Iv9OGXOiB8GNBFdtsTST-WrSfP6ixL0LQ0klALl5t~ay~RK~WO-2v8iQwoJ2AtkN4bs3Iy8JGwdbuzxYEthf49gNaGa09st58UD7KBFXTuXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The siphonous alga, Caulerpa lentillifera. Cultivated C. lentillifera (photo by Dr Ken Maeda). The alga consists of many grape-like vesicles connected by stolons, and the entire alga comprises one cell with many nuclei. Scale bar, 10 mm." path-from-xml="dsz002f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009617/dsz002f1.tif" path-from-xml="dsz002f1.tif" target="_blank">Open in new tab</a><a section="134009617" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f1.png?Expires=1576485542&amp;Signature=obdd2rXQ~UnXfq8j8IcR1LisaHDVohisBK8W-wGyRxz6Y8OHyMgQ5eW8T2In66m~xmNdiJARMRYqQwhEB5hxcRG7P7CaH3vafy--8DTS~gS~ThnXj7aLOjPGGyCvArVizTkSlceejcQpoqTMx~njdmH8mCZNoevhH788Tz6sxykoStvkovdyHsJ5Hp-O7o1EHwcfQgM613EYcVYuSkrp9j7kZeEGfdT6DTOUKmECkeR5ASeMWvQDKnHsBXgfX3cvFNM9XPIbkofhQVqwuu4tCPD59dyH15Ef-bx1lfTILvTXthsutUCjeEk0n24tNXlEHYFjRq14f9tVOjFBaRv8-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009617&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The siphonous alga, <em>Caulerpa lentillifera.</em> Cultivated <em>C. lentillifera</em> (photo by Dr Ken Maeda). The alga consists of many grape-like vesicles connected by stolons, and the entire alga comprises one cell with many nuclei. Scale bar, 10 mm.</p></div>
</div>
<p>Some <em>Caulerpa</em> species become invasive, causing environmental disturbances in coastal waters worldwide.<a reveal-id="dsz002-B3" data-open="dsz002-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> A recent report on microbial communities of marine sediments suggests that the relationship between <em>Caulerpa</em> and the microbial community is an important determinant of invasiveness.<a reveal-id="dsz002-B4" data-open="dsz002-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Cultivation and invasiveness depend upon asexual (vegetative) reproduction of macroscopic cells.<a reveal-id="dsz002-B5" data-open="dsz002-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Although multinucleate cells are also present in some clades of the Viridiplantae, including even land plants, homologies of molecular mechanisms associated with their development are little known.<a reveal-id="dsz002-B6" data-open="dsz002-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz002-B7" data-open="dsz002-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p>
<p>Inter- and intra-class relationships of ulvophytes, including six major orders (Bryopsidales, Cladophorales, Dasycladales, Trentepohliales, Ulotrichales and Ulvales), remain controversial,<a reveal-id="dsz002-B8 dsz002-B9 dsz002-B10" data-open="dsz002-B8 dsz002-B9 dsz002-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> although all members of this class are likely included in the monophyletic UTC (classes Ulvophyceae, Trebouxiophyceae and Chlorophyceae) clade.<a reveal-id="dsz002-B11" data-open="dsz002-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Ulvophyceans are useful to understand evolutionary processes underlying cellularity and the diverse structural plans of green plants, because they display a variety of architectures, as well as having unicellular, multicellular and siphonous species. In addition, multicellular bodies composed of multinucleate cells are seen in the Cladophorales. It is thought that the hypothetical common ancestor was unicellular, and that siphonous body plans were not intermediate states in the development of multicellularity.<a reveal-id="dsz002-B12" data-open="dsz002-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> These various types of morphogenesis have been studied extensively. For example, <em>Ulva</em>, which is multicellular, is one of the major experimental systems to explore mechanisms by which multicellularity evolved, as in <em>Volvox</em>.<a reveal-id="dsz002-B13" data-open="dsz002-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The draft genome of <em>Ulva mutabilis</em> has recently been published and provides insight into the evolution of multicellularity.<a reveal-id="dsz002-B14" data-open="dsz002-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The giant, mononuclear, unicellular <em>Acetabularia</em>, which belongs to the Dasycladales, has enabled elegant experiments for cutting and grafting of single cells to test whether the nucleus contains genetic materials.<a reveal-id="dsz002-B15" data-open="dsz002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><em>Caulerpa</em> likely provides a third experimental system among ulvophytes, since it is unicellular with multiple nuclei. As such, it should offer interesting comparisons with other major green plants, including land plants. However, no siphonous algal genome sequences were available for comparative genomic and developmental studies. Therefore, in order to explore evolution of plant structural plans and development, we decoded the genome of <em>C</em>. <em>lentillifera</em> and performed comparative genomic analyses using available green macroalgal genomes.</p>                    <h2 scrollto-destination="134009620" id="134009620" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="134009621" id="134009621" class="section-title">2.1. Algal samples and nucleic acid extraction</h3>
<p><em>Caulerpa lentillifera</em>, which originated from a population in Okinawa and has been cultivated in aquaria at the Onna Village Fisheries Cooperative, Okinawa, Japan, was used in this study. It is cultivated under natural light and is harvested during the daytime. Frond specimens for DNA extraction were washed with flowing UV-sterilized seawater for 4 days after harvesting, and those for RNA extraction were rinsed with 0.22-µm-filtered seawater. Any debris was removed from washed samples using tweezers. Fronds and stolons were separated for differential gene expression analysis. Algal samples were immediately frozen in liquid nitrogen and stored at −80° C until nucleic acid extraction.</p>
<p>High-molecular-weight genomic DNA was prepared from isolated nuclei. Isolation of nuclei followed the protocol of Zhang et al.<a reveal-id="dsz002-B16" data-open="dsz002-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> with some modifications. Frozen samples were ground with a mortar and pestle in liquid nitrogen and dissolved in pre-chilled nuclear isolation buffer [0.5 M sucrose, 10 mM Tris (pH 9.0), 80 mM KCl, 10 mM EDTA, 0.5% (v/v) Triton X-100, 5 mM dithiothreitol]. The purified nuclear pellet was dissolved in buffer G2 (Qiagen, 1014636). DNA was extracted using Qiagen Genomic-tips (10223, 10243). RNase and proteinase treatment and column purification followed manufacturer instructions. DNA was quantified using a dsDNA HS Assay Kit (ThermoFisher, Q32851) and DNA purity was verified with a NanoDrop 2000 Spectrophotometer (ThermoFisher). DNA integrity was checked with 0.7% (w/v) agarose gel electrophoresis.</p>
<p>Total RNA was extracted from ground macroscopic cells using Plant RNA reagent (ThermoFisher, 12322012) and a Qiagen RNeasy Plant Mini Kit (74904) with DNase treatment. RNA purity and quantity were verified with a NanoDrop 2000 Spectrophotometer. RNA integrity was confirmed with an Agilent 2100 Bioanalyzer.</p>                    <h3 scrollto-destination="134009625" id="134009625" class="section-title">2.2. Library preparation and sequencing</h3>
<p>All sequencing libraries were prepared with protocols provided by the manufacturers, except for slight modifications described below. A whole-genome shotgun sequencing library was constructed using a KAPA Hyper Prep Kit (Kapa Biosystems, KK8502) with a PCR-free method. Genomic DNA was sheared to a target size of 550 bp using the Covaris M220 system. After Illumina-sequencing adapter ligation, fragmented DNA was size selected in 1.5% agarose gel cassettes (Sage Science, BDF1510) utilizing BluePippin (Sage Science). Two mate-pair libraries were prepared using a Nextera Mate Pair Library Prep Kit (Illumina, FC-132-1001). Gel-free and gel-based methods were used for 3-kb and 6-kb libraries, respectively. Size selection of gel-based libraries was performed with 0.75% agarose gel cassettes (Sage Science, BLF7510) and BluePippin. The library for RNA sequencing was constructed with a TruSeq Stranded mRNA Library Prep Kit (Illumina, RS-122-2101). Fragmentation of purified mRNA was performed using the manufacturer’s standard protocol. Short-read DNA and RNA libraries were sequenced using Illumina MiSeq and HiSeq platforms, respectively. BluePippin sizing of a long-read DNA library was performed using 0.75% agarose gel cassettes for adjustment of the read length to 20 kb before being loaded into a PacBio RSII sequencer.</p>                    <h3 scrollto-destination="134009627" id="134009627" class="section-title">2.3. Nuclear genome assembly and genome size estimation</h3>
<p>In this study, we wished to study the <em>Caulerpa lentillifera</em> nuclear genome. To this end, we attempted to separate chloroplast genome sequences and possible contamination of bacterial sequences from nuclear sequences. First, adapter sequences and low-quality (&lt;Q20) regions in Illumina data were removed with Trimmomatic 0.33<a reveal-id="dsz002-B17" data-open="dsz002-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and Sickle 1.33 (<a class="link link-uri" href="https://github.com/najoshi/sickle" target="_blank">https://github.com/najoshi/sickle</a> (6 October 2017, date last accessed)), respectively. Sequencing data from mate-pair libraries were filtered with NextClip 1.3.1.<a reveal-id="dsz002-B18" data-open="dsz002-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p>
<p><em>(a) Chloroplast genome assembly and read removal:</em> A blastn search with default settings was performed to find PacBio reads that encode the <em>C</em>. <em>lentillifera rbcL</em> gene (accession number JN034416.1). We extracted a total of 3 Mb of longer sequences from rbcL-encoding reads. Extracted reads were assembled using sprai 0.9.9.19 (<a class="link link-uri" href="http://zombie.cb.k.u-tokyo.ac.jp/sprai/" target="_blank">http://zombie.cb.k.u-tokyo.ac.jp/sprai/</a> (12 September 2016, date last accessed)) with default settings and the circularity of assembled contigs was automatically checked in the sprai pipeline. A single linear contig was constructed. To extend the contig, all PacBio reads encoding rbcL were aligned in the assembled contig using BLASR version 5.3.574e1c2<a reveal-id="dsz002-B19" data-open="dsz002-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and flanking sequences at both ends were extracted. These sequences were assembled with the first-round PacBio reads with the sprai assembler to obtain a finalized circular contig. Sequence accuracy of the contig was polished with Arrow software (<a class="link link-uri" href="https://github.com/PacificBiosciences/GenomicConsensus" target="_blank">https://github.com/PacificBiosciences/GenomicConsensus</a> (24 July 2017, date last accessed)). All reads in the three genomic libraries that mapped onto the chloroplast genome using BWA 0.7.12,<a reveal-id="dsz002-B20" data-open="dsz002-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> were removed.</p>
<p><em>(b) Bacterial sequence assembly and read removal:</em> Remaining reads were assembled using MetaPlatanus 1.0.3 (<a class="link link-uri" href="http://platanus.bio.titech.ac.jp" target="_blank">http://platanus.bio.titech.ac.jp</a> (4 July 2018, date last accessed)) to identify bacterial sequences. MetaPlatanus performs di-codon-based clustering by considering inter-/intra-cluster linkages. Using this method as well as GC content of each cluster, bacterial sequences were identified and removed.</p>
<p><em>(c) The nuclear genome assembly:</em> The nuclear genome assembly without chloroplast and bacterial reads were generated with built-in programs in Platanus 1.2.4.<a reveal-id="dsz002-B21" data-open="dsz002-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Redundancy in primary assembly was removed using redundans 0.13c.<a reveal-id="dsz002-B22" data-open="dsz002-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> After reduction, mis-assemblies were corrected using BIGMAC 5.1<a reveal-id="dsz002-B23" data-open="dsz002-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and contaminant sequences in the scaffolds were removed using BinSanity<a reveal-id="dsz002-B24" data-open="dsz002-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> (commit version: f29c60). Cleaned scaffolds were treated with an iterative scaffolding pipeline employing three software packages, PBJelly in PBSuite 15.8.24,<a reveal-id="dsz002-B25" data-open="dsz002-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> BESST 2.2.6<a reveal-id="dsz002-B26" data-open="dsz002-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and LINKS 1.8.5.<a reveal-id="dsz002-B27" data-open="dsz002-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The improved scaffolds were polished using BWA mapping of PCR-free Illumina reads and Pilon 1.22.<a reveal-id="dsz002-B28" data-open="dsz002-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> BUSCO 3.0.2<a reveal-id="dsz002-B29" data-open="dsz002-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> with a Eukaryota dataset and CEGMA 2.5<a reveal-id="dsz002-B30" data-open="dsz002-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> were used to evaluate the final genome assembly. The genome size of <em>C. lentillifera</em> was calculated from k-mer histograms using Jellyfish 2.2.3<a reveal-id="dsz002-B31" data-open="dsz002-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and the GenomeScope web tool.<a reveal-id="dsz002-B32" data-open="dsz002-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p>                    <h3 scrollto-destination="134009632" id="134009632" class="section-title">2.4. Variant site detection and repeat analysis</h3>
<p>The number of variant sites was determined using another sequencing library from a single specimen. After adapter and low-quality region removal, reads were mapped onto the assembled genome using BWA. Ambiguous variant sites were removed using VCFtools 0.1.15<a reveal-id="dsz002-B33" data-open="dsz002-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with ‘+/d = 3/q = 30’ options.</p>
<p>RepeatModeler 1.0.8 (<a class="link link-uri" href="http://www.repeatmasker.org/RepeatModeler" target="_blank">http://www.repeatmasker.org/RepeatModeler</a> (15 September 2017, date last accessed)) and RepeatMasker 4.0.6 (<a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a> (15 September 2017, date last accessed)) were used to identify repeated elements in the assembled genome. Kimura substitution level of transposable elements (TEs) was predicted using utility scripts bundled with RepeatMasker.</p>                    <h3 scrollto-destination="134009635" id="134009635" class="section-title">2.5. Transcriptome assembly, gene prediction and annotation</h3>
<p>Transcriptomes were assembled <em>de novo</em> using Trinity 2.1.1.<a reveal-id="dsz002-B34" data-open="dsz002-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> In addition, STAR 2.5.2a<a reveal-id="dsz002-B35" data-open="dsz002-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and Trinity were combined to assemble a genome-guided transcriptome. The two types of transcriptomes were further integrated considering strand information and genome sequences using PASA 2.0.2.<a reveal-id="dsz002-B36" data-open="dsz002-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> To create a training set for gene prediction, two additional datasets were generated. Preliminary gene models were prepared from RNA-seq mapping for genome-guided transcriptome assembly using BRAKER1 pipeline 1.9.<a reveal-id="dsz002-B37" data-open="dsz002-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Protein sequences in the UniProtKB/Swiss-Prot database were aligned with the genome using Exonerate 2.2.0<a reveal-id="dsz002-B38" data-open="dsz002-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> with the ‘–percent 80’ option to find a putative conserved gene set. EVidenceModeler 1.1.1<a reveal-id="dsz002-B39" data-open="dsz002-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> was used to create a training set with weight settings for results of PASA being 5, BRAKER1 being 2 and Exonerate being 1. Hint data about exon–exon junctions and repeat information for gene prediction were obtained with Bowtie 2 version 2.2.6<a reveal-id="dsz002-B40" data-open="dsz002-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> combined with STAR and RepeatModeler/RepeatMasker pipeline, respectively. A final set of gene models was generated using AUGUSTUS 3.2.1<a reveal-id="dsz002-B41" data-open="dsz002-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> with the softmasked genome and incorporated hint data after training.</p>
<p>Sequence similarities between <em>C</em>. <em>lentillifera</em> gene models and NCBI RefSeq protein database release 79 were explored using BLAST searches (<em>E</em>-value cut-off of 10<sup>−5</sup>). HMMER 3.1b2 (<a class="link link-uri" href="http://hmmer.org" target="_blank">http://hmmer.org</a> (4 March 2015, date last accessed)) and Pfam-A 29.0 were used with default parameters to detect protein domains in gene models. Results of BLAST and HMMER searches were incorporated into gene annotations, which are available on the genome browser at <a class="link link-uri" href="http://marinegenomics.oist.jp/gallery/" target="_blank">http://marinegenomics.oist.jp/gallery/</a>. The results of the HMMER search were used to find homeodomains (Pfam accessions PF00046 and PF05920) in gene models of each species. Gene Ontology (GO) IDs were assigned using InterProScan 5.22-61.0 (<a class="link link-uri" href="https://www.ebi.ac.uk/interpro" target="_blank">https://www.ebi.ac.uk/interpro</a> (15 March 2017, date last accessed)) with a database provided by the InterProScan distributer. Transcription factors (TF), transcription regulators (TR) and protein kinases were annotated using iTAK 1.7 alpha.<a reveal-id="dsz002-B42" data-open="dsz002-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Genes involved in plant hormone biosynthesis and signalling were predicted by reciprocal BLAST search analysis with an <em>E</em>-value cut-off of 10<sup>−3</sup>. Plant hormone-related genes in <em>Arabidopsis</em> (Ref. <a reveal-id="dsz002-B43" data-open="dsz002-B43" class="link link-ref link-reveal xref-bibr">43</a>; <a class="link link-uri" href="http://www.genome.jp/kegg" target="_blank">http://www.genome.jp/kegg</a> (23 November 2017, date last accessed); <a class="link link-uri" href="http://hormones.psc.riken.jp" target="_blank">http://hormones.psc.riken.jp</a> (6 February 2017, date last accessed.)) were employed as query sequences in the BLAST search.</p>                    <h3 scrollto-destination="134009638" id="134009638" class="section-title">2.6. Expression analysis</h3>
<p>RNA-seq libraries for expression analysis were prepared from fronds and stolons of three different specimens. Expression levels of genes were based on the normalized count per million (CPM) from RNA-seq data. A trimmed mean of m-values (TMM) method was used for normalization. TMM normalization, calculation of CPM and detection of significant differences between tissues were performed using edgeR 3.20.1.<a reveal-id="dsz002-B44" data-open="dsz002-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p>                    <h3 scrollto-destination="134009640" id="134009640" class="section-title">2.7. Phylogenetic analysis</h3>
<p>OrthoFinder 1.1.4<a reveal-id="dsz002-B45" data-open="dsz002-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> was used to identify orthologous gene clusters with default settings. The longest protein sequences in each locus were used. Gene models of 16 genomes were employed for genome-based species tree reconstruction (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). An additional dataset was analysed to compare species tree topologies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Single-copy orthologous gene clusters were aligned with MAFFT 7.305<a reveal-id="dsz002-B46" data-open="dsz002-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> under default settings and extracted gapless regions using trimAl 1.4.1<a reveal-id="dsz002-B47" data-open="dsz002-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> with the ‘-nogaps’ option. IQ-TREE 1.5.3<a reveal-id="dsz002-B48" data-open="dsz002-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> was used to find partitions in concatenated alignments and optimal substitution models for each partition. These optimal parameters were used for both maximum-likelihood (ML) and Bayesian analyses. A ML phylogenetic tree was inferred with 1,000 bootstrap replications using IQ-TREE. A Bayesian phylogenetic tree was created with MrBayes 3.2.6.<a reveal-id="dsz002-B49" data-open="dsz002-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The chain length for MCMC analysis was 1,000,000 and sampling frequency was each 1,000 generations. The first 25% of sampled trees were discarded as burn-in. Both trees were constructed using <em>Cyanidioschyzon merolae</em> as an outgroup. Topology of the tree was scored further using ASTRAL 5.5.6<a reveal-id="dsz002-B50" data-open="dsz002-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> and BUCKy 1.4.4<a reveal-id="dsz002-B51" data-open="dsz002-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> based on a ML method and a Bayesian method, respectively. Amino acid alignments of each single-copy gene cluster were re-used to assess concatenated tree topologies. ML trees for plant hormone-related genes were constructed as described previously.<a reveal-id="dsz002-B52" data-open="dsz002-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> All generated trees were visualized using iTOL 3.5 (<a class="link link-uri" href="https://itol.embl.de" target="_blank">https://itol.embl.de</a> (28 November 2017, date last accessed)).</p>                    <h3 scrollto-destination="134009642" id="134009642" class="section-title">2.8. Comparative gene family analysis and synteny</h3>
<p>An UpSet plot of orthologous groups (OGs) assigned with OrthoFinder was visualized with UpSetR 1.3.3.<a reveal-id="dsz002-B53" data-open="dsz002-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> The Dollop programme in PHYLIP 3.696<a reveal-id="dsz002-B54" data-open="dsz002-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> was used to determine the presence or absence of OGs at ancestral nodes. Gains and losses of OGs in each clade were assumed from ancestral states based on least parsimony. The heat map of conserved OGs among green plants was visualized using the heat map function in R 3.6.0. Numbers of genes assigned within OGs were converted to robust <em>z</em>-scores defined by the median and interquartile range to calculate the relative abundances of genes.</p>
<p>Putative lost genes were determined by the following method. The initial dataset was a set of amino acid sequences that were not classified into the same OG with <em>Caulerpa lentillifera</em> gene models or transcriptomes that were translated to proteins. These query sequences were aligned to the <em>C</em>. <em>lentillifera</em> genome using Exonerate under default settings. Sequences aligned longer than 50% of query length were excluded as candidates of lost genes. OGs including at least one chlorophyte gene were collected. Collected sequences were assigned as <em>C</em>. <em>lentillifera</em> lost homologues. GO IDs of putative lost homologues were assigned based on Pfam domains encoded in query sequences using Pfam2GO (<a class="link link-uri" href="http://supfam.org/SUPERFAMILY/dcGO" target="_blank">http://supfam.org/SUPERFAMILY/dcGO</a> (25 October 2018, date last accessed)).</p>
<p>Genes located within 50 kb upstream or downstream of <em>BEL</em> or <em>KNOX</em> were used to trace the duplication history of the homeobox genes. Synteny among <em>BEL</em> and <em>KNOX</em> paralogous loci was estimated based on conserved OGs and/or protein domains. Paralogous conserved OGs were given by analyses using OrthoFinder as described above. Conserved protein domains in gene models were identified from the result of HMMER searches using default settings.</p>                    <h2 scrollto-destination="134009646" id="134009646" class="section-title">3. Results</h2>
                    <h3 scrollto-destination="134009647" id="134009647" class="section-title">3.1. Genome sequencing and annotation</h3>
<p>We performed whole-genome shotgun sequencing of genomic DNA using Illumina short-read and PacBio long-read platforms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) and assembled the sequences using a Platanus<a reveal-id="dsz002-B21" data-open="dsz002-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>-based hybrid pipeline (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The <em>Caulerpa lentillifera</em> genome was estimated to be ∼26 Mb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and the resulting assembly comprised ∼29 Mb (scaffold and contig N50 lengths, 948 kb and 324 kb, respectively) (<a reveal-id="dsz002-T1" data-open="dsz002-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>). Statistics regarding the assembly are comparable to those of the <em>Chlorella variabilis</em> and <em>Ostreococcus tauri</em> genomes (<a reveal-id="dsz002-T1" data-open="dsz002-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The heterozygosity rate estimated from the frequency of variant sites was ∼0.4%. The GC content of the genome was 40%, lower than in other green algae, most of which have GC contents close to 60% (<a reveal-id="dsz002-T1" data-open="dsz002-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Repetitive sequences constituted 6.7% of the genome, which is slightly higher than those in the genomes of <em>Coccomyxa subellipsoidea</em> (5.4%) and <em>Ostreococcus tauri</em> (5.3%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Comparison of the substitution level of TEs among the four algal genomes showed a low frequency of low-substitution-rate TEs in <em>C</em>. <em>lentillifera</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The substitution level in the <em>C</em>. <em>lentillifera</em> genome is more similar to that of <em>Ostreococcus</em> than <em>Chlamydomonas</em> or <em>Chlorella</em>. The <em>C</em>. <em>lentillifera</em> genome contains 4.6% unknown repeats (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), which need to be further characterized in comparison with sequences of other ulvophytes. </p>                        <a id="134009649" scrollto-destination="134009649"></a>
<div content-id="dsz002-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz002-T1" class="table-wrap-title" data-id="dsz002-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Assembly and annotation features of the <em>Caulerpa lentillifera</em> and other algal genomes</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Genome features </th>
<th>
<em>Caulerpa lentillifera</em> </th>
<th>
<em>Chlamydomonas reinhardtii</em><a reveal-id="dsz002-B58" data-open="dsz002-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> </th>
<th>
<em>Chlorella variabilis</em><a reveal-id="dsz002-B60" data-open="dsz002-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </th>
<th>
<em>Ostreococcus tauri</em><a reveal-id="dsz002-B62" data-open="dsz002-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Assembled genome size (Mb) </td>
<td>28.7 </td>
<td>111.1 </td>
<td>46.2 </td>
<td>12.6 </td>
</tr>
<tr>
<td>Number of scaffolds (≥500 bp) </td>
<td>185 </td>
<td>54 </td>
<td>414 </td>
<td>103 </td>
</tr>
<tr>
<td>Number of N50 scaffolds </td>
<td>14 </td>
<td>7 </td>
<td>12 </td>
<td>7 </td>
</tr>
<tr>
<td>N50 scaffold length (kb) </td>
<td>948 </td>
<td>7,783 </td>
<td>1,470 </td>
<td>739 </td>
</tr>
<tr>
<td>Longest scaffold (Mb) </td>
<td>1.29 </td>
<td>9.73 </td>
<td>3.12 </td>
<td>1.08 </td>
</tr>
<tr>
<td>Number of N50 contigs </td>
<td>28 </td>
<td>141 </td>
<td>438 </td>
<td>243 </td>
</tr>
<tr>
<td>N50 contig length (kb) </td>
<td>324 </td>
<td>215 </td>
<td>28 </td>
<td>15 </td>
</tr>
<tr>
<td>GC content (%) </td>
<td>40.4 </td>
<td>64.1 </td>
<td>67.1 </td>
<td>59.2 </td>
</tr>
<tr>
<td>Predicted protein-coding genes </td>
<td>9,311 </td>
<td>17,741 </td>
<td>9,791 </td>
<td>7,664 </td>
</tr>
<tr>
<td>Gene density (kb/gene) </td>
<td>3.2 </td>
<td>6.2 </td>
<td>4.7 </td>
<td>1.6 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz002-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Genome features </th>
<th>
<em>Caulerpa lentillifera</em> </th>
<th>
<em>Chlamydomonas reinhardtii</em><a reveal-id="dsz002-B58" data-open="dsz002-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> </th>
<th>
<em>Chlorella variabilis</em><a reveal-id="dsz002-B60" data-open="dsz002-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </th>
<th>
<em>Ostreococcus tauri</em><a reveal-id="dsz002-B62" data-open="dsz002-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Assembled genome size (Mb) </td>
<td>28.7 </td>
<td>111.1 </td>
<td>46.2 </td>
<td>12.6 </td>
</tr>
<tr>
<td>Number of scaffolds (≥500 bp) </td>
<td>185 </td>
<td>54 </td>
<td>414 </td>
<td>103 </td>
</tr>
<tr>
<td>Number of N50 scaffolds </td>
<td>14 </td>
<td>7 </td>
<td>12 </td>
<td>7 </td>
</tr>
<tr>
<td>N50 scaffold length (kb) </td>
<td>948 </td>
<td>7,783 </td>
<td>1,470 </td>
<td>739 </td>
</tr>
<tr>
<td>Longest scaffold (Mb) </td>
<td>1.29 </td>
<td>9.73 </td>
<td>3.12 </td>
<td>1.08 </td>
</tr>
<tr>
<td>Number of N50 contigs </td>
<td>28 </td>
<td>141 </td>
<td>438 </td>
<td>243 </td>
</tr>
<tr>
<td>N50 contig length (kb) </td>
<td>324 </td>
<td>215 </td>
<td>28 </td>
<td>15 </td>
</tr>
<tr>
<td>GC content (%) </td>
<td>40.4 </td>
<td>64.1 </td>
<td>67.1 </td>
<td>59.2 </td>
</tr>
<tr>
<td>Predicted protein-coding genes </td>
<td>9,311 </td>
<td>17,741 </td>
<td>9,791 </td>
<td>7,664 </td>
</tr>
<tr>
<td>Gene density (kb/gene) </td>
<td>3.2 </td>
<td>6.2 </td>
<td>4.7 </td>
<td>1.6 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134009649" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz002-T1" class="table-wrap-title" data-id="dsz002-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Assembly and annotation features of the <em>Caulerpa lentillifera</em> and other algal genomes</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Genome features </th>
<th>
<em>Caulerpa lentillifera</em> </th>
<th>
<em>Chlamydomonas reinhardtii</em><a reveal-id="dsz002-B58" data-open="dsz002-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> </th>
<th>
<em>Chlorella variabilis</em><a reveal-id="dsz002-B60" data-open="dsz002-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </th>
<th>
<em>Ostreococcus tauri</em><a reveal-id="dsz002-B62" data-open="dsz002-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Assembled genome size (Mb) </td>
<td>28.7 </td>
<td>111.1 </td>
<td>46.2 </td>
<td>12.6 </td>
</tr>
<tr>
<td>Number of scaffolds (≥500 bp) </td>
<td>185 </td>
<td>54 </td>
<td>414 </td>
<td>103 </td>
</tr>
<tr>
<td>Number of N50 scaffolds </td>
<td>14 </td>
<td>7 </td>
<td>12 </td>
<td>7 </td>
</tr>
<tr>
<td>N50 scaffold length (kb) </td>
<td>948 </td>
<td>7,783 </td>
<td>1,470 </td>
<td>739 </td>
</tr>
<tr>
<td>Longest scaffold (Mb) </td>
<td>1.29 </td>
<td>9.73 </td>
<td>3.12 </td>
<td>1.08 </td>
</tr>
<tr>
<td>Number of N50 contigs </td>
<td>28 </td>
<td>141 </td>
<td>438 </td>
<td>243 </td>
</tr>
<tr>
<td>N50 contig length (kb) </td>
<td>324 </td>
<td>215 </td>
<td>28 </td>
<td>15 </td>
</tr>
<tr>
<td>GC content (%) </td>
<td>40.4 </td>
<td>64.1 </td>
<td>67.1 </td>
<td>59.2 </td>
</tr>
<tr>
<td>Predicted protein-coding genes </td>
<td>9,311 </td>
<td>17,741 </td>
<td>9,791 </td>
<td>7,664 </td>
</tr>
<tr>
<td>Gene density (kb/gene) </td>
<td>3.2 </td>
<td>6.2 </td>
<td>4.7 </td>
<td>1.6 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz002-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Genome features </th>
<th>
<em>Caulerpa lentillifera</em> </th>
<th>
<em>Chlamydomonas reinhardtii</em><a reveal-id="dsz002-B58" data-open="dsz002-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> </th>
<th>
<em>Chlorella variabilis</em><a reveal-id="dsz002-B60" data-open="dsz002-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </th>
<th>
<em>Ostreococcus tauri</em><a reveal-id="dsz002-B62" data-open="dsz002-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Assembled genome size (Mb) </td>
<td>28.7 </td>
<td>111.1 </td>
<td>46.2 </td>
<td>12.6 </td>
</tr>
<tr>
<td>Number of scaffolds (≥500 bp) </td>
<td>185 </td>
<td>54 </td>
<td>414 </td>
<td>103 </td>
</tr>
<tr>
<td>Number of N50 scaffolds </td>
<td>14 </td>
<td>7 </td>
<td>12 </td>
<td>7 </td>
</tr>
<tr>
<td>N50 scaffold length (kb) </td>
<td>948 </td>
<td>7,783 </td>
<td>1,470 </td>
<td>739 </td>
</tr>
<tr>
<td>Longest scaffold (Mb) </td>
<td>1.29 </td>
<td>9.73 </td>
<td>3.12 </td>
<td>1.08 </td>
</tr>
<tr>
<td>Number of N50 contigs </td>
<td>28 </td>
<td>141 </td>
<td>438 </td>
<td>243 </td>
</tr>
<tr>
<td>N50 contig length (kb) </td>
<td>324 </td>
<td>215 </td>
<td>28 </td>
<td>15 </td>
</tr>
<tr>
<td>GC content (%) </td>
<td>40.4 </td>
<td>64.1 </td>
<td>67.1 </td>
<td>59.2 </td>
</tr>
<tr>
<td>Predicted protein-coding genes </td>
<td>9,311 </td>
<td>17,741 </td>
<td>9,791 </td>
<td>7,664 </td>
</tr>
<tr>
<td>Gene density (kb/gene) </td>
<td>3.2 </td>
<td>6.2 </td>
<td>4.7 </td>
<td>1.6 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134009649" target="_blank">Open in new tab</a></div>
</div></div>
<p>RNA-seq reads (29.2 Gb) assembled with the PASA pipeline<a reveal-id="dsz002-B36" data-open="dsz002-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S2</a></span>) were used to produce gene models with AUGUSTUS.<a reveal-id="dsz002-B41" data-open="dsz002-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> A final set of <em>C. lentillifera</em> protein-coding gene models numbered 9,311 (<a reveal-id="dsz002-T1" data-open="dsz002-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Mapping of RNA-seq reads to scaffolds confirmed that at least 6,231 of 9,311 genes (∼67%) were expressed at the macroscopic stage of <em>C. lentillifera</em>.</p>
<p>BUSCO<a reveal-id="dsz002-B29" data-open="dsz002-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> analysis, which was used to evaluate the genome assembly, showed that 86.4% of BUSCO single-copy orthologous queries were completely aligned with the assembled genome and 81.8% of the input queries were present in a single copy (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). These proportions of the <em>C</em>. <em>lentillifera</em> genome are comparable to those of other green algal genomes so far decoded, indicating the accuracy of the <em>C</em>. <em>lentillifera</em> genome assembly. The genome size and gene density of <em>C</em>. <em>lentillifera</em> (one gene/3.2 kb) appeared intermediate among the green algae (<a reveal-id="dsz002-T1" data-open="dsz002-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                    <h3 scrollto-destination="134009652" id="134009652" class="section-title">3.2. Phylogenetic analyses</h3>
<p>To examine the phylogenetic position of the ulvophycean <em>Caulerpa lentillifera</em> among green algae, we constructed a molecular phylogenetic tree of algae and land plants, based on a comparison of 107 nuclear proteins from <em>C</em>. <em>lentillifera</em> and 15 published genomes (<a reveal-id="dsz002-F2" data-open="dsz002-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S1</a></span>). Our analysis showed that the diversification of <em>C</em>. <em>lentillifera</em> occurred earlier than the split of the Chlorophyceae and Trebouxiophyceae with topological consistency among calculation methods. Some of the concordance factors from BUCKy were not high among the UTC clade due to limited taxonomic sampling. High-quality chlorophyte transcriptomic data were found in public databases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). By including an additional five species in the potential UTC clade, we confirmed that the topology of the early split of the <em>Caulerpa</em> lineage is maintained in the clade (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Therefore, an early branching of <em>Caulerpa</em> was evident among members of the UTC clade (<a reveal-id="dsz002-F2" data-open="dsz002-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4 and S5</a></span>). Our present result (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), based on nuclear genes, failed to support the Ulvophyceae (<em>Caulerpa</em> and <em>Ulva</em>) as a monophyletic group.<a reveal-id="dsz002-B55" data-open="dsz002-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsz002-B56" data-open="dsz002-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a></p>                        <a id="134009654" scrollto-destination="134009654"></a>
<div id="dsz002-F2" data-id="dsz002-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f2.png?Expires=1576485542&amp;Signature=ozu-greWYZYatXBhyOONMXUEn4mf4801aVR~1Sc~-vydJBJueW28~y6ZOhe0oJXx6UDjZVo113~Rcus4G0HBd46-RbXSI1jTuf0JsUy5p6HIXzs~fkOJUx67pU82yr-tvQ8ByqCQlkhAFHyUvCzqj5nm0CBb-qy67MRKPbSA~BPaX3MOCIiXN3F18fSa6s11uwlwkaRqAIxpr~h96IfknrwaQMC5x1qodHua0pnUIcRJCEXGXMlCT2MoCSFfSPAAQnUHJmBC5sJqm7G5PSMZ~Wh7YwFxb8aBorBNFmkU9TYYU57XN1c59zh6vlPzZB~gvHHrbZObMRgZY962YuCWzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolution of gene families in green algae showing predicted gains and losses of orthologous gene clusters in each branch. This phylogenetic tree was constructed with the optimal maximum-likelihood method using a concatenation of 107 nuclear-gene-encoded protein alignments. Detailed information regarding the tree is shown in Supplementary Fig. S4. Numbers of assigned OGs, orphans, and genes are shown next to species names. Numbers in parentheses are average gene numbers in OGs." path-from-xml="dsz002f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009654/dsz002f2.tif" path-from-xml="dsz002f2.tif" target="_blank">Open in new tab</a><a section="134009654" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f2.png?Expires=1576485542&amp;Signature=jUqvaCi9yAyJEdYjU-LsLrPkpbMqZcwwOr7W1kbqBX5JhX~WolyLzVIOOJPk7qFj39pv3RwbXfezKA-xWKmIXJIMnc~RyYjQkbvXfeI4TN-omDIO6CB55XRmj7sTzYmYNbu5gf5JqH4uq3Efk7591zDt16GqLDZJsXVUEps2nQ2alxl216z2Hp~Uyy1q2CG2xFmxeH5CFfUpn9FTahu4kMe65f8nA1zp1FSYkH3L3urq~2D5faH6NFhnUNq8fMOym2RdPKPaNkxGxbcWxv9~MNPXTraWyCsfI~gIx~eamjHcod8iTLvDCsCDDoNOYbYK7D1s0htAPqYpg1INHqKyTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009654&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Evolution of gene families in green algae showing predicted gains and losses of orthologous gene clusters in each branch. This phylogenetic tree was constructed with the optimal maximum-likelihood method using a concatenation of 107 nuclear-gene-encoded protein alignments. Detailed information regarding the tree is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>. Numbers of assigned OGs, orphans, and genes are shown next to species names. Numbers in parentheses are average gene numbers in OGs.</p></div>
</div>
<div content-id="dsz002-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f2.png?Expires=1576485542&amp;Signature=ozu-greWYZYatXBhyOONMXUEn4mf4801aVR~1Sc~-vydJBJueW28~y6ZOhe0oJXx6UDjZVo113~Rcus4G0HBd46-RbXSI1jTuf0JsUy5p6HIXzs~fkOJUx67pU82yr-tvQ8ByqCQlkhAFHyUvCzqj5nm0CBb-qy67MRKPbSA~BPaX3MOCIiXN3F18fSa6s11uwlwkaRqAIxpr~h96IfknrwaQMC5x1qodHua0pnUIcRJCEXGXMlCT2MoCSFfSPAAQnUHJmBC5sJqm7G5PSMZ~Wh7YwFxb8aBorBNFmkU9TYYU57XN1c59zh6vlPzZB~gvHHrbZObMRgZY962YuCWzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolution of gene families in green algae showing predicted gains and losses of orthologous gene clusters in each branch. This phylogenetic tree was constructed with the optimal maximum-likelihood method using a concatenation of 107 nuclear-gene-encoded protein alignments. Detailed information regarding the tree is shown in Supplementary Fig. S4. Numbers of assigned OGs, orphans, and genes are shown next to species names. Numbers in parentheses are average gene numbers in OGs." path-from-xml="dsz002f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009654/dsz002f2.tif" path-from-xml="dsz002f2.tif" target="_blank">Open in new tab</a><a section="134009654" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f2.png?Expires=1576485542&amp;Signature=jUqvaCi9yAyJEdYjU-LsLrPkpbMqZcwwOr7W1kbqBX5JhX~WolyLzVIOOJPk7qFj39pv3RwbXfezKA-xWKmIXJIMnc~RyYjQkbvXfeI4TN-omDIO6CB55XRmj7sTzYmYNbu5gf5JqH4uq3Efk7591zDt16GqLDZJsXVUEps2nQ2alxl216z2Hp~Uyy1q2CG2xFmxeH5CFfUpn9FTahu4kMe65f8nA1zp1FSYkH3L3urq~2D5faH6NFhnUNq8fMOym2RdPKPaNkxGxbcWxv9~MNPXTraWyCsfI~gIx~eamjHcod8iTLvDCsCDDoNOYbYK7D1s0htAPqYpg1INHqKyTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009654&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Evolution of gene families in green algae showing predicted gains and losses of orthologous gene clusters in each branch. This phylogenetic tree was constructed with the optimal maximum-likelihood method using a concatenation of 107 nuclear-gene-encoded protein alignments. Detailed information regarding the tree is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>. Numbers of assigned OGs, orphans, and genes are shown next to species names. Numbers in parentheses are average gene numbers in OGs.</p></div>
</div>
<p>Based on a comparison of chloroplast genes, Fučíková et al.<a reveal-id="dsz002-B11" data-open="dsz002-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> reported that siphonous ulvophytes, including <em>Caulerpa</em>, are sister to a part of the class Trebouxiophyceae. A mitochondrial multigene analysis supported a sister relationship between the Ulvophyceae and Chlorophyceae.<a reveal-id="dsz002-B57" data-open="dsz002-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> However, our analysis demonstrated the earlier diversification of Ulvophyceae (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4 and S5</a></span>) suggesting the paraphyly of Ulvophyceae to the other two clades. Specifically, our phylogenetic analysis was inconsistent with previous reports that used organelle-encoded genes.<a reveal-id="dsz002-B11" data-open="dsz002-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsz002-B57" data-open="dsz002-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Genomic information from <em>C</em>. <em>lentillifera</em> will facilitate further exploration of the phylogeny of green plants. In addition, genomic data from lineages that have ancestral cytomorphological characters, such as Chlorocystidales, Oltmannsiellopsidales, Scotinosphaerales or Ignatiales,<a reveal-id="dsz002-B10" data-open="dsz002-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsz002-B12" data-open="dsz002-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> are needed to provide a more detailed evolutionary history of ulvophytes, because a recent analysis of the <em>Ulva mutabilis</em> genome indicated a sister relationship between <em>Ulva</em> and the Chlorophyceae.<a reveal-id="dsz002-B14" data-open="dsz002-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p>                    <h3 scrollto-destination="134009656" id="134009656" class="section-title">3.3. Comparative analysis of gene families and lineage-specific gene expansions</h3>
<p>To understand the gene content of siphonous macroalga and to elucidate plant gene family evolution, we classified all <em>Caulerpa lentillifera</em> proteins and compared them using OrthoFinder,<a reveal-id="dsz002-B45" data-open="dsz002-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> with those of five other green algae (<em>Chlamydomonas reinhardtii</em>,<a reveal-id="dsz002-B58" data-open="dsz002-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><em>Volvox carteri</em>,<a reveal-id="dsz002-B59" data-open="dsz002-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><em>Chlorella variabilis</em>,<a reveal-id="dsz002-B60" data-open="dsz002-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><em>Coccomyxa subellipsoidea</em><a reveal-id="dsz002-B61" data-open="dsz002-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> and <em>Ostreococcus tauri</em><a reveal-id="dsz002-B62" data-open="dsz002-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a>) (<a reveal-id="dsz002-F2" data-open="dsz002-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S1</a></span>). <em>C. lentillifera</em> had 4,256 of the classified OGs. In this comparative analysis, we used the term ‘OG’ to define a gene family. The classified OGs are likely sets of highly similar sequences that were derived from a single gene in the last common ancestor.<a reveal-id="dsz002-B45" data-open="dsz002-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><em>C</em>. <em>lentillifera</em> OGs shared 3,846 of 8,547 <em>Chlamydomonas</em> OGs (45%), 3,779 of 8,273 <em>Volvox</em> OGs (46%), 3,470 of 5,407 <em>Chlorella</em> OGs (64%), 3,437 of 5,266 <em>Coccomyxa</em> OGs (65%) and 2,875 of 5,414 <em>Ostreococcus tauri</em> OGs (53%), respectively (<a reveal-id="dsz002-F2" data-open="dsz002-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). On the other hand, 131 of 4,256 <em>C</em>. <em>lentillifera</em> OGs were not found in the five other green algal OGs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>).</p>
<p>Further analysis of gene acquisitions and losses in <em>C</em>. <em>lentillifera</em> was performed, based on phylogenetic relationships mentioned in Section 3.2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). This analysis showed that OG losses have occurred &gt;76 times more frequently than OG acquisitions (<a reveal-id="dsz002-F2" data-open="dsz002-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). We annotated 2, 895 of putative OGs lost in <em>C</em>. <em>lentillifera</em>. Any green plant genes that were classified into 2,105 of the 2,895 OGs were not aligned on the <em>C</em>. <em>lentillifera</em> genome. These green plant genes did not show any similarity with the <em>C</em>. <em>lentillifera</em> transcriptome. Protein domains encoded in these putative lost genes in <em>C</em>. <em>lentillifera</em> were not uniformly conserved among other green plants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7a</a></span>). On the other hand, 17 of 1,223 <em>C</em>. <em>lentillifera</em> lost protein-domains were conserved among the twelve green plants. Functions of lost protein domains are likely associated with intracellular locality (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7b</a></span>).</p>
<p>The average number of genes per OG in <em>C</em>. <em>lentillifera</em> was 1.84, which is higher than in other green algae. This suggests more lineage-specific gene expansions in <em>C</em>. <em>lentillifera</em> (<a reveal-id="dsz002-F2" data-open="dsz002-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Heat map analyses of 14,946 OGs among 13 green plant genomes showed that the pattern of gene expansions in <em>C</em>. <em>lentillifera</em> did not overlap with those of other UTC algal genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span> and Dataset S1). This suggests that these gene expansions are specific to the <em>Caulerpa</em> lineage.</p>
<p>To further explore gene duplication in the <em>Caulerpa</em> lineage, we surveyed expanded OGs and performed GO analysis.<a reveal-id="dsz002-B63" data-open="dsz002-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> We compared <em>C</em>. <em>lentillifera</em> with the five aforementioned algal taxa. In both categories of biological process and molecular function, ubiquitination-related GOs (GO: 0016567, GO: 0004842) were expanded in <em>C</em>. <em>lentillifera</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>sets S2 and S3). <em>C</em>. <em>lentillifera</em> genes for peptidase (GO: 0008233) and peroxidase activity (GO: 0004601) were more abundant than those of other algae (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3). Analyses in cellular component categories also showed expansions of nuclear pore and COPI (coat protein complex I) vesicle coat (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S4). Interestingly, some nuclear pore-associated genes were expressed preferentially in fronds or stolons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). For example, the <em>GLE1-like</em> RNA-export mediator was expressed preferentially in fronds.</p>                    <h3 scrollto-destination="134009661" id="134009661" class="section-title">3.4. Evolution of <em>Caulerpa</em> genes for core regulatory networks</h3>
<p>One of the most prominent characteristics of <em>Caulerpa</em> is its differentiated physical structure as a single cell. However, molecular mechanisms involved in development of the plant architecture without multicellularity remain unknown.<a reveal-id="dsz002-B7" data-open="dsz002-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> To examine conservation and diversification of genes associated with core regulatory networks for morphology, we surveyed gene families for TF and TR. Comparisons with the plant database<a reveal-id="dsz002-B42" data-open="dsz002-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> showed that TF gene families were comparatively conserved between them (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S5). One of the expanded gene families was the C3H class, the expansion of which has also been reported in the land plant lineage.<a reveal-id="dsz002-B42" data-open="dsz002-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> In addition, homeobox genes of the HB-other class were abundant in the <em>C</em>. <em>lentillifera</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S5). Molecular phylogeny showed that BEL and KNOX subclasses, which form heterodimeric TF complexes through gamete fusion,<a reveal-id="dsz002-B64" data-open="dsz002-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> were expanded to eight genes in <em>C</em>. <em>lentillifera</em> (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Such gene duplications have not been reported in other chlorophyte genomes (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>), although the land plant lineage shows a greater number of duplications in this subclass.<a reveal-id="dsz002-B42" data-open="dsz002-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> DNA-recognition sites of some expanded BEL proteins deviated from conserved motifs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Unexpectedly, comparative analysis of eight loci paralogous to <em>BEL</em> and <em>KNOX</em> showed syntenic relationships with other genes within each of the BEL and KNOX families (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). The syntenic genes included 10 OGs based on conserved domains (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). In addition, syntenic property appeared between <em>BEL</em> and <em>KNOX</em> (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). Because BEL and KNOX families belong to the TALE homeobox class, this suggests that the ancestral TALE TF gene has been expanded by segmental duplication (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). </p>                        <a id="134009663" scrollto-destination="134009663"></a>
<div id="dsz002-F3" data-id="dsz002-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f3.png?Expires=1576485542&amp;Signature=znfwaGA9nB0tgaXhkgpRKPZ9qzE86nSGvcBNTtQcuqpd41ze7yk0JMK~tQjlInJ7j4BceCkFijgrwaNv4hZercO0MbUHfWTm-hP6KjQpHLG3b9jLEmJ-~uLzKepxij74r1JkQPZJupeg3bqMhmxCZ1Kns4wkxsfieoMqPqAn892aL~YpXCpNs-EdlHRLCCj74KTXTkuH-hR3QeYsG2jTK4ErWDXRhCORl4DwTLrStq38BsFF5M-dJhgQFtfmiO7SwlwyAVTabWXK9NtLVfFWa20Jl740GF8jlnvhpGcDPMdNBmP9SD~NypzB0BmSniG1joqCi~Fc8mHUE5HF6jBJmA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular phylogenetic analysis showing lineage-specific evolution of TALE class homeobox proteins in green plants and the ancient plant genomic structure. (a) The maximum-likelihood tree is constructed using multiple alignments of homeodomains. Green symbols show four members of the KNOX class and four of the BEL class in the Caulerpa genome. Nodes with more than 50 and 70% bootstrap support are marked with open and closed circles, respectively. Green, magenta and black lines correspond to chlorophyte, rhodophyte and Arabidopsis sequences, respectively. Ath, Arabidopsis thaliana. Cle, Caulerpa lentillifera. Cme, Cyanidioschyzon merolae. Cre, Chlamydomonas reinhardtii. Ota, Ostreococcus tauri. Vca, Volvox carteri. (b) Origin and evolution of the plant BEL/KNOX family and candidate genes for exploring the duplication mechanism of the ancestral gene. Simplified gene order in the Caulerpa genome is shown above. Scaffold numbers are indicated at the left side. Potential ancestral genomic structures are surrounded by dotted lines. Circles, rectangles and hexagonal boxes in diagrams of genomic sequences indicate protein domains contained in each locus. The synteny diagrams are not drawn to scale. (c) Expression levels of TALE homeobox genes. * indicates P-value &lt;0.05 and false discovery rate &lt;0.05. Two genes, g4802 of BEL, and g2165 of KNOX, show the difference of gene expression level between frond and stolon. Error bars and white circles show standard deviation and TMM-normalized CPM observed in each replicate, respectively." path-from-xml="dsz002f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009663/dsz002f3.tif" path-from-xml="dsz002f3.tif" target="_blank">Open in new tab</a><a section="134009663" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f3.png?Expires=1576485542&amp;Signature=CG5KO5aGzdNM23GuahZ1LV4atNXW-3de9AkdoJoj-tntt4TA-9sxe1IPC1~5sCp~~Rm2P6bdQwmxF6KbJfG-8xgdqVX23B6~Q-UGdY4AB3TW9TFlMWM00MIvM2TG0Z1K8BCVAm7TKSthLEqqGHxPz2MG7IOj-DdE4VdweELR~NrTeQezbDnTONrAVjKHQSOm8iN8JqQg53RJQKUvTg77mI12TdSo9YxnRbdaVM3sr9Wqpc~A9eqt06-7~chKh94GsDV8AWFRS6LOE~h3jEMs~Y~-ooEm3WsUW6K~oUTpN6Bp4gh9chUXnfB-WkzsTwcnh9DKNiMLnoEP32FHRPPu~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009663&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Molecular phylogenetic analysis showing lineage-specific evolution of TALE class homeobox proteins in green plants and the ancient plant genomic structure. (a) The maximum-likelihood tree is constructed using multiple alignments of homeodomains. Green symbols show four members of the KNOX class and four of the BEL class in the <em>Caulerpa</em> genome. Nodes with more than 50 and 70% bootstrap support are marked with open and closed circles, respectively. Green, magenta and black lines correspond to chlorophyte, rhodophyte and <em>Arabidopsis</em> sequences, respectively. Ath, <em>Arabidopsis thaliana</em>. Cle, <em>Caulerpa lentillifera</em>. Cme, <em>Cyanidioschyzon merolae</em>. Cre, <em>Chlamydomonas reinhardtii</em>. Ota, <em>Ostreococcus tauri</em>. Vca, <em>Volvox carteri</em>. (b) Origin and evolution of the plant BEL/KNOX family and candidate genes for exploring the duplication mechanism of the ancestral gene. Simplified gene order in the <em>Caulerpa</em> genome is shown above. Scaffold numbers are indicated at the left side. Potential ancestral genomic structures are surrounded by dotted lines. Circles, rectangles and hexagonal boxes in diagrams of genomic sequences indicate protein domains contained in each locus. The synteny diagrams are not drawn to scale. (c) Expression levels of TALE homeobox genes. * indicates <em>P</em>-value &lt;0.05 and false discovery rate &lt;0.05. Two genes, g4802 of <em>BEL</em>, and g2165 of <em>KNOX</em>, show the difference of gene expression level between frond and stolon. Error bars and white circles show standard deviation and TMM-normalized CPM observed in each replicate, respectively.</p></div>
</div>
<div content-id="dsz002-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f3.png?Expires=1576485542&amp;Signature=znfwaGA9nB0tgaXhkgpRKPZ9qzE86nSGvcBNTtQcuqpd41ze7yk0JMK~tQjlInJ7j4BceCkFijgrwaNv4hZercO0MbUHfWTm-hP6KjQpHLG3b9jLEmJ-~uLzKepxij74r1JkQPZJupeg3bqMhmxCZ1Kns4wkxsfieoMqPqAn892aL~YpXCpNs-EdlHRLCCj74KTXTkuH-hR3QeYsG2jTK4ErWDXRhCORl4DwTLrStq38BsFF5M-dJhgQFtfmiO7SwlwyAVTabWXK9NtLVfFWa20Jl740GF8jlnvhpGcDPMdNBmP9SD~NypzB0BmSniG1joqCi~Fc8mHUE5HF6jBJmA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Molecular phylogenetic analysis showing lineage-specific evolution of TALE class homeobox proteins in green plants and the ancient plant genomic structure. (a) The maximum-likelihood tree is constructed using multiple alignments of homeodomains. Green symbols show four members of the KNOX class and four of the BEL class in the Caulerpa genome. Nodes with more than 50 and 70% bootstrap support are marked with open and closed circles, respectively. Green, magenta and black lines correspond to chlorophyte, rhodophyte and Arabidopsis sequences, respectively. Ath, Arabidopsis thaliana. Cle, Caulerpa lentillifera. Cme, Cyanidioschyzon merolae. Cre, Chlamydomonas reinhardtii. Ota, Ostreococcus tauri. Vca, Volvox carteri. (b) Origin and evolution of the plant BEL/KNOX family and candidate genes for exploring the duplication mechanism of the ancestral gene. Simplified gene order in the Caulerpa genome is shown above. Scaffold numbers are indicated at the left side. Potential ancestral genomic structures are surrounded by dotted lines. Circles, rectangles and hexagonal boxes in diagrams of genomic sequences indicate protein domains contained in each locus. The synteny diagrams are not drawn to scale. (c) Expression levels of TALE homeobox genes. * indicates P-value &lt;0.05 and false discovery rate &lt;0.05. Two genes, g4802 of BEL, and g2165 of KNOX, show the difference of gene expression level between frond and stolon. Error bars and white circles show standard deviation and TMM-normalized CPM observed in each replicate, respectively." path-from-xml="dsz002f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009663/dsz002f3.tif" path-from-xml="dsz002f3.tif" target="_blank">Open in new tab</a><a section="134009663" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f3.png?Expires=1576485542&amp;Signature=CG5KO5aGzdNM23GuahZ1LV4atNXW-3de9AkdoJoj-tntt4TA-9sxe1IPC1~5sCp~~Rm2P6bdQwmxF6KbJfG-8xgdqVX23B6~Q-UGdY4AB3TW9TFlMWM00MIvM2TG0Z1K8BCVAm7TKSthLEqqGHxPz2MG7IOj-DdE4VdweELR~NrTeQezbDnTONrAVjKHQSOm8iN8JqQg53RJQKUvTg77mI12TdSo9YxnRbdaVM3sr9Wqpc~A9eqt06-7~chKh94GsDV8AWFRS6LOE~h3jEMs~Y~-ooEm3WsUW6K~oUTpN6Bp4gh9chUXnfB-WkzsTwcnh9DKNiMLnoEP32FHRPPu~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009663&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Molecular phylogenetic analysis showing lineage-specific evolution of TALE class homeobox proteins in green plants and the ancient plant genomic structure. (a) The maximum-likelihood tree is constructed using multiple alignments of homeodomains. Green symbols show four members of the KNOX class and four of the BEL class in the <em>Caulerpa</em> genome. Nodes with more than 50 and 70% bootstrap support are marked with open and closed circles, respectively. Green, magenta and black lines correspond to chlorophyte, rhodophyte and <em>Arabidopsis</em> sequences, respectively. Ath, <em>Arabidopsis thaliana</em>. Cle, <em>Caulerpa lentillifera</em>. Cme, <em>Cyanidioschyzon merolae</em>. Cre, <em>Chlamydomonas reinhardtii</em>. Ota, <em>Ostreococcus tauri</em>. Vca, <em>Volvox carteri</em>. (b) Origin and evolution of the plant BEL/KNOX family and candidate genes for exploring the duplication mechanism of the ancestral gene. Simplified gene order in the <em>Caulerpa</em> genome is shown above. Scaffold numbers are indicated at the left side. Potential ancestral genomic structures are surrounded by dotted lines. Circles, rectangles and hexagonal boxes in diagrams of genomic sequences indicate protein domains contained in each locus. The synteny diagrams are not drawn to scale. (c) Expression levels of TALE homeobox genes. * indicates <em>P</em>-value &lt;0.05 and false discovery rate &lt;0.05. Two genes, g4802 of <em>BEL</em>, and g2165 of <em>KNOX</em>, show the difference of gene expression level between frond and stolon. Error bars and white circles show standard deviation and TMM-normalized CPM observed in each replicate, respectively.</p></div>
</div>
<p>To examine expression differences among eight genes of the BEL/KNOX family, we performed RNA-seq analysis in fronds and stolons of a macroscopic cell. g4802 of <em>BEL</em> and g2165 of <em>KNOX</em> showed distinct and different expression levels between frond and stolon (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>). In addition, both had higher expression in stolons than the other six <em>BEL/KNOX</em> genes in the macroscopic stage (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>). Only <em>BEL</em> genes, abundantly expressed in the macroscopic stage, have diversified DNA-recognition sites (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). These results suggest that the possible BEL/KNOX heterodimers may be core regulators involved in morphological differentiation of a macroscopic cell.</p>                    <h3 scrollto-destination="134009665" id="134009665" class="section-title">3.5. Evolution of <em>Caulerpa</em> genes for plant hormone signalling</h3>
<p>In land plants, hormone signalling regulates growth, development and responses to environmental stress, although the evolutionary origins of the signalling machinery remain unknown.<a reveal-id="dsz002-B65" data-open="dsz002-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> It is tempting to ask whether homologues of <em>Caulerpa lentillifera</em> proteins are involved in formation and growth of multinucleated, giant cells, and/or adaptive responses. Although the majority of gene families for biosynthesis and hormone signalling in land plants<a reveal-id="dsz002-B65" data-open="dsz002-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> are not found in the <em>C</em>. <em>lentillifera</em> genome, most genes involved in abscisic acid (ABA) signalling in land plants were conserved (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>sets S6 and S7). In addition, some components of ABA signalling are expanded in <em>C</em>. <em>lentillifera</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S8). Phylogenetic analysis showed that genes for protein kinase have also been expanded in a lineage-specific manner, although the Chlorophyta has only one orthologue for <em>SnRK group 2 (SnRK2)</em>, an ABA signalling component (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12a</a></span>).</p>
<p>Interestingly, <em>Caulerpa</em>-specific genes similar to land plant <em>SnRK2</em> have been duplicated extensively, although the orthologues were likely lost in the land plant lineage. Additionally, 17 paralogs of <em>SnRKs</em> were tandemly clustered on scaffold 8 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12b</a></span>). Extensive analyses of protein kinases showed that, in addition to protein kinases (class: CAMK_OST1L) including SnRKs, only the tyrosine kinase family (class: TKL_CTR1-DRK-1) has likely been expanded in the <em>C</em>. <em>lentillifera</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S9). Another comprehensive survey of gene homologues for plant hormone biosynthesis clarified multiple tandem-duplicated genes of P450, which include a component of strigolactone (SL) biosynthesis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). It is likely that a pathway involved in SL biosynthesis is also diversified in the <em>Caulerpa</em> lineage.</p>                    <h2 scrollto-destination="134009668" id="134009668" class="section-title">4. Discussion</h2>
<p>We decoded the genome of a siphonous ulvophyte, <em>Caulerpa lentillifera</em>. Based on results of present and previous studies, we hypothesize the following cellular and molecular mechanisms involved in the structural differentiation in the single cell (<a reveal-id="dsz002-F4" data-open="dsz002-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). First, we found expansion of some OGs in the <em>C</em>. <em>lentillifera</em> genome that may be involved in the siphonous body plan. Differential expression patterns of nuclear pore-associated genes, such as a <em>GLE1-like</em> RNA-export mediator, suggest that RNA localization is regulated not only by transportation of RNA in the cytoplasm,<a reveal-id="dsz002-B66" data-open="dsz002-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> but also by nuclear position in the cell. The variations in expression profiles of transport proteins (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576485542&amp;Signature=C-4N~9TdVc26gsCYINUWX2daDjhRLS~sUe9U2-SNh7WrxkSaOFa4UxTdCcwaSjmXezd~CijBYokTVKVRw~flvn7gGjaiScAEdY~d~RE6b10GxnE-f4R7nLSZt-dJWzmqUTa2ehNp-NJ7pAOgNT1w0WdrvcUZFqO5WB56zg3DqevO~cbLM~NAffBdbJwWyq0pttkyhcnVBcdRLOFfnt-boY-xrSClUGDg7e6XqEEfr5pN5m-v5vYeJaWcGkjcimC2ww5XP4tJr8fltidRq0jv4e4ox97IgRWhVBZ~uGKnk16uzebpM0PjpXVbgvm1HO6vuYc~FYO7x5ZHn1mdCKsbKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>) imply that nuclei detect their own positions by unknown signals and express position-specific nuclear pore-associated genes (<a reveal-id="dsz002-F4" data-open="dsz002-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, left). Regulation of RNA transportation from nuclei by such genes function could play a role in different transcriptomic profiles in the structure, and structure-specific transcriptions may be regulated by nuclear import of TF using similar mechanisms (<a reveal-id="dsz002-F4" data-open="dsz002-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, left). It is likely that these phenomena enable structural patterning in siphonous algae without cell membrane boundaries, which are employed by multicellular organisms. </p>                        <a id="134009670" scrollto-destination="134009670"></a>
<div id="dsz002-F4" data-id="dsz002-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f4.png?Expires=1576485542&amp;Signature=h4jgAGwju2np~TL7udcr7ritfNu77qLinEduCeuTzpJpw8vwUHq9lOO89BcikTjhT5mleLhWCCKHfblEFdRkbvz8a453xhSo5pX30rf7g6RJDy88XcRkGIeqMbfg78nQL86I9c0l-78sKzlr-s179nQirQTZmX-a1L42HdgOjhqwWG5psI6ENSz8CGIPTkOHbiWTx3kO-RMzP-ZHKYdwDWko1le02oOfTEz3hzVJ5haqsfIT1bsq8DiAu6nOO4TQGi0lE4yk44xAfc5Jiln1u7QVYAJnnhKbnF5KucmAa1kmxrTX~gWGx74EAKtSJA6GE~G-6RHGAtAQkjvRSnDwuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hypothesized function of diversified gene families in siphonous macroalgae. In Caulerpa, divergent BEL and KNOX genes were formed by lineage-specific duplications, and they are differentially expressed in the macroscopic cell. By our prediction, KNOX/BEL genes of Caulerpa that have similar functions to those of Chlamydomonas are expressed specifically in gametes. In addition, nuclear pore-associated genes regulate RNA and/or protein transport in development of complex siphonous algae, as shown in balloons. Molecular localization is also controlled by vesicle transport proteins. These transporters generate molecular signatures in structure patterning without multicellularity. Expanded ABA signalling-associated genes are employed to adapt to environmental stress associated with a sessile lifestyle." path-from-xml="dsz002f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009670/dsz002f4.tif" path-from-xml="dsz002f4.tif" target="_blank">Open in new tab</a><a section="134009670" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f4.png?Expires=1576485542&amp;Signature=TgIRY-sIviGvAlgIeRAMZDMJe1mxf3ZGl1MwohtLnnpcBSgG13X0rAW7eJmJpiyv4yf44FTl3EowCmlSGiGdS64ChIrZHpsaa1j35j9kTrssACf8aHXz6euGMvc~XuHaSkF4v3j--zkOICXM24j6YVCkIEpEQ5lAYr4n7W6cwX5wtngQw61U5q1~2i5Qof-AGU-yIpDNIXgmXfnCg4iso3v-kUd9KBMciEiRI3dUO-azaYiFL1~eO-kJN410D1GqJPGmXfWNnlV7ku9cz8PbTUQ2EKBQgLrwr3ZR1b0pL9wLU56tHxGhsNfT7hQFMxrfu0v2Qvb6oUVVaR2RdOVQUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009670&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Hypothesized function of diversified gene families in siphonous macroalgae. In <em>Caulerpa</em>, divergent <em>BEL</em> and <em>KNOX</em> genes were formed by lineage-specific duplications, and they are differentially expressed in the macroscopic cell. By our prediction, <em>KNOX/BEL</em> genes of <em>Caulerpa</em> that have similar functions to those of <em>Chlamydomonas</em> are expressed specifically in gametes. In addition, nuclear pore-associated genes regulate RNA and/or protein transport in development of complex siphonous algae, as shown in balloons. Molecular localization is also controlled by vesicle transport proteins. These transporters generate molecular signatures in structure patterning without multicellularity. Expanded ABA signalling-associated genes are employed to adapt to environmental stress associated with a sessile lifestyle.</p></div>
</div>
<div content-id="dsz002-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/m_dsz002f4.png?Expires=1576485542&amp;Signature=h4jgAGwju2np~TL7udcr7ritfNu77qLinEduCeuTzpJpw8vwUHq9lOO89BcikTjhT5mleLhWCCKHfblEFdRkbvz8a453xhSo5pX30rf7g6RJDy88XcRkGIeqMbfg78nQL86I9c0l-78sKzlr-s179nQirQTZmX-a1L42HdgOjhqwWG5psI6ENSz8CGIPTkOHbiWTx3kO-RMzP-ZHKYdwDWko1le02oOfTEz3hzVJ5haqsfIT1bsq8DiAu6nOO4TQGi0lE4yk44xAfc5Jiln1u7QVYAJnnhKbnF5KucmAa1kmxrTX~gWGx74EAKtSJA6GE~G-6RHGAtAQkjvRSnDwuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hypothesized function of diversified gene families in siphonous macroalgae. In Caulerpa, divergent BEL and KNOX genes were formed by lineage-specific duplications, and they are differentially expressed in the macroscopic cell. By our prediction, KNOX/BEL genes of Caulerpa that have similar functions to those of Chlamydomonas are expressed specifically in gametes. In addition, nuclear pore-associated genes regulate RNA and/or protein transport in development of complex siphonous algae, as shown in balloons. Molecular localization is also controlled by vesicle transport proteins. These transporters generate molecular signatures in structure patterning without multicellularity. Expanded ABA signalling-associated genes are employed to adapt to environmental stress associated with a sessile lifestyle." path-from-xml="dsz002f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009670/dsz002f4.tif" path-from-xml="dsz002f4.tif" target="_blank">Open in new tab</a><a section="134009670" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002f4.png?Expires=1576485542&amp;Signature=TgIRY-sIviGvAlgIeRAMZDMJe1mxf3ZGl1MwohtLnnpcBSgG13X0rAW7eJmJpiyv4yf44FTl3EowCmlSGiGdS64ChIrZHpsaa1j35j9kTrssACf8aHXz6euGMvc~XuHaSkF4v3j--zkOICXM24j6YVCkIEpEQ5lAYr4n7W6cwX5wtngQw61U5q1~2i5Qof-AGU-yIpDNIXgmXfnCg4iso3v-kUd9KBMciEiRI3dUO-azaYiFL1~eO-kJN410D1GqJPGmXfWNnlV7ku9cz8PbTUQ2EKBQgLrwr3ZR1b0pL9wLU56tHxGhsNfT7hQFMxrfu0v2Qvb6oUVVaR2RdOVQUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009670&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz002f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Hypothesized function of diversified gene families in siphonous macroalgae. In <em>Caulerpa</em>, divergent <em>BEL</em> and <em>KNOX</em> genes were formed by lineage-specific duplications, and they are differentially expressed in the macroscopic cell. By our prediction, <em>KNOX/BEL</em> genes of <em>Caulerpa</em> that have similar functions to those of <em>Chlamydomonas</em> are expressed specifically in gametes. In addition, nuclear pore-associated genes regulate RNA and/or protein transport in development of complex siphonous algae, as shown in balloons. Molecular localization is also controlled by vesicle transport proteins. These transporters generate molecular signatures in structure patterning without multicellularity. Expanded ABA signalling-associated genes are employed to adapt to environmental stress associated with a sessile lifestyle.</p></div>
</div>
<p>Interestingly, expanded OGs in the <em>C</em>. <em>lentillifera</em> genome included the TALE class of homeobox proteins, including BEL and KNOX. These TALE homeobox genes are located at distant loci not only in <em>C</em>. <em>lentillifera</em> but also in <em>Chlamydomonas</em> and <em>Arabidopsis</em>. Homeobox genes in land plant genomes have been expanded by segmental duplications.<a reveal-id="dsz002-B67" data-open="dsz002-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Although the evolutionary mechanism of such homeobox genes has been extensively studied in land plants, the mechanism of the first duplication has not been proposed. Our comparative analysis demonstrated the ancient homeobox locus and adjacent gene sets evolved by segmental duplication (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). The <em>C</em>. <em>lentillifera</em> genome thus provides the first suggestion of how the ancestral TALE TF gene was duplicated during eukaryote evolution (<a reveal-id="dsz002-F3" data-open="dsz002-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). Ancestral BEL and KNOX may function in normal meiosis, mating and zygote development, as in <em>Chlamydomonas</em>.<a reveal-id="dsz002-B68" data-open="dsz002-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> We hypothesize that less expressed <em>BEL/KNOX</em> genes in <em>C</em>. <em>lentillifera</em> macroscopic cells preserve ancestral functions in gametes (<a reveal-id="dsz002-F4" data-open="dsz002-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). On the other hand, some of duplicated homeobox genes in <em>C</em>. <em>lentillifera</em> may have promoted neofunctionalization and those genes may not regulate alternation of life cycles but morphogenesis as in the land plant lineage<a reveal-id="dsz002-B64" data-open="dsz002-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsz002-B69" data-open="dsz002-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> (<a reveal-id="dsz002-F4" data-open="dsz002-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Differential expression of the homeobox genes in <em>C</em>. <em>lentillifera</em> can be interpreted as analogous to structural patterning through homeobox-protein antagonisms reported in land plants.<a reveal-id="dsz002-B69" data-open="dsz002-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a></p>
<p>In summary, our analyses illustrate the utility of the <em>C</em>. <em>lentillifera</em> genome for exploring possible evolutionary processes in green plants. Therefore, this siphonous algal genome will help to integrate knowledge from various model systems through comparisons with derived developmental traits and will accelerate understanding of apparent convergent evolution in plants.</p>                    <h2 scrollto-destination="134009673" id="134009673" class="section-title">Data availability</h2>
<p>All sequence data obtained in this study are accessible in the DDBJ/EMBL/NCBI database at the BioProject ID, PRJDB5734. A genome browser for assembled sequences, including annotation, is available at <a class="link link-uri" href="http://marinegenomics.oist.jp/gallery/" target="_blank">http://marinegenomics.oist.jp/gallery/</a>.</p>                    <h2 scrollto-destination="134009675" id="134009675" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>This study was funded by the Research and Development Program of Okinawa Institute of Science and Technology Graduate University to the Marine Genomics Unit. We thank Mr Munekazu Mekaru and Mr Makoto Toma (Onna Village Fisheries Cooperative in Okinawa) for providing samples and Dr Jun Inoue for useful comments about molecular phylogenetic analysis. DNA Sequencing Section and Scientific Computing &amp; Data Analysis Section at OIST were acknowledged. We also thank Dr Steven D. Aird for his help in preparation of the manuscript.</p>                    <h3 scrollto-destination="134009677" id="134009677" class="section-title">Accession number</h3>
<p>PRJDB5734</p>                    <h3 scrollto-destination="134009679" id="134009679" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="134009681" id="134009681" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz002-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Coneva</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Chitwood</div> <div class="given-names">D. H.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Plant architecture without multicellularity: quandaries over patterning and the soma-germline divide in siphonous algae</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">287</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Plant%20architecture%20without%20multicellularity%3A%20quandaries%20over%20patterning%20and%20the%20soma-germline%20divide%20in%20siphonous%20algae&amp;author=V.%20Coneva&amp;author=D.%20H.%20Chitwood&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=287" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2015.00287" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00287" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2015.00287"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25964794" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Plant%20architecture%20without%20multicellularity%3A%20quandaries%20over%20patterning%20and%20the%20soma-germline%20divide%20in%20siphonous%20algae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mary</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Mary</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Lorella</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Matias</div> <div class="given-names">J. R.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Rediscovery of naturally occurring seagrape <em>Caulerpa lentillifera</em> from the Gulf of Mannar and its mariculture</div>, <div class="source">Curr. Sci</div> ., <div class="volume">97</div>, <div class="fpage">1418</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Rediscovery%20of%20naturally%20occurring%20seagrape%20Caulerpa%20lentillifera%20from%20the%20Gulf%20of%20Mannar%20and%20its%20mariculture&amp;aulast=Mary&amp;date=2009&amp;spage=1418&amp;epage=20&amp;volume=97" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rediscovery%20of%20naturally%20occurring%20seagrape%20Caulerpa%20lentillifera%20from%20the%20Gulf%20of%20Mannar%20and%20its%20mariculture&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Galil</div> <div class="given-names">B. S.</div>
</div></div> <div class="year">2007</div>, 
<div class="article-title">Loss or gain? Invasive aliens and biodiversity in the Mediterranean Sea</div>, <div class="source">Mar. Pollut. Bull</div> ., <div class="volume">55</div>, <div class="fpage">314</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Loss%20or%20gain%3F%20Invasive%20aliens%20and%20biodiversity%20in%20the%20Mediterranean%20Sea&amp;author=B.%20S.%20Galil&amp;publication_year=2007&amp;journal=Mar.%20Pollut.%20Bull&amp;volume=55&amp;pages=314-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.marpolbul.2006.11.008" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.marpolbul.2006.11.008" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.marpolbul.2006.11.008"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17222869" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Loss%20or%20gain%3F%20Invasive%20aliens%20and%20biodiversity%20in%20the%20Mediterranean%20Sea&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gribben</div> <div class="given-names">P. E.</div>
</div>, <div class="name string-name">
<div class="surname">Nielsen</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Seymour</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">Bradley</div> <div class="given-names">D. J.</div>
</div>, <div class="name string-name">
<div class="surname">West</div> <div class="given-names">M. N.</div>
</div>, <div class="name string-name">
<div class="surname">Thomas</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Microbial communities in marine sediments modify success of an invasive macrophyte</div>, <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">9845</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Microbial%20communities%20in%20marine%20sediments%20modify%20success%20of%20an%20invasive%20macrophyte&amp;author=P.%20E.%20Gribben&amp;author=S.%20Nielsen&amp;author=J.%20R.%20Seymour&amp;author=D.%20J.%20Bradley&amp;author=M.%20N.%20West&amp;author=T.%20Thomas&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=9845" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41598-017-10231-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41598-017-10231-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41598-017-10231-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28852143" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Microbial%20communities%20in%20marine%20sediments%20modify%20success%20of%20an%20invasive%20macrophyte&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">de Gaillande</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Payri</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Remoissenet</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Zubia</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">
<em>Caulerpa</em> consumption, nutritional value and farming in the Indo-Pacific region</div>, <div class="source">J. Appl. Phycol</div> ., <div class="volume">29</div>, <div class="fpage">2249</div>–<div class="lpage">66</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Caulerpa%20consumption%2C%20nutritional%20value%20and%20farming%20in%20the%20Indo-Pacific%20region&amp;author=C.%20de%20Gaillande&amp;author=C.%20Payri&amp;author=G.%20Remoissenet&amp;author=M.%20Zubia&amp;publication_year=2017&amp;journal=J.%20Appl.%20Phycol&amp;volume=29&amp;pages=2249-66" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s10811-016-0912-6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10811-016-0912-6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs10811-016-0912-6"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Caulerpa%20consumption%2C%20nutritional%20value%20and%20farming%20in%20the%20Indo-Pacific%20region&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Niklas</div> <div class="given-names">K. J.</div>
</div>, <div class="name string-name">
<div class="surname">Kutschera</div> <div class="given-names">U.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">The evolution of the land plant life cycle</div>, <div class="source">New Phytol</div> ., <div class="volume">185</div>, <div class="fpage">27</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20the%20land%20plant%20life%20cycle&amp;author=K.%20J.%20Niklas&amp;author=U.%20Kutschera&amp;publication_year=2010&amp;journal=New%20Phytol&amp;volume=185&amp;pages=27-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1469-8137.2009.03054.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2009.03054.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1469-8137.2009.03054.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19863728" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20evolution%20of%20the%20land%20plant%20life%20cycle&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Niklas</div> <div class="given-names">K. J.</div>
</div>, <div class="name string-name">
<div class="surname">Cobb</div> <div class="given-names">E. D.</div>
</div>, <div class="name string-name">
<div class="surname">Crawford</div> <div class="given-names">D. R.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">The evo-devo of multinucleate cells, tissues, and organisms, and an alternative route to multicellularity</div>, <div class="source">Evol. Dev</div> ., <div class="volume">15</div>, <div class="fpage">466</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20evo-devo%20of%20multinucleate%20cells%2C%20tissues%2C%20and%20organisms%2C%20and%20an%20alternative%20route%20to%20multicellularity&amp;author=K.%20J.%20Niklas&amp;author=E.%20D.%20Cobb&amp;author=D.%20R.%20Crawford&amp;publication_year=2013&amp;journal=Evol.%20Dev&amp;volume=15&amp;pages=466-74" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/ede.v15.6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fede.v15.6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fede.v15.6"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24261447" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20evo-devo%20of%20multinucleate%20cells%2C%20tissues%2C%20and%20organisms%2C%20and%20an%20alternative%20route%20to%20multicellularity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pröschold</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Leliaert</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2007</div>. In: <div class="person-group">
<div class="name string-name">
<div class="surname">Brodie</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Lewis</div> <div class="given-names">J.</div>
</div>
</div> (eds) <div class="source">Unravelling the Algae: The Past, Present, and Future of Algal Systematics</div> , Chapter 7, pp. <div class="fpage">123</div>–<div class="lpage">153</div>. 
<div class="publisher-name">CRC Press</div>, 
<div class="publisher-loc">Boca Raton</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Unravelling%20the%20Algae%3A%20The%20Past%2C%20Present%2C%20and%20Future%20of%20Algal%20Systematics&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Unravelling%20the%20Algae%3A%20The%20Past%2C%20Present%2C%20and%20Future%20of%20Algal%20Systematics&amp;aulast=Pr%C3%B6schold&amp;date=2007&amp;spage=123&amp;epage=153" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Unravelling%20the%20Algae%3A%20The%20Past%2C%20Present%2C%20and%20Future%20of%20Algal%20Systematics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Unravelling%20the%20Algae%3A%20The%20Past%2C%20Present%2C%20and%20Future%20of%20Algal%20Systematics">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leliaert</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">D. R.</div>
</div>, <div class="name string-name">
<div class="surname">Moreau</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Phylogeny and molecular evolution of the green algae</div>, <div class="source">Crit. Rev. Plant Sci</div> ., <div class="volume">31</div>, <div class="fpage">1</div>–<div class="lpage">46</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20molecular%20evolution%20of%20the%20green%20algae&amp;author=F.%20Leliaert&amp;author=D.%20R.%20Smith&amp;author=H.%20Moreau&amp;publication_year=2012&amp;journal=Crit.%20Rev.%20Plant%20Sci&amp;volume=31&amp;pages=1-46" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/07352689.2011.615705" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07352689.2011.615705" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F07352689.2011.615705"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogeny%20and%20molecular%20evolution%20of%20the%20green%20algae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Guiry</div> <div class="given-names">M. D.</div>
</div>, <div class="name string-name">
<div class="surname">Guiry</div> <div class="given-names">G. M.</div>
</div>
</div> AlgaeBase. National University of Ireland, Galway. <a class="link link-uri" href="http://www.algaebase.org" target="_blank">http://www.algaebase.org</a> (26 April 2017, date last accessed).</div>
</div></div></div>
<div content-id="dsz002-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fučíková</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Leliaert</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Cooper</div> <div class="given-names">E. D.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">New phylogenetic hypotheses for the core Chlorophyta based on chloroplast sequence data</div>, <div class="source">Front. Ecol. Evol</div> ., <div class="volume">2</div>, <div class="fpage">63</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=New%20phylogenetic%20hypotheses%20for%20the%20core%20Chlorophyta%20based%20on%20chloroplast%20sequence%20data&amp;aulast=Fu%C4%8D%C3%ADkov%C3%A1&amp;date=2014&amp;spage=63&amp;volume=2" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20phylogenetic%20hypotheses%20for%20the%20core%20Chlorophyta%20based%20on%20chloroplast%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cocquyt</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Verbruggen</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Leliaert</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">De Clerck</div> <div class="given-names">O.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Evolution and cytological diversification of the green seaweeds (Ulvophyceae)</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">2052</div>–<div class="lpage">61</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20cytological%20diversification%20of%20the%20green%20seaweeds%20(Ulvophyceae)&amp;author=E.%20Cocquyt&amp;author=H.%20Verbruggen&amp;author=F.%20Leliaert&amp;author=O.%20De%20Clerck&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol&amp;volume=27&amp;pages=2052-61" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msq091" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq091" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsq091"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20368268" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20and%20cytological%20diversification%20of%20the%20green%20seaweeds%20(Ulvophyceae)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Coates</div> <div class="given-names">J. C.</div>
</div>, <div class="name string-name">
<div class="surname">Umm-E-Aiman Charrier</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Understanding “green” multicellularity: do seaweeds hold the key?,</div> <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">737</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Understanding%20%E2%80%9Cgreen%E2%80%9D%20multicellularity%3A%20do%20seaweeds%20hold%20the%20key%3F%2C&amp;author=J.%20C.%20Coates&amp;author=B.%20Umm-E-Aiman%20Charrier&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci&amp;volume=5&amp;pages=737" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25653653" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Understanding%20%E2%80%9Cgreen%E2%80%9D%20multicellularity%3A%20do%20seaweeds%20hold%20the%20key%3F%2C&amp;aulast=Coates&amp;date=2014&amp;spage=737&amp;volume=5" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Understanding%20%E2%80%9Cgreen%E2%80%9D%20multicellularity%3A%20do%20seaweeds%20hold%20the%20key%3F%2C&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">De Clerck</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Kao</div> <div class="given-names">S. M.</div>
</div>, <div class="name string-name">
<div class="surname">Bogaert</div> <div class="given-names">K. A.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Insights into the evolution of multicellularity from the sea lettuce genome</div>, <div class="source">Curr. Biol</div> ., <div class="volume">28</div>, <div class="fpage">2921</div>–<div class="lpage">33</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20evolution%20of%20multicellularity%20from%20the%20sea%20lettuce%20genome&amp;author=O.%20De%20Clerck&amp;author=S.%20M.%20Kao&amp;author=K.%20A.%20Bogaert&amp;publication_year=2018&amp;journal=Curr.%20Biol&amp;volume=28&amp;pages=2921-33" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.cub.2018.08.015" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2018.08.015" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.cub.2018.08.015"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30220504" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insights%20into%20the%20evolution%20of%20multicellularity%20from%20the%20sea%20lettuce%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Mandoli</div> <div class="given-names">D. F.</div>
</div></div> <div class="year">1998</div>, 
<div class="article-title">What ever happened to <em>Acetabularia</em>? Bringing a once-classic model system into the age of molecular genetics</div>, <div class="source">Int. Rev. Cytol</div> ., <div class="volume">182</div>, <div class="fpage">1</div>–<div class="lpage">67</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=What%20ever%20happened%20to%20Acetabularia%3F%20Bringing%20a%20once-classic%20model%20system%20into%20the%20age%20of%20molecular%20genetics&amp;author=D.%20F.%20Mandoli&amp;publication_year=1998&amp;journal=Int.%20Rev.%20Cytol&amp;volume=182&amp;pages=1-67" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0074-7696(08)62167-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0074-7696(08)62167-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0074-7696(08)62167-1"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:What%20ever%20happened%20to%20Acetabularia%3F%20Bringing%20a%20once-classic%20model%20system%20into%20the%20age%20of%20molecular%20genetics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Scheuring</div> <div class="given-names">C. F.</div>
</div>, <div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">C. C.</div>
</div>, <div class="name string-name">
<div class="surname">Dong</div> <div class="given-names">J. J.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">H. B.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Preparation of megabase-sized DNA from a variety of organisms using the nuclei method for advanced genomics research</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">7</div>, <div class="fpage">467</div>–<div class="lpage">78</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Preparation%20of%20megabase-sized%20DNA%20from%20a%20variety%20of%20organisms%20using%20the%20nuclei%20method%20for%20advanced%20genomics%20research&amp;author=M.%20Zhang&amp;author=Y.%20Zhang&amp;author=C.%20F.%20Scheuring&amp;author=C.%20C.%20Wu&amp;author=J.%20J.%20Dong&amp;author=H.%20B.%20Zhang&amp;publication_year=2012&amp;journal=Nat.%20Protoc&amp;volume=7&amp;pages=467-78" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nprot.2011.455" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2011.455" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnprot.2011.455"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22343429" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Preparation%20of%20megabase-sized%20DNA%20from%20a%20variety%20of%20organisms%20using%20the%20nuclei%20method%20for%20advanced%20genomics%20research&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bolger</div> <div class="given-names">A. M.</div>
</div>, <div class="name string-name">
<div class="surname">Lohse</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Usadel</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu170"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leggett</div> <div class="given-names">R. M.</div>
</div>, <div class="name string-name">
<div class="surname">Clavijo</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Clissold</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Clark</div> <div class="given-names">M. D.</div>
</div>, <div class="name string-name">
<div class="surname">Caccamo</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">NextClip: an analysis and read preparation tool for Nextera Long Mate Pair libraries</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">566</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=NextClip%3A%20an%20analysis%20and%20read%20preparation%20tool%20for%20Nextera%20Long%20Mate%20Pair%20libraries&amp;author=R.%20M.%20Leggett&amp;author=B.%20J.%20Clavijo&amp;author=L.%20Clissold&amp;author=M.%20D.%20Clark&amp;author=M.%20Caccamo&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=566-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt702" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt702" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt702"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297520" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:NextClip%3A%20an%20analysis%20and%20read%20preparation%20tool%20for%20Nextera%20Long%20Mate%20Pair%20libraries&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chaisson</div> <div class="given-names">M. J.</div>
</div>, <div class="name string-name">
<div class="surname">Tesler</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">13</div>, <div class="fpage">238</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mapping%20single%20molecule%20sequencing%20reads%20using%20basic%20local%20alignment%20with%20successive%20refinement%20(BLASR)%3A%20application%20and%20theory&amp;author=M.%20J.%20Chaisson&amp;author=G.%20Tesler&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=238" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-13-238" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-238" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-13-238"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22988817" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mapping%20single%20molecule%20sequencing%20reads%20using%20basic%20local%20alignment%20with%20successive%20refinement%20(BLASR)%3A%20application%20and%20theory&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div></div> <div class="year">2013</div>, Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM, arXiv preprint arXiv: 1303.3997v2 [q-bio.GN].</div>
</div></div></div>
<div content-id="dsz002-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kajitani</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Toshimoto</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Noguchi</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.170720.113"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pryszcz</div> <div class="given-names">L. P.</div>
</div>, <div class="name string-name">
<div class="surname">Gabaldón</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Redundans: an assembly pipeline for highly heterozygous genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">e113</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Redundans%3A%20an%20assembly%20pipeline%20for%20highly%20heterozygous%20genomes&amp;author=L.%20P.%20Pryszcz&amp;author=T.%20Gabald%C3%B3n&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=e113" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkw294" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw294" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkw294"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27131372" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Redundans%3A%20an%20assembly%20pipeline%20for%20highly%20heterozygous%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lam</div> <div class="given-names">K. K.</div>
</div>, <div class="name string-name">
<div class="surname">Hall</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Clum</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rao</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">BIGMAC: breaking inaccurate genomes and merging assembled contigs for long read metagenomic assembly</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">17</div>, <div class="fpage">435</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BIGMAC%3A%20breaking%20inaccurate%20genomes%20and%20merging%20assembled%20contigs%20for%20long%20read%20metagenomic%20assembly&amp;author=K.%20K.%20Lam&amp;author=R.%20Hall&amp;author=A.%20Clum&amp;author=S.%20Rao&amp;publication_year=2016&amp;journal=BMC%20Bioinformatics&amp;volume=17&amp;pages=435" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12859-016-1288-y" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12859-016-1288-y" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12859-016-1288-y"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27793084" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BIGMAC%3A%20breaking%20inaccurate%20genomes%20and%20merging%20assembled%20contigs%20for%20long%20read%20metagenomic%20assembly&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Graham</div> <div class="given-names">E. D.</div>
</div>, <div class="name string-name">
<div class="surname">Heidelberg</div> <div class="given-names">J. F.</div>
</div>, <div class="name string-name">
<div class="surname">Tully</div> <div class="given-names">B. J.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">BinSanity: unsupervised clustering of environmental microbial assemblies using coverage and affinity propagation</div>, <div class="source">PeerJ</div> ., <div class="volume">5</div>, <div class="fpage">e3035</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BinSanity%3A%20unsupervised%20clustering%20of%20environmental%20microbial%20assemblies%20using%20coverage%20and%20affinity%20propagation&amp;author=E.%20D.%20Graham&amp;author=J.%20F.%20Heidelberg&amp;author=B.%20J.%20Tully&amp;publication_year=2017&amp;journal=PeerJ&amp;volume=5&amp;pages=e3035" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.7717/peerj.3035" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7717%2Fpeerj.3035" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.7717%2Fpeerj.3035"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28289564" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BinSanity%3A%20unsupervised%20clustering%20of%20environmental%20microbial%20assemblies%20using%20coverage%20and%20affinity%20propagation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">English</div> <div class="given-names">A. C.</div>
</div>, <div class="name string-name">
<div class="surname">Richards</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Han</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Mind the gap: upgrading genomes with Pacific Biosciences RS long-read sequencing technology</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e47768</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mind%20the%20gap%3A%20upgrading%20genomes%20with%20Pacific%20Biosciences%20RS%20long-read%20sequencing%20technology&amp;author=A.%20C.%20English&amp;author=S.%20Richards&amp;author=Y.%20Han&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e47768" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0047768" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0047768" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0047768"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23185243" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mind%20the%20gap%3A%20upgrading%20genomes%20with%20Pacific%20Biosciences%20RS%20long-read%20sequencing%20technology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sahlin</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Chikhi</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Arvestad</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Assembly scaffolding with PE-contaminated mate-pair libraries</div>, <div class="source">Bioinformatics</div> , <div class="volume">32</div>, <div class="fpage">1925</div>–<div class="lpage">32</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assembly%20scaffolding%20with%20PE-contaminated%20mate-pair%20libraries&amp;author=K.%20Sahlin&amp;author=R.%20Chikhi&amp;author=L.%20Arvestad&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=1925-32" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btw064" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtw064" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtw064"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27153683" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assembly%20scaffolding%20with%20PE-contaminated%20mate-pair%20libraries&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Warren</div> <div class="given-names">R. L.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Vandervalk</div> <div class="given-names">B. P.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">LINKS: scalable, alignment-free scaffolding of draft genomes with long reads</div>, <div class="source">Gigascience</div> , <div class="volume">4</div>, <div class="fpage">35</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=LINKS%3A%20scalable%2C%20alignment-free%20scaffolding%20of%20draft%20genomes%20with%20long%20reads&amp;author=R.%20L.%20Warren&amp;author=C.%20Yang&amp;author=B.%20P.%20Vandervalk&amp;publication_year=2015&amp;journal=Gigascience&amp;volume=4&amp;pages=35" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13742-015-0076-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13742-015-0076-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13742-015-0076-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26244089" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:LINKS%3A%20scalable%2C%20alignment-free%20scaffolding%20of%20draft%20genomes%20with%20long%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Walker</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Abeel</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Shea</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e112963</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;author=B.%20J.%20Walker&amp;author=T.%20Abeel&amp;author=T.%20Shea&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e112963" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0112963" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112963" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0112963"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Waterhouse</div> <div class="given-names">R. M.</div>
</div>, <div class="name string-name">
<div class="surname">Seppey</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Simão</div> <div class="given-names">F. A.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">BUSCO applications from quality assessments to gene prediction and phylogenomics</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">35</div>, <div class="fpage">543</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BUSCO%20applications%20from%20quality%20assessments%20to%20gene%20prediction%20and%20phylogenomics&amp;author=R.%20M.%20Waterhouse&amp;author=M.%20Seppey&amp;author=F.%20A.%20Sim%C3%A3o&amp;publication_year=2018&amp;journal=Mol.%20Biol.%20Evol&amp;volume=35&amp;pages=543-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msx319" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsx319" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsx319"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BUSCO%20applications%20from%20quality%20assessments%20to%20gene%20prediction%20and%20phylogenomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Parra</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Bradnam</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Korf</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtm071"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Marçais</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Kingsford</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr011"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Vurture</div> <div class="given-names">G. W.</div>
</div>, <div class="name string-name">
<div class="surname">Sedlazeck</div> <div class="given-names">F. J.</div>
</div>, <div class="name string-name">
<div class="surname">Nattestad</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">GenomeScope: fast reference-free genome profiling from short reads</div>, <div class="source">Bioinformatics</div> , <div class="volume">33</div>, <div class="fpage">2202</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GenomeScope%3A%20fast%20reference-free%20genome%20profiling%20from%20short%20reads&amp;author=G.%20W.%20Vurture&amp;author=F.%20J.%20Sedlazeck&amp;author=M.%20Nattestad&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=2202-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btx153" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtx153" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtx153"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28369201" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GenomeScope%3A%20fast%20reference-free%20genome%20profiling%20from%20short%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Danecek</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Auton</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Abecasis</div> <div class="given-names">G.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">The variant call format and VCFtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2156</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20variant%20call%20format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr330"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20variant%20call%20format%20and%20VCFtools&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grabherr</div> <div class="given-names">M. G.</div>
</div>, <div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Yassour</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.%20G.%20Grabherr&amp;author=B.%20J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.1883"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dobin</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Davis</div> <div class="given-names">C. A.</div>
</div>, <div class="name string-name">
<div class="surname">Schlesinger</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">STAR: ultrafast universal RNA-seq aligner</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">15</div>–<div class="lpage">21</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;author=A.%20Dobin&amp;author=C.%20A.%20Davis&amp;author=F.%20Schlesinger&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bts635" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts635" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts635"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Delcher</div> <div class="given-names">A. L.</div>
</div>, <div class="name string-name">
<div class="surname">Mount</div> <div class="given-names">S. M.</div>
</div>
</div>, et al.  <div class="year">2003</div>, 
<div class="article-title">Improving the <em>Arabidopsis</em> genome annotation using maximal transcript alignment assemblies</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">5654</div>–<div class="lpage">66</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;author=B.%20J.%20Haas&amp;author=A.%20L.%20Delcher&amp;author=S.%20M.%20Mount&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=5654-66" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkg770" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg770" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkg770"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hoff</div> <div class="given-names">K. J.</div>
</div>, <div class="name string-name">
<div class="surname">Lange</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Lomsadze</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Borodovsky</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Stanke</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">BRAKER1: unsupervised RNA-Seq-based genome annotation with GeneMark-ET and AUGUSTUS</div>, <div class="source">Bioinformatics</div> , <div class="volume">32</div>, <div class="fpage">767</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BRAKER1%3A%20unsupervised%20RNA-Seq-based%20genome%20annotation%20with%20GeneMark-ET%20and%20AUGUSTUS&amp;author=K.%20J.%20Hoff&amp;author=S.%20Lange&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=767-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btv661" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv661" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtv661"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26559507" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BRAKER1%3A%20unsupervised%20RNA-Seq-based%20genome%20annotation%20with%20GeneMark-ET%20and%20AUGUSTUS&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Slater</div> <div class="given-names">G. S.</div>
</div>, <div class="name string-name">
<div class="surname">Birney</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Automated generation of heuristics for biological sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">6</div>, <div class="fpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.%20S.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-6-31"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">W.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</div>, <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2008-9-1-r7"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Langmead</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> ., <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1923"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Stanke</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Diekhans</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Baertsch</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Haussler</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtn013"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zheng</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Jiao</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">iTAK: a program for genome-wide prediction and classification of plant transcription factors, transcriptional regulators, and protein kinases</div>, <div class="source">Mol. Plant</div> , <div class="volume">9</div>, <div class="fpage">1667</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=iTAK%3A%20a%20program%20for%20genome-wide%20prediction%20and%20classification%20of%20plant%20transcription%20factors%2C%20transcriptional%20regulators%2C%20and%20protein%20kinases&amp;author=Y.%20Zheng&amp;author=C.%20Jiao&amp;author=H.%20Sun&amp;publication_year=2016&amp;journal=Mol.%20Plant&amp;volume=9&amp;pages=1667-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.molp.2016.09.014" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2016.09.014" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.molp.2016.09.014"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27717919" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:iTAK%3A%20a%20program%20for%20genome-wide%20prediction%20and%20classification%20of%20plant%20transcription%20factors%2C%20transcriptional%20regulators%2C%20and%20protein%20kinases&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Strigolactone signaling in <em>Arabidopsis</em> regulates shoot development by targeting D53-Like SMXL repressor proteins for ubiquitination and degradation</div>, <div class="source">Plant Cell</div> , <div class="volume">27</div>, <div class="fpage">3128</div>–<div class="lpage">42</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Strigolactone%20signaling%20in%20Arabidopsis%20regulates%20shoot%20development%20by%20targeting%20D53-Like%20SMXL%20repressor%20proteins%20for%20ubiquitination%20and%20degradation&amp;author=L.%20Wang&amp;author=B.%20Wang&amp;author=L.%20Jiang&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=3128-42" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.15.00605" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.15.00605" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.15.00605"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26546446" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Strigolactone%20signaling%20in%20Arabidopsis%20regulates%20shoot%20development%20by%20targeting%20D53-Like%20SMXL%20repressor%20proteins%20for%20ubiquitination%20and%20degradation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">McCarthy</div> <div class="given-names">D. J.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Smyth</div> <div class="given-names">G. K.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">4288</div>–<div class="lpage">97</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20of%20multifactor%20RNA-Seq%20experiments%20with%20respect%20to%20biological%20variation&amp;author=D.%20J.%20McCarthy&amp;author=Y.%20Chen&amp;author=G.%20K.%20Smyth&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=4288-97" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks042" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks042" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks042"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22287627" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Differential%20expression%20analysis%20of%20multifactor%20RNA-Seq%20experiments%20with%20respect%20to%20biological%20variation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Emms</div> <div class="given-names">D. M.</div>
</div>, <div class="name string-name">
<div class="surname">Kelly</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">157</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;author=D.%20M.%20Emms&amp;author=S.%20Kelly&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=157" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-015-0721-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0721-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0721-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26243257" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Katoh</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Misawa</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Kuma</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Miyata</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">3059</div>–<div class="lpage">66</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkf436" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkf436"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Capella-Gutiérrez</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Silla-Martíınez</div> <div class="given-names">J. M.</div>
</div>, <div class="name string-name">
<div class="surname">Gabaldón</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1972</div>–<div class="lpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=trimAl%3A%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large-scale%20phylogenetic%20analyses&amp;author=S.%20Capella-Guti%C3%A9rrez&amp;author=J.%20M.%20Silla-Mart%C3%AD%C4%B1nez&amp;author=T.%20Gabald%C3%B3n&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1972-3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp348" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp348" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp348"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505945" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:trimAl%3A%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large-scale%20phylogenetic%20analyses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nguyen</div> <div class="given-names">L. T.</div>
</div>, <div class="name string-name">
<div class="surname">Schmidt</div> <div class="given-names">H. A.</div>
</div>, <div class="name string-name">
<div class="surname">von Haeseler</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Minh</div> <div class="given-names">B. Q.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">268</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;author=L.%20T.%20Nguyen&amp;author=H.%20A.%20Schmidt&amp;author=A.%20von%20Haeseler&amp;author=B.%20Q.%20Minh&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=268-74" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msu300" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu300" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsu300"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371430" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ronquist</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Huelsenbeck</div> <div class="given-names">J. P.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">1572</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MrBayes%203%3A%20Bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;author=F.%20Ronquist&amp;author=J.%20P.%20Huelsenbeck&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1572-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg180" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtg180"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MrBayes%203%3A%20Bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Rabiee</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Sayyari</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Mirarab</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees</div>, <div class="source">BMC Bioinformatics</div> . <div class="volume">19</div>, <div class="fpage">153</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=BMC%20Bioinformatics&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=BMC%20Bioinformatics&amp;aulast=Zhang&amp;date=2018&amp;spage=153&amp;volume=19" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BMC%20Bioinformatics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=BMC%20Bioinformatics">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Larget</div> <div class="given-names">B. R.</div>
</div>, <div class="name string-name">
<div class="surname">Kotha</div> <div class="given-names">S. K.</div>
</div>, <div class="name string-name">
<div class="surname">Dewey</div> <div class="given-names">C. N.</div>
</div>, <div class="name string-name">
<div class="surname">Ané</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">BUCKy: gene tree/species tree reconciliation with Bayesian concordance analysis</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">2910</div>–<div class="lpage">1</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BUCKy%3A%20gene%20tree%2Fspecies%20tree%20reconciliation%20with%20Bayesian%20concordance%20analysis&amp;author=B.%20R.%20Larget&amp;author=S.%20K.%20Kotha&amp;author=C.%20N.%20Dewey&amp;author=C.%20An%C3%A9&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2910-1" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq539" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq539" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq539"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20861028" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BUCKy%3A%20gene%20tree%2Fspecies%20tree%20reconciliation%20with%20Bayesian%20concordance%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nishitsuji</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Arimoto</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Iwai</div> <div class="given-names">K.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">A draft genome of the brown alga, <em>Cladosiphon okamuranus</em>, S-strain: a platform for future studies of ‘mozuku’ biology</div>, <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">561</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20draft%20genome%20of%20the%20brown%20alga%2C%20Cladosiphon%20okamuranus%2C%20S-strain%3A%20a%20platform%20for%20future%20studies%20of%20%E2%80%98mozuku%E2%80%99%20biology&amp;author=K.%20Nishitsuji&amp;author=A.%20Arimoto&amp;author=K.%20Iwai&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=561-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsw039" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw039" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsw039"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27501718" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20draft%20genome%20of%20the%20brown%20alga%2C%20Cladosiphon%20okamuranus%2C%20S-strain%3A%20a%20platform%20for%20future%20studies%20of%20%E2%80%98mozuku%E2%80%99%20biology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Conway</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">Lex</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Gehlenborg</div> <div class="given-names">N.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">UpSetR: an R package for the visualization of intersecting sets and their properties</div>, <div class="source">Bioinformatics</div> , <div class="volume">33</div>, <div class="fpage">2938</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=UpSetR%3A%20an%20R%20package%20for%20the%20visualization%20of%20intersecting%20sets%20and%20their%20properties&amp;author=J.%20R.%20Conway&amp;author=A.%20Lex&amp;author=N.%20Gehlenborg&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=2938-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btx364" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtx364" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtx364"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28645171" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:UpSetR%3A%20an%20R%20package%20for%20the%20visualization%20of%20intersecting%20sets%20and%20their%20properties&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Felsenstein</div> <div class="given-names">J.</div>
</div></div> <div class="year">2005</div>, PHYLIP (Phylogeny Inference Package) version 3.6, University of Washington.</div>
</div></div></div>
<div content-id="dsz002-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Melton</div> <div class="given-names">J. T. I. I. I.</div>
</div>, <div class="name string-name">
<div class="surname">Leliaert</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Tronholm</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Lopez-Bautista</div> <div class="given-names">J. M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">The complete chloroplast and mitochondrial genomes of the green macroalga <em>Ulva</em> sp. UNA00071828 (Ulvophyceae, Chlorophyta)</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0121020</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20complete%20chloroplast%20and%20mitochondrial%20genomes%20of%20the%20green%20macroalga%20Ulva%20sp.%20UNA00071828%20(Ulvophyceae%2C%20Chlorophyta)&amp;author=J.%20T.%20I.%20I.%20I.%20Melton&amp;author=F.%20Leliaert&amp;author=A.%20Tronholm&amp;author=J.%20M.%20Lopez-Bautista&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0121020" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0121020" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0121020" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0121020"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25849557" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20complete%20chloroplast%20and%20mitochondrial%20genomes%20of%20the%20green%20macroalga%20Ulva%20sp.%20UNA00071828%20(Ulvophyceae%2C%20Chlorophyta)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B56" class="ref-content ">
<div class="label">56</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">L. Fang, L</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Chang</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Penny</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Zhong</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Chloroplast phylogenomic inference of green algae relationships</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">20528</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Chloroplast%20phylogenomic%20inference%20of%20green%20algae%20relationships&amp;author=S.%20L.%20Fang%2C%20L&amp;author=Z.%20Zhang&amp;author=X.%20Chang&amp;author=D.%20Penny&amp;author=B.%20Zhong&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=20528" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep20528" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep20528" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep20528"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26846729" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Chloroplast%20phylogenomic%20inference%20of%20green%20algae%20relationships&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B57" class="ref-content ">
<div class="label">57</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">D. R.</div>
</div>, <div class="name string-name">
<div class="surname">Burki</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Yamada</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">The GC-rich mitochondrial and plastid genomes of the green alga <em>Coccomyxa</em> give insight into the evolution of organelle DNA nucleotide landscape</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e23624</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20GC-rich%20mitochondrial%20and%20plastid%20genomes%20of%20the%20green%20alga%20Coccomyxa%20give%20insight%20into%20the%20evolution%20of%20organelle%20DNA%20nucleotide%20landscape&amp;author=D.%20R.%20Smith&amp;author=F.%20Burki&amp;author=T.%20Yamada&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e23624" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0023624" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0023624" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0023624"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21887287" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20GC-rich%20mitochondrial%20and%20plastid%20genomes%20of%20the%20green%20alga%20Coccomyxa%20give%20insight%20into%20the%20evolution%20of%20organelle%20DNA%20nucleotide%20landscape&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B58" class="ref-content ">
<div class="label">58</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Merchant</div> <div class="given-names">S. S.</div>
</div>, <div class="name string-name">
<div class="surname">Prochnik</div> <div class="given-names">S. E.</div>
</div>, <div class="name string-name">
<div class="surname">Vallon</div> <div class="given-names">O.</div>
</div>
</div>, et al.  <div class="year">2007</div>, 
<div class="article-title">The <em>Chlamydomonas</em> genome reveals the evolution of key animal and plant functions</div>, <div class="source">Science</div> , <div class="volume">318</div>, <div class="fpage">245</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Chlamydomonas%20genome%20reveals%20the%20evolution%20of%20key%20animal%20and%20plant%20functions&amp;author=S.%20S.%20Merchant&amp;author=S.%20E.%20Prochnik&amp;author=O.%20Vallon&amp;publication_year=2007&amp;journal=Science&amp;volume=318&amp;pages=245-50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1143609" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1143609" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1143609"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932292" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Chlamydomonas%20genome%20reveals%20the%20evolution%20of%20key%20animal%20and%20plant%20functions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B59" class="ref-content ">
<div class="label">59</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Prochnik</div> <div class="given-names">S. E.</div>
</div>, <div class="name string-name">
<div class="surname">Umen</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Nedelcu</div> <div class="given-names">A. M.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Genomic analysis of organismal complexity in the multicellular green alga <em>Volvox carteri</em>
</div>, <div class="source">Science</div> , <div class="volume">329</div>, <div class="fpage">223</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20organismal%20complexity%20in%20the%20multicellular%20green%20alga%20Volvox%20carteri&amp;author=S.%20E.%20Prochnik&amp;author=J.%20Umen&amp;author=A.%20M.%20Nedelcu&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=223-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1188800" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1188800" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1188800"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20616280" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomic%20analysis%20of%20organismal%20complexity%20in%20the%20multicellular%20green%20alga%20Volvox%20carteri&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B60" class="ref-content ">
<div class="label">60</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Blanc</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Duncan</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Agarkova</div> <div class="given-names">I.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">The <em>Chlorella variabilis</em> NC64A genome reveals adaptation to photosymbiosis, coevolution with viruses, and cryptic sex</div>, <div class="source">Plant Cell</div> , <div class="volume">22</div>, <div class="fpage">2943</div>–<div class="lpage">55</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Chlorella%20variabilis%20NC64A%20genome%20reveals%20adaptation%20to%20photosymbiosis%2C%20coevolution%20with%20viruses%2C%20and%20cryptic%20sex&amp;author=G.%20Blanc&amp;author=G.%20Duncan&amp;author=I.%20Agarkova&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=2943-55" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.110.076406" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.076406" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.110.076406"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20852019" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Chlorella%20variabilis%20NC64A%20genome%20reveals%20adaptation%20to%20photosymbiosis%2C%20coevolution%20with%20viruses%2C%20and%20cryptic%20sex&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B61" class="ref-content ">
<div class="label">61</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Blanc</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Agarkova</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Grimwood</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">The genome of the polar eukaryotic microalga <em>Coccomyxa subellipsoidea</em> reveals traits of cold adaptation</div>, <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">R39</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20polar%20eukaryotic%20microalga%20Coccomyxa%20subellipsoidea%20reveals%20traits%20of%20cold%20adaptation&amp;author=G.%20Blanc&amp;author=I.%20Agarkova&amp;author=J.%20Grimwood&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=R39" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2012-13-5-r39" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-5-r39" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2012-13-5-r39"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22630137" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genome%20of%20the%20polar%20eukaryotic%20microalga%20Coccomyxa%20subellipsoidea%20reveals%20traits%20of%20cold%20adaptation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B62" class="ref-content ">
<div class="label">62</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Palenik</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Grimwood</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Aerts</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2007</div>, 
<div class="article-title">The tiny eukaryote <em>Ostreococcus</em> provides genomic insights into the paradox of plankton speciation</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">104</div>, <div class="fpage">7705</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20tiny%20eukaryote%20Ostreococcus%20provides%20genomic%20insights%20into%20the%20paradox%20of%20plankton%20speciation&amp;author=B.%20Palenik&amp;author=J.%20Grimwood&amp;author=A.%20Aerts&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=104&amp;pages=7705-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0611046104" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611046104" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0611046104"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17460045" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20tiny%20eukaryote%20Ostreococcus%20provides%20genomic%20insights%20into%20the%20paradox%20of%20plankton%20speciation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B63" class="ref-content ">
<div class="label">63</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ashburner</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Ball</div> <div class="given-names">C. A.</div>
</div>, <div class="name string-name">
<div class="surname">Blake</div> <div class="given-names">J. A.</div>
</div>
</div>, et al.  <div class="year">2000</div>, 
<div class="article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium</div>, <div class="source">Nat. Genet</div> ., <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&amp;author=M.%20Ashburner&amp;author=C.%20A.%20Ball&amp;author=J.%20A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F75556"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B64" class="ref-content ">
<div class="label">64</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bowman</div> <div class="given-names">J. L.</div>
</div>, <div class="name string-name">
<div class="surname">Sakakibara</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Furumizu</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Dierschke</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Evolution in the cycles of life</div>, <div class="source">Annu. Rev. Genet</div> ., <div class="volume">50</div>, <div class="fpage">133</div>–<div class="lpage">54</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolution%20in%20the%20cycles%20of%20life&amp;author=J.%20L.%20Bowman&amp;author=K.%20Sakakibara&amp;author=C.%20Furumizu&amp;author=T.%20Dierschke&amp;publication_year=2016&amp;journal=Annu.%20Rev.%20Genet&amp;volume=50&amp;pages=133-54" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev-genet-120215-035227" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-120215-035227" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev-genet-120215-035227"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27617970" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20in%20the%20cycles%20of%20life&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B65" class="ref-content ">
<div class="label">65</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">S. S.</div>
</div>, <div class="name string-name">
<div class="surname">Han</div> <div class="given-names">G. Z.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Insights into the origin and evolution of the plant hormone signaling machinery</div>, <div class="source">Plant Physiol</div> ., <div class="volume">167</div>, <div class="fpage">872</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20origin%20and%20evolution%20of%20the%20plant%20hormone%20signaling%20machinery&amp;author=C.%20Wang&amp;author=Y.%20Liu&amp;author=S.%20S.%20Li&amp;author=G.%20Z.%20Han&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=167&amp;pages=872-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.114.247403" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.247403" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.114.247403"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25560880" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insights%20into%20the%20origin%20and%20evolution%20of%20the%20plant%20hormone%20signaling%20machinery&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B66" class="ref-content ">
<div class="label">66</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ranjan</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Townsley</div> <div class="given-names">B. T.</div>
</div>, <div class="name string-name">
<div class="surname">Ichihashi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Sinha</div> <div class="given-names">N. R.</div>
</div>, <div class="name string-name">
<div class="surname">Chitwood</div> <div class="given-names">D. H.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">An intracellular transcriptomic atlas of the giant coenocyte <em>Caulerpa taxifolia</em>
</div>, <div class="source">PLoS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1004900</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20intracellular%20transcriptomic%20atlas%20of%20the%20giant%20coenocyte%20Caulerpa%20taxifolia&amp;author=A.%20Ranjan&amp;author=B.%20T.%20Townsley&amp;author=Y.%20Ichihashi&amp;author=N.%20R.%20Sinha&amp;author=D.%20H.%20Chitwood&amp;publication_year=2015&amp;journal=PLoS%20Genet&amp;volume=11&amp;pages=e1004900" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1004900" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004900" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1004900"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25569326" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20intracellular%20transcriptomic%20atlas%20of%20the%20giant%20coenocyte%20Caulerpa%20taxifolia&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B67" class="ref-content ">
<div class="label">67</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bhattacharjee</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Ghangal</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Garg</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Jain</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Genome-wide analysis of homeobox gene family in legumes: identification, gene duplication and expression profiling</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0119198</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20homeobox%20gene%20family%20in%20legumes%3A%20identification%2C%20gene%20duplication%20and%20expression%20profiling&amp;author=A.%20Bhattacharjee&amp;author=R.%20Ghangal&amp;author=R.%20Garg&amp;author=M.%20Jain&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0119198" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0119198" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0119198" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0119198"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25745864" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20analysis%20of%20homeobox%20gene%20family%20in%20legumes%3A%20identification%2C%20gene%20duplication%20and%20expression%20profiling&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B68" class="ref-content ">
<div class="label">68</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">J. H.</div>
</div>, <div class="name string-name">
<div class="surname">Lin</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Joo</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Goodenough</div> <div class="given-names">U.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Early sexual origins of homeoprotein heterodimerization and evolution of the plant KNOX/BELL family</div>, <div class="source">Cell</div> , <div class="volume">133</div>, <div class="fpage">829</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Early%20sexual%20origins%20of%20homeoprotein%20heterodimerization%20and%20evolution%20of%20the%20plant%20KNOX%2FBELL%20family&amp;author=J.%20H.%20Lee&amp;author=H.%20Lin&amp;author=S.%20Joo&amp;author=U.%20Goodenough&amp;publication_year=2008&amp;journal=Cell&amp;volume=133&amp;pages=829-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.cell.2008.04.028" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.04.028" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.cell.2008.04.028"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18510927" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Early%20sexual%20origins%20of%20homeoprotein%20heterodimerization%20and%20evolution%20of%20the%20plant%20KNOX%2FBELL%20family&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz002-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz002-B69" class="ref-content ">
<div class="label">69</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Furumizu</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Alvarez</div> <div class="given-names">J. P.</div>
</div>, <div class="name string-name">
<div class="surname">Sakakibara</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Bowman</div> <div class="given-names">J. L.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Antagonistic roles for KNOX1 and KNOX2 genes in patterning the land plant body plan following an ancient gene duplication</div>, <div class="source">PLoS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1004980</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Antagonistic%20roles%20for%20KNOX1%20and%20KNOX2%20genes%20in%20patterning%20the%20land%20plant%20body%20plan%20following%20an%20ancient%20gene%20duplication&amp;author=C.%20Furumizu&amp;author=J.%20P.%20Alvarez&amp;author=K.%20Sakakibara&amp;author=J.%20L.%20Bowman&amp;publication_year=2015&amp;journal=PLoS%20Genet&amp;volume=11&amp;pages=e1004980" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1004980" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004980" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1004980"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25671434" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Antagonistic%20roles%20for%20KNOX1%20and%20KNOX2%20genes%20in%20patterning%20the%20land%20plant%20body%20plan%20following%20an%20ancient%20gene%20duplication&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-20126980" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Tabata%20Satoshi%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                            </span>
                        <br>
</div>
            </div>
        </div>
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5419551&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsz002/1/dsz002_supp.zip?Expires=1576701194&amp;Signature=kzjTKHn8NnJ8t7GDrgFFi2B63RSXHFAN-9a7sQT2m3dVp854xKRRgYtVklc-vUepGyWSvyggT8edPbLe~i0UoBwg8aE9deKUJyWYbSWfY-fuE5KGj2wUZpEauAiUPD5Sl2oP2fCKFT6dLwOs1rJUzK3oeaolVb~HZ~wvKxy2kgUnDX7ocYhcNRxSWFWG4Y3aRmePCvS~WIKTM-BcFYGNM7IF5Io4tOp42GaqIOSZ-mpxkwyJ9DFNdS~4cY3YB6S0Ry-NNHCTZag~vP6mjeRZynP1X7W0wuQYJbQm7RJxW6t~f8F5khEAspPnaTgVqORpbZKf9TayX6xslCWnlc0XAQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">8,100</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz002"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5419551"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493006800007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f2%2f183%2f5419551">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493006800007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F2%2F183%2F5419551" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5419551">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="siphonous macroalgal genome suggests convergent functions of homeobox genes in algae and land plants | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz002");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f183%2f5419551">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
