<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/4/313/5512299","@type":"ScholarlyArticle","name":"Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay","datePublished":"2019-06-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2019-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz011","keywords":["mate-pair sequencing","controlled polymerization","adapter-ligation","integrated platform"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/4/313/5512299","author":[{"name":"Dong, Zirui","affiliation":"Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China  Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China","@type":"Person"},{"name":"Zhao, Xia","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Li, Qiaoling","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Yang, Zhenjun","affiliation":"Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China  Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China","@type":"Person"},{"name":"Xi, Yang","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Alexeev, Andrei","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Shen, Hanjie","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Wang, Ou","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Ruan, Jie","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Ren, Han","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Wei, Hanmin","affiliation":"MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Qi, Xiaojuan","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Li, Jiguang","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Zhu, Xiaofan","affiliation":"Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China  Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China","@type":"Person"},{"name":"Zhang, Yanyan","affiliation":"MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Dai, Peng","affiliation":"Genetics and Prenatal Diagnosis Center, the First Affiliated Hospital of Zhengzhou University, Zhengzhou, China","@type":"Person"},{"name":"Kong, Xiangdong","affiliation":"Genetics and Prenatal Diagnosis Center, the First Affiliated Hospital of Zhengzhou University, Zhengzhou, China","@type":"Person"},{"name":"Kirkconnell, Killeen","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Alferov, Oleg","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Giles, Shane","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Yamtich, Jennifer","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Kermani, Bahram G","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Dong, Chao","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Liu, Pengjuan","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Mi, Zilan","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China","@type":"Person"},{"name":"Zhang, Wenwei","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  Guangdong High-Throughput Sequencing Research Center, Shenzhen, China","@type":"Person"},{"name":"Xu, Xun","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  James D. Watson Institute of Genome Sciences, Hangzhou, China","@type":"Person","sameAs":"http://orcid.org/0000-0002-5338-5173"},{"name":"Drmanac, Radoje","affiliation":"BGI-Shenzhen, Shenzhen, China  China National GeneBank, BGI-Shenzhen, Shenzhen, China  MGI, BGI-Shenzhen, Shenzhen, China  Complete Genomics Inc., San Jose, CA, USA","@type":"Person"},{"name":"Choy, Kwong Wai","affiliation":"Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China  Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China  The Chinese University of Hong Kong—Baylor College of Medicine Joint Center for Medical Genetics, Hong Kong, China","@type":"Person"},{"name":"Jiang, Yuan","affiliation":"Complete Genomics Inc., San Jose, CA, USA","@type":"Person","sameAs":"http://orcid.org/0000-0001-5170-2959"}],"description":"Abstract.  The diversity of disease presentations warrants one single assay for detection and delineation of various genomic disorders. Herein, we describe a ge","pageStart":"313","pageEnd":"325","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636238033&Signature=uaSYExkcYxo5nQlgOMk8fFv9J0e~iuPO4IYZzOieWJtndWh5qmRbiT5NbhDFTnrh6wzkhFG~Ne6oDI6t~46~KW9gL1F7uG78wFeJ~wFMAe9l0HfEa6t6Pkk5zZFrJv~nm0PJIP0b1IIU6zIuetTfE3Dve~a7nEQMmYX~OGiQFoRB2x7uYVdmcPbkWghRTIEq5lJZ-8KNfwpaHy11F3CkeyMhV8puW1zBqSrHYXpfc7t1R7aXYkDA6Iohrl1D8qu3-Lx4LXF1UfkQFjZQd5dQxTMCSlQRduL6SWMR3odyI6eRZSh5-eqhu8grgWAbiuBlviwfWJf2UqQ17ymRK1-gSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636238033&Signature=uaSYExkcYxo5nQlgOMk8fFv9J0e~iuPO4IYZzOieWJtndWh5qmRbiT5NbhDFTnrh6wzkhFG~Ne6oDI6t~46~KW9gL1F7uG78wFeJ~wFMAe9l0HfEa6t6Pkk5zZFrJv~nm0PJIP0b1IIU6zIuetTfE3Dve~a7nEQMmYX~OGiQFoRB2x7uYVdmcPbkWghRTIEq5lJZ-8KNfwpaHy11F3CkeyMhV8puW1zBqSrHYXpfc7t1R7aXYkDA6Iohrl1D8qu3-Lx4LXF1UfkQFjZQd5dQxTMCSlQRduL6SWMR3odyI6eRZSh5-eqhu8grgWAbiuBlviwfWJf2UqQ17ymRK1-gSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay">
<meta property="og:description" content="Abstract.  The diversity of disease presentations warrants one single assay for detection and delineation of various genomic disorders. Herein, we describe a ge">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/4/313/5512299">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636238033&amp;Signature=uaSYExkcYxo5nQlgOMk8fFv9J0e~iuPO4IYZzOieWJtndWh5qmRbiT5NbhDFTnrh6wzkhFG~Ne6oDI6t~46~KW9gL1F7uG78wFeJ~wFMAe9l0HfEa6t6Pkk5zZFrJv~nm0PJIP0b1IIU6zIuetTfE3Dve~a7nEQMmYX~OGiQFoRB2x7uYVdmcPbkWghRTIEq5lJZ-8KNfwpaHy11F3CkeyMhV8puW1zBqSrHYXpfc7t1R7aXYkDA6Iohrl1D8qu3-Lx4LXF1UfkQFjZQd5dQxTMCSlQRduL6SWMR3odyI6eRZSh5-eqhu8grgWAbiuBlviwfWJf2UqQ17ymRK1-gSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-06-07">
<meta name="citation_author" content="Dong, Zirui">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China">
<meta name="citation_author_institution" content="Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China">
<meta name="citation_author" content="Zhao, Xia">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Li, Qiaoling">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Yang, Zhenjun">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China">
<meta name="citation_author_institution" content="Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China">
<meta name="citation_author" content="Xi, Yang">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Alexeev, Andrei">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Shen, Hanjie">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Wang, Ou">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Ruan, Jie">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Ren, Han">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Wei, Hanmin">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Qi, Xiaojuan">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Li, Jiguang">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Zhu, Xiaofan">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China">
<meta name="citation_author_institution" content="Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China">
<meta name="citation_author" content="Zhang, Yanyan">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Dai, Peng">
<meta name="citation_author_institution" content="Genetics and Prenatal Diagnosis Center, the First Affiliated Hospital of Zhengzhou University, Zhengzhou, China">
<meta name="citation_author" content="Kong, Xiangdong">
<meta name="citation_author_institution" content="Genetics and Prenatal Diagnosis Center, the First Affiliated Hospital of Zhengzhou University, Zhengzhou, China">
<meta name="citation_author" content="Kirkconnell, Killeen">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Alferov, Oleg">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Giles, Shane">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Yamtich, Jennifer">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Kermani, Bahram G">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Dong, Chao">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Liu, Pengjuan">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Mi, Zilan">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author" content="Zhang, Wenwei">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="Guangdong High-Throughput Sequencing Research Center, Shenzhen, China">
<meta name="citation_author" content="Xu, Xun">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Hangzhou, China">
<meta name="citation_author" content="Drmanac, Radoje">
<meta name="citation_author_institution" content="BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="China National GeneBank, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="MGI, BGI-Shenzhen, Shenzhen, China">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_author" content="Choy, Kwong Wai">
<meta name="citation_author_institution" content="Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China">
<meta name="citation_author_institution" content="Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China">
<meta name="citation_author_institution" content="The Chinese University of Hong Kong—Baylor College of Medicine Joint Center for Medical Genetics, Hong Kong, China">
<meta name="citation_author" content="Jiang, Yuan">
<meta name="citation_author_institution" content="Complete Genomics Inc., San Jose, CA, USA">
<meta name="citation_title" content="Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay">
<meta name="citation_firstpage" content="313">
<meta name="citation_lastpage" content="325">
<meta name="citation_doi" content="10.1093/dnares/dsz011">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="4">
<meta name="citation_publication_date" content="2019/08/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=An integrated map of structural variation in 2,504 human genomes;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=526;  citation_pages=75-81; ">
<meta name="citation_reference" content="citation_title=Mechanisms underlying structural variant formation in genomic disorders;  citation_journal_title=Nat. Rev. Genet;  citation_year=2016;  citation_volume=17;  citation_pages=224-38; ">
<meta name="citation_reference" content="citation_title=TBX6 null variants and a common hypomorphic allele in congenital scoliosis;  citation_journal_title=N. Engl. J. Med;  citation_year=2015;  citation_volume=372;  citation_pages=341-50; ">
<meta name="citation_reference" content="citation_title=Identification of submicroscopic chromosomal aberrations in fetuses with increased nuchal translucency and apparently normal karyotype;  citation_journal_title=Ultrasound Obstet. Gynecol;  citation_year=2011;  citation_volume=38;  citation_pages=314-9; ">
<meta name="citation_reference" content="citation_title=Identification of a PTEN mutation with reduced protein stability, phosphatase activity, and nuclear localization in Hong Kong patients with autistic features, neurodevelopmental delays, and macrocephaly;  citation_journal_title=Autism Res;  citation_year=2018;  citation_volume=11;  citation_pages=1098-109; ">
<meta name="citation_reference" content="citation_title=The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies;  citation_journal_title=Nat. Genet;  citation_year=2017;  citation_volume=49;  citation_pages=36-45; ">
<meta name="citation_reference" content="citation_title=Discovering and understanding oncogenic gene fusions through data intensive computational approaches;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=4487-503; ">
<meta name="citation_reference" content="citation_title=Rare variants in the genetic background modulate cognitive and developmental phenotypes in individuals carrying disease-associated variants;  citation_journal_title=Genet. Med;  citation_year=201; ">
<meta name="citation_reference" content="citation_title=Increasing evidence of combinatory variant effects calls for revised classification of low-penetrance alleles;  citation_journal_title=Genet. Med;  citation_year=2018; ">
<meta name="citation_reference" content="citation_title=A survey of undetected, clinically relevant chromosome abnormalities when replacing postnatal karyotyping by whole genome sequencing;  citation_journal_title=Eur. J. Med. Genet;  citation_year=2018; ">
<meta name="citation_reference" content="citation_title=Improving mammalian genome scaffolding using large insert mate-pair next-generation sequencing;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=257">
<meta name="citation_reference" content="citation_title=Mapping translocation breakpoints by next-generation sequencing;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=1143-9; ">
<meta name="citation_reference" content="citation_title=Identification of balanced chromosomal rearrangements previously unknown among participants in the 1000 Genomes Project: implications for interpretation of structural variation in genomes and the future of clinical cytogenetics;  citation_journal_title=Genet. Med;  citation_year=2018;  citation_volume=20;  citation_pages=697-707; ">
<meta name="citation_reference" content="citation_title=Coming of age: ten years of next-generation sequencing technologies;  citation_journal_title=Nat. Rev. Genet;  citation_year=2016;  citation_volume=17;  citation_pages=333-51; ">
<meta name="citation_reference" content="citation_title=Next-generation sequencing strategies enable routine detection of balanced chromosome rearrangements for clinical diagnostics and genetic research;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2011;  citation_volume=88;  citation_pages=469-81; ">
<meta name="citation_reference" content="citation_title=A robust approach for blind detection of balanced chromosomal rearrangements with whole-genome low-coverage sequencing;  citation_journal_title=Hum. Mutat;  citation_year=2014;  citation_volume=35;  citation_pages=625-36; ">
<meta name="citation_reference" content="citation_title=NextClip: an analysis and read preparation tool for Nextera long mate pair libraries;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=566-8; ">
<meta name="citation_reference" content="citation_title=Meraculous: de novo genome assembly with short paired-end reads;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e23501">
<meta name="citation_reference" content="citation_title=Design of large-insert jumping libraries for structural variant detection using Illumina sequencing;  citation_journal_title=Curr. Protoc. Hum. Genet;  citation_year=2014;  citation_volume=80;  citation_pages=7.22.1-9; ">
<meta name="citation_reference" content="citation_title=A method to simultaneously construct up to 12 differently sized Illumina Nextera long mate pair libraries with reduced DNA input, time, and cost;  citation_journal_title=Biotechniques;  citation_year=2015;  citation_volume=59;  citation_pages=42-5; ">
<meta name="citation_reference" content="citation_title=Nanopore sequencing and assembly of a human genome with ultra-long reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2018;  citation_volume=36;  citation_pages=338-45; ">
<meta name="citation_reference" content="citation_title=Detection and visualization of complex structural variants from long reads;  citation_journal_title=BMC Bioinformatics;  citation_year=2018;  citation_volume=19;  citation_pages=508">
<meta name="citation_reference" content="citation_title=3' Branch ligation: a novel method to ligate non-complementary DNA to recessed or internal 3'OH ends in DNA or RNA;  citation_journal_title=DNA Res;  citation_year=201; ">
<meta name="citation_reference" content="citation_title=Accurate whole-genome sequencing and haplotyping from 10 to 20 human cells;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=487;  citation_pages=190-5; ">
<meta name="citation_reference" content="citation_title=Extensive sequencing of seven human genomes to characterize benchmark reference materials;  citation_journal_title=Sci. Data;  citation_year=2016;  citation_volume=3;  citation_pages=160025">
<meta name="citation_reference" content="citation_title=Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays;  citation_journal_title=Science;  citation_year=2010;  citation_volume=327;  citation_pages=78-81; ">
<meta name="citation_reference" content="citation_title=Computational techniques for human genome resequencing using mated gapped reads;  citation_journal_title=J. Comput. Biol;  citation_year=2012;  citation_volume=19;  citation_pages=279-92; ">
<meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; ">
<meta name="citation_reference" content="citation_title=Low-pass whole-genome sequencing in clinical cytogenetics: a validated approach;  citation_journal_title=Genet. Med;  citation_year=2016;  citation_volume=18;  citation_pages=940-8; ">
<meta name="citation_reference" content="citation_title=De novo assembly of a haplotype-resolved human genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=617-22; ">
<meta name="citation_reference" content="citation_title=3DIV: a 3D-genome interaction viewer and database;  citation_journal_title=Nucleic Acids Res;  citation_year=2018;  citation_volume=46;  citation_pages=D52-7; ">
<meta name="citation_reference" content="citation_title=Balanced chromosomal rearrangement detection by low-pass whole-genome sequencing;  citation_journal_title=Curr. Protoc. Hum. Genet;  citation_year=2018;  citation_volume=96;  citation_pages=8.18.1-16; ">
<meta name="citation_reference" content="citation_title=Accurate whole human genome sequencing using reversible terminator chemistry;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=53-9; ">
<meta name="citation_reference" content="citation_title=An integrated semiconductor device enabling non-optical genome sequencing;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=348-52; ">
<meta name="citation_reference" content="citation_title=cPAS-based sequencing on the BGISEQ-500 to explore small non-coding RNAs;  citation_journal_title=Clin. Epigenetics;  citation_year=2016;  citation_volume=8;  citation_pages=123">
<meta name="citation_reference" content="citation_title=A high-resolution, nucleosome position map of C. elegans reveals a lack of universal sequence-dictated positioning;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=1051-63; ">
<meta name="citation_reference" content="citation_title=Heterozygous mutation of eEF1A1b resulted in spermatogenesis arrest and infertility in male tilapia, Oreochromis niloticus;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=43733">
<meta name="citation_reference" content="citation_title=The accessible chromatin landscape of the human genome;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=489;  citation_pages=75-82; ">
<meta name="citation_reference" content="citation_title=Clinical diagnosis by whole-genome sequencing of a prenatal sample;  citation_journal_title=N. Engl. J. Med;  citation_year=2012;  citation_volume=367;  citation_pages=2226-32; ">
<meta name="citation_reference" content="citation_title=Whole-genome sequencing of cytogenetically balanced chromosome translocations identifies potentially pathological gene disruptions and highlights the importance of microhomology in the mechanism of formation;  citation_journal_title=Hum. Mutat;  citation_year=2017;  citation_volume=38;  citation_pages=180-92; ">
<meta name="citation_reference" content="citation_title=Evaluation of a transposase protocol for rapid generation of shotgun high-throughput sequencing libraries from nanogram quantities of DNA;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2011;  citation_volume=77;  citation_pages=8071-9; ">
<meta name="citation_reference" content="citation_title=Mechanisms for complex chromosomal insertions;  citation_journal_title=PLoS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1006446">
<meta name="citation_reference" content="citation_title=Gain-of-function mutations in ZIC1 are associated with coronal craniosynostosis and learning disability;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2015;  citation_volume=97;  citation_pages=378-88; ">
<meta name="citation_reference" content="citation_title=Tn5 transposase and tagmentation procedures for massively scaled sequencing projects;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=2033-40; ">
<meta name="citation_reference" content="citation_title=Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e24">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/4/313/29197165/dsz011.pdf">
<meta name="description" content="Abstract.  The diversity of disease presentations warrants one single assay for detection and delineation of various genomic disorders. Herein, we describe a ge">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/4/313/5512299">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/4/313/5512299">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/4/313/5512299");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2779951,2796125&authnips=128.235.85.32&doi=10.1093/dnares/dsz011';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f313%2f5512299" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f313%2f5512299" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=116309"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1576706134&amp;Signature=PFIg3fot6bFw-FQPKYxqRYBKXaOYgSIWb93v1sTw~t1~~Qe-SNA5tQREHynYjn6UTpIXJ8BWDWtt6q4glHXdNXgwAx4UndukH2SlAECgx9-AGE1pFf20ghDK9itv31PR5pKgiF2nqkdaB8LaUzVpsXNi~gePWM6Pmf-iriM6nI3N4U4hxILgT2eXO0F4FLCA1rxIF8n~VMtkPU-5Xn9CNq5g7XeCd29pbMjQPWhYIh-JFvUOVEzv-eMCpT8HVGmHOgxyymj-vPf8~TaVfJwSTegehZX8rc8Ye~Ucicpco1Uh-krkf3L1SqJmx5maqQmqqERlMVZKqCqHOHLH-BY8RA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/4">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="140465032">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465032">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465034">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465034">1. Background</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465039">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465039">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465064">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465064">3. Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465085">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465085">4. Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465094">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465094">Availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465096">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465096">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="140465102">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465102">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/4/301/5512305">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/4/327/5525352">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zirui Dong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China</div>
<div class="aff">Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zirui+Dong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dong%20Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Dong%20Zirui%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Zirui Dong</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xia Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xia+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao%20X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhao%20Xia%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Xia Zhao</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qiaoling Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qiaoling+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li%20Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Li%20Qiaoling%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Qiaoling Li</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhenjun Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China</div>
<div class="aff">Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhenjun+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang%20Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yang%20Zhenjun%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Zhenjun Yang</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yang Xi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yang+Xi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xi%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Xi%20Yang%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Yang Xi</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrei Alexeev</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrei+Alexeev">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Alexeev%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Alexeev%20Andrei%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Andrei Alexeev</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hanjie Shen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hanjie+Shen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shen%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shen%20Hanjie%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Hanjie Shen</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ou Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ou+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang%20O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wang%20Ou%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Ou Wang</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jie Ruan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jie+Ruan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ruan%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ruan%20Jie%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Jie Ruan</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Han Ren</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Han+Ren">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ren%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ren%20Han%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Han Ren</a><span class="delimiter-hidden">, </span>
                                </span>
                                    <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                        <span>...</span> Show more
                                    </a>
                                    <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hanmin Wei</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hanmin+Wei">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wei%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wei%20Hanmin%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Hanmin Wei</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaojuan Qi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaojuan+Qi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qi%20X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Qi%20Xiaojuan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Xiaojuan Qi</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiguang Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiguang+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Li%20Jiguang%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Jiguang Li</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaofan Zhu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China</div>
<div class="aff">Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaofan+Zhu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhu%20X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhu%20Xiaofan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Xiaofan Zhu</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanyan Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanyan+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20Yanyan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Yanyan Zhang</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Peng Dai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Genetics and Prenatal Diagnosis Center, the First Affiliated Hospital of Zhengzhou University, Zhengzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Peng+Dai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dai%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Dai%20Peng%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Peng Dai</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiangdong Kong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Genetics and Prenatal Diagnosis Center, the First Affiliated Hospital of Zhengzhou University, Zhengzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiangdong+Kong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kong%20X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kong%20Xiangdong%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Xiangdong Kong</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Killeen Kirkconnell</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Killeen+Kirkconnell">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kirkconnell%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kirkconnell%20Killeen%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Killeen Kirkconnell</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Oleg Alferov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Oleg+Alferov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Alferov%20O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Alferov%20Oleg%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Oleg Alferov</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shane Giles</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shane+Giles">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Giles%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Giles%20Shane%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Shane Giles</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jennifer Yamtich</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jennifer+Yamtich">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamtich%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yamtich%20Jennifer%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Jennifer Yamtich</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bahram G Kermani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bahram+G+Kermani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kermani%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kermani%20Bahram%20G%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Bahram G Kermani</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chao Dong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chao+Dong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dong%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Dong%20Chao%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Chao Dong</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pengjuan Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pengjuan+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Liu%20Pengjuan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Pengjuan Liu</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zilan Mi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zilan+Mi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mi%20Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mi%20Zilan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Zilan Mi</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wenwei Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">Guangdong High-Throughput Sequencing Research Center, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wenwei+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20Wenwei%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Wenwei Zhang</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xun Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">James D. Watson Institute of Genome Sciences, Hangzhou, China</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0002-5338-5173">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0002-5338-5173
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xun+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu%20X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Xu%20Xun%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Xun Xu</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Radoje Drmanac</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">China National GeneBank, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">MGI, BGI-Shenzhen, Shenzhen, China</div>
<div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 4086482560 3079. Fax. +1 4086482549. Email: <a href="mailto:yjiang@completegenomics.com" target="_blank">yjiang@completegenomics.com</a> (Y.J.); Tel. +852 35053099. Fax. +852 26360008. Email: <a href="mailto:richardchoy@cuhk.edu.hk" target="_blank">richardchoy@cuhk.edu.hk</a> (K.W.C.); Tel. +1 4088389539. Fax. +1 4086482549. Email: <a href="mailto:rade@completegenomics.com" target="_blank">rade@completegenomics.com</a> (R.D.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Radoje+Drmanac">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Drmanac%20R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Drmanac%20Radoje%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Radoje Drmanac</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kwong Wai Choy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Obstetrics and Gynaecology, The Chinese University of Hong Kong, Hong Kong, China</div>
<div class="aff">Shenzhen Research Institute, The Chinese University of Hong Kong, Shenzhen, China</div>
<div class="aff">The Chinese University of Hong Kong—Baylor College of Medicine Joint Center for Medical Genetics, Hong Kong, China</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 4086482560 3079. Fax. +1 4086482549. Email: <a href="mailto:yjiang@completegenomics.com" target="_blank">yjiang@completegenomics.com</a> (Y.J.); Tel. +852 35053099. Fax. +852 26360008. Email: <a href="mailto:richardchoy@cuhk.edu.hk" target="_blank">richardchoy@cuhk.edu.hk</a> (K.W.C.); Tel. +1 4088389539. Fax. +1 4086482549. Email: <a href="mailto:rade@completegenomics.com" target="_blank">rade@completegenomics.com</a> (R.D.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kwong+Wai+Choy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choy%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Choy%20Kwong%20Wai%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Kwong Wai Choy</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuan Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Complete Genomics Inc., San Jose, CA, USA</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 4086482560 3079. Fax. +1 4086482549. Email: <a href="mailto:yjiang@completegenomics.com" target="_blank">yjiang@completegenomics.com</a> (Y.J.); Tel. +852 35053099. Fax. +852 26360008. Email: <a href="mailto:richardchoy@cuhk.edu.hk" target="_blank">richardchoy@cuhk.edu.hk</a> (K.W.C.); Tel. +1 4088389539. Fax. +1 4086482549. Email: <a href="mailto:rade@completegenomics.com" target="_blank">rade@completegenomics.com</a> (R.D.)
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0001-5170-2959">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0001-5170-2959
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuan+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Jiang%20Yuan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Yuan Jiang</a><span class="delimiter"></span>
                                        </span>
                                    </div>
                            </div>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsz011-FM1"><div class="fn"><div class="footnote-content">Zirui Dong, Xia Zhao, Qiaoling Li and Zhenjun Yang authors contributed equally to this work.</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 4, August 2019, Pages 313–325, <a href="https://doi.org/10.1093/dnares/dsz011">https://doi.org/10.1093/dnares/dsz011</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">07 June 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">10 December 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">07 May 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">07 June 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5512299" href="/dnaresearch/article-pdf/26/4/313/29197165/dsz011.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Zirui Dong, Xia Zhao, Qiaoling Li, Zhenjun Yang, Yang Xi, Andrei Alexeev, Hanjie Shen, Ou Wang, Jie Ruan, Han Ren, Hanmin Wei, Xiaojuan Qi, Jiguang Li, Xiaofan Zhu, Yanyan Zhang, Peng Dai, Xiangdong Kong, Killeen Kirkconnell, Oleg Alferov, Shane Giles, Jennifer Yamtich, Bahram G Kermani, Chao Dong, Pengjuan Liu, Zilan Mi, Wenwei Zhang, Xun Xu, Radoje Drmanac, Kwong Wai Choy, Yuan Jiang,  Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay, <em>DNA Research</em>, Volume 26, Issue 4, August 2019, Pages 313–325, <a href="https://doi.org/10.1093/dnares/dsz011">https://doi.org/10.1093/dnares/dsz011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5512299&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20of%20coupling%20controlled%20polymerizations%20by%20adapter-ligation%20in%20mate-pair%20sequencing%20for%20detection%20of%20various%20genomic%20variants%20in%20one%20single%20assay&amp;publicationDate=2019-06-07&amp;volumeNum=26&amp;issueNum=4&amp;author=Dong%2C%20Zirui%3B%20Zhao%2C%20Xia&amp;startPage=313&amp;endPage=325&amp;contentId=10.1093%2Fdnares%2Fdsz011&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f313%2f5512299" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="140465032" id="140465032" class="abstract-title">Abstract</h2>
<section class="abstract"><p>The diversity of disease presentations warrants one single assay for detection and delineation of various genomic disorders. Herein, we describe a gel-free and biotin-capture-free mate-pair method through coupling Controlled Polymerizations by Adapter-Ligation (CP-AL). We first demonstrated the feasibility and ease-of-use in monitoring DNA nick translation and primer extension by limiting the nucleotide input. By coupling these two controlled polymerizations by a reported non-conventional adapter-ligation reaction 3′ branch ligation, we evidenced that CP-AL significantly increased DNA circularization efficiency (by 4-fold) and was applicable for different sequencing methods but at a faction of current cost. Its advantages were further demonstrated by fully elimination of small-insert-contaminated (by 39.3-fold) with a ∼50% increment of physical coverage, and producing uniform genome/exome coverage and the lowest chimeric rate. It achieved single-nucleotide variants detection with sensitivity and specificity up to 97.3 and 99.7%, respectively, compared with data from small-insert libraries. In addition, this method can provide a comprehensive delineation of structural rearrangements, evidenced by a potential diagnosis in a patient with oligo-atheno-terato-spermia. Moreover, it enables accurate mutation identification by integration of genomic variants from different aberration types. Overall, it provides a potential single-integrated solution for detecting various genomic variants, facilitating a genetic diagnosis in human diseases.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"mate-pair sequencing"'>mate-pair sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"controlled polymerization"'>controlled polymerization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="adapter-ligation">adapter-ligation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"integrated platform"'>integrated platform</a>
</div>                    <h2 scrollto-destination="140465034" id="140465034" class="section-title">1. Background</h2>
<p>Human genomic variants account for ∼300 Mb of sequence variation among individual human genomes and affect a variety of lengths ranging from single nucleotide to millions of base pairs,<a reveal-id="dsz011-B1" data-open="dsz011-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsz011-B2" data-open="dsz011-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> including single-nucleotide variants (SNVs), copy-number variants (CNVs) and structural variants (SVs). SNVs and CNVs are well known to be the causative factors in human diseases,<a reveal-id="dsz011-B3 dsz011-B4 dsz011-B5" data-open="dsz011-B3 dsz011-B4 dsz011-B5" class="link link-ref link-reveal xref-bibr"><sup>3–5</sup></a> while increasing studies demonstrates the importance of detecting SVs attributed to its pathogenicity by gene-dosage change, gene(s) disruption, gene fusion or dysregulation of disease-causing gene(s).<a reveal-id="dsz011-B2" data-open="dsz011-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsz011-B6" data-open="dsz011-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz011-B7" data-open="dsz011-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In addition, recent studies show that combination of different kinds of genomic variants in each single individual results in the variability and diversity of disease phenotypes.<a reveal-id="dsz011-B3" data-open="dsz011-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsz011-B8" data-open="dsz011-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsz011-B9" data-open="dsz011-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Thus, it warrants an integrated platform for comprehensive detection of various genomic variants for each single case.</p>
<p>Recently, 30-fold whole-genome sequencing (WGS) with small DNA fragments (300∼500 bp) has been suggested to be the first-tier testing assay for the genetic aetiological diagnosis in paediatric cases.<a reveal-id="dsz011-B10" data-open="dsz011-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> However, emerging studies have shown that at least 8.1% of those abnormalities routinely picked-up by karyotyping and chromosomal microarray analysis (CMA) would escape detection by this standard WGS,<a reveal-id="dsz011-B10" data-open="dsz011-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> letting alone those events beyond the resolutions. One of the reasons is that CNVs and SVs are predominantly mediated by repetitive elements (commonly &gt;1 kb),<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> causing the difficulty in identification of the flanking unique sequences by standard WGS.<a reveal-id="dsz011-B2" data-open="dsz011-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsz011-B12" data-open="dsz011-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsz011-B13" data-open="dsz011-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> To overcome this challenge,<a reveal-id="dsz011-B14" data-open="dsz011-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> sequencing of the junction sequences from large DNA fragments (i.e. 3∼8 kb) after circularization was introduced and named mate-pair sequencing.<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> It indicates that mate-pair sequencing with 30-fold read-depth would be an alternative strategy providing a solution to get most of the genomic variants detectable. Although there are several mate-pair methods available, limitations exist in each of them (<a reveal-id="dsz011-T1" data-open="dsz011-T1" class="link link-reveal link-table xref-fig">Table 1</a>), making clinical implementation impractical. For instance, the requirement of biotin labelling and enrichment results in high percentage of ‘inward’ sequences (read-pairs that do not cross the circularization junction)<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsz011-B17" data-open="dsz011-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsz011-B18" data-open="dsz011-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> while random fragmentation after DNA circularization leads to a high percentage of adapter-contaminated reads (harboring adapter sequence<a reveal-id="dsz011-B17" data-open="dsz011-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>). Both of them would significantly reduce the read utility. Furthermore, a high chimeric rate exists leading to a high probability of false positives, while short-read methods<a reveal-id="dsz011-B19" data-open="dsz011-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> report to have a failure of detection in ∼9% of cases.<a reveal-id="dsz011-B6" data-open="dsz011-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> In addition, gel-purification as a labourious procedure is mandatory in majority of methods (<a reveal-id="dsz011-T1" data-open="dsz011-T1" class="link link-reveal link-table xref-fig">Table 1</a>).<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsz011-B20" data-open="dsz011-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Attributed to the ability of generating long reads, sequencing on third-generation sequencing platforms [i.e. PromethION (Oxford Nanopore Tech) and Sequel (PacBio, Illumina)] seems to be an alternative strategy.<a reveal-id="dsz011-B21" data-open="dsz011-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsz011-B22" data-open="dsz011-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> However, high cost and high sequencing error rate presented prevent its application. </p>                        <a id="140465037" scrollto-destination="140465037"></a>
<div content-id="dsz011-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz011-T1" class="table-wrap-title" data-id="dsz011-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Method comparison</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>Reagent cost/sample (USD)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Size selection (i.e. gel) </th>
<th>Biotin label and enrichment </th>
<th>Turn-around-time (days) </th>
<th>DNA input amount (µg) </th>
<th>Chimeric rate (%) </th>
<th>Adapter-contaminated reads (%)<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>Longest read-length (bp) </th>
</tr></thead>
<tbody>
<tr>
<td>BLBEC </td>
<td>350 </td>
<td>Optional </td>
<td>Yes </td>
<td>3 </td>
<td>3|20<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>∼8.7 </td>
<td>NA </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>Custom Jumping Library </td>
<td>188 </td>
<td>Yes </td>
<td>Yes </td>
<td>3 </td>
<td>5∼20 </td>
<td>— </td>
<td>NA </td>
<td>∼26 </td>
</tr>
<tr>
<td>SOLiD </td>
<td>∼330 </td>
<td>Yes </td>
<td>Yes </td>
<td>2 </td>
<td>1∼5 </td>
<td>24.0–58.4 </td>
<td>NA </td>
<td>60 </td>
</tr>
<tr>
<td>Nextera </td>
<td>∼90|∼350 </td>
<td>Optional </td>
<td>Yes </td>
<td>1.5∼2 </td>
<td>1|4<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>∼3.0 </td>
<td>23.1∼23.7 </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>CP-AL </td>
<td>∼40 </td>
<td>No </td>
<td>No </td>
<td>2.5 </td>
<td>1 </td>
<td>∼2.9 </td>
<td>— </td>
<td>&gt;100 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz011-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>Reagent cost/sample (USD)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Size selection (i.e. gel) </th>
<th>Biotin label and enrichment </th>
<th>Turn-around-time (days) </th>
<th>DNA input amount (µg) </th>
<th>Chimeric rate (%) </th>
<th>Adapter-contaminated reads (%)<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>Longest read-length (bp) </th>
</tr></thead>
<tbody>
<tr>
<td>BLBEC </td>
<td>350 </td>
<td>Optional </td>
<td>Yes </td>
<td>3 </td>
<td>3|20<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>∼8.7 </td>
<td>NA </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>Custom Jumping Library </td>
<td>188 </td>
<td>Yes </td>
<td>Yes </td>
<td>3 </td>
<td>5∼20 </td>
<td>— </td>
<td>NA </td>
<td>∼26 </td>
</tr>
<tr>
<td>SOLiD </td>
<td>∼330 </td>
<td>Yes </td>
<td>Yes </td>
<td>2 </td>
<td>1∼5 </td>
<td>24.0–58.4 </td>
<td>NA </td>
<td>60 </td>
</tr>
<tr>
<td>Nextera </td>
<td>∼90|∼350 </td>
<td>Optional </td>
<td>Yes </td>
<td>1.5∼2 </td>
<td>1|4<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>∼3.0 </td>
<td>23.1∼23.7 </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>CP-AL </td>
<td>∼40 </td>
<td>No </td>
<td>No </td>
<td>2.5 </td>
<td>1 </td>
<td>∼2.9 </td>
<td>— </td>
<td>&gt;100 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>Library construction method: BLBEC [biotin-labeled blunt-end circularization<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a>]; custom jumping library;<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsz011-B19" data-open="dsz011-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> SOLiD mate-pair library construction;<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Nextera mate-pair library construction;<a reveal-id="dsz011-B25" data-open="dsz011-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> CP-AL (mate-pair method through coupling Controlled Polymerizations with a non-conventional Adapter-Ligation) demonstrated in this study.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Reagent cost was obtained from a published study (BLBEC and Custom Jumping Library)<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and supplier websites. Nextera: 4,233 USD for 48 gel-free samples or for 12 gel-plus samples, while SOLiD ∼4,000 USD for 12 samples.</p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">c</div>
<p>By searching with the Tn5 sequence<a reveal-id="dsz011-B45" data-open="dsz011-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> for the data with read-length no shorter than 100 bp.<a reveal-id="dsz011-B25" data-open="dsz011-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>
</div></div>
<div content-id="tblfn4"><div class="fn">
<div class="label">d</div>
<p>The numbers before and after the vertical line reflect DNA input for non-size-selected and size-selected library construction, respectively.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465037" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz011-T1" class="table-wrap-title" data-id="dsz011-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Method comparison</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>Reagent cost/sample (USD)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Size selection (i.e. gel) </th>
<th>Biotin label and enrichment </th>
<th>Turn-around-time (days) </th>
<th>DNA input amount (µg) </th>
<th>Chimeric rate (%) </th>
<th>Adapter-contaminated reads (%)<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>Longest read-length (bp) </th>
</tr></thead>
<tbody>
<tr>
<td>BLBEC </td>
<td>350 </td>
<td>Optional </td>
<td>Yes </td>
<td>3 </td>
<td>3|20<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>∼8.7 </td>
<td>NA </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>Custom Jumping Library </td>
<td>188 </td>
<td>Yes </td>
<td>Yes </td>
<td>3 </td>
<td>5∼20 </td>
<td>— </td>
<td>NA </td>
<td>∼26 </td>
</tr>
<tr>
<td>SOLiD </td>
<td>∼330 </td>
<td>Yes </td>
<td>Yes </td>
<td>2 </td>
<td>1∼5 </td>
<td>24.0–58.4 </td>
<td>NA </td>
<td>60 </td>
</tr>
<tr>
<td>Nextera </td>
<td>∼90|∼350 </td>
<td>Optional </td>
<td>Yes </td>
<td>1.5∼2 </td>
<td>1|4<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>∼3.0 </td>
<td>23.1∼23.7 </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>CP-AL </td>
<td>∼40 </td>
<td>No </td>
<td>No </td>
<td>2.5 </td>
<td>1 </td>
<td>∼2.9 </td>
<td>— </td>
<td>&gt;100 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz011-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Method<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>Reagent cost/sample (USD)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Size selection (i.e. gel) </th>
<th>Biotin label and enrichment </th>
<th>Turn-around-time (days) </th>
<th>DNA input amount (µg) </th>
<th>Chimeric rate (%) </th>
<th>Adapter-contaminated reads (%)<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>Longest read-length (bp) </th>
</tr></thead>
<tbody>
<tr>
<td>BLBEC </td>
<td>350 </td>
<td>Optional </td>
<td>Yes </td>
<td>3 </td>
<td>3|20<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>∼8.7 </td>
<td>NA </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>Custom Jumping Library </td>
<td>188 </td>
<td>Yes </td>
<td>Yes </td>
<td>3 </td>
<td>5∼20 </td>
<td>— </td>
<td>NA </td>
<td>∼26 </td>
</tr>
<tr>
<td>SOLiD </td>
<td>∼330 </td>
<td>Yes </td>
<td>Yes </td>
<td>2 </td>
<td>1∼5 </td>
<td>24.0–58.4 </td>
<td>NA </td>
<td>60 </td>
</tr>
<tr>
<td>Nextera </td>
<td>∼90|∼350 </td>
<td>Optional </td>
<td>Yes </td>
<td>1.5∼2 </td>
<td>1|4<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>∼3.0 </td>
<td>23.1∼23.7 </td>
<td>&gt;100 </td>
</tr>
<tr>
<td>CP-AL </td>
<td>∼40 </td>
<td>No </td>
<td>No </td>
<td>2.5 </td>
<td>1 </td>
<td>∼2.9 </td>
<td>— </td>
<td>&gt;100 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>Library construction method: BLBEC [biotin-labeled blunt-end circularization<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a>]; custom jumping library;<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsz011-B19" data-open="dsz011-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> SOLiD mate-pair library construction;<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Nextera mate-pair library construction;<a reveal-id="dsz011-B25" data-open="dsz011-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> CP-AL (mate-pair method through coupling Controlled Polymerizations with a non-conventional Adapter-Ligation) demonstrated in this study.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Reagent cost was obtained from a published study (BLBEC and Custom Jumping Library)<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and supplier websites. Nextera: 4,233 USD for 48 gel-free samples or for 12 gel-plus samples, while SOLiD ∼4,000 USD for 12 samples.</p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">c</div>
<p>By searching with the Tn5 sequence<a reveal-id="dsz011-B45" data-open="dsz011-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> for the data with read-length no shorter than 100 bp.<a reveal-id="dsz011-B25" data-open="dsz011-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>
</div></div>
<div content-id="tblfn4"><div class="fn">
<div class="label">d</div>
<p>The numbers before and after the vertical line reflect DNA input for non-size-selected and size-selected library construction, respectively.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465037" target="_blank">Open in new tab</a></div>
</div></div>
<p>Recently, we discovered a non-conventional ability of T4 DNA ligase to insert 5′ phosphorylated blunt-end double-stranded DNA (dsDNA) to DNA breaks at 3′-recessive ends, gaps or nicks to form a Y-shaped 3′-branch structure (named 3′ branch ligation or 3′BL).<a reveal-id="dsz011-B23" data-open="dsz011-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Given the high specificity of ligation and the achievement of 100% template usage theoretically, it provides a foundation for further development of mate-pair library construction. Herein, we describe a gel-free and biotin-capture-free mate-pair method through coupling two Controlled Polymerization reactions by this Adapter-Ligation reaction (CP-AL). Our study further demonstrates that CP-AL significantly increases circularization efficiency, fully eliminates small-inserts and produces a narrow range of insert-size with uniform genome/exome coverage, the lowest chimeric rate and a ∼50% increase of physical coverage. CP-AL was demonstrated the feasibility of being an integrated approach for detecting various genomic variants and at a fraction of current cost.</p>                    <h2 scrollto-destination="140465039" id="140465039" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="140465040" id="140465040" class="section-title">2.1. Ethics, consent and permissions</h3>
<p>The study protocol was approved by the Ethics Committee of BGI-Shenzhen and The Joint Chinese University of Hong Kong—New Territories East Cluster Clinical Research Ethics Committee (CREC Ref. Nos. 2016.713 and 2017.108). Six patients with infertility or recurrent miscarriages/foetal anomalies (detailed clinical indications were provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>) and with karyotyping and CMA results were enrolled, while two products of conceptions (POCs) with known trisomy were also recruited. Informed consent was obtained from each participant, ∼400 μl of peripheral blood was collected in an Ethylenediaminetetraacetic acid (EDTA) anticoagulant tube, while villus samples were collected from POCs. Genomic DNAs of NA19240 obtained from the Coriell Institute (Camden, NJ, USA)<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> was used for method evaluation, optimization and validation. In addition, published data from (i) four balanced translocations sequenced with non-size-selected biotin-labeled blunt-end circularization (BLBEC, 3∼8 kb) and (2) six cases prepared with the Nextera Mate-pair Library Construction Kit (hereafter called Nextera, Illumina, San Diego, CA, USA) released by the 1000 Genomes Project<a reveal-id="dsz011-B25" data-open="dsz011-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> were used for comparison.</p>                    <h3 scrollto-destination="140465042" id="140465042" class="section-title">2.2. DNA preparation and qualification</h3>
<p>Genomic DNA was extracted with DNeasy Blood &amp; Tissue Kit (Cat No./ID: 69506, Qiagen, Hilden, Germany), subsequently quantified with the Quant-iT dsDNA HS Assay kit (Invitrogen, Carlsbad, CA, USA) and QC checked. An aliquot of 1 μg DNA (OD260/OD280 &gt; 1.8; OD260/OD230 &gt; 2) from each sample was further sheared to a fragment size ranging from 3 to 8 kb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>) by HydroShear device (Digilab, Inc., Hopkinton, MA, USA) using the reported parameters.<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>                    <h3 scrollto-destination="140465044" id="140465044" class="section-title">2.3. Evaluation of DNA circularization efficiency</h3>
<p>DNA fragments (3∼8 kb) were purified with Agencourt AmpureXP beads (Beckman Coulter, Life Sciences, Indianapolis, IN, USA). After end-repair and the creation of an A-overhang, DNAs were ligated with adapter, adapter-1 (Ad1, <a reveal-id="dsz011-F1" data-open="dsz011-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, hereafter the adapter and primer sequences were provided in our previous study<a reveal-id="dsz011-B23" data-open="dsz011-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a>). Subsequently, after annealing, to generate dsDNA with adapters incorporating dUTPs in both strands (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary</a></span> Fig. S3), 320 ng of ligated DNA was amplified with a pair of primers, <em>Pfu</em> Turbo Cx (Agilent Technologies, Inc., Santa Clara, CA, USA) and extra dNTPs in a 200 µl of reaction volume according to the manufacturer’s protocol. Four micrograms of PCR products were treated at 37°C for 60 min with 10 units of UDG/EndoVIII cocktail [USER; New England Biolabs (NEB), Ipswich, MA, USA] to create a 14 bp complementary overhang. Double-stranded circles (dsCirs) were formed with one extendable gap (<a reveal-id="dsz011-F1" data-open="dsz011-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), and linear DNA was digested by plasmid safe (Epicentre, Illumina) at 37°C for 1.5 h. For comparison, 12 replicates were circularized by our reported BLBEC<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4)</a></span>). Circularization efficiency was calculated as the ratio of remained DNA to the original input. </p>                        <a id="140465046" scrollto-destination="140465046"></a>
<div id="dsz011-F1" data-id="dsz011-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f1.png?Expires=1576190032&amp;Signature=se-AeWxdpv1EtcGeZ1Qe6DWA1gYkzJcI4tuE7haQfBJrLaQ~KAG-EHeTcVY0hENmQ5aUHGlpn7a8H6CMyD-g0CbQRAdBHHIrV7Ep5UXhcArwhI3wXbPnRDjr8VajuEkjykFMS2PJJkaDkwo7XVHxT4szc4z84n-vy8GVqSZw6QSps9EmIcRpLDARanLgQwn9FbenY-Wovrd6RQH9Ky8xBNzgBum7YfFskrqPTt0pGyNrPckf05wod7kXi7gi4zHu4mnuI40l4qtxZfey7D1CBNRdmqc-PiEGbZMYydHmSDdBBqqnz4M-OKNhL9wQ~l1CaEboH6CiX44RdBrCBXWV8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of mate-pair library construction by coupling controlled polymerizations with a non-conventional adapter-ligation (CP-AL). The first adapter (Ad1, blue) with/without a barcode sequence (indicated by grey bar) is ligated to genomic DNA fragments (black lines). Various adapter designs may be used as the first adapter [e.g. blunt-end adapters, Y-shaped Illumina adapters, bubble adapters (Supplementary Fig. S2), etc.]. After Ad1 ligation and PCR, DNA ends are ligated together to form dsCirs containing a gap. naCNT is performed and results in DNA polymerization (dashed lines) and the movement of the gap into a selected length of the genomic DNA. 3′-branch ligation is used to ligate a 3′-end of the second adapter (Ad2_3′, yellow). The two strands of the dsCir are separated, and the single-stranded DNA (ssDNA) with Ad2_3′ at the 3′-end is used as a template for CPE (ntCPE or ttCPE). The Ad2_5′ sequence (green) is added to the 3′-end of the CPE product through 3′-branch ligation. After PCR, this results in genomic DNA with half of Ad2 at each end separated by Ad1." path-from-xml="dsz011f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465046/dsz011f1.tif" path-from-xml="dsz011f1.tif" target="_blank">Open in new tab</a><a section="140465046" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f1.png?Expires=1576190032&amp;Signature=0iyiaZgTT4iWD9VORm2Dgnw9RjSPZKQDP0dcahJ6yfpcQAnqXG2lKbwoT5c7WsJleT~tO2vh5EoY~FaEDDSpYvnsFQMsLO~REKO6NxUW9oVDrThtrzJER5zVCwY-GWPLjWkNKq-HQ1Du7iyLRBie1OB5HIz-swyN~ndXa5TkOa4-Ffvd~0YuVPjfv8xY1hu10mIl3aXoIIQkU4s3gCGiX1GkOHo2cE2qYqq1tqq4d~yMAxzXtZfsrW6WAnCliUJ4BEkIULjXnmZqRESAQG6DW1uxE21xYtOF7rcMo3XUNOmgbJDflDLVvTqAJ3t1ofK1pz91bCA9nEDkK4V0bZA6Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465046&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Schematic representation of mate-pair library construction by coupling controlled polymerizations with a non-conventional adapter-ligation (CP-AL). The first adapter (Ad1, blue) with/without a barcode sequence (indicated by grey bar) is ligated to genomic DNA fragments (black lines). Various adapter designs may be used as the first adapter [e.g. blunt-end adapters, Y-shaped Illumina adapters, bubble adapters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), etc.]. After Ad1 ligation and PCR, DNA ends are ligated together to form dsCirs containing a gap. naCNT is performed and results in DNA polymerization (dashed lines) and the movement of the gap into a selected length of the genomic DNA. 3′-branch ligation is used to ligate a 3′-end of the second adapter (Ad2_3′, yellow). The two strands of the dsCir are separated, and the single-stranded DNA (ssDNA) with Ad2_3′ at the 3′-end is used as a template for CPE (ntCPE or ttCPE). The Ad2_5′ sequence (green) is added to the 3′-end of the CPE product through 3′-branch ligation. After PCR, this results in genomic DNA with half of Ad2 at each end separated by Ad1.</p></div>
</div>
<div content-id="dsz011-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f1.png?Expires=1576190032&amp;Signature=se-AeWxdpv1EtcGeZ1Qe6DWA1gYkzJcI4tuE7haQfBJrLaQ~KAG-EHeTcVY0hENmQ5aUHGlpn7a8H6CMyD-g0CbQRAdBHHIrV7Ep5UXhcArwhI3wXbPnRDjr8VajuEkjykFMS2PJJkaDkwo7XVHxT4szc4z84n-vy8GVqSZw6QSps9EmIcRpLDARanLgQwn9FbenY-Wovrd6RQH9Ky8xBNzgBum7YfFskrqPTt0pGyNrPckf05wod7kXi7gi4zHu4mnuI40l4qtxZfey7D1CBNRdmqc-PiEGbZMYydHmSDdBBqqnz4M-OKNhL9wQ~l1CaEboH6CiX44RdBrCBXWV8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of mate-pair library construction by coupling controlled polymerizations with a non-conventional adapter-ligation (CP-AL). The first adapter (Ad1, blue) with/without a barcode sequence (indicated by grey bar) is ligated to genomic DNA fragments (black lines). Various adapter designs may be used as the first adapter [e.g. blunt-end adapters, Y-shaped Illumina adapters, bubble adapters (Supplementary Fig. S2), etc.]. After Ad1 ligation and PCR, DNA ends are ligated together to form dsCirs containing a gap. naCNT is performed and results in DNA polymerization (dashed lines) and the movement of the gap into a selected length of the genomic DNA. 3′-branch ligation is used to ligate a 3′-end of the second adapter (Ad2_3′, yellow). The two strands of the dsCir are separated, and the single-stranded DNA (ssDNA) with Ad2_3′ at the 3′-end is used as a template for CPE (ntCPE or ttCPE). The Ad2_5′ sequence (green) is added to the 3′-end of the CPE product through 3′-branch ligation. After PCR, this results in genomic DNA with half of Ad2 at each end separated by Ad1." path-from-xml="dsz011f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465046/dsz011f1.tif" path-from-xml="dsz011f1.tif" target="_blank">Open in new tab</a><a section="140465046" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f1.png?Expires=1576190032&amp;Signature=0iyiaZgTT4iWD9VORm2Dgnw9RjSPZKQDP0dcahJ6yfpcQAnqXG2lKbwoT5c7WsJleT~tO2vh5EoY~FaEDDSpYvnsFQMsLO~REKO6NxUW9oVDrThtrzJER5zVCwY-GWPLjWkNKq-HQ1Du7iyLRBie1OB5HIz-swyN~ndXa5TkOa4-Ffvd~0YuVPjfv8xY1hu10mIl3aXoIIQkU4s3gCGiX1GkOHo2cE2qYqq1tqq4d~yMAxzXtZfsrW6WAnCliUJ4BEkIULjXnmZqRESAQG6DW1uxE21xYtOF7rcMo3XUNOmgbJDflDLVvTqAJ3t1ofK1pz91bCA9nEDkK4V0bZA6Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465046&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Schematic representation of mate-pair library construction by coupling controlled polymerizations with a non-conventional adapter-ligation (CP-AL). The first adapter (Ad1, blue) with/without a barcode sequence (indicated by grey bar) is ligated to genomic DNA fragments (black lines). Various adapter designs may be used as the first adapter [e.g. blunt-end adapters, Y-shaped Illumina adapters, bubble adapters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), etc.]. After Ad1 ligation and PCR, DNA ends are ligated together to form dsCirs containing a gap. naCNT is performed and results in DNA polymerization (dashed lines) and the movement of the gap into a selected length of the genomic DNA. 3′-branch ligation is used to ligate a 3′-end of the second adapter (Ad2_3′, yellow). The two strands of the dsCir are separated, and the single-stranded DNA (ssDNA) with Ad2_3′ at the 3′-end is used as a template for CPE (ntCPE or ttCPE). The Ad2_5′ sequence (green) is added to the 3′-end of the CPE product through 3′-branch ligation. After PCR, this results in genomic DNA with half of Ad2 at each end separated by Ad1.</p></div>
</div>                    <h3 scrollto-destination="140465047" id="140465047" class="section-title">2.4. Evaluation of nick translation by limiting nucleotide quantities</h3>
<p>Nick translation by PolI (DNA polymerase I) was performed in a 20 μl reaction containing 0.23 pmol dsCirs, NEBuffer 2, 10 U DNA Polymerase I (NEB) and different concentrations (10, 5 or 3.33 μM) of dNTPs (Affymetrix, Inc., Santa Clara, CA, USA) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The dNTP mix contained equimolar quantities of each dNTP (dATP, dTTP, dCTP and dGTP). The reaction mixture was incubated at 10°C for 20 min. The products were purified with the MinElute PCR Purification Kit (Qiagen, Hilden, Germany). Exonucleases were used for digestion and to form linear DNA.<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Size analysis was performed for the MinElute purified Exo-treated products on 6% polyacrylamide TBE gels (Invitrogen) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3)</a></span>).</p>                    <h3 scrollto-destination="140465049" id="140465049" class="section-title">2.5. Evaluation of primer extension by limiting nucleotide quantities</h3>
<p>In brief, the 3′-ends of gDNA was ligated to the 3′-end of adapter, adapter-2 (Ad2, ON5 and ON6) and subsequently denatured at 96°C, annealed with a primer (ON10) at 56°C, and extended with <em>Taq</em> polymerase and serial dilutions of dNTPs (33.3, 25, 20, 16.7 and 14.3 pmol) at 72°C for 10 min. Products were purified with Agencourt Axygen beads and treated with Exonuclease VII, and bead purification. Size analysis was performed on a 6% TBE gel (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                    <h3 scrollto-destination="140465051" id="140465051" class="section-title">2.6. Comparison of different parameters in CP-AL and validation</h3>
<p>CP-AL was designed to couple nick translation and primer extension by 3′BL. For comparison, CP-ALs with different parameters (after formation of dsCir) were carried out with four DNA aliquots of NA19240<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> from a single source for comparison. dsCir DNA prepared as described above was subjected to Controlled Nick Translation (CNT) and Controlled Primer Extension (CPE) in different conditions. (i) Polymerase and additional dATP and dTTP for nick translation: for Libraries #1, #2 and #3, 1.5 pmol of dsCirs was incubated with Polymerase I and different amounts of dNTP mix; for Library #4, 0.45 U of PolI and 0.45 U of <em>Taq</em> Polymerase were used instead; an equal quantity of each dNTP was added (33 pmol of each) for Library #1, whereas 33 pmol of dCTP and dGTP and 100 pmol of dATP and dTTP (3-fold of dCTP/dGTP in total) were added for each of the other three libraries. After nick translation, 3′BL was performed by mixing the products with the 3′-end of Ad2 (ON5 and ON6) in a 120 µl of reaction followed by bead purification; (ii) Primer extension: CPE by adjusting reaction temperature and time (ttCPE) was used for Libraries #1 and #2; the reaction contained the 3′BL product, <em>Pfu</em> Turbo Cx (Agilent Technologies) and primer ON10. The mixture was incubated at 92°C for 5 min, 56°C for 30 s and 60°C for 30 s. CPE by adjustment of nucleotide amount (naCPE) was used in Libraries #3 and #4. The reaction consisted of 3′BL products, <em>Taq</em> Polymerase, primer ON10 and different amounts of dNTPs. The naCPE program was 96°C for 5 min, 56°C for 1 min and 72°C for 5 min. A sample of 40 μl purified product from either ttCPE or naCPE was ligated to the 5′-end of Ad2 (ON7 and ON8) and purified with Agencourt Axygen beads. (iii) Polymerase for Ad2 amplification: Libraries #1 and #4 were amplified using Q5 high-fidelity DNA polymerase (NEB). Library #2 was amplified using <em>Pfu</em> Turbo Cx. Library #3 was amplified using pyrophage polymerase (Lucigen, Middleton, WI, USA).</p>
<p>For all four libraries, single-stranded circles (ssCirs) and DNA nanoballs (DNBs) were prepared and subjected to improved cPAL (Combinatorial Probe-Anchor Ligation) sequencing on a DNB-based sequencing platform [Complete Genomics, Inc., (CGI) Mountain View, CA, USA]. We thus obtained 28 bases (2xSE28) from the 5′ ends of each adaptor, Ad1 and Ad2.<a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                    <h3 scrollto-destination="140465054" id="140465054" class="section-title">2.7. Optimized CP-AL</h3>
<p>Purification of DNAs, end-repair, A-tailing, adapter-ligation, PCR amplification (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span>), purification and dsCir DNA formation were performed for each DNA, according to the procedure described above. Of note, after purification of Ad1 PCR products, a total of 20 µg amplified DNA, which could be a mixture of each amplified DNA with equal quantity (from the six patients) or from one single sample (Trisomy 2 or 8), was used for dsCir DNA formation. naCNT was performed with 1 pmol dsCir DNA; Bst DNA Polymerase, Full Length (NEB); Klenow fragment (Enzymatics, Inc.); and limited dNTPs. 3′BL was performed to ligate the 3′-end of Ad2 to the naCNT products. Primer extension was accomplished using ttCPE as described above, and the reaction mixture was incubated at 92°C for 5 min, 56°C for 60 s and 60°C for 40 s; The sample was then purified with Agencourt AmpureXP beads. ttCPE products were ligated to the 5′-end of Ad2 and amplified with <em>Pfu</em> Turbo Cx (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>). ssCirs and DNBs were prepared for sequencing on the BGISEQ-500 platform (BGI-Wuhan, Wuhan, China).</p>
<p>For evaluation of repeatability, CP-AL was repeated for the six subjects after Ad1 PCR experiment. For these two libraries, one was subjected for sequencing with reads of 2 × 50 bp (equivalent to paired-end 50 bp, <a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span>), while the other was for 2 × 100 bp (equivalent to paired-end 100 bp). A minimum of 70 million read-pairs was obtained for each sample for 50 and 100 bp, respectively. For the POCs, a minimum of 30-fold read-depth was generated (2 × 100 bp, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) for each case. The robustness of CP-AL was evaluated by measurement of the DNA size for (i) DNA fragments after fragmentation with HydroShear; (ii) PCR products after Ad1 PCR and (iii) after Ad2 PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). </p>                        <a id="140465057" scrollto-destination="140465057"></a>
<div content-id="dsz011-T2" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz011-T2" class="table-wrap-title" data-id="dsz011-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Evaluation of different parameters/conditions for mate-pair library construction</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Library </th>
<th>Nick translation </th>
<th>Primer extension<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>% Autosomal exome by GC with &lt;60% of mean coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>% Faction of the genome with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>% Fraction of the coding region with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>#1 </td>
<td>PolI </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>6.6 </td>
<td>95.0 </td>
<td>95.9 </td>
</tr>
<tr>
<td>#2 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>1.8 </td>
<td>96.2 </td>
<td>97.8 </td>
</tr>
<tr>
<td>#3 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>0.7 </td>
<td>96.7 </td>
<td>98.1 </td>
</tr>
<tr>
<td>#4 </td>
<td>PolI/<em>Taq</em>+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>1.4 </td>
<td>96.1 </td>
<td>97.8 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz011-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Library </th>
<th>Nick translation </th>
<th>Primer extension<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>% Autosomal exome by GC with &lt;60% of mean coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>% Faction of the genome with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>% Fraction of the coding region with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>#1 </td>
<td>PolI </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>6.6 </td>
<td>95.0 </td>
<td>95.9 </td>
</tr>
<tr>
<td>#2 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>1.8 </td>
<td>96.2 </td>
<td>97.8 </td>
</tr>
<tr>
<td>#3 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>0.7 </td>
<td>96.7 </td>
<td>98.1 </td>
</tr>
<tr>
<td>#4 </td>
<td>PolI/<em>Taq</em>+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>1.4 </td>
<td>96.1 </td>
<td>97.8 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn5"><div class="fn">
<div class="label">a</div>
<p>ttCPE and naCPE refer to controlled primer extension by adjusting of reaction temperature and duration time or limiting the nucleotide input only.</p>
</div></div>
<div content-id="tblfn6"><div class="fn">
<div class="label">b</div>
<p>% Autosomal exome by GC with &lt;60% of mean coverage measures the fraction of the autosomal exome regions where the normalized coverage of exome by cumulative base GC percentage is &lt;60% of the mean coverage. Higher percentage indicates higher GC bias in the autosomal exome regions.</p>
</div></div>
<div content-id="tblfn7"><div class="fn">
<div class="label">c</div>
<p>Fully called genome and exome fractions; both alleles can be confidently called by Trait-o-Matic software.<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>
</div></div>
<div content-id="tblfn8"><div class="fn">
<div class="label">d</div>
<p>Additional 2-fold of dATP and dTTP relative to dGTP and dCTP.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465057" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz011-T2" class="table-wrap-title" data-id="dsz011-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Evaluation of different parameters/conditions for mate-pair library construction</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Library </th>
<th>Nick translation </th>
<th>Primer extension<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>% Autosomal exome by GC with &lt;60% of mean coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>% Faction of the genome with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>% Fraction of the coding region with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>#1 </td>
<td>PolI </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>6.6 </td>
<td>95.0 </td>
<td>95.9 </td>
</tr>
<tr>
<td>#2 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>1.8 </td>
<td>96.2 </td>
<td>97.8 </td>
</tr>
<tr>
<td>#3 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>0.7 </td>
<td>96.7 </td>
<td>98.1 </td>
</tr>
<tr>
<td>#4 </td>
<td>PolI/<em>Taq</em>+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>1.4 </td>
<td>96.1 </td>
<td>97.8 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz011-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Library </th>
<th>Nick translation </th>
<th>Primer extension<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>% Autosomal exome by GC with &lt;60% of mean coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>% Faction of the genome with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th>% Fraction of the coding region with fully called<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>#1 </td>
<td>PolI </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>6.6 </td>
<td>95.0 </td>
<td>95.9 </td>
</tr>
<tr>
<td>#2 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>ttCPE by <em>Pfu</em>Cx </td>
<td>1.8 </td>
<td>96.2 </td>
<td>97.8 </td>
</tr>
<tr>
<td>#3 </td>
<td>PolI+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>0.7 </td>
<td>96.7 </td>
<td>98.1 </td>
</tr>
<tr>
<td>#4 </td>
<td>PolI/<em>Taq</em>+2xAT<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
<td>naCPE by <em>Taq</em> </td>
<td>1.4 </td>
<td>96.1 </td>
<td>97.8 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn5"><div class="fn">
<div class="label">a</div>
<p>ttCPE and naCPE refer to controlled primer extension by adjusting of reaction temperature and duration time or limiting the nucleotide input only.</p>
</div></div>
<div content-id="tblfn6"><div class="fn">
<div class="label">b</div>
<p>% Autosomal exome by GC with &lt;60% of mean coverage measures the fraction of the autosomal exome regions where the normalized coverage of exome by cumulative base GC percentage is &lt;60% of the mean coverage. Higher percentage indicates higher GC bias in the autosomal exome regions.</p>
</div></div>
<div content-id="tblfn7"><div class="fn">
<div class="label">c</div>
<p>Fully called genome and exome fractions; both alleles can be confidently called by Trait-o-Matic software.<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>
</div></div>
<div content-id="tblfn8"><div class="fn">
<div class="label">d</div>
<p>Additional 2-fold of dATP and dTTP relative to dGTP and dCTP.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465057" target="_blank">Open in new tab</a></div>
</div></div>                        <a id="140465058" scrollto-destination="140465058"></a>
<div content-id="dsz011-T3" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz011-T3" class="table-wrap-title" data-id="dsz011-T3">
<div class="label">Table 3</div>
<div class="caption"><p>Performance comparison with paired-end 50 or 26 bp reads</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Sample/data </th>
<th>Karyotype </th>
<th>Results from whole-genome sequencing </th>
<th>Library construction method and sequencing platform<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>DNA input (µg) </th>
<th>Chimeric rate (%)<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Duplication rate (%) </th>
<th>Aligned distance &lt;1|kb (%) </th>
<th>Aligned distance &lt;1|kb (% uniquely aligned and non-duplicated) </th>
<th>Fraction of genome physically covered (%) </th>
</tr></thead>
<tbody>
<tr>
<td>BCA01<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;20)(p13;p13) </td>
<td>46,XX,t(2;20)(p15;p13) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>9.04 </td>
<td>15.7 </td>
<td>27.6 </td>
<td>5.2 </td>
<td>98.6|91.1<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA02<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;3)(p21;p25) </td>
<td>46,XX,t(2;3)(p22.1;p26.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>10.92 </td>
<td>15.9 </td>
<td>22.3 </td>
<td>3.9 </td>
<td>98.5|93.7<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA11<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(4;5; 14)(q31.3;q33; q13) </td>
<td>46,XX,t(4;5;14)(q32.3;q34;q21.3) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>6.70 </td>
<td>12.5 </td>
<td>29.7 </td>
<td>6.0 </td>
<td>96.7|91.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA16<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(14;18)(q24;q23) </td>
<td>46,XX,t(14;18)(q13.2;q22.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>8.04 </td>
<td>12.0 </td>
<td>30.5 </td>
<td>8.0 </td>
<td>96.7|91.2<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample01<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(10;13)(q11.2;q31q33) </td>
<td>46,XX,ins(10;13)(q21.3;q21.2q31.3) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.84 </td>
<td>16.8 </td>
<td>0.6 </td>
<td>0.1 </td>
<td>97.7|96.6<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample02<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;2)(q23;p13p22) </td>
<td>46,XY,ins(6;2)der(6)(6pter-&gt;6q13:: 6q13-&gt;6q21.3:: 2p16.1&lt;-2p16.1:: 6q21-&gt;6q22.31:: 2p16.1-&gt;2p11.2:: 2p22.2-&gt;2p16.1:: 6q22.31-&gt;6q26:: 6q13&lt;-6q13:: 6q26-&gt;6qter); der(2)(2pter-&gt;2p22.2:: 2p11.2-&gt;2qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.82 </td>
<td>17.8 </td>
<td>0.8 </td>
<td>0.2 </td>
<td>96.2|94.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample03<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(2;18)(q31;q21.1q23) </td>
<td>46, XX, ins(2; 18)(q32.2; q21.1q22.1); inv(18)(q21.1) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>3.52 </td>
<td>17.5 </td>
<td>0.5 </td>
<td>0.1 </td>
<td>98.0|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample04<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY, der(4)ins(4)(q21.1;q31.1q31.3)inv(4)(p12q13.3) </td>
<td>46,XY,der(4)(pter-&gt;p12:: q13.3&lt;-p12:: q31.1-&gt;q31.23:: q13.3-&gt;q31.1:: q31.23-&gt;qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.74 </td>
<td>19.0 </td>
<td>0.9 </td>
<td>0.4 </td>
<td>96.3|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample05<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;3)(q13; q21q24) </td>
<td>46,XY,ins(6;3)der(6)(6pter-&gt;6q14.3:: 3q21.1&lt;-3q21.1:: 3q24&lt;-3q21.3:: 3q21.1-&gt;3q21.3:: 6q14.3-&gt;6qter); der(3)(3pter-&gt;3q21.1:: 3q21.1-&gt;3q21.1:: 3q24-&gt;3qter)seq[hg19] del(3q24)chr3: g.146055006_148300124del </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.87 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>96.1|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample06<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX </td>
<td>46,XX </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.81 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>98.1|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz011-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Sample/data </th>
<th>Karyotype </th>
<th>Results from whole-genome sequencing </th>
<th>Library construction method and sequencing platform<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>DNA input (µg) </th>
<th>Chimeric rate (%)<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Duplication rate (%) </th>
<th>Aligned distance &lt;1|kb (%) </th>
<th>Aligned distance &lt;1|kb (% uniquely aligned and non-duplicated) </th>
<th>Fraction of genome physically covered (%) </th>
</tr></thead>
<tbody>
<tr>
<td>BCA01<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;20)(p13;p13) </td>
<td>46,XX,t(2;20)(p15;p13) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>9.04 </td>
<td>15.7 </td>
<td>27.6 </td>
<td>5.2 </td>
<td>98.6|91.1<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA02<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;3)(p21;p25) </td>
<td>46,XX,t(2;3)(p22.1;p26.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>10.92 </td>
<td>15.9 </td>
<td>22.3 </td>
<td>3.9 </td>
<td>98.5|93.7<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA11<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(4;5; 14)(q31.3;q33; q13) </td>
<td>46,XX,t(4;5;14)(q32.3;q34;q21.3) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>6.70 </td>
<td>12.5 </td>
<td>29.7 </td>
<td>6.0 </td>
<td>96.7|91.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA16<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(14;18)(q24;q23) </td>
<td>46,XX,t(14;18)(q13.2;q22.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>8.04 </td>
<td>12.0 </td>
<td>30.5 </td>
<td>8.0 </td>
<td>96.7|91.2<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample01<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(10;13)(q11.2;q31q33) </td>
<td>46,XX,ins(10;13)(q21.3;q21.2q31.3) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.84 </td>
<td>16.8 </td>
<td>0.6 </td>
<td>0.1 </td>
<td>97.7|96.6<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample02<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;2)(q23;p13p22) </td>
<td>46,XY,ins(6;2)der(6)(6pter-&gt;6q13:: 6q13-&gt;6q21.3:: 2p16.1&lt;-2p16.1:: 6q21-&gt;6q22.31:: 2p16.1-&gt;2p11.2:: 2p22.2-&gt;2p16.1:: 6q22.31-&gt;6q26:: 6q13&lt;-6q13:: 6q26-&gt;6qter); der(2)(2pter-&gt;2p22.2:: 2p11.2-&gt;2qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.82 </td>
<td>17.8 </td>
<td>0.8 </td>
<td>0.2 </td>
<td>96.2|94.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample03<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(2;18)(q31;q21.1q23) </td>
<td>46, XX, ins(2; 18)(q32.2; q21.1q22.1); inv(18)(q21.1) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>3.52 </td>
<td>17.5 </td>
<td>0.5 </td>
<td>0.1 </td>
<td>98.0|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample04<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY, der(4)ins(4)(q21.1;q31.1q31.3)inv(4)(p12q13.3) </td>
<td>46,XY,der(4)(pter-&gt;p12:: q13.3&lt;-p12:: q31.1-&gt;q31.23:: q13.3-&gt;q31.1:: q31.23-&gt;qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.74 </td>
<td>19.0 </td>
<td>0.9 </td>
<td>0.4 </td>
<td>96.3|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample05<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;3)(q13; q21q24) </td>
<td>46,XY,ins(6;3)der(6)(6pter-&gt;6q14.3:: 3q21.1&lt;-3q21.1:: 3q24&lt;-3q21.3:: 3q21.1-&gt;3q21.3:: 6q14.3-&gt;6qter); der(3)(3pter-&gt;3q21.1:: 3q21.1-&gt;3q21.1:: 3q24-&gt;3qter)seq[hg19] del(3q24)chr3: g.146055006_148300124del </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.87 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>96.1|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample06<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX </td>
<td>46,XX </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.81 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>98.1|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn9"><div class="fn">
<div class="label">a</div>
<p>BLBEC and CP-AL indicate library construction based on biotin-labeled blunt-end circularization and through coupling Controlled Polymerizations with Adapter-Ligation, respectively; HS and BS refer to HiSeq 2000 (Illumina) and BGISEQ-500 (BGI-Wuhan), respectively.</p>
</div></div>
<div content-id="tblfn10"><div class="fn">
<div class="label">b</div>
<p>Chimeric read-pairs were defined as the read-pairs aligned to different chromosomes or to the same chromosomes but with a distance &gt;10 kb.</p>
</div></div>
<div content-id="tblfn11"><div class="fn">
<div class="label">c</div>
<p>Sequenced with non-size-selected BLBEC libraries, published in our pilot study.<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>
</div></div>
<div content-id="tblfn12"><div class="fn">
<div class="label">d</div>
<p>The numbers before and after each vertical line reflect the fraction of the genome physically coverage rate by the data with paired-end 50 bp and trimmed 26 bp reads, respectively.</p>
</div></div>
<div content-id="tblfn13"><div class="fn">
<div class="label">e</div>
<p>Sequenced with CP-AL libraries and the detection results shown here are based on the data with 100 bp.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465058" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz011-T3" class="table-wrap-title" data-id="dsz011-T3">
<div class="label">Table 3</div>
<div class="caption"><p>Performance comparison with paired-end 50 or 26 bp reads</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Sample/data </th>
<th>Karyotype </th>
<th>Results from whole-genome sequencing </th>
<th>Library construction method and sequencing platform<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>DNA input (µg) </th>
<th>Chimeric rate (%)<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Duplication rate (%) </th>
<th>Aligned distance &lt;1|kb (%) </th>
<th>Aligned distance &lt;1|kb (% uniquely aligned and non-duplicated) </th>
<th>Fraction of genome physically covered (%) </th>
</tr></thead>
<tbody>
<tr>
<td>BCA01<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;20)(p13;p13) </td>
<td>46,XX,t(2;20)(p15;p13) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>9.04 </td>
<td>15.7 </td>
<td>27.6 </td>
<td>5.2 </td>
<td>98.6|91.1<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA02<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;3)(p21;p25) </td>
<td>46,XX,t(2;3)(p22.1;p26.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>10.92 </td>
<td>15.9 </td>
<td>22.3 </td>
<td>3.9 </td>
<td>98.5|93.7<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA11<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(4;5; 14)(q31.3;q33; q13) </td>
<td>46,XX,t(4;5;14)(q32.3;q34;q21.3) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>6.70 </td>
<td>12.5 </td>
<td>29.7 </td>
<td>6.0 </td>
<td>96.7|91.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA16<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(14;18)(q24;q23) </td>
<td>46,XX,t(14;18)(q13.2;q22.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>8.04 </td>
<td>12.0 </td>
<td>30.5 </td>
<td>8.0 </td>
<td>96.7|91.2<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample01<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(10;13)(q11.2;q31q33) </td>
<td>46,XX,ins(10;13)(q21.3;q21.2q31.3) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.84 </td>
<td>16.8 </td>
<td>0.6 </td>
<td>0.1 </td>
<td>97.7|96.6<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample02<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;2)(q23;p13p22) </td>
<td>46,XY,ins(6;2)der(6)(6pter-&gt;6q13:: 6q13-&gt;6q21.3:: 2p16.1&lt;-2p16.1:: 6q21-&gt;6q22.31:: 2p16.1-&gt;2p11.2:: 2p22.2-&gt;2p16.1:: 6q22.31-&gt;6q26:: 6q13&lt;-6q13:: 6q26-&gt;6qter); der(2)(2pter-&gt;2p22.2:: 2p11.2-&gt;2qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.82 </td>
<td>17.8 </td>
<td>0.8 </td>
<td>0.2 </td>
<td>96.2|94.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample03<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(2;18)(q31;q21.1q23) </td>
<td>46, XX, ins(2; 18)(q32.2; q21.1q22.1); inv(18)(q21.1) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>3.52 </td>
<td>17.5 </td>
<td>0.5 </td>
<td>0.1 </td>
<td>98.0|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample04<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY, der(4)ins(4)(q21.1;q31.1q31.3)inv(4)(p12q13.3) </td>
<td>46,XY,der(4)(pter-&gt;p12:: q13.3&lt;-p12:: q31.1-&gt;q31.23:: q13.3-&gt;q31.1:: q31.23-&gt;qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.74 </td>
<td>19.0 </td>
<td>0.9 </td>
<td>0.4 </td>
<td>96.3|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample05<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;3)(q13; q21q24) </td>
<td>46,XY,ins(6;3)der(6)(6pter-&gt;6q14.3:: 3q21.1&lt;-3q21.1:: 3q24&lt;-3q21.3:: 3q21.1-&gt;3q21.3:: 6q14.3-&gt;6qter); der(3)(3pter-&gt;3q21.1:: 3q21.1-&gt;3q21.1:: 3q24-&gt;3qter)seq[hg19] del(3q24)chr3: g.146055006_148300124del </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.87 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>96.1|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample06<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX </td>
<td>46,XX </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.81 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>98.1|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz011-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Sample/data </th>
<th>Karyotype </th>
<th>Results from whole-genome sequencing </th>
<th>Library construction method and sequencing platform<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th>DNA input (µg) </th>
<th>Chimeric rate (%)<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th>Duplication rate (%) </th>
<th>Aligned distance &lt;1|kb (%) </th>
<th>Aligned distance &lt;1|kb (% uniquely aligned and non-duplicated) </th>
<th>Fraction of genome physically covered (%) </th>
</tr></thead>
<tbody>
<tr>
<td>BCA01<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;20)(p13;p13) </td>
<td>46,XX,t(2;20)(p15;p13) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>9.04 </td>
<td>15.7 </td>
<td>27.6 </td>
<td>5.2 </td>
<td>98.6|91.1<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA02<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(2;3)(p21;p25) </td>
<td>46,XX,t(2;3)(p22.1;p26.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>10.92 </td>
<td>15.9 </td>
<td>22.3 </td>
<td>3.9 </td>
<td>98.5|93.7<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA11<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(4;5; 14)(q31.3;q33; q13) </td>
<td>46,XX,t(4;5;14)(q32.3;q34;q21.3) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>6.70 </td>
<td>12.5 </td>
<td>29.7 </td>
<td>6.0 </td>
<td>96.7|91.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>BCA16<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </td>
<td>46,XX,t(14;18)(q24;q23) </td>
<td>46,XX,t(14;18)(q13.2;q22.1) </td>
<td>BLBEC+HS </td>
<td>3 </td>
<td>8.04 </td>
<td>12.0 </td>
<td>30.5 </td>
<td>8.0 </td>
<td>96.7|91.2<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample01<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(10;13)(q11.2;q31q33) </td>
<td>46,XX,ins(10;13)(q21.3;q21.2q31.3) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.84 </td>
<td>16.8 </td>
<td>0.6 </td>
<td>0.1 </td>
<td>97.7|96.6<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample02<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;2)(q23;p13p22) </td>
<td>46,XY,ins(6;2)der(6)(6pter-&gt;6q13:: 6q13-&gt;6q21.3:: 2p16.1&lt;-2p16.1:: 6q21-&gt;6q22.31:: 2p16.1-&gt;2p11.2:: 2p22.2-&gt;2p16.1:: 6q22.31-&gt;6q26:: 6q13&lt;-6q13:: 6q26-&gt;6qter); der(2)(2pter-&gt;2p22.2:: 2p11.2-&gt;2qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.82 </td>
<td>17.8 </td>
<td>0.8 </td>
<td>0.2 </td>
<td>96.2|94.9<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample03<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX,ins(2;18)(q31;q21.1q23) </td>
<td>46, XX, ins(2; 18)(q32.2; q21.1q22.1); inv(18)(q21.1) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>3.52 </td>
<td>17.5 </td>
<td>0.5 </td>
<td>0.1 </td>
<td>98.0|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample04<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY, der(4)ins(4)(q21.1;q31.1q31.3)inv(4)(p12q13.3) </td>
<td>46,XY,der(4)(pter-&gt;p12:: q13.3&lt;-p12:: q31.1-&gt;q31.23:: q13.3-&gt;q31.1:: q31.23-&gt;qter) </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.74 </td>
<td>19.0 </td>
<td>0.9 </td>
<td>0.4 </td>
<td>96.3|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample05<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XY,ins(6;3)(q13; q21q24) </td>
<td>46,XY,ins(6;3)der(6)(6pter-&gt;6q14.3:: 3q21.1&lt;-3q21.1:: 3q24&lt;-3q21.3:: 3q21.1-&gt;3q21.3:: 6q14.3-&gt;6qter); der(3)(3pter-&gt;3q21.1:: 3q21.1-&gt;3q21.1:: 3q24-&gt;3qter)seq[hg19] del(3q24)chr3: g.146055006_148300124del </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.87 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>96.1|95.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
<tr>
<td>Sample06<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </td>
<td>46,XX </td>
<td>46,XX </td>
<td>CP-AL+BS </td>
<td>1 </td>
<td>2.81 </td>
<td>18.9 </td>
<td>0.7 </td>
<td>0.2 </td>
<td>98.1|97.0<a reveal-id="tblfn12" data-open="tblfn12" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn9"><div class="fn">
<div class="label">a</div>
<p>BLBEC and CP-AL indicate library construction based on biotin-labeled blunt-end circularization and through coupling Controlled Polymerizations with Adapter-Ligation, respectively; HS and BS refer to HiSeq 2000 (Illumina) and BGISEQ-500 (BGI-Wuhan), respectively.</p>
</div></div>
<div content-id="tblfn10"><div class="fn">
<div class="label">b</div>
<p>Chimeric read-pairs were defined as the read-pairs aligned to different chromosomes or to the same chromosomes but with a distance &gt;10 kb.</p>
</div></div>
<div content-id="tblfn11"><div class="fn">
<div class="label">c</div>
<p>Sequenced with non-size-selected BLBEC libraries, published in our pilot study.<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>
</div></div>
<div content-id="tblfn12"><div class="fn">
<div class="label">d</div>
<p>The numbers before and after each vertical line reflect the fraction of the genome physically coverage rate by the data with paired-end 50 bp and trimmed 26 bp reads, respectively.</p>
</div></div>
<div content-id="tblfn13"><div class="fn">
<div class="label">e</div>
<p>Sequenced with CP-AL libraries and the detection results shown here are based on the data with 100 bp.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465058" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="140465059" id="140465059" class="section-title">2.8. Data analysis, detection of genomics variants and validation</h3>
<p>Data from CGI<a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> were aligned to the human reference genome (hg19) with a reported method<a reveal-id="dsz011-B27" data-open="dsz011-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and detection of SNVs and InDels were carried out by Trait-o-Matic.<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> For evaluation of coverage uniformity, fraction of genome/exome/coding region was calculated as the size of region with both alleles confidently/fully called dividing by the full length (<a reveal-id="dsz011-T2" data-open="dsz011-T2" class="link link-reveal link-table xref-fig">Table 2</a>). In addition, sensitivity and specificity of SNV and InDel detection were determined by comparing the results from small-inserted libraries sequenced in the same platform.<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> For the data generated from HiSeq and BGISEQ (<a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>), alignment was performed with Burrows-Wheeler Aligner (BWA).<a reveal-id="dsz011-B28" data-open="dsz011-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Physical coverage was calculated as the sum of aligned distance between each uniquely aligned read-pair not resulted from PCR duplication. </p>
<p>The correlation of the GC (G and C nucleotides) percentage between the human reference genome and sequencing data was assessed (<a reveal-id="dsz011-F2" data-open="dsz011-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). In brief, equal number (<em>N</em> = 50 millions) of uniquely aligned read-pairs was randomly selected from each sample<a reveal-id="dsz011-B29" data-open="dsz011-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and merged into a dataset. Adjustable sliding windows (50 kb with 5 kb increments) and non-overlapping windows (5 kb) were determined by counting an equal number of read-pairs and the reference GC percentage of each window was calculated based on the reference genome. In each case, the GC percentage of each read-pair was calculated as the reference sequence between the aligned intervals, while GC percentage of each window was set as the median value of GC percentage from all read-pairs aligned to a particular window. </p>                        <a id="140465062" scrollto-destination="140465062"></a>
<div id="dsz011-F2" data-id="dsz011-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f2.png?Expires=1576190032&amp;Signature=oKDOmHpLVChpKWBUVS-PO5JQaFBI7t6ZfGNa0FKRz8GN1eevOPsZo2iqJ4e10epCO5q5aYUo~n1guHBtMs84a2ye09WfFIZMoYpy-LYc9nmuhlSfy9sDRfD7OL2dOkx8SWbax2v5IYux9moKHbsjW2GGg-G~9Ou-XiefqejGv82MNIJMBhlc0d99izK6lg40y65OymLbc9kEeX1jIjOtgi4ApxpC5pEnS~3xdUfFiuJUDU-zZsTryCXnm9xGTetSRWm2JWRb0CdEsKrD9oWT8coqEW173NAoKpu6~RH~SeCwACAOzGL5~~65mJO-mLlG4ENdfDSB98BqNLXFbWvrZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC concordance and insert-size distribution of data from CP-AL. (A–F) The correlation of GC percentage between human reference genome and sequencing reads in Samples01 to 06, respectively. The detailed method is described in Materials and methods. Each point indicates a particular window with the GC percentage (%) determined by sequencing read (X-axis) and human reference genome (Y-axis). Dotted line shows the ideal 100% correlation. Pearson correlation coefficient with P-value is shown in the left side of each figure. (G) The size distributions of four samples prepared with BLBEC and six prepared with CP-AL, respectively. Each sample was non-size-selected and sequenced with paired-end 50 bp reads. The X-axis shows the insert-size, and Y-axis indicates the percentage of the paired-end reads with certain insert-sizes. The figure shown in the left side with red dotted frame is the zoom in view. (H) Physical coverage distribution for the data from different library construction methods (BLBEC and CP-AL) for 50 million read-pairs per sample." path-from-xml="dsz011f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465062/dsz011f2.tif" path-from-xml="dsz011f2.tif" target="_blank">Open in new tab</a><a section="140465062" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f2.png?Expires=1576190032&amp;Signature=wcfxTbhQk0H1c5eWf3-bHaG5RdITsvlkTC5IL9lxb~LLL2spiLhM1Tc9REyImxGVxtE9dEoUo4jAr9pRQDR57L77a-rKYE-faCVkXTvWCBQ7DMY6wX6nbO9ALT681EE6Wc1V9~-pbH-n0tHOjjZH8SVPk-S3MTKNnvKZj-SnJXBQIIaxgxl8ZZm~lhD9oMAwUEicU2jIPVJXiLV9FuhnQ6DjDCQGdidxYGcILBrKOyWtxcPxP-tZ~lA~gsYHmD7QQOt8R9W6jqBYGIeZFRdICYBEg3QLZO6Y1NxBCJAr-WU5G6vL~w8t99y7XVwspr7vv0o18Jl4dOXT5L1~FG~mMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465062&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>GC concordance and insert-size distribution of data from CP-AL. (A–F) The correlation of GC percentage between human reference genome and sequencing reads in Samples01 to 06, respectively. The detailed method is described in Materials and methods. Each point indicates a particular window with the GC percentage (%) determined by sequencing read (<em>X</em>-axis) and human reference genome (<em>Y</em>-axis). Dotted line shows the ideal 100% correlation. Pearson correlation coefficient with <em>P</em>-value is shown in the left side of each figure. (G) The size distributions of four samples prepared with BLBEC and six prepared with CP-AL, respectively. Each sample was non-size-selected and sequenced with paired-end 50 bp reads. The <em>X</em>-axis shows the insert-size, and <em>Y</em>-axis indicates the percentage of the paired-end reads with certain insert-sizes. The figure shown in the left side with red dotted frame is the zoom in view. (H) Physical coverage distribution for the data from different library construction methods (BLBEC and CP-AL) for 50 million read-pairs per sample.</p></div>
</div>
<div content-id="dsz011-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f2.png?Expires=1576190032&amp;Signature=oKDOmHpLVChpKWBUVS-PO5JQaFBI7t6ZfGNa0FKRz8GN1eevOPsZo2iqJ4e10epCO5q5aYUo~n1guHBtMs84a2ye09WfFIZMoYpy-LYc9nmuhlSfy9sDRfD7OL2dOkx8SWbax2v5IYux9moKHbsjW2GGg-G~9Ou-XiefqejGv82MNIJMBhlc0d99izK6lg40y65OymLbc9kEeX1jIjOtgi4ApxpC5pEnS~3xdUfFiuJUDU-zZsTryCXnm9xGTetSRWm2JWRb0CdEsKrD9oWT8coqEW173NAoKpu6~RH~SeCwACAOzGL5~~65mJO-mLlG4ENdfDSB98BqNLXFbWvrZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC concordance and insert-size distribution of data from CP-AL. (A–F) The correlation of GC percentage between human reference genome and sequencing reads in Samples01 to 06, respectively. The detailed method is described in Materials and methods. Each point indicates a particular window with the GC percentage (%) determined by sequencing read (X-axis) and human reference genome (Y-axis). Dotted line shows the ideal 100% correlation. Pearson correlation coefficient with P-value is shown in the left side of each figure. (G) The size distributions of four samples prepared with BLBEC and six prepared with CP-AL, respectively. Each sample was non-size-selected and sequenced with paired-end 50 bp reads. The X-axis shows the insert-size, and Y-axis indicates the percentage of the paired-end reads with certain insert-sizes. The figure shown in the left side with red dotted frame is the zoom in view. (H) Physical coverage distribution for the data from different library construction methods (BLBEC and CP-AL) for 50 million read-pairs per sample." path-from-xml="dsz011f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465062/dsz011f2.tif" path-from-xml="dsz011f2.tif" target="_blank">Open in new tab</a><a section="140465062" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f2.png?Expires=1576190032&amp;Signature=wcfxTbhQk0H1c5eWf3-bHaG5RdITsvlkTC5IL9lxb~LLL2spiLhM1Tc9REyImxGVxtE9dEoUo4jAr9pRQDR57L77a-rKYE-faCVkXTvWCBQ7DMY6wX6nbO9ALT681EE6Wc1V9~-pbH-n0tHOjjZH8SVPk-S3MTKNnvKZj-SnJXBQIIaxgxl8ZZm~lhD9oMAwUEicU2jIPVJXiLV9FuhnQ6DjDCQGdidxYGcILBrKOyWtxcPxP-tZ~lA~gsYHmD7QQOt8R9W6jqBYGIeZFRdICYBEg3QLZO6Y1NxBCJAr-WU5G6vL~w8t99y7XVwspr7vv0o18Jl4dOXT5L1~FG~mMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465062&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>GC concordance and insert-size distribution of data from CP-AL. (A–F) The correlation of GC percentage between human reference genome and sequencing reads in Samples01 to 06, respectively. The detailed method is described in Materials and methods. Each point indicates a particular window with the GC percentage (%) determined by sequencing read (<em>X</em>-axis) and human reference genome (<em>Y</em>-axis). Dotted line shows the ideal 100% correlation. Pearson correlation coefficient with <em>P</em>-value is shown in the left side of each figure. (G) The size distributions of four samples prepared with BLBEC and six prepared with CP-AL, respectively. Each sample was non-size-selected and sequenced with paired-end 50 bp reads. The <em>X</em>-axis shows the insert-size, and <em>Y</em>-axis indicates the percentage of the paired-end reads with certain insert-sizes. The figure shown in the left side with red dotted frame is the zoom in view. (H) Physical coverage distribution for the data from different library construction methods (BLBEC and CP-AL) for 50 million read-pairs per sample.</p></div>
</div>
<p>CNV analysis was performed by our published method,<a reveal-id="dsz011-B29" data-open="dsz011-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> while the SVs identification was accomplished for each sample based on a four-step procedure: event clustering, systematic error filtering, random error filtering and aligned orientations.<a reveal-id="dsz011-B13" data-open="dsz011-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Reference sequences (1 kb) from both upstream and downstream of each breakpoint were used for primer design with Primer3 and Primer-Blast (Biotechnology Information). PCR was performed in the case and a well-established normal control (YH).<a reveal-id="dsz011-B13" data-open="dsz011-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsz011-B30" data-open="dsz011-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The products were subjected to Sanger sequencing on an ABI 3730 machine (Applied Biosystems, Wilmington, DE, USA), and the results were aligned with BLAT for confirmation. Regulatory analyses such as topological associated domains and distance normalized interaction frequency were performed by the web-based 3D-genome Interaction Viewer and Database (<a class="link link-uri" href="http://www.kobic.kr/3div/" target="_blank">http://www.kobic.kr/3div/</a>).<a reveal-id="dsz011-B31" data-open="dsz011-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> For the data from POCs, SNVs identification was carried out with Genome Analysis Toolkit HaplotypeCaller (version 3.4), and calculation of allelic ratio in chromosome-wide and genome-wide was described in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>.</p>                    <h2 scrollto-destination="140465064" id="140465064" class="section-title">3. Results</h2>
                    <h3 scrollto-destination="140465065" id="140465065" class="section-title">3.1. Establishment of a new mate-pair library method: CP-AL</h3>
<p>For clinical labs to conduct large-scale tests, an inexpensive and scalable mate-pair sequencing process is crucial. Herein, we described a new mate-pair library construction workflow through coupling two controlled polymerizations by 3′BL, which was demonstrated with higher process efficiency, lower cost and better uniformity of genome/exome coverage in following study. In the new schema (<a reveal-id="dsz011-F1" data-open="dsz011-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, steps with estimated turn-around-time were described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), gDNA was first fragmented into the size of interest, ligated to adapter-1 (Ad1, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and circularized to form double-stranded circles (dsCirs). To improve the efficiency of DNA circle formation, 14 bp overhangs were used. It resulted in median 12% circularization efficiency in 16 replicates compared with ∼3% from 12 replicates from BLBEC protocol<a reveal-id="dsz011-B32" data-open="dsz011-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), a 4-fold efficiency (Wilcoxon rank-sum test: <em>P</em> &lt; 0.0001),</p>
<p>The Ad1 dsCirs contained a gap that acted as the initiation site for a controlled polymerization CNT. Subsequently, the 3′-end of adapter-2 (Ad2, Ad2_3′) was directly added to the 3′OH termini of duplex DNA containing gaps using 3′BL.<a reveal-id="dsz011-B23" data-open="dsz011-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> After ligation, another controlled polymerization CPE was performed on the single-stranded DNA (ssDNA) template. This enzymatic target enrichment strategy can efficiently eliminate the ‘inward’ read-pairs that occur with the biotin capture method (<a reveal-id="dsz011-T1" data-open="dsz011-T1" class="link link-reveal link-table xref-fig">Table 1</a>). In CPE, a primer was hybridized to Ad2_3′ and extended onto the genomic template on the other side of Ad1. The 5′-end of Ad2 (Ad2_5′) was then added through another 3′BL to the newly synthesized CPE template (<a reveal-id="dsz011-F1" data-open="dsz011-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Each template of the mate-pair has a selected length (an example of a minimal length of 150 bp shown in Supplementary Fig. S6), resulting from CNT and CPE reactions, respectively and is separated by the Ad1 sequence with Ad2_5′ and 3′ sequences at either end. The ligation products were amplified using Ad2_5′ and Ad2_3′ primers resulting in a linear dsDNA library or a ssCir DNA library that can be sequenced by various technologies such as sequencing by synthesis on the Illumina,<a reveal-id="dsz011-B33" data-open="dsz011-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Ion Torrent<a reveal-id="dsz011-B34" data-open="dsz011-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> or BGI′s cPAS<a reveal-id="dsz011-B35" data-open="dsz011-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> platforms or sequencing by ligation using the SOLiD<a reveal-id="dsz011-B36" data-open="dsz011-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> or first-generation CGI<a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> platforms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>).</p>                    <h3 scrollto-destination="140465068" id="140465068" class="section-title">3.2. Feasibility testing of CP techniques for CP-AL schema</h3>
<p>As shown previously, polymerization speed strongly depends on polymerase Km and concentration, incubation time and temperature.<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Here we tested another hypothesis of controlling DNA extension by limiting nucleotide amount. We first used dsCirs with two gaps flanking the adapter on opposite strands as templates to assay nick translation extension (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>). Three different amounts of dNTPs (2.6, 4.0 and 8.0 pmol) were supplied in the PolI-mediated CNT reactions, which resulted in different observed extension lengths (24, 30 and 62 bp) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>), suggesting that nick translation can be controlled by nucleotide quantity. We calculated a predicted polymerization length per arm based on the ratio of total dNTPs to the number of naCNT initiation sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3C</a></span>). For PolI-mediated reactions, observed extension lengths (24, 30 and 62 bp) were close to the predicted lengths (23, 35 and 70 bp; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span> and C). It appears that PolI can utilize most, if not all, of the dNTPs in the nucleotide amount CNT (naCNT) reactions. Based on this observation, if we normalize the dsCir quantity to 1 pmol in the CP-AL schema, 100 pmol of total dNTPs will be needed in naCNT for an extension of ∼100 bp. We also found that naCNT products contain gaps of a few bp, since the 3′ exonuclease activity of PolI can cleave a few nucleotides from the nicks in a 5′ to 3′ direction (data not shown).</p>
<p>To replace the complicated biotin capture method for targets enrichment, we tried to use CPE to recover products of interest. We hypothesize that, as for CNT reactions, there are two ways to control primer extension length—by limiting the nucleotide amount (naCPE) or by managing the reaction time and temperature (ttCPE). We first assessed whether limiting dNTPs could control DNA extension length (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The naCPE template was 0.16 pmol of a 700∼3,000 bp gDNA fraction with 3′-ends ligated to an adapter sequence. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S5B</a></span> demonstrates that different naCPE reactions generated different sizes of primer-extended products (Lanes 1–5 in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), and the size correlates with the dNTP amount. Excess dNTPs (5,000 pmol, Lane 6) generated products of the original size range (700∼3,000 bp). Reactions performed with the lowest quantity of dNTPs (14.3 pmol, Lane 5 in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>) generating the smallest products (210∼300 bp). For ttCPE, the same amounts of CP-AL intermediate products after naCNT and 3′BL were extended for different times in the presence of excess dNTPs. An extension of 300∼450 bp was observed after a combination of 56°C for one min and 60°C for 40 s with <em>Pfu</em> Turbo Cx. After CPE, a partially duplex DNA with a 3′ recessive end is produced. 3′BL can efficiently add adapters to gapped or 3′-recessed duplex DNA.<a reveal-id="dsz011-B23" data-open="dsz011-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Therefore, both naCNT and CPE products are perfect templates for 3′BL.</p>                    <h3 scrollto-destination="140465071" id="140465071" class="section-title">3.3. Comparison of different parameters in CP-AL and validation</h3>
<p>We next compared different combinations of CP conditions and assayed the sequencing quality of the CP-AL libraries using a well-characterized sample NA19240, variants from which were reported.<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsz011-B26" data-open="dsz011-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> To minimize the GC bias in naCNT, two approaches were used (<a reveal-id="dsz011-T2" data-open="dsz011-T2" class="link link-reveal link-table xref-fig">Table 2</a>). First, more than one polymerase, such as a combination of PolI with <em>Taq</em> Pol, was included to process through most, if not all, polymerase-specific pausing sites to minimize bias. Second, dATP and dTTP were provided in 2-fold excess relative to dCTP and dGTP (3xAT in total), given a higher percentage of A/T compared with C/G in the human reference genome.</p>
<p>Four libraries with different CP conditions were constructed and sequenced at a read-depth of 50∼60-fold (<a reveal-id="dsz011-T2" data-open="dsz011-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Library #1 prepared with equal amounts of four dNTPs during naCNT demonstrated the highest fraction (6.6%) of autosomal exome at &lt;60% of the mean coverage, the lowest percentage (95.0%) of genome coverage with both alleles detected and the lowest sensitivity of SNV (95.2%) and InDel (80.7%) identification (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>)<a reveal-id="dsz011-B24" data-open="dsz011-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> compared with the other three libraries. It indicated that additional A/T provided during naCNT could improve the coverage uniformity. Compared with the data from small-insert libraries, with additional A/T provided, it resulted in detecting SNVs with sensitivity and specificity up to 97.3 and 99.7%, respectively, while it provided the sensitivity and specificity at 84.6 and 90.6%, respectively in InDels detection (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). No significant difference of genome/exome coverage was observed between Libraries #3 and #4, indicating no particular effect of using a combination of polymerases in naCNT. In addition, similar percentages of genome/exome coverage were observed for Library #2 (ttCPE) and Libraries #3 and #4 (naCPE), suggesting that ttCPE and naCPE were interchangeable. However, as it is difficult to precisely quantify the branched templates after naCPE and 3′BL, ttCPE was adopted. Taken together, we incorporated naCNT with 3xAT and ttCPE into CP-AL.</p>                    <h3 scrollto-destination="140465074" id="140465074" class="section-title">3.4. Validation of CP-AL for detection of CNVs and SVs</h3>
<p>One important application of mate-pair sequencing is CNVs and SVs detection. In this study, first, six samples (five with insertion translocations and one with normal karyotype, <a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>) were used for replicated CP-AL (1 µg input for fragmentation with end-products ranging from 3 to 8 kb) and sequenced for a minimum of 70 million read-pairs in 100 and 50 bp, respectively. Among these six subjects, Sample02 is a male subject diagnosed with oligo-atheno-terato-spermia, while the other five subjects have a history of recurrent miscarriages or foetal anomalies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>). Consistent with the performance during optimization stage as described above, GC bias was minimal indicated by the significantly high concordance (<em>r</em> &gt; 0.98) between the GC percentage in each window calculated by human reference genome and that of the aligned reads (<a reveal-id="dsz011-F2" data-open="dsz011-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>).</p>
<p>To evaluate the performance of CP-AL compared with the data generated from the most similar conditions (such as non-size-selection and 3∼8 kb initial fragment size), WGS data from four samples, which were with non-size-selected BLBEC libraries and sequenced for ∼50 million read-pairs (50 bp) on HiSeq 2000 platform<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> were used for comparison. First, we compared the data utility when with the same number of read-pairs (<em>N</em> = 50 million). Approximately 91.5% of the ‘inward’ read-pairs in the data with BLBEC were aligned to a distance &lt;1 kb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), while, based on the schema design, CP-AL would not generate such read-pairs (with inconsistent aligned orientations, <a reveal-id="dsz011-F1" data-open="dsz011-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Here, we compared the percentages of read-pairs with an aligned distance &lt;1 kb. The results showed that CP-AL produced only ∼0.7% of read-pairs with an aligned distance &lt;1 kb, compared with the ∼27.5% observed with BLBEC and ∼24.4% with Nextera (<a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>, <a reveal-id="dsz011-F2" data-open="dsz011-F2" class="link link-reveal link-table xref-fig">Fig. 2G</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Approximately 33% lower physical coverage was observed using BLBEC compared with CP-AL (<a reveal-id="dsz011-F2" data-open="dsz011-F2" class="link link-reveal link-table xref-fig">Fig. 2H</a>), indicating a much higher utility of sequencing data using CP-AL. Of note, although DNA input was reduced from 3 to 1 µg in CP-AL, a similar duplication rate was observed in samples with CP-AL compared with those with BLBEC (<a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The reduction of PCR duplication might be owing to the increased circularization efficiency, providing more DNA templates for further experiment. In addition, to evaluate performance using shorter reads such as the data generated using <em>Eco</em>P15I, the read-pairs from each sample were trimmed into 26 bp in length and subjected to the analysis under the same parameters. Although no significant difference of the fraction of genome covered physically was seen between the two methods when read-depth as 50 bp, the coverage was 4% higher with CP-AL compared with BLBEC when with the 26 bp reads (<a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>); this increased covered fraction can be attributed primarily to the higher data utility obtained from CP-AL.</p>
<p>Next we evaluated the ability of CP-AL in SVs detection with different read-lengths (100, 50 and 26 bp). Compared with the results identified with 100 bp sequencing, which fine-mapped a total number of 26 rearrangements in these five insertion translocations, the diagnostic yields of data with 50 and 26 bp (trimmed from 50 bp) were only 92.3% (24/26) and 84.6% (22/26), respectively (<a reveal-id="dsz011-F3" data-open="dsz011-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). It indicates that the longer read-length attained could enable a more comprehensive detection of structural rearrangements. Of note, in these five insertions the incidence of complex insertion translocations was 80.0% (4/5) (<a reveal-id="dsz011-F3" data-open="dsz011-F3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a>, <a reveal-id="dsz011-T3" data-open="dsz011-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Further, the importance of fine-mapping breakpoints can be evidenced by the potential genetic diagnosis of the patients. In Sample02, data from both 50 and 26 bp failed to identify the rearrangement in chromosome 6q13 between coordinate 74,138,669 and 74,343,934, the latter breakpoint is disrupting gene <em>SLC17A5</em> (<a reveal-id="dsz011-F4" data-open="dsz011-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Although compound heterozygous mutations have been found in <em>SLC17A5</em> causing Salla disease (OMIM: #604322), no direct evidence was proved for the contribution of its heterozygous loss to spermatogenesis. However, gene <em>EEF1A1</em>, whose super enhancer was found to be highly correlated with <em>SLC17A5</em> (both genes located in the same topological domain, <a reveal-id="dsz011-F4" data-open="dsz011-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>),<a reveal-id="dsz011-B31" data-open="dsz011-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> was likely haploinsufficient and might be essential for spermatogenesis.<a reveal-id="dsz011-B37" data-open="dsz011-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> It indicated that dysregulation of <em>EEF1A1</em> caused by disruption of <em>SLC17A5</em> would be the potential pathogenicity. In addition, it is also supported by the observation of disrupting 33.3% (5/15) cross-cell-type correlations between distal DNA I Hypersensitive sites and promoters of <em>EEF1A1</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>).<a reveal-id="dsz011-B38" data-open="dsz011-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> However, no sample type such as RNA is available from this subject for further investigation. </p>                        <a id="140465078" scrollto-destination="140465078"></a>
<div id="dsz011-F3" data-id="dsz011-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f3.png?Expires=1576190032&amp;Signature=iWkKUfJHErzdfH2w8GZK9yXb3RClkBXtPd2r7Y5ydMULl2B5ZYB1k~YZq4QCSxOpxtfP6wPkOYsZ~IYFIrjjqvikR4Qg-i4UQCXaTe7ViWJsqy37raGzFcnEC6qRMzd5gPTEIkKuSeNUxUfIEV8klFvHHSK~mfDNxIQsLEdcJJroqO3hPr1WPO7r9dvC3Qk1aO~qVSuvpwBG8~u-QuqwR1BTUwe9NHsPcHH1Y61te7HyKYXdGHEodMIpMXha2LjyBLS6dsg4AQK17DR3pj0DYNODVQ4bTFPtbADNwzL4ob90jXNTwJCkMEb~W7Y4~yIN1c~tkR1eIBq87nyVrH3mIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of chromosomal structural rearrangements. (A) Spectrum of chromosomal structural rearrangements. Each sample is indicated with the linkages with one particular colour and the event stated next to the linkages. All events are illustrated by the data from CP-AL with paired-end 100 bp sequencing and validated by Sanger sequencing. The events missed by the data from the same sample using shorter read-lengths are indicated by the red arrows (26 bp) and blue arrows (50 bp, outmost), respectively. Chromosomal nucleotide positions and bands are shown according to the University of California, Santa Cruz Genome Viewer Table Browser. The compositions of original chromosomes and the derivation chromosomes illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample03 and Sample05 are shown in figure (B) and (C), respectively. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation." path-from-xml="dsz011f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465078/dsz011f3.tif" path-from-xml="dsz011f3.tif" target="_blank">Open in new tab</a><a section="140465078" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f3.png?Expires=1576190032&amp;Signature=mGYR4OZqa78fn~lx~5Wg0X2BSrtR87Cc6srPsu7Ov8qndLXzaGz7JT2EmOuVaqoZx1EPVeYxXsfO2WW~BUa7wo7ncRpIANIDsYncJ7qTOWdpurGQxK09iMICXF1Ss2MTP8bqjTMmkTaEwTX2K1h6bVxUcDhPdE8E4~dvvNj7jwLgjw6A1eSXr4QO7nI7A-ROtaQ45XkxmTMW0OB8a9w~UIr0jsr2TzTpBsK1lu6otR26VmqCTa4MD-eGW7ZeCuom4uqP6BlJVN55J2Wg2gJo9WuRnvPmzxBxV2Hwp6kQXIbSeEvgluJKMz6L8C6Sa5btmaVeXrTkzpyS6q7dw9KYBQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465078&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detection of chromosomal structural rearrangements. (A) Spectrum of chromosomal structural rearrangements. Each sample is indicated with the linkages with one particular colour and the event stated next to the linkages. All events are illustrated by the data from CP-AL with paired-end 100 bp sequencing and validated by Sanger sequencing. The events missed by the data from the same sample using shorter read-lengths are indicated by the red arrows (26 bp) and blue arrows (50 bp, outmost), respectively. Chromosomal nucleotide positions and bands are shown according to the University of California, Santa Cruz Genome Viewer Table Browser. The compositions of original chromosomes and the derivation chromosomes illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample03 and Sample05 are shown in figure (B) and (C), respectively. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation.</p></div>
</div>
<div content-id="dsz011-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f3.png?Expires=1576190032&amp;Signature=iWkKUfJHErzdfH2w8GZK9yXb3RClkBXtPd2r7Y5ydMULl2B5ZYB1k~YZq4QCSxOpxtfP6wPkOYsZ~IYFIrjjqvikR4Qg-i4UQCXaTe7ViWJsqy37raGzFcnEC6qRMzd5gPTEIkKuSeNUxUfIEV8klFvHHSK~mfDNxIQsLEdcJJroqO3hPr1WPO7r9dvC3Qk1aO~qVSuvpwBG8~u-QuqwR1BTUwe9NHsPcHH1Y61te7HyKYXdGHEodMIpMXha2LjyBLS6dsg4AQK17DR3pj0DYNODVQ4bTFPtbADNwzL4ob90jXNTwJCkMEb~W7Y4~yIN1c~tkR1eIBq87nyVrH3mIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of chromosomal structural rearrangements. (A) Spectrum of chromosomal structural rearrangements. Each sample is indicated with the linkages with one particular colour and the event stated next to the linkages. All events are illustrated by the data from CP-AL with paired-end 100 bp sequencing and validated by Sanger sequencing. The events missed by the data from the same sample using shorter read-lengths are indicated by the red arrows (26 bp) and blue arrows (50 bp, outmost), respectively. Chromosomal nucleotide positions and bands are shown according to the University of California, Santa Cruz Genome Viewer Table Browser. The compositions of original chromosomes and the derivation chromosomes illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample03 and Sample05 are shown in figure (B) and (C), respectively. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation." path-from-xml="dsz011f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465078/dsz011f3.tif" path-from-xml="dsz011f3.tif" target="_blank">Open in new tab</a><a section="140465078" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f3.png?Expires=1576190032&amp;Signature=mGYR4OZqa78fn~lx~5Wg0X2BSrtR87Cc6srPsu7Ov8qndLXzaGz7JT2EmOuVaqoZx1EPVeYxXsfO2WW~BUa7wo7ncRpIANIDsYncJ7qTOWdpurGQxK09iMICXF1Ss2MTP8bqjTMmkTaEwTX2K1h6bVxUcDhPdE8E4~dvvNj7jwLgjw6A1eSXr4QO7nI7A-ROtaQ45XkxmTMW0OB8a9w~UIr0jsr2TzTpBsK1lu6otR26VmqCTa4MD-eGW7ZeCuom4uqP6BlJVN55J2Wg2gJo9WuRnvPmzxBxV2Hwp6kQXIbSeEvgluJKMz6L8C6Sa5btmaVeXrTkzpyS6q7dw9KYBQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465078&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detection of chromosomal structural rearrangements. (A) Spectrum of chromosomal structural rearrangements. Each sample is indicated with the linkages with one particular colour and the event stated next to the linkages. All events are illustrated by the data from CP-AL with paired-end 100 bp sequencing and validated by Sanger sequencing. The events missed by the data from the same sample using shorter read-lengths are indicated by the red arrows (26 bp) and blue arrows (50 bp, outmost), respectively. Chromosomal nucleotide positions and bands are shown according to the University of California, Santa Cruz Genome Viewer Table Browser. The compositions of original chromosomes and the derivation chromosomes illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample03 and Sample05 are shown in figure (B) and (C), respectively. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation.</p></div>
</div>                        <a id="140465079" scrollto-destination="140465079"></a>
<div id="dsz011-F4" data-id="dsz011-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f4.png?Expires=1576190032&amp;Signature=T5zIpUHh26WbqpSgkIp-NEMSb1m~ngobRwsbxqr8535RcPFJZBdpDvvbgqySOEEy5hKZWndtDdCKWZodyez0~HHuYAT1T5ARK9sqx3gMJj8WHCrWgZkHumImnMsYB9FxPBsuf~UktrbAai4-sxyj9cKex59TE5wXpxUDuYm~LKUbD6yN2HAKl7zGoPhQrP3RU2CUW-fpzZXVTkLWka68Qvw3TyIBwaRusXYPi1QFyEV-BDyoh3BzODxI4oH~jsaD6zARm1PYLckng~ibtvhoROfy5C4tEhd8NTXiUt6LbRyA~C-Vac-ZI7DZ2djUy5PTeDSqPgCDOiJ8QFtkgpWABA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of structural rearrangements in Sample02 a male subject with oligo-atheno-terato-spermia. (A) The compositions of normal chromosomes (shown in green dotted frame) and the derivation chromosomes (shown in red dotted frame) illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample02. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation. (B) Visualization (http://www.kobic.kr/3div/) of interaction between gene SLC17A5 and the other locations in PrEC normal Prostate epithelial cell, BglII, the most relevant cell line in the reported database. Bi. Distribution of topological associated domains (triangles in blue); Bii. Distributions of distance normalized interaction frequency and bias-removed interaction frequency; Biii. Present of super enhancers; Biv. Distribution of genes (RefSeq). The disruption location is indicated by red line (SLC17A5 indicated by a red arrow), while the locations of the certain topological associated domain (same as SLC17A5) are shown in orange dotted lines. Black arrow and green indicate the super enhancer of gene EEF1A1 and the gene itself, respectively." path-from-xml="dsz011f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465079/dsz011f4.tif" path-from-xml="dsz011f4.tif" target="_blank">Open in new tab</a><a section="140465079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f4.png?Expires=1576190032&amp;Signature=lym6aqFESVJ2cGEobPub3VyoU4yx~9yRsBBRyx0g3dFricGLnYSVTag3rEU5bb1zU86SvsuaUJw5U1B781ysQctsqH~a0z7tbmBX5vhg6SyY4kM3VFD1O7bsH7I5nJEvL-SnAUr7Y3O3ZGFsGyQN3-mMG4ae23Yt4MrrmVBkTdjiqNEIF4w7my4Ul70QBnSgVb-AP0kA81lDgrKctCPDJMz0ybftWxnHlWxPkOl69HpeTrh5JvryPZWR5jnnzjF5hkoDWrgSVfnlQxBAC09k9KDEacntOpAlPpnY7OaKv9Vwtk0Bg~5QCce7Pj0a0AP3KN7f~3ba4aUhMS4Lt1Fshw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465079&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detection of structural rearrangements in Sample02 a male subject with oligo-atheno-terato-spermia. (A) The compositions of normal chromosomes (shown in green dotted frame) and the derivation chromosomes (shown in red dotted frame) illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample02. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation. (B) Visualization (<a class="link link-uri" href="http://www.kobic.kr/3div/" target="_blank">http://www.kobic.kr/3div/</a>) of interaction between gene <em>SLC17A5</em> and the other locations in PrEC normal Prostate epithelial cell, BglII, the most relevant cell line in the reported database. Bi. Distribution of topological associated domains (triangles in blue); Bii. Distributions of distance normalized interaction frequency and bias-removed interaction frequency; Biii. Present of super enhancers; Biv. Distribution of genes (RefSeq). The disruption location is indicated by red line (<em>SLC17A5</em> indicated by a red arrow), while the locations of the certain topological associated domain (same as <em>SLC17A5</em>) are shown in orange dotted lines. Black arrow and green indicate the super enhancer of gene <em>EEF1A1</em> and the gene itself, respectively.</p></div>
</div>
<div content-id="dsz011-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f4.png?Expires=1576190032&amp;Signature=T5zIpUHh26WbqpSgkIp-NEMSb1m~ngobRwsbxqr8535RcPFJZBdpDvvbgqySOEEy5hKZWndtDdCKWZodyez0~HHuYAT1T5ARK9sqx3gMJj8WHCrWgZkHumImnMsYB9FxPBsuf~UktrbAai4-sxyj9cKex59TE5wXpxUDuYm~LKUbD6yN2HAKl7zGoPhQrP3RU2CUW-fpzZXVTkLWka68Qvw3TyIBwaRusXYPi1QFyEV-BDyoh3BzODxI4oH~jsaD6zARm1PYLckng~ibtvhoROfy5C4tEhd8NTXiUt6LbRyA~C-Vac-ZI7DZ2djUy5PTeDSqPgCDOiJ8QFtkgpWABA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of structural rearrangements in Sample02 a male subject with oligo-atheno-terato-spermia. (A) The compositions of normal chromosomes (shown in green dotted frame) and the derivation chromosomes (shown in red dotted frame) illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample02. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation. (B) Visualization (http://www.kobic.kr/3div/) of interaction between gene SLC17A5 and the other locations in PrEC normal Prostate epithelial cell, BglII, the most relevant cell line in the reported database. Bi. Distribution of topological associated domains (triangles in blue); Bii. Distributions of distance normalized interaction frequency and bias-removed interaction frequency; Biii. Present of super enhancers; Biv. Distribution of genes (RefSeq). The disruption location is indicated by red line (SLC17A5 indicated by a red arrow), while the locations of the certain topological associated domain (same as SLC17A5) are shown in orange dotted lines. Black arrow and green indicate the super enhancer of gene EEF1A1 and the gene itself, respectively." path-from-xml="dsz011f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465079/dsz011f4.tif" path-from-xml="dsz011f4.tif" target="_blank">Open in new tab</a><a section="140465079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f4.png?Expires=1576190032&amp;Signature=lym6aqFESVJ2cGEobPub3VyoU4yx~9yRsBBRyx0g3dFricGLnYSVTag3rEU5bb1zU86SvsuaUJw5U1B781ysQctsqH~a0z7tbmBX5vhg6SyY4kM3VFD1O7bsH7I5nJEvL-SnAUr7Y3O3ZGFsGyQN3-mMG4ae23Yt4MrrmVBkTdjiqNEIF4w7my4Ul70QBnSgVb-AP0kA81lDgrKctCPDJMz0ybftWxnHlWxPkOl69HpeTrh5JvryPZWR5jnnzjF5hkoDWrgSVfnlQxBAC09k9KDEacntOpAlPpnY7OaKv9Vwtk0Bg~5QCce7Pj0a0AP3KN7f~3ba4aUhMS4Lt1Fshw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465079&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detection of structural rearrangements in Sample02 a male subject with oligo-atheno-terato-spermia. (A) The compositions of normal chromosomes (shown in green dotted frame) and the derivation chromosomes (shown in red dotted frame) illustrated by the sequencing data from CP-AL with paired-end 100 bp reads in Sample02. Each DNA/chromosome segment is shown with a different colour and an arrow indicating the genomic orientation. (B) Visualization (<a class="link link-uri" href="http://www.kobic.kr/3div/" target="_blank">http://www.kobic.kr/3div/</a>) of interaction between gene <em>SLC17A5</em> and the other locations in PrEC normal Prostate epithelial cell, BglII, the most relevant cell line in the reported database. Bi. Distribution of topological associated domains (triangles in blue); Bii. Distributions of distance normalized interaction frequency and bias-removed interaction frequency; Biii. Present of super enhancers; Biv. Distribution of genes (RefSeq). The disruption location is indicated by red line (<em>SLC17A5</em> indicated by a red arrow), while the locations of the certain topological associated domain (same as <em>SLC17A5</em>) are shown in orange dotted lines. Black arrow and green indicate the super enhancer of gene <em>EEF1A1</em> and the gene itself, respectively.</p></div>
</div>
<p>Apart from the observation of improved diagnostic yield made by the analysis with longer read-length, another advantage is to provide more uniquely aligned read-pairs in supporting rearrangement event attributed to the increased specificity of alignment with longer read-length/sequence. This was evidenced by the observation of a 33.1% reduction (from 25.1 to 16.8) of the average number of supported read-pairs for each rearrangement event in the data with 26 bp trimmed from 50 bp.</p>
<p>To evaluate the performance of CNV detection by CP-AL, we compared the results with CMA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Consistent with our previous findings, CP-AL not only reported all 6 rare CNVs detected by CMA, but also additionally identified 11 rare CNVs (U-test <em>P</em> &lt; 0.0001)<a reveal-id="dsz011-B29" data-open="dsz011-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> including 9 deletions and 2 duplications. All of these additional CNVs were owing to the lack of probes within the targeted regions in CMA, but all of which were classified as benign or likely benign in public databases (such as ClinVar). Both assays defined a likely pathogenic deletion involving gene <em>ZIC1</em> in Sample05 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>), which was involved in the breakpoint of this complex rearrangement, indicated by the supporting read-pairs (<a reveal-id="dsz011-F3" data-open="dsz011-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). Overall, our study showed a high consistency of CNV detection between CP-AL and CMA in these six cases.</p>                    <h3 scrollto-destination="140465082" id="140465082" class="section-title">3.5. Establishment of a single assay for identification of various genomic variants</h3>
<p>To address the possibility of identifying SNV, CNVs and SVs in one single assay, we further carried out CP-AL with &gt;30-fold read-depth WGS for two POCs (<a reveal-id="dsz011-F5" data-open="dsz011-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). Detection of SNVs, CNVs and SVs were performed independently and integrated afterwards. In case Trisomy 2, the increased allelic ratio (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>) in chromosome 2 confirmed that there was one copy gain of whole chromosome 2 consistently with the result from CNV analysis, while SV analysis provides the information of the duplication/deletions supporting read-pairs (<a reveal-id="dsz011-F5" data-open="dsz011-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>) for narrow-downing the critical affected regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). Of note, the breakpoints of these deletions/duplications were predicted to locate in the regions fully with repetitive elements and segmental duplications (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>), which was unlikely detectable by standard WGS. The feasibility of detecting the same genomic variants by different analyses demonstrated an accurate detection, emphasizing the advantage of integration of all detection results. A similar finding was observed in case Trisomy 8 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>), demonstrating the robustness of this application. In addition, in case Trisomy 2, a 51.4 kb inversion was detected in chromosome 6q14.1, while a 187.9 kb inversion was observed in chromosome Xq28. Consistent with the findings of deletions/duplications, the breakpoints of both inversions were also predicted to locate in the regions densely populated with repetitive elements and segmental duplications (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). Overall, the analysis from these two samples emphasizes the advantages and superiorities of applying CP-AL with 30-fold WGS as an integrated method for potentially comprehensive study of genomic variants in human diseases, compared with standard WGS. </p>                        <a id="140465084" scrollto-destination="140465084"></a>
<div id="dsz011-F5" data-id="dsz011-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f5.png?Expires=1576190032&amp;Signature=XAgy9H~PpNeLLClZxZkRcGsHbmA338SWTuRi80DI~iI0LTHKD3FeRFDMSe58qr3Ifd4iln59-7NKFDEuRgYrSd1r1rIHk3Zf~goXA9P-KTyjm7CG68KwAlybGnM3zbSiNyLXbqkMN22ojsnZs2Q1H-Bj2LlLLSnGIi8~JI3JyAcbZf6Pyl-3~SwBVuxhI-FrpcBinF8BDuBuWvV29cdg1xZIQ1RFLEZoP4ZfwmPBKwdacXMclkpHda8LHRFTkhVOhKoIG424LKklyTMPbSlDUSx56-toPQQgJM4mW3amDKNdJffFHKse0m9w25674Ei7J19-IghfxW-~y7IsNXpm~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Whole-genome analysis of genomic variants in sample with Trisomy 2. The distributions of allelic ratio (window-size: 100 kb in log2 scale; Supplementary Material), copy-ratio and structural variants are shown from outer to inner circles accordingly. Allelic ratio of on average 4 is shown in chromosome 2 indicating there are two copies of one base type against only one copy of another base type. Karyotypical structures and cytogenetic band colours are shown according to the University of California, Santa Cruz Genome Viewer Table Browser and chromosome colour schemes (outmost circle). Rectangles in red and blue indicate copy-number losses and gains, respectively, while lines in red and blue also indicate copy-number losses and gains. Lines in black in SV analysis show two inversions detected in chromosome 6 and X, respectively." path-from-xml="dsz011f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465084/dsz011f5.tif" path-from-xml="dsz011f5.tif" target="_blank">Open in new tab</a><a section="140465084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f5.png?Expires=1576190032&amp;Signature=lxN54eRjMeUKAQXb8Ir1r-wDZLorQ-CLgnrNNgi4Erwp894roTw1EWE5sZ1D7-q1xzFNko19oFxdmdD0FFjqXpJ9sLcbha5wsqy9KR4BWZrkp8j73EyoRAYUBIHVfeJXytC7HRiyUl5KPHKWdLUNC5R90tCzayxtNNpyMfrqsdSUcG5dbga0gFKu6rxj8UOnZd0kIG4vQ~ds5DQ-iuYgoVEqD34vMOlbkPLmiFYqFns0HSGP9n79jyjd5skohYrvrq8JNYVHNTbWJacnbpcfh9T8iDPrw3WTMmhB49HIHmHs5sKVW9A14XnX86ZSXiJKPwAjvtb6H8b1rwh9OOT3Iw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465084&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Whole-genome analysis of genomic variants in sample with Trisomy 2. The distributions of allelic ratio (window-size: 100 kb in log2 scale; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>), copy-ratio and structural variants are shown from outer to inner circles accordingly. Allelic ratio of on average 4 is shown in chromosome 2 indicating there are two copies of one base type against only one copy of another base type. Karyotypical structures and cytogenetic band colours are shown according to the University of California, Santa Cruz Genome Viewer Table Browser and chromosome colour schemes (outmost circle). Rectangles in red and blue indicate copy-number losses and gains, respectively, while lines in red and blue also indicate copy-number losses and gains. Lines in black in SV analysis show two inversions detected in chromosome 6 and X, respectively.</p></div>
</div>
<div content-id="dsz011-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/m_dsz011f5.png?Expires=1576190032&amp;Signature=XAgy9H~PpNeLLClZxZkRcGsHbmA338SWTuRi80DI~iI0LTHKD3FeRFDMSe58qr3Ifd4iln59-7NKFDEuRgYrSd1r1rIHk3Zf~goXA9P-KTyjm7CG68KwAlybGnM3zbSiNyLXbqkMN22ojsnZs2Q1H-Bj2LlLLSnGIi8~JI3JyAcbZf6Pyl-3~SwBVuxhI-FrpcBinF8BDuBuWvV29cdg1xZIQ1RFLEZoP4ZfwmPBKwdacXMclkpHda8LHRFTkhVOhKoIG424LKklyTMPbSlDUSx56-toPQQgJM4mW3amDKNdJffFHKse0m9w25674Ei7J19-IghfxW-~y7IsNXpm~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Whole-genome analysis of genomic variants in sample with Trisomy 2. The distributions of allelic ratio (window-size: 100 kb in log2 scale; Supplementary Material), copy-ratio and structural variants are shown from outer to inner circles accordingly. Allelic ratio of on average 4 is shown in chromosome 2 indicating there are two copies of one base type against only one copy of another base type. Karyotypical structures and cytogenetic band colours are shown according to the University of California, Santa Cruz Genome Viewer Table Browser and chromosome colour schemes (outmost circle). Rectangles in red and blue indicate copy-number losses and gains, respectively, while lines in red and blue also indicate copy-number losses and gains. Lines in black in SV analysis show two inversions detected in chromosome 6 and X, respectively." path-from-xml="dsz011f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465084/dsz011f5.tif" path-from-xml="dsz011f5.tif" target="_blank">Open in new tab</a><a section="140465084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011f5.png?Expires=1576190032&amp;Signature=lxN54eRjMeUKAQXb8Ir1r-wDZLorQ-CLgnrNNgi4Erwp894roTw1EWE5sZ1D7-q1xzFNko19oFxdmdD0FFjqXpJ9sLcbha5wsqy9KR4BWZrkp8j73EyoRAYUBIHVfeJXytC7HRiyUl5KPHKWdLUNC5R90tCzayxtNNpyMfrqsdSUcG5dbga0gFKu6rxj8UOnZd0kIG4vQ~ds5DQ-iuYgoVEqD34vMOlbkPLmiFYqFns0HSGP9n79jyjd5skohYrvrq8JNYVHNTbWJacnbpcfh9T8iDPrw3WTMmhB49HIHmHs5sKVW9A14XnX86ZSXiJKPwAjvtb6H8b1rwh9OOT3Iw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465084&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz011f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Whole-genome analysis of genomic variants in sample with Trisomy 2. The distributions of allelic ratio (window-size: 100 kb in log2 scale; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>), copy-ratio and structural variants are shown from outer to inner circles accordingly. Allelic ratio of on average 4 is shown in chromosome 2 indicating there are two copies of one base type against only one copy of another base type. Karyotypical structures and cytogenetic band colours are shown according to the University of California, Santa Cruz Genome Viewer Table Browser and chromosome colour schemes (outmost circle). Rectangles in red and blue indicate copy-number losses and gains, respectively, while lines in red and blue also indicate copy-number losses and gains. Lines in black in SV analysis show two inversions detected in chromosome 6 and X, respectively.</p></div>
</div>                    <h2 scrollto-destination="140465085" id="140465085" class="section-title">4. Discussion</h2>
<p>Mate-pair sequencing provides a foundation for identifying various genomic variants, including SNVs, CNVs and SVs in one single assay, however, problems such as low data utility and high cost limit the clinical application of current methods (<a reveal-id="dsz011-T1" data-open="dsz011-T1" class="link link-reveal link-table xref-fig">Table 1</a>).<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> In this study, we developed a new mate-pair method CP-AL through coupling controlled polymerizations with a non-conventional adapter-ligation reaction recently developed in our previous study,<a reveal-id="dsz011-B23" data-open="dsz011-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and further demonstrated that its feasibility and advantages in detection and delineation of various genomic variants in human diseases.</p>
<p>In this study, based on the foundations provided by 3′BL reaction, we have demonstrated the effectiveness of controlling polymerizations during nick translation and primer extension and coupling these two reactions by 3′BL. Before building up the whole workflow, we demonstrated the feasibility of carrying out each key component, respectively: (i) conducting nick translation by adjusting the dNTP quantities (naCNT) instead of adjusting reaction temperature and duration time, the latter commonly results in a large size range of templates;<a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> (ii) performing primer extension by either adjusting the dNTP amount (naCPE) or adjusting reaction temperature and duration time (ttCPE). Further, during optimization, a more uniform genome/exome coverage was achieved by providing a 2-fold excess of dATPs/dTTPs in nick translation: the percentage of autosomal exome with &lt;60% of mean coverage was improved from 6.6 to 0.7%, whereas percentages of coding regions with both alleles detected in a well-characterized sample NA19240 were increased from 95.9 to 98.1% (<a reveal-id="dsz011-T2" data-open="dsz011-T2" class="link link-reveal link-table xref-fig">Table 2</a>).</p>
<p>Compared with the reported approaches,<a reveal-id="dsz011-B6" data-open="dsz011-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz011-B11" data-open="dsz011-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsz011-B39" data-open="dsz011-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsz011-B40" data-open="dsz011-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> CP-AL provides the following advantages: (i) an adapter with a 14 bp overhang is used increasing circularization efficiency (from ∼3 to ∼12% by 4-fold), thus decreasing the minimal DNA input amount; (ii) no gel-purification; (iii) biotin-capture-free fully eliminates the ‘inward-sequences’ (0.7%, or a 39.3-fold reduction from 27.5% for BLBEC; 24.4% for Nextera); (iv) secondary-fragmentation-free avoids the random presence of ‘adapter-contaminated’ (∼23% from Nextera) due to random fragmentation after DNA circularization; (v) sequencing with different read-lengths and different platforms demonstrates a more uniform genome/exome coverage attributed by minimal GC biases;<a reveal-id="dsz011-B41" data-open="dsz011-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and (vi) generation of a long read-length (100 bp or even longer) with the lowest chimeric rate provides the potential foundation for disease precision diagnosis and disease-causing gene discovery. It was evidenced by the observation of an 18.2% increased detection rate for SVs (4/22, four additional events detected from 22 detected with 26 bp), one of which potentially causes a positional effect for male infertility. In evaluation of the performances other than the consistency of variants detection,<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> it is important to have the same DNAs as input, however, no such material was available from the previous study for CP-AL experiments. Given the targeted performances were only related to the critical DNA fragment size instead of the certain mutations/variants pattern in a particular case and a consistent fragment size of the DNA input after fragmentation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>), our study is able to support the argument of CP-AL is superior to the existing mate-pair method in these related applications.</p>
<p>In addition, our data further suggests that good performance of SNV and InDel detection in CP-AL (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Together, CP-AL with WGS has been proved to be the most integrated assay in providing the detection results of point mutations (SNVs), deletions/duplications (CNVs), translocations, inversions and insertions (SVs). In addition, we further demonstrated its advantages by showing the higher detection accuracy attributed by detecting the same variants by different types of analysing methods (<a reveal-id="dsz011-F5" data-open="dsz011-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). For example, increased allelic ratio from SNV detection confirmed the copy-number gain while SV analysis provided the read-pairs supporting the deletions/duplications (<a reveal-id="dsz011-F5" data-open="dsz011-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). Since the breakpoints of those SVs identified by CP-AL are predicted to locate in the genomic regions densely populated with repetitive elements and segmental duplications, which are unlikely detectable by standard WGS, our study shows the potential of replacing standard WGS by CP-AL.</p>
<p>Apart from high read-depth application, we also show the feasibility of using CP-AL with low-pass (or low-coverage) WGS for CNV and SV detection application, given that an ∼50% increased physical coverage obtained from CP-AL. In this aim, only a limited number of read-pairs (50–70 million) is required in each sample; a pooling step is suitable.<a reveal-id="dsz011-B16" data-open="dsz011-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> In addition, due to the flexibility of sequencing orientation for ssDNA, pooling was performed soon after Ad1 PCR (<a reveal-id="dsz011-F1" data-open="dsz011-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). This change will reduce the labour cost for all subsequent procedures by treating the pooled library as a single sample. By modification of the adapter sequence, our study also shows the potential of wider applications in different sequencing platforms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). For the cost aspect, avoiding the usage of expensive reagents such as biotin and exonuclease (exonuclease digestion and/or streptavidin beads is costly and inefficient), the reagent cost of CP-AL is only a fraction of current cost, for instance: it is only approximately one-tenth of BLBEC (US $40 vs $350, <a reveal-id="dsz011-T1" data-open="dsz011-T1" class="link link-reveal link-table xref-fig">Table 1</a>).<a reveal-id="dsz011-B15" data-open="dsz011-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> It is important that reduction of mate-pair library cost can be achieved, given the era of 1,000 USD for standard WGS came, CP-AL would overcome the main challenges in detecting various genomic variants but with similar cost.</p>
<p>Moreover, the biologic interpretation of the genomic variants is important after identification. Five cases with insertion translocations known to result in abnormal embryos/pregnancies with high risk (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S15</a></span>) were studied in this study. CP-AL with 100 bp was able to detect and delineate all the rearrangements, providing a much higher incidence of complex insertion translocations as 80.0% (4/5) (<a reveal-id="dsz011-F3" data-open="dsz011-F3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a>) compared with the previous study.<a reveal-id="dsz011-B42" data-open="dsz011-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The importance of fine-mapping the related breakpoints was demonstrated in providing the potential foundation for disease precision diagnosis and disease-causing gene discovery, such as showing a potential positional effect in a patient with oligo-atheno-terato-spermia (Sample02, <a reveal-id="dsz011-F4" data-open="dsz011-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). In addition, in Sample05 with cryptic deletion (not detected by karyotyping) involving OMIM disease-causing gene <em>ZIC1</em> likely haploinsufficient (HI = 2) defined by ClinGen Genome Dosage Map. No clinical recognized malformation was identified in this patient. It might be due to incomplete penetrance, variable expressivity or the potential pathogenicity of this gene was contributed by gain of function instead,<a reveal-id="dsz011-B43" data-open="dsz011-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> warranting a comprehensive clinical assessment.</p>
<p>Although reduction of both labour work and reagent cost makes CP-AL more practical for routine practice, several limitations remain: (i) PCR amplification with primers incorporating dUTPs is used, which would be inefficient for DNA &gt;10 kb. Foreseeing that CP-AL with limited ability in identification of SVs mediated by low-copy repeat, application of third-generation sequencing platforms<a reveal-id="dsz011-B21" data-open="dsz011-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsz011-B22" data-open="dsz011-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> might be an alternative strategy; (ii) Read-length: given that improvement of variants detection and delineation were observed by longer read-length attributed to the improvement of alignment specificity (<a reveal-id="dsz011-F3" data-open="dsz011-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), it is valuable to provide longer read-depth (&gt;150 or 300 bp, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576190032&amp;Signature=qZLV1duj2Rdnq-8hr5iEDKl7GFxZASUWd5hxDt7qlqhwfCp6STNcSPYYil2UxaU-8hRr1ZugweDEBnhmjVW-P-nKwCl8C~TYv1tY0pm34ju90xc4salWc~kjbQeAKNSJczC6gmGdfsVDqxGCgWZR8sBvYnwb-YZU~u8sTaQLrOBnzwa-3Pa2khHS~1KF9~caPTIGdZrHxhpOrXelsgGbSnFazDbJZImUROQ7C5C6TpBXjpckxuI~6vE69r0xCfAU0hqebQuwAxvuZnMppgHJZl~jhuz7EZc7gj0dm6TK4917oY1htcBBvDtnuzEI9l~ujLP6qMsCwZvjjQQJ99bJFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>) for CP-AL; and (iii) DNA samples were fragmented physically (HydroShear); alternative methods such as fragmentation by Transposases (i.e. Tn5)<a reveal-id="dsz011-B44" data-open="dsz011-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> is warranted.</p>
<p>In conclusion, we provide a cost-effective and high-efficiency mate-pair method CP-AL that offers higher utility of read-pairs, better uniformity of genome/exome coverage at a fraction of current cost. With 30-fold read-depth, this approach shows promise as an integrated method and eligible first-tier WGS testing assay for detection and delineation of various genomic variants in human diseases.</p>                    <h2 scrollto-destination="140465094" id="140465094" class="section-title">Availability</h2>
<p>Whole-genome sequencing data used in this study have been made available in the CNGB Nucleotide Sequence Archive (CNSA: <a class="link link-uri" href="https://db.cngb.org/cnsa" target="_blank">https://db.cngb.org/cnsa</a>) under the accession number CNP0000078.</p>                    <h2 scrollto-destination="140465096" id="140465096" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>We would like to acknowledge the ongoing contributions and support of all Complete Genomics and BGI-Shenzhen employees, in particular the many highly skilled individuals that work in the libraries, reagents and sequencing groups that make it possible to generate high-quality whole-genome data. We thank Dr James Lupski, Dr Sau Wai Cheung, Dr Ya Gao and Dr Pengfei Liu for helpful discussions surrounding manuscript preparation. Z.D. thanks the support of the Dr Stanley Ho Medical Development Foundation.</p>                    <h3 scrollto-destination="140465098" id="140465098" class="section-title">Funding</h3>
<p>This work was supported in part by the National Natural Science Foundation of China (No. 31801042) and the Shenzhen Peacock Plan (No. KQTD20150330171505310).</p>                    <h3 scrollto-destination="140465100" id="140465100" class="section-title">Conflicts of interest</h3>
<p>Employees of BGI and Complete Genomics have stock holdings in BGI. The other authors declare no conflicts of interest.</p>                    <h2 scrollto-destination="140465102" id="140465102" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz011-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sudmant</div> <div class="given-names">P.H.</div>
</div>, <div class="name string-name">
<div class="surname">Rausch</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Gardner</div> <div class="given-names">E.J.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">An integrated map of structural variation in 2,504 human genomes</div>, <div class="source">Nature</div> , <div class="volume">526</div>, <div class="fpage">75</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20map%20of%20structural%20variation%20in%202%2C504%20human%20genomes&amp;author=P.H.%20Sudmant&amp;author=T.%20Rausch&amp;author=E.J.%20Gardner&amp;publication_year=2015&amp;journal=Nature&amp;volume=526&amp;pages=75-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature15394" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature15394" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature15394"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26432246" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20integrated%20map%20of%20structural%20variation%20in%202%2C504%20human%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Carvalho</div> <div class="given-names">C.M.</div>
</div>, <div class="name string-name">
<div class="surname">Lupski</div> <div class="given-names">J.R.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Mechanisms underlying structural variant formation in genomic disorders</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">17</div>, <div class="fpage">224</div>–<div class="lpage">38</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20underlying%20structural%20variant%20formation%20in%20genomic%20disorders&amp;author=C.M.%20Carvalho&amp;author=J.R.%20Lupski&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Genet&amp;volume=17&amp;pages=224-38" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg.2015.25" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2015.25" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg.2015.25"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26924765" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mechanisms%20underlying%20structural%20variant%20formation%20in%20genomic%20disorders&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Ming</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Xiao</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">TBX6 null variants and a common hypomorphic allele in congenital scoliosis</div>, <div class="source">N. Engl. J. Med</div> ., <div class="volume">372</div>, <div class="fpage">341</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=TBX6%20null%20variants%20and%20a%20common%20hypomorphic%20allele%20in%20congenital%20scoliosis&amp;author=N.%20Wu&amp;author=X.%20Ming&amp;author=J.%20Xiao&amp;publication_year=2015&amp;journal=N.%20Engl.%20J.%20Med&amp;volume=372&amp;pages=341-50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1056/NEJMoa1406829" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1406829" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1056%2FNEJMoa1406829"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25564734" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:TBX6%20null%20variants%20and%20a%20common%20hypomorphic%20allele%20in%20congenital%20scoliosis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leung</div> <div class="given-names">T.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Vogel</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Lau</div> <div class="given-names">T.K.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Identification of submicroscopic chromosomal aberrations in fetuses with increased nuchal translucency and apparently normal karyotype</div>, <div class="source">Ultrasound Obstet. Gynecol</div> ., <div class="volume">38</div>, <div class="fpage">314</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20submicroscopic%20chromosomal%20aberrations%20in%20fetuses%20with%20increased%20nuchal%20translucency%20and%20apparently%20normal%20karyotype&amp;author=T.Y.%20Leung&amp;author=I.%20Vogel&amp;author=T.K.%20Lau&amp;publication_year=2011&amp;journal=Ultrasound%20Obstet.%20Gynecol&amp;volume=38&amp;pages=314-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/uog.v38.3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fuog.v38.3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fuog.v38.3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21400624" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20submicroscopic%20chromosomal%20aberrations%20in%20fetuses%20with%20increased%20nuchal%20translucency%20and%20apparently%20normal%20karyotype&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wong</div> <div class="given-names">C.W.</div>
</div>, <div class="name string-name">
<div class="surname">Or</div> <div class="given-names">P.M.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Identification of a PTEN mutation with reduced protein stability, phosphatase activity, and nuclear localization in Hong Kong patients with autistic features, neurodevelopmental delays, and macrocephaly</div>, <div class="source">Autism Res</div> ., <div class="volume">11</div>, <div class="fpage">1098</div>–<div class="lpage">109</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20PTEN%20mutation%20with%20reduced%20protein%20stability%2C%20phosphatase%20activity%2C%20and%20nuclear%20localization%20in%20Hong%20Kong%20patients%20with%20autistic%20features%2C%20neurodevelopmental%20delays%2C%20and%20macrocephaly&amp;author=C.W.%20Wong&amp;author=P.M.Y.%20Or&amp;author=Y.%20Wang&amp;publication_year=2018&amp;journal=Autism%20Res&amp;volume=11&amp;pages=1098-109" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/aur.1950" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Faur.1950" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Faur.1950"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29608813" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20a%20PTEN%20mutation%20with%20reduced%20protein%20stability%2C%20phosphatase%20activity%2C%20and%20nuclear%20localization%20in%20Hong%20Kong%20patients%20with%20autistic%20features%2C%20neurodevelopmental%20delays%2C%20and%20macrocephaly&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Redin</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Brand</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Collins</div> <div class="given-names">R.L.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">The genomic landscape of balanced cytogenetic abnormalities associated with human congenital anomalies</div>, <div class="source">Nat. Genet</div> ., <div class="volume">49</div>, <div class="fpage">36</div>–<div class="lpage">45</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20landscape%20of%20balanced%20cytogenetic%20abnormalities%20associated%20with%20human%20congenital%20anomalies&amp;author=C.%20Redin&amp;author=H.%20Brand&amp;author=R.L.%20Collins&amp;publication_year=2017&amp;journal=Nat.%20Genet&amp;volume=49&amp;pages=36-45" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.3720" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3720" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.3720"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27841880" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genomic%20landscape%20of%20balanced%20cytogenetic%20abnormalities%20associated%20with%20human%20congenital%20anomalies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Latysheva</div> <div class="given-names">N.S.</div>
</div>, <div class="name string-name">
<div class="surname">Babu</div> <div class="given-names">M.M.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Discovering and understanding oncogenic gene fusions through data intensive computational approaches</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">4487</div>–<div class="lpage">503</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Discovering%20and%20understanding%20oncogenic%20gene%20fusions%20through%20data%20intensive%20computational%20approaches&amp;author=N.S.%20Latysheva&amp;author=M.M.%20Babu&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=4487-503" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkw282" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw282" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkw282"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27105842" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Discovering%20and%20understanding%20oncogenic%20gene%20fusions%20through%20data%20intensive%20computational%20approaches&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pizzo</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Jensen</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Polyak</div> <div class="given-names">A.</div>
</div>
</div>, et al. <div class="year">201</div>9, 
<div class="article-title">Rare variants in the genetic background modulate cognitive and developmental phenotypes in individuals carrying disease-associated variants</div>, <div class="source">Genet. Med</div> ., 4, 816-825.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Rare%20variants%20in%20the%20genetic%20background%20modulate%20cognitive%20and%20developmental%20phenotypes%20in%20individuals%20carrying%20disease-associated%20variants&amp;aulast=Pizzo&amp;date=201" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rare%20variants%20in%20the%20genetic%20background%20modulate%20cognitive%20and%20developmental%20phenotypes%20in%20individuals%20carrying%20disease-associated%20variants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">N.K.</div>
</div>, <div class="name string-name">
<div class="surname">Chiang</div> <div class="given-names">J.P.W.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">Increasing evidence of combinatory variant effects calls for revised classification of low-penetrance alleles</div>, <div class="source">Genet. Med</div> ., doi: 10.1038/s41436-018-0347-3. <div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Increasing%20evidence%20of%20combinatory%20variant%20effects%20calls%20for%20revised%20classification%20of%20low-penetrance%20alleles&amp;aulast=Wang&amp;date=2018" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Increasing%20evidence%20of%20combinatory%20variant%20effects%20calls%20for%20revised%20classification%20of%20low-penetrance%20alleles&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hochstenbach</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">van Binsbergen</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Schuring-Blom</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Buijs</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">van Amstel</div> <div class="given-names">H.K.P.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">A survey of undetected, clinically relevant chromosome abnormalities when replacing postnatal karyotyping by whole genome sequencing</div>, <div class="source">Eur. J. Med. Genet</div> ., doi: 10.1016/j.ejmg.2018.09.010.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20survey%20of%20undetected%2C%20clinically%20relevant%20chromosome%20abnormalities%20when%20replacing%20postnatal%20karyotyping%20by%20whole%20genome%20sequencing&amp;aulast=Hochstenbach&amp;date=2018" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20survey%20of%20undetected%2C%20clinically%20relevant%20chromosome%20abnormalities%20when%20replacing%20postnatal%20karyotyping%20by%20whole%20genome%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">van Heesch</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Kloosterman</div> <div class="given-names">W.P.</div>
</div>, <div class="name string-name">
<div class="surname">Lansu</div> <div class="given-names">N.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Improving mammalian genome scaffolding using large insert mate-pair next-generation sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">257</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improving%20mammalian%20genome%20scaffolding%20using%20large%20insert%20mate-pair%20next-generation%20sequencing&amp;author=S.%20van%20Heesch&amp;author=W.P.%20Kloosterman&amp;author=N.%20Lansu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=257" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-14-257" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-257" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-14-257"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23590730" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improving%20mammalian%20genome%20scaffolding%20using%20large%20insert%20mate-pair%20next-generation%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Kalscheuer</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Tzschach</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Mapping translocation breakpoints by next-generation sequencing</div>, <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">1143</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mapping%20translocation%20breakpoints%20by%20next-generation%20sequencing&amp;author=W.%20Chen&amp;author=V.%20Kalscheuer&amp;author=A.%20Tzschach&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=1143-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.076166.108" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.076166.108" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.076166.108"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18326688" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mapping%20translocation%20breakpoints%20by%20next-generation%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dong</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Identification of balanced chromosomal rearrangements previously unknown among participants in the 1000 Genomes Project: implications for interpretation of structural variation in genomes and the future of clinical cytogenetics</div>, <div class="source">Genet. Med</div> ., <div class="volume">20</div>, <div class="fpage">697</div>–<div class="lpage">707</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20balanced%20chromosomal%20rearrangements%20previously%20unknown%20among%20participants%20in%20the%201000%20Genomes%20Project%3A%20implications%20for%20interpretation%20of%20structural%20variation%20in%20genomes%20and%20the%20future%20of%20clinical%20cytogenetics&amp;author=Z.%20Dong&amp;author=H.%20Wang&amp;author=H.%20Chen&amp;publication_year=2018&amp;journal=Genet.%20Med&amp;volume=20&amp;pages=697-707" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/gim.2017.170" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fgim.2017.170" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fgim.2017.170"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29095815" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20balanced%20chromosomal%20rearrangements%20previously%20unknown%20among%20participants%20in%20the%201000%20Genomes%20Project%3A%20implications%20for%20interpretation%20of%20structural%20variation%20in%20genomes%20and%20the%20future%20of%20clinical%20cytogenetics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goodwin</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">McPherson</div> <div class="given-names">J.D.</div>
</div>, <div class="name string-name">
<div class="surname">McCombie</div> <div class="given-names">W.R.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Coming of age: ten years of next-generation sequencing technologies</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">17</div>, <div class="fpage">333</div>–<div class="lpage">51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;author=S.%20Goodwin&amp;author=J.D.%20McPherson&amp;author=W.R.%20McCombie&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Genet&amp;volume=17&amp;pages=333-51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg.2016.49" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2016.49" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg.2016.49"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27184599" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Talkowski</div> <div class="given-names">M.E.</div>
</div>, <div class="name string-name">
<div class="surname">Ernst</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Heilbut</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Next-generation sequencing strategies enable routine detection of balanced chromosome rearrangements for clinical diagnostics and genetic research</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">88</div>, <div class="fpage">469</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20strategies%20enable%20routine%20detection%20of%20balanced%20chromosome%20rearrangements%20for%20clinical%20diagnostics%20and%20genetic%20research&amp;author=M.E.%20Talkowski&amp;author=C.%20Ernst&amp;author=A.%20Heilbut&amp;publication_year=2011&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=88&amp;pages=469-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ajhg.2011.03.013" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2011.03.013" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ajhg.2011.03.013"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21473983" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Next-generation%20sequencing%20strategies%20enable%20routine%20detection%20of%20balanced%20chromosome%20rearrangements%20for%20clinical%20diagnostics%20and%20genetic%20research&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dong</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">A robust approach for blind detection of balanced chromosomal rearrangements with whole-genome low-coverage sequencing</div>, <div class="source">Hum. Mutat</div> ., <div class="volume">35</div>, <div class="fpage">625</div>–<div class="lpage">36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20robust%20approach%20for%20blind%20detection%20of%20balanced%20chromosomal%20rearrangements%20with%20whole-genome%20low-coverage%20sequencing&amp;author=Z.%20Dong&amp;author=L.%20Jiang&amp;author=C.%20Yang&amp;publication_year=2014&amp;journal=Hum.%20Mutat&amp;volume=35&amp;pages=625-36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/humu.2014.35.issue-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.2014.35.issue-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fhumu.2014.35.issue-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24610732" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20robust%20approach%20for%20blind%20detection%20of%20balanced%20chromosomal%20rearrangements%20with%20whole-genome%20low-coverage%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leggett</div> <div class="given-names">R.M.</div>
</div>, <div class="name string-name">
<div class="surname">Clavijo</div> <div class="given-names">B.J.</div>
</div>, <div class="name string-name">
<div class="surname">Clissold</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Clark</div> <div class="given-names">M.D.</div>
</div>, <div class="name string-name">
<div class="surname">Caccamo</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">NextClip: an analysis and read preparation tool for Nextera long mate pair libraries</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">566</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=NextClip%3A%20an%20analysis%20and%20read%20preparation%20tool%20for%20Nextera%20long%20mate%20pair%20libraries&amp;author=R.M.%20Leggett&amp;author=B.J.%20Clavijo&amp;author=L.%20Clissold&amp;author=M.D.%20Clark&amp;author=M.%20Caccamo&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=566-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt702" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt702" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt702"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297520" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:NextClip%3A%20an%20analysis%20and%20read%20preparation%20tool%20for%20Nextera%20long%20mate%20pair%20libraries&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chapman</div> <div class="given-names">J.A.</div>
</div>, <div class="name string-name">
<div class="surname">Ho</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Sunkara</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Luo</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Schroth</div> <div class="given-names">G.P.</div>
</div>, <div class="name string-name">
<div class="surname">Rokhsar</div> <div class="given-names">D.S.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Meraculous: de novo genome assembly with short paired-end reads</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e23501</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Meraculous%3A%20de%20novo%20genome%20assembly%20with%20short%20paired-end%20reads&amp;author=J.A.%20Chapman&amp;author=I.%20Ho&amp;author=S.%20Sunkara&amp;author=S.%20Luo&amp;author=G.P.%20Schroth&amp;author=D.S.%20Rokhsar&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e23501" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0023501" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0023501" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0023501"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21876754" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Meraculous%3A%20de%20novo%20genome%20assembly%20with%20short%20paired-end%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hanscom</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Talkowski</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Design of large-insert jumping libraries for structural variant detection using Illumina sequencing</div>, <div class="source">Curr. Protoc. Hum. Genet</div> ., <div class="volume">80</div>, <div class="fpage">7.22.1</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Design%20of%20large-insert%20jumping%20libraries%20for%20structural%20variant%20detection%20using%20Illumina%20sequencing&amp;author=C.%20Hanscom&amp;author=M.%20Talkowski&amp;publication_year=2014&amp;journal=Curr.%20Protoc.%20Hum.%20Genet&amp;volume=80&amp;pages=7.22.1-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/0471142905.hg0722s80" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F0471142905.hg0722s80" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2F0471142905.hg0722s80"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Design%20of%20large-insert%20jumping%20libraries%20for%20structural%20variant%20detection%20using%20Illumina%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Heavens</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Accinelli</div> <div class="given-names">G.G.</div>
</div>, <div class="name string-name">
<div class="surname">Clavijo</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Clark</div> <div class="given-names">M.D.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">A method to simultaneously construct up to 12 differently sized Illumina Nextera long mate pair libraries with reduced DNA input, time, and cost</div>, <div class="source">Biotechniques</div> , <div class="volume">59</div>, <div class="fpage">42</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20method%20to%20simultaneously%20construct%20up%20to%2012%20differently%20sized%20Illumina%20Nextera%20long%20mate%20pair%20libraries%20with%20reduced%20DNA%20input%2C%20time%2C%20and%20cost&amp;author=D.%20Heavens&amp;author=G.G.%20Accinelli&amp;author=B.%20Clavijo&amp;author=M.D.%20Clark&amp;publication_year=2015&amp;journal=Biotechniques&amp;volume=59&amp;pages=42-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2144/000114310" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000114310" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2144%2F000114310"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26156783" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20method%20to%20simultaneously%20construct%20up%20to%2012%20differently%20sized%20Illumina%20Nextera%20long%20mate%20pair%20libraries%20with%20reduced%20DNA%20input%2C%20time%2C%20and%20cost&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jain</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Koren</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Miga</div> <div class="given-names">K.H.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Nanopore sequencing and assembly of a human genome with ultra-long reads</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">36</div>, <div class="fpage">338</div>–<div class="lpage">45</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nanopore%20sequencing%20and%20assembly%20of%20a%20human%20genome%20with%20ultra-long%20reads&amp;author=M.%20Jain&amp;author=S.%20Koren&amp;author=K.H.%20Miga&amp;publication_year=2018&amp;journal=Nat.%20Biotechnol&amp;volume=36&amp;pages=338-45" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.4060" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.4060" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.4060"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29431738" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nanopore%20sequencing%20and%20assembly%20of%20a%20human%20genome%20with%20ultra-long%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Stephens</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Iyer</div> <div class="given-names">R.K.</div>
</div>, <div class="name string-name">
<div class="surname">Kocher</div> <div class="given-names">J.P.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">Detection and visualization of complex structural variants from long reads</div>, <div class="source">BMC Bioinformatics</div> ., <div class="volume">19</div>, <div class="fpage">508</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20visualization%20of%20complex%20structural%20variants%20from%20long%20reads&amp;author=Z.%20Stephens&amp;author=C.%20Wang&amp;author=R.K.%20Iyer&amp;author=J.P.%20Kocher&amp;publication_year=2018&amp;journal=BMC%20Bioinformatics&amp;volume=19&amp;pages=508" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12859-018-2539-x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12859-018-2539-x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12859-018-2539-x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30577744" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Detection%20and%20visualization%20of%20complex%20structural%20variants%20from%20long%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Xi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">W.</div>
</div>
</div>, et al. <div class="year">201</div>9, 
<div class="article-title">3' Branch ligation: a novel method to ligate non-complementary DNA to recessed or internal 3'OH ends in DNA or RNA</div>. <div class="source">DNA Res</div> ., 26, 45–53.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=3'%20Branch%20ligation%3A%20a%20novel%20method%20to%20ligate%20non-complementary%20DNA%20to%20recessed%20or%20internal%203'OH%20ends%20in%20DNA%20or%20RNA&amp;aulast=Wang&amp;date=201" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:3'%20Branch%20ligation%3A%20a%20novel%20method%20to%20ligate%20non-complementary%20DNA%20to%20recessed%20or%20internal%203'OH%20ends%20in%20DNA%20or%20RNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Peters</div> <div class="given-names">B.A.</div>
</div>, <div class="name string-name">
<div class="surname">Kermani</div> <div class="given-names">B.G.</div>
</div>, <div class="name string-name">
<div class="surname">Sparks</div> <div class="given-names">A.B.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Accurate whole-genome sequencing and haplotyping from 10 to 20 human cells</div>, <div class="source">Nature</div> , <div class="volume">487</div>, <div class="fpage">190</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accurate%20whole-genome%20sequencing%20and%20haplotyping%20from%2010%20to%2020%20human%20cells&amp;author=B.A.%20Peters&amp;author=B.G.%20Kermani&amp;author=A.B.%20Sparks&amp;publication_year=2012&amp;journal=Nature&amp;volume=487&amp;pages=190-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature11236" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11236" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature11236"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22785314" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accurate%20whole-genome%20sequencing%20and%20haplotyping%20from%2010%20to%2020%20human%20cells&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zook</div> <div class="given-names">J.M.</div>
</div>, <div class="name string-name">
<div class="surname">Catoe</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">McDaniel</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Extensive sequencing of seven human genomes to characterize benchmark reference materials</div>, <div class="source">Sci. Data</div> , <div class="volume">3</div>, <div class="fpage">160025</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Extensive%20sequencing%20of%20seven%20human%20genomes%20to%20characterize%20benchmark%20reference%20materials&amp;author=J.M.%20Zook&amp;author=D.%20Catoe&amp;author=J.%20McDaniel&amp;publication_year=2016&amp;journal=Sci.%20Data&amp;volume=3&amp;pages=160025" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/sdata.2016.25" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsdata.2016.25" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsdata.2016.25"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27271295" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Extensive%20sequencing%20of%20seven%20human%20genomes%20to%20characterize%20benchmark%20reference%20materials&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Drmanac</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Sparks</div> <div class="given-names">A.B.</div>
</div>, <div class="name string-name">
<div class="surname">Callow</div> <div class="given-names">M.J.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays</div>, <div class="source">Science</div> , <div class="volume">327</div>, <div class="fpage">78</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Human%20genome%20sequencing%20using%20unchained%20base%20reads%20on%20self-assembling%20DNA%20nanoarrays&amp;author=R.%20Drmanac&amp;author=A.B.%20Sparks&amp;author=M.J.%20Callow&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=78-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1181498" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181498" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1181498"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19892942" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Human%20genome%20sequencing%20using%20unchained%20base%20reads%20on%20self-assembling%20DNA%20nanoarrays&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Carnevali</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Baccash</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Halpern</div> <div class="given-names">A.L.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Computational techniques for human genome resequencing using mated gapped reads</div>, <div class="source">J. Comput. Biol</div> ., <div class="volume">19</div>, <div class="fpage">279</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Computational%20techniques%20for%20human%20genome%20resequencing%20using%20mated%20gapped%20reads&amp;author=P.%20Carnevali&amp;author=J.%20Baccash&amp;author=A.L.%20Halpern&amp;publication_year=2012&amp;journal=J.%20Comput.%20Biol&amp;volume=19&amp;pages=279-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1089/cmb.2011.0201" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2011.0201" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1089%2Fcmb.2011.0201"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22175250" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Computational%20techniques%20for%20human%20genome%20resequencing%20using%20mated%20gapped%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Durbin</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp324"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dong</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Hu</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Low-pass whole-genome sequencing in clinical cytogenetics: a validated approach</div>, <div class="source">Genet. Med</div> ., <div class="volume">18</div>, <div class="fpage">940</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Low-pass%20whole-genome%20sequencing%20in%20clinical%20cytogenetics%3A%20a%20validated%20approach&amp;author=Z.%20Dong&amp;author=J.%20Zhang&amp;author=P.%20Hu&amp;publication_year=2016&amp;journal=Genet.%20Med&amp;volume=18&amp;pages=940-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/gim.2015.199" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fgim.2015.199" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fgim.2015.199"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26820068" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Low-pass%20whole-genome%20sequencing%20in%20clinical%20cytogenetics%3A%20a%20validated%20approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cao</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Luo</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">De novo assembly of a haplotype-resolved human genome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">617</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20a%20haplotype-resolved%20human%20genome&amp;author=H.%20Cao&amp;author=H.%20Wu&amp;author=R.%20Luo&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=617-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.3200" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3200" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.3200"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26006006" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:De%20novo%20assembly%20of%20a%20haplotype-resolved%20human%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Jang</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Choi</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">3DIV: a 3D-genome interaction viewer and database</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">46</div>, <div class="fpage">D52</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=3DIV%3A%20a%203D-genome%20interaction%20viewer%20and%20database&amp;author=D.%20Yang&amp;author=I.%20Jang&amp;author=J.%20Choi&amp;publication_year=2018&amp;journal=Nucleic%20Acids%20Res&amp;volume=46&amp;pages=D52-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkx1017" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkx1017" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkx1017"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29106613" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:3DIV%3A%20a%203D-genome%20interaction%20viewer%20and%20database&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dong</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Ye</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">Z.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Balanced chromosomal rearrangement detection by low-pass whole-genome sequencing</div>, <div class="source">Curr. Protoc. Hum. Genet</div> ., <div class="volume">96</div>, <div class="fpage">8.18.1</div>–<div class="lpage">16</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Balanced%20chromosomal%20rearrangement%20detection%20by%20low-pass%20whole-genome%20sequencing&amp;author=Z.%20Dong&amp;author=L.%20Ye&amp;author=Z.%20Yang&amp;publication_year=2018&amp;journal=Curr.%20Protoc.%20Hum.%20Genet&amp;volume=96&amp;pages=8.18.1-16" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/0471142905.2018.96.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F0471142905.2018.96.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2F0471142905.2018.96.issue-1"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Balanced%20chromosomal%20rearrangement%20detection%20by%20low-pass%20whole-genome%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bentley</div> <div class="given-names">D.R.</div>
</div>, <div class="name string-name">
<div class="surname">Balasubramanian</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Swerdlow</div> <div class="given-names">H.P.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Accurate whole human genome sequencing using reversible terminator chemistry</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">53</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accurate%20whole%20human%20genome%20sequencing%20using%20reversible%20terminator%20chemistry&amp;author=D.R.%20Bentley&amp;author=S.%20Balasubramanian&amp;author=H.P.%20Swerdlow&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=53-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature07517" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07517" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature07517"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18987734" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accurate%20whole%20human%20genome%20sequencing%20using%20reversible%20terminator%20chemistry&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rothberg</div> <div class="given-names">J.M.</div>
</div>, <div class="name string-name">
<div class="surname">Hinz</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Rearick</div> <div class="given-names">T.M.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">An integrated semiconductor device enabling non-optical genome sequencing</div>, <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">348</div>–<div class="lpage">52</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20semiconductor%20device%20enabling%20non-optical%20genome%20sequencing&amp;author=J.M.%20Rothberg&amp;author=W.%20Hinz&amp;author=T.M.%20Rearick&amp;publication_year=2011&amp;journal=Nature&amp;volume=475&amp;pages=348-52" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature10242" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10242" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature10242"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21776081" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20integrated%20semiconductor%20device%20enabling%20non-optical%20genome%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fehlmann</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Reinheimer</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Geng</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">cPAS-based sequencing on the BGISEQ-500 to explore small non-coding RNAs</div>, <div class="source">Clin. Epigenetics</div> , <div class="volume">8</div>, <div class="fpage">123</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=cPAS-based%20sequencing%20on%20the%20BGISEQ-500%20to%20explore%20small%20non-coding%20RNAs&amp;author=T.%20Fehlmann&amp;author=S.%20Reinheimer&amp;author=C.%20Geng&amp;publication_year=2016&amp;journal=Clin.%20Epigenetics&amp;volume=8&amp;pages=123" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13148-016-0287-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13148-016-0287-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13148-016-0287-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27895807" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:cPAS-based%20sequencing%20on%20the%20BGISEQ-500%20to%20explore%20small%20non-coding%20RNAs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Valouev</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Ichikawa</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Tonthat</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">A high-resolution, nucleosome position map of <em>C. elegans</em> reveals a lack of universal sequence-dictated positioning</div>, <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">1051</div>–<div class="lpage">63</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%2C%20nucleosome%20position%20map%20of%20C.%20elegans%20reveals%20a%20lack%20of%20universal%20sequence-dictated%20positioning&amp;author=A.%20Valouev&amp;author=J.%20Ichikawa&amp;author=T.%20Tonthat&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=1051-63" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.076463.108" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.076463.108" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.076463.108"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18477713" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20high-resolution%2C%20nucleosome%20position%20map%20of%20C.%20elegans%20reveals%20a%20lack%20of%20universal%20sequence-dictated%20positioning&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Tan</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Heterozygous mutation of eEF1A1b resulted in spermatogenesis arrest and infertility in male tilapia, <em>Oreochromis niloticus</em>
</div>, <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">43733</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Heterozygous%20mutation%20of%20eEF1A1b%20resulted%20in%20spermatogenesis%20arrest%20and%20infertility%20in%20male%20tilapia%2C%20Oreochromis%20niloticus&amp;author=J.%20Chen&amp;author=D.%20Jiang&amp;author=D.%20Tan&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=43733" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep43733" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep43733" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep43733"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28266557" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Heterozygous%20mutation%20of%20eEF1A1b%20resulted%20in%20spermatogenesis%20arrest%20and%20infertility%20in%20male%20tilapia%2C%20Oreochromis%20niloticus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Thurman</div> <div class="given-names">R.E.</div>
</div>, <div class="name string-name">
<div class="surname">Rynes</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Humbert</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">The accessible chromatin landscape of the human genome</div>, <div class="source">Nature</div> , <div class="volume">489</div>, <div class="fpage">75</div>–<div class="lpage">82</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20accessible%20chromatin%20landscape%20of%20the%20human%20genome&amp;author=R.E.%20Thurman&amp;author=E.%20Rynes&amp;author=R.%20Humbert&amp;publication_year=2012&amp;journal=Nature&amp;volume=489&amp;pages=75-82" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature11232" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11232" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature11232"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955617" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20accessible%20chromatin%20landscape%20of%20the%20human%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Talkowski</div> <div class="given-names">M.E.</div>
</div>, <div class="name string-name">
<div class="surname">Ordulu</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Pillalamarri</div> <div class="given-names">V.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Clinical diagnosis by whole-genome sequencing of a prenatal sample</div>, <div class="source">N. Engl. J. Med</div> ., <div class="volume">367</div>, <div class="fpage">2226</div>–<div class="lpage">32</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Clinical%20diagnosis%20by%20whole-genome%20sequencing%20of%20a%20prenatal%20sample&amp;author=M.E.%20Talkowski&amp;author=Z.%20Ordulu&amp;author=V.%20Pillalamarri&amp;publication_year=2012&amp;journal=N.%20Engl.%20J.%20Med&amp;volume=367&amp;pages=2226-32" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1056/NEJMoa1208594" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1208594" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1056%2FNEJMoa1208594"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23215558" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Clinical%20diagnosis%20by%20whole-genome%20sequencing%20of%20a%20prenatal%20sample&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nilsson</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Pettersson</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Gustavsson</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Whole-genome sequencing of cytogenetically balanced chromosome translocations identifies potentially pathological gene disruptions and highlights the importance of microhomology in the mechanism of formation</div>, <div class="source">Hum. Mutat</div> ., <div class="volume">38</div>, <div class="fpage">180</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20cytogenetically%20balanced%20chromosome%20translocations%20identifies%20potentially%20pathological%20gene%20disruptions%20and%20highlights%20the%20importance%20of%20microhomology%20in%20the%20mechanism%20of%20formation&amp;author=D.%20Nilsson&amp;author=M.%20Pettersson&amp;author=P.%20Gustavsson&amp;publication_year=2017&amp;journal=Hum.%20Mutat&amp;volume=38&amp;pages=180-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/humu.2017.38.issue-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.2017.38.issue-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fhumu.2017.38.issue-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27862604" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Whole-genome%20sequencing%20of%20cytogenetically%20balanced%20chromosome%20translocations%20identifies%20potentially%20pathological%20gene%20disruptions%20and%20highlights%20the%20importance%20of%20microhomology%20in%20the%20mechanism%20of%20formation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Marine</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Polson</div> <div class="given-names">S.W.</div>
</div>, <div class="name string-name">
<div class="surname">Ravel</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Evaluation of a transposase protocol for rapid generation of shotgun high-throughput sequencing libraries from nanogram quantities of DNA</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">77</div>, <div class="fpage">8071</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20a%20transposase%20protocol%20for%20rapid%20generation%20of%20shotgun%20high-throughput%20sequencing%20libraries%20from%20nanogram%20quantities%20of%20DNA&amp;author=R.%20Marine&amp;author=S.W.%20Polson&amp;author=J.%20Ravel&amp;publication_year=2011&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=77&amp;pages=8071-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.05610-11" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.05610-11" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.05610-11"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21948828" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluation%20of%20a%20transposase%20protocol%20for%20rapid%20generation%20of%20shotgun%20high-throughput%20sequencing%20libraries%20from%20nanogram%20quantities%20of%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gu</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Szafranski</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Akdemir</div> <div class="given-names">Z.C.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Mechanisms for complex chromosomal insertions</div>, <div class="source">PLoS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1006446</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20for%20complex%20chromosomal%20insertions&amp;author=S.%20Gu&amp;author=P.%20Szafranski&amp;author=Z.C.%20Akdemir&amp;publication_year=2016&amp;journal=PLoS%20Genet&amp;volume=12&amp;pages=e1006446" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1006446" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1006446" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1006446"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27880765" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mechanisms%20for%20complex%20chromosomal%20insertions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Twigg</div> <div class="given-names">S.R.</div>
</div>, <div class="name string-name">
<div class="surname">Forecki</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Goos</div> <div class="given-names">J.A.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Gain-of-function mutations in ZIC1 are associated with coronal craniosynostosis and learning disability</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">97</div>, <div class="fpage">378</div>–<div class="lpage">88</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gain-of-function%20mutations%20in%20ZIC1%20are%20associated%20with%20coronal%20craniosynostosis%20and%20learning%20disability&amp;author=S.R.%20Twigg&amp;author=J.%20Forecki&amp;author=J.A.%20Goos&amp;publication_year=2015&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=97&amp;pages=378-88" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ajhg.2015.07.007" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2015.07.007" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ajhg.2015.07.007"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26340333" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gain-of-function%20mutations%20in%20ZIC1%20are%20associated%20with%20coronal%20craniosynostosis%20and%20learning%20disability&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Picelli</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Bjorklund</div> <div class="given-names">A.K.</div>
</div>, <div class="name string-name">
<div class="surname">Reinius</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Sagasser</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Winberg</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Sandberg</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Tn5 transposase and tagmentation procedures for massively scaled sequencing projects</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">2033</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tn5%20transposase%20and%20tagmentation%20procedures%20for%20massively%20scaled%20sequencing%20projects&amp;author=S.%20Picelli&amp;author=A.K.%20Bjorklund&amp;author=B.%20Reinius&amp;author=S.%20Sagasser&amp;author=G.%20Winberg&amp;author=R.%20Sandberg&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=2033-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.177881.114" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.177881.114" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.177881.114"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25079858" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tn5%20transposase%20and%20tagmentation%20procedures%20for%20massively%20scaled%20sequencing%20projects&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz011-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz011-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Van Nieuwerburgh</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Thompson</div> <div class="given-names">R.C.</div>
</div>, <div class="name string-name">
<div class="surname">Ledesma</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e24</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;author=F.%20Van%20Nieuwerburgh&amp;author=R.C.%20Thompson&amp;author=J.%20Ledesma&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e24" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1000" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1000" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1000"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127871" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="dsz011-FM1"><div class="fn"><div class="footnote-content">Zirui Dong, Xia Zhao, Qiaoling Li and Zhenjun Yang authors contributed equally to this work.</div></div></div>
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5512299&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz011/2/dsz011_supplementary_data.doc?Expires=1576701854&amp;Signature=3HZWZovTWdP-Ox0LaNmLbyZ5R~1BFK1ElJ6WxG3AGQxVzNuJpAAG5sxq3SUkS9G3QECWXm1exY2pq8m8Pky524blObwjoP0swbwapdcKi4UgMahQmytnT5nsZz~Ogwsw4~IVw~o7JNcCJxGat-yy-GiYmejo9jVfUAsTSTwThfoletWal9iegLR5Lahx4ZMr8djYk6v7v7vA4dQ6IRA8vRadxDPdvzJPLtjShPT9xj~3sFm7HQy-r4E3zWdF5XTco6uvkqVFoB0ZHFu9800oUqg1u6HVS~1a~JDEfBao89DapsEVPKr0d9qHMDRSoHTFTHitKvPG5THP6qpdAHC5gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz011_Supplementary_Data</div></a> - doc file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">498</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz011"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5512299"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493011000003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f4%2f313%2f5512299">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493011000003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F4%2F313%2F5512299" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5512299">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Development of coupling controlled polymerizations by adapter-ligation in mate-pair sequencing for detection of various genomic variants in one single assay | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz011");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f313%2f5512299">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
