<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Unraveling the complex genome of Saccharum spontaneum using Polyploid Gene Assembler | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/3/205/5320315","@type":"ScholarlyArticle","name":"Unraveling the complex genome of  Saccharum spontaneum  using Polyploid Gene Assembler","datePublished":"2019-02-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2019-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz001","keywords":["sugarcane","genome assembly","transcriptome","gene discovery","new assembler"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/3/205/5320315","author":[{"name":"Nascimento, Leandro Costa","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil  Laboratório Central de Tecnologias de Alto Desempenho (LaCTAD), Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"},{"name":"Yanagui, Karina","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"},{"name":"Jose, Juliana","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"},{"name":"Camargo, Eduardo L O","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil  Biocelere Agroindustrial Ltda, GranBio Investimentos S.A., Campinas, SP, Brazil","@type":"Person"},{"name":"Grassi, Maria Carolina B","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"},{"name":"Cunha, Camila P","affiliation":"Laboratório Nacional de Ciência e Tecnologia do Bioetanol (CTBE), Centro Nacional de Pesquisas em Energia e Materiais (CNPEM), Campinas, SP, Brazil","@type":"Person"},{"name":"Bressiani, José Antonio","affiliation":"Biocelere Agroindustrial Ltda, GranBio Investimentos S.A., Campinas, SP, Brazil","@type":"Person"},{"name":"Carvalho, Guilherme M A","affiliation":"Laboratório de Citogenética e Citometria, Departamento de Biologia Geral, Universidade Federal de Viçosa, Viçosa, MG, Brazil","@type":"Person"},{"name":"Carvalho, Carlos Roberto","affiliation":"Laboratório de Citogenética e Citometria, Departamento de Biologia Geral, Universidade Federal de Viçosa, Viçosa, MG, Brazil","@type":"Person"},{"name":"Prado, Paula F","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"},{"name":"Mieczkowski, Piotr","affiliation":"Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA","@type":"Person"},{"name":"Pereira, Gonçalo A G","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"},{"name":"Carazzolle, Marcelo F","affiliation":"Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil","@type":"Person"}],"description":"Abstract.  The Polyploid Gene Assembler (PGA), developed and tested in this study, represents a new strategy to perform gene-space assembly from complex genomes","pageStart":"205","pageEnd":"216","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636240084&Signature=wtONu3TsH6SuHVC4RokKcHwJVnLGjj0haW28H6tfQbDxfqdgpdQllK2ywxscHT1IAzYFG~HGNtPvAzKafRbhkEIX67MsLSW4X8c2fQR~NtQMFKwoeHs3cKf-yjzX5Ae6~XR8lwFRY-ZYBD7xgRYVoKB-uCRlFZKbCHR93zvfypjNnPfaF6wceMbvfjDtF8dz6SpgUcvsm1JiUVw2W0qwuOBhyE~juyIfZ-KUG3FJ8lfd8-fKs-f~pEYbhgx7NZF0KVSeMRz4Wk-Ny-ulFtNs6yZoKcatcuWzs-e2JA81v4o-LRR9Wm1upHAIb04ubDJdnAX~XbjneHAIH6mXzu~v-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Unraveling the complex genome of  Saccharum spontaneum  using Polyploid Gene Assembler","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636240084&Signature=wtONu3TsH6SuHVC4RokKcHwJVnLGjj0haW28H6tfQbDxfqdgpdQllK2ywxscHT1IAzYFG~HGNtPvAzKafRbhkEIX67MsLSW4X8c2fQR~NtQMFKwoeHs3cKf-yjzX5Ae6~XR8lwFRY-ZYBD7xgRYVoKB-uCRlFZKbCHR93zvfypjNnPfaF6wceMbvfjDtF8dz6SpgUcvsm1JiUVw2W0qwuOBhyE~juyIfZ-KUG3FJ8lfd8-fKs-f~pEYbhgx7NZF0KVSeMRz4Wk-Ny-ulFtNs6yZoKcatcuWzs-e2JA81v4o-LRR9Wm1upHAIb04ubDJdnAX~XbjneHAIH6mXzu~v-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Unraveling the complex genome of  Saccharum spontaneum  using Polyploid Gene Assembler">
<meta property="og:description" content="Abstract.  The Polyploid Gene Assembler (PGA), developed and tested in this study, represents a new strategy to perform gene-space assembly from complex genomes">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/3/205/5320315">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636240084&amp;Signature=wtONu3TsH6SuHVC4RokKcHwJVnLGjj0haW28H6tfQbDxfqdgpdQllK2ywxscHT1IAzYFG~HGNtPvAzKafRbhkEIX67MsLSW4X8c2fQR~NtQMFKwoeHs3cKf-yjzX5Ae6~XR8lwFRY-ZYBD7xgRYVoKB-uCRlFZKbCHR93zvfypjNnPfaF6wceMbvfjDtF8dz6SpgUcvsm1JiUVw2W0qwuOBhyE~juyIfZ-KUG3FJ8lfd8-fKs-f~pEYbhgx7NZF0KVSeMRz4Wk-Ny-ulFtNs6yZoKcatcuWzs-e2JA81v4o-LRR9Wm1upHAIb04ubDJdnAX~XbjneHAIH6mXzu~v-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-02-14">
<meta name="citation_author" content="Nascimento, Leandro Costa">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Laboratório Central de Tecnologias de Alto Desempenho (LaCTAD), Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Yanagui, Karina">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Jose, Juliana">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Camargo, Eduardo L O">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Biocelere Agroindustrial Ltda, GranBio Investimentos S.A., Campinas, SP, Brazil">
<meta name="citation_author" content="Grassi, Maria Carolina B">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Cunha, Camila P">
<meta name="citation_author_institution" content="Laboratório Nacional de Ciência e Tecnologia do Bioetanol (CTBE), Centro Nacional de Pesquisas em Energia e Materiais (CNPEM), Campinas, SP, Brazil">
<meta name="citation_author" content="Bressiani, José Antonio">
<meta name="citation_author_institution" content="Biocelere Agroindustrial Ltda, GranBio Investimentos S.A., Campinas, SP, Brazil">
<meta name="citation_author" content="Carvalho, Guilherme M A">
<meta name="citation_author_institution" content="Laboratório de Citogenética e Citometria, Departamento de Biologia Geral, Universidade Federal de Viçosa, Viçosa, MG, Brazil">
<meta name="citation_author" content="Carvalho, Carlos Roberto">
<meta name="citation_author_institution" content="Laboratório de Citogenética e Citometria, Departamento de Biologia Geral, Universidade Federal de Viçosa, Viçosa, MG, Brazil">
<meta name="citation_author" content="Prado, Paula F">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Mieczkowski, Piotr">
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA">
<meta name="citation_author" content="Pereira, Gonçalo A G">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Carazzolle, Marcelo F">
<meta name="citation_author_institution" content="Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil">
<meta name="citation_title" content="Unraveling the complex genome of Saccharum spontaneum using Polyploid Gene Assembler">
<meta name="citation_firstpage" content="205">
<meta name="citation_lastpage" content="216">
<meta name="citation_doi" content="10.1093/dnares/dsz001">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="3">
<meta name="citation_publication_date" content="2019/06/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Eukaryotic genome size databases;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D332-8; ">
<meta name="citation_reference" content="citation_title=A First-Generation Haplotype Map of Maize;  citation_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1115-1117; ">
<meta name="citation_reference" content="citation_title=Current challenges in de novo plant genome sequencing and assembly;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=243">
<meta name="citation_reference" content="citation_title=A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome;  citation_journal_title=Science;  citation_year=2014;  citation_volume=18;  citation_pages=1251788">
<meta name="citation_reference" content="citation_title=The Norway spruce genome sequence and conifer genome evolution;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=497;  citation_pages=579-84; ">
<meta name="citation_reference" content="citation_title=Reference-assisted chromosome assembly;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2013;  citation_volume=110;  citation_pages=1785-90; ">
<meta name="citation_reference" content="citation_title=Sugarcane genome analysis with molecular markers: a first decade of research;  citation_journal_title=Proc. Int. Soc. Sugar Cane Technol;  citation_year=2001;  citation_volume=24;  citation_pages=556-9; ">
<meta name="citation_reference" content="citation_title=High homologous gene conservation despite extreme autopolyploid redundancy in sugarcane;  citation_journal_title=New Phytol;  citation_year=2011;  citation_volume=189;  citation_pages=629-42; ">
<meta name="citation_reference" content="citation_title=Sugarcane Genomics: Depicting the Complex Genome of an Important Tropical Crop;  citation_title=Curr. Opin. Plant Biol.;  citation_year=2002;  citation_volume=5;  citation_pages=122-127; ">
<meta name="citation_reference" content="citation_title=The Potential of the Energy Cane as the Main Biomass Crop for the Cellulosic Industry;  citation_title=Chem. Biol. Technol. Agric.;  citation_year=2014;  citation_volume=1;  citation_pages=20">
<meta name="citation_reference" content="citation_publisher=Elsevier, Amsterdam;  citation_title=Sugarcane Improvement through Breeding;  citation_year=1987;  citation_pages=211-53; ">
<meta name="citation_reference" content="citation_title=The SUCEST-FUN regulatory network database: designing an energy grass;  citation_journal_title=Proc. Int. Soc. Sugar Cane Technol;  citation_year=2010;  citation_volume=27;  citation_pages=1-10; ">
<meta name="citation_reference" content="citation_title=Full-length enriched cDNA libraries and ORFeome analysis of sugarcane hybrid and ancestor genotypes;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e107351">
<meta name="citation_reference" content="citation_title=Initial genome sequencing of the sugarcane CP 96-1252 complex hybrid;  citation_journal_title=F1000Res;  citation_year=2017;  citation_volume=6;  citation_pages=688">
<meta name="citation_reference" content="citation_title=A mosaic monoploid reference sequence for the highly complex genome of sugarcane;  citation_journal_title=Nat. Commun;  citation_year=2018;  citation_volume=9;  citation_pages=2638">
<meta name="citation_reference" content="citation_title=Allele-defined genome of the autopolyploid sugarcane Saccharum spontaneum L;  citation_journal_title=Nat. Genet;  citation_year=2018;  citation_volume=50;  citation_pages=1565-1573; ">
<meta name="citation_reference" content="citation_title=RNA isolation from highly viscous samples rich in polyphenols and polysaccharides;  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=2002;  citation_volume=20;  citation_pages=417">
<meta name="citation_reference" content="citation_title=A micromethod for high throughput RNA extraction in forest trees;  citation_journal_title=Biol. Res;  citation_year=2007;  citation_volume=40;  citation_pages=291-7; ">
<meta name="citation_reference" content="citation_title=Genome size, base composition and karyotype of Jatropha curcas L., an important biofuel plant;  citation_journal_title=Plant Sci;  citation_year=2008;  citation_volume=174;  citation_pages=613-7; ">
<meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; ">
<meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; ">
<meta name="citation_reference" content="citation_title=L_RNA_scaffolder: scaffolding genomes with transcripts;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=604">
<meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; ">
<meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; ">
<meta name="citation_reference" content="citation_title=Construction of a map-based reference genome sequence for barley, Hordeum vulgare L;  citation_journal_title=Sci. Data;  citation_year=2017;  citation_volume=4;  citation_pages=170044">
<meta name="citation_reference" content="citation_title=Phytozome: a comparative platform for green plant genomics;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1178-86; ">
<meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison;  citation_journal_title=BMC Bioinform;  citation_year=2005;  citation_volume=6;  citation_pages=31">
<meta name="citation_reference" content="citation_title=AUGUSTUS: ab initio prediction of alternative transcripts;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W435-9; ">
<meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-2; ">
<meta name="citation_reference" content="citation_title=UniRef: comprehensive and non-redundant UniProt reference clusters;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1282-8; ">
<meta name="citation_reference" content="citation_title=The Sorghum bicolor genome and the diversification of grasses;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=457;  citation_pages=551-6; ">
<meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; ">
<meta name="citation_reference" content="citation_title=Genome sequence of foxtail millet (Setaria italica) provides insights into grass evolution and biofuel potential;  citation_journal_title=Nat. Biotechnol;  citation_year=2012;  citation_volume=30;  citation_pages=549-54; ">
<meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=265-72; ">
<meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R36">
<meta name="citation_reference" content="citation_title=The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=45-8; ">
<meta name="citation_reference" content="citation_title=The TIGR rice genome annotation resource: improvements and new features;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D883-7; ">
<meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1202-210; ">
<meta name="citation_reference" content="citation_title=De novo assembly and transcriptome analysis of contrasting sugarcane varieties;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e88462">
<meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=5654-66; ">
<meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-12; ">
<meta name="citation_reference" content="citation_title=Blast2GO: a comprehensive suite for functional analysis in plant genomics;  citation_journal_title=Int. J. Plant Genomics;  citation_year=2008;  citation_volume=2008;  citation_pages=619832">
<meta name="citation_reference" content="citation_title=Genome sequencing and analysis of the model grass Brachypodium distachyon;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463; ">
<meta name="citation_reference" content="citation_title=The draft genome of the fast-growing non-timber forest species moso bamboo (Phyllostachys heterocycla);  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=456-61; ">
<meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; ">
<meta name="citation_reference" content="citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=772-80; ">
<meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1312-13; ">
<meta name="citation_reference" content="citation_title=BadiRate: estimating family turnover rates by likelihood-based methods;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=279-81; ">
<meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=511-15; ">
<meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=323">
<meta name="citation_reference" content="citation_title=CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W345-9; ">
<meta name="citation_reference" content="citation_title=Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification;  citation_journal_title=Bioinform. Oxf. Engl;  citation_year=2005;  citation_volume=21;  citation_pages=650-9; ">
<meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; ">
<meta name="citation_reference" content="citation_title=The first near-complete assembly of the hexaploid bread wheat genome, Triticum aestivum;  citation_journal_title=GigaScience;  citation_year=2014;  citation_volume=6;  citation_pages=1-7; ">
<meta name="citation_reference" content="citation_title=New insights on leucine-rich repeats receptor-like kinase orthologous relationships in angiosperms;  citation_journal_title=Front. Plant Sci;  citation_year=2017;  citation_volume=8;  citation_pages=381">
<meta name="citation_reference" content="citation_title=GRAS-domain transcription factors that regulate plant development;  citation_journal_title=Plant Signal. Behav;  citation_year=2009;  citation_volume=4;  citation_pages=698-700; ">
<meta name="citation_reference" content="citation_title=ScGAI is a key regulator of culm development in sugarcane;  citation_journal_title=J. Exp. Bot;  citation_year=2018;  citation_volume=69;  citation_issue=16;  citation_pages=3823-3837; ">
<meta name="citation_reference" content="citation_title=Cellulose and callose synthesis and organization in focus, what’s new?;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2016;  citation_volume=34;  citation_pages=9-16; ">
<meta name="citation_reference" content="citation_title=Glucanocellulosic ethanol: the undiscovered biofuel potential in energy crops and marine biomass;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=13722">
<meta name="citation_reference" content="citation_title=Proteomic approach to understand the molecular physiology of symbiotic interaction between Piriformospora indica and Brassica napus;  citation_journal_title=Sci. Rep;  citation_year=2018;  citation_volume=8;  citation_pages=5773">
<meta name="citation_reference" content="citation_title=Calcium/calmodulin-dependent protein kinase is negatively and positively regulated by calcium, providing a mechanism for decoding calcium responses during symbiosis signaling;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=5053-66; ">
<meta name="citation_reference" content="citation_title=Overlaps in the transcriptional profiles of Medicago truncaluta roots inoculated with two different Glomus fungi provide insights into the genetic program activated during arbuscular mycorrhiza;  citation_journal_title=Plant Physiol;  citation_year=2005;  citation_volume=137;  citation_pages=1283-301; ">
<meta name="citation_reference" content="citation_title=Host and non-host roots in rice: cellular and molecular approaches reveal differential responses to arbuscular mycorrhizal fungi;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=636">
<meta name="citation_reference" content="citation_title=Genome-wide identification of the potato WRKY transcription factor family;  citation_journal_title=PLoS One;  citation_year=2017;  citation_volume=12;  citation_pages=e0181573">
<meta name="citation_reference" content="citation_title=he Medicago truncaluta N5 gene encoding a root-specific lipid transfer protein is required for the symbiotic interaction with Sinorhizobium meliloti;  citation_journal_title=Mol Plant Microbe Interact;  citation_year=2009;  citation_volume=22;  citation_issue=12;  citation_pages=1577-87; ">
<meta name="citation_reference" content="citation_title=Drought tolerance of bioenergy grass Saccharum spontaneum L. enhanced by arbuscular mycorrhizae;  citation_journal_title=Rhizosphere;  citation_year=2017;  citation_volume=3;  citation_pages=1-8; ">
<meta name="citation_reference" content="citation_title=Late Embryogenesis Abundant (LEA) proteins in legumes;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=4;  citation_pages=190">
<meta name="citation_reference" content="citation_title=Identification and expression analysis of BURP domain-containing genes in Medicago truncatula;  citation_journal_title=Front. Plant Sci;  citation_year=2016;  citation_volume=7;  citation_pages=485">
<meta name="citation_reference" content="citation_title=EUCANEXT: an integrated database for the exploration of genomic and transcriptomic data from Eucalyptus species;  citation_journal_title=Database J. Biol. Databases Curation;  citation_year=2017;  citation_volume=2017; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/3/205/28855575/dsz001.pdf">
<meta name="description" content="Abstract.  The Polyploid Gene Assembler (PGA), developed and tested in this study, represents a new strategy to perform gene-space assembly from complex genomes">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/3/205/5320315">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/3/205/5320315">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/3/205/5320315");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsz001';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f205%2f5320315" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f205%2f5320315" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115841"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1576706108&amp;Signature=S0sPFXsFvp8UxFxGCdckF32H~TZCH0ooiHAZdYcFIXka0MYS60xNwLmPzVlO-FRG9ZvpBuPc8IJHKUOup78ofgUsfwfKfqSzN2f3ckQVKpLmlGgYgTJYHlZwY2uhHD6c0m1N4jazce6SzbI2ZTEZmLRjJCfCbYUOhlgCdN4JZ-FR4ix82UDtqyWDWS20oar4QUSIKtzrilgNkTV~Tv7lW2xAIF7b~b8u3~JY1blL5J-IwHUv3nlvCwgE0UvvWkeRbTXNofrrsGKLkhfABovz4W0SB-rAlUZYB2xm07dWcPKfhXMidCNRNb-A4X9gu64JBfx69hwyHoFtRwa0i-UZlw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/3">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="137025512">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025512">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025514">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025514">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025519">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025519">2. Material and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025553">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025553">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025596">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025596">4. Conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025598">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025598">Availability of data</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025600">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025600">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="137025606">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025606">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/3/195/5303208">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/3/217/5475887">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Unraveling the complex genome of <em>Saccharum spontaneum</em> using Polyploid Gene Assembler
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Leandro Costa Nascimento</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
<div class="aff">Laboratório Central de Tecnologias de Alto Desempenho (LaCTAD), Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Leandro+Costa+Nascimento">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nascimento%20L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nascimento%20Leandro%20Costa%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Leandro Costa Nascimento</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karina Yanagui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karina+Yanagui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yanagui%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yanagui%20Karina%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Karina Yanagui</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juliana Jose</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juliana+Jose">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jose%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Jose%20Juliana%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Juliana Jose</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eduardo L O Camargo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
<div class="aff">Biocelere Agroindustrial Ltda, GranBio Investimentos S.A., Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eduardo+L+O+Camargo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Camargo%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Camargo%20Eduardo%20L%20O%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Eduardo L O Camargo</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Carolina B Grassi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Carolina+B+Grassi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Grassi%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Grassi%20Maria%20Carolina%20B%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Maria Carolina B Grassi</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Camila P Cunha</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório Nacional de Ciência e Tecnologia do Bioetanol (CTBE), Centro Nacional de Pesquisas em Energia e Materiais (CNPEM), Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Camila+P+Cunha">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cunha%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Cunha%20Camila%20P%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Camila P Cunha</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">José Antonio Bressiani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Biocelere Agroindustrial Ltda, GranBio Investimentos S.A., Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jos%c3%a9+Antonio+Bressiani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bressiani%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Bressiani%20Jos%C3%A9%20Antonio%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">José Antonio Bressiani</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guilherme M A Carvalho</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Citogenética e Citometria, Departamento de Biologia Geral, Universidade Federal de Viçosa, Viçosa, MG, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guilherme+M+A+Carvalho">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Carvalho%20G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Carvalho%20Guilherme%20M%20A%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Guilherme M A Carvalho</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Carlos Roberto Carvalho</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Citogenética e Citometria, Departamento de Biologia Geral, Universidade Federal de Viçosa, Viçosa, MG, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Carlos+Roberto+Carvalho">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Carvalho%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Carvalho%20Carlos%20Roberto%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Carlos Roberto Carvalho</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paula F Prado</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paula+F+Prado">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Prado%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Prado%20Paula%20F%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Paula F Prado</a><span class="delimiter-hidden">, </span>
                                </span>
                                    <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                        <span>...</span> Show more
                                    </a>
                                    <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Piotr Mieczkowski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Genetics, University of North Carolina at Chapel Hill, Chapel Hill, NC, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Piotr+Mieczkowski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mieczkowski%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mieczkowski%20Piotr%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Piotr Mieczkowski</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gonçalo A G Pereira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +55 19 3521 6237. Fax. +55 19 3521 6185. Email: <a href="mailto:goncalo@unicamp.br" target="_blank">goncalo@unicamp.br</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gon%c3%a7alo+A+G+Pereira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pereira%20G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Pereira%20Gon%C3%A7alo%20A%20G%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Gonçalo A G Pereira</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marcelo F Carazzolle</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Laboratório de Genômica e bioEnergia (LGE), Departamento de Genética, Evolução, Microbiologia e Imunologia, Instituto de Biologia, Universidade Estadual de Campinas, Campinas, SP, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marcelo+F+Carazzolle">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Carazzolle%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Carazzolle%20Marcelo%20F%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Marcelo F Carazzolle</a><span class="delimiter"></span>
                                        </span>
                                    </div>
                            </div>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 205–216, <a href="https://doi.org/10.1093/dnares/dsz001">https://doi.org/10.1093/dnares/dsz001</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">14 February 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">22 October 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">21 January 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">14 February 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5320315" href="/dnaresearch/article-pdf/26/3/205/28855575/dsz001.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Leandro Costa Nascimento, Karina Yanagui, Juliana Jose, Eduardo L O Camargo, Maria Carolina B Grassi, Camila P Cunha, José Antonio Bressiani, Guilherme M A Carvalho, Carlos Roberto Carvalho, Paula F Prado, Piotr Mieczkowski, Gonçalo A G Pereira, Marcelo F Carazzolle,  Unraveling the complex genome of <em>Saccharum spontaneum</em> using Polyploid Gene Assembler, <em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 205–216, <a href="https://doi.org/10.1093/dnares/dsz001">https://doi.org/10.1093/dnares/dsz001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5320315&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Unraveling%20the%20complex%20genome%20of%20%26lt%3Bem%26gt%3BSaccharum%20spontaneum%26lt%3B%2Fem%26gt%3B%20using%20Polyploid%20Gene%20Assembler&amp;publicationDate=2019-02-14&amp;volumeNum=26&amp;issueNum=3&amp;author=Nascimento%2C%20Leandro%20Costa%3B%20Yanagui%2C%20Karina&amp;startPage=205&amp;endPage=216&amp;contentId=10.1093%2Fdnares%2Fdsz001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f205%2f5320315" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="137025512" id="137025512" class="abstract-title">Abstract</h2>
<section class="abstract"><p>The Polyploid Gene Assembler (PGA), developed and tested in this study, represents a new strategy to perform gene-space assembly from complex genomes using low coverage DNA sequencing. The pipeline integrates reference-assisted loci and <em>de novo</em> assembly strategies to construct high-quality sequences focused on gene content. Pipeline validation was conducted with wheat (<em>Triticum aestivum</em>), a hexaploid species, using barley (<em>Hordeum vulgare</em>) as reference, that resulted in the identification of more than 90% of genes and several new genes. Moreover, PGA was used to assemble gene content in <em>Saccharum spontaneum</em> species, a parental lineage for hybrid sugarcane cultivars. <em>Saccharum spontaneum</em> gene sequence obtained was used to reference-guided transcriptome analysis of six different tissues. A total of 39,234 genes were identified, 60.4% clustered into known grass gene families. Thirty-seven gene families were expanded when compared with other grasses, three of them highlighted by the number of gene copies potentially involved in initial development and stress response. In addition, 3,108 promoters (many showing tissue specificity) were identified in this work. In summary, PGA can reconstruct high-quality gene sequences from polyploid genomes, as shown for wheat and <em>S. spontaneum</em> species, and it is more efficient than conventional genome assemblers using low coverage DNA sequencing.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword="sugarcane">sugarcane</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genome assembly"'>genome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"gene discovery"'>gene discovery</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"new assembler"'>new assembler</a>
</div>                    <h2 scrollto-destination="137025514" id="137025514" class="section-title">1. Introduction</h2>
<p>Even with the progress of the next-generation DNA sequencing technologies, the <em>de novo</em> genome assembly remains a bottleneck. It is observed mainly in the case of complex genomes, such as plants, which are more difficult to assemble without a reference genome. Usually, plant genomes are larger than most mammals genomes,<a reveal-id="dsz001-B1" data-open="dsz001-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> have high polyploidy, contain a high number of repetitive regions and a high level of heterozygosity.<a reveal-id="dsz001-B2" data-open="dsz001-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In addition, they contain a large number of gene families with numerous members and pseudogenes with nearly identical sequences due to effects of recent whole genome duplication and transposon activities. For these reasons, <em>de novo</em> assemblies of plants using only short reads generally create a highly fragmented genome split into hundred thousand scaffolds.<a reveal-id="dsz001-B3" data-open="dsz001-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Although the cost of sequencing has declined exponentially in recent years made it feasible to achieve high sequencing coverage for plant genomes, this strategy has not shown enough to generate quality genomes, such as the cases of <em>Triticum aestivum</em><a reveal-id="dsz001-B4" data-open="dsz001-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and <em>Picea abies</em>,<a reveal-id="dsz001-B5" data-open="dsz001-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> which genome assemblies are highly fragmented with a sequencing coverage around 150×. Among the possibilities to increase the quality of these genomes, the development of new bioinformatics efforts can contribute to overcoming these constraints, especially, by exploring the continuous growth of closely related genomes and reference-assisted loci algorithms.<a reveal-id="dsz001-B6" data-open="dsz001-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p>
<p>Plants from <em>Saccharum</em> genus have very complex genome with high level of polyploidy, among them the commercial cultivars of sugarcane have an economic relevance due to high amount of sucrose. They are generated from several crossings between <em>Saccharum officinarum</em> and <em>Saccharum spontaneum</em>, followed by artificial selections of hybrids.<a reveal-id="dsz001-B7" data-open="dsz001-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The genome profile of those hybrids consists of 70–80% of chromosomes from the <em>S. officinarum</em>, 10–20% of <em>S. spontaneum</em> and a few with inter-specific recombinations.<a reveal-id="dsz001-B8" data-open="dsz001-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsz001-B9" data-open="dsz001-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><em>Saccharum spontaneum</em> shows moderate sugar (8–10%) and high fibre (25–27%) contents, as well as high productivity (more than 180 t ha<sup>−1</sup>) and resistance to biotic and abiotic stresses.<a reveal-id="dsz001-B9" data-open="dsz001-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsz001-B10" data-open="dsz001-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Moreover, <em>S. spontaneum</em> promises to improve fibre content in lignocellulosic feedstock that has an economically attractiveness for biofuels and other biochemicals production using recent technologies developed for biomass deconstruction.<a reveal-id="dsz001-B10" data-open="dsz001-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The species is autopolyploid with chromosome numbers ranging from 40 to 128 and genome size ranging from 2.5 Gb to 12.5 Gb, with many aneuploidy forms.<a reveal-id="dsz001-B8" data-open="dsz001-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsz001-B11" data-open="dsz001-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p>
<p>Due to its economic relevance of sugar and first-generation ethanol production, several groups have worked to disclose the molecular basis of commercial sugarcane varieties and significant contributions for genome and transcriptome profile were achieved. The SUCEST-FUN<a reveal-id="dsz001-B12" data-open="dsz001-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and ORFeome<a reveal-id="dsz001-B13" data-open="dsz001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> databases are great examples of large initiatives of researchers groups to understand the molecular biology of sugarcane. This database is a large platform that supports the development of System Biology projects of sugarcane providing information regarding the genome, gene expression and regulatory network. Apart from the SUCEST-FUN project, many other studies have made noteworthy contributions on the molecular basis of sugarcane.<a reveal-id="dsz001-B14" data-open="dsz001-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Recently, a draft genome from sugarcane covering around 40% of the monoploid genome and more than 25,000 genes was published using a combination of BACs (Bacterial Artificial chromosomes), Illumina and PacBio sequencing.<a reveal-id="dsz001-B15" data-open="dsz001-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> However, most of these studies focused on commercial varieties of sugarcane, developed for the production of sugar and/or ethanol. A recent study of parental specie <em>S. spontaneum</em> investigated chromosomal structural rearrangements and disease resistance genes using whole genome sequencing.<a reveal-id="dsz001-B16" data-open="dsz001-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> However, the features related to productivity and accumulation of fibres were not analysed and are extremely necessary when we consider the production of biofuels from biomass and the transition for a green and sustainable economy.</p>
<p>In this work, we described a new and effective gene space assembly pipeline that allowed a reliable assembly of wheat (<em>T. aestivum</em>) and <em>S. spontaneum</em> genes using low coverage sequencing data. The identification of new genes, promoter regions, expanded gene families and transcriptome analysis of different tissues using reference-guided strategy were achieved. Our results showed the efficiency of Polyploid Gene Assembler (PGA) to reconstructed genes from complex genomes and contributed with new insights about the molecular basis of the <em>S. spontaneum</em> species, specifically involved with robustness and productivity of cane.</p>                    <h2 scrollto-destination="137025519" id="137025519" class="section-title">2. Material and methods</h2>
                    <h3 scrollto-destination="137025520" id="137025520" class="section-title">2.1. Plant material, genome and transcriptome sequencing</h3>
<p>This study was performed using DNA and mRNA from <em>S. spontaneum</em> accessions US851008 and IN8482. Leaf, apical meristem, node, bud, internode and root tissues from three biological replicates (individual plants) were harvested from 9-month old and greenhouse-grown plants, immediately frozen in liquid nitrogen and stored at −80°C. For the DNA extraction, 100 mg of leaf tissue were grinded in liquid nitrogen using the Mini-Beadbeater-96 (Biospec Products, Bartlesville, EUA), for 1.5 min. Each sample was resuspended in 700 μl of Extraction Buffer (100 mM Tris-HCl pH8, 20 mM EDTA, 2% CTAB, 1% PVP 40, 1.4 M NaCl, 0.3% 2-mercaptoethanol) and incubated at 65°C for 30 min, homogenizing every 10 min. Then, 800 μl of chloroform–isoamylalcohol (24:1) was added to the tubes, gently mixed and centrifuged at 16,000 × <em>g</em> for 10 min. The supernatant (450 μl) was transferred to a new 1.5 ml tube, containing 120 μl of Extraction Solution (5% CTAB, 1.4 M NaCl), added with 570 μl of chloroform–isoamylalcohol (24:1). Samples were centrifuged at 16,000 × <em>g</em> for 10 min and the supernatant (400) transferred to a new 1.5 ml tube, followed by the addition of 400 μl isopropanol. Samples were centrifuged at 16,000 × <em>g</em> for 5 min and the pellet washed with 1 ml of 70% ethanol. Air-dried pellets were resuspended in 300 μl of sterile water. Each sample was incubated with 2 μl of ribonuclease (RNAse 10 mg ml<sup>−1</sup>) at 37°C for 1 h. Quality control and quantification of the DNA were performed using dsDNA BR Assay (Invitrogen, Carlsbad, USA) and a Qubit fluorometer (Thermo Fisher Scientific, Waltham, USA), according to manufacturer’s instructions. Total RNA (only from US851008) was extracted following the protocol described by Zeng and Yang (2002)<a reveal-id="dsz001-B17" data-open="dsz001-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> modified by Le Provost et al. (2007).<a reveal-id="dsz001-B18" data-open="dsz001-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Quality control and quantification of RNA samples were performed using a Caliper LabChip XT micro capillary gel electrophoresis (Marshall Scientific, Hampton, USA) and a Qubit fluorometer (Thermo Fisher Scientific, Waltham, USA), respectively. Libraries were prepared with TruSeq Stranded mRNA Sample Preparation Kit (Illumina, San Diego, USA) using 1 µg of total RNA, according to manufacturer’s instructions. Quality control of the libraries was performed as previously described. Samples were sequenced at the High-Throughput Sequencing Facility at the University of North Carolina (UNC, USA) using HiSeq2500 (Illumina, San Diego, USA).</p>                    <h3 scrollto-destination="137025522" id="137025522" class="section-title">2.2. Genome size estimation</h3>
<p>Leaves from <em>Solanum lycopersicum</em> cultivar Stupicke, US851008 and IN8482 were processed for Flow Cytometry (FCM) at the Laboratory of Cytogenetics and Cytometry (Federal University of Viçosa, Viçosa, Brazil). Genome size of <em>S. spontaneum</em> was measured from nucleic suspensions prepared as described by Carvalho et al. (2008).<a reveal-id="dsz001-B19" data-open="dsz001-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> The suspensions were analysed with a Partec PAS Cytometer (Partec_GmbH, Munster, Germany), equipped with a laser source (488 nm, for 2<em>C</em> value). <em>Solanum lycopersicum</em> was used as standard (2<em>C</em> = 2 pg). FCM parameters (e.g. gain and channel) were determined for each DNA sample based on external FCM analyses of primary standard and sample (data not shown). Each analysis was conducted with three technical replicates, accounting for more than 10,000 nuclei. 2<em>C</em>-value of <em>S. spontaneum</em> was calculated by dividing the mean channel of the G0/G1 fluorescence peak from the primary standard by that of the sample.</p>                    <h3 scrollto-destination="137025524" id="137025524" class="section-title">2.3. PGA pipeline</h3>
<p>PGA was developed using PERL scripts for running in Linux system and integrates software to read mapping, <em>de novo</em> assembling and scaffolding. In mapping step, reads are split in sub-reads and mapped into the reference loci using Bowtie2<a reveal-id="dsz001-B20" data-open="dsz001-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (v. 2.2.1). Assemblies are performed by Trinity software<a reveal-id="dsz001-B21" data-open="dsz001-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> (v. 2013-02-25) allowing contigs greater than 500 bp. In the scaffolding steps, L_RNA_scaffolder<a reveal-id="dsz001-B22" data-open="dsz001-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> (allowing introns up to 500,000 bp) and BLAT<a reveal-id="dsz001-B23" data-open="dsz001-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (default parameters) are used for scaffolding using reference CDS and RNA sequences. SSPACE<a reveal-id="dsz001-B24" data-open="dsz001-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> (v. 2.0) with settings ‘–x 1-o 10 –v 2’ is used for scaffolding using raw DNA-Seq paired-end reads.</p>                    <h3 scrollto-destination="137025526" id="137025526" class="section-title">2.4. PGA validation using <em>T. aestivum</em> genome</h3>
<p>To validate PGA strategy, data from wheat (<em>T. aestivum</em>) were downloaded from Sequence Read Archive (SRA) from NCBI (National Center for Biotechnology Information) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The DNA-Seq reads were assembled with PGA using coverages of 3.6×, 5× and 7× (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) and the gene loci from barley (<em>Hordeum vulgare</em>) genome (24,243 sequences)<a reveal-id="dsz001-B25" data-open="dsz001-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> as reference. The wheat genes were downloaded from Phytozome<a reveal-id="dsz001-B26" data-open="dsz001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and compared with PGA assemblies using Exonerate<a reveal-id="dsz001-B27" data-open="dsz001-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> (v. 2.2.0) using the following settings ‘–refine region –refineboundary 100 –bestn 1 –percent 20 –quality 20 –geneseed 40 –seedrepeat 10 –minintron 30 –maxintron 500000’.</p>
<p>The gene prediction was performed using AUGUSTUS<a reveal-id="dsz001-B28" data-open="dsz001-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> (v. 3.2.1) with the following settings ‘–strand=both –genemodel=complete –species=wheat’. The AUGUSTUS results were compared with the wheat Phytozome gene prediction using BLASTn (<em>e</em>-value cutoff off 1e-20) and the ‘No hits’ sequences were selected. Among the ‘No hits’, we call new wheat genes those that met the following criteria: (i) non-aligned into the <em>T. aestivum</em> genome<a reveal-id="dsz001-B4" data-open="dsz001-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> with Exonerate<a reveal-id="dsz001-B27" data-open="dsz001-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> or, if aligned, lacking another predicted gene for at least 1,000 bp upstream or downstream by the windowBed tool from BEDTools<a reveal-id="dsz001-B29" data-open="dsz001-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (v. 2.17.0); (ii) have a hit in the Uniref90 database,<a reveal-id="dsz001-B30" data-open="dsz001-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> using BLASTp (<em>e</em>-value cutoff off 1e-5).</p>                    <h3 scrollto-destination="137025529" id="137025529" class="section-title">2.5. <em>Saccharum spontaneum</em> gene space assembly</h3>
<p><em>Saccharum spontaneum</em> DNA-Seq reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) were assembled by PGA using <em>Sorghum bicolor</em>,<a reveal-id="dsz001-B31" data-open="dsz001-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><em>Zea mays</em><a reveal-id="dsz001-B32" data-open="dsz001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and <em>Setaria italica</em><a reveal-id="dsz001-B33" data-open="dsz001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> genes as reference, chosen based on the phylogenetic distance. The gene space assembly was here referred as ‘<em>Saccharum spontaneum 1.0</em>’. To comparison, <em>S. spontaneum</em> reads were assembled by traditional approach using SOAPdenovo,<a reveal-id="dsz001-B34" data-open="dsz001-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> configured to use 73 K-mer, chosen by N50 maximization.</p>                    <h3 scrollto-destination="137025531" id="137025531" class="section-title">2.6. Splicing alignment of <em>S. spontaneum</em> RNA-Seq reads</h3>
<p>Paired-end and single-end reads from 18 RNA-Seq libraries (leaf, root, internode, node bud and apical meristem) were mapped against ‘<em>Saccharum spontaneum 1.0</em>’ using TopHat<a reveal-id="dsz001-B35" data-open="dsz001-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (v. 2.0.9) configured with the settings ‘–i 10 –I 500000 –library type fr-firststrand’. The percentage of mapped reads on each sample is available at <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. RNA-Seq data from wheat and maize were also mapped against their own reference genomes using software and parameters described above (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>).</p>                    <h3 scrollto-destination="137025533" id="137025533" class="section-title">2.7. <em>De novo</em> transcriptome assembly and annotation</h3>
<p>Trinity<a reveal-id="dsz001-B25" data-open="dsz001-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> (v. 2013-02-25) was used to perform <em>de novo</em> transcriptome assembly using 18 RNA-Seq libraries using the settings ‘–seqType fq –JM 100G –normalize_reads –normalize_max_read_cov 30’. Larger transcripts from each Trinity component were annotated using BLASTx (<em>e</em>-value cutoff &gt;1e-5) against the Uniref9030 and Swiss-Prot<a reveal-id="dsz001-B36" data-open="dsz001-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> databases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p>                    <h3 scrollto-destination="137025535" id="137025535" class="section-title">2.8. <em>Saccharum spontaneum</em> gene prediction</h3>
<p>Gene prediction in ‘<em>Saccharum spontaneum 1.0</em>’ was refined using the following datasets as extrinsic evidence: (i) 215,681 proteins from plants: 33,012 from great millet (<em>S. bicolor</em>),<a reveal-id="dsz001-B31" data-open="dsz001-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> 80,713 from maize (<em>Z. mays</em>),<a reveal-id="dsz001-B32" data-open="dsz001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> 35,471 from foxtail millet (<em>S. italica</em>),<a reveal-id="dsz001-B33" data-open="dsz001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> 39,049 from rice (<em>Oryza sativa</em>)<a reveal-id="dsz001-B37" data-open="dsz001-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and 27,436 from <em>Arabidopsis thaliana</em><a reveal-id="dsz001-B38" data-open="dsz001-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a>; (ii) 626,769 transcripts from <em>Saccharum</em> spp.: 358,695 from <em>S. spontaneum</em> (Section 4.7) and 268,034 from sugarcane<a reveal-id="dsz001-B13" data-open="dsz001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsz001-B39" data-open="dsz001-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and (iii) splicing alignments (BAM files) (Section 4.6). Datasets (i) and (ii) were mapped to <em>S. spontaneum</em> genome assembly using Exonerate<a reveal-id="dsz001-B27" data-open="dsz001-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> (v. 2.2.0) with the following settings ‘–refine region –refineboundary 100 –bestn 1 –percent 40 –quality 40 –geneseed 40 –seedrepeat 10 –minintron 30 –maxintron 500000’. The Exonerate<a reveal-id="dsz001-B27" data-open="dsz001-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> alignments were converted to GFF format with a custom PERL script. The BAM files from dataset (iii) were processed by bam2hints tools, from AUGUSTUS<a reveal-id="dsz001-B28" data-open="dsz001-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> package (v. 3.0.1) with the option ‘–intronsonly’, to convert the BAM files to GFF format. A custom PERL script filtered the hints files to retain only evidences of introns supported by three or more read alignments.</p>
<p>The prediction of the coding loci was also performed using AUGUSTUS<a reveal-id="dsz001-B28" data-open="dsz001-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> trained and configured as ‘–strand=both –genemodel=complete –alternatives-from-evidence=true –gff3 = on’. The training set was formed by 1,000 transcripts fully covered by one protein (the transcript must contains START and STOP codons in the correct position identified through comparison of BLASTx against the Uniref90 database<a reveal-id="dsz001-B30" data-open="dsz001-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>) from the Trinity<a reveal-id="dsz001-B21" data-open="dsz001-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><em>de novo</em> assembly (Section 4.7). GFF files generated by Exonerate<a reveal-id="dsz001-B27" data-open="dsz001-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and bam2hints programs were used as evidences of exons and introns. Predicted genes were mapped against CDD (Conserved Domain Database) using RPSBLAST and proteins with similarity superior to 30% to transposons (TE) domains were removed our dataset. The filtered gene models were submitted to PASA pipeline<a reveal-id="dsz001-B40" data-open="dsz001-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (v. 2.0.2) to improve the identification of untranslated regions (UTRs) and new splice variants.</p>
<p>The confidence of the predicted genes was determined by comparing coding sequences to 626,769 transcripts from <em>Saccharum</em> genus using BLASTn (<em>e</em>-value cutoff &lt;1e-5), with transcripts classified according to the alignment coverage: high confidence (HC; gene alignment coverage ≥60%). Medium confidence (MC; 30% ≤ alignment coverage ≥60%) and low confidence (LC; &lt; 30% alignment coverage or no hit). Finally, BUSCO pipeline<a reveal-id="dsz001-B41" data-open="dsz001-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> was performed using the plant dataset (Embryophyta odb9) to verify full-length HC proteins and compare their sequences with other homeologues in complex genomes, maize and sugarcane.</p>                    <h3 scrollto-destination="137025539" id="137025539" class="section-title">2.9. Read depth coverage</h3>
<p>To identify coverage from the gene regions in ‘<em>Saccharum spontaneum 1.0</em>’, the raw DNA reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) were compared with the <em>Saccharum spontaneum 1.0</em> assembly using Bowtie2<a reveal-id="dsz001-B20" data-open="dsz001-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (v. 2.2.1) using default parameters. The tools coverage and genomecov from BEDTools<a reveal-id="dsz001-B29" data-open="dsz001-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (v. 2.17.0) were used to extract the read depth coverage from the alignment results (BAM files).</p>                    <h3 scrollto-destination="137025541" id="137025541" class="section-title">2.10. Functional annotation of predicted genes</h3>
<p>The functional annotation of predicted <em>S. spontaneum</em> proteins was performed using BLASTp (<em>e</em>-value cutoff &lt;1e-5) against several databases, including Swiss-Prot,<a reveal-id="dsz001-B36" data-open="dsz001-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Uniref9030, NR from NCBI, TAIR<a reveal-id="dsz001-B38" data-open="dsz001-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and CDD. Gene ontology terms association and enrichment analysis were performed using BLAST2GO pipeline.<a reveal-id="dsz001-B42" data-open="dsz001-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                    <h3 scrollto-destination="137025543" id="137025543" class="section-title">2.11. Comparative genomics</h3>
<p>We used the proteins from assembled genome sequences of five Poaceae species available from the Phytozome v.12 database<a reveal-id="dsz001-B26" data-open="dsz001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a>: maize (PH207 v1.1),<a reveal-id="dsz001-B32" data-open="dsz001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> great millet (v3.1.1),<a reveal-id="dsz001-B31" data-open="dsz001-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> foxtail millet (v2.2),<a reveal-id="dsz001-B33" data-open="dsz001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> rice (v7)<a reveal-id="dsz001-B37" data-open="dsz001-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and purple false brome (<em>Brachypodium distachyon</em> v3.1)<a reveal-id="dsz001-B43" data-open="dsz001-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a>; and bamboo (<em>Phyllostachys heterocycla</em>) (EMBL, accession ERP001340).<a reveal-id="dsz001-B44" data-open="dsz001-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Homeologue gene groups were assigned between those species and <em>S. spontaneum</em> using the Markov clustering algorithm implemented in OrthoMCL.<a reveal-id="dsz001-B45" data-open="dsz001-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Single-copy orthologue (SCO) genes were used to a phylogenetic reconstruction. SCO genes were aligned individually with multiple alignment algorithms implemented in MAFFT<a reveal-id="dsz001-B46" data-open="dsz001-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> using the iterative refinement method and WSP and consistency scores (G-INS-i). All alignments were concatenated in a supermatrix for the maximum likelihood phylogenetic inference in RAxML v8<a reveal-id="dsz001-B47" data-open="dsz001-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> with the GTR+GAMMA model of substitutions and 1,000 bootstrap replicates for branch support. Gene groups with size changes during <em>S. spontaneum</em> evolution were identified by BadiRate<a reveal-id="dsz001-B48" data-open="dsz001-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> using the phylogenetic history, providing evolutionary inferences for expansion and retraction of gene families.</p>                    <h3 scrollto-destination="137025545" id="137025545" class="section-title">2.12. Transcript expression calculation</h3>
<p>The BAM files from RNA-Seq alignment to ‘<em>Saccharum spontaneum 1.0</em>’ were inputted into Cufflinks<a reveal-id="dsz001-B49" data-open="dsz001-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> (v. 2.1.1) to produce transcript assemblies, one for each sample. We used the settings ‘–I 500000 –multi-read-correct’ and the GFF file of <em>S. spontaneum</em> gene prediction to guide the assemblies. Subsequently, the cuffmerge/cuffcompare tools merged 18 cufflinks assemblies with the settings ‘–min-isoform-fraction 0.1’. The cuffcompare tool was used to classify the genes into new <em>S. spontaneum</em> isoforms, a transcript in the opposite strand than reference gene and as an intergenic transcript. The raw RNA-Seq reads were aligned against the transcriptome assembly using RSEM<a reveal-id="dsz001-B50" data-open="dsz001-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> (v. 1.2.19) with the parameter ‘–strand-specific’ to perform multi-mapping alignments and estimate the expression values for each transcript using RPKM (Reads Per Kilobase per Million mapped reads).</p>                    <h3 scrollto-destination="137025547" id="137025547" class="section-title">2.13. ncRNA identification</h3>
<p>ncRNAs were identified in the transcriptome assembly (Section 4.12) assuming four criteria: (i) classified as intergenic or in opposite strand than other transcript; (ii) no hits against NR database from NCBI (BLASTx <em>e</em>-value cutoff &lt;1e-5); (iii) Coding Potential Calculator (CPC)<a reveal-id="dsz001-B51" data-open="dsz001-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> score ≤ −1 (although CPC classifies ncRNA as transcripts with values ≤0, we decided to be more conservative to avoid false positives) and (iv) genomic location larger than 1,000 bp from scaffold tips to avoid partial genes not previously predicted.</p>                    <h3 scrollto-destination="137025549" id="137025549" class="section-title">2.14. Promoter sequences analysis</h3>
<p>A PERL script was developed to extract the start position of the alignment (first hit) from BLASTp/Uniref9030 output and, then, evaluate whether the predicted genes would be equivalent to complete protein prediction from the database. Because of the low conservation of the signal peptide, a margin of 20 amino acids was allowed at the beginning of each alignment. If the alignment started at up to 20 amino acids after the start codon, the protein beginning was in the correct position. The complete proteins were searched for the presence of at least 2,000 bp upstream of the start codon and those identified were subjected to a tissue-specific analysis using TAU metric.<a reveal-id="dsz001-B52" data-open="dsz001-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> TAU scores above 0.95 and RPKM average above 10 were potential candidates for tissue-specific promoters. A list with constitutive promoters was generated by calculating the coefficient of variance (standard deviation/mean) of each gene and it was considered constitutive when that coefficient was below 15%. Standard deviation and mean for each gene was calculated over RPKM values among all tissues.</p>                    <h3 scrollto-destination="137025551" id="137025551" class="section-title">2.15. <em>Saccharum</em> spp. databases comparison</h3>
<p>HC genes from <em>S. spontaneum</em> identified in this study were compared with three datasets of sugarcane genes: gene prediction from the sugarcane genome<a reveal-id="dsz001-B15" data-open="dsz001-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> (25,316 sequences), SUCEST-FUN<a reveal-id="dsz001-B12" data-open="dsz001-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> (43,141 sequences) and ORFeome<a reveal-id="dsz001-B13" data-open="dsz001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> (195,765 sequences) databases using BLASTn (<em>e</em>-value cut-off ≤ 1e<sup>−5</sup>). ‘No hits’ sequences were selected by a custom PERL script and the annotation from Uniref9030 from each one was selected. The expression profile was subjected to a clustering analysis using Seaborn’s clustermap package from Python.</p>                    <h2 scrollto-destination="137025553" id="137025553" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="137025554" id="137025554" class="section-title">3.1. Polyploid Gene Assembler</h3>
<p>PGA pipeline works with whole shotgun sequencing data of polyploidy and other complex genomes and focuses on gene assembly, including exons, introns, UTRs and promoters. It performs two main steps: (i) a reference-assisted assembly (here refereed as Reference Loci Assembly) and (ii) a <em>de novo</em> assembly with non-used reads in the first step. The pipeline integrates various software for read mapping, <em>de novo</em> assembling and scaffolding as described in <a reveal-id="dsz001-F1" data-open="dsz001-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>. PGA can be executed on a desktop machine with RAM memory configured according to the computational power available. Better performance is achieved using a server with at least 16 CPU cores, 64 GB of RAM memory and 500 GB of Hard Disk. Updates, new versions and results from PGA are available at the web site: <a class="link link-uri" href="http://www.lge.ibi.unicamp.br/pga" target="_blank">http://www.lge.ibi.unicamp.br/pga</a> (date last accessed 25 January 2019). PGA pipeline is valuable for any polyploid organism that has phylogenetically close-related species with available complete genome sequences. </p>                        <a id="137025556" scrollto-destination="137025556"></a>
<div id="dsz001-F1" data-id="dsz001-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f1.png?Expires=1576229860&amp;Signature=RgfDOaT6x7AM7ZbyFdEcDpGG9eN7Cy55SoVBRQrOKz41T8Fb-8hI3aVYckWwX-hv-EUzZEls5-19QveC5cwwr4YFQCogfbsrxSlTCCUMgdgyW05VRQxWplUF2Md5uGbyjLot~jT65m-xdAXTEE2X7h9F1pPX3VLeSPgSnUa7DpRurduMYkJOFr7B4mre4Gmrz5pfNuskFDprX1A4Jg0ac1tz6sBUvqVzZLL8aKY1NjttDGOZWyaFtAEcBbSSnShQmtnhizE4qNwW7Ajh543XG2IXr2u7QiRq2MrwjVxMHVtJW2fh2DPr3u6ImcSdlSjOAgc2ozNa5MSvyAoJg4OJ0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The PGA pipeline. (A) The three steps of the PGA pipeline are outlined, summarizing the methodology developed in this work. (B) The Reference Loci Assembly step of one locus (Loci 1) is show. Only three sub-reads (ReadA/1.1, ReadA/1.3 and ReadB/1.1) from two read-pairs (A and B) were mapped into the sequence of the Loci 1 (1 and 2), but the pipeline uses the both pairs (whole reads) in the local assembly (3) producing two contigs (4). (C) Validation of the PGA using sequencing data of T. aestivum." path-from-xml="dsz001f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025556/dsz001f1.tif" path-from-xml="dsz001f1.tif" target="_blank">Open in new tab</a><a section="137025556" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f1.png?Expires=1576229860&amp;Signature=XTVVBctDYlAQtT-H8zJ1p0cNBg4D-ie9SFD7wpwjY7MU8s~Ta7Rup7~SK7JjY3kGROn6Rh2k~EBkN0-utcar1HadQHxUt2OexXBJfCe~WWVVcOdAeXyhRuKrL8DrottuY8frTDzjv9nPi-MXegcE131PN4Hr1KA7CvYFx90O9IiNewV0qh6p7780Ficch1j3CYxzVkP5Oof440CowIwY3ZSNx2b4LowGwvggpi8ZDd7riGwiXz7n8RL6bW92xiE9KGiMEeeUePtD1NUCHpvOl9xLFSqf~XPy0tClL-TZE31zw0Goj5NmLwY58Gt4-ZimR5wVND0eiPdkgSHyaTxJDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025556&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The PGA pipeline. (A) The three steps of the PGA pipeline are outlined, summarizing the methodology developed in this work. (B) The Reference Loci Assembly step of one locus (Loci 1) is show. Only three sub-reads (ReadA/1.1, ReadA/1.3 and ReadB/1.1) from two read-pairs (A and B) were mapped into the sequence of the Loci 1 (1 and 2), but the pipeline uses the both pairs (whole reads) in the local assembly (3) producing two contigs (4). (C) Validation of the PGA using sequencing data of <em>T. aestivum</em>.</p></div>
</div>
<div content-id="dsz001-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f1.png?Expires=1576229860&amp;Signature=RgfDOaT6x7AM7ZbyFdEcDpGG9eN7Cy55SoVBRQrOKz41T8Fb-8hI3aVYckWwX-hv-EUzZEls5-19QveC5cwwr4YFQCogfbsrxSlTCCUMgdgyW05VRQxWplUF2Md5uGbyjLot~jT65m-xdAXTEE2X7h9F1pPX3VLeSPgSnUa7DpRurduMYkJOFr7B4mre4Gmrz5pfNuskFDprX1A4Jg0ac1tz6sBUvqVzZLL8aKY1NjttDGOZWyaFtAEcBbSSnShQmtnhizE4qNwW7Ajh543XG2IXr2u7QiRq2MrwjVxMHVtJW2fh2DPr3u6ImcSdlSjOAgc2ozNa5MSvyAoJg4OJ0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The PGA pipeline. (A) The three steps of the PGA pipeline are outlined, summarizing the methodology developed in this work. (B) The Reference Loci Assembly step of one locus (Loci 1) is show. Only three sub-reads (ReadA/1.1, ReadA/1.3 and ReadB/1.1) from two read-pairs (A and B) were mapped into the sequence of the Loci 1 (1 and 2), but the pipeline uses the both pairs (whole reads) in the local assembly (3) producing two contigs (4). (C) Validation of the PGA using sequencing data of T. aestivum." path-from-xml="dsz001f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025556/dsz001f1.tif" path-from-xml="dsz001f1.tif" target="_blank">Open in new tab</a><a section="137025556" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f1.png?Expires=1576229860&amp;Signature=XTVVBctDYlAQtT-H8zJ1p0cNBg4D-ie9SFD7wpwjY7MU8s~Ta7Rup7~SK7JjY3kGROn6Rh2k~EBkN0-utcar1HadQHxUt2OexXBJfCe~WWVVcOdAeXyhRuKrL8DrottuY8frTDzjv9nPi-MXegcE131PN4Hr1KA7CvYFx90O9IiNewV0qh6p7780Ficch1j3CYxzVkP5Oof440CowIwY3ZSNx2b4LowGwvggpi8ZDd7riGwiXz7n8RL6bW92xiE9KGiMEeeUePtD1NUCHpvOl9xLFSqf~XPy0tClL-TZE31zw0Goj5NmLwY58Gt4-ZimR5wVND0eiPdkgSHyaTxJDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025556&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The PGA pipeline. (A) The three steps of the PGA pipeline are outlined, summarizing the methodology developed in this work. (B) The Reference Loci Assembly step of one locus (Loci 1) is show. Only three sub-reads (ReadA/1.1, ReadA/1.3 and ReadB/1.1) from two read-pairs (A and B) were mapped into the sequence of the Loci 1 (1 and 2), but the pipeline uses the both pairs (whole reads) in the local assembly (3) producing two contigs (4). (C) Validation of the PGA using sequencing data of <em>T. aestivum</em>.</p></div>
</div>
<p><strong><em>PGA Reference Loci Assembly step</em></strong>. DNA-Seq reads are divided into three overlapping pieces with sizes equal to 50% of the original read size, called sub-reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The overlap between sub-reads prevents reads with tip differences (mismatches or gaps) from being unmapped, improving alignment rate in low conserved regions or intron–exon junctions from reference. The sub-reads are mapped against a selected reference loci, usually an evolutionary close-related species with available genome data. Reference loci must include exons, introns, UTRs and up to 500 bp upstream and downstream UTRs from one or more species to gene discovery maximization. On this step, PGA can be configured to allow differences (default is up to 10% of gaps or mismatches) between the sub-read and the reference sequence. Full reads corresponding to sub-reads mapped to a locus are retrieved and included in a <em>de novo</em> assembly performed to each locus separately (<a reveal-id="dsz001-F1" data-open="dsz001-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). In cases of sub-reads mapping to different loci, the locus with the highest number of occurrences is selected. Original CDS sequences from the reference(s) genome(s) can be used for scaffolding. This step is optional; however, a significant increase in longer contigs is observed (contigs ≥5,000 bp) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).</p>
<p><strong><em>PGA de novo Loci Assembly and Scaffolding steps</em></strong>. <em>De novo</em> assembly is performed only with reads that were discarded in the first step. Due the complexity of the polyploid genomes, this step generates very fragmented contigs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Despite that, we consider <em>de novo</em> assembly very important to identify genes that are exclusive in the studied organism. Optionally, is possible to perform a new round of scaffolding using raw DNA-Seq reads applied to <em>de novo</em> assembled contigs and reference-loci assembled contigs. After scaffolding step, it is possible to use RNA assemblies (assembled RNA-Seq or ESTs) to merge contigs. This step is also optional, depending on the availability of transcriptomic data of the organism of interest.</p>
<p>PGA uses a transcriptome assembler (Trinity<a reveal-id="dsz001-B21" data-open="dsz001-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>) instead of traditional genome assembler due to its best performance in comparison with conventional genome assemblers in most of the cases tested (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). It happened, probably, because Trinity was developed to support variations of coverage over the locus once it assembles the transcripts generated by alternative splicing events. All traditional genome assemblers, such as SOAPdenovo<a reveal-id="dsz001-B34" data-open="dsz001-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and Velvet,<a reveal-id="dsz001-B53" data-open="dsz001-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> are based on concept of uniform coverage, varying only on repetitive regions, and stop the formation of a contig when coverage changes significantly. This concept cannot be applied to polyploid genome, because homeologous chromosomes have regions with low (intergenic) and high (mainly exons) similarities,<a reveal-id="dsz001-B10" data-open="dsz001-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> which causes abrupt changes in coverage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p>
<p>The PGA strategy was validated using public available DNA and RNA sequence data (SRA, NCBI, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) from <em>T. aestivum</em> subsp. <em>aestivum</em> (bread wheat), a hexaploid species (size of 17 Gb) with high amount of near identical or duplicated sequences<a reveal-id="dsz001-B4" data-open="dsz001-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> using three different genome coverages (3.6, 5.0 and 7.7×) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) and <em>H. vulgare</em> (barley) genome as reference <a reveal-id="dsz001-B25" data-open="dsz001-B25" class="link link-ref link-reveal xref-bibr">25</a>. A total of 99,386 wheat genes available at Phytozome<a reveal-id="dsz001-B26" data-open="dsz001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> were used for comparison with the PGA scaffolds and resulted in almost 70% identified with up to 80% of alignment coverage (<a reveal-id="dsz001-F1" data-open="dsz001-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). In addition, the most part of the partially aligned genes (&lt;80% of alignment) have a very similar sequence but truncated in the borders (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), i.e. our assembly failed in the beginning or the end of the loci, possibly due the short scaffold sequences. Finally, the PGA assembly with coverage of 3.6× was submitted to gene prediction using AUGUSTUS<a reveal-id="dsz001-B28" data-open="dsz001-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> to verify if the pipeline was able to identify new wheat genes. This analysis resulted in a total of 6,378 predicted genes in the PGA assembly that were not identified by the Phytozome<a reveal-id="dsz001-B26" data-open="dsz001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> prediction. From these, 2,032 are probably new wheat genes because match in two criteria: (i) all of them have a BLASTp plant hits against Uniref9030 database; (ii) 171 genes did not align into the <em>T. aestivum</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) and 1,861 genes aligned into genome, but in the intergenic regions, lacking another predicted gene for at least 1,000 bp upstream or downstream (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). All these genes are new contributions for the complex wheat genome obtained by PGA pipeline using a very low sequencing coverage. The remaining 4,346 could be also real genes, but were discarded because did not match to one or the both criteria used.</p>                    <h3 scrollto-destination="137025561" id="137025561" class="section-title">3.2. Genome sequencing and gene-space assembly of <em>S. spontaneum</em>
</h3>
<p>There is a large variation in genome size among the varieties of <em>S. spontaneum</em> species (2<em>n</em> = 40–128).<a reveal-id="dsz001-B11" data-open="dsz001-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Therefore, the genome size of two <em>S. spontaneum</em> accessions, US851008 and IN8482, was experimentally estimated by flux cytometry resulting in 9.3 and 11.8 Gb, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Based on this result, DNA libraries of the smallest genome, i.e. <em>S. spontaneum</em> accession US851008, were sequenced by Illumina technology that produced 60 Gb of 100-bp paired end reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), representing a low coverage DNA sequencing of 6.4×. In addition, RNA-Seq libraries from leaf, apical meristem, node, bud, internode and root tissues were sequenced and produced around 20 million of 100-bp paired-end reads and 50 million of 100-bp single-end reads per library (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p>
<p>The application of PGA to <em>S. spontaneum</em> reads was performed using three closely related genomes as reference: <em>S. bicolor</em>,<a reveal-id="dsz001-B31" data-open="dsz001-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><em>Z. mays</em><a reveal-id="dsz001-B32" data-open="dsz001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and <em>S. italica.</em><a reveal-id="dsz001-B33" data-open="dsz001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Based on the phylogenetic distance, the 33,012 <em>S. bicolor</em> loci were used first, followed by 80,713 <em>Z. mays</em> loci and finished with 35,471 <em>S. italica</em> loci. The pipeline was executed for 23 days and used a maximum memory of 60 GB. PGA produced 79,690 scaffolds &gt;1,000 bp and this assembly was here referred as ‘<em>Saccharum spontaneum 1.0</em>’. Despite the ‘<em>Saccharum spontaneum 1.0</em>’ represents a high-fragmented draft genome, as observed by the number of sequences (≥500 bp) after scaffolding (179,429) and average sequence length (1,597 bp) (<a reveal-id="dsz001-T1" data-open="dsz001-T1" class="link link-reveal link-table xref-fig">Table 1</a>), it was expected that most of the sequences represent different protein-coding genes. To reinforce this hypothesis, the RNA-Seq reads were mapped against ‘<em>Saccharum spontaneum 1.0</em>’ resulting in around 70% of matches (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>), which it is similar value in comparison with wheat (average of 74.3%) and a close value in comparison with maize (average of 84%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>).</p>                        <a id="137025564" scrollto-destination="137025564"></a>
<div content-id="dsz001-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T1" class="table-wrap-title" data-id="dsz001-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Summary of three steps of PGA pipeline applied to <em>S. spontaneum</em>, including the <em>Saccharum spontaneum 1.0</em> assembly</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>Reference assembly </th>
<th>
<em>De novo</em> assembly </th>
<th>After scaffolding (<em>Saccharum spontaneum 1.0</em>) </th>
</tr></thead>
<tbody>
<tr>
<td>Sequences ≥ 500 bp </td>
<td>81,998 </td>
<td>132,518 </td>
<td>179,429 </td>
</tr>
<tr>
<td>Max sequence length (bp) </td>
<td>30,266 </td>
<td>26,551 </td>
<td>53,214 </td>
</tr>
<tr>
<td>Average sequence length (bp) </td>
<td>1,720 </td>
<td>1,053 </td>
<td>1,597 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>2,789 </td>
<td>1,130 </td>
<td>2,223 </td>
</tr>
<tr>
<td>Sequences ≥ 1,000 bp </td>
<td>39,213 </td>
<td>49,390 </td>
<td>79,690 </td>
</tr>
<tr>
<td>Sequences ≥ 5,000 bp </td>
<td>4,921 </td>
<td>454 </td>
<td>9,098 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>Reference assembly </th>
<th>
<em>De novo</em> assembly </th>
<th>After scaffolding (<em>Saccharum spontaneum 1.0</em>) </th>
</tr></thead>
<tbody>
<tr>
<td>Sequences ≥ 500 bp </td>
<td>81,998 </td>
<td>132,518 </td>
<td>179,429 </td>
</tr>
<tr>
<td>Max sequence length (bp) </td>
<td>30,266 </td>
<td>26,551 </td>
<td>53,214 </td>
</tr>
<tr>
<td>Average sequence length (bp) </td>
<td>1,720 </td>
<td>1,053 </td>
<td>1,597 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>2,789 </td>
<td>1,130 </td>
<td>2,223 </td>
</tr>
<tr>
<td>Sequences ≥ 1,000 bp </td>
<td>39,213 </td>
<td>49,390 </td>
<td>79,690 </td>
</tr>
<tr>
<td>Sequences ≥ 5,000 bp </td>
<td>4,921 </td>
<td>454 </td>
<td>9,098 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025564" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T1" class="table-wrap-title" data-id="dsz001-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Summary of three steps of PGA pipeline applied to <em>S. spontaneum</em>, including the <em>Saccharum spontaneum 1.0</em> assembly</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>Reference assembly </th>
<th>
<em>De novo</em> assembly </th>
<th>After scaffolding (<em>Saccharum spontaneum 1.0</em>) </th>
</tr></thead>
<tbody>
<tr>
<td>Sequences ≥ 500 bp </td>
<td>81,998 </td>
<td>132,518 </td>
<td>179,429 </td>
</tr>
<tr>
<td>Max sequence length (bp) </td>
<td>30,266 </td>
<td>26,551 </td>
<td>53,214 </td>
</tr>
<tr>
<td>Average sequence length (bp) </td>
<td>1,720 </td>
<td>1,053 </td>
<td>1,597 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>2,789 </td>
<td>1,130 </td>
<td>2,223 </td>
</tr>
<tr>
<td>Sequences ≥ 1,000 bp </td>
<td>39,213 </td>
<td>49,390 </td>
<td>79,690 </td>
</tr>
<tr>
<td>Sequences ≥ 5,000 bp </td>
<td>4,921 </td>
<td>454 </td>
<td>9,098 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>Reference assembly </th>
<th>
<em>De novo</em> assembly </th>
<th>After scaffolding (<em>Saccharum spontaneum 1.0</em>) </th>
</tr></thead>
<tbody>
<tr>
<td>Sequences ≥ 500 bp </td>
<td>81,998 </td>
<td>132,518 </td>
<td>179,429 </td>
</tr>
<tr>
<td>Max sequence length (bp) </td>
<td>30,266 </td>
<td>26,551 </td>
<td>53,214 </td>
</tr>
<tr>
<td>Average sequence length (bp) </td>
<td>1,720 </td>
<td>1,053 </td>
<td>1,597 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>2,789 </td>
<td>1,130 </td>
<td>2,223 </td>
</tr>
<tr>
<td>Sequences ≥ 1,000 bp </td>
<td>39,213 </td>
<td>49,390 </td>
<td>79,690 </td>
</tr>
<tr>
<td>Sequences ≥ 5,000 bp </td>
<td>4,921 </td>
<td>454 </td>
<td>9,098 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025564" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="137025565" id="137025565" class="section-title">3.3. Gene prediction and annotation</h3>
<p>A total of 53,436 genes and 59,890 transcripts were predicted on ‘<em>Saccharum spontaneum</em> 1.0’ using a combination of software and external evidence using proteins from closely related organisms and transcripts from <em>Saccharum</em> genus (as described in Section 2). The predicted proteins were annotated against public protein databases, resulting in more than 80% and 50% of genes supported by protein hit on the Uniref9030 and Swiss-Prot,<a reveal-id="dsz001-B36" data-open="dsz001-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> respectively (<a reveal-id="dsz001-F2" data-open="dsz001-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Also, the predictions were compared with grasses genes indicating high similarity with <em>S. bicolor</em><a reveal-id="dsz001-B31" data-open="dsz001-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and <em>S. italica</em><a reveal-id="dsz001-B33" data-open="dsz001-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). The 53,436 predicted genes in <em>Saccharum spontaneum 1.0</em> were classified considering the alignment coverage with 626,769 transcriptome sequences, resulting in 39,234 HC genes (alignment coverage ≥60%), 7,322 MC genes (30% ≤ alignment coverage ≥60%) and 6,880 as LC genes (2,903 genes with an &lt;30% alignment coverage and 3,977 no hits). A summary of the HC genes is shown in <a reveal-id="dsz001-T2" data-open="dsz001-T2" class="link link-reveal link-table xref-fig">Table 2</a>.</p>                        <a id="137025567" scrollto-destination="137025567"></a>
<div id="dsz001-F2" data-id="dsz001-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f2.png?Expires=1576229860&amp;Signature=hmIOV~-NfcHzDng97W1VlF45OUOk2CuFJK5kUeqhG-IwT2LcdHCg3z36~xIlu04l0mz3peTlTx9YoMTcEgzUNc4P~AJWkdoue5dgc9HqS0rjOPNDGoeFZ1ooV1d-HlwM4zCemvnWED3luvL~eaPYrFt-YIDHMw4-0M~yoyDhe5b0BZDdPDm8jf8HGEkJvqDq6DAUGM8idLx4-DFVYWK3oGmfs07SlFA7Z5yqFYaQqDOB8vK8ycgeq5Xjxg2EG6nbEbUy~5kVGyoKbsXW9Ig2umXUzT3rS07XLq5Ejqc70NeRRcXB3mwr1U0Dgy~CXvIADSk-Em9fTCAF~-zXBbRJDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Saccharum spontaneum gene prediction. (A) Annotation results from all genes against public protein databases. (B) BUSCO results using genes from sugarcane, S. spontaneum and Z. mays. (C) Accessing the completeness of the gene prediction from S. spontaneum, Z. mays and T. aestivum." path-from-xml="dsz001f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025567/dsz001f2.tif" path-from-xml="dsz001f2.tif" target="_blank">Open in new tab</a><a section="137025567" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f2.png?Expires=1576229860&amp;Signature=xusOE9TTkmnIrIMHhE4huEEtA21K2Srda2oP9pey-yYC0qya13-VbD93X4-AWa-OEOBhYPEj2KzTDaNu4XVU17j-01qYt-zw6rPu5WOp8O5yYC1Oix-ecCTT59jzQualZ6zAOqVN8Nf6lLHhyJXu4R71ylQ-UisZjfnua5I6oZKb6dNCMH2joUSYK0yQS97~T4atm0BRoDZR0k~RUaIo3tOce46Q64rmg4Styn6dVAlJnvZKytk~2FhZn11ygjlsFQ41xSDN97pvzf2ncjPMNN19u0eL~MFikMhdlPTtTFb6ohBGrsOjuLrNtedzQX4kMPBFatK-k0UWEOTPKfDsFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025567&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p><em>Saccharum spontaneum</em> gene prediction. (A) Annotation results from all genes against public protein databases. (B) BUSCO results using genes from sugarcane, <em>S. spontaneum</em> and <em>Z. mays</em>. (C) Accessing the completeness of the gene prediction from <em>S. spontaneum</em>, <em>Z. mays</em> and <em>T. aestivum</em>.</p></div>
</div>
<div content-id="dsz001-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f2.png?Expires=1576229860&amp;Signature=hmIOV~-NfcHzDng97W1VlF45OUOk2CuFJK5kUeqhG-IwT2LcdHCg3z36~xIlu04l0mz3peTlTx9YoMTcEgzUNc4P~AJWkdoue5dgc9HqS0rjOPNDGoeFZ1ooV1d-HlwM4zCemvnWED3luvL~eaPYrFt-YIDHMw4-0M~yoyDhe5b0BZDdPDm8jf8HGEkJvqDq6DAUGM8idLx4-DFVYWK3oGmfs07SlFA7Z5yqFYaQqDOB8vK8ycgeq5Xjxg2EG6nbEbUy~5kVGyoKbsXW9Ig2umXUzT3rS07XLq5Ejqc70NeRRcXB3mwr1U0Dgy~CXvIADSk-Em9fTCAF~-zXBbRJDA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Saccharum spontaneum gene prediction. (A) Annotation results from all genes against public protein databases. (B) BUSCO results using genes from sugarcane, S. spontaneum and Z. mays. (C) Accessing the completeness of the gene prediction from S. spontaneum, Z. mays and T. aestivum." path-from-xml="dsz001f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025567/dsz001f2.tif" path-from-xml="dsz001f2.tif" target="_blank">Open in new tab</a><a section="137025567" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f2.png?Expires=1576229860&amp;Signature=xusOE9TTkmnIrIMHhE4huEEtA21K2Srda2oP9pey-yYC0qya13-VbD93X4-AWa-OEOBhYPEj2KzTDaNu4XVU17j-01qYt-zw6rPu5WOp8O5yYC1Oix-ecCTT59jzQualZ6zAOqVN8Nf6lLHhyJXu4R71ylQ-UisZjfnua5I6oZKb6dNCMH2joUSYK0yQS97~T4atm0BRoDZR0k~RUaIo3tOce46Q64rmg4Styn6dVAlJnvZKytk~2FhZn11ygjlsFQ41xSDN97pvzf2ncjPMNN19u0eL~MFikMhdlPTtTFb6ohBGrsOjuLrNtedzQX4kMPBFatK-k0UWEOTPKfDsFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025567&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p><em>Saccharum spontaneum</em> gene prediction. (A) Annotation results from all genes against public protein databases. (B) BUSCO results using genes from sugarcane, <em>S. spontaneum</em> and <em>Z. mays</em>. (C) Accessing the completeness of the gene prediction from <em>S. spontaneum</em>, <em>Z. mays</em> and <em>T. aestivum</em>.</p></div>
</div>                        <a id="137025568" scrollto-destination="137025568"></a>
<div content-id="dsz001-T2" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T2" class="table-wrap-title" data-id="dsz001-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Summary of <em>S. spontaneum</em> gene prediction (only HC genes are shown)</p></div> </div>
<div class="table-overflow"><table><tbody>
<tr>
<td>#Gene loci </td>
<td>39,234 </td>
</tr>
<tr>
<td>#Transcripts </td>
<td>43,462 </td>
</tr>
<tr>
<td>Mean CDS length (bp) </td>
<td>902.85 </td>
</tr>
<tr>
<td>Max./min. CDS length (bp) </td>
<td>14,859/123 </td>
</tr>
<tr>
<td>Mean transcript length (bp) </td>
<td>1,082 </td>
</tr>
<tr>
<td>Mean exon length (bp) </td>
<td>247 </td>
</tr>
<tr>
<td>Max./min. exon length (bp) </td>
<td>6,702/3 </td>
</tr>
<tr>
<td>Single exon genes </td>
<td>9,683 </td>
</tr>
<tr>
<td>Swiss-Prot database support (&gt;30%/60%) </td>
<td>23,347/20,121 </td>
</tr>
<tr>
<td>Uniref90 database support (&gt;30%/60%) </td>
<td>33,945/32,148 </td>
</tr>
</tbody></table></div>
<div content-id="dsz001-T2" class="table-modal"><table><tbody>
<tr>
<td>#Gene loci </td>
<td>39,234 </td>
</tr>
<tr>
<td>#Transcripts </td>
<td>43,462 </td>
</tr>
<tr>
<td>Mean CDS length (bp) </td>
<td>902.85 </td>
</tr>
<tr>
<td>Max./min. CDS length (bp) </td>
<td>14,859/123 </td>
</tr>
<tr>
<td>Mean transcript length (bp) </td>
<td>1,082 </td>
</tr>
<tr>
<td>Mean exon length (bp) </td>
<td>247 </td>
</tr>
<tr>
<td>Max./min. exon length (bp) </td>
<td>6,702/3 </td>
</tr>
<tr>
<td>Single exon genes </td>
<td>9,683 </td>
</tr>
<tr>
<td>Swiss-Prot database support (&gt;30%/60%) </td>
<td>23,347/20,121 </td>
</tr>
<tr>
<td>Uniref90 database support (&gt;30%/60%) </td>
<td>33,945/32,148 </td>
</tr>
</tbody></table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025568" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T2" class="table-wrap-title" data-id="dsz001-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Summary of <em>S. spontaneum</em> gene prediction (only HC genes are shown)</p></div> </div>
<div class="table-overflow"><table><tbody>
<tr>
<td>#Gene loci </td>
<td>39,234 </td>
</tr>
<tr>
<td>#Transcripts </td>
<td>43,462 </td>
</tr>
<tr>
<td>Mean CDS length (bp) </td>
<td>902.85 </td>
</tr>
<tr>
<td>Max./min. CDS length (bp) </td>
<td>14,859/123 </td>
</tr>
<tr>
<td>Mean transcript length (bp) </td>
<td>1,082 </td>
</tr>
<tr>
<td>Mean exon length (bp) </td>
<td>247 </td>
</tr>
<tr>
<td>Max./min. exon length (bp) </td>
<td>6,702/3 </td>
</tr>
<tr>
<td>Single exon genes </td>
<td>9,683 </td>
</tr>
<tr>
<td>Swiss-Prot database support (&gt;30%/60%) </td>
<td>23,347/20,121 </td>
</tr>
<tr>
<td>Uniref90 database support (&gt;30%/60%) </td>
<td>33,945/32,148 </td>
</tr>
</tbody></table></div>
<div content-id="dsz001-T2" class="table-modal"><table><tbody>
<tr>
<td>#Gene loci </td>
<td>39,234 </td>
</tr>
<tr>
<td>#Transcripts </td>
<td>43,462 </td>
</tr>
<tr>
<td>Mean CDS length (bp) </td>
<td>902.85 </td>
</tr>
<tr>
<td>Max./min. CDS length (bp) </td>
<td>14,859/123 </td>
</tr>
<tr>
<td>Mean transcript length (bp) </td>
<td>1,082 </td>
</tr>
<tr>
<td>Mean exon length (bp) </td>
<td>247 </td>
</tr>
<tr>
<td>Max./min. exon length (bp) </td>
<td>6,702/3 </td>
</tr>
<tr>
<td>Single exon genes </td>
<td>9,683 </td>
</tr>
<tr>
<td>Swiss-Prot database support (&gt;30%/60%) </td>
<td>23,347/20,121 </td>
</tr>
<tr>
<td>Uniref90 database support (&gt;30%/60%) </td>
<td>33,945/32,148 </td>
</tr>
</tbody></table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025568" target="_blank">Open in new tab</a></div>
</div></div>
<p>To evaluate the completeness of HC proteins, the BUSCO<a reveal-id="dsz001-B41" data-open="dsz001-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> analysis was applied using a set of 1,440 highly conserved plant orthologous proteins. The analysis included <em>Z. mays</em><a reveal-id="dsz001-B32" data-open="dsz001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and Sugarcane proteins.<a reveal-id="dsz001-B15" data-open="dsz001-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> A majority (90%) of the 1,440 proteins in the plant BUSCO database was covered by HC proteins, and of these 80% were classified as complete and single copy. This result was similar to the <em>Z. mays</em> and better than Sugarcane proteins (<a reveal-id="dsz001-F2" data-open="dsz001-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Once the BUSCO analysis is based on a small set of single copy orthologous proteins, the completeness of HC proteins was expanded to all annotated proteins (BLASTp against Uniref9030) that were classified based on the alignment coverage: (i) ‘Complete’ predicted gene, when the predicted protein (query) covers very well the database subject (first hit from database); (ii) ‘3′ incomplete’ genes, when the predicted protein covers only the begin of database subject; (iii) ‘5′ incomplete’ genes, when the predicted protein covers only the final of the database subject and (iv) ‘Fragmented’ genes, when there is partial alignment on both sides. For comparison, the same procedure was performed using predicted proteins from <em>Z. mays</em><a reveal-id="dsz001-B32" data-open="dsz001-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and <em>T. aestivum</em><a reveal-id="dsz001-B4" data-open="dsz001-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> genomes available at Phytozome,<a reveal-id="dsz001-B26" data-open="dsz001-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> chosen for the complex genomes in the grass family. A larger amount of complete predicted genes in <em>S. spontaneum</em> were observed when compared with <em>Z. mays</em>, a largely studied genome, and <em>T. aestivum</em> (<a reveal-id="dsz001-F2" data-open="dsz001-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>).</p>
<p>PGA can assembly gene regions in polyploid organism using low sequencing coverage, because the high level of sequence conservation among homeologous genes increase the sequencing coverage in these regions. To show this effect, we calculated the sequencing coverage over gene regions by alignment of <em>S. spontaneum</em> reads against ‘<em>Saccharum spontaneum 1.0</em>’ and counting the read depth for each base pair in that regions (see Section 2). As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>, the distributions of read depth by base pair in the gene regions considering all predicted genes have peak at 82×. This new sequencing coverage represents ∼12 times the value previously calculated (6.4×) using the whole genome size and, interestingly, near the expected value of ploidy for <em>S. spontaneum</em>.</p>
<p>For comparison purposes, the <em>S. spontaneum</em> reads were assembled by conventional strategy using SOAPdenovo<a reveal-id="dsz001-B34" data-open="dsz001-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> (<a reveal-id="dsz001-T3" data-open="dsz001-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The SOAPdenovo was executed 12 days and used a maximum memory of 448 GB, 7.5× more memory than PGA. The assembled sequences were submitted for gene prediction using AUGUSTUS<a reveal-id="dsz001-B28" data-open="dsz001-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> (trained for <em>S. spontaneum</em> as described previously) that identified 54,658 genes. Despite this number is very similar than that obtained using PGA, the mean size of them (564.1 bp) was smaller than our dataset (53,436 genes with mean size of 840.3 bp). In addition, we performed completeness analysis using BUSCO<a reveal-id="dsz001-B41" data-open="dsz001-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> on SOAPdenovo gene prediction that showed a small covered over the BUSCO dataset (34.2%), being the majority fragmented (21.4%). These results reinforce the strategy based on reference-assisted loci that was used.</p>                        <a id="137025572" scrollto-destination="137025572"></a>
<div content-id="dsz001-T3" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T3" class="table-wrap-title" data-id="dsz001-T3">
<div class="label">Table 3</div>
<div class="caption"><p>Comparison between <em>S. spontaneum</em> assemblies using PGA and SOAPdenovo</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>PGA </th>
<th>SOAPdenovo </th>
</tr></thead>
<tbody>
<tr>
<td>Scaffolds (≥ 1,000 bp) </td>
<td>79,690 </td>
<td>135,024 </td>
</tr>
<tr>
<td>Mean scaffold size (bp) </td>
<td>2,720 </td>
<td>1,733 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>3,491 </td>
<td>1,744 </td>
</tr>
<tr>
<td>Largest scaffold (bp) </td>
<td>53,214 </td>
<td>54,032 </td>
</tr>
<tr>
<td>Total size (bp) </td>
<td>216,774,882 </td>
<td>234,014,899 </td>
</tr>
<tr>
<td>RAM memory used </td>
<td>60 GB </td>
<td>448 GB </td>
</tr>
<tr>
<td>Time to run </td>
<td>23 days </td>
<td>12 days </td>
</tr>
<tr>
<td>Gene loci </td>
<td>53,436 </td>
<td>54,658 </td>
</tr>
<tr>
<td>Mean size of genes (bp) </td>
<td>840.3 </td>
<td>564.1 </td>
</tr>
<tr>
<td>Complete and single copy (BUSCO) </td>
<td>79.6% </td>
<td>11.5% </td>
</tr>
<tr>
<td>Complete and duplicated (BUSCO) </td>
<td>2.8% </td>
<td>1.3% </td>
</tr>
<tr>
<td>Fragmented (BUSCO) </td>
<td>11.5% </td>
<td>21.4% </td>
</tr>
<tr>
<td>Missing (BUSCO) </td>
<td>6.1% </td>
<td>65.8% </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>PGA </th>
<th>SOAPdenovo </th>
</tr></thead>
<tbody>
<tr>
<td>Scaffolds (≥ 1,000 bp) </td>
<td>79,690 </td>
<td>135,024 </td>
</tr>
<tr>
<td>Mean scaffold size (bp) </td>
<td>2,720 </td>
<td>1,733 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>3,491 </td>
<td>1,744 </td>
</tr>
<tr>
<td>Largest scaffold (bp) </td>
<td>53,214 </td>
<td>54,032 </td>
</tr>
<tr>
<td>Total size (bp) </td>
<td>216,774,882 </td>
<td>234,014,899 </td>
</tr>
<tr>
<td>RAM memory used </td>
<td>60 GB </td>
<td>448 GB </td>
</tr>
<tr>
<td>Time to run </td>
<td>23 days </td>
<td>12 days </td>
</tr>
<tr>
<td>Gene loci </td>
<td>53,436 </td>
<td>54,658 </td>
</tr>
<tr>
<td>Mean size of genes (bp) </td>
<td>840.3 </td>
<td>564.1 </td>
</tr>
<tr>
<td>Complete and single copy (BUSCO) </td>
<td>79.6% </td>
<td>11.5% </td>
</tr>
<tr>
<td>Complete and duplicated (BUSCO) </td>
<td>2.8% </td>
<td>1.3% </td>
</tr>
<tr>
<td>Fragmented (BUSCO) </td>
<td>11.5% </td>
<td>21.4% </td>
</tr>
<tr>
<td>Missing (BUSCO) </td>
<td>6.1% </td>
<td>65.8% </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025572" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T3" class="table-wrap-title" data-id="dsz001-T3">
<div class="label">Table 3</div>
<div class="caption"><p>Comparison between <em>S. spontaneum</em> assemblies using PGA and SOAPdenovo</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>PGA </th>
<th>SOAPdenovo </th>
</tr></thead>
<tbody>
<tr>
<td>Scaffolds (≥ 1,000 bp) </td>
<td>79,690 </td>
<td>135,024 </td>
</tr>
<tr>
<td>Mean scaffold size (bp) </td>
<td>2,720 </td>
<td>1,733 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>3,491 </td>
<td>1,744 </td>
</tr>
<tr>
<td>Largest scaffold (bp) </td>
<td>53,214 </td>
<td>54,032 </td>
</tr>
<tr>
<td>Total size (bp) </td>
<td>216,774,882 </td>
<td>234,014,899 </td>
</tr>
<tr>
<td>RAM memory used </td>
<td>60 GB </td>
<td>448 GB </td>
</tr>
<tr>
<td>Time to run </td>
<td>23 days </td>
<td>12 days </td>
</tr>
<tr>
<td>Gene loci </td>
<td>53,436 </td>
<td>54,658 </td>
</tr>
<tr>
<td>Mean size of genes (bp) </td>
<td>840.3 </td>
<td>564.1 </td>
</tr>
<tr>
<td>Complete and single copy (BUSCO) </td>
<td>79.6% </td>
<td>11.5% </td>
</tr>
<tr>
<td>Complete and duplicated (BUSCO) </td>
<td>2.8% </td>
<td>1.3% </td>
</tr>
<tr>
<td>Fragmented (BUSCO) </td>
<td>11.5% </td>
<td>21.4% </td>
</tr>
<tr>
<td>Missing (BUSCO) </td>
<td>6.1% </td>
<td>65.8% </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th>PGA </th>
<th>SOAPdenovo </th>
</tr></thead>
<tbody>
<tr>
<td>Scaffolds (≥ 1,000 bp) </td>
<td>79,690 </td>
<td>135,024 </td>
</tr>
<tr>
<td>Mean scaffold size (bp) </td>
<td>2,720 </td>
<td>1,733 </td>
</tr>
<tr>
<td>N50 (bp) </td>
<td>3,491 </td>
<td>1,744 </td>
</tr>
<tr>
<td>Largest scaffold (bp) </td>
<td>53,214 </td>
<td>54,032 </td>
</tr>
<tr>
<td>Total size (bp) </td>
<td>216,774,882 </td>
<td>234,014,899 </td>
</tr>
<tr>
<td>RAM memory used </td>
<td>60 GB </td>
<td>448 GB </td>
</tr>
<tr>
<td>Time to run </td>
<td>23 days </td>
<td>12 days </td>
</tr>
<tr>
<td>Gene loci </td>
<td>53,436 </td>
<td>54,658 </td>
</tr>
<tr>
<td>Mean size of genes (bp) </td>
<td>840.3 </td>
<td>564.1 </td>
</tr>
<tr>
<td>Complete and single copy (BUSCO) </td>
<td>79.6% </td>
<td>11.5% </td>
</tr>
<tr>
<td>Complete and duplicated (BUSCO) </td>
<td>2.8% </td>
<td>1.3% </td>
</tr>
<tr>
<td>Fragmented (BUSCO) </td>
<td>11.5% </td>
<td>21.4% </td>
</tr>
<tr>
<td>Missing (BUSCO) </td>
<td>6.1% </td>
<td>65.8% </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025572" target="_blank">Open in new tab</a></div>
</div></div>
<p>In summary, the PGA pipeline produced high-quality <em>S. spontaneum</em> sequences focused on gene content in comparison with conventional genome assemblers (<a reveal-id="dsz001-T3" data-open="dsz001-T3" class="link link-reveal link-table xref-fig">Table 3</a>) and with other published complex genomes (<a reveal-id="dsz001-F2" data-open="dsz001-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The high-coverage sequencing over the gene regions, the use of transcriptome assembler and the criteria chosen for sequence selection (see Section 2) generated a representative sequence for each locus, avoiding erroneous duplication of homeologous alleles, that can be observed by low values of duplicated proteins (2.8%) in BUSCO<a reveal-id="dsz001-B41" data-open="dsz001-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> analysis.</p>                    <h3 scrollto-destination="137025574" id="137025574" class="section-title">3.4. Comparative genome and gene family analysis</h3>
<p>From the total of 39,234 HC genes predicted for <em>S. spontaneum</em>, 23,707 was clustered into gene families with genes from at least one other species and 15,527 genes were not assigned to any family and, thus, considered orphans (<a reveal-id="dsz001-F3" data-open="dsz001-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). SCOs appeared in 5,500 clusters that were used to infer the phylogeny in <a reveal-id="dsz001-F3" data-open="dsz001-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. </p>                        <a id="137025576" scrollto-destination="137025576"></a>
<div id="dsz001-F3" data-id="dsz001-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f3.png?Expires=1576229860&amp;Signature=p6YxuB03s-qd~QBpgifNS5XxzVCzz-vAsxG19g6xtlMJf65LFKjGRX091sz9QIIfutFjKygsbBMZfZ4~cE7dSrm2vEeobVYUC623RlPR8HNsR22rwsV2Ab9mQiDJa5TQREWZlxo6cQSNLAtV-fD8dDatjawaYrQH6obaFE0khL5uO9j7kI9GnLB2OsyPVRnktyOySzxDScB4A5JdcKCab-adbONgio1ntB1CNq75h8HFklTOMsTPgRvsAqoK8LOSM3RdS4nA1FL15ti3lcDb6gcXDIRuHOLsvVLHGKZP2pPWWO3FUsKun4D7cmuNVGIubi2nK2KBruJ1CujnxMPmrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic inference of the relationships among Poaceae species was compared with S. spontaneum. Phylogeny was obtained by Maximum Likelihood analysis using a concatenated matrix of the 5,500 single-copy gene families obtained in OrthoMCL. All branches got total support (100%) for the 1,000 bootstraps used. The total number of genes and the orphan genes for each species is on the right-side table. For each branch in phylogeny a pie chart reflects the proportion of gene gains and losses estimated in BadiRate analysis. The size of the pie charts reflects the absolute number of genes gained or lost for all families in that current or ancestral lineage." path-from-xml="dsz001f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025576/dsz001f3.tif" path-from-xml="dsz001f3.tif" target="_blank">Open in new tab</a><a section="137025576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f3.png?Expires=1576229860&amp;Signature=RXcyNWeCnyjRp7keWPv9laRhXFEixXv~lJNXHJVLTlZc4puo9ip4FRb27jbkljv70yvaXMLXvo4APRVJcXP~hr6W6LSOTynX46CmEyZxooQo46R1H9DO9~ghPmU9F8BK9QQVM-xQvzoD6wBRry-vpH-BgyZUIbsMyL1~CDGLMpKAXj~yFDFO5gWAmO2d~NTc5yO4u~DWizwFsB3ep5bVI3wqSwHDuCtmgzv-g42Di55k3NX3IBAYot4K0WwqiaGy60sgsfikjh8GWr2TBE5sq5J4YwS2Pg6a~44GOKtYFvtholAzfOCmh0eEYZOd51D3lWCM28HeUt1iSDZOoQTDGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025576&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Phylogenetic inference of the relationships among Poaceae species was compared with <em>S. spontaneum</em>. Phylogeny was obtained by Maximum Likelihood analysis using a concatenated matrix of the 5,500 single-copy gene families obtained in OrthoMCL. All branches got total support (100%) for the 1,000 bootstraps used. The total number of genes and the orphan genes for each species is on the right-side table. For each branch in phylogeny a pie chart reflects the proportion of gene gains and losses estimated in BadiRate analysis. The size of the pie charts reflects the absolute number of genes gained or lost for all families in that current or ancestral lineage.</p></div>
</div>
<div content-id="dsz001-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f3.png?Expires=1576229860&amp;Signature=p6YxuB03s-qd~QBpgifNS5XxzVCzz-vAsxG19g6xtlMJf65LFKjGRX091sz9QIIfutFjKygsbBMZfZ4~cE7dSrm2vEeobVYUC623RlPR8HNsR22rwsV2Ab9mQiDJa5TQREWZlxo6cQSNLAtV-fD8dDatjawaYrQH6obaFE0khL5uO9j7kI9GnLB2OsyPVRnktyOySzxDScB4A5JdcKCab-adbONgio1ntB1CNq75h8HFklTOMsTPgRvsAqoK8LOSM3RdS4nA1FL15ti3lcDb6gcXDIRuHOLsvVLHGKZP2pPWWO3FUsKun4D7cmuNVGIubi2nK2KBruJ1CujnxMPmrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic inference of the relationships among Poaceae species was compared with S. spontaneum. Phylogeny was obtained by Maximum Likelihood analysis using a concatenated matrix of the 5,500 single-copy gene families obtained in OrthoMCL. All branches got total support (100%) for the 1,000 bootstraps used. The total number of genes and the orphan genes for each species is on the right-side table. For each branch in phylogeny a pie chart reflects the proportion of gene gains and losses estimated in BadiRate analysis. The size of the pie charts reflects the absolute number of genes gained or lost for all families in that current or ancestral lineage." path-from-xml="dsz001f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025576/dsz001f3.tif" path-from-xml="dsz001f3.tif" target="_blank">Open in new tab</a><a section="137025576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f3.png?Expires=1576229860&amp;Signature=RXcyNWeCnyjRp7keWPv9laRhXFEixXv~lJNXHJVLTlZc4puo9ip4FRb27jbkljv70yvaXMLXvo4APRVJcXP~hr6W6LSOTynX46CmEyZxooQo46R1H9DO9~ghPmU9F8BK9QQVM-xQvzoD6wBRry-vpH-BgyZUIbsMyL1~CDGLMpKAXj~yFDFO5gWAmO2d~NTc5yO4u~DWizwFsB3ep5bVI3wqSwHDuCtmgzv-g42Di55k3NX3IBAYot4K0WwqiaGy60sgsfikjh8GWr2TBE5sq5J4YwS2Pg6a~44GOKtYFvtholAzfOCmh0eEYZOd51D3lWCM28HeUt1iSDZOoQTDGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025576&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Phylogenetic inference of the relationships among Poaceae species was compared with <em>S. spontaneum</em>. Phylogeny was obtained by Maximum Likelihood analysis using a concatenated matrix of the 5,500 single-copy gene families obtained in OrthoMCL. All branches got total support (100%) for the 1,000 bootstraps used. The total number of genes and the orphan genes for each species is on the right-side table. For each branch in phylogeny a pie chart reflects the proportion of gene gains and losses estimated in BadiRate analysis. The size of the pie charts reflects the absolute number of genes gained or lost for all families in that current or ancestral lineage.</p></div>
</div>
<p><em>Saccharum spontaneum</em> showed a high amount of orphan genes, like <em>Z. mays</em> e <em>O. sativa</em>, and 80% of them were assembled in the <em>de novo</em> step of our pipeline showing the importance of this step to reconstruct species-specific genes. Orphan genes represent genes that are in either a neutral process of pseudogenization or an adaptive process of evolutionary novelties of the lineage. Many orphan genes in plants have been reported as evolutionary novelties related to environmental stress response and species-specific traits<a reveal-id="dsz001-B54" data-open="dsz001-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and the large amount of orphan genes in <em>S. spontaneum</em> may be also related to traits selected during domestication. We evaluated the gene expression profile of the orphan genes and 52.5% have RPKM &gt;= 3 in at least one RNA-Seq libraries, being 31.8% with highly expression values (RPKM &gt;= 10). In addition, the most of 15,527 genes (53.3%) have similarity with the gene prediction from sugarcane (BLASTn, <em>e</em>-value cutoff of 1e-10) with some of them covering more than 90% of the sugarcane gene, such as: SS46658 (annotated as Peroxidase), SS21041 (annotated as Alpha-galactosidase) and SS4010 (annotated as transcriptional factor).</p>
<p>A total of 29,268 gene families with two or more genes were obtained, revealing 14,965 gene families with recent paralogues in at least one species, 138 families with recent paralogues exclusive to <em>S. spontaneum</em>, summing up 143 new duplication events in <em>S. spontaneum</em>: 133 duplications and 5 triplications (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). We also found 238 exclusive or expanded (or exclusive/expanded) gene families from <em>S. spontaneum</em>, with ∼95% representing unknown-domain protein or no hits in blast searches against NR/NCBI database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). We detected expression in 122 out of 167 genes among no hits families in at least one RNA-Seq library, which indicates that those families are not artefacts of the technique.</p>
<p>Expansions and retractions of gene families, analysed using phylogeny and the birth and death models, were used to calculate rates of gain and loss of genes through evolution (<a reveal-id="dsz001-F3" data-open="dsz001-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). In general, Poaceae species showed a higher gene gain than loss, with absolute numbers varying from 5,000 to 10,000 genes for each lineage. All ancestral lineages showed a lower amount of both gain and loss, suggesting that the current lineages have higher gene innovations. Those patterns of innovation may be associated with the domestication history of each crop. The highest proportion of gains was found within the grass clade, grouping <em>S. spontaneum</em>, <em>S. bicolor</em>, <em>Z. mays</em> and <em>S. italica</em>.</p>
<p>A total of 37 gene families were expanded in <em>S. spontaneum</em> when compared with its ancestral lineages. Most of them (30) were classified as no hits, unknown function, mobile element or ribosomal related in blast searches against NR/NCBI databases, although expression evidence was found in the RNA-Seq data. Three families were selected as promising candidates of expansion in <em>S. spontaneum</em> due to the number of genes, sequence completeness and agronomical relevance: leucine-rich repeat receptor-like kinase (LRR-RLK) family (ORTHOMCL0, 35 putative evolutionary novelties out of 61 genes), GRAS-domain protein family (ORTHOMCL18, 9 out of 26) and callose synthase (CalS) family (ORTHOMCL23, 12 out of 19) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S9 and S10</a></span>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S12</a></span>).</p>
<p>The LRR-RLK and GRAS domain are multigene and complex gene families, regulating many features in plant growth and development and stress response. LRR-RLK family encodes cell surface receptors responsible for cell–cell and cell–environment communication, affecting axillary and shoot apical meristem (SAM), tillering and brassinosteroid signalling pathway.<a reveal-id="dsz001-B55" data-open="dsz001-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Whereas, GRAS domain protein family modulates shoot and root development, through gibberellin and light signal transduction, axillary meristem initiation, SAM maintenance and root radial patterning.<a reveal-id="dsz001-B56" data-open="dsz001-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Recently, ScGAI, a GRAS protein and hormonal hub from sugarcane, was found in SAM and elongating internodes, implicated in shoot-root ratio modulation and tillering.<a reveal-id="dsz001-B57" data-open="dsz001-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Although difficult to devise gene function through phylogeny, the LRR-RLK and GRAS domain gene novelties from <em>S. spontaneum</em> showed on average higher gene expression levels in node and/or buds (axillary meristem) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>), indicating a potential role in initial growth development, coherently with the species capacity for intense tillering and faster culm development.</p>
<p>Callose, a minor cell wall polymer, is usually associated with specialized tissues (cell plate during cytokinesis, vascular system, pollen and pollen tube formation and plasmodesmata) and response to pathogen and herbivore attacks.<a reveal-id="dsz001-B58" data-open="dsz001-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Although composed mainly of cellulose, callose is found in significant amounts in cell walls of energy crops, such as maize and <em>Miscanthus x giganteus</em> (up to 5%) leaves, in which callose fibrils intertwined in cellulose forms an atypical outer layer.<a reveal-id="dsz001-B58" data-open="dsz001-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Callose-enriched biomass is a new target for biotechnological engineering resulting in significant increase in second generation ethanol yield using an optimized system of enzymes and yeast.<a reveal-id="dsz001-B59" data-open="dsz001-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> In this study, putative CalS gene novelties were ubiquitously and highly expressed (RPKM values from 103 to 174; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>) in all tissues evaluated and might represent new and interesting targets for sugarcane and energy cane breeding.</p>                    <h3 scrollto-destination="137025583" id="137025583" class="section-title">3.5. Reference transcriptome assembly</h3>
<p>The transcriptome assembly was produced using the Cufflinks<a reveal-id="dsz001-B49" data-open="dsz001-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> package using <em>Saccharum spontaneum 1.0</em> as the reference. As a result, 156,531 transcripts were identified with a length &gt;200 bp. The comparison between the reference and <em>de novo</em> transcriptome assembly, shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>, reveals that a set of higher quality sequences is constructed when the ‘<em>Saccharum spontaneum 1.0</em>’ is used to guide the transcriptome assembly. The transcriptome assembly using our reference genome proved to return high-quality and more informative data, with longer transcripts and ORFs, and more hits against Swiss-Prot.<a reveal-id="dsz001-B36" data-open="dsz001-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In addition, there is high concordance in gene structure, such as exon–intron junctions and intron length, between gene predictions and splicing alignments of RNA-Seq reads, as shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>.</p>
<p>In addition to improving the quality of the transcripts, the use of <em>Saccharum spontaneum 1.0</em> allows the identification of candidates for isoforms and long non-coding RNAs. For that the transcripts identified as a transcript in the opposite strand than reference gene (5,566) and as an intergenic (41,931) by the cuffcompare were submitted to the ncRNA identification. A total of 3,178 transcripts (mean size: 624 bp) could be assigned as non-coding (2,822 bigger than 200 bp, i.e. lncRNAs) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). As expected, those transcripts have little expression values, as shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>, in all RNA-Seq samples around 50% of non-coding RNAs have RPKM &gt; 1 and around 20% have RPKM &gt; 3.</p>                    <h3 scrollto-destination="137025586" id="137025586" class="section-title">3.6. Promoter sequences</h3>
<p>PGA pipeline can extend the scaffold sequences upstream from the gene, providing information on the promoter regions that can be used for many biotechnology applications. The use of constitutive or tissue-specific promoters is essential to carry out studies to investigate plant metabolism under defined conditions. Moreover, the use of right promoter to express heterologous proteins in a controlled condition and specific tissues is mandatory for the development of genetically modified varieties.</p>
<p>In this study, a total of 3,108 promoter regions were identified considering at least 2,000 bp upstream from the beginning of the gene. The initial positions of predicted genes were verified by alignment against Uniref90 database (see Section 2). These promoter regions were classified according to their respective gene expression profiles (constitutive or tissue-specific) and intensities: weak (10 &lt; RPKM &lt; 100) and strong (RPKM &gt;= 100). A summary of the promoter regions identified is shown in <a reveal-id="dsz001-T4" data-open="dsz001-T4" class="link link-reveal link-table xref-fig">Table 4</a> and reveals that the most represented group is formed by constitutive promoters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>) and there are many interesting tissue-specific promoters distributed among leaf (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S2</a></span>), root (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S3</a></span>), node (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S4</a></span>), bud (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S5</a></span>) and apical meristem (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S6</a></span>).</p>                        <a id="137025589" scrollto-destination="137025589"></a>
<div content-id="dsz001-T4" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T4" class="table-wrap-title" data-id="dsz001-T4">
<div class="label">Table 4</div>
<div class="caption"><p>Summary of the <em>S. spontaneum</em> promoter regions (≥2,000 bp upstream from the gene) classified by gene expression profile and intensity</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3"> </th>
<th rowspan="3"> </th>
<th colspan="7">Gene expression profile<hr> </th>
</tr>
<tr>
<th rowspan="2">Constitutive </th>
<th colspan="6">Tissue specific<hr> </th>
</tr>
<tr>
<th>Leaf </th>
<th>Root </th>
<th>Node </th>
<th>Internode </th>
<th>Bud </th>
<th>Apical meristem </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">Gene expression intensity </td>
<td>Weak </td>
<td>80 </td>
<td>33 </td>
<td>20 </td>
<td>4 </td>
<td>0 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Strong </td>
<td>7 </td>
<td>25 </td>
<td>2 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>2 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T4" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3"> </th>
<th rowspan="3"> </th>
<th colspan="7">Gene expression profile<hr> </th>
</tr>
<tr>
<th rowspan="2">Constitutive </th>
<th colspan="6">Tissue specific<hr> </th>
</tr>
<tr>
<th>Leaf </th>
<th>Root </th>
<th>Node </th>
<th>Internode </th>
<th>Bud </th>
<th>Apical meristem </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">Gene expression intensity </td>
<td>Weak </td>
<td>80 </td>
<td>33 </td>
<td>20 </td>
<td>4 </td>
<td>0 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Strong </td>
<td>7 </td>
<td>25 </td>
<td>2 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>2 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025589" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T4" class="table-wrap-title" data-id="dsz001-T4">
<div class="label">Table 4</div>
<div class="caption"><p>Summary of the <em>S. spontaneum</em> promoter regions (≥2,000 bp upstream from the gene) classified by gene expression profile and intensity</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3"> </th>
<th rowspan="3"> </th>
<th colspan="7">Gene expression profile<hr> </th>
</tr>
<tr>
<th rowspan="2">Constitutive </th>
<th colspan="6">Tissue specific<hr> </th>
</tr>
<tr>
<th>Leaf </th>
<th>Root </th>
<th>Node </th>
<th>Internode </th>
<th>Bud </th>
<th>Apical meristem </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">Gene expression intensity </td>
<td>Weak </td>
<td>80 </td>
<td>33 </td>
<td>20 </td>
<td>4 </td>
<td>0 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Strong </td>
<td>7 </td>
<td>25 </td>
<td>2 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>2 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T4" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3"> </th>
<th rowspan="3"> </th>
<th colspan="7">Gene expression profile<hr> </th>
</tr>
<tr>
<th rowspan="2">Constitutive </th>
<th colspan="6">Tissue specific<hr> </th>
</tr>
<tr>
<th>Leaf </th>
<th>Root </th>
<th>Node </th>
<th>Internode </th>
<th>Bud </th>
<th>Apical meristem </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">Gene expression intensity </td>
<td>Weak </td>
<td>80 </td>
<td>33 </td>
<td>20 </td>
<td>4 </td>
<td>0 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Strong </td>
<td>7 </td>
<td>25 </td>
<td>2 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>2 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025589" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="137025590" id="137025590" class="section-title">3.7. Comparison with sugarcane databases</h3>
<p>To identify <em>S. spontaneum</em> genes that were not described previously, we compared the HC genes produced by <em>Saccharum spontaneum 1.0</em> with the genes from the Sugarcane genome<a reveal-id="dsz001-B15" data-open="dsz001-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> (25,316 sequences) and the transcriptome sequences from SUCEST-FUN<a reveal-id="dsz001-B12" data-open="dsz001-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> (43,141 sequences) and ORFeome<a reveal-id="dsz001-B13" data-open="dsz001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> (195,765 sequences) (<a reveal-id="dsz001-F4" data-open="dsz001-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), resulting in 1,913 genes that were identified only in our assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576229860&amp;Signature=MbzSlR2ca4To5WqoismI10dxPA3nmAmbEAJfxgH6ev2Za1a691u6c8QlC9xnz9nbCHCAzaazCGPD1apKY7fPyqXVH0fCjXJJJlp1Z2jO8h5UztKMJlEFuaf5ZoMDGcRPZ9fDKwTOtgZ7DA9~GRI9yA9BFuGiWXLbp7IzgK56lO~jTDjGK0dpUtFCHPxZNSwPuwbNMMJRKcemg4Zof~eKGn5T6wYZxgAHoS~yBAdElcOxjllqGF7Rh791S1l4UJ9ZpH5waRawMhTQz0tX4YR0p5HSe8vkVcclKZkF4~I5Uni9jlYbmIBSd1zNiYZPK-JRYsmhJRYhDrhlyZpPWoxxGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15</a></span>). From those genes, 536 have known functions with proteins already characterized in other plants (mainly monocots), 502 were classified as hypothetical proteins, and 926 were no hits (<a reveal-id="dsz001-F4" data-open="dsz001-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Moreover, the expression values of the 1,913 genes in <em>S. spontaneum</em> (<a reveal-id="dsz001-F4" data-open="dsz001-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>), show distinct pattern across the selected tissues (apical meristem, lateral bud, internode, node, leaf and root). </p>                        <a id="137025592" scrollto-destination="137025592"></a>
<div id="dsz001-F4" data-id="dsz001-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f4.png?Expires=1576229860&amp;Signature=JRWrT~t~PJkuJRMwR6o2xHw~YhnGIdfhqUxhALKU4UtOcTpALQQY~kh9uKg5eT-fAG8x0jj3~mB1XtOew5QknWpb2dhVfvO41nt8EiUAofdEdl--Bh-EqtnVmCQW~FCr8y220ixs69y2iCYAed1LTalAgt671WxrscOBjADjssZZUK9Z3NjLFLw7orO1GkL9AqLleNleXBmtJWIdEX5ksOStHW5S78x8b90P3IaFdI~VP1TIPEYidU-H0WZmfRQmZt4y5GvRVVO7FCVKl-DcvbTX61koZ~sv-8~TF0vd0i565GonYTnj-1c5Z0eUtec0tsmUYdcdE7zM0-avNkDKuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="New contributions for sugarcane databases. (A) Results from the comparison of Saccharum spontaneum 1.0 assemble against Sugarcane databases (Gene prediction from sugarcane genome, SUCEST-FUN and ORFeome) and (B) the expression pattern of new identified genes in different tissues." path-from-xml="dsz001f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025592/dsz001f4.tif" path-from-xml="dsz001f4.tif" target="_blank">Open in new tab</a><a section="137025592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f4.png?Expires=1576229860&amp;Signature=UA-dlthoucN81gmIvdAgHgj5mJn6qY79HWu4V2A1z0ZS4Cm1ZUzf4cS30LNqBEHo62OuZDkQ-vob56pwHOZNmC22mlaZCxuoUIESYasRUKmCHAy3ITTYInJ6N1HUbKe2BtomxhKa00JfY~T65HUAJ~7yOIeaUfnW3pbDtbHWVRmT-lYHV3ZmAWbid~vD0p79HaUfS5D1L2r~yq6ZKlb663-kQFsGfVL-fiq-iocQq99KuCiF3W2R0NkAxnAZ17YQAND3O~g~upEcz6skL86VUUKoYvZxOFiLLPf7kQwAfM5cmlhz8ZDVZ76rA8hs8IpN-Xs4HHHb6ulhDYOPegH4zQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025592&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>New contributions for sugarcane databases. (A) Results from the comparison of <em>Saccharum spontaneum 1.0</em> assemble against Sugarcane databases (Gene prediction from sugarcane genome, SUCEST-FUN and ORFeome) and (B) the expression pattern of new identified genes in different tissues.</p></div>
</div>
<div content-id="dsz001-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/m_dsz001f4.png?Expires=1576229860&amp;Signature=JRWrT~t~PJkuJRMwR6o2xHw~YhnGIdfhqUxhALKU4UtOcTpALQQY~kh9uKg5eT-fAG8x0jj3~mB1XtOew5QknWpb2dhVfvO41nt8EiUAofdEdl--Bh-EqtnVmCQW~FCr8y220ixs69y2iCYAed1LTalAgt671WxrscOBjADjssZZUK9Z3NjLFLw7orO1GkL9AqLleNleXBmtJWIdEX5ksOStHW5S78x8b90P3IaFdI~VP1TIPEYidU-H0WZmfRQmZt4y5GvRVVO7FCVKl-DcvbTX61koZ~sv-8~TF0vd0i565GonYTnj-1c5Z0eUtec0tsmUYdcdE7zM0-avNkDKuw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="New contributions for sugarcane databases. (A) Results from the comparison of Saccharum spontaneum 1.0 assemble against Sugarcane databases (Gene prediction from sugarcane genome, SUCEST-FUN and ORFeome) and (B) the expression pattern of new identified genes in different tissues." path-from-xml="dsz001f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025592/dsz001f4.tif" path-from-xml="dsz001f4.tif" target="_blank">Open in new tab</a><a section="137025592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001f4.png?Expires=1576229860&amp;Signature=UA-dlthoucN81gmIvdAgHgj5mJn6qY79HWu4V2A1z0ZS4Cm1ZUzf4cS30LNqBEHo62OuZDkQ-vob56pwHOZNmC22mlaZCxuoUIESYasRUKmCHAy3ITTYInJ6N1HUbKe2BtomxhKa00JfY~T65HUAJ~7yOIeaUfnW3pbDtbHWVRmT-lYHV3ZmAWbid~vD0p79HaUfS5D1L2r~yq6ZKlb663-kQFsGfVL-fiq-iocQq99KuCiF3W2R0NkAxnAZ17YQAND3O~g~upEcz6skL86VUUKoYvZxOFiLLPf7kQwAfM5cmlhz8ZDVZ76rA8hs8IpN-Xs4HHHb6ulhDYOPegH4zQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025592&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz001f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>New contributions for sugarcane databases. (A) Results from the comparison of <em>Saccharum spontaneum 1.0</em> assemble against Sugarcane databases (Gene prediction from sugarcane genome, SUCEST-FUN and ORFeome) and (B) the expression pattern of new identified genes in different tissues.</p></div>
</div>
<p><em>Saccharum spontaneum</em> is one of the most robust grasses, having a high tolerance to abiotic and biotic stresses, mainly drought resistance and the ability to grow in nutrient-poor soils.<a reveal-id="dsz001-B9" data-open="dsz001-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Remarkable, from the genes presenting high expression values (≥50 FPKM), several are involved in stress response and 135 new genes were identified in roots. The ability to cope with several stresses of <em>S. spontaneum</em> could be a consequence of its large and dense root system when compared with <em>S. officinarum</em> and the commercial varieties of sugarcane, which make possible the soil exploitation and the higher uptake of water and nutrients. In addition, several genes could be related to symbiosis establishment (<a reveal-id="dsz001-T5" data-open="dsz001-T5" class="link link-reveal link-table xref-fig">Table 5</a>), recently described to increased drought resistance of <em>S. spontaneum.</em><a reveal-id="dsz001-B66" data-open="dsz001-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Among those genes, we can highlight genes related to recognition and early steps of symbiosis establishment (SS34074, SS37878, SS50399); stress and disease response (SS11486, SS38841, SS35410, SS41259) and cell wall degradation (SS48435); electron transfer and primary metabolism (SS35024, SS38176). Genes coding for nutrient transporter between fungus and plant were also identified (SS28265, SS11782, SS31510, SS13699), although they were already annotated in SUCEST-FUN<a reveal-id="dsz001-B12" data-open="dsz001-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and ORFeome<a reveal-id="dsz001-B13" data-open="dsz001-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> databases.</p>                        <a id="137025594" scrollto-destination="137025594"></a>
<div content-id="dsz001-T5" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T5" class="table-wrap-title" data-id="dsz001-T5">
<div class="label">Table 5</div>
<div class="caption"><p>Probable symbiosis-related genes expressed in <em>S. spontaneum</em> roots</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Gene ID </th>
<th>Protein/function </th>
<th>Expression (RPKM) </th>
<th>Reference </th>
</tr></thead>
<tbody>
<tr>
<td>SS43844 </td>
<td>MATH domain protein </td>
<td>6.33 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS34074</p></li>
<li><p>SS37878</p></li>
<li><p>SS50399</p></li>
</ul> </td>
<td>LRR receptor </td>
<td>
<ul class="list-simple">
<li><p>6.66</p></li>
<li><p>2.88</p></li>
<li><p>2.04</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS37936 </td>
<td>LysM domain </td>
<td>2.81 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS50999 </td>
<td>Calcium and calmodulin-dependent protein kinase </td>
<td>4.59 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS11486 </td>
<td>Salt stress-induced proteins </td>
<td>19.09 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS38841</p></li>
<li><p>SS32869</p></li>
<li><p>SS35410</p></li>
</ul> </td>
<td>Jacalin-like lectin </td>
<td>
<ul class="list-simple">
<li><p>20.13</p></li>
<li><p>16.06</p></li>
<li><p>2.38</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B63" data-open="dsz001-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> </td>
</tr>
<tr>
<td>SS41259 </td>
<td>Glutathione S-transferase </td>
<td>1.31 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS15872</p></li>
<li><p>SS42099</p></li>
</ul> </td>
<td>WRKY transcription factor </td>
<td>
<ul class="list-simple">
<li><p>3.56</p></li>
<li><p>2.01</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B64" data-open="dsz001-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsz001-B65" data-open="dsz001-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> </td>
</tr>
<tr>
<td>SS35024 </td>
<td>Blue copper binding proteins </td>
<td>23.70 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS38176 </td>
<td>Lipid transfer protein </td>
<td>210.90 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS33290 </td>
<td>Arabinogalactan </td>
<td>10.50 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS48435</p></li>
<li><p>SS46591</p></li>
</ul> </td>
<td>Pectinesterase </td>
<td>
<ul class="list-simple">
<li><p>2.92</p></li>
<li><p>1.09</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS47675 </td>
<td>Xyloglucan fucosyltransferase </td>
<td>2.13 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T5" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Gene ID </th>
<th>Protein/function </th>
<th>Expression (RPKM) </th>
<th>Reference </th>
</tr></thead>
<tbody>
<tr>
<td>SS43844 </td>
<td>MATH domain protein </td>
<td>6.33 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS34074</p></li>
<li><p>SS37878</p></li>
<li><p>SS50399</p></li>
</ul> </td>
<td>LRR receptor </td>
<td>
<ul class="list-simple">
<li><p>6.66</p></li>
<li><p>2.88</p></li>
<li><p>2.04</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS37936 </td>
<td>LysM domain </td>
<td>2.81 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS50999 </td>
<td>Calcium and calmodulin-dependent protein kinase </td>
<td>4.59 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS11486 </td>
<td>Salt stress-induced proteins </td>
<td>19.09 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS38841</p></li>
<li><p>SS32869</p></li>
<li><p>SS35410</p></li>
</ul> </td>
<td>Jacalin-like lectin </td>
<td>
<ul class="list-simple">
<li><p>20.13</p></li>
<li><p>16.06</p></li>
<li><p>2.38</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B63" data-open="dsz001-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> </td>
</tr>
<tr>
<td>SS41259 </td>
<td>Glutathione S-transferase </td>
<td>1.31 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS15872</p></li>
<li><p>SS42099</p></li>
</ul> </td>
<td>WRKY transcription factor </td>
<td>
<ul class="list-simple">
<li><p>3.56</p></li>
<li><p>2.01</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B64" data-open="dsz001-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsz001-B65" data-open="dsz001-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> </td>
</tr>
<tr>
<td>SS35024 </td>
<td>Blue copper binding proteins </td>
<td>23.70 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS38176 </td>
<td>Lipid transfer protein </td>
<td>210.90 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS33290 </td>
<td>Arabinogalactan </td>
<td>10.50 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS48435</p></li>
<li><p>SS46591</p></li>
</ul> </td>
<td>Pectinesterase </td>
<td>
<ul class="list-simple">
<li><p>2.92</p></li>
<li><p>1.09</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS47675 </td>
<td>Xyloglucan fucosyltransferase </td>
<td>2.13 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025594" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz001-T5" class="table-wrap-title" data-id="dsz001-T5">
<div class="label">Table 5</div>
<div class="caption"><p>Probable symbiosis-related genes expressed in <em>S. spontaneum</em> roots</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Gene ID </th>
<th>Protein/function </th>
<th>Expression (RPKM) </th>
<th>Reference </th>
</tr></thead>
<tbody>
<tr>
<td>SS43844 </td>
<td>MATH domain protein </td>
<td>6.33 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS34074</p></li>
<li><p>SS37878</p></li>
<li><p>SS50399</p></li>
</ul> </td>
<td>LRR receptor </td>
<td>
<ul class="list-simple">
<li><p>6.66</p></li>
<li><p>2.88</p></li>
<li><p>2.04</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS37936 </td>
<td>LysM domain </td>
<td>2.81 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS50999 </td>
<td>Calcium and calmodulin-dependent protein kinase </td>
<td>4.59 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS11486 </td>
<td>Salt stress-induced proteins </td>
<td>19.09 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS38841</p></li>
<li><p>SS32869</p></li>
<li><p>SS35410</p></li>
</ul> </td>
<td>Jacalin-like lectin </td>
<td>
<ul class="list-simple">
<li><p>20.13</p></li>
<li><p>16.06</p></li>
<li><p>2.38</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B63" data-open="dsz001-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> </td>
</tr>
<tr>
<td>SS41259 </td>
<td>Glutathione S-transferase </td>
<td>1.31 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS15872</p></li>
<li><p>SS42099</p></li>
</ul> </td>
<td>WRKY transcription factor </td>
<td>
<ul class="list-simple">
<li><p>3.56</p></li>
<li><p>2.01</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B64" data-open="dsz001-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsz001-B65" data-open="dsz001-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> </td>
</tr>
<tr>
<td>SS35024 </td>
<td>Blue copper binding proteins </td>
<td>23.70 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS38176 </td>
<td>Lipid transfer protein </td>
<td>210.90 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS33290 </td>
<td>Arabinogalactan </td>
<td>10.50 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS48435</p></li>
<li><p>SS46591</p></li>
</ul> </td>
<td>Pectinesterase </td>
<td>
<ul class="list-simple">
<li><p>2.92</p></li>
<li><p>1.09</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS47675 </td>
<td>Xyloglucan fucosyltransferase </td>
<td>2.13 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz001-T5" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Gene ID </th>
<th>Protein/function </th>
<th>Expression (RPKM) </th>
<th>Reference </th>
</tr></thead>
<tbody>
<tr>
<td>SS43844 </td>
<td>MATH domain protein </td>
<td>6.33 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS34074</p></li>
<li><p>SS37878</p></li>
<li><p>SS50399</p></li>
</ul> </td>
<td>LRR receptor </td>
<td>
<ul class="list-simple">
<li><p>6.66</p></li>
<li><p>2.88</p></li>
<li><p>2.04</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS37936 </td>
<td>LysM domain </td>
<td>2.81 </td>
<td>
<a reveal-id="dsz001-B60" data-open="dsz001-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> </td>
</tr>
<tr>
<td>SS50999 </td>
<td>Calcium and calmodulin-dependent protein kinase </td>
<td>4.59 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS11486 </td>
<td>Salt stress-induced proteins </td>
<td>19.09 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS38841</p></li>
<li><p>SS32869</p></li>
<li><p>SS35410</p></li>
</ul> </td>
<td>Jacalin-like lectin </td>
<td>
<ul class="list-simple">
<li><p>20.13</p></li>
<li><p>16.06</p></li>
<li><p>2.38</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B63" data-open="dsz001-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> </td>
</tr>
<tr>
<td>SS41259 </td>
<td>Glutathione S-transferase </td>
<td>1.31 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS15872</p></li>
<li><p>SS42099</p></li>
</ul> </td>
<td>WRKY transcription factor </td>
<td>
<ul class="list-simple">
<li><p>3.56</p></li>
<li><p>2.01</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B64" data-open="dsz001-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsz001-B65" data-open="dsz001-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> </td>
</tr>
<tr>
<td>SS35024 </td>
<td>Blue copper binding proteins </td>
<td>23.70 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS38176 </td>
<td>Lipid transfer protein </td>
<td>210.90 </td>
<td>
<a reveal-id="dsz001-B61" data-open="dsz001-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> </td>
</tr>
<tr>
<td>SS33290 </td>
<td>Arabinogalactan </td>
<td>10.50 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>
<ul class="list-simple">
<li><p>SS48435</p></li>
<li><p>SS46591</p></li>
</ul> </td>
<td>Pectinesterase </td>
<td>
<ul class="list-simple">
<li><p>2.92</p></li>
<li><p>1.09</p></li>
</ul> </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
<tr>
<td>SS47675 </td>
<td>Xyloglucan fucosyltransferase </td>
<td>2.13 </td>
<td>
<a reveal-id="dsz001-B62" data-open="dsz001-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025594" target="_blank">Open in new tab</a></div>
</div></div>
<p>Between other new genes, expressed exclusively in roots cells, we identified one corresponding to a Late Embryogenesis Abundant (LEA) (SS1022; 28.01 ± 4.12 RPKM). Characterized as a hydrophilic protein widely distributed in the plant kingdom, LEA is accumulated in vegetative tissues as a response to water limitation due to salinity, drought or high temperatures.<a reveal-id="dsz001-B67" data-open="dsz001-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Other significant genes annotated by our analysis and expressed only in <em>S. spontaneum</em> roots are the BURP domain-containing genes, specifically; those related to families 10 and 13 (BURP10 and BURP13). BURP domain-containing genes are related to many functions in plants, in <em>O. sativa</em>, the BURP10 was reposted to be highly expressed in roots at the tillering stage.<a reveal-id="dsz001-B68" data-open="dsz001-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Our results also demonstrated the expression of BURP10 (SS23435) in roots tissue (around 17.23 ± 3.12 RPKM), suggesting that this gene could be responsible for the dense root system and tillering potential described in <em>S. spontaneum</em>.</p>                    <h2 scrollto-destination="137025596" id="137025596" class="section-title">4. Conclusions</h2>
<p>Even with large advances in the sequencing technologies, the assembly of complex genomes still represents a bottleneck, mainly due to polyploidy and high heterozygosity. The development of new bioinformatics efforts can contribute to overcoming these constraints, especially, for complete genomes of the closely related organism, in which the methods based on reference assembly can be applied. Using the PGA pipeline, we provided a high-quality assembly of gene regions in the <em>T. aestivum</em> and <em>S. spontaneum</em>, proving that PGA can be more efficient than conventional genome assemblers in cases of complex genomes and using low coverage DNA sequencing. The low memory requirement by PGA in comparison with <em>de novo</em> assembly strategy is also an advantage. Our findings of <em>S. spontaneum</em> genome highlighted for the first time the molecular basis of some noteworthy features of this biomass, like the high productivity and the resistance face biotic and abiotic stress. Those results can be employed in future functional and genetic studies beyond supporting the development of new varieties of sugarcane for the agronomic industry.</p>                    <h2 scrollto-destination="137025598" id="137025598" class="section-title">Availability of data</h2>
<p>The DNA and RNA Illumina reads were submitted to SRA (Sequence Read Archive) from NCBI under BioProject accession number PRJNA474618. Also, <em>S. spontaneum</em> genome and gene prediction are available to download at the Saccharum database (<a class="link link-uri" href="http://www.lge.ibi.unicamp.br/spontaneum" target="_blank">http://www.lge.ibi.unicamp.br/spontaneum</a> - date last accessed 25 January 2019) constructed using the scripts provided by EUCANEXT.<a reveal-id="dsz001-B69" data-open="dsz001-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a></p>                    <h2 scrollto-destination="137025600" id="137025600" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>We are also thankful to (i) Life Sciences Core Facility (LaCTAD), University of Campinas (UNICAMP) for the computational infrastructure and (ii) GranBio Investimentos S.A. for <em>S. spontaneum</em> accessions.</p>                    <h3 scrollto-destination="137025602" id="137025602" class="section-title">Funding</h3>
<p>We thank the Brazilian National Council for Scientific and Technological Development (CNPq–RHAE, grants number 350474/2013-3 and 35081/2015-1), the Sao Paulo Research Foundation (FAPESP, grant numbers 2014/09638-0 and 2012/05890-1) and Center for Computational Engineering and Sciences—FAPESP/Cepid (2013/08293-7) for the financial support and scholarships.</p>                    <h3 scrollto-destination="137025604" id="137025604" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="137025606" id="137025606" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz001-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gregory</div> <div class="given-names">T.R.</div>
</div>, <div class="name string-name">
<div class="surname">Nicol</div> <div class="given-names">J.A.</div>
</div>, <div class="name string-name">
<div class="surname">Tamm</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Eukaryotic genome size databases</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">D332</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Eukaryotic%20genome%20size%20databases&amp;author=T.R.%20Gregory&amp;author=J.A.%20Nicol&amp;author=H.%20Tamm&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D332-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkl828" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl828" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl828"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17090588" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Eukaryotic%20genome%20size%20databases&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gore</div> <div class="given-names">M.A.</div>
</div>, <div class="name string-name">
<div class="surname">Chia</div> <div class="given-names">J.-M.</div>
</div>, <div class="name string-name">
<div class="surname">Elshire</div> <div class="given-names">R.J.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">A First-Generation Haplotype Map of Maize</div>, <div class="source">Science</div> , <div class="volume">326</div>:<div class="fpage">1115</div>–<div class="lpage">1117</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Science&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Science&amp;aulast=Gore&amp;date=2009&amp;spage=1115&amp;epage=1117&amp;volume=326" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Science&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Science">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schatz</div> <div class="given-names">M.C.</div>
</div>, <div class="name string-name">
<div class="surname">Witkowski</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">McCombie</div> <div class="given-names">W.R.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Current challenges in <em>de novo</em> plant genome sequencing and assembly</div>, <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">243</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Current%20challenges%20in%20de%20novo%20plant%20genome%20sequencing%20and%20assembly&amp;author=M.C.%20Schatz&amp;author=J.%20Witkowski&amp;author=W.R.%20McCombie&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=243" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2012-13-4-243" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-4-243" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2012-13-4-243"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22546054" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Current%20challenges%20in%20de%20novo%20plant%20genome%20sequencing%20and%20assembly&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">International Wheat Genome Sequencing Consortium (IWGSC). <div class="year">2014</div>, 
<div class="article-title">A chromosome-based draft sequence of the hexaploid bread wheat (<em>Triticum aestivum</em>) genome</div>, <div class="source">Science</div> , <div class="volume">18</div>, <div class="fpage">1251788</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20chromosome-based%20draft%20sequence%20of%20the%20hexaploid%20bread%20wheat%20(Triticum%20aestivum)%20genome&amp;date=2014&amp;spage=1251788&amp;volume=18" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20chromosome-based%20draft%20sequence%20of%20the%20hexaploid%20bread%20wheat%20(Triticum%20aestivum)%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nystedt</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Street</div> <div class="given-names">N.R.</div>
</div>, <div class="name string-name">
<div class="surname">Wetterbom</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">The Norway spruce genome sequence and conifer genome evolution</div>, <div class="source">Nature</div> , <div class="volume">497</div>, <div class="fpage">579</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Norway%20spruce%20genome%20sequence%20and%20conifer%20genome%20evolution&amp;author=B.%20Nystedt&amp;author=N.R.%20Street&amp;author=A.%20Wetterbom&amp;publication_year=2013&amp;journal=Nature&amp;volume=497&amp;pages=579-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature12211" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12211" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature12211"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23698360" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Norway%20spruce%20genome%20sequence%20and%20conifer%20genome%20evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Larkin</div> <div class="given-names">D.M.</div>
</div>, <div class="name string-name">
<div class="surname">Cai</div> <div class="given-names">Q.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Reference-assisted chromosome assembly</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">110</div>, <div class="fpage">1785</div>–<div class="lpage">90</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Reference-assisted%20chromosome%20assembly&amp;author=J.%20Kim&amp;author=D.M.%20Larkin&amp;author=Q.%20Cai&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=110&amp;pages=1785-90" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1220349110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1220349110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1220349110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23307812" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Reference-assisted%20chromosome%20assembly&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">D’Hont</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Glaszmann</div> <div class="given-names">J.C.</div>
</div>
</div> <div class="year">2001</div>, 
<div class="article-title">Sugarcane genome analysis with molecular markers: a first decade of research</div>, <div class="source">Proc. Int. Soc. Sugar Cane Technol</div> ., <div class="volume">24</div>, <div class="fpage">556</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Sugarcane%20genome%20analysis%20with%20molecular%20markers%3A%20a%20first%20decade%20of%20research&amp;aulast=D%E2%80%99Hont&amp;date=2001&amp;spage=556&amp;epage=9&amp;volume=24" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sugarcane%20genome%20analysis%20with%20molecular%20markers%3A%20a%20first%20decade%20of%20research&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garsmeur</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Charron</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Bocs</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">High homologous gene conservation despite extreme autopolyploid redundancy in sugarcane</div>, <div class="source">New Phytol</div> ., <div class="volume">189</div>, <div class="fpage">629</div>–<div class="lpage">42</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=High%20homologous%20gene%20conservation%20despite%20extreme%20autopolyploid%20redundancy%20in%20sugarcane&amp;author=O.%20Garsmeur&amp;author=C.%20Charron&amp;author=S.%20Bocs&amp;publication_year=2011&amp;journal=New%20Phytol&amp;volume=189&amp;pages=629-42" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1469-8137.2010.03497.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03497.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1469-8137.2010.03497.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21039564" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:High%20homologous%20gene%20conservation%20despite%20extreme%20autopolyploid%20redundancy%20in%20sugarcane&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grivet</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Arruda</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">Sugarcane Genomics: Depicting the Complex Genome of an Important Tropical Crop</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">5</div>, <div class="fpage">122</div>–<div class="lpage">127</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Curr.%20Opin.%20Plant%20Biol.&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Curr.%20Opin.%20Plant%20Biol.&amp;aulast=Grivet&amp;date=2002&amp;spage=122&amp;epage=127&amp;volume=5" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Curr.%20Opin.%20Plant%20Biol.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Curr.%20Opin.%20Plant%20Biol.">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Carvalho-Netto</div> <div class="given-names">O.V.</div>
</div>, <div class="name string-name">
<div class="surname">Bressiani</div> <div class="given-names">J.A.</div>
</div>, <div class="name string-name">
<div class="surname">Soriano</div> <div class="given-names">H.L.</div>
</div>
</div>, et al. <div class="year">2014</div>, 
<div class="article-title">The Potential of the Energy Cane as the Main Biomass Crop for the Cellulosic Industry</div>, <div class="source">Chem. Biol. Technol. Agric.</div> , <div class="volume">1</div>:<div class="fpage">20</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Chem.%20Biol.%20Technol.%20Agric.&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Chem.%20Biol.%20Technol.%20Agric.&amp;aulast=Carvalho-Netto&amp;date=2014&amp;spage=20&amp;volume=1" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Chem.%20Biol.%20Technol.%20Agric.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Chem.%20Biol.%20Technol.%20Agric.">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sreenivasan</div> <div class="given-names">T.V.</div>
</div>, <div class="name string-name">
<div class="surname">Ahloowalia</div> <div class="given-names">B.S.</div>
</div>
</div> <div class="year">1987</div>, Cytogenetics. In: <div class="person-group"><div class="name string-name">
<div class="surname">Heinz</div> <div class="given-names">D.J.</div>
</div></div>, ed. <div class="source">Sugarcane Improvement through Breeding</div> , pp. <div class="fpage">211</div>–<div class="lpage">53</div>. 
<div class="publisher-name">Elsevier</div>: 
<div class="publisher-loc">Amsterdam</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Sugarcane%20Improvement%20through%20Breeding&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Sugarcane%20Improvement%20through%20Breeding&amp;aulast=Sreenivasan&amp;date=1987&amp;spage=211&amp;epage=53" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sugarcane%20Improvement%20through%20Breeding&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Sugarcane%20Improvement%20through%20Breeding">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nishiyama</div> <div class="given-names">M.Y.</div>Jr</div>, <div class="name string-name">
<div class="surname">Vicente</div> <div class="given-names">F.F.R.</div>
</div>, <div class="name string-name">
<div class="surname">Lembke</div> <div class="given-names">C.G.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">The SUCEST-FUN regulatory network database: designing an energy grass</div>, <div class="source">Proc. Int. Soc. Sugar Cane Technol</div> ., <div class="volume">27</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20SUCEST-FUN%20regulatory%20network%20database%3A%20designing%20an%20energy%20grass&amp;aulast=Nishiyama&amp;date=2010&amp;spage=1&amp;epage=10&amp;volume=27" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20SUCEST-FUN%20regulatory%20network%20database%3A%20designing%20an%20energy%20grass&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nishiyama</div> <div class="given-names">M.Y.</div>Jr</div>, <div class="name string-name">
<div class="surname">Ferreira</div> <div class="given-names">S.S.</div>
</div>, <div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">P.-Z.</div>
</div>, <div class="name string-name">
<div class="surname">Becker</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Pörtner-Taliana</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Souza</div> <div class="given-names">G.M.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Full-length enriched cDNA libraries and ORFeome analysis of sugarcane hybrid and ancestor genotypes</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e107351</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Full-length%20enriched%20cDNA%20libraries%20and%20ORFeome%20analysis%20of%20sugarcane%20hybrid%20and%20ancestor%20genotypes&amp;author=M.Y.%20Nishiyama&amp;author=S.S.%20Ferreira&amp;author=P.-Z.%20Tang&amp;author=S.%20Becker&amp;author=A.%20P%C3%B6rtner-Taliana&amp;author=G.M.%20Souza&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e107351" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0107351" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0107351" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0107351"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25222706" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Full-length%20enriched%20cDNA%20libraries%20and%20ORFeome%20analysis%20of%20sugarcane%20hybrid%20and%20ancestor%20genotypes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Miller</div> <div class="given-names">J.R.</div>
</div>, <div class="name string-name">
<div class="surname">Dilley</div> <div class="given-names">K.A.</div>
</div>, <div class="name string-name">
<div class="surname">Harkins</div> <div class="given-names">D.M.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Initial genome sequencing of the sugarcane CP 96-1252 complex hybrid</div>, <div class="source">F1000Res</div> ., <div class="volume">6</div>, <div class="fpage">688</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Initial%20genome%20sequencing%20of%20the%20sugarcane%20CP%2096-1252%20complex%20hybrid&amp;author=J.R.%20Miller&amp;author=K.A.%20Dilley&amp;author=D.M.%20Harkins&amp;publication_year=2017&amp;journal=F1000Res&amp;volume=6&amp;pages=688" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.12688/f1000research" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.12688%2Ff1000research" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.12688%2Ff1000research"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28721204" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Initial%20genome%20sequencing%20of%20the%20sugarcane%20CP%2096-1252%20complex%20hybrid&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garsmeur</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Droc</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Antonise</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">A mosaic monoploid reference sequence for the highly complex genome of sugarcane</div>, <div class="source">Nat. Commun</div> ., <div class="volume">9</div>, <div class="fpage">2638</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20mosaic%20monoploid%20reference%20sequence%20for%20the%20highly%20complex%20genome%20of%20sugarcane&amp;author=O.%20Garsmeur&amp;author=G.%20Droc&amp;author=R.%20Antonise&amp;publication_year=2018&amp;journal=Nat.%20Commun&amp;volume=9&amp;pages=2638" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41467-018-05051-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41467-018-05051-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41467-018-05051-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29980662" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20mosaic%20monoploid%20reference%20sequence%20for%20the%20highly%20complex%20genome%20of%20sugarcane&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">H.</div>
</div>
</div>, et al. <div class="year">2018</div>, 
<div class="article-title">Allele-defined genome of the autopolyploid sugarcane <em>Saccharum spontaneum</em> L</div>., <div class="source">Nat. Genet</div> ., <div class="volume">50</div>:<div class="fpage">1565</div>–<div class="lpage">1573</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Allele-defined%20genome%20of%20the%20autopolyploid%20sugarcane%20Saccharum%20spontaneum%20L&amp;author=J.%20Zhang&amp;author=X.%20Zhang&amp;author=H.%20Tang&amp;publication_year=2018&amp;journal=Nat.%20Genet&amp;volume=50&amp;pages=1565-1573" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41588-018-0237-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41588-018-0237-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41588-018-0237-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30297971" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Allele-defined%20genome%20of%20the%20autopolyploid%20sugarcane%20Saccharum%20spontaneum%20L&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zeng</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">RNA isolation from highly viscous samples rich in polyphenols and polysaccharides</div>, <div class="source">Plant Mol. Biol. Rep</div> ., <div class="volume">20</div>, <div class="fpage">417</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RNA%20isolation%20from%20highly%20viscous%20samples%20rich%20in%20polyphenols%20and%20polysaccharides&amp;author=Y.%20Zeng&amp;author=T.%20Yang&amp;publication_year=2002&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=20&amp;pages=417" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF02772130" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02772130" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02772130"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RNA%20isolation%20from%20highly%20viscous%20samples%20rich%20in%20polyphenols%20and%20polysaccharides&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Le Provost</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Herrera</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Paiva</div> <div class="given-names">J.A.</div>
</div>, <div class="name string-name">
<div class="surname">Chaumeil</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Salin</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Plomion</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">A micromethod for high throughput RNA extraction in forest trees</div>, <div class="source">Biol. Res</div> ., <div class="volume">40</div>, <div class="fpage">291</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20micromethod%20for%20high%20throughput%20RNA%20extraction%20in%20forest%20trees&amp;author=G.%20Le%20Provost&amp;author=R.%20Herrera&amp;author=J.A.%20Paiva&amp;author=P.%20Chaumeil&amp;author=F.%20Salin&amp;author=C.%20Plomion&amp;publication_year=2007&amp;journal=Biol.%20Res&amp;volume=40&amp;pages=291-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.4067/S0716-97602007000400003" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4067%2FS0716-97602007000400003" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.4067%2FS0716-97602007000400003"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18449457" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20micromethod%20for%20high%20throughput%20RNA%20extraction%20in%20forest%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Carvalho</div> <div class="given-names">C.R.</div>
</div>, <div class="name string-name">
<div class="surname">Clarindo</div> <div class="given-names">W.R.</div>
</div>, <div class="name string-name">
<div class="surname">Praça</div> <div class="given-names">M.M.</div>
</div>, <div class="name string-name">
<div class="surname">Araújo</div> <div class="given-names">F.S.</div>
</div>, <div class="name string-name">
<div class="surname">Carels</div> <div class="given-names">N.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Genome size, base composition and karyotype of <em>Jatropha curcas</em> L., an important biofuel plant</div>, <div class="source">Plant Sci</div> ., <div class="volume">174</div>, <div class="fpage">613</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20size%2C%20base%20composition%20and%20karyotype%20of%20Jatropha%20curcas%20L.%2C%20an%20important%20biofuel%20plant&amp;author=C.R.%20Carvalho&amp;author=W.R.%20Clarindo&amp;author=M.M.%20Pra%C3%A7a&amp;author=F.S.%20Ara%C3%BAjo&amp;author=N.%20Carels&amp;publication_year=2008&amp;journal=Plant%20Sci&amp;volume=174&amp;pages=613-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.plantsci.2008.03.010" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2008.03.010" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.plantsci.2008.03.010"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20size%2C%20base%20composition%20and%20karyotype%20of%20Jatropha%20curcas%20L.%2C%20an%20important%20biofuel%20plant&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Langmead</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1923"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grabherr</div> <div class="given-names">M.G.</div>
</div>, <div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B.J.</div>
</div>, <div class="name string-name">
<div class="surname">Yassour</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.1883"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xue</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">J.-T.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">Y.-P.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">L_RNA_scaffolder: scaffolding genomes with transcripts</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">604</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=L_RNA_scaffolder%3A%20scaffolding%20genomes%20with%20transcripts&amp;author=W.%20Xue&amp;author=J.-T.%20Li&amp;author=Y.-P.%20Zhu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=604" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-14-604" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-604" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-14-604"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24010822" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:L_RNA_scaffolder%3A%20scaffolding%20genomes%20with%20transcripts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Kent</div> <div class="given-names">W.J.</div>
</div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.229202"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Boetzer</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Henkel</div> <div class="given-names">C.V.</div>
</div>, <div class="name string-name">
<div class="surname">Jansen</div> <div class="given-names">H.J.</div>
</div>, <div class="name string-name">
<div class="surname">Butler</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Pirovano</div> <div class="given-names">W.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq683"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Beier</div> <div class="given-names">M.S.</div>
</div>, <div class="name string-name">
<div class="surname">Himmelbach</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Colmsee</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Construction of a map-based reference genome sequence for barley, <em>Hordeum vulgare L</em>
</div>., <div class="source">Sci. Data</div> , <div class="volume">4</div>, <div class="fpage">170044</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20map-based%20reference%20genome%20sequence%20for%20barley%2C%20Hordeum%20vulgare%20L&amp;author=M.S.%20Beier&amp;author=A.%20Himmelbach&amp;author=C.%20Colmsee&amp;publication_year=2017&amp;journal=Sci.%20Data&amp;volume=4&amp;pages=170044" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/sdata.2017.44" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsdata.2017.44" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsdata.2017.44"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28448065" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Construction%20of%20a%20map-based%20reference%20genome%20sequence%20for%20barley%2C%20Hordeum%20vulgare%20L&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goodstein</div> <div class="given-names">D.M.</div>
</div>, <div class="name string-name">
<div class="surname">Shu</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Howson</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Phytozome: a comparative platform for green plant genomics</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D1178</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phytozome%3A%20a%20comparative%20platform%20for%20green%20plant%20genomics&amp;author=D.M.%20Goodstein&amp;author=S.%20Shu&amp;author=R.%20Howson&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1178-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr944" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr944" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr944"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phytozome%3A%20a%20comparative%20platform%20for%20green%20plant%20genomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Slater</div> <div class="given-names">G.S.C.M.</div>
</div>, <div class="name string-name">
<div class="surname">Birney</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Automated generation of heuristics for biological sequence comparison</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">6</div>, <div class="fpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.S.C.M.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinform&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-6-31"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Stanke</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Keller</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Gunduz</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Hayes</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Waack</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Morgenstern</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">AUGUSTUS: <em>ab initio</em> prediction of alternative transcripts</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W435</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;author=M.%20Stanke&amp;author=O.%20Keller&amp;author=I.%20Gunduz&amp;author=A.%20Hayes&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W435-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkl200" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl200" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl200"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845043" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Quinlan</div> <div class="given-names">A.R.</div>
</div>, <div class="name string-name">
<div class="surname">Hall</div> <div class="given-names">I.M.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">841</div>–<div class="lpage">2</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.R.%20Quinlan&amp;author=I.M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq033"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Suzek</div> <div class="given-names">B.E.</div>
</div>, <div class="name string-name">
<div class="surname">Huang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">McGarvey</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Mazumder</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">C.H.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">UniRef: comprehensive and non-redundant UniProt reference clusters</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1282</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=UniRef%3A%20comprehensive%20and%20non-redundant%20UniProt%20reference%20clusters&amp;author=B.E.%20Suzek&amp;author=H.%20Huang&amp;author=P.%20McGarvey&amp;author=R.%20Mazumder&amp;author=C.H.%20Wu&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1282-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btm098" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm098" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtm098"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379688" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:UniRef%3A%20comprehensive%20and%20non-redundant%20UniProt%20reference%20clusters&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Paterson</div> <div class="given-names">A.H.</div>
</div>, <div class="name string-name">
<div class="surname">Bowers</div> <div class="given-names">J.E.</div>
</div>, <div class="name string-name">
<div class="surname">Bruggmann</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses</div>, <div class="source">Nature</div> , <div class="volume">457</div>, <div class="fpage">551</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;author=A.H.%20Paterson&amp;author=J.E.%20Bowers&amp;author=R.%20Bruggmann&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=551-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature07723" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07723" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature07723"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schnable</div> <div class="given-names">P.S.</div>
</div>, <div class="name string-name">
<div class="surname">Ware</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Fulton</div> <div class="given-names">R.S.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">The B73 maize genome: complexity, diversity, and dynamics</div>, <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">1112</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1178534" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1178534"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Genome sequence of foxtail millet (<em>Setaria italica</em>) provides insights into grass evolution and biofuel potential</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">30</div>, <div class="fpage">549</div>–<div class="lpage">54</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20foxtail%20millet%20(Setaria%20italica)%20provides%20insights%20into%20grass%20evolution%20and%20biofuel%20potential&amp;author=J.%20Wang&amp;author=G.%20Zhang&amp;author=X.%20Liu&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol&amp;volume=30&amp;pages=549-54" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.2195" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2195" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.2195"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22580950" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20of%20foxtail%20millet%20(Setaria%20italica)%20provides%20insights%20into%20grass%20evolution%20and%20biofuel%20potential&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Ruan</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">
<em>De novo</em> assembly of human genomes with massively parallel short read sequencing</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">265</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;author=R.%20Li&amp;author=H.%20Zhu&amp;author=J.%20Ruan&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=265-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.097261.109" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.097261.109" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.097261.109"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019144" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Pertea</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Trapnell</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Pimentel</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Kelley</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>, <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2013-14-4-r36"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bairoch</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Apweiler</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2000</div>, 
<div class="article-title">The SWISS-PROT protein sequence database and its supplement TrEMBL in 2000</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">45</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20SWISS-PROT%20protein%20sequence%20database%20and%20its%20supplement%20TrEMBL%20in%202000&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=45-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/28.1.45" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.45" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F28.1.45"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592178" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20SWISS-PROT%20protein%20sequence%20database%20and%20its%20supplement%20TrEMBL%20in%202000&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ouyang</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Hamilton</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2007</div>, 
<div class="article-title">The TIGR rice genome annotation resource: improvements and new features</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">D883</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20TIGR%20rice%20genome%20annotation%20resource%3A%20improvements%20and%20new%20features&amp;author=S.%20Ouyang&amp;author=W.%20Zhu&amp;author=J.%20Hamilton&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D883-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkl976" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl976" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl976"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17145706" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20TIGR%20rice%20genome%20annotation%20resource%3A%20improvements%20and%20new%20features&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lamesch</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Berardini</div> <div class="given-names">T.Z.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D1202</div>–<div class="lpage">210</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20improved%20gene%20annotation%20and%20new%20tools&amp;author=P.%20Lamesch&amp;author=T.Z.%20Berardini&amp;author=D.%20Li&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1202-210" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1090" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1090" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1090"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140109" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20improved%20gene%20annotation%20and%20new%20tools&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cardoso-Silva</div> <div class="given-names">C.B.</div>
</div>, <div class="name string-name">
<div class="surname">Costa</div> <div class="given-names">E.A.</div>
</div>, <div class="name string-name">
<div class="surname">Mancini</div> <div class="given-names">M.C.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">
<em>De novo</em> assembly and transcriptome analysis of contrasting sugarcane varieties</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e88462</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20transcriptome%20analysis%20of%20contrasting%20sugarcane%20varieties&amp;author=C.B.%20Cardoso-Silva&amp;author=E.A.%20Costa&amp;author=M.C.%20Mancini&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e88462" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0088462" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0088462" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0088462"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24523899" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:De%20novo%20assembly%20and%20transcriptome%20analysis%20of%20contrasting%20sugarcane%20varieties&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B.J.</div>
</div>, <div class="name string-name">
<div class="surname">Delcher</div> <div class="given-names">A.L.</div>
</div>, <div class="name string-name">
<div class="surname">Mount</div> <div class="given-names">S.M.</div>
</div>
</div>, et al.  <div class="year">2003</div>, 
<div class="article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">5654</div>–<div class="lpage">66</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;author=B.J.%20Haas&amp;author=A.L.%20Delcher&amp;author=S.M.%20Mount&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=5654-66" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkg770" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg770" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkg770"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Simão</div> <div class="given-names">F.A.</div>
</div>, <div class="name string-name">
<div class="surname">Waterhouse</div> <div class="given-names">R.M.</div>
</div>, <div class="name string-name">
<div class="surname">Ioannidis</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Kriventseva</div> <div class="given-names">E.V.</div>
</div>, <div class="name string-name">
<div class="surname">Zdobnov</div> <div class="given-names">E.M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">12</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.A.%20Sim%C3%A3o&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-12" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtv351"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Conesa</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Götz</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Blast2GO: a comprehensive suite for functional analysis in plant genomics</div>, <div class="source">Int. J. Plant Genomics</div> , <div class="volume">2008</div>, <div class="fpage">619832</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20comprehensive%20suite%20for%20functional%20analysis%20in%20plant%20genomics&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;publication_year=2008&amp;journal=Int.%20J.%20Plant%20Genomics&amp;volume=2008&amp;pages=619832" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1155/2008/619832" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2008%2F619832" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1155%2F2008%2F619832"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18483572" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Blast2GO%3A%20a%20comprehensive%20suite%20for%20functional%20analysis%20in%20plant%20genomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Initiative</div> <div class="given-names">I.B.</div>
</div></div> <div class="year">2010</div>, 
<div class="article-title">Genome sequencing and analysis of the model grass <em>Brachypodium distachyon</em>
</div>, <div class="source">Nature</div> , <div class="volume">463</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Genome%20sequencing%20and%20analysis%20of%20the%20model%20grass%20Brachypodium%20distachyon&amp;aulast=Initiative&amp;date=2010&amp;volume=463" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequencing%20and%20analysis%20of%20the%20model%20grass%20Brachypodium%20distachyon&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Peng</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">The draft genome of the fast-growing non-timber forest species moso bamboo (<em>Phyllostachys heterocycla</em>)</div>, <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">456</div>–<div class="lpage">61</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20the%20fast-growing%20non-timber%20forest%20species%20moso%20bamboo%20(Phyllostachys%20heterocycla)&amp;author=Z.%20Peng&amp;author=Y.%20Lu&amp;author=L.%20Li&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=456-61" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.2569" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2569" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.2569"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23435089" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20draft%20genome%20of%20the%20fast-growing%20non-timber%20forest%20species%20moso%20bamboo%20(Phyllostachys%20heterocycla)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Stoeckert</div> <div class="given-names">C.J.</div>
</div>, <div class="name string-name">
<div class="surname">Roos</div> <div class="given-names">D.S.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.J.%20Stoeckert&amp;author=D.S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.1224503"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Katoh</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Standley</div> <div class="given-names">D.M.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">772</div>–<div class="lpage">80</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;author=K.%20Katoh&amp;author=D.M.%20Standley&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=772-80" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/mst010" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst010" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmst010"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Stamatakis</div> <div class="given-names">A.</div>
</div></div> <div class="year">2014</div>, 
<div class="article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1312</div>–<div class="lpage">13</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1312-13" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu033" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu033" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu033"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Librado</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Vieira</div> <div class="given-names">F.G.</div>
</div>, <div class="name string-name">
<div class="surname">Rozas</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">BadiRate: estimating family turnover rates by likelihood-based methods</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">279</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BadiRate%3A%20estimating%20family%20turnover%20rates%20by%20likelihood-based%20methods&amp;author=P.%20Librado&amp;author=F.G.%20Vieira&amp;author=J.%20Rozas&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=279-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr623" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr623" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr623"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22080468" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BadiRate%3A%20estimating%20family%20turnover%20rates%20by%20likelihood-based%20methods&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Trapnell</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Williams</div> <div class="given-names">B.A.</div>
</div>, <div class="name string-name">
<div class="surname">Pertea</div> <div class="given-names">G.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">15</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=511-15" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.1621"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Dewey</div> <div class="given-names">C.N.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">323</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=C.N.%20Dewey&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=323" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-12-323" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-323" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-12-323"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kong</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Ye</div> <div class="given-names">Z.-Q.</div>
</div>
</div>, et al.  <div class="year">2007</div>, 
<div class="article-title">CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W345</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CPC%3A%20assess%20the%20protein-coding%20potential%20of%20transcripts%20using%20sequence%20features%20and%20support%20vector%20machine&amp;author=L.%20Kong&amp;author=Y.%20Zhang&amp;author=Z.-Q.%20Ye&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W345-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkm391" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm391" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkm391"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17631615" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CPC%3A%20assess%20the%20protein-coding%20potential%20of%20transcripts%20using%20sequence%20features%20and%20support%20vector%20machine&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yanai</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Benjamin</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Shmoish</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification</div>, <div class="source">Bioinform. Oxf. Engl</div> ., <div class="volume">21</div>, <div class="fpage">650</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20midrange%20transcription%20profiles%20reveal%20expression%20level%20relationships%20in%20human%20tissue%20specification&amp;author=I.%20Yanai&amp;author=H.%20Benjamin&amp;author=M.%20Shmoish&amp;publication_year=2005&amp;journal=Bioinform.%20Oxf.%20Engl&amp;volume=21&amp;pages=650-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bti042" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti042" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbti042"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20midrange%20transcription%20profiles%20reveal%20expression%20level%20relationships%20in%20human%20tissue%20specification&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zerbino</div> <div class="given-names">D.R.</div>
</div>, <div class="name string-name">
<div class="surname">Birney</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Velvet: algorithms for <em>de novo</em> short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">821</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.074492.107"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Arendsee</div> <div class="given-names">Z.W.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Wurtele</div> <div class="given-names">E.S.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">The first near-complete assembly of the hexaploid bread wheat genome, <em>Triticum aestivum</em>
</div>, <div class="source">GigaScience</div> , <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20first%20near-complete%20assembly%20of%20the%20hexaploid%20bread%20wheat%20genome%2C%20Triticum%20aestivum&amp;aulast=Arendsee&amp;date=2014&amp;spage=1&amp;epage=7&amp;volume=6" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20first%20near-complete%20assembly%20of%20the%20hexaploid%20bread%20wheat%20genome%2C%20Triticum%20aestivum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dufayard</div> <div class="given-names">J.-F.</div>
</div>, <div class="name string-name">
<div class="surname">Bettembourg</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Fischer</div> <div class="given-names">I.</div>
</div>
</div>, et al. <div class="year">2017</div>, 
<div class="article-title">New insights on leucine-rich repeats receptor-like kinase orthologous relationships in angiosperms</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">8</div>:<div class="fpage">381</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=New%20insights%20on%20leucine-rich%20repeats%20receptor-like%20kinase%20orthologous%20relationships%20in%20angiosperms&amp;author=J.-F.%20Dufayard&amp;author=M.%20Bettembourg&amp;author=I.%20Fischer&amp;publication_year=2017&amp;journal=Front.%20Plant%20Sci&amp;volume=8&amp;pages=381" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28424707" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=New%20insights%20on%20leucine-rich%20repeats%20receptor-like%20kinase%20orthologous%20relationships%20in%20angiosperms&amp;aulast=Dufayard&amp;date=2017&amp;spage=381&amp;volume=8" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20insights%20on%20leucine-rich%20repeats%20receptor-like%20kinase%20orthologous%20relationships%20in%20angiosperms&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B56" class="ref-content ">
<div class="label">56</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hirsch</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Oldroyd</div> <div class="given-names">G.E.D.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">GRAS-domain transcription factors that regulate plant development</div>, <div class="source">Plant Signal. Behav</div> ., <div class="volume">4</div>, <div class="fpage">698</div>–<div class="lpage">700</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GRAS-domain%20transcription%20factors%20that%20regulate%20plant%20development&amp;author=S.%20Hirsch&amp;author=G.E.D.%20Oldroyd&amp;publication_year=2009&amp;journal=Plant%20Signal.%20Behav&amp;volume=4&amp;pages=698-700" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.4161/psb.4.8.9176" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.4.8.9176" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.4161%2Fpsb.4.8.9176"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19820314" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GRAS-domain%20transcription%20factors%20that%20regulate%20plant%20development&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B57" class="ref-content ">
<div class="label">57</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garcia Tavares</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Lakshmanan</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Peiter</div> <div class="given-names">E.</div>
</div>
</div>, et al. <div class="year">2018</div>, 
<div class="article-title">ScGAI is a key regulator of culm development in sugarcane</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">69</div>(<div class="issue">16</div>):<div class="fpage">3823</div>–<div class="lpage">3837</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ScGAI%20is%20a%20key%20regulator%20of%20culm%20development%20in%20sugarcane&amp;author=R.%20Garcia%20Tavares&amp;author=P.%20Lakshmanan&amp;author=E.%20Peiter&amp;publication_year=2018&amp;journal=J.%20Exp.%20Bot&amp;volume=69&amp;pages=3823-3837" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/ery180" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fery180" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Fery180"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29767776" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ScGAI%20is%20a%20key%20regulator%20of%20culm%20development%20in%20sugarcane&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B58" class="ref-content ">
<div class="label">58</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schneider</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Hanak</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Persson</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Voigt</div> <div class="given-names">C.A.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Cellulose and callose synthesis and organization in focus, what’s new?</div> <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">34</div>, <div class="fpage">9</div>–<div class="lpage">16</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Cellulose%20and%20callose%20synthesis%20and%20organization%20in%20focus%2C%20what%E2%80%99s%20new%3F&amp;author=R.%20Schneider&amp;author=T.%20Hanak&amp;author=S.%20Persson&amp;author=C.A.%20Voigt&amp;publication_year=2016&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=34&amp;pages=9-16" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.pbi.2016.07.007" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2016.07.007" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.pbi.2016.07.007"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27479608" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cellulose%20and%20callose%20synthesis%20and%20organization%20in%20focus%2C%20what%E2%80%99s%20new%3F&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B59" class="ref-content ">
<div class="label">59</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Falter</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Zwikowics</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Eggert</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Glucanocellulosic ethanol: the undiscovered biofuel potential in energy crops and marine biomass</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">13722</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Glucanocellulosic%20ethanol%3A%20the%20undiscovered%20biofuel%20potential%20in%20energy%20crops%20and%20marine%20biomass&amp;author=C.%20Falter&amp;author=C.%20Zwikowics&amp;author=D.%20Eggert&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=13722" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep13722" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep13722" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep13722"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26324382" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Glucanocellulosic%20ethanol%3A%20the%20undiscovered%20biofuel%20potential%20in%20energy%20crops%20and%20marine%20biomass&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B60" class="ref-content ">
<div class="label">60</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shrivastava</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Proteomic approach to understand the molecular physiology of symbiotic interaction between <em>Piriformospora indica</em> and <em>Brassica napus</em>
</div>, <div class="source">Sci. Rep</div> ., <div class="volume">8</div>, <div class="fpage">5773</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Proteomic%20approach%20to%20understand%20the%20molecular%20physiology%20of%20symbiotic%20interaction%20between%20Piriformospora%20indica%20and%20Brassica%20napus&amp;author=N.%20Shrivastava&amp;author=L.%20Jiang&amp;author=P.%20Li&amp;publication_year=2018&amp;journal=Sci.%20Rep&amp;volume=8&amp;pages=5773" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41598-018-23994-z" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41598-018-23994-z" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41598-018-23994-z"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29636503" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Proteomic%20approach%20to%20understand%20the%20molecular%20physiology%20of%20symbiotic%20interaction%20between%20Piriformospora%20indica%20and%20Brassica%20napus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B61" class="ref-content ">
<div class="label">61</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Miller</div> <div class="given-names">J.B.</div>
</div>, <div class="name string-name">
<div class="surname">Pratap</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Miyahara</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Calcium/calmodulin-dependent protein kinase is negatively and positively regulated by calcium, providing a mechanism for decoding calcium responses during symbiosis signaling</div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">5053</div>–<div class="lpage">66</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Calcium%2Fcalmodulin-dependent%20protein%20kinase%20is%20negatively%20and%20positively%20regulated%20by%20calcium%2C%20providing%20a%20mechanism%20for%20decoding%20calcium%20responses%20during%20symbiosis%20signaling&amp;author=J.B.%20Miller&amp;author=A.%20Pratap&amp;author=A.%20Miyahara&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=5053-66" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.113.116921" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.116921" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.113.116921"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24368786" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Calcium%2Fcalmodulin-dependent%20protein%20kinase%20is%20negatively%20and%20positively%20regulated%20by%20calcium%2C%20providing%20a%20mechanism%20for%20decoding%20calcium%20responses%20during%20symbiosis%20signaling&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B62" class="ref-content ">
<div class="label">62</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hohnjec</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Vieweg</div> <div class="given-names">M.F.</div>
</div>, <div class="name string-name">
<div class="surname">Pühler</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Becker</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Küster</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Overlaps in the transcriptional profiles of <em>Medicago truncaluta</em> roots inoculated with two different Glomus fungi provide insights into the genetic program activated during arbuscular mycorrhiza</div>, <div class="source">Plant Physiol</div> ., <div class="volume">137</div>, <div class="fpage">1283</div>–<div class="lpage">301</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Overlaps%20in%20the%20transcriptional%20profiles%20of%20Medicago%20truncaluta%20roots%20inoculated%20with%20two%20different%20Glomus%20fungi%20provide%20insights%20into%20the%20genetic%20program%20activated%20during%20arbuscular%20mycorrhiza&amp;author=N.%20Hohnjec&amp;author=M.F.%20Vieweg&amp;author=A.%20P%C3%BChler&amp;author=A.%20Becker&amp;author=H.%20K%C3%BCster&amp;publication_year=2005&amp;journal=Plant%20Physiol&amp;volume=137&amp;pages=1283-301" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.104.056572" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.056572" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.104.056572"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15778460" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Overlaps%20in%20the%20transcriptional%20profiles%20of%20Medicago%20truncaluta%20roots%20inoculated%20with%20two%20different%20Glomus%20fungi%20provide%20insights%20into%20the%20genetic%20program%20activated%20during%20arbuscular%20mycorrhiza&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B63" class="ref-content ">
<div class="label">63</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fiorilli</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Vallino</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Biselli</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Faccio</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Bagnaresi</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Bonfante</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Host and non-host roots in rice: cellular and molecular approaches reveal differential responses to arbuscular mycorrhizal fungi</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">636</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Host%20and%20non-host%20roots%20in%20rice%3A%20cellular%20and%20molecular%20approaches%20reveal%20differential%20responses%20to%20arbuscular%20mycorrhizal%20fungi&amp;author=V.%20Fiorilli&amp;author=M.%20Vallino&amp;author=C.%20Biselli&amp;author=A.%20Faccio&amp;author=P.%20Bagnaresi&amp;author=P.%20Bonfante&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=636" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2015.00636" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00636" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2015.00636"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26322072" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Host%20and%20non-host%20roots%20in%20rice%3A%20cellular%20and%20molecular%20approaches%20reveal%20differential%20responses%20to%20arbuscular%20mycorrhizal%20fungi&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B64" class="ref-content ">
<div class="label">64</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Genome-wide identification of the potato WRKY transcription factor family</div>, <div class="source">PLoS One</div> , <div class="volume">12</div>, <div class="fpage">e0181573</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20the%20potato%20WRKY%20transcription%20factor%20family&amp;author=C.%20Zhang&amp;author=D.%20Wang&amp;author=C.%20Yang&amp;publication_year=2017&amp;journal=PLoS%20One&amp;volume=12&amp;pages=e0181573" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0181573" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0181573" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0181573"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28727761" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20identification%20of%20the%20potato%20WRKY%20transcription%20factor%20family&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B65" class="ref-content ">
<div class="label">65</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pii</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Astegno</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Peroni</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Zaccardelli</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Pandolfini</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Crimi</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2009</div>, T
<div class="article-title">he <em>Medicago truncaluta</em> N5 gene encoding a root-specific lipid transfer protein is required for the symbiotic interaction with <em>Sinorhizobium meliloti</em>
</div>, <div class="source">Mol Plant Microbe Interact</div> , <div class="volume">22</div>(<div class="issue">12</div>), <div class="fpage">1577</div>–<div class="lpage">87</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=he%20Medicago%20truncaluta%20N5%20gene%20encoding%20a%20root-specific%20lipid%20transfer%20protein%20is%20required%20for%20the%20symbiotic%20interaction%20with%20Sinorhizobium%20meliloti&amp;author=Y.%20Pii&amp;author=A.%20Astegno&amp;author=E.%20Peroni&amp;author=M.%20Zaccardelli&amp;author=T.%20Pandolfini&amp;author=M.%20Crimi&amp;publication_year=2009&amp;journal=Mol%20Plant%20Microbe%20Interact&amp;volume=22&amp;pages=1577-87" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1094/MPMI-22-12-1577" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-22-12-1577" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1094%2FMPMI-22-12-1577"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19888823" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:he%20Medicago%20truncaluta%20N5%20gene%20encoding%20a%20root-specific%20lipid%20transfer%20protein%20is%20required%20for%20the%20symbiotic%20interaction%20with%20Sinorhizobium%20meliloti&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B66" class="ref-content ">
<div class="label">66</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mirshad</div> <div class="given-names">P.P.</div>
</div>, <div class="name string-name">
<div class="surname">Puthur</div> <div class="given-names">J.T.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Drought tolerance of bioenergy grass <em>Saccharum spontaneum</em> L. enhanced by arbuscular mycorrhizae</div>, <div class="source">Rhizosphere</div> , <div class="volume">3</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Drought%20tolerance%20of%20bioenergy%20grass%20Saccharum%20spontaneum%20L.%20enhanced%20by%20arbuscular%20mycorrhizae&amp;author=P.P.%20Mirshad&amp;author=J.T.%20Puthur&amp;publication_year=2017&amp;journal=Rhizosphere&amp;volume=3&amp;pages=1-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.rhisph.2016.09.004" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.rhisph.2016.09.004" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.rhisph.2016.09.004"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Drought%20tolerance%20of%20bioenergy%20grass%20Saccharum%20spontaneum%20L.%20enhanced%20by%20arbuscular%20mycorrhizae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B67" class="ref-content ">
<div class="label">67</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Battaglia</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Covarrubias</div> <div class="given-names">A.A.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Late Embryogenesis Abundant (LEA) proteins in legumes</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">4</div>, <div class="fpage">190</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Late%20Embryogenesis%20Abundant%20(LEA)%20proteins%20in%20legumes&amp;author=M.%20Battaglia&amp;author=A.A.%20Covarrubias&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci&amp;volume=4&amp;pages=190" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2013.00190" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2013.00190" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2013.00190"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23805145" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Late%20Embryogenesis%20Abundant%20(LEA)%20proteins%20in%20legumes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B68" class="ref-content ">
<div class="label">68</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Cai</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Xiang</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Identification and expression analysis of BURP domain-containing genes in <em>Medicago truncatula</em>
</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">485</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20expression%20analysis%20of%20BURP%20domain-containing%20genes%20in%20Medicago%20truncatula&amp;author=Y.%20Li&amp;author=X.%20Chen&amp;author=Z.%20Chen&amp;author=R.%20Cai&amp;author=H.%20Zhang&amp;author=Y.%20Xiang&amp;publication_year=2016&amp;journal=Front.%20Plant%20Sci&amp;volume=7&amp;pages=485" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27148311" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Identification%20and%20expression%20analysis%20of%20BURP%20domain-containing%20genes%20in%20Medicago%20truncatula&amp;aulast=Li&amp;date=2016&amp;spage=485&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20and%20expression%20analysis%20of%20BURP%20domain-containing%20genes%20in%20Medicago%20truncatula&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz001-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz001-B69" class="ref-content ">
<div class="label">69</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nascimento</div> <div class="given-names">L.C.</div>
</div>, <div class="name string-name">
<div class="surname">Salazar</div> <div class="given-names">M.M.</div>
</div>, <div class="name string-name">
<div class="surname">Lepikson-Neto</div> <div class="given-names">J.</div>
</div>
</div>, et al. <div class="year">2017</div>, 
<div class="article-title">EUCANEXT: an integrated database for the exploration of genomic and transcriptomic data from <em>Eucalyptus</em> species</div>, <div class="source">Database J. Biol. Databases Curation</div> , <div class="volume">2017</div>:doi.org/10.1093/database/bax079.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=EUCANEXT%3A%20an%20integrated%20database%20for%20the%20exploration%20of%20genomic%20and%20transcriptomic%20data%20from%20Eucalyptus%20species&amp;aulast=Nascimento&amp;date=2017&amp;volume=2017" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:EUCANEXT%3A%20an%20integrated%20database%20for%20the%20exploration%20of%20genomic%20and%20transcriptomic%20data%20from%20Eucalyptus%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5320315&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz001/1/dsz001_supplementary_data.zip?Expires=1576706108&amp;Signature=f-F9o5t0Aac1ca-mtaW5G5q-osJgmuEdZnVY8fNxKKlKbg1oX5RDKCZIdmJxyG3A9QUG9L1Q21Dtl5mGdKxQloySezPSA7rqkslbN-wQupZdnlFqoE8sLfa4sN8oliMPdPtJQIJGqv6XaCHz-n2JUvmavvgSW~VMhkXDamjYp~vLWLYZgbLqzo6redYhYrgNPo90avdU6degv9LBMI157hojroGk3nLPm5PqkzmsojJyXBb7lmRWGwefWc5PgmChYUmpBse707p60lJ4QsbbsKT1xv5SXyP-hndIY--tptVhs464dJ6CHqd1egbDTRaAjZaL2rI-p~fm0icuy0m~LA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz001_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,457</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz001"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5320315"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f3%2f205%2f5320315">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F3%2F205%2F5320315" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5320315">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Unraveling the complex genome of Saccharum spontaneum using Polyploid Gene Assembler | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz001");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f205%2f5320315">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
