<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>complexity of alternative splicing and landscape of tissue-specific expression in lotus (Nelumbo nucifera) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/4/301/5512305","@type":"ScholarlyArticle","name":"The complexity of alternative splicing and landscape of tissue-specific expression in lotus ( Nelumbo nucifera ) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing","datePublished":"2019-06-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2019-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz010","keywords":["SMRT sequencing","Illumina RNA-seq","full length","alternative splicing","isoform variants"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/4/301/5512305","author":[{"name":"Zhang, Yue","affiliation":"CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China  University of Chinese Academy of Sciences, Beijing, China","@type":"Person"},{"name":"Nyong'A, Tonny Maraga","affiliation":"CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China  University of Chinese Academy of Sciences, Beijing, China","@type":"Person"},{"name":"Shi, Tao","affiliation":"CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China","@type":"Person"},{"name":"Yang, Pingfang","affiliation":"State Key Laboratory of Biocatalysis and Enzyme Engineering, School of Life Sciences, Hubei University, Wuhan, China","@type":"Person","sameAs":"http://orcid.org/0000-0003-3526-4543"}],"description":"Abstract.  Alternative splicing (AS) plays a critical role in regulating different physiological and developmental processes in eukaryotes, by dramatically incr","pageStart":"301","pageEnd":"311","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636265479&Signature=Wa8mOUWC~3aX5C6HPKaYDisAAYDQtKDpqozcRfwGnn3f0e2Wf6w~9q-OFCd~3ia2aA0aiYQ8-xSE~Nn5qH6cu5MvzP8lPu2ha9~qOyNOmIa5wWKLpyRwrpuUcmHSNxHZRLIdq6MKq1XgS-v5Y6HS9kpoeGOfSkhGsnJ53mLfDg1Fkh6ejXTJuTvmSmLqAUpuaemdcKAJN1WwN2BimZQlb7Y1KCnrcBROJMH2VoxEDe2itUJgpxRXA5EzWUFeE6Hjp3bNrYvzcgf1Jw~LzCdJUW3N3A-CPmzGVK1K1Q5ctx-5tIjgDxOqsfpKpZLuhbnTcCAMs6-J5gZdiesHeJmNwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The complexity of alternative splicing and landscape of tissue-specific expression in lotus ( Nelumbo nucifera ) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636265479&Signature=Wa8mOUWC~3aX5C6HPKaYDisAAYDQtKDpqozcRfwGnn3f0e2Wf6w~9q-OFCd~3ia2aA0aiYQ8-xSE~Nn5qH6cu5MvzP8lPu2ha9~qOyNOmIa5wWKLpyRwrpuUcmHSNxHZRLIdq6MKq1XgS-v5Y6HS9kpoeGOfSkhGsnJ53mLfDg1Fkh6ejXTJuTvmSmLqAUpuaemdcKAJN1WwN2BimZQlb7Y1KCnrcBROJMH2VoxEDe2itUJgpxRXA5EzWUFeE6Hjp3bNrYvzcgf1Jw~LzCdJUW3N3A-CPmzGVK1K1Q5ctx-5tIjgDxOqsfpKpZLuhbnTcCAMs6-J5gZdiesHeJmNwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="complexity of alternative splicing and landscape of tissue-specific expression in lotus ( Nelumbo nucifera ) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing">
<meta property="og:description" content="Abstract.  Alternative splicing (AS) plays a critical role in regulating different physiological and developmental processes in eukaryotes, by dramatically incr">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/4/301/5512305">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636265479&amp;Signature=Wa8mOUWC~3aX5C6HPKaYDisAAYDQtKDpqozcRfwGnn3f0e2Wf6w~9q-OFCd~3ia2aA0aiYQ8-xSE~Nn5qH6cu5MvzP8lPu2ha9~qOyNOmIa5wWKLpyRwrpuUcmHSNxHZRLIdq6MKq1XgS-v5Y6HS9kpoeGOfSkhGsnJ53mLfDg1Fkh6ejXTJuTvmSmLqAUpuaemdcKAJN1WwN2BimZQlb7Y1KCnrcBROJMH2VoxEDe2itUJgpxRXA5EzWUFeE6Hjp3bNrYvzcgf1Jw~LzCdJUW3N3A-CPmzGVK1K1Q5ctx-5tIjgDxOqsfpKpZLuhbnTcCAMs6-J5gZdiesHeJmNwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-06-07">
<meta name="citation_author" content="Zhang, Yue">
<meta name="citation_author_institution" content="CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Nyong'A, Tonny Maraga">
<meta name="citation_author_institution" content="CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Shi, Tao">
<meta name="citation_author_institution" content="CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China">
<meta name="citation_author" content="Yang, Pingfang">
<meta name="citation_author_institution" content="State Key Laboratory of Biocatalysis and Enzyme Engineering, School of Life Sciences, Hubei University, Wuhan, China">
<meta name="citation_title" content="The complexity of alternative splicing and landscape of tissue-specific expression in lotus (Nelumbo nucifera) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing">
<meta name="citation_firstpage" content="301">
<meta name="citation_lastpage" content="311">
<meta name="citation_doi" content="10.1093/dnares/dsz010">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="4">
<meta name="citation_publication_date" content="2019/08/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=The evolution of posttranscriptional regulation;  citation_journal_title=Wires. RNA;  citation_year=2018;  citation_volume=9;  citation_pages=e1485">
<meta name="citation_reference" content="citation_title=Deciphering the splicing code;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=465;  citation_pages=53-9; ">
<meta name="citation_reference" content="citation_title=Alternative splicing of transcription factors in plant responses to low temperature stress: mechanisms and functions;  citation_journal_title=Planta;  citation_year=2013;  citation_volume=237;  citation_pages=1415-24; ">
<meta name="citation_reference" content="citation_title=Alternative splicing and evolution: diversification, exon definition and function;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=345-55; ">
<meta name="citation_reference" content="citation_title=Transcriptome survey reveals increased complexity of the alternative splicing landscape in Arabidopsis;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=1184-95; ">
<meta name="citation_reference" content="citation_title=Alternative isoform regulation in human tissue transcriptomes;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=470-6; ">
<meta name="citation_reference" content="citation_title=Complexity of the alternative splicing landscape in plants;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3657-83; ">
<meta name="citation_reference" content="citation_title=Cotranscriptional splicing efficiency differs dramatically between Drosophila and mouse;  citation_journal_title=RNA;  citation_year=2012;  citation_volume=18;  citation_pages=2174-86; ">
<meta name="citation_reference" content="citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing;  citation_journal_title=Nat. Genet;  citation_year=2008;  citation_volume=40;  citation_pages=1413">
<meta name="citation_reference" content="citation_title=Regulation of alternative splicing of Bcl-x by IL-6, GM-CSF and TPA;  citation_journal_title=Cell Res;  citation_year=2004;  citation_volume=14;  citation_pages=473">
<meta name="citation_reference" content="citation_title=A major facilitator superfamily transporter plays a dual role in polar auxin transport and drought stress tolerance in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=901-26; ">
<meta name="citation_reference" content="citation_title=Alternative splicing of the auxin biosynthesis gene YUCCA4 determines its subcellular compartmentation;  citation_journal_title=Plant J;  citation_year=2012;  citation_volume=70;  citation_pages=292-302; ">
<meta name="citation_reference" content="citation_title=Polypyrimidine tract binding protein homologs from Arabidopsis are key regulators of alternative splicing with implications in fundamental developmental processes;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=4360-75; ">
<meta name="citation_reference" content="citation_title=Dynamic regulation of genome-wide pre-mRNA splicing and stress tolerance by the Sm-like protein LSm5 in Arabidopsis;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=R1">
<meta name="citation_reference" content="citation_title=Genome-wide detection of condition-sensitive alternative splicing in Arabidopsis roots;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=162;  citation_pages=1750-63; ">
<meta name="citation_reference" content="citation_title=Alternative splicing at the intersection of biological timing, development, and stress responses;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3640-56; ">
<meta name="citation_reference" content="citation_title=Unproductive alternative splicing and nonsense mRNAs: a widespread phenomenon among plant circadian clock genes;  citation_journal_title=Biol. Direct;  citation_year=2012;  citation_volume=7;  citation_pages=20">
<meta name="citation_reference" content="citation_title=Two different transcripts of a LAMMER kinase gene play opposite roles in disease resistance;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=172;  citation_pages=1959-72; ">
<meta name="citation_reference" content="citation_title=Maize rough endosperm3 encodes an RNA splicing factor required for endosperm cell differentiation and has a nonautonomous effect on embryo development;  citation_journal_title=Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=4280-97; ">
<meta name="citation_reference" content="citation_title=Deep RNA sequencing at single base-pair resolution reveals high complexity of the rice transcriptome;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=646-54; ">
<meta name="citation_reference" content="citation_title=Genome-wide analysis of alternative splicing in Zea mays: landscape and genetic regulation;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=3472-87; ">
<meta name="citation_reference" content="citation_title=Global dissection of alternative splicing in paleopolyploid soybean;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=996-1008; ">
<meta name="citation_reference" content="citation_title=Assessment of transcript reconstruction methods for RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1177-84; ">
<meta name="citation_reference" content="citation_title=A single-molecule long-read survey of the human transcriptome;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=1009-14; ">
<meta name="citation_reference" content="citation_title=Exploiting single-molecule transcript sequencing for eukaryotic gene prediction;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=184">
<meta name="citation_reference" content="citation_title=Full-length transcriptome sequences and splice variants obtained by a combination of sequencing platforms applied to different root tissues of Salvia miltiorrhiza and tanshinone biosynthesis;  citation_journal_title=Plant J;  citation_year=2015;  citation_volume=82;  citation_pages=951-61; ">
<meta name="citation_reference" content="citation_title=A survey of the sorghum transcriptome using single-molecule long reads;  citation_journal_title=Nat Commun;  citation_year=2016;  citation_volume=7;  citation_pages=11706">
<meta name="citation_reference" content="citation_title=Global identification of alternative splicing via comparative analysis of SMRT- and Illumina-based RNA-seq in strawberry;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=90;  citation_pages=164-76; ">
<meta name="citation_reference" content="citation_title=Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7;  citation_pages=11708">
<meta name="citation_reference" content="citation_title=A global survey of alternative splicing in allopolyploid cotton: landscape, complexity and regulation;  citation_journal_title=New Phytol;  citation_year=2018;  citation_volume=217;  citation_pages=163-78; ">
<meta name="citation_reference" content="citation_title=Whole genome re-sequencing reveals evolutionary patterns of sacred lotus (Nelumbo nucifera);  citation_journal_title=J. Integr. Plant Biol;  citation_year=2018;  citation_volume=60;  citation_pages=2-15; ">
<meta name="citation_reference" content="citation_title=The evolution of plant microRNAs: insights from a basal eudicot sacred lotus;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=89;  citation_pages=442">
<meta name="citation_reference" content="citation_title=Genome of the long-living sacred lotus (Nelumbo nucifera Gaertn.);  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R41">
<meta name="citation_reference" content="citation_title=Thermal-stable proteins of fruit of long-living sacred lotus Nelumbo nucifera Gaertn var. China Antique;  citation_journal_title=Trop. Plant Biol;  citation_year=2013;  citation_volume=6;  citation_pages=69-84; ">
<meta name="citation_reference" content="citation_title=The sacred lotus genome provides insights into the evolution of flowering plants;  citation_journal_title=Plant J;  citation_year=2013;  citation_volume=76;  citation_pages=557-67; ">
<meta name="citation_reference" content="citation_title=LOTUS-DB: an integrative and interactive database for Nelumbo nucifera study;  citation_journal_title=Database (Oxford);  citation_year=2015;  citation_volume=2015;  citation_pages=bav023">
<meta name="citation_reference" content="citation_title=LoRDEC: accurate and efficient long read error correction;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3506-14; ">
<meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; ">
<meta name="citation_reference" content="citation_title=PRAPI: post-transcriptional regulation analysis pipeline for Iso-Seq;  citation_journal_title=Bioinformatics (Oxford, England);  citation_year=2018;  citation_volume=34;  citation_pages=1580-2; ">
<meta name="citation_reference" content="citation_title=TACO produces robust multisample transcriptome assemblies from RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2017;  citation_volume=14;  citation_pages=68-70; ">
<meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7">
<meta name="citation_reference" content="citation_title=Widespread polycistronic transcripts in fungi revealed by single-molecule mRNA sequencing;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0132628">
<meta name="citation_reference" content="citation_title=SUPPA2: fast, accurate, and uncertainty-aware differential splicing analysis across multiple conditions;  citation_journal_title=Genome Biol;  citation_year=2018;  citation_volume=19;  citation_pages=40">
<meta name="citation_reference" content="citation_title=rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=E5593-5601; ">
<meta name="citation_reference" content="citation_title=KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=316-22; ">
<meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology;  citation_journal_title=Nat. Genet;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; ">
<meta name="citation_reference" content="citation_title=The KEGG resource for deciphering the genome;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=D277">
<meta name="citation_reference" content="citation_title=Gene ontology analysis for RNA-seq: accounting for selection bias;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=R14">
<meta name="citation_reference" content="citation_title=Characterization of the human ESC transcriptome by hybrid sequencing;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=E4821-30; ">
<meta name="citation_reference" content="citation_title=PRAPI: post-transcriptional regulation analysis pipeline for Iso-Seq;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=34;  citation_pages=1580-2; ">
<meta name="citation_reference" content="citation_title=Systematic transcriptomic analysis provides insights into lotus (Nelumbo nucifera) seed development;  citation_journal_title=Plant Growth Regul;  citation_year=2018;  citation_volume=86;  citation_pages=339-50; ">
<meta name="citation_reference" content="citation_title=The ABC model of floral development;  citation_journal_title=J. Yunnan Univ;  citation_year=2001;  citation_volume=23;  citation_pages=102-5; ">
<meta name="citation_reference" content="citation_title=The ABCs of flower development: mutational analysis of AP1/FUL-like genes in rice provides evidence for a homeotic (A)-function in grasses;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=89;  citation_pages=310-24; ">
<meta name="citation_reference" content="citation_title=Expression of B-class MADS-box genes in response to variations in photoperiod is associated with chasmogamous and cleistogamous flower development in Viola philippica;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=151">
<meta name="citation_reference" content="citation_title=Gene duplication and transference of function in the paleoAP3 lineage of floral organ identity genes;  citation_journal_title=Front Plant Sci;  citation_year=2018;  citation_volume=9;  citation_pages=334">
<meta name="citation_reference" content="citation_title=Identification and characterization of the MADS-box genes and their contribution to flower organ in carnation (Dianthus caryophyllus L.);  citation_journal_title=Genes;  citation_year=2018;  citation_volume=9,; ">
<meta name="citation_reference" content="citation_title=Automated update, revision and quality control of the Zea mays genome annotations using MAKER-P improves the B73 RefGen_v3 gene models and identifies new genes;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=167;  citation_pages=25">
<meta name="citation_reference" content="citation_title=High impact gene discovery: simple strand-specific mRNA library construction and differential regulatory analysis based on gene co-expression network;  citation_journal_title=Methods Mol. Biol;  citation_year=2018;  citation_volume=1830;  citation_pages=163-89; ">
<meta name="citation_reference" content="citation_title=Next-generation sequencing sheds new light on small RNAs in plant reproductive development;  citation_journal_title=Curr. Issues Mol. Biol;  citation_year=2018;  citation_volume=27;  citation_pages=143">
<meta name="citation_reference" content="citation_title=Annotating genomes with massive-scale RNA sequencing;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=1-12; ">
<meta name="citation_reference" content="citation_title=Genome re-annotation of the wild strawberry Fragaria vesca using extensive Illumina- and SMRT-based RNA-seq datasets;  citation_journal_title=DNA Res;  citation_year=2018;  citation_volume=25;  citation_pages=61-70; ">
<meta name="citation_reference" content="citation_title=Transcriptomic analysis identifies the key genes involved in stamen petaloid in lotus (Nelumbo nucifera);  citation_journal_title=BMC Genomics;  citation_year=2018;  citation_volume=19;  citation_pages=554">
<meta name="citation_reference" content="citation_title=Identification and analysis of glutathione S-transferase gene family in sweet potato reveal divergent GST -mediated networks in aboveground and underground tissues in response to abiotic stresses;  citation_journal_title=BMC Plant Biol;  citation_year=2017;  citation_volume=17;  citation_pages=225">
<meta name="citation_reference" content="citation_title=Molecular mechanisms of acclimatization to phosphorus, starvation and recovery underlying full-length transcriptome profiling in barley (Hordeum vulgare L.);  citation_journal_title=Front. Plant Sci;  citation_year=2018;  citation_volume=9;  citation_pages=500">
<meta name="citation_reference" content="citation_title=RNA-seq uncovers SNPs and alternative splicing events in Asian lotus (Nelumbo nucifera);  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0125702">
<meta name="citation_reference" content="citation_title=Analysis of pollen-specific alternative splicing in Arabidopsis thaliana via semi-quantitative PCR;  citation_journal_title=Peer J;  citation_year=2015;  citation_volume=3;  citation_pages=e919">
<meta name="citation_reference" content="citation_title=Tissue-specific expression of telomerase reverse transcriptase gene variants in Nicotiana tabacum;  citation_journal_title=Planta;  citation_year=2017;  citation_volume=245;  citation_pages=549-61; ">
<meta name="citation_reference" content="citation_title=Bombyx E75 isoforms display stage- and tissue-specific responses to 20-hydroxyecdysone;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=12114">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/4/301/29197200/dsz010.pdf">
<meta name="description" content="Abstract.  Alternative splicing (AS) plays a critical role in regulating different physiological and developmental processes in eukaryotes, by dramatically incr">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/4/301/5512305">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/4/301/5512305">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/4/301/5512305");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsz010';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f301%2f5512305" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f301%2f5512305" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=116309"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1576706132&amp;Signature=F2WzqY5Rvw~5fnpnOvrqWmeUMWqA6TuHlcXGX5cKMZHC~icn0X1kJeAYo3hIjul1IYqABU5PFelPNYB7icUBfC2Utuqe0Kanc6xUPdQTVSqmHs8XGD0SD1jSofLoGYkyX4fvyBnDT6DegKNRAlK0p3CMB3IF5m7bw4ZfJ0hDTWLv8tmuUuQL6s7NPysvupS6F7Bi2JshmCvK0~2D16GAFc8Z0ZzUtHzETSwnvLON6PshV~D1U5sZReZew8O7QtN01WiuIN3o83L12vAOkYBx79sQQQquD3T4JorhpCEJNC83N3L0L-vg1O7y4o3hyzSvVe4q2sJ6SO7fYQ0CEMfTEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/4">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="140465355">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465355">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465357">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465357">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465363">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465363">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465382">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465382">3. Result</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465411">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465411">4. Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465419">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465419">5. Data availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465421">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465421">Funding</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465423">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465423">Conflict of interest</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="140465425">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465425">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/4/287/5490643">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/4/313/5512299">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    The complexity of alternative splicing and landscape of tissue-specific expression in lotus (<em>Nelumbo nucifera</em>) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yue Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China</div>
<div class="aff">University of Chinese Academy of Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yue+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20Yue%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Yue Zhang</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tonny Maraga Nyong'A</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China</div>
<div class="aff">University of Chinese Academy of Sciences, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tonny+Maraga+Nyong%27A">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nyong'A%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nyong'A%20Tonny%20Maraga%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Tonny Maraga Nyong'A</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tao Shi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">CAS Key Laboratory of Aquatic Botany and Watershed Ecology, Wuhan Botanical Garden, Chinese Academy of Sciences, Wuhan, CN, China</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed: Tel. +86-27-87700860. Fax. 86-27-87700880. Email: <a href="mailto:yangpf@hubu.edu.cn" target="_blank">yangpf@hubu.edu.cn</a> (P.Y.); <a href="mailto:shitao323@wbgcas.cn" target="_blank">shitao323@wbgcas.cn</a> (T.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tao+Shi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shi%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shi%20Tao%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Tao Shi</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pingfang Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Biocatalysis and Enzyme Engineering, School of Life Sciences, Hubei University, Wuhan, China</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed: Tel. +86-27-87700860. Fax. 86-27-87700880. Email: <a href="mailto:yangpf@hubu.edu.cn" target="_blank">yangpf@hubu.edu.cn</a> (P.Y.); <a href="mailto:shitao323@wbgcas.cn" target="_blank">shitao323@wbgcas.cn</a> (T.S.)
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-3526-4543">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0003-3526-4543
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pingfang+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yang%20Pingfang%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Pingfang Yang</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 4, August 2019, Pages 301–311, <a href="https://doi.org/10.1093/dnares/dsz010">https://doi.org/10.1093/dnares/dsz010</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">07 June 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">23 January 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">03 May 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">07 June 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5512305" href="/dnaresearch/article-pdf/26/4/301/29197200/dsz010.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yue Zhang, Tonny Maraga Nyong'A, Tao Shi, Pingfang Yang,  The complexity of alternative splicing and landscape of tissue-specific expression in lotus (<em>Nelumbo nucifera</em>) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing, <em>DNA Research</em>, Volume 26, Issue 4, August 2019, Pages 301–311, <a href="https://doi.org/10.1093/dnares/dsz010">https://doi.org/10.1093/dnares/dsz010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5512305&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20complexity%20of%20alternative%20splicing%20and%20landscape%20of%20tissue-specific%20expression%20in%20lotus%20(%26lt%3Bem%26gt%3BNelumbo%20nucifera%26lt%3B%2Fem%26gt%3B)%20unveiled%20by%20Illumina-%20and%20single-molecule%20real-time-based%20RNA-sequencing&amp;publicationDate=2019-06-07&amp;volumeNum=26&amp;issueNum=4&amp;author=Zhang%2C%20Yue%3B%20Nyong'A%2C%20Tonny%20Maraga&amp;startPage=301&amp;endPage=311&amp;contentId=10.1093%2Fdnares%2Fdsz010&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f301%2f5512305" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="140465355" id="140465355" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) plays a critical role in regulating different physiological and developmental processes in eukaryotes, by dramatically increasing the diversity of the transcriptome and the proteome. However, the saturation and complexity of AS remain unclear in lotus due to its limitation of rare obtainment of full-length multiple-splice isoforms. In this study, we apply a hybrid assembly strategy by combining single-molecule real-time sequencing and Illumina RNA-seq to get a comprehensive insight into the lotus transcriptomic landscape. We identified 211,802 high-quality full-length non-chimeric reads, with 192,690 non-redundant isoforms, and updated the lotus reference gene model. Moreover, our analysis identified a total of 104,288 AS events from 16,543 genes, with alternative 3ʹ splice-site being the predominant model, following by intron retention. By exploring tissue datasets, 370 tissue-specific AS events were identified among 12 tissues. Both the tissue-specific genes and isoforms might play important roles in tissue or organ development, and are suitable for ‘ABCE’ model partly in floral tissues. A large number of AS events and isoform variants identified in our study enhance the understanding of transcriptional diversity in lotus, and provide valuable resource for further functional genomic studies.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"SMRT sequencing"'>SMRT sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"Illumina RNA-seq"'>Illumina RNA-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"full length"'>full length</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"alternative splicing"'>alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"isoform variants"'>isoform variants</a>
</div>                    <h2 scrollto-destination="140465357" id="140465357" class="section-title">1. Introduction</h2>
<p>Gene expression is generally defined as the decoding of DNA information into proteins, which includes two basic steps: transcription of DNA into RNA and translation of RNA into protein. Although these two steps occur in the same spot in prokaryotes, they are separated at different subcellular locations in eukaryote.<a reveal-id="dsz010-B1" data-open="dsz010-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> This separation, along with the complicated gene structure, enables more fine and complex regulations on the eukaryotic gene expression. Among all regulations, alternative splicing (AS), defined as the inclusion of different exons into mature mRNA, is an important mechanism that could dramatically increase the diversity of transcriptome and proteome, and hence regulating different physiological and developmental processes in eukaryote.<a reveal-id="dsz010-B2" data-open="dsz010-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsz010-B3" data-open="dsz010-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Furthermore, AS could also regulate gene expression level and spatio-temporal specificity.<a reveal-id="dsz010-B4" data-open="dsz010-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Studies have shown that &gt;90% genes in human and &gt;60% multi-exon genes in Arabidopsis undergo AS.<a reveal-id="dsz010-B5" data-open="dsz010-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsz010-B6" data-open="dsz010-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> There are five major AS groups, intron retention (IR), exon skipping (ES), alternative donor (AD), alternative acceptor (AA) and alternative position (AP), of which IR is prevalent in plants,<a reveal-id="dsz010-B7" data-open="dsz010-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> whereas, ES is more common in mammals.<a reveal-id="dsz010-B8" data-open="dsz010-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsz010-B9" data-open="dsz010-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p>
<p>Although the functions of most multiple isoforms generated by AS remain uncovered, several studies have demonstrated the biological importance of AS in different physiological and developmental processes. Studies showed that the <em>Bcl-x</em> gene in <em>Drosophila</em> could produce two isoforms playing contradicting roles with one activating apoptosis while the other one inhibiting apoptosis.<a reveal-id="dsz010-B10" data-open="dsz010-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Studies have also shown the function differentiation of the same genes with different transcripts in plants. In Arabidopsis, AS of <em>zinc-induced facilitator-like 1</em> (<em>ZIFL1</em>) gene could generate two isoforms: <em>ZIFL1.1</em> and <em>ZIFL1.3</em>, of which the former influences polar transportation of auxin in roots, whereas the latter regulates the movement of stomatal.<a reveal-id="dsz010-B11" data-open="dsz010-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> In other examples, different transcripts of <em>YUCCA4</em> show different features with respect to subcellular localization and tissue specificity,<a reveal-id="dsz010-B12" data-open="dsz010-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and also different <em>PHYTOCHROME INTERACTING FACTOR6</em> (<em>PIF6</em>) transcripts in the ABA-dependent seed germination play different functions.<a reveal-id="dsz010-B13" data-open="dsz010-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Different AS isoforms are involved in stress response and circadian in Arabidopsis.<a reveal-id="dsz010-B14 dsz010-B15 dsz010-B16 dsz010-B17" data-open="dsz010-B14 dsz010-B15 dsz010-B16 dsz010-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> Similar discoveries have also been obtained in cereals, such as the <em>Rough endosperm3</em> (<em>Rgh3</em>) gene (&gt;19 transcript isoforms) of maize involved in endosperm differentiation and two transcripts of <em>OsDR11</em> gene of rice involved in the process of disease resistance.<a reveal-id="dsz010-B18" data-open="dsz010-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsz010-B19" data-open="dsz010-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p>
<p>Due to the biological importance of AS and the reduction of sequencing cost, comprehensive surveys of AS have been conducted widely in plants using next-generation sequencing (NGS) technology, which include Arabidopsis and other types of crops.<a reveal-id="dsz010-B20 dsz010-B21 dsz010-B22" data-open="dsz010-B20 dsz010-B21 dsz010-B22" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> Although, the NGS has been proven to be powerful, it still faces limitation in accurate identification of each isoform because of its short sequencing reads,<a reveal-id="dsz010-B23" data-open="dsz010-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> which is much more serious in polyploid species or species with abundant repetitive genome sequences. However, with the advent of PacBio single-molecule real-time (SMRT) sequencing technology, known as the third generation of sequencing technology, full-length sequence of mRNA could be obtained without <em>in silico</em> sequence assembly.<a reveal-id="dsz010-B24" data-open="dsz010-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Because of its ability to generate much longer reads, it can easily detect the existence of different transcripts in each gene, making it appropriate in gene annotation and AS analysis compared with the new-generation sequencing technology.<a reveal-id="dsz010-B25" data-open="dsz010-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> This SMART technology has been successfully applied in the analysis of AS in different plant species, and emerged to be efficient in enhancing the accuracy of AS identification.<a reveal-id="dsz010-B26 dsz010-B27 dsz010-B28 dsz010-B29 dsz010-B30" data-open="dsz010-B26 dsz010-B27 dsz010-B28 dsz010-B29 dsz010-B30" class="link link-ref link-reveal xref-bibr"><sup>26–30</sup></a></p>
<p>Lotus (<em>Nelumbo nucifera</em>), which belongs to the family of <em>Nelumbonaceae</em>, is a basal eudicot.<a reveal-id="dsz010-B31" data-open="dsz010-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> It is a crucial species in plant phylogenetics study because it is a slowly evolved and early branching eudicot and retained many ancestral phenotypic features similar to monocots.<a reveal-id="dsz010-B32" data-open="dsz010-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsz010-B33" data-open="dsz010-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Besides, it is also an important aquatic horticultural plant,<a reveal-id="dsz010-B34" data-open="dsz010-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> thus increasing the necessity to get a better comprehension of lotus. After the successful sequencing and annotation of the genome of two wild strains of sacred lotus, ‘China Antique’ and ‘Chinese Tai-zi’,<a reveal-id="dsz010-B33" data-open="dsz010-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsz010-B35" data-open="dsz010-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> the genome database for ‘China Antique’ has been constructed.<a reveal-id="dsz010-B36" data-open="dsz010-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> However, the genome is poorly annotated because of the short reads and redundant repeat sequences. Although the construction of high resolutions genetic map could aid in genome assembly, more information at transcription level is required to achieve a fine annotation.</p>
<p>To get a comprehensive survey of AS events in sacred lotus, we applied both PacBio SMRT and Illumina-based sequencing of the transcriptome from 16 different tissues of ‘China Antique’. The isoforms identified by both techniques were compared, and the tissue and developmental stage specific AS transcripts were identified and annotated in different tissues. The findings showed the complexity of AS events, while providing comprehensive transcriptome data for further genome annotation on sacred lotus.</p>                    <h2 scrollto-destination="140465363" id="140465363" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="140465364" id="140465364" class="section-title">2.1. Plant materials</h3>
<p><em>Nelumbo nucifera</em> ‘China Antique’ was cultivated in Wuhan Botanical Garden, CAS (114°30ʹE, 30°60ʹN) China. Root, leaf, petiole, and rhizome (elongation zone, internode, and apical meristem) tissues, which were used for sequencing, were collected. Samples of petal, receptacle, immature stamen, and unpollinated carpel were sampled on the blooming day (0 d post-anthesis, DPA), while samples of pollinated carpel were collected from 12 h after pollination. Mature stamen and receptacle at 2 DPA, seed-coat at 6, 12, 18 DPA, and cotyledon at 9, 12, and 15 DPA were collected. In total, there are 19 different samples which were immediately frozen in liquid nitrogen for further RNA extraction.</p>                    <h3 scrollto-destination="140465366" id="140465366" class="section-title">2.2. RNA preparation and Illumina RNA-seq</h3>
<p>For each sample, total RNA was extracted using the RNAprep pure Plant Kit (TIANGEN). After quality checking by 1% agarose gels, the RNA concentration and integrity were examined using an Qubit<sup>®</sup> RNA Assay Kit in a Qubit<sup>®</sup> 2.0 Flurometer (Life Technologies, CA, USA) and an Agilent 2100 Bioanalyzer (Agilent Technologies, CA, USA), respectively. Eligible RNA of each sample (3 μg) was used for constructing the Illumina sequencing library following the recommendations of NEBNext<sup>®</sup> Ultra™ RNA Library Prep Kit for Illumina<sup>®</sup> (NEB, USA). The library was sequenced on the Illumina HiSeq 2000 platform at Novogene company (Nanjing, China) and 150 bp paired-end reads were generated.</p>                    <h3 scrollto-destination="140465368" id="140465368" class="section-title">2.3. Single-molecule sequencing of pooled RNA samples</h3>
<p>In order to obtain the complete information of full-length transcriptome, total RNA from 19 different samples were pooled together in an equal quantity to construct libraries for PacBio sequencing. The mixed RNA sample was reverse transcribed using the Clontech SMARTer polymerase chain reaction (PCR) cDNA Synthesis Kit and oligo (dT). Large-scale PCR amplification was carried out to generate barcode full-length cDNA. The BluePippin Size Selection System protocol as described by Pacific Biosciences (PN 100-092-800-03) was used to separate the size of PCR selection for mix sample &gt;4 kb. After damage repairing and end joining, the full-length cDNA is ligated to the SMRT dump bell joint. These cDNA products were purified for Iso-Seq SMRTBell library preparation. A SMRT cells were sequenced on the PacBio Sequel System platform.</p>                    <h3 scrollto-destination="140465370" id="140465370" class="section-title">2.4. PacBio reads mapping</h3>
<p>The raw data generated from PacBio Sequel sequencing system was processed using the SMRTlinks 5.0 software with min-Length of 200 and min-Read-Score of 0.75. The Circular Consensus sequences (CCSs) were generated from subreads with pass 1 and accuracy &gt;0.8. According to whether the 5ʹ- and 3ʹ-cDNA primers were present and whether there was a poly-A tail signal preceding the 3ʹ-primer, a CCS or subread sequence was divided into full length, non-full length, and chimeric reads. The method of Iterative Clustering for Error Correction (ICE) was employed for iterative clustering to obtain the consensus reads. Polished consensus reads were acquired from the original consensus corrected with non-full-length read using Arrow. To improve consensus accuracy, we use Illumina RNA-seq data to correct the consensus reads with LoRDEC.<a reveal-id="dsz010-B37" data-open="dsz010-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Aligning high-quality full-length consensus reads to the reference using GMAP<a reveal-id="dsz010-B38" data-open="dsz010-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> with &gt;85% alignment coverage and &gt;90% alignment identity. PRAP,<a reveal-id="dsz010-B39" data-open="dsz010-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> a one-stop solution for Iso-Seq analysis, was used to identify alternative transcription initiation (ATI), AS, alternative cleavage and polyadenylation (APA) by full-length consensus reads. And TACO<a reveal-id="dsz010-B40" data-open="dsz010-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> was used to combine transcripts from SMRT and Illumina RNA-seq into a consensus merged transcriptome and TransDecoder v5.3.0 was applied to predict the open reading frame. EVidenceModeler (EVM)<a reveal-id="dsz010-B41" data-open="dsz010-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> was used to predict confident consensus gene models by using the merged transcriptome.</p>                    <h3 scrollto-destination="140465372" id="140465372" class="section-title">2.5. Identification of fusion transcripts</h3>
<p>To identify fusion transcripts, a Python script in the PBTRANSCRIPT-ToFU<a reveal-id="dsz010-B42" data-open="dsz010-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> package was used. The criteria for a candidate fusion transcript was as follows: transcript must be mapped at least two loci in the reference genome and the interval of each loci should be longer than 10 kb; the sequence from the mapped loci should cover more than 99% of the fusion transcript, with sequence from each locus in the fusion transcript covering at least 10%. To further obtain high-quality candidates, the transcripts whose loci in scaffolds other than the nine megascaffolds were filtrated.</p>                    <h3 scrollto-destination="140465374" id="140465374" class="section-title">2.6. Differential expression analysis based on Illumina data</h3>
<p>After quality control, clean reads were obtained by removing reads containing adapters, reads containing ploy-Ns and low-quality reads from raw data. The clean reads were mapped to the lotus reference genome v1.1<a reveal-id="dsz010-B33" data-open="dsz010-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with Tophat2 v2.1.0 with the default setting. Cufflinks v2.2.1 was used to quantify the expression level of genes and isoforms using the GTF annotation file generated by SMRT and RNA-seq merged transcriptome. To explore tissue-bias expression patterns of isoforms, Mean FPKM &gt;0.1 and coefficient of variation of FPKM (standard deviation divided by mean) &gt;2 were used as the threshold to filter isoforms for hierarchical and <em>k</em>-means clustering analysis in <em>R</em>. The value of <em>k</em> was set to 12, which equals to the number of tissues used in this study. Isoforms or genes with significantly differential expression were identified under the threshold FC (fold change) of FPKM &gt;2 by using the Cuffdiff command between pairwise samples. The significant can be either ‘yes’ or ‘no’, depending on whether <em>P</em> is greater than the FDR after Benjamini–Hochberg correction for multiple-testing. Comparing with other tissues, the genes or isoforms which showed significantly higher expression were identified as tissue-specific genes or isoforms.</p>                    <h3 scrollto-destination="140465376" id="140465376" class="section-title">2.7. AS analysis</h3>
<p>SUPPA<a reveal-id="dsz010-B43" data-open="dsz010-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> was employed to classify the AS events with the parameter <em>-f ioe</em>. It produces seven kinds of local event types including Skipping Exon (SE), Alternative 5ʹ Splice Sites (A5), Alternative 3ʹ Splice Sites (A3), Mutually Exclusive Exons (MX), Retained Intron (RI), Alternative First Exons (AF) and Alternative Last Exons (AL). Quantification of the differential expression by AS events was carried out using the rMATS.<a reveal-id="dsz010-B44" data-open="dsz010-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The output from rMATS, included the differential expression of five main AS events as SE, A5, A3, MX, and RI, was filtered for Inclusion Level Difference &gt;5% and FDR &lt;0.01.</p>                    <h3 scrollto-destination="140465378" id="140465378" class="section-title">2.8. Functional annotation and gene ontology enrichment analyses</h3>
<p>To investigate the functions of all isoforms, KOBAS2.0<a reveal-id="dsz010-B45" data-open="dsz010-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> was used to map all the isoform sequence to gene ontology (GO),<a reveal-id="dsz010-B46" data-open="dsz010-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> KEGG,<a reveal-id="dsz010-B47" data-open="dsz010-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> KOG, Pfam, Swiss-Prot, TrEMBL, and Nr database. A GOseq R package based on Wallenius non-central hyper-geometric distribution<a reveal-id="dsz010-B48" data-open="dsz010-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> was run to obtain the significantly enriched GO terms. Concurrently, all lotus genes were searched against Arabidopsis TAIR10 with BLAST (<em>e</em>-value &lt;1e−6) to obtain best hits for further functional annotation.</p>                    <h3 scrollto-destination="140465380" id="140465380" class="section-title">2.9. Reverse transcription-PCR and quantitative real-time PCR</h3>
<p>Total RNA (1 µg) from samples were used for first strand cDNA with reverse transcription system, and the product of reaction systems was diluted to 200 µL before reverse transcription-PCR (RT-PCR). Primers were designed to span the splicing events using Primer Premier software (v.5.0) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The PCR fragments were tested in 1% agarose gel and sequenced by Sanger method. The qRT-PCR reactions were performed on the Bio-Rad using the SYBR. The reaction procedure was initiated at 95°C for 10 s, followed by 40 cycles of 95°C for 15 s, 60°C for 30 s, and 72°C for 30 s with fluorescence detection. Relative gene expressions were analysed using 2<sup>−△△CT</sup> method, with the lotus <em>β-Actin</em> gene used as an internal standard.</p>                    <h2 scrollto-destination="140465382" id="140465382" class="section-title">3. Result</h2>
                    <h3 scrollto-destination="140465383" id="140465383" class="section-title">3.1. Sequencing of <em>N. nucifera</em> transcriptome through SMRT</h3>
<p>In order to obtain a comprehensive full-length transcriptome for <em>N</em>. <em>nucifera</em>, high-quality RNA was extracted from the above-mentioned 19 different samples including some tissues at different developmental stages. The same amount of RNA per sample was pooled, and size-fractionated library was sequenced on one SMRT cell using PacBio Sequel system. The raw data were then subjected to a series of analyses sequentially (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). After filtering out low-quality raw reads (length &lt;200 bp and sequence accuracy &lt;0.75) a total of 10,481,648 Reads of Inserts (ROI, 14.55 G) were obtained (<a reveal-id="dsz010-T1" data-open="dsz010-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Further analysis using the Iso-seq protocol generated 670,929 CCSs (<a reveal-id="dsz010-T1" data-open="dsz010-T1" class="link link-reveal link-table xref-fig">Table 1</a>), among which 151,262 and 519,667 were classified as non-full-length and full-length sequences (the reads with 5ʹ primers, 3ʹ primers and poly-A tails), respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>; <a reveal-id="dsz010-T1" data-open="dsz010-T1" class="link link-reveal link-table xref-fig">Table 1</a>). After excluding the chimeric reads, 457,565 full-length non-chimeric (FLNC) reads were clustered using the consensus sequences based on ICE algorithm, accumulating 211,802 polished consensus reads (<a reveal-id="dsz010-T1" data-open="dsz010-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The length of the 211,802 polished consensus reads ranged from 137 bp to 17,645 bp with an average of 2,233 bp (<a reveal-id="dsz010-T1" data-open="dsz010-T1" class="link link-reveal link-table xref-fig">Table 1</a>; <a reveal-id="dsz010-F1" data-open="dsz010-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>), similar to the 80th percentile value of the reference annotation gene length of 2,184 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). </p>                        <a id="140465385" scrollto-destination="140465385"></a>
<div id="dsz010-F1" data-id="dsz010-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f1.png?Expires=1576221205&amp;Signature=G7oyhVRH-dE0hU3efMSWpCDshCv3vAF3zRfpx-lLoQ1~IffJiyCkbVUyeoQtPfKKvtbxV9prrOtm9p-vFZronzgTCTYuF4az0riyKe8yjo38QFgM9HSY74N0caI3Rak2uyZ0mW-ruJ98l1ckgH0mXXtN9FOxxoAt5en41WbWuT0cApYF8AzRL7haI5xxEVSue8q5wx1FhMGNJZFrS7i5zPPYi9IXnty4D-TIGH9~IkLr-AZFhMiWWWLYYHM6PMqpIwnmNQ9UlkLnqRH4K21ky8u~PJ5fMdHK7uI-Wn2HJv9eSRnirQyeeQSMaD93FZ~5pnqI~CqEYLAz3YDaM--QaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of lotus transcriptome using Iso-Seq. (a) Density distribution of FLNC reads length obtained by SMART sequencing. (b) The statistics of GMAP mapping by full-length consensus reads. (c) Density distribution of full-length reads in reference genome of lotus. The green is the ‘+’ strand and the orange is the ‘−’ strand. Only the top 10 and other megascaffolds with suitable size and condense mapped reads were shown here. (d) Distribution of the number of poly(A) sites per gene. (e) Ratio of intra- and inter-chromosome fusion transcripts." path-from-xml="dsz010f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465385/dsz010f1.tif" path-from-xml="dsz010f1.tif" target="_blank">Open in new tab</a><a section="140465385" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f1.png?Expires=1576221205&amp;Signature=hMaTH4~QamkKHEof1DAtzTn7f2pcxe8G1spLk0y-sWt1d6VzqVW3s1OezSAwcbrqCs3Nu72VzMpa418UDt84Se39BBPtVFowyi93ZgmtEicGB~s0oaxKVBlVSch8FUIru7vWjH-oPvbpK3JNocCcy~lp5zsGGeQqGvtcnq7su37mNCpxGsF8FFwctcacTUlTGXQOFx-3iY1NjAyxMEuvc~DSjwa~orinwOJI3cVXM4C7nmelSBavTOLvcZ~9ltwK-QFNeWqYmVydvwunSn-0au9xrRGih7iHsnu3w71Dpypgnd-zFppwfWFk-kBAPyIu9ftkdFzDtnHxoth4QEw-oQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465385&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of lotus transcriptome using Iso-Seq. (a) Density distribution of FLNC reads length obtained by SMART sequencing. (b) The statistics of GMAP mapping by full-length consensus reads. (c) Density distribution of full-length reads in reference genome of lotus. The green is the ‘+’ strand and the orange is the ‘−’ strand. Only the top 10 and other megascaffolds with suitable size and condense mapped reads were shown here. (d) Distribution of the number of poly(A) sites per gene. (e) Ratio of intra- and inter-chromosome fusion transcripts.</p></div>
</div>
<div content-id="dsz010-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f1.png?Expires=1576221205&amp;Signature=G7oyhVRH-dE0hU3efMSWpCDshCv3vAF3zRfpx-lLoQ1~IffJiyCkbVUyeoQtPfKKvtbxV9prrOtm9p-vFZronzgTCTYuF4az0riyKe8yjo38QFgM9HSY74N0caI3Rak2uyZ0mW-ruJ98l1ckgH0mXXtN9FOxxoAt5en41WbWuT0cApYF8AzRL7haI5xxEVSue8q5wx1FhMGNJZFrS7i5zPPYi9IXnty4D-TIGH9~IkLr-AZFhMiWWWLYYHM6PMqpIwnmNQ9UlkLnqRH4K21ky8u~PJ5fMdHK7uI-Wn2HJv9eSRnirQyeeQSMaD93FZ~5pnqI~CqEYLAz3YDaM--QaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of lotus transcriptome using Iso-Seq. (a) Density distribution of FLNC reads length obtained by SMART sequencing. (b) The statistics of GMAP mapping by full-length consensus reads. (c) Density distribution of full-length reads in reference genome of lotus. The green is the ‘+’ strand and the orange is the ‘−’ strand. Only the top 10 and other megascaffolds with suitable size and condense mapped reads were shown here. (d) Distribution of the number of poly(A) sites per gene. (e) Ratio of intra- and inter-chromosome fusion transcripts." path-from-xml="dsz010f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465385/dsz010f1.tif" path-from-xml="dsz010f1.tif" target="_blank">Open in new tab</a><a section="140465385" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f1.png?Expires=1576221205&amp;Signature=hMaTH4~QamkKHEof1DAtzTn7f2pcxe8G1spLk0y-sWt1d6VzqVW3s1OezSAwcbrqCs3Nu72VzMpa418UDt84Se39BBPtVFowyi93ZgmtEicGB~s0oaxKVBlVSch8FUIru7vWjH-oPvbpK3JNocCcy~lp5zsGGeQqGvtcnq7su37mNCpxGsF8FFwctcacTUlTGXQOFx-3iY1NjAyxMEuvc~DSjwa~orinwOJI3cVXM4C7nmelSBavTOLvcZ~9ltwK-QFNeWqYmVydvwunSn-0au9xrRGih7iHsnu3w71Dpypgnd-zFppwfWFk-kBAPyIu9ftkdFzDtnHxoth4QEw-oQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465385&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of lotus transcriptome using Iso-Seq. (a) Density distribution of FLNC reads length obtained by SMART sequencing. (b) The statistics of GMAP mapping by full-length consensus reads. (c) Density distribution of full-length reads in reference genome of lotus. The green is the ‘+’ strand and the orange is the ‘−’ strand. Only the top 10 and other megascaffolds with suitable size and condense mapped reads were shown here. (d) Distribution of the number of poly(A) sites per gene. (e) Ratio of intra- and inter-chromosome fusion transcripts.</p></div>
</div>                        <a id="140465386" scrollto-destination="140465386"></a>
<div content-id="dsz010-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz010-T1" class="table-wrap-title" data-id="dsz010-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Summary of PacBio Iso-Seq dataset</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Category </th>
<th>Dataset </th>
</tr></thead>
<tbody>
<tr>
<td>Subreads number </td>
<td>10,481,648 </td>
</tr>
<tr>
<td>Average subreads length </td>
<td>1,389 </td>
</tr>
<tr>
<td>Subreads N50 </td>
<td>2,034 </td>
</tr>
<tr>
<td>CCS </td>
<td>670,929 </td>
</tr>
<tr>
<td>5ʹ-primer </td>
<td>585,493 </td>
</tr>
<tr>
<td>3ʹ-primer </td>
<td>605,695 </td>
</tr>
<tr>
<td>Poly-A </td>
<td>588,183 </td>
</tr>
<tr>
<td>Full length </td>
<td>519,667 </td>
</tr>
<tr>
<td>FLNC </td>
<td>457,565 </td>
</tr>
<tr>
<td>Average FLNC read length </td>
<td>2,080 </td>
</tr>
<tr>
<td>FLNC/CCS </td>
<td>0.68 </td>
</tr>
<tr>
<td>Polished consensus reads </td>
<td>211,802 </td>
</tr>
<tr>
<td>Min. length </td>
<td>137 </td>
</tr>
<tr>
<td>Max. length </td>
<td>17,645 </td>
</tr>
<tr>
<td>Average consensus reads length </td>
<td>2,233 </td>
</tr>
<tr>
<td>Polished N50 </td>
<td>2,326 </td>
</tr>
<tr>
<td>Total mapped polished consensus reads </td>
<td>196,329 </td>
</tr>
<tr>
<td>Multiple mapped reads </td>
<td>3,639 </td>
</tr>
<tr>
<td>Uniquely mapped reads </td>
<td>192,690 </td>
</tr>
<tr>
<td>Reads map to ‘+’ </td>
<td>105,814 </td>
</tr>
<tr>
<td>Reads map to ‘−’ </td>
<td>86,876 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz010-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Category </th>
<th>Dataset </th>
</tr></thead>
<tbody>
<tr>
<td>Subreads number </td>
<td>10,481,648 </td>
</tr>
<tr>
<td>Average subreads length </td>
<td>1,389 </td>
</tr>
<tr>
<td>Subreads N50 </td>
<td>2,034 </td>
</tr>
<tr>
<td>CCS </td>
<td>670,929 </td>
</tr>
<tr>
<td>5ʹ-primer </td>
<td>585,493 </td>
</tr>
<tr>
<td>3ʹ-primer </td>
<td>605,695 </td>
</tr>
<tr>
<td>Poly-A </td>
<td>588,183 </td>
</tr>
<tr>
<td>Full length </td>
<td>519,667 </td>
</tr>
<tr>
<td>FLNC </td>
<td>457,565 </td>
</tr>
<tr>
<td>Average FLNC read length </td>
<td>2,080 </td>
</tr>
<tr>
<td>FLNC/CCS </td>
<td>0.68 </td>
</tr>
<tr>
<td>Polished consensus reads </td>
<td>211,802 </td>
</tr>
<tr>
<td>Min. length </td>
<td>137 </td>
</tr>
<tr>
<td>Max. length </td>
<td>17,645 </td>
</tr>
<tr>
<td>Average consensus reads length </td>
<td>2,233 </td>
</tr>
<tr>
<td>Polished N50 </td>
<td>2,326 </td>
</tr>
<tr>
<td>Total mapped polished consensus reads </td>
<td>196,329 </td>
</tr>
<tr>
<td>Multiple mapped reads </td>
<td>3,639 </td>
</tr>
<tr>
<td>Uniquely mapped reads </td>
<td>192,690 </td>
</tr>
<tr>
<td>Reads map to ‘+’ </td>
<td>105,814 </td>
</tr>
<tr>
<td>Reads map to ‘−’ </td>
<td>86,876 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465386" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz010-T1" class="table-wrap-title" data-id="dsz010-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Summary of PacBio Iso-Seq dataset</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Category </th>
<th>Dataset </th>
</tr></thead>
<tbody>
<tr>
<td>Subreads number </td>
<td>10,481,648 </td>
</tr>
<tr>
<td>Average subreads length </td>
<td>1,389 </td>
</tr>
<tr>
<td>Subreads N50 </td>
<td>2,034 </td>
</tr>
<tr>
<td>CCS </td>
<td>670,929 </td>
</tr>
<tr>
<td>5ʹ-primer </td>
<td>585,493 </td>
</tr>
<tr>
<td>3ʹ-primer </td>
<td>605,695 </td>
</tr>
<tr>
<td>Poly-A </td>
<td>588,183 </td>
</tr>
<tr>
<td>Full length </td>
<td>519,667 </td>
</tr>
<tr>
<td>FLNC </td>
<td>457,565 </td>
</tr>
<tr>
<td>Average FLNC read length </td>
<td>2,080 </td>
</tr>
<tr>
<td>FLNC/CCS </td>
<td>0.68 </td>
</tr>
<tr>
<td>Polished consensus reads </td>
<td>211,802 </td>
</tr>
<tr>
<td>Min. length </td>
<td>137 </td>
</tr>
<tr>
<td>Max. length </td>
<td>17,645 </td>
</tr>
<tr>
<td>Average consensus reads length </td>
<td>2,233 </td>
</tr>
<tr>
<td>Polished N50 </td>
<td>2,326 </td>
</tr>
<tr>
<td>Total mapped polished consensus reads </td>
<td>196,329 </td>
</tr>
<tr>
<td>Multiple mapped reads </td>
<td>3,639 </td>
</tr>
<tr>
<td>Uniquely mapped reads </td>
<td>192,690 </td>
</tr>
<tr>
<td>Reads map to ‘+’ </td>
<td>105,814 </td>
</tr>
<tr>
<td>Reads map to ‘−’ </td>
<td>86,876 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz010-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Category </th>
<th>Dataset </th>
</tr></thead>
<tbody>
<tr>
<td>Subreads number </td>
<td>10,481,648 </td>
</tr>
<tr>
<td>Average subreads length </td>
<td>1,389 </td>
</tr>
<tr>
<td>Subreads N50 </td>
<td>2,034 </td>
</tr>
<tr>
<td>CCS </td>
<td>670,929 </td>
</tr>
<tr>
<td>5ʹ-primer </td>
<td>585,493 </td>
</tr>
<tr>
<td>3ʹ-primer </td>
<td>605,695 </td>
</tr>
<tr>
<td>Poly-A </td>
<td>588,183 </td>
</tr>
<tr>
<td>Full length </td>
<td>519,667 </td>
</tr>
<tr>
<td>FLNC </td>
<td>457,565 </td>
</tr>
<tr>
<td>Average FLNC read length </td>
<td>2,080 </td>
</tr>
<tr>
<td>FLNC/CCS </td>
<td>0.68 </td>
</tr>
<tr>
<td>Polished consensus reads </td>
<td>211,802 </td>
</tr>
<tr>
<td>Min. length </td>
<td>137 </td>
</tr>
<tr>
<td>Max. length </td>
<td>17,645 </td>
</tr>
<tr>
<td>Average consensus reads length </td>
<td>2,233 </td>
</tr>
<tr>
<td>Polished N50 </td>
<td>2,326 </td>
</tr>
<tr>
<td>Total mapped polished consensus reads </td>
<td>196,329 </td>
</tr>
<tr>
<td>Multiple mapped reads </td>
<td>3,639 </td>
</tr>
<tr>
<td>Uniquely mapped reads </td>
<td>192,690 </td>
</tr>
<tr>
<td>Reads map to ‘+’ </td>
<td>105,814 </td>
</tr>
<tr>
<td>Reads map to ‘−’ </td>
<td>86,876 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/140465386" target="_blank">Open in new tab</a></div>
</div></div>
<p>To further improve the accuracy of the consensus reads, Illumina RNA-seq was also applied on the same 19 samples separately. A total of 1.6 billion high-quality short reads were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Subsequently, we utilized LoRDEC to improve the accuracy of transcripts. The corrected consensus reads were then mapped to the lotus reference genome using GMAP. A total of 192,690 (90.98%) non-redundant isoforms were mapped to unique sequences on the genome, including 105,814 and 86,876 to the ‘+’ and ‘−’ strands, respectively, and 3,639 (1.72%) were mapped to multi-sequences, while the rest were not mapped onto the genome (<a reveal-id="dsz010-T1" data-open="dsz010-T1" class="link link-reveal link-table xref-fig">Table 1</a>; <a reveal-id="dsz010-F1" data-open="dsz010-F1" class="link link-reveal link-table xref-fig">Fig.1b</a>). Moreover, the distribution densities of these full-length consensus reads in reference genome are similar among different megascaffolds, except for megascaffold 4 being the densest one (<a reveal-id="dsz010-F1" data-open="dsz010-F1" class="link link-reveal link-table xref-fig">Fig.1c</a>).</p>                    <h3 scrollto-destination="140465388" id="140465388" class="section-title">3.2. Improving <em>N. nucifera</em> genome annotation by SMRT sequencing</h3>
<p>Considering the advantage of the sequencing length of SMRT technique, much focus has been directed towards the optimization of gene structure and discovery new transcript isoforms, thus leading to the development of various data analysis tools and pipelines, such as TAPIS<a reveal-id="dsz010-B27" data-open="dsz010-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and IDP.<a reveal-id="dsz010-B49" data-open="dsz010-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Post-transcriptional Regulation Analysis Pipeline for Iso-Seq (PRAPI) was used in the identification of post-transcriptional regulation for its celerity and one-stop operation employed in this study. APA and ATI has been deemed to have capacity for regulating gene expression and increasing transcriptome complexity.<a reveal-id="dsz010-B50" data-open="dsz010-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Using the RPAPI, 22,314 genes detected by PacBio have at least one poly(A) site, with 4,850 genes having at least five ploy(A) sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>; Figs 1d and 2). The average number of poly(A) site per gene was 3.07. The ATI events were identified in 11,719 genes detected by PacBio (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>; <a reveal-id="dsz010-F2" data-open="dsz010-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). In addition, 271 transcripts mapped at different chromosomes or different loci in the same chromosome were identified as fusion transcripts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>; <a reveal-id="dsz010-F2" data-open="dsz010-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), with the inter-chromosome fusion transcripts being predominant (<a reveal-id="dsz010-F1" data-open="dsz010-F1" class="link link-reveal link-table xref-fig">Fig. 1e</a>).</p>
<p>To obtain comprehensive information of their function, all the isoforms were mapped in different database as described in M&amp;M. The functions of total 32,121 genes identified by Illumina and PacBio data were annotated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The existing version of lotus reference genome contains 26,685 annotated genes.<a reveal-id="dsz010-B33" data-open="dsz010-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> In this study, 192,690 unique full-length isoforms were aligned to the reference genome. Further analysis revealed that 19,332 of the reference annotated genes, corresponding to 75,068 transcripts, were supported by one or more full-length reads. A lot of isoforms could not be matched to any of the reference genes, indicating the existence of novel genes. Combining the existing mass of multiple RNA-seq datasets and the new sequencing reads in this study to map against the reference version (<a reveal-id="dsz010-F2" data-open="dsz010-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), 6,881 novel genes were identified. Approximately one-third (2,456) of the novel genes were supported by both Illumina and PacBio datasets, and the other two-thirds were supported by Illumina (2,298) and PacBio (2,127) data, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S7</a></span>). To verify the existence of these new genes, a total of 20 new genes were randomly selected and identified by RT-PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). It seems that Illumina-seq and PacBio datasets had nearly similar contribution in discovering novel genes, although the depth of these two was different. To evaluate the homology of these novel genes in Arabidopsis, BLAST analyses were carried out using blastn as described in the Materials and methods section. There are only 396 novel genes with at least one significant match against Arabidopsis cDNA. </p>                        <a id="140465391" scrollto-destination="140465391"></a>
<div id="dsz010-F2" data-id="dsz010-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f2.png?Expires=1576221205&amp;Signature=IJqKCLtlDY6Nb56mSzTfZExHl7tpu~LP4L3IVhytnjDG3v1m9isyTu4jUcv1KWcqFtV9vZh1qaE1d~geQx3g2JjrAMZq1wC9PIpEHn5O7RtQo5syJYF4mHApMOzkumP8DVuyjYsXdtzY4ltfyEMOZCk2yxgc-F2Q0-uzLOItjsb1xueiDOtJKlVRLVVHOWHLHRsBHpO84c7QUmS8omeJ1plMjac75Tqm7c4Nrv8Tn~ck2774bX25S0gsuAey-~O6YPT5pJ0NCDFn-Dh4oXBvBmC2d2eV~ENIhwM17yQB7Vb2pJYTnNsLPFZ8Of1S3BpSgOpoSfrVTcnra9W9Tu9N6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CIRCOS visualization of distribution of different data at genome-wide level. (a) Gene density of reference genome. (b) Density of the predicted genes from PacBio data. (c) Density of predicted genes from hybrid data consisting of PacBio and Illumina data. (d) Distribution of the loci of predicted TF. (e) Distribution of genes with alternative poly(A) site. (f) Distribution of genes with ATI site. Linkage of fusion transcripts was showed in the most inside, with inter- and intra-chromosome, respectively. Based on the mapping results and the distribution of the fusion transcripts, only the biggest nine megascaffolds were shown here." path-from-xml="dsz010f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465391/dsz010f2.tif" path-from-xml="dsz010f2.tif" target="_blank">Open in new tab</a><a section="140465391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f2.png?Expires=1576221205&amp;Signature=t-Y9rftNahLVRF6E4ZA4rYmbknVbEZp-kiTVl~3Ce0XVFPMrWaL3PRlrGT4KB9M14q-a-CDnOF00E6VD4U1BMpPVY4iF9m1FZn8rb-N2rEO~iqqdwztefjphwtWADnWnu9xUudMAuQwC0dY-tGV6nlFF7tgopEmdPH-~kL2bKbBGFOzxKUkv24u91oytN9WhICLdHAF2ypWOfLRZAxolNvf~rXzxa0rZd4jxGQ~PSsofAlWJ6Nkn3nGf2K7F6LydBRBIBvcqDAzoZ58DTo89qTiBbRd~5MXafCSCwasXdUa~NbWCwbd4qYR6alAepmVIjBzO6vb4pl8Wmmo9iB~s-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465391&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>CIRCOS visualization of distribution of different data at genome-wide level. (a) Gene density of reference genome. (b) Density of the predicted genes from PacBio data. (c) Density of predicted genes from hybrid data consisting of PacBio and Illumina data. (d) Distribution of the loci of predicted TF. (e) Distribution of genes with alternative poly(A) site. (f) Distribution of genes with ATI site. Linkage of fusion transcripts was showed in the most inside, with inter- and intra-chromosome, respectively. Based on the mapping results and the distribution of the fusion transcripts, only the biggest nine megascaffolds were shown here.</p></div>
</div>
<div content-id="dsz010-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f2.png?Expires=1576221205&amp;Signature=IJqKCLtlDY6Nb56mSzTfZExHl7tpu~LP4L3IVhytnjDG3v1m9isyTu4jUcv1KWcqFtV9vZh1qaE1d~geQx3g2JjrAMZq1wC9PIpEHn5O7RtQo5syJYF4mHApMOzkumP8DVuyjYsXdtzY4ltfyEMOZCk2yxgc-F2Q0-uzLOItjsb1xueiDOtJKlVRLVVHOWHLHRsBHpO84c7QUmS8omeJ1plMjac75Tqm7c4Nrv8Tn~ck2774bX25S0gsuAey-~O6YPT5pJ0NCDFn-Dh4oXBvBmC2d2eV~ENIhwM17yQB7Vb2pJYTnNsLPFZ8Of1S3BpSgOpoSfrVTcnra9W9Tu9N6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CIRCOS visualization of distribution of different data at genome-wide level. (a) Gene density of reference genome. (b) Density of the predicted genes from PacBio data. (c) Density of predicted genes from hybrid data consisting of PacBio and Illumina data. (d) Distribution of the loci of predicted TF. (e) Distribution of genes with alternative poly(A) site. (f) Distribution of genes with ATI site. Linkage of fusion transcripts was showed in the most inside, with inter- and intra-chromosome, respectively. Based on the mapping results and the distribution of the fusion transcripts, only the biggest nine megascaffolds were shown here." path-from-xml="dsz010f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465391/dsz010f2.tif" path-from-xml="dsz010f2.tif" target="_blank">Open in new tab</a><a section="140465391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f2.png?Expires=1576221205&amp;Signature=t-Y9rftNahLVRF6E4ZA4rYmbknVbEZp-kiTVl~3Ce0XVFPMrWaL3PRlrGT4KB9M14q-a-CDnOF00E6VD4U1BMpPVY4iF9m1FZn8rb-N2rEO~iqqdwztefjphwtWADnWnu9xUudMAuQwC0dY-tGV6nlFF7tgopEmdPH-~kL2bKbBGFOzxKUkv24u91oytN9WhICLdHAF2ypWOfLRZAxolNvf~rXzxa0rZd4jxGQ~PSsofAlWJ6Nkn3nGf2K7F6LydBRBIBvcqDAzoZ58DTo89qTiBbRd~5MXafCSCwasXdUa~NbWCwbd4qYR6alAepmVIjBzO6vb4pl8Wmmo9iB~s-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465391&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>CIRCOS visualization of distribution of different data at genome-wide level. (a) Gene density of reference genome. (b) Density of the predicted genes from PacBio data. (c) Density of predicted genes from hybrid data consisting of PacBio and Illumina data. (d) Distribution of the loci of predicted TF. (e) Distribution of genes with alternative poly(A) site. (f) Distribution of genes with ATI site. Linkage of fusion transcripts was showed in the most inside, with inter- and intra-chromosome, respectively. Based on the mapping results and the distribution of the fusion transcripts, only the biggest nine megascaffolds were shown here.</p></div>
</div>
<p>In addition, the iTAK software, a programme for classification of plant transcription factor (TF), was used in the identification of TFs by binding to specific <em>cis</em>-elements in promoter regions. A total of 7,554 TFs of 50 families were predicted in lotus genome using iTAK tool, and their distribution and gene loci are shown in (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>; <a reveal-id="dsz010-F2" data-open="dsz010-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The main TFs identified in our research belong to the MYB (655), AP2 (562), bHLH (531), and C3H (494) families.</p>                    <h3 scrollto-destination="140465393" id="140465393" class="section-title">3.3. Analysis of AS</h3>
<p>Based on the method described in M&amp;M, a total of 104,288 AS events were identified from 16,543 reference gene models, accounting for 63.47% of the total annotated protein-coding genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). To obtain detailed information, the frequency of seven primary AS types, including skipping exon (SE), mutually exclusive exon (MX), alternative 5ʹ splice-site (A5), alternative 3ʹ splice-site (A3), retained intron (RI), alternative first exon (AF), alternative last exon (AL), and the number of corresponding annotated genes were calculated (<a reveal-id="dsz010-F3" data-open="dsz010-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). The most abundant AS events in lotus is A3 followed by RI and A5, which is different from other plants having RI as the major AS event. Among the genes regulated by AS, about two-thirds of them (67.33%) were associated with A3 events, and only 5.4% of them were associated with MX events. We found 4,944 (44.38%) of genes regulated by A3 events having one A3 gene loci, the highest number of A3 event identified in a single gene was 38 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). We analysed the AS based on PacBio and Illumina datasets, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>), in order to gain further insights on the effect of different sequencing techniques on the AS events. Although the number of AS events identified by RNA-seq is more compared with the AS identified by PacBio-Seq, the ratios among different AS events are very similar, with A3 being the major AS event (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6a</a></span>). </p>                        <a id="140465395" scrollto-destination="140465395"></a>
<div id="dsz010-F3" data-id="dsz010-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f3.png?Expires=1576221205&amp;Signature=RHE1GmhL4wGtpXApvzzNZrSY-17q0xl5b5AL~agfaKebRXBfuAY2aQpZUABjTqzu0hjCET8kh~OWzkoul1VEM32jMK3Eqc-jh57ZlFaXodC2jpD9LHFJ6wShMKoB8sXv6YaKGE0ENXwaX3SO5eNhKOko1X83OqXPssiNB3qveJWBCRujTyosl1f-MXgAYqLmAuN~Z4~JJdxi6ELFc1wAEyFw7PrY4UUTeJtN0vDbsyNU7YQvwqWZs5fcmsUeJ8382vwVvNJnthpSHCle9Tk78da5Te6sXz7CjrjqdQnMUwfPBI0NTuf4Y8j2LU5RcEKmgpr-z8VeYey7dxzzKO0t0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of AS events and percentage of AS events in tissues. (a) Characterization of AS events. Seven different types of AS events were showed. The number of AS events and corresponding percentage of all AS events were calculated. The number of gene associated with AS event and proportions of these genes occupying all genes undergoing AS. (b) Distribution of five main types of AS events in different tissues." path-from-xml="dsz010f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465395/dsz010f3.tif" path-from-xml="dsz010f3.tif" target="_blank">Open in new tab</a><a section="140465395" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f3.png?Expires=1576221205&amp;Signature=DTKqyYkWgtOpjYZlp~sCdbuSmMNVYaY~G8CvOrO-JAojFZF7w2j~L4T9qJ5XPLMmPdITxlzeBJwS4oqnUzcziiCSylwPbbLGAsLdCUNHk-X7UhcSmzP5BVcYMwpPNqZ150yvvM~nPHHjSn34ggauDKntU6V-k0qLGgB2D~pzj7xNyXDRYDQQiU0pkifnUXU-REpo~s7CDK8YelkkK-rZx37S1ZaPT1EB3hg9KRT06QgFO15ju-iBeWdyybv9dse9UMpwvidGqZjpDDdVwpvKpUoGGS9X2xa0fuIhDutrMjliLEI-y0~5he4G8HYirJcs0w2jen0oTuyg3mdTMkMiCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465395&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of AS events and percentage of AS events in tissues. (a) Characterization of AS events. Seven different types of AS events were showed. The number of AS events and corresponding percentage of all AS events were calculated. The number of gene associated with AS event and proportions of these genes occupying all genes undergoing AS. (b) Distribution of five main types of AS events in different tissues.</p></div>
</div>
<div content-id="dsz010-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f3.png?Expires=1576221205&amp;Signature=RHE1GmhL4wGtpXApvzzNZrSY-17q0xl5b5AL~agfaKebRXBfuAY2aQpZUABjTqzu0hjCET8kh~OWzkoul1VEM32jMK3Eqc-jh57ZlFaXodC2jpD9LHFJ6wShMKoB8sXv6YaKGE0ENXwaX3SO5eNhKOko1X83OqXPssiNB3qveJWBCRujTyosl1f-MXgAYqLmAuN~Z4~JJdxi6ELFc1wAEyFw7PrY4UUTeJtN0vDbsyNU7YQvwqWZs5fcmsUeJ8382vwVvNJnthpSHCle9Tk78da5Te6sXz7CjrjqdQnMUwfPBI0NTuf4Y8j2LU5RcEKmgpr-z8VeYey7dxzzKO0t0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of AS events and percentage of AS events in tissues. (a) Characterization of AS events. Seven different types of AS events were showed. The number of AS events and corresponding percentage of all AS events were calculated. The number of gene associated with AS event and proportions of these genes occupying all genes undergoing AS. (b) Distribution of five main types of AS events in different tissues." path-from-xml="dsz010f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465395/dsz010f3.tif" path-from-xml="dsz010f3.tif" target="_blank">Open in new tab</a><a section="140465395" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f3.png?Expires=1576221205&amp;Signature=DTKqyYkWgtOpjYZlp~sCdbuSmMNVYaY~G8CvOrO-JAojFZF7w2j~L4T9qJ5XPLMmPdITxlzeBJwS4oqnUzcziiCSylwPbbLGAsLdCUNHk-X7UhcSmzP5BVcYMwpPNqZ150yvvM~nPHHjSn34ggauDKntU6V-k0qLGgB2D~pzj7xNyXDRYDQQiU0pkifnUXU-REpo~s7CDK8YelkkK-rZx37S1ZaPT1EB3hg9KRT06QgFO15ju-iBeWdyybv9dse9UMpwvidGqZjpDDdVwpvKpUoGGS9X2xa0fuIhDutrMjliLEI-y0~5he4G8HYirJcs0w2jen0oTuyg3mdTMkMiCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465395&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of AS events and percentage of AS events in tissues. (a) Characterization of AS events. Seven different types of AS events were showed. The number of AS events and corresponding percentage of all AS events were calculated. The number of gene associated with AS event and proportions of these genes occupying all genes undergoing AS. (b) Distribution of five main types of AS events in different tissues.</p></div>
</div>
<p>To ascertain the relative importance of the five main models of AS (RI, SE, A3, A5, and MX) in different developmental stages, the RNA-seq datasets of different samples was used to identify AS events, respectively (<a reveal-id="dsz010-F3" data-open="dsz010-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). Splicing mode was generally uniform across the 19 samples with little variation on the ratio of different AS events among different tissues. Excluding the two fast growing tissues (elongation zone and apical meristem), RI was the common in all samples, accounting for approximately 31% of alternative transcripts. The number of A3 was the second most predominant. Moreover, the degree of the gene model regulated by AS events in every sample is different, indicating the potential existence of specifically expressive AS events (<a reveal-id="dsz010-F3" data-open="dsz010-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). To filtrate the tissue-specifically expressed AS events (SEASs), rMATS were used in our study.<a reveal-id="dsz010-B44" data-open="dsz010-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Among the 12 tissues, a total of 370 SEASs were predicted according to the expression of the splice junctions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6b</a></span>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). In particular, the largest number of SEASs found in stamen and carpel tissues, followed by receptacle and seed cotyledon, while no SEASs is identified in the elongation zone. Out of the 370 SEASs, SE was the major one among the five main types of AS events. These results showed that AS might be more preferentially occurring in fast-changing and rapid developing tissues.</p>
<p>To avoid the possible interference by false assembly or informatics artefacts, AS events were generated randomly through the selection of 10 genes to perform independent validation by RT-PCR (<a reveal-id="dsz010-F4" data-open="dsz010-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). We designed primers for the neighbouring regions of these AS junctions and judged the existence of AS events on the basis of the multiple bands. The gel banding pattern and the expected size of the fragments were consistent with the amplified splice isoforms. Some splicing fragments have precedence for high expression in specific tissue. For example, an AS junction from gene <em>XLOC_025058</em>, predicted as SEASs, had tissue-preferential high expression in the petal (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). An isoform from gene <em>XLOC_019602</em>, produced by an SE event, exhibited high expression level in the petal, stamen, and petiole but low or none in other tissues (<a reveal-id="dsz010-F4" data-open="dsz010-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). </p>                        <a id="140465398" scrollto-destination="140465398"></a>
<div id="dsz010-F4" data-id="dsz010-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f4.png?Expires=1576221205&amp;Signature=Yax-VfjjMK3~pktqhK~qdClGQMadlyZKwt3hu~YaleMSAzPEkn7anMJ3z3ZaHSPO1bJ34pHye1sVSu916WYH84a6bFnEM9jywjcJIP58BnWm~78Qi2VMLRZUgX9OXw~fmVL1yKJTOBrlaSEOyRMFHAQ2s-GWRH9rL1K6YiBsVMPvqQFHdW68jsbykKKCSO3S-FamA3EJrbZSAMTZx9nKfWmmdLe4KLRGbjGbCPGNODyaWXxCrleFYmkJ1TxNjrqITSxCI5hyCAd44Rl0H0zQfen32TAULqI2oeEXYYybWqXCv6jyQB3ssqhTu3jKp-rccptnVreGj4XCY5wCRbvEfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT-PCR validation of AS events for five genes. Gel bands in each image show DNA makers and PCR results in 13 different samples. Predicted structure of each isoform is shown in right panel. Boxes and line with arrows show exons and introns, respectively. The predicted size of each full length is shown at the end of each line. The loci of PCR primers (F, forward and R, reverse) are shown on the first isoform of each gene." path-from-xml="dsz010f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465398/dsz010f4.tif" path-from-xml="dsz010f4.tif" target="_blank">Open in new tab</a><a section="140465398" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f4.png?Expires=1576221205&amp;Signature=Vtxe8QdH01YyPzk7RGQjRPNSsDIpF9Elr2~OdjEPBeo~BF3FpWgHe-nUnBANs4mJeln1LmAQRcrFnjRv0u4O1Um31qvz2r-j0aDdVRcpU7Ex0tN5U8xKDVHfqbOj2tJoKvRViMW8Rz-Btczn7cdQrmv-lvJV77zhyYEoJu36CvCO4XW-bnm9Mkp6nz171pWTppXASFUp0iTLn-Y-G2iDvYEHBWqqDmn-hUEtTNbwVmlRNH4NAMV9EA2wSHwSN1xqYKdrMKlbwea7Jed3yDR9ZNlMajr8vscIZMOnLHDbZ-uT5czJ1wlrwen83WH1-FGdaa~2s8~2CbhqYbmGy0C34g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465398&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>RT-PCR validation of AS events for five genes. Gel bands in each image show DNA makers and PCR results in 13 different samples. Predicted structure of each isoform is shown in right panel. Boxes and line with arrows show exons and introns, respectively. The predicted size of each full length is shown at the end of each line. The loci of PCR primers (F, forward and R, reverse) are shown on the first isoform of each gene.</p></div>
</div>
<div content-id="dsz010-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f4.png?Expires=1576221205&amp;Signature=Yax-VfjjMK3~pktqhK~qdClGQMadlyZKwt3hu~YaleMSAzPEkn7anMJ3z3ZaHSPO1bJ34pHye1sVSu916WYH84a6bFnEM9jywjcJIP58BnWm~78Qi2VMLRZUgX9OXw~fmVL1yKJTOBrlaSEOyRMFHAQ2s-GWRH9rL1K6YiBsVMPvqQFHdW68jsbykKKCSO3S-FamA3EJrbZSAMTZx9nKfWmmdLe4KLRGbjGbCPGNODyaWXxCrleFYmkJ1TxNjrqITSxCI5hyCAd44Rl0H0zQfen32TAULqI2oeEXYYybWqXCv6jyQB3ssqhTu3jKp-rccptnVreGj4XCY5wCRbvEfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT-PCR validation of AS events for five genes. Gel bands in each image show DNA makers and PCR results in 13 different samples. Predicted structure of each isoform is shown in right panel. Boxes and line with arrows show exons and introns, respectively. The predicted size of each full length is shown at the end of each line. The loci of PCR primers (F, forward and R, reverse) are shown on the first isoform of each gene." path-from-xml="dsz010f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465398/dsz010f4.tif" path-from-xml="dsz010f4.tif" target="_blank">Open in new tab</a><a section="140465398" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f4.png?Expires=1576221205&amp;Signature=Vtxe8QdH01YyPzk7RGQjRPNSsDIpF9Elr2~OdjEPBeo~BF3FpWgHe-nUnBANs4mJeln1LmAQRcrFnjRv0u4O1Um31qvz2r-j0aDdVRcpU7Ex0tN5U8xKDVHfqbOj2tJoKvRViMW8Rz-Btczn7cdQrmv-lvJV77zhyYEoJu36CvCO4XW-bnm9Mkp6nz171pWTppXASFUp0iTLn-Y-G2iDvYEHBWqqDmn-hUEtTNbwVmlRNH4NAMV9EA2wSHwSN1xqYKdrMKlbwea7Jed3yDR9ZNlMajr8vscIZMOnLHDbZ-uT5czJ1wlrwen83WH1-FGdaa~2s8~2CbhqYbmGy0C34g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465398&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>RT-PCR validation of AS events for five genes. Gel bands in each image show DNA makers and PCR results in 13 different samples. Predicted structure of each isoform is shown in right panel. Boxes and line with arrows show exons and introns, respectively. The predicted size of each full length is shown at the end of each line. The loci of PCR primers (F, forward and R, reverse) are shown on the first isoform of each gene.</p></div>
</div>                    <h3 scrollto-destination="140465399" id="140465399" class="section-title">3.4. Tissue-specific genes and isoforms</h3>
<p>One of the primary analyses with Illumina sequencing data involves the identification of differentially expressed genes. Pairwise comparisons among all different tissue samples were performed according to the criteria of fold change &gt;2 and FDR &lt;0.05, to identify the differentially expressed genes and tissue-specific genes. A total of 13,425 genes were identified as tissue-specific genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>), and five randomly selected tissue-specific genes were identified by qRT-PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). The number of seed-coat-specific genes (2,477) is obviously more than any other tissues (<a reveal-id="dsz010-F5" data-open="dsz010-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), indicating the unique feature of seed-coat. Interestingly, there are very few elongation zone and apical meristem specific genes (<a reveal-id="dsz010-F5" data-open="dsz010-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), revealing they are not fully differentiated tissues. </p>                        <a id="140465401" scrollto-destination="140465401"></a>
<div id="dsz010-F5" data-id="dsz010-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f5.png?Expires=1576221205&amp;Signature=mWlboY-KJFwImF0-TnCDoPmvxk1XGlbKtYzfoUuMIiWtFK5wBGISmHzjBe9hvK6~24ah3RPUIAd16NKf2BKaKs3w7HpQb1f21dAu6lO~3gFooEznRBFzR784UJ7f4IXLEY9IGr53lfaS74FbLsp2u0YrY9CvKZCmj9atlDTMS~wvsX6n4-Ah6SEJQPOKV5YYT658k9WuEZZqsXsg~LqIayY8nVIIjaV4W2uGna032OYSahumhccZfOj1J-amEziUMMBrrFpojIIoumQShQZ3FSrQyc4z8911cRV5JkMlUPYCmJg2NF1uGViesVusCuQjs6eo~eTcvk~HX6k0ZtuVxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of tissue-specific genes and isoforms. (a) Distribution of the number of tissue-specific gene and tissue-specific isoform. (b) Percentage of tissue-specific isoforms corresponding non-tissue-specific gene and tissue-specific gene." path-from-xml="dsz010f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465401/dsz010f5.tif" path-from-xml="dsz010f5.tif" target="_blank">Open in new tab</a><a section="140465401" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f5.png?Expires=1576221205&amp;Signature=pZJ6HA6TVanzLP2W-VCCJ2rUSG~PxNe57XWVXQOA5tm-NlSj6mdvb~f1drb95-ZQiA8G91uuQ-9xi2lH141nHQwajYjp~fQRlVUmlHPK6oGCZPd9tEBcuFBqBR9kn0XEH1xBK7S-eQvVgIdMDtPccc13C87SSesS7B60TmKYhOdpA3QjmtzTxW7n4GL9aBTGJ7JtiDvWaDXTBMJEWMf1aMXKpnxXa23q1TkOePTlsV78~25SokmvKADw1knzUiijtKYnQ~5~CCR73dln5ncsaggaBP4Wjtn1~Oto597kRmcUF0d-pcu4ABEUajgy7OmUUlxwTOROSLP0q7-~5XzgYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465401&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of tissue-specific genes and isoforms. (a) Distribution of the number of tissue-specific gene and tissue-specific isoform. (b) Percentage of tissue-specific isoforms corresponding non-tissue-specific gene and tissue-specific gene.</p></div>
</div>
<div content-id="dsz010-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f5.png?Expires=1576221205&amp;Signature=mWlboY-KJFwImF0-TnCDoPmvxk1XGlbKtYzfoUuMIiWtFK5wBGISmHzjBe9hvK6~24ah3RPUIAd16NKf2BKaKs3w7HpQb1f21dAu6lO~3gFooEznRBFzR784UJ7f4IXLEY9IGr53lfaS74FbLsp2u0YrY9CvKZCmj9atlDTMS~wvsX6n4-Ah6SEJQPOKV5YYT658k9WuEZZqsXsg~LqIayY8nVIIjaV4W2uGna032OYSahumhccZfOj1J-amEziUMMBrrFpojIIoumQShQZ3FSrQyc4z8911cRV5JkMlUPYCmJg2NF1uGViesVusCuQjs6eo~eTcvk~HX6k0ZtuVxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of tissue-specific genes and isoforms. (a) Distribution of the number of tissue-specific gene and tissue-specific isoform. (b) Percentage of tissue-specific isoforms corresponding non-tissue-specific gene and tissue-specific gene." path-from-xml="dsz010f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465401/dsz010f5.tif" path-from-xml="dsz010f5.tif" target="_blank">Open in new tab</a><a section="140465401" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f5.png?Expires=1576221205&amp;Signature=pZJ6HA6TVanzLP2W-VCCJ2rUSG~PxNe57XWVXQOA5tm-NlSj6mdvb~f1drb95-ZQiA8G91uuQ-9xi2lH141nHQwajYjp~fQRlVUmlHPK6oGCZPd9tEBcuFBqBR9kn0XEH1xBK7S-eQvVgIdMDtPccc13C87SSesS7B60TmKYhOdpA3QjmtzTxW7n4GL9aBTGJ7JtiDvWaDXTBMJEWMf1aMXKpnxXa23q1TkOePTlsV78~25SokmvKADw1knzUiijtKYnQ~5~CCR73dln5ncsaggaBP4Wjtn1~Oto597kRmcUF0d-pcu4ABEUajgy7OmUUlxwTOROSLP0q7-~5XzgYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465401&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of tissue-specific genes and isoforms. (a) Distribution of the number of tissue-specific gene and tissue-specific isoform. (b) Percentage of tissue-specific isoforms corresponding non-tissue-specific gene and tissue-specific gene.</p></div>
</div>
<p>In order to explore the functions of these tissue-specific genes, GO functional enrichment analysis was conducted for these 12 tissues, separately (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Tissue-specific genes were classified into three categories (biological processes, molecular functions, and cellular components). In the seed-coat, the most enriched GO terms were ‘regulation of meristem growth’, ‘cysteine biosynthetic process’, ‘response to karrikin’, ‘plant-type cell wall’, and ‘integral component of membrane’. The elongation zone of the rhizome had the least tissue-specific genes, and the functions of these genes were enriched in ‘iron ion binding’ and ‘oxidoreductase activity’. The function of these tissue-specific genes was closely related to the physiological status of the corresponding tissues. For example, in leaf, the most enriched GO terms in cellular component category was ‘chloroplast stroma’, ‘chloroplast thylakoid membrane’, and ‘chloroplast envelope’, otherwise in biological processes category was ‘chlorophyll biosynthetic process’ (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>).</p>
<p>In lotus, previous transcriptomic studies mainly focused on the identification of differential expression genes in specific tissues at different developmental stages.<a reveal-id="dsz010-B51" data-open="dsz010-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> However, it is clear that most post-transcriptional regulation can change the product of gene expression. So, studies focusing on the level of different isoforms are also important. Based on the criterion described in M&amp;M, the distribution of tissue-specific isoforms generated by AS was analysed in this study to uncover the potential contribution of AS in tissue specialization. Further analyses of 12 tissues revealed that the seed-coat had the most abundant tissue-specific isoforms (662 isoforms), but the elongation zone and apical meristem had none (<a reveal-id="dsz010-F5" data-open="dsz010-F5" class="link link-reveal link-table xref-fig">Fig. 5a</a>). Interestingly, the distribution of tissue-specific isoforms among different tissues had similar patterns with the tissue-specific genes (<a reveal-id="dsz010-F5" data-open="dsz010-F5" class="link link-reveal link-table xref-fig">Fig. 5a</a>). On average, 97.3% of the number of tissue-specific isoforms had their corresponding genes being tissue-specific (<a reveal-id="dsz010-F5" data-open="dsz010-F5" class="link link-reveal link-table xref-fig">Fig. 5b</a>). However, in the cotyledon, the corresponding proportion was 88.5%, lower than other tissues, indicating the possible role of AS in the transcriptional diversity of cotyledon.</p>
<p>GO analyses showed that tissue-specific isoforms are enriched in particular biological processes varying across tissues (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S11</a></span>). In the carpel, tissue-specific isoforms were highly enriched in growth and development process, such as ‘response to growth hormone’ and ‘response to karrikin’, consistent with the role of carpel in the seed growth. More importantly, the leaf-specific transcript isoforms were enriched in chlorophyll biosynthetic process and photosynthesis. In contrast, stamen-specific transcript isoforms corresponding genes were enriched in the regulation of double fertilization forming a zygote and an endosperm.</p>
<p>To further understand the isoforms tissue-bias expression patterns, a total of 13,798 isoforms were selected for hierarchical and <em>k</em>-means clustering analysis having a mean FPKM &gt;0.1 and standard deviation/mean FPKM &gt;2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Based on the <em>k</em>-means clustering analysis, filtered isoforms were assigned into 12 clusters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S12</a></span>). The isoform levels being clustered to K1 and K8 were enriched in the cotyledon as compared with other tissues. The isoforms being clustered to K4 was enriched in the leaf and the petiole compared with other tissues, indicating strong association of isoforms in K4 to leaf and petiole. Notably, isoforms belonging to all other clusters revealed tissue-biased expression in one or two tissues other than cluster K3.</p>                    <h3 scrollto-destination="140465406" id="140465406" class="section-title">3.5. Differentially expressed genes and isoforms in floral organ specificity</h3>
<p>To identify the genes and isoforms associated with the floral development, pairwise comparison among petal, immature stamen, mature stamen, unpollinated carpel, pollinated carpel, immature receptacle, and mature receptacle was conducted. Differential expressed genes and isoforms were identified and classed into three categories: isoform differentially expressed but gene not differentially expressed (type A), both gene and isoform differentially expressed (type B), only gene differentially expressed (type C) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). In total, there were 19,021 genes and 18,399 isoforms identified as differentially expressed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). Although there were many differentially expressed isoforms, the number of type A were quite few. This indicates that regulations at isoform level and gene level are mostly correlated, except a few cases (Type A).</p>
<p>To further understand what roles these differential expressed genes and isoforms play in floral organs, GO enrichment analysis was conducted. Between immature and mature stamens, the differential expressed genes were enriched in during floral development, such as ‘inflorescence development’, ‘positive regulation of long-day photoperiodism, flowering’, and ‘response to brassinosteroid’. Between unpollinated receptacle and pollinated receptacle, the function of type A genes was enriched during ‘suspensor development’, ‘actin filament organization’, and ‘response to abscisic acid’, indicating differential expressed transcripts, corresponding gene excluding the differential expressed, had important roles in floral development. To further validate isoforms generated by AS, the RT-PCR experiment was performed to test two isoforms, TCONS_00013274 generated form XLOC_003531 gene and TCONS_00009721 generated from XLOC_002613 gene (<a reveal-id="dsz010-F6" data-open="dsz010-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). </p>                        <a id="140465409" scrollto-destination="140465409"></a>
<div id="dsz010-F6" data-id="dsz010-F6" class="fig fig-section">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f6.png?Expires=1576221205&amp;Signature=1FphxM2MfPoEPHQ~yQIH-MlFzYoL5B2YZFVX6ITpn5e7RIk8sVhyHiftUUZ4SQYciZs3rLjayhVgameeL-AGbj9c1WLkODCUW-HwLhysJqcYUwhXgiv8I4-lxdK05JAlKr~dHkZK7E3McgZHhEGE4x4mMV4odk-EFoKHAcJJZ~VHPhxKPMkbVJibcBsK0b9z2NEFpxPRWgSFy-CmF7osw9DEBCH0xvYJtM2VdE-vUMWLGtC9NJrF7YhHBeqGaMe-MQ0WmILDaPIlHIHdA5YIwzeklOq3WI8yhHbN5errSyQPgSGNnzJct3F-BKehMDHjKttvGdFNYcA4t~a-KBXihw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="qRT-PCR validation of differential expressed isoforms corresponding gene not differential expressed. (a) Isoform structure of gene XLOC_002613 was shown in left panel; the qRT-PCR results of isoform TCONS_00009721 and gene XLOC_002613 were in right panel. (b) Isoform structure of gene XLOC_003531 was shown in left panel; the qRT-PCR result of isoform TCONS_00013274 and gene XLOC_003531 were in the right panel. Grey peaks indicated read coverage. Black box and line show exons and introns. The region of the two arrows at left side is the specific region of validated isoforms. The region of the two arrows at right side is the comment region of all isoforms, the expression of this region was regarded as expression of this gene. The inserted panels are the enlarged view of the rectangle regions." path-from-xml="dsz010f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465409/dsz010f6.tif" path-from-xml="dsz010f6.tif" target="_blank">Open in new tab</a><a section="140465409" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f6.png?Expires=1576221205&amp;Signature=BPZNdmsN2ZhTuOtARJCigU3SwDyzb~x8NVkHVgImJCWbLxAMvEB2c4StgMrnlnwuZw~ntompXvKn6MiUuMs6pl0J74e-raTL5b51XfSto6XaUIS5H4YVkzW5rwsz7IJw0KkI8TlCGPPyFsZiyd6hgUSuMIQ~Ex--oll9E6fXPy3UM148j0p02edQMlAdLcR6WWV-g4idw94T-56BVzzdCUnOfkbsWQsDfjofI2Pzx4jhiQxo~TXtOKkkBY7i8nKlalTcjreCh0E2xFdJgMrZhISAgBQEcnAp1zQUsqVzhlnR2zj1ML9sQP~DbwtLyuPokU9XkY4Y1-SlVs-KEOk1-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465409&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>qRT-PCR validation of differential expressed isoforms corresponding gene not differential expressed. (a) Isoform structure of gene XLOC_002613 was shown in left panel; the qRT-PCR results of isoform TCONS_00009721 and gene XLOC_002613 were in right panel. (b) Isoform structure of gene XLOC_003531 was shown in left panel; the qRT-PCR result of isoform TCONS_00013274 and gene XLOC_003531 were in the right panel. Grey peaks indicated read coverage. Black box and line show exons and introns. The region of the two arrows at left side is the specific region of validated isoforms. The region of the two arrows at right side is the comment region of all isoforms, the expression of this region was regarded as expression of this gene. The inserted panels are the enlarged view of the rectangle regions.</p></div>
</div>
<div content-id="dsz010-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/m_dsz010f6.png?Expires=1576221205&amp;Signature=1FphxM2MfPoEPHQ~yQIH-MlFzYoL5B2YZFVX6ITpn5e7RIk8sVhyHiftUUZ4SQYciZs3rLjayhVgameeL-AGbj9c1WLkODCUW-HwLhysJqcYUwhXgiv8I4-lxdK05JAlKr~dHkZK7E3McgZHhEGE4x4mMV4odk-EFoKHAcJJZ~VHPhxKPMkbVJibcBsK0b9z2NEFpxPRWgSFy-CmF7osw9DEBCH0xvYJtM2VdE-vUMWLGtC9NJrF7YhHBeqGaMe-MQ0WmILDaPIlHIHdA5YIwzeklOq3WI8yhHbN5errSyQPgSGNnzJct3F-BKehMDHjKttvGdFNYcA4t~a-KBXihw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="qRT-PCR validation of differential expressed isoforms corresponding gene not differential expressed. (a) Isoform structure of gene XLOC_002613 was shown in left panel; the qRT-PCR results of isoform TCONS_00009721 and gene XLOC_002613 were in right panel. (b) Isoform structure of gene XLOC_003531 was shown in left panel; the qRT-PCR result of isoform TCONS_00013274 and gene XLOC_003531 were in the right panel. Grey peaks indicated read coverage. Black box and line show exons and introns. The region of the two arrows at left side is the specific region of validated isoforms. The region of the two arrows at right side is the comment region of all isoforms, the expression of this region was regarded as expression of this gene. The inserted panels are the enlarged view of the rectangle regions." path-from-xml="dsz010f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465409/dsz010f6.tif" path-from-xml="dsz010f6.tif" target="_blank">Open in new tab</a><a section="140465409" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010f6.png?Expires=1576221205&amp;Signature=BPZNdmsN2ZhTuOtARJCigU3SwDyzb~x8NVkHVgImJCWbLxAMvEB2c4StgMrnlnwuZw~ntompXvKn6MiUuMs6pl0J74e-raTL5b51XfSto6XaUIS5H4YVkzW5rwsz7IJw0KkI8TlCGPPyFsZiyd6hgUSuMIQ~Ex--oll9E6fXPy3UM148j0p02edQMlAdLcR6WWV-g4idw94T-56BVzzdCUnOfkbsWQsDfjofI2Pzx4jhiQxo~TXtOKkkBY7i8nKlalTcjreCh0E2xFdJgMrZhISAgBQEcnAp1zQUsqVzhlnR2zj1ML9sQP~DbwtLyuPokU9XkY4Y1-SlVs-KEOk1-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465409&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz010f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>qRT-PCR validation of differential expressed isoforms corresponding gene not differential expressed. (a) Isoform structure of gene XLOC_002613 was shown in left panel; the qRT-PCR results of isoform TCONS_00009721 and gene XLOC_002613 were in right panel. (b) Isoform structure of gene XLOC_003531 was shown in left panel; the qRT-PCR result of isoform TCONS_00013274 and gene XLOC_003531 were in the right panel. Grey peaks indicated read coverage. Black box and line show exons and introns. The region of the two arrows at left side is the specific region of validated isoforms. The region of the two arrows at right side is the comment region of all isoforms, the expression of this region was regarded as expression of this gene. The inserted panels are the enlarged view of the rectangle regions.</p></div>
</div>
<p>The ‘ABCE’ model controlling organ specificity has been identified in many plants, and the floral organ genes involved in ‘ABCE’ model have been identified in Arabidopsis.<a reveal-id="dsz010-B52 dsz010-B53 dsz010-B54 dsz010-B55 dsz010-B56" data-open="dsz010-B52 dsz010-B53 dsz010-B54 dsz010-B55 dsz010-B56" class="link link-ref link-reveal xref-bibr"><sup>52–56</sup></a> Therefore, we further checked the expression of key ‘ABCE’ genes in lotus base floral organs. Based on the phylogenetic tree analysis of MADS-box gene family from the Arabidopsis and lotus, the lotus ‘ABCE’ model genes were identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). According to the heatmap of MADS-box gene family in lotus, E-class genes were all highly expressed in four floral organ samples, while C-class genes were highly expressed in the carpel (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576221205&amp;Signature=NvY5F~X6RPTfCrmfChed075rVXIKcmuLJUg1HMB4TCzu~FbRtYaXHSF8Z7uSIE-XzD93by4leXEVlPe0CXBhGVzoLvTagDJ8D4-fEycsdFtLzVety09cct11Hyr-aGtcac9wXpPDyzIyu9i~-9ztL8bN2wlsqia7YivCbSOJTk79rOxNbxfNECt1pIyQ-~ywpMAVpABg0Sh2N3ek38V2WCijmkdpp~nJjsHcsB3o0X5G0MLekmiwUDg3BITzb133VJC4rL1YYNVAPtyvijW8a0PgU5nh-7qR5HOJbGGmM0lrtWXDhZX3X0W7~AGjE3lQK5rYdDFBz~5~C6RM83-kLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). In addition, B-class genes and C-class genes were highly expressed in the stamen, whereas both A- and B-class genes were highly expressed in the petal, consistent with the Arabidopsis model. Additionally, C-class genes were highly expressed in receptacle. These observations provided important insights into the ‘ABCE’ mechanism of lotus floral organ differentiation.</p>                    <h2 scrollto-destination="140465411" id="140465411" class="section-title">4. Discussion</h2>
<p>NGS has greatly boosted the understanding of gene structure,<a reveal-id="dsz010-B57" data-open="dsz010-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> transcriptional network<a reveal-id="dsz010-B58" data-open="dsz010-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a>, and small RNAs.<a reveal-id="dsz010-B59" data-open="dsz010-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> However, the limitation of sequencing length makes the assembly of full-length transcripts challenging, full-length transcript is the approved standard for gene annotation.<a reveal-id="dsz010-B60" data-open="dsz010-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> In this work, we applied SMRT sequencing in the identification of full-length splice isoforms while investigating the transcriptomic variants in lotus comprehensively, thus overcoming the disadvantage of inaccuracy and difficulty in NGS assembly. The expression of full-length transcripts and loci information would greatly improve the annotation information of lotus genome and provide deep insights into lotus transcriptional landscape.</p>
<p>In order to maximize the transcript diversity and investigate comprehensive splicing isoforms, we broadly harvested 19 samples from 12 tissues/organs. Compared with previous studies in lotus, the largest number of samples was used in our study potentially for covering the most comprehensive transcripts up-to-date. PacBio data from the combined samples might not contain sequencing depth high enough, but the sheer volume Illumina datasets partly made up for exploring low-abundance isoforms when comparing with previous SMRT transcriptomes<a reveal-id="dsz010-B27" data-open="dsz010-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsz010-B30" data-open="dsz010-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsz010-B61" data-open="dsz010-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> studies. Sparse analysis could prove the achieved near-saturation at gene level of the final read depth.<a reveal-id="dsz010-B29" data-open="dsz010-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> In this study, less than 10% of FLNCs was excavated in the &gt;6,000 bp, indicating the ability of few lotus genes in translating too long FLNCs. Furthermore, the mechanism of fusion transcripts seldom occurred.</p>
<p>Previous transcriptomic study on lotus revealed some novel genes compared with the existing annotation.<a reveal-id="dsz010-B62" data-open="dsz010-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> However, the annotation information is poor in the existing reference genome, full-length transcriptome resource replenished reference annotation is lacking. The annotation could provide aid in the discovery of novel or previously unrecognized protein-coding genes and isoforms. Through a hybrid sequencing approach combining SMRT and Illumina data, this analysis indicates that full-length transcriptome has a great potential in improving the current lotus annotation. The 6,881 novel genes here account for more than a quarter of the reference genes number, and the number of the genes supported by SMRT data and Illumina data is almost similar. The analysis shows that the initial advantage of discovering novel genes based on Iso-Seq is reduced along with the size of the Illumina sequences getting big enough. Therefore, this hybrid strategy has made considerable trade-off between the cost and efficiency and has been applied extensively in other plants, such as sweet potato,<a reveal-id="dsz010-B63" data-open="dsz010-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> barley<a reveal-id="dsz010-B64" data-open="dsz010-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> and <em>Salvia miltiorrhiza</em>.<a reveal-id="dsz010-B26" data-open="dsz010-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>
<p>AS is a crucial regulatory mechanism at the post-transcriptional level, since it contributes in transcriptome and proteome diversity.<a reveal-id="dsz010-B5" data-open="dsz010-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In lotus, AS events have been identified in four Asian lotus cultivars and the finding revealed alternative 5ʹ first exon as the predominant type of AS events, accounting for 41.2%.<a reveal-id="dsz010-B65" data-open="dsz010-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> However, it is different with other plant transcriptome study in AS. This study has a comprehensive and systematic analysis of AS in lotus, based on high-quality full-length isoforms and short reads of RNA-seq datasets. Through SMRT and Illumina sequencing, we identified a mass of putative AS events covering 49.3% genes. The ratio is less than 61% reported in Arabidopsis<a reveal-id="dsz010-B5" data-open="dsz010-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and 60.5% in cotton.<a reveal-id="dsz010-B30" data-open="dsz010-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Alternative 3ʹ splice-site events represent the largest proportion of AS and the number of genes with A3 events occupied 67.33% of all the genes undergoing AS in lotus. This kind of distribution of AS event was inconsistent with other plant studies,<a reveal-id="dsz010-B20" data-open="dsz010-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsz010-B22" data-open="dsz010-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsz010-B27" data-open="dsz010-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> where RI is the major AS event. This research also contradicts with the findings showing the predominance of RI as a mode of AS. In contrast, the RI event was the second largest category based on lotus annotation. Further AS analysis in different tissues indicated the occurrence of A3 events in the elongation zone and apical meristem of rhizome could increase the number of A3 events in the overall AS events. However, this research cannot exclude the possibility of deeper sequencing in identifying other isoforms that update the frequency of different types of AS events. Based on the SEASs analysis,<a reveal-id="dsz010-B66" data-open="dsz010-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> 94.05% SEASs predominantly occurred in the rapid growing and differentiating tissue, such as seed, carpel, and stamen. This study speculates that the genes regulatory network is complex in these tissues.</p>
<p>This study detected the functional and expressed specificity of both genes and isoforms involved in AS by predicting the annotation and expression quantity. Because the seed-coat features the largest number of tissue-specific genes and the function of these genes are enriched in development and cell wall, there is a possibility that these genes could greatly contribute to the firm structure of the lotus seed-coat which keeps the seed longevity up to a millennium. Our analysis showed that approximately one-quarter tissue-specific isoforms were associated with AS events in per tissue, highlighting AS’ important role in regulating tissue development. In addition, a small part of tissue-specific isoform generated by non-tissue-specific genes as evidenced in this study, excluding the cotyledon, which reveals the strong connection between the expression of tissue-specific isoform and tissue-specific genes. Our result showed that both the function of tissue-specific genes and isoforms were enriched in specific biological processes which vary in different tissues similar to the results reported in other plants.<a reveal-id="dsz010-B29" data-open="dsz010-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsz010-B67" data-open="dsz010-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a><sup>,</sup><a reveal-id="dsz010-B68" data-open="dsz010-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Additionally, the filtered isoforms showed a tissue-biased expression patterns based on hierarchical and <em>k</em>-means clustering analysis other than the tissue-specific isoforms. Therefore, there is a high possibility that the development of tissue or organ was regulated by both the expression level of genes and AS.</p>
<p>Through differential expression analyses conducted in this study, we filtered some differential expressed isoforms, whose corresponding genes were not differentially expressed different from the previous studies. Concurrently, these isoforms were involved in the development of the corresponding tissue samples involved in the key biological process, providing further evidence of the isoforms regulatory functions. In addition, through homologous analyses with Arabidopsis, ‘ABCE’ model genes identified in MADS-box gene family in lotus, proved the compatibility of the ‘ABCE’ model for some floral organs in lotus. Moreover, the intumescent receptacle peculiar to lotus was hypothetically regulated by SHP genes and E-class genes. However, more sampling from different time points in floral organs is needed in investigating the expression of the ‘ABCE’ genes in lotus.</p>
<p>In summary, this study improves the understanding of the transcriptional diversity in lotus by updating the existing gene models and annotations. As transcriptional diversity and complexity is unveiled by both expressional and splicing pattern, a global view of gene regulation in lotus is obtained, thus benefiting gene functional studies and molecular breeding of this economically important aquatic plant.</p>                    <h2 scrollto-destination="140465419" id="140465419" class="section-title">5. Data availability</h2>
<p>The Illumina data are accessible through NCBI Sequence Read Archive under accession numbers PRJNA503979 and PRJNA492157. The Iso-Seq sequencing data are accessible through NCBI Sequence Read Archive under accession number PRJNA503979.</p>                    <h2 scrollto-destination="140465421" id="140465421" class="section-title">Funding</h2>
<p>This study was supported by National Natural Science Foundation of China [grant number 31700197) and Key Research Program of Frontier Sciences, CAS, [grant number QYZDB-SSW-SMC017].</p>                    <h2 scrollto-destination="140465423" id="140465423" class="section-title">Conflict of interest</h2>
<p>None declared.</p>                    <h2 scrollto-destination="140465425" id="140465425" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz010-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schaefke</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y.S.</div>
</div>, <div class="name string-name">
<div class="surname">Fang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">W.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">The evolution of posttranscriptional regulation</div>, <div class="source">Wires. RNA</div> , <div class="volume">9</div>, <div class="fpage">e1485</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20posttranscriptional%20regulation&amp;author=B.%20Schaefke&amp;author=W.%20Sun&amp;author=Y.S.%20Li&amp;author=L.%20Fang&amp;author=W.%20Chen&amp;publication_year=2018&amp;journal=Wires.%20RNA&amp;volume=9&amp;pages=e1485" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/wrna.2018.9.issue-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.2018.9.issue-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fwrna.2018.9.issue-5"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20evolution%20of%20posttranscriptional%20regulation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Barash</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Calarco</div> <div class="given-names">J.A.</div>
</div>, <div class="name string-name">
<div class="surname">Gao</div> <div class="given-names">W.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Deciphering the splicing code</div>, <div class="source">Nature</div> , <div class="volume">465</div>, <div class="fpage">53</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Deciphering%20the%20splicing%20code&amp;author=Y.%20Barash&amp;author=J.A.%20Calarco&amp;author=W.%20Gao&amp;publication_year=2010&amp;journal=Nature&amp;volume=465&amp;pages=53-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature09000" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09000" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature09000"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20445623" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Deciphering%20the%20splicing%20code&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Seo</div> <div class="given-names">P.J.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">M.J.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">C.M.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Alternative splicing of transcription factors in plant responses to low temperature stress: mechanisms and functions</div>, <div class="source">Planta</div> , <div class="volume">237</div>, <div class="fpage">1415</div>–<div class="lpage">24</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20transcription%20factors%20in%20plant%20responses%20to%20low%20temperature%20stress%3A%20mechanisms%20and%20functions&amp;author=P.J.%20Seo&amp;author=M.J.%20Park&amp;author=C.M.%20Park&amp;publication_year=2013&amp;journal=Planta&amp;volume=237&amp;pages=1415-24" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00425-013-1882-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-013-1882-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00425-013-1882-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624977" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Alternative%20splicing%20of%20transcription%20factors%20in%20plant%20responses%20to%20low%20temperature%20stress%3A%20mechanisms%20and%20functions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Keren</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Lev-Maor</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Ast</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Alternative splicing and evolution: diversification, exon definition and function</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">345</div>–<div class="lpage">55</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20evolution%3A%20diversification%2C%20exon%20definition%20and%20function&amp;author=H.%20Keren&amp;author=G.%20Lev-Maor&amp;author=G.%20Ast&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=345-55" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg2776" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2776" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg2776"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20376054" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Alternative%20splicing%20and%20evolution%3A%20diversification%2C%20exon%20definition%20and%20function&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Marquez</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Brown</div> <div class="given-names">J.W.</div>
</div>, <div class="name string-name">
<div class="surname">Simpson</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Transcriptome survey reveals increased complexity of the alternative splicing landscape in Arabidopsis</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">1184</div>–<div class="lpage">95</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20survey%20reveals%20increased%20complexity%20of%20the%20alternative%20splicing%20landscape%20in%20Arabidopsis&amp;author=Y.%20Marquez&amp;author=J.W.%20Brown&amp;author=C.%20Simpson&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=1184-95" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.134106.111" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.134106.111" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.134106.111"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22391557" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcriptome%20survey%20reveals%20increased%20complexity%20of%20the%20alternative%20splicing%20landscape%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">E.T.</div>
</div>, <div class="name string-name">
<div class="surname">Sandberg</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Luo</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Alternative isoform regulation in human tissue transcriptomes</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">470</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;author=E.T.%20Wang&amp;author=R.%20Sandberg&amp;author=S.%20Luo&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=470-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature07509" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07509" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature07509"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978772" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Reddy</div> <div class="given-names">A.S.</div>
</div>, <div class="name string-name">
<div class="surname">Marquez</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Kalyna</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Complexity of the alternative splicing landscape in plants</div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3657</div>–<div class="lpage">83</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Complexity%20of%20the%20alternative%20splicing%20landscape%20in%20plants&amp;author=A.S.%20Reddy&amp;author=Y.%20Marquez&amp;author=M.%20Kalyna&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3657-83" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.113.117523" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.117523" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.113.117523"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179125" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Complexity%20of%20the%20alternative%20splicing%20landscape%20in%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Khodor</div> <div class="given-names">Y.L.</div>
</div>, <div class="name string-name">
<div class="surname">Menet</div> <div class="given-names">J.S.</div>
</div>, <div class="name string-name">
<div class="surname">Tolan</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Cotranscriptional splicing efficiency differs dramatically between Drosophila and mouse</div>, <div class="source">RNA</div> , <div class="volume">18</div>, <div class="fpage">2174</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Cotranscriptional%20splicing%20efficiency%20differs%20dramatically%20between%20Drosophila%20and%20mouse&amp;author=Y.L.%20Khodor&amp;author=J.S.%20Menet&amp;author=M.%20Tolan&amp;publication_year=2012&amp;journal=RNA&amp;volume=18&amp;pages=2174-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1261/rna.034090.112" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.034090.112" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1261%2Frna.034090.112"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23097425" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cotranscriptional%20splicing%20efficiency%20differs%20dramatically%20between%20Drosophila%20and%20mouse&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pan</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Shai</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">L.J.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</div>, <div class="source">Nat. Genet</div> ., <div class="volume">40</div>, <div class="fpage">1413</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=L.J.%20Lee&amp;publication_year=2008&amp;journal=Nat.%20Genet&amp;volume=40&amp;pages=1413" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.259" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.259" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.259"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978789" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">C.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Chu</div> <div class="given-names">J.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">J.K.</div>
</div>
</div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Regulation of alternative splicing of Bcl-x by IL-6, GM-CSF and TPA</div>, <div class="source">Cell Res</div> ., <div class="volume">14</div>, <div class="fpage">473</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20alternative%20splicing%20of%20Bcl-x%20by%20IL-6%2C%20GM-CSF%20and%20TPA&amp;author=C.Y.%20Li&amp;author=J.Y.%20Chu&amp;author=J.K.%20Yu&amp;publication_year=2004&amp;journal=Cell%20Res&amp;volume=14&amp;pages=473" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/sj.cr.7290250" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.cr.7290250" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsj.cr.7290250"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15625014" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Regulation%20of%20alternative%20splicing%20of%20Bcl-x%20by%20IL-6%2C%20GM-CSF%20and%20TPA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Estelle</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Cabrito</div> <div class="given-names">T.R.</div>
</div>, <div class="name string-name">
<div class="surname">Pawel</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">A major facilitator superfamily transporter plays a dual role in polar auxin transport and drought stress tolerance in Arabidopsis</div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">901</div>–<div class="lpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20major%20facilitator%20superfamily%20transporter%20plays%20a%20dual%20role%20in%20polar%20auxin%20transport%20and%20drought%20stress%20tolerance%20in%20Arabidopsis&amp;author=R.%20Estelle&amp;author=T.R.%20Cabrito&amp;author=B.%20Pawel&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=901-26" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.113.110353" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.110353" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.113.110353"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23524662" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20major%20facilitator%20superfamily%20transporter%20plays%20a%20dual%20role%20in%20polar%20auxin%20transport%20and%20drought%20stress%20tolerance%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Verena</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Pengwei</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Chris</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Alternative splicing of the auxin biosynthesis gene YUCCA4 determines its subcellular compartmentation</div>, <div class="source">Plant J</div> ., <div class="volume">70</div>, <div class="fpage">292</div>–<div class="lpage">302</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20the%20auxin%20biosynthesis%20gene%20YUCCA4%20determines%20its%20subcellular%20compartmentation&amp;author=K.%20Verena&amp;author=W.%20Pengwei&amp;author=H.%20Chris&amp;publication_year=2012&amp;journal=Plant%20J&amp;volume=70&amp;pages=292-302" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.2012.70.issue-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2012.70.issue-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.2012.70.issue-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22233288" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Alternative%20splicing%20of%20the%20auxin%20biosynthesis%20gene%20YUCCA4%20determines%20its%20subcellular%20compartmentation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ruhl</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Stauffer</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Kahles</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Polypyrimidine tract binding protein homologs from Arabidopsis are key regulators of alternative splicing with implications in fundamental developmental processes</div>, <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">4360</div>–<div class="lpage">75</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Polypyrimidine%20tract%20binding%20protein%20homologs%20from%20Arabidopsis%20are%20key%20regulators%20of%20alternative%20splicing%20with%20implications%20in%20fundamental%20developmental%20processes&amp;author=C.%20Ruhl&amp;author=E.%20Stauffer&amp;author=A.%20Kahles&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=4360-75" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.112.103622" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.103622" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.112.103622"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192226" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Polypyrimidine%20tract%20binding%20protein%20homologs%20from%20Arabidopsis%20are%20key%20regulators%20of%20alternative%20splicing%20with%20implications%20in%20fundamental%20developmental%20processes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cui</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Ding</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Dynamic regulation of genome-wide pre-mRNA splicing and stress tolerance by the Sm-like protein LSm5 in Arabidopsis</div>, <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">R1</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dynamic%20regulation%20of%20genome-wide%20pre-mRNA%20splicing%20and%20stress%20tolerance%20by%20the%20Sm-like%20protein%20LSm5%20in%20Arabidopsis&amp;author=P.%20Cui&amp;author=S.%20Zhang&amp;author=F.%20Ding&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=R1" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2014-15-1-r1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-1-r1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2014-15-1-r1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24393432" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dynamic%20regulation%20of%20genome-wide%20pre-mRNA%20splicing%20and%20stress%20tolerance%20by%20the%20Sm-like%20protein%20LSm5%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Lin</div> <div class="given-names">W.-D.</div>
</div>, <div class="name string-name">
<div class="surname">Ray</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genome-wide detection of condition-sensitive alternative splicing in Arabidopsis roots</div>, <div class="source">Plant Physiol</div> ., <div class="volume">162</div>, <div class="fpage">1750</div>–<div class="lpage">63</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20detection%20of%20condition-sensitive%20alternative%20splicing%20in%20Arabidopsis%20roots&amp;author=W.%20Li&amp;author=W.-D.%20Lin&amp;author=P.%20Ray&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=162&amp;pages=1750-63" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.113.217778" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.217778" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.113.217778"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23735510" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20detection%20of%20condition-sensitive%20alternative%20splicing%20in%20Arabidopsis%20roots&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Staiger</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Brown</div> <div class="given-names">J.W.S.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Alternative splicing at the intersection of biological timing, development, and stress responses</div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3640</div>–<div class="lpage">56</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20at%20the%20intersection%20of%20biological%20timing%2C%20development%2C%20and%20stress%20responses&amp;author=D.%20Staiger&amp;author=J.W.S.%20Brown&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3640-56" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.113.113803" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.113803" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.113.113803"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179132" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Alternative%20splicing%20at%20the%20intersection%20of%20biological%20timing%2C%20development%2C%20and%20stress%20responses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Filichkin</div> <div class="given-names">S.A.</div>
</div>, <div class="name string-name">
<div class="surname">Mockler</div> <div class="given-names">T.C.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Unproductive alternative splicing and nonsense mRNAs: a widespread phenomenon among plant circadian clock genes</div>, <div class="source">Biol. Direct</div> , <div class="volume">7</div>, <div class="fpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Unproductive%20alternative%20splicing%20and%20nonsense%20mRNAs%3A%20a%20widespread%20phenomenon%20among%20plant%20circadian%20clock%20genes&amp;author=S.A.%20Filichkin&amp;author=T.C.%20Mockler&amp;publication_year=2012&amp;journal=Biol.%20Direct&amp;volume=7&amp;pages=20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1745-6150-7-20" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-7-20" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1745-6150-7-20"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747664" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Unproductive%20alternative%20splicing%20and%20nonsense%20mRNAs%3A%20a%20widespread%20phenomenon%20among%20plant%20circadian%20clock%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Duan</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Xiao</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Xia</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Two different transcripts of a LAMMER kinase gene play opposite roles in disease resistance</div>, <div class="source">Plant Physiol</div> ., <div class="volume">172</div>, <div class="fpage">1959</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Two%20different%20transcripts%20of%20a%20LAMMER%20kinase%20gene%20play%20opposite%20roles%20in%20disease%20resistance&amp;author=L.%20Duan&amp;author=W.%20Xiao&amp;author=F.%20Xia&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=172&amp;pages=1959-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.16.01245" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.16.01245" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.16.01245"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27621422" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Two%20different%20transcripts%20of%20a%20LAMMER%20kinase%20gene%20play%20opposite%20roles%20in%20disease%20resistance&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Romain</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Federico</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Fajardo</div> <div class="given-names">D.S.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Maize rough endosperm3 encodes an RNA splicing factor required for endosperm cell differentiation and has a nonautonomous effect on embryo development</div>, <div class="source">Plant Cell</div> , <div class="volume">23</div>, <div class="fpage">4280</div>–<div class="lpage">97</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Maize%20rough%20endosperm3%20encodes%20an%20RNA%20splicing%20factor%20required%20for%20endosperm%20cell%20differentiation%20and%20has%20a%20nonautonomous%20effect%20on%20embryo%20development&amp;author=F.%20Romain&amp;author=M.%20Federico&amp;author=D.S.%20Fajardo&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=4280-97" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.111.092163" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.092163" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.111.092163"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22138152" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Maize%20rough%20endosperm3%20encodes%20an%20RNA%20splicing%20factor%20required%20for%20endosperm%20cell%20differentiation%20and%20has%20a%20nonautonomous%20effect%20on%20embryo%20development&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Guo</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Hu</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Deep RNA sequencing at single base-pair resolution reveals high complexity of the rice transcriptome</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">646</div>–<div class="lpage">54</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Deep%20RNA%20sequencing%20at%20single%20base-pair%20resolution%20reveals%20high%20complexity%20of%20the%20rice%20transcriptome&amp;author=G.%20Zhang&amp;author=G.%20Guo&amp;author=X.%20Hu&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=646-54" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.100677.109" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.100677.109" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.100677.109"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20305017" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Deep%20RNA%20sequencing%20at%20single%20base-pair%20resolution%20reveals%20high%20complexity%20of%20the%20rice%20transcriptome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Thatcher</div> <div class="given-names">S.R.</div>
</div>, <div class="name string-name">
<div class="surname">Zhou</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Leonard</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide analysis of alternative splicing in <em>Zea mays</em>: landscape and genetic regulation</div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">3472</div>–<div class="lpage">87</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20alternative%20splicing%20in%20Zea%20mays%3A%20landscape%20and%20genetic%20regulation&amp;author=S.R.%20Thatcher&amp;author=W.%20Zhou&amp;author=A.%20Leonard&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=3472-87" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.114.130773" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.130773" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.114.130773"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25248552" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20analysis%20of%20alternative%20splicing%20in%20Zea%20mays%3A%20landscape%20and%20genetic%20regulation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shen</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Zhou</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Z.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Global dissection of alternative splicing in paleopolyploid soybean</div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">996</div>–<div class="lpage">1008</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Global%20dissection%20of%20alternative%20splicing%20in%20paleopolyploid%20soybean&amp;author=Y.%20Shen&amp;author=Z.%20Zhou&amp;author=Z.%20Wang&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=996-1008" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.114.122739" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.122739" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.114.122739"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24681622" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Global%20dissection%20of%20alternative%20splicing%20in%20paleopolyploid%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Steijger</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Abril</div> <div class="given-names">J.F.</div>
</div>, <div class="name string-name">
<div class="surname">Engström</div> <div class="given-names">P.G.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Assessment of transcript reconstruction methods for RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1177</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=J.F.%20Abril&amp;author=P.G.%20Engstr%C3%B6m&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1177-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.2714" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2714" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.2714"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sharon</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Tilgner</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Grubert</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">A single-molecule long-read survey of the human transcriptome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">1009</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20single-molecule%20long-read%20survey%20of%20the%20human%20transcriptome&amp;author=D.%20Sharon&amp;author=H.%20Tilgner&amp;author=F.%20Grubert&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=1009-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.2705" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2705" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.2705"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24108091" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20single-molecule%20long-read%20survey%20of%20the%20human%20transcriptome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Minoche</div> <div class="given-names">A.E.</div>
</div>, <div class="name string-name">
<div class="surname">Dohm</div> <div class="given-names">J.C.</div>
</div>, <div class="name string-name">
<div class="surname">Schneider</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Exploiting single-molecule transcript sequencing for eukaryotic gene prediction</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">184</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Exploiting%20single-molecule%20transcript%20sequencing%20for%20eukaryotic%20gene%20prediction&amp;author=A.E.%20Minoche&amp;author=J.C.%20Dohm&amp;author=J.%20Schneider&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=184" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-015-0729-7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0729-7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0729-7"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26328666" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Exploiting%20single-molecule%20transcript%20sequencing%20for%20eukaryotic%20gene%20prediction&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Peters</div> <div class="given-names">R.J.</div>
</div>, <div class="name string-name">
<div class="surname">Weirather</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Full-length transcriptome sequences and splice variants obtained by a combination of sequencing platforms applied to different root tissues of <em>Salvia miltiorrhiza</em> and tanshinone biosynthesis</div>, <div class="source">Plant J</div> ., <div class="volume">82</div>, <div class="fpage">951</div>–<div class="lpage">61</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20sequences%20and%20splice%20variants%20obtained%20by%20a%20combination%20of%20sequencing%20platforms%20applied%20to%20different%20root%20tissues%20of%20Salvia%20miltiorrhiza%20and%20tanshinone%20biosynthesis&amp;author=Z.%20Xu&amp;author=R.J.%20Peters&amp;author=J.%20Weirather&amp;publication_year=2015&amp;journal=Plant%20J&amp;volume=82&amp;pages=951-61" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.12865" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.12865" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.12865"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25912611" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Full-length%20transcriptome%20sequences%20and%20splice%20variants%20obtained%20by%20a%20combination%20of%20sequencing%20platforms%20applied%20to%20different%20root%20tissues%20of%20Salvia%20miltiorrhiza%20and%20tanshinone%20biosynthesis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Abdel-Ghany</div> <div class="given-names">S.E.</div>
</div>, <div class="name string-name">
<div class="surname">Hamilton</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Jacobi</div> <div class="given-names">J.L.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">A survey of the sorghum transcriptome using single-molecule long reads</div>, <div class="source">Nat Commun</div> ., <div class="volume">7</div>, <div class="fpage">11706</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20the%20sorghum%20transcriptome%20using%20single-molecule%20long%20reads&amp;author=S.E.%20Abdel-Ghany&amp;author=M.%20Hamilton&amp;author=J.L.%20Jacobi&amp;publication_year=2016&amp;journal=Nat%20Commun&amp;volume=7&amp;pages=11706" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ncomms11706" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms11706" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fncomms11706"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27339290" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20survey%20of%20the%20sorghum%20transcriptome%20using%20single-molecule%20long%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Dai</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Hu</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Global identification of alternative splicing via comparative analysis of SMRT- and Illumina-based RNA-seq in strawberry</div>, <div class="source">Plant J</div> ., <div class="volume">90</div>, <div class="fpage">164</div>–<div class="lpage">76</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Global%20identification%20of%20alternative%20splicing%20via%20comparative%20analysis%20of%20SMRT-%20and%20Illumina-based%20RNA-seq%20in%20strawberry&amp;author=Y.%20Li&amp;author=C.%20Dai&amp;author=C.%20Hu&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=90&amp;pages=164-76" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.2017.90.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2017.90.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.2017.90.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27997733" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Global%20identification%20of%20alternative%20splicing%20via%20comparative%20analysis%20of%20SMRT-%20and%20Illumina-based%20RNA-seq%20in%20strawberry&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Tseng</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Regulski</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing</div>, <div class="source">Nat. Commun</div> ., <div class="volume">7</div>, <div class="fpage">11708</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Unveiling%20the%20complexity%20of%20the%20maize%20transcriptome%20by%20single-molecule%20long-read%20sequencing&amp;author=B.%20Wang&amp;author=E.%20Tseng&amp;author=M.%20Regulski&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=11708" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ncomms11708" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms11708" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fncomms11708"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27339440" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Unveiling%20the%20complexity%20of%20the%20maize%20transcriptome%20by%20single-molecule%20long-read%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Liang</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">A global survey of alternative splicing in allopolyploid cotton: landscape, complexity and regulation</div>, <div class="source">New Phytol</div> ., <div class="volume">217</div>, <div class="fpage">163</div>–<div class="lpage">78</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20global%20survey%20of%20alternative%20splicing%20in%20allopolyploid%20cotton%3A%20landscape%2C%20complexity%20and%20regulation&amp;author=M.%20Wang&amp;author=P.%20Wang&amp;author=F.%20Liang&amp;publication_year=2018&amp;journal=New%20Phytol&amp;volume=217&amp;pages=163-78" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.2018.217.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2018.217.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.2018.217.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28892169" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20global%20survey%20of%20alternative%20splicing%20in%20allopolyploid%20cotton%3A%20landscape%2C%20complexity%20and%20regulation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Huang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Whole genome re-sequencing reveals evolutionary patterns of sacred lotus (<em>Nelumbo nucifera</em>)</div>, <div class="source">J. Integr. Plant Biol</div> ., <div class="volume">60</div>, <div class="fpage">2</div>–<div class="lpage">15</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20re-sequencing%20reveals%20evolutionary%20patterns%20of%20sacred%20lotus%20(Nelumbo%20nucifera)&amp;author=L.%20Huang&amp;author=M.%20Yang&amp;author=L.%20Li&amp;publication_year=2018&amp;journal=J.%20Integr.%20Plant%20Biol&amp;volume=60&amp;pages=2-15" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/jipb.v60.1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.v60.1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fjipb.v60.1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29052958" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Whole%20genome%20re-sequencing%20reveals%20evolutionary%20patterns%20of%20sacred%20lotus%20(Nelumbo%20nucifera)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shi</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">The evolution of plant microRNAs: insights from a basal eudicot sacred lotus</div>, <div class="source">Plant J</div> ., <div class="volume">89</div>, <div class="fpage">442</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20plant%20microRNAs%3A%20insights%20from%20a%20basal%20eudicot%20sacred%20lotus&amp;author=T.%20Shi&amp;author=K.%20Wang&amp;author=P.%20Yang&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=89&amp;pages=442" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.13394" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.13394" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.13394"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27743419" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20evolution%20of%20plant%20microRNAs%3A%20insights%20from%20a%20basal%20eudicot%20sacred%20lotus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ming</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Vanburen</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genome of the long-living sacred lotus (<em>Nelumbo nucifera</em> Gaertn.)</div>, <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20of%20the%20long-living%20sacred%20lotus%20(Nelumbo%20nucifera%20Gaertn.)&amp;author=R.%20Ming&amp;author=R.%20Vanburen&amp;author=Y.%20Liu&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2013-14-5-r41" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-5-r41" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2013-14-5-r41"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23663246" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20of%20the%20long-living%20sacred%20lotus%20(Nelumbo%20nucifera%20Gaertn.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shen-Miller</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Xie</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Villa</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Thermal-stable proteins of fruit of long-living sacred lotus <em>Nelumbo nucifera</em> Gaertn var. China Antique</div>, <div class="source">Trop. Plant Biol</div> ., <div class="volume">6</div>, <div class="fpage">69</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Thermal-stable%20proteins%20of%20fruit%20of%20long-living%20sacred%20lotus%20Nelumbo%20nucifera%20Gaertn%20var.%20China%20Antique&amp;author=J.%20Shen-Miller&amp;author=Y.%20Xie&amp;author=S.%20Villa&amp;publication_year=2013&amp;journal=Trop.%20Plant%20Biol&amp;volume=6&amp;pages=69-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s12042-013-9124-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12042-013-9124-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs12042-013-9124-2"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Thermal-stable%20proteins%20of%20fruit%20of%20long-living%20sacred%20lotus%20Nelumbo%20nucifera%20Gaertn%20var.%20China%20Antique&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Fan</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">The sacred lotus genome provides insights into the evolution of flowering plants</div>, <div class="source">Plant J</div> ., <div class="volume">76</div>, <div class="fpage">557</div>–<div class="lpage">67</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20sacred%20lotus%20genome%20provides%20insights%20into%20the%20evolution%20of%20flowering%20plants&amp;author=Y.%20Wang&amp;author=G.%20Fan&amp;author=Y.%20Liu&amp;publication_year=2013&amp;journal=Plant%20J&amp;volume=76&amp;pages=557-67" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.2013.76.issue-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.76.issue-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.2013.76.issue-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23952714" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20sacred%20lotus%20genome%20provides%20insights%20into%20the%20evolution%20of%20flowering%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Deng</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Damaris</div> <div class="given-names">R.N.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">LOTUS-DB: an integrative and interactive database for <em>Nelumbo nucifera</em> study</div>, <div class="source">Database (Oxford)</div> , <div class="volume">2015</div>, <div class="fpage">bav023</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=LOTUS-DB%3A%20an%20integrative%20and%20interactive%20database%20for%20Nelumbo%20nucifera%20study&amp;author=K.%20Wang&amp;author=J.%20Deng&amp;author=R.N.%20Damaris&amp;publication_year=2015&amp;journal=Database%20(Oxford)&amp;volume=2015&amp;pages=bav023" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25819075" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=LOTUS-DB%3A%20an%20integrative%20and%20interactive%20database%20for%20Nelumbo%20nucifera%20study&amp;aulast=Wang&amp;date=2015&amp;spage=bav023&amp;volume=2015" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:LOTUS-DB%3A%20an%20integrative%20and%20interactive%20database%20for%20Nelumbo%20nucifera%20study&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leena</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Eric</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">LoRDEC: accurate and efficient long read error correction</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3506</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=LoRDEC%3A%20accurate%20and%20efficient%20long%20read%20error%20correction&amp;author=S.%20Leena&amp;author=R.%20Eric&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3506-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu538" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu538" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu538"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25165095" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:LoRDEC%3A%20accurate%20and%20efficient%20long%20read%20error%20correction&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">T.D.</div>
</div>, <div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">C.K.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1859</div>–<div class="lpage">75</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbti310"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gao</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">PRAPI: post-transcriptional regulation analysis pipeline for Iso-Seq</div>, <div class="source">Bioinformatics (Oxford, England)</div> , <div class="volume">34</div>, <div class="fpage">1580</div>–<div class="lpage">2</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PRAPI%3A%20post-transcriptional%20regulation%20analysis%20pipeline%20for%20Iso-Seq&amp;author=Y.%20Gao&amp;author=H.%20Wang&amp;author=H.%20Zhang&amp;publication_year=2018&amp;journal=Bioinformatics%20(Oxford%2C%20England)&amp;volume=34&amp;pages=1580-2" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btx830" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtx830" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtx830"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29280994" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PRAPI%3A%20post-transcriptional%20regulation%20analysis%20pipeline%20for%20Iso-Seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Niknafs</div> <div class="given-names">Y.S.</div>
</div>, <div class="name string-name">
<div class="surname">Pandian</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Iyer</div> <div class="given-names">H.K.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">TACO produces robust multisample transcriptome assemblies from RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">14</div>, <div class="fpage">68</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=TACO%20produces%20robust%20multisample%20transcriptome%20assemblies%20from%20RNA-seq&amp;author=Y.S.%20Niknafs&amp;author=B.%20Pandian&amp;author=H.K.%20Iyer&amp;publication_year=2017&amp;journal=Nat.%20Methods&amp;volume=14&amp;pages=68-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.4078" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.4078" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.4078"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27869815" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:TACO%20produces%20robust%20multisample%20transcriptome%20assemblies%20from%20RNA-seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B.J.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">W.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</div>, <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;author=B.J.%20Haas&amp;author=S.L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2008-9-1-r7"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gordon</div> <div class="given-names">S.P.</div>
</div>, <div class="name string-name">
<div class="surname">Tseng</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Salamov</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Widespread polycistronic transcripts in fungi revealed by single-molecule mRNA sequencing</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0132628</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Widespread%20polycistronic%20transcripts%20in%20fungi%20revealed%20by%20single-molecule%20mRNA%20sequencing&amp;author=S.P.%20Gordon&amp;author=E.%20Tseng&amp;author=A.%20Salamov&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0132628" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0132628" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0132628" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0132628"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26177194" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Widespread%20polycistronic%20transcripts%20in%20fungi%20revealed%20by%20single-molecule%20mRNA%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Trincado</div> <div class="given-names">J.L.</div>
</div>, <div class="name string-name">
<div class="surname">Entizne</div> <div class="given-names">J.C.</div>
</div>, <div class="name string-name">
<div class="surname">Hysenaj</div> <div class="given-names">G.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">SUPPA2: fast, accurate, and uncertainty-aware differential splicing analysis across multiple conditions</div>, <div class="source">Genome Biol</div> ., <div class="volume">19</div>, <div class="fpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=SUPPA2%3A%20fast%2C%20accurate%2C%20and%20uncertainty-aware%20differential%20splicing%20analysis%20across%20multiple%20conditions&amp;author=J.L.%20Trincado&amp;author=J.C.%20Entizne&amp;author=G.%20Hysenaj&amp;publication_year=2018&amp;journal=Genome%20Biol&amp;volume=19&amp;pages=40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-018-1417-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-018-1417-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-018-1417-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29571299" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:SUPPA2%3A%20fast%2C%20accurate%2C%20and%20uncertainty-aware%20differential%20splicing%20analysis%20across%20multiple%20conditions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shen</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">J.W.</div>
</div>, <div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">Z.X.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">E5593</div>–<div class="lpage">5601</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=rMATS%3A%20robust%20and%20flexible%20detection%20of%20differential%20alternative%20splicing%20from%20replicate%20RNA-Seq%20data&amp;author=S.%20Shen&amp;author=J.W.%20Park&amp;author=Z.X.%20Lu&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=E5593-5601" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1419161111" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1419161111" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1419161111"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:rMATS%3A%20robust%20and%20flexible%20detection%20of%20differential%20alternative%20splicing%20from%20replicate%20RNA-Seq%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xie</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Mao</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Huang</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">316</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=KOBAS%202.0%3A%20a%20web%20server%20for%20annotation%20and%20identification%20of%20enriched%20pathways%20and%20diseases&amp;author=C.%20Xie&amp;author=X.%20Mao&amp;author=J.%20Huang&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=316-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr483" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr483" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr483"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:KOBAS%202.0%3A%20a%20web%20server%20for%20annotation%20and%20identification%20of%20enriched%20pathways%20and%20diseases&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ashburner</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Ball</div> <div class="given-names">C.A.</div>
</div>, <div class="name string-name">
<div class="surname">Blake</div> <div class="given-names">J.A.</div>
</div>
</div>, et al.  <div class="year">2000</div>, 
<div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet</div> ., <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F75556"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kanehisa</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Goto</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Kawashima</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2004</div>, 
<div class="article-title">The KEGG resource for deciphering the genome</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">D277</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20KEGG%20resource%20for%20deciphering%20the%20genome&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;author=S.%20Kawashima&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=D277" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkh063" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh063" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkh063"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681412" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20KEGG%20resource%20for%20deciphering%20the%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Young</div> <div class="given-names">M.D.</div>
</div>, <div class="name string-name">
<div class="surname">Wakefield</div> <div class="given-names">M.J.</div>
</div>, <div class="name string-name">
<div class="surname">Smyth</div> <div class="given-names">G.K.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Gene ontology analysis for RNA-seq: accounting for selection bias</div>, <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">R14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%20analysis%20for%20RNA-seq%3A%20accounting%20for%20selection%20bias&amp;author=M.D.%20Young&amp;author=M.J.%20Wakefield&amp;author=G.K.%20Smyth&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2010-11-2-r14" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-2-r14" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2010-11-2-r14"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20132535" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20ontology%20analysis%20for%20RNA-seq%3A%20accounting%20for%20selection%20bias&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Au</div> <div class="given-names">K.F.</div>
</div>, <div class="name string-name">
<div class="surname">Sebastiano</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Afshar</div> <div class="given-names">P.T.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Characterization of the human ESC transcriptome by hybrid sequencing</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">E4821</div>–<div class="lpage">30</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20human%20ESC%20transcriptome%20by%20hybrid%20sequencing&amp;author=K.F.%20Au&amp;author=V.%20Sebastiano&amp;author=P.T.%20Afshar&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=E4821-30" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1320101110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1320101110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1320101110"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterization%20of%20the%20human%20ESC%20transcriptome%20by%20hybrid%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gao</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">PRAPI: post-transcriptional regulation analysis pipeline for Iso-Seq</div>. <div class="source">Bioinformatics</div> , <div class="volume">34</div>, <div class="fpage">1580</div>–<div class="lpage">2</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PRAPI%3A%20post-transcriptional%20regulation%20analysis%20pipeline%20for%20Iso-Seq&amp;author=Y.%20Gao&amp;author=H.%20Wang&amp;author=H.%20Zhang&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=34&amp;pages=1580-2" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btx830" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtx830" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtx830"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PRAPI%3A%20post-transcriptional%20regulation%20analysis%20pipeline%20for%20Iso-Seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Shi</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Huang</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Systematic transcriptomic analysis provides insights into lotus (<em>Nelumbo nucifera</em>) seed development</div>, <div class="source">Plant Growth Regul</div> ., <div class="volume">86</div>, <div class="fpage">339</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Systematic%20transcriptomic%20analysis%20provides%20insights%20into%20lotus%20(Nelumbo%20nucifera)%20seed%20development&amp;author=J.%20Li&amp;author=T.%20Shi&amp;author=L.%20Huang&amp;publication_year=2018&amp;journal=Plant%20Growth%20Regul&amp;volume=86&amp;pages=339-50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s10725-018-0433-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10725-018-0433-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs10725-018-0433-1"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Systematic%20transcriptomic%20analysis%20provides%20insights%20into%20lotus%20(Nelumbo%20nucifera)%20seed%20development&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">W.M.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">S.N.</div>
</div>
</div> <div class="year">2001</div>, 
<div class="article-title">The ABC model of floral development</div>, <div class="source">J. Yunnan Univ</div> ., <div class="volume">23</div>, <div class="fpage">102</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20ABC%20model%20of%20floral%20development&amp;aulast=Zhang&amp;date=2001&amp;spage=102&amp;epage=5&amp;volume=23" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20ABC%20model%20of%20floral%20development&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Shi</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Lin</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">The ABCs of flower development: mutational analysis of AP1/FUL-like genes in rice provides evidence for a homeotic (A)-function in grasses</div>, <div class="source">Plant J</div> ., <div class="volume">89</div>, <div class="fpage">310</div>–<div class="lpage">24</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20ABCs%20of%20flower%20development%3A%20mutational%20analysis%20of%20AP1%2FFUL-like%20genes%20in%20rice%20provides%20evidence%20for%20a%20homeotic%20(A)-function%20in%20grasses&amp;author=F.%20Wu&amp;author=X.%20Shi&amp;author=X.%20Lin&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=89&amp;pages=310-24" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.13386" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.13386" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.13386"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27689766" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20ABCs%20of%20flower%20development%3A%20mutational%20analysis%20of%20AP1%2FFUL-like%20genes%20in%20rice%20provides%20evidence%20for%20a%20homeotic%20(A)-function%20in%20grasses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Huo</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Expression of B-class MADS-box genes in response to variations in photoperiod is associated with chasmogamous and cleistogamous flower development in Viola philippica</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">151</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20B-class%20MADS-box%20genes%20in%20response%20to%20variations%20in%20photoperiod%20is%20associated%20with%20chasmogamous%20and%20cleistogamous%20flower%20development%20in%20Viola%20philippica&amp;author=Q.%20Li&amp;author=Q.%20Huo&amp;author=J.%20Wang&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=151" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12870-016-0832-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0832-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12870-016-0832-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27388887" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Expression%20of%20B-class%20MADS-box%20genes%20in%20response%20to%20variations%20in%20photoperiod%20is%20associated%20with%20chasmogamous%20and%20cleistogamous%20flower%20development%20in%20Viola%20philippica&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Galimba</div> <div class="given-names">K.D.</div>
</div>, <div class="name string-name">
<div class="surname">Martinez-Gomez</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Di Stilio</div> <div class="given-names">V.S.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">Gene duplication and transference of function in the paleoAP3 lineage of floral organ identity genes</div>, <div class="source">Front Plant Sci</div> ., <div class="volume">9</div>, <div class="fpage">334</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20transference%20of%20function%20in%20the%20paleoAP3%20lineage%20of%20floral%20organ%20identity%20genes&amp;author=K.D.%20Galimba&amp;author=J.%20Martinez-Gomez&amp;author=V.S.%20Di%20Stilio&amp;publication_year=2018&amp;journal=Front%20Plant%20Sci&amp;volume=9&amp;pages=334" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2018.00334" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2018.00334" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2018.00334"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29628932" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20duplication%20and%20transference%20of%20function%20in%20the%20paleoAP3%20lineage%20of%20floral%20organ%20identity%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B56" class="ref-content ">
<div class="label">56</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">S.</div>
</div>
</div>, et al. <div class="year">2018</div>, 
<div class="article-title">Identification and characterization of the MADS-box genes and their contribution to flower organ in carnation (<em>Dianthus caryophyllus</em> L.)</div>. <div class="source">Genes</div> , <div class="volume">9,</div> 193.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Identification%20and%20characterization%20of%20the%20MADS-box%20genes%20and%20their%20contribution%20to%20flower%20organ%20in%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;aulast=Zhang&amp;date=2018&amp;volume=9," href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20and%20characterization%20of%20the%20MADS-box%20genes%20and%20their%20contribution%20to%20flower%20organ%20in%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B57" class="ref-content ">
<div class="label">57</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Law</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Childs</div> <div class="given-names">K.L.</div>
</div>, <div class="name string-name">
<div class="surname">Campbell</div> <div class="given-names">M.S.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Automated update, revision and quality control of the Zea mays genome annotations using MAKER-P improves the B73 RefGen_v3 gene models and identifies new genes</div>, <div class="source">Plant Physiol</div> ., <div class="volume">167</div>, <div class="fpage">25</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Automated%20update%2C%20revision%20and%20quality%20control%20of%20the%20Zea%20mays%20genome%20annotations%20using%20MAKER-P%20improves%20the%20B73%20RefGen_v3%20gene%20models%20and%20identifies%20new%20genes&amp;author=M.%20Law&amp;author=K.L.%20Childs&amp;author=M.S.%20Campbell&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=167&amp;pages=25" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.114.245027" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.245027" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.114.245027"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25384563" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Automated%20update%2C%20revision%20and%20quality%20control%20of%20the%20Zea%20mays%20genome%20annotations%20using%20MAKER-P%20improves%20the%20B73%20RefGen_v3%20gene%20models%20and%20identifies%20new%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B58" class="ref-content ">
<div class="label">58</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ichihashi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Fukushima</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Shibata</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">High impact gene discovery: simple strand-specific mRNA library construction and differential regulatory analysis based on gene co-expression network</div>. <div class="source">Methods Mol. Biol</div> ., <div class="volume">1830</div>, <div class="fpage">163</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=High%20impact%20gene%20discovery%3A%20simple%20strand-specific%20mRNA%20library%20construction%20and%20differential%20regulatory%20analysis%20based%20on%20gene%20co-expression%20network&amp;author=Y.%20Ichihashi&amp;author=A.%20Fukushima&amp;author=A.%20Shibata&amp;publication_year=2018&amp;journal=Methods%20Mol.%20Biol&amp;volume=1830&amp;pages=163-89" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/978-1-4939-8657-6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-4939-8657-6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2F978-1-4939-8657-6"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30043371" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:High%20impact%20gene%20discovery%3A%20simple%20strand-specific%20mRNA%20library%20construction%20and%20differential%20regulatory%20analysis%20based%20on%20gene%20co-expression%20network&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B59" class="ref-content ">
<div class="label">59</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Li</div> <div class="given-names">X.</div>
</div></div> <div class="year">2018</div>, 
<div class="article-title">Next-generation sequencing sheds new light on small RNAs in plant reproductive development</div>, <div class="source">Curr. Issues Mol. Biol</div> ., <div class="volume">27</div>, <div class="fpage">143</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20sheds%20new%20light%20on%20small%20RNAs%20in%20plant%20reproductive%20development&amp;author=X.%20Li&amp;publication_year=2018&amp;journal=Curr.%20Issues%20Mol.%20Biol&amp;volume=27&amp;pages=143" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.21775/cimb.027.143" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.21775%2Fcimb.027.143" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.21775%2Fcimb.027.143"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28885180" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Next-generation%20sequencing%20sheds%20new%20light%20on%20small%20RNAs%20in%20plant%20reproductive%20development&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B60" class="ref-content ">
<div class="label">60</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Denoeud</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Aury</div> <div class="given-names">J.M.</div>
</div>, <div class="name string-name">
<div class="surname">Da</div> <div class="given-names">S.C.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Annotating genomes with massive-scale RNA sequencing</div>, <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Annotating%20genomes%20with%20massive-scale%20RNA%20sequencing&amp;author=F.%20Denoeud&amp;author=J.M.%20Aury&amp;author=S.C.%20Da&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=1-12" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2008-9-12-r175" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-12-r175" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2008-9-12-r175"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Annotating%20genomes%20with%20massive-scale%20RNA%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B61" class="ref-content ">
<div class="label">61</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wei</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Feng</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Genome re-annotation of the wild strawberry <em>Fragaria vesca</em> using extensive Illumina- and SMRT-based RNA-seq datasets</div>, <div class="source">DNA Res</div> ., <div class="volume">25</div>, <div class="fpage">61</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20re-annotation%20of%20the%20wild%20strawberry%20Fragaria%20vesca%20using%20extensive%20Illumina-%20and%20SMRT-based%20RNA-seq%20datasets&amp;author=Y.%20Li&amp;author=W.%20Wei&amp;author=J.%20Feng&amp;publication_year=2018&amp;journal=DNA%20Res&amp;volume=25&amp;pages=61-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsx038" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsx038" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsx038"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20re-annotation%20of%20the%20wild%20strawberry%20Fragaria%20vesca%20using%20extensive%20Illumina-%20and%20SMRT-based%20RNA-seq%20datasets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B62" class="ref-content ">
<div class="label">62</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lin</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Damaris</div> <div class="given-names">R.N.</div>
</div>, <div class="name string-name">
<div class="surname">Shi</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Transcriptomic analysis identifies the key genes involved in stamen petaloid in lotus (<em>Nelumbo nucifera</em>)</div>, <div class="source">BMC Genomics</div> , <div class="volume">19</div>, <div class="fpage">554</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20identifies%20the%20key%20genes%20involved%20in%20stamen%20petaloid%20in%20lotus%20(Nelumbo%20nucifera)&amp;author=Z.%20Lin&amp;author=R.N.%20Damaris&amp;author=T.%20Shi&amp;publication_year=2018&amp;journal=BMC%20Genomics&amp;volume=19&amp;pages=554" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-018-4950-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-018-4950-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-018-4950-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30053802" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcriptomic%20analysis%20identifies%20the%20key%20genes%20involved%20in%20stamen%20petaloid%20in%20lotus%20(Nelumbo%20nucifera)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B63" class="ref-content ">
<div class="label">63</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ding</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Identification and analysis of glutathione S-transferase gene family in sweet potato reveal divergent GST -mediated networks in aboveground and underground tissues in response to abiotic stresses</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">17</div>, <div class="fpage">225</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20glutathione%20S-transferase%20gene%20family%20in%20sweet%20potato%20reveal%20divergent%20GST%20-mediated%20networks%20in%20aboveground%20and%20underground%20tissues%20in%20response%20to%20abiotic%20stresses&amp;author=N.%20Ding&amp;author=A.%20Wang&amp;author=X.%20Zhang&amp;publication_year=2017&amp;journal=BMC%20Plant%20Biol&amp;volume=17&amp;pages=225" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12870-017-1179-z" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-017-1179-z" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12870-017-1179-z"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29179697" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20and%20analysis%20of%20glutathione%20S-transferase%20gene%20family%20in%20sweet%20potato%20reveal%20divergent%20GST%20-mediated%20networks%20in%20aboveground%20and%20underground%20tissues%20in%20response%20to%20abiotic%20stresses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B64" class="ref-content ">
<div class="label">64</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ren</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Meng</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Molecular mechanisms of acclimatization to phosphorus, starvation and recovery underlying full-length transcriptome profiling in barley (<em>Hordeum vulgare</em> L.)</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">9</div>, <div class="fpage">500</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20acclimatization%20to%20phosphorus%2C%20starvation%20and%20recovery%20underlying%20full-length%20transcriptome%20profiling%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=P.%20Ren&amp;author=Y.%20Meng&amp;author=B.%20Li&amp;publication_year=2018&amp;journal=Front.%20Plant%20Sci&amp;volume=9&amp;pages=500" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2018.00500" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2018.00500" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2018.00500"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29720989" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20mechanisms%20of%20acclimatization%20to%20phosphorus%2C%20starvation%20and%20recovery%20underlying%20full-length%20transcriptome%20profiling%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B65" class="ref-content ">
<div class="label">65</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">RNA-seq uncovers SNPs and alternative splicing events in Asian lotus (<em>Nelumbo nucifera</em>)</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0125702</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RNA-seq%20uncovers%20SNPs%20and%20alternative%20splicing%20events%20in%20Asian%20lotus%20(Nelumbo%20nucifera)&amp;author=M.%20Yang&amp;author=L.%20Xu&amp;author=Y.%20Liu&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0125702" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0125702" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0125702" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0125702"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25928215" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RNA-seq%20uncovers%20SNPs%20and%20alternative%20splicing%20events%20in%20Asian%20lotus%20(Nelumbo%20nucifera)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B66" class="ref-content ">
<div class="label">66</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Estrada</div> <div class="given-names">A.D.</div>
</div>, <div class="name string-name">
<div class="surname">Freese</div> <div class="given-names">N.H.</div>
</div>, <div class="name string-name">
<div class="surname">Blakley</div> <div class="given-names">I.C.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Analysis of pollen-specific alternative splicing in <em>Arabidopsis thaliana</em> via semi-quantitative PCR</div>, <div class="source">Peer J</div> ., <div class="volume">3</div>, <div class="fpage">e919</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20pollen-specific%20alternative%20splicing%20in%20Arabidopsis%20thaliana%20via%20semi-quantitative%20PCR&amp;author=A.D.%20Estrada&amp;author=N.H.%20Freese&amp;author=I.C.%20Blakley&amp;publication_year=2015&amp;journal=Peer%20J&amp;volume=3&amp;pages=e919" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.7717/peerj.919" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7717%2Fpeerj.919" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.7717%2Fpeerj.919"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25945312" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analysis%20of%20pollen-specific%20alternative%20splicing%20in%20Arabidopsis%20thaliana%20via%20semi-quantitative%20PCR&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B67" class="ref-content ">
<div class="label">67</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jureckova</div> <div class="given-names">J.F.</div>
</div>, <div class="name string-name">
<div class="surname">Sykorova</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Hafidh</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Tissue-specific expression of telomerase reverse transcriptase gene variants in <em>Nicotiana tabacum</em>
</div>, <div class="source">Planta</div> , <div class="volume">245</div>, <div class="fpage">549</div>–<div class="lpage">61</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tissue-specific%20expression%20of%20telomerase%20reverse%20transcriptase%20gene%20variants%20in%20Nicotiana%20tabacum&amp;author=J.F.%20Jureckova&amp;author=E.%20Sykorova&amp;author=S.%20Hafidh&amp;publication_year=2017&amp;journal=Planta&amp;volume=245&amp;pages=549-61" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00425-016-2624-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-016-2624-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00425-016-2624-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27900472" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tissue-specific%20expression%20of%20telomerase%20reverse%20transcriptase%20gene%20variants%20in%20Nicotiana%20tabacum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz010-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz010-B68" class="ref-content ">
<div class="label">68</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Guo</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Hossain</div> <div class="given-names">M.S.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Bombyx E75 isoforms display stage- and tissue-specific responses to 20-hydroxyecdysone</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">12114</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bombyx%20E75%20isoforms%20display%20stage-%20and%20tissue-specific%20responses%20to%2020-hydroxyecdysone&amp;author=K.%20Li&amp;author=E.%20Guo&amp;author=M.S.%20Hossain&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=12114" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep12114" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep12114" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep12114"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26166384" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bombyx%20E75%20isoforms%20display%20stage-%20and%20tissue-specific%20responses%20to%2020-hydroxyecdysone&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5512305&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz010/2/dsz010_supplementary_data.zip?Expires=1576697891&amp;Signature=u5RBS1o0Q~7-B3nmYcOceR3pE2KJqNB5rUptMVw-on-3uG4iqk-6L3-9fKo2~lv0PU8Zc92GcvaNGgsSgD0rS8NkCgcjlVq6Jt7Pp7~-FqOkw2TnAVXnvuBU8~obWLoNy8uRT6REIf5V89QQC8O6CVqU29zjAoRkLNf3ilxxIYwX~Iw4VBbVXApgsYJCPhAFZoCEL3Hogf2xMIjk9crseGkUMZDbKP8XtAtxsaV~u8g0xfxlZDmi5Cy60z3eMfmaRNscmrxANsuUN2-0pII-eDjwz8tTneYSpz52zXLzD69pjNL81G3T1ie~QP5rtW9kInwWRq5ZB8eeYPDMNJZucQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz010_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">577</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz010"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5512305"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493011000002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f4%2f301%2f5512305">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493011000002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F4%2F301%2F5512305" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5512305">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="complexity of alternative splicing and landscape of tissue-specific expression in lotus (Nelumbo nucifera) unveiled by Illumina- and single-molecule real-time-based RNA-sequencing | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz010");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f301%2f5512305">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
