<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Landscape of gene transposition–duplication within the Brassicaceae family | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/1/21/5151297","@type":"ScholarlyArticle","name":"Landscape of gene transposition–duplication within the Brassicaceae family","datePublished":"2018-10-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2019-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy035","keywords":["gene transposition","gene duplication","synteny","co-linearity","OrthNet"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/1/21/5151297","author":[{"name":"Oh, Dong-Ha","affiliation":"Department of Biological Sciences, Louisiana State University, Baton Rouge, LA, USA","@type":"Person","sameAs":"http://orcid.org/0000-0003-1526-9814"},{"name":"Dassanayake, Maheshi","affiliation":"Department of Biological Sciences, Louisiana State University, Baton Rouge, LA, USA","@type":"Person","sameAs":"http://orcid.org/0000-0003-3123-3731"}],"description":"Abstract.  We developed the CLfinder-OrthNet pipeline that detects co-linearity among multiple closely related genomes, finds orthologous gene groups, and encod","pageStart":"21","pageEnd":"36","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636325253&Signature=uhNPRfhhrjVDx5vmQj~FfOzD3tSx1BgfBUFqUjFR00xTaSavSuJ2pkQL7bkw3yc2GAV~493RfsP4jodLe30LOJbJvq4OSkXHfvH5d0SAsvTyIhDHqx3OaU~qdSj4ZMhv2mt-UKrdFqoZsvAov6E-Wwz47KR6mHAutPrBPjOgjrHKFIJlxJTXu1Mb51t3811BaxzY4dRPINMzid8f0iB2UZvIEJFEWS5Lf3YugjPol4kBuXB0GKu0qFzkruxDTtNeniK6JdcvHrhzTJgGfrhbRwh7MIgLbydj0PcUC6oEzOn2aHsnSajgj2jSuTQVzlfWOjFdxdXMkOurOaaevKBqfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Landscape of gene transposition–duplication within the Brassicaceae family","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636325253&Signature=uhNPRfhhrjVDx5vmQj~FfOzD3tSx1BgfBUFqUjFR00xTaSavSuJ2pkQL7bkw3yc2GAV~493RfsP4jodLe30LOJbJvq4OSkXHfvH5d0SAsvTyIhDHqx3OaU~qdSj4ZMhv2mt-UKrdFqoZsvAov6E-Wwz47KR6mHAutPrBPjOgjrHKFIJlxJTXu1Mb51t3811BaxzY4dRPINMzid8f0iB2UZvIEJFEWS5Lf3YugjPol4kBuXB0GKu0qFzkruxDTtNeniK6JdcvHrhzTJgGfrhbRwh7MIgLbydj0PcUC6oEzOn2aHsnSajgj2jSuTQVzlfWOjFdxdXMkOurOaaevKBqfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Landscape of gene transposition–duplication within the Brassicaceae family">
<meta property="og:description" content="Abstract.  We developed the CLfinder-OrthNet pipeline that detects co-linearity among multiple closely related genomes, finds orthologous gene groups, and encod">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/1/21/5151297">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636325253&amp;Signature=uhNPRfhhrjVDx5vmQj~FfOzD3tSx1BgfBUFqUjFR00xTaSavSuJ2pkQL7bkw3yc2GAV~493RfsP4jodLe30LOJbJvq4OSkXHfvH5d0SAsvTyIhDHqx3OaU~qdSj4ZMhv2mt-UKrdFqoZsvAov6E-Wwz47KR6mHAutPrBPjOgjrHKFIJlxJTXu1Mb51t3811BaxzY4dRPINMzid8f0iB2UZvIEJFEWS5Lf3YugjPol4kBuXB0GKu0qFzkruxDTtNeniK6JdcvHrhzTJgGfrhbRwh7MIgLbydj0PcUC6oEzOn2aHsnSajgj2jSuTQVzlfWOjFdxdXMkOurOaaevKBqfQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2018-10-30">
<meta name="citation_author" content="Oh, Dong-Ha">
<meta name="citation_author_institution" content="Department of Biological Sciences, Louisiana State University, Baton Rouge, LA, USA">
<meta name="citation_author" content="Dassanayake, Maheshi">
<meta name="citation_author_institution" content="Department of Biological Sciences, Louisiana State University, Baton Rouge, LA, USA">
<meta name="citation_title" content="Landscape of gene transposition–duplication within the Brassicaceae family">
<meta name="citation_firstpage" content="21">
<meta name="citation_lastpage" content="36">
<meta name="citation_doi" content="10.1093/dnares/dsy035">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/02/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Patching gaps in plant genomes results in gene movement and erosion of colinearity;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1229-37; ">
<meta name="citation_reference" content="citation_title=Genome and gene duplications and gene expression divergence: a view from plants;  citation_journal_title=Ann. N.Y. Acad. Sci;  citation_year=2012;  citation_volume=1256;  citation_pages=1-14; ">
<meta name="citation_reference" content="citation_title=Neofunctionalization of young duplicate genes in Drosophila;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2013;  citation_volume=110;  citation_pages=17409-14; ">
<meta name="citation_reference" content="citation_title=Evolution of cis-regulatory elements and regulatory networks in duplicated genes of Arabidopsis thaliana;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=169;  citation_pages=2982-91; ">
<meta name="citation_reference" content="citation_title=Genome structures and transcriptomes signify niche adaptation for the multiple-ion-tolerant extremophyte Schrenkiella parvula;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=164;  citation_pages=2123-38; ">
<meta name="citation_reference" content="citation_title=Rapid establishment of genetic incompatibility through natural epigenetic variation;  citation_journal_title=Curr. Biol;  citation_year=2012;  citation_volume=22;  citation_pages=326-31; ">
<meta name="citation_reference" content="citation_title=Why the activity of a gene depends on its neighbors;  citation_journal_title=Trends Genet;  citation_year=2015;  citation_volume=31;  citation_pages=483-90; ">
<meta name="citation_reference" content="citation_title=Genome-wide prediction and validation of intergenic enhancers in Arabidopsis using open chromatin signatures;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=2415-26; ">
<meta name="citation_reference" content="citation_title=Enhancers and chromatin structures: regulatory hubs in gene expression and diseases;  citation_journal_title=Biosci. Rep;  citation_year=2017;  citation_volume=37; ">
<meta name="citation_reference" content="citation_title=Genome structures and halophyte-specific gene expression of the extremophile Thellungiella parvula in comparison with Thellungiella salsuginea (Thellungiella halophila) and Arabidopsis;  citation_journal_title=Plant Physiol;  citation_year=2010;  citation_volume=154;  citation_pages=1040-52; ">
<meta name="citation_reference" content="citation_title=Protein subcellular relocalization of duplicated genes in Arabidopsis;  citation_journal_title=Genome Biol. Evol;  citation_year=2014;  citation_volume=6;  citation_pages=2501-15; ">
<meta name="citation_reference" content="citation_title=Evolution of gene duplication in plants;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=171;  citation_pages=2294-316; ">
<meta name="citation_reference" content="citation_title=Evolutionary fates and dynamic functionalization of young duplicate genes in Arabidopsis genomes;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=172;  citation_pages=427-40; ">
<meta name="citation_reference" content="citation_title=Turning a hobby into a job: how duplicated genes find new functions;  citation_journal_title=Nat. Rev. Genet;  citation_year=2008;  citation_volume=9;  citation_pages=938-50; ">
<meta name="citation_reference" content="citation_title=Gene duplication as a mechanism of genomic adaptation to a changing environment;  citation_journal_title=Proc. Biol. Sci;  citation_year=2012;  citation_volume=279;  citation_pages=5048-57; ">
<meta name="citation_reference" content="citation_title=Copy number variation of multiple genes at Rhg1 mediates nematode resistance in soybean;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1206-9; ">
<meta name="citation_reference" content="citation_title=The Cardamine hirsuta genome offers insight into the evolution of morphological diversity;  citation_journal_title=Nat. Plants;  citation_year=2016;  citation_volume=2;  citation_pages=16167">
<meta name="citation_reference" content="citation_title=A highly specific genome-wide association study integrated with transcriptome data reveals the contribution of copy number variations to specialized metabolites in Arabidopsis thaliana accessions;  citation_journal_title=Mol. Biol. Evol;  citation_year=2017;  citation_volume=34;  citation_pages=3111-22; ">
<meta name="citation_reference" content="citation_title=Evolution of CONSTANS regulation and function after gene duplication produced a photoperiodic flowering switch in the Brassicaceae;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=2284-301; ">
<meta name="citation_reference" content="citation_title=Transposable element-assisted evolution and adaptation to host plant within the Leptosphaeria maculans–Leptosphaeria biglobosa species complex of fungal pathogens;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1-27; ">
<meta name="citation_reference" content="citation_title=Gene translocation links insects and crustaceans;  citation_journal_title=Nature;  citation_year=1998;  citation_volume=392;  citation_pages=667-8; ">
<meta name="citation_reference" content="citation_title=Bias in plant gene content following different sorts of duplication: tandem, whole-genome, segmental, or by transposition;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2009;  citation_volume=60;  citation_pages=433-53; ">
<meta name="citation_reference" content="citation_title=Independent and parallel evolution of new genes by gene duplication in two origins of C4 photosynthesis provides new insight into the mechanism of phloem loading in C4 species;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=1796-806; ">
<meta name="citation_reference" content="citation_title=Leaf shape evolution through duplication, regulatory diversification, and loss of a homeobox gene;  citation_journal_title=Science;  citation_year=2014;  citation_volume=343;  citation_pages=780-3; ">
<meta name="citation_reference" content="citation_title=Phylogenomic synteny network analysis of MADS-box transcription factor genes reveals lineage-specific transpositions, ancient tandem duplications, and deep positional conservation;  citation_journal_title=Plant Cell;  citation_year=2017;  citation_volume=29;  citation_pages=1278-92; ">
<meta name="citation_reference" content="citation_title=Comparative phylogenomic synteny network analysis of mammalian and angiosperm genomes;  citation_journal_title=bioRxiv;  citation_year=2018;  citation_pages=246736">
<meta name="citation_reference" content="citation_title=Hierarchically aligning 10 legume genomes establishes a family-level genomics platform;  citation_journal_title=Plant Physiol;  citation_year=2017;  citation_volume=174;  citation_pages=284-300; ">
<meta name="citation_reference" content="citation_title=Beyond the thale: comparative genomics and genetics of Arabidopsis relatives;  citation_journal_title=Nat. Rev. Genet;  citation_year=2015;  citation_volume=16;  citation_pages=285-98; ">
<meta name="citation_reference" content="citation_title=The genome sequence of African rice (Oryza glaberrima) and evidence for independent domestication;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=982-8; ">
<meta name="citation_reference" content="citation_title=Sequencing and de novo assembly of a near complete indica rice genome;  citation_journal_title=Nat. Commun;  citation_year=2017;  citation_volume=8;  citation_pages=15324">
<meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and colinearity;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49">
<meta name="citation_reference" content="citation_title=SynFind: compiling syntenic regions across any set of genomes on demand;  citation_journal_title=Genome Biol. Evol;  citation_year=2015;  citation_volume=7;  citation_pages=3286-98; ">
<meta name="citation_reference" content="citation_title=i-ADHoRe 3.0-fast and sensitive detection of genomic homology in extremely large data sets;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e11">
<meta name="citation_reference" content="citation_title=OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D363-8; ">
<meta name="citation_reference" content="citation_title=How to usefully compare homologous plant genes and chromosomes as DNA sequences;  citation_journal_title=Plant J;  citation_year=2008;  citation_volume=53;  citation_pages=661-73; ">
<meta name="citation_reference" content="citation_title=A guide to the PLAZA 3.0 plant comparative genomic database;  citation_journal_title=Methods Mol. Biol;  citation_year=2017;  citation_volume=1533;  citation_pages=183-200; ">
<meta name="citation_reference" content="citation_title=GCView: the genomic context viewer for protein homology searches;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=353-6; ">
<meta name="citation_reference" content="citation_title=PlantTribes: a gene and gene family resource for comparative genomics in plants;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=970-6; ">
<meta name="citation_reference" content="citation_title=An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=891-8; ">
<meta name="citation_reference" content="citation_title=Araport11: a complete reannotation of the Arabidopsis thaliana reference genome;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=89;  citation_pages=789-804; ">
<meta name="citation_reference" content="citation_title=The genome of the extremophile crucifer Thellungiella parvula;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=913-8; ">
<meta name="citation_reference" content="citation_title=Insights into salt tolerance from the genome of Thellungiella salsuginea;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2012;  citation_volume=109;  citation_pages=12219-24; ">
<meta name="citation_reference" content="citation_title=The reference genome of the halophytic plant Eutrema salsugineum;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=4;  citation_pages=46">
<meta name="citation_reference" content="citation_title=A comparative study of salt tolerance parameters in 11 wild relatives of Arabidopsis thaliana;  citation_journal_title=J. Exp. Bot;  citation_year=2010;  citation_volume=61;  citation_pages=3787-98; ">
<meta name="citation_reference" content="citation_title=Salt cress. A halophyte and cryophyte Arabidopsis relative model system and its applicability to molecular genetic analyses of growth and development of extremophiles;  citation_journal_title=Plant Physiol;  citation_year=2004;  citation_volume=135;  citation_pages=1718-37; ">
<meta name="citation_reference" content="citation_title=Learning from evolution: Thellungiella generates new knowledge on essential and critical components of abiotic stress tolerance in plants;  citation_journal_title=Mol. Plant;  citation_year=2009;  citation_volume=2;  citation_pages=3-12; ">
<meta name="citation_reference" content="citation_title=Genomes of extremophile crucifers: new platforms for comparative genomics and beyond;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=166">
<meta name="citation_reference" content="citation_title=Life at the extreme: lessons from the genome;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=241">
<meta name="citation_reference" content="citation_title=MCScanX-transposed: detecting transposed gene duplications based on multiple colinearity scans;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=1458-60; ">
<meta name="citation_reference" content="citation_title=BLAST+: architecture and applications;  citation_journal_title=BMC Bioinform;  citation_year=2009;  citation_volume=10;  citation_pages=421">
<meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes. Genome;  citation_journal_title=Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; ">
<meta name="citation_reference" content="citation_title=Agalma: an automated phylogenomics workflow;  citation_journal_title=BMC Bioinform;  citation_year=2013;  citation_volume=14;  citation_pages=330">
<meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; ">
<meta name="citation_reference" content="citation_title=Using MCL to extract clusters from networks;  citation_journal_title=Methods Mol. Biol;  citation_year=2012;  citation_volume=804;  citation_pages=281-95; ">
<meta name="citation_reference" content="citation_title=Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown;  citation_journal_title=Nat. Protoc;  citation_year=2016;  citation_volume=11;  citation_pages=1650-67; ">
<meta name="citation_reference" content="citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=157">
<meta name="citation_reference" content="citation_title=Detecting gene clusters under evolutionary constraint in a large number of genomes;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=571-7; ">
<meta name="citation_reference" content="citation_title=CYNTENATOR: progressive gene order alignment of 17 vertebrate genomes;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e8861">
<meta name="citation_reference" content="citation_title=Network approaches for plant phylogenomic synteny analysis;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2017;  citation_volume=36;  citation_pages=129-34; ">
<meta name="citation_reference" content="citation_title=Mechanisms of change in gene copy number;  citation_journal_title=Nat. Rev. Genet;  citation_year=2009;  citation_volume=10;  citation_pages=551-64; ">
<meta name="citation_reference" content="citation_title=Nonallelic homologous recombination between retrotransposable elements is a driver of de novo unbalanced translocations;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=411-8; ">
<meta name="citation_reference" content="citation_title=Not born equal: increased rate asymmetry in relocated and retrotransposed rodent gene duplicates;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=679-86; ">
<meta name="citation_reference" content="citation_title=The fate of retrotransposed processed genes in Arabidopsis thaliana;  citation_journal_title=Gene;  citation_year=2017;  citation_volume=609;  citation_pages=1-8; ">
<meta name="citation_reference" content="citation_title=Gene duplication and exon shuffling by helitron-like transposons generate intraspecies diversity in maize;  citation_journal_title=Nat. Genet;  citation_year=2005;  citation_volume=37;  citation_pages=997-1002; ">
<meta name="citation_reference" content="citation_title=Transposed genes in Arabidopsis are often associated with flanking repeats;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=26">
<meta name="citation_reference" content="citation_title=TsHKT1;2, a HKT1 Homolog from the extremophile Arabidopsis relative Thellungiella salsuginea, shows K+ specificity in the presence of NaCl;  citation_journal_title=Plant Physiol;  citation_year=2012;  citation_volume=158;  citation_pages=1463-74; ">
<meta name="citation_reference" content="citation_title=Loss of halophytism by interference with SOS1 expression;  citation_journal_title=Plant Physiol;  citation_year=2009;  citation_volume=151;  citation_pages=210-22; ">
<meta name="citation_reference" content="citation_title=Isolation and functional characterization of the Arabidopsis salt tolerance 32 (AtSAT32) gene associated with salt tolerance and ABA signaling;  citation_journal_title=Physiol. Plant;  citation_year=2009;  citation_volume=135;  citation_pages=426-35; ">
<meta name="citation_reference" content="citation_title=RNA splicing regulates the temporal order of TNF-induced gene expression;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2013;  citation_volume=110;  citation_pages=11934-9; ">
<meta name="citation_reference" content="citation_title=Higher frequency of intron loss from the promoter proximally paused genes of Drosophila melanogaster: evidence consistent with delays in intron splicing as a selective force;  citation_journal_title=Fly (Austin);  citation_year=2014;  citation_volume=8;  citation_pages=120-5; ">
<meta name="citation_reference" content="citation_title=The evolution of spliceosomal introns: patterns, puzzles and progress;  citation_journal_title=Nat. Rev. Genet;  citation_year=2006;  citation_volume=7;  citation_pages=211-21; ">
<meta name="citation_reference" content="citation_title=The ZIP family of metal transporters;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2000;  citation_volume=1465;  citation_pages=190-8; ">
<meta name="citation_reference" content="citation_title=Zinc deficiency as a practical problem in plant and human nutrition in Turkey: a NATO-science for stability project;  citation_journal_title=Field Crops Res;  citation_year=1999;  citation_volume=60;  citation_pages=175-88; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/1/21/28033473/dsy035.pdf">
<meta name="description" content="Abstract.  We developed the CLfinder-OrthNet pipeline that detects co-linearity among multiple closely related genomes, finds orthologous gene groups, and encod">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/1/21/5151297">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/1/21/5151297">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/1/21/5151297");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy035';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f21%2f5151297" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f21%2f5151297" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=114871"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1576706058&amp;Signature=SZjA6b8DhviXcX5ic-90hmHMm~eYBklLHuEUUl9n9UXR8nh1XZAdfdepDO3zT28g2ZraNoGSharFnk1tz27So~nJurJJBU857fZFNNV-YA~goPvQx5~9hZ3O9JHiTN0-DLhRHxk-1mT3WO~esh3X78Af9yfDl0BFNX1dANnqBCv2UP-cTPNBQgmxGnjCxHRhY0WJ-S~zAO2EP6FqvWYUfecmY9~pVFFKd2bM1RYLmuj31etSlORJcjEwHvArXZJDHqI5Jr4iCjFlvt78tZeMlQwtvJg9t2vfJIHE4xvhCTpOoS1A0zXTQDjYAHTtjyPD6hI6cZQs7pJDQnlKyCKk4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/1">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="131979896">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979896">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979898">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979898">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979905">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979905">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979925">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979925">3. Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979956">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979956">4. Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979971">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979971">Funding</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979973">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979973">Supplementary data</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979975">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979975">Conflict of interest</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="131979977">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979977">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/1/13/5142401">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/1/37/5161192">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Landscape of gene transposition–duplication within the Brassicaceae family
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dong-Ha Oh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biological Sciences, Louisiana State University, Baton Rouge, LA, USA</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 225 578 4918. Fax. +1 225 578 2597. Email: <a href="mailto:ohdongha@gmail.com" target="_blank">ohdongha@gmail.com</a> (D.-H.O.); Tel. +1 225 578 0960. Fax. +1 225 578 2597. Email: <a href="mailto:maheshid@lsu.edu" target="_blank">maheshid@lsu.edu</a> (M.D.)
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-1526-9814">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0003-1526-9814
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dong-Ha+Oh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oh%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Oh%20Dong-Ha%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Dong-Ha Oh</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maheshi Dassanayake</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biological Sciences, Louisiana State University, Baton Rouge, LA, USA</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +1 225 578 4918. Fax. +1 225 578 2597. Email: <a href="mailto:ohdongha@gmail.com" target="_blank">ohdongha@gmail.com</a> (D.-H.O.); Tel. +1 225 578 0960. Fax. +1 225 578 2597. Email: <a href="mailto:maheshid@lsu.edu" target="_blank">maheshid@lsu.edu</a> (M.D.)
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-3123-3731">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0003-3123-3731
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maheshi+Dassanayake">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dassanayake%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Dassanayake%20Maheshi%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Maheshi Dassanayake</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 21–36, <a href="https://doi.org/10.1093/dnares/dsy035">https://doi.org/10.1093/dnares/dsy035</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">30 October 2018</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">06 March 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">03 October 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">30 October 2018</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5151297" href="/dnaresearch/article-pdf/26/1/21/28033473/dsy035.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Dong-Ha Oh, Maheshi Dassanayake,  Landscape of gene transposition–duplication within the Brassicaceae family, <em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 21–36, <a href="https://doi.org/10.1093/dnares/dsy035">https://doi.org/10.1093/dnares/dsy035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5151297&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Landscape%20of%20gene%20transposition%E2%80%93duplication%20within%20the%20Brassicaceae%20family&amp;publicationDate=2018-10-30&amp;volumeNum=26&amp;issueNum=1&amp;author=Oh%2C%20Dong-Ha%3B%20Dassanayake%2C%20Maheshi&amp;startPage=21&amp;endPage=36&amp;contentId=10.1093%2Fdnares%2Fdsy035&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f21%2f5151297" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="131979896" id="131979896" class="abstract-title">Abstract</h2>
<section class="abstract"><p>We developed the CLfinder-OrthNet pipeline that detects co-linearity among multiple closely related genomes, finds orthologous gene groups, and encodes the evolutionary history of each orthologue group into a representative network (OrthNet). Using a search based on network topology, we identified 1,394 OrthNets that included gene transposition–duplication (<em>tr–d</em>) events, out of 17,432 identified in six Brassicaceae genomes. Occurrences of <em>tr–d</em> shared by subsets of Brassicaceae genomes mirrored the divergence times between the genomes and their repeat contents. The majority of <em>tr–d</em> events resulted in truncated open reading frames (ORFs) in the duplicated loci. However, the duplicates with complete ORFs were significantly more frequent than expected from random events. These were derived from older <em>tr–d</em> events and had a higher chance of being expressed. We also found an enrichment of <em>tr–d</em> events with complete loss of intergenic sequence conservation between the original and duplicated loci. Finally, we identified <em>tr–d</em> events uniquely found in two extremophytes among the six Brassicaceae genomes, including <em>tr–d</em> of <em>SALT TOLERANCE 32</em> and <em>ZINC TRANSPORTER 3</em> that relate to their adaptive evolution. CLfinder-OrthNet provides a flexible toolkit to compare gene order, visualize evolutionary paths among orthologues as networks, and identify gene loci that share an evolutionary history.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"gene transposition"'>gene transposition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"gene duplication"'>gene duplication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="synteny">synteny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="co-linearity">co-linearity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="OrthNet">OrthNet</a>
</div>                    <h2 scrollto-destination="131979898" id="131979898" class="section-title" data-legacy-id="SEC1">1. Introduction</h2>
<p>Co-linearity among closely related genomes erodes over time due to the accumulation of mutations including gene duplication, deletion, and transposition.<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Gene duplication affects gene dosage, which may lead to divergence of expression and functions among duplicates.<a reveal-id="dsy035-B2" data-open="dsy035-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy035-B3" data-open="dsy035-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Gene transposition events modify expression strength and tissue-specificity through changes in regulatory sequences,<a reveal-id="dsy035-B4" data-open="dsy035-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> local epigenetic environment,<a reveal-id="dsy035-B6" data-open="dsy035-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and proximity to enhancers and chromatin structural contexts.<a reveal-id="dsy035-B7 dsy035-B8 dsy035-B9" data-open="dsy035-B7 dsy035-B8 dsy035-B9" class="link link-ref link-reveal xref-bibr"><sup>7–9</sup></a> Such events have been associated with variation in copy numbers of genes and transcripts,<a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsy035-B10" data-open="dsy035-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> as well as localization<a reveal-id="dsy035-B11" data-open="dsy035-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and functions<a reveal-id="dsy035-B12 dsy035-B13 dsy035-B14" data-open="dsy035-B12 dsy035-B13 dsy035-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> of encoded proteins among orthologous genes. A large number of studies have reported examples of gene level duplications and transpositions as key underlying sources for adaptations to specific environments or speciation.<a reveal-id="dsy035-B15 dsy035-B16 dsy035-B17 dsy035-B18 dsy035-B19 dsy035-B20 dsy035-B21" data-open="dsy035-B15 dsy035-B16 dsy035-B17 dsy035-B18 dsy035-B19 dsy035-B20 dsy035-B21" class="link link-ref link-reveal xref-bibr"><sup>15–21</sup></a></p>
<p>Comparative analysis of co-linearity and its erosion identifies modes of gene duplications<a reveal-id="dsy035-B22" data-open="dsy035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy035-B23" data-open="dsy035-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and traces the origin of genes or gene families and their evolutionary history.<a reveal-id="dsy035-B24 dsy035-B25 dsy035-B26" data-open="dsy035-B24 dsy035-B25 dsy035-B26" class="link link-ref link-reveal xref-bibr"><sup>24–26</sup></a> Such comparative analyses are facilitated by <em>de novo</em> assembled genomes released at unprecedented rates today,<a reveal-id="dsy035-B27 dsy035-B28 dsy035-B29 dsy035-B30" data-open="dsy035-B27 dsy035-B28 dsy035-B29 dsy035-B30" class="link link-ref link-reveal xref-bibr"><sup>27–30</sup></a> enabling detection of gene gain and loss as well as duplication and transposition among closely related taxa. These resources also call for novel methods and tools for systematic comparative analysis of genomes.</p>
<p>A number of tools are available for identification of gene blocks or large genomic regions co-linear among multiple genomes.<a reveal-id="dsy035-B31 dsy035-B32 dsy035-B33" data-open="dsy035-B31 dsy035-B32 dsy035-B33" class="link link-ref link-reveal xref-bibr"><sup>31–33</sup></a> Another set of tools can identify orthologues in related genomes for a gene of interest and visualize synteny and evolutionary events, such as gene duplication and transposition, associated with them.<a reveal-id="dsy035-B34 dsy035-B35 dsy035-B36 dsy035-B37 dsy035-B38" data-open="dsy035-B34 dsy035-B35 dsy035-B36 dsy035-B37 dsy035-B38" class="link link-ref link-reveal xref-bibr"><sup>34–38</sup></a> However, currently we do not have a method that can retrieve all orthologue loci within multiple genomes that have likely undergone the same set of evolutionary events without a prior assignment of a gene of interest.</p>
<p>To address this, we introduce the CLfinder-OrthNet pipeline, which identifies co-linearity (CL) in the gene order among multiple genomes, identify ‘orthologue groups’ based on co-linearity, and encodes genes in each orthologue group as a network of orthologues (OrthNet). Each orthologue group includes orthologues and paralogues likely derived from a single ancestral locus in multiple target genomes. All evolutionary events in each orthologue group, such as gene duplication, deletion, transposition, and any combination of them, in addition to gene synteny conservation, are captured as the topology of an OrthNet. Our pipeline enables detection of all orthologue groups from multiple genome that seemingly underwent the same evolutionary events, by searching OrthNets essentially based on their topologies.</p>
<p>As a proof-of-concept, we applied the CLfinder-OrthNet pipeline to six Brassicaceae (crucifer) genomes that share the same whole genome duplication history.<a reveal-id="dsy035-B39" data-open="dsy035-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The six Brassicaceae genomes included the model species <em>Arabidopsis thaliana</em><a reveal-id="dsy035-B40" data-open="dsy035-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and extremophytes <em>Schrenkiella parvula</em><a reveal-id="dsy035-B41" data-open="dsy035-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and <em>Eutrema salsugineum</em>,<a reveal-id="dsy035-B42" data-open="dsy035-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsy035-B43" data-open="dsy035-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> the two most salt-tolerant Brassicaceae species so far tested.<a reveal-id="dsy035-B44" data-open="dsy035-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><em>S. parvula</em> and <em>E. salsugineum</em> are biogeographicaly seperated and represent taxa adapted to multi-ion salt strsses in soils near a hypersaline lake in central Anatolia<a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and combined salt and freezing stresses in salt pans of high latitude regions in the northern hemisphere,<a reveal-id="dsy035-B45" data-open="dsy035-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsy035-B46" data-open="dsy035-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> respectively. These two extremophytes provide optimal models for comparative analyses to study plant adaptations to environmental challenges.<a reveal-id="dsy035-B47" data-open="dsy035-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsy035-B48" data-open="dsy035-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a></p>
<p>The CLfinder-OrthNet pipeline detects any combination of gene synteny conservation, duplications, deletions, and transpositions. For this study, we focused on the relatively under-studied transposition–duplication (<em>tr–d</em>) events within the six Brassicaceae genomes. A <em>tr–d</em> is an event where a single non-transposon gene is duplicated and relocated to a position where all evidence of common ancestry (i.e. synteny) is lost, i.e. ‘Duplication mode IV’ as defined by Freeling.<a reveal-id="dsy035-B22" data-open="dsy035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> A <em>tr–d</em> event results in variations in both copy numbers and co-linearity and, unlike other types of gene duplications, requires a systematic comparison of multiple genomes for detection.<a reveal-id="dsy035-B22" data-open="dsy035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy035-B49" data-open="dsy035-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Our pipeline identified <em>tr–d</em> events unique to a genome or shared by any subset of the six Brassicaceae genomes, as well as the original donor and duplicate loci in each <em>tr–d</em> event including loci with truncated coding regions. Using this pipeline, we aim to identify the landscape of lineage-specific and shared <em>tr–d</em> events among the target genomes; test whether there is a signature of selective retention among lineage-specific <em>tr–d</em> events; and characterize <em>tr–d</em> events unique to the extremophyte genomes, which may have contributed to their adaptive evolution.</p>                    <h2 scrollto-destination="131979905" id="131979905" class="section-title" data-legacy-id="SEC2">2. Materials and methods</h2>
                    <h3 scrollto-destination="131979906" id="131979906" class="section-title" data-legacy-id="SEC2.1">2.1. Genome and gene models</h3>
<p>We obtained genome annotations of <em>Arabidopsis lyrata</em> (Aly, version 1.0), <em>A. thaliana</em> (Ath, v. ‘TAIR10’), <em>Capsella rubella</em> (Cru, v. 1.0), and <em>E. salsugineum</em> (Esa, v. 1.0) from Phytozome v. 11 (<a class="link link-uri" href="http://genome.jgi.doe.gov/" target="_blank">http://genome.jgi.doe.gov/</a> (10 October 2018, date last accessed)), while genomes of <em>Sisymbrium irio</em> (Sir, v. 0.2; CoGE genome id 19579) and <em>S. parvula</em> (v. 2.0) were downloaded from CoGE (<a class="link link-uri" href="https://genomevolution.org/coge/" target="_blank">https://genomevolution.org/coge/</a> (10 October 2018, date last accessed)) and thellungiella.org (<a class="link link-uri" href="http://thellungiella.org/data/" target="_blank">http://thellungiella.org/data/</a> (10 October 2018, date last accessed)), respectively. For <em>S. irio</em> annotation, we used a combination of RepeatMasker, a Basic Local Alignment Search Tool for Nucleotides (BLASTN)<a reveal-id="dsy035-B50" data-open="dsy035-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> search vs Repbase v. 20170127 (<a class="link link-uri" href="http://www.girinst.org/repbase/" target="_blank">http://www.girinst.org/repbase/</a> (10 October 2018, date last accessed)), and OrthoMCL<a reveal-id="dsy035-B51" data-open="dsy035-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> to further filter out gene models most likely unannotated transposable elements (TEs). This step was added to remove the large number of repetitive lineage-specific genes (34.9% of 49,956 gene models), many of them showing sequence similarities with known TEs, in the <em>S. irio</em> genome annotation. To generate a species tree of Brassicaceae genomes including the six target species (<a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), we used Agalma<a reveal-id="dsy035-B52" data-open="dsy035-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> which built a maximum likelihood tree based on 14,614 alignments of homologous protein-coding gene clusters. Each cluster contained sequences from more than four crucifer genomes. We listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277251&amp;Signature=nN68KE~o0~ndVWZECN5xHdTTk3P7uj0LYTA5pRMhcjZkaU3s~-6izm-mIjcpc73SzmkXojk8TwKRtGGNaCxoa-FbMwcSPR5u8KyCu8-oyON3~luSbdFDWj2ni14niMQJV5BklxSefsi2vl22Ai32HgQxebcgvjXE1KTHbo~kC192SuFcWVM-yKN1Bv8nBTTG3q~HVgMN60QTcEb1n2AOfnjJAR71g5PvQ430iG9XNzE1VFKkc235a2OdV7v5VNiMqmqLD~JFl8KtfUZmPNpVkfel6XJ6upGs7jXQUAcZkjfX0MeaWbvq~JGQItp6CiYQ0FsGAXirGiJD7xUOWaTmcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> additional genome resources used for the Brassicaceae species tree. </p>                        <a id="131979908" scrollto-destination="131979908"></a>
<div id="dsy035-F1" data-id="dsy035-F1" class="fig fig-section">
<div class="label fig-label">Figure 1.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f1.png?Expires=1576277252&amp;Signature=oOoxwQ3KXtLBgnSv75Ay3eW2Fw7MMqAvyJkyuekrLvJ6AMCbDJGhFSIr5WfhJjm2XCejS97NxMmvIAfxxfOU0XikqkRXrYksUep9YNcCbBU0nusrFtWpEObhjBOvUikyulwkMdIGiArYY7txkUZPgL0ExaqwjIskW0ooJZo84AlkU-uIJ8Eucot1tChiRgyv6~mIoedObF1PAShTp82ExrrNOTQVczS~nV1dgt10IwyUezQCY8OFl8bMXL8XyWUv~WUG2nebKRcc0F2KXMWM4vR5r247f31szyoPJQAYTAjOmI9xL-FHugmr-CqbEzTiSbdyci-dnqa4jC19p6BfjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A comparative genomics framework including the two extremophyte/halophyte crucifers, S. parvula and E. salsugineum. Boxes and stars indicate the six Brassicaceae species selected for this work and halophytes, respectively. The tree was based on an alignment of 14,614 homologous gene clusters, as described in Materials and Methods." path-from-xml="dsy035f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979908/dsy035f1.tif" path-from-xml="dsy035f1.tif" target="_blank">Open in new tab</a><a section="131979908" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f1.png?Expires=1576277251&amp;Signature=00sY99VZh-xKgJlqQUCzJtynPQOy6-dphutjMRWXsnTvSg4JZyXSYKaPvDvOoRG7tXM7H1g8JOMtNfa3EcmZd6PvfFajvMoPolpo3RCkQdtzmMDotqgfFMxguGzQUTboiw5YK4Q4Y5OR1zSO7P2i-uJV0tZsroHmn9ecfMO~7nfrZwPLeq6AJvsD6EjjO7c~gH1p0gE4Ec3Dh8kmsE8Nq5qc5kQGKAj1zD1IYlog1WcM3nx057Aq2k7xea4C4ssiiNhF9yLZvi7xS9mAywBPBqEGF7WFd2cBkI-99IMGgysDxhR-wJzSFdDfmWj0qvv1VHhmScPbNLnAS8XRQfhIuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979908&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A comparative genomics framework including the two extremophyte/halophyte crucifers, <em>S. parvula</em> and <em>E. salsugineum</em>. Boxes and stars indicate the six Brassicaceae species selected for this work and halophytes, respectively. The tree was based on an alignment of 14,614 homologous gene clusters, as described in Materials and Methods.</p></div>
</div>
<div content-id="dsy035-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f1.png?Expires=1576277252&amp;Signature=oOoxwQ3KXtLBgnSv75Ay3eW2Fw7MMqAvyJkyuekrLvJ6AMCbDJGhFSIr5WfhJjm2XCejS97NxMmvIAfxxfOU0XikqkRXrYksUep9YNcCbBU0nusrFtWpEObhjBOvUikyulwkMdIGiArYY7txkUZPgL0ExaqwjIskW0ooJZo84AlkU-uIJ8Eucot1tChiRgyv6~mIoedObF1PAShTp82ExrrNOTQVczS~nV1dgt10IwyUezQCY8OFl8bMXL8XyWUv~WUG2nebKRcc0F2KXMWM4vR5r247f31szyoPJQAYTAjOmI9xL-FHugmr-CqbEzTiSbdyci-dnqa4jC19p6BfjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A comparative genomics framework including the two extremophyte/halophyte crucifers, S. parvula and E. salsugineum. Boxes and stars indicate the six Brassicaceae species selected for this work and halophytes, respectively. The tree was based on an alignment of 14,614 homologous gene clusters, as described in Materials and Methods." path-from-xml="dsy035f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979908/dsy035f1.tif" path-from-xml="dsy035f1.tif" target="_blank">Open in new tab</a><a section="131979908" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f1.png?Expires=1576277251&amp;Signature=00sY99VZh-xKgJlqQUCzJtynPQOy6-dphutjMRWXsnTvSg4JZyXSYKaPvDvOoRG7tXM7H1g8JOMtNfa3EcmZd6PvfFajvMoPolpo3RCkQdtzmMDotqgfFMxguGzQUTboiw5YK4Q4Y5OR1zSO7P2i-uJV0tZsroHmn9ecfMO~7nfrZwPLeq6AJvsD6EjjO7c~gH1p0gE4Ec3Dh8kmsE8Nq5qc5kQGKAj1zD1IYlog1WcM3nx057Aq2k7xea4C4ssiiNhF9yLZvi7xS9mAywBPBqEGF7WFd2cBkI-99IMGgysDxhR-wJzSFdDfmWj0qvv1VHhmScPbNLnAS8XRQfhIuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979908&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A comparative genomics framework including the two extremophyte/halophyte crucifers, <em>S. parvula</em> and <em>E. salsugineum</em>. Boxes and stars indicate the six Brassicaceae species selected for this work and halophytes, respectively. The tree was based on an alignment of 14,614 homologous gene clusters, as described in Materials and Methods.</p></div>
</div>
<p>For the analysis of co-linearity erosion (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1</a></span>), we compared all protein-coding gene loci in five crucifer genomes to those in <em>A. thaliana</em> using BLASTN. Four degenerate site (4d) substitution rate distributions were analysed for all reciprocal BLASTN pairs of protein-coding genes between the five crucifer genomes and <em>A. thaliana</em>, using codeml<a reveal-id="dsy035-B53" data-open="dsy035-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and custom scripts. TE contents were determined using RepeatMasker v. 4.0.7 and RepeatModeler v. 1.0.8 (<a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a> (10 October 2018, date last accessed)), for all six target species.</p>                    <h3 scrollto-destination="131979910" id="131979910" class="section-title" data-legacy-id="SEC2.2">2.2. Detecting co-linearity in gene orders among Brassicaceae genomes</h3>
<p>We developed the CLfinder process to scan the order of homologous gene loci between a pair of genomes and detect co-linearity among them. The CLfinder module performs this process automatically for all pairs among multiple closely related genomes. The CLfinder uses three inputs (<a reveal-id="dsy035-F2" data-open="dsy035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), for which we used only primary transcript models for protein-coding gene loci. <strong>Input 1</strong>: A custom script (‘parse_gtf.py’) parsed genome annotation in Genomic Transfer Format (GTF). <strong>Input 2</strong>: Intra-species paralogous loci detected by clustering primary protein-coding gene model sequences within each species using OrthoMCL,<a reveal-id="dsy035-B51" data-open="dsy035-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> as previously described in ref. <a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr">5</a>. <strong>Input 3:</strong> Results of pairwise reciprocal BLASTN (<em>e</em> &lt; 10<sup>−5</sup>) between all primary protein-coding gene model ORFs to obtain the ten most similar BLASTN hits (‘best-hits’) in the target species, for each gene model in the query species. </p>                        <a id="131979912" scrollto-destination="131979912"></a>
<div id="dsy035-F2" data-id="dsy035-F2" class="fig fig-section">
<div class="label fig-label">Figure 2.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f2.png?Expires=1576277252&amp;Signature=RdnvhY72iIcPNf0-JF~mgUG1T5sZX0~WIFRKuaYo19BmxUHK58XS1tCfy9IZiP981-duIlX6oS5BTHM1AIsANFEBBc694YFMjmuqqypmhmCy4db18MmSujT3hv5KoBwpJCUx-WVB2cZi~J4keyU3g5wNZffXL7-fgUYZ1lGbq9HPZBDYcHC6LxRqyxTgt7ISYcQ7wNtluM1BkWlgWtrpvuYHo~hK4AiabfYQMgKT2RRx6iOB7J~7H8prBz-FwR6IPUSjJ2B8~z3BKpCKL~ef4ea6kX9EwLsqR-SsJv0zfqG~kh~~AlHtoYf7QqxGv340hVOSmSiAv~I3ZzgxTj3xlw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The CLfinder-OrthNet pipeline consists of two modules, CLfinder (Co-Linearity finder) and ONfinder (OrthNet finder). In the cartoon next to the box for the CLfinder module, blue squares indicate gene loci with disrupted co-linearity due to mutations and indels in neighbouring loci, while orange squares are those transposed and lost the synteny. See Supplementary Fig. S1 for details. Next to the box for the ONfinder module is an exemplary OrthNet. (A) For the detailed method to determine co-linearity (CL) relationship between the query loci and their most homologous counterpart (‘best-hits’) in the target genome, see Materials and Methods (2.2) and Figs S2 and S3. CLfinder output for the six crucifer species highlighted in Fig. 1 are summarized in Table 1, with the full results given as Supplementary Dataset S1. (B) See Figures 3–6, for examples of OrthNets with different evolutionary histories represented as different network topologies, e.g. transposition (tr) and transposition–duplication (tr–d) unique to each species or a group of species." path-from-xml="dsy035f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979912/dsy035f2.tif" path-from-xml="dsy035f2.tif" target="_blank">Open in new tab</a><a section="131979912" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f2.png?Expires=1576277252&amp;Signature=B22IvWLliS5hZ9yTzDrBLhKjH12wqCuugblgs3nSNuaVcCAskzG3rJbRehnRi06o2GJN-eWTzqgFDrraVh0lgzyC7aKZIwrmPbwMcVB~~LNKVpM6nspTnMhnU~xaY4-nTi9ihzQiY8qjCBmN-Wwak6kXpLH6jDJNM3LEThafpb0T1mHivpNSk3~0wV7fhGJwIPVRV0I9YfgE32GTAvNxjj-sc~O~zM7eHp7VYNnPN7x0ehHakziGogYqdpcwmSbKtse~IlZhtOh2iWad~XM4222QYRlmb3OObqAKfW1re0z7oBZbeDovClDL0CvipT0AJVqohtEU8P3YuUP2dCQRvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979912&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The CLfinder-OrthNet pipeline consists of two modules, CLfinder (Co-Linearity finder) and ONfinder (OrthNet finder). In the cartoon next to the box for the CLfinder module, blue squares indicate gene loci with disrupted co-linearity due to mutations and indels in neighbouring loci, while orange squares are those transposed and lost the synteny. See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> for details. Next to the box for the ONfinder module is an exemplary OrthNet. (A) For the detailed method to determine co-linearity (CL) relationship between the query loci and their most homologous counterpart (‘best-hits’) in the target genome, see Materials and Methods (2.2) and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Figs S2 and S3</a></span>. CLfinder output for the six crucifer species highlighted in <a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> are summarized in <a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, with the full results given as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S1. (B) See <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Figures 3–6</a>, for examples of OrthNets with different evolutionary histories represented as different network topologies, e.g. transposition (<em>tr</em>) and transposition–duplication (<em>tr–d</em>) unique to each species or a group of species.</p></div>
</div>
<div content-id="dsy035-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f2.png?Expires=1576277252&amp;Signature=RdnvhY72iIcPNf0-JF~mgUG1T5sZX0~WIFRKuaYo19BmxUHK58XS1tCfy9IZiP981-duIlX6oS5BTHM1AIsANFEBBc694YFMjmuqqypmhmCy4db18MmSujT3hv5KoBwpJCUx-WVB2cZi~J4keyU3g5wNZffXL7-fgUYZ1lGbq9HPZBDYcHC6LxRqyxTgt7ISYcQ7wNtluM1BkWlgWtrpvuYHo~hK4AiabfYQMgKT2RRx6iOB7J~7H8prBz-FwR6IPUSjJ2B8~z3BKpCKL~ef4ea6kX9EwLsqR-SsJv0zfqG~kh~~AlHtoYf7QqxGv340hVOSmSiAv~I3ZzgxTj3xlw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The CLfinder-OrthNet pipeline consists of two modules, CLfinder (Co-Linearity finder) and ONfinder (OrthNet finder). In the cartoon next to the box for the CLfinder module, blue squares indicate gene loci with disrupted co-linearity due to mutations and indels in neighbouring loci, while orange squares are those transposed and lost the synteny. See Supplementary Fig. S1 for details. Next to the box for the ONfinder module is an exemplary OrthNet. (A) For the detailed method to determine co-linearity (CL) relationship between the query loci and their most homologous counterpart (‘best-hits’) in the target genome, see Materials and Methods (2.2) and Figs S2 and S3. CLfinder output for the six crucifer species highlighted in Fig. 1 are summarized in Table 1, with the full results given as Supplementary Dataset S1. (B) See Figures 3–6, for examples of OrthNets with different evolutionary histories represented as different network topologies, e.g. transposition (tr) and transposition–duplication (tr–d) unique to each species or a group of species." path-from-xml="dsy035f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979912/dsy035f2.tif" path-from-xml="dsy035f2.tif" target="_blank">Open in new tab</a><a section="131979912" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f2.png?Expires=1576277252&amp;Signature=B22IvWLliS5hZ9yTzDrBLhKjH12wqCuugblgs3nSNuaVcCAskzG3rJbRehnRi06o2GJN-eWTzqgFDrraVh0lgzyC7aKZIwrmPbwMcVB~~LNKVpM6nspTnMhnU~xaY4-nTi9ihzQiY8qjCBmN-Wwak6kXpLH6jDJNM3LEThafpb0T1mHivpNSk3~0wV7fhGJwIPVRV0I9YfgE32GTAvNxjj-sc~O~zM7eHp7VYNnPN7x0ehHakziGogYqdpcwmSbKtse~IlZhtOh2iWad~XM4222QYRlmb3OObqAKfW1re0z7oBZbeDovClDL0CvipT0AJVqohtEU8P3YuUP2dCQRvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979912&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The CLfinder-OrthNet pipeline consists of two modules, CLfinder (Co-Linearity finder) and ONfinder (OrthNet finder). In the cartoon next to the box for the CLfinder module, blue squares indicate gene loci with disrupted co-linearity due to mutations and indels in neighbouring loci, while orange squares are those transposed and lost the synteny. See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> for details. Next to the box for the ONfinder module is an exemplary OrthNet. (A) For the detailed method to determine co-linearity (CL) relationship between the query loci and their most homologous counterpart (‘best-hits’) in the target genome, see Materials and Methods (2.2) and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Figs S2 and S3</a></span>. CLfinder output for the six crucifer species highlighted in <a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> are summarized in <a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, with the full results given as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S1. (B) See <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Figures 3–6</a>, for examples of OrthNets with different evolutionary histories represented as different network topologies, e.g. transposition (<em>tr</em>) and transposition–duplication (<em>tr–d</em>) unique to each species or a group of species.</p></div>
</div>
<p>The sensitivity and stringency of the CLfinder process are adjustable based on three user-defined parameters, the window size (<em>W</em>), the number of co-linear loci-in-chain threshold (<em>N</em>), and the maximum gap allowed between co-linear loci-in chain (<em>G</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). For each query locus (from Input 1), CLfinder scans and counts the number of ‘loci-in-chain’, whose best-hits (from Input 3) are separated by the same or less than <em>G</em> from the best-hit of their immediate neighbours. The scan starts from the query locus and moves towards both up- and downstream of the query genome. If more than <em>N</em> loci-in-chain, including the query locus, are found within a window size of 2 × <em>W</em> (i.e. <em>W</em> to the up- and downstream), the query locus is declared co-linear (<em>cl</em>) with its best-hit locus in the target species. When loci-in-chain were found only towards one direction, the query-target best-hit pair is considered to represent an end of a co-linear genome segment derived from inversions, indels, and segmental duplications involving multiple gene loci, and designated as ‘<em>cl_end</em>’. We excluded lineage-specific (<em>ls</em>) loci that did not have a best-hit in the target species from the search for co-linear loci-in-chain. Tandem duplicated (<em>td</em>) loci, defined as adjacent loci separated by the same or less than <em>T</em> loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and in the same paralogue cluster defined in Input 2, were counted as a single locus during the co-linearity search step. A query locus neither co-linear (<em>cl</em>) nor lineage-specific (<em>ls</em>) was considered to be transposed (<em>tr</em>). If the query locus is in a genome assembly scaffold that contains less than <em>W</em> loci, the pipeline will mark the query locus as not-determined (<em>nd</em>) instead of transposed (<em>tr</em>). Segmental rearrangements such as local inversion are identified based on loci-in-chain detected exclusively in either up- or down-stream. In such cases, CLfinder declares the query and its best-hit as <em>cl_end</em>.</p>
<p>For the analysis of six crucifer genomes, CLfinder parameters (<em>W</em> = 20, <em>N</em> = 3, <em>G</em> = 20, and <em>T</em> = 4) were decided based on the distribution of protein-coding gene locus content in the scaffolds of the most fragmented genome and the results from the analysis of co-linearity erosion. The window size (<em>W</em>) was chosen based on the most fragmented <em>S. irio</em> genome, where &gt;63% of total genes were in scaffolds that contained at least 20 gene loci. Maximum gap between co-linear loci-in-chain (<em>G</em> = 20) was decided to distinguish gene order displacements likely caused by local indels from those by transpositions, based on the result of co-linearity erosion analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1</a></span>). With <em>W</em> and <em>N</em> parameters set, we tested multiple number of co-linear loci-in-chain threshold (<em>N</em>) parameters to detect co-linear genes between simulated genomes of 27,000 gene loci, assuming a complete random shuffling and lack of a common ancestor between genomes. With <em>N</em> = 3, after 10,000 simulations comparing randomly shuffled simulated genomes, CLfinder found on average only 39.43 ± 9.85 genes (0.15 ± 0.04% of the simulated genome) as false-positive co-linear genes, indicating that the selected parameters can effectively rule out co-linearity by chance and without a common ancestry (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). We determined the maximum tandem duplication distance (<em>T</em> = 4) to enable detection of nested tandem duplication events while filtering out tandem duplications with too many unrelated genes inserted in between. The CLfinder process (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) was performed for all possible query-target pairs for the six crucifer species using a wrapper script in the CLfinder module (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, ‘CLfinder_multi.py’). The results are combined into a single table and given as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>.</p>                    <h3 scrollto-destination="131979915" id="131979915" class="section-title" data-legacy-id="SEC2.3">2.3. Construction of OrthNets for Brassicaceae genomes</h3>
<p>The second module of the pipeline, ONfinder (OrthNet finder), combines the output of the CLfinder module and produces networks of orthologous loci (OrthNet). OrthNets represent primary protein-coding gene loci from all species (nodes) connected with their best-hits with directional edges, and the co-linearity relationship (i.e. either <em>cl</em>, <em>tr</em>, or <em>nd</em>) between them as the edge property. In addition, OrthNets include tandem duplicated paralogues connected with undirected edges (<em>td</em>) among themselves. The ONfinder module also compares the ORF size of a node to the median ORF size of all neighbouring nodes to detect truncated ORFs (i.e. of sizes less than 40 and 80% of the median ORF size) in each OrthNet.</p>
<p>ONfinder first clusters all nodes from all species connected with an edge (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, ‘create_OrthNet.py’). This process often results in an aggregation of multiple loci connected with unidirectional edges in a single large OrthNet, due to duplicated paralogues, lineage-specific deletions, and partial gene models that lead to non-reciprocal best-hit pairs. We employed two methods to alleviate this issue. First, for all unidirectional edges from node A to node B, ONfinder searches for an alternative best-hit for B, among the list of the ten best-hit candidates, that makes A and B a reciprocal best-hit pair (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, ‘update_BestHitPairs.py’). The best-hit candidates were obtained by running BLASTN with the ‘-max_target_seqs 10’ option when creating the Input 2 for the CLfinder module. The ONfinder module records how many best-hit candidates it tested before identifying a reciprocal best-hit pair (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplmentary Dataset S1</a></span>, legend). Only when a reciprocal best-hit does not exist among all ten alternatives, the OrthNet will have a unidirectional edge. Second, each OrthNet is subjected to further clustering to finer OrthNets using Markov Chain Clustering (MCL),<a reveal-id="dsy035-B54" data-open="dsy035-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> with an inflation parameter (1.2) and a higher edge weight given in the order of <em>td</em> (1.5), reciprocal <em>cl</em> (1.2), unidirectional <em>cl</em> (0.6), reciprocal <em>tr</em> (0.5), and unidirectional <em>tr</em> (0.25) edges (default edge weight values used in this study are in parentheses). Edge weights for the MCL process were decided as detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S4</a></span>, with the following aims: (i) to separate networks of out-paralogues derived from multiple loci duplicated prior to the divergence of the six target genomes (as exemplified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), while (ii) keeping paralogues that underwent tandem duplication (<em>td</em>), transposition–duplication (<em>tr–d</em>), and combinations of <em>td</em> and <em>tr–d</em>, together with the core set of co-linear orthologues in the same OrthNet. Once an OrthNet was separated into multiple smaller OrthNets, any edge removed by MCL was replaced by an alternative edge connecting nodes within each of new OrthNets, by searching for an alternative best-hit pair among the list of ten best-hit candidates (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4C</a></span>). The ONfinder module records the number of best-hit candidates tested before the final edge was decided (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>). The MCL process effectively removed spurious unidirectional edges that connect nodes from different loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. ‘update_OrthNet_afterMCL.py’, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S4</a></span> for detailed discussion and an example). The final network units (OrthNets) were converted to the Simple Interaction File (SIF) format for visualization using Cytoscape (<a class="link link-uri" href="http://cytoscape.org/" target="_blank">http://cytoscape.org/</a> (10 October 2018, date last accessed)). All OrthNets derived from this analysis are included as archived SIFs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2</a></span>).</p>                    <h3 scrollto-destination="131979918" id="131979918" class="section-title" data-legacy-id="SEC2.4">2.4. Searching OrthNets that share an evolutionary history</h3>
<p>While CLfinder and ONfinder can operate independently, when used in a pipeline, the ONfinder module adds the results of its analysis to the CLfinder result for each locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>). For each gene locus that belonged to an OrthNet, ONfinder adds the unique ID of the OrthNet that includes the locus, number of nodes derived from each genome (i.e. gene copy numbers) in the OrthNet, and the edge types connecting the locus to its best-hit nodes in other genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>). This information enabled identifying OrthNets with the same or similar topologies that represent the same set of evolutionary events.</p>
<p>The ONfinder module offers two methods to search OrthNets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). To systematically identify OrthNets including <em>tr–d</em> events uniquely found in subsets of the six crucifer genomes, we searched for OrthNets that (i) showed a node copy number pattern consistent with duplications specific to the subset and (ii) contain nodes that are connected to all of their orthologous neighbours through unidirectional <em>tr</em> edges. We used the ‘search_OrthNet_pattern.py’ script (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and searched for node copy number and edge property patterns as listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>, in the CLfinder-OrthNet output (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>). For evolutionary history patterns presented in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, the second script, ‘search_OrthNet_topology.py’ (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), found all OrthNets with either exactly the same or similar topologies with the query OrthNet. </p>                        <a id="131979921" scrollto-destination="131979921"></a>
<div id="dsy035-F3" data-id="dsy035-F3" class="fig fig-section">
<div class="label fig-label">Figure 3.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f3.png?Expires=1576277252&amp;Signature=fJepzbVmqI3KgbO~c8M-8xIOsaa4nOC3Parejmpn0gXq-ml9-qfaSw3lGL0YKge9yJtKYxKA9QSHC~RfOWMXVZZQYAXeepCKVjJkY38jZZk1bvmL4WUhNURJQ5jxrFAJgNgoEh3EsnwvaRXUuzsL2h~xnghKwriNg2w3DfiyApw9btR4S3epRoOT0sr4Z628~cpGwuQ7F3yDRi~UpJIzeELoIGP5bBwBbhoMI1Iucl4YoOMyrQ7GZ5evgV7V~l4R-MewZkkrrLNyV9tvyLZiDcLuaWnVc2xJNRzFLFXhrM8PSfzW26y1jBJr0UtFbAgnndQ~G3HR2XNBYsH9Kz1HcA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ONfinder module encodes the evolutionary history of an orthologous gene group into an OrthNet. (A) OrthNet examples representing five different evolutionary histories of orthologous gene groups derived from the six Brassicaceae genomes (Fig. 1, highlighted with boxes). Nodes are colour-coded according to the species. Transparent nodes with dashed borders indicate loci with truncated ORFs, i.e. ORF sizes smaller than either 80% or 50% compared with the median ORF size of nodes they are connected to. Edges show properties either co-linear (cl) or transposed (tr), reciprocally (rc) or unidirectionally (uni). Tandem duplicated (td) paralogues are connected by undirected edges. The lower panel shows the ID and annotation for representative OrthNets, as well as the number of OrthNets representing the same evolutionary history among 17,432 OrthNets identified for the six Brassicaceae genomes. (B) A histogram showing the size distribution of OrthNets." path-from-xml="dsy035f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979921/dsy035f3.tif" path-from-xml="dsy035f3.tif" target="_blank">Open in new tab</a><a section="131979921" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f3.png?Expires=1576277252&amp;Signature=FaAGmW0ZWamEBSetxZ5-~vBPqAsqlUCFSKfxJn3KkewGGePOnAERUuQK6h7C62P1nlv4QEcdE8A9BCJTfPept4j1TunvaeVWj0kwLNQ-cDHC5ltpOnX112OsIz7OgFxGQRfK0pQM2CA-uyHLRZSEaaQW5SxdXL4kymjYZ0NhvqTII1rba7DZGC2P94km7ZHN-Ko8YW0WwTaHn0tqdqI9x-yDufKx5Eah-CXDArpbF1EPyYnxwY1hEE0LZnkKkxdLoSCfjJjUNb0rGTAQ7OwQRiLJMQi8IlXxMVRMPKJSLPPAZAnrHx9BRGspwUS31MmfQndNJdvDVDYFZ3iDCj9vRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979921&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The ONfinder module encodes the evolutionary history of an orthologous gene group into an OrthNet. (A) OrthNet examples representing five different evolutionary histories of orthologous gene groups derived from the six Brassicaceae genomes (<a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, highlighted with boxes). Nodes are colour-coded according to the species. Transparent nodes with dashed borders indicate loci with truncated ORFs, i.e. ORF sizes smaller than either 80% or 50% compared with the median ORF size of nodes they are connected to. Edges show properties either co-linear (<em>cl</em>) or transposed (<em>tr</em>), reciprocally (rc) or unidirectionally (uni). Tandem duplicated (<em>td</em>) paralogues are connected by undirected edges. The lower panel shows the ID and annotation for representative OrthNets, as well as the number of OrthNets representing the same evolutionary history among 17,432 OrthNets identified for the six Brassicaceae genomes. (B) A histogram showing the size distribution of OrthNets.</p></div>
</div>
<div content-id="dsy035-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f3.png?Expires=1576277252&amp;Signature=fJepzbVmqI3KgbO~c8M-8xIOsaa4nOC3Parejmpn0gXq-ml9-qfaSw3lGL0YKge9yJtKYxKA9QSHC~RfOWMXVZZQYAXeepCKVjJkY38jZZk1bvmL4WUhNURJQ5jxrFAJgNgoEh3EsnwvaRXUuzsL2h~xnghKwriNg2w3DfiyApw9btR4S3epRoOT0sr4Z628~cpGwuQ7F3yDRi~UpJIzeELoIGP5bBwBbhoMI1Iucl4YoOMyrQ7GZ5evgV7V~l4R-MewZkkrrLNyV9tvyLZiDcLuaWnVc2xJNRzFLFXhrM8PSfzW26y1jBJr0UtFbAgnndQ~G3HR2XNBYsH9Kz1HcA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ONfinder module encodes the evolutionary history of an orthologous gene group into an OrthNet. (A) OrthNet examples representing five different evolutionary histories of orthologous gene groups derived from the six Brassicaceae genomes (Fig. 1, highlighted with boxes). Nodes are colour-coded according to the species. Transparent nodes with dashed borders indicate loci with truncated ORFs, i.e. ORF sizes smaller than either 80% or 50% compared with the median ORF size of nodes they are connected to. Edges show properties either co-linear (cl) or transposed (tr), reciprocally (rc) or unidirectionally (uni). Tandem duplicated (td) paralogues are connected by undirected edges. The lower panel shows the ID and annotation for representative OrthNets, as well as the number of OrthNets representing the same evolutionary history among 17,432 OrthNets identified for the six Brassicaceae genomes. (B) A histogram showing the size distribution of OrthNets." path-from-xml="dsy035f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979921/dsy035f3.tif" path-from-xml="dsy035f3.tif" target="_blank">Open in new tab</a><a section="131979921" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f3.png?Expires=1576277252&amp;Signature=FaAGmW0ZWamEBSetxZ5-~vBPqAsqlUCFSKfxJn3KkewGGePOnAERUuQK6h7C62P1nlv4QEcdE8A9BCJTfPept4j1TunvaeVWj0kwLNQ-cDHC5ltpOnX112OsIz7OgFxGQRfK0pQM2CA-uyHLRZSEaaQW5SxdXL4kymjYZ0NhvqTII1rba7DZGC2P94km7ZHN-Ko8YW0WwTaHn0tqdqI9x-yDufKx5Eah-CXDArpbF1EPyYnxwY1hEE0LZnkKkxdLoSCfjJjUNb0rGTAQ7OwQRiLJMQi8IlXxMVRMPKJSLPPAZAnrHx9BRGspwUS31MmfQndNJdvDVDYFZ3iDCj9vRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979921&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The ONfinder module encodes the evolutionary history of an orthologous gene group into an OrthNet. (A) OrthNet examples representing five different evolutionary histories of orthologous gene groups derived from the six Brassicaceae genomes (<a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, highlighted with boxes). Nodes are colour-coded according to the species. Transparent nodes with dashed borders indicate loci with truncated ORFs, i.e. ORF sizes smaller than either 80% or 50% compared with the median ORF size of nodes they are connected to. Edges show properties either co-linear (<em>cl</em>) or transposed (<em>tr</em>), reciprocally (rc) or unidirectionally (uni). Tandem duplicated (<em>td</em>) paralogues are connected by undirected edges. The lower panel shows the ID and annotation for representative OrthNets, as well as the number of OrthNets representing the same evolutionary history among 17,432 OrthNets identified for the six Brassicaceae genomes. (B) A histogram showing the size distribution of OrthNets.</p></div>
</div>                    <h3 scrollto-destination="131979922" id="131979922" class="section-title" data-legacy-id="SEC2.5">2.5. Analysis of transposition–duplication (<em>tr–d</em>) events</h3>
<p>Within an OrthNet with a <em>tr–d</em> event, the <em>tr–d</em> donor or ‘CL copy’ was the node connected to orthologous nodes with the most <em>cl</em> edges, while the remaining were <em>tr–d</em> acceptors or ‘Tr copies’. When multiple CL copies existed due to tandem duplication, we used the one with the longest ORF as the representing CL copy. Homologous Genome Segments (HGSs) were detected between the gene and ±5 kb intergenic regions of the CL copy and each of Tr copies, using LASTZ with chaining as previously described in ref. <a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr">5</a>. A <em>tr–d</em> event was ‘complete’ if the HGS included the entire CL copy gene. A ‘gene-only’ <em>tr–d</em> was defined as a complete <em>tr–d</em> event with the size of HGS less than 120% of the coding region of the CL copy gene. We determined the expected occurrences of complete and gene-only <em>tr–d</em> by random shuffling and overlapping of HGSs and CL copy genes. Specifically, we assumed a random HGS selected among the observed HGS length distribution occurred at a random position in the vicinity of a CL copy gene, with a minimum 60 bps overlap between the genic region and a HGS. The distribution of such occurrences from 10,000 iterations was fitted to a normal distribution to calculate the <em>P</em>-value of the observed occurrence, using the fitdistr function in R MASS package (<a class="link link-uri" href="https://cran.r-project.org/web/packages/MASS" target="_blank">https://cran.r-project.org/web/packages/MASS</a> (10 October 2018, date last accessed)).</p>
<p>The four degenerate site (4d) substitution rates were calculated for all CL and Tr copy pairs where the Tr copy contained a complete ORF, using codeml<a reveal-id="dsy035-B53" data-open="dsy035-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and a custom script (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, ‘pairwiseKs_by_codeml.py’). All custom scripts used in this study are available at the CLfinder-OrthNet GitHub page (<a class="link link-uri" href="https://github.com/ohdongha/CL_finder" target="_blank">https://github.com/ohdongha/CL_finder</a>). To determine Tr copies with expression evidence, we used RNA-seq data for leaf and root tissues obtained from Wang et al.<a reveal-id="dsy035-B13" data-open="dsy035-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> for <em>A. lyrata</em>, <em>A. thaliana</em>, and <em>C. rubella</em>, and Oh et al.<a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> for <em>A. thaliana</em> and <em>S. parvula</em> as well as samples prepared for this study (for <em>E. salsugineum</em>, BioProject ID PRJNA63667) as previously described in Oh et al.<a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> FPKM values of representative transcript models were estimated using Stringtie (v. 1.3.1c) with the ‘-e’ option,<a reveal-id="dsy035-B55" data-open="dsy035-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> after RNAseq reads were aligned to the genome using HISAT2 (v. 2.0.5).<a reveal-id="dsy035-B55" data-open="dsy035-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p>                    <h2 scrollto-destination="131979925" id="131979925" class="section-title" data-legacy-id="SEC3">3. Results</h2>
                    <h3 scrollto-destination="131979926" id="131979926" class="section-title" data-legacy-id="SEC3.1">3.1. Patterns of co-linearity erosion within the six Brassicaceae genomes</h3>
<p>We selected a set of six genomes with the same whole genome duplication history sampled from the Brassicaceae Lineages I and II for this study (<a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). This set includes the model plant <em>A. thaliana</em> (Ath) and its relatives in Lineage I, <em>A. lyrata</em> (Aly) and <em>Capsella rubella</em> (Cru), as well as <em>Sisymbrium irio</em> (Sir) and the two extremophytes, <em>E. salsugineum</em> (Esa) and <em>S. parvula</em> (Spa), in Lineage II. <a reveal-id="dsy035-F1" data-open="dsy035-F1" class="link link-reveal link-table xref-fig">Figure 1</a> shows the phylogenetic relationships of the target species with other published genomes in Brassicaceae based on amino acid sequence alignments of 14,614 homolog clusters (see Materials and Methods, 2.1).</p>
<p>Before applying the CLfinder-OrthNet pipeline, we analysed the degree of co-linearity erosion among the target Brassicaceae genomes by comparing gene orders, as detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1</a></span>. Our analysis revealed that two-thirds of genes identified as non-transposable element (non-TE) and non-lineage-specific (non-LS) genes in the Brassicaceae genomes showed a conservation of gene order with their immediate neighbours when compared with the genome of <em>A. thaliana</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>, <em>d<sub>n</sub></em><sub>,</sub><sub><em>n</em></sub><sub>+1</sub> ≤1). The proportion of non-TE and non-LS gene loci showing a proximal (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>, <em>d<sub>n</sub></em><sub>,</sub><sub><em>n</em></sub><sub>+1</sub> = 2–20) and distal (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>, <em>d<sub>n</sub></em><sub>,</sub><sub><em>n</em></sub><sub>+1</sub> &gt;20 and ‘Diff Chr’) gene order displacement, compared with their immediate neighbours, was correlated with the divergence time between genomes and their TE contents, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1D</a></span>). This suggested two different models for co-linearity erosion, as summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1E</a></span>. The proximal gene order displacements were likely resulted from mutations and indels accumulated in the neighbouring loci over time (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>, blue loci). In this model, larger gene order displacements requiring multiple mutations in the neighbouring gene loci are rarer, which explains the rapid decline of genes in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span> as <em>d<sub>n</sub></em><sub>,</sub><em><sub>n+</sub></em><sub>1</sub> increases from 1, i.e. perfect co-linearity, to larger values (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1E</a></span>, blue line and arrow). The second model involves transposition of mostly single locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>, orange loci), which may have been initiated by the presence of repetitive sequences and TE activities. Transposition based on mechanisms such as the repair of double strand breaks by non-homologous end-joining (DSB-NHEJ)<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> can occur ubiquitously between any pair of genomic locations with a frequency lower than that of point mutations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1E</a></span>, orange line and arrow). Genes showing larger <em>d<sub>n</sub></em><sub>,</sub><em><sub>n+</sub></em><sub>1</sub> with their neighbours in both directions are most likely transposed via the second model. We used the result from co-linearity erosion (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span>) to determine parameters suitable for detecting co-linear loci between a pair of genomes, as detailed in Materials and Methods (2.2) and Supplementary Text S1.</p>                    <h3 scrollto-destination="131979929" id="131979929" class="section-title" data-legacy-id="SEC3.2">3.2. Development of the CLfinder-OrthNet pipeline</h3>
<p>Our pipeline consists of two modules: CLfinder and ONfinder (<a reveal-id="dsy035-F2" data-open="dsy035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The first module, CLfinder, compares all possible pairs of query and target genomes and test whether each homologous gene pair (i.e. ‘best-hit’ pair, Supplementary Text Glossary) is co-linear (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S2 and S3</a></span>, and Dataset S1). CLfinder accepts three inputs: representative gene models for all loci in each genome, clusters of paralogues within each genome, and lists of best-hits between all possible query-target genome pairs (<a reveal-id="dsy035-F2" data-open="dsy035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Users can select the methods and criteria for defining paralogue clusters and best-hit pairs, as well as the sensitivity and stringency for the co-linearity detection by controlling three parameters: the window size (<em>W</em>), the number of co-linear loci-in-chain threshold (<em>N</em>), and the maximum gap allowed between co-linear loci-in chain (<em>G</em>) (see Materials and Methods, 2.2). Based on these parameters, the CLfinder module searches both up- and downstream of each locus in the query genome for ‘loci-in-chain’ based on the order of their best-hits in the target genome, to determine whether a query-target best-hit pair is either co-linear (<em>cl</em>), transposed (<em>tr</em>), or not able to determine (<em>nd</em>) due to the query genome assembly scaffold being too short. When co-linearity was detected only towards one direction, the query-target best-hit pair is considered representing an end of a co-linear genome segment (<em>cl_end</em>) derived from inversions, indels, and segmental duplications involving multiple gene loci. A query locus without a best-hit in the target genome is marked lineage-specific (<em>ls</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p>
<p>The second module, ONfinder, combines all pairwise comparisons by CLfinder and encodes co-linearity relationships among orthologues into networks (OrthNets), with gene loci as nodes connected by an edge to their best-hits in other genomes (<a reveal-id="dsy035-F2" data-open="dsy035-F2" class="link link-reveal link-table xref-fig">Figs 2 and 3</a>). Each edge has a property of either co-linear (<em>cl</em>), transposed (<em>tr</em>), or not determined (<em>nd</em>). The <em>cl</em> and <em>tr</em> edges can be either reciprocal or unidirectional (<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, ‘rc’ and ‘uni’, respectively). OrthNets also include tandem duplicated (<em>td</em>) paralogues, connected by undirected edges [e.g. panel (4) in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>]. ONfinder uses Markov clustering (MCL),<a reveal-id="dsy035-B54" data-open="dsy035-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> based on edge weights assigned according to edge properties, to divide large networks that are often a result of expanded gene families with a large number of paralogues into smaller clusters likely derived from a single ancestral locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span> and Text S2). Each cluster of orthologues, separated by MCL, is given an OrthNet ID and represented as an orthologue network or an OrthNet. Finally, ONfinder can search with a user-defined pattern of orthologue copy numbers, edge characteristics, and network topology, to retrieve all OrthNets sharing a given set of evolutionary events (see Materials and Methods, 2.4). Several selected examples of OrthNets representing different evolutionary histories are shown in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>.</p>                    <h3 scrollto-destination="131979932" id="131979932" class="section-title" data-legacy-id="SEC3.3">3.3. Identification of OrthNets among six Brassicaceae genomes</h3>
<p>We tested the CLfinder-OrthNet pipeline on the six Brassicaceae genomes using parameters and input files as described in Materials and Methods. The CLfinder module summarizes all reciprocal query-target genome pairwise analyses as exemplified for the six Brassicaceae genomes in <a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>. For simplicity, we considered <em>cl_end</em> loci pairs as <em>cl</em> in this summary. All query-target genome pairs showed a comparable number of <em>cl</em> loci pairs, ranging from 19,015 (<a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, Sir–Aly) to 24,296 (<a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, Aly–Ath). The number of <em>cl</em> pairs follows the division of the Lineage I (<a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, Aly, Ath, and Cru) and II (<a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, Esa, Sir, and Spa), with higher numbers observed between query-target pairs within each Lineage. The number of <em>tr</em> loci pairs was proportional to the repeat contents of the query genomes. For example, <em>A. lyrata</em> and <em>E. salsugineum</em> are the query genomes with the highest content of <em>tr</em> pairs (<a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, Aly and Esa), which correlated with the higher content of repeats in these two genomes than in <em>A. thaliana</em>, <em>C. rubella</em>, or <em>S. parvula</em> genomes [‘TE(%)’ row in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>]. When <em>S. irio</em> was the query, the proportion of <em>nd</em> pairs was higher than all other genomes (<a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>, Sir), because it had the most fragmented genome assembly among the six genomes. The entire CLfinder results for all query-target genome pairs are in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>. </p>
<div content-id="dsy035-T1" class="table-modal"><div class="table-wrap" content-id="dsy035-T1">
<div id="dsy035-T1" data-id="dsy035-T1" class="table-wrap-title">
<div class="label">Table 1.</div>
<div class="caption"><p>Summary of CLfinder results showing pairwise comparisons among 6 crucifer species</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Query species </th>
<th rowspan="2"># Protein-coding genes </th>
<th rowspan="2">CL type<sup>a</sup> </th>
<th colspan="6">Target species </th>
<th rowspan="2"># <em>td</em><sup>b</sup> events (# <em>td</em> genes) </th>
</tr>
<tr>
<th>Aly </th>
<th>Ath </th>
<th>Cru </th>
<th>Esa </th>
<th>Sir </th>
<th>Spa </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="4">Aly </td>
<td rowspan="4">32,657 </td>
<td>
<em>cl</em> </td>
<td rowspan="4"> </td>
<td>24,296 </td>
<td>23,055 </td>
<td>21,416 </td>
<td>19,988 </td>
<td>21,032 </td>
<td rowspan="4">2,163 (5,733) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,881 </td>
<td>5,375 </td>
<td>6,668 </td>
<td>8,104 </td>
<td>6,478 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>2,876 </td>
<td>3,611 </td>
<td>3,954 </td>
<td>3,902 </td>
<td>4,530 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>604 </td>
<td>616 </td>
<td>619 </td>
<td>663 </td>
<td>617 </td>
</tr>
<tr>
<td rowspan="4">Ath </td>
<td rowspan="4">27,206 </td>
<td>
<em>cl</em> </td>
<td>23,436 </td>
<td rowspan="4"> </td>
<td>22,683 </td>
<td>21,187 </td>
<td>19,821 </td>
<td>20,851 </td>
<td rowspan="4">1,747 (4,770) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,431 </td>
<td>2,804 </td>
<td>4,032 </td>
<td>5,355 </td>
<td>4,064 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,339 </td>
<td>1,719 </td>
<td>1,987 </td>
<td>2,030 </td>
<td>2,291 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td rowspan="4">Cru </td>
<td rowspan="4">26,521 </td>
<td>
<em>cl</em> </td>
<td>22,371 </td>
<td>22,836 </td>
<td rowspan="4"> </td>
<td>20,906 </td>
<td>19,350 </td>
<td>20,436 </td>
<td rowspan="4">1,752 (4,996) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,036 </td>
<td>2,836 </td>
<td>4,338 </td>
<td>5,817 </td>
<td>4,267 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>950 </td>
<td>666 </td>
<td>1,112 </td>
<td>1,154 </td>
<td>1,646 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>164 </td>
<td>183 </td>
<td>165 </td>
<td>200 </td>
<td>172 </td>
</tr>
<tr>
<td rowspan="4">Esa </td>
<td rowspan="4">26,351 </td>
<td>
<em>cl</em> </td>
<td>20,384 </td>
<td>20,884 </td>
<td>20,460 </td>
<td rowspan="4"> </td>
<td>19,699 </td>
<td>20,612 </td>
<td rowspan="4">1,646 (4,461) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,465 </td>
<td>4,137 </td>
<td>4,460 </td>
<td>5,431 </td>
<td>4,046 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,452 </td>
<td>1,274 </td>
<td>1,377 </td>
<td>1,146 </td>
<td>1,631 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>50 </td>
<td>56 </td>
<td>54 </td>
<td>75 </td>
<td>62 </td>
</tr>
<tr>
<td rowspan="4">Sir </td>
<td rowspan="4">32,524 </td>
<td>
<em>cl</em> </td>
<td>19,015 </td>
<td>19,538 </td>
<td>19,068 </td>
<td>19,728 </td>
<td rowspan="4"> </td>
<td>19,766 </td>
<td rowspan="4">1,795 (4,586) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,062 </td>
<td>2,860 </td>
<td>2,998 </td>
<td>2,722 </td>
<td>2,697 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>5,520 </td>
<td>5,148 </td>
<td>5,496 </td>
<td>5,054 </td>
<td>4,225 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>4,927 </td>
<td>4,978 </td>
<td>4,962 </td>
<td>5,020 </td>
<td>5,836 </td>
</tr>
<tr>
<td rowspan="4">Spa </td>
<td rowspan="4">26,847 </td>
<td>
<em>cl</em> </td>
<td>19,849 </td>
<td>20,358 </td>
<td>19,934 </td>
<td>20,380 </td>
<td>19,546 </td>
<td rowspan="4"> </td>
<td rowspan="4">1,242 (3,049) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,830 </td>
<td>2,452 </td>
<td>2,718 </td>
<td>2,534 </td>
<td>4,097 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>3,649 </td>
<td>3,541 </td>
<td>3,688 </td>
<td>3,432 </td>
<td>2,526 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>519 </td>
<td>496 </td>
<td>507 </td>
<td>501 </td>
<td>678 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy035-T1" class="table-modal"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Query species </th>
<th rowspan="2"># Protein-coding genes </th>
<th rowspan="2">CL type<sup>a</sup> </th>
<th colspan="6">Target species </th>
<th rowspan="2"># <em>td</em><sup>b</sup> events (# <em>td</em> genes) </th>
</tr>
<tr>
<th>Aly </th>
<th>Ath </th>
<th>Cru </th>
<th>Esa </th>
<th>Sir </th>
<th>Spa </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="4">Aly </td>
<td rowspan="4">32,657 </td>
<td>
<em>cl</em> </td>
<td rowspan="4"> </td>
<td>24,296 </td>
<td>23,055 </td>
<td>21,416 </td>
<td>19,988 </td>
<td>21,032 </td>
<td rowspan="4">2,163 (5,733) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,881 </td>
<td>5,375 </td>
<td>6,668 </td>
<td>8,104 </td>
<td>6,478 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>2,876 </td>
<td>3,611 </td>
<td>3,954 </td>
<td>3,902 </td>
<td>4,530 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>604 </td>
<td>616 </td>
<td>619 </td>
<td>663 </td>
<td>617 </td>
</tr>
<tr>
<td rowspan="4">Ath </td>
<td rowspan="4">27,206 </td>
<td>
<em>cl</em> </td>
<td>23,436 </td>
<td rowspan="4"> </td>
<td>22,683 </td>
<td>21,187 </td>
<td>19,821 </td>
<td>20,851 </td>
<td rowspan="4">1,747 (4,770) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,431 </td>
<td>2,804 </td>
<td>4,032 </td>
<td>5,355 </td>
<td>4,064 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,339 </td>
<td>1,719 </td>
<td>1,987 </td>
<td>2,030 </td>
<td>2,291 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td rowspan="4">Cru </td>
<td rowspan="4">26,521 </td>
<td>
<em>cl</em> </td>
<td>22,371 </td>
<td>22,836 </td>
<td rowspan="4"> </td>
<td>20,906 </td>
<td>19,350 </td>
<td>20,436 </td>
<td rowspan="4">1,752 (4,996) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,036 </td>
<td>2,836 </td>
<td>4,338 </td>
<td>5,817 </td>
<td>4,267 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>950 </td>
<td>666 </td>
<td>1,112 </td>
<td>1,154 </td>
<td>1,646 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>164 </td>
<td>183 </td>
<td>165 </td>
<td>200 </td>
<td>172 </td>
</tr>
<tr>
<td rowspan="4">Esa </td>
<td rowspan="4">26,351 </td>
<td>
<em>cl</em> </td>
<td>20,384 </td>
<td>20,884 </td>
<td>20,460 </td>
<td rowspan="4"> </td>
<td>19,699 </td>
<td>20,612 </td>
<td rowspan="4">1,646 (4,461) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,465 </td>
<td>4,137 </td>
<td>4,460 </td>
<td>5,431 </td>
<td>4,046 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,452 </td>
<td>1,274 </td>
<td>1,377 </td>
<td>1,146 </td>
<td>1,631 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>50 </td>
<td>56 </td>
<td>54 </td>
<td>75 </td>
<td>62 </td>
</tr>
<tr>
<td rowspan="4">Sir </td>
<td rowspan="4">32,524 </td>
<td>
<em>cl</em> </td>
<td>19,015 </td>
<td>19,538 </td>
<td>19,068 </td>
<td>19,728 </td>
<td rowspan="4"> </td>
<td>19,766 </td>
<td rowspan="4">1,795 (4,586) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,062 </td>
<td>2,860 </td>
<td>2,998 </td>
<td>2,722 </td>
<td>2,697 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>5,520 </td>
<td>5,148 </td>
<td>5,496 </td>
<td>5,054 </td>
<td>4,225 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>4,927 </td>
<td>4,978 </td>
<td>4,962 </td>
<td>5,020 </td>
<td>5,836 </td>
</tr>
<tr>
<td rowspan="4">Spa </td>
<td rowspan="4">26,847 </td>
<td>
<em>cl</em> </td>
<td>19,849 </td>
<td>20,358 </td>
<td>19,934 </td>
<td>20,380 </td>
<td>19,546 </td>
<td rowspan="4"> </td>
<td rowspan="4">1,242 (3,049) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,830 </td>
<td>2,452 </td>
<td>2,718 </td>
<td>2,534 </td>
<td>4,097 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>3,649 </td>
<td>3,541 </td>
<td>3,688 </td>
<td>3,432 </td>
<td>2,526 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>519 </td>
<td>496 </td>
<td>507 </td>
<td>501 </td>
<td>678 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>Co-linear (<em>cl</em>), transposed (<em>tr</em>), lineage-specific (<em>ls</em>), or not determined due to too small genome scaffold (<em>nd</em>), with CLfinder parameters {window_size, num_CL_trshld, gap_CL_trshld} = { 20, 3, 20 }.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Tandem duplication (<em>td</em>) detected using the parameter max_TD_loci_dist = 4.</p>
</div></div>
</div>
</div></div>
<div class="table-wrap" content-id="dsy035-T1">
<div id="dsy035-T1" data-id="dsy035-T1" class="table-wrap-title">
<div class="label">Table 1.</div>
<div class="caption"><p>Summary of CLfinder results showing pairwise comparisons among 6 crucifer species</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Query species </th>
<th rowspan="2"># Protein-coding genes </th>
<th rowspan="2">CL type<sup>a</sup> </th>
<th colspan="6">Target species </th>
<th rowspan="2"># <em>td</em><sup>b</sup> events (# <em>td</em> genes) </th>
</tr>
<tr>
<th>Aly </th>
<th>Ath </th>
<th>Cru </th>
<th>Esa </th>
<th>Sir </th>
<th>Spa </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="4">Aly </td>
<td rowspan="4">32,657 </td>
<td>
<em>cl</em> </td>
<td rowspan="4"> </td>
<td>24,296 </td>
<td>23,055 </td>
<td>21,416 </td>
<td>19,988 </td>
<td>21,032 </td>
<td rowspan="4">2,163 (5,733) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,881 </td>
<td>5,375 </td>
<td>6,668 </td>
<td>8,104 </td>
<td>6,478 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>2,876 </td>
<td>3,611 </td>
<td>3,954 </td>
<td>3,902 </td>
<td>4,530 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>604 </td>
<td>616 </td>
<td>619 </td>
<td>663 </td>
<td>617 </td>
</tr>
<tr>
<td rowspan="4">Ath </td>
<td rowspan="4">27,206 </td>
<td>
<em>cl</em> </td>
<td>23,436 </td>
<td rowspan="4"> </td>
<td>22,683 </td>
<td>21,187 </td>
<td>19,821 </td>
<td>20,851 </td>
<td rowspan="4">1,747 (4,770) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,431 </td>
<td>2,804 </td>
<td>4,032 </td>
<td>5,355 </td>
<td>4,064 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,339 </td>
<td>1,719 </td>
<td>1,987 </td>
<td>2,030 </td>
<td>2,291 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td rowspan="4">Cru </td>
<td rowspan="4">26,521 </td>
<td>
<em>cl</em> </td>
<td>22,371 </td>
<td>22,836 </td>
<td rowspan="4"> </td>
<td>20,906 </td>
<td>19,350 </td>
<td>20,436 </td>
<td rowspan="4">1,752 (4,996) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,036 </td>
<td>2,836 </td>
<td>4,338 </td>
<td>5,817 </td>
<td>4,267 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>950 </td>
<td>666 </td>
<td>1,112 </td>
<td>1,154 </td>
<td>1,646 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>164 </td>
<td>183 </td>
<td>165 </td>
<td>200 </td>
<td>172 </td>
</tr>
<tr>
<td rowspan="4">Esa </td>
<td rowspan="4">26,351 </td>
<td>
<em>cl</em> </td>
<td>20,384 </td>
<td>20,884 </td>
<td>20,460 </td>
<td rowspan="4"> </td>
<td>19,699 </td>
<td>20,612 </td>
<td rowspan="4">1,646 (4,461) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,465 </td>
<td>4,137 </td>
<td>4,460 </td>
<td>5,431 </td>
<td>4,046 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,452 </td>
<td>1,274 </td>
<td>1,377 </td>
<td>1,146 </td>
<td>1,631 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>50 </td>
<td>56 </td>
<td>54 </td>
<td>75 </td>
<td>62 </td>
</tr>
<tr>
<td rowspan="4">Sir </td>
<td rowspan="4">32,524 </td>
<td>
<em>cl</em> </td>
<td>19,015 </td>
<td>19,538 </td>
<td>19,068 </td>
<td>19,728 </td>
<td rowspan="4"> </td>
<td>19,766 </td>
<td rowspan="4">1,795 (4,586) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,062 </td>
<td>2,860 </td>
<td>2,998 </td>
<td>2,722 </td>
<td>2,697 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>5,520 </td>
<td>5,148 </td>
<td>5,496 </td>
<td>5,054 </td>
<td>4,225 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>4,927 </td>
<td>4,978 </td>
<td>4,962 </td>
<td>5,020 </td>
<td>5,836 </td>
</tr>
<tr>
<td rowspan="4">Spa </td>
<td rowspan="4">26,847 </td>
<td>
<em>cl</em> </td>
<td>19,849 </td>
<td>20,358 </td>
<td>19,934 </td>
<td>20,380 </td>
<td>19,546 </td>
<td rowspan="4"> </td>
<td rowspan="4">1,242 (3,049) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,830 </td>
<td>2,452 </td>
<td>2,718 </td>
<td>2,534 </td>
<td>4,097 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>3,649 </td>
<td>3,541 </td>
<td>3,688 </td>
<td>3,432 </td>
<td>2,526 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>519 </td>
<td>496 </td>
<td>507 </td>
<td>501 </td>
<td>678 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy035-T1" class="table-modal"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Query species </th>
<th rowspan="2"># Protein-coding genes </th>
<th rowspan="2">CL type<sup>a</sup> </th>
<th colspan="6">Target species </th>
<th rowspan="2"># <em>td</em><sup>b</sup> events (# <em>td</em> genes) </th>
</tr>
<tr>
<th>Aly </th>
<th>Ath </th>
<th>Cru </th>
<th>Esa </th>
<th>Sir </th>
<th>Spa </th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="4">Aly </td>
<td rowspan="4">32,657 </td>
<td>
<em>cl</em> </td>
<td rowspan="4"> </td>
<td>24,296 </td>
<td>23,055 </td>
<td>21,416 </td>
<td>19,988 </td>
<td>21,032 </td>
<td rowspan="4">2,163 (5,733) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,881 </td>
<td>5,375 </td>
<td>6,668 </td>
<td>8,104 </td>
<td>6,478 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>2,876 </td>
<td>3,611 </td>
<td>3,954 </td>
<td>3,902 </td>
<td>4,530 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>604 </td>
<td>616 </td>
<td>619 </td>
<td>663 </td>
<td>617 </td>
</tr>
<tr>
<td rowspan="4">Ath </td>
<td rowspan="4">27,206 </td>
<td>
<em>cl</em> </td>
<td>23,436 </td>
<td rowspan="4"> </td>
<td>22,683 </td>
<td>21,187 </td>
<td>19,821 </td>
<td>20,851 </td>
<td rowspan="4">1,747 (4,770) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,431 </td>
<td>2,804 </td>
<td>4,032 </td>
<td>5,355 </td>
<td>4,064 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,339 </td>
<td>1,719 </td>
<td>1,987 </td>
<td>2,030 </td>
<td>2,291 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td rowspan="4">Cru </td>
<td rowspan="4">26,521 </td>
<td>
<em>cl</em> </td>
<td>22,371 </td>
<td>22,836 </td>
<td rowspan="4"> </td>
<td>20,906 </td>
<td>19,350 </td>
<td>20,436 </td>
<td rowspan="4">1,752 (4,996) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,036 </td>
<td>2,836 </td>
<td>4,338 </td>
<td>5,817 </td>
<td>4,267 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>950 </td>
<td>666 </td>
<td>1,112 </td>
<td>1,154 </td>
<td>1,646 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>164 </td>
<td>183 </td>
<td>165 </td>
<td>200 </td>
<td>172 </td>
</tr>
<tr>
<td rowspan="4">Esa </td>
<td rowspan="4">26,351 </td>
<td>
<em>cl</em> </td>
<td>20,384 </td>
<td>20,884 </td>
<td>20,460 </td>
<td rowspan="4"> </td>
<td>19,699 </td>
<td>20,612 </td>
<td rowspan="4">1,646 (4,461) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>4,465 </td>
<td>4,137 </td>
<td>4,460 </td>
<td>5,431 </td>
<td>4,046 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>1,452 </td>
<td>1,274 </td>
<td>1,377 </td>
<td>1,146 </td>
<td>1,631 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>50 </td>
<td>56 </td>
<td>54 </td>
<td>75 </td>
<td>62 </td>
</tr>
<tr>
<td rowspan="4">Sir </td>
<td rowspan="4">32,524 </td>
<td>
<em>cl</em> </td>
<td>19,015 </td>
<td>19,538 </td>
<td>19,068 </td>
<td>19,728 </td>
<td rowspan="4"> </td>
<td>19,766 </td>
<td rowspan="4">1,795 (4,586) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>3,062 </td>
<td>2,860 </td>
<td>2,998 </td>
<td>2,722 </td>
<td>2,697 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>5,520 </td>
<td>5,148 </td>
<td>5,496 </td>
<td>5,054 </td>
<td>4,225 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>4,927 </td>
<td>4,978 </td>
<td>4,962 </td>
<td>5,020 </td>
<td>5,836 </td>
</tr>
<tr>
<td rowspan="4">Spa </td>
<td rowspan="4">26,847 </td>
<td>
<em>cl</em> </td>
<td>19,849 </td>
<td>20,358 </td>
<td>19,934 </td>
<td>20,380 </td>
<td>19,546 </td>
<td rowspan="4"> </td>
<td rowspan="4">1,242 (3,049) </td>
</tr>
<tr>
<td>
<em>tr</em> </td>
<td>2,830 </td>
<td>2,452 </td>
<td>2,718 </td>
<td>2,534 </td>
<td>4,097 </td>
</tr>
<tr>
<td>
<em>ls</em> </td>
<td>3,649 </td>
<td>3,541 </td>
<td>3,688 </td>
<td>3,432 </td>
<td>2,526 </td>
</tr>
<tr>
<td>
<em>nd</em> </td>
<td>519 </td>
<td>496 </td>
<td>507 </td>
<td>501 </td>
<td>678 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>Co-linear (<em>cl</em>), transposed (<em>tr</em>), lineage-specific (<em>ls</em>), or not determined due to too small genome scaffold (<em>nd</em>), with CLfinder parameters {window_size, num_CL_trshld, gap_CL_trshld} = { 20, 3, 20 }.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Tandem duplication (<em>td</em>) detected using the parameter max_TD_loci_dist = 4.</p>
</div></div>
</div>
</div>
<p>The ONfinder module combined all pairwise CLfinder analyses and developed OrthNets representing the evolutionary history of each set of orthologous loci as the network topology (<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). For an analysis of N genomes, a perfect polygon (e.g. hexagon in this study) with each of N nodes connected to other nodes by N-1 bidirectional solid grey edges represents a single-copy co-linear orthologous gene in all genomes [<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, panel (1)]. We identified a total of 7,034 OrthNets that showed single-copy loci co-linear to each other in all genomes. Panel (2) of <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> is an example from 50 OrthNets with co-linearity found within each of the Lineages I and II while loci between the two Lineages were transposed, representing a transposition event following the lineage divergence. Panel (3) shows one of the nine OrthNets with only the locus in <em>S. parvula</em> transposed compared with all other species. We found 44 OrthNets with the same evolutionary history depicted in panel (4), i.e. <em>E. salsugineum</em>-specific tandem duplication, and 86 OrthNets for <em>S. parvula</em>-specific transposition–duplication (<em>tr–d</em>) events shown in panel (5) of <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. ONfinder also compares the ORF size of a node with the median ORF size of all other orthologous nodes to which the node is connected, to identify truncated ORFs [e.g. panels (3), (4), and (5) in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>]. We included all OrthNets together with the CLfinder results identified among the six Brassicaceae genomes in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>.</p>
<p>An OrthNet may include a disproportionately large number of duplicated gene loci in specific genomes. For example, an OrthNet showing <em>A. lyrata</em>-specific <em>tr–d</em> events included 82 nodes representing additional <em>A. lyrata</em> transposed–duplicated paralogue copies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Such duplication events, as well as large gene families where exact reciprocal orthologue pairs were hard to identify among multiple paralogues, may result in an OrthNet with a large number of nodes. However, more than 85% of OrthNets contain the same or less than 12 nodes per OrthNet (14,849 out of total 17,432 OrthNets), likely derived from single ancestral loci with duplications restricted in a subset of the six Brassicaceae genomes (<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The size distribution of OrthNets was also comparable with that of orthologous gene clusters detected based on sequence homology by OrthoFinder<a reveal-id="dsy035-B56" data-open="dsy035-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6A</a></span>). The majority of OrthNets was matched 1-vs-1 with an OrthoFinder cluster (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6B</a></span>). In all OrthNet-OrthoFinder cluster pairs, 70.1% total OrthNets contained the set of genes identical to, and additional 12.3% OrthNets showed more than 80% overlap with, their corresponding OrthoFinder clusters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6C</a></span>, dashed box). An example of OrthNet different from the orthologous gene cluster detected by OrthoFinder is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span> and Text S2, respectively.</p>                    <h3 scrollto-destination="131979936" id="131979936" class="section-title" data-legacy-id="SEC3.4">3.4. Characterization of lineage-specific and shared <em>tr–d</em> events among Brassicaceae genomes</h3>
<p>We used the ‘search_OrthNet_pattern.py’ script in the ONfinder (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S3</a></span>) to detect OrthNets representing <em>tr</em> and <em>tr–d</em> events either unique to each of the six Brassicaceae genomes or shared by more than one genome (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Figs 4 and 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3). The number of OrthNets that showed <em>tr</em> events was smaller than those with <em>tr–d</em> events for all subsets of genomes including lineage-specific events (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). This observation agrees with the postulation that a <em>tr</em> event is a result of a deletion of the original/donor copy after a <em>tr–d</em> event.<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a></p>                        <a id="131979938" scrollto-destination="131979938"></a>
<div id="dsy035-F4" data-id="dsy035-F4" class="fig fig-section">
<div class="label fig-label">Figure 4.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f4.png?Expires=1576277252&amp;Signature=Xue4PWEZ3vSXOtvxNxHvfFx6k9f08-DPxHa0MUjSYuIfaCqiVRwlSNC3g8si4k2nrGbyyam7Ls55hVCTG0sevyhiPzY8-KsB7EBng7h3wFDiLEowfI4ety6VLwF1xyyF9qUf0wJU7gusAUvgWrzk68BmdNZX1j8KGIyAuuYAwJfKy2WdCA~KFR4pBGhTiOB4VZQh3UzrZNqHYljSBsvvCHaaC~SAzGZAm5irQRWQa1RvPfCEha9suA-HUyN6JKnTGLtjFs2LEuqy8OFuuj8Bb1zE25DFfX-G06OK~zLvTWYdFYzXcaXX30s7PZK8uRxzoFU9-Ii9yCU-ZwiBy~iTlQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of lineage-specific transposition–duplication (tr–d) events among the six Brassicaceae genomes. (A) Examples of OrthNets with tr–d events specific for S. parvula (WRKY DNA-BINDING PROTEIN 72/WRKY72) and C. rubella (AGAMOUS-LIKE 87/AGL87). Within a tr–d event, the original donor copy (CL copy) is reciprocally co-linear to orthologues in other genomes (Orthologs), while transposed and duplicated paralogues (Tr copies) are not. Nodes and edges are as described in Fig. 3A. (B) Comparison between the ORF size of all loci involved in a tr–d event (i.e. both CL and Tr copies) with the median ORF size of Orthologs. Blue (filled) dots indicate CL copies. (C and D) ORF size comparison between Tr copies and their corresponding CL copy within each of the tr–d events, as a scatterplot (C) and a histogram of ORF size ratio (D). Pink shades indicate Tr copies with complete ORFs whose sizes are comparable (±20% in proportion) to that of the CL copy. Panel D also shows numbers and percentages (in parentheses) of Tr copies with complete ORFs below the species labels. The entire list of OrthNets showing lineage-specific tr–d, including CL and Tr copies, is in Supplementary Dataset S3." path-from-xml="dsy035f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979938/dsy035f4.tif" path-from-xml="dsy035f4.tif" target="_blank">Open in new tab</a><a section="131979938" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f4.png?Expires=1576277252&amp;Signature=a9uEeM6hYPJ151z-pA2FU1QYoX01GmxaU~3zgdI4IIHKThd8DdIulU3Q8--32F4vxiQKZazls5BTkq02XxMEXhkXu~XVeszBuuEUJgL36-jfXtRPlC9qUYsRjwY62Qj89Th3rpP9UYk2AdxqL8bX4nelvni5OKSX4eqeOCSkJjDgnkZl88~osmHL6STnD9keNXw4F2u2XE5Qo3Y-bUc5KALQa-x8E6rWo6HSohN5d71LQ9cz5ded7xOx9D8QSVGnID3N5Zj8Sp15oYk~VNB9MuQRnHHtng-XEywBUp~bYyFl3QRB-lN5lLqgSC5g6fG8dVnZmtGCyBGNJa4e~nJciQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979938&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of lineage-specific transposition–duplication (<em>tr–d</em>) events among the six Brassicaceae genomes. (A) Examples of OrthNets with <em>tr–d</em> events specific for <em>S. parvula</em> (<em>WRKY DNA-BINDING PROTEIN 72</em>/<em>WRKY72</em>) and <em>C. rubella</em> (<em>AGAMOUS-LIKE 87</em>/<em>AGL87</em>). Within a <em>tr–d</em> event, the original donor copy (CL copy) is reciprocally co-linear to orthologues in other genomes (Orthologs), while transposed and duplicated paralogues (Tr copies) are not. Nodes and edges are as described in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. (B) Comparison between the ORF size of all loci involved in a <em>tr–d</em> event (i.e. both CL and Tr copies) with the median ORF size of Orthologs. Blue (filled) dots indicate CL copies. (C and D) ORF size comparison between Tr copies and their corresponding CL copy within each of the <em>tr–d</em> events, as a scatterplot (C) and a histogram of ORF size ratio (D). Pink shades indicate Tr copies with complete ORFs whose sizes are comparable (±20% in proportion) to that of the CL copy. Panel D also shows numbers and percentages (in parentheses) of Tr copies with complete ORFs below the species labels. The entire list of OrthNets showing lineage-specific <em>tr–d</em>, including CL and Tr copies, is in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3.</p></div>
</div>
<div content-id="dsy035-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f4.png?Expires=1576277252&amp;Signature=Xue4PWEZ3vSXOtvxNxHvfFx6k9f08-DPxHa0MUjSYuIfaCqiVRwlSNC3g8si4k2nrGbyyam7Ls55hVCTG0sevyhiPzY8-KsB7EBng7h3wFDiLEowfI4ety6VLwF1xyyF9qUf0wJU7gusAUvgWrzk68BmdNZX1j8KGIyAuuYAwJfKy2WdCA~KFR4pBGhTiOB4VZQh3UzrZNqHYljSBsvvCHaaC~SAzGZAm5irQRWQa1RvPfCEha9suA-HUyN6JKnTGLtjFs2LEuqy8OFuuj8Bb1zE25DFfX-G06OK~zLvTWYdFYzXcaXX30s7PZK8uRxzoFU9-Ii9yCU-ZwiBy~iTlQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of lineage-specific transposition–duplication (tr–d) events among the six Brassicaceae genomes. (A) Examples of OrthNets with tr–d events specific for S. parvula (WRKY DNA-BINDING PROTEIN 72/WRKY72) and C. rubella (AGAMOUS-LIKE 87/AGL87). Within a tr–d event, the original donor copy (CL copy) is reciprocally co-linear to orthologues in other genomes (Orthologs), while transposed and duplicated paralogues (Tr copies) are not. Nodes and edges are as described in Fig. 3A. (B) Comparison between the ORF size of all loci involved in a tr–d event (i.e. both CL and Tr copies) with the median ORF size of Orthologs. Blue (filled) dots indicate CL copies. (C and D) ORF size comparison between Tr copies and their corresponding CL copy within each of the tr–d events, as a scatterplot (C) and a histogram of ORF size ratio (D). Pink shades indicate Tr copies with complete ORFs whose sizes are comparable (±20% in proportion) to that of the CL copy. Panel D also shows numbers and percentages (in parentheses) of Tr copies with complete ORFs below the species labels. The entire list of OrthNets showing lineage-specific tr–d, including CL and Tr copies, is in Supplementary Dataset S3." path-from-xml="dsy035f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979938/dsy035f4.tif" path-from-xml="dsy035f4.tif" target="_blank">Open in new tab</a><a section="131979938" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f4.png?Expires=1576277252&amp;Signature=a9uEeM6hYPJ151z-pA2FU1QYoX01GmxaU~3zgdI4IIHKThd8DdIulU3Q8--32F4vxiQKZazls5BTkq02XxMEXhkXu~XVeszBuuEUJgL36-jfXtRPlC9qUYsRjwY62Qj89Th3rpP9UYk2AdxqL8bX4nelvni5OKSX4eqeOCSkJjDgnkZl88~osmHL6STnD9keNXw4F2u2XE5Qo3Y-bUc5KALQa-x8E6rWo6HSohN5d71LQ9cz5ded7xOx9D8QSVGnID3N5Zj8Sp15oYk~VNB9MuQRnHHtng-XEywBUp~bYyFl3QRB-lN5lLqgSC5g6fG8dVnZmtGCyBGNJa4e~nJciQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979938&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of lineage-specific transposition–duplication (<em>tr–d</em>) events among the six Brassicaceae genomes. (A) Examples of OrthNets with <em>tr–d</em> events specific for <em>S. parvula</em> (<em>WRKY DNA-BINDING PROTEIN 72</em>/<em>WRKY72</em>) and <em>C. rubella</em> (<em>AGAMOUS-LIKE 87</em>/<em>AGL87</em>). Within a <em>tr–d</em> event, the original donor copy (CL copy) is reciprocally co-linear to orthologues in other genomes (Orthologs), while transposed and duplicated paralogues (Tr copies) are not. Nodes and edges are as described in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. (B) Comparison between the ORF size of all loci involved in a <em>tr–d</em> event (i.e. both CL and Tr copies) with the median ORF size of Orthologs. Blue (filled) dots indicate CL copies. (C and D) ORF size comparison between Tr copies and their corresponding CL copy within each of the <em>tr–d</em> events, as a scatterplot (C) and a histogram of ORF size ratio (D). Pink shades indicate Tr copies with complete ORFs whose sizes are comparable (±20% in proportion) to that of the CL copy. Panel D also shows numbers and percentages (in parentheses) of Tr copies with complete ORFs below the species labels. The entire list of OrthNets showing lineage-specific <em>tr–d</em>, including CL and Tr copies, is in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3.</p></div>
</div>                        <a id="131979939" scrollto-destination="131979939"></a>
<div id="dsy035-F5" data-id="dsy035-F5" class="fig fig-section">
<div class="label fig-label">Figure 5.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f5.png?Expires=1576277252&amp;Signature=amXGt0Oa1cRgEGZfeLLU4HbVu7Cu9znAzKANoxkobDKKsmZHaNsxliK~WbrfIBAIeyji8qKpfVdL1kWuQvYJi~B0pUxj-UJCQ3XHZfAOIn4YyxmxezGyL3myupXD0Bmk1tiOgQC9qVGCGVnEl3mLk5VLbiSDpe80d8FW3-VTpC7aDNAKwZkPRsrcKN68Vio6ceJKz-GTMsOSrmRHoKdlb0J14o70jYabv4vhofjG-iWp7EjKa8yufsY9p9DPWgKuQdWRszHW7sHuI4ROkEhIQ4h2jirez9XqCiXOCSkR6SX--vzYVy-GeJBISam9q7RC~kRRUwjc68jLaL8Wpv8iTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transposition–duplication (tr–d) events shared by a pair of Brassicaceae genomes. (A) Example OrthNets with tr–d events shared by E. salsugineum and S. parvula, representing two categories: (i) independent-parallel (‘Ind-parallel’) tr–d events and (ii) tr–d events with Tr copies co-linear to each other (‘Tr–cl’). Nodes and edges are as described in Fig. 3A. (B) Number of OrthNets shared by pairs of genomes in ‘Ind-parallel’ and ‘Tr–cl’ categories, with the cladogram of the six crucifer genomes on the top. Heatmap colours visualize the rank of each cell based on the number of OrthNets in each category. (C) Proportion of Tr copies with complete ORFs (i.e. ORF size ±20% of the CL copy in proportion) within OrthNets presented in (B). The genome 1 (row)-genome 2 (column) position shows the number of Tr copies with complete ORF/total Tr copies in the genome 1, found in all OrthNets with tr–d shared by genomes 1 and 2. For example, the Aly–Ath and Ath–Aly positions in ‘Ind-parallel’ category indicate 6 out of 29 A. lyrata (Aly) Tr copies and 6 out of 22 A. thaliana (Ath) Tr copies, respectively, have complete ORFs in the 17 OrthNets with ‘Ind-parallel’ tr–d events shared by A. lyrata and A. thaliana. Heatmap colours indicate the percentage of Tr copies with complete ORFs, for cells with &gt;5 total Tr copies." path-from-xml="dsy035f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979939/dsy035f5.tif" path-from-xml="dsy035f5.tif" target="_blank">Open in new tab</a><a section="131979939" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f5.png?Expires=1576277252&amp;Signature=1VHXFdZoSicQjxGu18x1424hRpwYqUEtLOHIz0RznGJMcqLCykfcEzQaoqIarHY5m8vQC27F3gK5w98CnAwbG6aaoTAOe5fNG~5IBH4ixsZta1l8yXtafAi1X9gEc-oc19hrKG7OIcKD05G-qqMP1A4w1xfLDjafQxcpvyWTum-fj3SP8i0sIovSNP6OaGvOpK1tpzI3RgbWV1xaRKsQAZH6pnZ~Wq5nb-kx4Ps-DwRjs5ydv7vtNsPfC6nKjjXZpqXNxMxi3BABRIi3qa5Dfu00D8sJafDvzOfzKasGy6UJPBW-QlMekqOC4~eSQkr5xs43R7g23wylIp3JzJjLSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979939&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Transposition–duplication (<em>tr–d</em>) events shared by a pair of Brassicaceae genomes. (A) Example OrthNets with <em>tr–d</em> events shared by <em>E. salsugineum</em> and <em>S. parvula</em>, representing two categories: (i) independent-parallel (‘Ind-parallel’) <em>tr–d</em> events and (ii) <em>tr–d</em> events with Tr copies co-linear to each other (‘Tr–<em>cl</em>’). Nodes and edges are as described in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. (B) Number of OrthNets shared by pairs of genomes in ‘Ind-parallel’ and ‘Tr–<em>cl</em>’ categories, with the cladogram of the six crucifer genomes on the top. Heatmap colours visualize the rank of each cell based on the number of OrthNets in each category. (C) Proportion of Tr copies with complete ORFs (i.e. ORF size ±20% of the CL copy in proportion) within OrthNets presented in (B). The genome 1 (row)-genome 2 (column) position shows the number of Tr copies with complete ORF/total Tr copies in the genome 1, found in all OrthNets with <em>tr–d</em> shared by genomes 1 and 2. For example, the Aly–Ath and Ath–Aly positions in ‘Ind-parallel’ category indicate 6 out of 29 <em>A. lyrata</em> (Aly) Tr copies and 6 out of 22 <em>A. thaliana</em> (Ath) Tr copies, respectively, have complete ORFs in the 17 OrthNets with ‘Ind-parallel’ <em>tr–d</em> events shared by <em>A. lyrata</em> and <em>A. thaliana</em>. Heatmap colours indicate the percentage of Tr copies with complete ORFs, for cells with &gt;5 total Tr copies.</p></div>
</div>
<div content-id="dsy035-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f5.png?Expires=1576277252&amp;Signature=amXGt0Oa1cRgEGZfeLLU4HbVu7Cu9znAzKANoxkobDKKsmZHaNsxliK~WbrfIBAIeyji8qKpfVdL1kWuQvYJi~B0pUxj-UJCQ3XHZfAOIn4YyxmxezGyL3myupXD0Bmk1tiOgQC9qVGCGVnEl3mLk5VLbiSDpe80d8FW3-VTpC7aDNAKwZkPRsrcKN68Vio6ceJKz-GTMsOSrmRHoKdlb0J14o70jYabv4vhofjG-iWp7EjKa8yufsY9p9DPWgKuQdWRszHW7sHuI4ROkEhIQ4h2jirez9XqCiXOCSkR6SX--vzYVy-GeJBISam9q7RC~kRRUwjc68jLaL8Wpv8iTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Transposition–duplication (tr–d) events shared by a pair of Brassicaceae genomes. (A) Example OrthNets with tr–d events shared by E. salsugineum and S. parvula, representing two categories: (i) independent-parallel (‘Ind-parallel’) tr–d events and (ii) tr–d events with Tr copies co-linear to each other (‘Tr–cl’). Nodes and edges are as described in Fig. 3A. (B) Number of OrthNets shared by pairs of genomes in ‘Ind-parallel’ and ‘Tr–cl’ categories, with the cladogram of the six crucifer genomes on the top. Heatmap colours visualize the rank of each cell based on the number of OrthNets in each category. (C) Proportion of Tr copies with complete ORFs (i.e. ORF size ±20% of the CL copy in proportion) within OrthNets presented in (B). The genome 1 (row)-genome 2 (column) position shows the number of Tr copies with complete ORF/total Tr copies in the genome 1, found in all OrthNets with tr–d shared by genomes 1 and 2. For example, the Aly–Ath and Ath–Aly positions in ‘Ind-parallel’ category indicate 6 out of 29 A. lyrata (Aly) Tr copies and 6 out of 22 A. thaliana (Ath) Tr copies, respectively, have complete ORFs in the 17 OrthNets with ‘Ind-parallel’ tr–d events shared by A. lyrata and A. thaliana. Heatmap colours indicate the percentage of Tr copies with complete ORFs, for cells with &gt;5 total Tr copies." path-from-xml="dsy035f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979939/dsy035f5.tif" path-from-xml="dsy035f5.tif" target="_blank">Open in new tab</a><a section="131979939" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f5.png?Expires=1576277252&amp;Signature=1VHXFdZoSicQjxGu18x1424hRpwYqUEtLOHIz0RznGJMcqLCykfcEzQaoqIarHY5m8vQC27F3gK5w98CnAwbG6aaoTAOe5fNG~5IBH4ixsZta1l8yXtafAi1X9gEc-oc19hrKG7OIcKD05G-qqMP1A4w1xfLDjafQxcpvyWTum-fj3SP8i0sIovSNP6OaGvOpK1tpzI3RgbWV1xaRKsQAZH6pnZ~Wq5nb-kx4Ps-DwRjs5ydv7vtNsPfC6nKjjXZpqXNxMxi3BABRIi3qa5Dfu00D8sJafDvzOfzKasGy6UJPBW-QlMekqOC4~eSQkr5xs43R7g23wylIp3JzJjLSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979939&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Transposition–duplication (<em>tr–d</em>) events shared by a pair of Brassicaceae genomes. (A) Example OrthNets with <em>tr–d</em> events shared by <em>E. salsugineum</em> and <em>S. parvula</em>, representing two categories: (i) independent-parallel (‘Ind-parallel’) <em>tr–d</em> events and (ii) <em>tr–d</em> events with Tr copies co-linear to each other (‘Tr–<em>cl</em>’). Nodes and edges are as described in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. (B) Number of OrthNets shared by pairs of genomes in ‘Ind-parallel’ and ‘Tr–<em>cl</em>’ categories, with the cladogram of the six crucifer genomes on the top. Heatmap colours visualize the rank of each cell based on the number of OrthNets in each category. (C) Proportion of Tr copies with complete ORFs (i.e. ORF size ±20% of the CL copy in proportion) within OrthNets presented in (B). The genome 1 (row)-genome 2 (column) position shows the number of Tr copies with complete ORF/total Tr copies in the genome 1, found in all OrthNets with <em>tr–d</em> shared by genomes 1 and 2. For example, the Aly–Ath and Ath–Aly positions in ‘Ind-parallel’ category indicate 6 out of 29 <em>A. lyrata</em> (Aly) Tr copies and 6 out of 22 <em>A. thaliana</em> (Ath) Tr copies, respectively, have complete ORFs in the 17 OrthNets with ‘Ind-parallel’ <em>tr–d</em> events shared by <em>A. lyrata</em> and <em>A. thaliana</em>. Heatmap colours indicate the percentage of Tr copies with complete ORFs, for cells with &gt;5 total Tr copies.</p></div>
</div>
<p>In OrthNets including <em>tr–d</em> events, we identified the original donor or co-linear (CL) copy, or copies if the donor locus included tandem duplications, and the acceptor or transposed (Tr) copies, based on properties of edges connecting each of the duplicated paralogues to its neighbouring nodes in other genomes (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Figs 4A and 5A</a>). <a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Figure 4A</a> represents OrthNets with <em>S. parvula</em> and <em>C. rubella</em> lineage-specific <em>tr–d</em> events for orthologues of the <em>WRKY72</em> and <em>AGL87</em>, respectively. The CL copy (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, ‘CL copy’) was a part of the hexagon and mostly reciprocally co-linear to its orthologue nodes (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, ‘Orthologs’) from other genomes. A Tr copy was connected to orthologue nodes in the hexagon through unidirectional <em>tr</em> edges (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, ‘Tr copies’). An OrthNet may contain a single lineage-specific <em>tr–d</em> event as in the OrthNet for <em>WRKY72</em> (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, left) or multiple events featuring one CL copy associated with multiple Tr copies. Also, Tr copies may further undergo tandem duplication as shown in the OrthNet for <em>AGL87</em> (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, right).</p>
<p>We compared the ORF sizes between CL and Tr copies with the median ORF size of the orthologues from other genomes in the hexagon for all OrthNets representing lineage-specific <em>tr–d</em> events. We observed a conservation of ORF sizes between most CL copies and their co-linear orthologues (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, blue/filled dots), while the majority of Tr copies had truncated ORFs (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, grey dots). We also found a small proportion of Tr copies which had ORFs that were of similar size to their respective CL copy (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a>, pink shades). The distribution of the ORF size ratio between Tr and the CL copy showed peaks at 80–120% (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>, pink shades). These Tr copies that showed conservation in maintaining the original size of the ORFs were more abundant in <em>A. thaliana</em>, <em>C. rubella</em>, <em>E. salsugineum</em>, and <em>S. parvula</em>. These contributed to more than 24% of all Tr copies found in lineage-specific <em>tr–d</em> events in these genomes (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>).</p>
<p>For <em>tr–d</em> events shared between any pair within the six genomes, we identified two categories with different evolutionary contexts: (i) parallel <em>tr–d</em> events independently occuring in two genomes (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, ‘Ind-parallel’) and (ii) <em>tr–d</em> events where Tr copies from two genomes showing co-linearity between them (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, ‘Tr–<em>cl</em>’). <a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Figure 5A</a> depicts examples of OrthNets including <em>tr–d</em> events in the two categories. We found a total of seven and six OrthNets with <em>tr–d</em> events in ‘Ind-parallel’ and ‘Tr–<em>cl</em>’ categories, respectively, shared between <em>E. salsugineum</em> and <em>S. parvula</em>. Genomes with higher TE and repetitive sequence contents, such as <em>A. lyrata</em>, <em>S. irio</em>, and, to a lesser extent, <em>E. salsugineum</em>, included more ‘ind-parallel’ <em>tr–d</em> events shared with other genomes (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>, left panel). Among Tr copies in ‘ind-parallel’ <em>tr–d</em> events, the proportion of complete ORFs (i.e. ORF size within ±20% of the ORF of the corresponding CL copy) were comparable with Tr copies in LS <em>tr–d</em> events (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>, left panel and <a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>).</p>
<p>The ‘Tr–<em>cl</em>’ type <em>tr–d</em> events were mostly found between pairs of more recently diverged genomes, e.g. <em>A. lyrata</em>–<em>A. thaliana</em> and <em>S. irio</em>–<em>S. parvula</em>. The number of ‘Tr–<em>cl</em>’ <em>tr–d</em> events detected between Lineages I and II genomes were very low (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>, right panel). This observation was consistent with the notion that such a rare event must involve a <em>tr–d</em> event before the divergence of the two Lineages followed by deletions in all species except for the two genomes compared. The proportion of Tr copies that retained complete ORFs compared with the CL copy in ‘Tr–<em>cl</em>’ type <em>tr–d</em> events was higher (≥50%) than that found for ‘ind-parallel’ type <em>tr–d</em> events (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>).</p>                    <h3 scrollto-destination="131979944" id="131979944" class="section-title" data-legacy-id="SEC3.5">3.5. Tr copies with complete ORFs were rare, but significantly more frequent than random chance</h3>
<p>We hypothesized that selection has favoured conservation of beneficial Tr copies to preserve the ORF in additional gene copies (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a>, pink shades; <a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>), while majority of the Tr copies was either originally duplicated incompletely or have undergone mutations over time that had led to truncated ORFs. An alternative hypothesis is that these Tr copies with complete ORFs may have been easier to duplicate in their complete form by random chance due to their smaller gene size. Indeed, genes associated with Tr copies with complete ORFs were significantly shorter than those with Tr copies that had truncated ORFs (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). </p>                        <a id="131979946" scrollto-destination="131979946"></a>
<div id="dsy035-F6" data-id="dsy035-F6" class="fig fig-section">
<div class="label fig-label">Figure 6.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f6.png?Expires=1576277252&amp;Signature=ArTV0-ZuZROjWNU1ni2rr8MMSCisL4iG0vK9kYbnqF-YqJWv2jaES6nwCwq8zoYGE3FFp6KMaltPJoD7NEhld5ocYg2-qvDd71yvQZCjvF~pfhzlSD8J4J0HkBsR5RzmrT4378ZKliHBzW58fD25aNFlt70W54yFEau~f~kyY-OvcxAp8SWorzzduiTOb4FSP2E7nJ4f6Cwpche-xeEwR9QhXiE6N1SzRCer0psnt5Ms2htLZw~qaoD6XdzGIdUprSNZnQCxiugQSO95cRb3M~HVr8xiKGu8c3m7uqXv4XICsWxineBPtuJCoHhmAWHi6zdLMvlxjELZaU86sT8PDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of duplicated genomic regions in transposition–duplication (tr–d) events. (A) We identified Homologous Genome Segments (HGSs) between the CL and Tr copy genes and adjacent genomic regions (±5 kb) in a tr–d event as described in Materials and Methods (2.5). A tr–d event is either complete or incomplete, based on whether the HGS included the full CL copy gene or not. A subset of complete tr–d events had HGS coinciding with the start and end of the CL copy gene without extending to the intergenic regions (‘gene-only’ tr–d). (B) Histograms and box-and-whisker plots (inlets) showing size distributions of HGSs and CL copy genes for complete and incomplete tr–d events. (C) Comparison of observed (Obs.) and expected (Exp.) occurrences of complete and gene-only tr–d events. Upper panel shows the distribution of expected occurrences from 10,000 random shuffling of HGSs and CL copy genes for A. lyrata (Aly)-specific complete and gene-only tr–d events. Fitting the random shuffling results to normal distributions (upper panel, blue curves) generated P-values of observed occurrences for tr–d events unique to each genome and the genus Arabidopsis (lower panel). (D) Four degenerate site (4d) substitution rates between ORFs of CL and Tr copy genes in different types of complete tr–d events. Complete tr–d events were either gene-only (‘go’) or with HGSs detected in both gene and intergenic regions (‘g+i’). We compared lineage-specific (LS) and shared tr–d events that are either independent-parallel (‘Ind-par’) or with Tr copies co-linear to each other (‘Tr–cl’). Lines and ‘x’ marks in the box indicate medians and means, respectively. (E) Proportion of Tr copy genes with expression evidences (RNA-seq FPKM&gt;0) in all tr–d events either lineage-specific or shared by a pair of genomes. The tr–d type is as described in (A) and Tr copy ORF type is as in Fig. 4C and D (pink shade) and Fig. 6C. S. irio genes were excluded due to the lack of RNA-seq data." path-from-xml="dsy035f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979946/dsy035f6.tif" path-from-xml="dsy035f6.tif" target="_blank">Open in new tab</a><a section="131979946" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f6.png?Expires=1576277252&amp;Signature=nw-8Qr9J0yHo-68AkZkHEVV0TSTqi~dBraIGsWsB5~Afr0QH5eHgQdKRJhZQtYrhqFAWCY7qcpb31qAJsKDY1D11kNwMUR1xRSiOf505gLTDylH7kOr7SCRyo4Ycb8NwsbLN7eCjiMpo0dEL17BuBL~HnYtD6-q2yw8d6UKf6D-u0F9QIYmMSTC4K~TB7RHWuJjGgKZmjtKW4APWIiFT-f4JslGQMPbOqAottSUnIkbFV1hTzSCHj3mQ3u69UlPVKT0dAYuK4LFpq6s3yPEAQv01a9Of9xow0ziodiMSk-qKvU5wrmRmlK2TFVuaVgap8QzubAO5I3AES2cds7SkFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979946&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of duplicated genomic regions in transposition–duplication (<em>tr–d</em>) events. (A) We identified Homologous Genome Segments (HGSs) between the CL and Tr copy genes and adjacent genomic regions (±5 kb) in a <em>tr–d</em> event as described in Materials and Methods (2.5). A <em>tr–d</em> event is either complete or incomplete, based on whether the HGS included the full CL copy gene or not. A subset of complete <em>tr–d</em> events had HGS coinciding with the start and end of the CL copy gene without extending to the intergenic regions (‘gene-only’ <em>tr–d</em>). (B) Histograms and box-and-whisker plots (inlets) showing size distributions of HGSs and CL copy genes for complete and incomplete <em>tr–d</em> events. (C) Comparison of observed (Obs.) and expected (Exp.) occurrences of complete and gene-only <em>tr–d</em> events. Upper panel shows the distribution of expected occurrences from 10,000 random shuffling of HGSs and CL copy genes for <em>A. lyrata</em> (Aly)-specific complete and gene-only <em>tr–d</em> events. Fitting the random shuffling results to normal distributions (upper panel, blue curves) generated <em>P</em>-values of observed occurrences for <em>tr–d</em> events unique to each genome and the genus <em>Arabidopsis</em> (lower panel). (D) Four degenerate site (4d) substitution rates between ORFs of CL and Tr copy genes in different types of complete <em>tr–d</em> events. Complete <em>tr–d</em> events were either gene-only (‘go’) or with HGSs detected in both gene and intergenic regions (‘g+i’). We compared lineage-specific (LS) and shared <em>tr–d</em> events that are either independent-parallel (‘Ind-par’) or with Tr copies co-linear to each other (‘Tr–<em>cl</em>’). Lines and ‘x’ marks in the box indicate medians and means, respectively. (E) Proportion of Tr copy genes with expression evidences (RNA-seq FPKM&gt;0) in all <em>tr–d</em> events either lineage-specific or shared by a pair of genomes. The <em>tr–d</em> type is as described in (A) and Tr copy ORF type is as in <a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a> (pink shade) and <a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>. <em>S. irio</em> genes were excluded due to the lack of RNA-seq data.</p></div>
</div>
<div content-id="dsy035-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f6.png?Expires=1576277252&amp;Signature=ArTV0-ZuZROjWNU1ni2rr8MMSCisL4iG0vK9kYbnqF-YqJWv2jaES6nwCwq8zoYGE3FFp6KMaltPJoD7NEhld5ocYg2-qvDd71yvQZCjvF~pfhzlSD8J4J0HkBsR5RzmrT4378ZKliHBzW58fD25aNFlt70W54yFEau~f~kyY-OvcxAp8SWorzzduiTOb4FSP2E7nJ4f6Cwpche-xeEwR9QhXiE6N1SzRCer0psnt5Ms2htLZw~qaoD6XdzGIdUprSNZnQCxiugQSO95cRb3M~HVr8xiKGu8c3m7uqXv4XICsWxineBPtuJCoHhmAWHi6zdLMvlxjELZaU86sT8PDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of duplicated genomic regions in transposition–duplication (tr–d) events. (A) We identified Homologous Genome Segments (HGSs) between the CL and Tr copy genes and adjacent genomic regions (±5 kb) in a tr–d event as described in Materials and Methods (2.5). A tr–d event is either complete or incomplete, based on whether the HGS included the full CL copy gene or not. A subset of complete tr–d events had HGS coinciding with the start and end of the CL copy gene without extending to the intergenic regions (‘gene-only’ tr–d). (B) Histograms and box-and-whisker plots (inlets) showing size distributions of HGSs and CL copy genes for complete and incomplete tr–d events. (C) Comparison of observed (Obs.) and expected (Exp.) occurrences of complete and gene-only tr–d events. Upper panel shows the distribution of expected occurrences from 10,000 random shuffling of HGSs and CL copy genes for A. lyrata (Aly)-specific complete and gene-only tr–d events. Fitting the random shuffling results to normal distributions (upper panel, blue curves) generated P-values of observed occurrences for tr–d events unique to each genome and the genus Arabidopsis (lower panel). (D) Four degenerate site (4d) substitution rates between ORFs of CL and Tr copy genes in different types of complete tr–d events. Complete tr–d events were either gene-only (‘go’) or with HGSs detected in both gene and intergenic regions (‘g+i’). We compared lineage-specific (LS) and shared tr–d events that are either independent-parallel (‘Ind-par’) or with Tr copies co-linear to each other (‘Tr–cl’). Lines and ‘x’ marks in the box indicate medians and means, respectively. (E) Proportion of Tr copy genes with expression evidences (RNA-seq FPKM&gt;0) in all tr–d events either lineage-specific or shared by a pair of genomes. The tr–d type is as described in (A) and Tr copy ORF type is as in Fig. 4C and D (pink shade) and Fig. 6C. S. irio genes were excluded due to the lack of RNA-seq data." path-from-xml="dsy035f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979946/dsy035f6.tif" path-from-xml="dsy035f6.tif" target="_blank">Open in new tab</a><a section="131979946" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f6.png?Expires=1576277252&amp;Signature=nw-8Qr9J0yHo-68AkZkHEVV0TSTqi~dBraIGsWsB5~Afr0QH5eHgQdKRJhZQtYrhqFAWCY7qcpb31qAJsKDY1D11kNwMUR1xRSiOf505gLTDylH7kOr7SCRyo4Ycb8NwsbLN7eCjiMpo0dEL17BuBL~HnYtD6-q2yw8d6UKf6D-u0F9QIYmMSTC4K~TB7RHWuJjGgKZmjtKW4APWIiFT-f4JslGQMPbOqAottSUnIkbFV1hTzSCHj3mQ3u69UlPVKT0dAYuK4LFpq6s3yPEAQv01a9Of9xow0ziodiMSk-qKvU5wrmRmlK2TFVuaVgap8QzubAO5I3AES2cds7SkFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979946&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characterization of duplicated genomic regions in transposition–duplication (<em>tr–d</em>) events. (A) We identified Homologous Genome Segments (HGSs) between the CL and Tr copy genes and adjacent genomic regions (±5 kb) in a <em>tr–d</em> event as described in Materials and Methods (2.5). A <em>tr–d</em> event is either complete or incomplete, based on whether the HGS included the full CL copy gene or not. A subset of complete <em>tr–d</em> events had HGS coinciding with the start and end of the CL copy gene without extending to the intergenic regions (‘gene-only’ <em>tr–d</em>). (B) Histograms and box-and-whisker plots (inlets) showing size distributions of HGSs and CL copy genes for complete and incomplete <em>tr–d</em> events. (C) Comparison of observed (Obs.) and expected (Exp.) occurrences of complete and gene-only <em>tr–d</em> events. Upper panel shows the distribution of expected occurrences from 10,000 random shuffling of HGSs and CL copy genes for <em>A. lyrata</em> (Aly)-specific complete and gene-only <em>tr–d</em> events. Fitting the random shuffling results to normal distributions (upper panel, blue curves) generated <em>P</em>-values of observed occurrences for <em>tr–d</em> events unique to each genome and the genus <em>Arabidopsis</em> (lower panel). (D) Four degenerate site (4d) substitution rates between ORFs of CL and Tr copy genes in different types of complete <em>tr–d</em> events. Complete <em>tr–d</em> events were either gene-only (‘go’) or with HGSs detected in both gene and intergenic regions (‘g+i’). We compared lineage-specific (LS) and shared <em>tr–d</em> events that are either independent-parallel (‘Ind-par’) or with Tr copies co-linear to each other (‘Tr–<em>cl</em>’). Lines and ‘x’ marks in the box indicate medians and means, respectively. (E) Proportion of Tr copy genes with expression evidences (RNA-seq FPKM&gt;0) in all <em>tr–d</em> events either lineage-specific or shared by a pair of genomes. The <em>tr–d</em> type is as described in (A) and Tr copy ORF type is as in <a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a> (pink shade) and <a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>. <em>S. irio</em> genes were excluded due to the lack of RNA-seq data.</p></div>
</div>
<p>To test our hypotheses, we shuffled duplicated genomic regions and duplicated genes in <em>tr–d</em> events. Then, we compared the occurrences of randomized <em>tr–d</em> events showing complete duplication of the entire CL copy gene with those observed among actual <em>tr–d</em> events (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). First, to detect duplicated genomic regions in a <em>tr–d</em> event, we compared adjacent genomic regions, i.e. 5 kb up- and downstream regions, including the gene, of the CL copy and each of Tr copies. In this comparison, we searched for Homologous Genome Segments (HGSs) between the CL and Tr copy loci. As depicted in <a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>, an incomplete <em>tr–d</em> event results in a HGS carrying only a part of the CL copy gene (HGS ⊅ CL copy), while in a complete <em>tr–d</em>, the HGS encompasses the entire CL copy gene (HGS ⊇ CL copy). Interestingly, we found a subset of complete <em>tr–d</em> events where the start and end positions of the HGS appeared to overlap with the start and end of the CL copy gene (HGS ≈ CL copy). We named this subset ‘gene-only’ <em>tr–d</em> (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>) since the sequence homology was not detectable in the intergenic region further from the CL copy coding regions by more than 20% of the CL copy coding region size. A total of 224 complete <em>tr–d</em> events showed a shift towards longer HGSs, while their CL copy genes (coding regions) were significantly shorter (<em>P</em> &lt; 0.001, two-tailed <em>t</em>-test), compared with those in the 1,166 incomplete <em>tr–d</em> events (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>).</p>
<p>Following random shuffling of all HSGs and CL copy genes as described in Materials and Methods (2.5), we counted the occurrences of incomplete, complete, and gene-only <em>tr–d</em> events for each iteration. <a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Figure 6C</a> shows the comparison between the observed and expected occurrences of complete and gene-only <em>tr–d</em> events, where expected values were the mean values from 10,000 iterations. Assuming a normal distribution for the expected values, we estimated the <em>P</em>-value for the observed numbers of complete and gene-only LS <em>tr–d</em> events for each genome. Both complete and gene-only <em>tr–d</em> events were much more frequent than expected due to random chance. The gene-only <em>tr–d</em> events had smaller <em>P</em>-values than complete <em>tr–d</em> events in all categories tested except in <em>A. thaliana</em> lineage-specific <em>tr–d</em> events (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>, table in the lower panel). We observe a smaller number of lineage-specific <em>tr–d</em> events in <em>A. thaliana</em> than in any other target genome. This may be a result of <em>A. thaliana</em> and <em>A. lyrata</em> being the closest among all pairs, included in the same genus. Hence, we included the <em>Arabidopsis</em> genus-specific <em>tr–d</em> events into consideration, which led to numbers and enrichment of complete and gene-only <em>tr-d</em> events comparable to other genomes (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>, ‘Aly–Ath’).</p>
<p>Random occurrence of duplications cannot explain the observed proportion of complete <em>tr–d</em> events, which in &gt;90% of the cases also resulted in complete ORFs in the Tr copy loci (e.g. <a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a>, pink shades). More likely, the observed proportion of complete and gene-only duplications was the sequential result of random duplications and selective retention of beneficial coding regions over time. This explanation is consistent with 4d substitution rates between complete ORFs of Tr copies and CL copies in <em>tr–d</em> events. Higher 4d substitution rates, as a proxy for older duplications, were found between ORFs of Tr and CL copy pairs in gene-only <em>tr–d</em> events (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>, ‘go’). This was contrasting to <em>tr–d</em> events where HGSs comprised both gene and intergenic regions (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>, ‘g + i’), for both lineage-specific (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>, ‘LS’) and indepedent parallel (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>, ‘Ind-par’) shared <em>tr–d</em> events. The 4d substitution rates associated with ‘Tr–<em>cl</em>’ type shared <em>tr–d</em> events (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>, ‘Tr–<em>cl</em>’) showed median values comparable or higher than the median 4d substitution rates that represent the divergence between <em>A. thaliana</em> and Lineage II genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1D</a></span>). This further agreed with the notion that a ‘Tr–<em>cl</em>’ type <em>tr–d</em> event was derived from duplications dated prior to the divergence of genomes that shared the events.</p>
<p>Complete <em>tr–d</em> events also included a higher number of Tr copy genes that showed evidence of expression compared with incomplete <em>tr–d</em> events (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6E</a>). Incomplete <em>tr–d</em> was associated with most of the Tr copies with truncated ORFs, which comprised the majority of Tr copies in both lineage-specific <em>tr–d</em> (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a>) and independent parallel <em>tr–d</em> events shared by a pair of genomes (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). Out of total 1,706 Tr copies with truncated ORFs, only 135 were derived from complete <em>tr–d</em> events, in which the ORFs were most likely truncated by null mutations after the duplication (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6E</a>). We found no enrichment of single exon genes, a signature of retrotransposons, among <em>tr–d</em> events (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>).</p>                    <h3 scrollto-destination="131979951" id="131979951" class="section-title" data-legacy-id="SEC3.6">3.6. Genes associated with lineage-specific and shared <em>tr–d</em> events</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> presents a partial list of OrthNets associated with lineage-specific <em>tr–d</em> events for each of the six Brassicaceae genomes, selected based on the most number of Tr copies with complete ORFs and expression evidences, except for <em>S. irio</em>, for which RNAseq data was not available. For each OrthNet listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>, we included numbers of Tr copies tandem duplicated, with complete ORFs, and with expression evidences, as detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S3</a></span>. The complete list of OrthNets including lineage-specific <em>tr–d</em> events is available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3. We described genes and gene ontology terms enriched among them in lineage-specific <em>tr–d</em> events in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S3</a></span> and Dataset S4.</p>
<p>We selected the largest OrthNets with <em>E. salsugineum</em>-specific <em>tr–d</em> events (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>) and independently visualized the extent of gene duplications using the GEvo tool in the CoGE database<a reveal-id="dsy035-B35" data-open="dsy035-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). The <em>E. salsugineum</em> genome included six copies of <em>SALT TOLERANCE 32</em> (<em>SAT32</em>), which exists as a single copy in each of the other Brassicaceae genomes. Among five Tr copies detected for <em>EsSAT32</em>, we found three tandem duplicates (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a>, ‘Tr copies’). Four of the Tr copies had complete ORFs and three of them showed expression in either root or shoot tissues (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The GEvo plot illustrates extensive sequence similarity among all loci and adjacent genomic regions that are reciprocally co-linear among them (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>, <em>AtSAT32</em>, <em>SpSAT32</em>, and <em>EsSAT32;1</em>). Similar patterns were observed in comparisons with the <em>S. irio</em>, <em>C. rubella</em>, and <em>A. lyrata</em> co-linear orthologues (data not shown). The <em>EsSAT32</em> Tr copies (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>, <em>EsSAT32;2/3/4/5</em>) represented examples of gene-only <em>tr–d</em> events (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a>), where sequence similarities were restricted to the expected border regions of the gene models (i.e. ±20% of the coding region size). Interestingly, the Tr copies <em>EsSAT32;3/4/5</em> also exhibited intron losses, resulting in 9, 1, and 1 exons, respectively, compared with the 13 exons in the CL copy <em>EsSAT32;1</em> (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>), while maintaining high deduced amino acid similarities over most of the coding region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Among all <em>EsSAT</em> paralogues, the highest average expression was observed for one of the Tr copies, <em>EsSAT32;2</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>, OrthNet ID ON_2516 and gene ID 20188564). </p>                        <a id="131979954" scrollto-destination="131979954"></a>
<div id="dsy035-F7" data-id="dsy035-F7" class="fig fig-section">
<div class="label fig-label">Figure 7.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f7.png?Expires=1576277252&amp;Signature=Ss2WtEVdEJNweU5SaUSLpmqOd2z~XVFIRc9khrGNIfyIbIrLXtTSK2-UENxpaJJcUb8vc-NTYw9hV-0rt0UkPr5qsgpKiA-DVPZDkhQks7fG5XsSg4gUj6n4R9UhtA-IcjDqZOva1YaZd0HpbZeMdMQFefcaq1OQkYJZTdl9wJq8NvDhMM3uSGnuI366s3mtNSASnmmZsu19gB4c8Mr8nYYxJfyq7mg09ivqogE7OJhx2rN0ms9neJdOvPQC~S-C7JkHuW6hSrHr60Qna6sJJx7h0gN9PCXBjDWTaf6RwStO9x47jjIUJwCgEV-g1UPWYry0uK7fzL13bDwCNIfIVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An exemplary OrthNet showing E. salsugineum lineage-specific tr–d events. (A) OrthNet for the SALT-TOLERANCEE 32 (SAT32). Nodes and edges are as described in Fig. 3A. The OrthNet showed E. salsugineum-specific tr–d, with three of the five Tr copies tandem duplicated. (B) Comparison of SAT32 loci and adjacent ±30 kb genomic regions between A. thaliana, S. parvula, and E. salsugineum as a GEvo plot (https://genomevolution.org/r/maxx (10 October 2018, date last accessed)). Pink/semi-transparent ribbons connect Homologous Genomic Segments (HGSs) between genomes, while gene models, mRNAs, and coding sequences are depicted as cylinders underneath HGSs (for detailed legends, see https://genomevolution.org/wiki/index.php/GEvo (10 October 2018, date last accessed)). EsSAT32;1 is the CL copy (Esa|20186362), while EsSAT32;2 (Esa|20188564), EsSAT32;3 (Esa|20189286), EsSAT32;4 (Esa|20179565), and EsSAT32;5 (Esa|20180739) indicate the four Tr copies with complete ORFs. The yellow arrow marks the position of Esa|20181274, which contains a truncated ORF. EsSAT32;3/4/5 showed intron losses without compromising gene products (see text and Supplementary Fig. S9 for details)." path-from-xml="dsy035f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979954/dsy035f7.tif" path-from-xml="dsy035f7.tif" target="_blank">Open in new tab</a><a section="131979954" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f7.png?Expires=1576277252&amp;Signature=wtmMwjNaFc7a2dtMmB62mA8eaPp5ZK9dMXEpqmUCrgV0icAK5CkzFPt8cPKQMIkRjpneRXHC1KqBMWP04UPyaSQiky9AVXYM4IiNT5~Y6Q2G8iRSJFuWFDuv~uzP~07152oCKzLKmtMxhkiUwa~QBEqDwkBuv-A1TRV3-jwp-mQ1P3fA5HXVrm2DvkSwK5sXdP9By9PXpxVfGkXOdYo-THowvkP09KRm7Tc8C6vSJ-IJXw-LHtNIrZbeHQSP3Bq0E6sayHO7pTljTk8~odMiLrL-vEBV~rhPNcZUNvSawXrScE39W3u8neGlALcAV6ohHKR1kKlunwJLPWQ6Pf9HJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979954&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>An exemplary OrthNet showing <em>E. salsugineum</em> lineage-specific <em>tr–d</em> events. (A) OrthNet for the <em>SALT-TOLERANCEE 32</em> (<em>SAT32</em>). Nodes and edges are as described in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. The OrthNet showed <em>E. salsugineum</em>-specific <em>tr–d</em>, with three of the five Tr copies tandem duplicated. (B) Comparison of <em>SAT32</em> loci and adjacent ±30 kb genomic regions between <em>A. thaliana</em>, <em>S. parvula</em>, and <em>E. salsugineum</em> as a GEvo plot (<a class="link link-uri" href="https://genomevolution.org/r/maxx" target="_blank">https://genomevolution.org/r/maxx</a> (10 October 2018, date last accessed)). Pink/semi-transparent ribbons connect Homologous Genomic Segments (HGSs) between genomes, while gene models, mRNAs, and coding sequences are depicted as cylinders underneath HGSs (for detailed legends, see <a class="link link-uri" href="https://genomevolution.org/wiki/index.php/GEvo" target="_blank">https://genomevolution.org/wiki/index.php/GEvo</a> (10 October 2018, date last accessed)). <em>EsSAT32;1</em> is the CL copy (Esa|20186362), while <em>EsSAT32;2</em> (Esa|20188564), <em>EsSAT32;3</em> (Esa|20189286), <em>EsSAT32;4</em> (Esa|20179565), and <em>EsSAT32;5</em> (Esa|20180739) indicate the four Tr copies with complete ORFs. The yellow arrow marks the position of Esa|20181274, which contains a truncated ORF. <em>EsSAT32;3/4/5</em> showed intron losses without compromising gene products (see text and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span> for details).</p></div>
</div>
<div content-id="dsy035-F7" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 7.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/m_dsy035f7.png?Expires=1576277252&amp;Signature=Ss2WtEVdEJNweU5SaUSLpmqOd2z~XVFIRc9khrGNIfyIbIrLXtTSK2-UENxpaJJcUb8vc-NTYw9hV-0rt0UkPr5qsgpKiA-DVPZDkhQks7fG5XsSg4gUj6n4R9UhtA-IcjDqZOva1YaZd0HpbZeMdMQFefcaq1OQkYJZTdl9wJq8NvDhMM3uSGnuI366s3mtNSASnmmZsu19gB4c8Mr8nYYxJfyq7mg09ivqogE7OJhx2rN0ms9neJdOvPQC~S-C7JkHuW6hSrHr60Qna6sJJx7h0gN9PCXBjDWTaf6RwStO9x47jjIUJwCgEV-g1UPWYry0uK7fzL13bDwCNIfIVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An exemplary OrthNet showing E. salsugineum lineage-specific tr–d events. (A) OrthNet for the SALT-TOLERANCEE 32 (SAT32). Nodes and edges are as described in Fig. 3A. The OrthNet showed E. salsugineum-specific tr–d, with three of the five Tr copies tandem duplicated. (B) Comparison of SAT32 loci and adjacent ±30 kb genomic regions between A. thaliana, S. parvula, and E. salsugineum as a GEvo plot (https://genomevolution.org/r/maxx (10 October 2018, date last accessed)). Pink/semi-transparent ribbons connect Homologous Genomic Segments (HGSs) between genomes, while gene models, mRNAs, and coding sequences are depicted as cylinders underneath HGSs (for detailed legends, see https://genomevolution.org/wiki/index.php/GEvo (10 October 2018, date last accessed)). EsSAT32;1 is the CL copy (Esa|20186362), while EsSAT32;2 (Esa|20188564), EsSAT32;3 (Esa|20189286), EsSAT32;4 (Esa|20179565), and EsSAT32;5 (Esa|20180739) indicate the four Tr copies with complete ORFs. The yellow arrow marks the position of Esa|20181274, which contains a truncated ORF. EsSAT32;3/4/5 showed intron losses without compromising gene products (see text and Supplementary Fig. S9 for details)." path-from-xml="dsy035f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979954/dsy035f7.tif" path-from-xml="dsy035f7.tif" target="_blank">Open in new tab</a><a section="131979954" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035f7.png?Expires=1576277252&amp;Signature=wtmMwjNaFc7a2dtMmB62mA8eaPp5ZK9dMXEpqmUCrgV0icAK5CkzFPt8cPKQMIkRjpneRXHC1KqBMWP04UPyaSQiky9AVXYM4IiNT5~Y6Q2G8iRSJFuWFDuv~uzP~07152oCKzLKmtMxhkiUwa~QBEqDwkBuv-A1TRV3-jwp-mQ1P3fA5HXVrm2DvkSwK5sXdP9By9PXpxVfGkXOdYo-THowvkP09KRm7Tc8C6vSJ-IJXw-LHtNIrZbeHQSP3Bq0E6sayHO7pTljTk8~odMiLrL-vEBV~rhPNcZUNvSawXrScE39W3u8neGlALcAV6ohHKR1kKlunwJLPWQ6Pf9HJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979954&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy035f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>An exemplary OrthNet showing <em>E. salsugineum</em> lineage-specific <em>tr–d</em> events. (A) OrthNet for the <em>SALT-TOLERANCEE 32</em> (<em>SAT32</em>). Nodes and edges are as described in <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>. The OrthNet showed <em>E. salsugineum</em>-specific <em>tr–d</em>, with three of the five Tr copies tandem duplicated. (B) Comparison of <em>SAT32</em> loci and adjacent ±30 kb genomic regions between <em>A. thaliana</em>, <em>S. parvula</em>, and <em>E. salsugineum</em> as a GEvo plot (<a class="link link-uri" href="https://genomevolution.org/r/maxx" target="_blank">https://genomevolution.org/r/maxx</a> (10 October 2018, date last accessed)). Pink/semi-transparent ribbons connect Homologous Genomic Segments (HGSs) between genomes, while gene models, mRNAs, and coding sequences are depicted as cylinders underneath HGSs (for detailed legends, see <a class="link link-uri" href="https://genomevolution.org/wiki/index.php/GEvo" target="_blank">https://genomevolution.org/wiki/index.php/GEvo</a> (10 October 2018, date last accessed)). <em>EsSAT32;1</em> is the CL copy (Esa|20186362), while <em>EsSAT32;2</em> (Esa|20188564), <em>EsSAT32;3</em> (Esa|20189286), <em>EsSAT32;4</em> (Esa|20179565), and <em>EsSAT32;5</em> (Esa|20180739) indicate the four Tr copies with complete ORFs. The yellow arrow marks the position of Esa|20181274, which contains a truncated ORF. <em>EsSAT32;3/4/5</em> showed intron losses without compromising gene products (see text and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span> for details).</p></div>
</div>
<p>For <em>tr–d</em> events shared by multiple genomes, we present the entire list of such OrthNets for all pairs of genomes, as well as those associated with Lineages I- and II-specific <em>tr–d</em> events, in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3. The <em>tr–d</em> events shared by <em>E. salsugineum</em> and <em>S. parvula</em> were of particular interest, because they may indicate signatures of convergent evolution between these two species independently adapted to high salinity.<a reveal-id="dsy035-B47" data-open="dsy035-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsy035-B48" data-open="dsy035-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span> lists all OrthNets with ‘Ind-parallel’ and ‘Tr–<em>cl</em>’ type <em>tr–d</em> events (as defined in <a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) shared between these two extremophytes. We also included all Lineage II (i.e. <em>E. salsugineum</em>, <em>S. irio</em>, and <em>S. parvula</em>)-specific <em>tr–d</em> events that had truncated Tr copy ORFs only for <em>S. irio</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>, marked by superscript ‘e’). Among the ‘Ind-parallel’ <em>tr–d</em> events detected, three out of eight events were associated with stress signalling or response-related functions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>, <em>CDPK1</em>, <em>5PTASE11</em>, and <em>ABI1</em>). However, none of them included complete Tr copy ORFs in both <em>E. salsugineum</em> and <em>S. parvula</em>. Interestingly, the ‘Tr–<em>cl</em>’ category included more loci with Lineage II-specific <em>tr–d</em> followed by truncation of the Tr copy ORF in <em>S. irio</em>, leaving complete ORFs in the Tr copy loci only for <em>E. salsugineum</em> and <em>S. parvula</em>. Here, we found loci encoding orthologues of a putative basic helix-loop-helix (bHLH) type transcription factor, a NAC transcription factor (NAC058), and a calcineurin B-like protein 10 (CBL10). All Tr copies encoding these regulatory proteins showed expression evidence in both halophytes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p>                    <h2 scrollto-destination="131979956" id="131979956" class="section-title" data-legacy-id="SEC4">4. Discussion</h2>
                    <h3 scrollto-destination="131979957" id="131979957" class="section-title" data-legacy-id="SEC4.1">4.1. A systematic identification of orthologous loci with the same evolutionary history</h3>
<p>While a number of tools are available to detect co-linearity or synteny blocks among multiple genomes,<a reveal-id="dsy035-B32" data-open="dsy035-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy035-B33" data-open="dsy035-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsy035-B49" data-open="dsy035-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsy035-B57" data-open="dsy035-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><a reveal-id="dsy035-B58" data-open="dsy035-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> there has been a lack of methods which can systematically identify all groups of orthologous gene loci among multiple genomes that underwent the same set of evolutionary events, such as gene duplications and transpositions in a certain lineage or multiple lineages that are either mono-, para-, or polyphyletic. In an attempt to fill this gap, we developed the CLfinder-OrthNet pipeline. The CLfinder module detects co-linearity and transposition for individual loci, rather than for synteny blocks, to facilitate identification of single-gene transposition that consist the majority of transposition events<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy035-B22" data-open="dsy035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and automatically generates a summary of all pairwise genome comparisons as exemplified in <a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>. The ONfinder module follows the CLfinder and produces OrthNets connecting orthologous genes with edges representing the presence or absence of co-linearity among them, enabling a search based on the network topology for all orthologous gene groups sharing an evolutionary history.</p>
<p>Previous works have suggested network representation of synteny among orthologues as an effective method to combine and summarize synteny blocks identified by all-to-all pairwise comparisons among multiple genomes.<a reveal-id="dsy035-B59" data-open="dsy035-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> Synteny networks connecting co-linear orthologues from multiple genomes with undirected edges traced the evolutionary path of a gene family.<a reveal-id="dsy035-B25" data-open="dsy035-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> This approach has been used to compare the extent of gene duplications and lineage-specific expansion of gene families between mammalian and plant genomes.<a reveal-id="dsy035-B26" data-open="dsy035-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> While the CLfinder module similarly performs all-to-all pairwise analyses to detect co-linearity in gene order, OrthNets detected by the CLfinder-OrthNet pipeline are different from synteny networks<a reveal-id="dsy035-B59" data-open="dsy035-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> in a number of ways. For example, while synteny networks connected co-linear nodes with undirected edges, OrthNets connected nodes with directional edges with co-linearity or lack of it (i.e. transposed) encoded as edge properties. An OrthNet includes orthologues connected by reciprocal edges as well as paralogues derived from duplications connected by unidirectional edges to their neighbouring nodes found in other genomes [<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>, panels (4) and (5); <a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Figs 4A and 5A</a>]. We aimed to separate each OrthNet into a unit that represents a group of orthologues and paralogues likely derived from a single ancestral locus, by employing Markov clustering (MCL) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S2</a></span>). We chose MCL to control edge weights to prefer undirected tandem duplicated edges and reciprocal edges over unidirectional edges during the clustering process. In this way, each of the majority of OrthNets, e.g. &gt;85% of all OrthNets in case of the six Brassicaceae genomes (<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), represents the evolutionary history of genes derived from a single ancestral locus as the network topology. Essentially, OrthNets enable detection of all loci from multiple genomes that share the same evolutionary history by a search using a given network topology as the query (e.g. <a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Figs 3A and 5A</a>). We used this functionality to characterized transposition–duplication (<em>tr–d</em>) events in six Brassicaceae genomes, as a proof-of-concept.</p>                    <h3 scrollto-destination="131979960" id="131979960" class="section-title" data-legacy-id="SEC4.2">4.2. Transposition–duplication as a major mechanism for erosion of co-linearity</h3>
<p>For the transposition and transposition–duplication (<em>tr–d</em>) of non-TE gene loci, two types of models, retrotransposon-associated and DNA repair or replication-associated models have been suggested as the main mechanisms.<a reveal-id="dsy035-B60" data-open="dsy035-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsy035-B61" data-open="dsy035-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> A <em>tr–d</em> event derived from retrotransposons often leads to duplication of single exon genes.<a reveal-id="dsy035-B62 dsy035-B63 dsy035-B64" data-open="dsy035-B62 dsy035-B63 dsy035-B64" class="link link-ref link-reveal xref-bibr"><sup>62–64</sup></a> Transposition–duplication may also arise during the non-homologous end-joining (NHEJ) repair process of DNA double-strand breaks (DSB), where a short sequence motif may act as an anchor to a foreign sequence to fill-in a gap.<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> In agreement with this model, a previous comparison of <em>A. lyrata</em> and <em>A. thaliana</em> found a significant enrichment of flanking repeats, as short as 15 bps, among transposed genes.<a reveal-id="dsy035-B65" data-open="dsy035-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> The correlation between the proportion of query gene loci showing distal displacement (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>, <em>d<sub>n</sub></em><sub>,</sub><em><sub>n+</sub></em><sub>1</sub>&gt;20 or ‘Diff Chr’) and overall TE contents of the query genome, rather than divergence time (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1D</a></span>), supports the DSB-repair model. Higher TE contents likely provide a higher frequency of short repeat anchors required for the NHEJ DSB-repair, and TE activities themselves may also cause the DSB that lead to such repairs.<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The DSB-repair model can explain <em>tr–d</em> of multi-exon genes, which constitute the majority of <em>tr–d</em> events found in this study (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>).</p>
<p>Among lineage-specific <em>tr–d</em> events captured in OrthNets, we found a subset of transposed–duplicated gene loci (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, Tr copies) retaining similar ORF sizes compared with their respective donor locus (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, CL copy), as well as to orthologues in other species (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4B–D</a>). The DSB repair model of <em>tr–d</em> suggests that the duplicated region may start and end virtually at any random position in a genome, given that the short sequence motif needed for the repair is likely ubiquitously available and can be as short as several nucleotides.<a reveal-id="dsy035-B1" data-open="dsy035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy035-B65" data-open="dsy035-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> However, our simulation reveled that both ‘complete’ and ‘gene-only’ <em>tr–d</em> events were far more frequent than what was expected from a random duplication model alone (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a>). We are not aware of a gene duplication mechanism that preferably duplicates non-TE, protein-coding, multi-exon genes as entire units. Rather, our observation common to all six tested crucifer genomes is likely a result of random <em>tr–d</em> events (e.g. through DSB repair), followed by accumulation of mutations throughout the duplicated regions, except where the complete coding sequences were selectively retained. Supporting this notion, Tr copy genes with complete ORFs were more frequent among shared <em>tr–d</em> of older ‘Tr–<em>cl</em>’ type events (<a reveal-id="dsy035-F5" data-open="dsy035-F5" class="link link-reveal link-table xref-fig">Figs 5C and 6D</a>). These were also more likely to be expressed, hence less likely to be pseudogenes, compared with Tr copies with truncated ORFs (<a reveal-id="dsy035-F6" data-open="dsy035-F6" class="link link-reveal link-table xref-fig">Fig. 6E</a>). See Supplementary Texts for further discussions on the age of ‘gene-only’ <em>tr–d</em> events (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S4</a></span>) and on <em>tr–d</em> frequencies and TE contents (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S5</a></span>). Overall, our analyses depicted the landscape of <em>tr–d</em> events among Brassicaceae genomes, where the majority of <em>tr–d</em> was incomplete, while small numbers of <em>tr–d</em> including complete Tr copy ORFs and gene-only <em>tr–d</em> were likely to have resulted from random duplication events followed by selective retention of coding sequences over time.</p>                    <h3 scrollto-destination="131979963" id="131979963" class="section-title" data-legacy-id="SEC4.3">4.3. Search for extremophyte-specific <em>tr–d</em> events using CLfinder-OrthNet</h3>
<p>One possible application of the CLfinder-OrthNet pipeline is to retrieve orthologues sharing evolutionary events unique to a lineage with a specific trait or multiple lineages exhibiting a convergent trait, e.g. the two extremophyte <em>S. parvula</em> and <em>E. salsugineum</em>. As detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S6</a></span>, these two genomes have been identified with gene copy number and structural variations compared with <em>A. thaliana</em> that were associated with stress-adapted traits.<a reveal-id="dsy035-B10" data-open="dsy035-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy035-B66" data-open="dsy035-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a><sup>,</sup><a reveal-id="dsy035-B67" data-open="dsy035-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> In this study, CLfinder-OrthNet identified 63, 26, and 14 orthologue groups, represented as OrthNets, showing gene copy number increases through <em>tr–d</em> events specific to <em>E. salsugineum</em>, <em>S. parvula</em>, and both, respectively (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>, and Dataset S3). These numbers are orders of magnitude fewer than previous searches from a pairwise comparison with <em>A. thaliana</em>,<a reveal-id="dsy035-B5" data-open="dsy035-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> signifying the vastly improved resolution in finding extremophyte-specific events.</p>
<p>The OrthNet for the <em>SALT-TOLERANCE 32</em> (<em>SAT32</em>) locus (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>, ON_2516) represents the largest <em>E. salsugineum</em>-specific <em>tr–d</em> event. <em>SAT32</em> encodes a transcription regulator, whose expression level positively correlated with the survival rate of the model plant <em>A. thaliana</em> under salt stress.<a reveal-id="dsy035-B68" data-open="dsy035-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Three of the four <em>EsSAT32</em> paralogues with complete ORFs exhibited intron losses (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7B and</a><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Intron losses and smaller transcript sizes are reported to enable regulation of expression timing in Drosophila and mouse.<a reveal-id="dsy035-B69" data-open="dsy035-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a><sup>,</sup><a reveal-id="dsy035-B70" data-open="dsy035-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> It is not clear whether ‘gene-only’ <em>tr–d</em> events (<a reveal-id="dsy035-F7" data-open="dsy035-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>) among <em>EsSAT32</em> paralogues is indicative of reverse transcriptase-mediated duplication leading to intron losses<a reveal-id="dsy035-B71" data-open="dsy035-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> or different rate of mutation between gene and intergenic regions. Either way, such variation in intergenic regions including promoter regions may lead to sub-functionalization.<a reveal-id="dsy035-B2" data-open="dsy035-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> At least three <em>EsSAT32</em> paralogues exhibited different basal expression strengths in root and shoot tissues (data not shown).</p>
<p>A notable example of <em>S. parvula-</em>specific <em>tr–d</em>, with copy number increases in complete ORFs, is the <em>ZRT/IRT-LIKE PROTEIN 3</em> (<em>ZIP3</em>) locus encoding a zinc transporter (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). This particular <em>tr–d</em> may be a signature of an adaptation in <em>S. parvula</em>, to soils that are highly saline and also depleted in micronutrients such as zinc and iron in central Anatolia.<a reveal-id="dsy035-B72" data-open="dsy035-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a><sup>,</sup><a reveal-id="dsy035-B73" data-open="dsy035-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> See <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S7</a></span> and S8 for discussions on genes involved in <em>tr–d</em> unique to each extremophyte, as well as <em>tr–d</em> shared by the two extremophytes.</p>                    <h3 scrollto-destination="131979967" id="131979967" class="section-title" data-legacy-id="SEC4.4">4.4. Concluding remarks: CLfinder-OrthNet, a flexible toolkit for comparative genomics</h3>
<p>The CLfinder-OrthNet pipeline, in a proof-of-concept application, successfully encodes more than 85% of entire loci among six Brassicaceae genomes into OrthNet units in which evolutionary histories of genes derived from single ancestral loci can be traced (<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). Using a network topology-based search, we identified groups of orthologues, represented as OrthNets that share the same evolutionary histories (<a reveal-id="dsy035-F3" data-open="dsy035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), including <em>tr–d</em> unique to any subset of the six Brassicaceae genomes (<a reveal-id="dsy035-F4" data-open="dsy035-F4" class="link link-reveal link-table xref-fig">Figs 4 and 5</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>set S3).</p>
<p>As detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text S9</a></span>, CLfinder-OrthNet offers multiple options to apply the pipeline flexibly depending on target genomes and goals of the study. The sensitivity and stringency of co-linearity detection are adjustable by controlling parameters depending on the range of target genomes. The CLfinder module can use results from any method of sequence clustering and comparison, as well as genomic features other than protein-coding genes, as inputs. Moreover, the two modules can be used separately. For example, researchers can use the CLfinder module to quickly summarize the distribution of co-linear, tandem duplicated, and transposed genes among multiple genomes (e.g. <a reveal-id="dsy035-T1" data-open="dsy035-T1" class="link link-reveal link-table xref-fig">Table 1</a>), while the ONfinder module can accept locus-level synteny information from other methods to generate OrthNets.</p>
<p>Overall, the CLfinder-OrthNet pipeline offers a flexible toolkit to compare the arrangement of gene and other genomic features among multiple genomes. Future applications include, but not limited to, tracing evolutionary histories of a gene or gene families; inference of orthology based on both sequence homology and co-linearity; studying incongruence between sequence homology and synteny; and identification of candidate gene copy number variations associated with specific hypothesis-driven evolutionary mechanisms or traits.</p>                    <h2 scrollto-destination="131979971" id="131979971" class="section-title">Funding</h2>
<p>This work was supported by National Science Foundation (MCB 1616827) and the Next Generation BioGreen21 Program (PJ01317301) of the Rural Development Administration, Republic of Korea.</p>                    <h2 scrollto-destination="131979973" id="131979973" class="section-title">Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576277252&amp;Signature=uSEK2pH~~2FCdblJkfto-rEqjkUvLLQhjkjOH5DlRdy583cST5aag4Fg77lIPFla0NZfAa9z6KTr7LepD~nFcpCJFWEc9G50epUOaxnYPGKVM568~3a6ANLg-9WA5FE7YNrjF1UkNU4wh4uFUt1q6RHNil6M2L1ty10ywXDIygdEQrAIw10efY3L8Qi-HCXnAkAIK0I-4ikouT9Zj0iYwzp2ZUmpXsRUZbnOQ8xkqDz6SC0lnJwWRkpTyo-uEfrL~a9Wp3GgV2tFRJNsbVkExy~sp3O7yHTQIga7qhF32PvYpkJXbeh5AJD9inMbRLN5zsqxpIJQ7XG8JuRW4rfUHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> (Supplementary Texts S1-S9, Tables S1-S6, Figures S1-S9, and Dataset S2-S4) are available at <em>DNARES</em> online. Supplementary Dataset S1 is deposited at figshare (<a class="link" href="https://doi.org/10.6084/m9.figshare.6959435.v1" target="_blank">https://doi.org/10.6084/m9.figshare.6959435.v1</a>). The CLfinder-OrthNet pipeline is available in a GitHub (<a class="link link-uri" href="https://github.com/ohdongha/CL_finder" target="_blank">https://github.com/ohdongha/CL_finder</a>).</p>                    <h2 scrollto-destination="131979975" id="131979975" class="section-title">Conflict of interest</h2>
<p>None declared. </p>                    <h2 scrollto-destination="131979977" id="131979977" class="backreferences-title" data-legacy-id="Ref1">References</h2>
<div class="ref-list">
<div content-id="dsy035-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wicker</div> <div class="given-names">T.M.</div>
</div>, <div class="name string-name">
<div class="surname">Buchmann</div> <div class="given-names">J.P.</div>
</div>, <div class="name string-name">
<div class="surname">Keller</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Patching gaps in plant genomes results in gene movement and erosion of colinearity</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1229</div>–<div class="lpage">37</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Patching%20gaps%20in%20plant%20genomes%20results%20in%20gene%20movement%20and%20erosion%20of%20colinearity&amp;author=T.M.%20Wicker&amp;author=J.P.%20Buchmann&amp;author=B.%20Keller&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1229-37" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.107284.110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107284.110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.107284.110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20530251" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Patching%20gaps%20in%20plant%20genomes%20results%20in%20gene%20movement%20and%20erosion%20of%20colinearity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Paterson</div> <div class="given-names">A.H.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Genome and gene duplications and gene expression divergence: a view from plants</div>, <div class="source">Ann. N.Y. Acad. Sci</div> ., <div class="volume">1256</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20gene%20duplications%20and%20gene%20expression%20divergence%3A%20a%20view%20from%20plants&amp;author=Y.%20Wang&amp;author=X.%20Wang&amp;author=A.H.%20Paterson&amp;publication_year=2012&amp;journal=Ann.%20N.Y.%20Acad.%20Sci&amp;volume=1256&amp;pages=1-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1749-6632.2011.06384.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1749-6632.2011.06384.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1749-6632.2011.06384.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22257007" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20and%20gene%20duplications%20and%20gene%20expression%20divergence%3A%20a%20view%20from%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Assis</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Bachtrog</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Neofunctionalization of young duplicate genes in Drosophila</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">110</div>, <div class="fpage">17409</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Neofunctionalization%20of%20young%20duplicate%20genes%20in%20Drosophila&amp;author=R.%20Assis&amp;author=D.%20Bachtrog&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=110&amp;pages=17409-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1313759110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1313759110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1313759110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24101476" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Neofunctionalization%20of%20young%20duplicate%20genes%20in%20Drosophila&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Arsovski</div> <div class="given-names">A.A.</div>
</div>, <div class="name string-name">
<div class="surname">Pradinuk</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Guo</div> <div class="given-names">X.Q.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Adams</div> <div class="given-names">K.L.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Evolution of cis-regulatory elements and regulatory networks in duplicated genes of <em>Arabidopsis thaliana</em>
</div>. <div class="source">Plant Physiol</div> ., <div class="volume">169</div>, <div class="fpage">2982</div>–<div class="lpage">91</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20cis-regulatory%20elements%20and%20regulatory%20networks%20in%20duplicated%20genes%20of%20Arabidopsis%20thaliana&amp;author=A.A.%20Arsovski&amp;author=J.%20Pradinuk&amp;author=X.Q.%20Guo&amp;author=S.%20Wang&amp;author=K.L.%20Adams&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=169&amp;pages=2982-91" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26474639" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Evolution%20of%20cis-regulatory%20elements%20and%20regulatory%20networks%20in%20duplicated%20genes%20of%20Arabidopsis%20thaliana&amp;aulast=Arsovski&amp;date=2015&amp;spage=2982&amp;epage=91&amp;volume=169" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20of%20cis-regulatory%20elements%20and%20regulatory%20networks%20in%20duplicated%20genes%20of%20Arabidopsis%20thaliana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oh</div> <div class="given-names">D.-H.</div>
</div>, <div class="name string-name">
<div class="surname">Hong</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">S.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Yun</div> <div class="given-names">D.-J.</div>
</div>, <div class="name string-name">
<div class="surname">Bohnert</div> <div class="given-names">H.J.</div>
</div>, <div class="name string-name">
<div class="surname">Dassanayake</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Genome structures and transcriptomes signify niche adaptation for the multiple-ion-tolerant extremophyte <em>Schrenkiella parvula</em>
</div>, <div class="source">Plant Physiol</div> ., <div class="volume">164</div>, <div class="fpage">2123</div>–<div class="lpage">38</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20structures%20and%20transcriptomes%20signify%20niche%20adaptation%20for%20the%20multiple-ion-tolerant%20extremophyte%20Schrenkiella%20parvula&amp;author=D.-H.%20Oh&amp;author=H.%20Hong&amp;author=S.Y.%20Lee&amp;author=D.-J.%20Yun&amp;author=H.J.%20Bohnert&amp;author=M.%20Dassanayake&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=164&amp;pages=2123-38" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.113.233551" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.233551" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.113.233551"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24563282" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20structures%20and%20transcriptomes%20signify%20niche%20adaptation%20for%20the%20multiple-ion-tolerant%20extremophyte%20Schrenkiella%20parvula&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Durand</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Bouché</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Perez Strand</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Loudet</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Camilleri</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Rapid establishment of genetic incompatibility through natural epigenetic variation</div>, <div class="source">Curr. Biol</div> ., <div class="volume">22</div>, <div class="fpage">326</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rapid%20establishment%20of%20genetic%20incompatibility%20through%20natural%20epigenetic%20variation&amp;author=S.%20Durand&amp;author=N.%20Bouch%C3%A9&amp;author=E.%20Perez%20Strand&amp;author=O.%20Loudet&amp;author=C.%20Camilleri&amp;publication_year=2012&amp;journal=Curr.%20Biol&amp;volume=22&amp;pages=326-31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.cub.2011.12.054" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2011.12.054" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.cub.2011.12.054"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22285031" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rapid%20establishment%20of%20genetic%20incompatibility%20through%20natural%20epigenetic%20variation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Feuerborn</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Cook</div> <div class="given-names">P.R.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Why the activity of a gene depends on its neighbors</div>, <div class="source">Trends Genet</div> ., <div class="volume">31</div>, <div class="fpage">483</div>–<div class="lpage">90</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Why%20the%20activity%20of%20a%20gene%20depends%20on%20its%20neighbors&amp;author=A.%20Feuerborn&amp;author=P.R.%20Cook&amp;publication_year=2015&amp;journal=Trends%20Genet&amp;volume=31&amp;pages=483-90" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tig.2015.07.001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2015.07.001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tig.2015.07.001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26259670" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Why%20the%20activity%20of%20a%20gene%20depends%20on%20its%20neighbors&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Genome-wide prediction and validation of intergenic enhancers in <em>Arabidopsis</em> using open chromatin signatures</div>, <div class="source">Plant Cell</div> , <div class="volume">27</div>, <div class="fpage">2415</div>–<div class="lpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20prediction%20and%20validation%20of%20intergenic%20enhancers%20in%20Arabidopsis%20using%20open%20chromatin%20signatures&amp;author=B.%20Zhu&amp;author=W.%20Zhang&amp;author=T.%20Zhang&amp;author=B.%20Liu&amp;author=J.%20Jiang&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=2415-26" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.15.00537" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.15.00537" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.15.00537"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26373455" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20prediction%20and%20validation%20of%20intergenic%20enhancers%20in%20Arabidopsis%20using%20open%20chromatin%20signatures&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hu</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Tee</div> <div class="given-names">W.W.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Enhancers and chromatin structures: regulatory hubs in gene expression and diseases</div>. <div class="source">Biosci. Rep</div> ., <div class="volume">37</div>, BSR20160183.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Enhancers%20and%20chromatin%20structures%3A%20regulatory%20hubs%20in%20gene%20expression%20and%20diseases&amp;aulast=Hu&amp;date=2017&amp;volume=37" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Enhancers%20and%20chromatin%20structures%3A%20regulatory%20hubs%20in%20gene%20expression%20and%20diseases&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oh</div> <div class="given-names">D.-H.</div>
</div>, <div class="name string-name">
<div class="surname">Dassanayake</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">J.S.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Genome structures and halophyte-specific gene expression of the extremophile <em>Thellungiella parvula</em> in comparison with <em>Thellungiella salsuginea</em> (<em>Thellungiella halophila</em>) and Arabidopsis</div>. <div class="source">Plant Physiol</div> ., <div class="volume">154</div>, <div class="fpage">1040</div>–<div class="lpage">52</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20structures%20and%20halophyte-specific%20gene%20expression%20of%20the%20extremophile%20Thellungiella%20parvula%20in%20comparison%20with%20Thellungiella%20salsuginea%20(Thellungiella%20halophila)%20and%20Arabidopsis&amp;author=D.-H.%20Oh&amp;author=M.%20Dassanayake&amp;author=J.S.%20Haas&amp;publication_year=2010&amp;journal=Plant%20Physiol&amp;volume=154&amp;pages=1040-52" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.110.163923" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.163923" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.110.163923"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20833729" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20structures%20and%20halophyte-specific%20gene%20expression%20of%20the%20extremophile%20Thellungiella%20parvula%20in%20comparison%20with%20Thellungiella%20salsuginea%20(Thellungiella%20halophila)%20and%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">S.-L.</div>
</div>, <div class="name string-name">
<div class="surname">Pan</div> <div class="given-names">A.Q.</div>
</div>, <div class="name string-name">
<div class="surname">Adams</div> <div class="given-names">K.L.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Protein subcellular relocalization of duplicated genes in <em>Arabidopsis</em>
</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">6</div>, <div class="fpage">2501</div>–<div class="lpage">15</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Protein%20subcellular%20relocalization%20of%20duplicated%20genes%20in%20Arabidopsis&amp;author=S.-L.%20Liu&amp;author=A.Q.%20Pan&amp;author=K.L.%20Adams&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol&amp;volume=6&amp;pages=2501-15" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/gbe/evu191" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu191" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgbe%2Fevu191"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25193306" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Protein%20subcellular%20relocalization%20of%20duplicated%20genes%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Panchy</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Lehti-Shiu</div> <div class="given-names">M.D.</div>
</div>, <div class="name string-name">
<div class="surname">Shiu</div> <div class="given-names">S.-H.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Evolution of gene duplication in plants</div>, <div class="source">Plant Physiol</div> ., <div class="volume">171</div>, <div class="fpage">2294</div>–<div class="lpage">316</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20gene%20duplication%20in%20plants&amp;author=N.%20Panchy&amp;author=M.D.%20Lehti-Shiu&amp;author=S.-H.%20Shiu&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=171&amp;pages=2294-316" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27288366" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Evolution%20of%20gene%20duplication%20in%20plants&amp;aulast=Panchy&amp;date=2016&amp;spage=2294&amp;epage=316&amp;volume=171" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20of%20gene%20duplication%20in%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Tao</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Marowsky</div> <div class="given-names">N.C.</div>
</div>, <div class="name string-name">
<div class="surname">Fan</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Evolutionary fates and dynamic functionalization of young duplicate genes in <em>Arabidopsis</em> genomes</div>, <div class="source">Plant Physiol</div> ., <div class="volume">172</div>, <div class="fpage">427</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20fates%20and%20dynamic%20functionalization%20of%20young%20duplicate%20genes%20in%20Arabidopsis%20genomes&amp;author=J.%20Wang&amp;author=F.%20Tao&amp;author=N.C.%20Marowsky&amp;author=C.%20Fan&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=172&amp;pages=427-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.16.01177" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.16.01177" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.16.01177"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27485883" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolutionary%20fates%20and%20dynamic%20functionalization%20of%20young%20duplicate%20genes%20in%20Arabidopsis%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Conant</div> <div class="given-names">G.C.</div>
</div>, <div class="name string-name">
<div class="surname">Wolfe</div> <div class="given-names">K.H.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Turning a hobby into a job: how duplicated genes find new functions</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">9</div>, <div class="fpage">938</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Turning%20a%20hobby%20into%20a%20job%3A%20how%20duplicated%20genes%20find%20new%20functions&amp;author=G.C.%20Conant&amp;author=K.H.%20Wolfe&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet&amp;volume=9&amp;pages=938-50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg2482" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2482" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg2482"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015656" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Turning%20a%20hobby%20into%20a%20job%3A%20how%20duplicated%20genes%20find%20new%20functions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Kondrashov</div> <div class="given-names">F.A.</div>
</div></div> <div class="year">2012</div>, 
<div class="article-title">Gene duplication as a mechanism of genomic adaptation to a changing environment</div>, <div class="source">Proc. Biol. Sci</div> ., <div class="volume">279</div>, <div class="fpage">5048</div>–<div class="lpage">57</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20as%20a%20mechanism%20of%20genomic%20adaptation%20to%20a%20changing%20environment&amp;author=F.A.%20Kondrashov&amp;publication_year=2012&amp;journal=Proc.%20Biol.%20Sci&amp;volume=279&amp;pages=5048-57" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rspb.2012.1108" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2012.1108" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frspb.2012.1108"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22977152" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20duplication%20as%20a%20mechanism%20of%20genomic%20adaptation%20to%20a%20changing%20environment&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cook</div> <div class="given-names">D.E.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">T.G.</div>
</div>, <div class="name string-name">
<div class="surname">Guo</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Copy number variation of multiple genes at <em>Rhg1</em> mediates nematode resistance in soybean</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">1206</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20of%20multiple%20genes%20at%20Rhg1%20mediates%20nematode%20resistance%20in%20soybean&amp;author=D.E.%20Cook&amp;author=T.G.%20Lee&amp;author=X.%20Guo&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1206-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1228746" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1228746" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1228746"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23065905" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Copy%20number%20variation%20of%20multiple%20genes%20at%20Rhg1%20mediates%20nematode%20resistance%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gan</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Hay</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Kwantes</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">The <em>Cardamine hirsuta</em> genome offers insight into the evolution of morphological diversity</div>, <div class="source">Nat. Plants</div> , <div class="volume">2</div>, <div class="fpage">16167</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Cardamine%20hirsuta%20genome%20offers%20insight%20into%20the%20evolution%20of%20morphological%20diversity&amp;author=X.%20Gan&amp;author=A.%20Hay&amp;author=M.%20Kwantes&amp;publication_year=2016&amp;journal=Nat.%20Plants&amp;volume=2&amp;pages=16167" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nplants.2016.167" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2016.167" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnplants.2016.167"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27797353" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Cardamine%20hirsuta%20genome%20offers%20insight%20into%20the%20evolution%20of%20morphological%20diversity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shirai</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Matsuda</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Nakabayashi</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">A highly specific genome-wide association study integrated with transcriptome data reveals the contribution of copy number variations to specialized metabolites in <em>Arabidopsis thaliana</em> accessions</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">34</div>, <div class="fpage">3111</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20highly%20specific%20genome-wide%20association%20study%20integrated%20with%20transcriptome%20data%20reveals%20the%20contribution%20of%20copy%20number%20variations%20to%20specialized%20metabolites%20in%20Arabidopsis%20thaliana%20accessions&amp;author=K.%20Shirai&amp;author=F.%20Matsuda&amp;author=R.%20Nakabayashi&amp;publication_year=2017&amp;journal=Mol.%20Biol.%20Evol&amp;volume=34&amp;pages=3111-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msx234" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsx234" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsx234"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28961930" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20highly%20specific%20genome-wide%20association%20study%20integrated%20with%20transcriptome%20data%20reveals%20the%20contribution%20of%20copy%20number%20variations%20to%20specialized%20metabolites%20in%20Arabidopsis%20thaliana%20accessions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Simon</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Ruhl</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">De Montaigu</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Wotzel</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Coupland</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Evolution of CONSTANS regulation and function after gene duplication produced a photoperiodic flowering switch in the Brassicaceae</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">2284</div>–<div class="lpage">301</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20CONSTANS%20regulation%20and%20function%20after%20gene%20duplication%20produced%20a%20photoperiodic%20flowering%20switch%20in%20the%20Brassicaceae&amp;author=S.%20Simon&amp;author=M.%20Ruhl&amp;author=A.%20De%20Montaigu&amp;author=S.%20Wotzel&amp;author=G.%20Coupland&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=2284-301" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msv110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsv110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25972346" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20of%20CONSTANS%20regulation%20and%20function%20after%20gene%20duplication%20produced%20a%20photoperiodic%20flowering%20switch%20in%20the%20Brassicaceae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grandaubert</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Lowe</div> <div class="given-names">R.G.T.</div>
</div>, <div class="name string-name">
<div class="surname">Soyer</div> <div class="given-names">J.L.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Transposable element-assisted evolution and adaptation to host plant within the <em>Leptosphaeria maculans–Leptosphaeria biglobosa</em> species complex of fungal pathogens</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1</div>–<div class="lpage">27</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transposable%20element-assisted%20evolution%20and%20adaptation%20to%20host%20plant%20within%20the%20Leptosphaeria%20maculans%E2%80%93Leptosphaeria%20biglobosa%20species%20complex%20of%20fungal%20pathogens&amp;author=J.%20Grandaubert&amp;author=R.G.T.%20Lowe&amp;author=J.L.%20Soyer&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1-27" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-15-891" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-891" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-15-891"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24382143" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transposable%20element-assisted%20evolution%20and%20adaptation%20to%20host%20plant%20within%20the%20Leptosphaeria%20maculans%E2%80%93Leptosphaeria%20biglobosa%20species%20complex%20of%20fungal%20pathogens&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Boore</div> <div class="given-names">J.L.</div>
</div>, <div class="name string-name">
<div class="surname">Lavrov</div> <div class="given-names">D.V.</div>
</div>, <div class="name string-name">
<div class="surname">Brown</div> <div class="given-names">W.M.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Gene translocation links insects and crustaceans</div>, <div class="source">Nature</div> , <div class="volume">392</div>, <div class="fpage">667</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20translocation%20links%20insects%20and%20crustaceans&amp;author=J.L.%20Boore&amp;author=D.V.%20Lavrov&amp;author=W.M.%20Brown&amp;publication_year=1998&amp;journal=Nature&amp;volume=392&amp;pages=667-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/33577" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F33577" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F33577"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9565028" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20translocation%20links%20insects%20and%20crustaceans&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Freeling</div> <div class="given-names">M.</div>
</div></div> <div class="year">2009</div>, 
<div class="article-title">Bias in plant gene content following different sorts of duplication: tandem, whole-genome, segmental, or by transposition</div>, <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">60</div>, <div class="fpage">433</div>–<div class="lpage">53</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bias%20in%20plant%20gene%20content%20following%20different%20sorts%20of%20duplication%3A%20tandem%2C%20whole-genome%2C%20segmental%2C%20or%20by%20transposition&amp;author=M.%20Freeling&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=60&amp;pages=433-53" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev.arplant.043008.092122" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.043008.092122" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev.arplant.043008.092122"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19575588" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bias%20in%20plant%20gene%20content%20following%20different%20sorts%20of%20duplication%3A%20tandem%2C%20whole-genome%2C%20segmental%2C%20or%20by%20transposition&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Emms</div> <div class="given-names">D.M.</div>
</div>, <div class="name string-name">
<div class="surname">Covshoff</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Hibberd</div> <div class="given-names">J.M.</div>
</div>, <div class="name string-name">
<div class="surname">Kelly</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Independent and parallel evolution of new genes by gene duplication in two origins of C4 photosynthesis provides new insight into the mechanism of phloem loading in C4 species</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">1796</div>–<div class="lpage">806</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Independent%20and%20parallel%20evolution%20of%20new%20genes%20by%20gene%20duplication%20in%20two%20origins%20of%20C4%20photosynthesis%20provides%20new%20insight%20into%20the%20mechanism%20of%20phloem%20loading%20in%20C4%20species&amp;author=D.M.%20Emms&amp;author=S.%20Covshoff&amp;author=J.M.%20Hibberd&amp;author=S.%20Kelly&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=1796-806" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msw057" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw057" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsw057"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27016024" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Independent%20and%20parallel%20evolution%20of%20new%20genes%20by%20gene%20duplication%20in%20two%20origins%20of%20C4%20photosynthesis%20provides%20new%20insight%20into%20the%20mechanism%20of%20phloem%20loading%20in%20C4%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Vlad</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Kierzkowski</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Rast</div> <div class="given-names">M.I.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Leaf shape evolution through duplication, regulatory diversification, and loss of a homeobox gene</div>, <div class="source">Science</div> , <div class="volume">343</div>, <div class="fpage">780</div>–<div class="lpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Leaf%20shape%20evolution%20through%20duplication%2C%20regulatory%20diversification%2C%20and%20loss%20of%20a%20homeobox%20gene&amp;author=D.%20Vlad&amp;author=D.%20Kierzkowski&amp;author=M.I.%20Rast&amp;publication_year=2014&amp;journal=Science&amp;volume=343&amp;pages=780-3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1248384" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1248384" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1248384"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24531971" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Leaf%20shape%20evolution%20through%20duplication%2C%20regulatory%20diversification%2C%20and%20loss%20of%20a%20homeobox%20gene&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Holmer</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">de Bruijn</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Angenent</div> <div class="given-names">G.C.</div>
</div>, <div class="name string-name">
<div class="surname">van den Burg</div> <div class="given-names">H.A.</div>
</div>, <div class="name string-name">
<div class="surname">Schranz</div> <div class="given-names">M.E.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Phylogenomic synteny network analysis of MADS-box transcription factor genes reveals lineage-specific transpositions, ancient tandem duplications, and deep positional conservation</div>, <div class="source">Plant Cell</div> , <div class="volume">29</div>, <div class="fpage">1278</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogenomic%20synteny%20network%20analysis%20of%20MADS-box%20transcription%20factor%20genes%20reveals%20lineage-specific%20transpositions%2C%20ancient%20tandem%20duplications%2C%20and%20deep%20positional%20conservation&amp;author=T.%20Zhao&amp;author=R.%20Holmer&amp;author=S.%20de%20Bruijn&amp;author=G.C.%20Angenent&amp;author=H.A.%20van%20den%20Burg&amp;author=M.E.%20Schranz&amp;publication_year=2017&amp;journal=Plant%20Cell&amp;volume=29&amp;pages=1278-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.17.00595" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.17.00595" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.17.00595"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28584165" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenomic%20synteny%20network%20analysis%20of%20MADS-box%20transcription%20factor%20genes%20reveals%20lineage-specific%20transpositions%2C%20ancient%20tandem%20duplications%2C%20and%20deep%20positional%20conservation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Schranz</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">Comparative phylogenomic synteny network analysis of mammalian and angiosperm genomes</div>, <div class="source">bioRxiv</div> , <div class="fpage">246736</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Comparative%20phylogenomic%20synteny%20network%20analysis%20of%20mammalian%20and%20angiosperm%20genomes&amp;aulast=Zhao&amp;date=2018&amp;spage=246736" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparative%20phylogenomic%20synteny%20network%20analysis%20of%20mammalian%20and%20angiosperm%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Hierarchically aligning 10 legume genomes establishes a family-level genomics platform</div>, <div class="source">Plant Physiol</div> , <div class="volume">174</div>, <div class="fpage">284</div>–<div class="lpage">300</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hierarchically%20aligning%2010%20legume%20genomes%20establishes%20a%20family-level%20genomics%20platform&amp;author=J.%20Wang&amp;author=P.%20Sun&amp;author=Y.%20Li&amp;publication_year=2017&amp;journal=Plant%20Physiol&amp;volume=174&amp;pages=284-300" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.16.01981" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.16.01981" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.16.01981"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28325848" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hierarchically%20aligning%2010%20legume%20genomes%20establishes%20a%20family-level%20genomics%20platform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Koenig</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Weigel</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Beyond the thale: comparative genomics and genetics of <em>Arabidopsis</em> relatives</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">16</div>, <div class="fpage">285</div>–<div class="lpage">98</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Beyond%20the%20thale%3A%20comparative%20genomics%20and%20genetics%20of%20Arabidopsis%20relatives&amp;author=D.%20Koenig&amp;author=D.%20Weigel&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet&amp;volume=16&amp;pages=285-98" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg3883" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3883" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg3883"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25854181" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Beyond%20the%20thale%3A%20comparative%20genomics%20and%20genetics%20of%20Arabidopsis%20relatives&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Haberer</div> <div class="given-names">G.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">The genome sequence of African rice (<em>Oryza glaberrima</em>) and evidence for independent domestication</div>, <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">982</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20African%20rice%20(Oryza%20glaberrima)%20and%20evidence%20for%20independent%20domestication&amp;author=M.%20Wang&amp;author=Y.%20Yu&amp;author=G.%20Haberer&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=982-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.3044" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3044" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.3044"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25064006" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genome%20sequence%20of%20African%20rice%20(Oryza%20glaberrima)%20and%20evidence%20for%20independent%20domestication&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Du</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Ma</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Sequencing and <em>de novo</em> assembly of a near complete indica rice genome</div>, <div class="source">Nat. Commun</div> ., <div class="volume">8</div>, <div class="fpage">15324</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20de%20novo%20assembly%20of%20a%20near%20complete%20indica%20rice%20genome&amp;author=H.%20Du&amp;author=Y.%20Yu&amp;author=Y.%20Ma&amp;publication_year=2017&amp;journal=Nat.%20Commun&amp;volume=8&amp;pages=15324" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ncomms15324" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms15324" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fncomms15324"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28469237" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequencing%20and%20de%20novo%20assembly%20of%20a%20near%20complete%20indica%20rice%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Debarry</div> <div class="given-names">J.D.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and colinearity</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20colinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.D.%20Debarry&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1293"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20colinearity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Bomhoff</div> <div class="given-names">M.D.</div>
</div>, <div class="name string-name">
<div class="surname">Briones</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Schnable</div> <div class="given-names">J.C.</div>
</div>, <div class="name string-name">
<div class="surname">Lyons</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">SynFind: compiling syntenic regions across any set of genomes on demand</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">7</div>, <div class="fpage">3286</div>–<div class="lpage">98</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=SynFind%3A%20compiling%20syntenic%20regions%20across%20any%20set%20of%20genomes%20on%20demand&amp;author=H.%20Tang&amp;author=M.D.%20Bomhoff&amp;author=E.%20Briones&amp;author=L.%20Zhang&amp;author=J.C.%20Schnable&amp;author=E.%20Lyons&amp;publication_year=2015&amp;journal=Genome%20Biol.%20Evol&amp;volume=7&amp;pages=3286-98" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/gbe/evv219" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevv219" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgbe%2Fevv219"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26560340" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:SynFind%3A%20compiling%20syntenic%20regions%20across%20any%20set%20of%20genomes%20on%20demand&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Proost</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Fostier</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">De Witte</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">i-ADHoRe 3.0-fast and sensitive detection of genomic homology in extremely large data sets</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e11</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=i-ADHoRe%203.0-fast%20and%20sensitive%20detection%20of%20genomic%20homology%20in%20extremely%20large%20data%20sets&amp;author=S.%20Proost&amp;author=J.%20Fostier&amp;author=D.%20De%20Witte&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e11" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr955" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr955" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr955"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22102584" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:i-ADHoRe%203.0-fast%20and%20sensitive%20detection%20of%20genomic%20homology%20in%20extremely%20large%20data%20sets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Mackey</div> <div class="given-names">A.J.</div>
</div>, <div class="name string-name">
<div class="surname">Stoeckert</div> <div class="given-names">C.J.</div>
</div>, <div class="name string-name">
<div class="surname">Roos</div> <div class="given-names">D.S.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">OrthoMCL-DB: querying a comprehensive multi-species collection of ortholog groups</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">D363</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=OrthoMCL-DB%3A%20querying%20a%20comprehensive%20multi-species%20collection%20of%20ortholog%20groups&amp;author=F.%20Chen&amp;author=A.J.%20Mackey&amp;author=C.J.%20Stoeckert&amp;author=D.S.%20Roos&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=D363-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkj123" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj123" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkj123"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381887" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoMCL-DB%3A%20querying%20a%20comprehensive%20multi-species%20collection%20of%20ortholog%20groups&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lyons</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Freeling</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">How to usefully compare homologous plant genes and chromosomes as DNA sequences</div>, <div class="source">Plant J</div> ., <div class="volume">53</div>, <div class="fpage">661</div>–<div class="lpage">73</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=How%20to%20usefully%20compare%20homologous%20plant%20genes%20and%20chromosomes%20as%20DNA%20sequences&amp;author=E.%20Lyons&amp;author=M.%20Freeling&amp;publication_year=2008&amp;journal=Plant%20J&amp;volume=53&amp;pages=661-73" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1365-313X.2007.03326.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03326.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1365-313X.2007.03326.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18269575" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:How%20to%20usefully%20compare%20homologous%20plant%20genes%20and%20chromosomes%20as%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Vandepoele</div> <div class="given-names">K.</div>
</div></div> <div class="year">2017</div>, 
<div class="article-title">A guide to the PLAZA 3.0 plant comparative genomic database</div>, <div class="source">Methods Mol. Biol</div> ., <div class="volume">1533</div>, <div class="fpage">183</div>–<div class="lpage">200</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20guide%20to%20the%20PLAZA%203.0%20plant%20comparative%20genomic%20database&amp;author=K.%20Vandepoele&amp;publication_year=2017&amp;journal=Methods%20Mol.%20Biol&amp;volume=1533&amp;pages=183-200" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/978-1-4939-6658-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-4939-6658-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2F978-1-4939-6658-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27987171" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20guide%20to%20the%20PLAZA%203.0%20plant%20comparative%20genomic%20database&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grin</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Linke</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">GCView: the genomic context viewer for protein homology searches</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">353</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GCView%3A%20the%20genomic%20context%20viewer%20for%20protein%20homology%20searches&amp;author=I.%20Grin&amp;author=D.%20Linke&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=353-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr364" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr364" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr364"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GCView%3A%20the%20genomic%20context%20viewer%20for%20protein%20homology%20searches&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wall</div> <div class="given-names">P.K.</div>
</div>, <div class="name string-name">
<div class="surname">Leebens-Mack</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Müller</div> <div class="given-names">K.F.</div>
</div>, <div class="name string-name">
<div class="surname">Field</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Altman</div> <div class="given-names">N.S.</div>
</div>, <div class="name string-name">
<div class="surname">Depamphilis</div> <div class="given-names">C.W.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">PlantTribes: a gene and gene family resource for comparative genomics in plants</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">970</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PlantTribes%3A%20a%20gene%20and%20gene%20family%20resource%20for%20comparative%20genomics%20in%20plants&amp;author=P.K.%20Wall&amp;author=J.%20Leebens-Mack&amp;author=K.F.%20M%C3%BCller&amp;author=D.%20Field&amp;author=N.S.%20Altman&amp;author=C.W.%20Depamphilis&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=970-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkm972" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm972" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkm972"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18096613" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PlantTribes%3A%20a%20gene%20and%20gene%20family%20resource%20for%20comparative%20genomics%20in%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Haudry</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Platts</div> <div class="given-names">A.A.E.</div>
</div>, <div class="name string-name">
<div class="surname">Vello</div> <div class="given-names">E.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions</div>, <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">891</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20atlas%20of%20over%2090%2C000%20conserved%20noncoding%20sequences%20provides%20insight%20into%20crucifer%20regulatory%20regions&amp;author=A.%20Haudry&amp;author=A.A.E.%20Platts&amp;author=E.%20Vello&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=891-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.2684" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2684" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.2684"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23817568" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20atlas%20of%20over%2090%2C000%20conserved%20noncoding%20sequences%20provides%20insight%20into%20crucifer%20regulatory%20regions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cheng</div> <div class="given-names">C.-Y.</div>
</div>, <div class="name string-name">
<div class="surname">Krishnakumar</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Chan</div> <div class="given-names">A.P.</div>
</div>, <div class="name string-name">
<div class="surname">Thibaud-Nissen</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Schobel</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Town</div> <div class="given-names">C.D.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Araport11: a complete reannotation of the <em>Arabidopsis thaliana</em> reference genome</div>, <div class="source">Plant J</div> ., <div class="volume">89</div>, <div class="fpage">789</div>–<div class="lpage">804</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Araport11%3A%20a%20complete%20reannotation%20of%20the%20Arabidopsis%20thaliana%20reference%20genome&amp;author=C.-Y.%20Cheng&amp;author=V.%20Krishnakumar&amp;author=A.P.%20Chan&amp;author=F.%20Thibaud-Nissen&amp;author=S.%20Schobel&amp;author=C.D.%20Town&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=89&amp;pages=789-804" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.13415" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.13415" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.13415"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27862469" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Araport11%3A%20a%20complete%20reannotation%20of%20the%20Arabidopsis%20thaliana%20reference%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dassanayake</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Oh</div> <div class="given-names">D.-H.</div>
</div>, <div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">J.S.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">The genome of the extremophile crucifer <em>Thellungiella parvula</em>
</div>, <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">913</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20extremophile%20crucifer%20Thellungiella%20parvula&amp;author=M.%20Dassanayake&amp;author=D.-H.%20Oh&amp;author=J.S.%20Haas&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=913-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.889" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.889" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.889"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21822265" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genome%20of%20the%20extremophile%20crucifer%20Thellungiella%20parvula&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">H.-J.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.-Y.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Insights into salt tolerance from the genome of <em>Thellungiella salsuginea</em>
</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">109</div>, <div class="fpage">12219</div>–<div class="lpage">24</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20salt%20tolerance%20from%20the%20genome%20of%20Thellungiella%20salsuginea&amp;author=H.-J.%20Wu&amp;author=Z.%20Zhang&amp;author=J.-Y.%20Wang&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=109&amp;pages=12219-24" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1209954109" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1209954109" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1209954109"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22778405" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insights%20into%20salt%20tolerance%20from%20the%20genome%20of%20Thellungiella%20salsuginea&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Jarvis</div> <div class="given-names">D.E.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">The reference genome of the halophytic plant <em>Eutrema salsugineum</em>
</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">4</div>, <div class="fpage">46</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genome%20of%20the%20halophytic%20plant%20Eutrema%20salsugineum&amp;author=R.%20Yang&amp;author=D.E.%20Jarvis&amp;author=H.%20Chen&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci&amp;volume=4&amp;pages=46" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23518688" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20reference%20genome%20of%20the%20halophytic%20plant%20Eutrema%20salsugineum&amp;aulast=Yang&amp;date=2013&amp;spage=46&amp;volume=4" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20reference%20genome%20of%20the%20halophytic%20plant%20Eutrema%20salsugineum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Orsini</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">D'Urzo</div> <div class="given-names">M.P.</div>
</div>, <div class="name string-name">
<div class="surname">Inan</div> <div class="given-names">G.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">A comparative study of salt tolerance parameters in 11 wild relatives of <em>Arabidopsis thaliana</em>
</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">61</div>, <div class="fpage">3787</div>–<div class="lpage">98</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20study%20of%20salt%20tolerance%20parameters%20in%2011%20wild%20relatives%20of%20Arabidopsis%20thaliana&amp;author=F.%20Orsini&amp;author=M.P.%20D'Urzo&amp;author=G.%20Inan&amp;publication_year=2010&amp;journal=J.%20Exp.%20Bot&amp;volume=61&amp;pages=3787-98" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/erq188" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferq188" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Ferq188"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20595237" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20comparative%20study%20of%20salt%20tolerance%20parameters%20in%2011%20wild%20relatives%20of%20Arabidopsis%20thaliana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Inan</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Salt cress. A halophyte and cryophyte <em>Arabidopsis</em> relative model system and its applicability to molecular genetic analyses of growth and development of extremophiles</div>, <div class="source">Plant Physiol</div> ., <div class="volume">135</div>, <div class="fpage">1718</div>–<div class="lpage">37</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Salt%20cress.%20A%20halophyte%20and%20cryophyte%20Arabidopsis%20relative%20model%20system%20and%20its%20applicability%20to%20molecular%20genetic%20analyses%20of%20growth%20and%20development%20of%20extremophiles&amp;author=G.%20Inan&amp;author=Q.%20Zhang&amp;author=P.%20Li&amp;publication_year=2004&amp;journal=Plant%20Physiol&amp;volume=135&amp;pages=1718-37" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.104.041723" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.041723" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.104.041723"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247369" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Salt%20cress.%20A%20halophyte%20and%20cryophyte%20Arabidopsis%20relative%20model%20system%20and%20its%20applicability%20to%20molecular%20genetic%20analyses%20of%20growth%20and%20development%20of%20extremophiles&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Amtmann</div> <div class="given-names">A.</div>
</div></div> <div class="year">2009</div>, 
<div class="article-title">Learning from evolution: <em>Thellungiella</em> generates new knowledge on essential and critical components of abiotic stress tolerance in plants</div>, <div class="source">Mol. Plant</div> , <div class="volume">2</div>, <div class="fpage">3</div>–<div class="lpage">12</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Learning%20from%20evolution%3A%20Thellungiella%20generates%20new%20knowledge%20on%20essential%20and%20critical%20components%20of%20abiotic%20stress%20tolerance%20in%20plants&amp;author=A.%20Amtmann&amp;publication_year=2009&amp;journal=Mol.%20Plant&amp;volume=2&amp;pages=3-12" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/mp/ssn094" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fssn094" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmp%2Fssn094"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19529830" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Learning%20from%20evolution%3A%20Thellungiella%20generates%20new%20knowledge%20on%20essential%20and%20critical%20components%20of%20abiotic%20stress%20tolerance%20in%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dittami</div> <div class="given-names">S.M.</div>
</div>, <div class="name string-name">
<div class="surname">Tonon</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Genomes of extremophile crucifers: new platforms for comparative genomics and beyond</div>, <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">166</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomes%20of%20extremophile%20crucifers%3A%20new%20platforms%20for%20comparative%20genomics%20and%20beyond&amp;author=S.M.%20Dittami&amp;author=T.%20Tonon&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=166" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2012-13-8-166" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-8-166" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2012-13-8-166"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22897896" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomes%20of%20extremophile%20crucifers%3A%20new%20platforms%20for%20comparative%20genomics%20and%20beyond&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oh</div> <div class="given-names">D.-H.</div>
</div>, <div class="name string-name">
<div class="surname">Dassanayake</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Bohnert</div> <div class="given-names">H.J.</div>
</div>, <div class="name string-name">
<div class="surname">Cheeseman</div> <div class="given-names">J.M.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Life at the extreme: lessons from the genome</div>, <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">241</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Life%20at%20the%20extreme%3A%20lessons%20from%20the%20genome&amp;author=D.-H.%20Oh&amp;author=M.%20Dassanayake&amp;author=H.J.%20Bohnert&amp;author=J.M.%20Cheeseman&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=241" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb4003" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb4003" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb4003"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22390828" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Life%20at%20the%20extreme%3A%20lessons%20from%20the%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Paterson</div> <div class="given-names">A.H.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">MCScanX-transposed: detecting transposed gene duplications based on multiple colinearity scans</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">1458</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MCScanX-transposed%3A%20detecting%20transposed%20gene%20duplications%20based%20on%20multiple%20colinearity%20scans&amp;author=Y.%20Wang&amp;author=J.%20Li&amp;author=A.H.%20Paterson&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=1458-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt150" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt150" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt150"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23539305" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MCScanX-transposed%3A%20detecting%20transposed%20gene%20duplications%20based%20on%20multiple%20colinearity%20scans&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Camacho</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Coulouris</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Avagyan</div> <div class="given-names">V.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">BLAST+: architecture and applications</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">10</div>, <div class="fpage">421</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BLAST%2B%3A%20architecture%20and%20applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;publication_year=2009&amp;journal=BMC%20Bioinform&amp;volume=10&amp;pages=421" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-10-421" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-421" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-10-421"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BLAST%2B%3A%20architecture%20and%20applications&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Stoeckert</div> <div class="given-names">C.J.</div>
</div>, <div class="name string-name">
<div class="surname">Roos</div> <div class="given-names">D.S.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes. <em>Genome</em>
</div> <div class="source">Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes.%20Genome&amp;aulast=Li&amp;date=2003&amp;spage=2178&amp;epage=89&amp;volume=13" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes.%20Genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dunn</div> <div class="given-names">C.W.</div>
</div>, <div class="name string-name">
<div class="surname">Howison</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Zapata</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Agalma: an automated phylogenomics workflow</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">14</div>, <div class="fpage">330</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Agalma%3A%20an%20automated%20phylogenomics%20workflow&amp;author=C.W.%20Dunn&amp;author=M.%20Howison&amp;author=F.%20Zapata&amp;publication_year=2013&amp;journal=BMC%20Bioinform&amp;volume=14&amp;pages=330" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-14-330" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-330" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-14-330"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Agalma%3A%20an%20automated%20phylogenomics%20workflow&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">Z.</div>
</div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsm088"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">van Dongen</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Abreu-Goodger</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Using MCL to extract clusters from networks</div>, <div class="source">Methods Mol. Biol</div> ., <div class="volume">804</div>, <div class="fpage">281</div>–<div class="lpage">95</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Using%20MCL%20to%20extract%20clusters%20from%20networks&amp;author=S.%20van%20Dongen&amp;author=C.%20Abreu-Goodger&amp;publication_year=2012&amp;journal=Methods%20Mol.%20Biol&amp;volume=804&amp;pages=281-95" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/978-1-61779-361-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-61779-361-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2F978-1-61779-361-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22144159" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20MCL%20to%20extract%20clusters%20from%20networks&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pertea</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Pertea</div> <div class="given-names">G.M.</div>
</div>, <div class="name string-name">
<div class="surname">Leek</div> <div class="given-names">J.T.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">11</div>, <div class="fpage">1650</div>–<div class="lpage">67</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcript-level%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20HISAT%2C%20StringTie%20and%20Ballgown&amp;author=M.%20Pertea&amp;author=D.%20Kim&amp;author=G.M.%20Pertea&amp;author=J.T.%20Leek&amp;author=S.L.%20Salzberg&amp;publication_year=2016&amp;journal=Nat.%20Protoc&amp;volume=11&amp;pages=1650-67" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nprot.2016.095" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2016.095" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnprot.2016.095"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27560171" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcript-level%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20HISAT%2C%20StringTie%20and%20Ballgown&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B56" class="ref-content ">
<div class="label">56</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Emms</div> <div class="given-names">D.M.</div>
</div>, <div class="name string-name">
<div class="surname">Kelly</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">157</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;author=D.M.%20Emms&amp;author=S.%20Kelly&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=157" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-015-0721-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0721-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0721-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26243257" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B57" class="ref-content ">
<div class="label">57</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ling</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">He</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Xin</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Detecting gene clusters under evolutionary constraint in a large number of genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">571</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Detecting%20gene%20clusters%20under%20evolutionary%20constraint%20in%20a%20large%20number%20of%20genomes&amp;author=X.%20Ling&amp;author=X.%20He&amp;author=D.%20Xin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=571-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp027" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp027" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp027"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19158161" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Detecting%20gene%20clusters%20under%20evolutionary%20constraint%20in%20a%20large%20number%20of%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B58" class="ref-content ">
<div class="label">58</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rödelsperger</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Dieterich</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">CYNTENATOR: progressive gene order alignment of 17 vertebrate genomes</div>, <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e8861</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CYNTENATOR%3A%20progressive%20gene%20order%20alignment%20of%2017%20vertebrate%20genomes&amp;author=C.%20R%C3%B6delsperger&amp;author=C.%20Dieterich&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e8861" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0008861" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008861" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0008861"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20126624" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CYNTENATOR%3A%20progressive%20gene%20order%20alignment%20of%2017%20vertebrate%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B59" class="ref-content ">
<div class="label">59</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Schranz</div> <div class="given-names">M.E.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Network approaches for plant phylogenomic synteny analysis</div>, <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">36</div>, <div class="fpage">129</div>–<div class="lpage">34</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Network%20approaches%20for%20plant%20phylogenomic%20synteny%20analysis&amp;author=T.%20Zhao&amp;author=M.E.%20Schranz&amp;publication_year=2017&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=36&amp;pages=129-34" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.pbi.2017.03.001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2017.03.001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.pbi.2017.03.001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28327435" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Network%20approaches%20for%20plant%20phylogenomic%20synteny%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B60" class="ref-content ">
<div class="label">60</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hastings</div> <div class="given-names">P.J.</div>
</div>, <div class="name string-name">
<div class="surname">Lupski</div> <div class="given-names">J.R.</div>
</div>, <div class="name string-name">
<div class="surname">Rosenberg</div> <div class="given-names">S.M.</div>
</div>, <div class="name string-name">
<div class="surname">Ira</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Mechanisms of change in gene copy number</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">10</div>, <div class="fpage">551</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20change%20in%20gene%20copy%20number&amp;author=P.J.%20Hastings&amp;author=J.R.%20Lupski&amp;author=S.M.%20Rosenberg&amp;author=G.%20Ira&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet&amp;volume=10&amp;pages=551-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg2593" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2593" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg2593"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19597530" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mechanisms%20of%20change%20in%20gene%20copy%20number&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B61" class="ref-content ">
<div class="label">61</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Robberecht</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Voet</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Esteki</div> <div class="given-names">M.Z.</div>
</div>, <div class="name string-name">
<div class="surname">Nowakowska</div> <div class="given-names">B.A.</div>
</div>, <div class="name string-name">
<div class="surname">Vermeesch</div> <div class="given-names">J.R.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Nonallelic homologous recombination between retrotransposable elements is a driver of <em>de novo</em> unbalanced translocations</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">411</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nonallelic%20homologous%20recombination%20between%20retrotransposable%20elements%20is%20a%20driver%20of%20de%20novo%20unbalanced%20translocations&amp;author=C.%20Robberecht&amp;author=T.%20Voet&amp;author=M.Z.%20Esteki&amp;author=B.A.%20Nowakowska&amp;author=J.R.%20Vermeesch&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=411-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.145631.112" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.145631.112" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.145631.112"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23212949" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nonallelic%20homologous%20recombination%20between%20retrotransposable%20elements%20is%20a%20driver%20of%20de%20novo%20unbalanced%20translocations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B62" class="ref-content ">
<div class="label">62</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cusack</div> <div class="given-names">B.P.</div>
</div>, <div class="name string-name">
<div class="surname">Wolfe</div> <div class="given-names">K.H.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Not born equal: increased rate asymmetry in relocated and retrotransposed rodent gene duplicates</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">679</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Not%20born%20equal%3A%20increased%20rate%20asymmetry%20in%20relocated%20and%20retrotransposed%20rodent%20gene%20duplicates&amp;author=B.P.%20Cusack&amp;author=K.H.%20Wolfe&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=679-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msl199" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl199" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsl199"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17179139" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Not%20born%20equal%3A%20increased%20rate%20asymmetry%20in%20relocated%20and%20retrotransposed%20rodent%20gene%20duplicates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B63" class="ref-content ">
<div class="label">63</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Abdelkarim</div> <div class="given-names">B.T.M.</div>
</div>, <div class="name string-name">
<div class="surname">Maranda</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Drouin</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">The fate of retrotransposed processed genes in <em>Arabidopsis thaliana</em>
</div>, <div class="source">Gene</div> , <div class="volume">609</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20fate%20of%20retrotransposed%20processed%20genes%20in%20Arabidopsis%20thaliana&amp;author=B.T.M.%20Abdelkarim&amp;author=V.%20Maranda&amp;author=G.%20Drouin&amp;publication_year=2017&amp;journal=Gene&amp;volume=609&amp;pages=1-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gene.2017.01.029" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2017.01.029" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2017.01.029"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28131821" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20fate%20of%20retrotransposed%20processed%20genes%20in%20Arabidopsis%20thaliana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B64" class="ref-content ">
<div class="label">64</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Morgante</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Brunner</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Pea</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Fengler</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Zuccolo</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rafalski</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Gene duplication and exon shuffling by helitron-like transposons generate intraspecies diversity in maize</div>, <div class="source">Nat. Genet</div> ., <div class="volume">37</div>, <div class="fpage">997</div>–<div class="lpage">1002</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20exon%20shuffling%20by%20helitron-like%20transposons%20generate%20intraspecies%20diversity%20in%20maize&amp;author=M.%20Morgante&amp;author=S.%20Brunner&amp;author=G.%20Pea&amp;author=K.%20Fengler&amp;author=A.%20Zuccolo&amp;author=A.%20Rafalski&amp;publication_year=2005&amp;journal=Nat.%20Genet&amp;volume=37&amp;pages=997-1002" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng1615" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1615" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng1615"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056225" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20duplication%20and%20exon%20shuffling%20by%20helitron-like%20transposons%20generate%20intraspecies%20diversity%20in%20maize&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B65" class="ref-content ">
<div class="label">65</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Woodhouse</div> <div class="given-names">M.R.</div>
</div>, <div class="name string-name">
<div class="surname">Pedersen</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Freeling</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Transposed genes in <em>Arabidopsis</em> are often associated with flanking repeats</div>, <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transposed%20genes%20in%20Arabidopsis%20are%20often%20associated%20with%20flanking%20repeats&amp;author=M.R.%20Woodhouse&amp;author=B.%20Pedersen&amp;author=M.%20Freeling&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=26" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1000949" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000949" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1000949"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transposed%20genes%20in%20Arabidopsis%20are%20often%20associated%20with%20flanking%20repeats&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B66" class="ref-content ">
<div class="label">66</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ali</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">H.C.</div>
</div>, <div class="name string-name">
<div class="surname">Ali</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">TsHKT1;2, a HKT1 Homolog from the extremophile <em>Arabidopsis</em> relative <em>Thellungiella salsuginea</em>, shows K<sup>+</sup> specificity in the presence of NaCl</div>, <div class="source">Plant Physiol</div> ., <div class="volume">158</div>, <div class="fpage">1463</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=TsHKT1%3B2%2C%20a%20HKT1%20Homolog%20from%20the%20extremophile%20Arabidopsis%20relative%20Thellungiella%20salsuginea%2C%20shows%20K%2B%20specificity%20in%20the%20presence%20of%20NaCl&amp;author=Z.%20Ali&amp;author=H.C.%20Park&amp;author=A.%20Ali&amp;publication_year=2012&amp;journal=Plant%20Physiol&amp;volume=158&amp;pages=1463-74" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.111.193110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.111.193110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.111.193110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22238420" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:TsHKT1%3B2%2C%20a%20HKT1%20Homolog%20from%20the%20extremophile%20Arabidopsis%20relative%20Thellungiella%20salsuginea%2C%20shows%20K%2B%20specificity%20in%20the%20presence%20of%20NaCl&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B67" class="ref-content ">
<div class="label">67</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oh</div> <div class="given-names">D.-H.</div>
</div>, <div class="name string-name">
<div class="surname">Leidi</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Q.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Loss of halophytism by interference with SOS1 expression</div>, <div class="source">Plant Physiol</div> ., <div class="volume">151</div>, <div class="fpage">210</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Loss%20of%20halophytism%20by%20interference%20with%20SOS1%20expression&amp;author=D.-H.%20Oh&amp;author=E.%20Leidi&amp;author=Q.%20Zhang&amp;publication_year=2009&amp;journal=Plant%20Physiol&amp;volume=151&amp;pages=210-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.109.137802" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.137802" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.109.137802"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19571313" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Loss%20of%20halophytism%20by%20interference%20with%20SOS1%20expression&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B68" class="ref-content ">
<div class="label">68</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Park</div> <div class="given-names">M.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Chung</div> <div class="given-names">M.S.</div>
</div>, <div class="name string-name">
<div class="surname">Koh</div> <div class="given-names">H.S.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">D.J.</div>
</div>, <div class="name string-name">
<div class="surname">Ahn</div> <div class="given-names">S.J.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">C.S.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Isolation and functional characterization of the <em>Arabidopsis salt tolerance 32 (AtSAT32)</em> gene associated with salt tolerance and ABA signaling</div>, <div class="source">Physiol. Plant</div> ., <div class="volume">135</div>, <div class="fpage">426</div>–<div class="lpage">35</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20functional%20characterization%20of%20the%20Arabidopsis%20salt%20tolerance%2032%20(AtSAT32)%20gene%20associated%20with%20salt%20tolerance%20and%20ABA%20signaling&amp;author=M.Y.%20Park&amp;author=M.S.%20Chung&amp;author=H.S.%20Koh&amp;author=D.J.%20Lee&amp;author=S.J.%20Ahn&amp;author=C.S.%20Kim&amp;publication_year=2009&amp;journal=Physiol.%20Plant&amp;volume=135&amp;pages=426-35" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/ppl.2009.135.issue-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppl.2009.135.issue-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fppl.2009.135.issue-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19210750" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Isolation%20and%20functional%20characterization%20of%20the%20Arabidopsis%20salt%20tolerance%2032%20(AtSAT32)%20gene%20associated%20with%20salt%20tolerance%20and%20ABA%20signaling&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B69" class="ref-content ">
<div class="label">69</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hao</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Baltimore</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">RNA splicing regulates the temporal order of TNF-induced gene expression</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">110</div>, <div class="fpage">11934</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RNA%20splicing%20regulates%20the%20temporal%20order%20of%20TNF-induced%20gene%20expression&amp;author=S.%20Hao&amp;author=D.%20Baltimore&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=110&amp;pages=11934-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1309990110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1309990110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1309990110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23812748" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RNA%20splicing%20regulates%20the%20temporal%20order%20of%20TNF-induced%20gene%20expression&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B70" class="ref-content ">
<div class="label">70</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">X.N.</div>
</div>, <div class="name string-name">
<div class="surname">Niu</div> <div class="given-names">D.K.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Higher frequency of intron loss from the promoter proximally paused genes of <em>Drosophila melanogaster</em>: evidence consistent with delays in intron splicing as a selective force</div>, <div class="source">Fly (Austin)</div> , <div class="volume">8</div>, <div class="fpage">120</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Higher%20frequency%20of%20intron%20loss%20from%20the%20promoter%20proximally%20paused%20genes%20of%20Drosophila%20melanogaster%3A%20evidence%20consistent%20with%20delays%20in%20intron%20splicing%20as%20a%20selective%20force&amp;author=L.%20Jiang&amp;author=X.N.%20Li&amp;author=D.K.%20Niu&amp;publication_year=2014&amp;journal=Fly%20(Austin)&amp;volume=8&amp;pages=120-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.4161/fly.29489" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.29489" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.4161%2Ffly.29489"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25483256" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Higher%20frequency%20of%20intron%20loss%20from%20the%20promoter%20proximally%20paused%20genes%20of%20Drosophila%20melanogaster%3A%20evidence%20consistent%20with%20delays%20in%20intron%20splicing%20as%20a%20selective%20force&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B71" class="ref-content ">
<div class="label">71</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">William Roy</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Gilbert</div> <div class="given-names">W.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">The evolution of spliceosomal introns: patterns, puzzles and progress</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">7</div>, <div class="fpage">211</div>–<div class="lpage">21</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20spliceosomal%20introns%3A%20patterns%2C%20puzzles%20and%20progress&amp;author=S.%20William%20Roy&amp;author=W.%20Gilbert&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet&amp;volume=7&amp;pages=211-21" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg1807" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1807" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg1807"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16485020" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20evolution%20of%20spliceosomal%20introns%3A%20patterns%2C%20puzzles%20and%20progress&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B72" class="ref-content ">
<div class="label">72</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Guerinot</div> <div class="given-names">M.L.</div>
</div></div> <div class="year">2000</div>, 
<div class="article-title">The ZIP family of metal transporters</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1465</div>, <div class="fpage">190</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20ZIP%20family%20of%20metal%20transporters&amp;author=M.L.%20Guerinot&amp;publication_year=2000&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1465&amp;pages=190-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0005-2736(00)00138-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0005-2736(00)00138-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0005-2736(00)00138-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10748254" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20ZIP%20family%20of%20metal%20transporters&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy035-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy035-B73" class="ref-content ">
<div class="label">73</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cakmak</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Kalayci</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Ekiz</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Braun</div> <div class="given-names">H.J.</div>
</div>, <div class="name string-name">
<div class="surname">Kilinç</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Yilmaz</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">1999</div>, 
<div class="article-title">Zinc deficiency as a practical problem in plant and human nutrition in Turkey: a NATO-science for stability project</div>, <div class="source">Field Crops Res</div> ., <div class="volume">60</div>, <div class="fpage">175</div>–<div class="lpage">88</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Zinc%20deficiency%20as%20a%20practical%20problem%20in%20plant%20and%20human%20nutrition%20in%20Turkey%3A%20a%20NATO-science%20for%20stability%20project&amp;author=I.%20Cakmak&amp;author=M.%20Kalayci&amp;author=H.%20Ekiz&amp;author=H.J.%20Braun&amp;author=Y.%20Kilin%C3%A7&amp;author=A.%20Yilmaz&amp;publication_year=1999&amp;journal=Field%20Crops%20Res&amp;volume=60&amp;pages=175-88" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0378-4290(98)00139-7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-4290(98)00139-7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0378-4290(98)00139-7"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Zinc%20deficiency%20as%20a%20practical%20problem%20in%20plant%20and%20human%20nutrition%20in%20Turkey%3A%20a%20NATO-science%20for%20stability%20project&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5151297&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy035/2/dsy035_supp.zip?Expires=1576701153&amp;Signature=o~D6SGRsYRs8dHdu2r2mBEr1wf4Rr-wNpU4PLJkdmMVnWkeVh-pO8SHaGm~091D~M7matlhafsN5OGx0xR7nCmsIzAxcVUZS8ysS6SkhAGln8eNmu~fJj5-JjdiqiJoG~Cr4KWEvQrin3140rue-PbCEtzde4nuxJA~6WsCOgf36qM8zD8EThtzTsGbRiAK6ABvUmj76t~wu4yuDdN8~NOFzq1iv-MTbpS93rP-Tmlc6muuqo5RiW-R8ZqdMjBf8oZtWEj-1hYcH4hBWCzlU1jsKyiMBnPZJRLPrfMKt4lPD-D095Fq0K8Zp31czp~BOhigumknhRN9e91AAer1TKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,152</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy035"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5151297"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f1%2f21%2f5151297">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F1%2F21%2F5151297" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5151297">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Landscape of gene transposition–duplication within the Brassicaceae family | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy035");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f21%2f5151297">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
