<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Evaluation and application of RNA-Seq by MinION | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/1/55/5194314","@type":"ScholarlyArticle","name":"Evaluation and application of RNA-Seq by MinION","datePublished":"2018-11-20","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2019-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy038","keywords":["nanopore sequencing","transcriptome","transcript isoform","allelic expression"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/1/55/5194314","author":[{"name":"Seki, Masahide","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Katsumata, Eri","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Suzuki, Ayako","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Sereewattanawoot, Sarun","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Sakamoto, Yoshitaka","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Mizushima-Sugano, Junko","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan  Department of Chemistry and Life Science, School of Advanced Engineering, Kogakuin University, Shinjuku-ku, Tokyo, Japan","@type":"Person"},{"name":"Sugano, Sumio","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan  Department of Molecular Epidemiology, Medical Research Institute, Tokyo Medical and Dental University, Bunkyo-ku, Tokyo, Japan","@type":"Person"},{"name":"Kohno, Takashi","affiliation":"Division of Genome Biology, National Cancer Center Research Institute, Chuo-Ku, Tokyo, Japan","@type":"Person"},{"name":"Frith, Martin C","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan  Artificial Intelligence Research Center AIST, Koto-ku, Tokyo, Japan  Computational Bio Big-Data Open Innovation Laboratory (CBBD-OIL), AIST, Shinjuku-ku, Tokyo, Japan","@type":"Person"},{"name":"Tsuchihara, Katsuya","affiliation":"Division of Translational Informatics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"}],"description":"Abstract.  The current RNA-Seq method analyses fragments of mRNAs, from which it is occasionally difficult to reconstruct the entire transcript structure. Here,","pageStart":"55","pageEnd":"65","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636230619&Signature=nTL5O2XX1XZVWHV91bsAX6noAlAzkZbQIA49Hveh~eq7FV0tCA6k26BtyXZ0XBzNe4hpxfO~An1yLLK-UC~an-3A0-W91Iv0Un86ty6DO2UsRwUsmfgwXTM-r-YhcNsUBko2NcC3uEaBBVPYiePzCXVP9OUKDpRwsg2gRlQxH34UxaTFNmeYWv2fWC2pflA-MA04viUy1~iORxzLu5gTeZTyBQ-3daZxHsoUQfgXP4EdubdkUiw--Bpoc9RWt5IuFwcLHFXRVUSfPT2x6M4DwsxAUtLz6yxyJ0uEmUXYRAs80PoGaFtqTX~IQbj0Hb10oRHIQ4-gwSTMo54FG45apw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Evaluation and application of RNA-Seq by MinION","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636230619&Signature=nTL5O2XX1XZVWHV91bsAX6noAlAzkZbQIA49Hveh~eq7FV0tCA6k26BtyXZ0XBzNe4hpxfO~An1yLLK-UC~an-3A0-W91Iv0Un86ty6DO2UsRwUsmfgwXTM-r-YhcNsUBko2NcC3uEaBBVPYiePzCXVP9OUKDpRwsg2gRlQxH34UxaTFNmeYWv2fWC2pflA-MA04viUy1~iORxzLu5gTeZTyBQ-3daZxHsoUQfgXP4EdubdkUiw--Bpoc9RWt5IuFwcLHFXRVUSfPT2x6M4DwsxAUtLz6yxyJ0uEmUXYRAs80PoGaFtqTX~IQbj0Hb10oRHIQ4-gwSTMo54FG45apw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Evaluation and application of RNA-Seq by MinION">
<meta property="og:description" content="Abstract.  The current RNA-Seq method analyses fragments of mRNAs, from which it is occasionally difficult to reconstruct the entire transcript structure. Here,">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/1/55/5194314">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636230619&amp;Signature=nTL5O2XX1XZVWHV91bsAX6noAlAzkZbQIA49Hveh~eq7FV0tCA6k26BtyXZ0XBzNe4hpxfO~An1yLLK-UC~an-3A0-W91Iv0Un86ty6DO2UsRwUsmfgwXTM-r-YhcNsUBko2NcC3uEaBBVPYiePzCXVP9OUKDpRwsg2gRlQxH34UxaTFNmeYWv2fWC2pflA-MA04viUy1~iORxzLu5gTeZTyBQ-3daZxHsoUQfgXP4EdubdkUiw--Bpoc9RWt5IuFwcLHFXRVUSfPT2x6M4DwsxAUtLz6yxyJ0uEmUXYRAs80PoGaFtqTX~IQbj0Hb10oRHIQ4-gwSTMo54FG45apw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2018-11-20">
<meta name="citation_author" content="Seki, Masahide">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Katsumata, Eri">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Suzuki, Ayako">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Sereewattanawoot, Sarun">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Sakamoto, Yoshitaka">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Mizushima-Sugano, Junko">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author_institution" content="Department of Chemistry and Life Science, School of Advanced Engineering, Kogakuin University, Shinjuku-ku, Tokyo, Japan">
<meta name="citation_author" content="Sugano, Sumio">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author_institution" content="Department of Molecular Epidemiology, Medical Research Institute, Tokyo Medical and Dental University, Bunkyo-ku, Tokyo, Japan">
<meta name="citation_author" content="Kohno, Takashi">
<meta name="citation_author_institution" content="Division of Genome Biology, National Cancer Center Research Institute, Chuo-Ku, Tokyo, Japan">
<meta name="citation_author" content="Frith, Martin C">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_author_institution" content="Artificial Intelligence Research Center AIST, Koto-ku, Tokyo, Japan">
<meta name="citation_author_institution" content="Computational Bio Big-Data Open Innovation Laboratory (CBBD-OIL), AIST, Shinjuku-ku, Tokyo, Japan">
<meta name="citation_author" content="Tsuchihara, Katsuya">
<meta name="citation_author_institution" content="Division of Translational Informatics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Suzuki, Yutaka">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan">
<meta name="citation_title" content="Evaluation and application of RNA-Seq by MinION">
<meta name="citation_firstpage" content="55">
<meta name="citation_lastpage" content="65">
<meta name="citation_doi" content="10.1093/dnares/dsy038">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/02/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=RNA-Seq: a revolutionary tool for transcriptomics;  citation_journal_title=Nat. Rev. Genet;  citation_year=2009;  citation_volume=10;  citation_pages=57-63; ">
<meta name="citation_reference" content="citation_title=Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing;  citation_journal_title=Nat. Genet;  citation_year=2008;  citation_volume=40;  citation_pages=1413-5; ">
<meta name="citation_reference" content="citation_title=Reliable identification of genomic variants from RNA-seq data;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2013;  citation_volume=93;  citation_pages=641-51; ">
<meta name="citation_reference" content="citation_title=Assessment of transcript reconstruction methods for RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1177-84; ">
<meta name="citation_reference" content="citation_title=Coming of age: ten years of next-generation sequencing technologies;  citation_journal_title=Nat. Rev. Genet;  citation_year=2016;  citation_volume=17;  citation_pages=333-51; ">
<meta name="citation_reference" content="citation_title=Haplotype phasing: existing methods and new developments;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=703-14; ">
<meta name="citation_reference" content="citation_title=A single-molecule long-read survey of the human transcriptome;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=1009-14; ">
<meta name="citation_reference" content="citation_title=PacBio sequencing and its applications;  citation_journal_title=Genomics. Proteomics Bioinformatics;  citation_year=2015;  citation_volume=13;  citation_pages=278-89; ">
<meta name="citation_reference" content="Pacific Biosciences. Revolutionize Genomics with SMRT Sequencing. https://www.pacb.com/wp-content/uploads/SMRT-Sequencing-Brochure-Revolutionize-genomics-with-SMRT-Sequencing.pdf (accessed 21 June 2018). ">
<meta name="citation_reference" content="citation_title=The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=239">
<meta name="citation_reference" content="citation_title=Nanopore sequencing and assembly of a human genome with ultra-long reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2018;  citation_volume=36;  citation_pages=338-45; ">
<meta name="citation_reference" content="citation_title=Benchmarking of the Oxford Nanopore MinION sequencing for quantitative and qualitative assessment of cDNA populations;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=31602">
<meta name="citation_reference" content="citation_title=Nanopore long-read RNAseq reveals widespread transcriptional variation among the surface receptors of individual B cells;  citation_journal_title=Nat. Commun;  citation_year=2017;  citation_volume=8;  citation_pages=16027">
<meta name="citation_reference" content="citation_title=Smart-seq2 for sensitive full-length transcriptome profiling in single cells;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1096-8; ">
<meta name="citation_reference" content="citation_title=BLAT—The BLAST-like alignment tool;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; ">
<meta name="citation_reference" content="citation_title=Aberrant transcriptional regulations in cancers: genome, transcriptome and epigenome analysis of lung adenocarcinoma cell lines;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=13557-72; ">
<meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=491-501; ">
<meta name="citation_reference" content="citation_title=Training alignment parameters for arbitrary sequencers with LAST-TRAIN;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_pages=926-8; ">
<meta name="citation_reference" content="citation_title=Split-alignment of genomes finds orthologies more accurately;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=106">
<meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; ">
<meta name="citation_reference" content="citation_title=Highly parallel direct RN A sequencing on an array of nanopores;  citation_journal_title=Nat. Methods;  citation_year=2018;  citation_volume=15;  citation_pages=201-6; ">
<meta name="citation_reference" content="citation_title=EGFR mutation and resistance of non–small-cell lung cancer to Gefitinib;  citation_journal_title=N. Engl. J. Med;  citation_year=2005;  citation_volume=352;  citation_pages=786-92; ">
<meta name="citation_reference" content="citation_title=Bidirectional imprinting of a single gene: gNAS1 encodes maternally, paternally, and biallelically derived proteins;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1998;  citation_volume=95;  citation_pages=15475-80; ">
<meta name="citation_reference" content="citation_title=The GNAS complex locus and human diseases associated with loss-of-function mutations or epimutations within this imprinted gene;  citation_journal_title=Horm. Res. Paediatr;  citation_year=2013;  citation_volume=80;  citation_pages=229-41; ">
<meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons, Inc, Hoboken, NJ;  citation_title=Current Protocols in Molecular Biology;  citation_year=2016; ">
<meta name="citation_reference" content="citation_title=Long-term expansion of alveolar stem cells derived from human iPS cells in organoids;  citation_journal_title=Nat. Methods;  citation_year=2017;  citation_volume=14;  citation_pages=1097-106; ">
<meta name="citation_reference" content="citation_title=Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer;  citation_journal_title=DNA Res;  citation_year=2017;  citation_volume=24;  citation_pages=585-96; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/1/55/28033447/dsy038.pdf">
<meta name="description" content="Abstract.  The current RNA-Seq method analyses fragments of mRNAs, from which it is occasionally difficult to reconstruct the entire transcript structure. Here,">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/1/55/5194314">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/1/55/5194314">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/1/55/5194314");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy038';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f55%2f5194314" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f55%2f5194314" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=114871"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1576706054&amp;Signature=vBWNddoR50MXunHaIOWR4NKGA3RzdnFj2w0qk3NvgIypxbeQEyInkkoEtdnasX6NtO2S2NYf3wqojJKdpkSFXETU8YoNv8da7ngvqlrRkx27avSNfvtlH9MM6arJXHCP4x1EGEd1qpfCP0KZ6NvYu4FFgfaXjimkXV8u1WHsuaW8YiKN2unIWtrryrrA0p4NYAcZr3k6hjEdf7~MV5W2ILlSUUxJe64AmhGJjhHplwvyT53I4xxJUHwHLOuGWmMVUAVTCG8Z6Ft8d9JyoaR2TXsCMWL7Tp6L0bHZ4xsRiMFopapA7x0wOLVPo~2Cy0erwXXcezAGNXmQA8A-lZh-yQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/1">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="131979765">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979765">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979767">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979767">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979773">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979773">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979789">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979789">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979819">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979819">4. Conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979821">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979821">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="131979829">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979829">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/1/45/5181390">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/1/67/5193709">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Evaluation and application of RNA-Seq by MinION
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahide Seki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahide+Seki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seki%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Seki%20Masahide%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Masahide Seki</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eri Katsumata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eri+Katsumata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katsumata%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Katsumata%20Eri%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Eri Katsumata</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayako Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayako+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Suzuki%20Ayako%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Ayako Suzuki</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sarun Sereewattanawoot</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sarun+Sereewattanawoot">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sereewattanawoot%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sereewattanawoot%20Sarun%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Sarun Sereewattanawoot</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitaka Sakamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitaka+Sakamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakamoto%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sakamoto%20Yoshitaka%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Yoshitaka Sakamoto</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junko Mizushima-Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
<div class="aff">Department of Chemistry and Life Science, School of Advanced Engineering, Kogakuin University, Shinjuku-ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Junko+Mizushima-Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mizushima-Sugano%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mizushima-Sugano%20Junko%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Junko Mizushima-Sugano</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumio Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
<div class="aff">Department of Molecular Epidemiology, Medical Research Institute, Tokyo Medical and Dental University, Bunkyo-ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumio+Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugano%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sugano%20Sumio%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Sumio Sugano</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Kohno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Genome Biology, National Cancer Center Research Institute, Chuo-Ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Kohno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohno%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kohno%20Takashi%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Takashi Kohno</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin C Frith</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
<div class="aff">Artificial Intelligence Research Center AIST, Koto-ku, Tokyo, Japan</div>
<div class="aff">Computational Bio Big-Data Open Innovation Laboratory (CBBD-OIL), AIST, Shinjuku-ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+C+Frith">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Frith%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Frith%20Martin%20C%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Martin C Frith</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuya Tsuchihara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Translational Informatics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuya+Tsuchihara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuchihara%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Tsuchihara%20Katsuya%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Katsuya Tsuchihara</a><span class="delimiter-hidden">, </span>
                                </span>
                                    <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                        <span>...</span> Show more
                                    </a>
                                    <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 4 7136 3607. Fax. +81 4 7136 3607. Email: <a href="mailto:ysuzuki@k.u-tokyo.ac.jp" target="_blank">ysuzuki@k.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Suzuki%20Yutaka%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Yutaka Suzuki</a><span class="delimiter"></span>
                                        </span>
                                    </div>
                            </div>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy038-FM1"><div class="fn"><div class="footnote-content">Masahide Seki and Eri Katsumata authors equally contributed to this article.</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 55–65, <a href="https://doi.org/10.1093/dnares/dsy038">https://doi.org/10.1093/dnares/dsy038</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">20 November 2018</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">22 June 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">15 October 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">20 November 2018</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5194314" href="/dnaresearch/article-pdf/26/1/55/28033447/dsy038.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masahide Seki, Eri Katsumata, Ayako Suzuki, Sarun Sereewattanawoot, Yoshitaka Sakamoto, Junko Mizushima-Sugano, Sumio Sugano, Takashi Kohno, Martin C Frith, Katsuya Tsuchihara, Yutaka Suzuki,  Evaluation and application of RNA-Seq by MinION, <em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 55–65, <a href="https://doi.org/10.1093/dnares/dsy038">https://doi.org/10.1093/dnares/dsy038</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5194314&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Evaluation%20and%20application%20of%20RNA-Seq%20by%20MinION&amp;publicationDate=2018-11-20&amp;volumeNum=26&amp;issueNum=1&amp;author=Seki%2C%20Masahide%3B%20Katsumata%2C%20Eri&amp;startPage=55&amp;endPage=65&amp;contentId=10.1093%2Fdnares%2Fdsy038&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f55%2f5194314" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="131979765" id="131979765" class="abstract-title">Abstract</h2>
<section class="abstract"><p>The current RNA-Seq method analyses fragments of mRNAs, from which it is occasionally difficult to reconstruct the entire transcript structure. Here, we performed and evaluated the recent procedure for full-length cDNA sequencing using the Nanopore sequencer MinION. We applied MinION RNA-Seq for various applications, which would not always be easy using the usual RNA-Seq by Illumina. First, we examined and found that even though the sequencing accuracy was still limited to 92.3%, practically useful RNA-Seq analysis is possible. Particularly, taking advantage of the long-read nature of MinION, we demonstrate the identification of splicing patterns and their combinations as a form of full-length cDNAs without losing precise information concerning their expression levels. Transcripts of fusion genes in cancer cells can also be identified and characterized. Furthermore, the full-length cDNA information can be used for phasing of the SNPs detected by WES on the transcripts, providing essential information to identify allele-specific transcriptional events. We constructed a catalogue of full-length cDNAs in seven major organs for two particular individuals and identified allele-specific transcription and splicing. Finally, we demonstrate that single-cell sequencing is also possible. RNA-Seq on the MinION platform should provide a novel approach that is complementary to the current RNA-Seq.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"nanopore sequencing"'>nanopore sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"transcript isoform"'>transcript isoform</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"allelic expression"'>allelic expression</a>
</div>                    <h2 scrollto-destination="131979767" id="131979767" class="section-title" data-legacy-id="SEC1">1. Introduction</h2>
<p>RNA-Seq has revolutionized transcriptome analysis. Without detailed information for each of the transcripts, that is, comprehensive information on the transcripts, the transcriptome information can be obtained by massively parallel sequencing of mRNA fragments.<a reveal-id="dsy038-B1" data-open="dsy038-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Indeed, there are numerous successful examples of the application of RNA-Seq for measuring gene expression levels and detecting splicing variants and mutations in transcripts.<a reveal-id="dsy038-B2" data-open="dsy038-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy038-B3" data-open="dsy038-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, there is a large flaw in this method: it requires prior fragmentation of the mRNA molecules. When the entire transcript structure of a given transcript is studied, the fragmented sequence reads should be computationally assembled to form an ‘isoform’.<a reveal-id="dsy038-B4" data-open="dsy038-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> However, precise isoform information is not always represented in the assembled transcripts. If repetitive sequences are included in its internal sequence or its expression level is low, providing an insufficient number of sequence tags, the obtained results are occasionally unreliable without further validation analyses. Moreover, it is essentially impossible to analyse the relationship between multiple sites of splicing, if any.</p>
<p>Recently, to overcome the limitations of the current short-read sequencing approach, several new long-read technologies have been developed, such as RSII and Sequel of Pacific Biosciences, MinION of Oxford Nanopore Technologies, and the linked-read method of 10X Genomics.<a reveal-id="dsy038-B5" data-open="dsy038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In several aspects of genomic studies, long-read sequencing has appeared to have particular advantages. Namely, while it is occasionally difficult to align short reads uniquely to repetitive and duplicated regions, long reads can be effectively aligned even to these regions. Taking advantage of this feature, complementary use of long-read sequencing data analyses have now become authentic methods when <em>de novo</em> genome assembly or analysis of structural variations is contemplated. Since the long reads occasionally cover multiple SNPs in a single sequence, the mutual relationship of SNPs can also be obtained as ‘phasing information’.<a reveal-id="dsy038-B6" data-open="dsy038-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p>
<p>For the transcriptome study, PacBio RSII has been used for long-read sequencing of transcripts. In the ‘Iso-Seq’ method, the mRNAs are subjected to cDNA synthesis and amplification by the template-switching method.<a reveal-id="dsy038-B7" data-open="dsy038-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> From the sequence data obtained from PacBio RSII, it is possible to determine the entire structure of the transcripts. However, in this protocol, extensive prior size fractionation is necessary, perhaps because of the bias of the sequencer toward reading shorter fragments.<a reveal-id="dsy038-B8" data-open="dsy038-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The procedure, including the sequencing reaction itself, is sometimes technically difficult and cannot be performed in a small laboratory. More importantly, current or provisioned sequencing throughput is inadequate to obtain sufficient numbers of sequences to extract the gene expression information.<a reveal-id="dsy038-B9" data-open="dsy038-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p>
<p>The MinION sequencer is a newly developed portable long-read sequencer. After several rounds of updates, its sequence accuracy has now exceeded 90%, and the expected number of reads per flow cell is over hundreds of thousands per flow cell.<a reveal-id="dsy038-B10" data-open="dsy038-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The read length has also been improved, having achieved a N50 read length of more than 100 kb.<a reveal-id="dsy038-B11" data-open="dsy038-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Pioneering work on the application of MinION to RNA-Seq using R7.3 MinION flow cell and single-cell RNA-Seq using R7.3 and 9.4 flow cells has been reported.<a reveal-id="dsy038-B12" data-open="dsy038-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsy038-B13" data-open="dsy038-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> However, in these study, the number of mapped reads was limited to only ∼100,000. Throughput of MinION is growing by improvement of squenecing chemistry and software, and further evaluation is needed.</p>
<p>In this study, we evaluated and optimized the RNA-Seq procedure using the MinION sequencer and applied the update procedure for transcriptome analyses for various purposes. First, we used seven lung cancer cell lines and MinION R9, R9.4, and R9.5 for developing the experimental and computational procedure. We employed LAST for mapping MinION reads to RefSeq transcripts or directly to the human genome. We demonstrate that reasonably accurate information on gene expression levels as well as unique information on the entire transcript structure, including the transcript products of fusion genes in cancers, can be obtained using this approach. We also successfully attempted phasing of SNP, which were identified by whole genome sequencing (WGS), using a series of RNA materials obtained from representative organs of particular male and female individuals.</p>                    <h2 scrollto-destination="131979773" id="131979773" class="section-title" data-legacy-id="SEC2">2. Materials and methods</h2>
                    <h3 scrollto-destination="131979774" id="131979774" class="section-title" data-legacy-id="SEC2.1">2.1. Total RNA and DNA</h3>
<p>Total RNA purified from seven lung adenocarcinoma cell lines (PC-7, PC-9, H1975, H2228, VMRC-LCD, LC2/ad, and A549) were used. We purchased 14 total RNAs from seven organs, liver, kidney, skeletal muscle, pancreas, colon, heart, and lung, derived from two individuals (BioChain Institute). We also purchased two genomic DNAs derived from them. We conducted a quality check of the total RNA using the Agilent RNA Nano Kit.</p>                    <h3 scrollto-destination="131979776" id="131979776" class="section-title" data-legacy-id="SEC2.2">2.2. FL-cDNA synthesis and amplification</h3>
<p>Total RNA (50 ng) was used for FL-cDNA synthesis and amplification. We employed the SMART-Seq v4 Ultra Low Input RNA Kit (Takara Bio), except for the template switching oligo, poly-dT primer, and PCR primer. We used custom oligos with the same sequences as Smart-Seq2, instead of them.<a reveal-id="dsy038-B14" data-open="dsy038-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The synthesized first-strand cDNA was amplified by 16 cycles of PCR. FL-cDNA was quantified using the Agilent DNA 7500 Kit (Agilent Technologies).</p>                    <h3 scrollto-destination="131979778" id="131979778" class="section-title" data-legacy-id="SEC2.3">2.3. FL-cDNA preparation from a single cell</h3>
<p>LC2ad were dissociated to single cells by Accumax treatment (Innovative Cell Technologies). We employed the C1 single-cell auto prep system (Fluidigm) and SMART-Seq v4 Ultra Low Input RNA Kit with oligo DNA, identically to Smart-Seq2. We executed the C1 protocol ‘SMART-Seq v4 Rev B’. We acquired approximately 1 ng of FL-cDNA. As a control, we also conducted cDNA synthesis from bulk cells using a general thermal cycler with the same reaction conditions. Using SeqAmp DNA Polymerase (TAKARA Bio), we conducted a further amplification (1 cycle of 1 min at 96 °C, 5 cycles of 30 s at 95 °C, 65 °C for 30 s, 7 min at 68 °C, 1 cycle of 10 min at 72 °C). We purified the re-amplified cDNA using Agencourt AMPureXP (Beckman Coulter). Approximately 300 ng of FL-cDNA was obtained.</p>                    <h3 scrollto-destination="131979780" id="131979780" class="section-title" data-legacy-id="SEC2.4">2.4. MinION sequencing of FL-cDNA</h3>
<p>For 2D sequencing of FL-cDNA generated from total RNA, 1 μg of FL-cDNA was applied for library preparation of MinION with the protocol ‘2D genomic DNA by ligation’ and the Sequencing kit (SQK_MAP007), and the R9 flow cell (FLO-MAP104 or FLO-MIN105) or Sequencing kit (SQK-LSK208), and the R9.4 flow cell (FLO-MIN106). R9 flow cells were used for FL-cDNA-Seq of PC-7, PC-9, H1975, H2228, and VMRC-LCD. Both R9 and R9.4 flow cells were used for FL-cDNA-Seq of LC2ad. For 1D<sup>2</sup> sequencing of FL-cDNA, 1 µg of FL-cDNA of A549 was applied for library preparation of MinION using the protocol ‘1D∘2 sequencing of genomic DNA’ with some modifications with the 1D<sup>2</sup> sequencing kit (SQK-LSK308) and R9.5 flow cell (FLO-MIN107). For the purification steps after ligation of the 1D<sup>2</sup> adapter and Barcode Adapter Mix, the same volume of AMPure XP beads was added to the sample, instead of adding 40% of its volume. For single cell samples, 300 ng of re-amplified FL-cDNA was applied using the protocol ‘Native barcoding genomic DNA’ with the Native Barcoding Kit (EXP-NBD002), Sequencing kit (SQK-LSK208), and R9.4 flow cell (FLO-MIN106). We conducted multiplex sequencing of 11 samples.</p>
<p>For direct RNA-Seq, we performed direct RNA-Seq of LC2/ad using the R9.5 flow cell, Direct RNA Sequencing Kit (SQK-RNA001), and 500 ng polyA-selected RNA of LC2/ad, following the manufacturer’s instructions.</p>                    <h3 scrollto-destination="131979783" id="131979783" class="section-title" data-legacy-id="SEC2.5">2.5. Data analysis of MinION reads</h3>
<p>We aligned the FL-cDNA-Seq reads of LC2/ad sequenced by R9.4 flow cells to all RefSeq transcripts of mRNA (NM) and ncRNA (NR) or the longest coding isoforms of each RefGene. BWA-MEM version 0.7.15 had the following parameters: ‘-x ont2d’. We also aligned the MinION reads to these references by lastal and last-split of LAST version 833 with the parameter tuned by last train. To calculate the coverage and identity, we used the MinION reads aligned to all RefSeq transcripts. Coverage was defined as the fraction of RefSeq transcript length covered by the single read. Identity was defined as the percentage of matched bases to the sum of matched bases, substitutions, insertions, and deletions. For gene expression estimation, we used all the MinION reads aligned to the longest coding isoforms of each RefGene. The number of reads mapped to each isoform was counted. We calculated the rpm (reads per million) value for each gene as the expression unit. MinION reads were also aligned to the hg38 analysis set by LAST with the parameter of lastal tuned by last-train and ‘-d90 -m50 -D10’ and last-split ‘-m1 -d2’.</p>
<p>For splice isoform detection, we used the alignments of all pass 2d reads of LC2/ad (R9.4) to the reference genome by LAST. To remove pseudogene mapping, low-quality reads and alignment results, we discarded the aligned reads with any of the following five conditions. (1) Reads did not have an intron, which was defined as a gap of more than 50 bp. (2) Strand score of LAST was zero. (3) Unmapped length of reads within splice junctions was more than 10 bp. (4) Reads overlapped with pseudogenes of Genocode v27. (5) Isoforms with the same exon–intron structures existed on the reverse strand. Junctions of the remaining reads compared with RefSeq transcripts allowed a gap within 20 bp. If the corresponding RefSeq transcripts did not exist and the isoform was covered by more than ten reads, we judged the read to be derived from a novel isoform.</p>
<p>For detection of fusion transcript, we mapped all pass 2d reads to the reference genome by Blat with the following parameter: ‘-noHead -mask=lower’.<a reveal-id="dsy038-B5" data-open="dsy038-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsy038-B15" data-open="dsy038-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>We extracted reads with multiple alignments to multiple chromosomes, different strands of the same chromosome, or more than 1 Mb away from the same chromosome without an alternative alignment within 1 Mb of the same chromosome strand. If both ends of the reads were aligned with exons of RefSeq transcripts, we judged it as a fusion candidate.</p>
<p>For hetero SNP phasing of lung cancer cells, we used all pass 2d reads aligned to the genome by LAST and the hetero SNP data from our previous study.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> We selected reads with possible patterns of multiple SNP according to hetero SNPs and counted the number of reads with each SNP pattern. For each gene, we selected two major SNP patterns that did not mutually contradict one another, and were covered by more than five reads, respectively.</p>
<p>To detect differential allelic expression of two individuals, we used all 2d reads aligned to the genome by LAST and the hetero SNP data from this study (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>).<a reveal-id="dsy038-B17" data-open="dsy038-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> First, we used all tissue-merged reads of each individual. We called the two major patterns of single or multiple SNPs from these reads as performed for the lung cancer cell line data. We counted the number of reads in each organ according to each phased pattern. To detect genes with significant differential allelic expression, we employed the binomial test.</p>                    <h2 scrollto-destination="131979789" id="131979789" class="section-title" data-legacy-id="SEC3">3. Results and discussion</h2>
                    <h3 scrollto-destination="131979790" id="131979790" class="section-title" data-legacy-id="SEC3.1">3.1. Optimization of the cDNA-Seq procedure</h3>
<p>Using the SMART-Seq v4 Ultra Low Input RNA Kit from Takara Bio, which is based on the Smart-Seq2 method, we synthesized full-length cDNA (FL-cDNA) from 50 ng total RNA isolated from seven lung adenocarcinoma-derived cell lines: PC-7, PC-9, H1975, H2228, VMRC-LCD, LC2/ad, and A549 (<a reveal-id="dsy038-F1" data-open="dsy038-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>).<a reveal-id="dsy038-B14" data-open="dsy038-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> In these cell lines, we have previously performed RNA-Seq and WGS.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> By cDNA amplification via 16 cycles of PCR, we obtained 5–9 μg of double-stranded FL-cDNA, starting from 50 nanograms of total RNA. Although we also conducted cDNA preparation by Smart-Seq2, the yield of FL-cDNA was lower than that of SMART-Seq v4 (data not shown). We used 1 μg of FL-cDNA for library preparation of the MinION sequencing. We conducted one or two runs of 2D sequencing for approximately each six cell lines except for A549 by R9 flow cells (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). We used four R9.4 flow cells for 2D sequencing of LC2/ad. Averages of all runs using the R9 and R9.4 flow cells were 161,325 and 302,059 reads of total reads and 42,761 and 180,938 reads of pass 2d reads, respectively. We used only pass 2d reads, which had a higher quality in the following analyses. As a result, we obtained 723,750 reads of pass 2d from four runs of LC2/ad by R9.4 flow cells in total. We also conducted a single run of 1D<sup>2</sup> sequencing for A549 and obtained 994,111 1d reads and 931 pass 1dsq reads (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Statistics for the FL-cDNA-Seq analysis of the other cell lines are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. For the following analyses, we mainly focused on the data obtained from LC2/ad but obtained essentially similar results from the other cell lines. </p>                        <a id="131979792" scrollto-destination="131979792"></a>
<div id="dsy038-F1" data-id="dsy038-F1" class="fig fig-section">
<div class="label fig-label">Figure 1.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f1.png?Expires=1576183819&amp;Signature=K954Pk~t-sKq2z6MnfArhqr1sDjUqAnbOI6Cwq9MPu3JNxDEDhmQMt2ll5Y3PmA3Tl5-VYhWhFZB932fm7lYn5iTDeoEBeyrtP4K6w9uLAu3wcmp0CWidQEEa~6e7cH4TuZLMeCjYplcFWLvu8PeVZQ0pHgXtG2soec4LV3e-9ieJrLmGomhlAcLFf0LSK-aYTUA0zpa6Bg5sSHHi2rdOCKzYeZ6Tr64-YsLg1i6jNUr5~5MklBsyhPNgaG9G9Tof76Nst2es2fW9a0KGvoqayv1B7ZJjx9DBnH8xFAnc2gvz3cCbmNMi-3vgk8qn0EKAHYXNkfzTcw7rUS6e4ubgQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MinION sequencing of full-length cDNA. (A) Schematic of the FL-cDNA-Seq method. (B) Distribution of the concentration bioanalyzer and frequency of pass 2d reads of LC2/ad (R9.4) in each range of lengths. (C, D) Distribution of the coverage (C) and identity (D) of pass 2D reads of LC2ad, aligned by LAST (top) and BWA MEM (bottom). The average coverage or identity and percentages of reads with a coverage or identity greater than 0.8 is shown on the graphs. The coverage was defined as the ratio of the length of the RefSeq transcript covered by a single MinION read." path-from-xml="dsy038f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979792/dsy038f1.tif" path-from-xml="dsy038f1.tif" target="_blank">Open in new tab</a><a section="131979792" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f1.png?Expires=1576183819&amp;Signature=L3pU1hXTCQy1EpU5iAzY8IeT7nXnb3qmGINSSTZbTyJCp~FRiPkTzqN8FGO-XO2fj1WJOE1hxSZMSMnfWe0Uq-M9kVRsUzkK~ND~hrloKNnKcv~ksE8rODKCyMQiEKLUBbqeR5RDhsZ2YMPv~ewiPwDUGbE2rmLs9oB49~kxtZVvFeFPq3F2lvGPpDELpBeJDUXZLs0tknkL~Owgx7W9SgxARcOE3I41rejNaGTlYWOiPAcyCdcc-jD1tvGHUwzyWTKQGVWCIQleOuMcZx7tDRS10YvkQvTq1w84cH4hxi6yv2Rqpavs4dZaaAETNF9xCChF23Vq-zOCaftLCPeYsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979792&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>MinION sequencing of full-length cDNA. (A) Schematic of the FL-cDNA-Seq method. (B) Distribution of the concentration bioanalyzer and frequency of pass 2d reads of LC2/ad (R9.4) in each range of lengths. (C, D) Distribution of the coverage (C) and identity (D) of pass 2D reads of LC2ad, aligned by LAST (top) and BWA MEM (bottom). The average coverage or identity and percentages of reads with a coverage or identity greater than 0.8 is shown on the graphs. The coverage was defined as the ratio of the length of the RefSeq transcript covered by a single MinION read.</p></div>
</div>
<div content-id="dsy038-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f1.png?Expires=1576183819&amp;Signature=K954Pk~t-sKq2z6MnfArhqr1sDjUqAnbOI6Cwq9MPu3JNxDEDhmQMt2ll5Y3PmA3Tl5-VYhWhFZB932fm7lYn5iTDeoEBeyrtP4K6w9uLAu3wcmp0CWidQEEa~6e7cH4TuZLMeCjYplcFWLvu8PeVZQ0pHgXtG2soec4LV3e-9ieJrLmGomhlAcLFf0LSK-aYTUA0zpa6Bg5sSHHi2rdOCKzYeZ6Tr64-YsLg1i6jNUr5~5MklBsyhPNgaG9G9Tof76Nst2es2fW9a0KGvoqayv1B7ZJjx9DBnH8xFAnc2gvz3cCbmNMi-3vgk8qn0EKAHYXNkfzTcw7rUS6e4ubgQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MinION sequencing of full-length cDNA. (A) Schematic of the FL-cDNA-Seq method. (B) Distribution of the concentration bioanalyzer and frequency of pass 2d reads of LC2/ad (R9.4) in each range of lengths. (C, D) Distribution of the coverage (C) and identity (D) of pass 2D reads of LC2ad, aligned by LAST (top) and BWA MEM (bottom). The average coverage or identity and percentages of reads with a coverage or identity greater than 0.8 is shown on the graphs. The coverage was defined as the ratio of the length of the RefSeq transcript covered by a single MinION read." path-from-xml="dsy038f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979792/dsy038f1.tif" path-from-xml="dsy038f1.tif" target="_blank">Open in new tab</a><a section="131979792" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f1.png?Expires=1576183819&amp;Signature=L3pU1hXTCQy1EpU5iAzY8IeT7nXnb3qmGINSSTZbTyJCp~FRiPkTzqN8FGO-XO2fj1WJOE1hxSZMSMnfWe0Uq-M9kVRsUzkK~ND~hrloKNnKcv~ksE8rODKCyMQiEKLUBbqeR5RDhsZ2YMPv~ewiPwDUGbE2rmLs9oB49~kxtZVvFeFPq3F2lvGPpDELpBeJDUXZLs0tknkL~Owgx7W9SgxARcOE3I41rejNaGTlYWOiPAcyCdcc-jD1tvGHUwzyWTKQGVWCIQleOuMcZx7tDRS10YvkQvTq1w84cH4hxi6yv2Rqpavs4dZaaAETNF9xCChF23Vq-zOCaftLCPeYsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979792&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>MinION sequencing of full-length cDNA. (A) Schematic of the FL-cDNA-Seq method. (B) Distribution of the concentration bioanalyzer and frequency of pass 2d reads of LC2/ad (R9.4) in each range of lengths. (C, D) Distribution of the coverage (C) and identity (D) of pass 2D reads of LC2ad, aligned by LAST (top) and BWA MEM (bottom). The average coverage or identity and percentages of reads with a coverage or identity greater than 0.8 is shown on the graphs. The coverage was defined as the ratio of the length of the RefSeq transcript covered by a single MinION read.</p></div>
</div>
<p>First, we compared the results of LC2/ad between R9 and R9.4. The read lengths of R9 and R9.4 were distributed similarly to the average read lengths of 908 and 917 bp, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1</a></span>). The average read length of the other cell lines, which were sequenced by R9, was also approximately 1 kb long (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). We also compared the distributions of read length with the electropherogram of the Agilent Bioanalyzer (<a reveal-id="dsy038-F1" data-open="dsy038-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). We found that the distributions mostly overlapped, except for short fragments. These results might indicate that base calling of short fragment is unfavorable for the base caller of MinION. These results collectively suggested that transcriptome information can be read on the MinION sequencer without imposing serious length bias.</p>                    <h3 scrollto-destination="131979794" id="131979794" class="section-title" data-legacy-id="SEC3.2">3.2. The analytical pipeline</h3>
<p>To associate the obtained sequences to the reference genes or transcripts, we constructed a computational pipeline. Currently, most alignment programs are optimized for short and accurate reads, assuming the process of Illumina reads. We examined and found that these programs, as they are, are not suited for mapping the longer and less-accurate MinION reads.</p>
<p>First, to select an appropriate mapping software for this purpose, the pass 2d reads of LC2ad sequenced by R9.4 were aligned to the RefSeq transcripts by LAST and BWA.<a reveal-id="dsy038-B18 dsy038-B19 dsy038-B20" data-open="dsy038-B18 dsy038-B19 dsy038-B20" class="link link-ref link-reveal xref-bibr"><sup>18–20</sup></a> For LAST, we employed the parameters tuned by LAST-TRAIN and aligned using last-al and -split. For BWA, we used the parameter optimized for pass 2d reads of MinION (-x ont2d option) and aligned them using BWA-MEM.<a reveal-id="dsy038-B20" data-open="dsy038-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The 532,956 reads (74%) and 690,528 reads (95%) were first aligned to the RefSeq transcripts using both LAST and BWA (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). We evaluated the obtained data regarding their coverages as the RefSeq coverage (the fraction of RefSeq transcript length covered by a single read). For LAST and BWA, the average coverage was 38.3% and 33.9% of the mapped reads, respectively, showing a RefSeq coverage of more than 0.80 (<a reveal-id="dsy038-F1" data-open="dsy038-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). These results indicated that LAST could align the greater part of the read sequence to the RefSeq transcript, although the mapping rate of BWA was higher than LAST. Concomitantly, it is suggested that many of the transcripts were sequenced in their full-length forms. Based on these results, we decided to employ LAST as a default alignment software in the following analyses. When considering such alignment results, it is important to bear in mind that we can trivially obtain more and/or longer alignments, by using less-stringent alignment criteria.</p>                    <h3 scrollto-destination="131979797" id="131979797" class="section-title" data-legacy-id="SEC3.3">3.3. The overall sequence accuracy</h3>
<p>Based on the obtained sequence alignments, we evaluated the sequence accuracy (<a reveal-id="dsy038-F1" data-open="dsy038-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>). We found that the pass 2d reads obtained an overall accuracy of 92.3%. The mismatches consisted of 12.4%, 14.2%, and 73.4% base substitutions and insertions and deletions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). We examined the base substitution pattern and found that (G to A) substitutions were the most frequent (29.4%), although the preference was not substantial. We further compared the accuracy and RefSeq coverage between the 2d of 2 D sequencing, 1d and pass 1dsq reads of recent 1D<sup>2</sup> sequencing (<a reveal-id="dsy038-F1" data-open="dsy038-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Although the pass 1dsq reads (95.1%) were the most accurate compared with pass 2d reads (92.3%) and 1d reads (87.2%), there was no significant difference in the RefSeq coverage between them. However, the rate of pass 1dsq reads was quite low on this sequencing run (0.1%). In a typical run of FL-cDNA-Seq by 1D<sup>2</sup> sequencing, the rate of pass 1dsq reads was 0.4-0.9% (data not shown).</p>                    <h3 scrollto-destination="131979799" id="131979799" class="section-title" data-legacy-id="SEC3.4">3.4. Gene expression information</h3>
<p>To evaluate the precise representation of the transcriptome in MinION reads, we compared the gene expression levels in LC2/ad between those measured by the MinION reads and by the conventional Illumina TruSeq RNA method, which had been read by HiSeq2500 in our previous study.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> To minimize the influence of multi-mapped reads and differences in alignment software, both reads were aligned to the longest RefSeq transcripts of each gene by LAST (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). We calculated the rpm (reads per million) and tpm (transcripts per million) as the gene expression unit for the MinION and Illumina data sets. A high correlation coefficient (<em>R</em> = 0.88) was observed between them (also see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>, showing that similar results were also obtained for the BWA-processed data; <em>R</em> = 0.88). To further examine the influence of the mRNA fragmentation process, which is unique to TruSeq of Illumina, we used the same cDNA amplicons that were used for MinION sequencing and prepared an Illumina library by using the SMART-Seq v4 and the Nextera XT kits, referred to as SMART-Seq (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> and see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>). The comparison revealed a higher correlation (<em>R</em> = 0.91), suggesting that an influence of the difference in template preparation procedure could partly explain the difference in the represented expression information (<a reveal-id="dsy038-F2" data-open="dsy038-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). To further validate the influence regarding the generally smaller numbers of MinION reads (‘shallow sequencing depth’), we randomly sampled the reads of FL-cDNA-Seq and SMART-Seq and calculated the correlation efficiencies and number of detected genes as &gt;1 rpm or tpm. On MinION, the correlation efficiency with TruSeq roughly reached a plateau at a sequencing depth of 40,000 reads, although the number of detected genes continued to increase thereafter (<a reveal-id="dsy038-F2" data-open="dsy038-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). We also performed a similar comparison with SMART-Seq. SMART-Seq showed almost the same trend as FL-cDNA-Seq. </p>                        <a id="131979801" scrollto-destination="131979801"></a>
<div id="dsy038-F2" data-id="dsy038-F2" class="fig fig-section">
<div class="label fig-label">Figure 2.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f2.png?Expires=1576183819&amp;Signature=qYoKP0GD7gvDC8cgRHv~cOgqI2o~6f1lIyvwz3QL9En5an0iR8KKnd7LE1IXEEzCHK1pYa0gc8lYcei7K-V8JAMQqGnyaHCd0uTKsnTyB2Mw1vLXlf7dbWncyLjSjmfjOTFyvV61JBkFxYpa~yzI67FgRXQo4CFjXqgiTVZA645fIrbBAxXLw15JzzaFanUbmid-m3sR92cXw6f1l5cBQiqxo0f8yVDJew0YvPqbU2ZrUOuD4cdPyBTc2Tf1YShvbapEJXYteM2fWr4JMDdtI8SbT1Q7EfBEt5-woTRLzf-~Mjn9DemcrE8VIBumyb4aOBl8wNXv8eUMkRFho-iZIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of FL-cDNA-Seq and Illumina RNA-Seq. (A) The gene expression of FL-cDNA-Seq of LC2/ad (R9.4) was compared with that of TruSeq RNA (left) and SMART-Seq (right). Pearson correlation coefficients are shown on the graph. (B) Influence of sequencing depth on the estimation of gene expression level and gene detection. Reads for each method were randomly sampled in triplicate. The average of the Pearson correlation coefficients between TruSeq RNA and randomly sampled data for FL-cDNA-Seq and SMART-Seq is shown (left). The average number of genes with an expression level of more than 1 tpm or ppm is shown (right). (C) Comparison to qRT-qPCR. Forty-four genes detected by all methods were analyzed. The gene expression of these genes was normalized to GAPDH. Pearson correlation coefficients are shown on the graph. qRT-qPCR data of LC2/ad was obtained as in our previous study.16" path-from-xml="dsy038f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979801/dsy038f2.tif" path-from-xml="dsy038f2.tif" target="_blank">Open in new tab</a><a section="131979801" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f2.png?Expires=1576183819&amp;Signature=oJqLiMAnK0q4~xBL1jKzqCRAMd-4A64aVLTB4UY3rDmS4zpkS9ayXEzbyGTEqYPLgbABS~d3-neTpZ~eBUa5bH8krdzDsTnnuXDvzT5nUqCC3U9jvJP9rQ8loVcWeN~5Va8yx-z4It9yut71cX30vITwcDoDPHwRENXimRHKXHtWzJeD0Tw6FUn3d3dprSrGiI5nkMSBFmvdfnjR6GjS5N1l~d2I4QbJ6S00cdK8jE3QIcBZQJHbWurbZY12agx5Tof03d9CqIbFWx5l0-6JpTqJCaVAfwAoqwY8BO023eQEfPZx9Hg8ZXkzAX-7n82Kd7EQO3ppQyxLr6hHHWCmgg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979801&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of FL-cDNA-Seq and Illumina RNA-Seq. (A) The gene expression of FL-cDNA-Seq of LC2/ad (R9.4) was compared with that of TruSeq RNA (left) and SMART-Seq (right). Pearson correlation coefficients are shown on the graph. (B) Influence of sequencing depth on the estimation of gene expression level and gene detection. Reads for each method were randomly sampled in triplicate. The average of the Pearson correlation coefficients between TruSeq RNA and randomly sampled data for FL-cDNA-Seq and SMART-Seq is shown (left). The average number of genes with an expression level of more than 1 tpm or ppm is shown (right). (C) Comparison to qRT-qPCR. Forty-four genes detected by all methods were analyzed. The gene expression of these genes was normalized to GAPDH. Pearson correlation coefficients are shown on the graph. qRT-qPCR data of LC2/ad was obtained as in our previous study.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p></div>
</div>
<div content-id="dsy038-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f2.png?Expires=1576183819&amp;Signature=qYoKP0GD7gvDC8cgRHv~cOgqI2o~6f1lIyvwz3QL9En5an0iR8KKnd7LE1IXEEzCHK1pYa0gc8lYcei7K-V8JAMQqGnyaHCd0uTKsnTyB2Mw1vLXlf7dbWncyLjSjmfjOTFyvV61JBkFxYpa~yzI67FgRXQo4CFjXqgiTVZA645fIrbBAxXLw15JzzaFanUbmid-m3sR92cXw6f1l5cBQiqxo0f8yVDJew0YvPqbU2ZrUOuD4cdPyBTc2Tf1YShvbapEJXYteM2fWr4JMDdtI8SbT1Q7EfBEt5-woTRLzf-~Mjn9DemcrE8VIBumyb4aOBl8wNXv8eUMkRFho-iZIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of FL-cDNA-Seq and Illumina RNA-Seq. (A) The gene expression of FL-cDNA-Seq of LC2/ad (R9.4) was compared with that of TruSeq RNA (left) and SMART-Seq (right). Pearson correlation coefficients are shown on the graph. (B) Influence of sequencing depth on the estimation of gene expression level and gene detection. Reads for each method were randomly sampled in triplicate. The average of the Pearson correlation coefficients between TruSeq RNA and randomly sampled data for FL-cDNA-Seq and SMART-Seq is shown (left). The average number of genes with an expression level of more than 1 tpm or ppm is shown (right). (C) Comparison to qRT-qPCR. Forty-four genes detected by all methods were analyzed. The gene expression of these genes was normalized to GAPDH. Pearson correlation coefficients are shown on the graph. qRT-qPCR data of LC2/ad was obtained as in our previous study.16" path-from-xml="dsy038f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979801/dsy038f2.tif" path-from-xml="dsy038f2.tif" target="_blank">Open in new tab</a><a section="131979801" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f2.png?Expires=1576183819&amp;Signature=oJqLiMAnK0q4~xBL1jKzqCRAMd-4A64aVLTB4UY3rDmS4zpkS9ayXEzbyGTEqYPLgbABS~d3-neTpZ~eBUa5bH8krdzDsTnnuXDvzT5nUqCC3U9jvJP9rQ8loVcWeN~5Va8yx-z4It9yut71cX30vITwcDoDPHwRENXimRHKXHtWzJeD0Tw6FUn3d3dprSrGiI5nkMSBFmvdfnjR6GjS5N1l~d2I4QbJ6S00cdK8jE3QIcBZQJHbWurbZY12agx5Tof03d9CqIbFWx5l0-6JpTqJCaVAfwAoqwY8BO023eQEfPZx9Hg8ZXkzAX-7n82Kd7EQO3ppQyxLr6hHHWCmgg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979801&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of FL-cDNA-Seq and Illumina RNA-Seq. (A) The gene expression of FL-cDNA-Seq of LC2/ad (R9.4) was compared with that of TruSeq RNA (left) and SMART-Seq (right). Pearson correlation coefficients are shown on the graph. (B) Influence of sequencing depth on the estimation of gene expression level and gene detection. Reads for each method were randomly sampled in triplicate. The average of the Pearson correlation coefficients between TruSeq RNA and randomly sampled data for FL-cDNA-Seq and SMART-Seq is shown (left). The average number of genes with an expression level of more than 1 tpm or ppm is shown (right). (C) Comparison to qRT-qPCR. Forty-four genes detected by all methods were analyzed. The gene expression of these genes was normalized to GAPDH. Pearson correlation coefficients are shown on the graph. qRT-qPCR data of LC2/ad was obtained as in our previous study.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p></div>
</div>
<p>Then, we validated the gene expression levels of 44 genes by RT-qPCR.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> We compared the obtained results with FL-cDNA-Seq and Illumina RNA-Seq methods. Although the number of FL-cDNA-Seq reads was generally smaller than that of Illumina reads, the FL-cDNA-Seq data showed a comparably high correlation efficiency with the RT-qPCR data [<em>R</em> = 0.82 (TruSeq), 0.81 (SMART-Seq), and 0.82 (FL-cDNA-Seq), respectively] (<a reveal-id="dsy038-F2" data-open="dsy038-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Taken together, we concluded that FL-cDNA-Seq is effective for the purposes of quantitative transcriptome analysis.</p>                    <h3 scrollto-destination="131979803" id="131979803" class="section-title" data-legacy-id="SEC3.5">3.5. Comparison with direct RNA sequencing</h3>
<p>We also conducted recent direct RNA-Seq of LC2/ad<a reveal-id="dsy038-B21" data-open="dsy038-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). We obtained 556,195 1d reads from a single run. The average length of the direct RNA-Seq reads was 926 bp, which was comparable to the average length of the above RNA-Seq. Similarly, the direct RNA-Seq reads were aligned to all RefSeq transcripts using LAST. The direct RNA-Seq reads showed comparable coverage (57.0%) of the reference transcripts to FL-cDNA-Seq (58.4%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>). Furthermore, the direct RNA-Seq showed a high correlation (<em>R</em> = 0.92 and 0.90) with TruSeq and the FL-cDNA-Seq, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span>). In contrast, direct RNA-Seq showed lower fidelity (83.2%) than FL-cDNA-Seq (92.3%) (<a reveal-id="dsy038-F1" data-open="dsy038-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>). Based on these results, we conclude that direct RNA-Seq is also useful to estimate expression levels of the whole transcriptome. An obvious advantage of direct RNA-Seq is that it is free from PCR bias and may be able to analyse mRNA for which reverse transcription is not efficient. However, there is a limitation to this method, which is that 500 ng of polyA RNA is required as starting material, which may limit its application to rare samples, such as the subtle amounts of RNA present in some clinical samples.</p>                    <h3 scrollto-destination="131979805" id="131979805" class="section-title" data-legacy-id="SEC3.6">3.6. Applications of MinION transcriptome sequencing</h3>
<p>In addition to the above applications, we found that FL-cDNA-Seq may have several applications that are not easily performed using Illumina RNA-Seq.</p>                    <h4 scrollto-destination="131979807" id="131979807" class="section-title" data-legacy-id="SEC3.6.1">3.6.1. Detection of transcript variants and cancerous fusion genes</h4>
<p>We examined whether the FL-cDNA-Seq reads could be used to detect transcript variants and mutations. For this purpose, the FL-cDNA-Seq reads of LC2/ad (R9.4) were aligned to the reference genome by LAST. We calculated the distance between the detected and RefSeq transcript TSS (transcription start site) and TTS (transcription termination site), and 82% and 63% of the TSS and TTS detected by MinION were located within ±500 bp from those of the RefSeq transcripts, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S6 and S7</a></span>). We compared the detected transcripts with transcript models of RefSeq. Although 6,018 of the detected transcripts matched the RefSeq models, 158 of them represented unannotated isoforms (<a reveal-id="dsy038-T1" data-open="dsy038-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). We also compared the detected transcripts by FL-cDNA-Seq and RNA-Seq. Although only 33 (21%) of the novel isoforms detected by FL-cDNA-Seq were detected by cufflinks with guide of RefSeq annotation from the RNA-Seq data, all the splice junctions of 137 (87%) of them were covered by the RNA-Seq reads (<a reveal-id="dsy038-T1" data-open="dsy038-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Examples are novel isoforms of STARP and ACTG1 genes that were difficult to detect by Illumina reads alone (<a reveal-id="dsy038-F3" data-open="dsy038-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). </p>
<div content-id="dsy038-T1" class="table-modal"><div class="table-wrap" content-id="dsy038-T1">
<div id="dsy038-T1" data-id="dsy038-T1" class="table-wrap-title">
<div class="label">Table 1</div>
<div class="caption"><p>Application of MinION transcriptome sequencing: statistics for the isoform detected by FL-cDNA-Seq</p></div>
</div>
<div class="table-overflow"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">No. of mapped reads </th>
<th rowspan="2">No. of PF reads </th>
<th rowspan="2">% PF reads </th>
<th colspan="3">Known isoform </th>
<th colspan="5">Novel isoform </th>
</tr>
<tr>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of cufflinks-supported isoforms </th>
<th>No. of Illumina-supported isoforms </th>
</tr>
</thead>
<tbody><tr>
<td>556,195 </td>
<td>297,865 </td>
<td>54% </td>
<td>294,780 </td>
<td>99% </td>
<td>6,018 </td>
<td>3,085 </td>
<td>1% </td>
<td>158 </td>
<td>33 </td>
<td>137 </td>
</tr></tbody>
</table></div>
<div content-id="dsy038-T1" class="table-modal"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">No. of mapped reads </th>
<th rowspan="2">No. of PF reads </th>
<th rowspan="2">% PF reads </th>
<th colspan="3">Known isoform </th>
<th colspan="5">Novel isoform </th>
</tr>
<tr>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of cufflinks-supported isoforms </th>
<th>No. of Illumina-supported isoforms </th>
</tr>
</thead>
<tbody><tr>
<td>556,195 </td>
<td>297,865 </td>
<td>54% </td>
<td>294,780 </td>
<td>99% </td>
<td>6,018 </td>
<td>3,085 </td>
<td>1% </td>
<td>158 </td>
<td>33 </td>
<td>137 </td>
</tr></tbody>
</table></div>
</div></div>
<div class="table-wrap" content-id="dsy038-T1">
<div id="dsy038-T1" data-id="dsy038-T1" class="table-wrap-title">
<div class="label">Table 1</div>
<div class="caption"><p>Application of MinION transcriptome sequencing: statistics for the isoform detected by FL-cDNA-Seq</p></div>
</div>
<div class="table-overflow"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">No. of mapped reads </th>
<th rowspan="2">No. of PF reads </th>
<th rowspan="2">% PF reads </th>
<th colspan="3">Known isoform </th>
<th colspan="5">Novel isoform </th>
</tr>
<tr>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of cufflinks-supported isoforms </th>
<th>No. of Illumina-supported isoforms </th>
</tr>
</thead>
<tbody><tr>
<td>556,195 </td>
<td>297,865 </td>
<td>54% </td>
<td>294,780 </td>
<td>99% </td>
<td>6,018 </td>
<td>3,085 </td>
<td>1% </td>
<td>158 </td>
<td>33 </td>
<td>137 </td>
</tr></tbody>
</table></div>
<div content-id="dsy038-T1" class="table-modal"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">No. of mapped reads </th>
<th rowspan="2">No. of PF reads </th>
<th rowspan="2">% PF reads </th>
<th colspan="3">Known isoform </th>
<th colspan="5">Novel isoform </th>
</tr>
<tr>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of reads </th>
<th>% reads </th>
<th>No. of isoforms </th>
<th>No. of cufflinks-supported isoforms </th>
<th>No. of Illumina-supported isoforms </th>
</tr>
</thead>
<tbody><tr>
<td>556,195 </td>
<td>297,865 </td>
<td>54% </td>
<td>294,780 </td>
<td>99% </td>
<td>6,018 </td>
<td>3,085 </td>
<td>1% </td>
<td>158 </td>
<td>33 </td>
<td>137 </td>
</tr></tbody>
</table></div>
</div>                        <a id="131979809" scrollto-destination="131979809"></a>
<div id="dsy038-F3" data-id="dsy038-F3" class="fig fig-section">
<div class="label fig-label">Figure 3.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f3.png?Expires=1576183819&amp;Signature=vE~913L0kmRuFIxlNRFnpnd4udiq52s71vkoPTIOkw0-T-gTHr0~u6iz9~OUesDvsBsE22Qn2r3s5y6XofrypRTrtsrewyiLzj1szvF5dHuexP9ni9dMAweSyQEgYehyuCm89y1BqNXl2VWIQXRB7IoE-0Iuyt8GIUgaxalV6vAFcxI~cZ-sdfkwszFv4IF97tIKdURS6IBq4WouaaxeTjYBihUSiov2lrWprQq7ytRsBlHTc-vM~eEhNEqONvsGFablrQpUYA2eJXX7UUDZEVeTAfO5oGnh2W~lEnN8z4CpzW1UK6tv7XOptRWK59duapG9CUq2TNs3YJ3rvgnXkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Applications of MinION transcriptome sequencing. (A) Novel isoforms of the STRAP gene (left) and the ACTG1 gene (right). RefSeq transcripts and cufflink assembles using Illumina RNA-Seq are shown in the upper panel. FL-cDNA-Seq reads annotated as novel isoforms are shown in the middle panel. Illumina RNASeq reads are shown in the lower panel. (B) Three fusion genes detected by FL-cDNA. CCDC6-RET is known as a fusion gene of LC2/ad. Only the fusion chromosome of CCDC6-RET harbors a G to T SNP. WAC-SFMBT2 and ZSCAN22-CHMP2A were detected on LC2/ad and PC-9, respectively. (C) Sanger sequencing of the three fusion junctions. (D) Phased hetero SNP by FL-cDNA-Seq reads. We exemplified two phased genes, DSG2 (left) and SEPT9 (right), detected by the R9.4 reads of LC2ad. The distance between the phased SNP on the transcript and genome and hetero SNP patterns are shown at the top. The FL-cDNA-Seq reads are shown in the middle. The phased SNP pattern and number of reads covering all the SNPs are shown at the bottom." path-from-xml="dsy038f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979809/dsy038f3.tif" path-from-xml="dsy038f3.tif" target="_blank">Open in new tab</a><a section="131979809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f3.png?Expires=1576183819&amp;Signature=fkpS9ItQ6p7GucsWNFyBDsSHRfFrkpe0Fs2WEGHTilI98WEROPH6Gw3vWwIwuyAtNA3re7MDziSCUea22pIGklK5~ulYOjQN9Xa55dzSW5cJAwi3v0UixEjqRlBLMJUKogjRLJ14xQQRntda8-dAIf~ffhiBRISGtVt8Vm9oxdEXCOyeQOq4P81sQCFgBQw1HCVWrOQui0vRc9OPwG3RQxTK5YgE6Au~Wqs~Bhf-hVJkWXQY3-uDskzmEWLHQFCuRim~TU6nfwtILoC6FKV-zDZMJH6By~5-30cs~qIlI7V2333THF7y1KA2F7H0As9bbCy0ows2lfkv5zP-36QntQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979809&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Applications of MinION transcriptome sequencing. (A) Novel isoforms of the STRAP gene (left) and the ACTG1 gene (right). RefSeq transcripts and cufflink assembles using Illumina RNA-Seq are shown in the upper panel. FL-cDNA-Seq reads annotated as novel isoforms are shown in the middle panel. Illumina RNASeq reads are shown in the lower panel. (B) Three fusion genes detected by FL-cDNA. CCDC6-RET is known as a fusion gene of LC2/ad. Only the fusion chromosome of CCDC6-RET harbors a G to T SNP. WAC-SFMBT2 and ZSCAN22-CHMP2A were detected on LC2/ad and PC-9, respectively. (C) Sanger sequencing of the three fusion junctions. (D) Phased hetero SNP by FL-cDNA-Seq reads. We exemplified two phased genes, DSG2 (left) and SEPT9 (right), detected by the R9.4 reads of LC2ad. The distance between the phased SNP on the transcript and genome and hetero SNP patterns are shown at the top. The FL-cDNA-Seq reads are shown in the middle. The phased SNP pattern and number of reads covering all the SNPs are shown at the bottom.</p></div>
</div>
<div content-id="dsy038-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f3.png?Expires=1576183819&amp;Signature=vE~913L0kmRuFIxlNRFnpnd4udiq52s71vkoPTIOkw0-T-gTHr0~u6iz9~OUesDvsBsE22Qn2r3s5y6XofrypRTrtsrewyiLzj1szvF5dHuexP9ni9dMAweSyQEgYehyuCm89y1BqNXl2VWIQXRB7IoE-0Iuyt8GIUgaxalV6vAFcxI~cZ-sdfkwszFv4IF97tIKdURS6IBq4WouaaxeTjYBihUSiov2lrWprQq7ytRsBlHTc-vM~eEhNEqONvsGFablrQpUYA2eJXX7UUDZEVeTAfO5oGnh2W~lEnN8z4CpzW1UK6tv7XOptRWK59duapG9CUq2TNs3YJ3rvgnXkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Applications of MinION transcriptome sequencing. (A) Novel isoforms of the STRAP gene (left) and the ACTG1 gene (right). RefSeq transcripts and cufflink assembles using Illumina RNA-Seq are shown in the upper panel. FL-cDNA-Seq reads annotated as novel isoforms are shown in the middle panel. Illumina RNASeq reads are shown in the lower panel. (B) Three fusion genes detected by FL-cDNA. CCDC6-RET is known as a fusion gene of LC2/ad. Only the fusion chromosome of CCDC6-RET harbors a G to T SNP. WAC-SFMBT2 and ZSCAN22-CHMP2A were detected on LC2/ad and PC-9, respectively. (C) Sanger sequencing of the three fusion junctions. (D) Phased hetero SNP by FL-cDNA-Seq reads. We exemplified two phased genes, DSG2 (left) and SEPT9 (right), detected by the R9.4 reads of LC2ad. The distance between the phased SNP on the transcript and genome and hetero SNP patterns are shown at the top. The FL-cDNA-Seq reads are shown in the middle. The phased SNP pattern and number of reads covering all the SNPs are shown at the bottom." path-from-xml="dsy038f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979809/dsy038f3.tif" path-from-xml="dsy038f3.tif" target="_blank">Open in new tab</a><a section="131979809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f3.png?Expires=1576183819&amp;Signature=fkpS9ItQ6p7GucsWNFyBDsSHRfFrkpe0Fs2WEGHTilI98WEROPH6Gw3vWwIwuyAtNA3re7MDziSCUea22pIGklK5~ulYOjQN9Xa55dzSW5cJAwi3v0UixEjqRlBLMJUKogjRLJ14xQQRntda8-dAIf~ffhiBRISGtVt8Vm9oxdEXCOyeQOq4P81sQCFgBQw1HCVWrOQui0vRc9OPwG3RQxTK5YgE6Au~Wqs~Bhf-hVJkWXQY3-uDskzmEWLHQFCuRim~TU6nfwtILoC6FKV-zDZMJH6By~5-30cs~qIlI7V2333THF7y1KA2F7H0As9bbCy0ows2lfkv5zP-36QntQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979809&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Applications of MinION transcriptome sequencing. (A) Novel isoforms of the STRAP gene (left) and the ACTG1 gene (right). RefSeq transcripts and cufflink assembles using Illumina RNA-Seq are shown in the upper panel. FL-cDNA-Seq reads annotated as novel isoforms are shown in the middle panel. Illumina RNASeq reads are shown in the lower panel. (B) Three fusion genes detected by FL-cDNA. CCDC6-RET is known as a fusion gene of LC2/ad. Only the fusion chromosome of CCDC6-RET harbors a G to T SNP. WAC-SFMBT2 and ZSCAN22-CHMP2A were detected on LC2/ad and PC-9, respectively. (C) Sanger sequencing of the three fusion junctions. (D) Phased hetero SNP by FL-cDNA-Seq reads. We exemplified two phased genes, DSG2 (left) and SEPT9 (right), detected by the R9.4 reads of LC2ad. The distance between the phased SNP on the transcript and genome and hetero SNP patterns are shown at the top. The FL-cDNA-Seq reads are shown in the middle. The phased SNP pattern and number of reads covering all the SNPs are shown at the bottom.</p></div>
</div>
<p>We further attempted to detect fusion transcripts in cancer cells. We aligned the FL-cDNA-Seq reads of six cell lines to the reference genome using BLAT (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The transcripts with two or more alignments that were mapped to different chromosomes, or those that were mapped to regions that were mutually separated by more than 1 Mb, were considered. A total 151 fusion transcript candidates were detected from the LC2/ad reads of R9.4 (<a reveal-id="dsy038-T2" data-open="dsy038-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The detected fusion transcripts included the previously reported cancer driver fusion transcript of the CCDC6-RET gene. As shown in <a reveal-id="dsy038-F3" data-open="dsy038-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, seven reads directly represented the junction of the CCDC6-RET fusion transcript. Interestingly, the fusion point harbored a hetero germline SNP, which gave us a clue regarding the identities of the alleles in which the fusion event occurred. Considering all six cell lines together, we identified 194 fusion gene candidates (<a reveal-id="dsy038-T2" data-open="dsy038-T2" class="link link-reveal link-table xref-fig">Table 2</a>). For example, WAC-SFMBT2 and ZSCAN22-CHMP2A fusion transcripts were detected from LC2/ad and PC-9, respectively (<a reveal-id="dsy038-F3" data-open="dsy038-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). These genes could be validated using Sanger sequencing (<a reveal-id="dsy038-F3" data-open="dsy038-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> and see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>). These results indicate that MinION sequencing is a convenient method to detect fusion gene transcripts. </p>
<div content-id="dsy038-T2" class="table-modal"><div class="table-wrap" content-id="dsy038-T2">
<div id="dsy038-T2" data-id="dsy038-T2" class="table-wrap-title">
<div class="label">Table 2</div>
<div class="caption"><p>Application of MinION transcriptome sequencing: number of fusion gene candidates</p></div>
</div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of reads of fusion gene candidates </th>
<th>No. of fusion gene candidates </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>8 </td>
<td>8 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>5 </td>
<td>5 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>7 </td>
<td>7 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>14 </td>
<td>11 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>10 </td>
<td>10 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>158 </td>
<td>151 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy038-T2" class="table-modal"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of reads of fusion gene candidates </th>
<th>No. of fusion gene candidates </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>8 </td>
<td>8 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>5 </td>
<td>5 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>7 </td>
<td>7 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>14 </td>
<td>11 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>10 </td>
<td>10 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>158 </td>
<td>151 </td>
</tr>
</tbody>
</table></div>
</div></div>
<div class="table-wrap" content-id="dsy038-T2">
<div id="dsy038-T2" data-id="dsy038-T2" class="table-wrap-title">
<div class="label">Table 2</div>
<div class="caption"><p>Application of MinION transcriptome sequencing: number of fusion gene candidates</p></div>
</div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of reads of fusion gene candidates </th>
<th>No. of fusion gene candidates </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>8 </td>
<td>8 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>5 </td>
<td>5 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>7 </td>
<td>7 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>14 </td>
<td>11 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>10 </td>
<td>10 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>158 </td>
<td>151 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy038-T2" class="table-modal"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of reads of fusion gene candidates </th>
<th>No. of fusion gene candidates </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>8 </td>
<td>8 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>5 </td>
<td>5 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>7 </td>
<td>7 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>14 </td>
<td>11 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>10 </td>
<td>10 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>158 </td>
<td>151 </td>
</tr>
</tbody>
</table></div>
</div>                    <h4 scrollto-destination="131979811" id="131979811" class="section-title" data-legacy-id="SEC3.6.2">3.6.2. Haplotype phasing from MinION reads</h4>
<p>In lung cancer, there are well-known driver mutations in the tyrosine kinase domain of the EGFR gene, which are targets for molecular target drugs such as gefitinib. There are also several secondary mutations known for resistance to drugs in the proximal region.<a reveal-id="dsy038-B22" data-open="dsy038-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> For those mutations, it is important to obtain phase information (allele context of the mutations) since the clinical relevance will be distinct regardless of whether the mutations are located on the same allele. We expected long MinION reads to overcome the difficulties in phasing using Illumina reads. To avoid false detection due to the relatively high error rates of MinION, we employed the heterozygous SNV information for lung cancer cell lines previously detected from WGS using Illumina HiSeq.<a reveal-id="dsy038-B16" data-open="dsy038-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Considering all the cell lines together, multiple SNVs were phased on 291 genes (<a reveal-id="dsy038-T3" data-open="dsy038-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Interestingly, the mutual distance between those SNVs was occasionally greater than 100 kb because of the intron between them. This approach may enable rather efficient SNP phasing than that at the genomic level as far as the location in transcript regions (<a reveal-id="dsy038-F3" data-open="dsy038-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>). </p>
<div content-id="dsy038-T3" class="table-modal"><div class="table-wrap" content-id="dsy038-T3">
<div id="dsy038-T3" data-id="dsy038-T3" class="table-wrap-title">
<div class="label">Table 3</div>
<div class="caption"><p>Application of MinION transcriptome sequencing: phased gene number and heterozygous site distance</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of phased gene </th>
<th>Average of phase distance </th>
<th>Average of phase distance without intron </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>17 </td>
<td>5,791 </td>
<td>737 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>5,256 </td>
<td>570 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>2 </td>
<td>1,546 </td>
<td>725 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>12 </td>
<td>6,528 </td>
<td>484 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>2 </td>
<td>2,246 </td>
<td>102 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>16 </td>
<td>1,542 </td>
<td>219 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>237 </td>
<td>8,872 </td>
<td>569 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy038-T3" class="table-modal"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of phased gene </th>
<th>Average of phase distance </th>
<th>Average of phase distance without intron </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>17 </td>
<td>5,791 </td>
<td>737 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>5,256 </td>
<td>570 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>2 </td>
<td>1,546 </td>
<td>725 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>12 </td>
<td>6,528 </td>
<td>484 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>2 </td>
<td>2,246 </td>
<td>102 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>16 </td>
<td>1,542 </td>
<td>219 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>237 </td>
<td>8,872 </td>
<td>569 </td>
</tr>
</tbody>
</table></div>
</div></div>
<div class="table-wrap" content-id="dsy038-T3">
<div id="dsy038-T3" data-id="dsy038-T3" class="table-wrap-title">
<div class="label">Table 3</div>
<div class="caption"><p>Application of MinION transcriptome sequencing: phased gene number and heterozygous site distance</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of phased gene </th>
<th>Average of phase distance </th>
<th>Average of phase distance without intron </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>17 </td>
<td>5,791 </td>
<td>737 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>5,256 </td>
<td>570 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>2 </td>
<td>1,546 </td>
<td>725 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>12 </td>
<td>6,528 </td>
<td>484 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>2 </td>
<td>2,246 </td>
<td>102 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>16 </td>
<td>1,542 </td>
<td>219 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>237 </td>
<td>8,872 </td>
<td>569 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy038-T3" class="table-modal"><table>
<thead name="thead" align="left"><tr>
<th>Cell line </th>
<th>Flow cell </th>
<th>No. of phased gene </th>
<th>Average of phase distance </th>
<th>Average of phase distance without intron </th>
</tr></thead>
<tbody>
<tr>
<td>H1975 </td>
<td>R9 </td>
<td>17 </td>
<td>5,791 </td>
<td>737 </td>
</tr>
<tr>
<td>PC-9 </td>
<td>R9 </td>
<td>5 </td>
<td>5,256 </td>
<td>570 </td>
</tr>
<tr>
<td>PC-7 </td>
<td>R9 </td>
<td>2 </td>
<td>1,546 </td>
<td>725 </td>
</tr>
<tr>
<td>H2228 </td>
<td>R9 </td>
<td>12 </td>
<td>6,528 </td>
<td>484 </td>
</tr>
<tr>
<td>VMRC-LCD </td>
<td>R9 </td>
<td>2 </td>
<td>2,246 </td>
<td>102 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9 </td>
<td>16 </td>
<td>1,542 </td>
<td>219 </td>
</tr>
<tr>
<td>LC2/ad </td>
<td>R9.4 </td>
<td>237 </td>
<td>8,872 </td>
<td>569 </td>
</tr>
</tbody>
</table></div>
</div>                    <h4 scrollto-destination="131979813" id="131979813" class="section-title" data-legacy-id="SEC3.6.3">3.6.3. FL-cDNA-Seq of multi-organs of two single individuals</h4>
<p>By using FL-cDNA-Seq, we attempted to detect allele-biased expression in various organs of given individuals. We obtained a total of 14 total RNA sets from seven organs of two healthy individuals, one male and one female. We also obtained their corresponding genomic DNAs. We performed FL-cDNA-Seq for these 14 RNA samples and aligned the reads to the human reference genome by LAST (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S10 and S11</a></span>). We also performed whole exome sequencing of two genomic DNA samples by Illumina (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span> and see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>). Using GATK best practices, 437,646 and 395,486 hetero SNPs were called from the genomic DNAs of the male and the female samples, respectively. Using the obtained hetero SNP data, we phased the FL-cDNA-Seq reads of all tissue-merged data. We phased 2,928 genes, and 306 genes showed significant allelic expression based on the binomial test (<a reveal-id="dsy038-T4" data-open="dsy038-T4" class="link link-reveal link-table xref-fig">Table 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). For instance, the UBXN4 gene was showed differential allelic expression in the female. Although the C allele showed higher expression than the A allele in most tissues of the female, only the liver showed the opposite allelic expression pattern (<a reveal-id="dsy038-F4" data-open="dsy038-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). In the male, we also detected the GNAS gene, which has a known complex allelic imbalance expression pattern<a reveal-id="dsy038-B23" data-open="dsy038-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsy038-B24" data-open="dsy038-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> (<a reveal-id="dsy038-F4" data-open="dsy038-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). This gene has one maternal chromosome-specific first exon, two paternal chromosome-specific first exons, and neutral first exons. Paternal first exons and the maternal first exon were only detected on the C allele and T allele, showing that the C and T alleles are on the paternal chromosome and maternal chromosome, respectively. </p>
<div content-id="dsy038-T4" class="table-modal"><div class="table-wrap" content-id="dsy038-T4">
<div id="dsy038-T4" data-id="dsy038-T4" class="table-wrap-title">
<div class="label">Table 4</div>
<div class="caption"><p>Number of genes with allelic imbalance expression</p></div>
</div>
<div class="table-overflow"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Tissue </th>
<th colspan="2">Male </th>
<th colspan="2">Female </th>
</tr>
<tr>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Merged data </td>
<td>1,219 </td>
<td>105 </td>
<td>1,707 </td>
<td>201 </td>
</tr>
<tr>
<td>Colon </td>
<td>625 </td>
<td>64 </td>
<td>1,394 </td>
<td>161 </td>
</tr>
<tr>
<td>Heart </td>
<td>836 </td>
<td>78 </td>
<td>1,227 </td>
<td>161 </td>
</tr>
<tr>
<td>Kidney </td>
<td>842 </td>
<td>82 </td>
<td>1,305 </td>
<td>165 </td>
</tr>
<tr>
<td>Liver </td>
<td>854 </td>
<td>85 </td>
<td>1,387 </td>
<td>172 </td>
</tr>
<tr>
<td>Lung </td>
<td>986 </td>
<td>87 </td>
<td>1,563 </td>
<td>175 </td>
</tr>
<tr>
<td>Pancreas </td>
<td>760 </td>
<td>83 </td>
<td>818 </td>
<td>143 </td>
</tr>
<tr>
<td>Skeletal muscle </td>
<td>792 </td>
<td>63 </td>
<td>1,215 </td>
<td>157 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy038-T4" class="table-modal"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Tissue </th>
<th colspan="2">Male </th>
<th colspan="2">Female </th>
</tr>
<tr>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Merged data </td>
<td>1,219 </td>
<td>105 </td>
<td>1,707 </td>
<td>201 </td>
</tr>
<tr>
<td>Colon </td>
<td>625 </td>
<td>64 </td>
<td>1,394 </td>
<td>161 </td>
</tr>
<tr>
<td>Heart </td>
<td>836 </td>
<td>78 </td>
<td>1,227 </td>
<td>161 </td>
</tr>
<tr>
<td>Kidney </td>
<td>842 </td>
<td>82 </td>
<td>1,305 </td>
<td>165 </td>
</tr>
<tr>
<td>Liver </td>
<td>854 </td>
<td>85 </td>
<td>1,387 </td>
<td>172 </td>
</tr>
<tr>
<td>Lung </td>
<td>986 </td>
<td>87 </td>
<td>1,563 </td>
<td>175 </td>
</tr>
<tr>
<td>Pancreas </td>
<td>760 </td>
<td>83 </td>
<td>818 </td>
<td>143 </td>
</tr>
<tr>
<td>Skeletal muscle </td>
<td>792 </td>
<td>63 </td>
<td>1,215 </td>
<td>157 </td>
</tr>
</tbody>
</table></div>
</div></div>
<div class="table-wrap" content-id="dsy038-T4">
<div id="dsy038-T4" data-id="dsy038-T4" class="table-wrap-title">
<div class="label">Table 4</div>
<div class="caption"><p>Number of genes with allelic imbalance expression</p></div>
</div>
<div class="table-overflow"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Tissue </th>
<th colspan="2">Male </th>
<th colspan="2">Female </th>
</tr>
<tr>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Merged data </td>
<td>1,219 </td>
<td>105 </td>
<td>1,707 </td>
<td>201 </td>
</tr>
<tr>
<td>Colon </td>
<td>625 </td>
<td>64 </td>
<td>1,394 </td>
<td>161 </td>
</tr>
<tr>
<td>Heart </td>
<td>836 </td>
<td>78 </td>
<td>1,227 </td>
<td>161 </td>
</tr>
<tr>
<td>Kidney </td>
<td>842 </td>
<td>82 </td>
<td>1,305 </td>
<td>165 </td>
</tr>
<tr>
<td>Liver </td>
<td>854 </td>
<td>85 </td>
<td>1,387 </td>
<td>172 </td>
</tr>
<tr>
<td>Lung </td>
<td>986 </td>
<td>87 </td>
<td>1,563 </td>
<td>175 </td>
</tr>
<tr>
<td>Pancreas </td>
<td>760 </td>
<td>83 </td>
<td>818 </td>
<td>143 </td>
</tr>
<tr>
<td>Skeletal muscle </td>
<td>792 </td>
<td>63 </td>
<td>1,215 </td>
<td>157 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy038-T4" class="table-modal"><table>
<thead name="thead" align="left">
<tr>
<th rowspan="2">Tissue </th>
<th colspan="2">Male </th>
<th colspan="2">Female </th>
</tr>
<tr>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
<th>No. of phased gene </th>
<th>No. of genes with allelic expression (<em>P</em> &lt; 0.01) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Merged data </td>
<td>1,219 </td>
<td>105 </td>
<td>1,707 </td>
<td>201 </td>
</tr>
<tr>
<td>Colon </td>
<td>625 </td>
<td>64 </td>
<td>1,394 </td>
<td>161 </td>
</tr>
<tr>
<td>Heart </td>
<td>836 </td>
<td>78 </td>
<td>1,227 </td>
<td>161 </td>
</tr>
<tr>
<td>Kidney </td>
<td>842 </td>
<td>82 </td>
<td>1,305 </td>
<td>165 </td>
</tr>
<tr>
<td>Liver </td>
<td>854 </td>
<td>85 </td>
<td>1,387 </td>
<td>172 </td>
</tr>
<tr>
<td>Lung </td>
<td>986 </td>
<td>87 </td>
<td>1,563 </td>
<td>175 </td>
</tr>
<tr>
<td>Pancreas </td>
<td>760 </td>
<td>83 </td>
<td>818 </td>
<td>143 </td>
</tr>
<tr>
<td>Skeletal muscle </td>
<td>792 </td>
<td>63 </td>
<td>1,215 </td>
<td>157 </td>
</tr>
</tbody>
</table></div>
</div>                        <a id="131979815" scrollto-destination="131979815"></a>
<div id="dsy038-F4" data-id="dsy038-F4" class="fig fig-section">
<div class="label fig-label">Figure 4.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f4.png?Expires=1576183819&amp;Signature=jJwC9fVuzdOIDjXksVWhwZxcYW1hTuKixieAVmRAjptIbKqV5N-tdKwZxF9WuTNVUqkteXxubhS8WT1WA~0OhBFKbgNcpV66OVmhwgK-PIxEiPSr9Zzvuig~k1qEkBYYI6LpDnM2Zg~hQgMUZQCfpxKJR8zsG0UkbdL9beyJeQA~R25L4ycTN6UErnESfiqyIKb~Rj64y8YpO0~1jdm6BX~22mZ3FDK1BgnIiYJxIHRYNugB1Mb-b~oL8Xfrzt5bEOqnu5SajXWw9uifvdDdCU0Xwv-nVD7aYVhkc4K5HiWbui-3I6u3pZEhLDgbD2VtU3dYQ16zRN3~ATa6ptAc6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of differential allelic expression using FL-cDNA-Seq reads. We exemplified two differentially expressed allelic genes: UBXN4 (A) in colon and liver of the female sample and GNAS (B) in merged data of the male sample. We showed the FL-cDNA-Seq reads separately by discrete SNP patterns. The positions of the SNPs are marked by arrows. (A) We also showed the expression patterns of UBXN4 in each tissues of the female sample. *P &lt; 0.01." path-from-xml="dsy038f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979815/dsy038f4.tif" path-from-xml="dsy038f4.tif" target="_blank">Open in new tab</a><a section="131979815" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f4.png?Expires=1576183819&amp;Signature=c5DfKXnNZndhon8UwKh0y0RjiRkxY8Jg90APKu8CODqRZZNegAX2P-SizljrPRQGTRqZXIV7pyB-3d2w3yIpDGOYchoO7c-WbPEmIwUNzDSN3b9e1uQOUuPj4IuVW3ligh5GMOe5hJlxuoscMToCetaj2zmDAZ2ObKHE8v2NmGjvLKsyWNCgfI0W6CmcF3n5en9cyKdtIQ8qsq6uP62wP9fisUMwEPeHY7KyuiDD-jLx0ubQR5023hB5CF-ew5cZ4BaWglel5b7AH5y4HrIJpkLI7D4w7bWMidV-FIYExqLc1TK96xWW28Hztbl~MrpD2UOQxsiaf0f12ZAx8VleBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979815&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detection of differential allelic expression using FL-cDNA-Seq reads. We exemplified two differentially expressed allelic genes: UBXN4 (A) in colon and liver of the female sample and GNAS (B) in merged data of the male sample. We showed the FL-cDNA-Seq reads separately by discrete SNP patterns. The positions of the SNPs are marked by arrows. (A) We also showed the expression patterns of UBXN4 in each tissues of the female sample. *<em>P</em> &lt; 0.01.</p></div>
</div>
<div content-id="dsy038-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f4.png?Expires=1576183819&amp;Signature=jJwC9fVuzdOIDjXksVWhwZxcYW1hTuKixieAVmRAjptIbKqV5N-tdKwZxF9WuTNVUqkteXxubhS8WT1WA~0OhBFKbgNcpV66OVmhwgK-PIxEiPSr9Zzvuig~k1qEkBYYI6LpDnM2Zg~hQgMUZQCfpxKJR8zsG0UkbdL9beyJeQA~R25L4ycTN6UErnESfiqyIKb~Rj64y8YpO0~1jdm6BX~22mZ3FDK1BgnIiYJxIHRYNugB1Mb-b~oL8Xfrzt5bEOqnu5SajXWw9uifvdDdCU0Xwv-nVD7aYVhkc4K5HiWbui-3I6u3pZEhLDgbD2VtU3dYQ16zRN3~ATa6ptAc6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of differential allelic expression using FL-cDNA-Seq reads. We exemplified two differentially expressed allelic genes: UBXN4 (A) in colon and liver of the female sample and GNAS (B) in merged data of the male sample. We showed the FL-cDNA-Seq reads separately by discrete SNP patterns. The positions of the SNPs are marked by arrows. (A) We also showed the expression patterns of UBXN4 in each tissues of the female sample. *P &lt; 0.01." path-from-xml="dsy038f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979815/dsy038f4.tif" path-from-xml="dsy038f4.tif" target="_blank">Open in new tab</a><a section="131979815" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f4.png?Expires=1576183819&amp;Signature=c5DfKXnNZndhon8UwKh0y0RjiRkxY8Jg90APKu8CODqRZZNegAX2P-SizljrPRQGTRqZXIV7pyB-3d2w3yIpDGOYchoO7c-WbPEmIwUNzDSN3b9e1uQOUuPj4IuVW3ligh5GMOe5hJlxuoscMToCetaj2zmDAZ2ObKHE8v2NmGjvLKsyWNCgfI0W6CmcF3n5en9cyKdtIQ8qsq6uP62wP9fisUMwEPeHY7KyuiDD-jLx0ubQR5023hB5CF-ew5cZ4BaWglel5b7AH5y4HrIJpkLI7D4w7bWMidV-FIYExqLc1TK96xWW28Hztbl~MrpD2UOQxsiaf0f12ZAx8VleBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979815&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detection of differential allelic expression using FL-cDNA-Seq reads. We exemplified two differentially expressed allelic genes: UBXN4 (A) in colon and liver of the female sample and GNAS (B) in merged data of the male sample. We showed the FL-cDNA-Seq reads separately by discrete SNP patterns. The positions of the SNPs are marked by arrows. (A) We also showed the expression patterns of UBXN4 in each tissues of the female sample. *<em>P</em> &lt; 0.01.</p></div>
</div>                    <h4 scrollto-destination="131979816" id="131979816" class="section-title" data-legacy-id="SEC3.6.4">3.6.4. Single cell FL-cDNA-Seq</h4>
<p>Although another group has already applied RNA-Seq using MinION to single-cell RNA-Seq, we also considered whether FL-cDNA-Seq is also applicable to this technique.<a reveal-id="dsy038-B13" data-open="dsy038-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Some existing methods for single-cell RNA-Seq also uses cDNA templates prepared by the same SMART-Seq v4.<a reveal-id="dsy038-B25" data-open="dsy038-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsy038-B26" data-open="dsy038-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Full-length cDNA was synthesized and amplified using the Fluidigm C1 system. To obtain sufficient amounts of input template for FL-cDNA-Seq, five additional cycles of PCR were employed. Approximately 250–450 ng of cDNA templates were obtained from each of the single cells and subjected to FL-cDNA-Seq. The libraries obtained from one bulk and one doublet cell, as well as nine single cells, were barcoded and loaded onto a single R9.4 flow cell. A total of 24,817 reads (469–6,522 per samples) were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). For comparison, we also conducted Illumina sequencing using Nextera XT-processed cDNA obtained from the C1 platform (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576183819&amp;Signature=C0zFQWq8J6U4aOL0iE2M77v-w3mLqwswVFHkaenIPbSVQpVJFG8gD8t3KuH2ZjSjy3HsGyUdMpWkCcG9yR1fSAiP19Y-z7qQKffD7JlP6PnJxrgn-pCKrEelFLYe5TU~j2K8PlKu--QoMGJiJ5tNWAL8YiQIlY2PcCubE1rsrEXnbBSPz-FsZNKa9xISAbUqu2Nq98jVGoKFVDf~xuBUKDyM9bRsuemVVtU63oW28e5L8Wo7w1QeSoA3g-KFDsL810LN86EYF-c6tXv6HSbYMrInyT1L2LCPVxErLsFJTjNCK~eqGikK6lHyOFaTXoj0aVfEmmhC3ET9qayIYiRkEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15</a></span>). We compared the Illumina SMART-Seq and FL-cDNA-Seq of single cells. When we considered the correlation between SMART-Seq and FL-cDNA-Seq of the aggregated reads of nine single cells (‘virtual bulks’), the correlation was reasonably high (<em>R</em> = 0.63) (<a reveal-id="dsy038-F5" data-open="dsy038-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). However, when calculated at the single cell level, the correlation was <em>R</em> = 0.32–0.69 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). The low correlation efficiencies reflected the fact that the number of reads was small for the FL-cDNA-Seq reads (<a reveal-id="dsy038-F5" data-open="dsy038-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). We also compared the single cell data and bulk data of FL-cDNA-Seq, prepared using the same protocol. Between the virtual bulk of single cells and bulk, the correlation was moderately high (<em>R</em> = 0.69) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9A</a></span>). When calculated between the single cells and bulk, the correlation was <em>R</em> = 0.11–0.67 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576183819&amp;Signature=d9ldrW9vQYFJMkPNVUQ~1IouQdjLcZ-3cRTbE~izZIxvAuEbZYwZeTA1mdwd3UZ4lahsQQPhl51-NbxJvWk5LFBf1FoK~KkF1ZE~wSmQ2XdZe5DTyMJCIi4mO9IKc3WilEergkPPkwPUTsZ5bbMxF2GgaAS4SaeeWW3lSWOtSu5~YZNDsM4-nznQn5jeoJfbYs7Ej5apIjDiRGyYyu6IaLFf4cSikyzho1DNTpttzohj~e3S2fOAAAkrLT2Mcw36vCjWq6rBuMz1kg8A2LLQVGCnDLvgKO-yXdpjocZ1w8lkL~lkF73DWg-vikl8KiZnqmirrKYnwkBPJWBKzLr0MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9B</a></span>). The overall accuracy and RefSeq coverage of FL-cDNA-Seq reads were 93.2% and 43.2% (<a reveal-id="dsy038-F5" data-open="dsy038-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). Although the still limited throughput of the MinION sequencer is not suitable for gene expression analysis at the single cell level, we found that some full-length transcript variants were over-represented in particular individual cells. MinION sequencing can be complementarily used for this purpose. </p>                        <a id="131979818" scrollto-destination="131979818"></a>
<div id="dsy038-F5" data-id="dsy038-F5" class="fig fig-section">
<div class="label fig-label">Figure 5.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f5.png?Expires=1576183819&amp;Signature=BMsT4mA68n4Ku5lnr2DaI7R7D9VKvZB4iQDpd9AnEZFB-SxvPOgmLbMMq4dFy4KaVjm-Qshl0yg6p7GgEJdu9yaacOSGIKmowP8oueNZC4JFTscEC534xvWiIvb-69KaxVhDSJxMudSyf6Lw-Y1ISTHgtiwDqSwDTwUxYZpA1Xs2oaKn6wXosuiIwCaydtx6hLOpaVurD42n5SkgwId8uS9OxpCSbRcygD8tCNcvI8gkc2~5EEbOnQnmrzsAZuX~zMHwgmJRguCV1t6io-Fchc-aJuhz~jUqtTfkCya4qo3wtBBMdQl02dnqvwVrS~ojI5SSNpL-CTF37wmQhn-wOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Single cell FL-cDNA-Seq using C1. (A) Comparison of the expression level of the virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq and SMART-Seq. (B) Comparison of the expression level of bulk and virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq. (C) Correlation of the number of mapped reads of FL-cDNA-Seq and the correlation coefficient between the same single cell data quantified by FL-cDNA-Seq using MinION. The Pearson correlation coefficient between them is shown in the graph. (A, B) The Pearson correlation coefficient was calculated using the genes detected for both data sets." path-from-xml="dsy038f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979818/dsy038f5.tif" path-from-xml="dsy038f5.tif" target="_blank">Open in new tab</a><a section="131979818" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f5.png?Expires=1576183819&amp;Signature=qEBSuuhAZQazSxfoYUTAXDzMVUY5sgEhLgr4I7nzrxC9mB91H2I6xNMD3ODXr~ul6RhDIBFBTD7dBK~FTQy6lW-iMjsf7C1Hh1yHSw-3Ud6s9THAQu5bCUIHilAoRbs3PuwjmCMgBCJ98DTSlN6CAjF26UVn9-z-lw1Wco2578tn~AdEcDD7PVZkIv5gYQvlxH6m6~BIpC7cgDvI1lfUM3aXrj90G8q2HMB~DtVmjS9nesM3RwWKTw80ZVY4VFapmr6vCMaXwe4yUDE53ARvXsM8C-yUZZUuEOY4dhI9rPMiudxm9bI9qA610rr~SJpTIHq6~m7VMQpGY~zIKY6kxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979818&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Single cell FL-cDNA-Seq using C1. (A) Comparison of the expression level of the virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq and SMART-Seq. (B) Comparison of the expression level of bulk and virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq. (C) Correlation of the number of mapped reads of FL-cDNA-Seq and the correlation coefficient between the same single cell data quantified by FL-cDNA-Seq using MinION. The Pearson correlation coefficient between them is shown in the graph. (A, B) The Pearson correlation coefficient was calculated using the genes detected for both data sets.</p></div>
</div>
<div content-id="dsy038-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/m_dsy038f5.png?Expires=1576183819&amp;Signature=BMsT4mA68n4Ku5lnr2DaI7R7D9VKvZB4iQDpd9AnEZFB-SxvPOgmLbMMq4dFy4KaVjm-Qshl0yg6p7GgEJdu9yaacOSGIKmowP8oueNZC4JFTscEC534xvWiIvb-69KaxVhDSJxMudSyf6Lw-Y1ISTHgtiwDqSwDTwUxYZpA1Xs2oaKn6wXosuiIwCaydtx6hLOpaVurD42n5SkgwId8uS9OxpCSbRcygD8tCNcvI8gkc2~5EEbOnQnmrzsAZuX~zMHwgmJRguCV1t6io-Fchc-aJuhz~jUqtTfkCya4qo3wtBBMdQl02dnqvwVrS~ojI5SSNpL-CTF37wmQhn-wOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Single cell FL-cDNA-Seq using C1. (A) Comparison of the expression level of the virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq and SMART-Seq. (B) Comparison of the expression level of bulk and virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq. (C) Correlation of the number of mapped reads of FL-cDNA-Seq and the correlation coefficient between the same single cell data quantified by FL-cDNA-Seq using MinION. The Pearson correlation coefficient between them is shown in the graph. (A, B) The Pearson correlation coefficient was calculated using the genes detected for both data sets." path-from-xml="dsy038f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979818/dsy038f5.tif" path-from-xml="dsy038f5.tif" target="_blank">Open in new tab</a><a section="131979818" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038f5.png?Expires=1576183819&amp;Signature=qEBSuuhAZQazSxfoYUTAXDzMVUY5sgEhLgr4I7nzrxC9mB91H2I6xNMD3ODXr~ul6RhDIBFBTD7dBK~FTQy6lW-iMjsf7C1Hh1yHSw-3Ud6s9THAQu5bCUIHilAoRbs3PuwjmCMgBCJ98DTSlN6CAjF26UVn9-z-lw1Wco2578tn~AdEcDD7PVZkIv5gYQvlxH6m6~BIpC7cgDvI1lfUM3aXrj90G8q2HMB~DtVmjS9nesM3RwWKTw80ZVY4VFapmr6vCMaXwe4yUDE53ARvXsM8C-yUZZUuEOY4dhI9rPMiudxm9bI9qA610rr~SJpTIHq6~m7VMQpGY~zIKY6kxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979818&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy038f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Single cell FL-cDNA-Seq using C1. (A) Comparison of the expression level of the virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq and SMART-Seq. (B) Comparison of the expression level of bulk and virtual bulk of nine single cells of LC2/ad quantified by FL-cDNA-Seq. (C) Correlation of the number of mapped reads of FL-cDNA-Seq and the correlation coefficient between the same single cell data quantified by FL-cDNA-Seq using MinION. The Pearson correlation coefficient between them is shown in the graph. (A, B) The Pearson correlation coefficient was calculated using the genes detected for both data sets.</p></div>
</div>                    <h2 scrollto-destination="131979819" id="131979819" class="section-title" data-legacy-id="SEC4">4. Conclusions</h2>
<p>In this article, we evaluated current MinION sequencing and incorporated some modifications depending on the various applications. To our knowledge, this is the first article to survey the potential application of FL-cDNA-Seq, which may not be consistently easy with the current Illumina RNA-Seq. Indeed, our current knowledge regarding the transcriptome is almost limited to that obtained from Illumina or microarrays, which generally only represents information about fragmented transcripts. In contrast, FL-cDNA-Seq has enabled analysis of the entire transcript with respect to expression and base information at the isoform level. In this article, we demonstrated that these features first could enable one to take advantage of MinION long reads to gain novel insights in addition to the current transcriptome view. A no less important unique feature of the MinION sequencer is that it permits on-site sequencing without requiring any laboratory instruments for the sequencing. Because the flow cell as well as the data process software have been improved, now the sequencing yield has reached to more than 1 million reads. In our previous study, MinION sequencing of PCR amplicon from cDNA of clinical samples was performed.<a reveal-id="dsy038-B27" data-open="dsy038-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> We could detect fusion genes and phase the hetero SNVs. It is assumed that FL-cDNA-Seq is also applicable for clinical samples. These advantages with the portable feature will further accelerated the compilation of transcriptome data. With the expansion of transcriptome information, we will be able to understand how the code of the genome and its mutations are linked to gene expression.</p>                    <h2 scrollto-destination="131979821" id="131979821" class="backacknowledgements-title" data-legacy-id="Ack1">Acknowledgements</h2>
<p>We sincerely thank Hiroyuki Wakaguri, Terumi Horiuchi, and Yuta Kuze for assistance with the data analysis, and Kazumi Abe, Kiyomi Imamura, Yuni Ishikawa, Mari Tsubaki, Sachie Shimazu, Megumi Kombu, Etsuko Kobayashi, and Yusuke Ogishi for their experimental support.</p>                    <h3 scrollto-destination="131979823" id="131979823" class="section-title">Accession numbers</h3>
<p>The raw sequence reads generated in this study have been deposited at DDBJ under accession numbers RA006943D and DRA006942.</p>                    <h3 scrollto-destination="131979825" id="131979825" class="section-title">Funding</h3>
<p>This work was supported by JSPS KAKENHI Grant Number 16H06279 and JST CREST Grant Number JPMJCR15G3.</p>                    <h3 scrollto-destination="131979827" id="131979827" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="131979829" id="131979829" class="backreferences-title" data-legacy-id="Ref1">References</h2>
<div class="ref-list">
<div content-id="dsy038-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Gerstein</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Snyder</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">RNA-Seq: a revolutionary tool for transcriptomics</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">10</div>, <div class="fpage">57</div>–<div class="lpage">63</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RNA-Seq%3A%20a%20revolutionary%20tool%20for%20transcriptomics&amp;author=Z.%20Wang&amp;author=M.%20Gerstein&amp;author=M.%20Snyder&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet&amp;volume=10&amp;pages=57-63" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg2484" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2484" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg2484"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015660" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RNA-Seq%3A%20a%20revolutionary%20tool%20for%20transcriptomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pan</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Shai</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">L. J.</div>
</div>, <div class="name string-name">
<div class="surname">Frey</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Blencowe</div> <div class="given-names">B. J.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Deep surveying of alternative splicing complexity in the human transcriptome by high-throughput sequencing</div>, <div class="source">Nat. Genet</div> ., <div class="volume">40</div>, <div class="fpage">1413</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;author=Q.%20Pan&amp;author=O.%20Shai&amp;author=L.%20J.%20Lee&amp;author=B.%20J.%20Frey&amp;author=B.%20J.%20Blencowe&amp;publication_year=2008&amp;journal=Nat.%20Genet&amp;volume=40&amp;pages=1413-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.259" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.259" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.259"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978789" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Deep%20surveying%20of%20alternative%20splicing%20complexity%20in%20the%20human%20transcriptome%20by%20high-throughput%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Piskol</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Ramaswami</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">J. B.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Reliable identification of genomic variants from RNA-seq data</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">93</div>, <div class="fpage">641</div>–<div class="lpage">51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Reliable%20identification%20of%20genomic%20variants%20from%20RNA-seq%20data&amp;author=R.%20Piskol&amp;author=G.%20Ramaswami&amp;author=J.%20B.%20Li&amp;publication_year=2013&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=93&amp;pages=641-51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ajhg.2013.08.008" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2013.08.008" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ajhg.2013.08.008"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24075185" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Reliable%20identification%20of%20genomic%20variants%20from%20RNA-seq%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Steijger</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Abril</div> <div class="given-names">J. F.</div>
</div>, <div class="name string-name">
<div class="surname">Engström</div> <div class="given-names">P. G.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Assessment of transcript reconstruction methods for RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1177</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=J.%20F.%20Abril&amp;author=P.%20G.%20Engstr%C3%B6m&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1177-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.2714" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2714" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.2714"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goodwin</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">McPherson</div> <div class="given-names">J. D.</div>
</div>, <div class="name string-name">
<div class="surname">McCombie</div> <div class="given-names">W. R.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Coming of age: ten years of next-generation sequencing technologies</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">17</div>, <div class="fpage">333</div>–<div class="lpage">51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;author=S.%20Goodwin&amp;author=J.%20D.%20McPherson&amp;author=W.%20R.%20McCombie&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Genet&amp;volume=17&amp;pages=333-51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg.2016.49" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2016.49" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg.2016.49"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27184599" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Browning</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Browning</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Haplotype phasing: existing methods and new developments</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">703</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Haplotype%20phasing%3A%20existing%20methods%20and%20new%20developments&amp;author=S.%20Browning&amp;author=B.%20Browning&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=703-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg3054" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3054" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg3054"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921926" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Haplotype%20phasing%3A%20existing%20methods%20and%20new%20developments&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sharon</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Tilgner</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Grubert</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Snyder</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">A single-molecule long-read survey of the human transcriptome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">1009</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20single-molecule%20long-read%20survey%20of%20the%20human%20transcriptome&amp;author=D.%20Sharon&amp;author=H.%20Tilgner&amp;author=F.%20Grubert&amp;author=M.%20Snyder&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=1009-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.2705" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2705" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.2705"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24108091" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20single-molecule%20long-read%20survey%20of%20the%20human%20transcriptome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rhoads</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Au</div> <div class="given-names">K. F.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">PacBio sequencing and its applications</div>, <div class="source">Genomics. Proteomics Bioinformatics</div> ., <div class="volume">13</div>, <div class="fpage">278</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PacBio%20sequencing%20and%20its%20applications&amp;author=A.%20Rhoads&amp;author=K.%20F.%20Au&amp;publication_year=2015&amp;journal=Genomics.%20Proteomics%20Bioinformatics&amp;volume=13&amp;pages=278-89" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gpb.2015.08.002" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gpb.2015.08.002" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gpb.2015.08.002"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26542840" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PacBio%20sequencing%20and%20its%20applications&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">Pacific Biosciences. Revolutionize Genomics with SMRT Sequencing. <a class="link link-uri" href="https://www.pacb.com/wp-content/uploads/SMRT-Sequencing-Brochure-Revolutionize-genomics-with-SMRT-Sequencing.pdf" target="_blank">https://www.pacb.com/wp-content/uploads/SMRT-Sequencing-Brochure-Revolutionize-genomics-with-SMRT-Sequencing.pdf</a> (accessed 21 June 2018).</div>
</div></div></div>
<div content-id="dsy038-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jain</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Olsen</div> <div class="given-names">H. E.</div>
</div>, <div class="name string-name">
<div class="surname">Paten</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">The Oxford Nanopore MinION: delivery of nanopore sequencing to the genomics community</div>, <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">239</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Oxford%20Nanopore%20MinION%3A%20delivery%20of%20nanopore%20sequencing%20to%20the%20genomics%20community&amp;author=M.%20Jain&amp;author=H.%20E.%20Olsen&amp;author=B.%20Paten&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=239" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-016-1103-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-1103-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-016-1103-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27887629" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Oxford%20Nanopore%20MinION%3A%20delivery%20of%20nanopore%20sequencing%20to%20the%20genomics%20community&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jain</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Koren</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Miga</div> <div class="given-names">K. H.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Nanopore sequencing and assembly of a human genome with ultra-long reads</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">36</div>, <div class="fpage">338</div>–<div class="lpage">45</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nanopore%20sequencing%20and%20assembly%20of%20a%20human%20genome%20with%20ultra-long%20reads&amp;author=M.%20Jain&amp;author=S.%20Koren&amp;author=K.%20H.%20Miga&amp;publication_year=2018&amp;journal=Nat.%20Biotechnol&amp;volume=36&amp;pages=338-45" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.4060" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.4060" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.4060"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29431738" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nanopore%20sequencing%20and%20assembly%20of%20a%20human%20genome%20with%20ultra-long%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oikonomopoulos</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y. C.</div>
</div>, <div class="name string-name">
<div class="surname">Djambazian</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Badescu</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Ragoussis</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Benchmarking of the Oxford Nanopore MinION sequencing for quantitative and qualitative assessment of cDNA populations</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">31602</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Benchmarking%20of%20the%20Oxford%20Nanopore%20MinION%20sequencing%20for%20quantitative%20and%20qualitative%20assessment%20of%20cDNA%20populations&amp;author=S.%20Oikonomopoulos&amp;author=Y.%20C.%20Wang&amp;author=H.%20Djambazian&amp;author=D.%20Badescu&amp;author=J.%20Ragoussis&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=31602" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep31602" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep31602" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep31602"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27554526" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Benchmarking%20of%20the%20Oxford%20Nanopore%20MinION%20sequencing%20for%20quantitative%20and%20qualitative%20assessment%20of%20cDNA%20populations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Byrne</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Beaudin</div> <div class="given-names">A. E.</div>
</div>, <div class="name string-name">
<div class="surname">Olsen</div> <div class="given-names">H. E.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Nanopore long-read RNAseq reveals widespread transcriptional variation among the surface receptors of individual B cells</div>, <div class="source">Nat. Commun</div> , <div class="volume">8</div>, <div class="fpage">16027</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nanopore%20long-read%20RNAseq%20reveals%20widespread%20transcriptional%20variation%20among%20the%20surface%20receptors%20of%20individual%20B%20cells&amp;author=A.%20Byrne&amp;author=A.%20E.%20Beaudin&amp;author=H.%20E.%20Olsen&amp;publication_year=2017&amp;journal=Nat.%20Commun&amp;volume=8&amp;pages=16027" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ncomms16027" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms16027" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fncomms16027"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28722025" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nanopore%20long-read%20RNAseq%20reveals%20widespread%20transcriptional%20variation%20among%20the%20surface%20receptors%20of%20individual%20B%20cells&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Picelli</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Björklund</div> <div class="given-names">Å. K.</div>
</div>, <div class="name string-name">
<div class="surname">Faridani</div> <div class="given-names">O. R.</div>
</div>, <div class="name string-name">
<div class="surname">Sagasser</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Winberg</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Sandberg</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Smart-seq2 for sensitive full-length transcriptome profiling in single cells</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1096</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Smart-seq2%20for%20sensitive%20full-length%20transcriptome%20profiling%20in%20single%20cells&amp;author=S.%20Picelli&amp;author=%C3%85.%20K.%20Bj%C3%B6rklund&amp;author=O.%20R.%20Faridani&amp;author=S.%20Sagasser&amp;author=G.%20Winberg&amp;author=R.%20Sandberg&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1096-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.2639" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2639" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.2639"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24056875" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Smart-seq2%20for%20sensitive%20full-length%20transcriptome%20profiling%20in%20single%20cells&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Kent</div> <div class="given-names">W. J.</div>
</div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT—The BLAST-like alignment tool</div>, <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94The%20BLAST-like%20alignment%20tool&amp;author=W.%20J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.229202"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BLAT%E2%80%94The%20BLAST-like%20alignment%20tool&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Suzuki</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Makinoshima</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Wakaguri</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Aberrant transcriptional regulations in cancers: genome, transcriptome and epigenome analysis of lung adenocarcinoma cell lines</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">13557</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Aberrant%20transcriptional%20regulations%20in%20cancers%3A%20genome%2C%20transcriptome%20and%20epigenome%20analysis%20of%20lung%20adenocarcinoma%20cell%20lines&amp;author=A.%20Suzuki&amp;author=H.%20Makinoshima&amp;author=H.%20Wakaguri&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=13557-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gku885" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku885" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgku885"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25378332" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Aberrant%20transcriptional%20regulations%20in%20cancers%3A%20genome%2C%20transcriptome%20and%20epigenome%20analysis%20of%20lung%20adenocarcinoma%20cell%20lines&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Depristo</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Banks</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Poplin</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>, <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">501</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.%20A.%20Depristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=491-501" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.806"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hamada</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Ono</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Asai</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Frith</div> <div class="given-names">M. C.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Training alignment parameters for arbitrary sequencers with LAST-TRAIN</div>, <div class="source">Bioinformatics</div> , <div class="volume">33</div>, <div class="fpage">926</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Training%20alignment%20parameters%20for%20arbitrary%20sequencers%20with%20LAST-TRAIN&amp;author=M.%20Hamada&amp;author=Y.%20Ono&amp;author=K.%20Asai&amp;author=M.%20C.%20Frith&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=926-8" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28039163" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Training%20alignment%20parameters%20for%20arbitrary%20sequencers%20with%20LAST-TRAIN&amp;aulast=Hamada&amp;date=2017&amp;spage=926&amp;epage=8&amp;volume=33" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Training%20alignment%20parameters%20for%20arbitrary%20sequencers%20with%20LAST-TRAIN&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Frith</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Kawaguchi</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Split-alignment of genomes finds orthologies more accurately</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">106</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Split-alignment%20of%20genomes%20finds%20orthologies%20more%20accurately&amp;author=M.%20C.%20Frith&amp;author=R.%20Kawaguchi&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=106" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-015-0670-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0670-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0670-9"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25994148" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Split-alignment%20of%20genomes%20finds%20orthologies%20more%20accurately&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Durbin</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp324"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garalde</div> <div class="given-names">D. R.</div>
</div>, <div class="name string-name">
<div class="surname">Snell</div> <div class="given-names">E. A.</div>
</div>, <div class="name string-name">
<div class="surname">Jachimowicz</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Highly parallel direct RN A sequencing on an array of nanopores</div>, <div class="source">Nat. Methods</div> , <div class="volume">15</div>, <div class="fpage">201</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Highly%20parallel%20direct%20RN%20A%20sequencing%20on%20an%20array%20of%20nanopores&amp;author=D.%20R.%20Garalde&amp;author=E.%20A.%20Snell&amp;author=D.%20Jachimowicz&amp;publication_year=2018&amp;journal=Nat.%20Methods&amp;volume=15&amp;pages=201-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.4577" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.4577" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.4577"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29334379" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Highly%20parallel%20direct%20RN%20A%20sequencing%20on%20an%20array%20of%20nanopores&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kobayashi</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Boggon</div> <div class="given-names">T. J.</div>
</div>, <div class="name string-name">
<div class="surname">Dayaram</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2005</div>, 
<div class="article-title">
<em>EGFR</em> mutation and resistance of non–small-cell lung cancer to Gefitinib</div>, <div class="source">N. Engl. J. Med</div> ., <div class="volume">352</div>, <div class="fpage">786</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=EGFR%20mutation%20and%20resistance%20of%20non%E2%80%93small-cell%20lung%20cancer%20to%20Gefitinib&amp;author=S.%20Kobayashi&amp;author=T.%20J.%20Boggon&amp;author=T.%20Dayaram&amp;publication_year=2005&amp;journal=N.%20Engl.%20J.%20Med&amp;volume=352&amp;pages=786-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1056/NEJMoa044238" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa044238" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1056%2FNEJMoa044238"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728811" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:EGFR%20mutation%20and%20resistance%20of%20non%E2%80%93small-cell%20lung%20cancer%20to%20Gefitinib&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hayward</div> <div class="given-names">B. E.</div>
</div>, <div class="name string-name">
<div class="surname">Moran</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Strain</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Bonthron</div> <div class="given-names">D. T.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Bidirectional imprinting of a single gene: gNAS1 encodes maternally, paternally, and biallelically derived proteins</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">95</div>, <div class="fpage">15475</div>–<div class="lpage">80</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bidirectional%20imprinting%20of%20a%20single%20gene%3A%20gNAS1%20encodes%20maternally%2C%20paternally%2C%20and%20biallelically%20derived%20proteins&amp;author=B.%20E.%20Hayward&amp;author=V.%20Moran&amp;author=L.%20Strain&amp;author=D.%20T.%20Bonthron&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=95&amp;pages=15475-80" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.95.26.15475" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.26.15475" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.95.26.15475"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9860993" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bidirectional%20imprinting%20of%20a%20single%20gene%3A%20gNAS1%20encodes%20maternally%2C%20paternally%2C%20and%20biallelically%20derived%20proteins&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Turan</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Bastepe</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">The GNAS complex locus and human diseases associated with loss-of-function mutations or epimutations within this imprinted gene</div>, <div class="source">Horm. Res. Paediatr</div> ., <div class="volume">80</div>, <div class="fpage">229</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20GNAS%20complex%20locus%20and%20human%20diseases%20associated%20with%20loss-of-function%20mutations%20or%20epimutations%20within%20this%20imprinted%20gene&amp;author=S.%20Turan&amp;author=M.%20Bastepe&amp;publication_year=2013&amp;journal=Horm.%20Res.%20Paediatr&amp;volume=80&amp;pages=229-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1159/000355384" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000355384" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1159%2F000355384"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24107509" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20GNAS%20complex%20locus%20and%20human%20diseases%20associated%20with%20loss-of-function%20mutations%20or%20epimutations%20within%20this%20imprinted%20gene&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fish</div> <div class="given-names">R. N.</div>
</div>, <div class="name string-name">
<div class="surname">Bostick</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Lehman</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Farmer</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2016</div>, Transcriptome analysis at the single-cell level using SMART technology. In: <div class="source">Current Protocols in Molecular Biology</div> . 
<div class="publisher-name">John Wiley &amp; Sons, Inc</div>.: 
<div class="publisher-loc">Hoboken, NJ</div>, pp. 4.26.1–4.26.24.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Current%20Protocols%20in%20Molecular%20Biology&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Current%20Protocols%20in%20Molecular%20Biology&amp;aulast=Fish&amp;date=2016" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Current%20Protocols%20in%20Molecular%20Biology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Current%20Protocols%20in%20Molecular%20Biology">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yamamoto</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Gotoh</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Korogi</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Long-term expansion of alveolar stem cells derived from human iPS cells in organoids</div>, <div class="source">Nat. Methods</div> , <div class="volume">14</div>, <div class="fpage">1097</div>–<div class="lpage">106</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Long-term%20expansion%20of%20alveolar%20stem%20cells%20derived%20from%20human%20iPS%20cells%20in%20organoids&amp;author=Y.%20Yamamoto&amp;author=S.%20Gotoh&amp;author=Y.%20Korogi&amp;publication_year=2017&amp;journal=Nat.%20Methods&amp;volume=14&amp;pages=1097-106" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.4448" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.4448" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.4448"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28967890" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Long-term%20expansion%20of%20alveolar%20stem%20cells%20derived%20from%20human%20iPS%20cells%20in%20organoids&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy038-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy038-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Suzuki</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Suzuki</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Mizushima-Sugano</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer</div>, <div class="source">DNA Res</div> ., <div class="volume">24</div>, <div class="fpage">585</div>–<div class="lpage">96</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20phasing%20cancer%20mutations%20in%20lung%20cancers%20using%20a%20long-read%20portable%20sequencer&amp;author=A.%20Suzuki&amp;author=M.%20Suzuki&amp;author=J.%20Mizushima-Sugano&amp;publication_year=2017&amp;journal=DNA%20Res&amp;volume=24&amp;pages=585-96" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsx027" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsx027" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsx027"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29117310" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequencing%20and%20phasing%20cancer%20mutations%20in%20lung%20cancers%20using%20a%20long-read%20portable%20sequencer&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="dsy038-FM1"><div class="fn"><div class="footnote-content">Masahide Seki and Eri Katsumata authors equally contributed to this article.</div></div></div>
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5194314&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementary_methods.pdf?Expires=1576679287&amp;Signature=EoPWJckss6STJ6rpVwSBCTz~CW2EU4m-IzwsiT4Nd6EPzz~~~wNGQSOHqXXPnupUMckxxgYaC00PeW-NFUXJaYJAOC3GZkqXHkZtz-SoUr4vul3JgJOWLfzMrwhpfts4Jn~go~Oi4DMXn60wkuN6YFUFmU82W-7LnBU0SBdHXO3IQU0ThDZVmSl3cW1tdZXownU7RRCZMkkO4rPbj6VpP7BfL4JtFBeTWm58o7HJ-QteqxpUY4ujqQO6xvRl8CWb~JSojLqioaesZX4hMvim1HHvPIuGa1d4lcarRqQBaHvoq5GDuQeZOEsd0MDQgMiyv0Cim-JlfE47YOfYx-Hqxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Methods</div></a> - pdf file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy038/2/dsy038_supplementray_tables.xlsx?Expires=1576679287&amp;Signature=FJeHQ0~rjP8K-TpcCzVylEeqqswHaHZ1kffu54ncOmEdhG~~gWkAg83a2ASP3fw~SB6PlM8EqPdY4SPI~gmE~s1KS4Fmjj8hfBaVUqQjmflo1dmGvEOBXmjUToFkzjSlfAgm2qubpSq65Ms-sjWQfUCTGDo6qqo0wILYdft38MeaQ1tlfjcZQM6KtMRZLDXMVq-bQ5-GxnzMWp71j4dMhFVBweXqsCHAwAZjlloLbBWgZbz43NH332vug1Fw0wjQNhYZi-Vaq4GpGkLjSAGcN3Fi8hDbXAA7Uz671~BjbgRULuh2mDsqdHFWdV1xDUt0bAUd2GRGzygUDvv6ZU9nPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Tables</div></a> - xlsx file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">5,972</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy038"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5194314"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f1%2f55%2f5194314">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F1%2F55%2F5194314" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5194314">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Evaluation and application of RNA-Seq by MinION | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy038");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f55%2f5194314">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
