<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Search for potential reading frameshifts in cds from Arabidopsis thaliana and other genomes | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/2/157/5306610","@type":"ScholarlyArticle","name":"Search for potential reading frameshifts in cds from  Arabidopsis thaliana  and other genomes","datePublished":"2019-02-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2019-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy046","keywords":["reading frameshift","periodicity","dynamic programming","genetic algorithm"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/2/157/5306610","author":[{"name":"Suvorova, Y M","affiliation":"Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia","@type":"Person"},{"name":"Korotkova, M A","affiliation":"National Research Nuclear University MEPhI (Moscow Engineering Physics Institute), Moscow, Russia","@type":"Person"},{"name":"Skryabin, K G","affiliation":"Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia","@type":"Person"},{"name":"Korotkov, E V","affiliation":"Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia  National Research Nuclear University MEPhI (Moscow Engineering Physics Institute), Moscow, Russia","@type":"Person"}],"description":"Abstract.  A new mathematical method for potential reading frameshift detection in protein-coding sequences (cds) was developed. The algorithm is adjusted to th","pageStart":"157","pageEnd":"170","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636262148&Signature=CKdpCc5sTlPdjmLWt5RVE5ZXdFKmm-um93zqrYCz2biV-m0hnA2Y622l6pxS4la-dsZDLZagTdYbXmk1A61Tokseu2we~vHc4F6eATGoXgdhxcaqOVown3NlUBrKi9x09TKlJKKM0yNVhw6Lr3nLo88NHCQWxKamk4zusJxX2xrTM63E2ersLEye5dUYdCMLTwVyNk9jaBWkPyJ1mAV4CWH4zulM~oZYhYeQLqF6u9XvfIGQ7mGO98PgdDfdkL4ywremu6MXzVD51GDFcQ1Q4rNFX8hYH7dhXJzZoZWpzinpynWM5pYhtsDVFgGVgxulYKAgILJuDZbRW1H2DUUUEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Search for potential reading frameshifts in cds from  Arabidopsis thaliana  and other genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636262148&Signature=CKdpCc5sTlPdjmLWt5RVE5ZXdFKmm-um93zqrYCz2biV-m0hnA2Y622l6pxS4la-dsZDLZagTdYbXmk1A61Tokseu2we~vHc4F6eATGoXgdhxcaqOVown3NlUBrKi9x09TKlJKKM0yNVhw6Lr3nLo88NHCQWxKamk4zusJxX2xrTM63E2ersLEye5dUYdCMLTwVyNk9jaBWkPyJ1mAV4CWH4zulM~oZYhYeQLqF6u9XvfIGQ7mGO98PgdDfdkL4ywremu6MXzVD51GDFcQ1Q4rNFX8hYH7dhXJzZoZWpzinpynWM5pYhtsDVFgGVgxulYKAgILJuDZbRW1H2DUUUEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Search for potential reading frameshifts in cds from  Arabidopsis thaliana  and other genomes">
<meta property="og:description" content="Abstract.  A new mathematical method for potential reading frameshift detection in protein-coding sequences (cds) was developed. The algorithm is adjusted to th">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/2/157/5306610">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636262148&amp;Signature=CKdpCc5sTlPdjmLWt5RVE5ZXdFKmm-um93zqrYCz2biV-m0hnA2Y622l6pxS4la-dsZDLZagTdYbXmk1A61Tokseu2we~vHc4F6eATGoXgdhxcaqOVown3NlUBrKi9x09TKlJKKM0yNVhw6Lr3nLo88NHCQWxKamk4zusJxX2xrTM63E2ersLEye5dUYdCMLTwVyNk9jaBWkPyJ1mAV4CWH4zulM~oZYhYeQLqF6u9XvfIGQ7mGO98PgdDfdkL4ywremu6MXzVD51GDFcQ1Q4rNFX8hYH7dhXJzZoZWpzinpynWM5pYhtsDVFgGVgxulYKAgILJuDZbRW1H2DUUUEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-02-04">
<meta name="citation_author" content="Suvorova, Y M">
<meta name="citation_author_institution" content="Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia">
<meta name="citation_author" content="Korotkova, M A">
<meta name="citation_author_institution" content="National Research Nuclear University MEPhI (Moscow Engineering Physics Institute), Moscow, Russia">
<meta name="citation_author" content="Skryabin, K G">
<meta name="citation_author_institution" content="Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia">
<meta name="citation_author" content="Korotkov, E V">
<meta name="citation_author_institution" content="Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia">
<meta name="citation_author_institution" content="National Research Nuclear University MEPhI (Moscow Engineering Physics Institute), Moscow, Russia">
<meta name="citation_title" content="Search for potential reading frameshifts in cds from Arabidopsis thaliana and other genomes">
<meta name="citation_firstpage" content="157">
<meta name="citation_lastpage" content="170">
<meta name="citation_doi" content="10.1093/dnares/dsy046">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="2">
<meta name="citation_publication_date" content="2019/04/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_publisher=Cold Spring Harbor, New York, Cold Spring Harbor Laboratory Press;  citation_title=Molecular Biology of the Gene.;  citation_year=2013; ">
<meta name="citation_reference" content="citation_title=A frameshift mutation in NOD2 associated with susceptibility to Crohn’s disease;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=411;  citation_pages=603-6; ">
<meta name="citation_reference" content="citation_title=Two frameshift mutations in the cystic fibrosis gene;  citation_journal_title=Am. J. Hum. Genet;  citation_year=1991;  citation_volume=48;  citation_pages=227-31; ">
<meta name="citation_reference" content="citation_title=Novel frameshift mutation in Troponin C (TNNC1) associated with hypertrophic cardiomyopathy and sudden death;  citation_journal_title=Cardiol. Young;  citation_year=2011;  citation_volume=21;  citation_pages=345-8; ">
<meta name="citation_reference" content="citation_title=Identification of somatic mutations in human prostate cancer by RNA-Seq;  citation_journal_title=Gene;  citation_year=2013;  citation_volume=519;  citation_pages=343-7; ">
<meta name="citation_reference" content="citation_title=Exon recognition in vertebrate splicing;  citation_journal_title=J. Biol. Chem;  citation_year=1995;  citation_volume=270;  citation_pages=2411-4; ">
<meta name="citation_reference" content="citation_publisher=World Scientific Publishing Co. Pte. Ltd, ;  citation_title=DNA Damage Sens. Signal;  citation_year=2007;  citation_pages=1-22; ">
<meta name="citation_reference" content="citation_title=Frequent appearance of novel protein-coding sequences by frameshift translation;  citation_journal_title=Genomics;  citation_year=2006;  citation_volume=88;  citation_pages=690-7; ">
<meta name="citation_reference" content="citation_title=Functional divergence of proteins through frameshift mutations;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=428-31; ">
<meta name="citation_reference" content="citation_title=Frameshift alignment: statistics and post-genomic applications;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3575-3582; ">
<meta name="citation_reference" content="citation_title=Improve homology search sensitivity of PacBio data by correcting frameshifts;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=529-537; ">
<meta name="citation_reference" content="citation_title=HMM-FRAME: accurate protein domain classification for metagenomic sequences containing frameshift errors;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=198">
<meta name="citation_reference" content="citation_title=On programmed ribosomal frameshifting: the alternative proteomes;  citation_journal_title=Front. Genet;  citation_year=2012;  citation_volume=3;  citation_pages=1-10; ">
<meta name="citation_reference" content="citation_title=Pro-Frame: similarity-based gene recognition in eukaryotic DNA sequences with errors;  citation_journal_title=Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=13-15; ">
<meta name="citation_reference" content="Gîrdea M., Noé L., Kucherov G.  2010 , Back-translation for discovering distant protein homologies in the presence of frameshift mutation. Algorithms Mol. Biol., 5 (1), 6. ">
<meta name="citation_reference" content="citation_title=FrameD: a flexible program for quality check and gene prediction in prokaryotic genomes and noisy mature deukaryotic sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=3738-3741; ">
<meta name="citation_reference" content="citation_title=Genetack: frameshift identification in protein-coding sequences by the Viterbi algorithm;  citation_journal_title=J. Bioinform. Comput. Biol;  citation_year=2010;  citation_volume=8;  citation_pages=535-551; ">
<meta name="citation_reference" content="citation_title=Using triplet periodicity of nucleotide sequences for finding potential reading frame shifts in genes;  citation_journal_title=DNA Res;  citation_year=2009;  citation_volume=16;  citation_pages=105-14; ">
<meta name="citation_reference" content="citation_title=FrameDP: sensitive peptide detection on noisy matured sequences;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=670-671; ">
<meta name="citation_reference" content="citation_title=FragGeneScan: predicting genes in short and error-prone reads;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=e191">
<meta name="citation_reference" content="citation_title=GeneTack database: genes with frameshifts in prokaryotic genomes and eukaryotic mRNA sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=41;  citation_pages=D152-6; ">
<meta name="citation_reference" content="citation_title=Probabilistic methods of identifying genes in prokaryotic genomes: connections to the HMM theory;  citation_journal_title=Brief. Bioinform;  citation_year=2004;  citation_volume=5;  citation_pages=118-30; ">
<meta name="citation_reference" content="citation_title=On the origin of the periodicity of three in protein coding DNA sequences;  citation_journal_title=J. Theor. Biol;  citation_year=1994;  citation_volume=167;  citation_pages=413-4; ">
<meta name="citation_reference" content="citation_title=Search of hidden periodicities in DNA sequences;  citation_journal_title=J. Theor. Biol;  citation_year=1995;  citation_volume=175;  citation_pages=477-94; ">
<meta name="citation_reference" content="citation_title=Protein coding sequence identification by simultaneously characterizing the periodic and random features of DNA sequences;  citation_journal_title=J. Biomed. Biotechnol;  citation_year=2005;  citation_volume=2005;  citation_pages=139-46; ">
<meta name="citation_reference" content="citation_title=Prediction of protein coding regions by the 3-base periodicity analysis of a DNA sequence;  citation_journal_title=J. Theor. Biol;  citation_year=2007;  citation_volume=247;  citation_pages=687-94; ">
<meta name="citation_reference" content="Masoom H., Datta S., Asif A., Cunningham L., Wu G.  2006 , A fast algorithm for detecting frame shifts in DNA sequences. In: Proceedings of the 2006 IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology, CIBCB’06. IEEE Computer Society Press, Toronto, vol. 1, pp. 1–8, Sep. 28–29, 2006. ">
<meta name="citation_reference" content="citation_title=Localizing triplet periodicity in DNA and cDNA sequences;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=550">
<meta name="citation_reference" content="citation_title=Using triplet periodicity of nucleotide sequences for finding potential reading frame shifts in genes;  citation_journal_title=DNA Res;  citation_year=2009;  citation_volume=16;  citation_pages=105-114; ">
<meta name="citation_reference" content="citation_title=An approach for searching insertions in bacterial genes leading to the phase shift of triplet periodicity;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2011;  citation_volume=9;  citation_pages=158-70; ">
<meta name="citation_reference" content="citation_title=Classification analysis of triplet periodicity in protein-coding regions of genes;  citation_journal_title=Gene;  citation_year=2008;  citation_volume=421;  citation_pages=52-60; ">
<meta name="citation_reference" content="citation_title=Search of latent periodicity in amino acid sequences by means of genetic algorithm and dynamic programming;  citation_journal_title=Stat. Appl. Genet. Mol. Biol;  citation_year=2016;  citation_volume=15;  citation_pages=381-400; ">
<meta name="citation_reference" content="citation_title=Classification analysis of triplet periodicity in protein-coding regions of genes;  citation_journal_title=Gene;  citation_year=2008;  citation_volume=421;  citation_pages=52-60; ">
<meta name="citation_reference" content="Pugacheva V., Korotkov A., Korotkov E.  2016 , Search for latent periodicity in amino acid sequences with insertions and deletions. In: BIOINFORMATICS 2016—7th International Conference on Bioinformatics Models, Methods and Algorithms, Proceedings; Part of 9th International Joint Conference on Biomedical Engineering Systems and Technologies, BIOSTEC 2016. SCITEPRESS - Science and technology Publications, Ltd., Vol.3, 117–127. ">
<meta name="citation_reference" content="citation_title=The locally optimal method of cyclic alignment to reveal latent periodicities in genetic texts: the NAD-binding protein sites;  citation_journal_title=Mol. Biol;  citation_year=2003;  citation_volume=37;  citation_pages=663-673; ">
<meta name="citation_reference" content="citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins;  citation_journal_title=J. Mol. Biol;  citation_year=1970;  citation_volume=48;  citation_pages=443-53; ">
<meta name="citation_reference" content="citation_title=Identification of the nature of reading frame transitions observed in prokaryotic genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=6514-30; ">
<meta name="citation_reference" content="citation_title=Classification of triplet periodicity in the DNA sequences of genes from KEGG databank;  citation_journal_title=Mol. Biol;  citation_year=2008;  citation_volume=42;  citation_pages=707-720; ">
<meta name="citation_reference" content="citation_title=Ensembl 2015;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D662-9; ">
<meta name="citation_reference" content="citation_title=Study of the triplet periodicity phase shifts in genes;  citation_journal_title=J. Integr. Bioinform;  citation_year=2010;  citation_volume=7;  citation_pages=131-142; ">
<meta name="citation_reference" content="citation_publisher=Springer Berlin Heidelberg, Berlin, Heidelberg;  citation_title=Evolution by Gene Duplication.;  citation_year=1970; ">
<meta name="citation_reference" content="citation_title=Orthologs, paralogs, and evolutionary genomics;  citation_journal_title=Annu. Rev. Genet;  citation_year=2005;  citation_volume=39;  citation_pages=309-38; ">
<meta name="citation_reference" content="citation_title=Fusion of the human gene for the polyubiquitination coeffector UEV1 with Kua, a newly identified gene;  citation_journal_title=Genome Res;  citation_year=2000;  citation_volume=10;  citation_pages=1743-56; ">
<meta name="citation_reference" content="citation_title=Why genes in pieces?,;  citation_journal_title=Nature;  citation_year=1978;  citation_volume=271;  citation_pages=501">
<meta name="citation_reference" content="citation_title=Creation and disruption of protein features by alternative splicing—a novel mechanism to modulate function;  citation_journal_title=Genome Biol;  citation_year=2005;  citation_volume=6;  citation_pages=R58">
<meta name="citation_reference" content="citation_title=Lateral and oblique gene transfer;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2001;  citation_volume=11;  citation_pages=616-9; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/2/157/28495616/dsy046.pdf">
<meta name="description" content="Abstract.  A new mathematical method for potential reading frameshift detection in protein-coding sequences (cds) was developed. The algorithm is adjusted to th">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/2/157/5306610">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/2/157/5306610">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Behind_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Behind_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Behind_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/2/157/5306610");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy046';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f157%2f5306610" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f157%2f5306610" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115366"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1576706089&amp;Signature=Drq6wwqAdVeKbPwhVWu8zD4eEtLUMneAoa7qgyLrFoSYdcjLziC2QEf9qJsl7HDkcKd-TakUMY2eKvx0CyRSuj0JB3tS~144boC8Kb7oDzqRuyc4Z9uVP95S95ybblIpfwy-IuovYxZ3Lt49wu4~T~0C0iqLhCfCDAI0tY8GZ4HIms3p1MqAYE05KUpceWUFFBxxMKbH5PLGSEpFHgkuaEpBJkjdA6Gmmm0llJ9am6EqAKy5Uy0-I~~y8OEznZbfyQwZirWRGjBiJO9J4xN2pmgZ~zEa8WfxHYMKXBTwkHRBa40P3l1Mi89hkaglXuw2OPX5-ICWNkw2xYBfrnEbgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/2">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="134010150">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010150">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010152">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010152">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010165">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010165">2. Mathematical methods and algorithms</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010203">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010203">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010240">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010240">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="134010243">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010243">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/2/147/5280828">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/2/171/5304673">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Search for potential reading frameshifts in cds from <em>Arabidopsis thaliana</em> and other genomes
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Y M Suvorova</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Y+M+Suvorova">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suvorova%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Suvorova%20Y%20M%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Y M Suvorova</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M A Korotkova</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">National Research Nuclear University MEPhI (Moscow Engineering Physics Institute), Moscow, Russia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M+A+Korotkova">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Korotkova%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Korotkova%20M%20A%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">M A Korotkova</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">K G Skryabin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=K+G+Skryabin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Skryabin%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Skryabin%20K%20G%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">K G Skryabin</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">E V Korotkov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bioengineering, Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia</div>
<div class="aff">National Research Nuclear University MEPhI (Moscow Engineering Physics Institute), Moscow, Russia</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +7 926 724 8271. Fax. +7 499 135 0571. Email: <a href="mailto:genekorotkov@gmail.com" target="_blank">genekorotkov@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=E+V+Korotkov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Korotkov%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Korotkov%20E%20V%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">E V Korotkov</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 157–170, <a href="https://doi.org/10.1093/dnares/dsy046">https://doi.org/10.1093/dnares/dsy046</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">04 February 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">27 April 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">07 December 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">04 February 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5306610" href="/dnaresearch/article-pdf/26/2/157/28495616/dsy046.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Y M Suvorova, M A Korotkova, K G Skryabin, E V Korotkov,  Search for potential reading frameshifts in cds from <em>Arabidopsis thaliana</em> and other genomes, <em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 157–170, <a href="https://doi.org/10.1093/dnares/dsy046">https://doi.org/10.1093/dnares/dsy046</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5306610&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Search%20for%20potential%20reading%20frameshifts%20in%20cds%20from%20%26lt%3Bem%26gt%3BArabidopsis%20thaliana%26lt%3B%2Fem%26gt%3B%20and%20other%20genomes&amp;publicationDate=2019-02-04&amp;volumeNum=26&amp;issueNum=2&amp;author=Suvorova%2C%20Y%20M%3B%20Korotkova%2C%20M%20A&amp;startPage=157&amp;endPage=170&amp;contentId=10.1093%2Fdnares%2Fdsy046&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f157%2f5306610" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="134010150" id="134010150" class="abstract-title">Abstract</h2>
<section class="abstract"><p>A new mathematical method for potential reading frameshift detection in protein-coding sequences (cds) was developed. The algorithm is adjusted to the triplet periodicity of each analysed sequence using dynamic programming and a genetic algorithm. This does not require any preliminary training. Using the developed method, cds from the <em>Arabidopsis thaliana</em> genome were analysed. In total, the algorithm found 9,930 sequences containing one or more potential reading frameshift(s). This is ∼21% of all analysed sequences of the genome. The Type I and Type II error rates were estimated as 11% and 30%, respectively. Similar results were obtained for the genomes of <em>Caenorhabditis elegans,</em> Drosophila <em>melanogaster, Homo sapiens, Rattus norvegicus</em> and <em>Xenopus tropicalis</em>. Also, the developed algorithm was tested on 17 bacterial genomes. We compared our results with the previously obtained data on the search for potential reading frameshifts in these genomes. This study discussed the possibility that the reading frameshift seems like a relatively frequently encountered mutation; and this mutation could participate in the creation of new genes and proteins.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"reading frameshift"'>reading frameshift</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="periodicity">periodicity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"dynamic programming"'>dynamic programming</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genetic algorithm"'>genetic algorithm</a>
</div>                    <h2 scrollto-destination="134010152" id="134010152" class="section-title">1. Introduction</h2>
<p>The occurrence of reading frameshifts in a gene is a very serious mutation, and it results in the creation of mutant proteins.<a reveal-id="dsy046-B1" data-open="dsy046-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> This may result in various hereditary<a reveal-id="dsy046-B2 dsy046-B3 dsy046-B4" data-open="dsy046-B2 dsy046-B3 dsy046-B4" class="link link-ref link-reveal xref-bibr"><sup>2–4</sup></a> or oncological diseases.<a reveal-id="dsy046-B5" data-open="dsy046-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The occurrence of reading frameshifts in a gene is caused by the insertion or deletion of nucleotides, which is not a multiple of 3. Besides, in eukaryotic genes, reading frameshifts could arise through the shift of boundaries between exons and introns, when the splicing point is under mutation.<a reveal-id="dsy046-B6" data-open="dsy046-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> In the evolution of genetic sequences, it is common to observe the insertions and deletions of small fragments.<a reveal-id="dsy046-B7" data-open="dsy046-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy046-B8" data-open="dsy046-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> In this case, within the translation stage there is a complete change in the amino acid sequence beyond the frameshift position.<a reveal-id="dsy046-B9" data-open="dsy046-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Consequently, the encoded amino acid sequence can completely lose its biological function and become a pseudogenе. This sequence could either acquire another biological function or its function may remain the same. The study of such mutations is important for understanding the mechanisms of protein sequence evolution.</p>
<p>Errors of genome sequencing and assembly processes could also cause reading frameshifts in genes present in modern databases.<a reveal-id="dsy046-B10" data-open="dsy046-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> For example, in pyrosequencing methods and PacBio sequencing technology, frameshift errors appear very frequently. In this case, the encoded amino acid sequence beyond the frameshift position would be incorrect, which makes further annotation difficult. To find and correct shifts for these sequencing methods, special programs have been developed.<a reveal-id="dsy046-B11" data-open="dsy046-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsy046-B12" data-open="dsy046-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The identification of such errors is important for improving annotation and ensuring further research of new genomes.</p>
<p>In the literature, cases of programmed ribosomal frameshifts have been reported. In this case, the ribosome itself shifts within the protein synthesis process on a single base, which results in the emergence of an alternative protein.<a reveal-id="dsy046-B13" data-open="dsy046-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> However, such events are beyond the scope of this study. The present study is focused on developing a mathematical algorithm for the detection of potential reading frameshifts in protein-coding sequences (cds).</p>
<p>The methods currently used to find reading frameshifts can be divided into two classes. The methods of the first class are based on the comparison (alignment) of sequences. These methods use protein database search to find sequences which are homologous to the sequence of interest but are devoid of the frameshift. Such methods suggest the use of the BLAST program and analogues.<a reveal-id="dsy046-B8" data-open="dsy046-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy046-B9" data-open="dsy046-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy046-B14" data-open="dsy046-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> An obvious limitation of these methods is that in the absence of a homologous sequence without a frameshift, it is impossible to determine the presence of a frameshift. The large number of substitutions that occurred after the frameshift event could also make it difficult to find similarities. In order to search for remote similarities, while taking the possible frameshifts into account, special programs are being developed.<a reveal-id="dsy046-B15" data-open="dsy046-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p>
<p>The methods of the second class are aimed at finding the frameshift directly by the sequence (<em>ab initio</em>).<a reveal-id="dsy046-B16 dsy046-B17 dsy046-B18" data-open="dsy046-B16 dsy046-B17 dsy046-B18" class="link link-ref link-reveal xref-bibr"><sup>16–18</sup></a> There are several methods for the prediction of protein-coding regions in genome sequences that consider the probable reading frameshifts. These methods are used by programs: FrameD,<a reveal-id="dsy046-B16" data-open="dsy046-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy046-B19" data-open="dsy046-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> based on the Markov models, program for predicting genes by taking into account frameshifts and searching for frameshifts in known genes. The FragGeneScan program<a reveal-id="dsy046-B20" data-open="dsy046-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> was created to search for coding regions in short reads taking into account the frameshifts and based on the Hidden Markov Models (HMMs). Also, the HMM-frame program<a reveal-id="dsy046-B12" data-open="dsy046-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> uses the HMMs to search for protein domains in the metagenomics sequences while taking into consideration the probable reading frameshifts.</p>
<p>The GeneTack program is among the most widely used tools.<a reveal-id="dsy046-B17" data-open="dsy046-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> This program can identify reading frameshifts resulting from mutations and sequencing errors. The idea of the algorithm is that several genes are located on the chromosome one after another and are represented in databases as independent genes which may have originated from a single coding sequence separated as a result of the reading frameshift. The GeneTack algorithm is based on the HMM and the Viterbi algorithm, and possesses several parameters that require adjustment prior to sequence processing. Unfortunately, the methods are all limited because they require a training sample for determination of the HMM parameters. In the result some statistical properties of the gene could be averaged, which significantly reduces the capability of the methods. In particular, the frequencies of <em>k</em>-words belong to these statistical properties. Training is carried out using another program of the authors—GeneMarkS.<a reveal-id="dsy046-B22" data-open="dsy046-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The training sample is usually the entire sequence of the genome under study (for prokaryotes) or a set of coding sequences of this genome (for eukaryotes). Besides, a prepared model from the authors’ website, having a similar level of GC content, could also be used. The GC content of the sequences under examination has a significant effect on the search for reading frameshifts. The given method was designed to search for reading frameshifts both at the genome annotation stage (as part of the genome annotation integrated programs) and in known sequences.</p>
<p>Most of the methods used for the detection of frameshifts <em>ab initio</em> are based on the well-known property of the cds, i.e. the triplet periodicity (TP). TP is exclusively present in the cds of virtually all organisms and is considered to be a consequence of the preferred use of synonymous codons by various organisms.<a reveal-id="dsy046-B23" data-open="dsy046-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsy046-B24" data-open="dsy046-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> This property is used by many programs designed for cds prediction.<a reveal-id="dsy046-B25" data-open="dsy046-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsy046-B26" data-open="dsy046-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> It has been shown that the reading frameshift in a gene resulted in TP phase shift in the corresponding position.<a reveal-id="dsy046-B18" data-open="dsy046-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> TP phase shift is the shift of TP matrix columns relative to the positions of the bases in the codons. An example is presented in <a reveal-id="dsy046-F1" data-open="dsy046-F1" class="link link-reveal link-table xref-fig">Fig. 1A and B</a>. <a reveal-id="dsy046-F1" data-open="dsy046-F1" class="link link-reveal link-table xref-fig">Figure 1A</a> shows the TP matrix, which was constructed from the coding sequence ‘atcatcatc …’. The first column of this matrix corresponds to the first codon base, the second column corresponds to the second codon base, and the third column corresponds to the third codon base. Conditionally, this can be shown as: 1 =&gt; 1, 2 =&gt; 2, 3 =&gt; 3. Such a relationship can be called Phase 0, or <em>Fa </em>=<em> </em>0. After inserting one base (G in the middle of the sequence, <a reveal-id="dsy046-F1" data-open="dsy046-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>), the TP phase shifts to one base. In this case, the correspondence of positions in the codons and columns of the matrix is 1 =&gt; 2, 2 =&gt; 3, 3 =&gt; 1. This correspondence is called Phase 1 or <em>Fa </em>=<em> </em>1. The value <em>Fa </em>=<em> </em>2 is also possible. In this case, 1 =&gt; 3, 2 =&gt; 1, 3 =&gt; 2. Such a correspondence will be observed when inserting any two DNA bases. After inserting <em>n</em> DNA bases, <em>Fa</em> becomes <em>n</em> − 3int(<em>n</em>/3). Here, ‘int’ is the operation of computing the integer part of a number. </p>                        <a id="134010160" scrollto-destination="134010160"></a>
<div id="dsy046-F1" data-id="dsy046-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f1.png?Expires=1576262930&amp;Signature=t8CjEf0K2l-uuR3mInQ~IS5xeGKRCxD0v9257iKy6oPqySRgjpJjkera~qRjo70tw~fzLjkeizH~j2bArX8fMJjN4xWduXzqUI74wEQ-CBU4eMc8JgIoNHlwU6NW2sudFM~v4Zc25XvBV8Wv3L~s5GVQ18enckbJBgMVZ7ALnYKZ9mOY21xP-s1fr60cJhaa3KidfKB8Yb~uC1l1p2rVoUwxnSJhgpHzL0PSaFGkaJiiL18xZXxbwZePYM0PYoaaIi-SJfTVmaesEXDdiK~WbEWjVH9tt4Iw1hCWcUg1baGfe-RiT7C1k8mc19O3E7JcAjxtPPdDvZjOb1buWG1l6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The TP matrix is shown for the sequence S={atc}50. (B) The phase TP in a fragment of the sequence S. A base g is inserted in the middle of the sequence. The phase Fa = 0 was before the insertion of g, because there is an agreement between the columns of the matrix and the positions of the codons as 1 =&gt; 1, 2 =&gt; 2, 3 =&gt; 3, respectively. Fa = 1 after the insertion of g, because the following agreement is observed: 1 =&gt; 2, 2 =&gt; 3, 3 =&gt; 1. (C) The TP matrix for the sequence S={atcgga}25." path-from-xml="dsy046f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010160/dsy046f1.tif" path-from-xml="dsy046f1.tif" target="_blank">Open in new tab</a><a section="134010160" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f1.png?Expires=1576262930&amp;Signature=LVbeWreIMJl3tHV~PfenTciQU0aCfllFnPBN3EalnC2mzRlUMfhdfFczPvx5~3XBFF9I5r0CTpi-NhOQXRL3JJ8DsCZLbaMZpfPP65pYxMpXII4VPmVq6-AILlQpua~UoMOd~ULfZPU5aZRyMysl4j2yTWZuQD1zdyED4vnS7TCqcHvBpke332ejasF~Y28VN3ywsXHzsq5PJK~yy53RMtxkl2AonTVI90KHeEGcbuOrystFOvBYB9oXct0D2FjicdtIq0800PNeYq8dW0FtH3tH5H1dHTlBcu0DBMmKPmKXipGxdaBykhBErMXsvmOjDG9sZ6dVgVDXiq2Tl4E~zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010160&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>(A) The TP matrix is shown for the sequence <em>S</em>={<em>atc</em>}<sub>50</sub>. (B) The phase TP in a fragment of the sequence S. A base <em>g</em> is inserted in the middle of the sequence. The phase <em>Fa </em>=<em> </em>0 was before the insertion of <em>g</em>, because there is an agreement between the columns of the matrix and the positions of the codons as 1 =&gt; 1, 2 =&gt; 2, 3 =&gt; 3, respectively. <em>Fa </em>=<em> </em>1 after the insertion of <em>g</em>, because the following agreement is observed: 1 =&gt; 2, 2 =&gt; 3, 3 =&gt; 1. (C) The TP matrix for the sequence <em>S</em>={atcgga}<sub>25</sub>.</p></div>
</div>
<div content-id="dsy046-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f1.png?Expires=1576262930&amp;Signature=t8CjEf0K2l-uuR3mInQ~IS5xeGKRCxD0v9257iKy6oPqySRgjpJjkera~qRjo70tw~fzLjkeizH~j2bArX8fMJjN4xWduXzqUI74wEQ-CBU4eMc8JgIoNHlwU6NW2sudFM~v4Zc25XvBV8Wv3L~s5GVQ18enckbJBgMVZ7ALnYKZ9mOY21xP-s1fr60cJhaa3KidfKB8Yb~uC1l1p2rVoUwxnSJhgpHzL0PSaFGkaJiiL18xZXxbwZePYM0PYoaaIi-SJfTVmaesEXDdiK~WbEWjVH9tt4Iw1hCWcUg1baGfe-RiT7C1k8mc19O3E7JcAjxtPPdDvZjOb1buWG1l6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The TP matrix is shown for the sequence S={atc}50. (B) The phase TP in a fragment of the sequence S. A base g is inserted in the middle of the sequence. The phase Fa = 0 was before the insertion of g, because there is an agreement between the columns of the matrix and the positions of the codons as 1 =&gt; 1, 2 =&gt; 2, 3 =&gt; 3, respectively. Fa = 1 after the insertion of g, because the following agreement is observed: 1 =&gt; 2, 2 =&gt; 3, 3 =&gt; 1. (C) The TP matrix for the sequence S={atcgga}25." path-from-xml="dsy046f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010160/dsy046f1.tif" path-from-xml="dsy046f1.tif" target="_blank">Open in new tab</a><a section="134010160" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f1.png?Expires=1576262930&amp;Signature=LVbeWreIMJl3tHV~PfenTciQU0aCfllFnPBN3EalnC2mzRlUMfhdfFczPvx5~3XBFF9I5r0CTpi-NhOQXRL3JJ8DsCZLbaMZpfPP65pYxMpXII4VPmVq6-AILlQpua~UoMOd~ULfZPU5aZRyMysl4j2yTWZuQD1zdyED4vnS7TCqcHvBpke332ejasF~Y28VN3ywsXHzsq5PJK~yy53RMtxkl2AonTVI90KHeEGcbuOrystFOvBYB9oXct0D2FjicdtIq0800PNeYq8dW0FtH3tH5H1dHTlBcu0DBMmKPmKXipGxdaBykhBErMXsvmOjDG9sZ6dVgVDXiq2Tl4E~zw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010160&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>(A) The TP matrix is shown for the sequence <em>S</em>={<em>atc</em>}<sub>50</sub>. (B) The phase TP in a fragment of the sequence S. A base <em>g</em> is inserted in the middle of the sequence. The phase <em>Fa </em>=<em> </em>0 was before the insertion of <em>g</em>, because there is an agreement between the columns of the matrix and the positions of the codons as 1 =&gt; 1, 2 =&gt; 2, 3 =&gt; 3, respectively. <em>Fa </em>=<em> </em>1 after the insertion of <em>g</em>, because the following agreement is observed: 1 =&gt; 2, 2 =&gt; 3, 3 =&gt; 1. (C) The TP matrix for the sequence <em>S</em>={atcgga}<sub>25</sub>.</p></div>
</div>
<p>For the determination of TP phase shifts, mathematical methods such as the Fourier transform,<a reveal-id="dsy046-B27" data-open="dsy046-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> wavelet transform,<a reveal-id="dsy046-B28" data-open="dsy046-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> dynamic programming<a reveal-id="dsy046-B29" data-open="dsy046-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> or methods based on comparison of periodicity matrices<a reveal-id="dsy046-B30" data-open="dsy046-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> have been employed. The Fourier transform method produced good results on artificial sequences and genes with a high level of TP. However, the method requires a window of rather long length (the authors recommended a window size of about 750 nt).<a reveal-id="dsy046-B27" data-open="dsy046-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Methods based on dynamic programming or matrices comparison are characterized by enhanced sensitivity, but are incapable of detecting frameshifts in sequences with a low level of TP.<a reveal-id="dsy046-B18" data-open="dsy046-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p>
<p>We developed a new approach for TP phase shift detection in cds from prokaryotic and eukaryotic genomes. By this method, an attempt was made to eliminate the disadvantages of the HMM approaches connected to the requirement of the HMM configuration in the sampling of genes. Such adjustment significantly averages all the characteristics of the genes TP, because different classes of TP exist in the genome, and the combination thereof decreases the statistical significance of TP.<a reveal-id="dsy046-B31" data-open="dsy046-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> In addition, the use of HMM could identify reading frameshifts only in those cds possessing the same base correlations, as the training sample. If the correlations between bases of cds are different, then it could be impossible to detect the frameshifts. The HMM limitations are considered in more details in the ‘3.4. Comparison with the Genetack-GM program’ section.</p>
<p>We developed a method that enables determination of the best TP matrix for each sequence while taking into account the correlation of the adjacent DNA bases along with the possibility of nucleotides insertion or deletion. The genetic algorithm and the dynamic programming method<a reveal-id="dsy046-B32" data-open="dsy046-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> were used in a similar way, as described in our previous work on amino acid sequences. However, in the given case, the algorithm was modified by introducing a matrix that considers the correlation of neighbouring bases. After identifying such a matrix, we performed the final alignment of the sequence of interest with respect to the best matrix and found the probable reading frameshift positions.</p>
<p>As a result of our research, we identified an unexpectedly large number of probable reading frameshifts in the <em>Arabidopsisthaliana</em> genome cds, which constituted 9,930 with the error level of the first and second types being ∼11% and 30%. This is about 21% of all the registered cds from this genome. We obtained similar results for the genomes of Caenorhabditis <em>elegans,</em> Drosophila <em>melanogaster, Homo sapiens,</em> Rattus <em>norvegicus</em> and <em>Xenopus tropicalis</em>. It was assumed that the reading frameshift is a relatively frequently found mutation, and this mutation could take part in the creation of new genes and proteins.</p>                    <h2 scrollto-destination="134010165" id="134010165" class="section-title">2. Mathematical methods and algorithms</h2>
                    <h3 scrollto-destination="134010166" id="134010166" class="section-title">2.1. General description of the mathematical algorithm used in this work</h3>
<p>The task of reading frameshift detection in a protein-coding sequence could be mathematically solved, if it becomes possible to relate the statistical properties of the sequence with the reading frame. In this case, one could find the reading frameshift in a sequence without using any kind of training and without involving any additional information. Such a property could be presented by the TP of genes.<a reveal-id="dsy046-B33" data-open="dsy046-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> In this case, the reading frameshift would appear as a TP phase shift.</p>
<p>Let us consider a protein-coding sequence <em>S</em> having a length <em>N.</em> The TP of a sequence is usually set in the form of the <em>MT</em>(3, 4) matrix. Here, the columns indicate the three positions of codons (1, 2 or 3), and the rows represent four types of nucleotides.<a reveal-id="dsy046-B33" data-open="dsy046-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><a reveal-id="dsy046-F1" data-open="dsy046-F1" class="link link-reveal link-table xref-fig">Figure 1A and C</a> presents an example of a TP matrix created for the sequences <em>S</em>={atc}<sub>50</sub> and <em>S</em>={atcgga}<sub>25</sub>. In the first case, only three matrix elements are filled, whereas six elements are filled in the second case. The TP of the <em>S</em> sequence set in the form of the <em>MT</em> matrix perfectly reflects the difference in base frequencies at each position of the codon from the base frequencies throughout the entire nucleotide sequence. However, the <em>MT</em> matrix does not consider the correlation between adjacent bases, as illustrated by the following example. Let us assume that in our sequence only four codons are equally likely to be used: ATA, TAT, CGC and GCG, and they are arranged in a sequence in some random order. Then, the DNA base frequencies in each column of the <em>MT</em> matrix would be equal to the frequencies of bases in the entire sequence, and the matrix constructed according to this sequence would show that there is no TP in the sequence.</p>
<p>Therefore, it is more appropriate to use another matrix <em>M</em>(<em>i, n</em>) while searching for TP. The matrix contains 16 rows and 3 columns. The columns of the matrix represent the pairs of positions in the codon: 3-1, 1-2 and 2-3. Here, the <em>i</em> column of the <em>M</em> matrix takes the 1, 2 and 3 values for pairs of positions in codon: 3-1, 1-2 and 2-3, respectively. This means that the <em>M</em> matrix columns are numbered by the last position out of the two. The <em>n</em> row number shows the frequency of the base pairs in columns, and <em>n</em> takes values from 1 to 16. In order to fill the <em>M</em> matrix, the row number is calculated as follows: <em>n </em>=<em> s</em>(<em>j</em> − <em>1</em>)+4(<em>s</em>(<em>j</em>) − 1). This study utilized the <em>a </em>=<em> </em>1, <em>t </em>=<em> </em>2, <em>c </em>=<em> </em>3 and <em>g </em>=<em> </em>4 nucleotides numerical coding. Here, <em>s</em>(<em>j</em>) is the base of the <em>S</em> sequence in the <em>j</em> position, which corresponds to the <em>i</em> position in the codon calculated with the following equation: <em>i </em>=<em> j</em> − 3int(<em>j</em>/3), where ‘int’ is the integer part of the number. Then, the <em>s</em>(<em>j</em> − 1) base corresponds to the previous neighbouring codon position. Thus, <em>j</em> ranges from 2 to <em>N</em> values. For each <em>j</em>, <em>M</em>(<em>i, n</em>) = <em>M</em>(<em>i, n</em>) + 1.Therefore, the <em>M</em>(3, 16) matrix contains two types of statistical regularities of the coding regions. The first one is the difference between the frequencies of nucleotides in each position of a codon and the nucleotide frequencies of the entire <em>S</em> sequence. The second one is the correlation of two neighbouring nucleotides positions of the codon (3-1, 1-2 and 2-3).</p>
<p>Assuming we have a sequence <em>S</em> that is a cds. The sequence <em>S</em><sub>0</sub> is sequence <em>S</em> prior to the reading frameshifts, that is, <em>S</em><sub>0</sub> is the ancestor of sequence <em>S</em>. We create the matrix <em>M</em>(3, 16) for <em>S</em> and we create the matrix <em>M</em><sub>0</sub>(3, 16) for <em>S</em><sub>0</sub> as described above. If one knows the <em>M</em><sub>0</sub>(3, 16), then such TP periodicity shift could be found using the alignment of sequence <em>S</em> with the position-weight matrix (PWM) <em>W</em><sub>0</sub> (see Equation 1). This matrix was created using <em>M</em><sub>0</sub>(3, 16), as it was performed previously for the <em>MT</em>(3, 4) matrix<a reveal-id="dsy046-B34" data-open="dsy046-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and is shown below in Equation (1). To create the <em>W</em><sub>0</sub>(3, 16) matrix, each element of the <em>M</em><sub>0</sub>(3, 16) matrix was transformed to the argument of the normal distribution, using the normal approximation for the binomial distribution. Dynamic programming could be used to find the global alignment of the <em>S</em> sequence with respect to the <em>W</em><sub>0</sub>(3, 16) matrix. This alignment can be performed by cyclic alignment, as was done previously.<a reveal-id="dsy046-B35" data-open="dsy046-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> However, the problem is that there are no stored <em>M</em><sub>0</sub>(3, 16) matrix and the corresponding <em>W</em><sub>0</sub>(3, 16) matrix for the sequence <em>S</em><sub>0</sub> and sequence <em>S</em> is unknown. Using the sequence of an existing gene, it is impossible to identify <em>M</em><sub>0</sub>(3, 16) and <em>W</em><sub>0</sub>(3, 16) matrices. The reason why it is impossible to calculate this matrix is the uncertainty of the reading frameshift positions in the gene.</p>
<p>For example, let us consider a sequence <em>S</em><sub>0</sub>:{atg}<sub>50.</sub> If we delete a single nucleotide in the middle of <em>S</em><sub>0</sub>, we obtain a sequence <em>S</em>={atg}<sub>25</sub>{tga}<sub>25</sub>. The matrices constructed for the sequences <em>S</em><sub>0</sub> and <em>S</em> [<em>M</em><sub>0</sub>(3, 16) and <em>M</em>(3, 16)] are shown in <a reveal-id="dsy046-T1" data-open="dsy046-T1" class="link link-reveal link-table xref-fig">Table 1A and B</a>, respectively. <a reveal-id="dsy046-T1" data-open="dsy046-T1" class="link link-reveal link-table xref-fig">Table 1B</a> shows that for the <em>S</em> sequence, all available base pairs that could be constructed from the existing reading frame would receive a similar positive weight, when the <em>M</em>(3, 16) matrix is transformed into the <em>W</em>(3, 16) matrix (PWM). Thus, using the global alignment of the <em>S</em> sequence with respect to the <em>W</em>(3, 16) matrix<a reveal-id="dsy046-B36" data-open="dsy046-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> (or of the HMM<a reveal-id="dsy046-B37" data-open="dsy046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a>), it would be impossible to find the TP periodicity shift. This is because the deletion in position 76 of the <em>S</em> sequence shall not be created by global alignment with the <em>W</em>(3, 16) matrix. Such a shift could only be found if the <em>M</em><sub>0</sub>(3, 16) matrix is known, and the <em>W</em><sub>0</sub> matrix is created on its base. Therefore, the task is to develop a mathematical method to find the <em>W</em><sub>0</sub> matrix and perform a global alignment of the <em>S</em> sequence relative to <em>W</em><sub>0</sub>.</p>                        <a id="134010172" scrollto-destination="134010172"></a>
<div content-id="dsy046-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T1" class="table-wrap-title" data-id="dsy046-T1">
<div class="label">Table 1</div>
<div class="caption"><p>(A) Matrixes <em>M</em><sub>0</sub>(3, 16) and (B) <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>M</mi><mrow><mo>(</mo><mrow><mn>3</mn><mo>,</mo><mn>16</mn></mrow><mo>)</mo></mrow></mrow></math></span></p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
</tr>
</thead>
<tbody>
<tr>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ga </td>
<td>4 </td>
<td>50 </td>
<td>0 </td>
<td>0 </td>
<td>ga </td>
<td>4 </td>
<td>25 </td>
<td>0 </td>
<td>25 </td>
</tr>
<tr>
<td>at </td>
<td>5 </td>
<td>0 </td>
<td>50 </td>
<td>0 </td>
<td>at </td>
<td>5 </td>
<td>25 </td>
<td>25 </td>
<td>0 </td>
</tr>
<tr>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>0 </td>
<td>50 </td>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>25 </td>
<td>25 </td>
</tr>
<tr>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
</tr>
</thead>
<tbody>
<tr>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ga </td>
<td>4 </td>
<td>50 </td>
<td>0 </td>
<td>0 </td>
<td>ga </td>
<td>4 </td>
<td>25 </td>
<td>0 </td>
<td>25 </td>
</tr>
<tr>
<td>at </td>
<td>5 </td>
<td>0 </td>
<td>50 </td>
<td>0 </td>
<td>at </td>
<td>5 </td>
<td>25 </td>
<td>25 </td>
<td>0 </td>
</tr>
<tr>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>0 </td>
<td>50 </td>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>25 </td>
<td>25 </td>
</tr>
<tr>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010172" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T1" class="table-wrap-title" data-id="dsy046-T1">
<div class="label">Table 1</div>
<div class="caption"><p>(A) Matrixes <em>M</em><sub>0</sub>(3, 16) and (B) <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>M</mi><mrow><mo>(</mo><mrow><mn>3</mn><mo>,</mo><mn>16</mn></mrow><mo>)</mo></mrow></mrow></math></span></p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
</tr>
</thead>
<tbody>
<tr>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ga </td>
<td>4 </td>
<td>50 </td>
<td>0 </td>
<td>0 </td>
<td>ga </td>
<td>4 </td>
<td>25 </td>
<td>0 </td>
<td>25 </td>
</tr>
<tr>
<td>at </td>
<td>5 </td>
<td>0 </td>
<td>50 </td>
<td>0 </td>
<td>at </td>
<td>5 </td>
<td>25 </td>
<td>25 </td>
<td>0 </td>
</tr>
<tr>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>0 </td>
<td>50 </td>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>25 </td>
<td>25 </td>
</tr>
<tr>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
<th> </th>
<th>
<em>N</em> </th>
<th>1 </th>
<th>2 </th>
<th>3 </th>
</tr>
</thead>
<tbody>
<tr>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>aa </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ta </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ca </td>
<td>3 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ga </td>
<td>4 </td>
<td>50 </td>
<td>0 </td>
<td>0 </td>
<td>ga </td>
<td>4 </td>
<td>25 </td>
<td>0 </td>
<td>25 </td>
</tr>
<tr>
<td>at </td>
<td>5 </td>
<td>0 </td>
<td>50 </td>
<td>0 </td>
<td>at </td>
<td>5 </td>
<td>25 </td>
<td>25 </td>
<td>0 </td>
</tr>
<tr>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tt </td>
<td>6 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ct </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gt </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ac </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>tc </td>
<td>10 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cc </td>
<td>11 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gc </td>
<td>12 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>ag </td>
<td>13 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>0 </td>
<td>50 </td>
<td>tg </td>
<td>14 </td>
<td>0 </td>
<td>25 </td>
<td>25 </td>
</tr>
<tr>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>cg </td>
<td>15 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>gg </td>
<td>16 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010172" target="_blank">Open in new tab</a></div>
</div></div>
<p>This study partially utilized the algorithm that we developed in our previous work.<a reveal-id="dsy046-B32" data-open="dsy046-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> This algorithm employs a particular property of the <em>P</em>(<em>x </em>&gt;<em> F</em>) probability, which is calculated for the global alignment of the <em>S</em> sequence with the PWM. It was shown that the value <em>P</em>(<em>x </em>&gt;<em> F</em>) is lower for the <em>W</em><sub>0</sub>(3, 16) matrix compared with the <em>W</em>(3, 16) matrix. Here, <em>F</em> is the similarity function value of the global alignment (see Equation 2). This means that we must elaborate on a procedure for optimizing the <em>W</em>(3, 16) matrix to find the <em>W</em><sub>1</sub>(3, 16) matrix that is closest to the <em>W</em><sub>0</sub>(3, 16) matrix. In this case, we obtained the maximum value of the <em>F</em> similarity function and the lowest value of the <em>P</em>(<em>x </em>&gt;<em> F</em>) probability, as well as the global alignment of the <em>S</em> sequence with the <em>W</em><sub>1</sub>(3, 16) matrix. This global alignment allows us to determine the coordinates of the potential reading frameshifts. The optimization was performed using the algorithm shown in <a reveal-id="dsy046-F2" data-open="dsy046-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>. The idea of the algorithm is to create a random matrix <em>W</em><sub>r</sub>(3, 16) (or a set of random matrices), followed by optimization with a genetic algorithm and obtaining the <em>W</em><sub>1</sub>(3, 16) matrix as a result of optimization. Matrices have 3 columns and 16 rows, hence optimization takes place in a 48-dimensional space. Schematically, this optimization is presented in <a reveal-id="dsy046-F3" data-open="dsy046-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. At each optimization step, we move along the 48-dimensional space and obtain the intermediate matrices, which are represented by asterisks. Let us consider the crucial points of this algorithm. </p>                        <a id="134010174" scrollto-destination="134010174"></a>
<div id="dsy046-F2" data-id="dsy046-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f2.png?Expires=1576262930&amp;Signature=yIFlM1K9ZvGfVUTttaY73y0YdpeyhoHdP08DYX2D0iS1mTgY-X083tDE1LfUrrpdlh8XW9O5NSMFa7VUGcEM~9BYmqWVBuQ3PjcW67wVqDkZLF4S4~riYSCwuQGaIpcPyybFlVOzwzTCzfdmr3u5ZcxuhqGeK71qyWeFw1X0Pps-wFIzXWhy0Rq53pGsqzPHjLjZx49dd7uxJsHUGt-ggy1IqAs2h9v5ZYgt~uJuSfJ0yozUFCUz8zYLXNpLyM~UBt298HkLtCgMSVpO06YOBPCXI~bj2-AENa-XQYbTLQfJX4ouc~UWeI6YGpxO-pWYsNQENn8RWLxEhIG5UwUXtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A block diagram of the algorithm for the optimization of random matrices M(3,16) from the MR set, used to search a matrix with the largest mFmax." path-from-xml="dsy046f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010174/dsy046f2.tif" path-from-xml="dsy046f2.tif" target="_blank">Open in new tab</a><a section="134010174" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f2.png?Expires=1576262930&amp;Signature=n9ba7hikc-z4KeKmlZjEzQowpkfW5ppS6Mx5e2kjjWlGNTJAH~JJrqr8O0tlwe8tQmw1iiS1PxI4~SqBep1M4D5RJJoNUQk6rp3nCrycwTpaPPEHbhtsgsjd5pDQN72wrV-pXoMp1L-O8Sm4eAJplHPtvZxr5qJ6POTkWJa6qd7E9cEY-tuU4eY7iC4gGT3VyUw7P0mcnbGplUhLJSyQillqkHblG-zipm7tmC5pmYeb75qJZh7o6G0SOegjy5eFL06ZtHuCe6myPWm~mTJhIi0MfwFX9X7LMoFAN4GPkwLhqQF~vPzxpWGfUqPrpnsNW6QVN89wVIVfDEJhNruATw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010174&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A block diagram of the algorithm for the optimization of random matrices <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>M</mi><mrow><mo>(</mo><mrow><mn>3</mn><mo>,</mo><mn>16</mn></mrow><mo>)</mo></mrow></mrow></math></span> from the <em>MR</em> set, used to search a matrix with the largest <em>mF</em><sub>max</sub>.</p></div>
</div>
<div content-id="dsy046-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f2.png?Expires=1576262930&amp;Signature=yIFlM1K9ZvGfVUTttaY73y0YdpeyhoHdP08DYX2D0iS1mTgY-X083tDE1LfUrrpdlh8XW9O5NSMFa7VUGcEM~9BYmqWVBuQ3PjcW67wVqDkZLF4S4~riYSCwuQGaIpcPyybFlVOzwzTCzfdmr3u5ZcxuhqGeK71qyWeFw1X0Pps-wFIzXWhy0Rq53pGsqzPHjLjZx49dd7uxJsHUGt-ggy1IqAs2h9v5ZYgt~uJuSfJ0yozUFCUz8zYLXNpLyM~UBt298HkLtCgMSVpO06YOBPCXI~bj2-AENa-XQYbTLQfJX4ouc~UWeI6YGpxO-pWYsNQENn8RWLxEhIG5UwUXtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A block diagram of the algorithm for the optimization of random matrices M(3,16) from the MR set, used to search a matrix with the largest mFmax." path-from-xml="dsy046f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010174/dsy046f2.tif" path-from-xml="dsy046f2.tif" target="_blank">Open in new tab</a><a section="134010174" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f2.png?Expires=1576262930&amp;Signature=n9ba7hikc-z4KeKmlZjEzQowpkfW5ppS6Mx5e2kjjWlGNTJAH~JJrqr8O0tlwe8tQmw1iiS1PxI4~SqBep1M4D5RJJoNUQk6rp3nCrycwTpaPPEHbhtsgsjd5pDQN72wrV-pXoMp1L-O8Sm4eAJplHPtvZxr5qJ6POTkWJa6qd7E9cEY-tuU4eY7iC4gGT3VyUw7P0mcnbGplUhLJSyQillqkHblG-zipm7tmC5pmYeb75qJZh7o6G0SOegjy5eFL06ZtHuCe6myPWm~mTJhIi0MfwFX9X7LMoFAN4GPkwLhqQF~vPzxpWGfUqPrpnsNW6QVN89wVIVfDEJhNruATw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010174&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A block diagram of the algorithm for the optimization of random matrices <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>M</mi><mrow><mo>(</mo><mrow><mn>3</mn><mo>,</mo><mn>16</mn></mrow><mo>)</mo></mrow></mrow></math></span> from the <em>MR</em> set, used to search a matrix with the largest <em>mF</em><sub>max</sub>.</p></div>
</div>                        <a id="134010175" scrollto-destination="134010175"></a>
<div id="dsy046-F3" data-id="dsy046-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f3.png?Expires=1576262930&amp;Signature=z9Uyn~UA2Wgo1324LW8rt8S9H~k2a0aORQZf8QLtLVKZ-ZDiciw3cgmgu5KfS384O0t0ZsFXg4LsveWUvQUfNO3AXUrHnI8I~vrW9eaNQ9sO341gXiXk0nvoWB0Gl6hQFHajzgpvXgMvHZxlOw0eJsz-6FIXT-v0IzjWzTRl9x9CHrBZfYtv~cADx95lV12qYruukCs5y6HHjXA5YgxZdPUAHZv6beIlRW7fiY3Ol1SGpKCawze7Nejd10-C5y825ueSzVBSjhwem1T7G8fa0vdk9zJslwuw8HSP4JmIMC2hXTIpqKfdQyiG2sCF0OW32dfY8Le8wPgis3~qyuaJ5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The idea of the algorithm is to create a random matrix Wr, (or a set of random matrices), then optimize it with a genetic algorithm and get the W1 matrix as a result of optimization." path-from-xml="dsy046f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010175/dsy046f3.tif" path-from-xml="dsy046f3.tif" target="_blank">Open in new tab</a><a section="134010175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f3.png?Expires=1576262930&amp;Signature=gwvi6V3NQAI4cBIXEBgIc0bzA35gX-Rlf8MXTiQbXk8vG6LprOF6D7ZXrDOqaEahOX5xFxh10f1OqPuEYhv5ZuecZHfiHgcYEXkii6pQzEZa8OfTQjuAYHZ6Dhbw9rxrytwt83tggX8H-89IsnNyfUvA4hCKkq33mC3I0SSu7jVsZpO0j7JIJfCAesbOCppoeg-9CrxyX1RawJG6z32b5pyogHE7LktfOElEMdKtcRFHlvsvO11jK1eM~qNnA9ADP7UYMmj75WL7nO-xorFEw9ftFLklQzUQb9NCM2DoMZipdvusRm0hXce8~XR3f3moomK0O1wOg7dtWAupg7ynRg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010175&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The idea of the algorithm is to create a random matrix <em>W</em><sub>r</sub>, (or a set of random matrices), then optimize it with a genetic algorithm and get the <em>W</em><sub>1</sub> matrix as a result of optimization.</p></div>
</div>
<div content-id="dsy046-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f3.png?Expires=1576262930&amp;Signature=z9Uyn~UA2Wgo1324LW8rt8S9H~k2a0aORQZf8QLtLVKZ-ZDiciw3cgmgu5KfS384O0t0ZsFXg4LsveWUvQUfNO3AXUrHnI8I~vrW9eaNQ9sO341gXiXk0nvoWB0Gl6hQFHajzgpvXgMvHZxlOw0eJsz-6FIXT-v0IzjWzTRl9x9CHrBZfYtv~cADx95lV12qYruukCs5y6HHjXA5YgxZdPUAHZv6beIlRW7fiY3Ol1SGpKCawze7Nejd10-C5y825ueSzVBSjhwem1T7G8fa0vdk9zJslwuw8HSP4JmIMC2hXTIpqKfdQyiG2sCF0OW32dfY8Le8wPgis3~qyuaJ5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The idea of the algorithm is to create a random matrix Wr, (or a set of random matrices), then optimize it with a genetic algorithm and get the W1 matrix as a result of optimization." path-from-xml="dsy046f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010175/dsy046f3.tif" path-from-xml="dsy046f3.tif" target="_blank">Open in new tab</a><a section="134010175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f3.png?Expires=1576262930&amp;Signature=gwvi6V3NQAI4cBIXEBgIc0bzA35gX-Rlf8MXTiQbXk8vG6LprOF6D7ZXrDOqaEahOX5xFxh10f1OqPuEYhv5ZuecZHfiHgcYEXkii6pQzEZa8OfTQjuAYHZ6Dhbw9rxrytwt83tggX8H-89IsnNyfUvA4hCKkq33mC3I0SSu7jVsZpO0j7JIJfCAesbOCppoeg-9CrxyX1RawJG6z32b5pyogHE7LktfOElEMdKtcRFHlvsvO11jK1eM~qNnA9ADP7UYMmj75WL7nO-xorFEw9ftFLklQzUQb9NCM2DoMZipdvusRm0hXce8~XR3f3moomK0O1wOg7dtWAupg7ynRg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010175&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The idea of the algorithm is to create a random matrix <em>W</em><sub>r</sub>, (or a set of random matrices), then optimize it with a genetic algorithm and get the <em>W</em><sub>1</sub> matrix as a result of optimization.</p></div>
</div>                    <h3 scrollto-destination="134010176" id="134010176" class="section-title">2.2. Methods for creating a <em>W</em><sub>2</sub>(3, 16) random matrix</h3>
<div class="block-child-p">Assuming we have a DNA sequence <em>S</em> (cds), to create a <em>W</em><sub>2</sub>(3, 16) random matrix, we used the <em>S</em><sub>2</sub> sequence, which was obtained from the <em>S</em> sequence by randomly shuffling the nucleotides. The random shuffling algorithm has already been described in detail.<a reveal-id="dsy046-B32" data-open="dsy046-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> To do this, a sequence of <em>RS</em> of the same length as the <em>S</em> sequence was generated by a random number generator. Thereafter, we sorted the sequence <em>RS</em> in ascending order and memorized the permutations. Thereafter, these permutations were performed in the sequence <em>S</em> and the sequence <em>S</em><sub>2</sub> was obtained<em>.</em> Then, using the <em>S</em><sub>2</sub> sequence, we filled the <em>M</em><sub>2</sub>(3, 16) matrix. We filled the <em>W</em><sub>2</sub>(3, 16) matrix according to the equation: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>W</mi></mrow><mn>2</mn></msub><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>k</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><msub><mrow><mi>M</mi></mrow><mn>2</mn></msub><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>k</mi></mrow><mo>)</mo></mrow><mo>−</mo><mo stretchy="false">(</mo><mi>N</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><msub><mrow><mi>p</mi></mrow><mn>2</mn></msub><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo></mrow><mrow><msqrt><mrow><mo stretchy="false">(</mo><mi>N</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><msub><mrow><mi>p</mi></mrow><mn>2</mn></msub><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><msub><mrow><mi>p</mi></mrow><mn>2</mn></msub><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></msqrt></mrow></mfrac></mrow></math></div></div>
<div class="label">(1)</div>
</div>
</div>
<p>Here <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>p</mi></mrow><mn>2</mn></msub><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo><mo>=</mo><mi>p</mi><mo stretchy="false">(</mo><mi>l</mi><mo stretchy="false">)</mo><mi>p</mi><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></math>⁠</span>, where <em>p</em>(<em>l</em>) and <em>p</em>(<em>m</em>) are the probabilities of the <em>l-</em> or <em>m-</em>type nucleotides in the <em>S<sub>2</sub></em>sequence (<em>l, m</em>∈{a, t, c, g}); <em>p</em>(<em>l</em>) <em>= q</em>(<em>l</em>)<em>/N</em>, <em>q</em>(<em>l</em>) is the number of <em>l-</em>type nucleotides in the <em>S</em><sub>2</sub> sequence, and <em>N</em> is the <em>S</em><sub>2</sub> sequence length. This PWM calculation considers two types of statistical regularities in the <em>S</em><sub>2</sub> sequence. On the one hand, it considers heterogeneities in the nucleotide frequencies at each codon position, because the <em>p</em><sub>2</sub> probability without specificity to each codon position is used. On the other hand, this matrix also considers the correlation of neighbouring bases, because the expected number of each of the 16 pairs is calculated for each <em>i</em> position in the codon, <em>i</em>∈{1, 2, 3}.</p>                    <h3 scrollto-destination="134010179" id="134010179" class="section-title">2.3. Optimization of the <em>W</em><sub>2</sub>(3, 16) matrix using genetic algorithm and dynamic programming</h3>
                    <h4 scrollto-destination="134010180" id="134010180" class="section-title">2.3.1. Application of dynamic programming for <em>W</em><sub>2</sub>(3, 16) matrix optimization</h4>
<div class="block-child-p">Assuming we have a DNA sequence <em>S</em>, then we optimized the corresponding <em>W</em><sub>2</sub>(3, 16) matrix by a genetic algorithm to maximize the similarity function <em>F</em>. To calculate the similarity function, the <em>S</em> sequence was aligned with respect to the <em>W</em><sub>2</sub>(3, 16) matrix by an iterative procedure, and the <em>F</em> value was calculated: <a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable xmlns=""><mtr><mtd><mi>F</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo>=</mo><mi>max</mi><mrow><mo>{</mo><mtable><mtr><mtd><mi>F</mi><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>n</mi><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>F</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>−</mo><mi>d</mi></mtd></mtr><mtr><mtd><mi>F</mi><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo>−</mo><mi>d</mi></mtd></mtr></mtable><mo>}</mo></mrow><mo>.</mo></mtd></mtr><mtr><mtd></mtd></mtr></mtable></math></div></div>
<div class="label">(2)</div>
</div>
</div>
<div class="block-child-p">Here, <em>i</em> runs from 1 to <em>N + h</em>, and <em>j</em> runs from 2 to <em>N</em>. The <em>h</em> constant indicates the maximum number of insertions and deletions that could appear in the final alignment. We choose it as equal to 50, because among the studied sequences there was no cds with the number of insertions or deletions &gt;50. Here, <em>a</em>(<em>i</em>) was calculated as <em>i</em> − 3int(<em>i</em>/3.0). Therefore, <em>a</em>(<em>i</em>) takes the values 1, 2, 3, 1, 2, 3, 1, 2, 3, … for <em>i </em>=<em> </em>1, 2, 3, 4, 5, 6, 7, 8, 9, … . <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> is the transformed <em>W</em><sub>2</sub> matrix. The transformation (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>W</mi></mrow><mn>2</mn></msub></mrow></math></span><em>=&gt;</em><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>) was carried out so that all matrices <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> (which are used in Equation 2) would have the same value <em>R</em><sup>2</sup> = <em>R</em><sub>0</sub> and <em>K<sub>d</sub></em> = <em>K</em><sub>0.</sub> For any matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>, the constants <em>R</em> and <em>K<sub>d</sub></em>can be calculated using the following equations: <a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mrow><mi>R</mi></mrow><mn>2</mn></msup><mo>=</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></munderover><mrow><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>16</mn></mrow></munderover><mrow><msubsup><mrow><mi>w</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></mrow><msup><mrow><mrow><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></mrow><mn>2</mn></msup><mo>,</mo></mrow></math></div></div>
<div class="label">(3)</div>
</div>
<a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>K</mi></mrow><mi>d</mi></msub><mo>=</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></munderover><mrow><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>16</mn></mrow></munderover><mrow><msubsup><mrow><mi>w</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></mrow><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>k</mi><mo stretchy="false">)</mo><mi>p</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><msub><mrow><mi>p</mi></mrow><mn>2</mn></msub><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo><mo>.</mo></mrow></math></div></div>
<div class="label">(4)</div>
</div>
<em>p</em>(<em>i</em>) is the probability of encountering the symbols 1, 2 and 3 in the sequence <em>a</em>(<em>i</em>) and it is 1/3 for any <em>i</em>. The probability <em>p</em><sub>2</sub>(<em>k</em>) is defined in Equation (1) above. For all calculations in this work, we used <em>R</em><sub>0</sub> = 1,050 and <em>K</em><sub>0</sub> = −1.8. These values were chosen based on the following considerations. <em>R</em><sub>0</sub> was calculated as the sum of squares of the matrix <em>W</em><sub>2</sub>. The matrix <em>W</em><sub>2</sub> was created for the sequence <em>S</em> = {<em>atg</em>}<sub>400</sub> with the introduction of 2.0 random substitutions per nucleotide. The sequence <em>S</em> had a level of TP, which is average for the genes under study. We estimated the level of TP based on the <em>MT</em> matrix using mutual information converted into an argument of normal distribution <em>X</em>. The average level of <em>X</em> for the analysed genes was about 6.2. These calculations have earlier been described in detail.<a reveal-id="dsy046-B38" data-open="dsy046-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a>
</div>
<div class="block-child-p">The value <em>K</em><sub>0</sub> was chosen using a <em>Gr</em> set of artificially created sequences. The value of <em>K</em><sub>0</sub> reflects the accuracy of determining the beginning and the end of the local alignment and the number of values which are greater or less than zero in the matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>.<sup>3</sup><sup>2</sup> The volume of the set <em>Gr</em> was 500 sequences. Each sequence had a length of 1,200 nt. The first 400 bases of this sequence were random. The next 400 bases correspond to some random matrix <em>MT</em> for which the TP level <em>X </em>=<em> </em>6.2 (see the previous section). Then again, follow a random sequence of 400 bases. The matrix <em>MT</em> was used to calculate the weight matrix <em>WT</em> according to the equation: <a id="E5" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>W</mi><mi>T</mi><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mi>M</mi><mi>T</mi><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>L</mi><mi>p</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo></mrow><mrow><msqrt><mrow><mi>L</mi><mi>p</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>p</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></msqrt></mrow></mfrac><mo>.</mo></mrow></math></div></div>
<div class="label">(5)</div>
</div>
</div>
<div class="block-child-p">Here, <em>p</em>(<em>i</em>, <em>j</em>) = <em>x</em>(<em>i</em>)<em>y</em>(<em>j</em>)/<em>L</em><sup>2</sup>, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>x</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></msubsup><mrow><mi>M</mi><mi>T</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></mrow></math>⁠</span>, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>y</mi><mo stretchy="false">(</mo><mi>j</mi><mo stretchy="false">)</mo><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><mi>M</mi><mi>T</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></mrow></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>L</mi><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></msubsup><mrow><mi>M</mi><mi>T</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo></mrow></mrow></mrow></math>⁠</span>. Then, for each <em>WT</em> matrix, we constructed a local alignment according to Equation (6). Here we use the sequences <em>a</em>(<em>i</em>) also, the indices <em>i</em> and <em>j</em> are the same as in Equation (2). <a id="E6" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable xmlns=""><mtr><mtd><mi>E</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo>=</mo><mi>max</mi><mrow><mo>{</mo><mtable><mtr><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mi>E</mi><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mi>W</mi><mi>T</mi><mo stretchy="false">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo></mtd></mtr><mtr><mtd><mi>E</mi><mo stretchy="false">(</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>−</mo><mi>d</mi></mtd></mtr><mtr><mtd><mi>E</mi><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo stretchy="false">)</mo><mo>−</mo><mi>d</mi></mtd></mtr></mtable><mo>}</mo></mrow><mo>.</mo></mtd></mtr><mtr><mtd></mtd></mtr></mtable></math></div></div>
<div class="label">(6)</div>
</div>
</div>
<p>We found <em>E</em><sub>max</sub>, the coordinates of the maximum <em>i</em><sub>max</sub> and <em>j</em><sub>max</sub> as well as the coordinates of the beginning of the alignment <em>i</em><sub>0</sub> and <em>j</em><sub>0,</sub> for each sequence from the set <em>Gr</em>. We tested different values of <em>K</em><sub>0</sub> and chose <em>K</em><sub>0</sub> = −1.8. In this case, the sum of differences <em>i</em><sub>0</sub> and <em>j</em><sub>0</sub> from 400, plus the sum of the differences <em>i</em><sub>max</sub> and <em>j</em><sub>max</sub> from 800 were minimal and equal to 46 nt.</p>
<p>Let us continue to consider Equation (2). For this equation <em>n </em>=<em> s</em>(<em>k</em>) + 4(<em>s</em>(<em>j</em>) − 1) ranges from 1 to 16. The index <em>k</em> is calculated using transitions already created in the <em>F</em> matrix. As the matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>n</mi><mo stretchy="false">)</mo></mrow></math></span> is calculated for pairs of bases, to calculate <em>k</em>, it is necessary to determine the previous base of sequence <em>S</em>, which is included in the alignment. The previous base can be found by calculating the path to the cell with coordinates (<em>i</em>, <em>j</em>). If we get to the (<em>i, j</em>) cell from the (<em>i</em> − 1<em>, j</em> − 1) cell and we get to the (<em>i</em> – 1<em>, j</em> – 1) cell from the (<em>i</em> – 2<em>, j</em> – 2) cell, then <em>k </em>=<em> j</em> – 1. This corresponds to the transitions <em>F</em>(<em>i</em> – 2, <em>j</em> – 2) =&gt; <em>F</em>(<em>i</em> – 1, <em>j</em> – 1) =&gt; <em>F</em>(<em>i</em>, <em>j</em>) in the matrix <em>F</em>. Such a move corresponds to the diagonal move, and there are no insertions or deletions. If we get to the (<em>i</em>, <em>j</em>) cell from the (<em>i</em> − 1, <em>j</em> − 1) cell, to the (<em>i</em> − 1, <em>j</em> − 1) cell from the (<em>i</em> − 1, <em>j</em> − 2) cell and to the (<em>i</em> − 1, <em>j</em> − 2) cell from the (<em>i</em> − 2, <em>j</em> − 3) cell, then <em>k </em>=<em> j</em> − 2. This move corresponds to the skipping of a single base in the sequence <em>S</em>. Therefore, the transitions in the matrix <em>F</em> are <em>F</em>(<em>i</em> − 2, <em>j</em> − 3) =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 2) =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em>, <em>j</em>). The longer deletions in the sequence <em>S</em> (not longer than <em>h</em>) are treated similarly. Assuming that there is a deletion of length <em>q</em> in the sequence <em>S</em>; this means that <em>k </em>=<em> j</em> − 1 − <em>q</em> and the path to the cell (<em>i</em>, <em>j</em>) is <em>F</em>(<em>i</em> − 2, <em>j</em> − 2 − <em>q</em>) =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 1 − <em>q</em>) =&gt; … =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 2) =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em>, <em>j</em>).</p>
<div class="block-child-p">Deletions can also occur in the sequence <em>a</em>(<em>i</em>). These deletions correspond to a skipping of columns of the matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>n</mi><mo stretchy="false">)</mo></mrow></math></span>in the resulting alignment. The path <em>F</em>(<em>i</em> − 3, <em>j</em> − 2) =&gt; <em>F</em>(<em>i</em> − 2, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em>, <em>j</em>) corresponds to a skipping of a single column of the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> matrix. In this case, we cannot use the matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> because it uses pairs of neighbouring bases. Therefore we need a weight matrix calculated for pairs of bases separated by one base, i.e. the pairs <em>a</em>(<em>i</em>)<em>a</em>(<em>i </em>+<em> </em>2), <em>i </em>=<em> </em>1, 2, …, <em>N </em>+<em> h</em> − 2 [<em>N </em>+<em> h</em> is the length of the sequence <em>a</em>(<em>i</em>)]. Therefore, we need another matrix (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>3</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>) which could be obtained from the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> matrix. The <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>3</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> matrix contains weights for pairs of bases which are in codon positions 2-1, 3-2 and 1-3 (i.e. separated by one codon position). <a id="E7" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E7"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>3</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>i</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>l</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>=</mo><mn>0.25</mn><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></munderover><mrow><mo stretchy="false">(</mo><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>i</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>j</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>+</mo><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>j</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>l</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>/</mo><mn>2.0</mn></mrow><mo>.</mo></mrow></math></div></div>
<div class="label">(7)</div>
</div>
<em>i</em>, <em>j</em> and <em>l</em> denote the bases: 1 = <em>a</em>, 2 = <em>t</em>, 3 = <em>c</em>, 4 = <em>g</em>. Equation (7) is based on the assumption that a weight of the pair of bases (<em>i</em>)(<em>l</em>) separated by the single base <em>j</em> can be calculated using the weights of two intersecting pairs of bases (<em>i</em>)(<em>j</em>) and (<em>j</em>)(<em>l</em>)<em>.</em> Therefore, one can use the averaged weight of four possible intersecting pairs to estimate the weight of the pair (<em>i</em>)(<em>l</em>). Here <em>i</em> and <em>l</em> are fixed, and <em>j</em> ranges from 1 to 4.</div>
<div class="block-child-p">A deletion of two columns of the matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> corresponds to the path <em>F</em>(<em>i</em> − 4, <em>j</em> − 2) =&gt; <em>F</em>(<em>i</em> − 3, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em> − 2, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em> − 1, <em>j</em> − 1) =&gt; <em>F</em>(<em>i</em>, <em>j</em>). In this case, the equation for the calculation of the matrix<span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>3</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> is as follows: <a id="E8" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E8"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>3</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>i</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>l</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>=</mo><mn>0.0625</mn><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></munderover><mrow><munderover><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mn>4</mn></munderover><mrow><mo stretchy="false">(</mo><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>i</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>j</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>+</mo><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>k</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>l</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>/</mo><mn>2.0</mn></mrow></mrow><mo>.</mo></mrow></math></div></div>
<div class="label">(8)</div>
</div>
</div>
<p>Here <em>i</em>, <em>j</em>, <em>k</em> and <em>l</em> also denote the bases: 1 = <em>a</em>, 2 = <em>t</em>, 3 = <em>c</em>, 4 = <em>g</em>. Equation (8) is based on the assumption that the weight of a pair of bases (<em>i</em>)(<em>l</em>) separated by two bases (<em>j</em> and <em>k</em>) can be calculated based on the weights of the neighbouring pairs (<em>i</em>)(<em>j</em>) and (<em>k</em>)(<em>l</em>). Therefore, to estimate the weight of the pair separated by two positions, one can use the average weight of 16 possible neighbouring pairs. Here <em>i</em> and <em>l</em> are fixed, and <em>j</em> and <em>k</em> run from 1 to 4. If we delete three columns of the matrix, we return to the matrix<span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>. This will be correct for all deletions that are multiples of 3. Therefore, we used Equation (7) for column deletions of length 1, 4, 7, … and we used Equation (8) for column deletions of length 2, 5, 8, ….</p>
<div class="block-child-p">The zero row and column of the <em>F</em> matrix were filled with negative numbers, and the <em>F</em>(0, 0), <em>F</em>(1, 0), …, <em>F</em>(<em>h</em>, 0) were 0. For transition from the zero column to the first column of the <em>F</em> matrix and from the zero row to the first row of the <em>F</em> matrix, the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>4</mn><mo>'</mo></msubsup></mrow></math></span><em>F</em> matrix was used. It was determined as follows: <a id="E9" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E9"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mi>W</mi></mrow><mn>4</mn><mtext>t</mtext></msubsup><mo stretchy="false" xmlns="">(</mo><mi xmlns="">x</mi><mo xmlns="">,</mo><mi xmlns="">n</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mn xmlns="">0.25</mn><mstyle displaystyle="true" xmlns=""><munder><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>4</mn></mrow></munder><mrow><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mtext>t</mtext></msubsup><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>i</mi><mo>+</mo><mn>4</mn><mo stretchy="false">(</mo><mi>n</mi><mo>−</mo><mtext>1</mtext><mo stretchy="false">)</mo><mo stretchy="false">)</mo><mo>.</mo></mrow></mstyle></math></div></div>
<div class="label">(9)</div>
</div>
</div>
<p>Here, averaging over the four previous bases occurs and the weight depends only on the base in position <em>j</em>. In this case, the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>4</mn><mo>'</mo></msubsup></mrow></math></span> matrix replaces the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> matrix, but <em>n </em>=<em> s</em>(<em>j</em>) according to Equation (2).</p>
<p>The <em>d</em> constant (see Equation 2) plays an important role. It is intuitively clear that the smaller the statistical significance of TP in the <em>S</em> sequence, the higher the <em>d</em> value should be. To select the <em>d</em> value, we generated 1,000 sequences of 600 nt long for each level of the TP in the form of the <em>x</em> normal distribution argument<a reveal-id="dsy046-B33" data-open="dsy046-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> in the interval from 0 to 20 and with a step of 1. Then in a random position of this sequence, no closer than 100 nt from the beginning and end, a deletion of one base was introduced. Let us call each set of such sequences <em>MP</em>(<em>x</em>). For each <em>MP</em>(<em>x</em>) set, we selected the <em>d</em> value in such a way that the number of insertions or deletions that were made by the method outside the distance (±50) from the artificial deletion did not exceed 5%. The obtained <em>d</em> values are presented in <a reveal-id="dsy046-F4" data-open="dsy046-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>. </p>                        <a id="134010193" scrollto-destination="134010193"></a>
<div id="dsy046-F4" data-id="dsy046-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f4.png?Expires=1576262930&amp;Signature=ZhJPGsDsvHYpWfatGTFBPzfxlylrXt3D0u6zgyhXtvE0Eo9vdohFMzFWNHqJ~i6mCEbpg3lKPvQU3RvR8ie30WLiw2eorhyX4dhtd9m97JbrhMKOUdm2OERZg0r5C7AFPp-KQEQeFgubAZiCzEIP8anR2ok5sXsfkG1aEuduNxRwrRF-ImuBn7efJxPrSKiK2CROLfpssXHU806xmU7YU5tzSJTTKwK55Xi66Djq9VoPD-jmc11ttLRSgr8pz2fwQa4OvCODnCcug1NdBnr7ZFdm81yf7-SL4BpYr1y8DosU6tLb8bgTXM8pz5xkIcL8Uzmi8wwyTHlDl4ntHVw8Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The dependence of the d value (see Equation 2) on the TP, of the analysed sequence. The TP was calculated using the TP matrix33 and is expressed in the arguments of the normal distribution and is shown along the x-axis." path-from-xml="dsy046f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010193/dsy046f4.tif" path-from-xml="dsy046f4.tif" target="_blank">Open in new tab</a><a section="134010193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f4.png?Expires=1576262930&amp;Signature=IdmvVqJjxepxGTNPXioGjxmYPWdoP695m4ooKOC8kTfPm8iWWSSpdOIbmodpK7N3hlvj4JWHDlD7YJ5BaP28rgUDj-JgTQSgXWXMVAbFG6j0AiWRLFhWlbihK~lEuSYfd61117cMnMXT0PJNMTjE-vo2unuAVJYshhSsuz~P0pRQ0aWBAfCKg1gvsivoAlU~FypJOAbLxx76l5FNvIdXRK4yhq78xKMFB1~kkGv5cAVImhBmOzzEcmU-2LniuqvRRWCgrYJmi1JFzog0AxzmgWINPtH7zpfAbB51Bb6eXNBeg127jddm~CaTpwwdLiVuChqIVGgO76SDaV2n0FmoZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010193&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The dependence of the <em>d</em> value (see Equation 2) on the TP, of the analysed sequence. The TP was calculated using the TP matrix<a reveal-id="dsy046-B33" data-open="dsy046-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and is expressed in the arguments of the normal distribution and is shown along the <em>x</em>-axis.</p></div>
</div>
<div content-id="dsy046-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f4.png?Expires=1576262930&amp;Signature=ZhJPGsDsvHYpWfatGTFBPzfxlylrXt3D0u6zgyhXtvE0Eo9vdohFMzFWNHqJ~i6mCEbpg3lKPvQU3RvR8ie30WLiw2eorhyX4dhtd9m97JbrhMKOUdm2OERZg0r5C7AFPp-KQEQeFgubAZiCzEIP8anR2ok5sXsfkG1aEuduNxRwrRF-ImuBn7efJxPrSKiK2CROLfpssXHU806xmU7YU5tzSJTTKwK55Xi66Djq9VoPD-jmc11ttLRSgr8pz2fwQa4OvCODnCcug1NdBnr7ZFdm81yf7-SL4BpYr1y8DosU6tLb8bgTXM8pz5xkIcL8Uzmi8wwyTHlDl4ntHVw8Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The dependence of the d value (see Equation 2) on the TP, of the analysed sequence. The TP was calculated using the TP matrix33 and is expressed in the arguments of the normal distribution and is shown along the x-axis." path-from-xml="dsy046f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010193/dsy046f4.tif" path-from-xml="dsy046f4.tif" target="_blank">Open in new tab</a><a section="134010193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f4.png?Expires=1576262930&amp;Signature=IdmvVqJjxepxGTNPXioGjxmYPWdoP695m4ooKOC8kTfPm8iWWSSpdOIbmodpK7N3hlvj4JWHDlD7YJ5BaP28rgUDj-JgTQSgXWXMVAbFG6j0AiWRLFhWlbihK~lEuSYfd61117cMnMXT0PJNMTjE-vo2unuAVJYshhSsuz~P0pRQ0aWBAfCKg1gvsivoAlU~FypJOAbLxx76l5FNvIdXRK4yhq78xKMFB1~kkGv5cAVImhBmOzzEcmU-2LniuqvRRWCgrYJmi1JFzog0AxzmgWINPtH7zpfAbB51Bb6eXNBeg127jddm~CaTpwwdLiVuChqIVGgO76SDaV2n0FmoZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010193&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The dependence of the <em>d</em> value (see Equation 2) on the TP, of the analysed sequence. The TP was calculated using the TP matrix<a reveal-id="dsy046-B33" data-open="dsy046-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and is expressed in the arguments of the normal distribution and is shown along the <em>x</em>-axis.</p></div>
</div>
<p>Simultaneously with the <em>F</em> matrix, the inverse transition matrix was also filled, as is usually the case when searching for global alignment. Then, using the inverse transition matrix, the alignment of the <em>S</em> sequence with respect to the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> matrix was constructed, and the value of the <em>F</em><sub>max</sub> similarity function in the cells of the matrix <em>F</em>(<em>N</em>, <em>N</em>)<em>, F</em>(<em>N </em>+<em> </em>1, <em>N</em>), …, <em>F</em>(<em>N </em>+<em> h</em>, <em>N</em>) was determined.</p>
<p>The transformation of matrix <em>W</em><sub>2</sub> into matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> makes possible the achievement of similar distributions for the values of <em>F</em><sub>max</sub> on the set of random sequences of <em>S</em> for different <em>W</em><sub>2</sub> matrices. This is the essence of the transformation of matrix <em>W</em><sub>2</sub> into matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>. The similarity of distributions enables us to use <em>F</em><sub>max</sub> as a measure of the similarity of the matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> and the analysed sequence <em>S</em>. This allows consideration of the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi>t</mi></msubsup></mrow></math></span> matrix having the largest value of <em>F</em><sub>max</sub> as the matrix that best represents periodicity in sequence <em>S</em>. It is possible to carry out all the calculations without such transformation and using <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math>⁠</span>. Also, it is possible to use matrix <em>W</em><sub>2</sub> instead of matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup></mrow></math></span> in Equation (2). However, the comparison of <em>F</em><sub>max</sub> for different <em>W</em><sub>2</sub> matrices will have to be done by the Monte Carlo method, which considerably slows down implementation of the genetic algorithm (Section 2.3.2).</p>                    <h4 scrollto-destination="134010196" id="134010196" class="section-title">2.3.2. Application of the genetic algorithm for <em>W</em><sub>2</sub>(3, 16) matrix optimization</h4>
<p>When implementing the genetic algorithm, we considered the <em>F</em><sub>max</sub> value as the target function whereas the <em>W</em><sub>2</sub>(3, 16) matrix was considered as the ‘organism’. The use of the genetic algorithm for optimizing the <em>W</em><sub>2</sub>(3, 16) matrix was examined in detail in Ref. <a reveal-id="dsy046-B32" data-open="dsy046-B32" class="link link-ref link-reveal xref-bibr">32</a>, and the reader is hereby referred to that publication. Let us consider in general the operating process of the genetic algorithm. First, the <em>MR</em> set of the <em>W</em><sub>2</sub>(3, 16) random matrices with a volume of 500 matrices was generated. Each matrix was created as described in Section 2.1. Thereafter, each matrix was transformed to obtain a set of matrices with the same <em>R</em><sup>2</sup> and <em>K<sub>d</sub></em>, as described in Section 2.3.1 and in Ref. <a reveal-id="dsy046-B32" data-open="dsy046-B32" class="link link-ref link-reveal xref-bibr">32</a>. Consequently, a number of <em>MR</em><sup>t</sup> matrices were obtained. Each matrix from the <em>MR</em><sup>t</sup> set was subjected to the dynamic programming procedure to align with the <em>S</em> sequence, and the <em>F</em><sub>max</sub> value was calculated as described in Section 2.3.1. <em>F</em><sub>max</sub> was considered as the target function. Thereafter, two matrices from the <em>MR</em><sup>t</sup> set were selected, and the higher the probability of selecting these matrices, the greater was the value of the <em>F</em><sub>max</sub> objective function thereof. These two matrices ‘intercrossed’, and a ‘descendant’ was created. A descendant is a matrix, which possesses part of the cells from one matrix and a part of the cells from another. Then, one matrix from the <em>MR</em><sup>t</sup> set ‘perished’, and the probability of collapse was greater, the lower the <em>F</em><sub>max</sub> value for this matrix, and its place was taken by the descendant. In addition, random ‘point’ mutations were introduced in 10% of the randomly selected matrices from the <em>MR</em><sup>t</sup> set. Also, the greater the probability that the chosen matrix would be selected to introduce random mutations, the lower was its <em>F</em><sub>max</sub>. Mutations were introduced into a random cell, and the number contained there was changed to a random number in a uniform interval from −5 to +5. Let us call the entire process an iteration. Therefore, within a single iteration, the <em>F</em><sub>max</sub> value is calculated in 500 matrices, one matrix is deleted, one descendant is created, and random ‘point’ mutations are introduced into 50 matrices. Let us call <em>mF</em><sub>max</sub> the maximum value for <em>F</em><sub>max</sub>, which is obtained within a single iteration of the <em>MR</em> set. Then, the <em>MR</em> set is replaced by the <em>MR</em><sup>t</sup> set and the process is repeated from the very beginning.</p>
<p>In the result of the genetic algorithm operation, the <em>mF</em><sub>max</sub> was continuously increased. The genetic algorithm operation was stopped after the <em>mF</em><sub>max</sub> value stopped increasing during 50 iterations. On average, about 9 × 10<sup>3</sup> iterations are required to reach this point.</p>                    <h3 scrollto-destination="134010199" id="134010199" class="section-title">2.4. Developing a measure of significance for the phase shifts of TP</h3>
<p>After carrying out the genetic algorithm, we obtained a single <em>W</em><sub>1</sub>(3, 16) matrix (Section 2.1) which possesses the maximum similarity function (<em>mF</em><sub>max</sub>) and the alignment of the <em>S</em> sequence with respect to the columns of the <em>W</em><sub>1</sub>(3, 16) matrix. We are not interested in the <em>mF</em><sub>max</sub> values itself (it characterizes the TP level in the sequence), but rather in the statistical significance of the found TP phase shifts. To estimate it, the <em>mF</em><sub>max</sub> value was divided into three parts using the alignment sequence <em>S</em> and the <em>W</em><sub>1</sub>(3, 16) matrix. The first part is the region of the <em>S</em> sequence, where the positions of the <em>W</em><sub>1</sub>(3, 16) matrix columns and the reading frame in the <em>S</em> sequence coincide. In <a reveal-id="dsy046-F5" data-open="dsy046-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, this area is designated as <em>V</em>1. The second part (<a reveal-id="dsy046-F5" data-open="dsy046-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, <em>V</em>2) accounts for coincidences of the following form: 1 =&gt; 2, 2 =&gt; 3, 3 =&gt; 1, and the third part (<a reveal-id="dsy046-F5" data-open="dsy046-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, <em>V</em>3) falls on coincidences of the following form: 1 =&gt; 3, 2 =&gt; 1, 3 =&gt; 2. The sum of <em>V</em>1 + <em>V</em>2 + <em>V</em>3 − <em>kd</em> is equal to <em>mF</em><sub>max</sub>, where <em>k</em> is the number of inserts or deletions, and <em>d</em> is the price for insertion or deletion from Equation (2). Initially, the <em>W</em><sub>1</sub>(3, 16) matrix is unconnected with the reading frame. Therefore, cyclic rearrangements of the <em>W</em><sub>1</sub>(3, 16) matrix were carried out, so that <em>V</em>1 <em>≥ V</em>2 and <em>V</em>1 <em>≥ V</em>3. As an indicator that could tell us about the presence of phase shifts of TP, we assumed <em>V</em>2 + <em>V</em>3. </p>                        <a id="134010201" scrollto-destination="134010201"></a>
<div id="dsy046-F5" data-id="dsy046-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f5.png?Expires=1576262930&amp;Signature=YaqvGFfIyf7wHV~FuSw12m1FbDksoq39Z9WBraXTgEaTIJXIl-fbxUCNSg~o8TFcv1xO00CZPAXhJ2t736kG7i5tNmVLK54uPhPKya1RwO9B8XqxbQuOi~BX-Q0lp0esvI93XH6RqCM5FrT7NVkKXbVzxf~smHHqSht~q7zERq9ZkcGFUBPkKZrm3AmdpUSoelWpgkjNL532o~TsG4ftZOY3sbKMpDzNSKJgrF99JP1HjbLAVE2UbkfWy3TiZuF48WuFgZkpssb8Z0uCv-NrEdb3LgA2mP6c-Kbac3VhHV8EUOGd6djp7v44qSpf5X9MVdHr3beMgcnaEblGJm6MBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme of division of mFmax on V1, V2 and V3 (see Section 2.4)." path-from-xml="dsy046f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010201/dsy046f5.tif" path-from-xml="dsy046f5.tif" target="_blank">Open in new tab</a><a section="134010201" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f5.png?Expires=1576262930&amp;Signature=0MPXtNWG8Xzrt08G6vgofasfgIysahmkAXl1naZneOy1ahx3WSYAxqMjnCYwuPOzUrGpmc9g3O-gTITJ1~7j78hHmlGnhzwBP2zMj3KTTiMKL4Orbdp7hwAEgJF6H2j8RjJZfDYOCJSTUg9X~SzWgt6MnXK-zD0lwbEQrLhXYoCyrEHeb-4IGqBGO3qb2ysZtkXokCnoNHcDsXdzfbTbsR-4rSMTprPnUy-w6yEWZKt8yKQ30zf0TUK8wxRId2MpxfTLdRL90wxbfI49aAhaiemV2gt-1aLc7YRBAnt7GThTOkt0gI3EXIMq2Fmx4Fy86aaRpslhyeeR3utjGdY5XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010201&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Scheme of division of <em>mF</em><sub>max</sub> on <em>V</em>1, <em>V</em>2 and <em>V</em>3 (see Section 2.4).</p></div>
</div>
<div content-id="dsy046-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f5.png?Expires=1576262930&amp;Signature=YaqvGFfIyf7wHV~FuSw12m1FbDksoq39Z9WBraXTgEaTIJXIl-fbxUCNSg~o8TFcv1xO00CZPAXhJ2t736kG7i5tNmVLK54uPhPKya1RwO9B8XqxbQuOi~BX-Q0lp0esvI93XH6RqCM5FrT7NVkKXbVzxf~smHHqSht~q7zERq9ZkcGFUBPkKZrm3AmdpUSoelWpgkjNL532o~TsG4ftZOY3sbKMpDzNSKJgrF99JP1HjbLAVE2UbkfWy3TiZuF48WuFgZkpssb8Z0uCv-NrEdb3LgA2mP6c-Kbac3VhHV8EUOGd6djp7v44qSpf5X9MVdHr3beMgcnaEblGJm6MBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scheme of division of mFmax on V1, V2 and V3 (see Section 2.4)." path-from-xml="dsy046f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010201/dsy046f5.tif" path-from-xml="dsy046f5.tif" target="_blank">Open in new tab</a><a section="134010201" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f5.png?Expires=1576262930&amp;Signature=0MPXtNWG8Xzrt08G6vgofasfgIysahmkAXl1naZneOy1ahx3WSYAxqMjnCYwuPOzUrGpmc9g3O-gTITJ1~7j78hHmlGnhzwBP2zMj3KTTiMKL4Orbdp7hwAEgJF6H2j8RjJZfDYOCJSTUg9X~SzWgt6MnXK-zD0lwbEQrLhXYoCyrEHeb-4IGqBGO3qb2ysZtkXokCnoNHcDsXdzfbTbsR-4rSMTprPnUy-w6yEWZKt8yKQ30zf0TUK8wxRId2MpxfTLdRL90wxbfI49aAhaiemV2gt-1aLc7YRBAnt7GThTOkt0gI3EXIMq2Fmx4Fy86aaRpslhyeeR3utjGdY5XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010201&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Scheme of division of <em>mF</em><sub>max</sub> on <em>V</em>1, <em>V</em>2 and <em>V</em>3 (see Section 2.4).</p></div>
</div>
<p>For each gene we defined a threshold <em>V</em>0 = <em>V</em>2 + <em>V</em>3, below which it could be said that there were no TP phase shifts in the sequence. The <em>V</em>0 value was selected for each <em>S</em> sequence, using the <em>SR</em> sequences set. As gene sequences possess various lengths and different <em>MT</em> matrixes,<a reveal-id="dsy046-B29" data-open="dsy046-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> the selection has to be done for each gene. The <em>SR</em> set contained 10<sup>3</sup><em>S</em> sequences, where the codons were randomly shuffled. Such a shuffling obviously destroys all probable TP phase shifts. We selected the <em>V</em>0 value that provided no more than 20 sequences with random insertions or deletions in the <em>SR</em> set. Such insertions or deletions were considered to be significant [<em>N</em>(<em>V</em>0) ≤ 20], where <em>N</em> is the number of sequences with a random shift of TP. If <em>V</em>2 + <em>V</em>3 = 0, then the <em>SR</em> set was not created, because there were no TP phase shifts in the sequence.</p>                    <h2 scrollto-destination="134010203" id="134010203" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="134010204" id="134010204" class="section-title">3.1. Estimation of the first and second type error rates of the developed method</h3>
<p>In order to determine the number of errors of the first and the second type, we used all the coding sequences in the <em>A. thaliana</em> genome, which constitutes 48,322 cds. These sequences were downloaded from the Ensembl website (<a class="link link-uri" href="http://ftp://ftp.ensemblgenomes.org/pub/release-38/plants/fasta/arabidopsis_thaliana/cds/" target="_blank">ftp://ftp.ensemblgenomes.org/pub/release-38/plants/fasta/arabidopsis_thaliana/cds/</a>). We replaced all symbols (except <em>a</em>, <em>t</em>, <em>c</em> and <em>g</em>) in these sequences with a randomly selected nucleotide. Such a replacement was made for all studied cds.</p>
<p>Then, the <em>RN</em> set was created by the random shuffling of codons of the sequences from the initial set. Sequences from the random set should not contain TP phase shifts while having the same statistical properties as sequences from the initial set. The <em>RN</em> set allows estimation of the number of errors of the first type (true positive). We applied the approach developed by us to sequences from the <em>RN</em> set. In the result, we found 1,098 sequences with the TP phase shift, with <em>V</em>2 + <em>V</em>3 value higher than the threshold level. The total number of TP phase shifts in these sequences was 1,549.</p>
<p>It is also interesting to determine the number of errors of the second type and the power of the method. To do this, another test set (<em>RD</em>) was created using coding sequences from the <em>A. thaliana</em> genome having a length longer than 500 nt (40,621 sequences). The codons in these sequences were randomly shuffled. Then, a single-base deletion was introduced in a random position of each sequence not closer than 100 nt from the beginning or the end of the sequence. These sequences were analysed using the developed algorithm, and the results are presented in <a reveal-id="dsy046-T2" data-open="dsy046-T2" class="link link-reveal link-table xref-fig">Table 2</a>. This table shows that the method identified 28,357 sequences in the <em>RD</em> set, where <em>V</em>2 + <em>V</em>3 ≥ <em>V</em>0, and the predicted position is in the range of ±50 from the artificial deletion. This shows that second type errors constitute 30%, and so the power of the method constitutes 70%. Also, the method rather accurately predicts the location of the frameshifts, because it was only in 1,128 sequences that frameshifts were found outside the region ±50 from the deletion point. It should also be noted that the method does not create a significant number of random frameshifts. This is because 29,485 sequences with statistically significant frameshifts (28,357 + 1,128) contain 29,888 shifts, i.e. 403 frameshifts are due to purely random factors.</p>                        <a id="134010208" scrollto-destination="134010208"></a>
<div content-id="dsy046-T2" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T2" class="table-wrap-title" data-id="dsy046-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Search for phase shifts in the set RD</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Name of organism </th>
<th rowspan="2">The total number of sequences in the <em>RD</em> set </th>
<th rowspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≠ 0 </th>
<th colspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≥ <em>V</em>0<hr> </th>
<th>Total number of shifts </th>
</tr>
<tr>
<th>Within ±50 </th>
<th>Out ± 50 </th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>1. <em>Arabidopsis thaliana</em> </td>
<td>40,612 </td>
<td>31,203 </td>
<td>28,357 </td>
<td>1,128 </td>
<td>29,888 </td>
</tr>
<tr>
<td>2. <em>Anaeromyxobacter dehalogenans</em> </td>
<td>3,460 </td>
<td>3,458 </td>
<td>2,975 </td>
<td>68 </td>
<td>3,668 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T2" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Name of organism </th>
<th rowspan="2">The total number of sequences in the <em>RD</em> set </th>
<th rowspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≠ 0 </th>
<th colspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≥ <em>V</em>0<hr> </th>
<th>Total number of shifts </th>
</tr>
<tr>
<th>Within ±50 </th>
<th>Out ± 50 </th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>1. <em>Arabidopsis thaliana</em> </td>
<td>40,612 </td>
<td>31,203 </td>
<td>28,357 </td>
<td>1,128 </td>
<td>29,888 </td>
</tr>
<tr>
<td>2. <em>Anaeromyxobacter dehalogenans</em> </td>
<td>3,460 </td>
<td>3,458 </td>
<td>2,975 </td>
<td>68 </td>
<td>3,668 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010208" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T2" class="table-wrap-title" data-id="dsy046-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Search for phase shifts in the set RD</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Name of organism </th>
<th rowspan="2">The total number of sequences in the <em>RD</em> set </th>
<th rowspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≠ 0 </th>
<th colspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≥ <em>V</em>0<hr> </th>
<th>Total number of shifts </th>
</tr>
<tr>
<th>Within ±50 </th>
<th>Out ± 50 </th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>1. <em>Arabidopsis thaliana</em> </td>
<td>40,612 </td>
<td>31,203 </td>
<td>28,357 </td>
<td>1,128 </td>
<td>29,888 </td>
</tr>
<tr>
<td>2. <em>Anaeromyxobacter dehalogenans</em> </td>
<td>3,460 </td>
<td>3,458 </td>
<td>2,975 </td>
<td>68 </td>
<td>3,668 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T2" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Name of organism </th>
<th rowspan="2">The total number of sequences in the <em>RD</em> set </th>
<th rowspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≠ 0 </th>
<th colspan="2">Number of sequences which have <em>V</em>2 + <em>V</em>3 ≥ <em>V</em>0<hr> </th>
<th>Total number of shifts </th>
</tr>
<tr>
<th>Within ±50 </th>
<th>Out ± 50 </th>
<th> </th>
</tr>
</thead>
<tbody>
<tr>
<td>1. <em>Arabidopsis thaliana</em> </td>
<td>40,612 </td>
<td>31,203 </td>
<td>28,357 </td>
<td>1,128 </td>
<td>29,888 </td>
</tr>
<tr>
<td>2. <em>Anaeromyxobacter dehalogenans</em> </td>
<td>3,460 </td>
<td>3,458 </td>
<td>2,975 </td>
<td>68 </td>
<td>3,668 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010208" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="134010209" id="134010209" class="section-title">3.2. Searching for potential frameshift mutations in coding sequences from the <em>Arabidopsis thaliana</em> genome and several other eukaryotic genomes</h3>
<p>All coding sequences were analysed from the <em>A. thaliana</em> genome. In the result, we identified 9,930 cds with one or more TP phase shifts, which could indicate the presence of frameshift mutations in these sequences. A total of 14,951 TP phase shifts were found in these cds. This indicates that in many cds we detected multiple TP phase shifts. As the number of false positives within the <em>RN</em> set constitutes 1,549 TP shifts (see Section 3.1), then the first type error rate could be estimated as about ∼11%. As we are dealing with cds derived from mRNA, we also excluded the TP phase shifts found in the cds obtained by the alternative splicing of the same gene. In this case, 6,624 unique cds remain in the <em>A. thaliana</em> genome.</p>
<p>For further analysis, the sequences where the TP phase shifts were found were divided into subsequences in accordance with the TP phase shift coordinates. Therefore, each sequence was split at least into two subsequences. Further, subsequences longer than 60 nt were translated into amino acid sequences in accordance with two frames (except the one, which was already present in the original gene). Two alternative frames must be considered, because we do not know which of the subsequences possesses the correct frame. If we have a single TP phase shift in the <em>x</em> coordinate, the reading frameshift could be registered in the sequence from 1 to <em>x</em> or in the sequence from <em>x</em> to the end of cds. The developed method is incapable of distinguishing between these two cases, and we cannot exclude the possibility that there was a frameshift at the beginning of the gene. As a result, 43,499 sequences longer than 20 amino acids were obtained. Next, for these sequences, the blastP program was employed to search against the Swiss-Prot database (<em>E</em>-value cut-off 0.1). Consequently, a similarity was found for 824 subsequences from the 774 cds. This means that for ∼774 cds, the frameshift was also confirmed by the amino acid sequence similarity.</p>
<p>Let us consider an example of a cds with a TP phase shift from the <em>A. thaliana</em> genome, for which a similarity was found using an alternative reading frame. The sequence identifier is AT1G79920.2, and the corresponding amino acid sequence identifier in the Swiss-Prot is F4HQD5_ARATH. A TP phase shift was found at the 1933 position, and thereafter, the frame changes from the first to the third. <a reveal-id="dsy046-T3" data-open="dsy046-T3" class="link link-reveal link-table xref-fig">Table 3</a> presents the resulting <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mn>3</mn><mo>,</mo><mn>16</mn><mo stretchy="false">)</mo></mrow></math></span> matrix. For the third reading frame, an amino acid sequence was also obtained. <a reveal-id="dsy046-T4" data-open="dsy046-T4" class="link link-reveal link-table xref-fig">Table 4</a> shows that the F4HQD5_ARATH sequence possesses a similarity to the HS105_CRIGR sequence from the <em>Cricetulus griseus</em> (Chinese hamster) genome only after 642 amino acids, which corresponds to the coordinate of the discovered frameshift. The <em>E</em>-value for the similarity found constitutes 4.6e<sup>−144</sup>. Beyond the 656 amino acids of the HS105_CRIGR sequence, its similarity was observed with the amino acid sequence created by the third reading frame, after the 1933 position from the AT1G79920.2 cds (<a reveal-id="dsy046-T5" data-open="dsy046-T5" class="link link-reveal link-table xref-fig">Table 5</a>). The <em>E</em>-value for this similarity constitutes 4.6e<sup>−19</sup>. This example obviously demonstrates that simultaneously there are two similar proteins, one of which possesses a reading frameshift, while the other does not. The first one is the heat shock protein 70 (F4HQD5_ARATH) from the <em>A. thaliana</em> genome, while the second one is the heat shock protein 105 kDa (HS105_CRIGR) from the <em>Cricetulus griseus</em> (Chinese hamster) genome.</p>                        <a id="134010213" scrollto-destination="134010213"></a>
<div content-id="dsy046-T3" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T3" class="table-wrap-title" data-id="dsy046-T3">
<div class="label">Table 3</div>
<div class="caption"><p>The matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>n</mi><mo stretchy="false">)</mo></mrow></math></span> (see Equation 2) that was used for the construction of the final global alignment of cds AT1G79920.2 (point 2.4)</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th> </th>
<th>A </th>
<th>T </th>
<th>C </th>
<th>G </th>
</tr></thead>
<tbody>
<tr>
<td>1 </td>
<td>A </td>
<td>−1.4 </td>
<td>1.6 </td>
<td>0.3 </td>
<td>5.3 </td>
</tr>
<tr>
<td>1 </td>
<td>T </td>
<td>−7.9 </td>
<td>3.5 </td>
<td>2.4 </td>
<td>1.7 </td>
</tr>
<tr>
<td>1 </td>
<td>C </td>
<td>−0.8 </td>
<td>7.1 </td>
<td>−0.5 </td>
<td>−3.4 </td>
</tr>
<tr>
<td>1 </td>
<td>G </td>
<td>−6.3 </td>
<td>−2.0 </td>
<td>−1.6 </td>
<td>−4.5 </td>
</tr>
<tr>
<td>2 </td>
<td>A </td>
<td>−3.5 </td>
<td>−4.6 </td>
<td>−4.7 </td>
<td>0.9 </td>
</tr>
<tr>
<td>2 </td>
<td>T </td>
<td>−2.4 </td>
<td>−1.9 </td>
<td>0.7 </td>
<td>11.3 </td>
</tr>
<tr>
<td>2 </td>
<td>C </td>
<td>1.3 </td>
<td>−0.3 </td>
<td>−0.3 </td>
<td>−1.4 </td>
</tr>
<tr>
<td>2 </td>
<td>G </td>
<td>0.1 </td>
<td>−3.7 </td>
<td>−0.3 </td>
<td>1.8 </td>
</tr>
<tr>
<td>3 </td>
<td>A </td>
<td>2.7 </td>
<td>0.4 </td>
<td>−0.3 </td>
<td>−2.9 </td>
</tr>
<tr>
<td>3 </td>
<td>T </td>
<td>−6.8 </td>
<td>0.2 </td>
<td>−0.3 </td>
<td>−5.4 </td>
</tr>
<tr>
<td>3 </td>
<td>C </td>
<td>−1.7 </td>
<td>−1.1 </td>
<td>0.4 </td>
<td>−3.7 </td>
</tr>
<tr>
<td>3 </td>
<td>G </td>
<td>6.4 </td>
<td>1.5 </td>
<td>4.3 </td>
<td>−0.9 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th> </th>
<th>A </th>
<th>T </th>
<th>C </th>
<th>G </th>
</tr></thead>
<tbody>
<tr>
<td>1 </td>
<td>A </td>
<td>−1.4 </td>
<td>1.6 </td>
<td>0.3 </td>
<td>5.3 </td>
</tr>
<tr>
<td>1 </td>
<td>T </td>
<td>−7.9 </td>
<td>3.5 </td>
<td>2.4 </td>
<td>1.7 </td>
</tr>
<tr>
<td>1 </td>
<td>C </td>
<td>−0.8 </td>
<td>7.1 </td>
<td>−0.5 </td>
<td>−3.4 </td>
</tr>
<tr>
<td>1 </td>
<td>G </td>
<td>−6.3 </td>
<td>−2.0 </td>
<td>−1.6 </td>
<td>−4.5 </td>
</tr>
<tr>
<td>2 </td>
<td>A </td>
<td>−3.5 </td>
<td>−4.6 </td>
<td>−4.7 </td>
<td>0.9 </td>
</tr>
<tr>
<td>2 </td>
<td>T </td>
<td>−2.4 </td>
<td>−1.9 </td>
<td>0.7 </td>
<td>11.3 </td>
</tr>
<tr>
<td>2 </td>
<td>C </td>
<td>1.3 </td>
<td>−0.3 </td>
<td>−0.3 </td>
<td>−1.4 </td>
</tr>
<tr>
<td>2 </td>
<td>G </td>
<td>0.1 </td>
<td>−3.7 </td>
<td>−0.3 </td>
<td>1.8 </td>
</tr>
<tr>
<td>3 </td>
<td>A </td>
<td>2.7 </td>
<td>0.4 </td>
<td>−0.3 </td>
<td>−2.9 </td>
</tr>
<tr>
<td>3 </td>
<td>T </td>
<td>−6.8 </td>
<td>0.2 </td>
<td>−0.3 </td>
<td>−5.4 </td>
</tr>
<tr>
<td>3 </td>
<td>C </td>
<td>−1.7 </td>
<td>−1.1 </td>
<td>0.4 </td>
<td>−3.7 </td>
</tr>
<tr>
<td>3 </td>
<td>G </td>
<td>6.4 </td>
<td>1.5 </td>
<td>4.3 </td>
<td>−0.9 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>n </em>=<em> s</em>(<em>k</em>)+4(<em>s</em>(<em>i</em>) − 1), and <em>a</em>(<em>i</em>) was calculated as <em>i</em> – 3int(<em>i</em>/3) for <em>i </em>=<em> </em>1,…, <em>N.</em> The index <em>k</em> is calculated using the already created transitions in the matrix <em>F</em> (see the text under Equation 2). Columns 3 through 6 correspond to the bases <em>s</em>(<em>i</em>), the second column corresponds to the bases <em>s</em>(<em>k</em>), and the first column shows the positions <em>a</em>(<em>i</em>).</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010213" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T3" class="table-wrap-title" data-id="dsy046-T3">
<div class="label">Table 3</div>
<div class="caption"><p>The matrix <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mrow><mi>W</mi></mrow><mn>2</mn><mi mathvariant="normal">t</mi></msubsup><mo stretchy="false">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>n</mi><mo stretchy="false">)</mo></mrow></math></span> (see Equation 2) that was used for the construction of the final global alignment of cds AT1G79920.2 (point 2.4)</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th> </th>
<th>A </th>
<th>T </th>
<th>C </th>
<th>G </th>
</tr></thead>
<tbody>
<tr>
<td>1 </td>
<td>A </td>
<td>−1.4 </td>
<td>1.6 </td>
<td>0.3 </td>
<td>5.3 </td>
</tr>
<tr>
<td>1 </td>
<td>T </td>
<td>−7.9 </td>
<td>3.5 </td>
<td>2.4 </td>
<td>1.7 </td>
</tr>
<tr>
<td>1 </td>
<td>C </td>
<td>−0.8 </td>
<td>7.1 </td>
<td>−0.5 </td>
<td>−3.4 </td>
</tr>
<tr>
<td>1 </td>
<td>G </td>
<td>−6.3 </td>
<td>−2.0 </td>
<td>−1.6 </td>
<td>−4.5 </td>
</tr>
<tr>
<td>2 </td>
<td>A </td>
<td>−3.5 </td>
<td>−4.6 </td>
<td>−4.7 </td>
<td>0.9 </td>
</tr>
<tr>
<td>2 </td>
<td>T </td>
<td>−2.4 </td>
<td>−1.9 </td>
<td>0.7 </td>
<td>11.3 </td>
</tr>
<tr>
<td>2 </td>
<td>C </td>
<td>1.3 </td>
<td>−0.3 </td>
<td>−0.3 </td>
<td>−1.4 </td>
</tr>
<tr>
<td>2 </td>
<td>G </td>
<td>0.1 </td>
<td>−3.7 </td>
<td>−0.3 </td>
<td>1.8 </td>
</tr>
<tr>
<td>3 </td>
<td>A </td>
<td>2.7 </td>
<td>0.4 </td>
<td>−0.3 </td>
<td>−2.9 </td>
</tr>
<tr>
<td>3 </td>
<td>T </td>
<td>−6.8 </td>
<td>0.2 </td>
<td>−0.3 </td>
<td>−5.4 </td>
</tr>
<tr>
<td>3 </td>
<td>C </td>
<td>−1.7 </td>
<td>−1.1 </td>
<td>0.4 </td>
<td>−3.7 </td>
</tr>
<tr>
<td>3 </td>
<td>G </td>
<td>6.4 </td>
<td>1.5 </td>
<td>4.3 </td>
<td>−0.9 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th> </th>
<th> </th>
<th>A </th>
<th>T </th>
<th>C </th>
<th>G </th>
</tr></thead>
<tbody>
<tr>
<td>1 </td>
<td>A </td>
<td>−1.4 </td>
<td>1.6 </td>
<td>0.3 </td>
<td>5.3 </td>
</tr>
<tr>
<td>1 </td>
<td>T </td>
<td>−7.9 </td>
<td>3.5 </td>
<td>2.4 </td>
<td>1.7 </td>
</tr>
<tr>
<td>1 </td>
<td>C </td>
<td>−0.8 </td>
<td>7.1 </td>
<td>−0.5 </td>
<td>−3.4 </td>
</tr>
<tr>
<td>1 </td>
<td>G </td>
<td>−6.3 </td>
<td>−2.0 </td>
<td>−1.6 </td>
<td>−4.5 </td>
</tr>
<tr>
<td>2 </td>
<td>A </td>
<td>−3.5 </td>
<td>−4.6 </td>
<td>−4.7 </td>
<td>0.9 </td>
</tr>
<tr>
<td>2 </td>
<td>T </td>
<td>−2.4 </td>
<td>−1.9 </td>
<td>0.7 </td>
<td>11.3 </td>
</tr>
<tr>
<td>2 </td>
<td>C </td>
<td>1.3 </td>
<td>−0.3 </td>
<td>−0.3 </td>
<td>−1.4 </td>
</tr>
<tr>
<td>2 </td>
<td>G </td>
<td>0.1 </td>
<td>−3.7 </td>
<td>−0.3 </td>
<td>1.8 </td>
</tr>
<tr>
<td>3 </td>
<td>A </td>
<td>2.7 </td>
<td>0.4 </td>
<td>−0.3 </td>
<td>−2.9 </td>
</tr>
<tr>
<td>3 </td>
<td>T </td>
<td>−6.8 </td>
<td>0.2 </td>
<td>−0.3 </td>
<td>−5.4 </td>
</tr>
<tr>
<td>3 </td>
<td>C </td>
<td>−1.7 </td>
<td>−1.1 </td>
<td>0.4 </td>
<td>−3.7 </td>
</tr>
<tr>
<td>3 </td>
<td>G </td>
<td>6.4 </td>
<td>1.5 </td>
<td>4.3 </td>
<td>−0.9 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>n </em>=<em> s</em>(<em>k</em>)+4(<em>s</em>(<em>i</em>) − 1), and <em>a</em>(<em>i</em>) was calculated as <em>i</em> – 3int(<em>i</em>/3) for <em>i </em>=<em> </em>1,…, <em>N.</em> The index <em>k</em> is calculated using the already created transitions in the matrix <em>F</em> (see the text under Equation 2). Columns 3 through 6 correspond to the bases <em>s</em>(<em>i</em>), the second column corresponds to the bases <em>s</em>(<em>k</em>), and the first column shows the positions <em>a</em>(<em>i</em>).</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010213" target="_blank">Open in new tab</a></div>
</div></div>                        <a id="134010214" scrollto-destination="134010214"></a>
<div content-id="dsy046-T4" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T4" class="table-wrap-title" data-id="dsy046-T4">
<div class="label">Table 4</div>
<div class="caption"><p>Alignment of the amino acid sequence F4HQD5_ARATH, which is encoded by cds AT1G79920.2 from the <em>Arabidopsis thaliana</em> genome with the amino acid sequence HS105_CRIGR from the genome <em>Cricetulus griseus</em> (Chinese hamster) </p></div> </div>
<div class="table-overflow"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf1.png?Expires=1576262930&amp;Signature=N6CVZ9Y4zOSfYnnDtzoH3ckA6FLR5ux6f0BJPuL5m8jJ2PxiIEykf2-AcrX-KCIjLTEiIIZiZ7qA-rPYxQnBeuo~Zppw7knA3jRuWZ4YKD70JLihxIbpzIvxPCljazMhCDRPAJTxrEpvuGENZ8DJymXcWJArl-YNBV1oZxk6GlsGZ9-YDUcpUCfbVeo4ySLumG8KZbC9NRFtSH8Q3fKfuWZzjEEzduvfBLNI4Vo8S4dR7Y5CmrHYtJP-oogEvgsSwnXPVsWutybJ6V56Gj1H7rsQhbMa1BqyLalpty07OtFks3-pqFsfkazLVIsR-~iEiCXPW1iw~KyhUY64XPOCsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf1.tif"> </td></tr></tbody></table></div>
<div content-id="dsy046-T4" class="table-modal"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf1.png?Expires=1576262930&amp;Signature=N6CVZ9Y4zOSfYnnDtzoH3ckA6FLR5ux6f0BJPuL5m8jJ2PxiIEykf2-AcrX-KCIjLTEiIIZiZ7qA-rPYxQnBeuo~Zppw7knA3jRuWZ4YKD70JLihxIbpzIvxPCljazMhCDRPAJTxrEpvuGENZ8DJymXcWJArl-YNBV1oZxk6GlsGZ9-YDUcpUCfbVeo4ySLumG8KZbC9NRFtSH8Q3fKfuWZzjEEzduvfBLNI4Vo8S4dR7Y5CmrHYtJP-oogEvgsSwnXPVsWutybJ6V56Gj1H7rsQhbMa1BqyLalpty07OtFks3-pqFsfkazLVIsR-~iEiCXPW1iw~KyhUY64XPOCsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf1.tif"> </td></tr></tbody></table></div>
<div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>This alignment can be found from 1 to 642 amino acids for F4HQD5_ARATH and from 1 to 655 amino acids for HS105_CRIGR.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010214" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T4" class="table-wrap-title" data-id="dsy046-T4">
<div class="label">Table 4</div>
<div class="caption"><p>Alignment of the amino acid sequence F4HQD5_ARATH, which is encoded by cds AT1G79920.2 from the <em>Arabidopsis thaliana</em> genome with the amino acid sequence HS105_CRIGR from the genome <em>Cricetulus griseus</em> (Chinese hamster) </p></div> </div>
<div class="table-overflow"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf1.png?Expires=1576262930&amp;Signature=N6CVZ9Y4zOSfYnnDtzoH3ckA6FLR5ux6f0BJPuL5m8jJ2PxiIEykf2-AcrX-KCIjLTEiIIZiZ7qA-rPYxQnBeuo~Zppw7knA3jRuWZ4YKD70JLihxIbpzIvxPCljazMhCDRPAJTxrEpvuGENZ8DJymXcWJArl-YNBV1oZxk6GlsGZ9-YDUcpUCfbVeo4ySLumG8KZbC9NRFtSH8Q3fKfuWZzjEEzduvfBLNI4Vo8S4dR7Y5CmrHYtJP-oogEvgsSwnXPVsWutybJ6V56Gj1H7rsQhbMa1BqyLalpty07OtFks3-pqFsfkazLVIsR-~iEiCXPW1iw~KyhUY64XPOCsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf1.tif"> </td></tr></tbody></table></div>
<div content-id="dsy046-T4" class="table-modal"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf1.png?Expires=1576262930&amp;Signature=N6CVZ9Y4zOSfYnnDtzoH3ckA6FLR5ux6f0BJPuL5m8jJ2PxiIEykf2-AcrX-KCIjLTEiIIZiZ7qA-rPYxQnBeuo~Zppw7knA3jRuWZ4YKD70JLihxIbpzIvxPCljazMhCDRPAJTxrEpvuGENZ8DJymXcWJArl-YNBV1oZxk6GlsGZ9-YDUcpUCfbVeo4ySLumG8KZbC9NRFtSH8Q3fKfuWZzjEEzduvfBLNI4Vo8S4dR7Y5CmrHYtJP-oogEvgsSwnXPVsWutybJ6V56Gj1H7rsQhbMa1BqyLalpty07OtFks3-pqFsfkazLVIsR-~iEiCXPW1iw~KyhUY64XPOCsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf1.tif"> </td></tr></tbody></table></div>
<div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>This alignment can be found from 1 to 642 amino acids for F4HQD5_ARATH and from 1 to 655 amino acids for HS105_CRIGR.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010214" target="_blank">Open in new tab</a></div>
</div></div>                        <a id="134010215" scrollto-destination="134010215"></a>
<div content-id="dsy046-T5" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T5" class="table-wrap-title" data-id="dsy046-T5">
<div class="label">Table 5</div>
<div class="caption"><p>Alignment of the amino acid sequence obtained by the third reading frame of the cds AT1G79920.2 from the position 1933 to the end of the sequence from the <em>Arabidopsis thaliana</em> genome, with the amino acid sequence HS105_CRIGR from the genome <em>Cricetulus griseus</em> (Chinese hamster)</p></div> </div>
<div class="table-overflow"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf2.png?Expires=1576262930&amp;Signature=e6SZtej0eKAi2L1okg8TfTu7LBcgEdUB056sFZaKPOdVB2jDpSmBiHDWJ6j7H9BlO6xK4-M6-rZ2F6h7-bXb1f-FRtkDvzoG3BIgBCQar1eyWaqQhGaH9lj1OrNfCeHYoo1BfkCEBQRJrwZ7WO-wd5~jsG0Hq~3YsYxXVSWHSno17iDgZw-2Y1GC7Z4CsrN8B~ja7QanTWLKruS-TNHkJ2HsQh4UEv4PqsFHL7RcTTTJt6NchlaN6Z-jBf8rSypDqjyLmUL4K~p0AZVoQ0GPmTN3~QpXuXN~JnilD2X382-Q~qaAPnSPmxjlagp2wU2pI7rtYbBelNA~KL42neCVsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf2.tif"> </td></tr></tbody></table></div>
<div content-id="dsy046-T5" class="table-modal"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf2.png?Expires=1576262930&amp;Signature=e6SZtej0eKAi2L1okg8TfTu7LBcgEdUB056sFZaKPOdVB2jDpSmBiHDWJ6j7H9BlO6xK4-M6-rZ2F6h7-bXb1f-FRtkDvzoG3BIgBCQar1eyWaqQhGaH9lj1OrNfCeHYoo1BfkCEBQRJrwZ7WO-wd5~jsG0Hq~3YsYxXVSWHSno17iDgZw-2Y1GC7Z4CsrN8B~ja7QanTWLKruS-TNHkJ2HsQh4UEv4PqsFHL7RcTTTJt6NchlaN6Z-jBf8rSypDqjyLmUL4K~p0AZVoQ0GPmTN3~QpXuXN~JnilD2X382-Q~qaAPnSPmxjlagp2wU2pI7rtYbBelNA~KL42neCVsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf2.tif"> </td></tr></tbody></table></div>
<div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>This alignment was found from 656 to 752 amino acids for the sequence HS105_CRIGR.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010215" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T5" class="table-wrap-title" data-id="dsy046-T5">
<div class="label">Table 5</div>
<div class="caption"><p>Alignment of the amino acid sequence obtained by the third reading frame of the cds AT1G79920.2 from the position 1933 to the end of the sequence from the <em>Arabidopsis thaliana</em> genome, with the amino acid sequence HS105_CRIGR from the genome <em>Cricetulus griseus</em> (Chinese hamster)</p></div> </div>
<div class="table-overflow"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf2.png?Expires=1576262930&amp;Signature=e6SZtej0eKAi2L1okg8TfTu7LBcgEdUB056sFZaKPOdVB2jDpSmBiHDWJ6j7H9BlO6xK4-M6-rZ2F6h7-bXb1f-FRtkDvzoG3BIgBCQar1eyWaqQhGaH9lj1OrNfCeHYoo1BfkCEBQRJrwZ7WO-wd5~jsG0Hq~3YsYxXVSWHSno17iDgZw-2Y1GC7Z4CsrN8B~ja7QanTWLKruS-TNHkJ2HsQh4UEv4PqsFHL7RcTTTJt6NchlaN6Z-jBf8rSypDqjyLmUL4K~p0AZVoQ0GPmTN3~QpXuXN~JnilD2X382-Q~qaAPnSPmxjlagp2wU2pI7rtYbBelNA~KL42neCVsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf2.tif"> </td></tr></tbody></table></div>
<div content-id="dsy046-T5" class="table-modal"><table><tbody><tr><td>
<img class="contentFigures" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046ilf2.png?Expires=1576262930&amp;Signature=e6SZtej0eKAi2L1okg8TfTu7LBcgEdUB056sFZaKPOdVB2jDpSmBiHDWJ6j7H9BlO6xK4-M6-rZ2F6h7-bXb1f-FRtkDvzoG3BIgBCQar1eyWaqQhGaH9lj1OrNfCeHYoo1BfkCEBQRJrwZ7WO-wd5~jsG0Hq~3YsYxXVSWHSno17iDgZw-2Y1GC7Z4CsrN8B~ja7QanTWLKruS-TNHkJ2HsQh4UEv4PqsFHL7RcTTTJt6NchlaN6Z-jBf8rSypDqjyLmUL4K~p0AZVoQ0GPmTN3~QpXuXN~JnilD2X382-Q~qaAPnSPmxjlagp2wU2pI7rtYbBelNA~KL42neCVsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsy046ilf2.tif"> </td></tr></tbody></table></div>
<div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>This alignment was found from 656 to 752 amino acids for the sequence HS105_CRIGR.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010215" target="_blank">Open in new tab</a></div>
</div></div>
<p>In addition to the <em>A. thaliana</em> genome, we applied our method to cds from five eukaryotic genomes. The cds were also obtained from the Ensembl database (<a class="link link-uri" href="http://ftp://ftp.ensembl.org/pub/release-91/fasta/" target="_blank">ftp://ftp.ensembl.org/pub/release-91/fasta/</a>). These genomes include those of <em>C. elegans, D. melanogaster, H. sapiens, R. norvegicus</em> and <em>X. tropicalis</em>. <a reveal-id="dsy046-T6" data-open="dsy046-T6" class="link link-reveal link-table xref-fig">Table 6</a> presents data on the number of potential reading frameshifts. The level of errors of the first and second type corresponds to those estimated for the <em>A. thaliana</em> genome, with accuracy of ±5%. From <a reveal-id="dsy046-T6" data-open="dsy046-T6" class="link link-reveal link-table xref-fig">Table 6</a>, it can be seen that these genomes contain on average from ∼1.5 to 3 frameshifts in one <em>cds.</em></p>                        <a id="134010217" scrollto-destination="134010217"></a>
<div content-id="dsy046-T6" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T6" class="table-wrap-title" data-id="dsy046-T6">
<div class="label">Table 6</div>
<div class="caption"><p>The number of cds with potential frameshift mutations in the six eukaryotic genomes examined, disregarding and taking into account alternative splicing</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Organism name </th>
<th>Number of potential frameshift mutations </th>
<th>Number of cds with the potential frameshift mutations </th>
<th>Number of cds with potential frameshift mutations take into account an alternative splicing </th>
<th>Number of cds with potential frameshift mutations from the work<sup>21</sup> </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>Arabidopsis thaliana</em> </td>
<td>14,954 </td>
<td>9,930 </td>
<td>6,624 </td>
<td>2,067 </td>
</tr>
<tr>
<td>Caenorhabditis <em>elegans</em> </td>
<td>11,103 </td>
<td>6,370 </td>
<td>3,788 </td>
<td>611 </td>
</tr>
<tr>
<td>Drosophila <em>melanogaster</em> </td>
<td>31,873 </td>
<td>8,833 </td>
<td>3,649 </td>
<td>2,616 </td>
</tr>
<tr>
<td>
<em>Homo sapiens</em> </td>
<td>33,336 </td>
<td>21,363 </td>
<td>9,456 </td>
<td>7,395 </td>
</tr>
<tr>
<td>Rattus <em>norvegicus</em> </td>
<td>9,752 </td>
<td>5,689 </td>
<td>4,608 </td>
<td>703 </td>
</tr>
<tr>
<td>
<em>Xenopus tropicalis</em> </td>
<td>6,348 </td>
<td>4,014 </td>
<td>3,401 </td>
<td>529 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T6" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Organism name </th>
<th>Number of potential frameshift mutations </th>
<th>Number of cds with the potential frameshift mutations </th>
<th>Number of cds with potential frameshift mutations take into account an alternative splicing </th>
<th>Number of cds with potential frameshift mutations from the work<sup>21</sup> </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>Arabidopsis thaliana</em> </td>
<td>14,954 </td>
<td>9,930 </td>
<td>6,624 </td>
<td>2,067 </td>
</tr>
<tr>
<td>Caenorhabditis <em>elegans</em> </td>
<td>11,103 </td>
<td>6,370 </td>
<td>3,788 </td>
<td>611 </td>
</tr>
<tr>
<td>Drosophila <em>melanogaster</em> </td>
<td>31,873 </td>
<td>8,833 </td>
<td>3,649 </td>
<td>2,616 </td>
</tr>
<tr>
<td>
<em>Homo sapiens</em> </td>
<td>33,336 </td>
<td>21,363 </td>
<td>9,456 </td>
<td>7,395 </td>
</tr>
<tr>
<td>Rattus <em>norvegicus</em> </td>
<td>9,752 </td>
<td>5,689 </td>
<td>4,608 </td>
<td>703 </td>
</tr>
<tr>
<td>
<em>Xenopus tropicalis</em> </td>
<td>6,348 </td>
<td>4,014 </td>
<td>3,401 </td>
<td>529 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>The data obtained in the work<a reveal-id="dsy046-B37" data-open="dsy046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> are also shown.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010217" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T6" class="table-wrap-title" data-id="dsy046-T6">
<div class="label">Table 6</div>
<div class="caption"><p>The number of cds with potential frameshift mutations in the six eukaryotic genomes examined, disregarding and taking into account alternative splicing</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Organism name </th>
<th>Number of potential frameshift mutations </th>
<th>Number of cds with the potential frameshift mutations </th>
<th>Number of cds with potential frameshift mutations take into account an alternative splicing </th>
<th>Number of cds with potential frameshift mutations from the work<sup>21</sup> </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>Arabidopsis thaliana</em> </td>
<td>14,954 </td>
<td>9,930 </td>
<td>6,624 </td>
<td>2,067 </td>
</tr>
<tr>
<td>Caenorhabditis <em>elegans</em> </td>
<td>11,103 </td>
<td>6,370 </td>
<td>3,788 </td>
<td>611 </td>
</tr>
<tr>
<td>Drosophila <em>melanogaster</em> </td>
<td>31,873 </td>
<td>8,833 </td>
<td>3,649 </td>
<td>2,616 </td>
</tr>
<tr>
<td>
<em>Homo sapiens</em> </td>
<td>33,336 </td>
<td>21,363 </td>
<td>9,456 </td>
<td>7,395 </td>
</tr>
<tr>
<td>Rattus <em>norvegicus</em> </td>
<td>9,752 </td>
<td>5,689 </td>
<td>4,608 </td>
<td>703 </td>
</tr>
<tr>
<td>
<em>Xenopus tropicalis</em> </td>
<td>6,348 </td>
<td>4,014 </td>
<td>3,401 </td>
<td>529 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T6" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Organism name </th>
<th>Number of potential frameshift mutations </th>
<th>Number of cds with the potential frameshift mutations </th>
<th>Number of cds with potential frameshift mutations take into account an alternative splicing </th>
<th>Number of cds with potential frameshift mutations from the work<sup>21</sup> </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>Arabidopsis thaliana</em> </td>
<td>14,954 </td>
<td>9,930 </td>
<td>6,624 </td>
<td>2,067 </td>
</tr>
<tr>
<td>Caenorhabditis <em>elegans</em> </td>
<td>11,103 </td>
<td>6,370 </td>
<td>3,788 </td>
<td>611 </td>
</tr>
<tr>
<td>Drosophila <em>melanogaster</em> </td>
<td>31,873 </td>
<td>8,833 </td>
<td>3,649 </td>
<td>2,616 </td>
</tr>
<tr>
<td>
<em>Homo sapiens</em> </td>
<td>33,336 </td>
<td>21,363 </td>
<td>9,456 </td>
<td>7,395 </td>
</tr>
<tr>
<td>Rattus <em>norvegicus</em> </td>
<td>9,752 </td>
<td>5,689 </td>
<td>4,608 </td>
<td>703 </td>
</tr>
<tr>
<td>
<em>Xenopus tropicalis</em> </td>
<td>6,348 </td>
<td>4,014 </td>
<td>3,401 </td>
<td>529 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>The data obtained in the work<a reveal-id="dsy046-B37" data-open="dsy046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> are also shown.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010217" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="134010218" id="134010218" class="section-title">3.3. Searching for potential frameshift mutations in coding sequences from prokaryotic genomes</h3>
<p>We also studied the presence of TP phase shifts in cds from 17 bacterial genomes. For example, using the <em>A. deehalogenans</em> genome, we studied the <em>RN</em> and <em>RD</em> sets (see Section 3.1) created from the cds set of this genome. The results of examining the <em>RN</em> set (codon-shuffled sequences) demonstrate that only 61 TP shifts could be identified. Thus, the number of errors of the first type (false positives) for this genome is about 8% of the number of cds found with the TP phase shift (<a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 3). In the remaining 16 genomes, fluctuations in first type error rates ranged from 6% to 14%.</p>                        <a id="134010220" scrollto-destination="134010220"></a>
<div content-id="dsy046-T7" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T7" class="table-wrap-title" data-id="dsy046-T7">
<div class="label">Table 7</div>
<div class="caption"><p>The number of potential frameshifts in cds from the 17 bacterial genomes (Column 2) obtained in the present study</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th colspan="2">This work<hr> </th>
<th colspan="4">GeneTack-GM program<hr> </th>
</tr>
<tr>
<th>1. Name of bacteria </th>
<th>2. Number of cds with frameshifts </th>
<th>3. Number of frameshifts </th>
<th>4. Inside the gene, no closer than 50 nt to the border </th>
<th>5. The beginning or end of the gene is not more than 50 nt (the number of cases with the addition of a neighbouring gene) </th>
<th>6. Between genes </th>
<th>7. Total </th>
</tr>
</thead>
<tbody>
<tr>
<td>
<em>Anaeromyxobacter dehalogenans</em> </td>
<td>425 </td>
<td>768 </td>
<td>49 </td>
<td>101 (62) </td>
<td>32 </td>
<td>182 </td>
</tr>
<tr>
<td>
<em>Archaeoglobus fulgidus</em> </td>
<td>77 </td>
<td>174 </td>
<td>44 </td>
<td>299 (251) </td>
<td>53 </td>
<td>396 </td>
</tr>
<tr>
<td>
<em>Bacillus subtilis</em> </td>
<td>126 </td>
<td>434 </td>
<td>35 </td>
<td>79 (57) </td>
<td>27 </td>
<td>141 </td>
</tr>
<tr>
<td>
<em>Campylobacter jejuni</em> </td>
<td>48 </td>
<td>86 </td>
<td>8 </td>
<td>104 (92) </td>
<td>16 </td>
<td>128 </td>
</tr>
<tr>
<td>
<em>Caulobacter crescentus</em> </td>
<td>306 </td>
<td>651 </td>
<td>43 </td>
<td>57 (30) </td>
<td>3 </td>
<td>103 </td>
</tr>
<tr>
<td>
<em>Clavibacter michiganensis</em> </td>
<td>357 </td>
<td>736 </td>
<td>35 </td>
<td>65 (40) </td>
<td>63 </td>
<td>163 </td>
</tr>
<tr>
<td>
<em>Methanopyrus kandleri</em> </td>
<td>150 </td>
<td>436 </td>
<td>76 </td>
<td>96 (68) </td>
<td>38 </td>
<td>210 </td>
</tr>
<tr>
<td>
<em>Methanosphaera stadtmanae</em> </td>
<td>111 </td>
<td>272 </td>
<td>5 </td>
<td>37 (27) </td>
<td>13 </td>
<td>55 </td>
</tr>
<tr>
<td>
<em>Pasteurella multocida</em> </td>
<td>46 </td>
<td>146 </td>
<td>9 </td>
<td>56 (48) </td>
<td>4 </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Picrophilus torridus</em> </td>
<td>15 </td>
<td>62 </td>
<td>7 </td>
<td>154 (130) </td>
<td>9 </td>
<td>170 </td>
</tr>
<tr>
<td>
<em>Pyrobaculum aerophilum</em> </td>
<td>71 </td>
<td>249 </td>
<td>65 </td>
<td>235 (163) </td>
<td>48 </td>
<td>348 </td>
</tr>
<tr>
<td>
<em>Ralstonia solanacearum</em> </td>
<td>330 </td>
<td>607 </td>
<td>38 </td>
<td>66 (41) </td>
<td>19 </td>
<td>123 </td>
</tr>
<tr>
<td>
<em>Salmonella enterica,</em> </td>
<td>115 </td>
<td>457 </td>
<td>52 </td>
<td>162 (112) </td>
<td>38 </td>
<td>252 </td>
</tr>
<tr>
<td>
<em>Staphylococcus aureus</em> </td>
<td>130 </td>
<td>381 </td>
<td>4 </td>
<td>52 (43) </td>
<td>16 </td>
<td>72 </td>
</tr>
<tr>
<td>
<em>Streptococcus pyogenes</em> </td>
<td>60 </td>
<td>209 </td>
<td>17 </td>
<td>52 (39) </td>
<td>6 </td>
<td>75 </td>
</tr>
<tr>
<td>
<em>Thermococcus kodakarensis</em> </td>
<td>85 </td>
<td>172 </td>
<td>43 </td>
<td>57 (38) </td>
<td>4 </td>
<td>104 </td>
</tr>
<tr>
<td>
<em>Thermotoga maritima</em> </td>
<td>37 </td>
<td>127 </td>
<td>10 </td>
<td>76 (64) </td>
<td>5 </td>
<td>91 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T7" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th colspan="2">This work<hr> </th>
<th colspan="4">GeneTack-GM program<hr> </th>
</tr>
<tr>
<th>1. Name of bacteria </th>
<th>2. Number of cds with frameshifts </th>
<th>3. Number of frameshifts </th>
<th>4. Inside the gene, no closer than 50 nt to the border </th>
<th>5. The beginning or end of the gene is not more than 50 nt (the number of cases with the addition of a neighbouring gene) </th>
<th>6. Between genes </th>
<th>7. Total </th>
</tr>
</thead>
<tbody>
<tr>
<td>
<em>Anaeromyxobacter dehalogenans</em> </td>
<td>425 </td>
<td>768 </td>
<td>49 </td>
<td>101 (62) </td>
<td>32 </td>
<td>182 </td>
</tr>
<tr>
<td>
<em>Archaeoglobus fulgidus</em> </td>
<td>77 </td>
<td>174 </td>
<td>44 </td>
<td>299 (251) </td>
<td>53 </td>
<td>396 </td>
</tr>
<tr>
<td>
<em>Bacillus subtilis</em> </td>
<td>126 </td>
<td>434 </td>
<td>35 </td>
<td>79 (57) </td>
<td>27 </td>
<td>141 </td>
</tr>
<tr>
<td>
<em>Campylobacter jejuni</em> </td>
<td>48 </td>
<td>86 </td>
<td>8 </td>
<td>104 (92) </td>
<td>16 </td>
<td>128 </td>
</tr>
<tr>
<td>
<em>Caulobacter crescentus</em> </td>
<td>306 </td>
<td>651 </td>
<td>43 </td>
<td>57 (30) </td>
<td>3 </td>
<td>103 </td>
</tr>
<tr>
<td>
<em>Clavibacter michiganensis</em> </td>
<td>357 </td>
<td>736 </td>
<td>35 </td>
<td>65 (40) </td>
<td>63 </td>
<td>163 </td>
</tr>
<tr>
<td>
<em>Methanopyrus kandleri</em> </td>
<td>150 </td>
<td>436 </td>
<td>76 </td>
<td>96 (68) </td>
<td>38 </td>
<td>210 </td>
</tr>
<tr>
<td>
<em>Methanosphaera stadtmanae</em> </td>
<td>111 </td>
<td>272 </td>
<td>5 </td>
<td>37 (27) </td>
<td>13 </td>
<td>55 </td>
</tr>
<tr>
<td>
<em>Pasteurella multocida</em> </td>
<td>46 </td>
<td>146 </td>
<td>9 </td>
<td>56 (48) </td>
<td>4 </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Picrophilus torridus</em> </td>
<td>15 </td>
<td>62 </td>
<td>7 </td>
<td>154 (130) </td>
<td>9 </td>
<td>170 </td>
</tr>
<tr>
<td>
<em>Pyrobaculum aerophilum</em> </td>
<td>71 </td>
<td>249 </td>
<td>65 </td>
<td>235 (163) </td>
<td>48 </td>
<td>348 </td>
</tr>
<tr>
<td>
<em>Ralstonia solanacearum</em> </td>
<td>330 </td>
<td>607 </td>
<td>38 </td>
<td>66 (41) </td>
<td>19 </td>
<td>123 </td>
</tr>
<tr>
<td>
<em>Salmonella enterica,</em> </td>
<td>115 </td>
<td>457 </td>
<td>52 </td>
<td>162 (112) </td>
<td>38 </td>
<td>252 </td>
</tr>
<tr>
<td>
<em>Staphylococcus aureus</em> </td>
<td>130 </td>
<td>381 </td>
<td>4 </td>
<td>52 (43) </td>
<td>16 </td>
<td>72 </td>
</tr>
<tr>
<td>
<em>Streptococcus pyogenes</em> </td>
<td>60 </td>
<td>209 </td>
<td>17 </td>
<td>52 (39) </td>
<td>6 </td>
<td>75 </td>
</tr>
<tr>
<td>
<em>Thermococcus kodakarensis</em> </td>
<td>85 </td>
<td>172 </td>
<td>43 </td>
<td>57 (38) </td>
<td>4 </td>
<td>104 </td>
</tr>
<tr>
<td>
<em>Thermotoga maritima</em> </td>
<td>37 </td>
<td>127 </td>
<td>10 </td>
<td>76 (64) </td>
<td>5 </td>
<td>91 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>Columns 4–7 show the data obtained in the work.<a reveal-id="dsy046-B37" data-open="dsy046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Column 3 shows the shifts found within known genes (no closer than 50 nt before the start/end of the sequence). Column 4 shows the shifts that occur on the edges of known genes (no more than 50 nt from the beginning or end of a known gene). In parentheses, the number of shifts is indicated, when the association with the adjacent gene occurs. The number of shifts occurring in the area between known genes is shown in Column 5.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010220" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T7" class="table-wrap-title" data-id="dsy046-T7">
<div class="label">Table 7</div>
<div class="caption"><p>The number of potential frameshifts in cds from the 17 bacterial genomes (Column 2) obtained in the present study</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th colspan="2">This work<hr> </th>
<th colspan="4">GeneTack-GM program<hr> </th>
</tr>
<tr>
<th>1. Name of bacteria </th>
<th>2. Number of cds with frameshifts </th>
<th>3. Number of frameshifts </th>
<th>4. Inside the gene, no closer than 50 nt to the border </th>
<th>5. The beginning or end of the gene is not more than 50 nt (the number of cases with the addition of a neighbouring gene) </th>
<th>6. Between genes </th>
<th>7. Total </th>
</tr>
</thead>
<tbody>
<tr>
<td>
<em>Anaeromyxobacter dehalogenans</em> </td>
<td>425 </td>
<td>768 </td>
<td>49 </td>
<td>101 (62) </td>
<td>32 </td>
<td>182 </td>
</tr>
<tr>
<td>
<em>Archaeoglobus fulgidus</em> </td>
<td>77 </td>
<td>174 </td>
<td>44 </td>
<td>299 (251) </td>
<td>53 </td>
<td>396 </td>
</tr>
<tr>
<td>
<em>Bacillus subtilis</em> </td>
<td>126 </td>
<td>434 </td>
<td>35 </td>
<td>79 (57) </td>
<td>27 </td>
<td>141 </td>
</tr>
<tr>
<td>
<em>Campylobacter jejuni</em> </td>
<td>48 </td>
<td>86 </td>
<td>8 </td>
<td>104 (92) </td>
<td>16 </td>
<td>128 </td>
</tr>
<tr>
<td>
<em>Caulobacter crescentus</em> </td>
<td>306 </td>
<td>651 </td>
<td>43 </td>
<td>57 (30) </td>
<td>3 </td>
<td>103 </td>
</tr>
<tr>
<td>
<em>Clavibacter michiganensis</em> </td>
<td>357 </td>
<td>736 </td>
<td>35 </td>
<td>65 (40) </td>
<td>63 </td>
<td>163 </td>
</tr>
<tr>
<td>
<em>Methanopyrus kandleri</em> </td>
<td>150 </td>
<td>436 </td>
<td>76 </td>
<td>96 (68) </td>
<td>38 </td>
<td>210 </td>
</tr>
<tr>
<td>
<em>Methanosphaera stadtmanae</em> </td>
<td>111 </td>
<td>272 </td>
<td>5 </td>
<td>37 (27) </td>
<td>13 </td>
<td>55 </td>
</tr>
<tr>
<td>
<em>Pasteurella multocida</em> </td>
<td>46 </td>
<td>146 </td>
<td>9 </td>
<td>56 (48) </td>
<td>4 </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Picrophilus torridus</em> </td>
<td>15 </td>
<td>62 </td>
<td>7 </td>
<td>154 (130) </td>
<td>9 </td>
<td>170 </td>
</tr>
<tr>
<td>
<em>Pyrobaculum aerophilum</em> </td>
<td>71 </td>
<td>249 </td>
<td>65 </td>
<td>235 (163) </td>
<td>48 </td>
<td>348 </td>
</tr>
<tr>
<td>
<em>Ralstonia solanacearum</em> </td>
<td>330 </td>
<td>607 </td>
<td>38 </td>
<td>66 (41) </td>
<td>19 </td>
<td>123 </td>
</tr>
<tr>
<td>
<em>Salmonella enterica,</em> </td>
<td>115 </td>
<td>457 </td>
<td>52 </td>
<td>162 (112) </td>
<td>38 </td>
<td>252 </td>
</tr>
<tr>
<td>
<em>Staphylococcus aureus</em> </td>
<td>130 </td>
<td>381 </td>
<td>4 </td>
<td>52 (43) </td>
<td>16 </td>
<td>72 </td>
</tr>
<tr>
<td>
<em>Streptococcus pyogenes</em> </td>
<td>60 </td>
<td>209 </td>
<td>17 </td>
<td>52 (39) </td>
<td>6 </td>
<td>75 </td>
</tr>
<tr>
<td>
<em>Thermococcus kodakarensis</em> </td>
<td>85 </td>
<td>172 </td>
<td>43 </td>
<td>57 (38) </td>
<td>4 </td>
<td>104 </td>
</tr>
<tr>
<td>
<em>Thermotoga maritima</em> </td>
<td>37 </td>
<td>127 </td>
<td>10 </td>
<td>76 (64) </td>
<td>5 </td>
<td>91 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T7" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th colspan="2">This work<hr> </th>
<th colspan="4">GeneTack-GM program<hr> </th>
</tr>
<tr>
<th>1. Name of bacteria </th>
<th>2. Number of cds with frameshifts </th>
<th>3. Number of frameshifts </th>
<th>4. Inside the gene, no closer than 50 nt to the border </th>
<th>5. The beginning or end of the gene is not more than 50 nt (the number of cases with the addition of a neighbouring gene) </th>
<th>6. Between genes </th>
<th>7. Total </th>
</tr>
</thead>
<tbody>
<tr>
<td>
<em>Anaeromyxobacter dehalogenans</em> </td>
<td>425 </td>
<td>768 </td>
<td>49 </td>
<td>101 (62) </td>
<td>32 </td>
<td>182 </td>
</tr>
<tr>
<td>
<em>Archaeoglobus fulgidus</em> </td>
<td>77 </td>
<td>174 </td>
<td>44 </td>
<td>299 (251) </td>
<td>53 </td>
<td>396 </td>
</tr>
<tr>
<td>
<em>Bacillus subtilis</em> </td>
<td>126 </td>
<td>434 </td>
<td>35 </td>
<td>79 (57) </td>
<td>27 </td>
<td>141 </td>
</tr>
<tr>
<td>
<em>Campylobacter jejuni</em> </td>
<td>48 </td>
<td>86 </td>
<td>8 </td>
<td>104 (92) </td>
<td>16 </td>
<td>128 </td>
</tr>
<tr>
<td>
<em>Caulobacter crescentus</em> </td>
<td>306 </td>
<td>651 </td>
<td>43 </td>
<td>57 (30) </td>
<td>3 </td>
<td>103 </td>
</tr>
<tr>
<td>
<em>Clavibacter michiganensis</em> </td>
<td>357 </td>
<td>736 </td>
<td>35 </td>
<td>65 (40) </td>
<td>63 </td>
<td>163 </td>
</tr>
<tr>
<td>
<em>Methanopyrus kandleri</em> </td>
<td>150 </td>
<td>436 </td>
<td>76 </td>
<td>96 (68) </td>
<td>38 </td>
<td>210 </td>
</tr>
<tr>
<td>
<em>Methanosphaera stadtmanae</em> </td>
<td>111 </td>
<td>272 </td>
<td>5 </td>
<td>37 (27) </td>
<td>13 </td>
<td>55 </td>
</tr>
<tr>
<td>
<em>Pasteurella multocida</em> </td>
<td>46 </td>
<td>146 </td>
<td>9 </td>
<td>56 (48) </td>
<td>4 </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Picrophilus torridus</em> </td>
<td>15 </td>
<td>62 </td>
<td>7 </td>
<td>154 (130) </td>
<td>9 </td>
<td>170 </td>
</tr>
<tr>
<td>
<em>Pyrobaculum aerophilum</em> </td>
<td>71 </td>
<td>249 </td>
<td>65 </td>
<td>235 (163) </td>
<td>48 </td>
<td>348 </td>
</tr>
<tr>
<td>
<em>Ralstonia solanacearum</em> </td>
<td>330 </td>
<td>607 </td>
<td>38 </td>
<td>66 (41) </td>
<td>19 </td>
<td>123 </td>
</tr>
<tr>
<td>
<em>Salmonella enterica,</em> </td>
<td>115 </td>
<td>457 </td>
<td>52 </td>
<td>162 (112) </td>
<td>38 </td>
<td>252 </td>
</tr>
<tr>
<td>
<em>Staphylococcus aureus</em> </td>
<td>130 </td>
<td>381 </td>
<td>4 </td>
<td>52 (43) </td>
<td>16 </td>
<td>72 </td>
</tr>
<tr>
<td>
<em>Streptococcus pyogenes</em> </td>
<td>60 </td>
<td>209 </td>
<td>17 </td>
<td>52 (39) </td>
<td>6 </td>
<td>75 </td>
</tr>
<tr>
<td>
<em>Thermococcus kodakarensis</em> </td>
<td>85 </td>
<td>172 </td>
<td>43 </td>
<td>57 (38) </td>
<td>4 </td>
<td>104 </td>
</tr>
<tr>
<td>
<em>Thermotoga maritima</em> </td>
<td>37 </td>
<td>127 </td>
<td>10 </td>
<td>76 (64) </td>
<td>5 </td>
<td>91 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>Columns 4–7 show the data obtained in the work.<a reveal-id="dsy046-B37" data-open="dsy046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Column 3 shows the shifts found within known genes (no closer than 50 nt before the start/end of the sequence). Column 4 shows the shifts that occur on the edges of known genes (no more than 50 nt from the beginning or end of a known gene). In parentheses, the number of shifts is indicated, when the association with the adjacent gene occurs. The number of shifts occurring in the area between known genes is shown in Column 5.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010220" target="_blank">Open in new tab</a></div>
</div></div>
<p><a reveal-id="dsy046-T2" data-open="dsy046-T2" class="link link-reveal link-table xref-fig">Table 2</a> presents the results obtained from studying the <em>RD</em> set. This table shows that from 3,460 codon-shuffled cds with an artificial frameshift, in 2,975 sequences, TP phase shifts were found within the ±50 nt interval. In 68 cases, TP phase shifts were predicted outside the ±50 nt region from the artificially created deletion. This implies that the number of errors of the second type constitutes ∼14%, and the power of the method is ∼86%. The results of analysis of the TP phase shifts in the remaining 16 genomes are presented in <a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 2. The total number of TP phase shifts is shown in <a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 3. The table shows that the number of TP phase shifts in bacterial genes ranges from several dozens to hundreds per single genome.</p>                    <h3 scrollto-destination="134010222" id="134010222" class="section-title">3.4. Comparison with the Genetack-GM program</h3>
<p>It would be interesting to compare the obtained results with the results obtained previously while searching for reading frameshifts. It has been recorded that the Genetack-GM program showed the best results among other frameshift prediction methods, hence our results were compared with this program.<a reveal-id="dsy046-B17" data-open="dsy046-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> For this purpose, the complete sequences of the 17 prokaryotic genomes were downloaded from the Ensembl database<a reveal-id="dsy046-B39" data-open="dsy046-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and submitted to the GeneTack-GM software program. GeneTack-GM is a combination of the GeneMark program designed to indicate coding sequences in a genome, and the Genetack designed to search for potential frameshifts.<a reveal-id="dsy046-B37" data-open="dsy046-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In the case of prokaryotic sequences, the Genetack software program searches for cases of potential frameshifts that resulted in the splitting of a single coding sequence into two independent ones (the author claimed that, in modern databases they are usually represented by two separate genes). The result of the GeneTack-GM software is the predicted coordinates of a gene (usually a hypothetical gene) and the coordinate of a frameshift within the gene.</p>
<p>We compared the coordinates of frameshifts obtained by Genetack for the 17 bacterial genomes with the boundaries of known genes indicated in the annotation to the corresponding genomes in the Ensembl database. Frameshifts found by the GeneTack-GM were divided into three categories according to their position in the known genes. The results are presented in <a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>. The first category includes frameshifts found within the known genes (not closer than 50 nt to the start/end of the gene) (<a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 4). The second category includes frameshifts that occur at the edges of the known genes (not more than 50 nt from the start/end of the gene). For this category, it is also indicated in parentheses whether this frameshift leads to uniting a gene with the neighbouring one (i.e. to predicting a hypothetical gene), if the coordinate of the end of a hypothetical gene predicted by the software program captures the following gene by more than 100 nt (<a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 5). The third category includes frameshifts occurring in the area between the known genes (<a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 6). It is evident that most of the frameshifts found by GeneTack are pertaining to the second and third categories. This implies that the presence of these frameshifts is connected to the fact that the two adjacent genes that are indicated in the Ensembl database are combined by the GeneTack-GM software program into a single gene or into one gene that captures parts of these genes.</p>
<p>However, this study evaluated the presence of reading frameshifts in known genes (cds). At a distance no closer than 50 nt from the boundaries, our method found more than 70% of the frameshifts. This 70% relate to the first category of frameshifts (70% of the data in <a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>, Column 3). A comparison of 70% of Columns 3 and 4 shows that our approach found a significantly large number of reading frameshifts within the already known genes, compared with the GeneTack-GM software program, with a lower false positive rate (8–14% for our program versus 32% for the GeneTack software program).</p>
<p>Similar results were obtained when comparing the results obtained in the present study for the <em>A. thaliana</em> genome with the data presented in the GeneTack database for the genome. A total of 2,067 potential reading frameshifts were found in the <em>A. thaliana</em> genome by the authors of GeneTack, whereas we were able to detect 14,951 TP shift cases (see <a reveal-id="dsy046-T6" data-open="dsy046-T6" class="link link-reveal link-table xref-fig">Table 6</a>). It should be noted that we analysed only the cds, whereas the GeneTack database contains data for mRNA sequences, which also includes the non-coding sequences (5′ and 3′ untranslated regions). Therefore, we also divided the 2,067 reading frameshifts into three groups, as shown in <a reveal-id="dsy046-T7" data-open="dsy046-T7" class="link link-reveal link-table xref-fig">Table 7</a>. The first group includes frameshifts which are located inside the cds not closer than 50 nt from the beginning and end of the coding section. The second group includes frameshifts that are located at a distance not more than 50 nt from the ends of cds, and in the third group the frameshift corresponds to the mRNA non-coding regions. The first group includes 485, the second group includes 710, and the third group includes 872 reading frameshifts.</p>
<p>A more detailed study of the distribution of frameshifts by position in genes from the <em>A. thaliana</em> genome is shown in <a reveal-id="dsy046-F6" data-open="dsy046-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>. The increase in the number of frameshifts at the end of cds may be due to the fact that the sequences at the end of the gene do not greatly affect the structure of the encoded protein. However, it is surprising that such an increase is also found at the beginning of the gene. It is difficult to imagine that such mutations will not change the biological function of the encoded protein. Rather, it can be assumed that the observed frameshifts are compensating, which return the reading frame to its original position. The initial frameshift could be at the very beginning of the gene and we were unable to see it using this method. Our approach may not find a frameshift due to a large penalty for insertion or deletion (<em>d</em> in Equation 2) if it occurs at a distance &lt;20–30 bases from the start of the gene. However, the second, already found frameshift, just compensates it. In this case, the distance to the frameshift revealed at the beginning of the gene should be similar to the distance between the pairwise compensating frameshifts that we find in cds. We constructed a distribution between the pair compensating frameshifts. This distribution is shown in <a reveal-id="dsy046-F7" data-open="dsy046-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>. It can be seen from the figure that the average distance between the compensating frameshifts is &lt;0.1 of the corresponding gene size. This result supports our hypothesis that paired compensating frameshifts often occur at the beginning of a gene. This hypothesis explains the surprisingly large number of TP phase shifts, which was revealed at the beginning of the genes in <a reveal-id="dsy046-F6" data-open="dsy046-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>. </p>                        <a id="134010228" scrollto-destination="134010228"></a>
<div id="dsy046-F6" data-id="dsy046-F6" class="fig fig-section">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f6.png?Expires=1576262930&amp;Signature=e5NslBmr9jI15vFcGqH-TtSMbFYhMbbqd1zgqhSzvswSzRRQw4uVqF90tJV-GuNKEUMOM9LYWIu4W2rwFT2GHXMwSMuhkLPwzInKhuNeUjKu-aMVmPJqNgP5K6wmyMkl3OnoZjNKMTAuSoJHYlQoLl5ZWYUXjngKCyoq44bEluyx1-aj31L16MC1VhWgAx83WjrjRyKsbaMK2gd0f6z7oKDYu4L1q~ccZxQhaYgJuYlgRZ0En6uvBClgQHc4osDQXFJdx-65C7~JmOwI5strXhnT~LQ6X4OdU-h1RPOQT3XgtIfTegktpWJ4yoayPJYPMgNgVZ8DOdrWSX8WrnpWyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of shifts position in the sequence of a gene. The x-axis shows the distance as a percentage of the beginning of the gene (with step equals to 5%), the y-axis shows the percentage of shifts per interval of 5%. The black bars—the data from the work,21 the white—the data of our work. The leftmost and rightmost bars show the number of frameshifts found outside the cds from the work.21" path-from-xml="dsy046f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010228/dsy046f6.tif" path-from-xml="dsy046f6.tif" target="_blank">Open in new tab</a><a section="134010228" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f6.png?Expires=1576262930&amp;Signature=RVMqZYEOZJ0sv6B9wHyKp7oCz3YXozk86OAyUK4p~kIzBSJqurwEwVACqD9GZ-gB33YlhUl2W16WksujH9FLwqPNf5jNvS8JlE7RbcxXAKai1oK8JJlrpE5Xr2k8e8DN~D5ru915qa46UJPCFAkvK3WidxcWOuHmZ6t~mBMjiq1zXK6v3fc631IPi9FnEx-g61DSU9sk8JeM7cp2S7zMR~Wrwd0LgSYNFBtxEtJQ5zGj9AA~wzYXIGjKJv0Zz7~KSDkIVXhbW7-EfvVgofL3Dj4G8v8FBzJIJnnXU-rOkdAgVP50itWoLvY3QaBA9YAJ-SNclfPtm2usm7LkHD9hFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010228&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of shifts position in the sequence of a gene. The <em>x</em>-axis shows the distance as a percentage of the beginning of the gene (with step equals to 5%), the <em>y</em>-axis shows the percentage of shifts per interval of 5%. The black bars—the data from the work,<a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> the white—the data of our work. The leftmost and rightmost bars show the number of frameshifts found outside the cds from the work.<a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p></div>
</div>
<div content-id="dsy046-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f6.png?Expires=1576262930&amp;Signature=e5NslBmr9jI15vFcGqH-TtSMbFYhMbbqd1zgqhSzvswSzRRQw4uVqF90tJV-GuNKEUMOM9LYWIu4W2rwFT2GHXMwSMuhkLPwzInKhuNeUjKu-aMVmPJqNgP5K6wmyMkl3OnoZjNKMTAuSoJHYlQoLl5ZWYUXjngKCyoq44bEluyx1-aj31L16MC1VhWgAx83WjrjRyKsbaMK2gd0f6z7oKDYu4L1q~ccZxQhaYgJuYlgRZ0En6uvBClgQHc4osDQXFJdx-65C7~JmOwI5strXhnT~LQ6X4OdU-h1RPOQT3XgtIfTegktpWJ4yoayPJYPMgNgVZ8DOdrWSX8WrnpWyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of shifts position in the sequence of a gene. The x-axis shows the distance as a percentage of the beginning of the gene (with step equals to 5%), the y-axis shows the percentage of shifts per interval of 5%. The black bars—the data from the work,21 the white—the data of our work. The leftmost and rightmost bars show the number of frameshifts found outside the cds from the work.21" path-from-xml="dsy046f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010228/dsy046f6.tif" path-from-xml="dsy046f6.tif" target="_blank">Open in new tab</a><a section="134010228" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f6.png?Expires=1576262930&amp;Signature=RVMqZYEOZJ0sv6B9wHyKp7oCz3YXozk86OAyUK4p~kIzBSJqurwEwVACqD9GZ-gB33YlhUl2W16WksujH9FLwqPNf5jNvS8JlE7RbcxXAKai1oK8JJlrpE5Xr2k8e8DN~D5ru915qa46UJPCFAkvK3WidxcWOuHmZ6t~mBMjiq1zXK6v3fc631IPi9FnEx-g61DSU9sk8JeM7cp2S7zMR~Wrwd0LgSYNFBtxEtJQ5zGj9AA~wzYXIGjKJv0Zz7~KSDkIVXhbW7-EfvVgofL3Dj4G8v8FBzJIJnnXU-rOkdAgVP50itWoLvY3QaBA9YAJ-SNclfPtm2usm7LkHD9hFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010228&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of shifts position in the sequence of a gene. The <em>x</em>-axis shows the distance as a percentage of the beginning of the gene (with step equals to 5%), the <em>y</em>-axis shows the percentage of shifts per interval of 5%. The black bars—the data from the work,<a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> the white—the data of our work. The leftmost and rightmost bars show the number of frameshifts found outside the cds from the work.<a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p></div>
</div>                        <a id="134010229" scrollto-destination="134010229"></a>
<div id="dsy046-F7" data-id="dsy046-F7" class="fig fig-section">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f7.png?Expires=1576262930&amp;Signature=P2VKfLzt2m~CdeUCX4TgxUA-f5vsiv~fJIZMpbWOYCSrwZ9qEFC-~w6kgYWhCWvXWBJbjmx3XnXkPto6ILdlRmXONGn--CDhRGja39y2RkIUqVU9KWxSBkAIaWueM11fEJH~2AQgNClW-z~aiAoUiUmqid~DreVYfWpixVY9ebkqP-Awfmh3eqaU-dSrdSt1QNYneR94LBgpBksg9cQAW6f~gWd3c1s5vQSkkAdJCWMxc9-O48yD3YzC44KRGQ0ZT7rjgWfRiXt~Z-zEfqcDc8zDCw0GRXqoytDRxHeRF4~4m0SxDM69n7F4dUxaHoGHmHNZm82bzgAb93osAawt4g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the distance between paired compensating shifts of the TP phase in the Arabidopsis thaliana genome." path-from-xml="dsy046f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010229/dsy046f7.tif" path-from-xml="dsy046f7.tif" target="_blank">Open in new tab</a><a section="134010229" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f7.png?Expires=1576262930&amp;Signature=etkfZlnjGlqxYMTljlfhnHVzBeMIUDJYz2nXfJiKD7Tq9tjKWvG3PibXzNAdNWxvcz6rG3PQtQoNNyIgCOdjS~EzZnsu0DHGBDNpBN1os1WvLT01BSwMeMwtcMfN8RzmVs4AwIG82sOJfL4VSJ1syQzTiNImNQUCNqXO51znBVsRDqYcUeWmUoYsFXjIjf-jXVtIfzYo~3tuQAWglR8bDqKSrboUvhh7w22z7mwlszLe0YFUfZJ5EaQFUy~XRoB4LELpujBbMeQb-nNbtktl420zi531HSCmvITmvkGNIVUkKuC2D-KHr5d0P1DAXY-1iajy1pZs3P5pLtI-CTycyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010229&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of the distance between paired compensating shifts of the TP phase in the <em>Arabidopsis thaliana</em> genome.</p></div>
</div>
<div content-id="dsy046-F7" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/m_dsy046f7.png?Expires=1576262930&amp;Signature=P2VKfLzt2m~CdeUCX4TgxUA-f5vsiv~fJIZMpbWOYCSrwZ9qEFC-~w6kgYWhCWvXWBJbjmx3XnXkPto6ILdlRmXONGn--CDhRGja39y2RkIUqVU9KWxSBkAIaWueM11fEJH~2AQgNClW-z~aiAoUiUmqid~DreVYfWpixVY9ebkqP-Awfmh3eqaU-dSrdSt1QNYneR94LBgpBksg9cQAW6f~gWd3c1s5vQSkkAdJCWMxc9-O48yD3YzC44KRGQ0ZT7rjgWfRiXt~Z-zEfqcDc8zDCw0GRXqoytDRxHeRF4~4m0SxDM69n7F4dUxaHoGHmHNZm82bzgAb93osAawt4g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the distance between paired compensating shifts of the TP phase in the Arabidopsis thaliana genome." path-from-xml="dsy046f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010229/dsy046f7.tif" path-from-xml="dsy046f7.tif" target="_blank">Open in new tab</a><a section="134010229" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy046/1/dsy046f7.png?Expires=1576262930&amp;Signature=etkfZlnjGlqxYMTljlfhnHVzBeMIUDJYz2nXfJiKD7Tq9tjKWvG3PibXzNAdNWxvcz6rG3PQtQoNNyIgCOdjS~EzZnsu0DHGBDNpBN1os1WvLT01BSwMeMwtcMfN8RzmVs4AwIG82sOJfL4VSJ1syQzTiNImNQUCNqXO51znBVsRDqYcUeWmUoYsFXjIjf-jXVtIfzYo~3tuQAWglR8bDqKSrboUvhh7w22z7mwlszLe0YFUfZJ5EaQFUy~XRoB4LELpujBbMeQb-nNbtktl420zi531HSCmvITmvkGNIVUkKuC2D-KHr5d0P1DAXY-1iajy1pZs3P5pLtI-CTycyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010229&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy046f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of the distance between paired compensating shifts of the TP phase in the <em>Arabidopsis thaliana</em> genome.</p></div>
</div>
<p>As we have identified 14,951 potential reading frameshifts in cds from the genome, our algorithm is about seven times more efficient than the GeneTack-GM software program. If we compare the results in relation to the first group, then this difference will be greater, because more than 70% of the frameshifts detected are related to the first group (<a reveal-id="dsy046-F6" data-open="dsy046-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Similar results were also obtained for all other eukaryotic genomes (<a reveal-id="dsy046-T6" data-open="dsy046-T6" class="link link-reveal link-table xref-fig">Table 6</a>). The result could be explained by the fact that if the HMM is trained on a set of selected mRNAs,<a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> statistical properties, such as <em>k</em>-mer frequencies, are averaged over the set. In the result of the averaging, the HMM parameters are changed so that the number of errors of the first and the second type could be increased. Consequently, some frameshifts could be missed by the HMM-based method. Let us illustrate this statement for brevity and simplicity using the classical Markov models. Consider two sequences, <em>seq</em>1 = ‘<em>ttgccagagcagattgcccagattgccagatt</em>’ and <em>seq</em>2 = ‘<em>aactcggtaacggtctaaactcggtacggtcta</em>’. The conditional probabilities of the nucleotide pairs of these sequences are presented in <em>N</em>1 and <em>N</em>2 (<a reveal-id="dsy046-T8" data-open="dsy046-T8" class="link link-reveal link-table xref-fig">Table 8A and B</a>). The matrix represents the probability <em>P</em>(<em>X<sub>n</sub></em><sub>+1</sub> = <em>i<sub>n</sub></em><sub>+1</sub>|<em>X<sub>n</sub></em> = <em>i <sub>n</sub></em>), where <em>X<sub>n</sub></em><sub>+1</sub> corresponds to the rows of the matrix, and <em>X<sub>n</sub></em> corresponds to the columns. A Markov model was built using the conditional probability matrix <em>N</em>1 or the <em>N</em>2. The matrices <em>N</em>1 and <em>N</em>2 can be used to search for sequences with similar nucleotide correlations as the sequences <em>seq</em>1 and <em>seq</em>2. Let the probabilities <em>P</em>11 and <em>P</em>22 be the probabilities that the sequences <em>seq</em>1 and <em>seq</em>2 are generated by the <em>N</em>1 and <em>N</em>2 matrices, respectively [<em>P</em>11 = (0.5)<sup>22</sup> and <em>P</em>22 = (0.5)<sup>22</sup>]. The probability <em>P</em>12 that the sequence <em>seq</em>1 was generated using the matrix <em>N</em>2 is equal to zero [<em>P</em>12 = (0)<sup>22</sup> = 0] as well as the probability <em>P</em>21 (that the sequence <em>seq</em>2 was generated using the matrix <em>N</em>1). However, the probability that a randomly shuffled sequence was generated using the matrix <em>N</em>1 <em>PR</em>1 = 0 is zero, <em>PR</em>2 = 0 because in the randomly shuffled sequence there will be pairs of nucleotides, for which <em>P</em>(<em>X<sub>n</sub></em><sub>+1</sub> = <em>i <sub>n</sub></em><sub>+1</sub>|<em>X<sub>n</sub></em> = <em>i<sub>n</sub></em>) = 0 both in the matrix <em>N</em>1 and in the matrix <em>N</em>2. Therefore, one can find sequences similar to <em>seq</em>1 or <em>seq</em>2 surrounded by random sequences using the Markov model at a statistically significant level.</p>                        <a id="134010231" scrollto-destination="134010231"></a>
<div content-id="dsy046-T8" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T8" class="table-wrap-title" data-id="dsy046-T8">
<div class="label">Table 8</div>
<div class="caption"><p>The matrices of conditional probabilities <em>P</em>(<em>X<sub>n</sub></em><sub>+1</sub>=<em>i <sub>n</sub></em><sub>+1</sub>|<em>X<sub>n</sub></em>=<em>i <sub>n</sub></em>), created by the sequences <em>seq</em>1 = ttgccagagcagattgcccagattgccagatt (A), <em>seq</em>2 = aactcggtaacggtctaaactcggtacggtcta (B) and unification of these sequences (C)</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th> </th>
<th> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(C)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
</tr>
</thead>
<tbody>
<tr>
<td>a </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>a </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>a </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>t </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>t </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>t </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>c </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>c </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>c </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>g </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>g </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>g </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T8" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th> </th>
<th> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(C)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
</tr>
</thead>
<tbody>
<tr>
<td>a </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>a </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>a </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>t </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>t </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>t </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>c </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>c </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>c </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>g </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>g </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>g </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p><em>X<sub>n</sub></em><sub>+1</sub> corresponds to the rows of the matrix, and <em>X<sub>n</sub></em> corresponds to the columns of the matrix.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010231" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy046-T8" class="table-wrap-title" data-id="dsy046-T8">
<div class="label">Table 8</div>
<div class="caption"><p>The matrices of conditional probabilities <em>P</em>(<em>X<sub>n</sub></em><sub>+1</sub>=<em>i <sub>n</sub></em><sub>+1</sub>|<em>X<sub>n</sub></em>=<em>i <sub>n</sub></em>), created by the sequences <em>seq</em>1 = ttgccagagcagattgcccagattgccagatt (A), <em>seq</em>2 = aactcggtaacggtctaaactcggtacggtcta (B) and unification of these sequences (C)</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th> </th>
<th> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(C)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
</tr>
</thead>
<tbody>
<tr>
<td>a </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>a </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>a </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>t </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>t </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>t </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>c </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>c </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>c </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>g </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>g </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>g </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy046-T8" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>(A)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th> </th>
<th> </th>
<th>(B)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>(C)<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
<th>
<hr> </th>
</tr>
<tr>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
<th> </th>
<th>a </th>
<th>t </th>
<th>c </th>
<th>g </th>
</tr>
</thead>
<tbody>
<tr>
<td>a </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>a </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>a </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>t </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td> </td>
<td>t </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>t </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>c </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>c </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>c </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
<tr>
<td>g </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td> </td>
<td>g </td>
<td>0 </td>
<td>0.5 </td>
<td>0 </td>
<td>0.5 </td>
<td>g </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
<td>0.25 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn6"><div class="fn"><p><em>X<sub>n</sub></em><sub>+1</sub> corresponds to the rows of the matrix, and <em>X<sub>n</sub></em> corresponds to the columns of the matrix.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010231" target="_blank">Open in new tab</a></div>
</div></div>
<p>But if a Markov model is trained using both sequences <em>seq</em>1 and <em>seq</em>2, the matrix <em>N</em>3 would be constructed (<a reveal-id="dsy046-T8" data-open="dsy046-T8" class="link link-reveal link-table xref-fig">Table 8C</a>). In this case, the probabilities of generating <em>seq</em>1 and <em>seq</em>2 are (0.25)<sup>22</sup> and the same probability will be obtained for any other sequence of the same length, including the randomly shuffled sequence. So, the identification of <em>seq</em>1 and <em>seq</em>2 using the Markov model is impossible at a statistically significant level. We can say that the statistical properties of these sequences are averaged.</p>
<p>The same phenomenon can be observed in the case of real genes, when a training sample is created for HMM from many genes. This effect was employed on the Genetack-GM program. We created two sets of artificial genes (<em>Q</em>1 and <em>Q</em>2) with different types of triplet frequency using different synonymous codons and used the sets to train HMM. Each artificial gene had a length of 1,500 bases and contained a start codon as well as a stop codon. Each set has a volume of 1,000 sequences. Also, we created a set <em>Q</em>3 of 1,000 sequences, half of which were of type <em>Q</em>1, while the other half were of type <em>Q</em>2. Additionally we created two sets <em>D</em>1 and <em>D</em>2, which had the same TP as the sets <em>Q</em>1 and <em>Q</em>2, respectively. But each sequence contained one deletion in a random position, but not closer than 100 bases from the beginning or the end. In order to exclude the effect of the stop codons resulting from the frameshifts, we replaced them with randomly selected coding codons. The volume of sets <em>D</em>1 and <em>D</em>2 was 100 sequences each.</p>
<p>First, we trained the Genetack-GM program on sets <em>Q</em>1 and <em>Q</em>2 and searched for frameshifts in sets <em>D</em>1 and <em>D</em>2, respectively. The program found shifts in 75 sequences from set <em>D</em>1 and 17 from set <em>D</em>2. Trained <em>Q</em>3 Genetack-GM was applied to sets <em>D</em>1 or <em>D</em>2. In the result, the program found 8 and 0 sequences with frameshifts, respectively. This example shows that combining different genes into one training set can significantly degrade the capabilities of the HMM. In our method, such averaging does not occur because we analyse each cds without using a training set; the method is adjusted to the TP that exists in each considered cds. This means that the mathematical method finds an optimal correlation matrix considering the possibility of insertions or deletions for each analysed cds. The final alignment of the studied sequence against the obtained matrix provides an alignment and coordinates of the potential reading frameshifts. It is also important to note that to search for potential frameshifts using our method, one needs only cds without any other information or training set. This is the main improvement of the method in comparison with HMM to determine the reading frameshifts.</p>
<p>Genetack performance was also tested on a set of cds without additional non-coding regions. We randomly selected 1,000 cds from the <em>A. thaliana</em> genome. In each of these cds, we deleted a single nucleotide in a random position (but no closer than 100 nt from the beginning or end of the sequence). Then, we inserted a random nucleotide just before the stop-codon to keep the sequence length. Genetack and the method developed here were applied to this set. In the result, Genetack predicted frameshift in 615 sequences and our method in 676 sequences (inside ± 50 nt from the actual deletion position). The HMM for Genetack was trained by GeneMark on a set of 100,000 cds from <em>A. thaliana.</em> Then in each cds with deletion, we changed the stop and start codons that occurred after the deletion in the first frame to a randomly chosen coding codon. Then, we again applied both programs to this set. In the result, Genetack correctly predicts frameshift only in 277 sequences and our method in 624. The reduced number of frameshifts found by our method could be explained by the decreasing TP quality after the codon changes. The result demonstrates that Genetack is more suitable for the detection of frameshift that separates a single coding sequence into two (or more) independent genes as it was pointed by the authors.<a reveal-id="dsy046-B17" data-open="dsy046-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> But Genetack weakly predicts frameshifts if they do not lead to the formation of the premature stop-codon or if we have only cds without surrounding non-coding regions.</p>
<p>We identified TP phase shifts in 660 genes, which constituted 83% of the first group, out of 1,183 genes discovered by the GeneTack software program in cds (2,080 genes found in cds and non-coding regions in the work<a reveal-id="dsy046-B21" data-open="dsy046-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>). Besides, more than 70% of the frameshifts found in our earlier publications for prokaryotic genes were identified using the developed algorithm.<a reveal-id="dsy046-B30" data-open="dsy046-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsy046-B40" data-open="dsy046-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> At the same time, for prokaryotic genes, a higher number of potential frameshift mutations was discovered using this method. This is because the developed method works much better with sequences having a low level of TP.</p>                    <h3 scrollto-destination="134010237" id="134010237" class="section-title">3.5. Discussion of the possibility of creating new genes through frameshift mutations</h3>
<p>For many years, a study of the evolution of genes has attracted the attention of researchers. After determining the sequences of many prokaryotic and eukaryotic genes, the research in this area has significantly expanded. It is now believed that new genes are created by duplicating already existing genes.<a reveal-id="dsy046-B41" data-open="dsy046-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Therefore, a large number of genes are grouped in families based on the similarity of the amino acid or nucleotide sequences.<a reveal-id="dsy046-B42" data-open="dsy046-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Processes such as gene fusion,<a reveal-id="dsy046-B43" data-open="dsy046-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> exon shuffling,<a reveal-id="dsy046-B44" data-open="dsy046-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> alternative splicing<a reveal-id="dsy046-B45" data-open="dsy046-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and lateral gene transfer<a reveal-id="dsy046-B46" data-open="dsy046-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> are considered to be the principal mechanisms of the creation of a variety of genes in a genome and the corresponding proteins. However, using such mechanisms, it is difficult to create a fundamentally new sequence, but a frameshift mutation could do this efficiently. It has previously been suggested that frameshift mutations could play a significant role in the process of creating new genes.<a reveal-id="dsy046-B8" data-open="dsy046-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy046-B9" data-open="dsy046-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> The authors of the presented works have proposed that if a protein for some reasons is not under the selection pressure, frameshift mutation could persist and eventually lead to functional divergence. It is mainly this phenomenon that we observed in the present work. More than 20% of the studied eukaryotic cds contain potential frameshift mutations. The question that remains is, how does a protein sequence bear any biological sense after the reading frameshift? It could be assumed that the genetic code could be perfectly adapted to such changes, and it allows the frameshift mutations to obtain biologically meaningful sequences.</p>
<p>Search for frameshifts in genes by the developed method can be done online at: <a class="link link-uri" href="http://victoria.biengi.ac.ru/fsfinder" target="_blank">http://victoria.biengi.ac.ru/fsfinder</a>.</p>                    <h2 scrollto-destination="134010240" id="134010240" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>50% of this research was supported by the RSF grant 14-24-00175, while the other 50% was performed within the framework of the state order No. 01201371082.</p>
<p><em>Conflict of interest</em> None declared.</p>                    <h2 scrollto-destination="134010243" id="134010243" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsy046-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Watson</div> <div class="given-names">J.D.</div>
</div>, <div class="name string-name">
<div class="surname">Baker</div> <div class="given-names">T.A.</div>
</div>, <div class="name string-name">
<div class="surname">Bell</div> <div class="given-names">S.P.</div>
</div>, <div class="name string-name">
<div class="surname">Gann</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Levine</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Losick</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2013</div>, <div class="source">Molecular Biology of the Gene.</div>  
<div class="publisher-loc">Cold Spring Harbor Laboratory Press</div>: 
<div class="publisher-name">Cold Spring Harbor, New York</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Molecular%20Biology%20of%20the%20Gene.&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Molecular%20Biology%20of%20the%20Gene.&amp;aulast=Watson&amp;date=2013" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20Biology%20of%20the%20Gene.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Molecular%20Biology%20of%20the%20Gene.">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ogura</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Bonen</div> <div class="given-names">D.K.</div>
</div>, <div class="name string-name">
<div class="surname">Inohara</div> <div class="given-names">N.</div>
</div>
</div>, et al.  <div class="year">2001</div>, 
<div class="article-title">A frameshift mutation in <em>NOD2</em> associated with susceptibility to Crohn’s disease</div>, <div class="source">Nature</div> , <div class="volume">411</div>, <div class="fpage">603</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20frameshift%20mutation%20in%20NOD2%20associated%20with%20susceptibility%20to%20Crohn%E2%80%99s%20disease&amp;author=Y.%20Ogura&amp;author=D.K.%20Bonen&amp;author=N.%20Inohara&amp;publication_year=2001&amp;journal=Nature&amp;volume=411&amp;pages=603-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/35079114" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35079114" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F35079114"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11385577" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20frameshift%20mutation%20in%20NOD2%20associated%20with%20susceptibility%20to%20Crohn%E2%80%99s%20disease&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Iannuzzi</div> <div class="given-names">M.C.</div>
</div>, <div class="name string-name">
<div class="surname">Stern</div> <div class="given-names">R.C.</div>
</div>, <div class="name string-name">
<div class="surname">Collins</div> <div class="given-names">F.S.</div>
</div>
</div>, et al.  <div class="year">1991</div>, 
<div class="article-title">Two frameshift mutations in the cystic fibrosis gene</div>, <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">48</div>, <div class="fpage">227</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Two%20frameshift%20mutations%20in%20the%20cystic%20fibrosis%20gene&amp;author=M.C.%20Iannuzzi&amp;author=R.C.%20Stern&amp;author=F.S.%20Collins&amp;publication_year=1991&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=48&amp;pages=227-31" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1990834" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Two%20frameshift%20mutations%20in%20the%20cystic%20fibrosis%20gene&amp;aulast=Iannuzzi&amp;date=1991&amp;spage=227&amp;epage=31&amp;volume=48" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Two%20frameshift%20mutations%20in%20the%20cystic%20fibrosis%20gene&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chung</div> <div class="given-names">W.K.</div>
</div>, <div class="name string-name">
<div class="surname">Kitner</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Maron</div> <div class="given-names">B.J.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Novel frameshift mutation in Troponin C (TNNC1) associated with hypertrophic cardiomyopathy and sudden death</div>, <div class="source">Cardiol. Young</div> , <div class="volume">21</div>, <div class="fpage">345</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Novel%20frameshift%20mutation%20in%20Troponin%20C%20(TNNC1)%20associated%20with%20hypertrophic%20cardiomyopathy%20and%20sudden%20death&amp;author=W.K.%20Chung&amp;author=C.%20Kitner&amp;author=B.J.%20Maron&amp;publication_year=2011&amp;journal=Cardiol.%20Young&amp;volume=21&amp;pages=345-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1017/S1047951110001927" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1047951110001927" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1017%2FS1047951110001927"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21262074" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Novel%20frameshift%20mutation%20in%20Troponin%20C%20(TNNC1)%20associated%20with%20hypertrophic%20cardiomyopathy%20and%20sudden%20death&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Identification of somatic mutations in human prostate cancer by RNA-Seq</div>, <div class="source">Gene</div> , <div class="volume">519</div>, <div class="fpage">343</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20somatic%20mutations%20in%20human%20prostate%20cancer%20by%20RNA-Seq&amp;author=X.%20Xu&amp;author=K.%20Zhu&amp;author=F.%20Liu&amp;publication_year=2013&amp;journal=Gene&amp;volume=519&amp;pages=343-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gene.2013.01.046" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2013.01.046" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2013.01.046"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23434521" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20somatic%20mutations%20in%20human%20prostate%20cancer%20by%20RNA-Seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Berget</div> <div class="given-names">S.M.</div>
</div></div> <div class="year">1995</div>, 
<div class="article-title">Exon recognition in vertebrate splicing</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">270</div>, <div class="fpage">2411</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Exon%20recognition%20in%20vertebrate%20splicing&amp;author=S.M.%20Berget&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem&amp;volume=270&amp;pages=2411-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1074/jbc.270.6.2411" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.6.2411" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1074%2Fjbc.270.6.2411"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7852296" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Exon%20recognition%20in%20vertebrate%20splicing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wood</div> <div class="given-names">J.L.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2007</div>, DNA repair, genetic instability, and cancer. <div class="source">DNA Damage Sens. Signal</div> . 
<div class="publisher-name">World Scientific Publishing Co. Pte. Ltd</div>.,: Singapore, pp. <div class="fpage">1</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=DNA%20Damage%20Sens.%20Signal&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=DNA%20Damage%20Sens.%20Signal&amp;aulast=Wood&amp;date=2007&amp;spage=1&amp;epage=22" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DNA%20Damage%20Sens.%20Signal&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=DNA%20Damage%20Sens.%20Signal">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Okamura</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Feuk</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Marquès-Bonet</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Navarro</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Scherer</div> <div class="given-names">S.W.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Frequent appearance of novel protein-coding sequences by frameshift translation</div>, <div class="source">Genomics</div> , <div class="volume">88</div>, <div class="fpage">690</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Frequent%20appearance%20of%20novel%20protein-coding%20sequences%20by%20frameshift%20translation&amp;author=K.%20Okamura&amp;author=L.%20Feuk&amp;author=T.%20Marqu%C3%A8s-Bonet&amp;author=A.%20Navarro&amp;author=S.W.%20Scherer&amp;publication_year=2006&amp;journal=Genomics&amp;volume=88&amp;pages=690-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ygeno.2006.06.009" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2006.06.009" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ygeno.2006.06.009"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16890400" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Frequent%20appearance%20of%20novel%20protein-coding%20sequences%20by%20frameshift%20translation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Raes</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Van De Peer</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Functional divergence of proteins through frameshift mutations</div>, <div class="source">Trends Genet</div> ., <div class="volume">21</div>, <div class="fpage">428</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Functional%20divergence%20of%20proteins%20through%20frameshift%20mutations&amp;author=J.%20Raes&amp;author=Y.%20Van%20De%20Peer&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=428-31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tig.2005.05.013" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.05.013" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tig.2005.05.013"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15951050" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Functional%20divergence%20of%20proteins%20through%20frameshift%20mutations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sheetlin</div> <div class="given-names">S.L.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Frith</div> <div class="given-names">M.C.</div>
</div>, <div class="name string-name">
<div class="surname">Spouge</div> <div class="given-names">J.L.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Frameshift alignment: statistics and post-genomic applications</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3575</div>–<div class="lpage">3582</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Frameshift%20alignment%3A%20statistics%20and%20post-genomic%20applications&amp;author=S.L.%20Sheetlin&amp;author=Y.%20Park&amp;author=M.C.%20Frith&amp;author=J.L.%20Spouge&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3575-3582" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu576" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu576" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu576"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25172925" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Frameshift%20alignment%3A%20statistics%20and%20post-genomic%20applications&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Du</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Improve homology search sensitivity of PacBio data by correcting frameshifts</div>, <div class="source">Bioinformatics</div> , <div class="volume">32</div>, <div class="fpage">529</div>–<div class="lpage">537</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improve%20homology%20search%20sensitivity%20of%20PacBio%20data%20by%20correcting%20frameshifts&amp;author=N.%20Du&amp;author=Y.%20Sun&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=529-537" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btw458" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtw458" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtw458"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improve%20homology%20search%20sensitivity%20of%20PacBio%20data%20by%20correcting%20frameshifts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">HMM-FRAME: accurate protein domain classification for metagenomic sequences containing frameshift errors</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">198</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=HMM-FRAME%3A%20accurate%20protein%20domain%20classification%20for%20metagenomic%20sequences%20containing%20frameshift%20errors&amp;author=Y.%20Zhang&amp;author=Y.%20Sun&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=198" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-12-198" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-198" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-12-198"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21609463" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:HMM-FRAME%3A%20accurate%20protein%20domain%20classification%20for%20metagenomic%20sequences%20containing%20frameshift%20errors&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Ketteler</div> <div class="given-names">R.</div>
</div></div> <div class="year">2012</div>, 
<div class="article-title">On programmed ribosomal frameshifting: the alternative proteomes</div>, <div class="source">Front. Genet</div> , <div class="volume">3</div>, article 242, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=On%20programmed%20ribosomal%20frameshifting%3A%20the%20alternative%20proteomes&amp;author=R.%20Ketteler&amp;publication_year=2012&amp;journal=Front.%20Genet&amp;volume=3&amp;pages=1-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fgene.2012.00242" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2012.00242" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffgene.2012.00242"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22303408" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:On%20programmed%20ribosomal%20frameshifting%3A%20the%20alternative%20proteomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mironov</div> <div class="given-names">A.A.</div>
</div>, <div class="name string-name">
<div class="surname">Novichkov</div> <div class="given-names">P.S.</div>
</div>, <div class="name string-name">
<div class="surname">Gelfand</div> <div class="given-names">M.S.</div>
</div>
</div> <div class="year">2001</div>, 
<div class="article-title">Pro-Frame: similarity-based gene recognition in eukaryotic DNA sequences with errors</div>. <div class="source">Bioinformatics</div> , <div class="volume">17</div>, <div class="fpage">13</div>–<div class="lpage">15</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Pro-Frame%3A%20similarity-based%20gene%20recognition%20in%20eukaryotic%20DNA%20sequences%20with%20errors&amp;author=A.A.%20Mironov&amp;author=P.S.%20Novichkov&amp;author=M.S.%20Gelfand&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=13-15" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/17.1.13" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.1.13" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F17.1.13"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11222258" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Pro-Frame%3A%20similarity-based%20gene%20recognition%20in%20eukaryotic%20DNA%20sequences%20with%20errors&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gîrdea</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Noé</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Kucherov</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Back-translation for discovering distant protein homologies in the presence of frameshift mutation</div>. <div class="source">Algorithms Mol. Biol</div> ., <div class="volume">5</div>(1), 6.</div>
</div></div></div>
<div content-id="dsy046-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schiex</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Gouzy</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Moisan</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">de Oliveira</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">FrameD: a flexible program for quality check and gene prediction in prokaryotic genomes and noisy mature deukaryotic sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">3738</div>–<div class="lpage">3741</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=FrameD%3A%20a%20flexible%20program%20for%20quality%20check%20and%20gene%20prediction%20in%20prokaryotic%20genomes%20and%20noisy%20mature%20deukaryotic%20sequences&amp;author=T.%20Schiex&amp;author=J.%20Gouzy&amp;author=A.%20Moisan&amp;author=Y.%20de%20Oliveira&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=3738-3741" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkg610" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg610" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkg610"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824407" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FrameD%3A%20a%20flexible%20program%20for%20quality%20check%20and%20gene%20prediction%20in%20prokaryotic%20genomes%20and%20noisy%20mature%20deukaryotic%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Antonov</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Borodovsky</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Genetack: frameshift identification in protein-coding sequences by the Viterbi algorithm</div>, <div class="source">J. Bioinform. Comput. Biol</div> ., <div class="volume">8</div>, <div class="fpage">535</div>–<div class="lpage">551</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetack%3A%20frameshift%20identification%20in%20protein-coding%20sequences%20by%20the%20Viterbi%20algorithm&amp;author=I.%20Antonov&amp;author=M.%20Borodovsky&amp;publication_year=2010&amp;journal=J.%20Bioinform.%20Comput.%20Biol&amp;volume=8&amp;pages=535-551" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1142/S0219720010004847" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1142%2FS0219720010004847" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1142%2FS0219720010004847"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20556861" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetack%3A%20frameshift%20identification%20in%20protein-coding%20sequences%20by%20the%20Viterbi%20algorithm&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Frenkel</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Using triplet periodicity of nucleotide sequences for finding potential reading frame shifts in genes</div>, <div class="source">DNA Res</div> ., <div class="volume">16</div>, <div class="fpage">105</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Using%20triplet%20periodicity%20of%20nucleotide%20sequences%20for%20finding%20potential%20reading%20frame%20shifts%20in%20genes&amp;author=F.%20Frenkel&amp;author=E.V.%20Korotkov&amp;publication_year=2009&amp;journal=DNA%20Res&amp;volume=16&amp;pages=105-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsp002" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp002" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsp002"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261626" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20triplet%20periodicity%20of%20nucleotide%20sequences%20for%20finding%20potential%20reading%20frame%20shifts%20in%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gouzy</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Carrere</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Schiex</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">FrameDP: sensitive peptide detection on noisy matured sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">670</div>–<div class="lpage">671</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=FrameDP%3A%20sensitive%20peptide%20detection%20on%20noisy%20matured%20sequences&amp;author=J.%20Gouzy&amp;author=S.%20Carrere&amp;author=T.%20Schiex&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=670-671" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp024" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp024" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp024"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19153134" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FrameDP%3A%20sensitive%20peptide%20detection%20on%20noisy%20matured%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rho</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Ye</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">FragGeneScan: predicting genes in short and error-prone reads</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">38</div>, <div class="fpage">e191</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=FragGeneScan%3A%20predicting%20genes%20in%20short%20and%20error-prone%20reads&amp;author=M.%20Rho&amp;author=H.%20Tang&amp;author=Y.%20Ye&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=e191" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkq747" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq747" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkq747"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805240" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FragGeneScan%3A%20predicting%20genes%20in%20short%20and%20error-prone%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Antonov</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Baranov</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Borodovsky</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">GeneTack database: genes with frameshifts in prokaryotic genomes and eukaryotic mRNA sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D152</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GeneTack%20database%3A%20genes%20with%20frameshifts%20in%20prokaryotic%20genomes%20and%20eukaryotic%20mRNA%20sequences&amp;author=I.%20Antonov&amp;author=P.%20Baranov&amp;author=M.%20Borodovsky&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D152-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks1062" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1062" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks1062"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161689" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GeneTack%20database%3A%20genes%20with%20frameshifts%20in%20prokaryotic%20genomes%20and%20eukaryotic%20mRNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Azad</div> <div class="given-names">R.K.</div>
</div>, <div class="name string-name">
<div class="surname">Borodovsky</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Probabilistic methods of identifying genes in prokaryotic genomes: connections to the HMM theory</div>, <div class="source">Brief. Bioinform</div> ., <div class="volume">5</div>, <div class="fpage">118</div>–<div class="lpage">30</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Probabilistic%20methods%20of%20identifying%20genes%20in%20prokaryotic%20genomes%3A%20connections%20to%20the%20HMM%20theory&amp;author=R.K.%20Azad&amp;author=M.%20Borodovsky&amp;publication_year=2004&amp;journal=Brief.%20Bioinform&amp;volume=5&amp;pages=118-30" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bib/5.2.118" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2F5.2.118" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbib%2F5.2.118"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15260893" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Probabilistic%20methods%20of%20identifying%20genes%20in%20prokaryotic%20genomes%3A%20connections%20to%20the%20HMM%20theory&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gutiérrez</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Oliver</div> <div class="given-names">J.L.</div>
</div>, <div class="name string-name">
<div class="surname">Marín</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">1994</div>, 
<div class="article-title">On the origin of the periodicity of three in protein coding DNA sequences</div>, <div class="source">J. Theor. Biol</div> ., <div class="volume">167</div>, <div class="fpage">413</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=On%20the%20origin%20of%20the%20periodicity%20of%20three%20in%20protein%20coding%20DNA%20sequences&amp;author=G.%20Guti%C3%A9rrez&amp;author=J.L.%20Oliver&amp;author=A.%20Mar%C3%ADn&amp;publication_year=1994&amp;journal=J.%20Theor.%20Biol&amp;volume=167&amp;pages=413-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1006/jtbi.1994.1080" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1994.1080" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1006%2Fjtbi.1994.1080"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8207954" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:On%20the%20origin%20of%20the%20periodicity%20of%20three%20in%20protein%20coding%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chechetkin</div> <div class="given-names">V.R.</div>
</div>, <div class="name string-name">
<div class="surname">Turygin</div> <div class="given-names">A.Yu.</div>
</div>
</div> <div class="year">1995</div>, 
<div class="article-title">Search of hidden periodicities in DNA sequences</div>, <div class="source">J. Theor. Biol</div> ., <div class="volume">175</div>, <div class="fpage">477</div>–<div class="lpage">94</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Search%20of%20hidden%20periodicities%20in%20DNA%20sequences&amp;author=V.R.%20Chechetkin&amp;author=A.Yu.%20Turygin&amp;publication_year=1995&amp;journal=J.%20Theor.%20Biol&amp;volume=175&amp;pages=477-94" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1006/jtbi.1995.0155" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1995.0155" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1006%2Fjtbi.1995.0155"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7475085" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Search%20of%20hidden%20periodicities%20in%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gao</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Qi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Cao</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Tung</div> <div class="given-names">W.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Protein coding sequence identification by simultaneously characterizing the periodic and random features of DNA sequences</div>, <div class="source">J. Biomed. Biotechnol</div> ., <div class="volume">2005</div>, <div class="fpage">139</div>–<div class="lpage">46</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Protein%20coding%20sequence%20identification%20by%20simultaneously%20characterizing%20the%20periodic%20and%20random%20features%20of%20DNA%20sequences&amp;author=J.%20Gao&amp;author=Y.%20Qi&amp;author=Y.%20Cao&amp;author=W.%20Tung&amp;publication_year=2005&amp;journal=J.%20Biomed.%20Biotechnol&amp;volume=2005&amp;pages=139-46" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1155/JBB.2005.139" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2FJBB.2005.139" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1155%2FJBB.2005.139"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16046819" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Protein%20coding%20sequence%20identification%20by%20simultaneously%20characterizing%20the%20periodic%20and%20random%20features%20of%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yin</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Yau</div> <div class="given-names">S.S.-T.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Prediction of protein coding regions by the 3-base periodicity analysis of a DNA sequence</div>, <div class="source">J. Theor. Biol</div> ., <div class="volume">247</div>, <div class="fpage">687</div>–<div class="lpage">94</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20protein%20coding%20regions%20by%20the%203-base%20periodicity%20analysis%20of%20a%20DNA%20sequence&amp;author=C.%20Yin&amp;author=S.S.-T.%20Yau&amp;publication_year=2007&amp;journal=J.%20Theor.%20Biol&amp;volume=247&amp;pages=687-94" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.jtbi.2007.03.038" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2007.03.038" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jtbi.2007.03.038"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17509616" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Prediction%20of%20protein%20coding%20regions%20by%20the%203-base%20periodicity%20analysis%20of%20a%20DNA%20sequence&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Masoom</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Datta</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Asif</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Cunningham</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2006</div>, A fast algorithm for detecting frame shifts in DNA sequences. In: <em>Proceedings of the 2006 IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology, CIBCB’06</em>. IEEE Computer Society Press, Toronto, vol. 1, pp. 1–8, Sep. 28–29, 2006.</div>
</div></div></div>
<div content-id="dsy046-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Stein</div> <div class="given-names">L.D.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Localizing triplet periodicity in DNA and cDNA sequences</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">550</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Localizing%20triplet%20periodicity%20in%20DNA%20and%20cDNA%20sequences&amp;author=L.%20Wang&amp;author=L.D.%20Stein&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=550" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-11-550" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-550" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-11-550"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21059240" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Localizing%20triplet%20periodicity%20in%20DNA%20and%20cDNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Frenkel</div> <div class="given-names">F.E.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Using triplet periodicity of nucleotide sequences for finding potential reading frame shifts in genes</div>, <div class="source">DNA Res</div> ., <div class="volume">16</div>, <div class="fpage">105</div>–<div class="lpage">114</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Using%20triplet%20periodicity%20of%20nucleotide%20sequences%20for%20finding%20potential%20reading%20frame%20shifts%20in%20genes&amp;author=F.E.%20Frenkel&amp;author=E.V.%20Korotkov&amp;publication_year=2009&amp;journal=DNA%20Res&amp;volume=16&amp;pages=105-114" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsp002" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp002" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsp002"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261626" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20triplet%20periodicity%20of%20nucleotide%20sequences%20for%20finding%20potential%20reading%20frame%20shifts%20in%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Korotkova</div> <div class="given-names">M.A.</div>
</div>, <div class="name string-name">
<div class="surname">Kudryashov</div> <div class="given-names">N.A.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">An approach for searching insertions in bacterial genes leading to the phase shift of triplet periodicity</div>, <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">158</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20approach%20for%20searching%20insertions%20in%20bacterial%20genes%20leading%20to%20the%20phase%20shift%20of%20triplet%20periodicity&amp;author=M.A.%20Korotkova&amp;author=N.A.%20Kudryashov&amp;author=E.V.%20Korotkov&amp;publication_year=2011&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=9&amp;pages=158-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S1672-0229(11)60019-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1672-0229(11)60019-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS1672-0229(11)60019-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22196359" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20approach%20for%20searching%20insertions%20in%20bacterial%20genes%20leading%20to%20the%20phase%20shift%20of%20triplet%20periodicity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Frenkel</div> <div class="given-names">F.E.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Classification analysis of triplet periodicity in protein-coding regions of genes</div>, <div class="source">Gene</div> , <div class="volume">421</div>, <div class="fpage">52</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Classification%20analysis%20of%20triplet%20periodicity%20in%20protein-coding%20regions%20of%20genes&amp;author=F.E.%20Frenkel&amp;author=E.V.%20Korotkov&amp;publication_year=2008&amp;journal=Gene&amp;volume=421&amp;pages=52-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gene.2008.06.012" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.06.012" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2008.06.012"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593596" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Classification%20analysis%20of%20triplet%20periodicity%20in%20protein-coding%20regions%20of%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pugacheva</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Search of latent periodicity in amino acid sequences by means of genetic algorithm and dynamic programming</div>, <div class="source">Stat. Appl. Genet. Mol. Biol</div> ., <div class="volume">15</div>, <div class="fpage">381</div>–<div class="lpage">400</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Search%20of%20latent%20periodicity%20in%20amino%20acid%20sequences%20by%20means%20of%20genetic%20algorithm%20and%20dynamic%20programming&amp;author=V.%20Pugacheva&amp;author=A.%20Korotkov&amp;author=E.%20Korotkov&amp;publication_year=2016&amp;journal=Stat.%20Appl.%20Genet.%20Mol.%20Biol&amp;volume=15&amp;pages=381-400" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1515/sagmb-2015-0079" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1515%2Fsagmb-2015-0079" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1515%2Fsagmb-2015-0079"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27337743" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Search%20of%20latent%20periodicity%20in%20amino%20acid%20sequences%20by%20means%20of%20genetic%20algorithm%20and%20dynamic%20programming&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Frenkel</div> <div class="given-names">F.E.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Classification analysis of triplet periodicity in protein-coding regions of genes</div>, <div class="source">Gene</div> , <div class="volume">421</div>, <div class="fpage">52</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Classification%20analysis%20of%20triplet%20periodicity%20in%20protein-coding%20regions%20of%20genes&amp;author=F.E.%20Frenkel&amp;author=E.V.%20Korotkov&amp;publication_year=2008&amp;journal=Gene&amp;volume=421&amp;pages=52-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gene.2008.06.012" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.06.012" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gene.2008.06.012"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593596" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Classification%20analysis%20of%20triplet%20periodicity%20in%20protein-coding%20regions%20of%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pugacheva</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2016</div>, Search for latent periodicity in amino acid sequences with insertions and deletions. In: <em>BIOINFORMATICS 2016—7th International Conference on Bioinformatics Models, Methods and Algorithms, Proceedings; Part of 9th International Joint Conference on Biomedical Engineering Systems and Technologies, BIOSTEC 2016</em>. SCITEPRESS - Science and technology Publications, Ltd., Vol.3, 117–127.</div>
</div></div></div>
<div content-id="dsy046-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Laskin</div> <div class="given-names">A.A.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>, <div class="name string-name">
<div class="surname">Chaley</div> <div class="given-names">M.B.</div>
</div>, <div class="name string-name">
<div class="surname">Kudryashov</div> <div class="given-names">N.A.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">The locally optimal method of cyclic alignment to reveal latent periodicities in genetic texts: the NAD-binding protein sites</div>, <div class="source">Mol. Biol</div> ., <div class="volume">37</div>, <div class="fpage">663</div>–<div class="lpage">673</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20locally%20optimal%20method%20of%20cyclic%20alignment%20to%20reveal%20latent%20periodicities%20in%20genetic%20texts%3A%20the%20NAD-binding%20protein%20sites&amp;author=A.A.%20Laskin&amp;author=E.V.%20Korotkov&amp;author=M.B.%20Chaley&amp;author=N.A.%20Kudryashov&amp;publication_year=2003&amp;journal=Mol.%20Biol&amp;volume=37&amp;pages=663-673" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1023/A:1025139427862" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1025139427862" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1023%2FA:1025139427862"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20locally%20optimal%20method%20of%20cyclic%20alignment%20to%20reveal%20latent%20periodicities%20in%20genetic%20texts%3A%20the%20NAD-binding%20protein%20sites&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Needleman</div> <div class="given-names">S.B.</div>
</div>, <div class="name string-name">
<div class="surname">Wunsch</div> <div class="given-names">C.D.</div>
</div>
</div> <div class="year">1970</div>, 
<div class="article-title">A general method applicable to the search for similarities in the amino acid sequence of two proteins</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">48</div>, <div class="fpage">443</div>–<div class="lpage">53</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20general%20method%20applicable%20to%20the%20search%20for%20similarities%20in%20the%20amino%20acid%20sequence%20of%20two%20proteins&amp;author=S.B.%20Needleman&amp;author=C.D.%20Wunsch&amp;publication_year=1970&amp;journal=J.%20Mol.%20Biol&amp;volume=48&amp;pages=443-53" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/0022-2836(70)90057-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(70)90057-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0022-2836(70)90057-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5420325" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20general%20method%20applicable%20to%20the%20search%20for%20similarities%20in%20the%20amino%20acid%20sequence%20of%20two%20proteins&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Antonov</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Coakley</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Atkins</div> <div class="given-names">J.F.</div>
</div>, <div class="name string-name">
<div class="surname">Baranov</div> <div class="given-names">P.V.</div>
</div>, <div class="name string-name">
<div class="surname">Borodovsky</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Identification of the nature of reading frame transitions observed in prokaryotic genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">6514</div>–<div class="lpage">30</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20nature%20of%20reading%20frame%20transitions%20observed%20in%20prokaryotic%20genomes&amp;author=I.%20Antonov&amp;author=A.%20Coakley&amp;author=J.F.%20Atkins&amp;author=P.V.%20Baranov&amp;author=M.%20Borodovsky&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=6514-30" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkt274" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt274" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkt274"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23649834" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20the%20nature%20of%20reading%20frame%20transitions%20observed%20in%20prokaryotic%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Frenkel</div> <div class="given-names">F.E.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Classification of triplet periodicity in the DNA sequences of genes from KEGG databank</div>, <div class="source">Mol. Biol</div> ., <div class="volume">42</div>, <div class="fpage">707</div>–<div class="lpage">720</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Classification%20of%20triplet%20periodicity%20in%20the%20DNA%20sequences%20of%20genes%20from%20KEGG%20databank&amp;author=F.E.%20Frenkel&amp;author=E.V.%20Korotkov&amp;publication_year=2008&amp;journal=Mol.%20Biol&amp;volume=42&amp;pages=707-720" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1134/S0026893308040201" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1134%2FS0026893308040201" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1134%2FS0026893308040201"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Classification%20of%20triplet%20periodicity%20in%20the%20DNA%20sequences%20of%20genes%20from%20KEGG%20databank&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cunningham</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Amode</div> <div class="given-names">M.R.</div>
</div>, <div class="name string-name">
<div class="surname">Barrell</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Ensembl 2015</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D662</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ensembl%202015&amp;author=F.%20Cunningham&amp;author=M.R.%20Amode&amp;author=D.%20Barrell&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D662-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gku1010" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1010" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgku1010"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25352552" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ensembl%202015&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Korotkov</div> <div class="given-names">E.V.</div>
</div>, <div class="name string-name">
<div class="surname">Korotkova</div> <div class="given-names">M.A.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Study of the triplet periodicity phase shifts in genes</div>, <div class="source">J. Integr. Bioinform</div> ., <div class="volume">7</div>, <div class="fpage">131</div>–<div class="lpage">142</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Study%20of%20the%20triplet%20periodicity%20phase%20shifts%20in%20genes&amp;aulast=Korotkov&amp;date=2010&amp;spage=131&amp;epage=142&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Study%20of%20the%20triplet%20periodicity%20phase%20shifts%20in%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Ohno</div> <div class="given-names">S.</div>
</div></div> <div class="year">1970</div>, <div class="source">Evolution by Gene Duplication.</div>  
<div class="publisher-name">Springer Berlin Heidelberg</div>: 
<div class="publisher-loc">Berlin, Heidelberg</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Evolution%20by%20Gene%20Duplication.&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Evolution%20by%20Gene%20Duplication.&amp;aulast=Ohno&amp;date=1970" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20by%20Gene%20Duplication.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Evolution%20by%20Gene%20Duplication.">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Koonin</div> <div class="given-names">E.V.</div>
</div></div> <div class="year">2005</div>, 
<div class="article-title">Orthologs, paralogs, and evolutionary genomics</div>, <div class="source">Annu. Rev. Genet</div> ., <div class="volume">39</div>, <div class="fpage">309</div>–<div class="lpage">38</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Orthologs%2C%20paralogs%2C%20and%20evolutionary%20genomics&amp;author=E.V.%20Koonin&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Genet&amp;volume=39&amp;pages=309-38" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev.genet.39.073003.114725" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.39.073003.114725" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev.genet.39.073003.114725"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16285863" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Orthologs%2C%20paralogs%2C%20and%20evolutionary%20genomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Thomson</div> <div class="given-names">T.M.</div>
</div>, <div class="name string-name">
<div class="surname">Lozano</div> <div class="given-names">J.J.</div>
</div>, <div class="name string-name">
<div class="surname">Loukili</div> <div class="given-names">N.</div>
</div>
</div>, et al.  <div class="year">2000</div>, 
<div class="article-title">Fusion of the human gene for the polyubiquitination coeffector UEV1 with Kua, a newly identified gene</div>, <div class="source">Genome Res</div> ., <div class="volume">10</div>, <div class="fpage">1743</div>–<div class="lpage">56</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fusion%20of%20the%20human%20gene%20for%20the%20polyubiquitination%20coeffector%20UEV1%20with%20Kua%2C%20a%20newly%20identified%20gene&amp;author=T.M.%20Thomson&amp;author=J.J.%20Lozano&amp;author=N.%20Loukili&amp;publication_year=2000&amp;journal=Genome%20Res&amp;volume=10&amp;pages=1743-56" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.GR-1405R" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.GR-1405R" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.GR-1405R"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076860" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fusion%20of%20the%20human%20gene%20for%20the%20polyubiquitination%20coeffector%20UEV1%20with%20Kua%2C%20a%20newly%20identified%20gene&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Gilbert</div> <div class="given-names">W.</div>
</div></div> <div class="year">1978</div>, 
<div class="article-title">Why genes in pieces?,</div> <div class="source">Nature</div> , <div class="volume">271</div>, <div class="fpage">501</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Why%20genes%20in%20pieces%3F%2C&amp;author=W.%20Gilbert&amp;publication_year=1978&amp;journal=Nature&amp;volume=271&amp;pages=501" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/271501a0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F271501a0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F271501a0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/622185" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Why%20genes%20in%20pieces%3F%2C&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hiller</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Huse</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Platzer</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Backofen</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Creation and disruption of protein features by alternative splicing—a novel mechanism to modulate function</div>, <div class="source">Genome Biol</div> ., <div class="volume">6</div>, <div class="fpage">R58</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Creation%20and%20disruption%20of%20protein%20features%20by%20alternative%20splicing%E2%80%94a%20novel%20mechanism%20to%20modulate%20function&amp;author=M.%20Hiller&amp;author=K.%20Huse&amp;author=M.%20Platzer&amp;author=R.%20Backofen&amp;publication_year=2005&amp;journal=Genome%20Biol&amp;volume=6&amp;pages=R58" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2005-6-7-r58" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-7-r58" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2005-6-7-r58"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15998447" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Creation%20and%20disruption%20of%20protein%20features%20by%20alternative%20splicing%E2%80%94a%20novel%20mechanism%20to%20modulate%20function&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy046-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy046-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Ochman</div> <div class="given-names">H.</div>
</div></div> <div class="year">2001</div>, 
<div class="article-title">Lateral and oblique gene transfer</div>, <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">11</div>, <div class="fpage">616</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Lateral%20and%20oblique%20gene%20transfer&amp;author=H.%20Ochman&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=11&amp;pages=616-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0959-437X(00)00243-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(00)00243-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0959-437X(00)00243-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11682303" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Lateral%20and%20oblique%20gene%20transfer&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-20127018" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Toh</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Toh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toh%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Toh%20Hiroyuki%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Hiroyuki Toh</a>
                            </span>
                        <br>
</div>
            </div>
        </div>
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5306610&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,130</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy046"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5306610"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f2%2f157%2f5306610">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F2%2F157%2F5306610" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5306610">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Search for potential reading frameshifts in cds from Arabidopsis thaliana and other genomes | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy046");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f157%2f5306610">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
