<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Intragenomic heterogeneity of intergenic ribosomal DNA spacers in Cucurbita moschata is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/3/273/5479970","@type":"ScholarlyArticle","name":"Intragenomic heterogeneity of intergenic ribosomal DNA spacers in  Cucurbita moschata  is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations","datePublished":"2019-04-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2019-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz008","keywords":["<italic>Cucurbita moschata</italic>","ribosomal DNA intergenic spacer","DNA-minisatellite","intragenomic structural heterogeneity","non-canonical DNA conformations"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/3/273/5479970","author":[{"name":"Matyášek, Roman","affiliation":"Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic","@type":"Person"},{"name":"Kuderová, Alena","affiliation":"Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic","@type":"Person"},{"name":"Kutílková, Eva","affiliation":"Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic","@type":"Person"},{"name":"Kučera, Marek","affiliation":"Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic","@type":"Person"},{"name":"Kovařík, Aleš","affiliation":"Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic","@type":"Person"}],"description":"Abstract.  The intergenic spacer (IGS) of rDNA is frequently built of long blocks of tandem repeats. To estimate the intragenomic variability of such knotty reg","pageStart":"273","pageEnd":"286","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636521572&Signature=G4Q-OgvwWUKbgjjxZ9h4fO7gNQH4CVspv~j3SdBtMQxIr8pvR8moPPxXJZYEClD3QMmg-BGK-s6K22VlpGAFnyd8evl8~tBg2rvk2PZ56Riz9jwtxFIBGB8XmtpRoQerUCfA2yho-WT8SqlSYXg~OcnvQtkpL1vt0UYGlLPSHUGm1SaALTtX0b1ZyuqneP42pS6dtyf5wXCm6hpQVNrM9naxgAep-8nCNBRcrV5O99gnTGGeRZtyET2Hny-sN0gXN8GLZhLCVFoQg1cws8QkM6eFwqJsE1HBrskRQu5z7vwBIblX1wOOHbYxzyDQUZcipcDN7uEefD033goPLf~qtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Intragenomic heterogeneity of intergenic ribosomal DNA spacers in  Cucurbita moschata  is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636521572&Signature=G4Q-OgvwWUKbgjjxZ9h4fO7gNQH4CVspv~j3SdBtMQxIr8pvR8moPPxXJZYEClD3QMmg-BGK-s6K22VlpGAFnyd8evl8~tBg2rvk2PZ56Riz9jwtxFIBGB8XmtpRoQerUCfA2yho-WT8SqlSYXg~OcnvQtkpL1vt0UYGlLPSHUGm1SaALTtX0b1ZyuqneP42pS6dtyf5wXCm6hpQVNrM9naxgAep-8nCNBRcrV5O99gnTGGeRZtyET2Hny-sN0gXN8GLZhLCVFoQg1cws8QkM6eFwqJsE1HBrskRQu5z7vwBIblX1wOOHbYxzyDQUZcipcDN7uEefD033goPLf~qtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Intragenomic heterogeneity of intergenic ribosomal DNA spacers in  Cucurbita moschata  is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations">
<meta property="og:description" content="Abstract.  The intergenic spacer (IGS) of rDNA is frequently built of long blocks of tandem repeats. To estimate the intragenomic variability of such knotty reg">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/3/273/5479970">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636521572&amp;Signature=G4Q-OgvwWUKbgjjxZ9h4fO7gNQH4CVspv~j3SdBtMQxIr8pvR8moPPxXJZYEClD3QMmg-BGK-s6K22VlpGAFnyd8evl8~tBg2rvk2PZ56Riz9jwtxFIBGB8XmtpRoQerUCfA2yho-WT8SqlSYXg~OcnvQtkpL1vt0UYGlLPSHUGm1SaALTtX0b1ZyuqneP42pS6dtyf5wXCm6hpQVNrM9naxgAep-8nCNBRcrV5O99gnTGGeRZtyET2Hny-sN0gXN8GLZhLCVFoQg1cws8QkM6eFwqJsE1HBrskRQu5z7vwBIblX1wOOHbYxzyDQUZcipcDN7uEefD033goPLf~qtQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-04-25">
<meta name="citation_author" content="Matyášek, Roman">
<meta name="citation_author_institution" content="Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic">
<meta name="citation_author" content="Kuderová, Alena">
<meta name="citation_author_institution" content="Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic">
<meta name="citation_author" content="Kutílková, Eva">
<meta name="citation_author_institution" content="Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic">
<meta name="citation_author" content="Kučera, Marek">
<meta name="citation_author_institution" content="Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic">
<meta name="citation_author" content="Kovařík, Aleš">
<meta name="citation_author_institution" content="Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic">
<meta name="citation_title" content="Intragenomic heterogeneity of intergenic ribosomal DNA spacers in Cucurbita moschata is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations">
<meta name="citation_firstpage" content="273">
<meta name="citation_lastpage" content="286">
<meta name="citation_doi" content="10.1093/dnares/dsz008">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="3">
<meta name="citation_publication_date" content="2019/06/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Structural organization and regulation of transcription by RNA polymerase I of plant nuclear ribosomal RNA genes;  citation_journal_title=Results Probl. Cell Differ;  citation_year=1994;  citation_volume=20;  citation_pages=3-24; ">
<meta name="citation_reference" content="citation_title=Interpopulation hybridization generates meiotically stable rDNA epigenetic variants in allotetraploid Tragopogon mirus;  citation_journal_title=Plant J;  citation_year=2016;  citation_volume=85;  citation_pages=362-77; ">
<meta name="citation_reference" content="citation_title=The differential expression of ribosomal-RNA genes;  citation_journal_title=Philos. Trans. Roy. Soc. B;  citation_year=1986; ">
<meta name="citation_reference" content="citation_title=Partial inactivation of wheat nucleolus organizers by the nucleolus organizer chromosomes from Aegilops-Umbellulata;  citation_journal_title=Chromosoma;  citation_year=1982;  citation_volume=84;  citation_pages=687-700; ">
<meta name="citation_reference" content="citation_title=Interplay of ribosomal DNA loci in nucleolar dominance: dominant NORs are up-regulated by chromatin dynamics in the wheat-rye system;  citation_journal_title=Plos One;  citation_year=2008; ">
<meta name="citation_reference" content="citation_title=Organization, differential expression and methylation of rDNA in artificial Solanum allopolyploids;  citation_journal_title=Plant Mol. Biol;  citation_year=2004;  citation_volume=56;  citation_pages=439-63; ">
<meta name="citation_reference" content="citation_title=Transcriptional analysis of nucleolar dominance in polyploid plants: biased expression/silencing of progenitor rRNA genes is developmentally regulated in Brassica;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=3442-7; ">
<meta name="citation_reference" content="citation_title=Ribosomal transcription units integrated via T-DNA transformation associate with the nucleolus and do not require upstream repeat sequences for activity in Arabidopsis thaliana;  citation_journal_title=Plant J;  citation_year=1997;  citation_volume=11;  citation_pages=1007-16; ">
<meta name="citation_reference" content="citation_title=Functional-analysis of Arabidopsis thaliana ribosomal-RNA gene and spacer promoters in-vivo and by transient expression;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=7528-32; ">
<meta name="citation_reference" content="citation_title=Complex-formation of nuclear proteins with the RNA polymerase-I promoter and repeated elements in the external transcribed spacer of Cucumis sativus ribosomal DNA;  citation_journal_title=Nucleic Acids Res;  citation_year=1992;  citation_volume=20;  citation_pages=3685-91; ">
<meta name="citation_reference" content="citation_title=Molecular evolution of 35S rDNA and taxonomic status of Lycopersicon within Solanum sect. Petota;  citation_journal_title=Plant Syst. Evol;  citation_year=2008;  citation_volume=276;  citation_pages=59-71; ">
<meta name="citation_reference" content="citation_title=Termination of transcription of ribosomal-RNA genes of mung bean occurs within a 175-bp repetitive element of the spacer region;  citation_journal_title=Mol. Gen. Genet;  citation_year=1989;  citation_volume=218;  citation_pages=302-7; ">
<meta name="citation_reference" content="citation_title=Finely orchestrated movements: evolution of the ribosomal RNA genes;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=175;  citation_pages=477-85; ">
<meta name="citation_reference" content="citation_title=Variation of 45S rDNA intergenic spacers in Arabidopsis thaliana;  citation_journal_title=Plant Mol. Biol;  citation_year=2016;  citation_volume=92;  citation_pages=457-71; ">
<meta name="citation_reference" content="citation_title=Epigenetic control of RNA polymerase I transcription in mammalian cells,;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2013;  citation_volume=1829;  citation_pages=393-404; ">
<meta name="citation_reference" content="citation_title=A duplicated NUCLEOLIN gene with antagonistic activity is required for chromatin organization of silent 45S rDNA in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=1330-44; ">
<meta name="citation_reference" content="citation_title=Dual-color FISH karyotype analyses using rDNAs in three Cucurbitaceae species;  citation_journal_title=Genes Genom;  citation_year=2011;  citation_volume=33;  citation_pages=521-8; ">
<meta name="citation_reference" content="citation_title=Molecular evolution of the intergenic spacer in the nuclear ribosomal-RNA genes of Cucurbitaceae;  citation_journal_title=J. Mol. Evol;  citation_year=1993;  citation_volume=36;  citation_pages=144-52; ">
<meta name="citation_reference" content="citation_title=Heterogeneity of pumpkin ribosomal DNA;  citation_journal_title=Plant Physiol;  citation_year=1983;  citation_volume=72;  citation_pages=166-71; ">
<meta name="citation_reference" content="citation_title=Analysis of two abundant, highly related satellites in the allotetraploid Nicotiana arentsii using double-strand conformation polymorphism analysis and sequencing;  citation_journal_title=New Phytol;  citation_year=2011;  citation_volume=192;  citation_pages=747-59; ">
<meta name="citation_reference" content="citation_title=Gene conversion of ribosomal DNA in Nicotiana tabacum is associated with undermethylated, decondensed and probably active gene units;  citation_journal_title=Chromosoma;  citation_year=2000;  citation_volume=109;  citation_pages=161-72; ">
<meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; ">
<meta name="citation_reference" content="citation_title=QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W676-82; ">
<meta name="citation_reference" content="citation_title=Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=477-86; ">
<meta name="citation_reference" content="citation_title=Estimation of the number of nucleotide substitutions in the control region of mitochondrial-DNA in humans and chimpanzees;  citation_journal_title=Mol. Biol. Evol;  citation_year=1993;  citation_volume=10;  citation_pages=512-26; ">
<meta name="citation_reference" content="citation_title=BAC ‘landing’ on chromosomes of Brachypodium distachyon for comparative genome alignment;  citation_journal_title=Nat. Protoc;  citation_year=2007;  citation_volume=2;  citation_pages=88-98; ">
<meta name="citation_reference" content="citation_title=Dynamic changes in the distribution of a satellite homologous to intergenic 26-18S rDNA spacer in the evolution of Nicotiana;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=1935-46; ">
<meta name="citation_reference" content="citation_title=Mini- and microsatellite expansions: the recombination connection;  citation_journal_title=EMBO Rep;  citation_year=2000;  citation_volume=1;  citation_pages=122-6; ">
<meta name="citation_reference" content="citation_title=Variable tandem repeats accelerate evolution of coding and regulatory sequences;  citation_journal_title=Annu. Rev. Genet;  citation_year=2010;  citation_volume=44;  citation_pages=445-77; ">
<meta name="citation_reference" content="citation_title=Multiple repeats of a promoter segment causes transcription factor autoregulation in red apples;  citation_journal_title=Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=168-83; ">
<meta name="citation_reference" content="citation_title=Specific binding of nuclear proteins to the promoter region of a maize nuclear ribosomal-RNA gene unit;  citation_journal_title=J. Biol. Chem;  citation_year=1989;  citation_volume=264;  citation_pages=1467-72; ">
<meta name="citation_reference" content="citation_title=Xenopus ribosomal RNA gene intergenic spacer elements conferring transcriptional enhancement and nucleolar dominance-like competition in oocytes;  citation_journal_title=J. Biol. Chem;  citation_year=2002;  citation_volume=277;  citation_pages=31577-84; ">
<meta name="citation_reference" content="citation_title=A 140-base pair repetitive sequence element in the mouse ribosomal-RNA gene spacer enhances transcription by RNA polymerase-I in a cell-free system;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1990;  citation_volume=87;  citation_pages=7527-31; ">
<meta name="citation_reference" content="citation_title=G quadruplex in plants: a ubiquitous regulatory element and its biological relevance;  citation_journal_title=Front Plant Sci;  citation_year=2017;  citation_volume=8,;  citation_pages=1163">
<meta name="citation_reference" content="citation_title=Z-DNA, an active element in the genome;  citation_journal_title=Front. Biosci;  citation_year=2007;  citation_volume=12;  citation_pages=4424-38; ">
<meta name="citation_reference" content="citation_title=G4IPDB: a database for G-quadruplex structure forming nucleic acid interacting proteins;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=38144">
<meta name="citation_reference" content="citation_title=Nucleophosmin mutations alter its nucleolar localization by impairing G-quadruplex binding at ribosomal DNA;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=3228-39; ">
<meta name="citation_reference" content="citation_title=Transient hoogsteen base pairs in canonical duplex DNA;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=470; ">
<meta name="citation_reference" content="citation_title=Evidence of the formation of G-quadruplex structures in the promoter region of the human vascular endothelial growth factor gene;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=1256-65; ">
<meta name="citation_reference" content="citation_title=Reaction path ensemble of the B-Z-DNA transition: a comprehensive atomistic study;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=33-43; ">
<meta name="citation_reference" content="citation_title=The role of methylation in the intrinsic dynamics of B- and Z-DNA;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e35558">
<meta name="citation_reference" content="citation_title=Silenced rRNA genes are activated and substitute for partially eliminated active homeologs in the recently formed allotetraploid, Tragopogon mirus (Asteraceae);  citation_journal_title=Heredity;  citation_year=2015;  citation_volume=114;  citation_pages=356-65; ">
<meta name="citation_reference" content="citation_title=Sequence complexity and DNA curvature;  citation_journal_title=Comput. Chem;  citation_year=1999;  citation_volume=23;  citation_pages=263-74; ">
<meta name="citation_reference" content="citation_title=Plant rDNA database: ribosomal DNA loci information goes online;  citation_journal_title=Chromosoma;  citation_year=2012;  citation_volume=121;  citation_pages=389-94; ">
<meta name="citation_reference" content="citation_title=Transcription activity of rRNA genes correlates with a tendency towards intergenomic homogenization in Nicotiana allotetraploids;  citation_journal_title=New Phytol;  citation_year=2007;  citation_volume=174;  citation_pages=658-68; ">
<meta name="citation_reference" content="citation_title=Gene conversion events and variable degree of homogenization of rDNA loci in cultivars of Brassica napus;  citation_journal_title=Ann. Bot;  citation_year=2017;  citation_volume=119;  citation_pages=13-26; ">
<meta name="citation_reference" content="citation_title=Nucleotide sequence, structural organization and length heterogeneity of ribosomal DNA intergenic spacer in Quercus petraea (Matt.) Liebl. and Q-robur L;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2009;  citation_volume=281;  citation_pages=207-21; ">
<meta name="citation_reference" content="citation_title=Structural and functional characterization of the intergenic spacer region of the rDNA in Daucus carota;  citation_journal_title=Plant Cell Physiol;  citation_year=1996;  citation_volume=37;  citation_pages=233-8; ">
<meta name="citation_reference" content="citation_title=Evolution of satellite DNAs from the genus Palorus - experimental evidence for the “library” hypothesis;  citation_journal_title=Mol. Biol. Evol;  citation_year=1998;  citation_volume=15;  citation_pages=1062-8; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/3/273/28855582/dsz008.pdf">
<meta name="description" content="Abstract.  The intergenic spacer (IGS) of rDNA is frequently built of long blocks of tandem repeats. To estimate the intragenomic variability of such knotty reg">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/3/273/5479970">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/3/273/5479970">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/3/273/5479970");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2779951,2796125&authnips=128.235.85.32&doi=10.1093/dnares/dsz008';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f273%2f5479970" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f273%2f5479970" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115841"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1576706120&amp;Signature=4Hxpowgov0hukhpdBfzbNfZicK1DU9TgKcB8YJdQ1yPiyZg~d6filgoD1BwTGIoMqWj3L9igJaHR1pys~HDZOPoVKaEAyhgIrzHEBntmZGQXV7AnwBZs1kp5nItgV6MpOrR0zvJkLVE9-3wvyNBFvRFsY-UvppHQNW4RVFMCXb6fKfIOu~RetF6aZm5b6J7-yRkFvgOo2wJb66v8uLsABlusWLyFk3VHnQK0DQBbw1w~giBMJ0iaArmQZmt9kuklyawgitee2cGjImTpvhLs4dr0IEmZFrbfyduhxZygo7O69PSLxCTMnjvbd0PLUk64nOgrNXr25DSPelDERmg4Dg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/3">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="137025611">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025611">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025613">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025613">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025618">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025618">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025636">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025636">3. Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025669">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025669">4. Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025682">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025682">5. Conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025684">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025684">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="137025692">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025692">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/3/261/5476584">&lt; Previous</a>
        </li>        
            <li class="empty arrow"> </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Intragenomic heterogeneity of intergenic ribosomal DNA spacers in <em>Cucurbita moschata</em> is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Roman Matyášek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +420 54 1517 230. Fax +420 54 1211 293. Email: <a href="mailto:matyasek@ibp.cz" target="_blank">matyasek@ibp.cz</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Roman+Maty%c3%a1%c5%a1ek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maty%C3%A1%C5%A1ek%20R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Maty%C3%A1%C5%A1ek%20Roman%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Roman Matyášek</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alena Kuderová</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alena+Kuderov%c3%a1">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuderov%C3%A1%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kuderov%C3%A1%20Alena%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Alena Kuderová</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eva Kutílková</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eva+Kut%c3%adlkov%c3%a1">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kut%C3%ADlkov%C3%A1%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kut%C3%ADlkov%C3%A1%20Eva%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Eva Kutílková</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marek Kučera</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marek+Ku%c4%8dera">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ku%C4%8Dera%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ku%C4%8Dera%20Marek%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Marek Kučera</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aleš Kovařík</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biophysics of the Czech Academy of Sciences, CZ Brno, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ale%c5%a1+Kova%c5%99%c3%adk">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kova%C5%99%C3%ADk%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kova%C5%99%C3%ADk%20Ale%C5%A1%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Aleš Kovařík</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 273–286, <a href="https://doi.org/10.1093/dnares/dsz008">https://doi.org/10.1093/dnares/dsz008</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">25 April 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">16 August 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">03 April 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">25 April 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5479970" href="/dnaresearch/article-pdf/26/3/273/28855582/dsz008.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Roman Matyášek, Alena Kuderová, Eva Kutílková, Marek Kučera, Aleš Kovařík,  Intragenomic heterogeneity of intergenic ribosomal DNA spacers in <em>Cucurbita moschata</em> is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations, <em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 273–286, <a href="https://doi.org/10.1093/dnares/dsz008">https://doi.org/10.1093/dnares/dsz008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5479970&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Intragenomic%20heterogeneity%20of%20intergenic%20ribosomal%20DNA%20spacers%20in%20%26lt%3Bem%26gt%3BCucurbita%20moschata%26lt%3B%2Fem%26gt%3B%20is%20determined%20by%20DNA%20minisatellites%20with%20variable%20potential%20to%20form%20non-canonical%20DNA%20conformations&amp;publicationDate=2019-04-25&amp;volumeNum=26&amp;issueNum=3&amp;author=Maty%C3%A1%C5%A1ek%2C%20Roman%3B%20Kuderov%C3%A1%2C%20Alena&amp;startPage=273&amp;endPage=286&amp;contentId=10.1093%2Fdnares%2Fdsz008&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f273%2f5479970" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="137025611" id="137025611" class="abstract-title">Abstract</h2>
<section class="abstract"><p>The intergenic spacer (IGS) of rDNA is frequently built of long blocks of tandem repeats. To estimate the intragenomic variability of such knotty regions, we employed PacBio sequencing of the <em>Cucurbita moschata</em> genome, in which thousands of rDNA copies are distributed across a number of loci. The rRNA coding regions are highly conserved, indicating intensive interlocus homogenization and/or high selection pressure. However, the IGS exhibits high intragenomic structural diversity. Two repeated blocks, R1 (300–1250 bp) and R2 (290–643 bp), account for most of the IGS variation. They exhibit minisatellite-like features built of multiple periodically spaced short GC-rich sequence motifs with the potential to adopt non-canonical DNA conformations, G-quadruplex-folded and left-handed Z-DNA. The mutual arrangement of these motifs can be used to classify IGS variants into five structural families. Subtle polymorphisms exist within each family due to a variable number of repeats, suggesting the coexistence of an enormous number of IGS variants. The substantial length and structural heterogeneity of IGS minisatellites suggests that the tempo of their divergence exceeds the tempo of the homogenization of rDNA arrays. As frequently occurring among plants, we hypothesize that their instability may influence transcription regulation and/or destabilize rDNA units, possibly spreading them across the genome.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"Cucurbita moschata"'><em>Cucurbita moschata</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"ribosomal DNA intergenic spacer"'>ribosomal DNA intergenic spacer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="DNA-minisatellite">DNA-minisatellite</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"intragenomic structural heterogeneity"'>intragenomic structural heterogeneity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"non-canonical DNA conformations"'>non-canonical DNA conformations</a>
</div>                    <h2 scrollto-destination="137025613" id="137025613" class="section-title">1. Introduction</h2>
<p>In eukaryotes, the 35S nuclear ribosomal DNA (rDNA) units are organized in tandem repeats located at one or more chromosomal loci, the so-called nucleolar organizing region (NOR).<a reveal-id="dsz008-B1" data-open="dsz008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The rDNA unit is composed of the coding regions for the 18S, 5.8S and 26S ribosomal RNA (rRNA); the internal transcribed spacers (ITS1 and ITS2); and the intergenic spacer (IGS). The IGS, located between the 3’ end of the 26S rRNA gene and the 5’ end of the 18S rRNA gene, comprises the 3′ external transcribed spacer (3′-ETS), the non-transcribed spacer, and the 5’-ETS). In general, the IGS is built of multiple blocks of tandem and dispersed repeats that may be arranged in highly complex patterns, resulting in variations between related species, populations and even within an individual.<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The presence of conserved structural features in the IGS, such as repeating elements, sequences with self-complementarity having the potential to generate a conserved secondary structure, the transcription initiation site (TIS) and the transcription termination site, all led to the recognition of the IGS as a functionally important region.<a reveal-id="dsz008-B1" data-open="dsz008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a></p>
<p>The presence of highly variable repeats in the IGS of most higher eukaryotes suggests that these fulfil an important function.<a reveal-id="dsz008-B1" data-open="dsz008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Their function as possible enhancers of RNA polymerase I (Pol I) transcription has been studied predominantly in interspecific hybrids.<a reveal-id="dsz008-B3" data-open="dsz008-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The rDNA loci with a longer IGS, containing more homologous repeats upstream of TIS, transcriptionally dominate over those with a shorter IGS in <em>Triticum-Aegilops</em>,<a reveal-id="dsz008-B4" data-open="dsz008-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>Triticum-Secale</em><a reveal-id="dsz008-B5" data-open="dsz008-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and <em>Tragopon mirus</em><a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> but not in <em>Solanum</em>,<a reveal-id="dsz008-B6" data-open="dsz008-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><em>Brassica</em><a reveal-id="dsz008-B7" data-open="dsz008-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and <em>Arabidopsis.</em><a reveal-id="dsz008-B8" data-open="dsz008-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Repeats from <em>Arabidopsis thaliana</em> can, however, substitute for <em>Xenopus</em> repeats to enhance Pol I transcription.<a reveal-id="dsz008-B9" data-open="dsz008-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Additionally, repeats within the 5′-ETS constitute a recognition site for DNA-interacting proteins, which may stimulate the transcription of rDNA.<a reveal-id="dsz008-B10" data-open="dsz008-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In <em>Tragopogon</em>,<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><em>Solanum</em><a reveal-id="dsz008-B11" data-open="dsz008-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and <em>Vigna radiate</em>,<a reveal-id="dsz008-B1" data-open="dsz008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> transcriptionally dominant rDNA variants harbour more repeats in the 5′-ETS. Repeats localized upstream of the TIS may contain multiple termination signals for Pol I.<a reveal-id="dsz008-B12" data-open="dsz008-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In many species, individual rDNA repeats do not evolve independently but in a concerted manner due to the process of sequence homogenization, resulting in the co-existence of numerous nearly identical units in the same genome.<a reveal-id="dsz008-B13" data-open="dsz008-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> However, accumulating data suggest that multiple rDNA repeat units with different sequence similarities and lengths, mainly due to variable repeats in IGS, can be simultaneously present in the same genome.<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsz008-B14" data-open="dsz008-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The origin of such diversity has not been explained. Nevertheless, only a subset of rRNA genes with distinct structures is transcriptionally active, a phenomenon known as nucleolar dominance in hybrids.<a reveal-id="dsz008-B7" data-open="dsz008-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The interplay of DNA methylation, histone modification, and chromatin remodelling activities guided by ncRNA derived from variable IGSs is required for the specific establishment of chromatin organization in silent 35S rDNA in animals<a reveal-id="dsz008-B15" data-open="dsz008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and plants.<a reveal-id="dsz008-B16" data-open="dsz008-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>
<p><em>Cucurbita pepo</em>, <em>Cucurbita moschata</em>, and <em>Cucurbita maxima</em> are the most economically important cultivated species within <em>Cucurbita</em> with relatively small genome sizes 0.55, 0.43 and 0.46 pg/1C, respectively (Kew Angiosperm DNA C-values database). Isozyme assays and high (relative to other Cucurbitaceae) chromosome numbers (<em>n</em> = 20–24) have led to the suggestion that the genus is of allopolyploid origin.<a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> In <em>C. moschata</em> (2<em>n</em> = 40), the 45S rDNA occupies large parts of five chromosome pairs.<a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> However, sequencing data for IGS are currently not available for this species, and only a single IGS has been completely sequenced in each related species <em>C. pepo</em> and <em>C. maxima</em>, showing that duplicated promoter-like sequences with TIS for Pol I are separated by multiple tandem repeats,<a reveal-id="dsz008-B18" data-open="dsz008-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> some of which may function in enhancing transcription, as shown for the related species <em>Cucumis sativus</em>.<a reveal-id="dsz008-B10" data-open="dsz008-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In <em>C. pepo</em>, restriction polymorphism showed that 3,400 rDNA units fall into seven classes that are distinct in length and/or nucleotide sequence.<a reveal-id="dsz008-B19" data-open="dsz008-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Therefore, nothing is currently known about either the structure of IGS in <em>C. moschata</em> or the distribution of 35S rDNA loci in <em>C. pepo</em> and <em>C. maxima</em> chromosomal complements.</p>
<p>To elucidate the regulation of the cellular rRNA level, we initiated a study of plant IGSs, particularly the regions of initiation and termination of transcription. To estimate the intragenomic structural variability of the IGS, we leveraged the advantage of long reads provided by PacBio sequencing methodology, which is relatively insensitive to highly structured, GC-rich and repetitive, DNA, characteristic of IGS. As a model plant, we selected <em>C. moschata</em>, a species with rDNA units distributed among multiple rDNA loci, suggesting high structural variability. We focussed on two extremely GC-rich repetitive regions located upstream of genic and spacer promoter-like sequences and evaluated their heterogeneity in the content and the distribution of sequence motifs with the potential to adopt non-canonical DNA structures, which are believed to participate in transcription regulation. We further evaluated the frequency of single nucleotide polymorphisms (SNPs) and length polymorphisms caused by short insertions, deletions or variable numbers of subrepeats along the entire rDNA unit, with a focus on comparing genic regions with an IGS and ITS.</p>                    <h2 scrollto-destination="137025618" id="137025618" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="137025619" id="137025619" class="section-title">2.1. Plant material, DNA extraction, and Southern and slot blot hybridizations</h3>
<p>Musquee de Provence (Muscat de Provence) cultivar of <em>C. moschata</em> Duch. ex Poir. and Rouge vif d’Etampes cultivar of <em>C. pepo</em> L. were grown in a garden. Fresh young leaves were ground in liquid nitrogen to a fine powder, which was mixed with 2.5 volumes (v/w) of extraction buffer [1.1 M NaCl, 1.4% cetyltrimethylammonium bromide, 71 mM Tris-HCl (pH 8.0), 14 mM ethylenediaminetetraacetic acid (EDTA), and 0.1% 2-mercaptoethanol] and incubated for 60 min at 60°C with occasional gentle mixing. The extract was gently mixed with an equal volume of chloroform:octanol (24:1). After centrifugation [13,000 × <em>g</em>, 20 min, room temperature (RT)], the DNA was precipitated with 2/3 volume of isopropanol, washed with 75% ethanol/10 mM NH<sub>4</sub>OAc several times for 6 h, dissolved in TE, treated with RNase A (100 µg.ml<sup>−</sup><a reveal-id="dsz008-B1" data-open="dsz008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>, SIGMA) for 60 min at 37°C and protease K (200 µg. ml<sup>−</sup><a reveal-id="dsz008-B1" data-open="dsz008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>, SIGMA) for 20 h at 52°C, extracted with phenol-chloroform and precipitated with ethanol. The quantity and quality of DNA were checked by spectrophotometry and gel electrophoresis. For Southern analysis, genomic DNA (2 µg) was digested with restriction endonucleases (10 U; 2 × 4 h), separated by gel electrophoresis and blotted onto nylon membranes (GE Healthcare). The hybridization was carried out in modified Church–Gilbert buffer.<a reveal-id="dsz008-B20" data-open="dsz008-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The 26S rDNA probe was a 220-bp PCR product derived from the 3′ end of the 26S rRNA gene.<a reveal-id="dsz008-B21" data-open="dsz008-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> This PCR product was also used as a standard for slot blot analysis. The IGS rDNA probes, covering the putative spacer promoter and R2 region, represent the PCR product amplified from GenBank accession MG744572 (cl7) by primers C<sub>F</sub> and C<sub>R</sub> (see PCR amplifications) and digested with NdeI. The hybridization signals were visualized by PhosphorImaging (Typhon; GE Healthcare) and quantitatively analysed by ImageQuant (GE Healthcare).</p>                    <h3 scrollto-destination="137025621" id="137025621" class="section-title">2.2. Fractionalization of genomic DNA and PacBio sequencing</h3>
<p>To enhance the content of rDNA in the sample, the rDNA was separated from appropriately restricted bulk genomic DNA by preparative electrophoresis. One milligram of DNA was digested with EcoRV (3 × 8 h; 3 × 1,000 U), and the completeness of digestion was checked by Southern blot hybridization against 26S rDNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>). The digested DNA (300 µg) was size-fractionated by electrophoresis in a 0.5% preparative agarose gel using Bio-Rad Model 491 Prep-Cell (Serva). When the xylene cyanol dye marker reached the bottom of the gel, individual fractions (1.3 ml/4 min) were collected under electrophoretic conditions of 70 V/5.5 h (fractions 1–83), followed by 100 V (fractions 84–174). Each fraction was lyophilized and dissolved in 100 µl TE + 1 mM DTT, and 2-µl aliquots were analysed by Southern hybridization to the 26S rDNA probe. The fractions with the highest ratio between the hybridization signal and total input DNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1B</a></span>; framed) were collected, desalted with NucAway<sup>TM</sup> Spin Columns (Ambion), lyophilized and checked for DNA quantity and quality (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>).</p>
<p>The PacBio RS library insert length was 2.1 kb on average (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). During sequencing, the library inserts were passed by the polymerase ∼12 times on average (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>), thus creating a mean read quality of the insert sequences of 99.61% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2C</a></span>). Using CLC bio (<a class="link link-uri" href="http://cgs.hku.hk/portal/index.php/software-for-next-generation-sequencing" target="_blank">http://cgs.hku.hk/portal/index.php/software-for-next-generation-sequencing</a>), reads of inserts (ROIs) were mapped to a reference sequence assembled from GenBank accessions for the <em>C. pepo</em> 26S rRNA gene (AF479108.1), IGS (X55960.1), 18S rRNA gene (AF206895.1) and ITS1-5.8S_rDNA_gene–ITS2 region (consensus sequence derived from AM981169.1; AM981168.1; AJ488214.1; KF835490.1; FJ915107.2; FJ915103.2; FJ915102.2 and EF595858.1) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The reference sequence was terminally delimited by a unique EcoRV restriction site located within the 5.8S rRNA gene. The mean insert length of the mapped ROIs was ∼2.1 kb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2D</a></span>). More than 100 ROIs, covering predominantly repetitive regions in the IGS, were difficult to map and were assigned manually based on dot-plot figures specific for pumpkin IGS. We therefore extracted the consensus sequence derived from all successfully mapped ROIs, and all available ROIs were again mapped to this consensus. When compared with the initial mapping, additional ROIs were mapped, particularly those homologous to IGS repeat. After the second mapping, we extracted the resulting consensus sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) and evaluated the frequency and distributions of abundant (&gt;10%) SNPs.</p>
<p>The read quality of each ROI, harbouring the R2 repeat, was estimated from the number of subreads. Only those subreads spanning the entire R2 region were considered for calculations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2E</a></span>). To evaluate the capacity of the PacBio to determine the correct length of GC-rich and highly repeated regions, the lengths of the longest R1 and R2 consensus sequences, determined by dot-plot analyses, were compared with the lengths in each corresponding subread (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span> and B).</p>                    <h3 scrollto-destination="137025625" id="137025625" class="section-title">2.3. Sequence analysis</h3>
<p>The copy numbers and lengths of monomeric units as well as the overall lengths of repetitive regions were determined by the Tandem Repeats Finder<a reveal-id="dsz008-B22" data-open="dsz008-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> (<a class="link link-uri" href="https://tandem.bu.edu/trf/trf.html" target="_blank">https://tandem.bu.edu/trf/trf.html</a>). The lengths of repetitive regions were also estimated from (i) the length of the DNA with an uninterrupted GC content higher than 75% determined by the bend.it<sup>®</sup> server (<a class="link link-uri" href="http://pongor.itk.ppke.hu/dna/bend_it.html#/bendit_intro" target="_blank">http://pongor.itk.ppke.hu/dna/bend_it.html#/bendit_intro</a>) (window size 31) and (ii) dot plot diagrams constructed by the YASS: genomic similarity search tool (<a class="link link-uri" href="http://bioinfo.lifl.fr/yass/yass.php" target="_blank">http://bioinfo.lifl.fr/yass/yass.php</a>). This approach was also used to determine duplicated promoter-like regions delimited by the conserved boundary sequences CCATCACCCATT and TGGGCATATGCTTGG. The size variability at repetitive regions was also evaluated from distances between the conserved boundary unique sequences GGAGGYTAACC and TACCAACA for R1 or TGGGCATATGCTTGG and CCATCACCCATT for R2. The variability in the distances between duplicated TIS sequences ATATAGGGGG was estimated as well.</p>
<p>The potential to form G-quadruplexes (pG4) was evaluated with QGRS Mapper<a reveal-id="dsz008-B23" data-open="dsz008-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (<a class="link link-uri" href="http://bioinformatics.ramapo.edu/QGRS/analyze.php" target="_blank">http://bioinformatics.ramapo.edu/QGRS/analyze.php</a>). Because the sequence propensity for forming Z-DNA was on the order of d(GC)<sub>n</sub> &gt; d(CA)<sub>n</sub> &gt; d(CGGG)<sub>n</sub> &gt; d(AT)<sub>n</sub>,<a reveal-id="dsz008-B24" data-open="dsz008-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> we separately evaluated the frequency and distribution of (CG)<sub>n</sub> repeats longer than 7 nt, more complex (PuPy)<sub>n</sub>, or CG<sub>3</sub> motifs. The frequency and distribution of short dispersed repeated motifs were evaluated manually.</p>
<p>Phylogenetic relationships were constructed from the SNP distribution among the promoter-like sequence variants linked to the R1 and/or R2 regions in the corresponding ROIs. The sequences of genic and spacer promoters were aligned altogether using the MUSCLE programme, and the phylogenetic tree was constructed with the MEGA 7 programme. The evolutionary history was inferred using a maximum likelihood method based on the Tamura-Nei model.<a reveal-id="dsz008-B25" data-open="dsz008-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> The confidence of inferred evolutionary relationships was assessed by bootstrap analysis with 500 repetitions.</p>
<p>For comparative statistics, the Mann-Whitney (<a class="link link-uri" href="http://vassarstats.net/utest.html" target="_blank">http://vassarstats.net/utest.html</a>) non-parametric U-test and Student’s parametric <em>t</em>-test (Microsoft Excel) were used.</p>                    <h3 scrollto-destination="137025630" id="137025630" class="section-title">2.4. Fluorescence <em>in situ</em> hybridization</h3>
<p>Chromosomes from squash root tips were prepared according to previous methods,<a reveal-id="dsz008-B26" data-open="dsz008-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> except for enzymatic treatment, which lasted 30–45 min. Squashes were treated with RNase A (100 µg/ml) in a humidified chamber (37°C/1 h), rinsed in 2× standard saline citrate (SSC) at RT for 3 × 5 min, treated with pepsin (50 µg/ml) at RT for 5–7 min, rinsed in 2× SSC (3 × 5 min), fixed in 3.7% formaldehyde/phosphate-buffered saline (PBS) for 10 min, rinsed in 2× SSC (3 × 5 min), dehydrated in an ethanol series (50, 70 and 100%) for 2 min each, and air-dried. The 18S probe (GenBank X51576.1) was labelled with SpectrumGreen dUTP using a Nick Translation Kit (Abbott Molecular, IL, USA). The 5S probe (GenBank JX101915.1) was labelled with Amersham FluoroLinkTM Cy3-dUTP (GE Healthcare, Chalfont, St Giles, England) using the Nick Translation Mix from Roche. The hybridization mix (40% formamide, 50% dextran sulphate, 2× SSC and 100–150 ng of each probe in a 30 µl volume) was heated (10 min/75°C), cooled on ice, pipetted onto a slide and covered with a plastic coverslip. The slide was incubated in a humidified chamber (75°C/5 min, 65°C/2 min, 55°C/2 min, 45°C/2 min and 37°C/17 h) and washed in 2× SSC (42°C/2 × 5 min), 0.1× SSC (42°C/2 × 5 min), 2× SSC (42°C/2 × 5 min), 2× SSC (RT/5 min) and 4× SSC/0.1% Tween 20 (RT/7 min) with a final brief wash in PBS. The chromosomes were counterstained in Vectashield (Vector Laboratories, Inc., Burlingame, CA, USA) containing 1.5 µg/ml 4’, 6-diamidino-2’-phenylindole (DAPI). Fluorescence images were captured using an Olympus AX 70 fluorescence microscope equipped with a digital camera. Images were analysed and processed using ISIS software (MetaSystems, Altlussheim, Germany).</p>                    <h3 scrollto-destination="137025632" id="137025632" class="section-title">2.5. PCR amplifications, cloning and sequencing</h3>
<p>For PCR amplification of R2 repetitive regions, primers C<sub>F</sub> (5′-ACTTGAAAGAATGACGCCGGT-3′) and C<sub>R</sub> (5′-CAAGAAAACAACAACTCCACATGTAA-3′) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) were designed. The PCR mixture was composed of 0.4 µM each primer, 0.5 ng/µl of template genomic DNA, 0.2 mM of each deoxyribonucleotide triphosphate, 1× KAPA Taq Buffer (2.5 mM MgCl<sub>2</sub>), 5% DMSO and 0.02 U/µl KAPA Taq DNA Polymerase (KAPABIOSYSTEMS). The cycling conditions were as follows: initial denaturation (95°C/180 s), followed by 35 cycles (95°C/30 s, 57°C/30 s, 72°C/60 s) and then 72°C/5 min. The PCR products were checked by agarose-gel electrophoresis, ligated into the pDrive Cloning Vector (QIAGEN) and sequenced by Sanger methodology.</p>                    <h3 scrollto-destination="137025634" id="137025634" class="section-title">2.6. Data availability</h3>
<p>The PacBio FASTQ data generated from this study were submitted to NCBI under the BioProject ID PRJNA400686 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/bioproject/400686" target="_blank">http://www.ncbi.nlm.nih.gov/bioproject/400686</a>). The data derived from Sanger sequencing have been submitted to GenBank under accessions MG744571-MG744575.</p>                    <h2 scrollto-destination="137025636" id="137025636" class="section-title">3. Results</h2>
                    <h3 scrollto-destination="137025637" id="137025637" class="section-title">3.1. Considerably high numbers of rDNA units distributed across multiple chromosomal loci suggest the coexistence of rDNA structural variants in the pumpkin genome</h3>
<p>Digestion of <em>C. moschata</em> genomic DNA with EcoRV resulted in almost completely restricted rDNA into a population of ∼10 kb long units that migrated during electrophoresis as a prominent band detected either by staining with ethidium bromide or by hybridization with rDNA probe (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>), suggesting considerably high rDNA copy numbers in the genome. For a detailed evaluation of rDNA intragenomic heterogeneity, the DNA-species forming this band were subjected to PacBio sequencing. Approximately one quarter (4,200) of the ROIs were successfully mapped to a 35S rDNA reference sequence, indicating ∼3-fold enhancement of the rDNA content in the sample in comparison to the native pumpkin genomic DNA, in which a fraction of ∼7.8% (6,500 copies) was estimated to be composed of 35S rDNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) distributed across five chromosomal loci of highly variable sizes as follows from fluorescence <em>in situ</em> hybridization (FISH) analyses performed previously by Waminal <em>et al</em>.<a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> (2011). To evaluate the variability in the chromosomal distribution of 35S rDNA within <em>Cucurbita</em>, we performed a similar analysis with <em>C. pepo</em>. As in <em>C. moschata</em>, hybridization signals at two loci (four signals) were significantly stronger than the remaining signals (<a reveal-id="dsz008-F1" data-open="dsz008-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) frequently present on extremely small chromosomes (<a reveal-id="dsz008-F1" data-open="dsz008-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>). Although the number of strong 35S rDNA signals was invariant, the total number of signals slightly varied (12–13) among ten here analysed metaphases, suggesting potential aneuploidy in <em>C. pepo</em>, which was supported by some metaphase preparations with counted 41 chromosomes (<a reveal-id="dsz008-F1" data-open="dsz008-F1" class="link link-reveal link-table xref-fig">Fig. 1F</a>). In addition, higher total number of 35S rDNA sites in <em>C. pepo</em>, than in <em>C. moschata</em> suggests some instability of 35S rDNA across <em>Cucurbita</em>. Similarly, two loci and one locus of the 5S rDNA detected in <em>C. moschata</em><a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and <em>C. pepo</em> (<a reveal-id="dsz008-F1" data-open="dsz008-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>), respectively, suggests that instability in rDNA is rather common in <em>Cucurbita</em>. Nevertheless, the distribution of 35S rDNA across multiple loci in both species might indicate the coexistence of 35S rDNA structural variants within <em>Cucurbita</em>. </p>                        <a id="137025639" scrollto-destination="137025639"></a>
<div id="dsz008-F1" data-id="dsz008-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f1.png?Expires=1576484115&amp;Signature=04ElcRHirymekaOE4PUIvtRyvvqBi8-gt1kXRkiT0ZH81U56T3n9S0XKofeprNRk6ltaFFES4qWRpzXAa0Y1l7kVrfLCO2W0a0J6i~MUp5wcqOAvbIg6fAKvJ4k7yFYl8TKtcnER1jQXzZkxZSliTSAlvXAX3EYXQO~wXWKR5HXPsHfwxULUqf2C049F2ocn3ykS5pegef8Yi2yl8Zp8pEUXz2vAj6xGp250G80VLpeIlrHHQiC3oB4rHpffmGGi17XfHg7torJBFjGQNJJc33zfNcBm66jWqLxQNQgjo~~~tTkat1R7lEffQIZoK59akC--5z0H3PXBLgiKX5-I3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FISH at metaphase chromosomes of C. pepo. Four very strong and nine significantly weaker signals for 35S rDNA (green; panels A and D) complemented with two strong signals for 5S rDNA (red; panels B and D) were detected. Chromosomes were counterstained with DAPI (C). To count all very small chromosomes hardly detected with colour, chromosomes were highlighted in inverted black and white format (panels E and F). Two metaphase preparations are shown, and panel E corresponds to panels (A–D). The chromosomes that harbour 35S rDNA are marked by arrowheads. Panel (F) shows metaphase preparation with all chromosomes well separated, counted and numbered from 1 through 41, approximately in the order of their relative sizes. Note that odd total number of chromosomes suggests an aneuploidy in this metaphase, which is in agreement with the odd number of rDNA signals shown in panels (A, D and E), although showing different metaphase. Scale bars (A–F) = 10 µm. " path-from-xml="dsz008f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025639/dsz008f1.tif" path-from-xml="dsz008f1.tif" target="_blank">Open in new tab</a><a section="137025639" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f1.png?Expires=1576484115&amp;Signature=fMY4YzJdg5ZA1ojMJW8AsrE2tf1j4GLvMTG75iMeeocvTADzRYrMNfuOFqV~ZrRfmaAd77EdUCqASP7kAVc7u8plu84wDnGH4s74JfkTen0NJZsSRWWaqnKaSxz1EyMhLcPDkVCr9xrftnEN60mJJbeZqrlIRUWoYgS50Weaw9hcLYxLRxkgqlvf4uNzkmJsbnL-V3B~2COWiRE6Js1fQyQKYNCMd7PWWyHVu7I793eNeDthYs9XAiM2TDFYoUCC2pqbapSD6c5im3dSs33vc9b7oQWBrJLVRf9Y-czyAf4eZHX3I-6hps0eMbOK5tGUi4eweWT6Ra6Nl0gWo~G7iA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025639&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>FISH at metaphase chromosomes of <em>C. pepo.</em> Four very strong and nine significantly weaker signals for 35S rDNA (green; panels A and D) complemented with two strong signals for 5S rDNA (red; panels B and D) were detected. Chromosomes were counterstained with DAPI (C). To count all very small chromosomes hardly detected with colour, chromosomes were highlighted in inverted black and white format (panels E and F). Two metaphase preparations are shown, and panel E corresponds to panels (A–D). The chromosomes that harbour 35S rDNA are marked by arrowheads. Panel (F) shows metaphase preparation with all chromosomes well separated, counted and numbered from 1 through 41, approximately in the order of their relative sizes. Note that odd total number of chromosomes suggests an aneuploidy in this metaphase, which is in agreement with the odd number of rDNA signals shown in panels (A, D and E), although showing different metaphase. Scale bars (A–F) = 10 µm. </p></div>
</div>
<div content-id="dsz008-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f1.png?Expires=1576484115&amp;Signature=04ElcRHirymekaOE4PUIvtRyvvqBi8-gt1kXRkiT0ZH81U56T3n9S0XKofeprNRk6ltaFFES4qWRpzXAa0Y1l7kVrfLCO2W0a0J6i~MUp5wcqOAvbIg6fAKvJ4k7yFYl8TKtcnER1jQXzZkxZSliTSAlvXAX3EYXQO~wXWKR5HXPsHfwxULUqf2C049F2ocn3ykS5pegef8Yi2yl8Zp8pEUXz2vAj6xGp250G80VLpeIlrHHQiC3oB4rHpffmGGi17XfHg7torJBFjGQNJJc33zfNcBm66jWqLxQNQgjo~~~tTkat1R7lEffQIZoK59akC--5z0H3PXBLgiKX5-I3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FISH at metaphase chromosomes of C. pepo. Four very strong and nine significantly weaker signals for 35S rDNA (green; panels A and D) complemented with two strong signals for 5S rDNA (red; panels B and D) were detected. Chromosomes were counterstained with DAPI (C). To count all very small chromosomes hardly detected with colour, chromosomes were highlighted in inverted black and white format (panels E and F). Two metaphase preparations are shown, and panel E corresponds to panels (A–D). The chromosomes that harbour 35S rDNA are marked by arrowheads. Panel (F) shows metaphase preparation with all chromosomes well separated, counted and numbered from 1 through 41, approximately in the order of their relative sizes. Note that odd total number of chromosomes suggests an aneuploidy in this metaphase, which is in agreement with the odd number of rDNA signals shown in panels (A, D and E), although showing different metaphase. Scale bars (A–F) = 10 µm. " path-from-xml="dsz008f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025639/dsz008f1.tif" path-from-xml="dsz008f1.tif" target="_blank">Open in new tab</a><a section="137025639" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f1.png?Expires=1576484115&amp;Signature=fMY4YzJdg5ZA1ojMJW8AsrE2tf1j4GLvMTG75iMeeocvTADzRYrMNfuOFqV~ZrRfmaAd77EdUCqASP7kAVc7u8plu84wDnGH4s74JfkTen0NJZsSRWWaqnKaSxz1EyMhLcPDkVCr9xrftnEN60mJJbeZqrlIRUWoYgS50Weaw9hcLYxLRxkgqlvf4uNzkmJsbnL-V3B~2COWiRE6Js1fQyQKYNCMd7PWWyHVu7I793eNeDthYs9XAiM2TDFYoUCC2pqbapSD6c5im3dSs33vc9b7oQWBrJLVRf9Y-czyAf4eZHX3I-6hps0eMbOK5tGUi4eweWT6Ra6Nl0gWo~G7iA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025639&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>FISH at metaphase chromosomes of <em>C. pepo.</em> Four very strong and nine significantly weaker signals for 35S rDNA (green; panels A and D) complemented with two strong signals for 5S rDNA (red; panels B and D) were detected. Chromosomes were counterstained with DAPI (C). To count all very small chromosomes hardly detected with colour, chromosomes were highlighted in inverted black and white format (panels E and F). Two metaphase preparations are shown, and panel E corresponds to panels (A–D). The chromosomes that harbour 35S rDNA are marked by arrowheads. Panel (F) shows metaphase preparation with all chromosomes well separated, counted and numbered from 1 through 41, approximately in the order of their relative sizes. Note that odd total number of chromosomes suggests an aneuploidy in this metaphase, which is in agreement with the odd number of rDNA signals shown in panels (A, D and E), although showing different metaphase. Scale bars (A–F) = 10 µm. </p></div>
</div>
<p>It is necessary to stress that such a mode of rDNA sample purification, based on a specific restriction pattern, may eliminate 35S rDNA pseudogenes or unusual IGS variants that are sometimes found dispersed in genomes outside regular NORs.<a reveal-id="dsz008-B27" data-open="dsz008-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> In theory, such a method is suitable for the enrichment of samples with each type of repeat, which might be restricted from the genome using an appropriate restriction endonuclease.</p>
<p>Although relatively low numbers of 35S rDNA PacBio ROIs were analysed compared with the high copy numbers of 35S rDNA in the pumpkin genome, we believe that even those ROIs associated with the region of lowest coverage (repeats in IGS) might depict the variability in prominent structural motifs with sufficient accuracy.</p>                    <h3 scrollto-destination="137025642" id="137025642" class="section-title">3.2. Common sequence blocks in the IGS are shared in considerably variable proportions among individual rDNA copies</h3>
<p>As the IGS represents the region with the highest sequence heterogeneity within the pumpkin rDNA unit (<a reveal-id="dsz008-F2" data-open="dsz008-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>), we concentrated our further efforts to estimate the intra-individual structural variability of this evolutionarily highly variable region. Each IGS copy is assembled from the three repeat regions R1–R3, which are separated by two copies of a unique duplicated sequence (Pg and Ps) (<a reveal-id="dsz008-F2" data-open="dsz008-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). These duplicates are mutually highly related (<a reveal-id="dsz008-F2" data-open="dsz008-F2" class="link link-reveal link-table xref-fig">Fig. 2A;</a> dot plot) and internally moderately variable nucleotide sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>) with sizes ranging from 442 to 459 bp. Both contain a highly conserved sequence motif (TATATAGGGG) related to the known plant TIS for Pol l, suggesting that they may function as putative genic (Pg) and spacer (Ps) Pol I promoters. Interestingly, both duplicates harbour another short, highly conserved motif, CCCCTATAT, which is located ∼40 bp downstream of each (genic and spacer) TIS and represents nearly perfect reverse-complemented reflection of the TIS. </p>                        <a id="137025644" scrollto-destination="137025644"></a>
<div id="dsz008-F2" data-id="dsz008-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f2.png?Expires=1576484115&amp;Signature=uT-iOoDPY1L~9IKiyIx2MEzVOUyp0QpNT6csMgp0EnvnosybxjOATmDT6OHdEdgb4YmoWWJqDjfxGHMWyYs4i8hua-lBI-77W5t-Fz5kRDQ9K6gMsfNKpPMucw4z24hJifZrzOpGw41dYjJ30tBZsd5~euH~ShgQgGpyqNUWLdPqrB~3NbvnZjzSfC9tZhVhyx~drTmodqY3BO~FBdPKm4~r8S6QmkxHF97yVaWI5XzrMKDTsw~3zGfN9FU8mAsmvOgAOcFsqAyC9aktpi4~nV9PQdfI88amYkTrAL4qE76HT17cZ3YVm8IaSmuls9zvsc8S8zEnDcLmyBVsZoeI6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repetitive organization of the IGS in the pumpkin 35S rDNA unit. (A) Three regions (R1–R3), each built of tandemly arranged repetitive units, are separated by two mutually highly homologous sequences (Ps and Pg), as demonstrated by the dot-plot diagram constructed from the consensus sequence derived from mapped ROIs. Motifs related to TISs are denoted by arrows. (B) The frequency of abundant (&gt;10%) SNPs plotted along the rDNA unit (100-bp window) shows that the R1 and R2 regions represent the most variable extremes in the rDNA unit. In contrast, all three genes are highly conserved and separated by the moderately variable ITS1 and ITS2. The pattern of all abundant SNPs along the source consensus sequence is shown in Supplementary Fig. S4. It is necessary to stress that the highest variability at repeats R1 and R2 may be artificially overestimated owing to possible incorrect mapping of the corresponding ROIs with variable copy numbers of repeats." path-from-xml="dsz008f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025644/dsz008f2.tif" path-from-xml="dsz008f2.tif" target="_blank">Open in new tab</a><a section="137025644" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f2.png?Expires=1576484115&amp;Signature=MwFrOChVRovdkZITEZyAOs9V2UJOsrsc2aArI-cisvAdI2Ez04laqObfBVSwAC~fii0HlDnrDaw54uXAv0aaEidRAm069WbKnWFEaviSiwzhQfRE8wkcvB2WNfjmgtFifAZBO3Q3bP3zcYU99uPgo4iHN36Nd6qXtOzupZpyihfNf-UyJF~cXq8Gr8GCjcuf6-lhmeheXS0O486VOVx3NBDXtzUuiTedV8iIIhg52SjoZLFM07Efmnvm0XZTws7xet6qzwKpreN0gXF0ZbhDuaHz54-CNSarpDIw82HOpD7VNBsSY1G4Jh3ZUs6PIZIMw38E8mn8XybcRDYxnX4uIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025644&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Repetitive organization of the IGS in the pumpkin 35S rDNA unit. (A) Three regions (R1–R3), each built of tandemly arranged repetitive units, are separated by two mutually highly homologous sequences (P<sub>s</sub> and P<sub>g</sub>), as demonstrated by the dot-plot diagram constructed from the consensus sequence derived from mapped ROIs. Motifs related to TISs are denoted by arrows. (B) The frequency of abundant (&gt;10%) SNPs plotted along the rDNA unit (100-bp window) shows that the R1 and R2 regions represent the most variable extremes in the rDNA unit. In contrast, all three genes are highly conserved and separated by the moderately variable ITS1 and ITS2. The pattern of all abundant SNPs along the source consensus sequence is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>. It is necessary to stress that the highest variability at repeats R1 and R2 may be artificially overestimated owing to possible incorrect mapping of the corresponding ROIs with variable copy numbers of repeats.</p></div>
</div>
<div content-id="dsz008-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f2.png?Expires=1576484115&amp;Signature=uT-iOoDPY1L~9IKiyIx2MEzVOUyp0QpNT6csMgp0EnvnosybxjOATmDT6OHdEdgb4YmoWWJqDjfxGHMWyYs4i8hua-lBI-77W5t-Fz5kRDQ9K6gMsfNKpPMucw4z24hJifZrzOpGw41dYjJ30tBZsd5~euH~ShgQgGpyqNUWLdPqrB~3NbvnZjzSfC9tZhVhyx~drTmodqY3BO~FBdPKm4~r8S6QmkxHF97yVaWI5XzrMKDTsw~3zGfN9FU8mAsmvOgAOcFsqAyC9aktpi4~nV9PQdfI88amYkTrAL4qE76HT17cZ3YVm8IaSmuls9zvsc8S8zEnDcLmyBVsZoeI6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repetitive organization of the IGS in the pumpkin 35S rDNA unit. (A) Three regions (R1–R3), each built of tandemly arranged repetitive units, are separated by two mutually highly homologous sequences (Ps and Pg), as demonstrated by the dot-plot diagram constructed from the consensus sequence derived from mapped ROIs. Motifs related to TISs are denoted by arrows. (B) The frequency of abundant (&gt;10%) SNPs plotted along the rDNA unit (100-bp window) shows that the R1 and R2 regions represent the most variable extremes in the rDNA unit. In contrast, all three genes are highly conserved and separated by the moderately variable ITS1 and ITS2. The pattern of all abundant SNPs along the source consensus sequence is shown in Supplementary Fig. S4. It is necessary to stress that the highest variability at repeats R1 and R2 may be artificially overestimated owing to possible incorrect mapping of the corresponding ROIs with variable copy numbers of repeats." path-from-xml="dsz008f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025644/dsz008f2.tif" path-from-xml="dsz008f2.tif" target="_blank">Open in new tab</a><a section="137025644" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f2.png?Expires=1576484115&amp;Signature=MwFrOChVRovdkZITEZyAOs9V2UJOsrsc2aArI-cisvAdI2Ez04laqObfBVSwAC~fii0HlDnrDaw54uXAv0aaEidRAm069WbKnWFEaviSiwzhQfRE8wkcvB2WNfjmgtFifAZBO3Q3bP3zcYU99uPgo4iHN36Nd6qXtOzupZpyihfNf-UyJF~cXq8Gr8GCjcuf6-lhmeheXS0O486VOVx3NBDXtzUuiTedV8iIIhg52SjoZLFM07Efmnvm0XZTws7xet6qzwKpreN0gXF0ZbhDuaHz54-CNSarpDIw82HOpD7VNBsSY1G4Jh3ZUs6PIZIMw38E8mn8XybcRDYxnX4uIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025644&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Repetitive organization of the IGS in the pumpkin 35S rDNA unit. (A) Three regions (R1–R3), each built of tandemly arranged repetitive units, are separated by two mutually highly homologous sequences (P<sub>s</sub> and P<sub>g</sub>), as demonstrated by the dot-plot diagram constructed from the consensus sequence derived from mapped ROIs. Motifs related to TISs are denoted by arrows. (B) The frequency of abundant (&gt;10%) SNPs plotted along the rDNA unit (100-bp window) shows that the R1 and R2 regions represent the most variable extremes in the rDNA unit. In contrast, all three genes are highly conserved and separated by the moderately variable ITS1 and ITS2. The pattern of all abundant SNPs along the source consensus sequence is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>. It is necessary to stress that the highest variability at repeats R1 and R2 may be artificially overestimated owing to possible incorrect mapping of the corresponding ROIs with variable copy numbers of repeats.</p></div>
</div>
<p>Although duplicated promoter-like sequences are rather invariable in length, repeated regions substantially contribute to the overall length variability of the IGS. Analyses of 134 and 436 ROIs by several independent methods consistently demonstrated extraordinarily high size variability at R1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span> and B) and R2 (<a reveal-id="dsz008-F3" data-open="dsz008-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) tandem repeats, respectively, which ranges with fine graduations of more than hundreds of bp and a multimodal size distribution. Such a broad size variability at R1 and R2 even resulted in reciprocal lengths of both repeats in some IGSs, as demonstrated by ROIs bearing either more abundant R1 or more abundant R2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). </p>                        <a id="137025646" scrollto-destination="137025646"></a>
<div id="dsz008-F3" data-id="dsz008-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f3.png?Expires=1576484115&amp;Signature=bkzvGHwEfPX8VZtQVfG0SY34OaE4d~yG5q5-bNR-WdhvvRQGQEr3uUGYPT1NzLqLMgyp5Ac4d6uvJIZVoD-vJceySxfMxPV60uHaK0~EF1YQQCO-t8cUvmtvPvBJc86XHDhAuzjSXFNqBDW5fkBjZh8VgC~M9QRNmrJPd0qQVKkYEBaom5qWp7255T8-twKm150aTg~EXawTGwk2jLFOU76oiB~XRe9-qaq0C6Z1ih8zmSt2jqWtJE~-EF-bfAhE5icCrxIdlapHcjOWFwSjPfJbds-A3HrV-vsSCWWpEUIiNYG8I9KSi5Fe-0A0gwdS6SWJ-9~I~Afk6ovUmfItfw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The multimodal distribution of size variants in the R2 repeated region. Three independent methods were used to evaluate the size variability: (A) Distance between two boundary unique sequences (see Section 2.3). (B) Overall size of all tandem repeats detected in the R2 region by Tandem Repeats Finder. (C) Size of the uninterrupted region with GC content higher than 75%. Significant correlations between the results obtained using these three approaches, are demonstrated in panels (D–F)." path-from-xml="dsz008f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025646/dsz008f3.tif" path-from-xml="dsz008f3.tif" target="_blank">Open in new tab</a><a section="137025646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f3.png?Expires=1576484115&amp;Signature=BjjeFlqAlDs7pPGSwISrKe8UsT7bA351vK~1NBLmhXw1TM5cI-GVZc5L59yNj8fUxtWHmzBZpZqRoG4vjT-bZW-KK4GrP4A4eQpXVe2V4b9KGhTCWZB2Pb-iJlX4O2XKyA3tqa9RKF16pQ6yHk3LYGN6Awyy2ezo91aa3hLeRj3K8vsD2pWuoH2YSQyeFKDe3K8kJaKq-FGrfxtODjcjC-3Z-Yf01wTkyM5J0edz4qSBg9B05QN30OVisSBRGhSrNJHU0BbVhNFBf5s3Bh66quu9IXbYbsfscQQhjTx23DGVgm5CAK~oVtZqfjRDMjKxFBvHPH7ghx0JmnL6r0GD1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025646&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The multimodal distribution of size variants in the R2 repeated region. Three independent methods were used to evaluate the size variability: (A) Distance between two boundary unique sequences (see Section 2.3). (B) Overall size of all tandem repeats detected in the R2 region by Tandem Repeats Finder. (C) Size of the uninterrupted region with GC content higher than 75%. Significant correlations between the results obtained using these three approaches, are demonstrated in panels (D–F).</p></div>
</div>
<div content-id="dsz008-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f3.png?Expires=1576484115&amp;Signature=bkzvGHwEfPX8VZtQVfG0SY34OaE4d~yG5q5-bNR-WdhvvRQGQEr3uUGYPT1NzLqLMgyp5Ac4d6uvJIZVoD-vJceySxfMxPV60uHaK0~EF1YQQCO-t8cUvmtvPvBJc86XHDhAuzjSXFNqBDW5fkBjZh8VgC~M9QRNmrJPd0qQVKkYEBaom5qWp7255T8-twKm150aTg~EXawTGwk2jLFOU76oiB~XRe9-qaq0C6Z1ih8zmSt2jqWtJE~-EF-bfAhE5icCrxIdlapHcjOWFwSjPfJbds-A3HrV-vsSCWWpEUIiNYG8I9KSi5Fe-0A0gwdS6SWJ-9~I~Afk6ovUmfItfw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The multimodal distribution of size variants in the R2 repeated region. Three independent methods were used to evaluate the size variability: (A) Distance between two boundary unique sequences (see Section 2.3). (B) Overall size of all tandem repeats detected in the R2 region by Tandem Repeats Finder. (C) Size of the uninterrupted region with GC content higher than 75%. Significant correlations between the results obtained using these three approaches, are demonstrated in panels (D–F)." path-from-xml="dsz008f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025646/dsz008f3.tif" path-from-xml="dsz008f3.tif" target="_blank">Open in new tab</a><a section="137025646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f3.png?Expires=1576484115&amp;Signature=BjjeFlqAlDs7pPGSwISrKe8UsT7bA351vK~1NBLmhXw1TM5cI-GVZc5L59yNj8fUxtWHmzBZpZqRoG4vjT-bZW-KK4GrP4A4eQpXVe2V4b9KGhTCWZB2Pb-iJlX4O2XKyA3tqa9RKF16pQ6yHk3LYGN6Awyy2ezo91aa3hLeRj3K8vsD2pWuoH2YSQyeFKDe3K8kJaKq-FGrfxtODjcjC-3Z-Yf01wTkyM5J0edz4qSBg9B05QN30OVisSBRGhSrNJHU0BbVhNFBf5s3Bh66quu9IXbYbsfscQQhjTx23DGVgm5CAK~oVtZqfjRDMjKxFBvHPH7ghx0JmnL6r0GD1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025646&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>The multimodal distribution of size variants in the R2 repeated region. Three independent methods were used to evaluate the size variability: (A) Distance between two boundary unique sequences (see Section 2.3). (B) Overall size of all tandem repeats detected in the R2 region by Tandem Repeats Finder. (C) Size of the uninterrupted region with GC content higher than 75%. Significant correlations between the results obtained using these three approaches, are demonstrated in panels (D–F).</p></div>
</div>
<p>In contrast to repeats R1 and R2, located upstream of the genic TIS, the unique repeat R3, located downstream (<a reveal-id="dsz008-F2" data-open="dsz008-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), seemed to be substantially more homogeneous in size (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7C</a></span>), ranging from 2.9 to 3.4 tandemly arranged repeats with a consensus sequence varying from 248 to 261 bp. The exception was a single R3 region composed of 4.3 repeats.</p>
<p>Size polymorphisms detected by PacBio sequencing in the R1 and R2 regions were verified in the genome-wide range by PCR and Southern blot analyses. PCR amplification of R2 regions provided multiple products with an overall shape characteristic of tandem repeats (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). They all were, however, shorter than expected and the origins of the PCR products were, therefore, verified by Sanger sequencing. All cloned PCR products were assembled of the spacer promoter and short repeats of different sizes related to the R2 region.</p>
<p>To evaluate the size variability at repeated regions by Southern blot analyses, genomic DNA was digested by restriction enzymes with targets in the vicinity of those repeats, and restriction fragments of interest were visualized by hybridization to the IGS probe(s) (<a reveal-id="dsz008-F4" data-open="dsz008-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Restriction with NdeI or DraI was used for R2 repeats, whereas double digestion with NdeI and HpaI showed overall size heterogeneity in both R1 and R2 regions. Each digest provided multiple rather diffuse hybridization signals (<a reveal-id="dsz008-F4" data-open="dsz008-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>), indicating the number of size variants present in both R1 and R2 regions and continuously spanning a considerably wide range of size distributions. Such pattern correlated well with the multimodal size distribution of PacBio ROIs (compare <a reveal-id="dsz008-F4" data-open="dsz008-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a> with <a reveal-id="dsz008-F3" data-open="dsz008-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). Evident structural variability across plants within single cultivar (<a reveal-id="dsz008-F4" data-open="dsz008-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>), suggests fast repeats spreading or contraction in pumpkin IGS. As expected from the restriction map (<a reveal-id="dsz008-F4" data-open="dsz008-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), both NdeI and DraI digests also provided long (&gt;5 kb) hybridization signals (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). The NdeI fragment (7617 bp) harbours the entire R1, whereas the DraI fragment (5,360 bp) contains the entire R3 repeat. More diffuse pattern provided by NdeI confirmed higher size variability in the R1 region than in the R3 region. </p>                        <a id="137025650" scrollto-destination="137025650"></a>
<div id="dsz008-F4" data-id="dsz008-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f4.png?Expires=1576484115&amp;Signature=uZSV-7csmaCON-XU5wvGVpuk-uu0yWbG1blJeVCMB1yQOZk0SWKxa8R1CJkFueAJcRn7rmZ6nZlTmVzyQkDoKk6DxGffEa9vxWk2Kj5qpEJhFgAgwjS~T1nErd-FfCCgl26l76dkrd6wzzW0wHt3T8VpPk86GZn0s16E-SP3F9sbZvK2qHdxjKTDONHBP6~dZcAN8e1mxEDlWGFeDtSUso6p6xxCUaMNRw2bcej2L~ggBiuDUAijF4K9qP0xDhE3rXpcfcL8sFkbRvr88AXsInvQ727SEwp4d~ZnwC8IQZcvMgd9JyNKeknKS6biV-mONfXzZLaaXbLXIjUVo2Yj8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Size polymorphisms at R2(R1) region(s) may be supported on a genome-wide range by Southern blot analyses. (A) Target(s) for IGS hybridization probe(s) are shown on the background of a restriction map for NdeI (N), HpaI (H) and DraI (D) constructed from the consensus sequence for the corresponding IGS regions (Supplementary Fig. S4). The restricted DNA fragments that are expected to hybridize with the probe(s) are drawn below. (B) Southern hybridization of IGS probe(s) to genomic DNA restricted by the quoted RE reveals multiple rather diffuse bands. For better resolution, only short (&lt;2 kb) fragments are shown (long fragments are shown in Supplementary Fig. S10). (C) Densitometric evaluations of hybridization profiles. Note that NdeI digests show size variability in R2 (Bands 1–2), whereas double digestion with NdeI and HpaI shows size variability in both R1 (Bands 3–4) and R2 (1–2) regions. (D) Longer exposition of NdeI digests highlighted structural variability (asterisk) across three plants selected within a single pumpkin cultivar. Plant marked by number 1 was sequenced and analysed in panel (B)." path-from-xml="dsz008f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025650/dsz008f4.tif" path-from-xml="dsz008f4.tif" target="_blank">Open in new tab</a><a section="137025650" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f4.png?Expires=1576484115&amp;Signature=VzjxE~HoAHBHhuRvisdSf7NQ-V85cbqwpaMa036vMxZrnnSa0fR7FMteFdmbAa9A3v~JSS52l7~Y03mpIEjNqNLlL3Zx1p-L5CC0EbhhU05W~-AZuDwwgfGdiHU9ky6bFkrIc3evlYmYBxwqKEZwBO5cc8EVpgBHDSC2QUY4cAzXI6hjSLQWu~XS4scvcUKGffyNgf~jVdDjEdfw1MEho4428kAjU7vRHzIBJkSeHPlHneT6UqRacapZE9zAVBxXeC48KkiO8o29zyaB39EoXb0NUgOS6x94v956tdQ9hBQTi0tX4-2GecG3ntphaTG2wMzd9ghLszWM-cPovTgUhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025650&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Size polymorphisms at R2(R1) region(s) may be supported on a genome-wide range by Southern blot analyses. (A) Target(s) for IGS hybridization probe(s) are shown on the background of a restriction map for NdeI (N), HpaI (H) and DraI (D) constructed from the consensus sequence for the corresponding IGS regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). The restricted DNA fragments that are expected to hybridize with the probe(s) are drawn below. (B) Southern hybridization of IGS probe(s) to genomic DNA restricted by the quoted RE reveals multiple rather diffuse bands. For better resolution, only short (&lt;2 kb) fragments are shown (long fragments are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). (C) Densitometric evaluations of hybridization profiles. Note that NdeI digests show size variability in R2 (Bands 1–2), whereas double digestion with NdeI and HpaI shows size variability in both R1 (Bands 3–4) and R2 (1–2) regions. (D) Longer exposition of NdeI digests highlighted structural variability (asterisk) across three plants selected within a single pumpkin cultivar. Plant marked by number 1 was sequenced and analysed in panel (B).</p></div>
</div>
<div content-id="dsz008-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f4.png?Expires=1576484115&amp;Signature=uZSV-7csmaCON-XU5wvGVpuk-uu0yWbG1blJeVCMB1yQOZk0SWKxa8R1CJkFueAJcRn7rmZ6nZlTmVzyQkDoKk6DxGffEa9vxWk2Kj5qpEJhFgAgwjS~T1nErd-FfCCgl26l76dkrd6wzzW0wHt3T8VpPk86GZn0s16E-SP3F9sbZvK2qHdxjKTDONHBP6~dZcAN8e1mxEDlWGFeDtSUso6p6xxCUaMNRw2bcej2L~ggBiuDUAijF4K9qP0xDhE3rXpcfcL8sFkbRvr88AXsInvQ727SEwp4d~ZnwC8IQZcvMgd9JyNKeknKS6biV-mONfXzZLaaXbLXIjUVo2Yj8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Size polymorphisms at R2(R1) region(s) may be supported on a genome-wide range by Southern blot analyses. (A) Target(s) for IGS hybridization probe(s) are shown on the background of a restriction map for NdeI (N), HpaI (H) and DraI (D) constructed from the consensus sequence for the corresponding IGS regions (Supplementary Fig. S4). The restricted DNA fragments that are expected to hybridize with the probe(s) are drawn below. (B) Southern hybridization of IGS probe(s) to genomic DNA restricted by the quoted RE reveals multiple rather diffuse bands. For better resolution, only short (&lt;2 kb) fragments are shown (long fragments are shown in Supplementary Fig. S10). (C) Densitometric evaluations of hybridization profiles. Note that NdeI digests show size variability in R2 (Bands 1–2), whereas double digestion with NdeI and HpaI shows size variability in both R1 (Bands 3–4) and R2 (1–2) regions. (D) Longer exposition of NdeI digests highlighted structural variability (asterisk) across three plants selected within a single pumpkin cultivar. Plant marked by number 1 was sequenced and analysed in panel (B)." path-from-xml="dsz008f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025650/dsz008f4.tif" path-from-xml="dsz008f4.tif" target="_blank">Open in new tab</a><a section="137025650" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f4.png?Expires=1576484115&amp;Signature=VzjxE~HoAHBHhuRvisdSf7NQ-V85cbqwpaMa036vMxZrnnSa0fR7FMteFdmbAa9A3v~JSS52l7~Y03mpIEjNqNLlL3Zx1p-L5CC0EbhhU05W~-AZuDwwgfGdiHU9ky6bFkrIc3evlYmYBxwqKEZwBO5cc8EVpgBHDSC2QUY4cAzXI6hjSLQWu~XS4scvcUKGffyNgf~jVdDjEdfw1MEho4428kAjU7vRHzIBJkSeHPlHneT6UqRacapZE9zAVBxXeC48KkiO8o29zyaB39EoXb0NUgOS6x94v956tdQ9hBQTi0tX4-2GecG3ntphaTG2wMzd9ghLszWM-cPovTgUhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025650&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Size polymorphisms at R2(R1) region(s) may be supported on a genome-wide range by Southern blot analyses. (A) Target(s) for IGS hybridization probe(s) are shown on the background of a restriction map for NdeI (N), HpaI (H) and DraI (D) constructed from the consensus sequence for the corresponding IGS regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). The restricted DNA fragments that are expected to hybridize with the probe(s) are drawn below. (B) Southern hybridization of IGS probe(s) to genomic DNA restricted by the quoted RE reveals multiple rather diffuse bands. For better resolution, only short (&lt;2 kb) fragments are shown (long fragments are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). (C) Densitometric evaluations of hybridization profiles. Note that NdeI digests show size variability in R2 (Bands 1–2), whereas double digestion with NdeI and HpaI shows size variability in both R1 (Bands 3–4) and R2 (1–2) regions. (D) Longer exposition of NdeI digests highlighted structural variability (asterisk) across three plants selected within a single pumpkin cultivar. Plant marked by number 1 was sequenced and analysed in panel (B).</p></div>
</div>                    <h3 scrollto-destination="137025651" id="137025651" class="section-title">3.3. Structurally divergent DNA minisatellites coexist in the R2 region</h3>
<p>Multimodal distribution of size variants in both the R1 and R2 regions, which were consistently detected by PacBio sequencing and Southern analyses, suggested the coexistence of two or more prominent lengths and/or structural variants in both regions. We further focussed on detailed structural analyses of the R2 repeat because it is adjacent to both promoter-like regions (<a reveal-id="dsz008-F2" data-open="dsz008-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), and the multimodal distribution of size variants appeared to be more pronounced compared with the R1 and R3 regions. Each R2-repeat variant was significantly enriched in GC content (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), and the sense DNA strand was enriched with C at the expense of G (C:G ∼ 7:3). We therefore inspected individual ROIs for the frequency and distribution of sequence motifs associated with GC-rich and inter-strand C/G bias as follows: (i) pG4- and pZ-motifs with the potential to form non-canonical G-quadruplexes and left-handed Z-DNA structures, respectively; (ii) tandemly arrayed partially degenerated short DNA motifs forming microsatellite-like DNA stretches; and (iii) the dispersed highly abundant permutated motifs C<sub>3</sub>G and GC<sub>3</sub>. Based on the highly variable number and spatial arrangement of these motifs, all R2 repeats might be classified into five structural families, A–E (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>), arranged according to their abundance (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). They all are formed of relatively short (34–79 bp) tandemly arrayed units, each consisting of equal numbers of pZ- and pG4-motif(s) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Therefore, each repeat unit has a polarised structure with pZ characteristics prevailing towards the spacer promoter, while pG4 structural characteristics are more prominent towards the genic promoter. Intrinsic structures of both pZ- (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) and pG4- (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) motifs were, however, specific for a particular family. An enormous structural variability in the R2 region was further emphasized by multiple subfamilies with a distinct number of tandemly arranged repetitive units and overall length, which might be defined within the majority of families (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Selected structural features that further discriminated individual families are summarized in <a reveal-id="dsz008-T1" data-open="dsz008-T1" class="link link-reveal link-table xref-fig">Table 1</a>.</p>                        <a id="137025653" scrollto-destination="137025653"></a>
<div id="dsz008-F5" data-id="dsz008-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f5.png?Expires=1576484115&amp;Signature=abN8lMyS~P4tA7BTWbgYLVuTDbvlJ4OZXanGaqO3Kzv2tB5KRaOTRxC41i2ncmttKn-MhR~PA5gVTS059jj0AMmNElY42pPULBpmCkBjStJFd-dfC5mtl7ES~9d3Xboctyq6aSM2fs03q-vaOW7xmyv6dPPuZNjuH6xrn7BX6~ZoQ7xw4T0md8A6vjJ5rM~0iLkEoEcQkLnM~XF8u7v~-1CdQFhD~DAf0-tcbpKttJD2YSsXGIR1YcQaEzKtngoAAnWAyiTg-3coPKuGnI0W5qq3rqtEMMi-olxLnZWjU1a6r5-m~9QQEMz7PSYGHJJWz1gaLpvUy8vRhPXhhFYmaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Five prominent repeated families coexist in R2 regions in pumpkin IGSs. (A) Each family (A–E) is characterized by a specific arrangement of two kinds of short sequence motifs, pZ and pG4. Individual abundant pZ- and pG4-variants are distinguished by small and numbered capital letters, respectively, within corresponding rectangles and specified in Supplementary Tables S3 and S4, respectively. C3/C4 means that two pG4-motifs (C3 and C4) are overlapping. Three pG4 prominent variants that are distinct in length of G tracts (G2–G4) are considered. A particular family may consist of several size variants (subfamilies) with a distinct number of basic tandem repeats (arrows). Each subfamily is represented by the most abundant distribution of pG4-motifs. (B) The relative abundance of individual subfamilies (left) and families (right) in the genome is estimated from the number of corresponding ROIs." path-from-xml="dsz008f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025653/dsz008f5.tif" path-from-xml="dsz008f5.tif" target="_blank">Open in new tab</a><a section="137025653" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f5.png?Expires=1576484115&amp;Signature=smeLn5E0G2XYCd4U~C57OMQO~OqSicM-Ewx2bVP5oTJ8ua23obF5vl7ZvdC5-DDkY8ZWknI~CK20qVQLzhe9Gnac2XLu~3lVV74UTYHCvxwiSi-3GQp60KXnf28zk~20ibYtYIgjHuncxGlzy-j2BzHdJr4DoS5-DpFNKp4jc8hIuFeR7ZHaRXtUuN6q951YsD~1CR59qy7p1kIb3qSqneH2zXj~~s16fC8IVnLPlqvqA0belaKhqs74TYZ3alEgQIqNzzAU-nkgACYUKxgyPB33-fvWqsrKdhm1EqlxOAyRHiK8Q~3VFCHgaht26nUJBeBIYocIYtI118AoooyARA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025653&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Five prominent repeated families coexist in R2 regions in pumpkin IGSs. (A) Each family (A–E) is characterized by a specific arrangement of two kinds of short sequence motifs, pZ and pG4. Individual abundant pZ- and pG4-variants are distinguished by small and numbered capital letters, respectively, within corresponding rectangles and specified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>, respectively. C3/C4 means that two pG4-motifs (C3 and C4) are overlapping. Three pG4 prominent variants that are distinct in length of G tracts (G<sub>2</sub>–G<sub>4</sub>) are considered. A particular family may consist of several size variants (subfamilies) with a distinct number of basic tandem repeats (arrows). Each subfamily is represented by the most abundant distribution of pG4-motifs. (B) The relative abundance of individual subfamilies (left) and families (right) in the genome is estimated from the number of corresponding ROIs.</p></div>
</div>
<div content-id="dsz008-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f5.png?Expires=1576484115&amp;Signature=abN8lMyS~P4tA7BTWbgYLVuTDbvlJ4OZXanGaqO3Kzv2tB5KRaOTRxC41i2ncmttKn-MhR~PA5gVTS059jj0AMmNElY42pPULBpmCkBjStJFd-dfC5mtl7ES~9d3Xboctyq6aSM2fs03q-vaOW7xmyv6dPPuZNjuH6xrn7BX6~ZoQ7xw4T0md8A6vjJ5rM~0iLkEoEcQkLnM~XF8u7v~-1CdQFhD~DAf0-tcbpKttJD2YSsXGIR1YcQaEzKtngoAAnWAyiTg-3coPKuGnI0W5qq3rqtEMMi-olxLnZWjU1a6r5-m~9QQEMz7PSYGHJJWz1gaLpvUy8vRhPXhhFYmaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Five prominent repeated families coexist in R2 regions in pumpkin IGSs. (A) Each family (A–E) is characterized by a specific arrangement of two kinds of short sequence motifs, pZ and pG4. Individual abundant pZ- and pG4-variants are distinguished by small and numbered capital letters, respectively, within corresponding rectangles and specified in Supplementary Tables S3 and S4, respectively. C3/C4 means that two pG4-motifs (C3 and C4) are overlapping. Three pG4 prominent variants that are distinct in length of G tracts (G2–G4) are considered. A particular family may consist of several size variants (subfamilies) with a distinct number of basic tandem repeats (arrows). Each subfamily is represented by the most abundant distribution of pG4-motifs. (B) The relative abundance of individual subfamilies (left) and families (right) in the genome is estimated from the number of corresponding ROIs." path-from-xml="dsz008f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025653/dsz008f5.tif" path-from-xml="dsz008f5.tif" target="_blank">Open in new tab</a><a section="137025653" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f5.png?Expires=1576484115&amp;Signature=smeLn5E0G2XYCd4U~C57OMQO~OqSicM-Ewx2bVP5oTJ8ua23obF5vl7ZvdC5-DDkY8ZWknI~CK20qVQLzhe9Gnac2XLu~3lVV74UTYHCvxwiSi-3GQp60KXnf28zk~20ibYtYIgjHuncxGlzy-j2BzHdJr4DoS5-DpFNKp4jc8hIuFeR7ZHaRXtUuN6q951YsD~1CR59qy7p1kIb3qSqneH2zXj~~s16fC8IVnLPlqvqA0belaKhqs74TYZ3alEgQIqNzzAU-nkgACYUKxgyPB33-fvWqsrKdhm1EqlxOAyRHiK8Q~3VFCHgaht26nUJBeBIYocIYtI118AoooyARA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025653&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Five prominent repeated families coexist in R2 regions in pumpkin IGSs. (A) Each family (A–E) is characterized by a specific arrangement of two kinds of short sequence motifs, pZ and pG4. Individual abundant pZ- and pG4-variants are distinguished by small and numbered capital letters, respectively, within corresponding rectangles and specified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>, respectively. C3/C4 means that two pG4-motifs (C3 and C4) are overlapping. Three pG4 prominent variants that are distinct in length of G tracts (G<sub>2</sub>–G<sub>4</sub>) are considered. A particular family may consist of several size variants (subfamilies) with a distinct number of basic tandem repeats (arrows). Each subfamily is represented by the most abundant distribution of pG4-motifs. (B) The relative abundance of individual subfamilies (left) and families (right) in the genome is estimated from the number of corresponding ROIs.</p></div>
</div>                        <a id="137025654" scrollto-destination="137025654"></a>
<div content-id="dsz008-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz008-T1" class="table-wrap-title" data-id="dsz008-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Selected features characterizing individual R2 families</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>Feature </th>
<th>Specification </th>
<th>Family </th>
<th> </th>
<th> </th>
<th> </th>
<th> </th>
</tr>
<tr>
<th> </th>
<th> </th>
<th>A </th>
<th>B </th>
<th>C </th>
<th>D </th>
<th>E </th>
</tr>
</thead>
<tbody>
<tr>
<td>Overall size<sup>a,b,c</sup> </td>
<td>mean [bp] </td>
<td>341; 294; 433 </td>
<td>303.5; 284.5; 396.5 </td>
<td>424; 427; 519 </td>
<td>207; 189; 295 </td>
<td>264; 215; 358 </td>
</tr>
<tr>
<td> </td>
<td>min. [bp] </td>
<td>199; 186; 286 </td>
<td>216; 205; 318 </td>
<td>309; 310; 401 </td>
<td>194; 133; 283 </td>
<td>200; 170; 291 </td>
</tr>
<tr>
<td> </td>
<td>max. [bp] </td>
<td>446; 435; 539 </td>
<td>415; 403; 511 </td>
<td>551; 553; 643 </td>
<td>239; 201; 327 </td>
<td>310; 275; 401 </td>
</tr>
<tr>
<td> </td>
<td>max. – min. [bp] </td>
<td>247; 249; 253 </td>
<td>199; 198; 193 </td>
<td>242; 243; 242 </td>
<td>45; 68; 44 </td>
<td>110; 105; 110 </td>
</tr>
<tr>
<td>Tandem unit<sup>a,d</sup> </td>
<td>consensus mean size [bp] </td>
<td>34 </td>
<td>79 [39]<sup>e</sup> </td>
<td>42 </td>
<td>38 </td>
<td>37 </td>
</tr>
<tr>
<td> </td>
<td>copy numbers range </td>
<td>4.8–12.6 </td>
<td>2.0–4.3 [7.1]<sup>e</sup> </td>
<td>6.4–13.5 </td>
<td>3.6 </td>
<td>2.4–3.8 </td>
</tr>
<tr>
<td>pZ-motifs<sup>a,f</sup> </td>
<td>copy numbers range </td>
<td>5–12 </td>
<td>6–10 </td>
<td>6–12 </td>
<td>5 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>7(4) </td>
<td>6(1) </td>
<td>5(3) </td>
<td>5(4) </td>
<td>5(2) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant(s) </td>
<td>(GC)<sub>6</sub> </td>
<td>(GC)<sub>6</sub>; (CG)<sub>6</sub>C </td>
<td>(CG)<sub>5</sub>C </td>
<td>(GC)<sub>5</sub> </td>
<td>(CG)<sub>5</sub>C; (CG)<sub>6</sub>C </td>
</tr>
<tr>
<td>pG4-motifs<sup>a,g,c</sup> </td>
<td>copy numbers range<sup>h</sup> </td>
<td>2–3 </td>
<td>4–7 </td>
<td>9–14 </td>
<td>4 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>8(6) </td>
<td>10(7) </td>
<td>9(6) </td>
<td>6(5) </td>
<td>8(4) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant<sup>i</sup> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>2</sub>; G<sub>3</sub> </td>
<td>G<sub>4</sub> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>3</sub> </td>
</tr>
<tr>
<td>Microsatellite </td>
<td>consensus </td>
<td>nd </td>
<td>c<sub>4</sub>g; c<sub>4</sub>gc<sub>3</sub>(gc)<sub>3</sub><sup>j</sup> </td>
<td>cgc<sub>3</sub> </td>
<td>nd </td>
<td>c<sub>4</sub>g; ccacgc </td>
</tr>
<tr>
<td> </td>
<td>abundance [% of ROIs] </td>
<td>nd </td>
<td>59; 30<sup>k</sup> </td>
<td>92 </td>
<td>nd </td>
<td>97; 76 </td>
</tr>
<tr>
<td> </td>
<td>copy number mean </td>
<td>nd </td>
<td>7.6; 5.6 </td>
<td>43.2 </td>
<td>nd </td>
<td>26.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number min. </td>
<td>nd </td>
<td>6.6; 5.6 </td>
<td>13.0 </td>
<td>nd </td>
<td>10.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number max. </td>
<td>nd </td>
<td>19.6; 8.6 </td>
<td>62.2 </td>
<td>nd </td>
<td>33.6; 5.3 </td>
</tr>
<tr>
<td>Ratio GC<sub>3</sub>/C<sub>3</sub>G<sup>c</sup> </td>
<td>mean </td>
<td>1.5 </td>
<td>1.1 [0.9]<sup>e</sup> </td>
<td>1.0 </td>
<td>1.2 </td>
<td>1.3 </td>
</tr>
<tr>
<td>RTA<sub>n</sub>T-size<sup>c,l</sup> </td>
<td>mean; min.; max. [bp] </td>
<td>39; 21; 49 </td>
<td>12; 11; 14 </td>
<td>8; 0; 8 </td>
<td>10; 9; 10 </td>
<td>9; 8; 9 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz008-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>Feature </th>
<th>Specification </th>
<th>Family </th>
<th> </th>
<th> </th>
<th> </th>
<th> </th>
</tr>
<tr>
<th> </th>
<th> </th>
<th>A </th>
<th>B </th>
<th>C </th>
<th>D </th>
<th>E </th>
</tr>
</thead>
<tbody>
<tr>
<td>Overall size<sup>a,b,c</sup> </td>
<td>mean [bp] </td>
<td>341; 294; 433 </td>
<td>303.5; 284.5; 396.5 </td>
<td>424; 427; 519 </td>
<td>207; 189; 295 </td>
<td>264; 215; 358 </td>
</tr>
<tr>
<td> </td>
<td>min. [bp] </td>
<td>199; 186; 286 </td>
<td>216; 205; 318 </td>
<td>309; 310; 401 </td>
<td>194; 133; 283 </td>
<td>200; 170; 291 </td>
</tr>
<tr>
<td> </td>
<td>max. [bp] </td>
<td>446; 435; 539 </td>
<td>415; 403; 511 </td>
<td>551; 553; 643 </td>
<td>239; 201; 327 </td>
<td>310; 275; 401 </td>
</tr>
<tr>
<td> </td>
<td>max. – min. [bp] </td>
<td>247; 249; 253 </td>
<td>199; 198; 193 </td>
<td>242; 243; 242 </td>
<td>45; 68; 44 </td>
<td>110; 105; 110 </td>
</tr>
<tr>
<td>Tandem unit<sup>a,d</sup> </td>
<td>consensus mean size [bp] </td>
<td>34 </td>
<td>79 [39]<sup>e</sup> </td>
<td>42 </td>
<td>38 </td>
<td>37 </td>
</tr>
<tr>
<td> </td>
<td>copy numbers range </td>
<td>4.8–12.6 </td>
<td>2.0–4.3 [7.1]<sup>e</sup> </td>
<td>6.4–13.5 </td>
<td>3.6 </td>
<td>2.4–3.8 </td>
</tr>
<tr>
<td>pZ-motifs<sup>a,f</sup> </td>
<td>copy numbers range </td>
<td>5–12 </td>
<td>6–10 </td>
<td>6–12 </td>
<td>5 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>7(4) </td>
<td>6(1) </td>
<td>5(3) </td>
<td>5(4) </td>
<td>5(2) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant(s) </td>
<td>(GC)<sub>6</sub> </td>
<td>(GC)<sub>6</sub>; (CG)<sub>6</sub>C </td>
<td>(CG)<sub>5</sub>C </td>
<td>(GC)<sub>5</sub> </td>
<td>(CG)<sub>5</sub>C; (CG)<sub>6</sub>C </td>
</tr>
<tr>
<td>pG4-motifs<sup>a,g,c</sup> </td>
<td>copy numbers range<sup>h</sup> </td>
<td>2–3 </td>
<td>4–7 </td>
<td>9–14 </td>
<td>4 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>8(6) </td>
<td>10(7) </td>
<td>9(6) </td>
<td>6(5) </td>
<td>8(4) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant<sup>i</sup> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>2</sub>; G<sub>3</sub> </td>
<td>G<sub>4</sub> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>3</sub> </td>
</tr>
<tr>
<td>Microsatellite </td>
<td>consensus </td>
<td>nd </td>
<td>c<sub>4</sub>g; c<sub>4</sub>gc<sub>3</sub>(gc)<sub>3</sub><sup>j</sup> </td>
<td>cgc<sub>3</sub> </td>
<td>nd </td>
<td>c<sub>4</sub>g; ccacgc </td>
</tr>
<tr>
<td> </td>
<td>abundance [% of ROIs] </td>
<td>nd </td>
<td>59; 30<sup>k</sup> </td>
<td>92 </td>
<td>nd </td>
<td>97; 76 </td>
</tr>
<tr>
<td> </td>
<td>copy number mean </td>
<td>nd </td>
<td>7.6; 5.6 </td>
<td>43.2 </td>
<td>nd </td>
<td>26.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number min. </td>
<td>nd </td>
<td>6.6; 5.6 </td>
<td>13.0 </td>
<td>nd </td>
<td>10.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number max. </td>
<td>nd </td>
<td>19.6; 8.6 </td>
<td>62.2 </td>
<td>nd </td>
<td>33.6; 5.3 </td>
</tr>
<tr>
<td>Ratio GC<sub>3</sub>/C<sub>3</sub>G<sup>c</sup> </td>
<td>mean </td>
<td>1.5 </td>
<td>1.1 [0.9]<sup>e</sup> </td>
<td>1.0 </td>
<td>1.2 </td>
<td>1.3 </td>
</tr>
<tr>
<td>RTA<sub>n</sub>T-size<sup>c,l</sup> </td>
<td>mean; min.; max. [bp] </td>
<td>39; 21; 49 </td>
<td>12; 11; 14 </td>
<td>8; 0; 8 </td>
<td>10; 9; 10 </td>
<td>9; 8; 9 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>For individual subfamilies, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Three values for each descriptive statistical parameter are derived from three independent methods used for repeat size estimations and representing: (i) region with GC content &gt;75%, (ii) Tandem Repeats Finder and (iii) distance between conserved unique bordering sequences, respectively.</p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">c</div>
<p>For comparative statistics, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>.</p>
</div></div>
<div content-id="tblfn4"><div class="fn">
<div class="label">d</div>
<p>For the consensus, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>
</div></div>
<div content-id="tblfn5"><div class="fn">
<div class="label">e</div>
<p>Values for the B8a subfamily are in square brackets, if significantly different from other B-subfamilies.</p>
</div></div>
<div content-id="tblfn6"><div class="fn">
<div class="label">f</div>
<p>Frequency of abundant pZ-motifs are comprehensively specified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p>
</div></div>
<div content-id="tblfn7"><div class="fn">
<div class="label">g</div>
<p>Frequency of abundant pG4-variants are comprehensively specified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>.</p>
</div></div>
<div content-id="tblfn8"><div class="fn">
<div class="label">h</div>
<p>Only motifs with G<sub>3</sub> and G<sub>4</sub> tracts are considered.</p>
</div></div>
<div content-id="tblfn9"><div class="fn">
<div class="label">i</div>
<p>Only variants with G-tracts of different lengths (G<sub>2</sub>–G<sub>4</sub>) are distinguished.</p>
</div></div>
<div content-id="tblfn10"><div class="fn">
<div class="label">j</div>
<p>Longer motif is specific for the B8a-subfamily.</p>
</div></div>
<div content-id="tblfn11"><div class="fn">
<div class="label">k</div>
<p>Computed from the number of ROIs in B8 and B8a subfamilies, respectively.</p>
</div></div>
<div content-id="tblfn12"><div class="fn">
<div class="label">l</div>
<p>Length of the spatially linked AT-region (RTA<sub>n</sub>T) located between the putative spacer promoter and R1 minisatellite (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025654" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz008-T1" class="table-wrap-title" data-id="dsz008-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Selected features characterizing individual R2 families</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>Feature </th>
<th>Specification </th>
<th>Family </th>
<th> </th>
<th> </th>
<th> </th>
<th> </th>
</tr>
<tr>
<th> </th>
<th> </th>
<th>A </th>
<th>B </th>
<th>C </th>
<th>D </th>
<th>E </th>
</tr>
</thead>
<tbody>
<tr>
<td>Overall size<sup>a,b,c</sup> </td>
<td>mean [bp] </td>
<td>341; 294; 433 </td>
<td>303.5; 284.5; 396.5 </td>
<td>424; 427; 519 </td>
<td>207; 189; 295 </td>
<td>264; 215; 358 </td>
</tr>
<tr>
<td> </td>
<td>min. [bp] </td>
<td>199; 186; 286 </td>
<td>216; 205; 318 </td>
<td>309; 310; 401 </td>
<td>194; 133; 283 </td>
<td>200; 170; 291 </td>
</tr>
<tr>
<td> </td>
<td>max. [bp] </td>
<td>446; 435; 539 </td>
<td>415; 403; 511 </td>
<td>551; 553; 643 </td>
<td>239; 201; 327 </td>
<td>310; 275; 401 </td>
</tr>
<tr>
<td> </td>
<td>max. – min. [bp] </td>
<td>247; 249; 253 </td>
<td>199; 198; 193 </td>
<td>242; 243; 242 </td>
<td>45; 68; 44 </td>
<td>110; 105; 110 </td>
</tr>
<tr>
<td>Tandem unit<sup>a,d</sup> </td>
<td>consensus mean size [bp] </td>
<td>34 </td>
<td>79 [39]<sup>e</sup> </td>
<td>42 </td>
<td>38 </td>
<td>37 </td>
</tr>
<tr>
<td> </td>
<td>copy numbers range </td>
<td>4.8–12.6 </td>
<td>2.0–4.3 [7.1]<sup>e</sup> </td>
<td>6.4–13.5 </td>
<td>3.6 </td>
<td>2.4–3.8 </td>
</tr>
<tr>
<td>pZ-motifs<sup>a,f</sup> </td>
<td>copy numbers range </td>
<td>5–12 </td>
<td>6–10 </td>
<td>6–12 </td>
<td>5 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>7(4) </td>
<td>6(1) </td>
<td>5(3) </td>
<td>5(4) </td>
<td>5(2) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant(s) </td>
<td>(GC)<sub>6</sub> </td>
<td>(GC)<sub>6</sub>; (CG)<sub>6</sub>C </td>
<td>(CG)<sub>5</sub>C </td>
<td>(GC)<sub>5</sub> </td>
<td>(CG)<sub>5</sub>C; (CG)<sub>6</sub>C </td>
</tr>
<tr>
<td>pG4-motifs<sup>a,g,c</sup> </td>
<td>copy numbers range<sup>h</sup> </td>
<td>2–3 </td>
<td>4–7 </td>
<td>9–14 </td>
<td>4 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>8(6) </td>
<td>10(7) </td>
<td>9(6) </td>
<td>6(5) </td>
<td>8(4) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant<sup>i</sup> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>2</sub>; G<sub>3</sub> </td>
<td>G<sub>4</sub> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>3</sub> </td>
</tr>
<tr>
<td>Microsatellite </td>
<td>consensus </td>
<td>nd </td>
<td>c<sub>4</sub>g; c<sub>4</sub>gc<sub>3</sub>(gc)<sub>3</sub><sup>j</sup> </td>
<td>cgc<sub>3</sub> </td>
<td>nd </td>
<td>c<sub>4</sub>g; ccacgc </td>
</tr>
<tr>
<td> </td>
<td>abundance [% of ROIs] </td>
<td>nd </td>
<td>59; 30<sup>k</sup> </td>
<td>92 </td>
<td>nd </td>
<td>97; 76 </td>
</tr>
<tr>
<td> </td>
<td>copy number mean </td>
<td>nd </td>
<td>7.6; 5.6 </td>
<td>43.2 </td>
<td>nd </td>
<td>26.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number min. </td>
<td>nd </td>
<td>6.6; 5.6 </td>
<td>13.0 </td>
<td>nd </td>
<td>10.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number max. </td>
<td>nd </td>
<td>19.6; 8.6 </td>
<td>62.2 </td>
<td>nd </td>
<td>33.6; 5.3 </td>
</tr>
<tr>
<td>Ratio GC<sub>3</sub>/C<sub>3</sub>G<sup>c</sup> </td>
<td>mean </td>
<td>1.5 </td>
<td>1.1 [0.9]<sup>e</sup> </td>
<td>1.0 </td>
<td>1.2 </td>
<td>1.3 </td>
</tr>
<tr>
<td>RTA<sub>n</sub>T-size<sup>c,l</sup> </td>
<td>mean; min.; max. [bp] </td>
<td>39; 21; 49 </td>
<td>12; 11; 14 </td>
<td>8; 0; 8 </td>
<td>10; 9; 10 </td>
<td>9; 8; 9 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz008-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>Feature </th>
<th>Specification </th>
<th>Family </th>
<th> </th>
<th> </th>
<th> </th>
<th> </th>
</tr>
<tr>
<th> </th>
<th> </th>
<th>A </th>
<th>B </th>
<th>C </th>
<th>D </th>
<th>E </th>
</tr>
</thead>
<tbody>
<tr>
<td>Overall size<sup>a,b,c</sup> </td>
<td>mean [bp] </td>
<td>341; 294; 433 </td>
<td>303.5; 284.5; 396.5 </td>
<td>424; 427; 519 </td>
<td>207; 189; 295 </td>
<td>264; 215; 358 </td>
</tr>
<tr>
<td> </td>
<td>min. [bp] </td>
<td>199; 186; 286 </td>
<td>216; 205; 318 </td>
<td>309; 310; 401 </td>
<td>194; 133; 283 </td>
<td>200; 170; 291 </td>
</tr>
<tr>
<td> </td>
<td>max. [bp] </td>
<td>446; 435; 539 </td>
<td>415; 403; 511 </td>
<td>551; 553; 643 </td>
<td>239; 201; 327 </td>
<td>310; 275; 401 </td>
</tr>
<tr>
<td> </td>
<td>max. – min. [bp] </td>
<td>247; 249; 253 </td>
<td>199; 198; 193 </td>
<td>242; 243; 242 </td>
<td>45; 68; 44 </td>
<td>110; 105; 110 </td>
</tr>
<tr>
<td>Tandem unit<sup>a,d</sup> </td>
<td>consensus mean size [bp] </td>
<td>34 </td>
<td>79 [39]<sup>e</sup> </td>
<td>42 </td>
<td>38 </td>
<td>37 </td>
</tr>
<tr>
<td> </td>
<td>copy numbers range </td>
<td>4.8–12.6 </td>
<td>2.0–4.3 [7.1]<sup>e</sup> </td>
<td>6.4–13.5 </td>
<td>3.6 </td>
<td>2.4–3.8 </td>
</tr>
<tr>
<td>pZ-motifs<sup>a,f</sup> </td>
<td>copy numbers range </td>
<td>5–12 </td>
<td>6–10 </td>
<td>6–12 </td>
<td>5 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>7(4) </td>
<td>6(1) </td>
<td>5(3) </td>
<td>5(4) </td>
<td>5(2) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant(s) </td>
<td>(GC)<sub>6</sub> </td>
<td>(GC)<sub>6</sub>; (CG)<sub>6</sub>C </td>
<td>(CG)<sub>5</sub>C </td>
<td>(GC)<sub>5</sub> </td>
<td>(CG)<sub>5</sub>C; (CG)<sub>6</sub>C </td>
</tr>
<tr>
<td>pG4-motifs<sup>a,g,c</sup> </td>
<td>copy numbers range<sup>h</sup> </td>
<td>2–3 </td>
<td>4–7 </td>
<td>9–14 </td>
<td>4 </td>
<td>5–8 </td>
</tr>
<tr>
<td> </td>
<td>no of variants (family-specific) </td>
<td>8(6) </td>
<td>10(7) </td>
<td>9(6) </td>
<td>6(5) </td>
<td>8(4) </td>
</tr>
<tr>
<td> </td>
<td>the most abundant variant<sup>i</sup> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>2</sub>; G<sub>3</sub> </td>
<td>G<sub>4</sub> </td>
<td>G<sub>2</sub> </td>
<td>G<sub>3</sub> </td>
</tr>
<tr>
<td>Microsatellite </td>
<td>consensus </td>
<td>nd </td>
<td>c<sub>4</sub>g; c<sub>4</sub>gc<sub>3</sub>(gc)<sub>3</sub><sup>j</sup> </td>
<td>cgc<sub>3</sub> </td>
<td>nd </td>
<td>c<sub>4</sub>g; ccacgc </td>
</tr>
<tr>
<td> </td>
<td>abundance [% of ROIs] </td>
<td>nd </td>
<td>59; 30<sup>k</sup> </td>
<td>92 </td>
<td>nd </td>
<td>97; 76 </td>
</tr>
<tr>
<td> </td>
<td>copy number mean </td>
<td>nd </td>
<td>7.6; 5.6 </td>
<td>43.2 </td>
<td>nd </td>
<td>26.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number min. </td>
<td>nd </td>
<td>6.6; 5.6 </td>
<td>13.0 </td>
<td>nd </td>
<td>10.2; 4.3 </td>
</tr>
<tr>
<td> </td>
<td>copy number max. </td>
<td>nd </td>
<td>19.6; 8.6 </td>
<td>62.2 </td>
<td>nd </td>
<td>33.6; 5.3 </td>
</tr>
<tr>
<td>Ratio GC<sub>3</sub>/C<sub>3</sub>G<sup>c</sup> </td>
<td>mean </td>
<td>1.5 </td>
<td>1.1 [0.9]<sup>e</sup> </td>
<td>1.0 </td>
<td>1.2 </td>
<td>1.3 </td>
</tr>
<tr>
<td>RTA<sub>n</sub>T-size<sup>c,l</sup> </td>
<td>mean; min.; max. [bp] </td>
<td>39; 21; 49 </td>
<td>12; 11; 14 </td>
<td>8; 0; 8 </td>
<td>10; 9; 10 </td>
<td>9; 8; 9 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>For individual subfamilies, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Three values for each descriptive statistical parameter are derived from three independent methods used for repeat size estimations and representing: (i) region with GC content &gt;75%, (ii) Tandem Repeats Finder and (iii) distance between conserved unique bordering sequences, respectively.</p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">c</div>
<p>For comparative statistics, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>.</p>
</div></div>
<div content-id="tblfn4"><div class="fn">
<div class="label">d</div>
<p>For the consensus, see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>
</div></div>
<div content-id="tblfn5"><div class="fn">
<div class="label">e</div>
<p>Values for the B8a subfamily are in square brackets, if significantly different from other B-subfamilies.</p>
</div></div>
<div content-id="tblfn6"><div class="fn">
<div class="label">f</div>
<p>Frequency of abundant pZ-motifs are comprehensively specified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p>
</div></div>
<div content-id="tblfn7"><div class="fn">
<div class="label">g</div>
<p>Frequency of abundant pG4-variants are comprehensively specified in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>.</p>
</div></div>
<div content-id="tblfn8"><div class="fn">
<div class="label">h</div>
<p>Only motifs with G<sub>3</sub> and G<sub>4</sub> tracts are considered.</p>
</div></div>
<div content-id="tblfn9"><div class="fn">
<div class="label">i</div>
<p>Only variants with G-tracts of different lengths (G<sub>2</sub>–G<sub>4</sub>) are distinguished.</p>
</div></div>
<div content-id="tblfn10"><div class="fn">
<div class="label">j</div>
<p>Longer motif is specific for the B8a-subfamily.</p>
</div></div>
<div content-id="tblfn11"><div class="fn">
<div class="label">k</div>
<p>Computed from the number of ROIs in B8 and B8a subfamilies, respectively.</p>
</div></div>
<div content-id="tblfn12"><div class="fn">
<div class="label">l</div>
<p>Length of the spatially linked AT-region (RTA<sub>n</sub>T) located between the putative spacer promoter and R1 minisatellite (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025654" target="_blank">Open in new tab</a></div>
</div></div>
<p>Families A and C represent highly heterogeneous populations of size variants built of variable numbers of monomeric units, each composed of single pZ- and single pG4-motifs. Both motifs, however, significantly differ between families in structure. In Family A, the majority of pG4-motifs are formed of short G<sub>2</sub> tracts, resulting in the shortest monomeric unit. In contrast, G<sub>3</sub> and G<sub>4</sub> tracts are characteristic for Family C, resulting in a substantially longer unit with higher potential to form G4-conformations. Nevertheless, in both families, pG4 variants with greater potential to form G4 conformations are accumulated towards the genic promoter. In Family A, the number of such pG4 variants appears to be independent of the considerably variable total number of pZ- and pG4-motifs in individual size subfamilies. Such an arrangement may result in a polarized structure of the R2 region as a whole. It is necessary to stress that the pG4 variant U1 with the highest potential to form G-quadruplexes is significantly more abundant in Family C in comparison to other families (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p>
<p>Family B is exceptional in carrying the longest basic monomeric repeated units, each built of alternating arrangements of two pZ- and two pG4-variants. Such an arrangement results in partial homology within each unit. Although both abundant subfamilies B8 and B8a share the same number of pZ-motifs, they differ substantially in the occurrence of specific pG4-motifs (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) and relative content of GC<sub>3</sub> and C<sub>3</sub>G motifs (<a reveal-id="dsz008-T1" data-open="dsz008-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>
<p>Family D is the shortest and the most homogeneous in size, built of a unique copy number of basic monomeric units, each composed of single pZ- and single pG4-motifs, both with specific structural features (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>).</p>
<p>In size, the moderately variable Family E is built of monomers that are closely related to those found in Family C; however, they are, on average, shorter due to the absence of pG4 motifs with G<sub>4</sub> tracts that are frequent in Family C. In addition, two highly conserved pZ-variants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) as well as four pG4-variants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) are specific for this family.</p>
<p>Considerably high structural variability occurred within almost each pG4 position along each subfamily (<a reveal-id="dsz008-F6" data-open="dsz008-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). In contrast, variability in adjacent pZ motifs appeared rather negligible, suggesting a diverse mode of evolution. Substantial variability within each family was further demonstrated using comprehensive descriptive statistics performed for the number of quantitative features within each corresponding subfamily (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Finally, significant differences among all five R2 families were supported by comparative statistics in all evaluated structural characteristics with only a few exceptions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). </p>                        <a id="137025660" scrollto-destination="137025660"></a>
<div id="dsz008-F6" data-id="dsz008-F6" class="fig fig-section">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f6.png?Expires=1576484115&amp;Signature=Avkh832NvuJvpDYLlcvHixBDrM1g64ZT4V9FuR3r0nuOx92odoQmORP-JRHeIga48575200Ntcfr9kiRNFga6s7WuRedet8Iknpe0rYSVPMl8W4BjNuHbEKLU9cRr-BblS6JM71py45vYP4di8~adkhL-9LzTEDDzEnCqWEFjh1W8Cie~VDVfiqPa67hVH0BRDoYTCYV8os6qoS527NI~o9iVbqC5qHrb7MgOZl0zxIpoR1KjxSChMYPGT8wH6uhJlYiSjQEdO6pzOxBrKqFonEfLNL61ntjG4B6mM-baX71EmZ9dWP5ruagJV8JLdwVbl3oxj0OcsJRCq~ehIV~Kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structural variability of pG4- and pZ-motifs in distinct positions along the R2 minisatellites. The relative abundance of particular pG4- and pZ-variants are represented by charts located above and under the corresponding positions, respectively, in selected A9- and C9-sub-families (for more annotation, see Fig. 5). The pG4-variants are distinguished by numerals representing the in silico computed potential (score) to form a G4-conformation. Each pG4-variant may, however, be composed of several sequence motifs with the same score. Only variants that occurred more frequently than in one ROI are distinguished." path-from-xml="dsz008f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025660/dsz008f6.tif" path-from-xml="dsz008f6.tif" target="_blank">Open in new tab</a><a section="137025660" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f6.png?Expires=1576484115&amp;Signature=RWZxMRPS6smpfFJ2M~1u1nNmm2IcuVpnGN~ZF0G2t06xo9~aaTcpNaTJF2tad~RnkcrWXQIzlXTuRhXvZQi1eXDmalnIhsz6LAAvYpT5bqnuH1Fsi0heNFVyyRcDpTythvdgfQK6oaOjY~S7V76Lop1ss86~rUU1JMLvlklstyzrR96jEbR66Vk0geqzfhkS~pcwTxQwgaC1gyOecS-RivuXzj4tcgPwBJLJs4UDIgodYp4TxjiQZgndVGv77mZNzVdmsHJCFQpgEQCN0lerxOmCnbMtlY7t1sjsoFBcJgrjM~GhlLy6EZWUHz1GAycomttnqutD2QBYUiJFwRhvcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025660&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Structural variability of pG4- and pZ-motifs in distinct positions along the R2 minisatellites. The relative abundance of particular pG4- and pZ-variants are represented by charts located above and under the corresponding positions, respectively, in selected A9- and C9-sub-families (for more annotation, see <a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The pG4-variants are distinguished by numerals representing the <em>in silico</em> computed potential (score) to form a G4-conformation. Each pG4-variant may, however, be composed of several sequence motifs with the same score. Only variants that occurred more frequently than in one ROI are distinguished.</p></div>
</div>
<div content-id="dsz008-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f6.png?Expires=1576484115&amp;Signature=Avkh832NvuJvpDYLlcvHixBDrM1g64ZT4V9FuR3r0nuOx92odoQmORP-JRHeIga48575200Ntcfr9kiRNFga6s7WuRedet8Iknpe0rYSVPMl8W4BjNuHbEKLU9cRr-BblS6JM71py45vYP4di8~adkhL-9LzTEDDzEnCqWEFjh1W8Cie~VDVfiqPa67hVH0BRDoYTCYV8os6qoS527NI~o9iVbqC5qHrb7MgOZl0zxIpoR1KjxSChMYPGT8wH6uhJlYiSjQEdO6pzOxBrKqFonEfLNL61ntjG4B6mM-baX71EmZ9dWP5ruagJV8JLdwVbl3oxj0OcsJRCq~ehIV~Kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structural variability of pG4- and pZ-motifs in distinct positions along the R2 minisatellites. The relative abundance of particular pG4- and pZ-variants are represented by charts located above and under the corresponding positions, respectively, in selected A9- and C9-sub-families (for more annotation, see Fig. 5). The pG4-variants are distinguished by numerals representing the in silico computed potential (score) to form a G4-conformation. Each pG4-variant may, however, be composed of several sequence motifs with the same score. Only variants that occurred more frequently than in one ROI are distinguished." path-from-xml="dsz008f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025660/dsz008f6.tif" path-from-xml="dsz008f6.tif" target="_blank">Open in new tab</a><a section="137025660" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f6.png?Expires=1576484115&amp;Signature=RWZxMRPS6smpfFJ2M~1u1nNmm2IcuVpnGN~ZF0G2t06xo9~aaTcpNaTJF2tad~RnkcrWXQIzlXTuRhXvZQi1eXDmalnIhsz6LAAvYpT5bqnuH1Fsi0heNFVyyRcDpTythvdgfQK6oaOjY~S7V76Lop1ss86~rUU1JMLvlklstyzrR96jEbR66Vk0geqzfhkS~pcwTxQwgaC1gyOecS-RivuXzj4tcgPwBJLJs4UDIgodYp4TxjiQZgndVGv77mZNzVdmsHJCFQpgEQCN0lerxOmCnbMtlY7t1sjsoFBcJgrjM~GhlLy6EZWUHz1GAycomttnqutD2QBYUiJFwRhvcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025660&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Structural variability of pG4- and pZ-motifs in distinct positions along the R2 minisatellites. The relative abundance of particular pG4- and pZ-variants are represented by charts located above and under the corresponding positions, respectively, in selected A9- and C9-sub-families (for more annotation, see <a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The pG4-variants are distinguished by numerals representing the <em>in silico</em> computed potential (score) to form a G4-conformation. Each pG4-variant may, however, be composed of several sequence motifs with the same score. Only variants that occurred more frequently than in one ROI are distinguished.</p></div>
</div>                    <h3 scrollto-destination="137025661" id="137025661" class="section-title">3.4. Correlation between the structural variability in R2 repeats and adjacent regions of pumpkin IGSs</h3>
<p>Suggesting the concerted evolution of IGSs as a unit, we questioned whether the structural variability at the R2 region was reflected in the distal R1 region with comparable multimodal length variability (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span> and B). Both regions are GC rich and share some motifs, particularly C<sub>4</sub>GC, in similar abundances, suggesting a common origin. Nevertheless, a rather low overall sequence similarity between regions was detected by dot plot analyses (<a reveal-id="dsz008-F2" data-open="dsz008-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8A</a></span>) and further supported by the occurrence of predominantly different short GC-rich motifs in both regions. (i) The pZ-motifs prevail in R2 as (CG)<sub>n</sub> variants (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), whereas more complex (PyPu)<sub>n</sub> variants are common in R1, and therefore, except for the (CG)<sub>4</sub>C motif, there is no shared pZ-motif variant in either region (compare <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> with <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). (ii) No pG4-motif variant with G<sub>4</sub> stretches is shared by the two regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). (iii) The ratio between G<sub>3</sub>C and C<sub>3</sub>G motifs is mostly reversed between R1 and R2 repeats.</p>
<p>The mutual distribution of the pZ- and pG4-motifs along the entire R1 region and its overall length and size variability suggest that five basic structural variants, S, T, U, V and X (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>), coexist in the R1 region. Families S, U and X are short with low size variability, whereas families T and V are highly variable in size but substantially different in the mutual distribution of pZ- and pG4-motifs. Family T is composed of a large number of pZ-motifs that are regularly distributed along the entire length and complemented with a few pG4-motifs adjacent to the spacer promoter. In contrast, Family V is full of pG4-motifs that are regularly arranged along the entire region and complemented with three pZ-motifs adjacent to the spacer promoter. Among the short R1 families, S is distinct, having a lower number of pZ-motifs in comparison to the pG4-motifs, whereas both the U and X families contain high accumulations of pZ-motifs, significantly exceeding the number of pG4-motifs. The U family has a lower number of pZ-motifs than the X family. Common, as well as specific pZ-motifs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>) are present in all R1 families.</p>
<p>To evaluate whether distinct R1 families are specifically linked to corresponding R2 families, we analysed a limited number of ROIs composed of both R1 and R2 regions (or the larger part) linked to a spacer promoter. We showed that for each R1 family, all corresponding ROIs fall into a unique R2 family and <em>vice versa</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). It is necessary to stress that the R2 satellite families A and C, which are rather long and variable in size, are linked to the short and size-invariant R1 families S and U, respectively. In contrast, the short and less variable in size R2 Families D and B are linked to the long and highly variable in size R1 families V and T, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 12</a></span>).</p>
<p>As both R1 and R2 repetitive regions show signs of common evolution, we extended our effort to evaluate structural variability at adjacent non-repetitive promoter regions. The phylogenetic analysis well separated the genic and spacer promoter duplicates. In addition, both genic and spacer promoters were split into five clusters 1<sup>g</sup>–5<sup>g</sup> and 1<sup>s</sup>–5<sup>s</sup>, respectively, supported by bootstrap values ranging from 52 to 96% (<a reveal-id="dsz008-F7" data-open="dsz008-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). All promoter sequences forming particular cluster were linked to a unique family from the R1 and R2 regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>) and <em>vice versa</em>, suggesting a common evolution of repeated and unique sequences in pumpkin IGS. </p>                        <a id="137025666" scrollto-destination="137025666"></a>
<div id="dsz008-F7" data-id="dsz008-F7" class="fig fig-section">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f7.png?Expires=1576484115&amp;Signature=Y6Nazp8ibE0iV-xDkzJnOT88iHQCVFD3-wsSCf9lCz2i6eHUQJmdBFE1TPHOugMvBDISwuDbK4PIavMu5mUICT2aYymE~jn4GgWAgvDMh0Oag3gEnXA1~kB1axbDUgdC9pWiA26O0Oomoo0pIa3b2D3AcDkv-LhzLMfh60q2GeS--E318NGgEK6eaFzbrL4mXZ1HNi6U9Jjtj4B0t-0dhkdVye3~L1~g5Mf8t2JSJOZtu4gF-9uWP1WfHqyc30YxVXG6QHURktAzDaADtu7Tv53z1mE-xJzXtn3Ae92ofc5Ksjomnamou62CE7ILewhBzK7Q39tZlMwejeUgORhgdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ML phylogenetic tree constructed from the SNP distribution along putative promoters. Genic and spacer promoter variants, determined in Supplementary Fig. S6, are well separated. In addition, both genic and spacer promoter sequences are split into the same number (five) of clusters of 1g–5g and 1s–5s, respectively. Owing to high number of sequences, individual clusters are represented by circles with areas proportional to the number (specified in brackets) of corresponding promoters, specified in more details in Supplementary Table S10. Only bootstrap values higher than 50% are shown." path-from-xml="dsz008f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025666/dsz008f7.tif" path-from-xml="dsz008f7.tif" target="_blank">Open in new tab</a><a section="137025666" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f7.png?Expires=1576484115&amp;Signature=sh~p6OBU3kfxcHiS120sb9zx~zt1Lu2jSBrR2YIurUnVUsmniLancqvyWu8Dwia7RfOoNyn0907PXFzjRTcCnF2S-ViigGr5B79PxKgQgBqbR-l-OpWI3r6JKuEfxl4t8vgBfWulWRxUntHcJKfDQlAe6GWSCCuxo-hHjdQySJ1o~zStpKIAfSoHIH3FdOwLYD86YOd7AjFGc7SIg~rH0UfqdCoVZGNRo2LndkoTA-RG6~mPZPowk4dkjq7VzdQx14VxTFJ7YDDcDVPQqqyULuOByE9qgWR3lJkjwR9h33z1iujemGoCzQ~nzdr3eP2lvnzRu2zVlxrmdHhVAfy9tQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025666&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>ML phylogenetic tree constructed from the SNP distribution along putative promoters. Genic and spacer promoter variants, determined in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>, are well separated. In addition, both genic and spacer promoter sequences are split into the same number (five) of clusters of 1<sup>g</sup>–5<sup>g</sup> and 1<sup>s</sup>–5<sup>s</sup>, respectively. Owing to high number of sequences, individual clusters are represented by circles with areas proportional to the number (specified in brackets) of corresponding promoters, specified in more details in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>. Only bootstrap values higher than 50% are shown.</p></div>
</div>
<div content-id="dsz008-F7" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/m_dsz008f7.png?Expires=1576484115&amp;Signature=Y6Nazp8ibE0iV-xDkzJnOT88iHQCVFD3-wsSCf9lCz2i6eHUQJmdBFE1TPHOugMvBDISwuDbK4PIavMu5mUICT2aYymE~jn4GgWAgvDMh0Oag3gEnXA1~kB1axbDUgdC9pWiA26O0Oomoo0pIa3b2D3AcDkv-LhzLMfh60q2GeS--E318NGgEK6eaFzbrL4mXZ1HNi6U9Jjtj4B0t-0dhkdVye3~L1~g5Mf8t2JSJOZtu4gF-9uWP1WfHqyc30YxVXG6QHURktAzDaADtu7Tv53z1mE-xJzXtn3Ae92ofc5Ksjomnamou62CE7ILewhBzK7Q39tZlMwejeUgORhgdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ML phylogenetic tree constructed from the SNP distribution along putative promoters. Genic and spacer promoter variants, determined in Supplementary Fig. S6, are well separated. In addition, both genic and spacer promoter sequences are split into the same number (five) of clusters of 1g–5g and 1s–5s, respectively. Owing to high number of sequences, individual clusters are represented by circles with areas proportional to the number (specified in brackets) of corresponding promoters, specified in more details in Supplementary Table S10. Only bootstrap values higher than 50% are shown." path-from-xml="dsz008f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025666/dsz008f7.tif" path-from-xml="dsz008f7.tif" target="_blank">Open in new tab</a><a section="137025666" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008f7.png?Expires=1576484115&amp;Signature=sh~p6OBU3kfxcHiS120sb9zx~zt1Lu2jSBrR2YIurUnVUsmniLancqvyWu8Dwia7RfOoNyn0907PXFzjRTcCnF2S-ViigGr5B79PxKgQgBqbR-l-OpWI3r6JKuEfxl4t8vgBfWulWRxUntHcJKfDQlAe6GWSCCuxo-hHjdQySJ1o~zStpKIAfSoHIH3FdOwLYD86YOd7AjFGc7SIg~rH0UfqdCoVZGNRo2LndkoTA-RG6~mPZPowk4dkjq7VzdQx14VxTFJ7YDDcDVPQqqyULuOByE9qgWR3lJkjwR9h33z1iujemGoCzQ~nzdr3eP2lvnzRu2zVlxrmdHhVAfy9tQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025666&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz008f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>ML phylogenetic tree constructed from the SNP distribution along putative promoters. Genic and spacer promoter variants, determined in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>, are well separated. In addition, both genic and spacer promoter sequences are split into the same number (five) of clusters of 1<sup>g</sup>–5<sup>g</sup> and 1<sup>s</sup>–5<sup>s</sup>, respectively. Owing to high number of sequences, individual clusters are represented by circles with areas proportional to the number (specified in brackets) of corresponding promoters, specified in more details in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>. Only bootstrap values higher than 50% are shown.</p></div>
</div>
<p>Finally, the size ranges of RTA<sub>n</sub>T-motifs, located between the spacer promoter and R1 repeat (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) are highly specific for each corresponding spatially linked R1/R2 family (<a reveal-id="dsz008-T1" data-open="dsz008-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>
<p>Classification of the pumpkin IGSs into multiple families appears to be highly legitimate because structural variants of R1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>) and R2 (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>) repeats, defined based on the arrangement of distinct structural blocks, correlate well with (i) adjacent promoter variants (<a reveal-id="dsz008-F7" data-open="dsz008-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>) that are independently defined from the distribution of SNPs and (ii) the size of A/T-rich region RTA<sub>n</sub>T (<a reveal-id="dsz008-T1" data-open="dsz008-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>                    <h2 scrollto-destination="137025669" id="137025669" class="section-title">4. Discussion</h2>
                    <h3 scrollto-destination="137025670" id="137025670" class="section-title">4.1. The minisatellite-like organization suggests genetic instability and accelerated structural evolution of R2 (R1) regions, which might affect transcription from adjacent Pol I promoters</h3>
<p>The repeat unit size, copy number, sequence purity, and GC content of R2/R1 repeats in pumpkin IGSs are typical for so-called DNA minisatellites. The origin of the extended copy number variability at R2/R1 regions may, therefore, be explained by mechanisms that are common for satellites where the number of repeated units expands or contracts at high frequencies by mechanisms such as intra- and inter-locus recombination and/or replication slippage.<a reveal-id="dsz008-B28" data-open="dsz008-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> In pumpkin IGSs, minisatellites that are highly variable in length (Families A and C) coexist with those with sizes that are rather stable (Family D), suggesting distinct mechanisms of evolution.</p>
<p>The occurrence of tandem repeats in the vicinity of Pol I promoters appears to be a common feature of IGSs across flowering plants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). Their structures (size of monomeric unit, copy number, nucleotide sequence and GC content) vary; however, with wide ranges even between closely related species. Nevertheless, significantly higher GC content, compared with adjacent promoter-like sequences, appeared to be the most striking feature of IGS satellites and the highest GC content within plants is characteristic for <em>Cucurbita</em>. Microsatellites (the most striking example represents <em>Capsicum</em>), minisatellites (<em>Olea</em>) and satellites (<em>Hordeum</em>) have evolved upstream of TIS, whereas only minisatellites (<em>Eruca</em>) and satellites (<em>Cucurbita</em>) were found downstream of TIS. The distance between TIS and adjacent satellites varies widely, and the corresponding nucleotide sequence is extremely variable even in the short ‘core’ Pol I region adjacent to TIS.</p>
<p>The instability of satellites with mutation frequencies from 10<sup>−2</sup> to 10<sup>−5</sup> per generation is within the range of frequencies of genetic point mutations (10<sup>−8</sup> to 10<sup>−9</sup>) and epigenetic switches (10<sup>−1</sup> to 10<sup>−2</sup>).<a reveal-id="dsz008-B29" data-open="dsz008-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Therefore, these repetitive elements share both genetic and epigenetic features. As tandem repeats can reversibly expand or contract without a complete loss of information, they resemble epigenetic transitions, which do not lead to severe changes in function or completely novel features. Generally, tandem repeats are frequently located in gene promoters where their variations can act as mediators of rapid and fine-tuned phenotypic changes, perhaps through alterations in chromatin structure.<a reveal-id="dsz008-B29" data-open="dsz008-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsz008-B30" data-open="dsz008-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The activation of Pol I transcription appears to be mediated by transcription factors that interact with both the promoter and the upstream located mini(satellites) (enhancers), although their structural features vary with a wide range between maize,<a reveal-id="dsz008-B31" data-open="dsz008-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><em>Xenopus</em><a reveal-id="dsz008-B32" data-open="dsz008-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and mice<a reveal-id="dsz008-B33" data-open="dsz008-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). In pumpkin, R2 minisatellites are built of rather atypical repeat units, each composed of two kinds of asymmetrically distributed GC-rich and short microsatellite-like sequences with the potential, although considerably variable, to form Z- or G4-non-canonical DNA structures. There are direct and indirect pieces of evidence that both G4- and Z-DNA conformations are formed in vivo: (i) functionally important genomic regions, including promoters, rDNA and sites with low nucleosome occupancy are enriched in these motifs.<a reveal-id="dsz008-B34" data-open="dsz008-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsz008-B35" data-open="dsz008-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (ii) Specific proteins with affinity to pZ- or pG4- or both-conformations participate in their regulation either by promotion or destabilization.<a reveal-id="dsz008-B36" data-open="dsz008-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Namely, two abundant nucleolus-specific proteins, nucleolin and nucleophosmin, show specific affinity to G4-conformations in rDNA.<a reveal-id="dsz008-B37" data-open="dsz008-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Related proteins might recognize the satellites in pumpkin IGSs that are rich in both motifs, although their distributions and proportions vary widely. During transcription, these proteins may interact with G4 structures formed by the non-template G-rich strand to prevent renaturation of the duplex and to render the template strand available for multiple rounds of transcription. The stability of the melted region may correlate with variable numbers or densities of pG4-motifs. (iii) Hoogsteen base pairs may transiently form in canonical double-stranded DNA,<a reveal-id="dsz008-B38" data-open="dsz008-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and G4 conformations can be formed in supercoiled duplex DNA and DNA in chromatin <em>in vivo</em>.<a reveal-id="dsz008-B39" data-open="dsz008-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> As local disruption of the regular double helix DNA structure is associated with B-Z transition,<a reveal-id="dsz008-B40" data-open="dsz008-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> G4-formation may be promoted by closely linked pZ-motifs. Diversely methylated pZ-motifs may undergo a B-Z transition with varying efficiency,<a reveal-id="dsz008-B41" data-open="dsz008-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and thereby increase the epigenetically determined structural heterogeneity of distinct IGS variants.</p>
<p>The occurrence of two significantly larger FISH signals in both <em>C. moschata</em> and <em>C. pepo</em> may suggest that these loci are transcriptionally active and form dominant NORs. However, FISH analyses did not show any secondary constriction at these loci in either <em>C. moschata</em><a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> or <em>C. pepo</em> (<a reveal-id="dsz008-F1" data-open="dsz008-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). We previously showed that there is no unequivocal correlation between the size of rDNA loci and their transcription activity, and even loci with highly reduced rDNA copy numbers may be transcriptionally dominant, forming active NORs.<a reveal-id="dsz008-B42" data-open="dsz008-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>
<p>Specific structural features of the short repeat unit together with its variable copy numbers may result in decreased sequence complexity along considerably long and variable DNA stretches associated with altered GC content, bendability, intrinsic curvature and, consequently, chromatin structure.<a reveal-id="dsz008-B43" data-open="dsz008-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Altered chromatin structure established at such repeats may consequently partially spread into adjacent regulatory elements and gradually sterically attract/repulse certain factors that participate in transcription or replication.</p>                    <h3 scrollto-destination="137025676" id="137025676" class="section-title">4.2. Origin and spatial arrangement of multiple 35S rDNA variants in the pumpkin genome</h3>
<p>Generally, tandem repeats in individual IGS variants may differ: (i) in the copy number of the common repeat, as observed between species within <em>Capsicum</em>, <em>Solanum</em> and <em>Quercus</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). (ii) In the spatial arrangement of common repeats, as found between <em>Tragopogon dubius</em> and <em>Tragopogon porrifolius</em>,<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> and (iii) in the occurrence of specific repeats, as has been described for <em>Brassica</em> species.<a reveal-id="dsz008-B7" data-open="dsz008-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Owing to a few IGSs analysed in each species, interspecies heterogeneity detected among highly related species may; however, be the consequence of broad size variability within each species, as documented recently for <em>Arabidopsis</em><a reveal-id="dsz008-B14" data-open="dsz008-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and here for pumpkin. We describe the coexistence of a number of IGS variants in a single genome, which differ not merely in overall length defined by the number of tandem repeats but also in their intrinsic structure determined by the specific arrangement of distinct short sequence motifs. Such enormous rDNA variability supports the hypothesis of a hybrid/polyploid origin of <em>C. moschata.</em><a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Alternatively, multiple rDNA loci may originate from rDNA translocations in a single genome, and the structural identity of rDNA loci may be maintained as a consequence of inefficient interlocus homogenization. On the whole-genome scale, the mutual arrangement of individual rDNA variants in <em>C. moschata</em> is currently unknown. Considering the equal number (5) of rDNA structural variants (<a reveal-id="dsz008-F5" data-open="dsz008-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>) and number of chromosomal rDNA loci,<a reveal-id="dsz008-B17" data-open="dsz008-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> we can suppose that individual variants are distributed separately among multiple chromosomal loci. However, possible homeologous recombinations may lead to homogenization of rDNA at some or all loci during <em>C. moschata</em> diploidization. As a result, individual rDNA families might be partially or completely spatially intermingled. Despite invariable chromosome number in allohexaploid wheat (2<em>n</em> = 6<em>x</em> = 42), the numbers of rDNA chromosomal sites range from 2 to 16 (Plant rDNA database),<a reveal-id="dsz008-B44" data-open="dsz008-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> suggesting variable modes of rDNA evolution during diploidization of wheat cultivars. In the recently formed allotetraploid <em>T. mirus</em>, both parental IGS variants have remained preserved.<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Among more ancient <em>Nicotiana</em> allotetraploids, the extent of interlocus rDNA homogenization decreased in the direction <em>N. arentsii</em> &gt; <em>N. tabacum</em> &gt; <em>N. rustica</em><a reveal-id="dsz008-B45" data-open="dsz008-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and both parental variants remained in <em>N. rustica</em>, whereas complete homogenization of rDNA towards one parental variant occurred in <em>N. arentsii.</em> Using specific FISH probes derived from the parental IGS, both homogenization and preservation of parental rDNA variants were demonstrated in different cultivars of <em>Brassica napus.</em><a reveal-id="dsz008-B46" data-open="dsz008-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Because of the extraordinarily high GC content of all R2 and R1 variants in pumpkin, they can hardly be distinguished by a similar approach. A significantly higher number of 35S rDNA than 5S rDNA suggests segmental duplications or 35S rDNA translocations during the pumpkin genome evolution. Alternatively, 5S rDNA has partially been eliminated, as may be inferred from only one 5S rDNA locus currently present in <em>C. pepo</em>.</p>                    <h3 scrollto-destination="137025678" id="137025678" class="section-title">4.3. The global repetitive architecture of <em>C. moschata</em> IGSs is conserved among thousands of rDNA copies but differs in closely related species</h3>
<p>Despite the considerably high structural variability within both R1 and R2 minisatellites, the overall repetitive organization of pumpkin IGSs is rather conserved, invariably formed of three prominent repeats intermingled with two highly homologous copies of considerably long non-repetitive sequences, suggesting selection constraints to maintain such complexly structured regions. Although rarely occurring within flowering plants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>), similar duplications of promoter-like sequences separated by tandem repeats were previously described in distantly related species <em>Tragopogon</em> (Asteraceae),<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><em>Arabidopsis</em> (Brassicaceae),<a reveal-id="dsz008-B14" data-open="dsz008-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><em>Quercus</em> (Fagaceae),<a reveal-id="dsz008-B47" data-open="dsz008-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><em>Daucus</em> (Apiaceae)<a reveal-id="dsz008-B48" data-open="dsz008-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and <em>Xanthisma</em> (Asteraceae), suggesting recurrent duplication events during dicot evolution. Comprehensive analysis of the structural heterogeneity of the 35S rDNA IGS has, however, been performed only for <em>A. thaliana</em> using PacBio sequencing of PCR amplicons.<a reveal-id="dsz008-B14" data-open="dsz008-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> As in pumpkin, high size heterogeneity is due to the variable number of short tandemly arrayed monomeric units, which, however, are significantly more homogeneous in nucleotide sequences compared with repeated units forming five families in pumpkin. In contrast to pumpkin, they are distributed among variable numbers (1–3) of tandem blocks separated by variable numbers of promoter-like duplicates (0–2). Intraspecific variability in the number of promoter-like sequences was also detected in <em>Quercus robur</em>. The number of duplicated promoter-like sequences significantly differs between the related species <em>T. porrifolius</em> (&gt;3 copies) and <em>T. dubius</em> (2 copies). The length of duplicated regions significantly varies between species, ranging from less than 100 bp (probably representing only a core promoter) to several hundred bp.</p>
<p>We presume that together with the non-repetitive sequence surrounding TIS, adjacent tandem repeats were also duplicated, as a whole or in part, during the evolution of the IGS. Alternatively, a unique promoter-like sequence may have been duplicated and translocated inside the ancestral unique tandem repeat. Perhaps due to the action of selective constraints, duplicated unique sequences evolved in a concerted manner to maintain a high degree of homology, both within and between them, in each species. In contrast, duplicated repeated regions could subsequently evolve rather independently either due to the absence of selective constraints or, more likely, their repetitive characteristics, which may have significantly accelerated their diversification. In pumpkin, both modern repetitive regions R1 and R2 share only a few signs: the extremely high GC content and the occurrence of short sequence motifs, including pZ and pG4, which may indicate their common origin. Duplicated satellites in each <em>Q. robur</em>, <em>T. porrifolius</em> and <em>A. thaliana</em>; however, share highly related nucleotide sequences, suggesting a more recent duplication or another mechanism of evolution, such as the impact of selection constraints on each satellite.</p>
<p>The R3 repeat is unrelated to R1 and R2 repeats, made up of significantly longer units with the expected GC content, which seems to be common among dicots where repeats located upstream of TIS are, on average, composed of higher copy numbers of shorter monomers with higher GC content than repeats localized downstream (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). In contrast, monocots harbour satellites composed of substantially long repeated units located both upstream and downstream of genic TIS. When compared with R1 and R2 minisatellites, the R3 satellite in <em>C. moschata</em> is significantly more homogeneous in size, composed almost exclusively of three monomeric units, reminiscent of results previously reported for <em>A. thaliana,</em><a reveal-id="dsz008-B14" data-open="dsz008-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and suggesting rather common features among plants. Despite high size homogeneity within <em>C. moschata</em>, the R3 repeat was significantly amplified in <em>C. maxima</em>, currently forming a tandem of nine units.<a reveal-id="dsz008-B18" data-open="dsz008-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Such interspecific differences might be explained on the basis of the so-called ‘library hypothesis’.<a reveal-id="dsz008-B49" data-open="dsz008-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Accordingly, both size variants of R3 probably evolved in a common ancestor, although perhaps at low copy numbers, followed by subsequent spreading of a shorter variant in <em>C. moschata</em> and <em>C. pepo,</em><a reveal-id="dsz008-B18" data-open="dsz008-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> while the longer variant was amplified in <em>C. maxima</em>. In contrast, the R1 repeat is almost completely deleted in <em>C. maxima</em>, whereas it is preserved in <em>C. pepo</em>. Nevertheless, both highly conserved sequences surrounding the R1 region in <em>C. pepo</em> and <em>C. moschata</em> are completely retained in <em>C. maxima</em>, surrounding the short non-repetitive region with signs of R1 repeats (CG-rich, pZ- and pG4-motifs) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576484115&amp;Signature=0Ub6gwdCM~irjCUDXGZLIivt5ysbb6C7xE0VI9MCdF9BdT22r0pxfhXprWWokk8p3hCwhe9VHmDdjRzyTDYXWHFl5MdtsgzC6itj2kuMGqvyIvSngHT-3OMtPddDIpJMN~Gec43QMDW2G4ZIBEx2wpb9ciiFeJI2zGeSzEWgDCu-qbX~U0j6bFxEKj7vtIKEcXPmJUJ5VTlhzS6rU5awg6kbbLPmNwHW~d4sMoWqghwUqPIR6Q-9B5iEiqF34ipnjnO6pUR2HhcK8vVjKk5lk4~mta9qTxEFh9u8YMmqWzFQ~20HOSa5kokgCI-dGV9IhJHg18lfVAVi2TF9e~WBsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The pumpkin R1 repeat may, therefore, originated by amplification of the short unique sequence present in the common ancestor rather than by the above mentioned duplication event. Similarly, <em>T. dubius</em> harbours relatively long repeats located downstream of TIS complemented with only one repeat located upstream of TIS, whereas the <em>T. porrifolius</em> IGS is composed of short downstream repeats complemented with multiple satellites located upstream.<a reveal-id="dsz008-B2" data-open="dsz008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a></p>                    <h2 scrollto-destination="137025682" id="137025682" class="section-title">5. Conclusions</h2>
<p>We showed that in the genome of <em>C. moschata</em>, five rDNA structural variants are distributed across five chromosomal loci. Structural variability between rDNA variants is determined by the variable repetitive arrangement of a few kinds of short GC-rich sequence blocks with the potential to form non-canonical DNA structures and together to form various minisatellites in the IGS. Because pZ- and pG4-conformations, as well as tandemly arranged repeats in IGSs, are known to participate in transcriptional regulation, we hypothesize that such higher order structural variability of IGS repeats may participate in a graduated affinity of TIFs for the adjacent Pol I promoters, potentially promoting the fine orchestration of rDNA transcription regulation in response to cell protein synthesis demand. We suppose that individual rDNA variants evolved in two or more parental genomes followed by hybridization or, alternatively, within a single genome at separated chromosomal rDNA loci as a consequence of inefficient inter-locus homogenization.</p>                    <h2 scrollto-destination="137025684" id="137025684" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>We thank Radka Vozarova for FISH analyses and helpful comments on the article.</p>                    <h3 scrollto-destination="137025686" id="137025686" class="section-title">Accession numbers</h3>
<p>BioProject ID PRJNA400686 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/bioproject/400686" target="_blank">http://www.ncbi.nlm.nih.gov/bioproject/400686</a>); GenBank MG744571-MG744575.</p>                    <h3 scrollto-destination="137025688" id="137025688" class="section-title">Funding</h3>
<p>The research was funded by the Czech Science Foundation (P501/17/11642S).</p>                    <h3 scrollto-destination="137025690" id="137025690" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="137025692" id="137025692" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz008-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hemleben</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Zentgraf</div> <div class="given-names">U.</div>
</div>
</div> <div class="year">1994</div>, 
<div class="article-title">Structural organization and regulation of transcription by RNA polymerase I of plant nuclear ribosomal RNA genes</div>, <div class="source">Results Probl. Cell Differ</div> ., <div class="volume">20</div>, <div class="fpage">3</div>–<div class="lpage">24</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Structural%20organization%20and%20regulation%20of%20transcription%20by%20RNA%20polymerase%20I%20of%20plant%20nuclear%20ribosomal%20RNA%20genes&amp;author=V.%20Hemleben&amp;author=U.%20Zentgraf&amp;publication_year=1994&amp;journal=Results%20Probl.%20Cell%20Differ&amp;volume=20&amp;pages=3-24" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/978-3-540-48037-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-540-48037-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2F978-3-540-48037-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8036320" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Structural%20organization%20and%20regulation%20of%20transcription%20by%20RNA%20polymerase%20I%20of%20plant%20nuclear%20ribosomal%20RNA%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Matyasek</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Dobesova</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Huska</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Interpopulation hybridization generates meiotically stable rDNA epigenetic variants in allotetraploid <em>Tragopogon mirus</em>
</div>, <div class="source">Plant J</div> ., <div class="volume">85</div>, <div class="fpage">362</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Interpopulation%20hybridization%20generates%20meiotically%20stable%20rDNA%20epigenetic%20variants%20in%20allotetraploid%20Tragopogon%20mirus&amp;author=R.%20Matyasek&amp;author=E.%20Dobesova&amp;author=D.%20Huska&amp;publication_year=2016&amp;journal=Plant%20J&amp;volume=85&amp;pages=362-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.2016.85.issue-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2016.85.issue-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.2016.85.issue-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26711705" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Interpopulation%20hybridization%20generates%20meiotically%20stable%20rDNA%20epigenetic%20variants%20in%20allotetraploid%20Tragopogon%20mirus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Flavell</div> <div class="given-names">R.B.</div>
</div>, <div class="name string-name">
<div class="surname">Odell</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Thompson</div> <div class="given-names">W.F.</div>
</div>
</div>, et al. <div class="year">1986</div>, 
<div class="article-title">The differential expression of ribosomal-RNA genes</div>, <div class="source">Philos. Trans. Roy. Soc. B</div> ., 314, 385-397. doi:10.1098/rstb.1986.0060. <div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20differential%20expression%20of%20ribosomal-RNA%20genes&amp;aulast=Flavell&amp;date=1986" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20differential%20expression%20of%20ribosomal-RNA%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Martini</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">O’Dell</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Flavell</div> <div class="given-names">R.B.</div>
</div>
</div> <div class="year">1982</div>, 
<div class="article-title">Partial inactivation of wheat nucleolus organizers by the nucleolus organizer chromosomes from <em>Aegilops</em>-<em>Umbellulata</em>
</div>, <div class="source">Chromosoma</div> , <div class="volume">84</div>, <div class="fpage">687</div>–<div class="lpage">700</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Partial%20inactivation%20of%20wheat%20nucleolus%20organizers%20by%20the%20nucleolus%20organizer%20chromosomes%20from%20Aegilops-Umbellulata&amp;author=G.%20Martini&amp;author=M.%20O%E2%80%99Dell&amp;author=R.B.%20Flavell&amp;publication_year=1982&amp;journal=Chromosoma&amp;volume=84&amp;pages=687-700" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF00286334" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00286334" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00286334"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Partial%20inactivation%20of%20wheat%20nucleolus%20organizers%20by%20the%20nucleolus%20organizer%20chromosomes%20from%20Aegilops-Umbellulata&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Silva</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Pereira</div> <div class="given-names">H.S.</div>
</div>, <div class="name string-name">
<div class="surname">Bento</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Interplay of ribosomal DNA loci in nucleolar dominance: dominant NORs are up-regulated by chromatin dynamics in the wheat-rye system</div>. <div class="source">Plos One</div> , 3, e3824. doi:10.1371/journal.pone.0003824.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Interplay%20of%20ribosomal%20DNA%20loci%20in%20nucleolar%20dominance%3A%20dominant%20NORs%20are%20up-regulated%20by%20chromatin%20dynamics%20in%20the%20wheat-rye%20system&amp;aulast=Silva&amp;date=2008" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Interplay%20of%20ribosomal%20DNA%20loci%20in%20nucleolar%20dominance%3A%20dominant%20NORs%20are%20up-regulated%20by%20chromatin%20dynamics%20in%20the%20wheat-rye%20system&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Komarova</div> <div class="given-names">N.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Grabe</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Huigen</div> <div class="given-names">D.J.</div>
</div>
</div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Organization, differential expression and methylation of rDNA in artificial <em>Solanum</em> allopolyploids</div>, <div class="source">Plant Mol. Biol</div> ., <div class="volume">56</div>, <div class="fpage">439</div>–<div class="lpage">63</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Organization%2C%20differential%20expression%20and%20methylation%20of%20rDNA%20in%20artificial%20Solanum%20allopolyploids&amp;author=N.Y.%20Komarova&amp;author=T.%20Grabe&amp;author=D.J.%20Huigen&amp;publication_year=2004&amp;journal=Plant%20Mol.%20Biol&amp;volume=56&amp;pages=439-63" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s11103-004-4678-x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-004-4678-x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs11103-004-4678-x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15604755" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Organization%2C%20differential%20expression%20and%20methylation%20of%20rDNA%20in%20artificial%20Solanum%20allopolyploids&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">Z.J.</div>
</div>, <div class="name string-name">
<div class="surname">Pikaard</div> <div class="given-names">C.S.</div>
</div>
</div> <div class="year">1997</div>, 
<div class="article-title">Transcriptional analysis of nucleolar dominance in polyploid plants: biased expression/silencing of progenitor rRNA genes is developmentally regulated in <em>Brassica</em>
</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">94</div>, <div class="fpage">3442</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20analysis%20of%20nucleolar%20dominance%20in%20polyploid%20plants%3A%20biased%20expression%2Fsilencing%20of%20progenitor%20rRNA%20genes%20is%20developmentally%20regulated%20in%20Brassica&amp;author=Z.J.%20Chen&amp;author=C.S.%20Pikaard&amp;publication_year=1997&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=3442-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.94.7.3442" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.7.3442" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.94.7.3442"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcriptional%20analysis%20of%20nucleolar%20dominance%20in%20polyploid%20plants%3A%20biased%20expression%2Fsilencing%20of%20progenitor%20rRNA%20genes%20is%20developmentally%20regulated%20in%20Brassica&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wanzenbock</div> <div class="given-names">E.M.</div>
</div>, <div class="name string-name">
<div class="surname">Schofer</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Schweizer</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Bachmair</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">1997</div>, 
<div class="article-title">Ribosomal transcription units integrated via T-DNA transformation associate with the nucleolus and do not require upstream repeat sequences for activity in <em>Arabidopsis thaliana</em>
</div>, <div class="source">Plant J</div> ., <div class="volume">11</div>, <div class="fpage">1007</div>–<div class="lpage">16</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ribosomal%20transcription%20units%20integrated%20via%20T-DNA%20transformation%20associate%20with%20the%20nucleolus%20and%20do%20not%20require%20upstream%20repeat%20sequences%20for%20activity%20in%20Arabidopsis%20thaliana&amp;author=E.M.%20Wanzenbock&amp;author=C.%20Schofer&amp;author=D.%20Schweizer&amp;author=A.%20Bachmair&amp;publication_year=1997&amp;journal=Plant%20J&amp;volume=11&amp;pages=1007-16" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1365-313X.1997.11051007.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1997.11051007.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1365-313X.1997.11051007.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9193072" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ribosomal%20transcription%20units%20integrated%20via%20T-DNA%20transformation%20associate%20with%20the%20nucleolus%20and%20do%20not%20require%20upstream%20repeat%20sequences%20for%20activity%20in%20Arabidopsis%20thaliana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Doelling</div> <div class="given-names">J.H.</div>
</div>, <div class="name string-name">
<div class="surname">Gaudino</div> <div class="given-names">R.J.</div>
</div>, <div class="name string-name">
<div class="surname">Pikaard</div> <div class="given-names">C.S.</div>
</div>
</div> <div class="year">1993</div>, 
<div class="article-title">Functional-analysis of <em>Arabidopsis thaliana</em> ribosomal-RNA gene and spacer promoters in-vivo and by transient expression</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">90</div>, <div class="fpage">7528</div>–<div class="lpage">32</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Functional-analysis%20of%20Arabidopsis%20thaliana%20ribosomal-RNA%20gene%20and%20spacer%20promoters%20in-vivo%20and%20by%20transient%20expression&amp;author=J.H.%20Doelling&amp;author=R.J.%20Gaudino&amp;author=C.S.%20Pikaard&amp;publication_year=1993&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=7528-32" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.90.16.7528" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.16.7528" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.90.16.7528"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Functional-analysis%20of%20Arabidopsis%20thaliana%20ribosomal-RNA%20gene%20and%20spacer%20promoters%20in-vivo%20and%20by%20transient%20expression&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zentgraf</div> <div class="given-names">U.</div>
</div>, <div class="name string-name">
<div class="surname">Hemleben</div> <div class="given-names">V.</div>
</div>
</div> <div class="year">1992</div>, 
<div class="article-title">Complex-formation of nuclear proteins with the RNA polymerase-I promoter and repeated elements in the external transcribed spacer of <em>Cucumis sativus</em> ribosomal DNA</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">20</div>, <div class="fpage">3685</div>–<div class="lpage">91</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Complex-formation%20of%20nuclear%20proteins%20with%20the%20RNA%20polymerase-I%20promoter%20and%20repeated%20elements%20in%20the%20external%20transcribed%20spacer%20of%20Cucumis%20sativus%20ribosomal%20DNA&amp;author=U.%20Zentgraf&amp;author=V.%20Hemleben&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res&amp;volume=20&amp;pages=3685-91" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/20.14.3685" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.14.3685" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F20.14.3685"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1641334" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Complex-formation%20of%20nuclear%20proteins%20with%20the%20RNA%20polymerase-I%20promoter%20and%20repeated%20elements%20in%20the%20external%20transcribed%20spacer%20of%20Cucumis%20sativus%20ribosomal%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Komarova</div> <div class="given-names">N.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Grimm</div> <div class="given-names">G.W.</div>
</div>, <div class="name string-name">
<div class="surname">Hemleben</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Volkov</div> <div class="given-names">R.A.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Molecular evolution of 35S rDNA and taxonomic status of Lycopersicon within <em>Solanum</em> sect. Petota</div>, <div class="source">Plant Syst. Evol</div> ., <div class="volume">276</div>, <div class="fpage">59</div>–<div class="lpage">71</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%2035S%20rDNA%20and%20taxonomic%20status%20of%20Lycopersicon%20within%20Solanum%20sect.%20Petota&amp;author=N.Y.%20Komarova&amp;author=G.W.%20Grimm&amp;author=V.%20Hemleben&amp;author=R.A.%20Volkov&amp;publication_year=2008&amp;journal=Plant%20Syst.%20Evol&amp;volume=276&amp;pages=59-71" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00606-008-0091-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00606-008-0091-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00606-008-0091-2"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20evolution%20of%2035S%20rDNA%20and%20taxonomic%20status%20of%20Lycopersicon%20within%20Solanum%20sect.%20Petota&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schiebel</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">von Waldburg</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Gerstner</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Hemleben</div> <div class="given-names">V.</div>
</div>
</div> <div class="year">1989</div>, 
<div class="article-title">Termination of transcription of ribosomal-RNA genes of mung bean occurs within a 175-bp repetitive element of the spacer region</div>, <div class="source">Mol. Gen. Genet</div> ., <div class="volume">218</div>, <div class="fpage">302</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Termination%20of%20transcription%20of%20ribosomal-RNA%20genes%20of%20mung%20bean%20occurs%20within%20a%20175-bp%20repetitive%20element%20of%20the%20spacer%20region&amp;author=K.%20Schiebel&amp;author=G.%20von%20Waldburg&amp;author=J.%20Gerstner&amp;author=V.%20Hemleben&amp;publication_year=1989&amp;journal=Mol.%20Gen.%20Genet&amp;volume=218&amp;pages=302-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF00331282" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00331282" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00331282"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2779517" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Termination%20of%20transcription%20of%20ribosomal-RNA%20genes%20of%20mung%20bean%20occurs%20within%20a%20175-bp%20repetitive%20element%20of%20the%20spacer%20region&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Eickbush</div> <div class="given-names">T.H.</div>
</div>, <div class="name string-name">
<div class="surname">Eickbush</div> <div class="given-names">D.G.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Finely orchestrated movements: evolution of the ribosomal RNA genes</div>, <div class="source">Genetics</div> , <div class="volume">175</div>, <div class="fpage">477</div>–<div class="lpage">85</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Finely%20orchestrated%20movements%3A%20evolution%20of%20the%20ribosomal%20RNA%20genes&amp;author=T.H.%20Eickbush&amp;author=D.G.%20Eickbush&amp;publication_year=2007&amp;journal=Genetics&amp;volume=175&amp;pages=477-85" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.107.071399" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.071399" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.107.071399"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17322354" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Finely%20orchestrated%20movements%3A%20evolution%20of%20the%20ribosomal%20RNA%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Havlová</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Dvořáčková</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Peiro</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Variation of 45S rDNA intergenic spacers in <em>Arabidopsis thaliana</em>
</div>, <div class="source">Plant Mol. Biol</div> ., <div class="volume">92</div>, <div class="fpage">457</div>–<div class="lpage">71</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Variation%20of%2045S%20rDNA%20intergenic%20spacers%20in%20Arabidopsis%20thaliana&amp;author=K.%20Havlov%C3%A1&amp;author=M.%20Dvo%C5%99%C3%A1%C4%8Dkov%C3%A1&amp;author=R.%20Peiro&amp;publication_year=2016&amp;journal=Plant%20Mol.%20Biol&amp;volume=92&amp;pages=457-71" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s11103-016-0524-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-016-0524-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs11103-016-0524-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27531496" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Variation%20of%2045S%20rDNA%20intergenic%20spacers%20in%20Arabidopsis%20thaliana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grummt</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Langst</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Epigenetic control of RNA polymerase I transcription in mammalian cells,</div> <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1829</div>, <div class="fpage">393</div>–<div class="lpage">404</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20control%20of%20RNA%20polymerase%20I%20transcription%20in%20mammalian%20cells%2C&amp;author=I.%20Grummt&amp;author=G.%20Langst&amp;publication_year=2013&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1829&amp;pages=393-404" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.bbagrm.2012.10.004" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagrm.2012.10.004" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.bbagrm.2012.10.004"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23063748" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Epigenetic%20control%20of%20RNA%20polymerase%20I%20transcription%20in%20mammalian%20cells%2C&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Durut</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Abou-Ellail</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Pontvianne</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">A duplicated NUCLEOLIN gene with antagonistic activity is required for chromatin organization of silent 45S rDNA in <em>Arabidopsis</em>
</div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">1330</div>–<div class="lpage">44</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20duplicated%20NUCLEOLIN%20gene%20with%20antagonistic%20activity%20is%20required%20for%20chromatin%20organization%20of%20silent%2045S%20rDNA%20in%20Arabidopsis&amp;author=N.%20Durut&amp;author=M.%20Abou-Ellail&amp;author=F.%20Pontvianne&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=1330-44" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.114.123893" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.123893" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.114.123893"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24668745" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20duplicated%20NUCLEOLIN%20gene%20with%20antagonistic%20activity%20is%20required%20for%20chromatin%20organization%20of%20silent%2045S%20rDNA%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Waminal</div> <div class="given-names">N.E.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">N.S.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">H.H.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Dual-color FISH karyotype analyses using rDNAs in three Cucurbitaceae species</div>, <div class="source">Genes Genom</div> ., <div class="volume">33</div>, <div class="fpage">521</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dual-color%20FISH%20karyotype%20analyses%20using%20rDNAs%20in%20three%20Cucurbitaceae%20species&amp;author=N.E.%20Waminal&amp;author=N.S.%20Kim&amp;author=H.H.%20Kim&amp;publication_year=2011&amp;journal=Genes%20Genom&amp;volume=33&amp;pages=521-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s13258-011-0046-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13258-011-0046-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs13258-011-0046-9"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dual-color%20FISH%20karyotype%20analyses%20using%20rDNAs%20in%20three%20Cucurbitaceae%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">King</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Torres</div> <div class="given-names">R.A.</div>
</div>, <div class="name string-name">
<div class="surname">Zentgraf</div> <div class="given-names">U.</div>
</div>, <div class="name string-name">
<div class="surname">Hemleben</div> <div class="given-names">V.</div>
</div>
</div> <div class="year">1993</div>, 
<div class="article-title">Molecular evolution of the intergenic spacer in the nuclear ribosomal-RNA genes of Cucurbitaceae</div>, <div class="source">J. Mol. Evol</div> ., <div class="volume">36</div>, <div class="fpage">144</div>–<div class="lpage">52</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%20the%20intergenic%20spacer%20in%20the%20nuclear%20ribosomal-RNA%20genes%20of%20Cucurbitaceae&amp;author=K.%20King&amp;author=R.A.%20Torres&amp;author=U.%20Zentgraf&amp;author=V.%20Hemleben&amp;publication_year=1993&amp;journal=J.%20Mol.%20Evol&amp;volume=36&amp;pages=144-52" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF00166250" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00166250" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00166250"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8433383" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20evolution%20of%20the%20intergenic%20spacer%20in%20the%20nuclear%20ribosomal-RNA%20genes%20of%20Cucurbitaceae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Siegel</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Kolacz</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">1983</div>, 
<div class="article-title">Heterogeneity of pumpkin ribosomal DNA</div>, <div class="source">Plant Physiol</div> ., <div class="volume">72</div>, <div class="fpage">166</div>–<div class="lpage">71</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Heterogeneity%20of%20pumpkin%20ribosomal%20DNA&amp;author=A.%20Siegel&amp;author=K.%20Kolacz&amp;publication_year=1983&amp;journal=Plant%20Physiol&amp;volume=72&amp;pages=166-71" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.72.1.166" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.72.1.166" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.72.1.166"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16662953" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Heterogeneity%20of%20pumpkin%20ribosomal%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Matyasek</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Fulnecek</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Leitch</div> <div class="given-names">A.R.</div>
</div>, <div class="name string-name">
<div class="surname">Kovarik</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Analysis of two abundant, highly related satellites in the allotetraploid <em>Nicotiana arentsii</em> using double-strand conformation polymorphism analysis and sequencing</div>, <div class="source">New Phytol</div> ., <div class="volume">192</div>, <div class="fpage">747</div>–<div class="lpage">59</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20two%20abundant%2C%20highly%20related%20satellites%20in%20the%20allotetraploid%20Nicotiana%20arentsii%20using%20double-strand%20conformation%20polymorphism%20analysis%20and%20sequencing&amp;author=R.%20Matyasek&amp;author=J.%20Fulnecek&amp;author=A.R.%20Leitch&amp;author=A.%20Kovarik&amp;publication_year=2011&amp;journal=New%20Phytol&amp;volume=192&amp;pages=747-59" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.2011.192.issue-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2011.192.issue-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.2011.192.issue-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21777247" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analysis%20of%20two%20abundant%2C%20highly%20related%20satellites%20in%20the%20allotetraploid%20Nicotiana%20arentsii%20using%20double-strand%20conformation%20polymorphism%20analysis%20and%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lim</div> <div class="given-names">K.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Kovarik</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Matyasek</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Bezdek</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Lichtenstein</div> <div class="given-names">C.P.</div>
</div>, <div class="name string-name">
<div class="surname">Leitch</div> <div class="given-names">A.R.</div>
</div>
</div> <div class="year">2000</div>, 
<div class="article-title">Gene conversion of ribosomal DNA in <em>Nicotiana tabacum</em> is associated with undermethylated, decondensed and probably active gene units</div>, <div class="source">Chromosoma</div> , <div class="volume">109</div>, <div class="fpage">161</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20conversion%20of%20ribosomal%20DNA%20in%20Nicotiana%20tabacum%20is%20associated%20with%20undermethylated%2C%20decondensed%20and%20probably%20active%20gene%20units&amp;author=K.Y.%20Lim&amp;author=A.%20Kovarik&amp;author=R.%20Matyasek&amp;author=M.%20Bezdek&amp;author=C.P.%20Lichtenstein&amp;author=A.R.%20Leitch&amp;publication_year=2000&amp;journal=Chromosoma&amp;volume=109&amp;pages=161-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s004120000074" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004120000074" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs004120000074"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10929194" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20conversion%20of%20ribosomal%20DNA%20in%20Nicotiana%20tabacum%20is%20associated%20with%20undermethylated%2C%20decondensed%20and%20probably%20active%20gene%20units&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Benson</div> <div class="given-names">G.</div>
</div></div> <div class="year">1999</div>, 
<div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">573</div>–<div class="lpage">80</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F27.2.573"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kikin</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">D’Antonio</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Bagga</div> <div class="given-names">P.S.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">QGRS Mapper: a web-based server for predicting G-quadruplexes in nucleotide sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W676</div>–<div class="lpage">82</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=QGRS%20Mapper%3A%20a%20web-based%20server%20for%20predicting%20G-quadruplexes%20in%20nucleotide%20sequences&amp;author=O.%20Kikin&amp;author=L.%20D%E2%80%99Antonio&amp;author=P.S.%20Bagga&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W676-82" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkl253" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl253" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl253"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845096" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:QGRS%20Mapper%3A%20a%20web-based%20server%20for%20predicting%20G-quadruplexes%20in%20nucleotide%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shin</div> <div class="given-names">S.I.</div>
</div>, <div class="name string-name">
<div class="surname">Ham</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Z-DNA-forming sites identified by ChIP-Seq are associated with actively transcribed regions in the human genome</div>, <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">477</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Z-DNA-forming%20sites%20identified%20by%20ChIP-Seq%20are%20associated%20with%20actively%20transcribed%20regions%20in%20the%20human%20genome&amp;author=S.I.%20Shin&amp;author=S.%20Ham&amp;author=J.%20Park&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=477-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsw031" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw031" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsw031"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Z-DNA-forming%20sites%20identified%20by%20ChIP-Seq%20are%20associated%20with%20actively%20transcribed%20regions%20in%20the%20human%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tamura</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Nei</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">1993</div>, 
<div class="article-title">Estimation of the number of nucleotide substitutions in the control region of mitochondrial-DNA in humans and chimpanzees</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">10</div>, <div class="fpage">512</div>–<div class="lpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20the%20number%20of%20nucleotide%20substitutions%20in%20the%20control%20region%20of%20mitochondrial-DNA%20in%20humans%20and%20chimpanzees&amp;author=K.%20Tamura&amp;author=M.%20Nei&amp;publication_year=1993&amp;journal=Mol.%20Biol.%20Evol&amp;volume=10&amp;pages=512-26" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8336541" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Estimation%20of%20the%20number%20of%20nucleotide%20substitutions%20in%20the%20control%20region%20of%20mitochondrial-DNA%20in%20humans%20and%20chimpanzees&amp;aulast=Tamura&amp;date=1993&amp;spage=512&amp;epage=26&amp;volume=10" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Estimation%20of%20the%20number%20of%20nucleotide%20substitutions%20in%20the%20control%20region%20of%20mitochondrial-DNA%20in%20humans%20and%20chimpanzees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jenkins</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Hasterok</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">BAC ‘landing’ on chromosomes of <em>Brachypodium distachyon</em> for comparative genome alignment</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">2</div>, <div class="fpage">88</div>–<div class="lpage">98</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BAC%20%E2%80%98landing%E2%80%99%20on%20chromosomes%20of%20Brachypodium%20distachyon%20for%20comparative%20genome%20alignment&amp;author=G.%20Jenkins&amp;author=R.%20Hasterok&amp;publication_year=2007&amp;journal=Nat.%20Protoc&amp;volume=2&amp;pages=88-98" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nprot.2006.490" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2006.490" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnprot.2006.490"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17401342" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BAC%20%E2%80%98landing%E2%80%99%20on%20chromosomes%20of%20Brachypodium%20distachyon%20for%20comparative%20genome%20alignment&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lim</div> <div class="given-names">K.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Skalicka</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Koukalova</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Dynamic changes in the distribution of a satellite homologous to intergenic 26-18S rDNA spacer in the evolution of Nicotiana</div>, <div class="source">Genetics</div> , <div class="volume">166</div>, <div class="fpage">1935</div>–<div class="lpage">46</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dynamic%20changes%20in%20the%20distribution%20of%20a%20satellite%20homologous%20to%20intergenic%2026-18S%20rDNA%20spacer%20in%20the%20evolution%20of%20Nicotiana&amp;author=K.Y.%20Lim&amp;author=K.%20Skalicka&amp;author=B.%20Koukalova&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=1935-46" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.166.4.1935" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.4.1935" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.166.4.1935"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15126410" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dynamic%20changes%20in%20the%20distribution%20of%20a%20satellite%20homologous%20to%20intergenic%2026-18S%20rDNA%20spacer%20in%20the%20evolution%20of%20Nicotiana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Richard</div> <div class="given-names">G.F.</div>
</div>, <div class="name string-name">
<div class="surname">Paques</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2000</div>, 
<div class="article-title">Mini- and microsatellite expansions: the recombination connection</div>, <div class="source">EMBO Rep</div> ., <div class="volume">1</div>, <div class="fpage">122</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mini-%20and%20microsatellite%20expansions%3A%20the%20recombination%20connection&amp;author=G.F.%20Richard&amp;author=F.%20Paques&amp;publication_year=2000&amp;journal=EMBO%20Rep&amp;volume=1&amp;pages=122-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/embo-reports/kvd031" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fembo-reports%2Fkvd031" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fembo-reports%2Fkvd031"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11265750" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mini-%20and%20microsatellite%20expansions%3A%20the%20recombination%20connection&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gemayel</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Vinces</div> <div class="given-names">M.D.</div>
</div>, <div class="name string-name">
<div class="surname">Legendre</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Verstrepen</div> <div class="given-names">K.J.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Variable tandem repeats accelerate evolution of coding and regulatory sequences</div>, <div class="source">Annu. Rev. Genet</div> ., <div class="volume">44</div>, <div class="fpage">445</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Variable%20tandem%20repeats%20accelerate%20evolution%20of%20coding%20and%20regulatory%20sequences&amp;author=R.%20Gemayel&amp;author=M.D.%20Vinces&amp;author=M.%20Legendre&amp;author=K.J.%20Verstrepen&amp;publication_year=2010&amp;journal=Annu.%20Rev.%20Genet&amp;volume=44&amp;pages=445-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev-genet-072610-155046" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-072610-155046" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev-genet-072610-155046"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20809801" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Variable%20tandem%20repeats%20accelerate%20evolution%20of%20coding%20and%20regulatory%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Espley</div> <div class="given-names">R.V.</div>
</div>, <div class="name string-name">
<div class="surname">Brendolise</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Chagne</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Multiple repeats of a promoter segment causes transcription factor autoregulation in red apples</div>, <div class="source">Plant Cell</div> , <div class="volume">21</div>, <div class="fpage">168</div>–<div class="lpage">83</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Multiple%20repeats%20of%20a%20promoter%20segment%20causes%20transcription%20factor%20autoregulation%20in%20red%20apples&amp;author=R.V.%20Espley&amp;author=C.%20Brendolise&amp;author=D.%20Chagne&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=168-83" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.108.059329" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.059329" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.108.059329"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151225" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Multiple%20repeats%20of%20a%20promoter%20segment%20causes%20transcription%20factor%20autoregulation%20in%20red%20apples&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schmitz</div> <div class="given-names">M.L.</div>
</div>, <div class="name string-name">
<div class="surname">Maier</div> <div class="given-names">U.G.</div>
</div>, <div class="name string-name">
<div class="surname">Brown</div> <div class="given-names">J.W.S.</div>
</div>, <div class="name string-name">
<div class="surname">Feix</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">1989</div>, 
<div class="article-title">Specific binding of nuclear proteins to the promoter region of a maize nuclear ribosomal-RNA gene unit</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">264</div>, <div class="fpage">1467</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Specific%20binding%20of%20nuclear%20proteins%20to%20the%20promoter%20region%20of%20a%20maize%20nuclear%20ribosomal-RNA%20gene%20unit&amp;author=M.L.%20Schmitz&amp;author=U.G.%20Maier&amp;author=J.W.S.%20Brown&amp;author=G.%20Feix&amp;publication_year=1989&amp;journal=J.%20Biol.%20Chem&amp;volume=264&amp;pages=1467-72" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2912967" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Specific%20binding%20of%20nuclear%20proteins%20to%20the%20promoter%20region%20of%20a%20maize%20nuclear%20ribosomal-RNA%20gene%20unit&amp;aulast=Schmitz&amp;date=1989&amp;spage=1467&amp;epage=72&amp;volume=264" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Specific%20binding%20of%20nuclear%20proteins%20to%20the%20promoter%20region%20of%20a%20maize%20nuclear%20ribosomal-RNA%20gene%20unit&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Caudy</div> <div class="given-names">A.A.</div>
</div>, <div class="name string-name">
<div class="surname">Pikaard</div> <div class="given-names">C.S.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">
<em>Xenopus</em> ribosomal RNA gene intergenic spacer elements conferring transcriptional enhancement and nucleolar dominance-like competition in oocytes</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">277</div>, <div class="fpage">31577</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Xenopus%20ribosomal%20RNA%20gene%20intergenic%20spacer%20elements%20conferring%20transcriptional%20enhancement%20and%20nucleolar%20dominance-like%20competition%20in%20oocytes&amp;author=A.A.%20Caudy&amp;author=C.S.%20Pikaard&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem&amp;volume=277&amp;pages=31577-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1074/jbc.M202737200" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M202737200" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1074%2Fjbc.M202737200"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12080048" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Xenopus%20ribosomal%20RNA%20gene%20intergenic%20spacer%20elements%20conferring%20transcriptional%20enhancement%20and%20nucleolar%20dominance-like%20competition%20in%20oocytes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kuhn</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Deppert</div> <div class="given-names">U.</div>
</div>, <div class="name string-name">
<div class="surname">Grummt</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">1990</div>, 
<div class="article-title">A 140-base pair repetitive sequence element in the mouse ribosomal-RNA gene spacer enhances transcription by RNA polymerase-I in a cell-free system</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">87</div>, <div class="fpage">7527</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20140-base%20pair%20repetitive%20sequence%20element%20in%20the%20mouse%20ribosomal-RNA%20gene%20spacer%20enhances%20transcription%20by%20RNA%20polymerase-I%20in%20a%20cell-free%20system&amp;author=A.%20Kuhn&amp;author=U.%20Deppert&amp;author=I.%20Grummt&amp;publication_year=1990&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=87&amp;pages=7527-31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.87.19.7527" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.87.19.7527" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.87.19.7527"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20140-base%20pair%20repetitive%20sequence%20element%20in%20the%20mouse%20ribosomal-RNA%20gene%20spacer%20enhances%20transcription%20by%20RNA%20polymerase-I%20in%20a%20cell-free%20system&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yadav</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Hemansi, Kim</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Tuteja</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Yadav</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">G quadruplex in plants: a ubiquitous regulatory element and its biological relevance</div>, <div class="source">Front Plant Sci</div> ., <div class="volume">8,</div> <div class="fpage">1163</div>, 10.3389/fpls<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=G%20quadruplex%20in%20plants%3A%20a%20ubiquitous%20regulatory%20element%20and%20its%20biological%20relevance&amp;author=V.%20Yadav&amp;author=N.%20Hemansi%2C%20Kim&amp;author=N.%20Tuteja&amp;author=P.%20Yadav&amp;publication_year=2017&amp;journal=Front%20Plant%20Sci&amp;volume=8,&amp;pages=1163" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2017.01163" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2017.01163" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2017.01163"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:G%20quadruplex%20in%20plants%3A%20a%20ubiquitous%20regulatory%20element%20and%20its%20biological%20relevance&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">G.L.</div>
</div>, <div class="name string-name">
<div class="surname">Vasquez</div> <div class="given-names">K.M.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Z-DNA, an active element in the genome</div>, <div class="source">Front. Biosci</div> ., <div class="volume">12</div>, <div class="fpage">4424</div>–<div class="lpage">38</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Z-DNA%2C%20an%20active%20element%20in%20the%20genome&amp;author=G.L.%20Wang&amp;author=K.M.%20Vasquez&amp;publication_year=2007&amp;journal=Front.%20Biosci&amp;volume=12&amp;pages=4424-38" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2741/2399" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2741%2F2399" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2741%2F2399"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485386" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Z-DNA%2C%20an%20active%20element%20in%20the%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mishra</div> <div class="given-names">S.K.</div>
</div>, <div class="name string-name">
<div class="surname">Tawani</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Mishra</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Kumar</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">G4IPDB: a database for G-quadruplex structure forming nucleic acid interacting proteins</div>. <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">38144</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=G4IPDB%3A%20a%20database%20for%20G-quadruplex%20structure%20forming%20nucleic%20acid%20interacting%20proteins&amp;author=S.K.%20Mishra&amp;author=A.%20Tawani&amp;author=A.%20Mishra&amp;author=A.%20Kumar&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=38144" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep38144" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep38144" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep38144"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27905517" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:G4IPDB%3A%20a%20database%20for%20G-quadruplex%20structure%20forming%20nucleic%20acid%20interacting%20proteins&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chiarella</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">De Cola</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Scaglione</div> <div class="given-names">G.L.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Nucleophosmin mutations alter its nucleolar localization by impairing G-quadruplex binding at ribosomal DNA</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">3228</div>–<div class="lpage">39</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nucleophosmin%20mutations%20alter%20its%20nucleolar%20localization%20by%20impairing%20G-quadruplex%20binding%20at%20ribosomal%20DNA&amp;author=S.%20Chiarella&amp;author=A.%20De%20Cola&amp;author=G.L.%20Scaglione&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=3228-39" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkt001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkt001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23328624" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nucleophosmin%20mutations%20alter%20its%20nucleolar%20localization%20by%20impairing%20G-quadruplex%20binding%20at%20ribosomal%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nikolova</div> <div class="given-names">E.N.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Wise</div> <div class="given-names">A.A.</div>
</div>, <div class="name string-name">
<div class="surname">O’Brien</div> <div class="given-names">P.J.</div>
</div>, <div class="name string-name">
<div class="surname">Andricioaei</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Al-Hashimi</div> <div class="given-names">H.M.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Transient hoogsteen base pairs in canonical duplex DNA</div>, <div class="source">Nature</div> , <div class="volume">470</div>, 498.U484.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Transient%20hoogsteen%20base%20pairs%20in%20canonical%20duplex%20DNA&amp;aulast=Nikolova&amp;date=2011&amp;volume=470" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transient%20hoogsteen%20base%20pairs%20in%20canonical%20duplex%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">D.K.</div>
</div>, <div class="name string-name">
<div class="surname">Guo</div> <div class="given-names">K.X.</div>
</div>, <div class="name string-name">
<div class="surname">Shin</div> <div class="given-names">Y.J.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Evidence of the formation of G-quadruplex structures in the promoter region of the human vascular endothelial growth factor gene</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">1256</div>–<div class="lpage">65</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20the%20formation%20of%20G-quadruplex%20structures%20in%20the%20promoter%20region%20of%20the%20human%20vascular%20endothelial%20growth%20factor%20gene&amp;author=D.K.%20Sun&amp;author=K.X.%20Guo&amp;author=Y.J.%20Shin&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=1256-65" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkq926" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq926" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkq926"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20959293" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evidence%20of%20the%20formation%20of%20G-quadruplex%20structures%20in%20the%20promoter%20region%20of%20the%20human%20vascular%20endothelial%20growth%20factor%20gene&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Moradi</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Babin</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Roland</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Sagui</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Reaction path ensemble of the B-Z-DNA transition: a comprehensive atomistic study</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">33</div>–<div class="lpage">43</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Reaction%20path%20ensemble%20of%20the%20B-Z-DNA%20transition%3A%20a%20comprehensive%20atomistic%20study&amp;author=M.%20Moradi&amp;author=V.%20Babin&amp;author=C.%20Roland&amp;author=C.%20Sagui&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=33-43" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks1003" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1003" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks1003"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104380" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Reaction%20path%20ensemble%20of%20the%20B-Z-DNA%20transition%3A%20a%20comprehensive%20atomistic%20study&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Temiz</div> <div class="given-names">N.A.</div>
</div>, <div class="name string-name">
<div class="surname">Donohue</div> <div class="given-names">D.E.</div>
</div>, <div class="name string-name">
<div class="surname">Bacolla</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Luke</div> <div class="given-names">B.T.</div>
</div>, <div class="name string-name">
<div class="surname">Collins</div> <div class="given-names">J.R.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">The role of methylation in the intrinsic dynamics of B- and Z-DNA</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e35558</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20methylation%20in%20the%20intrinsic%20dynamics%20of%20B-%20and%20Z-DNA&amp;author=N.A.%20Temiz&amp;author=D.E.%20Donohue&amp;author=A.%20Bacolla&amp;author=B.T.%20Luke&amp;author=J.R.%20Collins&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e35558" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0035558" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0035558" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0035558"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22530050" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20role%20of%20methylation%20in%20the%20intrinsic%20dynamics%20of%20B-%20and%20Z-DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dobešová</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Malinská</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Matyášek</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Silenced rRNA genes are activated and substitute for partially eliminated active homeologs in the recently formed allotetraploid, Tragopogon mirus (Asteraceae)</div>, <div class="source">Heredity</div> , <div class="volume">114</div>, <div class="fpage">356</div>–<div class="lpage">65</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Silenced%20rRNA%20genes%20are%20activated%20and%20substitute%20for%20partially%20eliminated%20active%20homeologs%20in%20the%20recently%20formed%20allotetraploid%2C%20Tragopogon%20mirus%20(Asteraceae)&amp;author=E.%20Dobe%C5%A1ov%C3%A1&amp;author=H.%20Malinsk%C3%A1&amp;author=R.%20Maty%C3%A1%C5%A1ek&amp;publication_year=2015&amp;journal=Heredity&amp;volume=114&amp;pages=356-65" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/hdy.2014.111" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2014.111" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fhdy.2014.111"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25537492" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Silenced%20rRNA%20genes%20are%20activated%20and%20substitute%20for%20partially%20eliminated%20active%20homeologs%20in%20the%20recently%20formed%20allotetraploid%2C%20Tragopogon%20mirus%20(Asteraceae)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gabrielian</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Bolshoy</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">1999</div>, 
<div class="article-title">Sequence complexity and DNA curvature</div>, <div class="source">Comput. Chem</div> ., <div class="volume">23</div>, <div class="fpage">263</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequence%20complexity%20and%20DNA%20curvature&amp;author=A.%20Gabrielian&amp;author=A.%20Bolshoy&amp;publication_year=1999&amp;journal=Comput.%20Chem&amp;volume=23&amp;pages=263-74" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0097-8485(99)00007-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0097-8485(99)00007-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0097-8485(99)00007-8"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10404619" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequence%20complexity%20and%20DNA%20curvature&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garcia</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Garnatje</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Kovařík</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Plant rDNA database: ribosomal DNA loci information goes online</div>, <div class="source">Chromosoma</div> , <div class="volume">121</div>, <div class="fpage">389</div>–<div class="lpage">94</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Plant%20rDNA%20database%3A%20ribosomal%20DNA%20loci%20information%20goes%20online&amp;author=S.%20Garcia&amp;author=T.%20Garnatje&amp;author=A.%20Kova%C5%99%C3%ADk&amp;publication_year=2012&amp;journal=Chromosoma&amp;volume=121&amp;pages=389-94" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00412-012-0368-7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00412-012-0368-7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00412-012-0368-7"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22527113" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Plant%20rDNA%20database%3A%20ribosomal%20DNA%20loci%20information%20goes%20online&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dadejova</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Lim</div> <div class="given-names">K.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Souckova</div> <div class="given-names">S.K.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Transcription activity of rRNA genes correlates with a tendency towards intergenomic homogenization in <em>Nicotiana</em> allotetraploids</div>, <div class="source">New Phytol</div> ., <div class="volume">174</div>, <div class="fpage">658</div>–<div class="lpage">68</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcription%20activity%20of%20rRNA%20genes%20correlates%20with%20a%20tendency%20towards%20intergenomic%20homogenization%20in%20Nicotiana%20allotetraploids&amp;author=M.%20Dadejova&amp;author=K.Y.%20Lim&amp;author=S.K.%20Souckova&amp;publication_year=2007&amp;journal=New%20Phytol&amp;volume=174&amp;pages=658-68" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.2007.174.issue-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2007.174.issue-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.2007.174.issue-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17447920" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcription%20activity%20of%20rRNA%20genes%20correlates%20with%20a%20tendency%20towards%20intergenomic%20homogenization%20in%20Nicotiana%20allotetraploids&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sochorova</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Coriton</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Kuderova</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Lunerova</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Chevre</div> <div class="given-names">A.M.</div>
</div>, <div class="name string-name">
<div class="surname">Kovarik</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Gene conversion events and variable degree of homogenization of rDNA loci in cultivars of <em>Brassica napus</em>
</div>, <div class="source">Ann. Bot</div> ., <div class="volume">119</div>, <div class="fpage">13</div>–<div class="lpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20conversion%20events%20and%20variable%20degree%20of%20homogenization%20of%20rDNA%20loci%20in%20cultivars%20of%20Brassica%20napus&amp;author=J.%20Sochorova&amp;author=O.%20Coriton&amp;author=A.%20Kuderova&amp;author=J.%20Lunerova&amp;author=A.M.%20Chevre&amp;author=A.%20Kovarik&amp;publication_year=2017&amp;journal=Ann.%20Bot&amp;volume=119&amp;pages=13-26" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/aob/mcw187" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcw187" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Faob%2Fmcw187"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27707747" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20conversion%20events%20and%20variable%20degree%20of%20homogenization%20of%20rDNA%20loci%20in%20cultivars%20of%20Brassica%20napus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bauer</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Horvat</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Birus</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Vicic</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Zoldos</div> <div class="given-names">V.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Nucleotide sequence, structural organization and length heterogeneity of ribosomal DNA intergenic spacer in <em>Quercus petraea</em> (Matt.) Liebl. and <em>Q-robur</em> L</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">281</div>, <div class="fpage">207</div>–<div class="lpage">21</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequence%2C%20structural%20organization%20and%20length%20heterogeneity%20of%20ribosomal%20DNA%20intergenic%20spacer%20in%20Quercus%20petraea%20(Matt.)%20Liebl.%20and%20Q-robur%20L&amp;author=N.%20Bauer&amp;author=T.%20Horvat&amp;author=I.%20Birus&amp;author=V.%20Vicic&amp;author=V.%20Zoldos&amp;publication_year=2009&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=281&amp;pages=207-21" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00438-008-0404-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-008-0404-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00438-008-0404-8"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19052776" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nucleotide%20sequence%2C%20structural%20organization%20and%20length%20heterogeneity%20of%20ribosomal%20DNA%20intergenic%20spacer%20in%20Quercus%20petraea%20(Matt.)%20Liebl.%20and%20Q-robur%20L&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Suzuki</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Tanifuji</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Komeda</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Kato</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">1996</div>, 
<div class="article-title">Structural and functional characterization of the intergenic spacer region of the rDNA in <em>Daucus carota</em>
</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">37</div>, <div class="fpage">233</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20characterization%20of%20the%20intergenic%20spacer%20region%20of%20the%20rDNA%20in%20Daucus%20carota&amp;author=A.%20Suzuki&amp;author=S.%20Tanifuji&amp;author=Y.%20Komeda&amp;author=A.%20Kato&amp;publication_year=1996&amp;journal=Plant%20Cell%20Physiol&amp;volume=37&amp;pages=233-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.pcp.a028937" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.pcp.a028937" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.pcp.a028937"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8665099" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Structural%20and%20functional%20characterization%20of%20the%20intergenic%20spacer%20region%20of%20the%20rDNA%20in%20Daucus%20carota&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz008-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz008-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mestrovic</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Plohl</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Mravinac</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Ugarkovic</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Evolution of satellite DNAs from the genus Palorus - experimental evidence for the “library” hypothesis</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">15</div>, <div class="fpage">1062</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20satellite%20DNAs%20from%20the%20genus%20Palorus%20-%20experimental%20evidence%20for%20the%20%E2%80%9Clibrary%E2%80%9D%20hypothesis&amp;author=N.%20Mestrovic&amp;author=M.%20Plohl&amp;author=B.%20Mravinac&amp;author=D.%20Ugarkovic&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol&amp;volume=15&amp;pages=1062-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026005" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026005" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a026005"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9718733" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolution%20of%20satellite%20DNAs%20from%20the%20genus%20Palorus%20-%20experimental%20evidence%20for%20the%20%E2%80%9Clibrary%E2%80%9D%20hypothesis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5479970&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz008/1/dsz008_supplementary_data.doc?Expires=1576702820&amp;Signature=yuAnneedQrg6p6VFMPMoRouHXjtm7KixuAX3kkMZMDFDdD4fs5ePH4InGxZSqOpxwxhTN9DK9vaYLLnfRaMu8ImYRxN4orjrqB~hd8rOdbd9nStSXdqde3MVQ70qF8hAECzKcYUuhS3y4pZ9ZreLxpHJHje0lfdt9VNC8i7620Ai8JGLlwR8qeADrvfJp1bGkqBHIO7TVoKXLSoFJooOQ~srcZqytQpENHzqZjDAHidQwSLHblZ2jXAjyoWKvz9apbMbmSU~DRnl4wDeVJrQT~fMPiwcYJfssnAXCufWcN6yXY8wx~kF~lhIKvtg3n7H4KV~wNZn8IG2ugZjVhQJzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz008_Supplementary_Data</div></a> - doc file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">802</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz008"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5479970"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f3%2f273%2f5479970">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F3%2F273%2F5479970" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5479970">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Intragenomic heterogeneity of intergenic ribosomal DNA spacers in Cucurbita moschata is determined by DNA minisatellites with variable potential to form non-canonical DNA conformations | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz008");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f273%2f5479970">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
