<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass (Lolium perenne) | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/1/1/5133005","@type":"ScholarlyArticle","name":"Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )","datePublished":"2018-10-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2019-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy033","keywords":["genomic diversity","targeted resequencing","variant calling","allele reconstruction","natural variation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/1/1/5133005","author":[{"name":"Veeckman, Elisabeth","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium  Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium  Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium","@type":"Person","sameAs":"http://orcid.org/0000-0003-0510-6317"},{"name":"Van Glabeke, Sabine","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium","@type":"Person"},{"name":"Haegeman, Annelies","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium","@type":"Person"},{"name":"Muylle, Hilde","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium","@type":"Person"},{"name":"van Parijs, Frederik R D","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium","@type":"Person"},{"name":"Byrne, Stephen L","affiliation":"Teagasc, Crop Science Department, Carlow, Ireland","@type":"Person"},{"name":"Asp, Torben","affiliation":"Department of Molecular Biology and Genetics, Faculty of Science and Technology, Research Center Flakkebjerg Aarhus University, DK Slagelse, Denmark","@type":"Person"},{"name":"Studer, Bruno","affiliation":"Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, CH Zurich, Switzerland","@type":"Person"},{"name":"Rohde, Antje","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium","@type":"Person"},{"name":"Roldán-Ruiz, Isabel","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium  Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium","@type":"Person"},{"name":"Vandepoele, Klaas","affiliation":"Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium  Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium  Center for Plant Systems Biology, VIB, B Ghent, Belgium","@type":"Person"},{"name":"Ruttink, Tom","affiliation":"ILVO, Plant Sciences Unit, B Melle, Belgium  Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium","@type":"Person"}],"description":"Abstract.  Revealing DNA sequence variation within the Lolium perenne genepool is important for genetic analysis and development of breeding applications. We re","pageStart":"1","pageEnd":"12","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636621597&Signature=4u-acLAlK5TsxlPmeaVtFEN-MtPdnlH~6orgdT0xBUirVxz5HrVhy7~DQ-IEqf9B0uEc4-tYShQBI5sX8G7vpnEqlvcU06ArsxzJeIUwx1Owe1m5nsLF4igGGSnHbcSAy~MCNKk7RUohC5iZ-SdMT7zushH~Byu2ehknhDJ8-sAR~-ruxQaJGkoOctXUT6ntp3BqAobK~y2Zpkw~zDkRcgiegdvmDjKOVyqunxlh5u0~H7JA9umf2eelWzQZWEAd9faBgbqKhDyHjEnM8QETmak~4JQipZojZ64Llcze9Rj-QBuq7S6uIYafeq-Fsi6kL38HX1VbhwSYkmfeKHFOzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636621597&Signature=4u-acLAlK5TsxlPmeaVtFEN-MtPdnlH~6orgdT0xBUirVxz5HrVhy7~DQ-IEqf9B0uEc4-tYShQBI5sX8G7vpnEqlvcU06ArsxzJeIUwx1Owe1m5nsLF4igGGSnHbcSAy~MCNKk7RUohC5iZ-SdMT7zushH~Byu2ehknhDJ8-sAR~-ruxQaJGkoOctXUT6ntp3BqAobK~y2Zpkw~zDkRcgiegdvmDjKOVyqunxlh5u0~H7JA9umf2eelWzQZWEAd9faBgbqKhDyHjEnM8QETmak~4JQipZojZ64Llcze9Rj-QBuq7S6uIYafeq-Fsi6kL38HX1VbhwSYkmfeKHFOzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )">
<meta property="og:description" content="Abstract.  Revealing DNA sequence variation within the Lolium perenne genepool is important for genetic analysis and development of breeding applications. We re">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/1/1/5133005">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636621597&amp;Signature=4u-acLAlK5TsxlPmeaVtFEN-MtPdnlH~6orgdT0xBUirVxz5HrVhy7~DQ-IEqf9B0uEc4-tYShQBI5sX8G7vpnEqlvcU06ArsxzJeIUwx1Owe1m5nsLF4igGGSnHbcSAy~MCNKk7RUohC5iZ-SdMT7zushH~Byu2ehknhDJ8-sAR~-ruxQaJGkoOctXUT6ntp3BqAobK~y2Zpkw~zDkRcgiegdvmDjKOVyqunxlh5u0~H7JA9umf2eelWzQZWEAd9faBgbqKhDyHjEnM8QETmak~4JQipZojZ64Llcze9Rj-QBuq7S6uIYafeq-Fsi6kL38HX1VbhwSYkmfeKHFOzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2018-10-15">
<meta name="citation_author" content="Veeckman, Elisabeth">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author_institution" content="Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium">
<meta name="citation_author_institution" content="Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium">
<meta name="citation_author" content="Van Glabeke, Sabine">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author" content="Haegeman, Annelies">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author" content="Muylle, Hilde">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author" content="van Parijs, Frederik R D">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author" content="Byrne, Stephen L">
<meta name="citation_author_institution" content="Teagasc, Crop Science Department, Carlow, Ireland">
<meta name="citation_author" content="Asp, Torben">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Faculty of Science and Technology, Research Center Flakkebjerg Aarhus University, DK Slagelse, Denmark">
<meta name="citation_author" content="Studer, Bruno">
<meta name="citation_author_institution" content="Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, CH Zurich, Switzerland">
<meta name="citation_author" content="Rohde, Antje">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author" content="Roldán-Ruiz, Isabel">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author_institution" content="Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium">
<meta name="citation_author" content="Vandepoele, Klaas">
<meta name="citation_author_institution" content="Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium">
<meta name="citation_author_institution" content="Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium">
<meta name="citation_author_institution" content="Center for Plant Systems Biology, VIB, B Ghent, Belgium">
<meta name="citation_author" content="Ruttink, Tom">
<meta name="citation_author_institution" content="ILVO, Plant Sciences Unit, B Melle, Belgium">
<meta name="citation_author_institution" content="Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium">
<meta name="citation_title" content="Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass (Lolium perenne)">
<meta name="citation_firstpage" content="1">
<meta name="citation_lastpage" content="12">
<meta name="citation_doi" content="10.1093/dnares/dsy033">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/02/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_publisher=Springer, Berlin, Germany;  citation_title=In: Sustainable Use of Genetic Diversity in Forage and Turf Breeding;  citation_year=2010;  citation_pages=383-390; ">
<meta name="citation_reference" content="citation_title=Drought changes and the mechanism analysis for the North American Prairie;  citation_journal_title=J. Arid Land;  citation_year=2013;  citation_volume=5;  citation_pages=1-14; ">
<meta name="citation_reference" content="citation_title=Nucleotide diversity and linkage disequilibrium in five Lolium perenne genes with putative role in shoot morphology;  citation_journal_title=Plant Sci;  citation_year=2010;  citation_volume=179;  citation_pages=194-201; ">
<meta name="citation_reference" content="citation_title=Linkage disequilibrium in synthetic varieties of perennial ryegrass;  citation_journal_title=Theor. Appl. Genet;  citation_year=2007;  citation_volume=115;  citation_pages=837-47; ">
<meta name="citation_reference" content="citation_title=Association of candidate genes with flowering time and water-soluble carbohydrate content in Lolium perenne (L.);  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=177;  citation_pages=535-47; ">
<meta name="citation_reference" content="citation_title=Nucleotide diversity and linkage disequilibrium of nine genes with putative effects on flowering time in perennial ryegrass (Lolium perenne L.);  citation_journal_title=Plant Sci;  citation_year=2011;  citation_volume=180;  citation_pages=228-37; ">
<meta name="citation_reference" content="citation_title=Allelic variation in the perennial ryegrass flowering locus T gene is associated with changes in flowering time across a range of populations;  citation_journal_title=Plant Physiol;  citation_year=2011;  citation_volume=155;  citation_pages=1013-22; ">
<meta name="citation_reference" content="citation_title=A transcriptome map of perennial ryegrass (Lolium perenne L.);  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=140">
<meta name="citation_reference" content="citation_title=Genetic-geographic correlation revealed across a broad European ecotypic sample of perennial ryegrass (Lolium perenne) using array-based SNP genotyping;  citation_journal_title=Theor. Appl. Genet;  citation_year=2015;  citation_volume=128;  citation_pages=1917-32; ">
<meta name="citation_reference" content="citation_title=Genotyping-by-sequencing: a promising tool for plant genetics research and breeding;  citation_journal_title=Hortic. Environ. Biotechnol;  citation_year=2017;  citation_volume=58;  citation_pages=425-31; ">
<meta name="citation_reference" content="citation_title=Nucleotide diversity and linkage disequilibrium in 11 expressed resistance candidate genes in Lolium perenne;  citation_journal_title=BMC Plant Biol;  citation_year=2007;  citation_volume=7;  citation_pages=43">
<meta name="citation_reference" content="citation_title=A synteny-based draft genome sequence of the forage grass Lolium perenne;  citation_journal_title=Plant J;  citation_year=2015;  citation_volume=84;  citation_pages=816-26; ">
<meta name="citation_reference" content="citation_title=Variant callers for next-generation sequencing data: a comparison study;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e75619">
<meta name="citation_reference" content="citation_title=Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing;  citation_journal_title=Genome Med;  citation_year=2013;  citation_volume=5;  citation_pages=28">
<meta name="citation_reference" content="citation_title=Validation and assessment of variant calling pipelines for next-generation sequencing;  citation_journal_title=Hum. Genomics;  citation_year=2014;  citation_volume=8;  citation_pages=14">
<meta name="citation_reference" content="citation_title=Comparing a few SNP calling algorithms using low-coverage sequencing data;  citation_journal_title=BMC Bioinformatics;  citation_year=2013;  citation_volume=14;  citation_pages=274">
<meta name="citation_reference" content="citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=341">
<meta name="citation_reference" content="citation_title=Automated reconstruction of whole-genome phylogenies from short-sequence reads;  citation_journal_title=Mol Biol Evol;  citation_year=2014;  citation_volume=31;  citation_pages=1077-88; ">
<meta name="citation_reference" content="citation_title=Multiple reference genomes and transcriptomes for Arabidopsis thaliana;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=477;  citation_pages=419">
<meta name="citation_reference" content="citation_title=PLAZA 3.0: an access point for plant comparative genomics;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D974-81; ">
<meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; ">
<meta name="citation_reference" content="citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0;  citation_journal_title=Syst Biol;  citation_year=2010;  citation_volume=59;  citation_pages=307-21; ">
<meta name="citation_reference" content="citation_title=De novo assembly of the perennial ryegrass transcriptome using an RNA-Seq strategy;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e103567">
<meta name="citation_reference" content="citation_title=Vernalization mediated changes in the Lolium perenne transcriptome;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e107365">
<meta name="citation_reference" content="citation_title=Orthology guided assembly in highly heterozygous crops: creating a reference transcriptome to uncover genetic diversity in Lolium perenne;  citation_journal_title=Plant Biotechnol. J;  citation_year=2013;  citation_volume=11;  citation_pages=605-17; ">
<meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; ">
<meta name="citation_reference" content="citation_title=Design optimization methods for genomic DNA tiling arrays;  citation_journal_title=Genome Res;  citation_year=2006;  citation_volume=16;  citation_pages=271-81; ">
<meta name="citation_reference" content="citation_title=Rapid isolation of high molecular-weight plant DNA;  citation_journal_title=Nucleic Acids Res;  citation_year=1980;  citation_volume=8;  citation_pages=4321-5; ">
<meta name="citation_reference" content="citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e62355">
<meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; ">
<meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; ">
<meta name="citation_reference" content="citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=873-81; ">
<meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-2; ">
<meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; ">
<meta name="citation_reference" content="citation_title=Haplotype-based variant detection from short-read sequencing;  citation_journal_title=arXiv Preprint arXiv;  citation_year=2012;  citation_volume=1207;  citation_pages=3907.">
<meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; ">
<meta name="citation_reference" content="citation_title=From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline;  citation_journal_title=Curr Protoc Bioinformatics;  citation_year=2013;  citation_volume=43; ">
<meta name="citation_reference" content="citation_title=The variant call format and VCFtools;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; ">
<meta name="citation_reference" content="citation_title=VariantMetaCaller: automated fusion of variant calling pipelines for quantitative, precision-based filtering;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=875">
<meta name="citation_reference" content="citation_title=UpSet: visualization of Intersecting Sets;  citation_journal_title=IEEE Trans. Vis. Comput. Graph;  citation_year=2014;  citation_volume=20;  citation_pages=1983-92; ">
<meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3;  citation_journal_title=Fly (Austin);  citation_year=2012;  citation_volume=6;  citation_pages=80-92; ">
<meta name="citation_reference" content="citation_title=Primer3-new capabilities and interfaces;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e115">
<meta name="citation_reference" content="citation_title=PEAR: a fast and accurate Illumina Paired-End reAd mergeR;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=614-20; ">
<meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program;  citation_journal_title=Genome Res;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; ">
<meta name="citation_reference" content="citation_title=Engineering a software tool for gene structure prediction in higher organisms;  citation_journal_title=Inform Software Tech;  citation_year=2005;  citation_volume=47;  citation_pages=965-78; ">
<meta name="citation_reference" content="citation_title=A gene encoding a DUF247 domain protein cosegregates with the S self-incompatibility locus in perennial ryegrass;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=870-84; ">
<meta name="citation_reference" content="citation_title=Are we there yet? Reliably estimating the completeness of plant genome sequences;  citation_journal_title=Plant Cell;  citation_year=2016;  citation_volume=28;  citation_pages=1759-68; ">
<meta name="citation_reference" content="citation_publisher=Springer International Publishing, Cham;  citation_title=Molecular Breeding of Forage and Turf: The Proceedings of the 8th International Symposium on the Molecular Breeding of Forage and Turf;  citation_year=2015;  citation_pages=93-109; ">
<meta name="citation_reference" content="citation_title=Coverage recommendation for genotyping analysis of highly heterologous species using next-generation sequencing technology;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=35736">
<meta name="citation_reference" content="citation_title=An analytical workflow for accurate variant discovery in highly divergent regions;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=703">
<meta name="citation_reference" content="citation_title=Comprehensive analysis to improve the validation rate for single nucleotide variants detected by next-generation sequencing;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e86664">
<meta name="citation_reference" content="citation_title=Cross-platform compatibility of Hi-Plex, a streamlined approach for targeted massively parallel sequencing;  citation_journal_title=Anal Biochem;  citation_year=2013;  citation_volume=442;  citation_pages=127-9; ">
<meta name="citation_reference" content="citation_title=Toward better understanding of artifacts in variant calling from high-coverage samples;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2843-51; ">
<meta name="citation_reference" content="citation_title=GIGANTEA – an emerging story;  citation_journal_title=Front. Plant Sci;  citation_year=2015;  citation_volume=6;  citation_pages=8">
<meta name="citation_reference" content="citation_title=A protein farnesyl transferase involved in abscisic acid signal transduction in Arabidopsis;  citation_journal_title=Science;  citation_year=1996;  citation_volume=273;  citation_pages=1239-41; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/1/1/28033430/dsy033.pdf">
<meta name="description" content="Abstract.  Revealing DNA sequence variation within the Lolium perenne genepool is important for genetic analysis and development of breeding applications. We re">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/1/1/5133005">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/1/1/5133005">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/1/1/5133005");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy033';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f1%2f5133005" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f1%2f5133005" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=114871"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1576706054&amp;Signature=vBWNddoR50MXunHaIOWR4NKGA3RzdnFj2w0qk3NvgIypxbeQEyInkkoEtdnasX6NtO2S2NYf3wqojJKdpkSFXETU8YoNv8da7ngvqlrRkx27avSNfvtlH9MM6arJXHCP4x1EGEd1qpfCP0KZ6NvYu4FFgfaXjimkXV8u1WHsuaW8YiKN2unIWtrryrrA0p4NYAcZr3k6hjEdf7~MV5W2ILlSUUxJe64AmhGJjhHplwvyT53I4xxJUHwHLOuGWmMVUAVTCG8Z6Ft8d9JyoaR2TXsCMWL7Tp6L0bHZ4xsRiMFopapA7x0wOLVPo~2Cy0erwXXcezAGNXmQA8A-lZh-yQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/1">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="131979638">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979638">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979640">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979640">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979647">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979647">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979663">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979663">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979702">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979702">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="131979710">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979710">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="empty arrow"> </li>
            <li class="next arrow">
            <a href="/dnaresearch/article/26/1/13/5142401">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass (<em>Lolium perenne</em>)
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elisabeth Veeckman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
<div class="aff">Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium</div>
<div class="aff">Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-0510-6317">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0003-0510-6317
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Elisabeth+Veeckman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Veeckman%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Veeckman%20Elisabeth%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Elisabeth Veeckman</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sabine Van Glabeke</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sabine+Van+Glabeke">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Van%20Glabeke%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Van%20Glabeke%20Sabine%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Sabine Van Glabeke</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Annelies Haegeman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Annelies+Haegeman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Haegeman%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Haegeman%20Annelies%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Annelies Haegeman</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hilde Muylle</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hilde+Muylle">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muylle%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Muylle%20Hilde%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Hilde Muylle</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Frederik R D van Parijs</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Frederik+R+D+van+Parijs">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=van%20Parijs%20F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22van%20Parijs%20Frederik%20R%20D%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Frederik R D van Parijs</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephen L Byrne</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Teagasc, Crop Science Department, Carlow, Ireland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephen+L+Byrne">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Byrne%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Byrne%20Stephen%20L%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Stephen L Byrne</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Torben Asp</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular Biology and Genetics, Faculty of Science and Technology, Research Center Flakkebjerg Aarhus University, DK Slagelse, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Torben+Asp">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Asp%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Asp%20Torben%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Torben Asp</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bruno Studer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Molecular Plant Breeding, Institute of Agricultural Sciences, ETH Zurich, CH Zurich, Switzerland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bruno+Studer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Studer%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Studer%20Bruno%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Bruno Studer</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antje Rohde</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
        </div>


        <div class="info-author-correspondence">
            Present address: BASF Agricultural Solutions, Technologiepark 38, B-9052 Ghent, Belgium.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antje+Rohde">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rohde%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Rohde%20Antje%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Antje Rohde</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Isabel Roldán-Ruiz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
<div class="aff">Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Isabel+Rold%c3%a1n-Ruiz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rold%C3%A1n-Ruiz%20I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Rold%C3%A1n-Ruiz%20Isabel%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Isabel Roldán-Ruiz</a><span class="delimiter-hidden">, </span>
                                </span>
                                    <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                        <span>...</span> Show more
                                    </a>
                                    <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Klaas Vandepoele</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium</div>
<div class="aff">Department of Plant Biotechnology and Bioinformatics, Ghent University, B Ghent, Belgium</div>
<div class="aff">Center for Plant Systems Biology, VIB, B Ghent, Belgium</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Klaas+Vandepoele">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vandepoele%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Vandepoele%20Klaas%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Klaas Vandepoele</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tom Ruttink</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">ILVO, Plant Sciences Unit, B Melle, Belgium</div>
<div class="aff">Bioinformatics Institute Ghent, Ghent University, B Ghent, Belgium</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +32 9 272 28 78. Fax. +32 9 272 29 01. Email: <a href="mailto:tom.ruttink@ilvo.vlaanderen.be" target="_blank">tom.ruttink@ilvo.vlaanderen.be</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tom+Ruttink">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ruttink%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ruttink%20Tom%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Tom Ruttink</a><span class="delimiter"></span>
                                        </span>
                                    </div>
                            </div>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 1–12, <a href="https://doi.org/10.1093/dnares/dsy033">https://doi.org/10.1093/dnares/dsy033</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">15 October 2018</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">26 June 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">06 September 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">15 October 2018</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5133005" href="/dnaresearch/article-pdf/26/1/1/28033430/dsy033.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Elisabeth Veeckman, Sabine Van Glabeke, Annelies Haegeman, Hilde Muylle, Frederik R D van Parijs, Stephen L Byrne, Torben Asp, Bruno Studer, Antje Rohde, Isabel Roldán-Ruiz, Klaas Vandepoele, Tom Ruttink,  Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass (<em>Lolium perenne</em>), <em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 1–12, <a href="https://doi.org/10.1093/dnares/dsy033">https://doi.org/10.1093/dnares/dsy033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5133005&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Overcoming%20challenges%20in%20variant%20calling%3A%20exploring%20sequence%20diversity%20in%20candidate%20genes%20for%20plant%20development%20in%20perennial%20ryegrass%20(%26lt%3Bem%26gt%3BLolium%20perenne%26lt%3B%2Fem%26gt%3B)&amp;publicationDate=2018-10-15&amp;volumeNum=26&amp;issueNum=1&amp;author=Veeckman%2C%20Elisabeth%3B%20Van%20Glabeke%2C%20Sabine&amp;startPage=1&amp;endPage=12&amp;contentId=10.1093%2Fdnares%2Fdsy033&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f1%2f5133005" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="131979638" id="131979638" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Revealing DNA sequence variation within the <em>Lolium perenne</em> genepool is important for genetic analysis and development of breeding applications. We reviewed current literature on plant development to select candidate genes in pathways that control agronomic traits, and identified 503 orthologues in <em>L. perenne</em>. Using targeted resequencing, we constructed a comprehensive catalogue of genomic variation for a <em>L. perenne</em> germplasm collection of 736 genotypes derived from current cultivars, breeding material and wild accessions. To overcome challenges of variant calling in heterogeneous outbreeding species, we used two complementary strategies to explore sequence diversity. First, four variant calling pipelines were integrated with the VariantMetaCaller to reach maximal sensitivity. Additional multiplex amplicon sequencing was used to empirically estimate an appropriate precision threshold. Second, a <em>de novo</em> assembly strategy was used to reconstruct divergent alleles for each gene. The advantage of this approach was illustrated by discovery of 28 novel alleles of <em>LpSDUF247</em>, a polymorphic gene co-segregating with the S-locus of the grass self-incompatibility system. Our approach is applicable to other genetically diverse outbreeding species. The resulting collection of functionally annotated variants can be mined for variants causing phenotypic variation, either through genetic association studies, or by selecting carriers of rare defective alleles for physiological analyses.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genomic diversity"'>genomic diversity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"targeted resequencing"'>targeted resequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"variant calling"'>variant calling</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"allele reconstruction"'>allele reconstruction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"natural variation"'>natural variation</a>
</div>                    <h2 scrollto-destination="131979640" id="131979640" class="section-title" data-legacy-id="SEC1">1. Introduction</h2>
<p>Perennial ryegrass (<em>Lolium perenne</em> L.) is one of the most widely cultivated grass species in Europe. It is of interest for grazing, hay and silage production as it has a long growing season, and relatively high yield and nutritive value. Because of its outbreeding nature, individual plants are highly heterozygous and the diploid perennial ryegrass genome is highly heterogeneous both within and across breeding populations and wild accessions. As genomic variation forms the foundation of phenotypic variation, revealing DNA sequence variation within the genepool is important for genetic analysis and development of breeding applications.<a reveal-id="dsy033-B1" data-open="dsy033-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a></p>
<p>Several studies used a candidate gene-based approach to associate sequence polymorphisms with phenotypic variation. Examples include the association of <em>Late embryogenesis abundant 3</em> (<em>LEA3</em>) with drought tolerance,<a reveal-id="dsy033-B2" data-open="dsy033-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><em>Brassinosteroid insensitive 1</em> (<em>BRI1</em>) with shoot morphology,<a reveal-id="dsy033-B3" data-open="dsy033-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><em>Gibberellic acid insensitive</em> (<em>GAI</em>) with organ growth,<a reveal-id="dsy033-B4" data-open="dsy033-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>Heading date 1</em> (<em>HD1</em>) with carbohydrate content<a reveal-id="dsy033-B5" data-open="dsy033-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and <em>Flowering locus T</em> (<em>FT</em>) with flowering time.<a reveal-id="dsy033-B5 dsy033-B6 dsy033-B7" data-open="dsy033-B5 dsy033-B6 dsy033-B7" class="link link-ref link-reveal xref-bibr"><sup>5–7</sup></a> While these studies show the power of testing gene–trait associations, the limited number of genes per study was mostly due to the high cost of genotyping at the time. However, this approach is not amenable to study complex traits related to plant development and phenology, which are typically regulated by the interaction of many genes. Therefore, we need versatile and cost-efficient methods to characterize the genetic variation in parallel for hundreds of candidate genes and hundreds of genotypes. This enables breeders to perform higher resolution screening of genetic diversity in their material and link genotypic and phenotypic variation.</p>
<p>Single-nucleotide polymorphisms (SNPs) are the most prevalent type of genomic variation and are convenient molecular markers. Two complementary SNP genotyping arrays are available for high-throughput screening in perennial ryegrass.<a reveal-id="dsy033-B8" data-open="dsy033-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy033-B9" data-open="dsy033-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> These arrays target SNPs in genic regions, but do not allow discovery of new sequence variants. In contrast, genotyping-by-sequencing (GBS) allows for simultaneous discovery of genome-wide SNPs and genotyping of a large number of individuals or pools, thereby avoiding ascertainment bias. Therefore, GBS has broad applications in plant breeding and genetics studies, including linkage maps, genome-wide association studies, genomic selection and genomic diversity studies.<a reveal-id="dsy033-B10" data-open="dsy033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Only short fragments (range about 100–300 bp) are sequenced and there is no <em>a priori</em> control over which genes are tagged. In combination with a local short linkage disequilibrium that is typical for an outbreeding species as <em>L. perenne</em>,<a reveal-id="dsy033-B3" data-open="dsy033-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy033-B11" data-open="dsy033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> it can be very difficult to identify SNPs that are causal for the phenotype of interest.</p>
<p>The large size of the <em>L. perenne</em> genome (2 Gbp) and high repetitive sequence content (76%)<a reveal-id="dsy033-B12" data-open="dsy033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> currently precludes whole genome sequencing at sufficient depth in hundreds of accessions as has been done in e.g. Arabidopsis, rice and soybean. To study trait genetics in forage and turf grasses, we identified hundreds of candidate genes in genetic pathways that control plant development and quality traits, and analysed their genome sequence using probe capture enrichment for targeted resequencing in a large germplasm collection of 736 genotypes. We specifically focussed on genes involved in pathways related to interesting agronomic traits, such as plant growth and architecture (important for biomass yield), development and transition to flowering (important for seasonal control of growth), cell wall biogenesis (important for digestibility) and phytohormone biosynthesis, signalling and response [including abscisic acid (ABA), auxin, brassinosteroids, cytokinins, ethylene, gibberellic acid and strigolactones]. Identifying sequence variants in these genes provides insights in the range of naturally occurring genomic diversity that can be expected in gene-rich regions of the genome. The variants can be used as markers for association genetics studies (as previously described for <em>LEA3</em>, <em>BRI1</em>, <em>GAI</em>, <em>HD1</em> and <em>FT3</em>), and/or to identify alleles with altered amino acid sequence, mRNA splicing or mRNA stability, hence altered gene function or regulation possibly resulting in an altered physiology and thereby affecting the phenotype.</p>
<p>Multiple bioinformatics methods are available to identify sequence variants using next-generation sequencing (NGS) data, but defining a complete and reliable variant set remains difficult. <em>De novo</em> discovery of genomic polymorphisms commonly relies on mapping reads to a single reference genome sequence. Although the GATK best practices are the most commonly used variant calling (VC) pipeline, there is no single best VC pipeline available with both good sensitivity and precision. Moreover, there is low concordance between VC pipelines, even with the same input data.<a reveal-id="dsy033-B13 dsy033-B14 dsy033-B15 dsy033-B16" data-open="dsy033-B13 dsy033-B14 dsy033-B15 dsy033-B16" class="link link-ref link-reveal xref-bibr"><sup>13–16</sup></a> In addition, each VC pipeline returns different variant annotations that can be used for quality filtering. Choosing the appropriate filtering criteria and thresholds [for instance, minimum read depth (RD)] is not straightforward as NGS data typically has a non-uniform distribution of coverage<a reveal-id="dsy033-B17" data-open="dsy033-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and estimated quality values may be dataset dependent so that optimal settings need to be calibrated for each dataset. The high density of sequence polymorphisms in the germplasm collection with respect to the <em>L. perenne</em> genome sequence could also hamper variant identification. More divergent alleles could contain the most interesting genomic variation, but are also the most difficult to detect as reads that are highly divergent from the reference genome may fail to map if the parameters for short read alignment are too stringent.<a reveal-id="dsy033-B18" data-open="dsy033-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Hence, if capture and/or mapping efficiency of highly divergent sequences precludes their detection, it is to be expected that routine workflows of mapping and VC lead to an underestimation of the genetic diversity at highly divergent regions, a known problem in genome resequencing studies.<a reveal-id="dsy033-B19" data-open="dsy033-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p>
<p>Here, we present the identification and annotation of 503 <em>L. perenne</em> orthologues of known genes that regulate plant growth and development. These genes were resequenced in a germplasm collection of 736 genotypes to describe the genomic variation in <em>L. perenne</em>. Two complementary strategies were used to obtain a reliable and complete catalogue of genomic variation. First, four VC pipelines were compared and automatically integrated to reach maximal sensitivity. The influence of mapping algorithms was assessed and hard filtering was compared with precision-based filtering to reach sufficient specificity. Additionally, an alternative strategy consisting of <em>de novo</em> assembly followed by overlap-layout-consensus (OLC) clustering was used to circumvent read mapping bias and to construct alternative alleles for each gene. This reference independent allele reconstruction is particularly important for gene families with highly divergent alleles. We demonstrated the benefit of this approach for <em>LpSDUF247</em> and identified 28 novel alleles that were not detected using traditional VC pipelines. This approach is broadly applicable to other highly heterozygous outbreeding species. Finally, we used all this information to create a comprehensive catalogue of functionally annotated genetic variation across many pathways that control growth, development and agricultural traits.</p>                    <h2 scrollto-destination="131979647" id="131979647" class="section-title" data-legacy-id="SEC2">2. Materials and methods</h2>
                    <h3 scrollto-destination="131979648" id="131979648" class="section-title" data-legacy-id="SEC2.1">2.1 Candidate gene identification and manual curation</h3>
<p>Gene families of <em>A. thaliana</em> candidate genes were identified using the comparative genomics platform PLAZA 3.0 Monocots.<a reveal-id="dsy033-B20" data-open="dsy033-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><em>Brachypodium distachyon</em> family members were used to identify homologous loci in the draft genome sequence of <em>L. perenne</em><a reveal-id="dsy033-B12" data-open="dsy033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> using BLASTx analysis (<em>E</em>-value 10e-5). At each <em>L. perenne</em> locus, predicted protein sequences were added to the corresponding PLAZA 3.0 Monocots gene family. Using protein sequences of all gene family members of <em>A. thaliana</em>, <em>B. distachyon</em> and <em>L. perenne</em>, a phylogenetic tree was built with MUSCLE (v3.8.31)<a reveal-id="dsy033-B21" data-open="dsy033-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and PhyML<a reveal-id="dsy033-B22" data-open="dsy033-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> using default settings, and for each <em>A. thaliana</em> candidate gene the closest orthologous <em>L. perenne</em> gene was selected for further manual curation. The <em>L. perenne</em> gene models were evaluated using multiple protein sequence alignments with MUSCLE using orthologous proteins from <em>B. distachyon</em>, <em>O. sativa</em>, <em>Z. mays</em> and <em>S. bicolor</em> according to PLAZA 3.0 Monocots. Additional RNA-seq data<a reveal-id="dsy033-B23 dsy033-B24 dsy033-B25" data-open="dsy033-B23 dsy033-B24 dsy033-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a> mapped with TopHat (v2.0.13)<a reveal-id="dsy033-B26" data-open="dsy033-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> using default settings, was used to refine gene models and delineate untranslated regions, or to design a new gene model if required (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S1).</p>                    <h3 scrollto-destination="131979650" id="131979650" class="section-title" data-legacy-id="SEC2.2">2.2 Probe design, library construction and sequencing</h3>
<p>The coding strand of each of the 503 target regions (gene model and 1,000 bp upstream promoter region) was tiled with 120 bp probes, starting every 40 bp using OligoTiler.<a reveal-id="dsy033-B27" data-open="dsy033-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Probes showing high sequence similarity to non-targets, other probes, repetitive sequences, mitochondrial or chloroplast sequences, or with extreme GC content (&lt;25% or &gt;65%) were removed. Finally, 57,693 SureSelect probes of 120 bp (Agilent) were retained, covering 2.3 Mb of the intended 2.8 Mb target region, at around 3× tiling.</p>
<p>Genomic DNA was extracted from freeze-dried leaf material from 736 <em>L. perenne</em> genotypes representing current cultivars, breeding material and wild accessions using the cetyltrimethylammonium bromide (CTAB) method.<a reveal-id="dsy033-B28" data-open="dsy033-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> DNA concentration was measured using the Quantus double-stranded DNA assay (Promega, Madison, WI, USA). For each genotype, an indexed shotgun sequencing library was prepared from 100 ng DNA by (i) Adaptive Focused Acoustic fragmentation on a Covaris S2 instrument (Covaris, Inc.), (ii) adapter ligation and (iii) magnetic bead purification using an adapted protocol of Uitdewilligen et al.<a reveal-id="dsy033-B29" data-open="dsy033-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The libraries were pooled without normalization into eight pools, each containing 96 libraries of individual genotypes. Each pool was used for a probe capture hybridization reaction according to the SureSelect protocol (Agilent SureSelectXT2 Target Enrichment for Illumina Paired-End Sequencing Library Protocol, v. 1.0). After PCR amplification of purified enriched pooled libraries, each pool of 96 libraries was sequenced on one lane of a HiSeq2000 instrument using 2 × 91 PE sequencing (BGI, Shenzhen, China). The raw data is available in the NCBI Sequence Read Archive (BioProject PRJNA434356, Accessions SRR6812717–SRR6813075).</p>                    <h3 scrollto-destination="131979653" id="131979653" class="section-title" data-legacy-id="SEC2.3">2.3 Read mapping and variant calling</h3>
<p>Raw reads were trimmed and quality filtered by Trimmomatic (v0.32)<a reveal-id="dsy033-B30" data-open="dsy033-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and mapped onto the draft perennial ryegrass genome sequence<a reveal-id="dsy033-B12" data-open="dsy033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> with default settings of BWA-MEM (version 0.7.8-r455)<a reveal-id="dsy033-B31" data-open="dsy033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and GSNAP (version 2016-09-23).<a reveal-id="dsy033-B32" data-open="dsy033-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Duplicate reads were marked using Picard-tools (release 1.113). Local realignment around indels was performed according to the best practices workflow of the Genome Analysis Toolkit (GATK) (v.3.7).<a reveal-id="dsy033-B36" data-open="dsy033-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsy033-B37" data-open="dsy033-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> RD and coverage were calculated on the resulting BAM files using BEDTools (v2.25.0).<a reveal-id="dsy033-B33" data-open="dsy033-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a></p>
<p>Four different VC pipelines were used: SAMtools (version 1.2-115-gb8ff342),<a reveal-id="dsy033-B34" data-open="dsy033-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Freebayes (v1.0.2-2-g7ceb532),<a reveal-id="dsy033-B35" data-open="dsy033-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> GATK Unified Genotyper (GATK UG) and GATK HaplotypeCaller (GATK HC).<a reveal-id="dsy033-B36" data-open="dsy033-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsy033-B37" data-open="dsy033-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Multi-allelic variants were removed using VCFtools (v0.1.14).<a reveal-id="dsy033-B38" data-open="dsy033-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> For hard filtering, a custom Python script was used to remove variant positions and genotype calls with a RD lower than 6 and a genotype quality (GQ) lower than 30. SNPs and indels were automatically integrated by the VMC (v1.0),<a reveal-id="dsy033-B39" data-open="dsy033-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> in 10 and 2 partitions, respectively. The estimated precision (EP) was calculated using a custom Python script based on the formulas given in Gézsi et al.<a reveal-id="dsy033-B39" data-open="dsy033-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The concordance of SNP and indel sets identified by four VC pipelines was determined using information in the INFO field of the VCF file returned by VMC and visually represented using Upset,<a reveal-id="dsy033-B40" data-open="dsy033-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> before and after precision-based filtering (EP &gt; 80%). Functional effects of sequence variants were predicted with SnpEff (version 4.3T).<a reveal-id="dsy033-B41" data-open="dsy033-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> To validate consistency of genotype calls in an F1 segregating population, Mendelian inheritance errors (MIE) were defined after precision-based filtering (EP &gt; 80%) using PLINK (v1.90b2t), for two parents and their F1 progeny of 29 individuals. Variants with a missing genotype call in either one of the parents were excluded from analysis, as were MIEs derived from a missing genotype call in one of the 29 F1 progeny.</p>                    <h3 scrollto-destination="131979656" id="131979656" class="section-title" data-legacy-id="SEC2.4">2.4 Hi-Plex amplicon sequencing</h3>
<p>To generate an independent variant set, 78 genotypes were selected for resequencing of 171 amplicon regions of 80–140 bp. Of these, 147 amplicons overlap with 28 candidate genes. Primers were designed with Primer3<a reveal-id="dsy033-B42" data-open="dsy033-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and divided into two highly multiplex (Hi-Plex) PCR-reactions according to their amplification efficiency (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S4). DNA was extracted using the CTAB method<a reveal-id="dsy033-B28" data-open="dsy033-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and DNA concentration was measured using the Quantus double-stranded DNA assay (Promega, Madison, WI, USA). Per sample, the final DNA concentration was adjusted to 40 ng/µl and the amplicons were PCR-amplified while adding sample specific indices. Libraries were prepared using the KAPA Hyper Prep PCR-free Kit according to manufacturer directions (Kapa Biosystems, USA). Hi-Plex amplification reactions and library preparations were done by Floodlight Genomics LLC (Knoxville, TN, USA). The libraries were sequenced with 2 × 150 PE on a HiSeq3000 instrument (OMRF, Oklahoma City, OK, USA). Paired-end reads were merged with PEAR (v0.9.8)<a reveal-id="dsy033-B43" data-open="dsy033-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> and adapter sequences were removed. The read data is available in the NCBI Sequence Read Archive (BioProject PRJNA437219, Accessions SRR6813540–SRR6813585). BWA-MEM was used for read mapping, and VC was done by running the four VC pipelines. Bi-allelic variants were extracted using VCFtools and combined by VMC and the EP was calculated as described above.</p>                    <h3 scrollto-destination="131979658" id="131979658" class="section-title" data-legacy-id="SEC2.5">2.5 Identification of divergent alleles of <em>LpSDUF247</em>
</h3>
<p>Per genotype, all reads were used for De Bruijn Graph assembly without scaffolding (CLC Genomics Workbench 9.5.3, <a class="link link-uri" href="https://www.qiagenbioinformatics.com" target="_blank">https://www.qiagenbioinformatics.com</a> (date last accessed 14 september 2018)). Contigs of at least 200 bp were retained and mapped onto the reference genome with BWA-MEM using default parameters, to group all contigs of the 736 genotypes per candidate gene. Per candidate gene, sequences of overlapping allelic fragments were extracted from the BAM files using BEDtools and clustered with the OLC assembler CAP3 (version date 02/10/15).<a reveal-id="dsy033-B44" data-open="dsy033-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Singlet sequences returned by CAP3 were removed from further analysis. All resulting alleles are assigned to their respective candidate gene and are available as (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S5), allowing the reader to repeat the analyses described below for any other candidate gene.</p>
<p>One of the candidate genes of the 503 gene set, <em>LpSDUF247</em>, is known to be highly polymorphic and was selected to demonstrate in-depth reconstruction of divergent alleles. The 34 contigs of <em>LpSDUF247</em> were aligned using MUSCLE and six highly similar sequences (&gt;98% identity) were removed. The reference gene model of <em>LpSDUF247</em> was projected onto the contigs using GenomeThreader (v 1.6.6)<a reveal-id="dsy033-B45" data-open="dsy033-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> to identify CDS regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S6) and corresponding protein sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S7).</p>
<p>All <em>B. distachyon</em> members of the DUF247 gene family (HOM03M000101) were used in a tBLASTn search against the perennial ryegrass genome sequence, and 25 <em>LpDUF247</em> genes were identified and manually annotated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S8). After multiple sequence alignment of all 25 LpDUF247 protein sequences with <em>B. distachyon</em> and <em>H. vulgare</em> gene family members using MUSCLE, a phylogenetic tree was built with PhyML using 100 rounds of bootstrapping (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Similarly, a phylogenetic tree was built using the reference protein sequences of LpDUF247-01, LpSDUF247, LpDUF247-03 and LpDUF247-04, the protein sequences of the LpSDUF247 alleles, and five LpSDUF247-02 alleles identified by Manzanares et al.<a reveal-id="dsy033-B46" data-open="dsy033-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>).</p>
<p>The 28 novel alleles were added to the reference genome sequence and read mapping was repeated for all 736 genotypes onto this multi-allelic reference genome. A matrix was created with the average RD per <em>LpSDUF247</em> allele per genotype using BEDtools. This matrix was normalized per genotype, by dividing the RD <em>per LpSDUF247</em> allele by the sum of RDs across <em>all LpSDUF247</em> alleles, to identify alleles with the highest relative RD for each genotype while correcting for differences in library size and capture efficiency across the set of 736 samples.</p>                    <h2 scrollto-destination="131979663" id="131979663" class="section-title" data-legacy-id="SEC3">3. Results and discussion</h2>
                    <h3 scrollto-destination="131979664" id="131979664" class="section-title" data-legacy-id="SEC3.1">3.1 Identification, classification and curation of target genes</h3>
<p>To identify <em>L. perenne</em> genes putatively involved in the regulation of plant growth and development, plant architecture, induction of flowering, cell wall biogenesis and phytohormone biosynthesis, signalling and response, we first searched the literature for <em>Arabidopsis thaliana</em> genes with a well-defined molecular and physiological function (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Next, the corresponding 174 gene families were identified with the comparative genomics platform PLAZA 3.0 Monocots.<a reveal-id="dsy033-B20" data-open="dsy033-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> For each of the <em>A. thaliana</em> candidate genes, a comprehensive list of orthologous loci in the draft genome sequence of <em>L. perenne</em><a reveal-id="dsy033-B12" data-open="dsy033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> was delineated. A phylogenetic tree was built for <em>A. thaliana</em> and <em>B. distachyon</em> gene family members of the 174 PLAZA gene families, to select the closest orthologous <em>L. perenne</em> sequences of the candidate genes. When no clear one-to-one orthologous pairs were found due to lineage-specific gene duplication or gene loss events, the best two or three <em>L. perenne</em> loci were selected from the respective clades. The final selection contained 503 <em>L. perenne</em> candidate genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). For 407 of these loci, an annotated gene model was available.<a reveal-id="dsy033-B12" data-open="dsy033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> For the other 96 loci, a gene model needed to be annotated <em>ab initio</em>, in line with previous observations that the annotated gene space of <em>L. perenne</em> is 76% complete.<a reveal-id="dsy033-B47" data-open="dsy033-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The available gene models were evaluated using multiple protein sequence alignments with all their monocot gene family members according to PLAZA 3.0 Monocots. In addition, mapped RNA-seq data<a reveal-id="dsy033-B23 dsy033-B24 dsy033-B25" data-open="dsy033-B23 dsy033-B24 dsy033-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a> was used to refine gene models and delineate untranslated regions. Taken together, manual curation of 503 gene models (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S1), showed that previously available gene models<a reveal-id="dsy033-B12" data-open="dsy033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> were correct for 272 loci (54%) and needed small adaptations for 135 loci (27%). A completely new gene model was annotated at 96 loci (19%) using RNA-seq data. The length of the protein sequences corresponds well to that of their closest <em>B. distachyon</em> orthologs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), showing that the 503 manually curated <em>L. perenne</em> gene models are of high quality (HQ). This was required to delineate regions for probe design and to correctly position variants relative to the reading frame in the CDS to functionally interpret the consequences of sequence polymorphism in the genic regions. Finally, the 503 candidate genes were assigned to biological processes based on the known function of their <em>A. thaliana</em> orthologs (<a reveal-id="dsy033-T1" data-open="dsy033-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). This HQ gene set can also be used to train and validate gene prediction algorithms to improve genome-wide gene annotation. </p>
<div content-id="dsy033-T1" class="table-modal"><div class="table-wrap" content-id="dsy033-T1">
<div id="dsy033-T1" data-id="dsy033-T1" class="table-wrap-title">
<div class="label">Table 1</div>
<div class="caption"><p>Assignment of 503 candidate genes to pathways and distribution of high impact mutations per pathway</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr>
<th>Pathway </th>
<th>Gene families </th>
<th># candidate genes </th>
<th>Stop gain </th>
<th>Splice site </th>
<th>Frame shift </th>
</tr></thead>
<tbody>
<tr><td colspan="6">
<strong>Plant development and architecture</strong> </td></tr>
<tr>
<td>Development </td>
<td>BCH1, BRIZ, CBP80, DRM1, HB13, HYL1, ING2, RSM1, SAMDC4 </td>
<td>14 </td>
<td>2 (14%) </td>
<td>4 (29%) </td>
<td>– </td>
</tr>
<tr>
<td>Cell wall </td>
<td>4CL, ALDH, C3H, C4H, CAD, CAD2, CCoAOMT, CCR, CES, COMT, F5H, HCT, HPRGP, IRX, LAC, OFP, PAL, POX, SND, XylS, XylT </td>
<td>121 </td>
<td>41 (34%) </td>
<td>16 (13%) </td>
<td>5 (4%) </td>
</tr>
<tr>
<td>Cell wall TF </td>
<td>ERF, WRKY </td>
<td>6 </td>
<td>2 (33%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Cell wall TF MYB </td>
<td>MYB </td>
<td>21 </td>
<td>3 (14%) </td>
<td>– </td>
<td>1 (5%) </td>
</tr>
<tr>
<td>Cell wall TF NAC </td>
<td>NAC </td>
<td>11 </td>
<td>2 (18%) </td>
<td>4 (36%) </td>
<td>1 (9%) </td>
</tr>
<tr>
<td>Chromatin remodelling </td>
<td>MET1, SWI </td>
<td>4 </td>
<td>3 (75%) </td>
<td>2 (50%) </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ initiation </td>
<td>ANT, SLOMO, TOP1A </td>
<td>6 </td>
<td>1 (17%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ patterning morphogenesis </td>
<td>AS, CLF, DOT5, GRF, KAN, NOV, SE, TRN1, YABBY, ZPR1, ZPR3 </td>
<td>30 </td>
<td>7 (23%) </td>
<td>3 (10%) </td>
<td>2 (7%) </td>
</tr>
<tr>
<td>Lateral organ identity </td>
<td>AN3, BOP, HDZIPIII </td>
<td>10 </td>
<td>4 (40%) </td>
<td>1 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Light signalling </td>
<td>bHLHABAI, CO1, COP9, CRY, DET1, HY5, LHY, PCI, PFT1, PHYB, PIF, SPA </td>
<td>29 </td>
<td>4 (14%) </td>
<td>7 (24%) </td>
<td>– </td>
</tr>
<tr>
<td>Shoot apical meristem </td>
<td>BARD1, BLH, CLPS3, FTA, KNAT, OBE1, ULT1, USP1, VEF2, WOX14, WUS </td>
<td>25 </td>
<td>8 (32%) </td>
<td>5 (20%) </td>
<td>3 (12%) </td>
</tr>
<tr>
<td>Self-incompatibility </td>
<td>DUF247, GK </td>
<td>4 </td>
<td>2 (50%) </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
</tr>
<tr>
<td>Transition to flowering </td>
<td>CCA, FCA, FIE, FKF1, FLD, FPA, FT, FVE, FWA, FY, GI, LHP1, MBD9, PHP, RAV, SDG8, SPL3, VIL3, VRN1, VRN1-like </td>
<td>45 </td>
<td>19 (42%) </td>
<td>12 (27%) </td>
<td>1 (2%) </td>
</tr>
<tr>
<td>Flower development </td>
<td>ESD4, HAC3, LFY3, LUG, MADS, RGA, SEU, SUF4, SUP </td>
<td>31 </td>
<td>2 (6%) </td>
<td>4 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Transcription factor </td>
<td>BIM2, TCP </td>
<td>8 </td>
<td>2 (25%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr><td colspan="6">
<strong>Phytohormone biosynthesis, signalling and response</strong> </td></tr>
<tr>
<td>ABA biosynthesis </td>
<td>NCED1, PDS1, PDS3 </td>
<td>4 </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
<td>– </td>
</tr>
<tr>
<td>ABA signalling </td>
<td>ABI1, ABI3, ABI5, ABI8, AIP3, DRIP, GBF, GPA, GTG2, HD2C, PSY, SAD1, SIR3, WIG, ZEP </td>
<td>29 </td>
<td>10 (34%) </td>
<td>3 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin biosynthesis </td>
<td>TAA1, TAR2, YUC </td>
<td>6 </td>
<td>3 (50%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Auxin signalling </td>
<td>ADA2B, AMP1, ARF, AUXIAA, AXR, AXR1, AXR4, AXR6, CAND1, GH3, TIR1 </td>
<td>20 </td>
<td>8 (40%) </td>
<td>3 (15%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin transport </td>
<td>AUX1, ENP, PGP4, PID2, PIN1, PIN1like, SPS </td>
<td>12 </td>
<td>1 (8%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid biosynthesis </td>
<td>DWF1, DWF3, DWF5, DWF7, SQS </td>
<td>8 </td>
<td>2 (25%) </td>
<td>1 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid signalling </td>
<td>BES1 </td>
<td>2 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Cytokinin signalling </td>
<td>ARR, CRE, GCR1, RR </td>
<td>11 </td>
<td>2 (18%) </td>
<td>1 (9%) </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene biosynthesis </td>
<td>ACS </td>
<td>2 </td>
<td>1 (50%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene signalling </td>
<td>EBF1, EBF2, EIL3, EIN2, ETO1, ETR1 </td>
<td>13 </td>
<td>7 (54%) </td>
<td>1 (8%) </td>
<td>1 (8%) </td>
</tr>
<tr>
<td>Gibberellin biosynthesis </td>
<td>GAOX </td>
<td>11 </td>
<td>4 (36%) </td>
<td>– </td>
<td>2 (18%) </td>
</tr>
<tr>
<td>Gibberellin signalling </td>
<td>GID1A, SHI, SPY </td>
<td>5 </td>
<td>– </td>
<td>1 (20%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton biosynthesis </td>
<td>D14, D27, MAX1, MAX3, MAX4 </td>
<td>11 </td>
<td>3 (27%) </td>
<td>2 (18%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton signalling </td>
<td>MAX2, TB1 </td>
<td>4 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>
<strong>Total</strong> </td>
<td>
<strong>180</strong> </td>
<td>
<strong>503</strong> </td>
<td>
<strong>144</strong> </td>
<td>
<strong>72</strong> </td>
<td>
<strong>19</strong> </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy033-T1" class="table-modal"><table>
<thead name="thead" align="left"><tr>
<th>Pathway </th>
<th>Gene families </th>
<th># candidate genes </th>
<th>Stop gain </th>
<th>Splice site </th>
<th>Frame shift </th>
</tr></thead>
<tbody>
<tr><td colspan="6">
<strong>Plant development and architecture</strong> </td></tr>
<tr>
<td>Development </td>
<td>BCH1, BRIZ, CBP80, DRM1, HB13, HYL1, ING2, RSM1, SAMDC4 </td>
<td>14 </td>
<td>2 (14%) </td>
<td>4 (29%) </td>
<td>– </td>
</tr>
<tr>
<td>Cell wall </td>
<td>4CL, ALDH, C3H, C4H, CAD, CAD2, CCoAOMT, CCR, CES, COMT, F5H, HCT, HPRGP, IRX, LAC, OFP, PAL, POX, SND, XylS, XylT </td>
<td>121 </td>
<td>41 (34%) </td>
<td>16 (13%) </td>
<td>5 (4%) </td>
</tr>
<tr>
<td>Cell wall TF </td>
<td>ERF, WRKY </td>
<td>6 </td>
<td>2 (33%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Cell wall TF MYB </td>
<td>MYB </td>
<td>21 </td>
<td>3 (14%) </td>
<td>– </td>
<td>1 (5%) </td>
</tr>
<tr>
<td>Cell wall TF NAC </td>
<td>NAC </td>
<td>11 </td>
<td>2 (18%) </td>
<td>4 (36%) </td>
<td>1 (9%) </td>
</tr>
<tr>
<td>Chromatin remodelling </td>
<td>MET1, SWI </td>
<td>4 </td>
<td>3 (75%) </td>
<td>2 (50%) </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ initiation </td>
<td>ANT, SLOMO, TOP1A </td>
<td>6 </td>
<td>1 (17%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ patterning morphogenesis </td>
<td>AS, CLF, DOT5, GRF, KAN, NOV, SE, TRN1, YABBY, ZPR1, ZPR3 </td>
<td>30 </td>
<td>7 (23%) </td>
<td>3 (10%) </td>
<td>2 (7%) </td>
</tr>
<tr>
<td>Lateral organ identity </td>
<td>AN3, BOP, HDZIPIII </td>
<td>10 </td>
<td>4 (40%) </td>
<td>1 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Light signalling </td>
<td>bHLHABAI, CO1, COP9, CRY, DET1, HY5, LHY, PCI, PFT1, PHYB, PIF, SPA </td>
<td>29 </td>
<td>4 (14%) </td>
<td>7 (24%) </td>
<td>– </td>
</tr>
<tr>
<td>Shoot apical meristem </td>
<td>BARD1, BLH, CLPS3, FTA, KNAT, OBE1, ULT1, USP1, VEF2, WOX14, WUS </td>
<td>25 </td>
<td>8 (32%) </td>
<td>5 (20%) </td>
<td>3 (12%) </td>
</tr>
<tr>
<td>Self-incompatibility </td>
<td>DUF247, GK </td>
<td>4 </td>
<td>2 (50%) </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
</tr>
<tr>
<td>Transition to flowering </td>
<td>CCA, FCA, FIE, FKF1, FLD, FPA, FT, FVE, FWA, FY, GI, LHP1, MBD9, PHP, RAV, SDG8, SPL3, VIL3, VRN1, VRN1-like </td>
<td>45 </td>
<td>19 (42%) </td>
<td>12 (27%) </td>
<td>1 (2%) </td>
</tr>
<tr>
<td>Flower development </td>
<td>ESD4, HAC3, LFY3, LUG, MADS, RGA, SEU, SUF4, SUP </td>
<td>31 </td>
<td>2 (6%) </td>
<td>4 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Transcription factor </td>
<td>BIM2, TCP </td>
<td>8 </td>
<td>2 (25%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr><td colspan="6">
<strong>Phytohormone biosynthesis, signalling and response</strong> </td></tr>
<tr>
<td>ABA biosynthesis </td>
<td>NCED1, PDS1, PDS3 </td>
<td>4 </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
<td>– </td>
</tr>
<tr>
<td>ABA signalling </td>
<td>ABI1, ABI3, ABI5, ABI8, AIP3, DRIP, GBF, GPA, GTG2, HD2C, PSY, SAD1, SIR3, WIG, ZEP </td>
<td>29 </td>
<td>10 (34%) </td>
<td>3 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin biosynthesis </td>
<td>TAA1, TAR2, YUC </td>
<td>6 </td>
<td>3 (50%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Auxin signalling </td>
<td>ADA2B, AMP1, ARF, AUXIAA, AXR, AXR1, AXR4, AXR6, CAND1, GH3, TIR1 </td>
<td>20 </td>
<td>8 (40%) </td>
<td>3 (15%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin transport </td>
<td>AUX1, ENP, PGP4, PID2, PIN1, PIN1like, SPS </td>
<td>12 </td>
<td>1 (8%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid biosynthesis </td>
<td>DWF1, DWF3, DWF5, DWF7, SQS </td>
<td>8 </td>
<td>2 (25%) </td>
<td>1 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid signalling </td>
<td>BES1 </td>
<td>2 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Cytokinin signalling </td>
<td>ARR, CRE, GCR1, RR </td>
<td>11 </td>
<td>2 (18%) </td>
<td>1 (9%) </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene biosynthesis </td>
<td>ACS </td>
<td>2 </td>
<td>1 (50%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene signalling </td>
<td>EBF1, EBF2, EIL3, EIN2, ETO1, ETR1 </td>
<td>13 </td>
<td>7 (54%) </td>
<td>1 (8%) </td>
<td>1 (8%) </td>
</tr>
<tr>
<td>Gibberellin biosynthesis </td>
<td>GAOX </td>
<td>11 </td>
<td>4 (36%) </td>
<td>– </td>
<td>2 (18%) </td>
</tr>
<tr>
<td>Gibberellin signalling </td>
<td>GID1A, SHI, SPY </td>
<td>5 </td>
<td>– </td>
<td>1 (20%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton biosynthesis </td>
<td>D14, D27, MAX1, MAX3, MAX4 </td>
<td>11 </td>
<td>3 (27%) </td>
<td>2 (18%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton signalling </td>
<td>MAX2, TB1 </td>
<td>4 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>
<strong>Total</strong> </td>
<td>
<strong>180</strong> </td>
<td>
<strong>503</strong> </td>
<td>
<strong>144</strong> </td>
<td>
<strong>72</strong> </td>
<td>
<strong>19</strong> </td>
</tr>
</tbody>
</table></div>
</div></div>
<div class="table-wrap" content-id="dsy033-T1">
<div id="dsy033-T1" data-id="dsy033-T1" class="table-wrap-title">
<div class="label">Table 1</div>
<div class="caption"><p>Assignment of 503 candidate genes to pathways and distribution of high impact mutations per pathway</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr>
<th>Pathway </th>
<th>Gene families </th>
<th># candidate genes </th>
<th>Stop gain </th>
<th>Splice site </th>
<th>Frame shift </th>
</tr></thead>
<tbody>
<tr><td colspan="6">
<strong>Plant development and architecture</strong> </td></tr>
<tr>
<td>Development </td>
<td>BCH1, BRIZ, CBP80, DRM1, HB13, HYL1, ING2, RSM1, SAMDC4 </td>
<td>14 </td>
<td>2 (14%) </td>
<td>4 (29%) </td>
<td>– </td>
</tr>
<tr>
<td>Cell wall </td>
<td>4CL, ALDH, C3H, C4H, CAD, CAD2, CCoAOMT, CCR, CES, COMT, F5H, HCT, HPRGP, IRX, LAC, OFP, PAL, POX, SND, XylS, XylT </td>
<td>121 </td>
<td>41 (34%) </td>
<td>16 (13%) </td>
<td>5 (4%) </td>
</tr>
<tr>
<td>Cell wall TF </td>
<td>ERF, WRKY </td>
<td>6 </td>
<td>2 (33%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Cell wall TF MYB </td>
<td>MYB </td>
<td>21 </td>
<td>3 (14%) </td>
<td>– </td>
<td>1 (5%) </td>
</tr>
<tr>
<td>Cell wall TF NAC </td>
<td>NAC </td>
<td>11 </td>
<td>2 (18%) </td>
<td>4 (36%) </td>
<td>1 (9%) </td>
</tr>
<tr>
<td>Chromatin remodelling </td>
<td>MET1, SWI </td>
<td>4 </td>
<td>3 (75%) </td>
<td>2 (50%) </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ initiation </td>
<td>ANT, SLOMO, TOP1A </td>
<td>6 </td>
<td>1 (17%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ patterning morphogenesis </td>
<td>AS, CLF, DOT5, GRF, KAN, NOV, SE, TRN1, YABBY, ZPR1, ZPR3 </td>
<td>30 </td>
<td>7 (23%) </td>
<td>3 (10%) </td>
<td>2 (7%) </td>
</tr>
<tr>
<td>Lateral organ identity </td>
<td>AN3, BOP, HDZIPIII </td>
<td>10 </td>
<td>4 (40%) </td>
<td>1 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Light signalling </td>
<td>bHLHABAI, CO1, COP9, CRY, DET1, HY5, LHY, PCI, PFT1, PHYB, PIF, SPA </td>
<td>29 </td>
<td>4 (14%) </td>
<td>7 (24%) </td>
<td>– </td>
</tr>
<tr>
<td>Shoot apical meristem </td>
<td>BARD1, BLH, CLPS3, FTA, KNAT, OBE1, ULT1, USP1, VEF2, WOX14, WUS </td>
<td>25 </td>
<td>8 (32%) </td>
<td>5 (20%) </td>
<td>3 (12%) </td>
</tr>
<tr>
<td>Self-incompatibility </td>
<td>DUF247, GK </td>
<td>4 </td>
<td>2 (50%) </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
</tr>
<tr>
<td>Transition to flowering </td>
<td>CCA, FCA, FIE, FKF1, FLD, FPA, FT, FVE, FWA, FY, GI, LHP1, MBD9, PHP, RAV, SDG8, SPL3, VIL3, VRN1, VRN1-like </td>
<td>45 </td>
<td>19 (42%) </td>
<td>12 (27%) </td>
<td>1 (2%) </td>
</tr>
<tr>
<td>Flower development </td>
<td>ESD4, HAC3, LFY3, LUG, MADS, RGA, SEU, SUF4, SUP </td>
<td>31 </td>
<td>2 (6%) </td>
<td>4 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Transcription factor </td>
<td>BIM2, TCP </td>
<td>8 </td>
<td>2 (25%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr><td colspan="6">
<strong>Phytohormone biosynthesis, signalling and response</strong> </td></tr>
<tr>
<td>ABA biosynthesis </td>
<td>NCED1, PDS1, PDS3 </td>
<td>4 </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
<td>– </td>
</tr>
<tr>
<td>ABA signalling </td>
<td>ABI1, ABI3, ABI5, ABI8, AIP3, DRIP, GBF, GPA, GTG2, HD2C, PSY, SAD1, SIR3, WIG, ZEP </td>
<td>29 </td>
<td>10 (34%) </td>
<td>3 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin biosynthesis </td>
<td>TAA1, TAR2, YUC </td>
<td>6 </td>
<td>3 (50%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Auxin signalling </td>
<td>ADA2B, AMP1, ARF, AUXIAA, AXR, AXR1, AXR4, AXR6, CAND1, GH3, TIR1 </td>
<td>20 </td>
<td>8 (40%) </td>
<td>3 (15%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin transport </td>
<td>AUX1, ENP, PGP4, PID2, PIN1, PIN1like, SPS </td>
<td>12 </td>
<td>1 (8%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid biosynthesis </td>
<td>DWF1, DWF3, DWF5, DWF7, SQS </td>
<td>8 </td>
<td>2 (25%) </td>
<td>1 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid signalling </td>
<td>BES1 </td>
<td>2 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Cytokinin signalling </td>
<td>ARR, CRE, GCR1, RR </td>
<td>11 </td>
<td>2 (18%) </td>
<td>1 (9%) </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene biosynthesis </td>
<td>ACS </td>
<td>2 </td>
<td>1 (50%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene signalling </td>
<td>EBF1, EBF2, EIL3, EIN2, ETO1, ETR1 </td>
<td>13 </td>
<td>7 (54%) </td>
<td>1 (8%) </td>
<td>1 (8%) </td>
</tr>
<tr>
<td>Gibberellin biosynthesis </td>
<td>GAOX </td>
<td>11 </td>
<td>4 (36%) </td>
<td>– </td>
<td>2 (18%) </td>
</tr>
<tr>
<td>Gibberellin signalling </td>
<td>GID1A, SHI, SPY </td>
<td>5 </td>
<td>– </td>
<td>1 (20%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton biosynthesis </td>
<td>D14, D27, MAX1, MAX3, MAX4 </td>
<td>11 </td>
<td>3 (27%) </td>
<td>2 (18%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton signalling </td>
<td>MAX2, TB1 </td>
<td>4 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>
<strong>Total</strong> </td>
<td>
<strong>180</strong> </td>
<td>
<strong>503</strong> </td>
<td>
<strong>144</strong> </td>
<td>
<strong>72</strong> </td>
<td>
<strong>19</strong> </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy033-T1" class="table-modal"><table>
<thead name="thead" align="left"><tr>
<th>Pathway </th>
<th>Gene families </th>
<th># candidate genes </th>
<th>Stop gain </th>
<th>Splice site </th>
<th>Frame shift </th>
</tr></thead>
<tbody>
<tr><td colspan="6">
<strong>Plant development and architecture</strong> </td></tr>
<tr>
<td>Development </td>
<td>BCH1, BRIZ, CBP80, DRM1, HB13, HYL1, ING2, RSM1, SAMDC4 </td>
<td>14 </td>
<td>2 (14%) </td>
<td>4 (29%) </td>
<td>– </td>
</tr>
<tr>
<td>Cell wall </td>
<td>4CL, ALDH, C3H, C4H, CAD, CAD2, CCoAOMT, CCR, CES, COMT, F5H, HCT, HPRGP, IRX, LAC, OFP, PAL, POX, SND, XylS, XylT </td>
<td>121 </td>
<td>41 (34%) </td>
<td>16 (13%) </td>
<td>5 (4%) </td>
</tr>
<tr>
<td>Cell wall TF </td>
<td>ERF, WRKY </td>
<td>6 </td>
<td>2 (33%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Cell wall TF MYB </td>
<td>MYB </td>
<td>21 </td>
<td>3 (14%) </td>
<td>– </td>
<td>1 (5%) </td>
</tr>
<tr>
<td>Cell wall TF NAC </td>
<td>NAC </td>
<td>11 </td>
<td>2 (18%) </td>
<td>4 (36%) </td>
<td>1 (9%) </td>
</tr>
<tr>
<td>Chromatin remodelling </td>
<td>MET1, SWI </td>
<td>4 </td>
<td>3 (75%) </td>
<td>2 (50%) </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ initiation </td>
<td>ANT, SLOMO, TOP1A </td>
<td>6 </td>
<td>1 (17%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Lateral organ patterning morphogenesis </td>
<td>AS, CLF, DOT5, GRF, KAN, NOV, SE, TRN1, YABBY, ZPR1, ZPR3 </td>
<td>30 </td>
<td>7 (23%) </td>
<td>3 (10%) </td>
<td>2 (7%) </td>
</tr>
<tr>
<td>Lateral organ identity </td>
<td>AN3, BOP, HDZIPIII </td>
<td>10 </td>
<td>4 (40%) </td>
<td>1 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Light signalling </td>
<td>bHLHABAI, CO1, COP9, CRY, DET1, HY5, LHY, PCI, PFT1, PHYB, PIF, SPA </td>
<td>29 </td>
<td>4 (14%) </td>
<td>7 (24%) </td>
<td>– </td>
</tr>
<tr>
<td>Shoot apical meristem </td>
<td>BARD1, BLH, CLPS3, FTA, KNAT, OBE1, ULT1, USP1, VEF2, WOX14, WUS </td>
<td>25 </td>
<td>8 (32%) </td>
<td>5 (20%) </td>
<td>3 (12%) </td>
</tr>
<tr>
<td>Self-incompatibility </td>
<td>DUF247, GK </td>
<td>4 </td>
<td>2 (50%) </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
</tr>
<tr>
<td>Transition to flowering </td>
<td>CCA, FCA, FIE, FKF1, FLD, FPA, FT, FVE, FWA, FY, GI, LHP1, MBD9, PHP, RAV, SDG8, SPL3, VIL3, VRN1, VRN1-like </td>
<td>45 </td>
<td>19 (42%) </td>
<td>12 (27%) </td>
<td>1 (2%) </td>
</tr>
<tr>
<td>Flower development </td>
<td>ESD4, HAC3, LFY3, LUG, MADS, RGA, SEU, SUF4, SUP </td>
<td>31 </td>
<td>2 (6%) </td>
<td>4 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Transcription factor </td>
<td>BIM2, TCP </td>
<td>8 </td>
<td>2 (25%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr><td colspan="6">
<strong>Phytohormone biosynthesis, signalling and response</strong> </td></tr>
<tr>
<td>ABA biosynthesis </td>
<td>NCED1, PDS1, PDS3 </td>
<td>4 </td>
<td>1 (25%) </td>
<td>1 (25%) </td>
<td>– </td>
</tr>
<tr>
<td>ABA signalling </td>
<td>ABI1, ABI3, ABI5, ABI8, AIP3, DRIP, GBF, GPA, GTG2, HD2C, PSY, SAD1, SIR3, WIG, ZEP </td>
<td>29 </td>
<td>10 (34%) </td>
<td>3 (10%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin biosynthesis </td>
<td>TAA1, TAR2, YUC </td>
<td>6 </td>
<td>3 (50%) </td>
<td>– </td>
<td>1 (17%) </td>
</tr>
<tr>
<td>Auxin signalling </td>
<td>ADA2B, AMP1, ARF, AUXIAA, AXR, AXR1, AXR4, AXR6, CAND1, GH3, TIR1 </td>
<td>20 </td>
<td>8 (40%) </td>
<td>3 (15%) </td>
<td>– </td>
</tr>
<tr>
<td>Auxin transport </td>
<td>AUX1, ENP, PGP4, PID2, PIN1, PIN1like, SPS </td>
<td>12 </td>
<td>1 (8%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid biosynthesis </td>
<td>DWF1, DWF3, DWF5, DWF7, SQS </td>
<td>8 </td>
<td>2 (25%) </td>
<td>1 (13%) </td>
<td>– </td>
</tr>
<tr>
<td>Brassinosteroid signalling </td>
<td>BES1 </td>
<td>2 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Cytokinin signalling </td>
<td>ARR, CRE, GCR1, RR </td>
<td>11 </td>
<td>2 (18%) </td>
<td>1 (9%) </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene biosynthesis </td>
<td>ACS </td>
<td>2 </td>
<td>1 (50%) </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>Ethylene signalling </td>
<td>EBF1, EBF2, EIL3, EIN2, ETO1, ETR1 </td>
<td>13 </td>
<td>7 (54%) </td>
<td>1 (8%) </td>
<td>1 (8%) </td>
</tr>
<tr>
<td>Gibberellin biosynthesis </td>
<td>GAOX </td>
<td>11 </td>
<td>4 (36%) </td>
<td>– </td>
<td>2 (18%) </td>
</tr>
<tr>
<td>Gibberellin signalling </td>
<td>GID1A, SHI, SPY </td>
<td>5 </td>
<td>– </td>
<td>1 (20%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton biosynthesis </td>
<td>D14, D27, MAX1, MAX3, MAX4 </td>
<td>11 </td>
<td>3 (27%) </td>
<td>2 (18%) </td>
<td>– </td>
</tr>
<tr>
<td>Strigolacton signalling </td>
<td>MAX2, TB1 </td>
<td>4 </td>
<td>– </td>
<td>– </td>
<td>– </td>
</tr>
<tr>
<td>
<strong>Total</strong> </td>
<td>
<strong>180</strong> </td>
<td>
<strong>503</strong> </td>
<td>
<strong>144</strong> </td>
<td>
<strong>72</strong> </td>
<td>
<strong>19</strong> </td>
</tr>
</tbody>
</table></div>
</div>                    <h3 scrollto-destination="131979666" id="131979666" class="section-title" data-legacy-id="SEC3.2">3.2 Design and efficacy of targeted resequencing by probe capture enrichment</h3>
<p>For each candidate gene, a target region was delineated spanning the curated gene model and an additional 1,000 bp upstream promoter region, as described previously.<a reveal-id="dsy033-B48" data-open="dsy033-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Probes were designed for a total length of 2.3 Mbp, corresponding to a coverage of 85% of each target region on average, as probes targeting repetitive regions were excluded (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S2). Targeted resequencing of 736 genotypes resulted in 3.2 million reads per genotype on average (range 20 thousand–31 million). After duplicate read removal, a mean of 1.9 million reads was retained per genotype, corresponding to a mean RD of 80× per position within the target regions. For VC analysis in heterozygous diploid species, a coverage of at least 6–10× is desirable to avoid false negative heterozygous calls.<a reveal-id="dsy033-B49" data-open="dsy033-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Saturation curves show a non-linear relationship between number of reads per library and target region coverage at a given RD threshold, as expected for probe capture enriched shotgun sequencing libraries (<a reveal-id="dsy033-F1" data-open="dsy033-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). At least 550,000 uniquely mapped reads per genotype were required to reach the probe region coverage plateau at 95% for RD ≥ 1. Further increasing the number of reads per sample did not substantially increase probe region coverage (see Ruttink et al.<a reveal-id="dsy033-B48" data-open="dsy033-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a>). The probe region coverage was slightly lower at higher RD thresholds (89% for RD ≥ 6 and 85% for RD ≥ 10 (boxplots in <a reveal-id="dsy033-F1" data-open="dsy033-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). </p>                        <a id="131979668" scrollto-destination="131979668"></a>
<div id="dsy033-F1" data-id="dsy033-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f1.png?Expires=1576700707&amp;Signature=JkCN6iqa3JNmSd5mxQfIlCyLEuP3ZYlH83ZfYz2Ifh6pAvd1tWM~X3wasC2EoBpC-cZdJSoTTNs3P1ZWue4lNTl~ti~YIXeHGLEth41zUUh5qxgfnpK-ByutFxoythbqOAZP52td9nGvIBJaeEk-gYzsIijEcsdFUyWFwCLS8QAclNbZiFIMSrpV8og0-Oaz4zMaoRZs2EqshE2aJASNp2-ZsxqZncVPXCgfBobNmlEUX0Qh4NrZ-~71W4Cj9-IuWf2PWzxqlxqg9Cd8EXnNcnfIOdY7CT381ESXiOFdFxyObizMV~nv1gP1LTOAomQQJvIYwOmZ2pGNXeytH9Ni0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Target region coverage per genotype. For each of the 503 candidate genes, the target region was delineated as the gene model and an additional 1,000 bp upstream promoter region. The mean fraction of the target region covered per genotype is shown in function of the number of uniquely mapped reads using BWA-MEM after duplicate removal, using different RD thresholds [RD ≥ 1 (blue), RD ≥ 6 (green) and RD ≥ 10 (orange)]." path-from-xml="dsy033f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979668/dsy033f1.tif" path-from-xml="dsy033f1.tif" target="_blank">Open in new tab</a><a section="131979668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f1.png?Expires=1576700707&amp;Signature=I-WRjC~6M6cOyWWHLdULNYg17OLTJd9VELwUG691lmcjIs1wMX0pQFS0qpQc950YVraKmcTNJc~VGdWJVACoCOgtJTkKqGIEm698leRo2qJjA-WT4I2ZB~utvGFwcHEZ~5nx-hPL9MZzs6ahRlfMmuhaAj~ASAApDAksrLIB9x5cX94PUg2Nu6p-ccA7brTH~ITgQtX7K4lRu1mA2M-Z7Fi762nfE6wkrPVihbWL6MNzGa5aEXcrmONu47Gnv0csJLqRckwJqBaOpd43PpmojeA8VIhR1iIfMsx4v3JEODWKMB5-yJd6JRulE2aW0eLeCufcaroX-LiUoBcETOTycw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979668&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Target region coverage per genotype. For each of the 503 candidate genes, the target region was delineated as the gene model and an additional 1,000 bp upstream promoter region. The mean fraction of the target region covered per genotype is shown in function of the number of uniquely mapped reads using BWA-MEM after duplicate removal, using different RD thresholds [RD ≥ 1 (blue), RD ≥ 6 (green) and RD ≥ 10 (orange)].</p></div>
</div>
<div content-id="dsy033-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f1.png?Expires=1576700707&amp;Signature=JkCN6iqa3JNmSd5mxQfIlCyLEuP3ZYlH83ZfYz2Ifh6pAvd1tWM~X3wasC2EoBpC-cZdJSoTTNs3P1ZWue4lNTl~ti~YIXeHGLEth41zUUh5qxgfnpK-ByutFxoythbqOAZP52td9nGvIBJaeEk-gYzsIijEcsdFUyWFwCLS8QAclNbZiFIMSrpV8og0-Oaz4zMaoRZs2EqshE2aJASNp2-ZsxqZncVPXCgfBobNmlEUX0Qh4NrZ-~71W4Cj9-IuWf2PWzxqlxqg9Cd8EXnNcnfIOdY7CT381ESXiOFdFxyObizMV~nv1gP1LTOAomQQJvIYwOmZ2pGNXeytH9Ni0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Target region coverage per genotype. For each of the 503 candidate genes, the target region was delineated as the gene model and an additional 1,000 bp upstream promoter region. The mean fraction of the target region covered per genotype is shown in function of the number of uniquely mapped reads using BWA-MEM after duplicate removal, using different RD thresholds [RD ≥ 1 (blue), RD ≥ 6 (green) and RD ≥ 10 (orange)]." path-from-xml="dsy033f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979668/dsy033f1.tif" path-from-xml="dsy033f1.tif" target="_blank">Open in new tab</a><a section="131979668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f1.png?Expires=1576700707&amp;Signature=I-WRjC~6M6cOyWWHLdULNYg17OLTJd9VELwUG691lmcjIs1wMX0pQFS0qpQc950YVraKmcTNJc~VGdWJVACoCOgtJTkKqGIEm698leRo2qJjA-WT4I2ZB~utvGFwcHEZ~5nx-hPL9MZzs6ahRlfMmuhaAj~ASAApDAksrLIB9x5cX94PUg2Nu6p-ccA7brTH~ITgQtX7K4lRu1mA2M-Z7Fi762nfE6wkrPVihbWL6MNzGa5aEXcrmONu47Gnv0csJLqRckwJqBaOpd43PpmojeA8VIhR1iIfMsx4v3JEODWKMB5-yJd6JRulE2aW0eLeCufcaroX-LiUoBcETOTycw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979668&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Target region coverage per genotype. For each of the 503 candidate genes, the target region was delineated as the gene model and an additional 1,000 bp upstream promoter region. The mean fraction of the target region covered per genotype is shown in function of the number of uniquely mapped reads using BWA-MEM after duplicate removal, using different RD thresholds [RD ≥ 1 (blue), RD ≥ 6 (green) and RD ≥ 10 (orange)].</p></div>
</div>                    <h3 scrollto-destination="131979669" id="131979669" class="section-title" data-legacy-id="SEC3.3">3.3 Optimization of variant calling pipelines to compile a reliable catalogue of sequence variation</h3>
<p>To obtain a complete and reliable variant set, we selected two mapping algorithms and four frequently used multi-sample VC pipelines to reach maximal sensitivity. Read mapping algorithm BWA-MEM<a reveal-id="dsy033-B31" data-open="dsy033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> was compared to GSNAP,<a reveal-id="dsy033-B32" data-open="dsy033-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> which is able to handle short and long insertions and deletions. Two alignment based VC pipelines were selected for their strength in SNP calling<a reveal-id="dsy033-B50" data-open="dsy033-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a>: GATK Unified Genotyper (GATK UG)<a reveal-id="dsy033-B36" data-open="dsy033-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsy033-B37" data-open="dsy033-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and SAMtools.<a reveal-id="dsy033-B34" data-open="dsy033-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Additionally, two haplotype-based VC pipelines were chosen for their strength in indel detection: GATK HaplotypeCaller (GATK HC) and Freebayes.<a reveal-id="dsy033-B35" data-open="dsy033-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> We compared the resulting variant sets and assessed the performance of hard filtering to improve the precision of variant sets. Finally, the four individual variant sets and corresponding variant quality annotations were merged by the VMC,<a reveal-id="dsy033-B39" data-open="dsy033-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> allowing for precision-based filtering as an alternative to hard filtering.</p>                    <h3 scrollto-destination="131979671" id="131979671" class="section-title" data-legacy-id="SEC3.4">3.4 Influence of read mapping algorithms and variant calling pipelines</h3>
<p>For each VC pipeline, the similarity of SNP and indel sets identified using BWA-MEM or GSNAP mappings was calculated using the Jaccard Index (<a reveal-id="dsy033-F2" data-open="dsy033-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The similarity of SNP<sub>BWA</sub> and SNP<sub>GSNAP</sub> sets was lowest for Freebayes (0.73) and highest for SAMtools (0.83). The similarity of indel<sub>BWA</sub> and indel<sub>GSNAP</sub> sets was lower than that of SNP<sub>BWA</sub> and SNP<sub>GSNAP</sub>, independent of the VC pipeline. Jaccard index between indel<sub>BWA</sub> and indel<sub>GSNAP</sub> sets ranged from 0.47 (Freebayes) to 0.70 (GATK HC). On average, 11% of the SNPs were uniquely identified in the SNP<sub>BWA</sub> set and 9% of the SNPs were uniquely identified in the SNP<sub>GSNAP</sub> set. Likewise, on average, 17% of the indels were uniquely identified in the indel<sub>BWA</sub> set and 19% of the indels were uniquely identified in the indel<sub>GSNAP</sub> set. In summary, the choice of read mapper did not affect the SNP and indel sets as much as the choice of VC pipeline. For results presented below, only variants identified on BWA-MEM mappings are shown, as the same trend was observed for GSNAP mappings. </p>                        <a id="131979673" scrollto-destination="131979673"></a>
<div id="dsy033-F2" data-id="dsy033-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f2.png?Expires=1576700707&amp;Signature=BN65Lfb5lOriv12~CptqLuw19~FsPdqo4c5OTRz02Yn9XxWdSCeAXwpzuFbH2TeuCrPGSah73lpA2y2Rv~ND2EKXC3glDLipBlcNsHdTmbRMpYH4LcYDrhBudcXUBdJyVrlQN3nB81Dl9DoO~tT8s6ogwE~JeRwjFwHWxDB9PNnNnLUv6b-sc9r4-NIze9ehrbRrZTY7h~FxSf1dNsd07sSRX9qu5xfzVzKpJ94l3BoBpic~8O6vprUDmLG8D3WBEoPYkFifUTxdjj1bilDfYf5G~t-bnUiMB-MQRSmpAjFAG6HeAvW~nUthSDRvorTdvFDM7f12K-FyB3qsRbQmpg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overlap of variant sets generated using BWA-MEM and GSNAP mappings as input for four VC pipelines. SNPs and indels were determined for 503 candidate genes in 736 genotypes for BWA-MEM and GSNAP mappings using four VC pipelines. The intersect of variants sets was calculated to determine common variants (dark grey) and uniquely identified variants using BWA-MEM mappings (light grey) or GSNAP mappings (black) as input. The Jaccard index value indicates the corresponding similarity." path-from-xml="dsy033f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979673/dsy033f2.tif" path-from-xml="dsy033f2.tif" target="_blank">Open in new tab</a><a section="131979673" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f2.png?Expires=1576700707&amp;Signature=V64G~nqBVqLntgPptVRstc8WVy3QdsmKTxzHoETuA-pQxeZMdWLMVKWfVTq4K452SfpBpcs88N~XlLwUaHLVFwjlETLk8CK-GXQxirbqXcXe7y-5vWy7CwLsqus9MbF3k99B2y8Om-8AFij0QKHXoLNZJ13mUieE875NJ-L8yJ1gNDpFfS1gnKyiXRn~bpS7oGppWtv0u1BkrnLlcs7QK9Z9o7z1UdA4MF6duYUHLfejhD25dG1ODvkdYnktGDN6IuHGq25y-JwXgWDAf173WZGr6VeMBc~ICMfwKDlTX5btxcMPbycQWEKYlLn-r0gLNcCje5BdLmdUfF5UCd9MQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979673&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Overlap of variant sets generated using BWA-MEM and GSNAP mappings as input for four VC pipelines. SNPs and indels were determined for 503 candidate genes in 736 genotypes for BWA-MEM and GSNAP mappings using four VC pipelines. The intersect of variants sets was calculated to determine common variants (dark grey) and uniquely identified variants using BWA-MEM mappings (light grey) or GSNAP mappings (black) as input. The Jaccard index value indicates the corresponding similarity.</p></div>
</div>
<div content-id="dsy033-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f2.png?Expires=1576700707&amp;Signature=BN65Lfb5lOriv12~CptqLuw19~FsPdqo4c5OTRz02Yn9XxWdSCeAXwpzuFbH2TeuCrPGSah73lpA2y2Rv~ND2EKXC3glDLipBlcNsHdTmbRMpYH4LcYDrhBudcXUBdJyVrlQN3nB81Dl9DoO~tT8s6ogwE~JeRwjFwHWxDB9PNnNnLUv6b-sc9r4-NIze9ehrbRrZTY7h~FxSf1dNsd07sSRX9qu5xfzVzKpJ94l3BoBpic~8O6vprUDmLG8D3WBEoPYkFifUTxdjj1bilDfYf5G~t-bnUiMB-MQRSmpAjFAG6HeAvW~nUthSDRvorTdvFDM7f12K-FyB3qsRbQmpg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overlap of variant sets generated using BWA-MEM and GSNAP mappings as input for four VC pipelines. SNPs and indels were determined for 503 candidate genes in 736 genotypes for BWA-MEM and GSNAP mappings using four VC pipelines. The intersect of variants sets was calculated to determine common variants (dark grey) and uniquely identified variants using BWA-MEM mappings (light grey) or GSNAP mappings (black) as input. The Jaccard index value indicates the corresponding similarity." path-from-xml="dsy033f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979673/dsy033f2.tif" path-from-xml="dsy033f2.tif" target="_blank">Open in new tab</a><a section="131979673" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f2.png?Expires=1576700707&amp;Signature=V64G~nqBVqLntgPptVRstc8WVy3QdsmKTxzHoETuA-pQxeZMdWLMVKWfVTq4K452SfpBpcs88N~XlLwUaHLVFwjlETLk8CK-GXQxirbqXcXe7y-5vWy7CwLsqus9MbF3k99B2y8Om-8AFij0QKHXoLNZJ13mUieE875NJ-L8yJ1gNDpFfS1gnKyiXRn~bpS7oGppWtv0u1BkrnLlcs7QK9Z9o7z1UdA4MF6duYUHLfejhD25dG1ODvkdYnktGDN6IuHGq25y-JwXgWDAf173WZGr6VeMBc~ICMfwKDlTX5btxcMPbycQWEKYlLn-r0gLNcCje5BdLmdUfF5UCd9MQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979673&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Overlap of variant sets generated using BWA-MEM and GSNAP mappings as input for four VC pipelines. SNPs and indels were determined for 503 candidate genes in 736 genotypes for BWA-MEM and GSNAP mappings using four VC pipelines. The intersect of variants sets was calculated to determine common variants (dark grey) and uniquely identified variants using BWA-MEM mappings (light grey) or GSNAP mappings (black) as input. The Jaccard index value indicates the corresponding similarity.</p></div>
</div>                    <h3 scrollto-destination="131979674" id="131979674" class="section-title" data-legacy-id="SEC3.5">3.5 Concordance of variant sets produced by four variant calling pipelines</h3>
<p>Next, the size and concordance of variant sets (bi-allelic SNPs and indels) identified by the four VC pipelines were compared (<a reveal-id="dsy033-F3" data-open="dsy033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The number of SNPs was highest for GATK UG and SAMtools and considerably lower for Freebayes. The number of indels was at least four times lower than the number of SNPs identified by the same VC pipeline. GATK UG and GATK HC identified the highest number of indels, and SAMtools the least. The concordance of all four VC pipelines was low: only 150k SNPs (33% of the total number of SNPs identified) and 6.8k indels (5% of the total number of indels identified) were commonly identified, in line with previous reports.<a reveal-id="dsy033-B14" data-open="dsy033-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p>                        <a id="131979676" scrollto-destination="131979676"></a>
<div id="dsy033-F3" data-id="dsy033-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f3.png?Expires=1576700707&amp;Signature=KEccPAM6Lh01OyS-B8ExcyVn723G-ty3pdkzLV-RcrxY7nLLdg0rzqS6Vna9Lc~WzK5rctFlm0lVPohckAtzqyF-NylQm5cOXRxBmwsIuwEet6vm04~z3JorDJ1ltCujRWeIe64hWUB4z1Aw-IOdgGQT~oIo86e4R~Ioetm1Y3O-4m5JSJJjS1Mg08TZOsnt3jNYNfN9pyMtKIcAtYIrGk1aF6yDAbFdP7cchHHp7fJhKx8eyBXFgNUSuTkIbiEHSVKoIheEibuK6AUARRFkuo8LcKucF-SDNIwEvW1Uozdz4WausvZwC6X63CacuMZ1A6skXTAVgxC2f6bpBYBXJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Size and concordance of bi-allelic SNP and indel sets of four VC pipelines, before and after precision-based filtering. SNPs and indels were identified for 503 candidate genes in 736 genotypes using four VC pipelines and concordance was calculated for bi-allelic SNPs (a) and indels (b). Per Upset plot, the lower left panel shows the total number of variants per VC pipeline; the lower right panel shows the overlap in call sets between the four VC pipelines. The bar graph shows the size per concordance group before (black) and after integration by VMC and precision-based filtering (EP &gt; 80%) (light grey)." path-from-xml="dsy033f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979676/dsy033f3.tif" path-from-xml="dsy033f3.tif" target="_blank">Open in new tab</a><a section="131979676" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f3.png?Expires=1576700707&amp;Signature=KpEWNiCAS574SC56STCWL1pGZipWmBIyRtYQMcYbnT~c8i0vBMNNEhKOkc0AcTTpLmhQA3CIhl4diVEmJ4BLBNtwebZCTcW8bOUqRexeVt3KmCssSvqHJjukVXo8ALFdhJAs3rqalsEn1T4nC55itomUOSqr-U7rsn4CIhkho28MGZFN~A4D1d1HCDWdklX0MkIkr6~i21itmUbFn6YUqSYNIl4zJ7E6ZoFz8jhHDpgREt~FOybOslY78CcifBQvtqzPeop7bG8e4fA5QvlDRsXFRYuCKyKFjXRsNOKjmcLqPLkIzYJ4C9qmAD132mOgQ36UCE~BmdJJkTO4bHsWCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979676&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Size and concordance of bi-allelic SNP and indel sets of four VC pipelines, before and after precision-based filtering. SNPs and indels were identified for 503 candidate genes in 736 genotypes using four VC pipelines and concordance was calculated for bi-allelic SNPs (a) and indels (b). Per Upset plot, the lower left panel shows the total number of variants per VC pipeline; the lower right panel shows the overlap in call sets between the four VC pipelines. The bar graph shows the size per concordance group before (black) and after integration by VMC and precision-based filtering (EP &gt; 80%) (light grey).</p></div>
</div>
<div content-id="dsy033-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f3.png?Expires=1576700707&amp;Signature=KEccPAM6Lh01OyS-B8ExcyVn723G-ty3pdkzLV-RcrxY7nLLdg0rzqS6Vna9Lc~WzK5rctFlm0lVPohckAtzqyF-NylQm5cOXRxBmwsIuwEet6vm04~z3JorDJ1ltCujRWeIe64hWUB4z1Aw-IOdgGQT~oIo86e4R~Ioetm1Y3O-4m5JSJJjS1Mg08TZOsnt3jNYNfN9pyMtKIcAtYIrGk1aF6yDAbFdP7cchHHp7fJhKx8eyBXFgNUSuTkIbiEHSVKoIheEibuK6AUARRFkuo8LcKucF-SDNIwEvW1Uozdz4WausvZwC6X63CacuMZ1A6skXTAVgxC2f6bpBYBXJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Size and concordance of bi-allelic SNP and indel sets of four VC pipelines, before and after precision-based filtering. SNPs and indels were identified for 503 candidate genes in 736 genotypes using four VC pipelines and concordance was calculated for bi-allelic SNPs (a) and indels (b). Per Upset plot, the lower left panel shows the total number of variants per VC pipeline; the lower right panel shows the overlap in call sets between the four VC pipelines. The bar graph shows the size per concordance group before (black) and after integration by VMC and precision-based filtering (EP &gt; 80%) (light grey)." path-from-xml="dsy033f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979676/dsy033f3.tif" path-from-xml="dsy033f3.tif" target="_blank">Open in new tab</a><a section="131979676" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f3.png?Expires=1576700707&amp;Signature=KpEWNiCAS574SC56STCWL1pGZipWmBIyRtYQMcYbnT~c8i0vBMNNEhKOkc0AcTTpLmhQA3CIhl4diVEmJ4BLBNtwebZCTcW8bOUqRexeVt3KmCssSvqHJjukVXo8ALFdhJAs3rqalsEn1T4nC55itomUOSqr-U7rsn4CIhkho28MGZFN~A4D1d1HCDWdklX0MkIkr6~i21itmUbFn6YUqSYNIl4zJ7E6ZoFz8jhHDpgREt~FOybOslY78CcifBQvtqzPeop7bG8e4fA5QvlDRsXFRYuCKyKFjXRsNOKjmcLqPLkIzYJ4C9qmAD132mOgQ36UCE~BmdJJkTO4bHsWCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979676&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Size and concordance of bi-allelic SNP and indel sets of four VC pipelines, before and after precision-based filtering. SNPs and indels were identified for 503 candidate genes in 736 genotypes using four VC pipelines and concordance was calculated for bi-allelic SNPs (a) and indels (b). Per Upset plot, the lower left panel shows the total number of variants per VC pipeline; the lower right panel shows the overlap in call sets between the four VC pipelines. The bar graph shows the size per concordance group before (black) and after integration by VMC and precision-based filtering (EP &gt; 80%) (light grey).</p></div>
</div>                    <h3 scrollto-destination="131979677" id="131979677" class="section-title" data-legacy-id="SEC3.6">3.6 Precision-based filtering is more reliable than hard filtering</h3>
<p>Hard filtering on e.g. minimal RD and GQ is a commonly used strategy to improve the precision of variant sets. As expected, both number of variant positions and call rate (number of genotype calls per position across 736 genotypes) decreased by filtering on minimal RD of six and minimal GQ score of 30 (<a reveal-id="dsy033-F4" data-open="dsy033-F4" class="link link-reveal link-table xref-fig">Fig. 4a and b</a>). Notably, hard filtering did not increase the concordance between VC pipelines (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), indicating that true variants were not necessarily identified by multiple VC pipelines. These results corroborate that it is difficult to build a reliable catalogue of sequence variation using a single VC pipeline and applying hard filtering.<a reveal-id="dsy033-B51" data-open="dsy033-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a></p>                        <a id="131979679" scrollto-destination="131979679"></a>
<div id="dsy033-F4" data-id="dsy033-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f4.png?Expires=1576700707&amp;Signature=npVR24t70WKnmz1970ROZN6LzISHceGlqtk-sD~Ee4DWnebfda1znFW6D7DnxmbFbSKsyB4ZVP8r4lotNWiUV6VOxgP0ANxgy8HU8gFfus1VMe3oPhysbFVMeVkJ8VskokRWuP3ttN0DNRjHj3QK27tDa1oPa3xO7GphFH8yMV2c44xX-VNX~q2ZFGt0fDKYG05gFHC7hBYPXkyU1FQz2mviReJYuc7Asaex-HNviNQX3TTxaM7M1KpTWMBqU4PUq2TJso9DeR3Ngwy~FJ~9SZzZftGnuO-FtR8VYkA6xAWJBfIZ1P5gsGajUPGDiCI53NqcuMMC40E-NenUUSy4dA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of hard filtering and precision-based filtering on the saturation of genotype calls across the 736 genotypes. SNPs and indels were determined for 503 candidate genes in 736 genotypes using four VC pipelines and integrated using the VMC. The genotype call rate was calculated as the number of genotype calls present for each variant, over the total number of genotypes, and plotted cumulatively to estimate the genotype call saturation. This was done for bi-allelic variant sets: (a) before and (b) after hard filtering (RD &gt; 6, GQ &gt; 30) of the variant sets returned by the four VC pipelines and (c) before and (d) after precision-based filtering (EP &gt; 80%) of the VMC output." path-from-xml="dsy033f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979679/dsy033f4.tif" path-from-xml="dsy033f4.tif" target="_blank">Open in new tab</a><a section="131979679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f4.png?Expires=1576700707&amp;Signature=AZALntyAXpG~f4n6kwFcCZanL0idhvmNp0nsycvft4sJnqOAUOYRRJM2L2nPDeTRL2TWjzXGXRqN9gGbY8iDfxo8ygVOXgbBEEsRuDbX22Bds82xjk6qUapMEhpwz7PLOmbMKUMXArn9~SDRq8iBm9xSNBmq9tJSYn9hBAPD~tWz9KnOuPAzJMDYik38S22pb3JNjupEWb8NhRmfRb6Ang~9OgoQlfLdwuF36e4AJcupBS4AHlye-9kN88jMK9MYIJieYjhUXG5XOd80nn6CDCw6u28QoEvA7JdcaJoI8f~Ir5Ma4s9rbZ0a5sRI8cF4~RBypXN~l0QxdWG6mfDIQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979679&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Effect of hard filtering and precision-based filtering on the saturation of genotype calls across the 736 genotypes. SNPs and indels were determined for 503 candidate genes in 736 genotypes using four VC pipelines and integrated using the VMC. The genotype call rate was calculated as the number of genotype calls present for each variant, over the total number of genotypes, and plotted cumulatively to estimate the genotype call saturation. This was done for bi-allelic variant sets: (a) before and (b) after hard filtering (RD &gt; 6, GQ &gt; 30) of the variant sets returned by the four VC pipelines and (c) before and (d) after precision-based filtering (EP &gt; 80%) of the VMC output.</p></div>
</div>
<div content-id="dsy033-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f4.png?Expires=1576700707&amp;Signature=npVR24t70WKnmz1970ROZN6LzISHceGlqtk-sD~Ee4DWnebfda1znFW6D7DnxmbFbSKsyB4ZVP8r4lotNWiUV6VOxgP0ANxgy8HU8gFfus1VMe3oPhysbFVMeVkJ8VskokRWuP3ttN0DNRjHj3QK27tDa1oPa3xO7GphFH8yMV2c44xX-VNX~q2ZFGt0fDKYG05gFHC7hBYPXkyU1FQz2mviReJYuc7Asaex-HNviNQX3TTxaM7M1KpTWMBqU4PUq2TJso9DeR3Ngwy~FJ~9SZzZftGnuO-FtR8VYkA6xAWJBfIZ1P5gsGajUPGDiCI53NqcuMMC40E-NenUUSy4dA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of hard filtering and precision-based filtering on the saturation of genotype calls across the 736 genotypes. SNPs and indels were determined for 503 candidate genes in 736 genotypes using four VC pipelines and integrated using the VMC. The genotype call rate was calculated as the number of genotype calls present for each variant, over the total number of genotypes, and plotted cumulatively to estimate the genotype call saturation. This was done for bi-allelic variant sets: (a) before and (b) after hard filtering (RD &gt; 6, GQ &gt; 30) of the variant sets returned by the four VC pipelines and (c) before and (d) after precision-based filtering (EP &gt; 80%) of the VMC output." path-from-xml="dsy033f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979679/dsy033f4.tif" path-from-xml="dsy033f4.tif" target="_blank">Open in new tab</a><a section="131979679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f4.png?Expires=1576700707&amp;Signature=AZALntyAXpG~f4n6kwFcCZanL0idhvmNp0nsycvft4sJnqOAUOYRRJM2L2nPDeTRL2TWjzXGXRqN9gGbY8iDfxo8ygVOXgbBEEsRuDbX22Bds82xjk6qUapMEhpwz7PLOmbMKUMXArn9~SDRq8iBm9xSNBmq9tJSYn9hBAPD~tWz9KnOuPAzJMDYik38S22pb3JNjupEWb8NhRmfRb6Ang~9OgoQlfLdwuF36e4AJcupBS4AHlye-9kN88jMK9MYIJieYjhUXG5XOd80nn6CDCw6u28QoEvA7JdcaJoI8f~Ir5Ma4s9rbZ0a5sRI8cF4~RBypXN~l0QxdWG6mfDIQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979679&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Effect of hard filtering and precision-based filtering on the saturation of genotype calls across the 736 genotypes. SNPs and indels were determined for 503 candidate genes in 736 genotypes using four VC pipelines and integrated using the VMC. The genotype call rate was calculated as the number of genotype calls present for each variant, over the total number of genotypes, and plotted cumulatively to estimate the genotype call saturation. This was done for bi-allelic variant sets: (a) before and (b) after hard filtering (RD &gt; 6, GQ &gt; 30) of the variant sets returned by the four VC pipelines and (c) before and (d) after precision-based filtering (EP &gt; 80%) of the VMC output.</p></div>
</div>
<p>As an alternative for hard filtering on individual VC pipelines, the VMC<a reveal-id="dsy033-B39" data-open="dsy033-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> uses support vector machines to automatically combine multiple information sources (including RD and GQ values) generated by the four VC pipelines, and estimates the probability that a variant is a true genetic variant and not a sequencing artefact. The unfiltered, VMC integrated probe capture variant set contained 444,222 SNPs and 132,766 indels, determined in 736 genotypes and 503 candidate genes. By ordering the variants according to their probability, an EP was calculated for each variant, which can be used for precision-based filtering. In general, variants identified by multiple VC pipelines were assigned higher EP scores. As precision in this context refers to the number of true called variants, choosing an EP threshold is equivalent to finding a dataset- and aim-specific balance between sensitivity and precision of VC.<a reveal-id="dsy033-B39" data-open="dsy033-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p>                    <h3 scrollto-destination="131979681" id="131979681" class="section-title" data-legacy-id="SEC3.7">3.7 Empirical determination of the EP threshold</h3>
<p>Instead of using an arbitrary EP threshold, we reasoned that the EP threshold should be determined empirically, based on the distributions of EP values of HQ and low quality variants (LQ). In the absence of a published reference set of variants for the genotypes and genes used in this study, we generated an independent variant set for a subset of 78 genotypes using a Hi-Plex amplicon sequencing assay<a reveal-id="dsy033-B52" data-open="dsy033-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> of 171 amplicons, of which 147 overlap with 28 out of the 503 candidate genes. Hi-Plex amplicon sequencing resulted in 126,000 reads per genotype on average (range 11,000–418,000), corresponding to an average RD of 619 reads per amplicon (range 24–20,000).</p>
<p>Using the four VC pipelines integrated by the VMC resulted in a Hi-Plex variant set containing 813 SNPs and 184 indels, compared with 775 SNPs and 246 indels in the probe capture variant set that overlap with these amplicons. In total, 593 SNPs and 60 indels were commonly identified by the two independent sequencing-based genotyping methods. Together, these variants were defined as the HQ subset of variants. Conversely, SNPs and indels that were unique to either set (i.e. non-reproducible and more likely to be random artefacts), were defined as the LQ subset of variants per genotyping method.</p>
<p>To further validate HQ variants, we compared genotype calls of two methods (probe capture vs Hi-Plex) at the individual genotype level. The mean genotype call consistency, calculated as percentage of identical genotype calls on the total of 593 HQ SNPs and 60 HQ indels, over all 78 genotypes was 97% (range 93–100%). This high level of genotype call consistency confirmed the HQ of commonly identified variants. Inconsistent genotype calls are most likely the result of failed probe capture, low RD, the complexity of the region potentially hampering read mapping, allele specific amplification bias in the amplicon sequencing data, or combinations thereof.</p>
<p>Comparison of EP value distributions of HQ and LQ variant positions (<a reveal-id="dsy033-F5" data-open="dsy033-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>) revealed that EP values of the Hi-Plex variant set were generally lower than those of the probe capture variant set, possibly because of higher RD and lower complexity of amplicon reads. Furthermore, EP values associated with HQ SNPs were higher than EP values of LQ SNPs for both Hi-Plex and probe capture SNP sets. Taken together, these data show that an EP threshold of 80% differentiates most HQ variants from LQ variants in the probe capture SNP set, whereas the EP threshold needs to be set at 70% to remove LQ variants from the Hi-Plex SNP set. This further indicates that different EP thresholds ought to be used depending on the genotyping method. In contrast to SNPs, there was no clear differentiation between EP values associated with HQ or LQ indels (<a reveal-id="dsy033-F5" data-open="dsy033-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). This shows that indel detection remains challenging because of mapping and/or realignment errors, and errors near repetitive regions,<a reveal-id="dsy033-B53" data-open="dsy033-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> thus leading to an incomplete indel set and underestimation of frameshift variants. </p>                        <a id="131979686" scrollto-destination="131979686"></a>
<div id="dsy033-F5" data-id="dsy033-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f5.png?Expires=1576700707&amp;Signature=LwQ0lttd67H1le4M312qq-ihqzmhu6vWA-y161INjHrfp-PfPo0O9Lg6WgI7KZoGHZSU1vQ2VP7ru5ehgiD-YkFboEbnfyHUii5LYGwPNlfhLUFQra6pmY9MMrf1VTb4hddcs37YG7boambD9MEde6TPWyGGWgOF9Toz8q4Pt3X4GZgItDf~0RdN~HAx2~GgpISjSFmxnlHDeWObpn3AVKUKWTZuDmBFj5AQCBM5-5BGKTPfsl1afQVin6dL2Ft6nffWKypcv9Pku5GI6ZW4VlbA6WxgNVmG2o6teqSQEuHoBRJtJMxCyJzJS5QmQLGsezaUtAuqAj~QYk8T3RCr3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of EP values in HQ and LQ variant sets. For variants present in 78 genotypes and 147 amplicons, box plots show the distributions of EP values for commonly identified (HQ) and uniquely identified variants (LQ) in the probe capture and Hi-Plex variant sets." path-from-xml="dsy033f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979686/dsy033f5.tif" path-from-xml="dsy033f5.tif" target="_blank">Open in new tab</a><a section="131979686" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f5.png?Expires=1576700707&amp;Signature=CY~LLviqapLi-kPXSvbQXZcsd85cT7ae6tNkaAhHwNSDnw77UM-go0ScSzeBTkFcq5zhJItpxeojbHbNaLbVtewsE0BBkcN4LyYXPrQL0R3T2DTGyGiy6AAaV~0nNOfECdnDX5jGCyXNfXugscpxgP6LKh-T8Gv6y3Gm~Oi2zJrJWqonXSCS565O7-0xkls6nRH3nNXxei2vLLoWYRBpRbVy776cJIKY1nGkO87kG~Bn3ZuRabVy~BLY-2WOGxQ~0jPe0N5edMG-vuWJyl-v26TRxGMo1LfdRUroLr2-CfGiJ2AAAYk3Mh30oXGp0dyTtvEOL7TU4I7DlaTY-aro2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979686&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of EP values in HQ and LQ variant sets. For variants present in 78 genotypes and 147 amplicons, box plots show the distributions of EP values for commonly identified (HQ) and uniquely identified variants (LQ) in the probe capture and Hi-Plex variant sets.</p></div>
</div>
<div content-id="dsy033-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f5.png?Expires=1576700707&amp;Signature=LwQ0lttd67H1le4M312qq-ihqzmhu6vWA-y161INjHrfp-PfPo0O9Lg6WgI7KZoGHZSU1vQ2VP7ru5ehgiD-YkFboEbnfyHUii5LYGwPNlfhLUFQra6pmY9MMrf1VTb4hddcs37YG7boambD9MEde6TPWyGGWgOF9Toz8q4Pt3X4GZgItDf~0RdN~HAx2~GgpISjSFmxnlHDeWObpn3AVKUKWTZuDmBFj5AQCBM5-5BGKTPfsl1afQVin6dL2Ft6nffWKypcv9Pku5GI6ZW4VlbA6WxgNVmG2o6teqSQEuHoBRJtJMxCyJzJS5QmQLGsezaUtAuqAj~QYk8T3RCr3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of EP values in HQ and LQ variant sets. For variants present in 78 genotypes and 147 amplicons, box plots show the distributions of EP values for commonly identified (HQ) and uniquely identified variants (LQ) in the probe capture and Hi-Plex variant sets." path-from-xml="dsy033f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979686/dsy033f5.tif" path-from-xml="dsy033f5.tif" target="_blank">Open in new tab</a><a section="131979686" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f5.png?Expires=1576700707&amp;Signature=CY~LLviqapLi-kPXSvbQXZcsd85cT7ae6tNkaAhHwNSDnw77UM-go0ScSzeBTkFcq5zhJItpxeojbHbNaLbVtewsE0BBkcN4LyYXPrQL0R3T2DTGyGiy6AAaV~0nNOfECdnDX5jGCyXNfXugscpxgP6LKh-T8Gv6y3Gm~Oi2zJrJWqonXSCS565O7-0xkls6nRH3nNXxei2vLLoWYRBpRbVy776cJIKY1nGkO87kG~Bn3ZuRabVy~BLY-2WOGxQ~0jPe0N5edMG-vuWJyl-v26TRxGMo1LfdRUroLr2-CfGiJ2AAAYk3Mh30oXGp0dyTtvEOL7TU4I7DlaTY-aro2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979686&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of EP values in HQ and LQ variant sets. For variants present in 78 genotypes and 147 amplicons, box plots show the distributions of EP values for commonly identified (HQ) and uniquely identified variants (LQ) in the probe capture and Hi-Plex variant sets.</p></div>
</div>
<p>Using the empirically validated EP threshold of 80% for the probe capture variant set containing variants of 736 genotypes and 503 candidate genes resulted in 252,406 SNPs and 5,074 indels (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S3). The high genotype call consistency indicates that the VMC, at least for SNPs, was able to reliably integrate variant sets without losing genotype call quality. Moreover, using the VMC for precision-based filtering led to a higher genotype call rate compared with hard filtering of individual VC pipelines (<a reveal-id="dsy033-F4" data-open="dsy033-F4" class="link link-reveal link-table xref-fig">Fig. 4c and d</a>).</p>                    <h3 scrollto-destination="131979688" id="131979688" class="section-title" data-legacy-id="SEC3.8">3.8 Validation of the resulting variant set in an F1 progeny</h3>
<p>Mendelian inheritance in a segregating F1 progeny derived from a bi-parental cross was used as an accuracy measurement for the precision-based filtered variant set: MIEs are most likely the result of erroneous genotype calls. The set of 736 individuals contained 2 parents and their respective F1 progeny of 29 individuals. The genotype calls of these individuals were used to calculate the number of MIEs. Out of the 257,480 variants, 10,669 contained a missing genotype call in either one or both parents (4%) and could not be tested. For the 246,881 remaining variants and 29 individuals, 89,789 MIEs were identified, of which 57,326 (63%) were due to a missing genotype call. The other 32,463 MIEs represent a genotype call error in only a fraction of all genotype calls among the 246,881 variants in this F1 progeny (&lt;0.5%). Moreover, these MIEs corresponded to 9,440 variant positions (4%) of which most had a MIE in a single individual (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p>                    <h3 scrollto-destination="131979690" id="131979690" class="section-title" data-legacy-id="SEC3.9">3.9 Effects of sequence variation on gene function</h3>
<p>We investigated the consequences of sequence variants on predicted gene function, using the manually curated HQ gene models to annotate the variants with SnpEff.<a reveal-id="dsy033-B41" data-open="dsy033-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> The complete annotation of functional effects for each of 252,406 SNPs and 5,074 indels is available as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S3. Out of the 257,480 variants, 65,225 resided in exon regions (25%) and 116,274 in intron regions (45%) corresponding to a density of 8.6 and 10.1 variants per 100 bp, respectively. Among the SNPs in coding regions, 38% were non-synonymous substitutions, which is consistent with previous observations in <em>L. perenne</em> transcriptomes.<a reveal-id="dsy033-B23 dsy033-B24 dsy033-B25" data-open="dsy033-B23 dsy033-B24 dsy033-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a></p>
<p>A general overview of the abundance of high impact effects on gene function, listed per functional category or pathway is presented in <a reveal-id="dsy033-T1" data-open="dsy033-T1" class="link link-reveal link-table xref-fig">Table 1</a>. These include gain of stop codons, frame shifts and alterations in splice sites, as they are most likely to disrupt protein function, possibly leading to loss of function (LOF), and causing phenotypic variation. For instance, the variant set contained 256 stop gain variants, affecting 144 out of the 503 candidate genes. The position of each stop gain relative to the total CDS length could indicate the degree to which the protein is affected (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700707&amp;Signature=dodVVvuGYWrccErd7D595DhuCkYWc23nWxwAjhDhs7LwrPZokuvOBWR5ARHNytEmLz7Fs7LYa67vfHIGfhbY59HT87J4-CtgQk4PBOxTCWkYXGxSkfy0F5~UXo9UzL-2VUjkCFulfOXpkMC-WGBsAiKqknBOfA~2lLAMAyswXY4mc6XcI3--l7fpw~NGYUerD9berx8qgARGNFpegIvEwGUZ9zDapWm2lHWouaK~gNhnC0h53pJRP1pWkQEohiDZi2ckm344RDnhKNemGf0r6ZvT0A4dg1MqKIOWaoLqcg0aYWgb-5dCBNGqzxCqyZo1kXLH0gIgxunk~PUEExN5Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Most of these stop gain variants occur at low allele frequency across the germplasm collection. Additionally, 72 candidate genes were affected by splice site variants: 40 variants affected donor splice sites and 47 variants affected acceptor splice sites. In line with the relatively low number of indels (5,074), only 20 frameshift variants were identified in 19 genes. In summary, naturally occurring LOF alleles could be readily identified in as much as one-third of the genes tested across various pathways that are important for plant growth and development.</p>
<p>This variant catalogue can be exploited in a dual fashion: (1) to associate genomic variation with phenotypic variation using an association mapping approach, which we are currently performing for architectural traits and cell wall digestibility, or (2) to mine for rare defective alleles, i.e. variants that disrupt gene function or regulation, and to subsequently select carriers of these variants for detailed phenotypic analysis. For example, we observed naturally occurring alleles for the single copy genes <em>GIGANTEA</em> (<em>LpGI-01</em>) and <em>ENHANCED RESPONSE TO ABSCISIC ACID 1</em> (<em>LpERA1-01</em>), in which a premature stop codon truncates translation at 5% and 23% of the protein length, respectively. Crosses with the carriers of these putative null alleles could help to clarify the function of <em>LpGI-01</em> in the regulation of flowering time, circadian clock, and/or hypocotyl elongation<a reveal-id="dsy033-B54" data-open="dsy033-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and <em>LpERA1-01</em> in meristem organization and the ABA-mediated signal transduction pathway.<a reveal-id="dsy033-B55" data-open="dsy033-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p>
<p>Sequence variants were determined in a germplasm collection representing commercial cultivars, breeding populations and wild accessions, so as to ensure the downstream application in current breeding programs. For instance, the 170 amplicons used to estimate the EP threshold, and to validate the genotype calls of the probe capture set, were designed to cover the genetic diversity in 28 genes putatively involved in flowering time and other phenotypic traits of interest to breeders. Design and validation of these amplicons is a clear illustration of the application of the variant set. Since a comprehensive set of SNPs and indels are now known for our breeding materials, detailed and customised design of PCR primers targeting specific SNPs in candidate genes spread across the genome, while avoiding polymorphisms in the flanking primer binding site, becomes feasible. Similar methods and criteria apply to the design of hybridization probes for high density SNP arrays. We are currently using Hi-Plex amplicon sequencing as a very cost- and time-efficient method to screen hundreds of variants simultaneously in a few thousand genotypes, a scale required to screen for putative associations with phenotypic traits in our current breeding populations.</p>                    <h3 scrollto-destination="131979695" id="131979695" class="section-title" data-legacy-id="SEC3.10">3.10 Reconstruction of divergent alleles enables better characterization of genetic variation</h3>
<p>The prime goal of targeted resequencing is to <em>de novo</em> discover alleles that are divergent from the reference genome sequence. However, the capture efficiency of a divergent sequence is reduced with increasing sequence dissimilarity to the reference sequence for which the probes were designed. Additionally, reads may fail to map to highly divergent regions if the parameters for short read alignment are too stringent.<a reveal-id="dsy033-B18" data-open="dsy033-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> To circumvent mapping short reads to a single reference sequence, on which classical VC pipelines rely, we devised a <em>de novo</em> assembly strategy to reconstruct full-length alleles. First, <em>de novo</em> assembly of the captured reads was performed per individual genotype to reconstruct alleles for each of the 503 candidate genes in parallel. Next, all <em>de novo</em> assembled contigs from all 736 genotypes were aligned to the reference genome to sort out and extract all corresponding allelic fragments per candidate gene. Per gene, all contigs were clustered using the OLC assembler CAP3<a reveal-id="dsy033-B44" data-open="dsy033-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> to collapse allelic redundancy and resolve fragmented gene sequences. This three-step approach results in a collection of alternative alleles assigned to each of the 503 candidate genes (on average 58 contigs per gene, range 4–203). The entire set of 29,320 CAP3 contigs is available as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700708&amp;Signature=O8w3eeeX1e0czEczhZmMLtW1s7UGt7JVXt7fA8eL1iIGZJhTwPDWRCBIfjPQwLLTD-Kf4VukrS2i081HurUdv8ecWSVAPelmfGhfxNri~yV9aa8TZmQSr~ZoBMCUx-m8ybCGzGND3GAE85xoaoWAYcD4N-McUqeCPhSQdcFO5t6sh3esFXCDCFSMmZQYh62yhEgohbl0FZMf~p4yLLNV7gVbRhJn3rxf-mNGb8ymoZ8I3S03xCqQesqDxPMZQKM8M9CinvjuGy6BCqAwD-4i~PUlPaFqGbu5kUmynMLtLuQJ8sf~zlNojy5mRo4~H9gYuKDE8efDd92Gt-~2hP2B~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S5. The value of this approach is that we can now characterize genetic variation in regions of high sequence diversity where traditional short read mapping-based VC pipelines fail.</p>
<p>We demonstrate the value of this approach for <em>LpSDUF247</em>, but all analyses described below may be repeated for any of the other 502 candidate genes, using the sequence data supplied as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700708&amp;Signature=O8w3eeeX1e0czEczhZmMLtW1s7UGt7JVXt7fA8eL1iIGZJhTwPDWRCBIfjPQwLLTD-Kf4VukrS2i081HurUdv8ecWSVAPelmfGhfxNri~yV9aa8TZmQSr~ZoBMCUx-m8ybCGzGND3GAE85xoaoWAYcD4N-McUqeCPhSQdcFO5t6sh3esFXCDCFSMmZQYh62yhEgohbl0FZMf~p4yLLNV7gVbRhJn3rxf-mNGb8ymoZ8I3S03xCqQesqDxPMZQKM8M9CinvjuGy6BCqAwD-4i~PUlPaFqGbu5kUmynMLtLuQJ8sf~zlNojy5mRo4~H9gYuKDE8efDd92Gt-~2hP2B~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S5. <em>LpSDUF247</em> is a highly polymorphic gene co-segregating with the <em>S</em>-locus that determines the grass self-incompatibility system.<a reveal-id="dsy033-B46" data-open="dsy033-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> CAP3 clustered the allelic diversity present in the 736 genotypes into 34 separate contigs. Alignment of CAP3 contigs identified a central region of the protein with high sequence divergence, while this region is virtually free from SNPs in the VC dataset, showing the limitations of read mapping-based VC. Six contigs displayed high sequence similarity (98%) to the reference sequence or to other contigs and were removed. Within the remaining allelic contigs, a single exon encoded for the DUF247 protein. The translated proteins showed only 73–84% global sequence identity with each other (<a reveal-id="dsy033-F6" data-open="dsy033-F6" class="link link-reveal link-table xref-fig">Fig. 6a</a>). These data are consistent with the previously reported identification of at least five unique alleles of <em>LpSDUF247</em> with 80–90% protein sequence identity.<a reveal-id="dsy033-B46" data-open="dsy033-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Phylogenetic analysis confirmed that <em>de novo</em> assembled contigs were indeed novel alleles of <em>LpSDUF247</em> at the <em>S</em>-locus, and not of any of the 24 other <em>DUF247 paralogs</em> in the <em>L. perenne</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700708&amp;Signature=O8w3eeeX1e0czEczhZmMLtW1s7UGt7JVXt7fA8eL1iIGZJhTwPDWRCBIfjPQwLLTD-Kf4VukrS2i081HurUdv8ecWSVAPelmfGhfxNri~yV9aa8TZmQSr~ZoBMCUx-m8ybCGzGND3GAE85xoaoWAYcD4N-McUqeCPhSQdcFO5t6sh3esFXCDCFSMmZQYh62yhEgohbl0FZMf~p4yLLNV7gVbRhJn3rxf-mNGb8ymoZ8I3S03xCqQesqDxPMZQKM8M9CinvjuGy6BCqAwD-4i~PUlPaFqGbu5kUmynMLtLuQJ8sf~zlNojy5mRo4~H9gYuKDE8efDd92Gt-~2hP2B~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S5 and S6</a></span>). </p>                        <a id="131979698" scrollto-destination="131979698"></a>
<div id="dsy033-F6" data-id="dsy033-F6" class="fig fig-section">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f6.png?Expires=1576700708&amp;Signature=I1k5lOWvq9pBQn9eBUb9FvU3SyBnNwX45~PndchWA3VeyUDnnOTRA-PwF1Caa9mA011Vi4bhHs~P80X5iEMfVDl6cO08diXYmc933VSajJhYWHVs6RfBlg7NcrVJPWdQgRnSCgr4c5TloRof0VUkJZC9zrUjRwsa~4MsTP-1jqzhwD-F8OXcLpOtgTlv7S0lnPYgMycj5xAvghuYzBDsmTQraVHc-G3NJpaJntSAXda0bSR6egtumg5FQTNRqsuxTlVbwiOF3J9fslv3udJOLy93PjYyA0Vp9CyrVCUnW0WSi6mNXRDEpQZcm6kMStAdyiMBAfucxXZI9PpCXsh0cA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence diversity and distribution of 28 newly identified alleles of LpSDUF247 across breeding populations and wild accessions. A similarity matrix (a) and phylogenetic tree (b) were built using the protein sequences of 28 alleles and reference sequence (R) of LpSDUF247, together with three additional DUF247 genes. Panel c gives an overview of the distribution of the LpSDUF247 alleles across the gene pool. The alleles present per genotype were identified by mapping the reads to a multi-allelic reference genome, and calculating the ratio of average RD per allele over the total number of reads mapping to LpSDUF247 alleles." path-from-xml="dsy033f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979698/dsy033f6.tif" path-from-xml="dsy033f6.tif" target="_blank">Open in new tab</a><a section="131979698" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f6.png?Expires=1576700708&amp;Signature=08RUsX--tta4oFCdygkuMEY-P9ZGkh94-Dt2Gb0V7pru3M8LYTCyJn~l8yK~T8wqLqg9OHPkvjtgtKzwrc4e6xTacIh37oQ5OHwvwfygYHsvr34HBynSm-xQ1KLDt4QydnCk2I55HhQa6dQFBCgdAWPUGT3rY6ShunnPoPh7~y36yrOf60hi9WW4RztHnBd~zGzYWS-43smcX0t9SxVslXlllh4mpPKIMWWFDhOaBNPTca5T96wNGynLFhOup642bgcDldsakyB1KvG2c5zg3hlYFnqgrvJ3s-w32Z6IhwEgQq~yAaLIzTxGtpLjEN5iOUcjr28AxYCHQCYYeM2VFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979698&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Sequence diversity and distribution of 28 newly identified alleles of <em>LpSDUF247</em> across breeding populations and wild accessions. A similarity matrix (a) and phylogenetic tree (b) were built using the protein sequences of 28 alleles and reference sequence (R) of <em>LpSDUF247</em>, together with three additional <em>DUF247</em> genes. Panel c gives an overview of the distribution of the <em>LpSDUF247</em> alleles across the gene pool. The alleles present per genotype were identified by mapping the reads to a multi-allelic reference genome, and calculating the ratio of average RD per allele over the total number of reads mapping to <em>LpSDUF247</em> alleles.</p></div>
</div>
<div content-id="dsy033-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/m_dsy033f6.png?Expires=1576700708&amp;Signature=I1k5lOWvq9pBQn9eBUb9FvU3SyBnNwX45~PndchWA3VeyUDnnOTRA-PwF1Caa9mA011Vi4bhHs~P80X5iEMfVDl6cO08diXYmc933VSajJhYWHVs6RfBlg7NcrVJPWdQgRnSCgr4c5TloRof0VUkJZC9zrUjRwsa~4MsTP-1jqzhwD-F8OXcLpOtgTlv7S0lnPYgMycj5xAvghuYzBDsmTQraVHc-G3NJpaJntSAXda0bSR6egtumg5FQTNRqsuxTlVbwiOF3J9fslv3udJOLy93PjYyA0Vp9CyrVCUnW0WSi6mNXRDEpQZcm6kMStAdyiMBAfucxXZI9PpCXsh0cA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence diversity and distribution of 28 newly identified alleles of LpSDUF247 across breeding populations and wild accessions. A similarity matrix (a) and phylogenetic tree (b) were built using the protein sequences of 28 alleles and reference sequence (R) of LpSDUF247, together with three additional DUF247 genes. Panel c gives an overview of the distribution of the LpSDUF247 alleles across the gene pool. The alleles present per genotype were identified by mapping the reads to a multi-allelic reference genome, and calculating the ratio of average RD per allele over the total number of reads mapping to LpSDUF247 alleles." path-from-xml="dsy033f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979698/dsy033f6.tif" path-from-xml="dsy033f6.tif" target="_blank">Open in new tab</a><a section="131979698" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033f6.png?Expires=1576700708&amp;Signature=08RUsX--tta4oFCdygkuMEY-P9ZGkh94-Dt2Gb0V7pru3M8LYTCyJn~l8yK~T8wqLqg9OHPkvjtgtKzwrc4e6xTacIh37oQ5OHwvwfygYHsvr34HBynSm-xQ1KLDt4QydnCk2I55HhQa6dQFBCgdAWPUGT3rY6ShunnPoPh7~y36yrOf60hi9WW4RztHnBd~zGzYWS-43smcX0t9SxVslXlllh4mpPKIMWWFDhOaBNPTca5T96wNGynLFhOup642bgcDldsakyB1KvG2c5zg3hlYFnqgrvJ3s-w32Z6IhwEgQq~yAaLIzTxGtpLjEN5iOUcjr28AxYCHQCYYeM2VFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979698&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy033f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Sequence diversity and distribution of 28 newly identified alleles of <em>LpSDUF247</em> across breeding populations and wild accessions. A similarity matrix (a) and phylogenetic tree (b) were built using the protein sequences of 28 alleles and reference sequence (R) of <em>LpSDUF247</em>, together with three additional <em>DUF247</em> genes. Panel c gives an overview of the distribution of the <em>LpSDUF247</em> alleles across the gene pool. The alleles present per genotype were identified by mapping the reads to a multi-allelic reference genome, and calculating the ratio of average RD per allele over the total number of reads mapping to <em>LpSDUF247</em> alleles.</p></div>
</div>
<p>Next, we analysed the distribution of <em>LpSDUF247</em> alleles across the <em>L. perenne</em> germplasm collection. The 28 novel alleles were added to the reference genome sequence, thus complementing the reference <em>LpSDUF247</em> allele, and giving reads the opportunity for near-perfect mapping at their respective allele. Differential read mapping <em>across</em> the alleles in a multi-allelic context was then used to score which alleles are present in each genotype. Mapping reads in a multi-allelic context eliminates the need for VC, but only if alleles are sufficiently divergent so that differential RD can be used to identify which alleles are present per genotype. Near-perfect mapping of the raw reads onto the newly constructed <em>LpSDUF247</em> alternative alleles confirmed their existence in the <em>L. perenne</em> germplasm collection, except for allele 31 which had no read support (<a reveal-id="dsy033-F6" data-open="dsy033-F6" class="link link-reveal link-table xref-fig">Fig. 6c</a>). This also shows that the capture efficiency of 120 bp probes was sufficient to detect alleles with as little as 80% sequence identity to the reference genome sequence.</p>
<p>In the vast majority of genotypes (501 out of 736) the reads almost exclusively mapped onto a combination of two <em>LpSDUF247</em> alleles, often at similar RD, and only a minor fraction (&lt;5%) of reads mapped to additional alleles (<a reveal-id="dsy033-F6" data-open="dsy033-F6" class="link link-reveal link-table xref-fig">Fig. 6c</a>). There was no bias for combinations of alleles across wild accessions, breeding populations and cultivars, and clear segregation of alleles was observed in the F1 progeny (<em>n</em> = 29) of a bi-parental cross that was included in the set of 736 individuals (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700708&amp;Signature=O8w3eeeX1e0czEczhZmMLtW1s7UGt7JVXt7fA8eL1iIGZJhTwPDWRCBIfjPQwLLTD-Kf4VukrS2i081HurUdv8ecWSVAPelmfGhfxNri~yV9aa8TZmQSr~ZoBMCUx-m8ybCGzGND3GAE85xoaoWAYcD4N-McUqeCPhSQdcFO5t6sh3esFXCDCFSMmZQYh62yhEgohbl0FZMf~p4yLLNV7gVbRhJn3rxf-mNGb8ymoZ8I3S03xCqQesqDxPMZQKM8M9CinvjuGy6BCqAwD-4i~PUlPaFqGbu5kUmynMLtLuQJ8sf~zlNojy5mRo4~H9gYuKDE8efDd92Gt-~2hP2B~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Furthermore, 57 genotypes displayed reads mapping only to a single allele, suggesting either homozygosity or the failure to capture and sequence yet undiscovered alternative alleles with even stronger sequence divergence to the reference genome sequence used to design the probes. Finally, 177 genotypes displayed RD spread over three or more alternative alleles. In 65 of them, the higher allele count could be explained by a consistent co-segregation of <em>LpSDUF247</em>-04 with <em>LpSDUF247</em>-28 suggesting a gene duplication, in combination with an additional, variable third allele. In the remaining 112 genotypes, the observation that reads map to more than two alleles in a multi-allelic reference genome, could indicate ambiguity of read mapping between closely related alleles, or the presence of additional alleles derived from cross-over events at the <em>LpSDUF247</em> locus.</p>
<p>Although <em>LpSDUF247</em> was the most extreme case of sequence divergence in alternative alleles, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700708&amp;Signature=O8w3eeeX1e0czEczhZmMLtW1s7UGt7JVXt7fA8eL1iIGZJhTwPDWRCBIfjPQwLLTD-Kf4VukrS2i081HurUdv8ecWSVAPelmfGhfxNri~yV9aa8TZmQSr~ZoBMCUx-m8ybCGzGND3GAE85xoaoWAYcD4N-McUqeCPhSQdcFO5t6sh3esFXCDCFSMmZQYh62yhEgohbl0FZMf~p4yLLNV7gVbRhJn3rxf-mNGb8ymoZ8I3S03xCqQesqDxPMZQKM8M9CinvjuGy6BCqAwD-4i~PUlPaFqGbu5kUmynMLtLuQJ8sf~zlNojy5mRo4~H9gYuKDE8efDd92Gt-~2hP2B~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S8</a></span> presents four other candidate genes with different levels of divergence, global or local. The alternative alleles of <em>LpMAX3-01</em> and <em>LpETR1-01</em> showed only local sequence divergence, at the introns and 5′UTR regions, respectively. The sequence variation of <em>LpFT-04</em> was captured in only four contigs, explaining why the variant density across the gene region was low, especially when low frequent SNPs were filtered out. Taken together, the analysis of <em>LpSDUF247</em> demonstrates the rich sequence diversity that can be mined for in this catalogue of genomic diversity across 503 candidate genes.</p>                    <h2 scrollto-destination="131979702" id="131979702" class="backacknowledgements-title" data-legacy-id="Ack1">Acknowledgements</h2>
<p>We thank the breeders of ILVO, Barenbrug and Eurograss for providing plant material, András Gézsi for support and advice for running the VMC and Kurt Lamour (Floodlight Genomics) for his services provided for the Hi-Plex amplicon sequencing assay. The computational resources (Stevin Supercomputer Infrastructure) and services used in this work were provided by the VSC (Flemish Supercomputer Center), funded by Ghent University, FWO and the Flemish Government—department EWI. This research was supported by the Agency for the promotion of Innovation by Science and Technology (IWT) in Flanders, Belgium (project LO-080510), and by Aarhus University and the Godfred Birkedal Hartmann foundation in Denmark. Elisabeth Veeckman was supported by an ILVO PhD fellowship.</p>                    <h3 scrollto-destination="131979704" id="131979704" class="section-title">Accession numbers</h3>
<p>SRR accession numbers: SRR6812717–SRR6813075, SRR6813540–SRR6813585.</p>                    <h3 scrollto-destination="131979706" id="131979706" class="section-title">Funding</h3>
<p>Agency for the promotion of Innovation by Science and Technology (IWT) project LO-080510, Aarhus University, Godfred Birkedal Hartmann foundation.</p>                    <h3 scrollto-destination="131979708" id="131979708" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="131979710" id="131979710" class="backreferences-title" data-legacy-id="Ref1">References</h2>
<div class="ref-list">
<div content-id="dsy033-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Roldán-Ruiz</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Kölliker</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2010</div>, Marker-assisted selection in forage crops and turf: a review. <div class="source">In: Sustainable Use of Genetic Diversity in Forage and Turf Breeding</div>  (pp. <div class="fpage">383</div>–<div class="lpage">390</div>), ed. <div class="person-group"><div class="name string-name">
<div class="surname">Huyghe</div> <div class="given-names">C.</div>
</div></div> 383–390. 
<div class="publisher-loc">Berlin, Germany</div>: 
<div class="publisher-name">Springer</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=In%3A%20Sustainable%20Use%20of%20Genetic%20Diversity%20in%20Forage%20and%20Turf%20Breeding&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=In%3A%20Sustainable%20Use%20of%20Genetic%20Diversity%20in%20Forage%20and%20Turf%20Breeding&amp;aulast=Rold%C3%A1n-Ruiz&amp;date=2010&amp;spage=383&amp;epage=390" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:In%3A%20Sustainable%20Use%20of%20Genetic%20Diversity%20in%20Forage%20and%20Turf%20Breeding&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=In%3A%20Sustainable%20Use%20of%20Genetic%20Diversity%20in%20Forage%20and%20Turf%20Breeding">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Sauchyn</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y. F.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Drought changes and the mechanism analysis for the North American Prairie</div>, <div class="source">J. Arid Land</div> , <div class="volume">5</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Drought%20changes%20and%20the%20mechanism%20analysis%20for%20the%20North%20American%20Prairie&amp;author=G.%20Yu&amp;author=D.%20Sauchyn&amp;author=Y.%20F.%20Li&amp;publication_year=2013&amp;journal=J.%20Arid%20Land&amp;volume=5&amp;pages=1-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s40333-013-0136-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs40333-013-0136-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs40333-013-0136-4"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Drought%20changes%20and%20the%20mechanism%20analysis%20for%20the%20North%20American%20Prairie&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Brazauskas</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Pašakinskienė</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Asp</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Lübberstedt</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Nucleotide diversity and linkage disequilibrium in five <em>Lolium perenne</em> genes with putative role in shoot morphology</div>, <div class="source">Plant Sci</div> , <div class="volume">179</div>, <div class="fpage">194</div>–<div class="lpage">201</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20diversity%20and%20linkage%20disequilibrium%20in%20five%20Lolium%20perenne%20genes%20with%20putative%20role%20in%20shoot%20morphology&amp;author=G.%20Brazauskas&amp;author=I.%20Pa%C5%A1akinskien%C4%97&amp;author=T.%20Asp&amp;author=T.%20L%C3%BCbberstedt&amp;publication_year=2010&amp;journal=Plant%20Sci&amp;volume=179&amp;pages=194-201" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.plantsci.2010.04.016" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2010.04.016" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.plantsci.2010.04.016"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nucleotide%20diversity%20and%20linkage%20disequilibrium%20in%20five%20Lolium%20perenne%20genes%20with%20putative%20role%20in%20shoot%20morphology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Auzanneau</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Huyghe</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Julier</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Barre</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Linkage disequilibrium in synthetic varieties of perennial ryegrass</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">115</div>, <div class="fpage">837</div>–<div class="lpage">47</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%20in%20synthetic%20varieties%20of%20perennial%20ryegrass&amp;author=J.%20Auzanneau&amp;author=C.%20Huyghe&amp;author=B.%20Julier&amp;author=P.%20Barre&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet&amp;volume=115&amp;pages=837-47" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00122-007-0612-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0612-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00122-007-0612-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17701396" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Linkage%20disequilibrium%20in%20synthetic%20varieties%20of%20perennial%20ryegrass&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Skot</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Humphreys</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Humphreys</div> <div class="given-names">M. O.</div>
</div>
</div> et al.  <div class="year">2007</div>, 
<div class="article-title">Association of candidate genes with flowering time and water-soluble carbohydrate content in <em>Lolium perenne</em> (L.)</div>, <div class="source">Genetics</div> , <div class="volume">177</div>, <div class="fpage">535</div>–<div class="lpage">47</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Association%20of%20candidate%20genes%20with%20flowering%20time%20and%20water-soluble%20carbohydrate%20content%20in%20Lolium%20perenne%20(L.)&amp;author=L.%20Skot&amp;author=J.%20Humphreys&amp;author=M.%20O.%20Humphreys&amp;publication_year=2007&amp;journal=Genetics&amp;volume=177&amp;pages=535-47" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.107.071522" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.071522" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.107.071522"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17660575" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Association%20of%20candidate%20genes%20with%20flowering%20time%20and%20water-soluble%20carbohydrate%20content%20in%20Lolium%20perenne%20(L.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fiil</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Lenk</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Petersen</div> <div class="given-names">K.</div>
</div>
</div> et al.  <div class="year">2011</div>, 
<div class="article-title">Nucleotide diversity and linkage disequilibrium of nine genes with putative effects on flowering time in perennial ryegrass (<em>Lolium perenne</em> L.)</div>, <div class="source">Plant Sci</div> , <div class="volume">180</div>, <div class="fpage">228</div>–<div class="lpage">37</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20diversity%20and%20linkage%20disequilibrium%20of%20nine%20genes%20with%20putative%20effects%20on%20flowering%20time%20in%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;author=A.%20Fiil&amp;author=I.%20Lenk&amp;author=K.%20Petersen&amp;publication_year=2011&amp;journal=Plant%20Sci&amp;volume=180&amp;pages=228-37" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.plantsci.2010.08.015" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2010.08.015" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.plantsci.2010.08.015"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21421365" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nucleotide%20diversity%20and%20linkage%20disequilibrium%20of%20nine%20genes%20with%20putative%20effects%20on%20flowering%20time%20in%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Skot</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Sanderson</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Thomas</div> <div class="given-names">A.</div>
</div>
</div> et al.  <div class="year">2011</div>, 
<div class="article-title">Allelic variation in the perennial ryegrass flowering locus T gene is associated with changes in flowering time across a range of populations</div>, <div class="source">Plant Physiol</div> , <div class="volume">155</div>, <div class="fpage">1013</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Allelic%20variation%20in%20the%20perennial%20ryegrass%20flowering%20locus%20T%20gene%20is%20associated%20with%20changes%20in%20flowering%20time%20across%20a%20range%20of%20populations&amp;author=L.%20Skot&amp;author=R.%20Sanderson&amp;author=A.%20Thomas&amp;publication_year=2011&amp;journal=Plant%20Physiol&amp;volume=155&amp;pages=1013-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.110.169870" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.169870" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.110.169870"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21115808" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Allelic%20variation%20in%20the%20perennial%20ryegrass%20flowering%20locus%20T%20gene%20is%20associated%20with%20changes%20in%20flowering%20time%20across%20a%20range%20of%20populations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Studer</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Byrne</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Nielsen</div> <div class="given-names">R. O.</div>
</div>
</div> et al.  <div class="year">2012</div>, 
<div class="article-title">A transcriptome map of perennial ryegrass (<em>Lolium perenne</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">140</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20transcriptome%20map%20of%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;author=B.%20Studer&amp;author=S.%20Byrne&amp;author=R.%20O.%20Nielsen&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=140" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-13-140" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-140" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-13-140"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22513206" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20transcriptome%20map%20of%20perennial%20ryegrass%20(Lolium%20perenne%20L.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Blackmore</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Thomas</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">McMahon</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Powell</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Hegarty</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Genetic-geographic correlation revealed across a broad European ecotypic sample of perennial ryegrass (<em>Lolium perenne</em>) using array-based SNP genotyping</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">128</div>, <div class="fpage">1917</div>–<div class="lpage">32</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetic-geographic%20correlation%20revealed%20across%20a%20broad%20European%20ecotypic%20sample%20of%20perennial%20ryegrass%20(Lolium%20perenne)%20using%20array-based%20SNP%20genotyping&amp;author=T.%20Blackmore&amp;author=I.%20Thomas&amp;author=R.%20McMahon&amp;author=W.%20Powell&amp;author=M.%20Hegarty&amp;publication_year=2015&amp;journal=Theor.%20Appl.%20Genet&amp;volume=128&amp;pages=1917-32" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00122-015-2556-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-015-2556-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00122-015-2556-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26093611" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetic-geographic%20correlation%20revealed%20across%20a%20broad%20European%20ecotypic%20sample%20of%20perennial%20ryegrass%20(Lolium%20perenne)%20using%20array-based%20SNP%20genotyping&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chung</div> <div class="given-names">Y. S.</div>
</div>, <div class="name string-name">
<div class="surname">Choi</div> <div class="given-names">S. C.</div>
</div>, <div class="name string-name">
<div class="surname">Jun</div> <div class="given-names">T. H.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Genotyping-by-sequencing: a promising tool for plant genetics research and breeding</div>, <div class="source">Hortic. Environ. Biotechnol</div> ., <div class="volume">58</div>, <div class="fpage">425</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%3A%20a%20promising%20tool%20for%20plant%20genetics%20research%20and%20breeding&amp;author=Y.%20S.%20Chung&amp;author=S.%20C.%20Choi&amp;author=T.%20H.%20Jun&amp;author=C.%20Kim&amp;publication_year=2017&amp;journal=Hortic.%20Environ.%20Biotechnol&amp;volume=58&amp;pages=425-31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s13580-017-0297-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13580-017-0297-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs13580-017-0297-8"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genotyping-by-sequencing%3A%20a%20promising%20tool%20for%20plant%20genetics%20research%20and%20breeding&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xing</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Frei</div> <div class="given-names">U.</div>
</div>, <div class="name string-name">
<div class="surname">Schejbel</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Asp</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Lubberstedt</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Nucleotide diversity and linkage disequilibrium in 11 expressed resistance candidate genes in <em>Lolium perenne</em>
</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">7</div>, <div class="fpage">43</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20diversity%20and%20linkage%20disequilibrium%20in%2011%20expressed%20resistance%20candidate%20genes%20in%20Lolium%20perenne&amp;author=Y.%20Xing&amp;author=U.%20Frei&amp;author=B.%20Schejbel&amp;author=T.%20Asp&amp;author=T.%20Lubberstedt&amp;publication_year=2007&amp;journal=BMC%20Plant%20Biol&amp;volume=7&amp;pages=43" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2229-7-43" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-7-43" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2229-7-43"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17683574" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nucleotide%20diversity%20and%20linkage%20disequilibrium%20in%2011%20expressed%20resistance%20candidate%20genes%20in%20Lolium%20perenne&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Byrne</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Nagy</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Pfeifer</div> <div class="given-names">M.</div>
</div>
</div> et al.  <div class="year">2015</div>, 
<div class="article-title">A synteny-based draft genome sequence of the forage grass <em>Lolium perenne</em>
</div>, <div class="source">Plant J</div> ., <div class="volume">84</div>, <div class="fpage">816</div>–<div class="lpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20synteny-based%20draft%20genome%20sequence%20of%20the%20forage%20grass%20Lolium%20perenne&amp;author=S.%20L.%20Byrne&amp;author=I.%20Nagy&amp;author=M.%20Pfeifer&amp;publication_year=2015&amp;journal=Plant%20J&amp;volume=84&amp;pages=816-26" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.13037" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.13037" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.13037"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26408275" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20synteny-based%20draft%20genome%20sequence%20of%20the%20forage%20grass%20Lolium%20perenne&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Han</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Gelernter</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">B.-Z.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Variant callers for next-generation sequencing data: a comparison study</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e75619</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Variant%20callers%20for%20next-generation%20sequencing%20data%3A%20a%20comparison%20study&amp;author=X.%20Liu&amp;author=S.%20Han&amp;author=Z.%20Wang&amp;author=J.%20Gelernter&amp;author=B.-Z.%20Yang&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e75619" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0075619" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0075619" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0075619"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24086590" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Variant%20callers%20for%20next-generation%20sequencing%20data%3A%20a%20comparison%20study&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">O'Rawe</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">G.</div>
</div>
</div> et al.  <div class="year">2013</div>, 
<div class="article-title">Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing</div>, <div class="source">Genome Med</div> ., <div class="volume">5</div>, <div class="fpage">28</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Low%20concordance%20of%20multiple%20variant-calling%20pipelines%3A%20practical%20implications%20for%20exome%20and%20genome%20sequencing&amp;author=J.%20O'Rawe&amp;author=T.%20Jiang&amp;author=G.%20Sun&amp;publication_year=2013&amp;journal=Genome%20Med&amp;volume=5&amp;pages=28" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gm432" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgm432" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgm432"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23537139" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Low%20concordance%20of%20multiple%20variant-calling%20pipelines%3A%20practical%20implications%20for%20exome%20and%20genome%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pirooznia</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Kramer</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Parla</div> <div class="given-names">J.</div>
</div>
</div> et al.  <div class="year">2014</div>, 
<div class="article-title">Validation and assessment of variant calling pipelines for next-generation sequencing</div>, <div class="source">Hum. Genomics</div> , <div class="volume">8</div>, <div class="fpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Validation%20and%20assessment%20of%20variant%20calling%20pipelines%20for%20next-generation%20sequencing&amp;author=M.%20Pirooznia&amp;author=M.%20Kramer&amp;author=J.%20Parla&amp;publication_year=2014&amp;journal=Hum.%20Genomics&amp;volume=8&amp;pages=14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1479-7364-8-14" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1479-7364-8-14" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1479-7364-8-14"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25078893" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Validation%20and%20assessment%20of%20variant%20calling%20pipelines%20for%20next-generation%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">X. Q.</div>
</div>, <div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">S. Y.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Comparing a few SNP calling algorithms using low-coverage sequencing data</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">274</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparing%20a%20few%20SNP%20calling%20algorithms%20using%20low-coverage%20sequencing%20data&amp;author=X.%20Q.%20Yu&amp;author=S.%20Y.%20Sun&amp;publication_year=2013&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=274" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-14-274" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-274" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-14-274"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24044377" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparing%20a%20few%20SNP%20calling%20algorithms%20using%20low-coverage%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Quail</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Coupland</div> <div class="given-names">P.</div>
</div>
</div> et al.  <div class="year">2012</div>, 
<div class="article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">341</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20tale%20of%20three%20next%20generation%20sequencing%20platforms%3A%20comparison%20of%20Ion%20Torrent%2C%20Pacific%20Biosciences%20and%20Illumina%20MiSeq%20sequencers&amp;author=M.%20A.%20Quail&amp;author=M.%20Smith&amp;author=P.%20Coupland&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=341" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-13-341" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-341" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-13-341"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22827831" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20tale%20of%20three%20next%20generation%20sequencing%20platforms%3A%20comparison%20of%20Ion%20Torrent%2C%20Pacific%20Biosciences%20and%20Illumina%20MiSeq%20sequencers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bertels</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Silander</div> <div class="given-names">O. K.</div>
</div>, <div class="name string-name">
<div class="surname">Pachkov</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Rainey</div> <div class="given-names">P. B.</div>
</div>, <div class="name string-name">
<div class="surname">van Nimwegen</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Automated reconstruction of whole-genome phylogenies from short-sequence reads</div>, <div class="source">Mol Biol Evol</div> , <div class="volume">31</div>, <div class="fpage">1077</div>–<div class="lpage">88</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Automated%20reconstruction%20of%20whole-genome%20phylogenies%20from%20short-sequence%20reads&amp;author=F.%20Bertels&amp;author=O.%20K.%20Silander&amp;author=M.%20Pachkov&amp;author=P.%20B.%20Rainey&amp;author=E.%20van%20Nimwegen&amp;publication_year=2014&amp;journal=Mol%20Biol%20Evol&amp;volume=31&amp;pages=1077-88" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msu088" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu088" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsu088"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24600054" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Automated%20reconstruction%20of%20whole-genome%20phylogenies%20from%20short-sequence%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gan</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Stegle</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Behr</div> <div class="given-names">J.</div>
</div>
</div> et al.  <div class="year">2011</div>, 
<div class="article-title">Multiple reference genomes and transcriptomes for <em>Arabidopsis thaliana</em>
</div>, <div class="source">Nature</div> , <div class="volume">477</div>, <div class="fpage">419</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Multiple%20reference%20genomes%20and%20transcriptomes%20for%20Arabidopsis%20thaliana&amp;author=X.%20Gan&amp;author=O.%20Stegle&amp;author=J.%20Behr&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=419" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature10414" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10414" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature10414"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874022" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Multiple%20reference%20genomes%20and%20transcriptomes%20for%20Arabidopsis%20thaliana&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Proost</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Van Bel</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Vaneechoutte</div> <div class="given-names">D.</div>
</div>
</div> et al.  <div class="year">2015</div>, 
<div class="article-title">PLAZA 3.0: an access point for plant comparative genomics</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">43</div>, <div class="fpage">D974</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PLAZA%203.0%3A%20an%20access%20point%20for%20plant%20comparative%20genomics&amp;author=S.%20Proost&amp;author=M.%20Van%20Bel&amp;author=D.%20Vaneechoutte&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D974-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gku986" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku986" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgku986"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25324309" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PLAZA%203.0%3A%20an%20access%20point%20for%20plant%20comparative%20genomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Edgar</div> <div class="given-names">R. C.</div>
</div></div> <div class="year">2004</div>, 
<div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">32</div>, <div class="fpage">1792</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.%20C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkh340"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Guindon</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Dufayard</div> <div class="given-names">J. F.</div>
</div>, <div class="name string-name">
<div class="surname">Lefort</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Anisimova</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Hordijk</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Gascuel</div> <div class="given-names">O.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</div>, <div class="source">Syst Biol</div> , <div class="volume">59</div>, <div class="fpage">307</div>–<div class="lpage">21</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=J.%20F.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Syst%20Biol&amp;volume=59&amp;pages=307-21" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syq010" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq010" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq010"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Farrell</div> <div class="given-names">J. D.</div>
</div>, <div class="name string-name">
<div class="surname">Byrne</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Paina</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Asp</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">De novo assembly of the perennial ryegrass transcriptome using an RNA-Seq strategy</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e103567</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20the%20perennial%20ryegrass%20transcriptome%20using%20an%20RNA-Seq%20strategy&amp;author=J.%20D.%20Farrell&amp;author=S.%20Byrne&amp;author=C.%20Paina&amp;author=T.%20Asp&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e103567" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0103567" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0103567" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0103567"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25126744" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:De%20novo%20assembly%20of%20the%20perennial%20ryegrass%20transcriptome%20using%20an%20RNA-Seq%20strategy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Paina</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Byrne</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Domnisoru</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Asp</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Vernalization mediated changes in the <em>Lolium perenne</em> transcriptome</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e107365</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Vernalization%20mediated%20changes%20in%20the%20Lolium%20perenne%20transcriptome&amp;author=C.%20Paina&amp;author=S.%20L.%20Byrne&amp;author=C.%20Domnisoru&amp;author=T.%20Asp&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e107365" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0107365" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0107365" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0107365"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25225807" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Vernalization%20mediated%20changes%20in%20the%20Lolium%20perenne%20transcriptome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ruttink</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Sterck</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Rohde</div> <div class="given-names">A.</div>
</div>
</div> et al.  <div class="year">2013</div>, 
<div class="article-title">Orthology guided assembly in highly heterozygous crops: creating a reference transcriptome to uncover genetic diversity in <em>Lolium perenne</em>
</div>, <div class="source">Plant Biotechnol. J</div> ., <div class="volume">11</div>, <div class="fpage">605</div>–<div class="lpage">17</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Orthology%20guided%20assembly%20in%20highly%20heterozygous%20crops%3A%20creating%20a%20reference%20transcriptome%20to%20uncover%20genetic%20diversity%20in%20Lolium%20perenne&amp;author=T.%20Ruttink&amp;author=L.%20Sterck&amp;author=A.%20Rohde&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.%20J&amp;volume=11&amp;pages=605-17" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/pbi.2013.11.issue-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2013.11.issue-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fpbi.2013.11.issue-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23433242" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Orthology%20guided%20assembly%20in%20highly%20heterozygous%20crops%3A%20creating%20a%20reference%20transcriptome%20to%20uncover%20genetic%20diversity%20in%20Lolium%20perenne&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Trapnell</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Pachter</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S. L.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.%20L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp120"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bertone</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Trifonov</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Rozowsky</div> <div class="given-names">J. S.</div>
</div>
</div> et al.  <div class="year">2006</div>, 
<div class="article-title">Design optimization methods for genomic DNA tiling arrays</div>, <div class="source">Genome Res</div> ., <div class="volume">16</div>, <div class="fpage">271</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Design%20optimization%20methods%20for%20genomic%20DNA%20tiling%20arrays&amp;author=P.%20Bertone&amp;author=V.%20Trifonov&amp;author=J.%20S.%20Rozowsky&amp;publication_year=2006&amp;journal=Genome%20Res&amp;volume=16&amp;pages=271-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.4452906" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4452906" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.4452906"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16365382" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Design%20optimization%20methods%20for%20genomic%20DNA%20tiling%20arrays&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Murray</div> <div class="given-names">M. G.</div>
</div>, <div class="name string-name">
<div class="surname">Thompson</div> <div class="given-names">W. F.</div>
</div>
</div> <div class="year">1980</div>, 
<div class="article-title">Rapid isolation of high molecular-weight plant DNA</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">8</div>, <div class="fpage">4321</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rapid%20isolation%20of%20high%20molecular-weight%20plant%20DNA&amp;author=M.%20G.%20Murray&amp;author=W.%20F.%20Thompson&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res&amp;volume=8&amp;pages=4321-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/8.19.4321" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.19.4321" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F8.19.4321"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433111" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rapid%20isolation%20of%20high%20molecular-weight%20plant%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Uitdewilligen</div> <div class="given-names">J. G. A. M. L.</div>
</div>, <div class="name string-name">
<div class="surname">Wolters</div> <div class="given-names">A.-M. A.</div>
</div>, <div class="name string-name">
<div class="surname">D’hoop</div> <div class="given-names">B. B.</div>
</div>, <div class="name string-name">
<div class="surname">Borm</div> <div class="given-names">T. J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Visser</div> <div class="given-names">R. G. F.</div>
</div>, <div class="name string-name">
<div class="surname">van Eck</div> <div class="given-names">H. J.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e62355</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20next-generation%20sequencing%20method%20for%20genotyping-by-sequencing%20of%20highly%20heterozygous%20autotetraploid%20potato&amp;author=J.%20G.%20A.%20M.%20L.%20Uitdewilligen&amp;author=A.-M.%20A.%20Wolters&amp;author=B.%20B.%20D%E2%80%99hoop&amp;author=T.%20J.%20A.%20Borm&amp;author=R.%20G.%20F.%20Visser&amp;author=H.%20J.%20van%20Eck&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e62355" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0062355" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0062355" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0062355"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23667470" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20next-generation%20sequencing%20method%20for%20genotyping-by-sequencing%20of%20highly%20heterozygous%20autotetraploid%20potato&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bolger</div> <div class="given-names">A. M.</div>
</div>, <div class="name string-name">
<div class="surname">Lohse</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Usadel</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu170"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Durbin</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp324"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">T. D.</div>
</div>, <div class="name string-name">
<div class="surname">Nacu</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">873</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20SNP-tolerant%20detection%20of%20complex%20variants%20and%20splicing%20in%20short%20reads&amp;author=T.%20D.%20Wu&amp;author=S.%20Nacu&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=873-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq057" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq057" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq057"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20147302" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20and%20SNP-tolerant%20detection%20of%20complex%20variants%20and%20splicing%20in%20short%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Quinlan</div> <div class="given-names">A. R.</div>
</div>, <div class="name string-name">
<div class="surname">Hall</div> <div class="given-names">I. M.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">841</div>–<div class="lpage">2</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.%20R.%20Quinlan&amp;author=I.%20M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq033"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Handsaker</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Wysoker</div> <div class="given-names">A.</div>
</div>
</div> et al.  <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp352"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garrison</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Marth</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Haplotype-based variant detection from short-read sequencing</div>, <div class="source">arXiv Preprint arXiv</div> , <div class="volume">1207</div>, <div class="fpage">3907.</div>
<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Haplotype-based%20variant%20detection%20from%20short-read%20sequencing&amp;aulast=Garrison&amp;date=2012&amp;spage=3907.&amp;volume=1207" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Haplotype-based%20variant%20detection%20from%20short-read%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">McKenna</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Hanna</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Banks</div> <div class="given-names">E.</div>
</div>
</div> et al.  <div class="year">2010</div>, 
<div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">303</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.107524.110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Van der Auwera</div> <div class="given-names">G. A.</div>
</div>, <div class="name string-name">
<div class="surname">Carneiro</div> <div class="given-names">M. O.</div>
</div>, <div class="name string-name">
<div class="surname">Hartl</div> <div class="given-names">C.</div>
</div>
</div> et al.  <div class="year">2013</div>, 
<div class="article-title">From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline</div>, <div class="source">Curr Protoc Bioinformatics</div> , <div class="volume">43</div>, 11 10–33.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=From%20FastQ%20data%20to%20high%20confidence%20variant%20calls%3A%20the%20Genome%20Analysis%20Toolkit%20best%20practices%20pipeline&amp;aulast=Van%20der%20Auwera&amp;date=2013&amp;volume=43" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:From%20FastQ%20data%20to%20high%20confidence%20variant%20calls%3A%20the%20Genome%20Analysis%20Toolkit%20best%20practices%20pipeline&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Danecek</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Auton</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Abecasis</div> <div class="given-names">G.</div>
</div>
</div> et al.  <div class="year">2011</div>, 
<div class="article-title">The variant call format and VCFtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2156</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20variant%20call%20format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr330"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20variant%20call%20format%20and%20VCFtools&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gézsi</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Bolgár</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Marx</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Sarkozy</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Szalai</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Antal</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">VariantMetaCaller: automated fusion of variant calling pipelines for quantitative, precision-based filtering</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">875</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=VariantMetaCaller%3A%20automated%20fusion%20of%20variant%20calling%20pipelines%20for%20quantitative%2C%20precision-based%20filtering&amp;author=A.%20G%C3%A9zsi&amp;author=B.%20Bolg%C3%A1r&amp;author=P.%20Marx&amp;author=P.%20Sarkozy&amp;author=C.%20Szalai&amp;author=P.%20Antal&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=875" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-015-2050-y" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2050-y" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-015-2050-y"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26510841" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:VariantMetaCaller%3A%20automated%20fusion%20of%20variant%20calling%20pipelines%20for%20quantitative%2C%20precision-based%20filtering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lex</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Gehlenborg</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Strobelt</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Vuillemot</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Pfister</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">UpSet: visualization of Intersecting Sets</div>, <div class="source">IEEE Trans. Vis. Comput. Graph</div> ., <div class="volume">20</div>, <div class="fpage">1983</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=UpSet%3A%20visualization%20of%20Intersecting%20Sets&amp;author=A.%20Lex&amp;author=N.%20Gehlenborg&amp;author=H.%20Strobelt&amp;author=R.%20Vuillemot&amp;author=H.%20Pfister&amp;publication_year=2014&amp;journal=IEEE%20Trans.%20Vis.%20Comput.%20Graph&amp;volume=20&amp;pages=1983-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1109/TVCG.2014.2346248" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTVCG.2014.2346248" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1109%2FTVCG.2014.2346248"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26356912" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:UpSet%3A%20visualization%20of%20Intersecting%20Sets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cingolani</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Platts</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Wang le</div> <div class="given-names">L.</div>
</div>
</div> et al.  <div class="year">2012</div>, 
<div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3</div>, <div class="source">Fly (Austin)</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.%20Wang%20le&amp;publication_year=2012&amp;journal=Fly%20(Austin)&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.4161%2Ffly.19695"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Untergasser</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Cutcutache</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Koressaar</div> <div class="given-names">T.</div>
</div>
</div> et al.  <div class="year">2012</div>, 
<div class="article-title">Primer3-new capabilities and interfaces</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e115</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Primer3-new%20capabilities%20and%20interfaces&amp;author=A.%20Untergasser&amp;author=I.%20Cutcutache&amp;author=T.%20Koressaar&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e115" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks596" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks596" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks596"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22730293" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Primer3-new%20capabilities%20and%20interfaces&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">J. J.</div>
</div>, <div class="name string-name">
<div class="surname">Kobert</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Flouri</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Stamatakis</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">PEAR: a fast and accurate Illumina Paired-End reAd mergeR</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">614</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PEAR%3A%20a%20fast%20and%20accurate%20Illumina%20Paired-End%20reAd%20mergeR&amp;author=J.%20J.%20Zhang&amp;author=K.%20Kobert&amp;author=T.%20Flouri&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=614-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt593" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt593" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt593"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24142950" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PEAR%3A%20a%20fast%20and%20accurate%20Illumina%20Paired-End%20reAd%20mergeR&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Huang</div> <div class="given-names">X. Q.</div>
</div>, <div class="name string-name">
<div class="surname">Madan</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">1999</div>, 
<div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res</div> ., <div class="volume">9</div>, <div class="fpage">868</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Q.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.9.9.868"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gremme</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Brendel</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Sparks</div> <div class="given-names">M. E.</div>
</div>, <div class="name string-name">
<div class="surname">Kurtz</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Engineering a software tool for gene structure prediction in higher organisms</div>, <div class="source">Inform Software Tech</div> , <div class="volume">47</div>, <div class="fpage">965</div>–<div class="lpage">78</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Engineering%20a%20software%20tool%20for%20gene%20structure%20prediction%20in%20higher%20organisms&amp;author=G.%20Gremme&amp;author=V.%20Brendel&amp;author=M.%20E.%20Sparks&amp;author=S.%20Kurtz&amp;publication_year=2005&amp;journal=Inform%20Software%20Tech&amp;volume=47&amp;pages=965-78" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.infsof.2005.09.005" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.infsof.2005.09.005" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.infsof.2005.09.005"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Engineering%20a%20software%20tool%20for%20gene%20structure%20prediction%20in%20higher%20organisms&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Manzanares</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Barth</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Thorogood</div> <div class="given-names">D.</div>
</div>
</div> et al.  <div class="year">2016</div>, 
<div class="article-title">A gene encoding a DUF247 domain protein cosegregates with the S self-incompatibility locus in perennial ryegrass</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">870</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20gene%20encoding%20a%20DUF247%20domain%20protein%20cosegregates%20with%20the%20S%20self-incompatibility%20locus%20in%20perennial%20ryegrass&amp;author=C.%20Manzanares&amp;author=S.%20Barth&amp;author=D.%20Thorogood&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=870-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msv335" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv335" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsv335"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26659250" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20gene%20encoding%20a%20DUF247%20domain%20protein%20cosegregates%20with%20the%20S%20self-incompatibility%20locus%20in%20perennial%20ryegrass&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Veeckman</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Ruttink</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Vandepoele</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Are we there yet? Reliably estimating the completeness of plant genome sequences</div>, <div class="source">Plant Cell</div> ., <div class="volume">28</div>, <div class="fpage">1759</div>–<div class="lpage">68</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Are%20we%20there%20yet%3F%20Reliably%20estimating%20the%20completeness%20of%20plant%20genome%20sequences&amp;author=E.%20Veeckman&amp;author=T.%20Ruttink&amp;author=K.%20Vandepoele&amp;publication_year=2016&amp;journal=Plant%20Cell&amp;volume=28&amp;pages=1759-68" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.16.00349" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.16.00349" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.16.00349"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27512012" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Are%20we%20there%20yet%3F%20Reliably%20estimating%20the%20completeness%20of%20plant%20genome%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ruttink</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Haegeman</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">van Parijs</div> <div class="given-names">F.R.D.</div>
</div>
</div> et al.  <div class="year">2015</div>, Genetic diversity in candidate genes for developmental traits and cell wall characteristics in perennial ryegrass (<em>Lolium perenne</em>). In: <div class="person-group">
<div class="name string-name">
<div class="surname">Budak</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Spangenberg</div> <div class="given-names">G.</div>
</div>
</div> (eds), <div class="source">Molecular Breeding of Forage and Turf: The Proceedings of the 8th International Symposium on the Molecular Breeding of Forage and Turf</div> , pp. <div class="fpage">93</div>–<div class="lpage">109</div>. 
<div class="publisher-name">Springer International Publishing</div>, 
<div class="publisher-loc">Cham</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Molecular%20Breeding%20of%20Forage%20and%20Turf%3A%20The%20Proceedings%20of%20the%208th%20International%20Symposium%20on%20the%20Molecular%20Breeding%20of%20Forage%20and%20Turf&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Molecular%20Breeding%20of%20Forage%20and%20Turf%3A%20The%20Proceedings%20of%20the%208th%20International%20Symposium%20on%20the%20Molecular%20Breeding%20of%20Forage%20and%20Turf&amp;aulast=Ruttink&amp;date=2015&amp;spage=93&amp;epage=109" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20Breeding%20of%20Forage%20and%20Turf%3A%20The%20Proceedings%20of%20the%208th%20International%20Symposium%20on%20the%20Molecular%20Breeding%20of%20Forage%20and%20Turf&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Molecular%20Breeding%20of%20Forage%20and%20Turf%3A%20The%20Proceedings%20of%20the%208th%20International%20Symposium%20on%20the%20Molecular%20Breeding%20of%20Forage%20and%20Turf">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Song</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Coverage recommendation for genotyping analysis of highly heterologous species using next-generation sequencing technology</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">35736</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Coverage%20recommendation%20for%20genotyping%20analysis%20of%20highly%20heterologous%20species%20using%20next-generation%20sequencing%20technology&amp;author=K.%20Song&amp;author=L.%20Li&amp;author=G.%20Zhang&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=35736" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep35736" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep35736" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep35736"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27760996" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Coverage%20recommendation%20for%20genotyping%20analysis%20of%20highly%20heterologous%20species%20using%20next-generation%20sequencing%20technology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tian</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Yan</div> <div class="given-names">H. H.</div>
</div>, <div class="name string-name">
<div class="surname">Neuhauser</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Slager</div> <div class="given-names">S. L.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">An analytical workflow for accurate variant discovery in highly divergent regions</div>, <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">703</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20analytical%20workflow%20for%20accurate%20variant%20discovery%20in%20highly%20divergent%20regions&amp;author=S.%20L.%20Tian&amp;author=H.%20H.%20Yan&amp;author=C.%20Neuhauser&amp;author=S.%20L.%20Slager&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=703" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-016-3045-z" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-3045-z" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-016-3045-z"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27590916" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20analytical%20workflow%20for%20accurate%20variant%20discovery%20in%20highly%20divergent%20regions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Park</div> <div class="given-names">M.-H.</div>
</div>, <div class="name string-name">
<div class="surname">Rhee</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">J. H.</div>
</div>
</div> et al.  <div class="year">2014</div>, 
<div class="article-title">Comprehensive analysis to improve the validation rate for single nucleotide variants detected by next-generation sequencing</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e86664</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20analysis%20to%20improve%20the%20validation%20rate%20for%20single%20nucleotide%20variants%20detected%20by%20next-generation%20sequencing&amp;author=M.-H.%20Park&amp;author=H.%20Rhee&amp;author=J.%20H.%20Park&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e86664" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0086664" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0086664" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0086664"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24489763" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comprehensive%20analysis%20to%20improve%20the%20validation%20rate%20for%20single%20nucleotide%20variants%20detected%20by%20next-generation%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nguyen-Dumont</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Pope</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Hammet</div> <div class="given-names">F.</div>
</div>
</div> et al.  <div class="year">2013</div>, 
<div class="article-title">Cross-platform compatibility of Hi-Plex, a streamlined approach for targeted massively parallel sequencing</div>, <div class="source">Anal Biochem</div> , <div class="volume">442</div>, <div class="fpage">127</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Cross-platform%20compatibility%20of%20Hi-Plex%2C%20a%20streamlined%20approach%20for%20targeted%20massively%20parallel%20sequencing&amp;author=T.%20Nguyen-Dumont&amp;author=B.%20J.%20Pope&amp;author=F.%20Hammet&amp;publication_year=2013&amp;journal=Anal%20Biochem&amp;volume=442&amp;pages=127-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ab.2013.07.046" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ab.2013.07.046" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ab.2013.07.046"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23933242" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cross-platform%20compatibility%20of%20Hi-Plex%2C%20a%20streamlined%20approach%20for%20targeted%20massively%20parallel%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div></div> <div class="year">2014</div>, 
<div class="article-title">Toward better understanding of artifacts in variant calling from high-coverage samples</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2843</div>–<div class="lpage">51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Toward%20better%20understanding%20of%20artifacts%20in%20variant%20calling%20from%20high-coverage%20samples&amp;author=H.%20Li&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2843-51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu356" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu356" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu356"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24974202" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Toward%20better%20understanding%20of%20artifacts%20in%20variant%20calling%20from%20high-coverage%20samples&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mishra</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Panigrahi</div> <div class="given-names">K. C.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">GIGANTEA – an emerging story</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">6</div>, <div class="fpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GIGANTEA%20%E2%80%93%20an%20emerging%20story&amp;author=P.%20Mishra&amp;author=K.%20C.%20Panigrahi&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci&amp;volume=6&amp;pages=8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fpls.2015.00008" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00008" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffpls.2015.00008"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25674098" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GIGANTEA%20%E2%80%93%20an%20emerging%20story&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy033-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy033-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cutler</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Ghassemian</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Bonetta</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Cooney</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">McCourt</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">1996</div>, 
<div class="article-title">A protein farnesyl transferase involved in abscisic acid signal transduction in Arabidopsis</div>, <div class="source">Science</div> , <div class="volume">273</div>, <div class="fpage">1239</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20protein%20farnesyl%20transferase%20involved%20in%20abscisic%20acid%20signal%20transduction%20in%20Arabidopsis&amp;author=S.%20Cutler&amp;author=M.%20Ghassemian&amp;author=D.%20Bonetta&amp;author=S.%20Cooney&amp;author=P.%20McCourt&amp;publication_year=1996&amp;journal=Science&amp;volume=273&amp;pages=1239-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.273.5279.1239" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.273.5279.1239" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.273.5279.1239"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8703061" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20protein%20farnesyl%20transferase%20involved%20in%20abscisic%20acid%20signal%20transduction%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5133005&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy033/2/dsy033_supp.zip?Expires=1576700708&amp;Signature=O8w3eeeX1e0czEczhZmMLtW1s7UGt7JVXt7fA8eL1iIGZJhTwPDWRCBIfjPQwLLTD-Kf4VukrS2i081HurUdv8ecWSVAPelmfGhfxNri~yV9aa8TZmQSr~ZoBMCUx-m8ybCGzGND3GAE85xoaoWAYcD4N-McUqeCPhSQdcFO5t6sh3esFXCDCFSMmZQYh62yhEgohbl0FZMf~p4yLLNV7gVbRhJn3rxf-mNGb8ymoZ8I3S03xCqQesqDxPMZQKM8M9CinvjuGy6BCqAwD-4i~PUlPaFqGbu5kUmynMLtLuQJ8sf~zlNojy5mRo4~H9gYuKDE8efDd92Gt-~2hP2B~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,118</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy033"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5133005"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f1%2f1%2f5133005">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F1%2F1%2F5133005" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5133005">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass (Lolium perenne) | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy033");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f1%2f5133005">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
