<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/5/391/5541856","@type":"ScholarlyArticle","name":"Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes","datePublished":"2019-07-31","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2019-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz017","keywords":["Illumina sequencing","sequencing bias","library preparation kits","bacterial genome sequencing","metagenome sequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/5/391/5541856","author":[{"name":"Sato, Mitsuhiko P","affiliation":"Department of Bacteriology, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"Department of Bacteriology, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan","@type":"Person"},{"name":"Nakamura, Keiji","affiliation":"Department of Bacteriology, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan","@type":"Person"},{"name":"Nishida, Ruriko","affiliation":"Department of Bacteriology, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan   Department of Medicine and Biosystemic Science, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan","@type":"Person"},{"name":"Gotoh, Yasuhiro","affiliation":"Department of Bacteriology, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan","@type":"Person"},{"name":"Hayashi, Masahiro","affiliation":"Division of Anaerobe Research, Life Science Research Center, Gifu University , Gifu, Gifu, Japan   Center for Conservation of Microbial Genetic Resource, Gifu University , Gifu, Gifu, Japan","@type":"Person"},{"name":"Hisatsune, Junzo","affiliation":"Project Research Center for Nosocomial Infectious Diseases,  Hiroshima University , Hiroshima, Hiroshima, Japan   Department of Bacteriology, Graduate School of Biomedical and Health Sciences , Hiroshima University, Hiroshima, Hiroshima, Japan   Antimicrobial Resistance Research Center, National Institute of Infectious Diseases , Tokyo, Japan","@type":"Person"},{"name":"Sugai, Motoyuki","affiliation":"Project Research Center for Nosocomial Infectious Diseases,  Hiroshima University , Hiroshima, Hiroshima, Japan   Department of Bacteriology, Graduate School of Biomedical and Health Sciences , Hiroshima University, Hiroshima, Hiroshima, Japan   Antimicrobial Resistance Research Center, National Institute of Infectious Diseases , Tokyo, Japan","@type":"Person"},{"name":"Takehiko, Itoh","affiliation":"Department of Biological Information, Tokyo Institute of Technology , Tokyo, Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"Department of Bacteriology, Graduate School of Medical Sciences , Kyushu University, Fukuoka, Fukuoka, Japan","@type":"Person"}],"description":"Abstract.  In bacterial genome and metagenome sequencing, Illumina sequencers are most frequently used due to their high throughput capacity, and multiple libra","pageStart":"391","pageEnd":"398","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1636232784&Signature=OxKMD4ReI9lWkazcEZpIvKb4jtKXrtNjvburBWQkO~-1phzSX91SdVIHHSKT-hGKNTubYO6DmmpBo0LbjTA0-rCbovkOTRw6-yan4x54zsyi80dju-xMvncbFGqpIZdpXGPGHOQdJ36LjxtFL2xSNp2UX1pcptpe-t2tzDMPIy2O-7-ZSh8JqyYG7ZFkR9~j2oUaE77Ql8S6SCGkLsLhLsGRA0f48hn1skgq9rqX2GxeGEfEc546mm8NdzIPiuJN1bly9W0ROpuGE6ahEy7OLvYAc7A3WhG~KYexzuOsR5aTO190cXLIN3Lxu17t1U~XiQvQe9Ahv2kGnGbH~zTiWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1636232784&Signature=OxKMD4ReI9lWkazcEZpIvKb4jtKXrtNjvburBWQkO~-1phzSX91SdVIHHSKT-hGKNTubYO6DmmpBo0LbjTA0-rCbovkOTRw6-yan4x54zsyi80dju-xMvncbFGqpIZdpXGPGHOQdJ36LjxtFL2xSNp2UX1pcptpe-t2tzDMPIy2O-7-ZSh8JqyYG7ZFkR9~j2oUaE77Ql8S6SCGkLsLhLsGRA0f48hn1skgq9rqX2GxeGEfEc546mm8NdzIPiuJN1bly9W0ROpuGE6ahEy7OLvYAc7A3WhG~KYexzuOsR5aTO190cXLIN3Lxu17t1U~XiQvQe9Ahv2kGnGbH~zTiWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes">
<meta property="og:description" content="Abstract.  In bacterial genome and metagenome sequencing, Illumina sequencers are most frequently used due to their high throughput capacity, and multiple libra">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/5/391/5541856">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1636232784&amp;Signature=OxKMD4ReI9lWkazcEZpIvKb4jtKXrtNjvburBWQkO~-1phzSX91SdVIHHSKT-hGKNTubYO6DmmpBo0LbjTA0-rCbovkOTRw6-yan4x54zsyi80dju-xMvncbFGqpIZdpXGPGHOQdJ36LjxtFL2xSNp2UX1pcptpe-t2tzDMPIy2O-7-ZSh8JqyYG7ZFkR9~j2oUaE77Ql8S6SCGkLsLhLsGRA0f48hn1skgq9rqX2GxeGEfEc546mm8NdzIPiuJN1bly9W0ROpuGE6ahEy7OLvYAc7A3WhG~KYexzuOsR5aTO190cXLIN3Lxu17t1U~XiQvQe9Ahv2kGnGbH~zTiWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-07-31">
<meta name="citation_author" content="Sato, Mitsuhiko P">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_author" content="Ogura, Yoshitoshi">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_author" content="Nakamura, Keiji">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_author" content="Nishida, Ruriko">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_author_institution" content="Department of Medicine and Biosystemic Science, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_author" content="Gotoh, Yasuhiro">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_author" content="Hayashi, Masahiro">
<meta name="citation_author_institution" content="Division of Anaerobe Research, Life Science Research Center, Gifu University, Gifu, Gifu, Japan">
<meta name="citation_author_institution" content="Center for Conservation of Microbial Genetic Resource, Gifu University, Gifu, Gifu, Japan">
<meta name="citation_author" content="Hisatsune, Junzo">
<meta name="citation_author_institution" content="Project Research Center for Nosocomial Infectious Diseases, Hiroshima University, Hiroshima, Hiroshima, Japan">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Biomedical and Health Sciences, Hiroshima University, Hiroshima, Hiroshima, Japan">
<meta name="citation_author_institution" content="Antimicrobial Resistance Research Center, National Institute of Infectious Diseases, Tokyo, Japan">
<meta name="citation_author" content="Sugai, Motoyuki">
<meta name="citation_author_institution" content="Project Research Center for Nosocomial Infectious Diseases, Hiroshima University, Hiroshima, Hiroshima, Japan">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Biomedical and Health Sciences, Hiroshima University, Hiroshima, Hiroshima, Japan">
<meta name="citation_author_institution" content="Antimicrobial Resistance Research Center, National Institute of Infectious Diseases, Tokyo, Japan">
<meta name="citation_author" content="Takehiko, Itoh">
<meta name="citation_author_institution" content="Department of Biological Information, Tokyo Institute of Technology, Tokyo, Japan">
<meta name="citation_author" content="Hayashi, Tetsuya">
<meta name="citation_author_institution" content="Department of Bacteriology, Graduate School of Medical Sciences, Kyushu University, Fukuoka, Fukuoka, Japan">
<meta name="citation_title" content="Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes">
<meta name="citation_firstpage" content="391">
<meta name="citation_lastpage" content="398">
<meta name="citation_doi" content="10.1093/dnares/dsz017">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="5">
<meta name="citation_publication_date" content="2019/10/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=De novo assembly of short sequence reads;  citation_journal_title=Brief. Bioinform;  citation_year=2010;  citation_volume=11;  citation_pages=457-72; ">
<meta name="citation_reference" content="citation_title=Coming of age: ten years of next-generation sequencing technologies;  citation_journal_title=Nat. Rev. Genet;  citation_year=2016;  citation_volume=17;  citation_pages=333-51; ">
<meta name="citation_reference" content="citation_title=Sequencing technologies—the next generation;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=31-46; ">
<meta name="citation_reference" content="citation_title=Library construction for next-generation sequencing: overviews and challenges;  citation_journal_title=Biotechniques;  citation_year=2014;  citation_volume=56;  citation_pages=61-77; ">
<meta name="citation_reference" content="citation_title=Evaluation of library preparation methods for Illumina next generation sequencing of small amounts of DNA from foodborne parasites;  citation_journal_title=J. Microbiol. Methods;  citation_year=2016;  citation_volume=130;  citation_pages=23-6; ">
<meta name="citation_reference" content="citation_title=Library preparation methods for next-generation sequencing: tone down the bias;  citation_journal_title=Exp. Cell Res;  citation_year=2014;  citation_volume=322;  citation_pages=12-20; ">
<meta name="citation_reference" content="citation_title=Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=R119">
<meta name="citation_reference" content="citation_title=Whole-genome sequencing and variant discovery in C. elegans;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=183-8; ">
<meta name="citation_reference" content="citation_title=A large genome center’s improvements to the Illumina sequencing system;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=1005-10; ">
<meta name="citation_reference" content="citation_title=Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes;  citation_journal_title=Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=291-5; ">
<meta name="citation_reference" content="citation_title=Characterizing and measuring bias in sequence data;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R51">
<meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; ">
<meta name="citation_reference" content="citation_title=Assembling genomes and mini-metagenomes from highly chimeric reads;  citation_year=2013;  citation_pages=158-70; ">
<meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; ">
<meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; ">
<meta name="citation_reference" content="citation_title=Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries;  citation_journal_title=Genome Biol;  citation_year=2011;  citation_volume=12;  citation_pages=R18">
<meta name="citation_reference" content="citation_title=Optimizing Illumina next-generation sequencing library preparation for extremely at-biased genomes;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1">
<meta name="citation_reference" content="citation_title=Comparison of sample preparation methods used for the next-generation sequencing of Mycobacterium tuberculosis;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0148676-14; ">
<meta name="citation_reference" content="citation_title=Library preparation methodology can influence genomic and functional predictions in human microbiome research;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=14024-9; ">
<meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12;  citation_journal_title=Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-62; ">
<meta name="citation_reference" content="citation_title=Complete genome sequence of enterohemorrhagic Escherichia coli O157: H7 and genomic comparison with a laboratory strain K-12;  citation_journal_title=DNA Res;  citation_year=2001;  citation_volume=8;  citation_pages=11-22; ">
<meta name="citation_reference" content="citation_title=Genome and virulence determinants of high virulence community-acquired MRSA;  citation_journal_title=Lancet;  citation_year=2002;  citation_volume=359;  citation_pages=1819-27; ">
<meta name="citation_reference" content="citation_title=Whole genome sequencing of meticillin-resistant Staphylococcus aureus;  citation_journal_title=Lancet;  citation_year=2001;  citation_volume=357;  citation_pages=1225-40; ">
<meta name="citation_reference" content="citation_title=Identification and molecular characterization of a gene homologous to epr (endopeptidase resistance gene) in Staphylococcus aureus;  citation_journal_title=Gene;  citation_year=1998;  citation_volume=224;  citation_pages=67-75; ">
<meta name="citation_reference" content="citation_title=FastUniq: a fast de novo duplicates removal tool for paired short reads;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e52249">
<meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; ">
<meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; ">
<meta name="citation_reference" content="citation_title=CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes;  citation_journal_title=Genome Res;  citation_year=2015;  citation_volume=25;  citation_pages=1043-55; ">
<meta name="citation_reference" content="citation_title=Insertion site preference of Mu, Tn5, and Tn7 transposons;  citation_journal_title=Mob. DNA;  citation_year=2012;  citation_volume=3;  citation_pages=3">
<meta name="citation_reference" content="citation_title=Impact of three Illumina library construction methods on GC bias and HLA genotype calling;  citation_journal_title=Hum. Immunol;  citation_year=2015;  citation_volume=76;  citation_pages=166-75; ">
<meta name="citation_reference" content="citation_title=Improved genome sequencing using an engineered transposase;  citation_journal_title=BMC Biotechnol;  citation_year=2017;  citation_volume=17;  citation_pages=6">
<meta name="citation_reference" content="citation_title=Bead-linked transposomes enable anormalization-free workflow for NGS library preparation;  citation_journal_title=BMC Genomics;  citation_year=2018;  citation_volume=19;  citation_pages=722">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/5/391/30159637/dsz017.pdf">
<meta name="description" content="Abstract.  In bacterial genome and metagenome sequencing, Illumina sequencers are most frequently used due to their high throughput capacity, and multiple libra">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/5/391/5541856">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/5/391/5541856">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/5/391/5541856");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2779951,2796125&authnips=128.235.85.32&doi=10.1093/dnares/dsz017';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f391%2f5541856" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f391%2f5541856" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=116936"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1576718524&amp;Signature=FQutN17-j1nGUBawbqXbDcoMu6K1c6qLC~qW48GfzF~lLtTrYr2bLC3YK0gEMACdA6eLMj1uzUDPtDU3i7WWpMkDpsXO6z868tE01CX7dJXQI79~xm9e8AFUWI-7rXCG11wv17iCD3rvPwLgTUCLa4doDzQOmCPBnVfevPqw6WyJOMKSGQoe-GAyed6cnayk4uUrE0meLB1R2FQTNf0063nC5X9S5UsdACIZmx7GEf46YRVLUcd6zL51D4xgy4rDjeKY8~hR1RTJY~~01xU7PIpZqj2SCEeXBJTJR~A82iV5AuCusQje4BLw8Gf6WxyRNh243fNybfuVcEuIJ6pPKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/5">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="164508215">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508215">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508217">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508217">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508221">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508221">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508238">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508238">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508261">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508261">4. Conclusion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508263">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508263">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="164508269">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508269">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/5/379/5537336">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/5/399/5543487">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuhiko P Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuhiko+P+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sato%20Mitsuhiko%20P%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Mitsuhiko P Sato</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ogura%20Yoshitoshi%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keiji Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keiji+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nakamura%20Keiji%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Keiji Nakamura</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ruriko Nishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
<div class="aff">
<div class="insititution">Department of Medicine and Biosystemic Science, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ruriko+Nishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishida%20R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nishida%20Ruriko%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Ruriko Nishida</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiro Gotoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiro+Gotoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gotoh%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Gotoh%20Yasuhiro%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Yasuhiro Gotoh</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahiro Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Division of Anaerobe Research, Life Science Research Center, Gifu University</div>, Gifu, Gifu, Japan</div>
<div class="aff">
<div class="insititution">Center for Conservation of Microbial Genetic Resource, Gifu University</div>, Gifu, Gifu, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahiro+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hayashi%20Masahiro%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Masahiro Hayashi</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junzo Hisatsune</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Project Research Center for Nosocomial Infectious Diseases, <div class="insititution">Hiroshima University</div>, Hiroshima, Hiroshima, Japan</div>
<div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Biomedical and Health Sciences</div>, Hiroshima University, Hiroshima, Hiroshima, Japan</div>
<div class="aff">
<div class="insititution">Antimicrobial Resistance Research Center, National Institute of Infectious Diseases</div>, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Junzo+Hisatsune">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hisatsune%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hisatsune%20Junzo%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Junzo Hisatsune</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motoyuki Sugai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Project Research Center for Nosocomial Infectious Diseases, <div class="insititution">Hiroshima University</div>, Hiroshima, Hiroshima, Japan</div>
<div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Biomedical and Health Sciences</div>, Hiroshima University, Hiroshima, Hiroshima, Japan</div>
<div class="aff">
<div class="insititution">Antimicrobial Resistance Research Center, National Institute of Infectious Diseases</div>, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motoyuki+Sugai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugai%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sugai%20Motoyuki%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Motoyuki Sugai</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Itoh Takehiko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Biological Information, Tokyo Institute of Technology</div>, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Itoh+Takehiko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takehiko%20I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Takehiko%20Itoh%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Itoh Takehiko</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">
<div class="insititution">Department of Bacteriology, Graduate School of Medical Sciences</div>, Kyushu University, Fukuoka, Fukuoka, Japan</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 92 642 6128. Fax. +81 92 642 6128. Email: <a href="mailto:thayash@bact.med.kyushu-u.ac.jp" target="_blank">thayash@bact.med.kyushu-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hayashi%20Tetsuya%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 5, October 2019, Pages 391–398, <a href="https://doi.org/10.1093/dnares/dsz017">https://doi.org/10.1093/dnares/dsz017</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">31 July 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">05 April 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">17 July 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">31 July 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5541856" href="/dnaresearch/article-pdf/26/5/391/30159637/dsz017.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mitsuhiko P Sato, Yoshitoshi Ogura, Keiji Nakamura, Ruriko Nishida, Yasuhiro Gotoh, Masahiro Hayashi, Junzo Hisatsune, Motoyuki Sugai, Itoh Takehiko, Tetsuya Hayashi,  Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes, <em>DNA Research</em>, Volume 26, Issue 5, October 2019, Pages 391–398, <a href="https://doi.org/10.1093/dnares/dsz017">https://doi.org/10.1093/dnares/dsz017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5541856&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparison%20of%20the%20sequencing%20bias%20of%20currently%20available%20library%20preparation%20kits%20for%20Illumina%20sequencing%20of%20bacterial%20genomes%20and%20metagenomes&amp;publicationDate=2019-07-31&amp;volumeNum=26&amp;issueNum=5&amp;author=Sato%2C%20Mitsuhiko%20P%3B%20Ogura%2C%20Yoshitoshi&amp;startPage=391&amp;endPage=398&amp;contentId=10.1093%2Fdnares%2Fdsz017&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f391%2f5541856" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="164508215" id="164508215" class="abstract-title">Abstract</h2>
<section class="abstract"><p>In bacterial genome and metagenome sequencing, Illumina sequencers are most frequently used due to their high throughput capacity, and multiple library preparation kits have been developed for Illumina platforms. Here, we systematically analysed and compared the sequencing bias generated by currently available library preparation kits for Illumina sequencing. Our analyses revealed that a strong sequencing bias is introduced in low-GC regions by the Nextera XT kit. The level of bias introduced is dependent on the level of GC content; stronger bias is generated as the GC content decreases. Other analysed kits did not introduce this strong sequencing bias. The GC content-associated sequencing bias introduced by Nextera XT was more remarkable in metagenome sequencing of a mock bacterial community and seriously affected estimation of the relative abundance of low-GC species. The results of our analyses highlight the importance of selecting proper library preparation kits according to the purposes and targets of sequencing, particularly in metagenome sequencing, where a wide range of microbial species with various degrees of GC content is present. Our data also indicate that special attention should be paid to which library preparation kit was used when analysing and interpreting publicly available metagenomic data.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"Illumina sequencing"'>Illumina sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"sequencing bias"'>sequencing bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"library preparation kits"'>library preparation kits</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"bacterial genome sequencing"'>bacterial genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"metagenome sequencing"'>metagenome sequencing</a>
</div>                    <h2 scrollto-destination="164508217" id="164508217" class="section-title">1. Introduction</h2>
<p>Next-generation sequencing (NGS) has revolutionized the field of genomics,<a reveal-id="dsz017-B1" data-open="dsz017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsz017-B2" data-open="dsz017-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> as it has much higher throughput (thus much lower cost) compared with traditional Sanger sequencing.<a reveal-id="dsz017-B3" data-open="dsz017-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The use of Illumina sequencing technology dominates the fields of bacterial genomics and metagenomics. Therein, library construction is an important process. While several library construction methods have been developed for Illumina sequencing,<a reveal-id="dsz017-B4" data-open="dsz017-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsz017-B5" data-open="dsz017-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> this process generally comprises three steps: DNA fragmentation, repairing and end-polishing of fragmented DNA, and ligation of platform-specific adaptors.<a reveal-id="dsz017-B6" data-open="dsz017-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Several library preparation kits are now commercially available, and they employ sonication or enzymatic digestion for DNA fragmentation. Nextera XT DNA Library Preparation Kit (XT), which employs a transposon to shear genomic DNA and simultaneously introduce adapter sequences,<a reveal-id="dsz017-B7" data-open="dsz017-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> is an alternative approach for streamlining the workflow, improving turnaround time and reducing DNA input.</p>
<p>In Illumina sequencing, extreme base composition, i.e. extremely GC-poor or rich sequences, has been reported to yield uneven or poor sequencing coverage.<a reveal-id="dsz017-B8 dsz017-B9 dsz017-B10" data-open="dsz017-B8 dsz017-B9 dsz017-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> For example, Illumina sequencing of <em>Plasmodium falciparum</em> (mean GC; 19%) and <em>Rhodobacter sphaeroides</em> (69%) favoured more GC-balanced regions and yielded fewer reads from regions of GC content at either extreme.<a reveal-id="dsz017-B11" data-open="dsz017-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Such a sequencing bias reduces the efficacy of data analyses; genome assembly,<a reveal-id="dsz017-B12" data-open="dsz017-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsz017-B13" data-open="dsz017-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> identification of sequence variations by read mapping,<a reveal-id="dsz017-B14" data-open="dsz017-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsz017-B15" data-open="dsz017-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and estimation of the copy numbers of sequences of interests. Lower-coverage regions may lead to a failure to identify single-nucleotide polymorphisms and genomic regions of functional or phylogenetical importance. Efforts to reduce gaps or low-coverage regions by obtaining more sequence reads inflate sequencing costs and may limit the effectiveness of genomic analyses, particularly, those aiming to analyse numerous samples. Thus, improving our knowledge of sequencing bias is essential to further improve the utility of sequencing by NGS.</p>
<p>Uneven coverage associated with GC bias can be introduced during PCR amplification of library, cluster amplification, or sequencing. Among these, library amplification is known as a major source.<a reveal-id="dsz017-B16" data-open="dsz017-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsz017-B17" data-open="dsz017-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The XT kit has been reported to introduce a substantial sequencing bias in <em>Mycobacterium tuberculosis</em> sequencing (mean GC; 66%)<a reveal-id="dsz017-B18" data-open="dsz017-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and yielded more unmapped open reading frames for <em>Clostridium beijerinckii</em> (30%) in a mock metagenome sample compared with other kits.<a reveal-id="dsz017-B19" data-open="dsz017-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> We also found inefficiency issues in assembling <em>Staphylococcus aureus</em> genomes (33%) from Illumina reads obtained by XT. Because studies on sequencing bias have been conducted in limited strains or species, systematic investigation of bacteria with a wide GC content range is required to understand the factors that introduce sequencing bias. Different sequencing kits and protocols may also be differentially affected by GC bias. The resulting sequence bias also impairs interpretation of metagenome sequencing data, which include many unknown species, particularly in estimating the relative abundances of genes/species in a microbial community based on read counting. Therefore, in this study, we compared currently available library preparation kits for Illumina sequencing, including the Nextera DNA Flex Library Prep Kit recently released by Illumina, to examine what kinds and what levels of sequencing bias are generated by these kits across a wide range of bacterial species. The impacts of sequencing bias on metagenome analysis were also evaluated using a mock bacterial community comprising species with a wide range of GC content levels.</p>                    <h2 scrollto-destination="164508221" id="164508221" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="164508222" id="164508222" class="section-title">2.1. Strains and genomic DNA preparation</h3>
<p>We used two <em>Escherichia coli</em> strains, K-12 MG1655 and O157 Sakai, and two <em>S. aureus</em> strains, MW2 and N315, as model microbial organisms to analyse sequencing bias introduced by different library preparation kits (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The complete genome sequences of these strains are available.<a reveal-id="dsz017-B20 dsz017-B21 dsz017-B22 dsz017-B23" data-open="dsz017-B20 dsz017-B21 dsz017-B22 dsz017-B23" class="link link-ref link-reveal xref-bibr"><sup>20–23</sup></a> In addition, 22 strains of non-<em>S. aureus</em> species in the genus <em>Staphylococcus</em> and seven species with various degrees of GC content from the strain collections of our laboratories were analysed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The <em>E. coli</em> strains were grown in Lysogeny broth (LB; Becton Dickinson Microbiology Systems, MD, USA) at 37°C. Genomic DNA was purified from 2 ml of overnight culture using a Genomic-tip 100/G and Genomic DNA buffer set (QIAGEN, CA, USA) according to the manufacturer’s instructions. <em>Staphylococcal</em> strains were grown at 37°C overnight with shaking in 2-ml tryptic soy broth (Becton Dickinson Microbiology Systems), and genomic DNA was isolated as previously described.<a reveal-id="dsz017-B24" data-open="dsz017-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Briefly, the cells were collected by centrifugation, resuspended in 500 µl of CS buffer (100 mM Tris-HCl pH 7.5, 150 mM NaCl, 10 mM EDTA), incubated sequentially with 20 µg/ml lysostaphin (Wako, Tokyo, Japan) for 30 min at 37°C and with 100 µg/ml Proteinase K and 1% SDS (both from Nacalai Tesque, Kyoto, Japan) at 55°C for 2 h, and then subjected to phenol-chloroform extraction and ethanol precipitation. The genomic DNA was finally cleaned up using a Genomic DNA Clean &amp; Concentrator kit (Zymo Research, CA, USA). Among the seven species with various degrees of GC content, <em>M. interjectum</em> and <em>M. malmoense</em> were grown in Middlebrook 7H9 broth (Becton Dickinson Microbiology Systems) with albumin dextrose catalase enrichment. <em>Tsukamurella pulmonis</em>, <em>Klebsiella aerogenes</em>, <em>Streptobacillus moniliformis</em>, and <em>Brachyspira pilosicoli</em> were grown on sheep blood agar at 37°C, and <em>Serratia liquefaciens</em> was grown on LB agar at 37°C. Genomic DNA from these bacteria was prepared using a NucleoSpin Microbial DNA kit (Takara Bio Inc., Shiga, Japan).</p>
<p>A mock microbial community DNA was composed of genomic DNA from <em>E. coli</em> MG1655, <em>S. aureus</em> N315, <em>M. interjectum</em>, <em>M. malmoense</em>, <em>T. pulmonis</em>, <em>K. aerogenes</em>, <em>S. liquefaciens</em>, <em>S. moniliformis</em>, and <em>B. pilosicoli</em>, representing an approximately equimolar mixture of these genomic DNAs.</p>                    <h3 scrollto-destination="164508225" id="164508225" class="section-title">2.2. Genome sequences obtained from a public database</h3>
<p>The genome sequencing data of 204 human pathogenic bacterial species were obtained from the DDBJ Sequence Read Archive database in fastq format. These strains were from the Gifu Type Culture (GTC) collection of Gifu University Center for Conservation of Microbial Genetic Resource (GCMR) and were sequenced by the National BioResource Project (NBRP) of Japan using an XT library preparation kit (Illumina, CA, USA) and the Illumina HiSeq 2500 platform (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p>                    <h3 scrollto-destination="164508227" id="164508227" class="section-title">2.3. Library preparation and sequencing</h3>
<p>Eight sequencing library preparation kits were used in this study: XT, KAPA HyperPlus (NIPPON Genetics Co. Ltd, Tokyo, Japan) with PCR or PCR-free workflow (referred to as KP and KPF, respectively), NEBNext Ultra II (referred to as NN; New England Biolabs Japan, Tokyo, Japan), QIAseq FX (QS; QIAGEN), TruSeq nano (TS; Illumina), TruSeq DNA PCR-Free (TSF; Illumina), and Nextera DNA Flex (FL; Illumina) (<a reveal-id="dsz017-T1" data-open="dsz017-T1" class="link link-reveal link-table xref-fig">Table 1</a>). For XT, KP, NN, QS, and FL, 1 ng of DNA, measured by Agilent TapeStation (Agilent Technologies, CA, USA), was used for input DNA, and the number of PCR cycles was fixed at 12. These parameters were set according to the XT protocol, for which the amount of input DNA and the number of PCR cycles were essentially unable to be modified. For the TS library preparation, 200 ng of input DNA and eight PCR cycles were employed. For KPF and TSF, 1 µg of input DNA was used. Other procedures for library preparation were performed according to the recommended protocols of each kit. The prepared libraries were sequenced on the Illumina MiSeq platform with a paired-end (PE) 600-cycle mode (Reagent Kit v3). The XT libraries of the <em>E. coli</em> and <em>S. aureus</em> genomes were prepared independently three times from the same genomic DNA preparation to generate technical replicates. The sequencing reads have been deposited in DDBJ (accession number: PRJDB8030). </p>                        <a id="164508229" scrollto-destination="164508229"></a>
<div content-id="dsz017-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz017-T1" class="table-wrap-title" data-id="dsz017-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Library preparation kits analysed in this study</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Kits </th>
<th>Abbreviation </th>
<th>Fragmentation methods </th>
<th>PCR cycles </th>
<th>Input DNA (ng) </th>
</tr></thead>
<tbody>
<tr>
<td>Nextera XT </td>
<td>XT </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>Nextera DNA Flex </td>
<td>FL </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>KAPA HyperPlus </td>
<td>KP </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>NEBNext Ultra II </td>
<td>NN </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>QIAseq FX </td>
<td>QS </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>TruSeq nano </td>
<td>TS </td>
<td>Sonication </td>
<td>8 </td>
<td>200 </td>
</tr>
<tr>
<td>KAPA HyperPlus PCR-free workflow </td>
<td>KPF </td>
<td>Enzymatic </td>
<td>0 </td>
<td>1,000 </td>
</tr>
<tr>
<td>TruSeq DNA PCR-free </td>
<td>TSF </td>
<td>Sonication </td>
<td>0 </td>
<td>1,000 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz017-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Kits </th>
<th>Abbreviation </th>
<th>Fragmentation methods </th>
<th>PCR cycles </th>
<th>Input DNA (ng) </th>
</tr></thead>
<tbody>
<tr>
<td>Nextera XT </td>
<td>XT </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>Nextera DNA Flex </td>
<td>FL </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>KAPA HyperPlus </td>
<td>KP </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>NEBNext Ultra II </td>
<td>NN </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>QIAseq FX </td>
<td>QS </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>TruSeq nano </td>
<td>TS </td>
<td>Sonication </td>
<td>8 </td>
<td>200 </td>
</tr>
<tr>
<td>KAPA HyperPlus PCR-free workflow </td>
<td>KPF </td>
<td>Enzymatic </td>
<td>0 </td>
<td>1,000 </td>
</tr>
<tr>
<td>TruSeq DNA PCR-free </td>
<td>TSF </td>
<td>Sonication </td>
<td>0 </td>
<td>1,000 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/164508229" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz017-T1" class="table-wrap-title" data-id="dsz017-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Library preparation kits analysed in this study</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Kits </th>
<th>Abbreviation </th>
<th>Fragmentation methods </th>
<th>PCR cycles </th>
<th>Input DNA (ng) </th>
</tr></thead>
<tbody>
<tr>
<td>Nextera XT </td>
<td>XT </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>Nextera DNA Flex </td>
<td>FL </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>KAPA HyperPlus </td>
<td>KP </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>NEBNext Ultra II </td>
<td>NN </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>QIAseq FX </td>
<td>QS </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>TruSeq nano </td>
<td>TS </td>
<td>Sonication </td>
<td>8 </td>
<td>200 </td>
</tr>
<tr>
<td>KAPA HyperPlus PCR-free workflow </td>
<td>KPF </td>
<td>Enzymatic </td>
<td>0 </td>
<td>1,000 </td>
</tr>
<tr>
<td>TruSeq DNA PCR-free </td>
<td>TSF </td>
<td>Sonication </td>
<td>0 </td>
<td>1,000 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz017-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Kits </th>
<th>Abbreviation </th>
<th>Fragmentation methods </th>
<th>PCR cycles </th>
<th>Input DNA (ng) </th>
</tr></thead>
<tbody>
<tr>
<td>Nextera XT </td>
<td>XT </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>Nextera DNA Flex </td>
<td>FL </td>
<td>Tagmentation by transposome </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>KAPA HyperPlus </td>
<td>KP </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>NEBNext Ultra II </td>
<td>NN </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>QIAseq FX </td>
<td>QS </td>
<td>Enzymatic </td>
<td>12 </td>
<td>1 </td>
</tr>
<tr>
<td>TruSeq nano </td>
<td>TS </td>
<td>Sonication </td>
<td>8 </td>
<td>200 </td>
</tr>
<tr>
<td>KAPA HyperPlus PCR-free workflow </td>
<td>KPF </td>
<td>Enzymatic </td>
<td>0 </td>
<td>1,000 </td>
</tr>
<tr>
<td>TruSeq DNA PCR-free </td>
<td>TSF </td>
<td>Sonication </td>
<td>0 </td>
<td>1,000 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/164508229" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="164508230" id="164508230" class="section-title">2.4. Genome assembly</h3>
<p>PCR duplicates were removed by FastUniq,<a reveal-id="dsz017-B25" data-open="dsz017-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and adapter and low-quality sequences were trimmed by trim galore! v0.4.2 (<a class="link link-uri" href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>) for all sequencing data. Trimmed reads from each sample were assembled using Velvet v1.2.10 and VelvetOptimiser v2.2.5<a reveal-id="dsz017-B12" data-open="dsz017-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> with k-mer lengths ranging from 19 to 199. Contigs shorter than 300 bp were excluded from this study. To compare the numbers of contigs obtained by the six library preparation kits under the same conditions (e.g. at the same sequencing depth), sequencing reads were randomly picked up from each of the four model genomes (two <em>E. coli</em> and two <em>S. aureus</em> genomes) to gain ×30 coverage, which was calculated based on the total genome length of each strain. This procedure was repeated ten times for each genome. The four model genomes were also assembled by SPAdes v3.9.0<a reveal-id="dsz017-B13" data-open="dsz017-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> with default parameters.</p>                    <h3 scrollto-destination="164508232" id="164508232" class="section-title">2.5. Sequencing bias analysis</h3>
<p>Insert sizes of each library were calculated using Picard tools version 2.7.0 (<a class="link link-uri" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>), and jellyfish v2.2.6<a reveal-id="dsz017-B26" data-open="dsz017-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was used to calculate the 30-mer frequency. Per-base coverage was computed by counting the reads mapped to a given base along the reference genome.<a reveal-id="dsz017-B11" data-open="dsz017-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> We used bowtie2<a reveal-id="dsz017-B14" data-open="dsz017-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and SAMtools<a reveal-id="dsz017-B27" data-open="dsz017-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> for read mapping and counting the coverage of each base, respectively. As references, the published complete sequences were used for the four model genomes from <em>E. coli</em> and <em>S. aureus</em>, while draft sequences were used for others. The draft sequences were generated by merging and assembling sequencing reads obtained by all kits in each strain, and contigs shorter than N90 were excluded from the references because they might be derived from multicopy genetic elements. The qualities of the assemblies from 204 strains obtained from the NBRP data were assessed by checkM,<a reveal-id="dsz017-B28" data-open="dsz017-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and 13 genomes with low completeness (&lt;85%) or high contamination (&gt;5%) were excluded from this study (final <em>n</em> = 191).</p>
<p>To evaluate coverage bias (a deviation from the uniform distribution of reads across the genome), we first calculated per-base relative coverage, which is the ratio of the coverage of a given reference base to the mean coverage across the genome.<a reveal-id="dsz017-B11" data-open="dsz017-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Then, ‘relative coverage’ was presented as the mean per-base relative coverage of each 200-bp window with no overlap.</p>
<div class="block-child-p">To evaluate the level of sequencing bias associated with various degrees of GC content, GC content was calculated for each 200-bp window. The 200 bp windows with similar GC content levels (defined by a 0.5% interval) were binned, and the mean relative coverage of each bin was calculated; here, the mean relative coverage of the bin with a given GC content was represented as <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math>⁠</span>. Finally, to quantify the overall sequencing bias associated with GC content in a given genome, the overall GC content-associated bias was defined as: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac xmlns=""><mrow><mrow><msub><mo stretchy="false">∑</mo><mrow><mi>i</mi></mrow></msub><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub><msqrt><msup><mrow><mfenced separators="|"><mrow><msub><mrow><mi>C</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>-</mo><mrow><mi>C</mi></mrow></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></msqrt></mrow></mrow></mrow><mrow><msub><mrow><mi>n</mi></mrow><mrow><mi>T</mi></mrow></msub><mi>N</mi></mrow></mfrac></math></div></div></div>where <em>i</em> is GC content defined at 0.5% intervals; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> of the bin with <em>i</em> % GC; <em>C</em> is the mean relative coverage across the genome (=1); <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the number of windows with <em>i</em> % GC; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>n</mi></mrow><mrow><mi>T</mi></mrow></msub></math></span> is the total number of windows; and <em>N</em> is the total number of analysed bins. We used linear regression models to investigate the relationship between the overall GC content-associated sequencing bias and mean GC content across the genome using R v3.2.4.</div>                    <h3 scrollto-destination="164508236" id="164508236" class="section-title">2.6. Digital droplet PCR and mock community analysis</h3>
<p>Accurate molecular ratios of the seven species in the pooled genomic DNA sample for the mock microbial community were determined by digital droplet PCR (ddPCR) using ddPCR™ EvaGreen Supermix (BioRad, CA, USA) and species-specific primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), which were designed in the arginyl-tRNA synthetase gene, one of the universal single-copy genes. Droplets were generated using a QX200™ droplet generator (BioRad), and each ddPCR sample was composed of sample DNA, primers, and ddPCR super mix. PCR was performed with the following conditions: 95°C for 5 min, 40 cycles of 94°C for 30 s, and 58°C for 90 s, and a final incubation at 90°C for 5 min. The data were analysed by QuantaSoft version 1.7.4 (BioRad). The established copy numbers of each genome in the pooled DNA sample were used to normalize the relative abundance of genomic DNA and per-base coverage for each species within the mock microbial community.</p>                    <h2 scrollto-destination="164508238" id="164508238" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="164508239" id="164508239" class="section-title">3.1. Comparison of six library preparation kits using <em>E. coli</em> and <em>S. aureus</em> as model bacterial genomes</h3>
<p>We first used two <em>E. coli</em> and two <em>S. aureus</em> genomes as model genomes for comparing the quality of the libraries prepared by different library preparation kits. The six kits compared in this study cover three fragmentation strategies (<a reveal-id="dsz017-T1" data-open="dsz017-T1" class="link link-reveal link-table xref-fig">Table 1</a>): tagmentation by transposome (XT and FL), enzymatic fragmentation (KP, NN, and QS), and sonication (TS). FL utilizes improved tagmentation chemistry to obtain a uniform fragment size; KP, NN, and QS each use different enzymes for fragmentation.</p>
<p>To assess the difference in the efficacy of sequence assembly between the kits, we first compared the number of contigs and L50 values of each assembly obtained by Velvet and SPAdes at the same sequencing depth (×30) (<a reveal-id="dsz017-F1" data-open="dsz017-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). In <em>E. coli</em>, the number of contigs and L50 values exhibited some level of variation between the kits and between the strains and assemblers used. In contrast, the levels of sequence assembly of both <em>S. aureus</em> genomes obtained by XT were much lower than those obtained by other kits, regardless of which assembler was used. Analysis of k-mer frequencies in each read data set revealed that a sharp peak was observed in <em>S. aureus</em> data sets with the exception of the data set obtained by XT (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). This result suggests that the inefficient sequence assembly observed for the XT-derived <em>S. aureus</em> assemblies was caused by uneven genome sequencing. In fact, sequence coverage across the genome in each library assessed by calculating the relative coverage in every 200-bp window (see Materials and methods) revealed that the XT-derived assemblies of <em>S. aureus</em> genomes showed remarkably uneven coverage along the entire genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Notable bias was also observed in the XT-derived assemblies of <em>E. coli</em> genomes but to a much lesser extent than in the <em>S. aureus</em> genomes. </p>                        <a id="164508242" scrollto-destination="164508242"></a>
<div id="dsz017-F1" data-id="dsz017-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f1.png?Expires=1576212293&amp;Signature=woLJE9288-14i2rYx~mX1SRb8zmAICT5-45lGvIIwja2DbR1IE1HGUB6QfhvnHMc0~bAfg6DqLeQisEnhFqOB7wx2jqpMjqb~2zh563TrwyKw02HpGhLaKoBOYpy09D~C2HrQSVc7nXF-x2krdGsSb7yyxJ2iq3Ea6aI4P1kxF7L1mfp7riJIV9RVoqCk2naS6X8B1gG9WBPFjcUL-Y5EWYUC9XmfVZee5J1MZG9E09ZVLd6nXHaNmBIuoFy9X8Qq9dDrOstt67nwT4tla~VRBSIQC7Yaca4oco1Pw-DI1z~SjpOHfPeuk4aOQ27vBso4FMZyav--DeUB1GPZmu4kw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quality comparison of E. coli and S. aureus genome assemblies obtained by library preparation kits. (A) Assembly statistics obtained by six library preparation kits were compared in E. coli and S. aureus. Two E. coli and two S. aureus genomes were analysed as model bacterial genomes to compare six library preparation kits. Illumina read sequences obtained from each library were assembled using Velvet and SPAdes, and the numbers of contigs and L50 values of each assembly are shown. In each sequence data set, assembly was repeated 10 times using Illumina reads randomly selected at 30× coverage. Error bars indicate standard deviations. The six kits used cover three fragmentation strategies (see the main text). XT, Nextera XT; FL, Nextera DNA Flex; KP, KAPA HyperPlus; NN, NEBNext Ultra II; QS, QIAseq FX; and TS, TruSeq nano. (B) Relative sequence coverage in relation to GC content was calculated in E. coli and S. aureus genomes obtained by three library preparation kits. Relative sequence coverage in the genome assemblies obtained by the XT, FL, and KP kits and GC content were calculated for every 200-bp window with no overlap. Only the first 120,000 bp regions of each genome are shown. (C) Relationships between GC content and sequence coverage in the E. coli and S. aureus genome assemblies obtained by six library preparation kits are shown. The relative abundance of 200 bp bins with a given GC content (defined by 0.5% interval) and the mean relative coverage of bins with a given GC content (CGC) were calculated and are shown along with GC content by black lines or lines coloured according to the library preparation kits, respectively. Black horizontal lines (CGC=1) represent unbiased coverage. The data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at DNARES online." path-from-xml="dsz017f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508242/dsz017f1.tif" path-from-xml="dsz017f1.tif" target="_blank">Open in new tab</a><a section="164508242" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f1.png?Expires=1576212293&amp;Signature=2DE~nJllP3fNXAK7Ag0M1gGzE27yDZTlfxWcJf0eFbqBA6N6mLlFi2e1K62WNFty2D89j-A-B11pTz44lDD6Lngg7prcasK3taVkE1GNi5nQ1os7o0ivUkknnYxR1ciMrgUTVso5e9YS27BS7WSjUX~K699erJgWQ3M1jsrYIsJTBrrUC96JOyVRhPQhY1oim8OtWbCOUedRBRslpwOg-ENCqzZHlrUF5m3lNHdFKfPbZjbaBD89rhUl~Pu-TkRGj7yYpqPQrat7gwI20BR45rLKaBirVLspJrpKf9TMMzKy0ceCoZpItmUky0-Aod8GVbzA0YwksCBRqpuUh-OiUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508242&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Quality comparison of <em>E. coli</em> and <em>S. aureus</em> genome assemblies obtained by library preparation kits. (A) Assembly statistics obtained by six library preparation kits were compared in <em>E. coli</em> and <em>S. aureus</em>. Two <em>E. coli</em> and two <em>S. aureus</em> genomes were analysed as model bacterial genomes to compare six library preparation kits. Illumina read sequences obtained from each library were assembled using Velvet and SPAdes, and the numbers of contigs and L50 values of each assembly are shown. In each sequence data set, assembly was repeated 10 times using Illumina reads randomly selected at 30× coverage. Error bars indicate standard deviations. The six kits used cover three fragmentation strategies (see the main text). XT, Nextera XT; FL, Nextera DNA Flex; KP, KAPA HyperPlus; NN, NEBNext Ultra II; QS, QIAseq FX; and TS, TruSeq nano. (B) Relative sequence coverage in relation to GC content was calculated in <em>E. coli</em> and <em>S. aureus</em> genomes obtained by three library preparation kits. Relative sequence coverage in the genome assemblies obtained by the XT, FL, and KP kits and GC content were calculated for every 200-bp window with no overlap. Only the first 120,000 bp regions of each genome are shown. (C) Relationships between GC content and sequence coverage in the <em>E. coli</em> and <em>S. aureus</em> genome assemblies obtained by six library preparation kits are shown. The relative abundance of 200 bp bins with a given GC content (defined by 0.5% interval) and the mean relative coverage of bins with a given GC content (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math>⁠</span>) were calculated and are shown along with GC content by black lines or lines coloured according to the library preparation kits, respectively. Black horizontal lines (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span>=1) represent unbiased coverage. The data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsz017-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f1.png?Expires=1576212293&amp;Signature=woLJE9288-14i2rYx~mX1SRb8zmAICT5-45lGvIIwja2DbR1IE1HGUB6QfhvnHMc0~bAfg6DqLeQisEnhFqOB7wx2jqpMjqb~2zh563TrwyKw02HpGhLaKoBOYpy09D~C2HrQSVc7nXF-x2krdGsSb7yyxJ2iq3Ea6aI4P1kxF7L1mfp7riJIV9RVoqCk2naS6X8B1gG9WBPFjcUL-Y5EWYUC9XmfVZee5J1MZG9E09ZVLd6nXHaNmBIuoFy9X8Qq9dDrOstt67nwT4tla~VRBSIQC7Yaca4oco1Pw-DI1z~SjpOHfPeuk4aOQ27vBso4FMZyav--DeUB1GPZmu4kw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quality comparison of E. coli and S. aureus genome assemblies obtained by library preparation kits. (A) Assembly statistics obtained by six library preparation kits were compared in E. coli and S. aureus. Two E. coli and two S. aureus genomes were analysed as model bacterial genomes to compare six library preparation kits. Illumina read sequences obtained from each library were assembled using Velvet and SPAdes, and the numbers of contigs and L50 values of each assembly are shown. In each sequence data set, assembly was repeated 10 times using Illumina reads randomly selected at 30× coverage. Error bars indicate standard deviations. The six kits used cover three fragmentation strategies (see the main text). XT, Nextera XT; FL, Nextera DNA Flex; KP, KAPA HyperPlus; NN, NEBNext Ultra II; QS, QIAseq FX; and TS, TruSeq nano. (B) Relative sequence coverage in relation to GC content was calculated in E. coli and S. aureus genomes obtained by three library preparation kits. Relative sequence coverage in the genome assemblies obtained by the XT, FL, and KP kits and GC content were calculated for every 200-bp window with no overlap. Only the first 120,000 bp regions of each genome are shown. (C) Relationships between GC content and sequence coverage in the E. coli and S. aureus genome assemblies obtained by six library preparation kits are shown. The relative abundance of 200 bp bins with a given GC content (defined by 0.5% interval) and the mean relative coverage of bins with a given GC content (CGC) were calculated and are shown along with GC content by black lines or lines coloured according to the library preparation kits, respectively. Black horizontal lines (CGC=1) represent unbiased coverage. The data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at DNARES online." path-from-xml="dsz017f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508242/dsz017f1.tif" path-from-xml="dsz017f1.tif" target="_blank">Open in new tab</a><a section="164508242" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f1.png?Expires=1576212293&amp;Signature=2DE~nJllP3fNXAK7Ag0M1gGzE27yDZTlfxWcJf0eFbqBA6N6mLlFi2e1K62WNFty2D89j-A-B11pTz44lDD6Lngg7prcasK3taVkE1GNi5nQ1os7o0ivUkknnYxR1ciMrgUTVso5e9YS27BS7WSjUX~K699erJgWQ3M1jsrYIsJTBrrUC96JOyVRhPQhY1oim8OtWbCOUedRBRslpwOg-ENCqzZHlrUF5m3lNHdFKfPbZjbaBD89rhUl~Pu-TkRGj7yYpqPQrat7gwI20BR45rLKaBirVLspJrpKf9TMMzKy0ceCoZpItmUky0-Aod8GVbzA0YwksCBRqpuUh-OiUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508242&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Quality comparison of <em>E. coli</em> and <em>S. aureus</em> genome assemblies obtained by library preparation kits. (A) Assembly statistics obtained by six library preparation kits were compared in <em>E. coli</em> and <em>S. aureus</em>. Two <em>E. coli</em> and two <em>S. aureus</em> genomes were analysed as model bacterial genomes to compare six library preparation kits. Illumina read sequences obtained from each library were assembled using Velvet and SPAdes, and the numbers of contigs and L50 values of each assembly are shown. In each sequence data set, assembly was repeated 10 times using Illumina reads randomly selected at 30× coverage. Error bars indicate standard deviations. The six kits used cover three fragmentation strategies (see the main text). XT, Nextera XT; FL, Nextera DNA Flex; KP, KAPA HyperPlus; NN, NEBNext Ultra II; QS, QIAseq FX; and TS, TruSeq nano. (B) Relative sequence coverage in relation to GC content was calculated in <em>E. coli</em> and <em>S. aureus</em> genomes obtained by three library preparation kits. Relative sequence coverage in the genome assemblies obtained by the XT, FL, and KP kits and GC content were calculated for every 200-bp window with no overlap. Only the first 120,000 bp regions of each genome are shown. (C) Relationships between GC content and sequence coverage in the <em>E. coli</em> and <em>S. aureus</em> genome assemblies obtained by six library preparation kits are shown. The relative abundance of 200 bp bins with a given GC content (defined by 0.5% interval) and the mean relative coverage of bins with a given GC content (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math>⁠</span>) were calculated and are shown along with GC content by black lines or lines coloured according to the library preparation kits, respectively. Black horizontal lines (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span>=1) represent unbiased coverage. The data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<p>Because sequencing bias associated with GC content has been reported for Illumina sequencing,<a reveal-id="dsz017-B8 dsz017-B9 dsz017-B10" data-open="dsz017-B8 dsz017-B9 dsz017-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> we next investigated the relationship between relative coverage and GC content across the <em>E. coli</em> and <em>S. aureus</em> genomes (<a reveal-id="dsz017-F1" data-open="dsz017-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). The results of this analysis clearly indicated that the variations in relative coverage in the XT libraries correlated well with the variations in GC content, particularly in the <em>S. aureus</em> genomes, with lower coverage in regions with lower GC content. In <a reveal-id="dsz017-F1" data-open="dsz017-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, only the data for a 120-kb region in each genome are shown, but similar patterns were observed along the entire genome (data not shown). To quantitatively evaluate the level of GC content-associated sequencing bias at the whole-genome level, we calculated <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math>⁠</span>, which represents the mean relative coverage of a 200-bp bin with a given GC content (see Materials and methods for more details), across the genome and analysed the GC content-associated variation in <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> in each library (<a reveal-id="dsz017-F1" data-open="dsz017-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). This analysis revealed that in the XT libraries of <em>S. aureus</em>, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> was drastically reduced as GC content decreased. A reduction in <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> was also observed in the XT libraries of <em>E. coli</em>, but only in the bins with lower GC content. In contrast, similar <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> values were observed in bins of all GC content values in the libraries prepared with the other five kits, including FL. We prepared XT libraries three times from all <em>E. coli</em> and <em>S. aureus</em> genomes and performed the same analyses as technical replicates. Although the results of <em>E. coli</em> libraries showed some variation between the samples, the results of <em>S. aureus</em> genomes were highly reproducible (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p>
<p>All these results indicate that a notable GC content-associated sequencing bias in Illumina sequencing is introduced by XT, particularly in <em>S. aureus</em> genome sequencing.</p>                    <h3 scrollto-destination="164508245" id="164508245" class="section-title">3.2. Sequencing bias in 22 strains from various species in the genus <em>Staphylococcus</em>
</h3>
<p>To examine whether the GC content-associated sequencing bias observed for XT is species-specific, we assessed the sequencing bias introduced by XT in other <em>Staphylococcus</em> species. We analysed 22 strains of 15 non-<em>S. aureus</em> species in the genus <em>Staphylococcus</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). KP was used as a control kit. In all strains analysed, while <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> values in the KP libraries were nearly even regardless of the GC content (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub><mi mathvariant="normal" xmlns=""> </mi></math></span>in the XT libraries drastically decreased as GC content decreased, as seen in <em>S. aureus</em>.</p>
<p>We quantified the overall GC content-associated sequencing bias by calculating the average deviation of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> (see Materials and methods) and performed a linear regression analysis of the relationship between overall GC content-associated bias and mean GC content of each genome (<a reveal-id="dsz017-F2" data-open="dsz017-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). These analyses revealed that, in all strains, the overall sequencing bias was much higher in XT libraries than in KP libraries. Moreover, the levels of overall sequencing bias correlated well with the mean GC content of genomes, particularly in the XT libraries; the regression coefficients of XT and KP were −0.034 (<em>P</em> value = 0.0010) and −0.0067 (<em>P</em> value = 0.0005), respectively. These results indicated that the strong GC content-associated sequencing bias introduced by XT is common to the members of the genus <em>Staphylococcus</em>. </p>                        <a id="164508248" scrollto-destination="164508248"></a>
<div id="dsz017-F2" data-id="dsz017-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f2.png?Expires=1576212293&amp;Signature=36Ez6VKvnTkwcZEeBOmbbR-5TT8jkyoiWKCiDsHcGxSURh6qcO6bKzu42MTV2KYoW6Y8J8lsQb-yiMPehePiayWIWibm~0Y-3cPV5Ld~xq6uhjMsRNXuQHG7-mbPRRFZUU1u3pj6bReD6zrr2CUOokGBgr9PtMZU0Hikpqzs6i5EyBLwOsbOKM4sqT9kpjYm14hf5oeHXmJLliv6nj2QRHf9kVSGy-49~CeOSJqc4JhqgV-aDYtBvXshNx6tAt6fgLZbwHAdnKssVYnPfATyLoQ~YiBvFSTWLT2HejkfxZWtGcyJ28dFOa2tegWeLCtqXYF1AOsIutCFleo0uUJUKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overall GC content-associated sequencing bias observed in 22 strains of non-S. aureus species in the genus Staphylococcus. Sequence reads were obtained from 22 strains of non-S. aureus species in the genus Staphylococcus using the XT and KP kits. The overall sequencing bias associated with GC content observed in the genome assemblies was quantified (see Materials and methods in the main text), and the relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown. Solid lines indicate regression lines, and the 95% confidence intervals are indicated in grey." path-from-xml="dsz017f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508248/dsz017f2.tif" path-from-xml="dsz017f2.tif" target="_blank">Open in new tab</a><a section="164508248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f2.png?Expires=1576212293&amp;Signature=yDqG~1rG07oe2zTtN921tP2FeJI~wsE5S9eXmbmq-O2-ThK6eXAP9x415Nqts6WZurC7ckWsSKbaNzNmakvMjXfb159zx7yJkLxcbsJpYJ6nQiTtYVlRDytquOxehjjLhl7SbObT6yTIzudDOVE01J-rH-2BbI1x-nOA44Gr0sN0H5g8KPU2eGb3ElZ9bjKiQjAFoCYcIjsIsB~l5qyPoa6wC7mxVLZgZzTGr4UC9wJbnkB3Gb71OnzTuMpHNm7u0YxeFgSg~dbtMZjXkq8OzJaxYn-kPUzAAS11fJHx4p-dmBj6YBvmuywfo99iilLXFl6ieksdk2pkPBuv9--NZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508248&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Overall GC content-associated sequencing bias observed in 22 strains of non-<em>S. aureus</em> species in the genus <em>Staphylococcus</em>. Sequence reads were obtained from 22 strains of non-<em>S. aureus</em> species in the genus <em>Staphylococcus</em> using the XT and KP kits. The overall sequencing bias associated with GC content observed in the genome assemblies was quantified (see Materials and methods in the main text), and the relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown. Solid lines indicate regression lines, and the 95% confidence intervals are indicated in grey.</p></div>
</div>
<div content-id="dsz017-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f2.png?Expires=1576212293&amp;Signature=36Ez6VKvnTkwcZEeBOmbbR-5TT8jkyoiWKCiDsHcGxSURh6qcO6bKzu42MTV2KYoW6Y8J8lsQb-yiMPehePiayWIWibm~0Y-3cPV5Ld~xq6uhjMsRNXuQHG7-mbPRRFZUU1u3pj6bReD6zrr2CUOokGBgr9PtMZU0Hikpqzs6i5EyBLwOsbOKM4sqT9kpjYm14hf5oeHXmJLliv6nj2QRHf9kVSGy-49~CeOSJqc4JhqgV-aDYtBvXshNx6tAt6fgLZbwHAdnKssVYnPfATyLoQ~YiBvFSTWLT2HejkfxZWtGcyJ28dFOa2tegWeLCtqXYF1AOsIutCFleo0uUJUKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overall GC content-associated sequencing bias observed in 22 strains of non-S. aureus species in the genus Staphylococcus. Sequence reads were obtained from 22 strains of non-S. aureus species in the genus Staphylococcus using the XT and KP kits. The overall sequencing bias associated with GC content observed in the genome assemblies was quantified (see Materials and methods in the main text), and the relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown. Solid lines indicate regression lines, and the 95% confidence intervals are indicated in grey." path-from-xml="dsz017f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508248/dsz017f2.tif" path-from-xml="dsz017f2.tif" target="_blank">Open in new tab</a><a section="164508248" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f2.png?Expires=1576212293&amp;Signature=yDqG~1rG07oe2zTtN921tP2FeJI~wsE5S9eXmbmq-O2-ThK6eXAP9x415Nqts6WZurC7ckWsSKbaNzNmakvMjXfb159zx7yJkLxcbsJpYJ6nQiTtYVlRDytquOxehjjLhl7SbObT6yTIzudDOVE01J-rH-2BbI1x-nOA44Gr0sN0H5g8KPU2eGb3ElZ9bjKiQjAFoCYcIjsIsB~l5qyPoa6wC7mxVLZgZzTGr4UC9wJbnkB3Gb71OnzTuMpHNm7u0YxeFgSg~dbtMZjXkq8OzJaxYn-kPUzAAS11fJHx4p-dmBj6YBvmuywfo99iilLXFl6ieksdk2pkPBuv9--NZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508248&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Overall GC content-associated sequencing bias observed in 22 strains of non-<em>S. aureus</em> species in the genus <em>Staphylococcus</em>. Sequence reads were obtained from 22 strains of non-<em>S. aureus</em> species in the genus <em>Staphylococcus</em> using the XT and KP kits. The overall sequencing bias associated with GC content observed in the genome assemblies was quantified (see Materials and methods in the main text), and the relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown. Solid lines indicate regression lines, and the 95% confidence intervals are indicated in grey.</p></div>
</div>                    <h3 scrollto-destination="164508249" id="164508249" class="section-title">3.3. Sequencing bias in 191 species representing a wide range of bacterial species</h3>
<p>We further investigated the overall GC content-associated sequencing bias introduced by XT across a wide range of bacterial species using 191 genomes from various species in the GTC collection (one strain from each species), which were sequenced by NBRP of Japan. These species were from Proteobacteria (<em>n</em> = 90), Actinobacteria (<em>n</em> = 40), Bacteroidetes (<em>n</em> = 11), Firmicutes (<em>n</em> = 47), and three other phyla (<em>n</em> = 3) and had a wide range of genome sizes (1.5–6.4 Mb) and GC content (25.7–71.9%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). As shown in <a reveal-id="dsz017-F3" data-open="dsz017-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span> for the data of each genome), although overall sequencing bias was observed in many species, a stronger bias was more frequently observed in the species with an extreme mean GC content. Particularly in low-GC species (&lt;40%), stronger bias was observed in all genomes, and the levels of bias correlated with the levels of GC content, as seen in the genus <em>Staphylococcus</em>. In many species with higher GC content, particularly those with extremely high GC content (&gt;65%), high levels of overall sequencing bias were also observed. In most of these species, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span> gradually decreased as the GC content increased, as opposed to the trend in low-GC species (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The results of this analysis indicated that XT introduces GC content-associated sequencing bias in a wide range of bacterial species, particularly those with extreme GC content, although sequencing bias in lower and higher GC regions may not be caused by the same mechanism. </p>                        <a id="164508251" scrollto-destination="164508251"></a>
<div id="dsz017-F3" data-id="dsz017-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f3.png?Expires=1576212293&amp;Signature=tePyG4q4IiOBebbVBxajX3yPvpf78oZwcyp3qssQcSAjHpgRfqbqTgli8UTBXjo~L4z3mo0DOdse~xNtpntuMcXDV1NTzsQ-RcmReS41eKg7gElkn7IeLGBVUKuxEFcDkGIDtjxN9ze8LLZP2pT5BB6pMNmQTKqWNTDS4hY8RxlJVbg25e2HUwf3sHOuD58T8VrPPeytS1GrPJ5numSX9O8EzC6mCf3aSDSAwGecRI5kcoCIvC~z1h-Jin6ITPd3WRzOUjMYeoS07W-pLP-OmxSIYpgEiwJZYVpsOLAPEaZOvNzaGp5zS3ieWcABX5LEuhoau2IcAXoWchE5xCmTSg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overall GC content-associated sequencing bias in the sequence data of 191 species obtained by the XT library preparation kit. Illumina sequencing data for 191 species (one strain from each species) produced using the XT kit from a project of NBRP of Japan were downloaded from the public database (DDBJ). The overall GC content-associated sequencing bias in each data set was quantified, and relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown." path-from-xml="dsz017f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508251/dsz017f3.tif" path-from-xml="dsz017f3.tif" target="_blank">Open in new tab</a><a section="164508251" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f3.png?Expires=1576212293&amp;Signature=QKOBwOe2mjFEPeuI2IkoLNF-AiYfC-uSsEsVubds4GyCoQqsBqQsyxBR36bx7nPuFvgIhCw3d22BSAR2CDTTjRPHZxv9WioJNpLQwVBb5-m162Nn1E6cjmM953gZYycysu~QNKCj8Zt5CuLzR-KR1whstSM9O~yZMEzk5AvY6d44AncAH8OjFWw5oGbirVLwn~HIejFh8YsoLVlsAYK1hr~q~-TccuWtwXFaGeeSJOeNjOyIDPAWPlMCT3AjO3WHYVMInJuw3IlLinxtfP-FTPrr5znJKG2scTLntC4ibDIGGWga0e-n7P1eo~PKKF1CwUswjKXYdpDwHvdergiwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508251&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Overall GC content-associated sequencing bias in the sequence data of 191 species obtained by the XT library preparation kit. Illumina sequencing data for 191 species (one strain from each species) produced using the XT kit from a project of NBRP of Japan were downloaded from the public database (DDBJ). The overall GC content-associated sequencing bias in each data set was quantified, and relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown.</p></div>
</div>
<div content-id="dsz017-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f3.png?Expires=1576212293&amp;Signature=tePyG4q4IiOBebbVBxajX3yPvpf78oZwcyp3qssQcSAjHpgRfqbqTgli8UTBXjo~L4z3mo0DOdse~xNtpntuMcXDV1NTzsQ-RcmReS41eKg7gElkn7IeLGBVUKuxEFcDkGIDtjxN9ze8LLZP2pT5BB6pMNmQTKqWNTDS4hY8RxlJVbg25e2HUwf3sHOuD58T8VrPPeytS1GrPJ5numSX9O8EzC6mCf3aSDSAwGecRI5kcoCIvC~z1h-Jin6ITPd3WRzOUjMYeoS07W-pLP-OmxSIYpgEiwJZYVpsOLAPEaZOvNzaGp5zS3ieWcABX5LEuhoau2IcAXoWchE5xCmTSg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overall GC content-associated sequencing bias in the sequence data of 191 species obtained by the XT library preparation kit. Illumina sequencing data for 191 species (one strain from each species) produced using the XT kit from a project of NBRP of Japan were downloaded from the public database (DDBJ). The overall GC content-associated sequencing bias in each data set was quantified, and relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown." path-from-xml="dsz017f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508251/dsz017f3.tif" path-from-xml="dsz017f3.tif" target="_blank">Open in new tab</a><a section="164508251" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f3.png?Expires=1576212293&amp;Signature=QKOBwOe2mjFEPeuI2IkoLNF-AiYfC-uSsEsVubds4GyCoQqsBqQsyxBR36bx7nPuFvgIhCw3d22BSAR2CDTTjRPHZxv9WioJNpLQwVBb5-m162Nn1E6cjmM953gZYycysu~QNKCj8Zt5CuLzR-KR1whstSM9O~yZMEzk5AvY6d44AncAH8OjFWw5oGbirVLwn~HIejFh8YsoLVlsAYK1hr~q~-TccuWtwXFaGeeSJOeNjOyIDPAWPlMCT3AjO3WHYVMInJuw3IlLinxtfP-FTPrr5znJKG2scTLntC4ibDIGGWga0e-n7P1eo~PKKF1CwUswjKXYdpDwHvdergiwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508251&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Overall GC content-associated sequencing bias in the sequence data of 191 species obtained by the XT library preparation kit. Illumina sequencing data for 191 species (one strain from each species) produced using the XT kit from a project of NBRP of Japan were downloaded from the public database (DDBJ). The overall GC content-associated sequencing bias in each data set was quantified, and relationships between the quantified overall sequencing bias and the mean GC content of each genome are shown.</p></div>
</div>                    <h3 scrollto-destination="164508252" id="164508252" class="section-title">3.4. Comparison of sequencing bias between the methods employing different fragmentation and amplification techniques</h3>
<p>To confirm the reproducibility of sequencing bias introduced by XT and to evaluate the effects of PCR amplification, we prepared genomic DNA from seven species that showed high levels of sequence bias in the analyses described above and prepared their sequencing libraries using four library preparation kits utilizing various fragmentation methods (XT, FL, KP, and TS) and two PCR-free kits (KPF and TSF). The seven analysed species were two low-GC species (<em>B. pilosicoli</em> and <em>S. moniliformis</em>), two intermediate-GC species (<em>S. liquefaciens</em> and <em>K. aerogenes</em>), and three high-GC species (<em>M. malmoense</em>, <em>M. interjectum</em>, and <em>T. pulmonis</em>). Analyses of the sequences obtained from each library revealed that strong GC content-associated sequencing bias was reproducibly observed in the XT libraries of the two low-GC species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). However, strong bias was not observed in the XT libraries of the two intermediate-GC species, although some bias was introduced in low-GC regions. In the three high-GC species, some levels of GC content-associated sequencing bias were detected in the XT libraries, but the observed bias was not as strong as that in the data set produced by NBRP of Japan. Although the reason(s) for the stronger bias observed in the NBRP data set is unknown, several factors, such as the accuracy of quantification of input DNA and lengths of input DNA, might have affected the quality of these libraries prepared with XT.</p>
<p>In the libraries prepared with other kits, including the recently released FL kit by Illumina, no strong bias was observed. Some GC content-associated sequencing bias was observed in the TS libraries of the two low-GC species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>), but no such bias was detected in the libraries prepared with TSF (PCR-free). This finding suggests that PCR amplification during library preparation can also introduce some levels of bias in low-GC regions. However, the effects are not as strong and are unlikely to cause serious problems in bacterial genome sequencing.</p>                    <h3 scrollto-destination="164508255" id="164508255" class="section-title">3.5. Analysis of the mock microbial community</h3>
<p>GC content-associated sequencing bias could be a more serious problem in metagenomics, where a wide range of microbial species with various degrees of GC content is sequenced. Therefore, we analysed the bias introduced in metagenome sequencing by XT and other kits (FL, KP, KPF, TS, and TSF) using a mock bacterial community DNA sample. The sample was composed of genomic DNA from the following nine bacteria: <em>S. aureus</em> (strain N315), <em>E. coli</em> (strain MG1655), and the seven species used in the analysis described above (<a reveal-id="dsz017-F4" data-open="dsz017-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). In the relative abundance analysis of the nine bacteria (<a reveal-id="dsz017-F4" data-open="dsz017-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), the abundance of three low-GC species, <em>S. aureus</em>, <em>B. pilosicoli</em>, and <em>S. moniliformis</em>, in the XT library was remarkably lower than the expected values, which were calculated and normalized based on their molecular ratios in the sample examined. Furthermore, in relative coverage analysis within each genome, strong GC content-associated sequencing bias was evident not only in the three low-GC species but also in other species (<a reveal-id="dsz017-F4" data-open="dsz017-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). </p>                        <a id="164508257" scrollto-destination="164508257"></a>
<div id="dsz017-F4" data-id="dsz017-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f4.png?Expires=1576212293&amp;Signature=fWhcOIwKPQEkyBgGZbuSf5S7LM2VMe4YQPXhOPVKgeq96U4SMGBFfEK0g0JE4AIgTnSQrfWR1OFG-Zl8ZWQURXTJtHGn9pswr8n75nZ0vOZ93hmKcoKqddokw6GH8PIn4deW2H78AizfC2Qdtib~BcWRhK1IrtzV4WSNr284eGpPr26riSVrk4UD968Q7JCISd79lnFTcLe2Byfz1MqVFn22EFIIuYyEmpOQDx85~~P5QQrCbse6c-n55XnCsE4c6vKzlkHZ3s72J9bIDQW~f7MPHr4WUTNGyzhvxMhHx85GRv3z83LQv2AKoj18z0iZtyq8sZ-B0h5m6IABRR36Fw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Metagenome sequencing of a mock bacterial community using six library preparation kits and the sequencing bias introduced by each kit. (A) Libraries of a mock bacterial community prepared by six library preparation kits were sequenced, and the relative genome abundance estimated in each data set obtained by six library preparation kits is shown. The mock community was composed of nine species with various levels of GC content. The relative abundances of each species were normalized by their genome sizes and the copy numbers of each species in the sample, which were determined by ddPCR. (B) Relationships between the GC content and sequence coverage in each genome in the mock community are shown. The mean relative coverage of each 200-bp bin with a given GC content (CGC) in each genome was calculated in each data set and is shown according to GC content by coloured lines. The colours of the lines correspond to the species shown in panel (A). Black horizontal lines in each plot (CGC=1) represent unbiased coverage. The relative coverage was normalized by the copy numbers in the sample determined by ddPCR. Data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at DNARES online." path-from-xml="dsz017f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508257/dsz017f4.tif" path-from-xml="dsz017f4.tif" target="_blank">Open in new tab</a><a section="164508257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f4.png?Expires=1576212293&amp;Signature=HLP92nRoY2VRVk02vNz~ui-6Vi3MP9mECXztb7JbhUY5js5UpcIoWEKEIlM-2t0dRnBeVX0COLmjduePT97RPVxJ3inh9xRNV42H4UPTwyuNRShMfWplv83UBybsjRqWVK0Wpj0UXP3e7ekUoemsdLtwpyJ22q9D0eew87seI9NNDaT3NMjZMM62zEZrP~1uGy2wl14RxD7fN0Fj64UvJ9~Zwq-wwIgCtYxDlUTppqicWyMII7x7EsO4pIMJ1C7SNUvTLMOorYqQSDNIk64yXR8t7-h7Kunrzcl5chzPgQRX6cUPqrdPATkBCrLotjyeBy-FYJiEXeWYsV2h88MEGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508257&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Metagenome sequencing of a mock bacterial community using six library preparation kits and the sequencing bias introduced by each kit. (A) Libraries of a mock bacterial community prepared by six library preparation kits were sequenced, and the relative genome abundance estimated in each data set obtained by six library preparation kits is shown. The mock community was composed of nine species with various levels of GC content. The relative abundances of each species were normalized by their genome sizes and the copy numbers of each species in the sample, which were determined by ddPCR. (B) Relationships between the GC content and sequence coverage in each genome in the mock community are shown. The mean relative coverage of each 200-bp bin with a given GC content (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math>⁠</span>) in each genome was calculated in each data set and is shown according to GC content by coloured lines. The colours of the lines correspond to the species shown in panel (A). Black horizontal lines in each plot (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span>=1) represent unbiased coverage. The relative coverage was normalized by the copy numbers in the sample determined by ddPCR. Data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsz017-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/m_dsz017f4.png?Expires=1576212293&amp;Signature=fWhcOIwKPQEkyBgGZbuSf5S7LM2VMe4YQPXhOPVKgeq96U4SMGBFfEK0g0JE4AIgTnSQrfWR1OFG-Zl8ZWQURXTJtHGn9pswr8n75nZ0vOZ93hmKcoKqddokw6GH8PIn4deW2H78AizfC2Qdtib~BcWRhK1IrtzV4WSNr284eGpPr26riSVrk4UD968Q7JCISd79lnFTcLe2Byfz1MqVFn22EFIIuYyEmpOQDx85~~P5QQrCbse6c-n55XnCsE4c6vKzlkHZ3s72J9bIDQW~f7MPHr4WUTNGyzhvxMhHx85GRv3z83LQv2AKoj18z0iZtyq8sZ-B0h5m6IABRR36Fw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Metagenome sequencing of a mock bacterial community using six library preparation kits and the sequencing bias introduced by each kit. (A) Libraries of a mock bacterial community prepared by six library preparation kits were sequenced, and the relative genome abundance estimated in each data set obtained by six library preparation kits is shown. The mock community was composed of nine species with various levels of GC content. The relative abundances of each species were normalized by their genome sizes and the copy numbers of each species in the sample, which were determined by ddPCR. (B) Relationships between the GC content and sequence coverage in each genome in the mock community are shown. The mean relative coverage of each 200-bp bin with a given GC content (CGC) in each genome was calculated in each data set and is shown according to GC content by coloured lines. The colours of the lines correspond to the species shown in panel (A). Black horizontal lines in each plot (CGC=1) represent unbiased coverage. The relative coverage was normalized by the copy numbers in the sample determined by ddPCR. Data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at DNARES online." path-from-xml="dsz017f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508257/dsz017f4.tif" path-from-xml="dsz017f4.tif" target="_blank">Open in new tab</a><a section="164508257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017f4.png?Expires=1576212293&amp;Signature=HLP92nRoY2VRVk02vNz~ui-6Vi3MP9mECXztb7JbhUY5js5UpcIoWEKEIlM-2t0dRnBeVX0COLmjduePT97RPVxJ3inh9xRNV42H4UPTwyuNRShMfWplv83UBybsjRqWVK0Wpj0UXP3e7ekUoemsdLtwpyJ22q9D0eew87seI9NNDaT3NMjZMM62zEZrP~1uGy2wl14RxD7fN0Fj64UvJ9~Zwq-wwIgCtYxDlUTppqicWyMII7x7EsO4pIMJ1C7SNUvTLMOorYqQSDNIk64yXR8t7-h7Kunrzcl5chzPgQRX6cUPqrdPATkBCrLotjyeBy-FYJiEXeWYsV2h88MEGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508257&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz017f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Metagenome sequencing of a mock bacterial community using six library preparation kits and the sequencing bias introduced by each kit. (A) Libraries of a mock bacterial community prepared by six library preparation kits were sequenced, and the relative genome abundance estimated in each data set obtained by six library preparation kits is shown. The mock community was composed of nine species with various levels of GC content. The relative abundances of each species were normalized by their genome sizes and the copy numbers of each species in the sample, which were determined by ddPCR. (B) Relationships between the GC content and sequence coverage in each genome in the mock community are shown. The mean relative coverage of each 200-bp bin with a given GC content (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math>⁠</span>) in each genome was calculated in each data set and is shown according to GC content by coloured lines. The colours of the lines correspond to the species shown in panel (A). Black horizontal lines in each plot (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi mathvariant="normal">GC</mi></mrow></msub></math></span>=1) represent unbiased coverage. The relative coverage was normalized by the copy numbers in the sample determined by ddPCR. Data for bins with extreme GC content (those representing &lt;0.5% of all 200 bp bins) are not shown. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<p>Other kits, including FL, did not introduce the strong bias observed in the XT library. However, a notable bias was observed in the TS library at regions with extreme GC content. The bias introduced by TS was reproducibly detected in a separate experiment (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Such a bias was not observed in the library prepared with TSF, which also employs sonication for fragmentation but does not include PCR amplification. This finding suggests that the bias observed in the TS libraries may be generated by a combination of some level of non-random DNA fragmentation during sonication and PCR amplification.</p>                    <h3 scrollto-destination="164508259" id="164508259" class="section-title">3.6. Possible mechanisms to generate sequencing bias</h3>
<p>The main mechanism to generate sequencing bias in XT libraries is probably uneven tagmentation affected by local GC-content, particularly AT-rich sequences, because the GC content-associated insertion bias of Tn5 transposase has been pointed out by several studies.<a reveal-id="dsz017-B29 dsz017-B30 dsz017-B31" data-open="dsz017-B29 dsz017-B30 dsz017-B31" class="link link-ref link-reveal xref-bibr"><sup>29–31</sup></a> It appears that less frequent insertion in GC-unbalanced regions than in balanced regions. As AT-richness increases, higher level of sequencing bias was observed in our data set. This observation suggests the gradual decrease in insertion frequency of Tn5 transposase according to the level of AT-richness, which resulted in the increase in sequencing bias in the genomes or genomic regions of bacterial strains with more extreme mean GC-contents as observed in the strains with &lt;40% GC content. No clear difference was observed between KP and KPF libraries. However, it appears that PCR amplification during library preparation also has some contribution to bias generation because some bias was observed in the TS libraries of low GC species but not in their TSF libraries (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576212293&amp;Signature=ZMHJBy83qU6OyET5rThPZOQxuxdgft2GNTzHTv7tq8BSACvikBbms3CpGy0yF0oMSf9ahRk0JG1igHwxH~keZYtFVjMr3H0K~5-74NsqPnxZFcMe4boDmkTXC~hEOmJH0BAxaZSGs2K0Alo-l~xM1wq1SOrb74kTbtF3RaNCuki0DudxEj34fymZ4MuM1L6zWoIZMgHDDmS-NUIhYURPvcdQxECc0xBjujmbG~iG4wVEL-pUIFp16I-ful8po9nyD0jp53VPkodkB5jpbkp~asECsW5nq4PrgF8sbrVrtZjFG1PAKFuYUbDvS99WUxBim7P9qw4cQbdk6ZBfqthzyQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). When single genomes were analysed, the effect was not so prominent, and practically not problematic. However, in the analysis of bacterial community DNA, significant bias was introduced by TS but not TSF. This bias observed in the TS library may be generated by a combination of some level of non-random DNA fragmentation during sonication and PCR amplification. Importantly, the problem of strong sequencing bias due to uneven tagmentation has been solved in FL by employing bead-linked transposases and by some other modifications,<a reveal-id="dsz017-B32" data-open="dsz017-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> although it is not open to users whether XT and FL use the same transposase or FL uses an improved enzyme.<a reveal-id="dsz017-B31" data-open="dsz017-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>                    <h2 scrollto-destination="164508261" id="164508261" class="section-title">4. Conclusion</h2>
<p>By systematic comparison of currently available library preparation kits for Illumina sequencing, we demonstrated that strong sequencing bias is introduced in low-GC regions by the Nextera XT kit. The level of bias introduced is dependent on the level of GC content; stronger bias is generated as the GC content decreases. More substantial GC content-associated sequencing bias was introduced by Nextera XT in metagenome sequencing of a mock bacterial community. Other kits, including the Nextera DNA Flex Library Prep Kit, a recently released kit from Illumina, did not introduce strong GC content-associated sequencing bias, but the TruSeq nano kit generated notable bias in regions with extreme GC content when used for metagenome sequencing. Our data indicate the importance of selecting proper library preparation kits according to the purposes and targets of genome sequencing, particularly sequencing of low-GC species and metagenome sequencing. Special attention should also be paid to which library preparation kit was used when analysing and interpreting publicly available data.</p>                    <h2 scrollto-destination="164508263" id="164508263" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>The authors thank M. Horiguchi, M. Kumagai, Y. Nagayoshi, Y. Morita, and K. Ozaki for providing technical assistance.</p>                    <h3 scrollto-destination="164508265" id="164508265" class="section-title">Funding</h3>
<p>This work was supported by the JSPS KAKENHI [221S0002 and 16H06279].</p>                    <h3 scrollto-destination="164508267" id="164508267" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="164508269" id="164508269" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz017-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Paszkiewicz</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Studholme</div> <div class="given-names">D.J.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">
<em>De novo</em> assembly of short sequence reads</div>, <div class="source">Brief. Bioinform</div> ., <div class="volume">11</div>, <div class="fpage">457</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20short%20sequence%20reads&amp;author=K.%20Paszkiewicz&amp;author=D.J.%20Studholme&amp;publication_year=2010&amp;journal=Brief.%20Bioinform&amp;volume=11&amp;pages=457-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bib/bbq020" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbq020" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbib%2Fbbq020"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20724458" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:De%20novo%20assembly%20of%20short%20sequence%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goodwin</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">McPherson</div> <div class="given-names">J.D.</div>
</div>, <div class="name string-name">
<div class="surname">McCombie</div> <div class="given-names">W.R.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Coming of age: ten years of next-generation sequencing technologies</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">17</div>, <div class="fpage">333</div>–<div class="lpage">51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;author=S.%20Goodwin&amp;author=J.D.%20McPherson&amp;author=W.R.%20McCombie&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Genet&amp;volume=17&amp;pages=333-51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg.2016.49" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2016.49" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg.2016.49"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27184599" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Metzker</div> <div class="given-names">M.L.</div>
</div></div> <div class="year">2010</div>, 
<div class="article-title">Sequencing technologies—the next generation</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">31</div>–<div class="lpage">46</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequencing%20technologies%E2%80%94the%20next%20generation&amp;author=M.L.%20Metzker&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=31-46" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nrg2626" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2626" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg2626"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997069" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequencing%20technologies%E2%80%94the%20next%20generation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Head</div> <div class="given-names">S.R.</div>
</div>, <div class="name string-name">
<div class="surname">Kiyomi Komori</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">LaMere</div> <div class="given-names">S.A.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Library construction for next-generation sequencing: overviews and challenges</div>, <div class="source">Biotechniques</div> , <div class="volume">56</div>, <div class="fpage">61</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Library%20construction%20for%20next-generation%20sequencing%3A%20overviews%20and%20challenges&amp;author=S.R.%20Head&amp;author=H.%20Kiyomi%20Komori&amp;author=S.A.%20LaMere&amp;publication_year=2014&amp;journal=Biotechniques&amp;volume=56&amp;pages=61-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2144/000114133" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000114133" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2144%2F000114133"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24502796" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Library%20construction%20for%20next-generation%20sequencing%3A%20overviews%20and%20challenges&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nascimento</div> <div class="given-names">F.S.</div>
</div>, <div class="name string-name">
<div class="surname">Wei-Pridgeon</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Arrowood</div> <div class="given-names">M.J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Evaluation of library preparation methods for Illumina next generation sequencing of small amounts of DNA from foodborne parasites</div>, <div class="source">J. Microbiol. Methods</div> , <div class="volume">130</div>, <div class="fpage">23</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20library%20preparation%20methods%20for%20Illumina%20next%20generation%20sequencing%20of%20small%20amounts%20of%20DNA%20from%20foodborne%20parasites&amp;author=F.S.%20Nascimento&amp;author=Y.%20Wei-Pridgeon&amp;author=M.J.%20Arrowood&amp;publication_year=2016&amp;journal=J.%20Microbiol.%20Methods&amp;volume=130&amp;pages=23-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.mimet.2016.08.020" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2016.08.020" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mimet.2016.08.020"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27553132" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluation%20of%20library%20preparation%20methods%20for%20Illumina%20next%20generation%20sequencing%20of%20small%20amounts%20of%20DNA%20from%20foodborne%20parasites&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Van Dijk</div> <div class="given-names">E.L.</div>
</div>, <div class="name string-name">
<div class="surname">Jaszczyszyn</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Thermes</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Library preparation methods for next-generation sequencing: tone down the bias</div>, <div class="source">Exp. Cell Res</div> ., <div class="volume">322</div>, <div class="fpage">12</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Library%20preparation%20methods%20for%20next-generation%20sequencing%3A%20tone%20down%20the%20bias&amp;author=E.L.%20Van%20Dijk&amp;author=Y.%20Jaszczyszyn&amp;author=C.%20Thermes&amp;publication_year=2014&amp;journal=Exp.%20Cell%20Res&amp;volume=322&amp;pages=12-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.yexcr.2014.01.008" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.yexcr.2014.01.008" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.yexcr.2014.01.008"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24440557" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Library%20preparation%20methods%20for%20next-generation%20sequencing%3A%20tone%20down%20the%20bias&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Adey</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Morrison</div> <div class="given-names">H.G.</div>
</div>, <div class="name string-name">
<div class="surname">Asan</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density <em>in vitro</em> transposition</div>, <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">R119</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rapid%2C%20low-input%2C%20low-bias%20construction%20of%20shotgun%20fragment%20libraries%20by%20high-density%20in%20vitro%20transposition&amp;author=A.%20Adey&amp;author=H.G.%20Morrison&amp;author=A.%20Asan&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R119" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2010-11-12-r119" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-12-r119" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2010-11-12-r119"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21143862" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rapid%2C%20low-input%2C%20low-bias%20construction%20of%20shotgun%20fragment%20libraries%20by%20high-density%20in%20vitro%20transposition&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hillier</div> <div class="given-names">L.W.</div>
</div>, <div class="name string-name">
<div class="surname">Marth</div> <div class="given-names">G.T.</div>
</div>, <div class="name string-name">
<div class="surname">Quinlan</div> <div class="given-names">A.R.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Whole-genome sequencing and variant discovery in <em>C. elegans</em>
</div>, <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">183</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20and%20variant%20discovery%20in%20C.%20elegans&amp;author=L.W.%20Hillier&amp;author=G.T.%20Marth&amp;author=A.R.%20Quinlan&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=183-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1179" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1179" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1179"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18204455" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Whole-genome%20sequencing%20and%20variant%20discovery%20in%20C.%20elegans&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Quail</div> <div class="given-names">M.A.</div>
</div>, <div class="name string-name">
<div class="surname">Kozarewa</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">A large genome center’s improvements to the Illumina sequencing system</div>, <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">1005</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20large%20genome%20center%E2%80%99s%20improvements%20to%20the%20Illumina%20sequencing%20system&amp;author=M.A.%20Quail&amp;author=I.%20Kozarewa&amp;author=F.%20Smith&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=1005-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1270" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1270" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1270"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19034268" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20large%20genome%20center%E2%80%99s%20improvements%20to%20the%20Illumina%20sequencing%20system&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kozarewa</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Ning</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Quail</div> <div class="given-names">M.A.</div>
</div>, <div class="name string-name">
<div class="surname">Sanders</div> <div class="given-names">M.J.</div>
</div>, <div class="name string-name">
<div class="surname">Berriman</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Turner</div> <div class="given-names">D.J.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes</div>, <div class="source">Nat. Methods</div> , <div class="volume">6</div>, <div class="fpage">291</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Amplification-free%20Illumina%20sequencing-library%20preparation%20facilitates%20improved%20mapping%20and%20assembly%20of%20(G%2BC)-biased%20genomes&amp;author=I.%20Kozarewa&amp;author=Z.%20Ning&amp;author=M.A.%20Quail&amp;author=M.J.%20Sanders&amp;author=M.%20Berriman&amp;author=D.J.%20Turner&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=291-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1311" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1311" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1311"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19287394" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Amplification-free%20Illumina%20sequencing-library%20preparation%20facilitates%20improved%20mapping%20and%20assembly%20of%20(G%2BC)-biased%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ross</div> <div class="given-names">M.G.</div>
</div>, <div class="name string-name">
<div class="surname">Russ</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Costello</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Characterizing and measuring bias in sequence data</div>, <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterizing%20and%20measuring%20bias%20in%20sequence%20data&amp;author=M.G.%20Ross&amp;author=C.%20Russ&amp;author=M.%20Costello&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2013-14-5-r51" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-5-r51" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2013-14-5-r51"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23718773" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterizing%20and%20measuring%20bias%20in%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zerbino</div> <div class="given-names">D.R.</div>
</div>, <div class="name string-name">
<div class="surname">Birney</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Velvet: algorithms for <em>de novo</em> short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">821</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.074492.107"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nurk</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Bankevich</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Antipov</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Assembling genomes and mini-metagenomes from highly chimeric reads</div>. In: Deng, M., Jiang, R., Sun, F., Zhang X. (eds) <em>Research in Computational Molecular Biology. RECOMB 2013</em>. <em>Lecture Notes in Computer Science</em>, Vol. 7821. <div class="fpage">158</div>–<div class="lpage">70</div>. Berlin, Heidelberg: Springer.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Assembling%20genomes%20and%20mini-metagenomes%20from%20highly%20chimeric%20reads&amp;aulast=Nurk&amp;date=2013&amp;spage=158&amp;epage=70" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assembling%20genomes%20and%20mini-metagenomes%20from%20highly%20chimeric%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Langmead</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1923"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Durbin</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp324"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Aird</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Ross</div> <div class="given-names">M.G.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">W.-S.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries</div>, <div class="source">Genome Biol</div> ., <div class="volume">12</div>, <div class="fpage">R18</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analyzing%20and%20minimizing%20PCR%20amplification%20bias%20in%20Illumina%20sequencing%20libraries&amp;author=D.%20Aird&amp;author=M.G.%20Ross&amp;author=W.-S.%20Chen&amp;publication_year=2011&amp;journal=Genome%20Biol&amp;volume=12&amp;pages=R18" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2011-12-2-r18" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-2-r18" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2011-12-2-r18"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21338519" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analyzing%20and%20minimizing%20PCR%20amplification%20bias%20in%20Illumina%20sequencing%20libraries&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oyola</div> <div class="given-names">S.O.</div>
</div>, <div class="name string-name">
<div class="surname">Otto</div> <div class="given-names">T.D.</div>
</div>, <div class="name string-name">
<div class="surname">Gu</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Optimizing Illumina next-generation sequencing library preparation for extremely at-biased genomes</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">1</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Optimizing%20Illumina%20next-generation%20sequencing%20library%20preparation%20for%20extremely%20at-biased%20genomes&amp;author=S.O.%20Oyola&amp;author=T.D.%20Otto&amp;author=Y.%20Gu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-13-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-13-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Optimizing%20Illumina%20next-generation%20sequencing%20library%20preparation%20for%20extremely%20at-biased%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tyler</div> <div class="given-names">A.D.</div>
</div>, <div class="name string-name">
<div class="surname">Christianson</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Knox</div> <div class="given-names">N.C.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Comparison of sample preparation methods used for the next-generation sequencing of <em>Mycobacterium tuberculosis</em>
</div>, <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0148676</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20sample%20preparation%20methods%20used%20for%20the%20next-generation%20sequencing%20of%20Mycobacterium%20tuberculosis&amp;author=A.D.%20Tyler&amp;author=S.%20Christianson&amp;author=N.C.%20Knox&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0148676-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0148676" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0148676" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0148676"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26849565" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparison%20of%20sample%20preparation%20methods%20used%20for%20the%20next-generation%20sequencing%20of%20Mycobacterium%20tuberculosis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jones</div> <div class="given-names">M.B.</div>
</div>, <div class="name string-name">
<div class="surname">Highlander</div> <div class="given-names">S.K.</div>
</div>, <div class="name string-name">
<div class="surname">Anderson</div> <div class="given-names">E.L.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Library preparation methodology can influence genomic and functional predictions in human microbiome research</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">14024</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Library%20preparation%20methodology%20can%20influence%20genomic%20and%20functional%20predictions%20in%20human%20microbiome%20research&amp;author=M.B.%20Jones&amp;author=S.K.%20Highlander&amp;author=E.L.%20Anderson&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=14024-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1519288112" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1519288112" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1519288112"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Library%20preparation%20methodology%20can%20influence%20genomic%20and%20functional%20predictions%20in%20human%20microbiome%20research&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Blattner</div> <div class="given-names">F.R.</div>
</div>, <div class="name string-name">
<div class="surname">Plunkett</div> <div class="given-names">I.I.G.</div>
</div>, <div class="name string-name">
<div class="surname">Bloch</div> <div class="given-names">C.A.</div>
</div>
</div>, et al.  <div class="year">1997</div>, 
<div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</div>, <div class="source">Science</div> , <div class="volume">277</div>, <div class="fpage">1453</div>–<div class="lpage">62</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;author=F.R.%20Blattner&amp;author=I.I.G.%20Plunkett&amp;author=C.A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-62" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.277.5331.1453"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hayashi</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Makino</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Ohnishi</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2001</div>, 
<div class="article-title">Complete genome sequence of enterohemorrhagic <em>Escherichia coli</em> O157: H7 and genomic comparison with a laboratory strain K-12</div>, <div class="source">DNA Res</div> ., <div class="volume">8</div>, <div class="fpage">11</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%3A%20H7%20and%20genomic%20comparison%20with%20a%20laboratory%20strain%20K-12&amp;author=T.%20Hayashi&amp;author=K.%20Makino&amp;author=M.%20Ohnishi&amp;publication_year=2001&amp;journal=DNA%20Res&amp;volume=8&amp;pages=11-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/8.1.11" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.1.11" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2F8.1.11"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11258796" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Complete%20genome%20sequence%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%3A%20H7%20and%20genomic%20comparison%20with%20a%20laboratory%20strain%20K-12&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Baba</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Takeuchi</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Kuroda</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2002</div>, 
<div class="article-title">Genome and virulence determinants of high virulence community-acquired MRSA</div>, <div class="source">Lancet</div> , <div class="volume">359</div>, <div class="fpage">1819</div>–<div class="lpage">27</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20virulence%20determinants%20of%20high%20virulence%20community-acquired%20MRSA&amp;author=T.%20Baba&amp;author=F.%20Takeuchi&amp;author=M.%20Kuroda&amp;publication_year=2002&amp;journal=Lancet&amp;volume=359&amp;pages=1819-27" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0140-6736(02)08713-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(02)08713-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0140-6736(02)08713-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12044378" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20and%20virulence%20determinants%20of%20high%20virulence%20community-acquired%20MRSA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kuroda</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Ohta</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Uchiyama</div> <div class="given-names">I.</div>
</div>
</div>, et al.  <div class="year">2001</div>, 
<div class="article-title">Whole genome sequencing of meticillin-resistant <em>Staphylococcus aureus</em>
</div>, <div class="source">Lancet</div> , <div class="volume">357</div>, <div class="fpage">1225</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20sequencing%20of%20meticillin-resistant%20Staphylococcus%20aureus&amp;author=M.%20Kuroda&amp;author=T.%20Ohta&amp;author=I.%20Uchiyama&amp;publication_year=2001&amp;journal=Lancet&amp;volume=357&amp;pages=1225-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0140-6736(00)04403-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(00)04403-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0140-6736(00)04403-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11418146" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Whole%20genome%20sequencing%20of%20meticillin-resistant%20Staphylococcus%20aureus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sugai</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Fujiwara</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Komatsuzawa</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Suginaka</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Identification and molecular characterization of a gene homologous to epr (endopeptidase resistance gene) in <em>Staphylococcus aureus</em>
</div>, <div class="source">Gene</div> , <div class="volume">224</div>, <div class="fpage">67</div>–<div class="lpage">75</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20molecular%20characterization%20of%20a%20gene%20homologous%20to%20epr%20(endopeptidase%20resistance%20gene)%20in%20Staphylococcus%20aureus&amp;author=M.%20Sugai&amp;author=T.%20Fujiwara&amp;author=H.%20Komatsuzawa&amp;author=H.%20Suginaka&amp;publication_year=1998&amp;journal=Gene&amp;volume=224&amp;pages=67-75" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0378-1119(98)00508-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(98)00508-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0378-1119(98)00508-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9931440" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20and%20molecular%20characterization%20of%20a%20gene%20homologous%20to%20epr%20(endopeptidase%20resistance%20gene)%20in%20Staphylococcus%20aureus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Luo</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Qian</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">FastUniq: a fast <em>de novo</em> duplicates removal tool for paired short reads</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e52249</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=FastUniq%3A%20a%20fast%20de%20novo%20duplicates%20removal%20tool%20for%20paired%20short%20reads&amp;author=H.%20Xu&amp;author=X.%20Luo&amp;author=J.%20Qian&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e52249" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0052249" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052249" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0052249"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23284954" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FastUniq%3A%20a%20fast%20de%20novo%20duplicates%20removal%20tool%20for%20paired%20short%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Marçais</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Kingsford</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr011"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Handsaker</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Wysoker</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp352"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Parks</div> <div class="given-names">D.H.</div>
</div>, <div class="name string-name">
<div class="surname">Imelfort</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Skennerton</div> <div class="given-names">C.T.</div>
</div>, <div class="name string-name">
<div class="surname">Hugenholtz</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Tyson</div> <div class="given-names">G.W.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes</div>, <div class="source">Genome Res</div> ., <div class="volume">25</div>, <div class="fpage">1043</div>–<div class="lpage">55</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&amp;author=D.H.%20Parks&amp;author=M.%20Imelfort&amp;author=C.T.%20Skennerton&amp;author=P.%20Hugenholtz&amp;author=G.W.%20Tyson&amp;publication_year=2015&amp;journal=Genome%20Res&amp;volume=25&amp;pages=1043-55" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.186072.114" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.186072.114" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.186072.114"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25977477" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Green</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Bouchier</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Fairhead</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Craig</div> <div class="given-names">N.L.</div>
</div>, <div class="name string-name">
<div class="surname">Cormack</div> <div class="given-names">B.P.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Insertion site preference of Mu, Tn5, and Tn7 transposons</div>, <div class="source">Mob. DNA</div> ., <div class="volume">3</div>, <div class="fpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insertion%20site%20preference%20of%20Mu%2C%20Tn5%2C%20and%20Tn7%20transposons&amp;author=B.%20Green&amp;author=C.%20Bouchier&amp;author=C.%20Fairhead&amp;author=N.L.%20Craig&amp;author=B.P.%20Cormack&amp;publication_year=2012&amp;journal=Mob.%20DNA&amp;volume=3&amp;pages=3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1759-8753-3-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1759-8753-3-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1759-8753-3-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22313799" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insertion%20site%20preference%20of%20Mu%2C%20Tn5%2C%20and%20Tn7%20transposons&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lan</div> <div class="given-names">J.H.</div>
</div>, <div class="name string-name">
<div class="surname">Yuxin Yin</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Elaine</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Reed</div> <div class="given-names">E.F.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Impact of three Illumina library construction methods on GC bias and HLA genotype calling</div>, <div class="source">Hum. Immunol</div> ., <div class="volume">76</div>, <div class="fpage">166</div>–<div class="lpage">75</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20three%20Illumina%20library%20construction%20methods%20on%20GC%20bias%20and%20HLA%20genotype%20calling&amp;author=J.H.%20Lan&amp;author=Y.%20Yuxin%20Yin&amp;author=F.%20Elaine&amp;author=E.F.%20Reed&amp;publication_year=2015&amp;journal=Hum.%20Immunol&amp;volume=76&amp;pages=166-75" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.humimm.2014.12.016" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.humimm.2014.12.016" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.humimm.2014.12.016"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25543015" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Impact%20of%20three%20Illumina%20library%20construction%20methods%20on%20GC%20bias%20and%20HLA%20genotype%20calling&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kai</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Gloeckner</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Osothprarop</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Improved genome sequencing using an engineered transposase</div>, <div class="source">BMC Biotechnol</div> ., <div class="volume">17</div>, <div class="fpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improved%20genome%20sequencing%20using%20an%20engineered%20transposase&amp;author=A.%20Kai&amp;author=C.%20Gloeckner&amp;author=T.%20Osothprarop&amp;publication_year=2017&amp;journal=BMC%20Biotechnol&amp;volume=17&amp;pages=6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12896-016-0326-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12896-016-0326-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12896-016-0326-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28095828" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improved%20genome%20sequencing%20using%20an%20engineered%20transposase&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz017-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz017-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bruinsma</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Burgess</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Schlingman</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Bead-linked transposomes enable anormalization-free workflow for NGS library preparation</div>, <div class="source">BMC Genomics</div> , <div class="volume">19</div>, <div class="fpage">722</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bead-linked%20transposomes%20enable%20anormalization-free%20workflow%20for%20NGS%20library%20preparation&amp;author=S.%20Bruinsma&amp;author=J.%20Burgess&amp;author=D.%20Schlingman&amp;publication_year=2018&amp;journal=BMC%20Genomics&amp;volume=19&amp;pages=722" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-018-5096-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-018-5096-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-018-5096-9"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30285621" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bead-linked%20transposomes%20enable%20anormalization-free%20workflow%20for%20NGS%20library%20preparation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5541856&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz017/1/dsz017_supplementary_data.zip?Expires=1576718525&amp;Signature=KwxxN~7OMNPftH9mwTe9Ar6KUnA6yA1RpAp845wgXG~8J4Duqdn~rYAyMTYOKxVoVW1nRVXIGrDlBjtnK5TN7vlRZmmWy86fdDMUM-Vvdzuye680lPXoLgParJVoll6EG56w2UjX1UfncQWwnn9I3VA29g-3dA8S4646Q7FSsCHJKhclFh15~UCYPMLEb4l3FwoLH1A5IJBdXH~cVxpCHFS~STGKJ5kYYIYh~NDY-zymD6IGFZBkLApI2S8S7rFBXI7nrMBREt1PwkXKVH-iiAQ8u6-YIECEWestoHtlBOkUNQfmKnVVru17AS~VlhVB8G0J4hx52JcF7gny0fhQ-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz017_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,364</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz017"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5541856"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f5%2f391%2f5541856">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F5%2F391%2F5541856" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5541856">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Comparison of the sequencing bias of currently available library preparation kits for Illumina sequencing of bacterial genomes and metagenomes | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz017");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f391%2f5541856">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
