<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean (Glycine max) | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/3/243/5474979","@type":"ScholarlyArticle","name":"Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean ( Glycine max )","datePublished":"2019-04-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2019-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz005","keywords":["AmpliSeq","target re-sequencing","genotyping","flowering time-related gene","soybean"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/3/243/5474979","author":[{"name":"Ogiso-Tanaka, Eri","affiliation":"Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Shimizu, Takehiko","affiliation":"Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Hajika, Makita","affiliation":"Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Kaga, Akito","affiliation":"Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Ishimoto, Masao","affiliation":"Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan","@type":"Person"}],"description":"Abstract.  Whole-genome re-sequencing is a powerful approach to detect gene variants, but it is expensive to analyse only the target genes. To circumvent this p","pageStart":"243","pageEnd":"260","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636255837&Signature=J15I9rU6fq9bOSJYNZQZisqHslwVEIE-~KeucjNjkVHnfhznCgE0iHEmH15nLo8AXjk7R2j6q3zkos3QjfNUlpOgz21Mo0p9yb5HEL9qqkTTOATFDgt2NK6S2veeTGxNSK-OovMNCztbEVns9lgEja-ykK-ufGCiTfSG2JjKyDjP4xhxsCRAfEEoc3rl9uhqo-l~3OER7mAnrXdmHCVN2YdmzXD4doGu-OQ71h-GLN~X3Y5-JMvGLh3bk2bfgmUKv~nYh2xLzff0aT0To-quWluaUx~N6UD3k2DQ11Tja68q5RgBicG6BTPWXhuVEHpCQmdrRiHRplKpe-GSPH~lCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean ( Glycine max )","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636255837&Signature=J15I9rU6fq9bOSJYNZQZisqHslwVEIE-~KeucjNjkVHnfhznCgE0iHEmH15nLo8AXjk7R2j6q3zkos3QjfNUlpOgz21Mo0p9yb5HEL9qqkTTOATFDgt2NK6S2veeTGxNSK-OovMNCztbEVns9lgEja-ykK-ufGCiTfSG2JjKyDjP4xhxsCRAfEEoc3rl9uhqo-l~3OER7mAnrXdmHCVN2YdmzXD4doGu-OQ71h-GLN~X3Y5-JMvGLh3bk2bfgmUKv~nYh2xLzff0aT0To-quWluaUx~N6UD3k2DQ11Tja68q5RgBicG6BTPWXhuVEHpCQmdrRiHRplKpe-GSPH~lCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean ( Glycine max )">
<meta property="og:description" content="Abstract.  Whole-genome re-sequencing is a powerful approach to detect gene variants, but it is expensive to analyse only the target genes. To circumvent this p">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/3/243/5474979">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636255837&amp;Signature=J15I9rU6fq9bOSJYNZQZisqHslwVEIE-~KeucjNjkVHnfhznCgE0iHEmH15nLo8AXjk7R2j6q3zkos3QjfNUlpOgz21Mo0p9yb5HEL9qqkTTOATFDgt2NK6S2veeTGxNSK-OovMNCztbEVns9lgEja-ykK-ufGCiTfSG2JjKyDjP4xhxsCRAfEEoc3rl9uhqo-l~3OER7mAnrXdmHCVN2YdmzXD4doGu-OQ71h-GLN~X3Y5-JMvGLh3bk2bfgmUKv~nYh2xLzff0aT0To-quWluaUx~N6UD3k2DQ11Tja68q5RgBicG6BTPWXhuVEHpCQmdrRiHRplKpe-GSPH~lCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-04-16">
<meta name="citation_author" content="Ogiso-Tanaka, Eri">
<meta name="citation_author_institution" content="Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Shimizu, Takehiko">
<meta name="citation_author_institution" content="Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Hajika, Makita">
<meta name="citation_author_institution" content="Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Kaga, Akito">
<meta name="citation_author_institution" content="Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Ishimoto, Masao">
<meta name="citation_author_institution" content="Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan">
<meta name="citation_title" content="Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean (Glycine max)">
<meta name="citation_firstpage" content="243">
<meta name="citation_lastpage" content="260">
<meta name="citation_doi" content="10.1093/dnares/dsz005">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="3">
<meta name="citation_publication_date" content="2019/06/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Regulation of seed yield and agronomic characters by photoperiod sensitivity and growth habit genes in soybean;  citation_journal_title=Theor. Appl. Genet;  citation_year=2010;  citation_volume=120;  citation_pages=1005-12; ">
<meta name="citation_reference" content="citation_title=QTL identification of yield-related traits and their association with flowering and maturity in soybean;  citation_journal_title=J. Crop Sci. Biotechnol;  citation_year=2011;  citation_volume=14;  citation_pages=65-70; ">
<meta name="citation_reference" content="citation_title=Natural variation in the genes responsible for maturity loci E1, E2, E3 and E4 in soybean;  citation_journal_title=Ann. Bot;  citation_year=2014;  citation_volume=113;  citation_pages=429-41; ">
<meta name="citation_reference" content="citation_title=Two major genes for time of flowering and maturity in soybean;  citation_journal_title=Crop Sci;  citation_year=1971;  citation_volume=11;  citation_pages=242-4; ">
<meta name="citation_reference" content="citation_title=Inheritance of a soybean flowering response to fluorescent daylength conditions;  citation_journal_title=Can. J. Genet. Cytol;  citation_year=1971;  citation_volume=13;  citation_pages=703-7; ">
<meta name="citation_reference" content="citation_title=Inheritance of insensitivity to long daylength;  citation_journal_title=Soyb. Genet. Newsl;  citation_year=1980;  citation_volume=7;  citation_pages=26-9; ">
<meta name="citation_reference" content="citation_title=A new soybean maturity and photoperiod-sensitivity locus linked to E1 and T;  citation_journal_title=Crop Sci;  citation_year=2001;  citation_volume=41;  citation_pages=698-701; ">
<meta name="citation_reference" content="citation_title=A new dominant gene E9 conditions early flowering and maturity in soybean;  citation_journal_title=Crop Sci;  citation_year=2014;  citation_volume=54;  citation_pages=2529-35; ">
<meta name="citation_reference" content="citation_title=Mapping and identification of a potential candidate gene for a novel maturity locus, E10, in soybean;  citation_journal_title=Theor. Appl. Genet;  citation_year=2017;  citation_volume=130;  citation_pages=377-90; ">
<meta name="citation_reference" content="citation_title=Positional cloning and characterization reveal the molecular basis for soybean maturity locus E1 that regulates photoperiodic flowering,;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=E2155-64; ">
<meta name="citation_reference" content="citation_title=A map-based cloning strategy employing a residual heterozygous line reveals that the GIGANTEA gene is involved in soybean maturity and flowering;  citation_journal_title=Genetics;  citation_year=2011;  citation_volume=188;  citation_pages=395-407; ">
<meta name="citation_reference" content="citation_title=Map-based cloning of the gene associated with the soybean maturity locus E3;  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=182;  citation_pages=1251-62; ">
<meta name="citation_reference" content="citation_title=Genetic redundancy in soybean photoresponses associated with duplication of the phytochrome A gene;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=180;  citation_pages=995-1007; ">
<meta name="citation_reference" content="citation_title=Genetic variation in four maturity genes affects photoperiod insensitivity and PHYA-regulated post-flowering responses of soybean;  citation_journal_title=BMC Plant Biol;  citation_year=2013;  citation_volume=13;  citation_pages=91">
<meta name="citation_reference" content="citation_title=Genetic variation in soybean at the maturity locus E4 is involved in adaptation to long days at high latitudes;  citation_journal_title=Agronomy;  citation_year=2013;  citation_volume=3;  citation_pages=117-34; ">
<meta name="citation_reference" content="citation_title=Two coordinately regulated homologs of FLOWERING LOCUS T are involved in the control of photoperiodic flowering in soybean;  citation_journal_title=Plant Physiol;  citation_year=2010;  citation_volume=154;  citation_pages=1220-31; ">
<meta name="citation_reference" content="citation_title=A recessive allele for delayed flowering at the soybean maturity locus E9 is a leaky allele of FT2a, a FLOWERING LOCUS T ortholog;  citation_journal_title=BMC Plant Biol;  citation_year=2016;  citation_volume=16;  citation_pages=20">
<meta name="citation_reference" content="citation_title=A soybean quantitative trait locus that promotes flowering under long days is identified as FT5a, a FLOWERING LOCUS T ortholog;  citation_journal_title=J. Exp. Bot;  citation_year=2016;  citation_volume=67;  citation_pages=5247-58; ">
<meta name="citation_reference" content=" Ion AmpliSeq Designer. https://ampliseq.com/help/startDesign.action (25 March 2019, date last accessed) ">
<meta name="citation_reference" content="AmpliSeq protocol. https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0006735_AmpliSeq_DNA_RNA_LibPrep_UG.pdf. ">
<meta name="citation_reference" content="citation_title=Evaluation of soybean germplasm conserved in NIAS genebank and development of mini core collections;  citation_journal_title=Breed. Sci;  citation_year=2012;  citation_volume=61;  citation_pages=566-92; ">
<meta name="citation_reference" content="citation_title=QTL mapping for salt tolerance and domestication-related traits in Vigna marina subsp. oblonga, a halophytic species;  citation_journal_title=Theor. Appl. Genet;  citation_year=2014;  citation_volume=127;  citation_pages=691-702; ">
<meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; ">
<meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3;  citation_journal_title=Fly (Austin);  citation_year=2012;  citation_volume=6;  citation_pages=80-92; ">
<meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation;  citation_journal_title=Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=308-11; ">
<meta name="citation_reference" content="citation_publisher=R Foundation for Statistical Computing, Vienna, Austria;  citation_title=R: A Language and Environment for Statistical Computing;  citation_year=2017; ">
<meta name="citation_reference" content="Troubleshooting Guides. https://www.ampliseq.com/help/troubleshooting.action. ">
<meta name="citation_reference" content="citation_title=Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=408-14; ">
<meta name="citation_reference" content="citation_title=Ectopic expression reveals a conserved PHYB homolog in soybean;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e27737">
<meta name="citation_reference" content="citation_title=GmFT2a, a soybean homolog of FLOWERING LOCUS T, is involved in flowering transition and maintenance;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e29238">
<meta name="citation_reference" content="citation_title=Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei;  citation_journal_title=DNA Res;  citation_year=2018;  citation_volume=25;  citation_pages=123-36; ">
<meta name="citation_reference" content="citation_title=Hd3a, a rice ortholog of the Arabidopsis FT gene, promotes transition to flowering downstream of Hd1 under short-day conditions;  citation_journal_title=Plant Cell Physiol;  citation_year=2002;  citation_volume=43;  citation_pages=1096-105; ">
<meta name="citation_reference" content="citation_title=Natural variation of the RICE FLOWERING LOCUS T 1 contributes to flowering time divergence in rice;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e75959">
<meta name="citation_reference" content="citation_title=The soybean stem growth habit gene Dt1 is an ortholog of Arabidopsis terminal flower1;  citation_journal_title=Plant Physiol;  citation_year=2010;  citation_volume=153;  citation_pages=198-210; ">
<meta name="citation_reference" content="citation_title=A putative CENTRORADIALIS/TERMINAL FLOWER 1-like gene, Ljcen1, plays a role in phase transition in Lotus japonicus;  citation_journal_title=J. Plant Physiol;  citation_year=2006;  citation_volume=163;  citation_pages=436-44; ">
<meta name="citation_reference" content="citation_title=DETERMINATE and LATE FLOWERING are two TERMINAL FLOWER1/CENTRORADIALIS homologs that control two distinct phases of flowering initiation and development in pea;  citation_journal_title=Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=2742-54; ">
<meta name="citation_reference" content="citation_title=Inflorescence commitment and architecture in Arabidopsis;  citation_journal_title=Science;  citation_year=1997;  citation_volume=275;  citation_pages=80-3; ">
<meta name="citation_reference" content="citation_title=Functional divergence of the TFL1-like gene family in Arabidopsis revealed by characterization of a novel homologue;  citation_journal_title=Genes Cells;  citation_year=2001;  citation_volume=6;  citation_pages=327-36; ">
<meta name="citation_reference" content="citation_title=Control of inflorescence architecture in Antirrhinum;  citation_journal_title=Nature;  citation_year=1996;  citation_volume=379;  citation_pages=791-7; ">
<meta name="citation_reference" content="citation_title=Ultra-dense SNP genetic map construction and identification of SiDt gene controlling the determinate growth habit in Sesamum indicum L;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=31556">
<meta name="citation_reference" content="citation_title=Arabidopsis clock-associated pseudo-response regulators PRR9, PRR7 and PRR5 coordinately and positively regulate flowering time through the canonical CONSTANS-dependent photoperiodic pathway;  citation_journal_title=Plant Cell Physiol;  citation_year=2007;  citation_volume=48;  citation_pages=822-32; ">
<meta name="citation_reference" content="citation_title=Casein kinases I and 2α phosphorylate Oryza sativa pseudo-response regulator 37 (OsPRR37) in photoperiodic flowering in rice;  citation_journal_title=Mol. Cells;  citation_year=2015;  citation_volume=38;  citation_pages=81-8; ">
<meta name="citation_reference" content="citation_title=The pseudo-response regulator Ppd-H1 provides adaptation to photoperiod in 799 barley;  citation_journal_title=Science;  citation_year=2005;  citation_volume=310;  citation_pages=1031-4; ">
<meta name="citation_reference" content="citation_title=Identification of additional QTLs for flowering time by removing the effect of the maturity gene E1 in soybean;  citation_journal_title=J. Integr. Agric;  citation_year=2016;  citation_volume=15;  citation_pages=42-9; ">
<meta name="citation_reference" content="citation_title=Control of lateral organ development and flowering time by the Arabidopsis thaliana MADS-box gene AGAMOUS-LIKE6;  citation_journal_title=Plant J;  citation_year=2010;  citation_volume=62;  citation_pages=807-16; ">
<meta name="citation_reference" content="citation_title=AGAMOUS-LIKE 6 is a floral promoter that negatively regulates the FLC/MAF clade genes and positively regulates FT in Arabidopsis;  citation_journal_title=Plant J;  citation_year=2011;  citation_volume=65;  citation_pages=62-76; ">
<meta name="citation_reference" content="citation_title=The WD-repeat protein superfamily in Arabidopsis: conservation and divergence in structure and function;  citation_journal_title=BMC Genomics;  citation_year=2003;  citation_volume=4;  citation_pages=50">
<meta name="citation_reference" content="citation_title=Allelic variations at four major maturity E genes and transcriptional abundance of the E1 gene are associated with flowering time and maturity of soybean cultivars;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e97636">
<meta name="citation_reference" content="citation_title=Genome-wide association study for flowering time, maturity dates and plant height in early maturing soybean (Glycine max) germplasm;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=217">
<meta name="citation_reference" content="citation_title=Two genes affecting stem termination in soybeans;  citation_journal_title=Crop Sci;  citation_year=1972;  citation_volume=12;  citation_pages=235-9; ">
<meta name="citation_reference" content="citation_title=Artificial selection for determinate growth habit in soybean;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=8563-8; ">
<meta name="citation_reference" content="citation_title=Conserved CO-FT regulons contribute to the photoperiod flowering control in soybean;  citation_journal_title=BMC Plant Biol;  citation_year=2014;  citation_volume=14;  citation_pages=9">
<meta name="citation_reference" content="citation_title=Molecular mechanisms of flowering under long days and stem growth habit in soybean;  citation_journal_title=J. Exp. Bot;  citation_year=2017;  citation_volume=68;  citation_pages=1873-84; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/3/243/28855592/dsz005.pdf">
<meta name="description" content="Abstract.  Whole-genome re-sequencing is a powerful approach to detect gene variants, but it is expensive to analyse only the target genes. To circumvent this p">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/3/243/5474979">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/3/243/5474979">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/3/243/5474979");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsz005';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f243%2f5474979" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f243%2f5474979" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115841"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1576706106&amp;Signature=fgMnyvVYW-N0hiRIUrbAiFxkaF5udsL5J6YtQeQB3EuivI38Vwd4-T8v6cweN~2RZPIskmCsYI9YfLbYWoMEqAqW20kj1Bww4Z5k1TQApn6jsedqUpYQxmH4lEfokLp5ukgsBB8QCx1Pwpr-ZJbWgn-6uPaCn6WMBxV8lSF2CD2J3bOeGiiHFzTB8VisJ4VJlloJMdf~mSk8wudKGZ129q9Vu4xrVcY0g9zFFt0Q-8IQRu6xLJxi2bfxMR0wKCPzjYO-IKNIWkx7Kl0QswaF3VwdZaged2vkiIHAuHFPkimCYXo091ahTGbPp5gCdaidjGS5OIY-s0SPT1ct7dxeVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/3">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="137025697">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025697">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025699">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025699">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025705">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025705">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025722">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025722">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025771">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025771">4. Conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025773">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025773">Data availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025775">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025775">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="137025781">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025781">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/3/231/5430876">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/3/261/5476584">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean (<em>Glycine max</em>)
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eri Ogiso-Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eri+Ogiso-Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogiso-Tanaka%20E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ogiso-Tanaka%20Eri%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Eri Ogiso-Tanaka</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shimizu%20Takehiko%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Takehiko Shimizu</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makita Hajika</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makita+Hajika">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hajika%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hajika%20Makita%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Makita Hajika</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akito Kaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel./Fax. +81 29 838 7452. Email: <a href="mailto:kaga@affrc.go.jp" target="_blank">kaga@affrc.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akito+Kaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kaga%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kaga%20Akito%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Akito Kaga</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masao Ishimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Crop Science (NICS), NARO (National Agriculture and Food Research Organization), 2-1-2 Kannondai, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masao+Ishimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishimoto%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ishimoto%20Masao%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Masao Ishimoto</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 243–260, <a href="https://doi.org/10.1093/dnares/dsz005">https://doi.org/10.1093/dnares/dsz005</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">16 April 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">17 December 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">11 March 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">16 April 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5474979" href="/dnaresearch/article-pdf/26/3/243/28855592/dsz005.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Eri Ogiso-Tanaka, Takehiko Shimizu, Makita Hajika, Akito Kaga, Masao Ishimoto,  Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean (<em>Glycine max</em>), <em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 243–260, <a href="https://doi.org/10.1093/dnares/dsz005">https://doi.org/10.1093/dnares/dsz005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5474979&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Highly%20multiplexed%20AmpliSeq%20technology%20identifies%20novel%20variation%20of%20flowering%20time-related%20genes%20in%20soybean%20(%26lt%3Bem%26gt%3BGlycine%20max%26lt%3B%2Fem%26gt%3B)&amp;publicationDate=2019-04-16&amp;volumeNum=26&amp;issueNum=3&amp;author=Ogiso-Tanaka%2C%20Eri%3B%20Shimizu%2C%20Takehiko&amp;startPage=243&amp;endPage=260&amp;contentId=10.1093%2Fdnares%2Fdsz005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f243%2f5474979" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="137025697" id="137025697" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Whole-genome re-sequencing is a powerful approach to detect gene variants, but it is expensive to analyse only the target genes. To circumvent this problem, we attempted to detect novel variants of flowering time-related genes and their homologues in soybean mini-core collection by target re-sequencing using AmpliSeq technology. The average depth of 382 amplicons targeting 29 genes was 1,237 with 99.85% of the sequence data mapped to the reference genome. Totally, 461 variants were detected, of which 150 sites were novel and not registered in dbSNP. Known and novel variants were detected in the classical maturity loci—<em>E1</em>, <em>E2</em>, <em>E3</em>, and <em>E4</em>. Additionally, large indel alleles, <em>E1-nl</em> and <em>E3-tr</em>, were successfully identified. Novel loss-of-function and missense variants were found in <em>FT2a</em>, <em>MADS-box</em>, <em>WDR61</em>, <em>phytochromes</em>, and <em>two-component response regulators</em>. The multiple regression analysis showed that four genes<em>—E2</em>, <em>E3</em>, <em>Dt1</em>, and <em>two-component response regulator</em>—can explain 51.1–52.3% of the variation in flowering time of the mini-core collection. Among them, the two-component response regulator with a premature stop codon is a novel gene that has not been reported as a soybean flowering time-related gene. These data suggest that the AmpliSeq technology is a powerful tool to identify novel alleles.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword="AmpliSeq">AmpliSeq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"target re-sequencing"'>target re-sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genotyping">genotyping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"flowering time-related gene"'>flowering time-related gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="soybean">soybean</a>
</div>                    <h2 scrollto-destination="137025699" id="137025699" class="section-title">1. Introduction</h2>
<p>Flowering time is critical for successful seed production by plants. Flowering time and maturity are the most important traits to determine the adaptability of soybean [<em>Glycine max</em> (L.) Merr.] cultivation. These not only restrict the cultivation area but also greatly affect plant architecture and yield.<a reveal-id="dsz005-B1" data-open="dsz005-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsz005-B2" data-open="dsz005-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Therefore, it is necessary to clarify the genetic factors affecting flowering time and maturity and control them using a combination of alleles with different genetic effects on flowering time. To combine such alleles freely based on DNA marker-assisted selection, a catalogue of alleles for breeding materials will be necessary. Soybean is a typical short-day plant. Several functional nucleotide polymorphisms responsible for diversity in flowering time among cultivars are already known.<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Classical maturity loci designated as <em>E</em> loci have been characterized, including <em>E1</em> and <em>E2</em>,<a reveal-id="dsz005-B4" data-open="dsz005-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>E3</em>,<a reveal-id="dsz005-B5" data-open="dsz005-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><em>E4</em>,<a reveal-id="dsz005-B6" data-open="dsz005-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><em>E7</em>,<a reveal-id="dsz005-B7" data-open="dsz005-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><em>E8</em>,<a reveal-id="dsz005-B1" data-open="dsz005-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><em>E9</em>,<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and <em>E10.</em><a reveal-id="dsz005-B9" data-open="dsz005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Of these, <em>E1</em>,<a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><em>E2</em>,<a reveal-id="dsz005-B11" data-open="dsz005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><em>E3</em>,<a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>E4</em>,<a reveal-id="dsz005-B13" data-open="dsz005-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and <em>E9</em><a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> have been isolated as flowering time-related genes. <em>E1</em> encodes putative transcriptional factor containing plant-specific B3 domain.<a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><em>E2</em> encodes a homologue of GIGANTEA.<a reveal-id="dsz005-B11" data-open="dsz005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><em>E3</em> and <em>E4</em> encode a homologue of the photoreceptor phytochrome A (PHYA).<a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>E9</em> encodes the florigen protein FT2a.<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p>
<p>In the <em>E1</em> gene, three alleles, namely <em>e1-as</em> (=<em>e1</em> designated by Bernard<a reveal-id="dsz005-B4" data-open="dsz005-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a>), <em>e1-fs</em>, and <em>e1-nl</em>, have been reported as early flowering phenotype under long-day conditions.<a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The <em>e1-as</em> allele has a single missense mutation (Arg15Thr) in the coding region. The <em>e1-as</em> genotype promotes flowering for ∼10 days compared with that by the <em>E1</em> genotype under natural day-length conditions at Matsudo, Japan (35°78′N, 139°90′E). The <em>e1-fs</em> allele has a 1-bp deletion, resulting in a premature stop codon in the cultivar Sakamotowase.<a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><em>e1-nl</em> is a null allele in which ∼142 kb, including the entire <em>E1</em> gene, is deleted in some early flowering cultivars.<a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In contrast, only one <em>e2</em> allele has been reported in the <em>E2</em> gene. The <em>e2</em> allele has one premature stop codon mutation due to single nucleotide polymorphism (SNP) in the 10th exon.<a reveal-id="dsz005-B11" data-open="dsz005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The <em>e2</em> genotype promotes flowering for ∼9 days under natural day-length conditions at Tsukuba, Japan (36°03′N, 140°04′E).<a reveal-id="dsz005-B11" data-open="dsz005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> In the <em>E3</em> gene, <em>e3-Mo</em>, <em>e3-fs</em>, <em>e3-tr</em> (=<em>e3</em> designated by Buzzell<a reveal-id="dsz005-B5" data-open="dsz005-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a>), and <em>e3-ns</em> have been reported as nonfunctional alleles.<a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The <em>e3-Mo</em> alleles have SNP for a non-synonymous amino acid substitution (G1050R) in the third exon.<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The <em>e3-fs</em> allele has a single base insertion in the exon, resulting in frameshift mutation.<a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The <em>e3-ns</em> allele has a nonsense mutation in which a single nucleotide substitution in exon 3 creates a stop codon.<a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The <em>e3-tr</em> alleles lack a 13.33-kb genomic region including a part of exons 3 and 4.<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> These nonfunctional alleles promote flowering under long-day conditions. In addition, <em>E3-Mi</em> and <em>E3-Ha</em> also have been reported as functional alleles.<a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The <em>E3-Mi</em> alleles have a 2.633-kb deletion in the third intron. As for the <em>E4</em> gene, five nonfunctional alleles, viz., <em>e4-SORE-1</em>, <em>e4-kam</em>, <em>e4-kes</em>, <em>e4-oto</em>, and <em>e4-tsu</em>, have been reported.<a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsz005-B15" data-open="dsz005-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The <em>e4-SORE-1</em> (<em>=e4</em> designated by Buzzell and Voldeng<a reveal-id="dsz005-B6" data-open="dsz005-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a>) alleles have a <em>Ty1/copia</em>-like retrotransposon (<em>SORE-1</em>) insertion in first exon, resulting in a nonfunctional allele.</p>
<p>These <em>E1–E4</em> genes can result in variation in flowering time by controlling the expression of <em>FLOWERING LOCUS T</em> (<em>FT</em>) genes, <em>FT2a</em> and <em>FT5a.</em><a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsz005-B11" data-open="dsz005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsz005-B14" data-open="dsz005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsz005-B16" data-open="dsz005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The florigen protein FT2a is encoded by <em>E9</em>. The <em>e9</em> allele has a <em>SORE-1</em> insertion in the first intron. Although eight SNPs and six InDels in the <em>E9</em> have been reported, the influence on gene function is unknown.<a reveal-id="dsz005-B17" data-open="dsz005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The <em>FT5a</em> gene was identified as <em>qDTF-J</em>, which promotes the flowering time for ∼5 days under natural day-length conditions at Hokkaido, Japan (43°07′N, 141°35′E).<a reveal-id="dsz005-B18" data-open="dsz005-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> In the <em>FT5a</em> gene, 13 SNPs and 3 InDels are reported only in the promoter and untranslated regions (UTRs) in 439 cultivated and wild soybean accessions.</p>
<p>The functional nucleotide polymorphisms of the four <em>E</em> genes (<em>E1</em>–<em>E4</em>) are useful to predict the flowering time and could explain ∼62–66% of the phenotypic variation in flowering time among 63 Japanese accessions under long-day conditions.<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, prediction of flowering time will be difficult if the breeding materials have unknown alleles affecting flowering phenotype. Therefore, development of a sequencing system that can easily capture as many alleles as possible is required. Recently, it became possible to obtain whole-genome information easily with the development of next-generation sequencing (NGS) technologies. However, it is still expensive for re-sequencing large genomes. Moreover, it is necessary to have analytical and storage environments to deal with enormous amounts of whole-genome sequence data of genetic resources. Target re-sequence is one of the alternative sequencing methods to obtain sequence data of a limited region, which can minimize cost and time for data analysis and decrease data storage. The AmpliSeq technology (Thermo Fisher Scientific, Waltham, MA, USA) is one of the target re-sequencing technologies, a multiplex polymerase chain reaction (PCR)-based assay targeting regions of interest. The AmpliSeq Designer<a reveal-id="dsz005-B19" data-open="dsz005-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> designs primer set that amplifies PCR products ranging from 75 to 375 bp in the target region, and multiplex-PCR products are sequenced by NGS. The method enables amplification of ∼6,000 amplicons by ultra-high multiplex PCR and constructs a targeted sequencing library in 10 h.<a reveal-id="dsz005-B20" data-open="dsz005-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> In routine genotyping of crop breeding, NGS-based techniques need to meet several criteria. The processing time between sample collection and interpretation of sequencing should be short. Furthermore, it is necessary to construct libraries using limited amount of input DNA including partially degraded DNA sample and the read depth must be deep enough to detect variant accurately. The Ion Torrent platform<a reveal-id="dsz005-B19" data-open="dsz005-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsz005-B20" data-open="dsz005-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> in combination with the AmpliSeq multiplex PCR can use DNA input of as low as 10 ng, and the processing time between sample collection and sequence analysis can be finished within 5 days.<a reveal-id="dsz005-B20" data-open="dsz005-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The AmpliSeq technology is frequently used for studying human inherited cancer, but it can also be applied to plant and agronomic research.</p>
<p>In this study, we applied AmpliSeq technology to clarify the alleles of flowering time-related genes and their homologues in diverse soybean germplasm to identify novel and known variations associated with flowering time.</p>                    <h2 scrollto-destination="137025705" id="137025705" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="137025706" id="137025706" class="section-title">2.1. Plant materials and DNA extraction</h3>
<p>DNA was extracted from 192 accessions of a soybean mini-core collection, provided by Genebank, NARO<a reveal-id="dsz005-B21" data-open="dsz005-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Of these, 122 accessions were sown and germinated in plastic pots on rock wool material ‘grodan’ (Nittobo, Tokyo, Japan) moistened with water. After 10 days under 12 h light/12 h dark conditions at 25°C, the first leaf was collected in a 2-ml tube. The leaf tissue of 38 samples was ground in liquid nitrogen and CTAB buffer, and then immediately used for DNA extraction manually.<a reveal-id="dsz005-B22" data-open="dsz005-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The remaining 84 samples were dried in a freeze dryer (FDU-2100, EYELA, Tokyo, Japan). These samples were lyophilized at −80°C for 12 h and stored at 4°C. The dried leaves were crushed using a ShakeMaster (Bio Medical Science Inc. Tokyo, Japan) and the leaf powder was used to extract DNA. DNA from 44 samples was extracted using the CTAB DNA extraction kit (NR-502, KURABO, Osaka, Japan) and DNA extraction robot PE-480 (GENE PREP STAR, KURABO). DNA from another 40 samples was extracted by the bead-based method of the BioSprint 96 DNA Plant Kit on robotic workstation (QIAGEN, Hilden, Germany). From the other 70 samples, DNA was extracted from the seed tissue using the BioSprint 96 DNA Plant Kit on robotic workstation, according to the manufacturer’s instruction (QIAGEN). The seed tissue samples were obtained by scraping dried seed and crushing using Zirconia beads and TissueLyser II (QIAGEN). The quality of extracted DNA was evaluated based on the DNA integrity number, which is an index showing the fragmentation degree of DNA using TapeStation (Agilent Technologies, Santa Clara, CA, USA). The DNA concentration was measured using the Qubit Fluorometer (Thermo Fisher Scientific) by exciting at 485 nm and measuring the fluorescence intensity at 520 nm. The instrument was calibrated with the Quant-iT dsDNA BR Assay kit (Thermo Fisher Scientific), according to the manufacturer’s instructions.</p>                    <h3 scrollto-destination="137025708" id="137025708" class="section-title">2.2. Ion AmpliSeq custom panel design</h3>
<p>A custom panel targeting 29 genes was designed based on Soybean reference genome version 1.1<a reveal-id="dsz005-B23" data-open="dsz005-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> using the Ion AmpliSeq Designer tool<a reveal-id="dsz005-B19" data-open="dsz005-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> version 1.2.9 using the standard DNA (125–275 bp amplicon target sizes) option. Two primer pools were designed to amplify 382 amplicons, covering 29 target genes of total length 64.98 kb (<a reveal-id="dsz005-T1" data-open="dsz005-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S3</a></span>). These included the coding regions of <em>B3 domain containing genes</em> (<em>E1</em> and homologue), <em>Phytochrome A genes</em> (<em>E3 and E4</em>), <em>Phytochrome B genes</em>, <em>FT/TERMINAL FLOWER 1</em> (<em>TFL1</em>) <em>family</em> genes (including <em>FT2a</em>, <em>FT5a</em>, <em>and Dt1</em>), <em>two-component response regulator</em>-like genes, MADS box gene, WD repeat-containing gene (<em>WDR61</em>), Achaete-scute transcription factor gene, and a part of the exon of <em>GIGANTEA</em> (<em>E2)</em>.</p>                        <a id="137025710" scrollto-destination="137025710"></a>
<div content-id="dsz005-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T1" class="table-wrap-title" data-id="dsz005-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Genomic regions or SNP targeted by the AmpliSeq design</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>  </th>
<th colspan="2">Gmax275 (ver. 2.0)<hr> </th>
<th>  </th>
<th>  </th>
<th colspan="3">Gmax189 (ver. 1.1)<hr> </th>
<th rowspan="2">Description<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Gene name<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Target </th>
<th rowspan="2">Total designed target length of amplicons<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">Target size<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th rowspan="2">Number of amplicon<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th>
<th rowspan="2">Coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> (%) </th>
<th rowspan="2">Reference </th>
</tr>
<tr>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>2 </td>
<td>6116379 </td>
<td>6117379 </td>
<td>Glyma02g07650 </td>
<td>2 </td>
<td>6041770 </td>
<td>6042409 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>Exon + UTR </td>
<td>861 </td>
<td>639 </td>
<td>6 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>3 </td>
<td>40522597 </td>
<td>40525110 </td>
<td>Glyma03g35250 </td>
<td>3 </td>
<td>42534078 </td>
<td>42535865 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>Exon + UTR </td>
<td>1,577 </td>
<td>1,003 </td>
<td>9 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>3 </td>
<td>42918771 </td>
<td>42923401 </td>
<td>Glyma03g38620 </td>
<td>3 </td>
<td>44925730 </td>
<td>44930360 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>Exon + UTR </td>
<td>3,688 </td>
<td>2,876 </td>
<td>20 </td>
<td>98.6 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.04G143300<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>4 </td>
<td>26120011 </td>
<td>26120532 </td>
<td>Glyma18g22670 </td>
<td>18 </td>
<td>25739929 </td>
<td>25740831 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1Lb</em> </td>
<td>Exon + UTR </td>
<td>1,037 </td>
<td>902 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>4 </td>
<td>36758125 </td>
<td>36758770 </td>
<td>Glyma04g24640 </td>
<td>4 </td>
<td>28293933 </td>
<td>28294806 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1La</em> </td>
<td>Exon + UTR </td>
<td>967 </td>
<td>873 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>6 </td>
<td>20207077 </td>
<td>20207940 </td>
<td>Glyma06g23026 </td>
<td>6 </td>
<td>20006928 </td>
<td>20007814 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1</em> </td>
<td>Exon + UTR </td>
<td>1,059 </td>
<td>886 </td>
<td>6 </td>
<td>100.0 </td>
<td>3, 4, 11 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>8 </td>
<td>47458142 </td>
<td>47459829 </td>
<td>Glyma08g47810 </td>
<td>8 </td>
<td>46606934 </td>
<td>46608654 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>Exon + UTR </td>
<td>1,300 </td>
<td>746 </td>
<td>8 </td>
<td>100.0 </td>
<td>20 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>8 </td>
<td>47472881 </td>
<td>47473362 </td>
<td>Glyma08g47823 </td>
<td>8 </td>
<td>46621704 </td>
<td>46622185 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>Exon + UTR </td>
<td>612 </td>
<td>265 </td>
<td>3 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>9 </td>
<td>2960395 </td>
<td>2967229 </td>
<td>Glyma09g03990 </td>
<td>9 </td>
<td>2919887 </td>
<td>2926740 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>Exon + UTR </td>
<td>5,150 </td>
<td>4,336 </td>
<td>27 </td>
<td>99.7 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>9 </td>
<td>35652219 </td>
<td>35653967 </td>
<td>Glyma09g26550 </td>
<td>9 </td>
<td>33049107 </td>
<td>33050904 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>Gene </td>
<td>1,813 </td>
<td>1,122 </td>
<td>11 </td>
<td>98.7 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>10 </td>
<td>37489560 </td>
<td>37495624 </td>
<td>Glyma10g28170 </td>
<td>10 </td>
<td>36962521 </td>
<td>36968813 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,212 </td>
<td>4,065 </td>
<td>29 </td>
<td>94.6 </td>
<td>14 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>10 </td>
<td>45294735 </td>
<td>45316121 </td>
<td>Glyma10g36600 </td>
<td>10 </td>
<td>44732730 </td>
<td> </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>SNP </td>
<td>222 </td>
<td>1 </td>
<td>1 </td>
<td>100.0 </td>
<td>3, 4, 12 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>12 </td>
<td>5508365 </td>
<td>5522772 </td>
<td>Glyma12g07861 </td>
<td>12 </td>
<td>5496565 </td>
<td>5511828 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>4,116 </td>
<td>2,871 </td>
<td>24 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>15 </td>
<td>11435551 </td>
<td>11442683 </td>
<td>Glyma15g14980 </td>
<td>15 </td>
<td>11415495 </td>
<td>11422656 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,445 </td>
<td>4,542 </td>
<td>28 </td>
<td>98.1 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>16 </td>
<td>4135885 </td>
<td>4137742 </td>
<td>Glyma16g04830 </td>
<td>16 </td>
<td>4115033 </td>
<td>4116923 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>Exon + UTR </td>
<td>1,683 </td>
<td>1,109 </td>
<td>9 </td>
<td>98.0 </td>
<td>15, 19, 37 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>16 </td>
<td>4162525 </td>
<td>4164824 </td>
<td>Glyma16g04840 </td>
<td>16 </td>
<td>4141774 </td>
<td>4144073 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>Exon + UTR </td>
<td>1,031 </td>
<td>486 </td>
<td>6 </td>
<td>92.7 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>16 </td>
<td>31109999 </td>
<td>31114963 </td>
<td>Glyma16g26660 </td>
<td>16 </td>
<td>30741660 </td>
<td>30746677 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Exon + UTR </td>
<td>1,515 </td>
<td>935 </td>
<td>9 </td>
<td>99.4 </td>
<td>9, 15, 18, 33, 37 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>16 </td>
<td>31148829 </td>
<td>31151842 </td>
<td>Glyma16g26690 </td>
<td>16 </td>
<td>30780496 </td>
<td>30783509 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>Exon + UTR </td>
<td>860 </td>
<td>464 </td>
<td>5 </td>
<td>88.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>16 </td>
<td>35777815 </td>
<td>35779317 </td>
<td>Glyma16g32080 </td>
<td>16 </td>
<td>35274147 </td>
<td>35275762 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>Exon + UTR </td>
<td>1,554 </td>
<td>1,038 </td>
<td>6 </td>
<td>99.0 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>16 </td>
<td>36179891 </td>
<td>36187469 </td>
<td>Glyma16g32540 </td>
<td>16 </td>
<td>35676581 </td>
<td>35684221 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>2,534 </td>
<td>1,095 </td>
<td>15 </td>
<td>98.8 </td>
<td>48, 49 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>17 </td>
<td>3955518 </td>
<td>3958432 </td>
<td>Glyma17g05990 </td>
<td>17 </td>
<td>4225839 </td>
<td>4228888 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Exon + UTR </td>
<td>1,743 </td>
<td>1,318 </td>
<td>9 </td>
<td>100.0 </td>
<td>50 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>17 </td>
<td>7052506 </td>
<td>7053858 </td>
<td>Glyma17g09810 </td>
<td>17 </td>
<td>7317271 </td>
<td>7318756 </td>
<td>Achaete-scute transcription factor-related<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>i</sup></a> </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>1,594 </td>
<td>1,166 </td>
<td>9 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>19 </td>
<td>36030632 </td>
<td>36032867 </td>
<td>Glyma19g28390 </td>
<td>19 </td>
<td>35849981 </td>
<td>35852216 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>Exon + UTR </td>
<td>952 </td>
<td>463 </td>
<td>5 </td>
<td>100.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>19 </td>
<td>36049111 </td>
<td>36051851 </td>
<td>Glyma19g28400 </td>
<td>19 </td>
<td>35868460 </td>
<td>35871203 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>Exon + UTR </td>
<td>1,113 </td>
<td>560 </td>
<td>6 </td>
<td>90.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>19 </td>
<td>45183357 </td>
<td>45185175 </td>
<td>Glyma19g37890 </td>
<td>19 </td>
<td>44979743 </td>
<td>44981657 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Exon + UTR </td>
<td>1,411 </td>
<td>1,078 </td>
<td>8 </td>
<td>100.0 </td>
<td>37, 53 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>19 </td>
<td>47633059 </td>
<td>47641958 </td>
<td>Glyma19g41210 </td>
<td>19 </td>
<td>47511095 </td>
<td>47520052 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Exon + UTR </td>
<td>5,235 </td>
<td>4,400 </td>
<td>27 </td>
<td>98.6 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>19 </td>
<td>50364718 </td>
<td>50369677 </td>
<td>Glyma19g44970 </td>
<td>19 </td>
<td>50244046 </td>
<td>50249070 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>Gene </td>
<td>5,673 </td>
<td>4,941 </td>
<td>31 </td>
<td>98.3 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>20 </td>
<td>33236018 </td>
<td>33241692 </td>
<td>Glyma20g22160 </td>
<td>20 </td>
<td>32087412 </td>
<td>32093306 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>Exon + UTR </td>
<td>4,871 </td>
<td>4,076 </td>
<td>25 </td>
<td>99.2 </td>
<td>3, 14 </td>
</tr>
<tr>
<td>Glyma.U034500<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>Scaffold_32 </td>
<td>197150 </td>
<td>220019 </td>
<td>Glyma11g15580 </td>
<td>11 </td>
<td>11232271 </td>
<td>11255186 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,352 </td>
<td>3,833 </td>
<td>30 </td>
<td>98.1 </td>
<td>  </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>  </th>
<th colspan="2">Gmax275 (ver. 2.0)<hr> </th>
<th>  </th>
<th>  </th>
<th colspan="3">Gmax189 (ver. 1.1)<hr> </th>
<th rowspan="2">Description<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Gene name<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Target </th>
<th rowspan="2">Total designed target length of amplicons<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">Target size<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th rowspan="2">Number of amplicon<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th>
<th rowspan="2">Coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> (%) </th>
<th rowspan="2">Reference </th>
</tr>
<tr>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>2 </td>
<td>6116379 </td>
<td>6117379 </td>
<td>Glyma02g07650 </td>
<td>2 </td>
<td>6041770 </td>
<td>6042409 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>Exon + UTR </td>
<td>861 </td>
<td>639 </td>
<td>6 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>3 </td>
<td>40522597 </td>
<td>40525110 </td>
<td>Glyma03g35250 </td>
<td>3 </td>
<td>42534078 </td>
<td>42535865 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>Exon + UTR </td>
<td>1,577 </td>
<td>1,003 </td>
<td>9 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>3 </td>
<td>42918771 </td>
<td>42923401 </td>
<td>Glyma03g38620 </td>
<td>3 </td>
<td>44925730 </td>
<td>44930360 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>Exon + UTR </td>
<td>3,688 </td>
<td>2,876 </td>
<td>20 </td>
<td>98.6 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.04G143300<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>4 </td>
<td>26120011 </td>
<td>26120532 </td>
<td>Glyma18g22670 </td>
<td>18 </td>
<td>25739929 </td>
<td>25740831 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1Lb</em> </td>
<td>Exon + UTR </td>
<td>1,037 </td>
<td>902 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>4 </td>
<td>36758125 </td>
<td>36758770 </td>
<td>Glyma04g24640 </td>
<td>4 </td>
<td>28293933 </td>
<td>28294806 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1La</em> </td>
<td>Exon + UTR </td>
<td>967 </td>
<td>873 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>6 </td>
<td>20207077 </td>
<td>20207940 </td>
<td>Glyma06g23026 </td>
<td>6 </td>
<td>20006928 </td>
<td>20007814 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1</em> </td>
<td>Exon + UTR </td>
<td>1,059 </td>
<td>886 </td>
<td>6 </td>
<td>100.0 </td>
<td>3, 4, 11 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>8 </td>
<td>47458142 </td>
<td>47459829 </td>
<td>Glyma08g47810 </td>
<td>8 </td>
<td>46606934 </td>
<td>46608654 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>Exon + UTR </td>
<td>1,300 </td>
<td>746 </td>
<td>8 </td>
<td>100.0 </td>
<td>20 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>8 </td>
<td>47472881 </td>
<td>47473362 </td>
<td>Glyma08g47823 </td>
<td>8 </td>
<td>46621704 </td>
<td>46622185 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>Exon + UTR </td>
<td>612 </td>
<td>265 </td>
<td>3 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>9 </td>
<td>2960395 </td>
<td>2967229 </td>
<td>Glyma09g03990 </td>
<td>9 </td>
<td>2919887 </td>
<td>2926740 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>Exon + UTR </td>
<td>5,150 </td>
<td>4,336 </td>
<td>27 </td>
<td>99.7 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>9 </td>
<td>35652219 </td>
<td>35653967 </td>
<td>Glyma09g26550 </td>
<td>9 </td>
<td>33049107 </td>
<td>33050904 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>Gene </td>
<td>1,813 </td>
<td>1,122 </td>
<td>11 </td>
<td>98.7 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>10 </td>
<td>37489560 </td>
<td>37495624 </td>
<td>Glyma10g28170 </td>
<td>10 </td>
<td>36962521 </td>
<td>36968813 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,212 </td>
<td>4,065 </td>
<td>29 </td>
<td>94.6 </td>
<td>14 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>10 </td>
<td>45294735 </td>
<td>45316121 </td>
<td>Glyma10g36600 </td>
<td>10 </td>
<td>44732730 </td>
<td> </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>SNP </td>
<td>222 </td>
<td>1 </td>
<td>1 </td>
<td>100.0 </td>
<td>3, 4, 12 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>12 </td>
<td>5508365 </td>
<td>5522772 </td>
<td>Glyma12g07861 </td>
<td>12 </td>
<td>5496565 </td>
<td>5511828 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>4,116 </td>
<td>2,871 </td>
<td>24 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>15 </td>
<td>11435551 </td>
<td>11442683 </td>
<td>Glyma15g14980 </td>
<td>15 </td>
<td>11415495 </td>
<td>11422656 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,445 </td>
<td>4,542 </td>
<td>28 </td>
<td>98.1 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>16 </td>
<td>4135885 </td>
<td>4137742 </td>
<td>Glyma16g04830 </td>
<td>16 </td>
<td>4115033 </td>
<td>4116923 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>Exon + UTR </td>
<td>1,683 </td>
<td>1,109 </td>
<td>9 </td>
<td>98.0 </td>
<td>15, 19, 37 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>16 </td>
<td>4162525 </td>
<td>4164824 </td>
<td>Glyma16g04840 </td>
<td>16 </td>
<td>4141774 </td>
<td>4144073 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>Exon + UTR </td>
<td>1,031 </td>
<td>486 </td>
<td>6 </td>
<td>92.7 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>16 </td>
<td>31109999 </td>
<td>31114963 </td>
<td>Glyma16g26660 </td>
<td>16 </td>
<td>30741660 </td>
<td>30746677 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Exon + UTR </td>
<td>1,515 </td>
<td>935 </td>
<td>9 </td>
<td>99.4 </td>
<td>9, 15, 18, 33, 37 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>16 </td>
<td>31148829 </td>
<td>31151842 </td>
<td>Glyma16g26690 </td>
<td>16 </td>
<td>30780496 </td>
<td>30783509 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>Exon + UTR </td>
<td>860 </td>
<td>464 </td>
<td>5 </td>
<td>88.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>16 </td>
<td>35777815 </td>
<td>35779317 </td>
<td>Glyma16g32080 </td>
<td>16 </td>
<td>35274147 </td>
<td>35275762 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>Exon + UTR </td>
<td>1,554 </td>
<td>1,038 </td>
<td>6 </td>
<td>99.0 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>16 </td>
<td>36179891 </td>
<td>36187469 </td>
<td>Glyma16g32540 </td>
<td>16 </td>
<td>35676581 </td>
<td>35684221 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>2,534 </td>
<td>1,095 </td>
<td>15 </td>
<td>98.8 </td>
<td>48, 49 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>17 </td>
<td>3955518 </td>
<td>3958432 </td>
<td>Glyma17g05990 </td>
<td>17 </td>
<td>4225839 </td>
<td>4228888 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Exon + UTR </td>
<td>1,743 </td>
<td>1,318 </td>
<td>9 </td>
<td>100.0 </td>
<td>50 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>17 </td>
<td>7052506 </td>
<td>7053858 </td>
<td>Glyma17g09810 </td>
<td>17 </td>
<td>7317271 </td>
<td>7318756 </td>
<td>Achaete-scute transcription factor-related<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>i</sup></a> </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>1,594 </td>
<td>1,166 </td>
<td>9 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>19 </td>
<td>36030632 </td>
<td>36032867 </td>
<td>Glyma19g28390 </td>
<td>19 </td>
<td>35849981 </td>
<td>35852216 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>Exon + UTR </td>
<td>952 </td>
<td>463 </td>
<td>5 </td>
<td>100.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>19 </td>
<td>36049111 </td>
<td>36051851 </td>
<td>Glyma19g28400 </td>
<td>19 </td>
<td>35868460 </td>
<td>35871203 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>Exon + UTR </td>
<td>1,113 </td>
<td>560 </td>
<td>6 </td>
<td>90.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>19 </td>
<td>45183357 </td>
<td>45185175 </td>
<td>Glyma19g37890 </td>
<td>19 </td>
<td>44979743 </td>
<td>44981657 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Exon + UTR </td>
<td>1,411 </td>
<td>1,078 </td>
<td>8 </td>
<td>100.0 </td>
<td>37, 53 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>19 </td>
<td>47633059 </td>
<td>47641958 </td>
<td>Glyma19g41210 </td>
<td>19 </td>
<td>47511095 </td>
<td>47520052 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Exon + UTR </td>
<td>5,235 </td>
<td>4,400 </td>
<td>27 </td>
<td>98.6 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>19 </td>
<td>50364718 </td>
<td>50369677 </td>
<td>Glyma19g44970 </td>
<td>19 </td>
<td>50244046 </td>
<td>50249070 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>Gene </td>
<td>5,673 </td>
<td>4,941 </td>
<td>31 </td>
<td>98.3 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>20 </td>
<td>33236018 </td>
<td>33241692 </td>
<td>Glyma20g22160 </td>
<td>20 </td>
<td>32087412 </td>
<td>32093306 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>Exon + UTR </td>
<td>4,871 </td>
<td>4,076 </td>
<td>25 </td>
<td>99.2 </td>
<td>3, 14 </td>
</tr>
<tr>
<td>Glyma.U034500<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>Scaffold_32 </td>
<td>197150 </td>
<td>220019 </td>
<td>Glyma11g15580 </td>
<td>11 </td>
<td>11232271 </td>
<td>11255186 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,352 </td>
<td>3,833 </td>
<td>30 </td>
<td>98.1 </td>
<td>  </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>Gene description from Phytozome 12.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Gene name refers to Kong et al.,<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Wu et al.,<a reveal-id="dsz005-B29" data-open="dsz005-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Fan et al.,<a reveal-id="dsz005-B52" data-open="dsz005-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and Cao et al.<a reveal-id="dsz005-B53" data-open="dsz005-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">c</div>
<p>Total size of amplified region by designed amplicon primers.</p>
</div></div>
<div content-id="tblfn4"><div class="fn">
<div class="label">d</div>
<p>Target size (bp) of amplicon based on soybean genome version 1.1 (Gmax189).</p>
</div></div>
<div content-id="tblfn5"><div class="fn">
<div class="label">e</div>
<p>Total number of designed amplicons on target gene.</p>
</div></div>
<div content-id="tblfn6"><div class="fn">
<div class="label">f</div>
<p>Percentage of target region covered by amplicon.</p>
</div></div>
<div content-id="tblfn7"><div class="fn">
<div class="label">g</div>
<p>Glyma.04G143300 and Glyma.U034500on Gmax275 genome version were different chromosome positions on Gmax189.</p>
</div></div>
<div content-id="tblfn8"><div class="fn">
<div class="label">h</div>
<p>The gene annotation manually curated.</p>
</div></div>
<div content-id="tblfn9"><div class="fn">
<div class="label">i</div>
<p>‘Achaete-scute transcription factor related’ gene was included as control for variant detection.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025710" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T1" class="table-wrap-title" data-id="dsz005-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Genomic regions or SNP targeted by the AmpliSeq design</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th>  </th>
<th colspan="2">Gmax275 (ver. 2.0)<hr> </th>
<th>  </th>
<th>  </th>
<th colspan="3">Gmax189 (ver. 1.1)<hr> </th>
<th rowspan="2">Description<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Gene name<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Target </th>
<th rowspan="2">Total designed target length of amplicons<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">Target size<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th rowspan="2">Number of amplicon<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th>
<th rowspan="2">Coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> (%) </th>
<th rowspan="2">Reference </th>
</tr>
<tr>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>2 </td>
<td>6116379 </td>
<td>6117379 </td>
<td>Glyma02g07650 </td>
<td>2 </td>
<td>6041770 </td>
<td>6042409 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>Exon + UTR </td>
<td>861 </td>
<td>639 </td>
<td>6 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>3 </td>
<td>40522597 </td>
<td>40525110 </td>
<td>Glyma03g35250 </td>
<td>3 </td>
<td>42534078 </td>
<td>42535865 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>Exon + UTR </td>
<td>1,577 </td>
<td>1,003 </td>
<td>9 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>3 </td>
<td>42918771 </td>
<td>42923401 </td>
<td>Glyma03g38620 </td>
<td>3 </td>
<td>44925730 </td>
<td>44930360 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>Exon + UTR </td>
<td>3,688 </td>
<td>2,876 </td>
<td>20 </td>
<td>98.6 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.04G143300<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>4 </td>
<td>26120011 </td>
<td>26120532 </td>
<td>Glyma18g22670 </td>
<td>18 </td>
<td>25739929 </td>
<td>25740831 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1Lb</em> </td>
<td>Exon + UTR </td>
<td>1,037 </td>
<td>902 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>4 </td>
<td>36758125 </td>
<td>36758770 </td>
<td>Glyma04g24640 </td>
<td>4 </td>
<td>28293933 </td>
<td>28294806 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1La</em> </td>
<td>Exon + UTR </td>
<td>967 </td>
<td>873 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>6 </td>
<td>20207077 </td>
<td>20207940 </td>
<td>Glyma06g23026 </td>
<td>6 </td>
<td>20006928 </td>
<td>20007814 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1</em> </td>
<td>Exon + UTR </td>
<td>1,059 </td>
<td>886 </td>
<td>6 </td>
<td>100.0 </td>
<td>3, 4, 11 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>8 </td>
<td>47458142 </td>
<td>47459829 </td>
<td>Glyma08g47810 </td>
<td>8 </td>
<td>46606934 </td>
<td>46608654 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>Exon + UTR </td>
<td>1,300 </td>
<td>746 </td>
<td>8 </td>
<td>100.0 </td>
<td>20 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>8 </td>
<td>47472881 </td>
<td>47473362 </td>
<td>Glyma08g47823 </td>
<td>8 </td>
<td>46621704 </td>
<td>46622185 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>Exon + UTR </td>
<td>612 </td>
<td>265 </td>
<td>3 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>9 </td>
<td>2960395 </td>
<td>2967229 </td>
<td>Glyma09g03990 </td>
<td>9 </td>
<td>2919887 </td>
<td>2926740 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>Exon + UTR </td>
<td>5,150 </td>
<td>4,336 </td>
<td>27 </td>
<td>99.7 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>9 </td>
<td>35652219 </td>
<td>35653967 </td>
<td>Glyma09g26550 </td>
<td>9 </td>
<td>33049107 </td>
<td>33050904 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>Gene </td>
<td>1,813 </td>
<td>1,122 </td>
<td>11 </td>
<td>98.7 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>10 </td>
<td>37489560 </td>
<td>37495624 </td>
<td>Glyma10g28170 </td>
<td>10 </td>
<td>36962521 </td>
<td>36968813 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,212 </td>
<td>4,065 </td>
<td>29 </td>
<td>94.6 </td>
<td>14 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>10 </td>
<td>45294735 </td>
<td>45316121 </td>
<td>Glyma10g36600 </td>
<td>10 </td>
<td>44732730 </td>
<td> </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>SNP </td>
<td>222 </td>
<td>1 </td>
<td>1 </td>
<td>100.0 </td>
<td>3, 4, 12 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>12 </td>
<td>5508365 </td>
<td>5522772 </td>
<td>Glyma12g07861 </td>
<td>12 </td>
<td>5496565 </td>
<td>5511828 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>4,116 </td>
<td>2,871 </td>
<td>24 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>15 </td>
<td>11435551 </td>
<td>11442683 </td>
<td>Glyma15g14980 </td>
<td>15 </td>
<td>11415495 </td>
<td>11422656 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,445 </td>
<td>4,542 </td>
<td>28 </td>
<td>98.1 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>16 </td>
<td>4135885 </td>
<td>4137742 </td>
<td>Glyma16g04830 </td>
<td>16 </td>
<td>4115033 </td>
<td>4116923 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>Exon + UTR </td>
<td>1,683 </td>
<td>1,109 </td>
<td>9 </td>
<td>98.0 </td>
<td>15, 19, 37 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>16 </td>
<td>4162525 </td>
<td>4164824 </td>
<td>Glyma16g04840 </td>
<td>16 </td>
<td>4141774 </td>
<td>4144073 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>Exon + UTR </td>
<td>1,031 </td>
<td>486 </td>
<td>6 </td>
<td>92.7 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>16 </td>
<td>31109999 </td>
<td>31114963 </td>
<td>Glyma16g26660 </td>
<td>16 </td>
<td>30741660 </td>
<td>30746677 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Exon + UTR </td>
<td>1,515 </td>
<td>935 </td>
<td>9 </td>
<td>99.4 </td>
<td>9, 15, 18, 33, 37 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>16 </td>
<td>31148829 </td>
<td>31151842 </td>
<td>Glyma16g26690 </td>
<td>16 </td>
<td>30780496 </td>
<td>30783509 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>Exon + UTR </td>
<td>860 </td>
<td>464 </td>
<td>5 </td>
<td>88.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>16 </td>
<td>35777815 </td>
<td>35779317 </td>
<td>Glyma16g32080 </td>
<td>16 </td>
<td>35274147 </td>
<td>35275762 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>Exon + UTR </td>
<td>1,554 </td>
<td>1,038 </td>
<td>6 </td>
<td>99.0 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>16 </td>
<td>36179891 </td>
<td>36187469 </td>
<td>Glyma16g32540 </td>
<td>16 </td>
<td>35676581 </td>
<td>35684221 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>2,534 </td>
<td>1,095 </td>
<td>15 </td>
<td>98.8 </td>
<td>48, 49 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>17 </td>
<td>3955518 </td>
<td>3958432 </td>
<td>Glyma17g05990 </td>
<td>17 </td>
<td>4225839 </td>
<td>4228888 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Exon + UTR </td>
<td>1,743 </td>
<td>1,318 </td>
<td>9 </td>
<td>100.0 </td>
<td>50 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>17 </td>
<td>7052506 </td>
<td>7053858 </td>
<td>Glyma17g09810 </td>
<td>17 </td>
<td>7317271 </td>
<td>7318756 </td>
<td>Achaete-scute transcription factor-related<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>i</sup></a> </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>1,594 </td>
<td>1,166 </td>
<td>9 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>19 </td>
<td>36030632 </td>
<td>36032867 </td>
<td>Glyma19g28390 </td>
<td>19 </td>
<td>35849981 </td>
<td>35852216 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>Exon + UTR </td>
<td>952 </td>
<td>463 </td>
<td>5 </td>
<td>100.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>19 </td>
<td>36049111 </td>
<td>36051851 </td>
<td>Glyma19g28400 </td>
<td>19 </td>
<td>35868460 </td>
<td>35871203 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>Exon + UTR </td>
<td>1,113 </td>
<td>560 </td>
<td>6 </td>
<td>90.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>19 </td>
<td>45183357 </td>
<td>45185175 </td>
<td>Glyma19g37890 </td>
<td>19 </td>
<td>44979743 </td>
<td>44981657 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Exon + UTR </td>
<td>1,411 </td>
<td>1,078 </td>
<td>8 </td>
<td>100.0 </td>
<td>37, 53 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>19 </td>
<td>47633059 </td>
<td>47641958 </td>
<td>Glyma19g41210 </td>
<td>19 </td>
<td>47511095 </td>
<td>47520052 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Exon + UTR </td>
<td>5,235 </td>
<td>4,400 </td>
<td>27 </td>
<td>98.6 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>19 </td>
<td>50364718 </td>
<td>50369677 </td>
<td>Glyma19g44970 </td>
<td>19 </td>
<td>50244046 </td>
<td>50249070 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>Gene </td>
<td>5,673 </td>
<td>4,941 </td>
<td>31 </td>
<td>98.3 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>20 </td>
<td>33236018 </td>
<td>33241692 </td>
<td>Glyma20g22160 </td>
<td>20 </td>
<td>32087412 </td>
<td>32093306 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>Exon + UTR </td>
<td>4,871 </td>
<td>4,076 </td>
<td>25 </td>
<td>99.2 </td>
<td>3, 14 </td>
</tr>
<tr>
<td>Glyma.U034500<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>Scaffold_32 </td>
<td>197150 </td>
<td>220019 </td>
<td>Glyma11g15580 </td>
<td>11 </td>
<td>11232271 </td>
<td>11255186 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,352 </td>
<td>3,833 </td>
<td>30 </td>
<td>98.1 </td>
<td>  </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th>  </th>
<th colspan="2">Gmax275 (ver. 2.0)<hr> </th>
<th>  </th>
<th>  </th>
<th colspan="3">Gmax189 (ver. 1.1)<hr> </th>
<th rowspan="2">Description<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Gene name<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Target </th>
<th rowspan="2">Total designed target length of amplicons<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">Target size<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th rowspan="2">Number of amplicon<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th>
<th rowspan="2">Coverage<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> (%) </th>
<th rowspan="2">Reference </th>
</tr>
<tr>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
<th>Gene ID </th>
<th>Chr. </th>
<th>Start </th>
<th>End </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>2 </td>
<td>6116379 </td>
<td>6117379 </td>
<td>Glyma02g07650 </td>
<td>2 </td>
<td>6041770 </td>
<td>6042409 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>Exon + UTR </td>
<td>861 </td>
<td>639 </td>
<td>6 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>3 </td>
<td>40522597 </td>
<td>40525110 </td>
<td>Glyma03g35250 </td>
<td>3 </td>
<td>42534078 </td>
<td>42535865 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>Exon + UTR </td>
<td>1,577 </td>
<td>1,003 </td>
<td>9 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>3 </td>
<td>42918771 </td>
<td>42923401 </td>
<td>Glyma03g38620 </td>
<td>3 </td>
<td>44925730 </td>
<td>44930360 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>Exon + UTR </td>
<td>3,688 </td>
<td>2,876 </td>
<td>20 </td>
<td>98.6 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.04G143300<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>4 </td>
<td>26120011 </td>
<td>26120532 </td>
<td>Glyma18g22670 </td>
<td>18 </td>
<td>25739929 </td>
<td>25740831 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1Lb</em> </td>
<td>Exon + UTR </td>
<td>1,037 </td>
<td>902 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>4 </td>
<td>36758125 </td>
<td>36758770 </td>
<td>Glyma04g24640 </td>
<td>4 </td>
<td>28293933 </td>
<td>28294806 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1La</em> </td>
<td>Exon + UTR </td>
<td>967 </td>
<td>873 </td>
<td>5 </td>
<td>100.0 </td>
<td>11, 54 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>6 </td>
<td>20207077 </td>
<td>20207940 </td>
<td>Glyma06g23026 </td>
<td>6 </td>
<td>20006928 </td>
<td>20007814 </td>
<td>B3 domain-containing protein<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>h</sup></a> </td>
<td>
<em>E1</em> </td>
<td>Exon + UTR </td>
<td>1,059 </td>
<td>886 </td>
<td>6 </td>
<td>100.0 </td>
<td>3, 4, 11 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>8 </td>
<td>47458142 </td>
<td>47459829 </td>
<td>Glyma08g47810 </td>
<td>8 </td>
<td>46606934 </td>
<td>46608654 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>Exon + UTR </td>
<td>1,300 </td>
<td>746 </td>
<td>8 </td>
<td>100.0 </td>
<td>20 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>8 </td>
<td>47472881 </td>
<td>47473362 </td>
<td>Glyma08g47823 </td>
<td>8 </td>
<td>46621704 </td>
<td>46622185 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>Exon + UTR </td>
<td>612 </td>
<td>265 </td>
<td>3 </td>
<td>100.0 </td>
<td>37, 38 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>9 </td>
<td>2960395 </td>
<td>2967229 </td>
<td>Glyma09g03990 </td>
<td>9 </td>
<td>2919887 </td>
<td>2926740 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>Exon + UTR </td>
<td>5,150 </td>
<td>4,336 </td>
<td>27 </td>
<td>99.7 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>9 </td>
<td>35652219 </td>
<td>35653967 </td>
<td>Glyma09g26550 </td>
<td>9 </td>
<td>33049107 </td>
<td>33050904 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>Gene </td>
<td>1,813 </td>
<td>1,122 </td>
<td>11 </td>
<td>98.7 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>10 </td>
<td>37489560 </td>
<td>37495624 </td>
<td>Glyma10g28170 </td>
<td>10 </td>
<td>36962521 </td>
<td>36968813 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,212 </td>
<td>4,065 </td>
<td>29 </td>
<td>94.6 </td>
<td>14 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>10 </td>
<td>45294735 </td>
<td>45316121 </td>
<td>Glyma10g36600 </td>
<td>10 </td>
<td>44732730 </td>
<td> </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>SNP </td>
<td>222 </td>
<td>1 </td>
<td>1 </td>
<td>100.0 </td>
<td>3, 4, 12 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>12 </td>
<td>5508365 </td>
<td>5522772 </td>
<td>Glyma12g07861 </td>
<td>12 </td>
<td>5496565 </td>
<td>5511828 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>4,116 </td>
<td>2,871 </td>
<td>24 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>15 </td>
<td>11435551 </td>
<td>11442683 </td>
<td>Glyma15g14980 </td>
<td>15 </td>
<td>11415495 </td>
<td>11422656 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,445 </td>
<td>4,542 </td>
<td>28 </td>
<td>98.1 </td>
<td>32 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>16 </td>
<td>4135885 </td>
<td>4137742 </td>
<td>Glyma16g04830 </td>
<td>16 </td>
<td>4115033 </td>
<td>4116923 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>Exon + UTR </td>
<td>1,683 </td>
<td>1,109 </td>
<td>9 </td>
<td>98.0 </td>
<td>15, 19, 37 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>16 </td>
<td>4162525 </td>
<td>4164824 </td>
<td>Glyma16g04840 </td>
<td>16 </td>
<td>4141774 </td>
<td>4144073 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>Exon + UTR </td>
<td>1,031 </td>
<td>486 </td>
<td>6 </td>
<td>92.7 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>16 </td>
<td>31109999 </td>
<td>31114963 </td>
<td>Glyma16g26660 </td>
<td>16 </td>
<td>30741660 </td>
<td>30746677 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Exon + UTR </td>
<td>1,515 </td>
<td>935 </td>
<td>9 </td>
<td>99.4 </td>
<td>9, 15, 18, 33, 37 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>16 </td>
<td>31148829 </td>
<td>31151842 </td>
<td>Glyma16g26690 </td>
<td>16 </td>
<td>30780496 </td>
<td>30783509 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>Exon + UTR </td>
<td>860 </td>
<td>464 </td>
<td>5 </td>
<td>88.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>16 </td>
<td>35777815 </td>
<td>35779317 </td>
<td>Glyma16g32080 </td>
<td>16 </td>
<td>35274147 </td>
<td>35275762 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>Exon + UTR </td>
<td>1,554 </td>
<td>1,038 </td>
<td>6 </td>
<td>99.0 </td>
<td>37 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>16 </td>
<td>36179891 </td>
<td>36187469 </td>
<td>Glyma16g32540 </td>
<td>16 </td>
<td>35676581 </td>
<td>35684221 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>2,534 </td>
<td>1,095 </td>
<td>15 </td>
<td>98.8 </td>
<td>48, 49 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>17 </td>
<td>3955518 </td>
<td>3958432 </td>
<td>Glyma17g05990 </td>
<td>17 </td>
<td>4225839 </td>
<td>4228888 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Exon + UTR </td>
<td>1,743 </td>
<td>1,318 </td>
<td>9 </td>
<td>100.0 </td>
<td>50 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>17 </td>
<td>7052506 </td>
<td>7053858 </td>
<td>Glyma17g09810 </td>
<td>17 </td>
<td>7317271 </td>
<td>7318756 </td>
<td>Achaete-scute transcription factor-related<a reveal-id="tblfn9" data-open="tblfn9" class="link link-ref link-reveal xref-fn"><sup>i</sup></a> </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>1,594 </td>
<td>1,166 </td>
<td>9 </td>
<td>100.0 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>19 </td>
<td>36030632 </td>
<td>36032867 </td>
<td>Glyma19g28390 </td>
<td>19 </td>
<td>35849981 </td>
<td>35852216 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>Exon + UTR </td>
<td>952 </td>
<td>463 </td>
<td>5 </td>
<td>100.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>19 </td>
<td>36049111 </td>
<td>36051851 </td>
<td>Glyma19g28400 </td>
<td>19 </td>
<td>35868460 </td>
<td>35871203 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>Exon + UTR </td>
<td>1,113 </td>
<td>560 </td>
<td>6 </td>
<td>90.0 </td>
<td>15, 37 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>19 </td>
<td>45183357 </td>
<td>45185175 </td>
<td>Glyma19g37890 </td>
<td>19 </td>
<td>44979743 </td>
<td>44981657 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Exon + UTR </td>
<td>1,411 </td>
<td>1,078 </td>
<td>8 </td>
<td>100.0 </td>
<td>37, 53 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>19 </td>
<td>47633059 </td>
<td>47641958 </td>
<td>Glyma19g41210 </td>
<td>19 </td>
<td>47511095 </td>
<td>47520052 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Exon + UTR </td>
<td>5,235 </td>
<td>4,400 </td>
<td>27 </td>
<td>98.6 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>19 </td>
<td>50364718 </td>
<td>50369677 </td>
<td>Glyma19g44970 </td>
<td>19 </td>
<td>50244046 </td>
<td>50249070 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>Gene </td>
<td>5,673 </td>
<td>4,941 </td>
<td>31 </td>
<td>98.3 </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>20 </td>
<td>33236018 </td>
<td>33241692 </td>
<td>Glyma20g22160 </td>
<td>20 </td>
<td>32087412 </td>
<td>32093306 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>Exon + UTR </td>
<td>4,871 </td>
<td>4,076 </td>
<td>25 </td>
<td>99.2 </td>
<td>3, 14 </td>
</tr>
<tr>
<td>Glyma.U034500<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>g</sup></a> </td>
<td>Scaffold_32 </td>
<td>197150 </td>
<td>220019 </td>
<td>Glyma11g15580 </td>
<td>11 </td>
<td>11232271 </td>
<td>11255186 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>Exon + UTR </td>
<td>5,352 </td>
<td>3,833 </td>
<td>30 </td>
<td>98.1 </td>
<td>  </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn">
<div class="label">a</div>
<p>Gene description from Phytozome 12.</p>
</div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">b</div>
<p>Gene name refers to Kong et al.,<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Wu et al.,<a reveal-id="dsz005-B29" data-open="dsz005-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Fan et al.,<a reveal-id="dsz005-B52" data-open="dsz005-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and Cao et al.<a reveal-id="dsz005-B53" data-open="dsz005-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">c</div>
<p>Total size of amplified region by designed amplicon primers.</p>
</div></div>
<div content-id="tblfn4"><div class="fn">
<div class="label">d</div>
<p>Target size (bp) of amplicon based on soybean genome version 1.1 (Gmax189).</p>
</div></div>
<div content-id="tblfn5"><div class="fn">
<div class="label">e</div>
<p>Total number of designed amplicons on target gene.</p>
</div></div>
<div content-id="tblfn6"><div class="fn">
<div class="label">f</div>
<p>Percentage of target region covered by amplicon.</p>
</div></div>
<div content-id="tblfn7"><div class="fn">
<div class="label">g</div>
<p>Glyma.04G143300 and Glyma.U034500on Gmax275 genome version were different chromosome positions on Gmax189.</p>
</div></div>
<div content-id="tblfn8"><div class="fn">
<div class="label">h</div>
<p>The gene annotation manually curated.</p>
</div></div>
<div content-id="tblfn9"><div class="fn">
<div class="label">i</div>
<p>‘Achaete-scute transcription factor related’ gene was included as control for variant detection.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025710" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="137025711" id="137025711" class="section-title">2.3. Library preparation and sequencing</h3>
<p>The Qubit dsDNA High Sensitivity Assay Kit (Thermo Fisher Scientific) was used to quantify DNA for NGS library construction. The NGS library was constructed using the Ion AmpliSeq Library Kit 2.0 (Thermo Fisher Scientific), according to the manufacturer’s protocol (Japanese version corresponding to Manual 2014.7 rev.B.0 version: <a class="link link-uri" href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0013432_Ion_AmpliSeq_Library_Prep_on_Ion_Chef_UG.pdf,%2025%20March%202019,%20date%20last%20accessed" target="_blank">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0013432_Ion_AmpliSeq_Library_Prep_on_Ion_Chef_UG.pdf, 25 March 2019, date last accessed</a>). For the multiplex-PCR amplification, 1–10 ng of each DNA was amplified using one primer pool (191 amplicon primer pairs) per reaction. This was performed using 4 µl of 5× Ion AmpliSeq HiFi Master Mix, 10 µl of 2× AmpliSeq Custom Primer Pool, 1–10 ng of DNA, and the volume was made up to 20 µl with nuclease-free water. The reaction mix was heated for 2 min at 99°C for enzyme activation, followed by 18 two-step cycles at 99°C for 15 s and at 60°C for 4 min, and ending with a holding period at 10°C. As for low-quality DNA samples, 21 cycles were subjected under similar conditions. The primers of amplicons were digested and phosphorylated for adapter ligation using 2 µl of FuPa enzyme per sample at 55°C for 10 min, followed by enzyme inactivation at 60°C for 20 min. To enable multiple libraries to be loaded per chip, 2 µl of a unique diluted mix, including Ion Xpress Barcode and Ion P1 Adapters at standard volumes, was ligated to the end of the digested amplicons using 2 µl of DNA ligase at 22°C for 30 min, followed by ligase inactivation for 10 min at 72°C. The resulting un-amplified adapter-ligated library was purified using 45 µl of Agencourt AMPure XP Reagent (Beckman Coulter, Brea, CA, USA), followed by washing using 150 µl of freshly prepared 70% ethanol. After purification, 50 µl of Platinum PCR SuperMix High Fidelity and 2 µl of Library Amplification Primer Mix of the Ion AmpliSeq Library Kit 2.0 were added to the dried AMPure XP beads, and then the reaction plate was placed on a magnetic rack to separate the beads from the supernatant. The amplicon library in the supernatant was further amplified to enrich the material for accurate quantification at 98°C for 2 min, followed by five two-step cycles at 98°C for 15 s and at 60°C for 1 min. The amplified amplicon library was then purified using 25 µl of AMPure XP, followed by a second purification step with 60 µl of AMPure XP and 150 µl of freshly prepared 70% ethanol. The concentration and size distribution of amplicons in the library were then determined using an Agilent BioAnalyzer DNA High-Sensitivity chip or TapeStation 4200 D1000 chip (Agilent Technologies), according to the instruction of the manufacturer. After quantification, each library was diluted to a concentration of 100 pM prior to template preparation. Subsequently, the libraries were pooled in equimolar amounts prior to further processing. Emulsion PCR, emulsion breaking, and enrichment for template preparation of ion sphere particles were performed using the Ion 520 &amp; 530 and 540 Kit-Chef (Thermo Fisher Scientific) according to the instruction of the manufacturer. After the preparation of ion sphere particles, sequencing was performed with an Ion Torrent Ion S5 or S5XL system using Ion 520 and 540 Chip (Thermo Fisher Scientific), according to the instruction of the manufacturer.</p>                    <h3 scrollto-destination="137025713" id="137025713" class="section-title">2.4. Data analysis</h3>
<div class="block-child-p">The Ion S5/S5XL sequence data were mapped to the soybean genome reference version 2.0 (Gmax275: https://genome.jgi.doe.gov/portal/pages/dynamicOrganismDownload.jsf?%20organism=Phytozome, 25 March 2019, date last accessed) using Ion Torrent Suite version 5.2.1 software. In typical genome databases of soybean (Williams 82), such as, Phytozome and Soybase, Gmax275 is widely used instead of Gmax189.<a reveal-id="dsz005-B23" data-open="dsz005-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The assembly size and number of predicted protein-coding loci of Gmax275 are 978 Mb and 56,044, which are higher than 969.6 Mb and 46,430 of Gmax189, respectively. In the Gmax275 assembly, several genes are located on chromosomes/scaffolds different from those of Gmax189. For example, 238 genes on the chromosomes of Gmax189 are located on the scaffolds of Gmax275, whereas 100 genes on the scaffolds of Gmax189 are located on the chromosomes of Gmax275 (<a class="link link-uri" href="http://www.soybase.org/correspondence/methods.txt" target="_blank">http://www.soybase.org/correspondence/methods.txt</a>, 25 March 2019, date last accessed). In this study, one of the target genes, <em>Glyma18g22670</em> (B3 domain-containing protein), on chromosome 18 of Gmax189 was located on chromosome 4 (<em>Glyma.04G143300</em>) in Gmax275. However, the structure of all target genes used in this study was the same between Gmax189 and Gmax275. Ion Torrent Suite software was optimized for Ion Torrent raw data analysis; alignment using Torrent Mapping Alignment Program (TMAP) version 5.2.25 and Coverage Analysis v.5.8.0.8, and variant calling using Torrent Variant Caller version 5.2.1.38 and plug-in version 5.2.25. To evaluate PCR amplification efficiency of each amplicon, the amplicons per 100k reads mapped (APKM) as the count scaled by the total number of amplicons sequenced <em>N</em> times per 100k reads as follows: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mtext>APKM</mtext><mo>=</mo><mfrac><mrow><msub><mrow><mi>X</mi></mrow><mi>i</mi></msub></mrow><mrow><mfrac><mi>N</mi><mrow><msup><mrow><mrow><mn>10</mn></mrow></mrow><mn>5</mn></msup></mrow></mfrac></mrow></mfrac><mo>=</mo><mfrac><mrow><msub><mrow><mi>X</mi></mrow><mi>i</mi></msub></mrow><mi>N</mi></mfrac><msup><mrow><mrow><mn>.10</mn></mrow></mrow><mn>5</mn></msup></mrow><mo xmlns="">,</mo></math></div></div></div>where <em>X<sub>i</sub></em> represents the read coverage <em>X</em> of target amplicon <em>i</em>.</div>
<p>Variant calling was performed using the default (low stringency) and custom parameters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). All accessions used in this study were propagated by the single seed descent method; therefore, all variants should be detected as homozygous theoretically. The parameter of TMAP in Torrent Variant Caller was changed to loosen the judgment condition of homozygous by setting ‘snp_min_allele_freq’ from 0.15 (default) to 0.3. In this condition, the SNP was detected with allele frequency of &gt;70% as homozygous. The InDels were detected as homozygous when allele frequency was &gt;75% (default parameter). Sequence variants detected as heterozygous under these conditions were excluded. The vcf files obtained were annotated and filtered using the snpEff version 4.0e.<a reveal-id="dsz005-B24" data-open="dsz005-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a></p>                    <h3 scrollto-destination="137025716" id="137025716" class="section-title">2.5. Detection of known and novel SNPs and InDels</h3>
<p>The known alleles <em>E1</em>, <em>E2</em>, <em>E3</em>, and <em>E4</em> were investigated as described above.<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The polymorphism information from the Single Nucleotide Polymorphism Database<a reveal-id="dsz005-B25" data-open="dsz005-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> (dbSNP: <a class="link link-uri" href="https://www.ncbi.nlm.nih.gov/" target="_blank">https://www.ncbi.nlm.nih.gov/</a>snp, 25 March 2019, date last accessed, data downloaded on 29 April 2016) was used to investigate whether the detected polymorphism has already been identified in the whole-genome sequence of soybean. As the position of soybean genome in the National Center for Biotechnology Information (NCBI) and Phytozome v12.1 are not consistent, we converted the position of SNP in the dbSNP from the NCBI to that of Gmax275 of Phytozome v12.1. The SNP ID number (rs; refSNP cluster) was used for the SNP name.</p>                    <h3 scrollto-destination="137025718" id="137025718" class="section-title">2.6. Validation of SNPs and InDels</h3>
<p>The detected variants of the <em>Phytochrome A</em> (<em>E3, E4</em>) and <em>FT</em> genes (<em>FT2a</em> and <em>FT5a</em>) were further confirmed by Sanger sequencing. The exon containing the novel variants was amplified by PCR using the primers shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>. The PCR product was purified using Affymetrix ExoSap-IT regent (ExoSap-IT, USB Corporation, Staufen, Germany) and directly sequenced for both sense and antisense strands using Big Dye Terminator version 3.1 (Applied Biosystems, Foster City, CA, USA) in an ABI 3500 Genetic Analyzer (Applied Biosystems), according to the manufacturer’s protocol. The sequences were analysed using Genetics software version 10.0.8 (GENETYX Corp., Japan).</p>                    <h3 scrollto-destination="137025720" id="137025720" class="section-title">2.7. Gene-based multiple regression association testing for flowering time</h3>
<p>Flowering time was evaluated from 2011 to 2013 at the National Institute of Crop Science (36°02′N, 140°11′E), Tsukuba, Japan. Seeds were sown on 12 July 2011, and 10 July 2012 and 2013. A starter fertilizer containing 3, 10, and 10 g m<sup>−2</sup> of N, P<sub>2</sub>O<sub>5</sub>, and K<sub>2</sub>O, respectively, was applied. Each accession was planted in single-row plots. Each row was paved 0.7 m apart and each plot comprised 12 plants that were spaced 0.13 m apart. The average days to flowering in each plot were used for analysis. Association between days to flowering and each polymorphic SNPs/InDels was assessed using linear regression, where the simulated trait values across the 190 individuals were regressed onto the numeric code of each SNP and InDel genotype; this tested the null hypothesis of the additive allelic effect on the trait. Regression analyses were performed using ‘lm()’ in R.<a reveal-id="dsz005-B26" data-open="dsz005-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> First, simple linear regression analysis was performed to assess the influence of the detected variant on flowering time at the significance level of <em>P </em>&lt;<em> </em>0.05. Subsequently, multiple linear regression analysis was performed using the significantly representative variants after removing redundant variants at the significance level of <em>P </em>&lt;<em> </em>0.05.</p>                    <h2 scrollto-destination="137025722" id="137025722" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="137025723" id="137025723" class="section-title">3.1. Amplicon design and comparison of library quality using DNA samples derived from the leaf and seed</h3>
<p>To evaluate the performance of AmpliSeq, we focused on gene region of 29 genes (<a reveal-id="dsz005-T1" data-open="dsz005-T1" class="link link-reveal link-table xref-fig">Table 1</a>) selected from known genes related to flowering time and their homologues in soybean. A total of 382 amplicon primer pairs consisting of two primer pools (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) were designed for the 64.98-kb target region using the AmpliSeq designer tool. These primer pairs covered 98.4% of the target region, ranging from 89.8% to 100%, by overlapping PCR products of total length 70,180 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The average amplicon size including primer region was 237.1 bp ranging from 125 to 275 bp (target region was 65–232 bp). The target gene with the lowest coverage (89.8%) was <em>Glyma.16g044200</em> (<em>FT</em>-like gene).</p>
<p>We examined the DNA quality necessary for AmpliSeq library construction because genotyping is commonly performed using low-quality DNA especially that derived from the seed of soybean for marker-assisted selection. Low-quality DNA derived from the seed was obtained at concentrations of 0.5–3 ng/µl, whereas high-quality DNA from the leaf was obtained at concentrations of 30–50 ng/µl (<a reveal-id="dsz005-F1" data-open="dsz005-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). We used 1–10 ng of seed-derived DNA and 10 ng of leaf-derived DNA for preparing AmpliSeq library. To confirm whether low-quality DNA can produce a library of sufficient quality, distribution range of amplicons in the libraries prepared using low-quality DNA was compared with that of high-quality DNA, which is recommended for sequencing using the Agilent 2100 Bioanalyzer or TapeStation 4200 (<a reveal-id="dsz005-F1" data-open="dsz005-F1" class="link link-reveal link-table xref-fig">Fig. 1A and B</a>). No difference was observed between low-quality DNA from the seed and high-quality DNA from the leaf in the size range of amplicons (130–370 bp) or maximum peak amplitude (<a reveal-id="dsz005-F1" data-open="dsz005-F1" class="link link-reveal link-table xref-fig">Fig. 1B and C</a>). These results reveal that the AmpliSeq library of sufficient yield and quality can be prepared from low-quality DNA. We then prepared sufficient amount of library using DNA derived from the leaf or seed of the soybean mini-core collection. </p>                        <a id="137025726" scrollto-destination="137025726"></a>
<div id="dsz005-F1" data-id="dsz005-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f1.png?Expires=1576535926&amp;Signature=Boxhq26TqMiHYc6y6HHLhlMfTEbMpdlUAWYePB2onGGGvp~97r3g5yrFGRpSYf6lkO8gIsVi2kGu8S6SDV2lhpk8BZRL8TrogyzT28qjn73M9RvzABtyJF35KTLe7KtSeXTRLD-5Z4tDjbcK8G2LK8FQU2o-J2Q~wTqUFBP2OzYDQbrXa-sc2FekaZkJu2rThgB-UFXwrym1YXhQ4cRyhb2VJK47XqGHBk17XvvGkZMg-776fgOIaXQvVK7yd~QG~q8EZLb~VuWBbR0WoIldNIjvRjLpcMOJM-mtV3uWBk-maPoB-v7DPgvoLWlWIXn4l~oNNSYqsfKGs6b7yK424w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of quality of DNA and AmpliSeq library prepared from the DNA using the Agilent 2200 TapeStation system. The AmpliSeq libraries were evaluated using the D1000 screen tape. (A) Quality of the DNA derived from the leaf and seeds. W and E indicate Williams 82 and Enrei, respectively. The numerical assessment of DNA quality ranged from 1 to 10 based on the DNA integrity number (DIN). A high DIN indicates highly intact DNA, whereas a low DIN indicates degraded DNA. (B) Distribution of amplicons in the AmpliSeq library shown as a gel image. (C) Electropherogram of the same AmpliSeq library as shown in (B). Lower (25 bp) and upper (1,500 bp) peak are the standard markers. The middle peak indicates the library." path-from-xml="dsz005f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025726/dsz005f1.tif" path-from-xml="dsz005f1.tif" target="_blank">Open in new tab</a><a section="137025726" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f1.png?Expires=1576535926&amp;Signature=QAUR-O~su8p510ELRNeRfEizS~FksgMyjVViPPd4w6MXTYRRbzYL4yxjD7aEHXcWtdJ~cpxNuEglwt1c6oPaIN0d9d6oMH9zt2alxkHheNKJ0-o2674xXU7~E50aPHDKqIa4ZrquAdDT3LkWOMDUmm-MFU6M2hQRe7~0PktyokABerJFIZOJ7duiF2CVMbluaLbf8yyk29lmDRuPPXwVhgl3jlAeX6SVjdVmmN1nW33eH3GWSdgsOcHHXkohtFpscM5HrgGIyi6w2TtsCiGWuRkUeNjaQDxe~RkMLyrtuWZVJxOMmE9CU7gWPYge0dlG8btJZzhiSceQvjLrl7Isqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025726&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Evaluation of quality of DNA and AmpliSeq library prepared from the DNA using the Agilent 2200 TapeStation system. The AmpliSeq libraries were evaluated using the D1000 screen tape. (A) Quality of the DNA derived from the leaf and seeds. W and E indicate Williams 82 and Enrei, respectively. The numerical assessment of DNA quality ranged from 1 to 10 based on the DNA integrity number (DIN). A high DIN indicates highly intact DNA, whereas a low DIN indicates degraded DNA. (B) Distribution of amplicons in the AmpliSeq library shown as a gel image. (C) Electropherogram of the same AmpliSeq library as shown in (B). Lower (25 bp) and upper (1,500 bp) peak are the standard markers. The middle peak indicates the library.</p></div>
</div>
<div content-id="dsz005-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f1.png?Expires=1576535926&amp;Signature=Boxhq26TqMiHYc6y6HHLhlMfTEbMpdlUAWYePB2onGGGvp~97r3g5yrFGRpSYf6lkO8gIsVi2kGu8S6SDV2lhpk8BZRL8TrogyzT28qjn73M9RvzABtyJF35KTLe7KtSeXTRLD-5Z4tDjbcK8G2LK8FQU2o-J2Q~wTqUFBP2OzYDQbrXa-sc2FekaZkJu2rThgB-UFXwrym1YXhQ4cRyhb2VJK47XqGHBk17XvvGkZMg-776fgOIaXQvVK7yd~QG~q8EZLb~VuWBbR0WoIldNIjvRjLpcMOJM-mtV3uWBk-maPoB-v7DPgvoLWlWIXn4l~oNNSYqsfKGs6b7yK424w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evaluation of quality of DNA and AmpliSeq library prepared from the DNA using the Agilent 2200 TapeStation system. The AmpliSeq libraries were evaluated using the D1000 screen tape. (A) Quality of the DNA derived from the leaf and seeds. W and E indicate Williams 82 and Enrei, respectively. The numerical assessment of DNA quality ranged from 1 to 10 based on the DNA integrity number (DIN). A high DIN indicates highly intact DNA, whereas a low DIN indicates degraded DNA. (B) Distribution of amplicons in the AmpliSeq library shown as a gel image. (C) Electropherogram of the same AmpliSeq library as shown in (B). Lower (25 bp) and upper (1,500 bp) peak are the standard markers. The middle peak indicates the library." path-from-xml="dsz005f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025726/dsz005f1.tif" path-from-xml="dsz005f1.tif" target="_blank">Open in new tab</a><a section="137025726" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f1.png?Expires=1576535926&amp;Signature=QAUR-O~su8p510ELRNeRfEizS~FksgMyjVViPPd4w6MXTYRRbzYL4yxjD7aEHXcWtdJ~cpxNuEglwt1c6oPaIN0d9d6oMH9zt2alxkHheNKJ0-o2674xXU7~E50aPHDKqIa4ZrquAdDT3LkWOMDUmm-MFU6M2hQRe7~0PktyokABerJFIZOJ7duiF2CVMbluaLbf8yyk29lmDRuPPXwVhgl3jlAeX6SVjdVmmN1nW33eH3GWSdgsOcHHXkohtFpscM5HrgGIyi6w2TtsCiGWuRkUeNjaQDxe~RkMLyrtuWZVJxOMmE9CU7gWPYge0dlG8btJZzhiSceQvjLrl7Isqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025726&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Evaluation of quality of DNA and AmpliSeq library prepared from the DNA using the Agilent 2200 TapeStation system. The AmpliSeq libraries were evaluated using the D1000 screen tape. (A) Quality of the DNA derived from the leaf and seeds. W and E indicate Williams 82 and Enrei, respectively. The numerical assessment of DNA quality ranged from 1 to 10 based on the DNA integrity number (DIN). A high DIN indicates highly intact DNA, whereas a low DIN indicates degraded DNA. (B) Distribution of amplicons in the AmpliSeq library shown as a gel image. (C) Electropherogram of the same AmpliSeq library as shown in (B). Lower (25 bp) and upper (1,500 bp) peak are the standard markers. The middle peak indicates the library.</p></div>
</div>                    <h3 scrollto-destination="137025727" id="137025727" class="section-title">3.2. Performance of NGS and uniformity of amplicon coverage</h3>
<p>Among 105,761,267 reads obtained, 105,603,249 (99.85%) reads were mapped to Williams 82 reference genome Gmax275 using TMAP and the average read depth across the target region was 1,237× (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). According to the on-target rate, 94.12% of the reads was mapped to the targeted regions. The average read length, average on-target rate, and uniformity (percent of reads &gt;0.2× of mean coverage in the sample) of the leaf and seed samples were similar, but a few seed samples showed lower average read length and uniformity (<a reveal-id="dsz005-F2" data-open="dsz005-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The highly fragmented DNA sample showed low amplification of long amplicons (&gt; 200 bp) and low uniformity (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A and B</a></span>). Low average amplicon length or low uniformity of the samples might be caused by DNA fragmentation or contamination of the DNA solution.<a reveal-id="dsz005-B27" data-open="dsz005-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p>                        <a id="137025729" scrollto-destination="137025729"></a>
<div id="dsz005-F2" data-id="dsz005-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f2.png?Expires=1576535926&amp;Signature=f2go77iUj9QQ3gIdGlhzMOFw-kyOpKShX1fhkxAzEe3PHdBKaTGBMkzx6k2hebmnzS8XnSyPNUG8KXuxHxo-4f0dMWBJ~HScaklLtMvjxApAHHk4BK1G6GetF72UdNiQpDg2pJOWHa60SLXJVpCKw6~lg~GLBcz0PpaVdXjnxB7xmwGeYxyKy0wjFrt9l4u9EdCxOzt0DwLWA-hi9zoae7ffSSfEC9MPYVlQXdT5EuN9fpSnDCoMpjLEA0BIGzUPWjvYTbqFJm27UD~d9Xt2cnr9ax9gOa1XCUpLnZYWlWyizeisoy48lgzDw194r8EcBY3NZGE~vq9IeD71TfTeIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of sequence performance metrics classified by the plant materials used for DNA extraction. (A) The box plot of average read length, on-target rate, and uniformity. The on-target rate is on-target percent of the aligned reads. Uniformity is the percent of bases in all the amplicon-targeted regions covered by at least 0.2× the mean base read depth. (B) Average normalized reads (APKM) per sample across 382 amplicons generated from the 192 mini-core collection. The X and Y axes indicate 382 amplicons sorted in their read coverage and APKM shown as the mean on a log scale, respectively." path-from-xml="dsz005f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025729/dsz005f2.tif" path-from-xml="dsz005f2.tif" target="_blank">Open in new tab</a><a section="137025729" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f2.png?Expires=1576535926&amp;Signature=WtMzX2n-Ewh2ReLlS-98kvPHruc98ZZj~6ar7rncW8sWs8DIFMNNvNhWyuaYuhH3Ck9M~bOdbleEE6O7HAeuGkU99Mnizo8kh0h6s7d6gULpbBOu~Id6aQ~w04ggKEfFvtU-nto09IUPlOys73xDrXq2OxBkLUUQ~dWW-39-w8QQVCcPvn8iXVQSXPJwpcG1l5m6xlluQL~cbRiA~uLiKkwSZJ3RevCSjZsMtd8s4aFPbAiGXWM9UVspuQUuKyAC2odm0-~9IxmvJ5GMI6LVTQ3IZTvDhUu-VQYBblKG5Dl6mWv6BVsOqtIcs0zgxd-fSEldE3-IcTtELZ1EKfCK6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025729&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of sequence performance metrics classified by the plant materials used for DNA extraction. (A) The box plot of average read length, on-target rate, and uniformity. The on-target rate is on-target percent of the aligned reads. Uniformity is the percent of bases in all the amplicon-targeted regions covered by at least 0.2× the mean base read depth. (B) Average normalized reads (APKM) per sample across 382 amplicons generated from the 192 mini-core collection. The <em>X</em> and <em>Y</em> axes indicate 382 amplicons sorted in their read coverage and APKM shown as the mean on a log scale, respectively.</p></div>
</div>
<div content-id="dsz005-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f2.png?Expires=1576535926&amp;Signature=f2go77iUj9QQ3gIdGlhzMOFw-kyOpKShX1fhkxAzEe3PHdBKaTGBMkzx6k2hebmnzS8XnSyPNUG8KXuxHxo-4f0dMWBJ~HScaklLtMvjxApAHHk4BK1G6GetF72UdNiQpDg2pJOWHa60SLXJVpCKw6~lg~GLBcz0PpaVdXjnxB7xmwGeYxyKy0wjFrt9l4u9EdCxOzt0DwLWA-hi9zoae7ffSSfEC9MPYVlQXdT5EuN9fpSnDCoMpjLEA0BIGzUPWjvYTbqFJm27UD~d9Xt2cnr9ax9gOa1XCUpLnZYWlWyizeisoy48lgzDw194r8EcBY3NZGE~vq9IeD71TfTeIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of sequence performance metrics classified by the plant materials used for DNA extraction. (A) The box plot of average read length, on-target rate, and uniformity. The on-target rate is on-target percent of the aligned reads. Uniformity is the percent of bases in all the amplicon-targeted regions covered by at least 0.2× the mean base read depth. (B) Average normalized reads (APKM) per sample across 382 amplicons generated from the 192 mini-core collection. The X and Y axes indicate 382 amplicons sorted in their read coverage and APKM shown as the mean on a log scale, respectively." path-from-xml="dsz005f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025729/dsz005f2.tif" path-from-xml="dsz005f2.tif" target="_blank">Open in new tab</a><a section="137025729" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f2.png?Expires=1576535926&amp;Signature=WtMzX2n-Ewh2ReLlS-98kvPHruc98ZZj~6ar7rncW8sWs8DIFMNNvNhWyuaYuhH3Ck9M~bOdbleEE6O7HAeuGkU99Mnizo8kh0h6s7d6gULpbBOu~Id6aQ~w04ggKEfFvtU-nto09IUPlOys73xDrXq2OxBkLUUQ~dWW-39-w8QQVCcPvn8iXVQSXPJwpcG1l5m6xlluQL~cbRiA~uLiKkwSZJ3RevCSjZsMtd8s4aFPbAiGXWM9UVspuQUuKyAC2odm0-~9IxmvJ5GMI6LVTQ3IZTvDhUu-VQYBblKG5Dl6mWv6BVsOqtIcs0zgxd-fSEldE3-IcTtELZ1EKfCK6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025729&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of sequence performance metrics classified by the plant materials used for DNA extraction. (A) The box plot of average read length, on-target rate, and uniformity. The on-target rate is on-target percent of the aligned reads. Uniformity is the percent of bases in all the amplicon-targeted regions covered by at least 0.2× the mean base read depth. (B) Average normalized reads (APKM) per sample across 382 amplicons generated from the 192 mini-core collection. The <em>X</em> and <em>Y</em> axes indicate 382 amplicons sorted in their read coverage and APKM shown as the mean on a log scale, respectively.</p></div>
</div>
<p>To compare the efficiency of PCR amplification of each amplicon, the APKM was calculated for each amplicon. The magnitude of APKM was similar irrespective of the type of DNA sample between the leaf and seed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). In contrast, there was no relationship between the APKM and read length of amplicons (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>). The APKM ranged from 0 to 3,333. Only one amplicon (AMPL1040290) had zero read. As the primer pair of AMPL1040290 was designed for the region flanking the TA-repeat microsatellite, it is difficult to amplify by multiplex PCR. We could amplify amplicons of 400–450 bp using the single primer pair of AMPL1040290.</p>
<p>We evaluated known alleles as an example to verify whether the reads were correctly mapped. Among two alleles with a large deletion, <em>e3-Mo</em> and <em>e3-tr</em>, at the <em>E3</em> gene on Chr19,<a reveal-id="dsz005-B12" data-open="dsz005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> the read mapping status of the <em>e3tr</em> allele, with a 15-kb deletion including the fourth exon, was examined by designing four amplicons, viz., AMPL1040313, AMPL1040314, AMPL1037722, and AMPL1036854 (<a reveal-id="dsz005-F3" data-open="dsz005-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The mapped reads from Williams 82 covered the entire fourth exon by the four amplicons (<a reveal-id="dsz005-F3" data-open="dsz005-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). In contrast, the reads from PGC010 were mapped to a part of the fourth exon, which could not be mapped because of absence of the fourth exon in PGC010. When we confirmed the sequence of the mapped reads, these are found to contain several polymorphic sites originated from another region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>). The primer pair of AMPL1040313 was found to have a similar (2- and 1-bp mismatches in the forward and reverse primers) sequence to that of the <em>E3</em> homologous gene on Chr3 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>). A comparison of sequence of the original amplicon designed for the <em>E3</em> gene on Chr19 with that designed for the <em>phyA</em> gene on Chr3 revealed that the amplicon of <em>phyA</em> could be mapped preferentially to <em>E3</em> in the absence of sequence information (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3C</a></span>). To preferentially output the alignment containing indel, we changed the penalty parameter by using the option ‘-A 10 -M 60 -O 50 -E 1’ to TMAP. The default parameters of TMAP option are as follows: ‘-A’, score for a match [default = 1]; ‘-M’, the mismatch penalty [3]; ‘-O’, the indel start penalty [5]; and ‘-E’, the indel extension penalty [2]. By increasing the penalty values related to base match and InDels, the miss-mapped reads on the fourth exon can be reduced from 2,426 (default parameter) to 3 reads (<a reveal-id="dsz005-F3" data-open="dsz005-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). By optimizing these parameters, the miss-mapped reads on the second exon of <em>E3</em> were also mapped to the correct position, <em>phyA</em> on Chr3 (second exon; <a reveal-id="dsz005-F3" data-open="dsz005-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). We also investigated whether the 15-kb deletion can be detected using the read coverage. The APKM of the four amplicons located at the fourth exon of <em>E3</em> was compared between the <em>E3</em> and <em>e3tr</em> alleles (<a reveal-id="dsz005-F3" data-open="dsz005-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The number of accessions classified as <em>E3</em> and <em>e3tr</em> by additional marker analysis was 152 and 36, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The APKM of four amplicons on the fourth exon of the <em>e3tr</em> allele was almost zero, whereas the APKM of the <em>E3</em> allele varied depending on the accessions, and it was difficult to judge the presence or absence of deletion from the APKM of each amplicon. However, it was possible to classify the presence or absence of deletion clearly (<a reveal-id="dsz005-F3" data-open="dsz005-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>) when the average APKM of the four amplicons was used instead, because differences in amplification efficiency due to sequence variation at the priming site can be cancelled using the APKM of multiple amplicons. </p>                        <a id="137025732" scrollto-destination="137025732"></a>
<div id="dsz005-F3" data-id="dsz005-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f3.png?Expires=1576535926&amp;Signature=KB3GrCf4u5D6q0fFEDx9KybG4Wcvygm6IbJLQuk~YXKmXx7SqX-wWfq8nmlNwf8m32PRwNlu0GfFZ8AvAWVHIXsWdqR71xaQVUQqC-sqyRNPUmeBbDKWQ7tkfEiU3IW561rxggk9eyw9SjgJzrKhOyAJCkRPOyWDY0sz-cOzFaoAWtvlhcvaGZcefCqPAwL3E1WExWLT5os6KnpziJwN3dcYgUA7oVn-r58hDqXoNAZqSyh34cIw5OVaLaoWAlXSmmE4FmWuzrpgB1iiwLCkKhEtf9jGJTrZk-Xfna4CelXeVYv9sArvPwlthAPlZ~9apqf3~ZYTtioZrI5PiXFepg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of mapped sequence reads of two different alleles in the E3 gene region. (A) Top: Read coverage of Williams 82 with the default parameters of TMAP. Bottom: Read coverage of PGC010 with the default and optimized parameters of TMAP. The E3 gene of PGC010 has a large deletion in the fourth exon. (B) APKM of amplicons from variant (e3T) and wild type (WT) on the fourth exon of E3. Average APKM of four amplicons (left most) and APKM of each amplicon—(a) AMPL1040313, (b) AMPL1040314, (c) AMPL1037722, and (d) AMPL1036854." path-from-xml="dsz005f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025732/dsz005f3.tif" path-from-xml="dsz005f3.tif" target="_blank">Open in new tab</a><a section="137025732" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f3.png?Expires=1576535926&amp;Signature=j1NfnC6GOLSSZB59W6vMQlUFvOU5cf94-4G-GPN~kkESR6lS90nYv~MUWWCN4BUQ94oaNSlEPDHZWTbYBw1e44SF0JqMaYgIRvrJiaXqKl9sF~EgBlza-GGy0qH~XFln7gIleRyKRU6wi2dR5uGp8JTFLp-suAShiNbXr8Ref90K6z9YViGqV7ZSAAyJ2PsScCZuD-SXCVhOcG~6X5k7-kvzc8Y6u8sOHo25ULinrDdR6DS8FjSMNt4t6NCdntWO0QOHdNi-v9WoqtQZaPQlxO0yLYkTW4CczmD1G1O-UXrZ36SFPclLs9PReYeqsw-VZucCwx8fHakcQqvdtK0hKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025732&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of mapped sequence reads of two different alleles in the <em>E3</em> gene region. (A) Top: Read coverage of Williams 82 with the default parameters of TMAP. Bottom: Read coverage of PGC010 with the default and optimized parameters of TMAP. The <em>E3</em> gene of PGC010 has a large deletion in the fourth exon. (B) APKM of amplicons from variant (<em>e3T</em>) and wild type (WT) on the fourth exon of <em>E3</em>. Average APKM of four amplicons (left most) and APKM of each amplicon—(a) AMPL1040313, (b) AMPL1040314, (c) AMPL1037722, and (d) AMPL1036854.</p></div>
</div>
<div content-id="dsz005-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f3.png?Expires=1576535926&amp;Signature=KB3GrCf4u5D6q0fFEDx9KybG4Wcvygm6IbJLQuk~YXKmXx7SqX-wWfq8nmlNwf8m32PRwNlu0GfFZ8AvAWVHIXsWdqR71xaQVUQqC-sqyRNPUmeBbDKWQ7tkfEiU3IW561rxggk9eyw9SjgJzrKhOyAJCkRPOyWDY0sz-cOzFaoAWtvlhcvaGZcefCqPAwL3E1WExWLT5os6KnpziJwN3dcYgUA7oVn-r58hDqXoNAZqSyh34cIw5OVaLaoWAlXSmmE4FmWuzrpgB1iiwLCkKhEtf9jGJTrZk-Xfna4CelXeVYv9sArvPwlthAPlZ~9apqf3~ZYTtioZrI5PiXFepg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of mapped sequence reads of two different alleles in the E3 gene region. (A) Top: Read coverage of Williams 82 with the default parameters of TMAP. Bottom: Read coverage of PGC010 with the default and optimized parameters of TMAP. The E3 gene of PGC010 has a large deletion in the fourth exon. (B) APKM of amplicons from variant (e3T) and wild type (WT) on the fourth exon of E3. Average APKM of four amplicons (left most) and APKM of each amplicon—(a) AMPL1040313, (b) AMPL1040314, (c) AMPL1037722, and (d) AMPL1036854." path-from-xml="dsz005f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025732/dsz005f3.tif" path-from-xml="dsz005f3.tif" target="_blank">Open in new tab</a><a section="137025732" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f3.png?Expires=1576535926&amp;Signature=j1NfnC6GOLSSZB59W6vMQlUFvOU5cf94-4G-GPN~kkESR6lS90nYv~MUWWCN4BUQ94oaNSlEPDHZWTbYBw1e44SF0JqMaYgIRvrJiaXqKl9sF~EgBlza-GGy0qH~XFln7gIleRyKRU6wi2dR5uGp8JTFLp-suAShiNbXr8Ref90K6z9YViGqV7ZSAAyJ2PsScCZuD-SXCVhOcG~6X5k7-kvzc8Y6u8sOHo25ULinrDdR6DS8FjSMNt4t6NCdntWO0QOHdNi-v9WoqtQZaPQlxO0yLYkTW4CczmD1G1O-UXrZ36SFPclLs9PReYeqsw-VZucCwx8fHakcQqvdtK0hKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025732&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Distribution of mapped sequence reads of two different alleles in the <em>E3</em> gene region. (A) Top: Read coverage of Williams 82 with the default parameters of TMAP. Bottom: Read coverage of PGC010 with the default and optimized parameters of TMAP. The <em>E3</em> gene of PGC010 has a large deletion in the fourth exon. (B) APKM of amplicons from variant (<em>e3T</em>) and wild type (WT) on the fourth exon of <em>E3</em>. Average APKM of four amplicons (left most) and APKM of each amplicon—(a) AMPL1040313, (b) AMPL1040314, (c) AMPL1037722, and (d) AMPL1036854.</p></div>
</div>
<p>As described above, appropriate parameters are required to map short amplicon reads to the correct genomic region. As the AmpliSeq technology has been mainly used in animals in which palaeopolyploidy is considerably rare, no such limitation has been reported. Soybean is an ancient tetraploid, which underwent two whole-genome duplications (palaeopolyploidy); most of the genes have paralogous genes with multiple copies.<a reveal-id="dsz005-B23" data-open="dsz005-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The information provided above would be useful when the AmpliSeq technology is applied to plant species, which have experienced whole-genome duplication or triplications.</p>                    <h3 scrollto-destination="137025734" id="137025734" class="section-title">3.3. Detected variants of flowering time-related genes</h3>
<p>A total of 192 soybean mini-core collection was analysed to detect novel variants in flowering time-related genes by AmpliSeq. Among the 461 variants (SNPs or InDels) detected in the target regions, 311 (67.5%) sites have already been reported or registered in dbSNP,<a reveal-id="dsz005-B25" data-open="dsz005-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> whereas 150 sites (32.5%) were novel (<a reveal-id="dsz005-T2" data-open="dsz005-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The variants detected were compared in depth with information of flowering time-related genes, <em>E1</em>, <em>E2</em>, <em>E3</em>, <em>E4</em>, <em>FT2a</em>, <em>FT5a</em>, and their homologues.<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsz005-B17" data-open="dsz005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsz005-B18" data-open="dsz005-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Further, we performed linear regression analysis to detect responsible variants associated with flowering time under long-day field conditions. Among the 461 variants, 207, 206, and 219 were found to be potentially associated with flowering time in 2011, 2012, and 2013 by the simple linear regression analysis, respectively (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The most significantly associated variant with flowering time was SNP (rs124971350) at <em>E2</em> (<em>e2</em> allele) (<em>P </em>&lt;<em> </em>2.0e<sup>−16</sup> for 3 yrs) (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The second most significant variant was a large deletion in <em>E3</em> (<em>e3-tr</em> allele) (<em>P </em>&lt;<em> </em>1.9e<sup>−13</sup>, 1.6e<sup>−13</sup>, and 5.2e<sup>−14</sup> for 2011, 2012, and 2013, respectively). The other seven genes, <em>WD repeat-containing protein 61</em>, <em>Dt1</em>, <em>MADS-box protein</em>, two genes of <em>two-component response regulator</em>-like genes, <em>FT2a</em>, and <em>PhyB</em>, showed highly significant association with flowering time (<em>P </em>&lt;<em> </em>0.0001) (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>).</p>                        <a id="137025736" scrollto-destination="137025736"></a>
<div content-id="dsz005-T2" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T2" class="table-wrap-title" data-id="dsz005-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Classification of detected variants of 28 flowering time-related genes by snpEff analysis</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Glyma ID </th>
<th>Description </th>
<th>Gene symbol </th>
<th>Number of variants in total </th>
<th>Number of novel variants </th>
<th>Number of known variants </th>
<th>Number of missense variants </th>
<th>Number of ‘high’ effect variants to gene function<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>10 </td>
<td>2 </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>2 </td>
<td>0 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>70 </td>
<td>5 </td>
<td>65 </td>
<td>35 </td>
<td>5 </td>
</tr>
<tr>
<td>Glyma.04G143300 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1Lb</em> </td>
<td>3 </td>
<td>1 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1La</em> </td>
<td>4 </td>
<td>0 </td>
<td>4 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1</em> </td>
<td>5 </td>
<td>2 </td>
<td>3 </td>
<td>3 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>9 </td>
<td>2 </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>6 </td>
<td>1 </td>
<td>5 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>10 </td>
<td>6 </td>
<td>4 </td>
<td>2 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>1 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>20 </td>
<td>8 </td>
<td>12 </td>
<td>3 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>2 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>18 </td>
<td>3 </td>
<td>15 </td>
<td>5 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>27 </td>
<td>9 </td>
<td>18 </td>
<td>7 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>8 </td>
<td>2 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>22 </td>
<td>13 </td>
<td>9 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>9 </td>
<td>3 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>20 </td>
<td>5 </td>
<td>15 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>45 </td>
<td>15 </td>
<td>30 </td>
<td>7 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>15 </td>
<td>1 </td>
<td>14 </td>
<td>0 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>Achaete-scute transcription factor-related </td>
<td>
<em> </em> </td>
<td>17 </td>
<td>10 </td>
<td>7 </td>
<td>0 </td>
<td>6 </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>11 </td>
<td>3 </td>
<td>8 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>5 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>16 </td>
<td>8 </td>
<td>8 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>42 </td>
<td>11 </td>
<td>31 </td>
<td>8 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>16 </td>
<td>2 </td>
<td>14 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>26 </td>
<td>26 </td>
<td>0 </td>
<td>5 </td>
<td>7 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Glyma ID </th>
<th>Description </th>
<th>Gene symbol </th>
<th>Number of variants in total </th>
<th>Number of novel variants </th>
<th>Number of known variants </th>
<th>Number of missense variants </th>
<th>Number of ‘high’ effect variants to gene function<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>10 </td>
<td>2 </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>2 </td>
<td>0 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>70 </td>
<td>5 </td>
<td>65 </td>
<td>35 </td>
<td>5 </td>
</tr>
<tr>
<td>Glyma.04G143300 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1Lb</em> </td>
<td>3 </td>
<td>1 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1La</em> </td>
<td>4 </td>
<td>0 </td>
<td>4 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1</em> </td>
<td>5 </td>
<td>2 </td>
<td>3 </td>
<td>3 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>9 </td>
<td>2 </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>6 </td>
<td>1 </td>
<td>5 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>10 </td>
<td>6 </td>
<td>4 </td>
<td>2 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>1 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>20 </td>
<td>8 </td>
<td>12 </td>
<td>3 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>2 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>18 </td>
<td>3 </td>
<td>15 </td>
<td>5 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>27 </td>
<td>9 </td>
<td>18 </td>
<td>7 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>8 </td>
<td>2 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>22 </td>
<td>13 </td>
<td>9 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>9 </td>
<td>3 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>20 </td>
<td>5 </td>
<td>15 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>45 </td>
<td>15 </td>
<td>30 </td>
<td>7 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>15 </td>
<td>1 </td>
<td>14 </td>
<td>0 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>Achaete-scute transcription factor-related </td>
<td>
<em> </em> </td>
<td>17 </td>
<td>10 </td>
<td>7 </td>
<td>0 </td>
<td>6 </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>11 </td>
<td>3 </td>
<td>8 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>5 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>16 </td>
<td>8 </td>
<td>8 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>42 </td>
<td>11 </td>
<td>31 </td>
<td>8 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>16 </td>
<td>2 </td>
<td>14 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>26 </td>
<td>26 </td>
<td>0 </td>
<td>5 </td>
<td>7 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn10"><div class="fn">
<div class="label">a</div>
<p>High-impact variant to gene function includes criteria of stop lost, stop gained, and frameshift.</p>
</div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025736" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T2" class="table-wrap-title" data-id="dsz005-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Classification of detected variants of 28 flowering time-related genes by snpEff analysis</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Glyma ID </th>
<th>Description </th>
<th>Gene symbol </th>
<th>Number of variants in total </th>
<th>Number of novel variants </th>
<th>Number of known variants </th>
<th>Number of missense variants </th>
<th>Number of ‘high’ effect variants to gene function<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>10 </td>
<td>2 </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>2 </td>
<td>0 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>70 </td>
<td>5 </td>
<td>65 </td>
<td>35 </td>
<td>5 </td>
</tr>
<tr>
<td>Glyma.04G143300 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1Lb</em> </td>
<td>3 </td>
<td>1 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1La</em> </td>
<td>4 </td>
<td>0 </td>
<td>4 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1</em> </td>
<td>5 </td>
<td>2 </td>
<td>3 </td>
<td>3 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>9 </td>
<td>2 </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>6 </td>
<td>1 </td>
<td>5 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>10 </td>
<td>6 </td>
<td>4 </td>
<td>2 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>1 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>20 </td>
<td>8 </td>
<td>12 </td>
<td>3 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>2 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>18 </td>
<td>3 </td>
<td>15 </td>
<td>5 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>27 </td>
<td>9 </td>
<td>18 </td>
<td>7 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>8 </td>
<td>2 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>22 </td>
<td>13 </td>
<td>9 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>9 </td>
<td>3 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>20 </td>
<td>5 </td>
<td>15 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>45 </td>
<td>15 </td>
<td>30 </td>
<td>7 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>15 </td>
<td>1 </td>
<td>14 </td>
<td>0 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>Achaete-scute transcription factor-related </td>
<td>
<em> </em> </td>
<td>17 </td>
<td>10 </td>
<td>7 </td>
<td>0 </td>
<td>6 </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>11 </td>
<td>3 </td>
<td>8 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>5 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>16 </td>
<td>8 </td>
<td>8 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>42 </td>
<td>11 </td>
<td>31 </td>
<td>8 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>16 </td>
<td>2 </td>
<td>14 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>26 </td>
<td>26 </td>
<td>0 </td>
<td>5 </td>
<td>7 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Glyma ID </th>
<th>Description </th>
<th>Gene symbol </th>
<th>Number of variants in total </th>
<th>Number of novel variants </th>
<th>Number of known variants </th>
<th>Number of missense variants </th>
<th>Number of ‘high’ effect variants to gene function<a reveal-id="tblfn10" data-open="tblfn10" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
</tr></thead>
<tbody>
<tr>
<td>Glyma.02G069500 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFTL7</em> </td>
<td>10 </td>
<td>2 </td>
<td>8 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G194700 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>GmTFL2</em> </td>
<td>2 </td>
<td>0 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.03G227300 </td>
<td>Phytochrome A </td>
<td>
<em>GmPHYA4</em> </td>
<td>70 </td>
<td>5 </td>
<td>65 </td>
<td>35 </td>
<td>5 </td>
</tr>
<tr>
<td>Glyma.04G143300 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1Lb</em> </td>
<td>3 </td>
<td>1 </td>
<td>2 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.04G156400 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1La</em> </td>
<td>4 </td>
<td>0 </td>
<td>4 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain-containing protein </td>
<td>
<em>E1</em> </td>
<td>5 </td>
<td>2 </td>
<td>3 </td>
<td>3 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.08G363100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT4</em> </td>
<td>9 </td>
<td>2 </td>
<td>7 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.08G363200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT6</em> </td>
<td>6 </td>
<td>1 </td>
<td>5 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G035500 </td>
<td>Phytochrome B </td>
<td>
<em>GmPHYB1</em> </td>
<td>10 </td>
<td>6 </td>
<td>4 </td>
<td>2 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.09G143500 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL4</em> </td>
<td>1 </td>
<td>1 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.10G141400 </td>
<td>Phytochrome A </td>
<td>
<em> </em> </td>
<td>20 </td>
<td>8 </td>
<td>12 </td>
<td>3 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>2 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>18 </td>
<td>3 </td>
<td>15 </td>
<td>5 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em> </em> </td>
<td>27 </td>
<td>9 </td>
<td>18 </td>
<td>7 </td>
<td>2 </td>
</tr>
<tr>
<td>Glyma.16G044100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5a/GmFTL4</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G044200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3a/GmFTL1</em> </td>
<td>8 </td>
<td>2 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>22 </td>
<td>13 </td>
<td>9 </td>
<td>2 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.16G151000 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2b/GmFTL5</em> </td>
<td>9 </td>
<td>3 </td>
<td>6 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G196300 </td>
<td>BROTHER OF FT AND TFL 1 </td>
<td>
<em>GmTFL3</em> </td>
<td>20 </td>
<td>5 </td>
<td>15 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em> </em> </td>
<td>45 </td>
<td>15 </td>
<td>30 </td>
<td>7 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>15 </td>
<td>1 </td>
<td>14 </td>
<td>0 </td>
<td>1 </td>
</tr>
<tr>
<td>Glyma.17G090500 </td>
<td>Achaete-scute transcription factor-related </td>
<td>
<em> </em> </td>
<td>17 </td>
<td>10 </td>
<td>7 </td>
<td>0 </td>
<td>6 </td>
</tr>
<tr>
<td>Glyma.19G108100 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT3b/GmFTL2</em> </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G108200 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT5b/GmFTL6</em> </td>
<td>11 </td>
<td>3 </td>
<td>8 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>14 </td>
<td>5 </td>
<td>9 </td>
<td>5 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>16 </td>
<td>8 </td>
<td>8 </td>
<td>5 </td>
<td>3 </td>
</tr>
<tr>
<td>Glyma.19G260400 </td>
<td>Pseudo-response regulator 5 </td>
<td>
<em> </em> </td>
<td>42 </td>
<td>11 </td>
<td>31 </td>
<td>8 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.20G090000 </td>
<td>Phytochrome A </td>
<td>
<em>E4</em> </td>
<td>16 </td>
<td>2 </td>
<td>14 </td>
<td>1 </td>
<td>0 </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em> </em> </td>
<td>26 </td>
<td>26 </td>
<td>0 </td>
<td>5 </td>
<td>7 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn10"><div class="fn">
<div class="label">a</div>
<p>High-impact variant to gene function includes criteria of stop lost, stop gained, and frameshift.</p>
</div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025736" target="_blank">Open in new tab</a></div>
</div></div>                        <a id="137025737" scrollto-destination="137025737"></a>
<div content-id="dsz005-T3" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T3" class="table-wrap-title" data-id="dsz005-T3">
<div class="label">Table 3</div>
<div class="caption"><p>DNA polymorphisms in nine genes significantly associated with flowering time variation in the mini-core collection as revealed by the single linear regression analysis</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Gene description </th>
<th rowspan="2">Gene symbol<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Chromosome </th>
<th rowspan="2">Position (bp) </th>
<th rowspan="2">Reference sequence </th>
<th rowspan="2">Alternative sequence </th>
<th rowspan="2">SNP name </th>
<th colspan="3">
<em>P</em> value of association tests<hr> </th>
<th rowspan="2">Functional effect </th>
<th rowspan="2">Amino acid change </th>
<th rowspan="2">Functional class </th>
</tr>
<tr>
<th>2011 </th>
<th>2012 </th>
<th>2013 </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain- containing protein </td>
<td>
<em>E1</em> </td>
<td>Chr06 </td>
<td>20207322 </td>
<td>C </td>
<td>G </td>
<td>rs123612969 (<em>e1-as</em>) </td>
<td>0.0226 </td>
<td>0.0153 </td>
<td>0.0333 </td>
<td>Missense_variant </td>
<td>p.Thr75Arg/c.224C&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>A </td>
<td>T </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>Stop-gained </td>
<td>p.Lys527*/c.1582A&gt;T </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Chr12 </td>
<td>5508242 </td>
<td>T </td>
<td>G </td>
<td>rs125308101 </td>
<td>0.00258 </td>
<td>0.0082 </td>
<td>0.0019 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508672 </td>
<td>CT </td>
<td>C </td>
<td>rs745192414 </td>
<td>0.00155 </td>
<td>0.00261 </td>
<td>0.00503 </td>
<td>Intron_variant </td>
<td>c.-120 + 219delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508702 </td>
<td>T </td>
<td>C </td>
<td>rs743387935 </td>
<td>0.0211 </td>
<td>0.0228 </td>
<td>0.0109 </td>
<td>Intron_variant </td>
<td>c.-119-198T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509310 </td>
<td>G </td>
<td>A </td>
<td>rs388619068 </td>
<td>0.0146 </td>
<td>0.0119 </td>
<td>0.00907 </td>
<td>Missense_variant </td>
<td>p.Asp98Asn/c.292G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509317 </td>
<td>C </td>
<td>T </td>
<td>rs125308103 </td>
<td>0.00107 </td>
<td>0.00148 </td>
<td>0.00319 </td>
<td>Missense_variant </td>
<td>p.Ser100Leu/c.299C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5519728 </td>
<td>A </td>
<td>C </td>
<td>rs125308115 </td>
<td>0.00229 </td>
<td>0.00749 </td>
<td>0.00169 </td>
<td>Missense_variant </td>
<td>p.Lys378Gln/c.1132A&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520578 </td>
<td>A </td>
<td>G </td>
<td>rs389122657 </td>
<td>0.014 </td>
<td>0.0112 </td>
<td>0.00865 </td>
<td>Synonymous_ variant </td>
<td>p.Ala504Ala/c.1512A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>T </td>
<td>C </td>
<td>rs125308117 </td>
<td>8.96E-10 </td>
<td>1.76E-08 </td>
<td>7.74E-11 </td>
<td>Stop_lost </td>
<td>p.Ter627Glnext*?/c.1879T&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5521029 </td>
<td>G </td>
<td>T </td>
<td>rs743179814 </td>
<td>0.0244 </td>
<td>0.0203 </td>
<td>0.0089 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*82G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em>-</em> </td>
<td>Chr15 </td>
<td>11436193 </td>
<td>G </td>
<td>A </td>
<td>rs388435741 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11441207 </td>
<td>C </td>
<td>T </td>
<td>rs126279495 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>Missense_variant </td>
<td>p.Val394Ile/c.1180G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11442400 </td>
<td>A </td>
<td>C </td>
<td>rs126279502 </td>
<td>0.0753 </td>
<td>0.0769 </td>
<td>0.0431 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-14T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGC </td>
<td>T </td>
<td>rs392064733_1 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-50_-59delATAAGAAAGC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGCA </td>
<td>TA </td>
<td>rs392064733_2 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-49_-58delTAAGAAAGCA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110991 </td>
<td>A </td>
<td>AATAT </td>
<td>rs864598505_1 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-56_202-55insATAT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111033 </td>
<td>T </td>
<td>A </td>
<td>rs126829817 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-15T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111042 </td>
<td>T </td>
<td>C </td>
<td>rs126829818 </td>
<td>0.0697 </td>
<td>0.21 </td>
<td>0.0419 </td>
<td>Splice_region_ variant </td>
<td>c.202T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111349 </td>
<td>A </td>
<td>G </td>
<td>rs126829819 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.304 + 72A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114633 </td>
<td>G </td>
<td>A </td>
<td>Chr16_31114633 </td>
<td>5.46E-06 </td>
<td>4.14E-06 </td>
<td>3.05E-07 </td>
<td>Missense_ variant </td>
<td>p.Gly169Asp/c.506G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114658 </td>
<td>AGA </td>
<td>AA </td>
<td>Chr16_31114658_2 </td>
<td>0.0765 </td>
<td>0.0538 </td>
<td>0.043 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*1delG </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114930 </td>
<td>G </td>
<td>T </td>
<td>rs126829846 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*272G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em>-</em> </td>
<td>Chr16 </td>
<td>36179909 </td>
<td>T </td>
<td>G </td>
<td>rs126888526 </td>
<td>1.13E-07 </td>
<td>3.18E-06 </td>
<td>3.35E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180002 </td>
<td>G </td>
<td>A </td>
<td>rs126888527 </td>
<td>1.09E-07 </td>
<td>1.63E-06 </td>
<td>1.98E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180087 </td>
<td>T </td>
<td>TGA </td>
<td>Chr16_36180087 </td>
<td>8.8E-08 </td>
<td>2.16E-06 </td>
<td>2.07E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*39_*40insTC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180122 </td>
<td>A </td>
<td>C </td>
<td>rs126888528 </td>
<td>4.19E-07 </td>
<td>9.01E-06 </td>
<td>1.03E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*5T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180183 </td>
<td>T </td>
<td>A </td>
<td>rs389022394 </td>
<td>4.96E-07 </td>
<td>1.03E-05 </td>
<td>1.32E-06 </td>
<td>Missense_variant </td>
<td>p.Thr219Ser/c.655A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182854 </td>
<td>T </td>
<td>TA </td>
<td>Chr16_36182854 </td>
<td>6.15E-06 </td>
<td>4.11E-05 </td>
<td>3.79E-06 </td>
<td>Intron_variant </td>
<td>c.511-49_511-50insT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182857 </td>
<td>T </td>
<td>A </td>
<td>rs126888574 </td>
<td>8.41E-07 </td>
<td>7.3E-06 </td>
<td>5.2E-07 </td>
<td>Intron_variant </td>
<td>c.511-52A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183423 </td>
<td>T </td>
<td>G </td>
<td>rs126888580_2 </td>
<td>6.9E-06 </td>
<td>5.27E-06 </td>
<td>2.61E-06 </td>
<td>Intron_variant </td>
<td>c.427-19A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183426 </td>
<td>G </td>
<td>C </td>
<td>rs126888581_1 </td>
<td>0.00286 </td>
<td>0.00159 </td>
<td>0.0018 </td>
<td>Intron_variant </td>
<td>c.427-22C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183435 </td>
<td>G </td>
<td>C </td>
<td>rs126888582 </td>
<td>6.8E-06 </td>
<td>5.07E-06 </td>
<td>2.66E-06 </td>
<td>Intron_variant </td>
<td>c.427-31C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183450 </td>
<td>G </td>
<td>A </td>
<td>rs126888583 </td>
<td>0.0104 </td>
<td>0.00721 </td>
<td>0.00804 </td>
<td>Intron_variant </td>
<td>c.427-46C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183510 </td>
<td>C </td>
<td>T </td>
<td>rs126888584 </td>
<td>0.000219 </td>
<td>0.00275 </td>
<td>0.000461 </td>
<td>Intron_variant </td>
<td>c.427-106G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183518 </td>
<td>A </td>
<td>G </td>
<td>rs126888585 </td>
<td>3.22E-10 </td>
<td>9.24E-09 </td>
<td>3.46E-10 </td>
<td>Intron_variant </td>
<td>c.427-114T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183541 </td>
<td>C </td>
<td>A </td>
<td>rs126888586 </td>
<td>2.98E-10 </td>
<td>8.83E-09 </td>
<td>3.27E-10 </td>
<td>Intron_variant </td>
<td>c.427-137G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183556 </td>
<td>T </td>
<td>C </td>
<td>Chr16_36183556 </td>
<td>0.0507 </td>
<td>0.0363 </td>
<td>0.048 </td>
<td>Intron_variant </td>
<td>c.427-152A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183568 </td>
<td>T </td>
<td>A </td>
<td>rs126888588 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-164A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183573 </td>
<td>T </td>
<td>A </td>
<td>rs126888589 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-169A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184165 </td>
<td>T </td>
<td>C </td>
<td>rs126888605 </td>
<td>4.71E-05 </td>
<td>3.54E-05 </td>
<td>0.000019 </td>
<td>Intron_variant </td>
<td>c.327-41A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184729 </td>
<td>T </td>
<td>C </td>
<td>rs126888609 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Missense_variant </td>
<td>p.Thr79Ala/c.235A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184733 </td>
<td>C </td>
<td>T </td>
<td>rs126888610 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser77Ser/c.231G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184819 </td>
<td>A </td>
<td>T </td>
<td>rs126888611 </td>
<td>0.0341 </td>
<td>0.0817 </td>
<td>0.0244 </td>
<td>Intron_variant </td>
<td>c.183-38T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187211 </td>
<td>A </td>
<td>C </td>
<td>rs744233319 </td>
<td>0.000287 </td>
<td>0.000744 </td>
<td>0.00118 </td>
<td>Synonymous_ variant </td>
<td>p.Ser36Ser/c.108T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187552 </td>
<td>T </td>
<td>G </td>
<td>rs126888636 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187600 </td>
<td>A </td>
<td>G </td>
<td>rs126888637 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Chr17 </td>
<td>3955280 </td>
<td>A </td>
<td>AT </td>
<td>rs126942305_1 </td>
<td>0.00214 </td>
<td>0.00191 </td>
<td>0.00197 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955411 </td>
<td>A </td>
<td>G </td>
<td>rs126942313 </td>
<td>2.56E-10 </td>
<td>3.12E-10 </td>
<td>1.07E-10 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955475 </td>
<td>TC </td>
<td>AA </td>
<td>rs126942314_1 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955476 </td>
<td>C </td>
<td>A </td>
<td>rs126942315 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955546 </td>
<td>T </td>
<td>A </td>
<td>rs126942316 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955554 </td>
<td>T </td>
<td>C </td>
<td>rs126942317 </td>
<td>0.031 </td>
<td>0.0299 </td>
<td>0.0358 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*117A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955716 </td>
<td>A </td>
<td>G </td>
<td>rs388258139 </td>
<td>0.00418 </td>
<td>0.00761 </td>
<td>0.00338 </td>
<td>Synonymous_ variant </td>
<td>p.Ala307Ala/c.921T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955763 </td>
<td>C </td>
<td>CG </td>
<td>Chr17_3955763 </td>
<td>0.00444 </td>
<td>0.00194 </td>
<td>0.00318 </td>
<td>Frameshift_ variant </td>
<td>p.Val291_Ala292fs/c.873_874insC </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955764 </td>
<td>A </td>
<td>G </td>
<td>rs126942318 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val291Val/c.873T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955884 </td>
<td>G </td>
<td>C </td>
<td>rs126942319 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val251Val/c.753C&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956142 </td>
<td>T </td>
<td>C </td>
<td>rs126942321 </td>
<td>6.36E-08 </td>
<td>3.72E-08 </td>
<td>1.25E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala165Ala/c.495A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956163 </td>
<td>T </td>
<td>C </td>
<td>rs126942322 </td>
<td>0.00644 </td>
<td>0.00295 </td>
<td>0.00466 </td>
<td>Synonymous_ variant </td>
<td>p.Lys158Lys/c.474A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958319 </td>
<td>C </td>
<td>G </td>
<td>rs126942340 </td>
<td>9.03E-10 </td>
<td>4.86E-10 </td>
<td>1.98E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser16Ser/c.48G&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958374 </td>
<td>T </td>
<td>G </td>
<td>rs126942341 </td>
<td>2.32E-10 </td>
<td>9.96E-11 </td>
<td>5.12E-11 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-8A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>T </td>
<td>A </td>
<td>rs127928573 (<em>dt1</em>) </td>
<td>0.00122 </td>
<td>0.00587 </td>
<td>0.000401 </td>
<td>Missense_variant </td>
<td>p.Arg166Trp/c.496A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183808 </td>
<td>C </td>
<td>T </td>
<td>rs745009806 </td>
<td>0.0319 </td>
<td>0.0184 </td>
<td>0.0153 </td>
<td>Missense_variant </td>
<td>p.Arg130Lys/c.389G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183859 </td>
<td>G </td>
<td>A </td>
<td>rs127928574 </td>
<td>4.29E-11 </td>
<td>3.71E-11 </td>
<td>6.02E-10 </td>
<td>Missense_variant </td>
<td>p.Pro113Leu/c.338C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45184581 </td>
<td>G </td>
<td>T </td>
<td>Chr19_45184581 </td>
<td>0.014 </td>
<td>0.153 </td>
<td>0.0428 </td>
<td>Splice_region_ variant </td>
<td>c.202C&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45185131 </td>
<td>C </td>
<td>T </td>
<td>Chr19_45185131 </td>
<td>0.00805 </td>
<td>0.00351 </td>
<td>0.00764 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-142G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Chr19 </td>
<td>47633086 </td>
<td>T </td>
<td>TA </td>
<td>Chr19_47633086 </td>
<td>0.000763 </td>
<td>0.000353 </td>
<td>0.00085 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-693_-694insA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47634596 </td>
<td>A </td>
<td>G </td>
<td>rs127944661 </td>
<td>0.0052 </td>
<td>0.00108 </td>
<td>0.00429 </td>
<td>Synonymous_ variant </td>
<td>p.Ser43Ser/c.129A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635025 </td>
<td>C </td>
<td>T </td>
<td>rs388644281 </td>
<td>3.18E-05 </td>
<td>3.43E-05 </td>
<td>3.69E-05 </td>
<td>Synonymous_ variant </td>
<td>p.Ile186Ile/c.558C&gt;T </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635737 </td>
<td>C </td>
<td>A </td>
<td>rs389001110 </td>
<td>0.0534 </td>
<td>0.052 </td>
<td>0.048 </td>
<td>Missense_variant </td>
<td>p.Leu424Ile/c.1270C&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636564 </td>
<td>G </td>
<td>A </td>
<td>rs127944664 </td>
<td>0.0098 </td>
<td>0.00232 </td>
<td>0.00888 </td>
<td>Intron_variant </td>
<td>c.2074 + 23G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636607 </td>
<td>G </td>
<td>T </td>
<td>rs127944665 </td>
<td>0.0127 </td>
<td>0.0031 </td>
<td>0.0113 </td>
<td>Intron_variant </td>
<td>c.2074 + 66G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47637258 </td>
<td>A </td>
<td>G </td>
<td>rs393405985_1 </td>
<td>0.044 </td>
<td>0.0447 </td>
<td>0.0427 </td>
<td>Missense_variant </td>
<td>p.Thr832Ala/c.2494A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638302 </td>
<td>G </td>
<td>A </td>
<td>Chr19_47638302 (<em>e3-Mo</em>) </td>
<td>0.0433 </td>
<td>0.0306 </td>
<td>0.0485 </td>
<td>Missense_variant </td>
<td>p.Gly1050Arg/c.3148G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638344 </td>
<td>A </td>
<td>T </td>
<td>rs389636522 </td>
<td>0.00508 </td>
<td>0.00113 </td>
<td>0.00433 </td>
<td>Splice_region_ variant </td>
<td>c.3183A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>  </td>
<td>15 kb deletion </td>
<td>
<em>e3-tr</em> </td>
<td>1.9E-13 </td>
<td>1.61E-13 </td>
<td>5.15E-14 </td>
<td>Loss of exon 4 </td>
<td>  </td>
<td>HIGH </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Scaffold_32 </td>
<td>197169 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_197169 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*958G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197421 </td>
<td>T </td>
<td>C </td>
<td>Scaffold_32_197421 </td>
<td>0.000034 </td>
<td>0.00007 </td>
<td>8.54E-05 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*706A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197459 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_197459 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*668T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198053 </td>
<td>T </td>
<td>A </td>
<td>Scaffold_32_198053 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.*133 + 29A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198773 </td>
<td>A </td>
<td>AT </td>
<td>Scaffold_32_198773 </td>
<td>1.85E-06 </td>
<td>3.75E-06 </td>
<td>2.85E-06 </td>
<td>Frameshift_variant </td>
<td>p.Met737_Ala738fs/c.2209_2210insA </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199551 </td>
<td>A </td>
<td>C </td>
<td>Scaffold_32_199551 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala529Ala/c.1587T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199605 </td>
<td>T </td>
<td>G </td>
<td>Scaffold_32_199605 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.1575-42A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199656 </td>
<td>TA </td>
<td>T </td>
<td>Scaffold_32_199656 </td>
<td>0.0096 </td>
<td>0.0104 </td>
<td>0.011 </td>
<td>Intron_variant </td>
<td>c.1574 + 2delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199722 </td>
<td>C </td>
<td>G </td>
<td>Scaffold_32_199722 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>Missense_variant </td>
<td>p.Gly504Ala/c.1511G&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>202754 </td>
<td>G </td>
<td>A </td>
<td>Scaffold_32_202754 </td>
<td>0.0142 </td>
<td>0.01 </td>
<td>0.0105 </td>
<td>Stop_gained </td>
<td>p.Arg308*/c.922C&gt;T </td>
<td> </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>206717 </td>
<td>A </td>
<td>G </td>
<td>Scaffold_32_206717 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>Intron_variant </td>
<td>c.792 + 27T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>216494 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_216494 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Val168Val/c.504G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>218380 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_218380 </td>
<td>3.29E-09 </td>
<td>4.29E-08 </td>
<td>8.4E-09 </td>
<td>Synonymous_ variant </td>
<td>p.Pro73Pro/c.219T&gt;A </td>
<td>SILENT </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T3" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Gene description </th>
<th rowspan="2">Gene symbol<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Chromosome </th>
<th rowspan="2">Position (bp) </th>
<th rowspan="2">Reference sequence </th>
<th rowspan="2">Alternative sequence </th>
<th rowspan="2">SNP name </th>
<th colspan="3">
<em>P</em> value of association tests<hr> </th>
<th rowspan="2">Functional effect </th>
<th rowspan="2">Amino acid change </th>
<th rowspan="2">Functional class </th>
</tr>
<tr>
<th>2011 </th>
<th>2012 </th>
<th>2013 </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain- containing protein </td>
<td>
<em>E1</em> </td>
<td>Chr06 </td>
<td>20207322 </td>
<td>C </td>
<td>G </td>
<td>rs123612969 (<em>e1-as</em>) </td>
<td>0.0226 </td>
<td>0.0153 </td>
<td>0.0333 </td>
<td>Missense_variant </td>
<td>p.Thr75Arg/c.224C&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>A </td>
<td>T </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>Stop-gained </td>
<td>p.Lys527*/c.1582A&gt;T </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Chr12 </td>
<td>5508242 </td>
<td>T </td>
<td>G </td>
<td>rs125308101 </td>
<td>0.00258 </td>
<td>0.0082 </td>
<td>0.0019 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508672 </td>
<td>CT </td>
<td>C </td>
<td>rs745192414 </td>
<td>0.00155 </td>
<td>0.00261 </td>
<td>0.00503 </td>
<td>Intron_variant </td>
<td>c.-120 + 219delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508702 </td>
<td>T </td>
<td>C </td>
<td>rs743387935 </td>
<td>0.0211 </td>
<td>0.0228 </td>
<td>0.0109 </td>
<td>Intron_variant </td>
<td>c.-119-198T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509310 </td>
<td>G </td>
<td>A </td>
<td>rs388619068 </td>
<td>0.0146 </td>
<td>0.0119 </td>
<td>0.00907 </td>
<td>Missense_variant </td>
<td>p.Asp98Asn/c.292G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509317 </td>
<td>C </td>
<td>T </td>
<td>rs125308103 </td>
<td>0.00107 </td>
<td>0.00148 </td>
<td>0.00319 </td>
<td>Missense_variant </td>
<td>p.Ser100Leu/c.299C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5519728 </td>
<td>A </td>
<td>C </td>
<td>rs125308115 </td>
<td>0.00229 </td>
<td>0.00749 </td>
<td>0.00169 </td>
<td>Missense_variant </td>
<td>p.Lys378Gln/c.1132A&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520578 </td>
<td>A </td>
<td>G </td>
<td>rs389122657 </td>
<td>0.014 </td>
<td>0.0112 </td>
<td>0.00865 </td>
<td>Synonymous_ variant </td>
<td>p.Ala504Ala/c.1512A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>T </td>
<td>C </td>
<td>rs125308117 </td>
<td>8.96E-10 </td>
<td>1.76E-08 </td>
<td>7.74E-11 </td>
<td>Stop_lost </td>
<td>p.Ter627Glnext*?/c.1879T&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5521029 </td>
<td>G </td>
<td>T </td>
<td>rs743179814 </td>
<td>0.0244 </td>
<td>0.0203 </td>
<td>0.0089 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*82G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em>-</em> </td>
<td>Chr15 </td>
<td>11436193 </td>
<td>G </td>
<td>A </td>
<td>rs388435741 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11441207 </td>
<td>C </td>
<td>T </td>
<td>rs126279495 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>Missense_variant </td>
<td>p.Val394Ile/c.1180G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11442400 </td>
<td>A </td>
<td>C </td>
<td>rs126279502 </td>
<td>0.0753 </td>
<td>0.0769 </td>
<td>0.0431 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-14T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGC </td>
<td>T </td>
<td>rs392064733_1 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-50_-59delATAAGAAAGC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGCA </td>
<td>TA </td>
<td>rs392064733_2 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-49_-58delTAAGAAAGCA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110991 </td>
<td>A </td>
<td>AATAT </td>
<td>rs864598505_1 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-56_202-55insATAT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111033 </td>
<td>T </td>
<td>A </td>
<td>rs126829817 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-15T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111042 </td>
<td>T </td>
<td>C </td>
<td>rs126829818 </td>
<td>0.0697 </td>
<td>0.21 </td>
<td>0.0419 </td>
<td>Splice_region_ variant </td>
<td>c.202T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111349 </td>
<td>A </td>
<td>G </td>
<td>rs126829819 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.304 + 72A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114633 </td>
<td>G </td>
<td>A </td>
<td>Chr16_31114633 </td>
<td>5.46E-06 </td>
<td>4.14E-06 </td>
<td>3.05E-07 </td>
<td>Missense_ variant </td>
<td>p.Gly169Asp/c.506G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114658 </td>
<td>AGA </td>
<td>AA </td>
<td>Chr16_31114658_2 </td>
<td>0.0765 </td>
<td>0.0538 </td>
<td>0.043 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*1delG </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114930 </td>
<td>G </td>
<td>T </td>
<td>rs126829846 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*272G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em>-</em> </td>
<td>Chr16 </td>
<td>36179909 </td>
<td>T </td>
<td>G </td>
<td>rs126888526 </td>
<td>1.13E-07 </td>
<td>3.18E-06 </td>
<td>3.35E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180002 </td>
<td>G </td>
<td>A </td>
<td>rs126888527 </td>
<td>1.09E-07 </td>
<td>1.63E-06 </td>
<td>1.98E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180087 </td>
<td>T </td>
<td>TGA </td>
<td>Chr16_36180087 </td>
<td>8.8E-08 </td>
<td>2.16E-06 </td>
<td>2.07E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*39_*40insTC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180122 </td>
<td>A </td>
<td>C </td>
<td>rs126888528 </td>
<td>4.19E-07 </td>
<td>9.01E-06 </td>
<td>1.03E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*5T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180183 </td>
<td>T </td>
<td>A </td>
<td>rs389022394 </td>
<td>4.96E-07 </td>
<td>1.03E-05 </td>
<td>1.32E-06 </td>
<td>Missense_variant </td>
<td>p.Thr219Ser/c.655A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182854 </td>
<td>T </td>
<td>TA </td>
<td>Chr16_36182854 </td>
<td>6.15E-06 </td>
<td>4.11E-05 </td>
<td>3.79E-06 </td>
<td>Intron_variant </td>
<td>c.511-49_511-50insT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182857 </td>
<td>T </td>
<td>A </td>
<td>rs126888574 </td>
<td>8.41E-07 </td>
<td>7.3E-06 </td>
<td>5.2E-07 </td>
<td>Intron_variant </td>
<td>c.511-52A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183423 </td>
<td>T </td>
<td>G </td>
<td>rs126888580_2 </td>
<td>6.9E-06 </td>
<td>5.27E-06 </td>
<td>2.61E-06 </td>
<td>Intron_variant </td>
<td>c.427-19A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183426 </td>
<td>G </td>
<td>C </td>
<td>rs126888581_1 </td>
<td>0.00286 </td>
<td>0.00159 </td>
<td>0.0018 </td>
<td>Intron_variant </td>
<td>c.427-22C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183435 </td>
<td>G </td>
<td>C </td>
<td>rs126888582 </td>
<td>6.8E-06 </td>
<td>5.07E-06 </td>
<td>2.66E-06 </td>
<td>Intron_variant </td>
<td>c.427-31C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183450 </td>
<td>G </td>
<td>A </td>
<td>rs126888583 </td>
<td>0.0104 </td>
<td>0.00721 </td>
<td>0.00804 </td>
<td>Intron_variant </td>
<td>c.427-46C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183510 </td>
<td>C </td>
<td>T </td>
<td>rs126888584 </td>
<td>0.000219 </td>
<td>0.00275 </td>
<td>0.000461 </td>
<td>Intron_variant </td>
<td>c.427-106G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183518 </td>
<td>A </td>
<td>G </td>
<td>rs126888585 </td>
<td>3.22E-10 </td>
<td>9.24E-09 </td>
<td>3.46E-10 </td>
<td>Intron_variant </td>
<td>c.427-114T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183541 </td>
<td>C </td>
<td>A </td>
<td>rs126888586 </td>
<td>2.98E-10 </td>
<td>8.83E-09 </td>
<td>3.27E-10 </td>
<td>Intron_variant </td>
<td>c.427-137G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183556 </td>
<td>T </td>
<td>C </td>
<td>Chr16_36183556 </td>
<td>0.0507 </td>
<td>0.0363 </td>
<td>0.048 </td>
<td>Intron_variant </td>
<td>c.427-152A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183568 </td>
<td>T </td>
<td>A </td>
<td>rs126888588 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-164A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183573 </td>
<td>T </td>
<td>A </td>
<td>rs126888589 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-169A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184165 </td>
<td>T </td>
<td>C </td>
<td>rs126888605 </td>
<td>4.71E-05 </td>
<td>3.54E-05 </td>
<td>0.000019 </td>
<td>Intron_variant </td>
<td>c.327-41A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184729 </td>
<td>T </td>
<td>C </td>
<td>rs126888609 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Missense_variant </td>
<td>p.Thr79Ala/c.235A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184733 </td>
<td>C </td>
<td>T </td>
<td>rs126888610 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser77Ser/c.231G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184819 </td>
<td>A </td>
<td>T </td>
<td>rs126888611 </td>
<td>0.0341 </td>
<td>0.0817 </td>
<td>0.0244 </td>
<td>Intron_variant </td>
<td>c.183-38T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187211 </td>
<td>A </td>
<td>C </td>
<td>rs744233319 </td>
<td>0.000287 </td>
<td>0.000744 </td>
<td>0.00118 </td>
<td>Synonymous_ variant </td>
<td>p.Ser36Ser/c.108T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187552 </td>
<td>T </td>
<td>G </td>
<td>rs126888636 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187600 </td>
<td>A </td>
<td>G </td>
<td>rs126888637 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Chr17 </td>
<td>3955280 </td>
<td>A </td>
<td>AT </td>
<td>rs126942305_1 </td>
<td>0.00214 </td>
<td>0.00191 </td>
<td>0.00197 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955411 </td>
<td>A </td>
<td>G </td>
<td>rs126942313 </td>
<td>2.56E-10 </td>
<td>3.12E-10 </td>
<td>1.07E-10 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955475 </td>
<td>TC </td>
<td>AA </td>
<td>rs126942314_1 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955476 </td>
<td>C </td>
<td>A </td>
<td>rs126942315 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955546 </td>
<td>T </td>
<td>A </td>
<td>rs126942316 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955554 </td>
<td>T </td>
<td>C </td>
<td>rs126942317 </td>
<td>0.031 </td>
<td>0.0299 </td>
<td>0.0358 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*117A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955716 </td>
<td>A </td>
<td>G </td>
<td>rs388258139 </td>
<td>0.00418 </td>
<td>0.00761 </td>
<td>0.00338 </td>
<td>Synonymous_ variant </td>
<td>p.Ala307Ala/c.921T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955763 </td>
<td>C </td>
<td>CG </td>
<td>Chr17_3955763 </td>
<td>0.00444 </td>
<td>0.00194 </td>
<td>0.00318 </td>
<td>Frameshift_ variant </td>
<td>p.Val291_Ala292fs/c.873_874insC </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955764 </td>
<td>A </td>
<td>G </td>
<td>rs126942318 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val291Val/c.873T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955884 </td>
<td>G </td>
<td>C </td>
<td>rs126942319 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val251Val/c.753C&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956142 </td>
<td>T </td>
<td>C </td>
<td>rs126942321 </td>
<td>6.36E-08 </td>
<td>3.72E-08 </td>
<td>1.25E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala165Ala/c.495A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956163 </td>
<td>T </td>
<td>C </td>
<td>rs126942322 </td>
<td>0.00644 </td>
<td>0.00295 </td>
<td>0.00466 </td>
<td>Synonymous_ variant </td>
<td>p.Lys158Lys/c.474A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958319 </td>
<td>C </td>
<td>G </td>
<td>rs126942340 </td>
<td>9.03E-10 </td>
<td>4.86E-10 </td>
<td>1.98E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser16Ser/c.48G&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958374 </td>
<td>T </td>
<td>G </td>
<td>rs126942341 </td>
<td>2.32E-10 </td>
<td>9.96E-11 </td>
<td>5.12E-11 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-8A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>T </td>
<td>A </td>
<td>rs127928573 (<em>dt1</em>) </td>
<td>0.00122 </td>
<td>0.00587 </td>
<td>0.000401 </td>
<td>Missense_variant </td>
<td>p.Arg166Trp/c.496A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183808 </td>
<td>C </td>
<td>T </td>
<td>rs745009806 </td>
<td>0.0319 </td>
<td>0.0184 </td>
<td>0.0153 </td>
<td>Missense_variant </td>
<td>p.Arg130Lys/c.389G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183859 </td>
<td>G </td>
<td>A </td>
<td>rs127928574 </td>
<td>4.29E-11 </td>
<td>3.71E-11 </td>
<td>6.02E-10 </td>
<td>Missense_variant </td>
<td>p.Pro113Leu/c.338C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45184581 </td>
<td>G </td>
<td>T </td>
<td>Chr19_45184581 </td>
<td>0.014 </td>
<td>0.153 </td>
<td>0.0428 </td>
<td>Splice_region_ variant </td>
<td>c.202C&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45185131 </td>
<td>C </td>
<td>T </td>
<td>Chr19_45185131 </td>
<td>0.00805 </td>
<td>0.00351 </td>
<td>0.00764 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-142G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Chr19 </td>
<td>47633086 </td>
<td>T </td>
<td>TA </td>
<td>Chr19_47633086 </td>
<td>0.000763 </td>
<td>0.000353 </td>
<td>0.00085 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-693_-694insA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47634596 </td>
<td>A </td>
<td>G </td>
<td>rs127944661 </td>
<td>0.0052 </td>
<td>0.00108 </td>
<td>0.00429 </td>
<td>Synonymous_ variant </td>
<td>p.Ser43Ser/c.129A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635025 </td>
<td>C </td>
<td>T </td>
<td>rs388644281 </td>
<td>3.18E-05 </td>
<td>3.43E-05 </td>
<td>3.69E-05 </td>
<td>Synonymous_ variant </td>
<td>p.Ile186Ile/c.558C&gt;T </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635737 </td>
<td>C </td>
<td>A </td>
<td>rs389001110 </td>
<td>0.0534 </td>
<td>0.052 </td>
<td>0.048 </td>
<td>Missense_variant </td>
<td>p.Leu424Ile/c.1270C&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636564 </td>
<td>G </td>
<td>A </td>
<td>rs127944664 </td>
<td>0.0098 </td>
<td>0.00232 </td>
<td>0.00888 </td>
<td>Intron_variant </td>
<td>c.2074 + 23G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636607 </td>
<td>G </td>
<td>T </td>
<td>rs127944665 </td>
<td>0.0127 </td>
<td>0.0031 </td>
<td>0.0113 </td>
<td>Intron_variant </td>
<td>c.2074 + 66G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47637258 </td>
<td>A </td>
<td>G </td>
<td>rs393405985_1 </td>
<td>0.044 </td>
<td>0.0447 </td>
<td>0.0427 </td>
<td>Missense_variant </td>
<td>p.Thr832Ala/c.2494A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638302 </td>
<td>G </td>
<td>A </td>
<td>Chr19_47638302 (<em>e3-Mo</em>) </td>
<td>0.0433 </td>
<td>0.0306 </td>
<td>0.0485 </td>
<td>Missense_variant </td>
<td>p.Gly1050Arg/c.3148G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638344 </td>
<td>A </td>
<td>T </td>
<td>rs389636522 </td>
<td>0.00508 </td>
<td>0.00113 </td>
<td>0.00433 </td>
<td>Splice_region_ variant </td>
<td>c.3183A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>  </td>
<td>15 kb deletion </td>
<td>
<em>e3-tr</em> </td>
<td>1.9E-13 </td>
<td>1.61E-13 </td>
<td>5.15E-14 </td>
<td>Loss of exon 4 </td>
<td>  </td>
<td>HIGH </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Scaffold_32 </td>
<td>197169 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_197169 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*958G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197421 </td>
<td>T </td>
<td>C </td>
<td>Scaffold_32_197421 </td>
<td>0.000034 </td>
<td>0.00007 </td>
<td>8.54E-05 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*706A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197459 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_197459 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*668T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198053 </td>
<td>T </td>
<td>A </td>
<td>Scaffold_32_198053 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.*133 + 29A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198773 </td>
<td>A </td>
<td>AT </td>
<td>Scaffold_32_198773 </td>
<td>1.85E-06 </td>
<td>3.75E-06 </td>
<td>2.85E-06 </td>
<td>Frameshift_variant </td>
<td>p.Met737_Ala738fs/c.2209_2210insA </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199551 </td>
<td>A </td>
<td>C </td>
<td>Scaffold_32_199551 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala529Ala/c.1587T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199605 </td>
<td>T </td>
<td>G </td>
<td>Scaffold_32_199605 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.1575-42A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199656 </td>
<td>TA </td>
<td>T </td>
<td>Scaffold_32_199656 </td>
<td>0.0096 </td>
<td>0.0104 </td>
<td>0.011 </td>
<td>Intron_variant </td>
<td>c.1574 + 2delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199722 </td>
<td>C </td>
<td>G </td>
<td>Scaffold_32_199722 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>Missense_variant </td>
<td>p.Gly504Ala/c.1511G&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>202754 </td>
<td>G </td>
<td>A </td>
<td>Scaffold_32_202754 </td>
<td>0.0142 </td>
<td>0.01 </td>
<td>0.0105 </td>
<td>Stop_gained </td>
<td>p.Arg308*/c.922C&gt;T </td>
<td> </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>206717 </td>
<td>A </td>
<td>G </td>
<td>Scaffold_32_206717 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>Intron_variant </td>
<td>c.792 + 27T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>216494 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_216494 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Val168Val/c.504G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>218380 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_218380 </td>
<td>3.29E-09 </td>
<td>4.29E-08 </td>
<td>8.4E-09 </td>
<td>Synonymous_ variant </td>
<td>p.Pro73Pro/c.219T&gt;A </td>
<td>SILENT </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn11"><div class="fn">
<div class="label">a</div>
<p>Gene names refer to those in Kong et al.,<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Fan et al.,<a reveal-id="dsz005-B52" data-open="dsz005-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and Cao et al.<a reveal-id="dsz005-B53" data-open="dsz005-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p>
</div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025737" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T3" class="table-wrap-title" data-id="dsz005-T3">
<div class="label">Table 3</div>
<div class="caption"><p>DNA polymorphisms in nine genes significantly associated with flowering time variation in the mini-core collection as revealed by the single linear regression analysis</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Gene description </th>
<th rowspan="2">Gene symbol<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Chromosome </th>
<th rowspan="2">Position (bp) </th>
<th rowspan="2">Reference sequence </th>
<th rowspan="2">Alternative sequence </th>
<th rowspan="2">SNP name </th>
<th colspan="3">
<em>P</em> value of association tests<hr> </th>
<th rowspan="2">Functional effect </th>
<th rowspan="2">Amino acid change </th>
<th rowspan="2">Functional class </th>
</tr>
<tr>
<th>2011 </th>
<th>2012 </th>
<th>2013 </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain- containing protein </td>
<td>
<em>E1</em> </td>
<td>Chr06 </td>
<td>20207322 </td>
<td>C </td>
<td>G </td>
<td>rs123612969 (<em>e1-as</em>) </td>
<td>0.0226 </td>
<td>0.0153 </td>
<td>0.0333 </td>
<td>Missense_variant </td>
<td>p.Thr75Arg/c.224C&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>A </td>
<td>T </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>Stop-gained </td>
<td>p.Lys527*/c.1582A&gt;T </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Chr12 </td>
<td>5508242 </td>
<td>T </td>
<td>G </td>
<td>rs125308101 </td>
<td>0.00258 </td>
<td>0.0082 </td>
<td>0.0019 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508672 </td>
<td>CT </td>
<td>C </td>
<td>rs745192414 </td>
<td>0.00155 </td>
<td>0.00261 </td>
<td>0.00503 </td>
<td>Intron_variant </td>
<td>c.-120 + 219delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508702 </td>
<td>T </td>
<td>C </td>
<td>rs743387935 </td>
<td>0.0211 </td>
<td>0.0228 </td>
<td>0.0109 </td>
<td>Intron_variant </td>
<td>c.-119-198T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509310 </td>
<td>G </td>
<td>A </td>
<td>rs388619068 </td>
<td>0.0146 </td>
<td>0.0119 </td>
<td>0.00907 </td>
<td>Missense_variant </td>
<td>p.Asp98Asn/c.292G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509317 </td>
<td>C </td>
<td>T </td>
<td>rs125308103 </td>
<td>0.00107 </td>
<td>0.00148 </td>
<td>0.00319 </td>
<td>Missense_variant </td>
<td>p.Ser100Leu/c.299C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5519728 </td>
<td>A </td>
<td>C </td>
<td>rs125308115 </td>
<td>0.00229 </td>
<td>0.00749 </td>
<td>0.00169 </td>
<td>Missense_variant </td>
<td>p.Lys378Gln/c.1132A&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520578 </td>
<td>A </td>
<td>G </td>
<td>rs389122657 </td>
<td>0.014 </td>
<td>0.0112 </td>
<td>0.00865 </td>
<td>Synonymous_ variant </td>
<td>p.Ala504Ala/c.1512A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>T </td>
<td>C </td>
<td>rs125308117 </td>
<td>8.96E-10 </td>
<td>1.76E-08 </td>
<td>7.74E-11 </td>
<td>Stop_lost </td>
<td>p.Ter627Glnext*?/c.1879T&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5521029 </td>
<td>G </td>
<td>T </td>
<td>rs743179814 </td>
<td>0.0244 </td>
<td>0.0203 </td>
<td>0.0089 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*82G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em>-</em> </td>
<td>Chr15 </td>
<td>11436193 </td>
<td>G </td>
<td>A </td>
<td>rs388435741 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11441207 </td>
<td>C </td>
<td>T </td>
<td>rs126279495 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>Missense_variant </td>
<td>p.Val394Ile/c.1180G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11442400 </td>
<td>A </td>
<td>C </td>
<td>rs126279502 </td>
<td>0.0753 </td>
<td>0.0769 </td>
<td>0.0431 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-14T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGC </td>
<td>T </td>
<td>rs392064733_1 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-50_-59delATAAGAAAGC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGCA </td>
<td>TA </td>
<td>rs392064733_2 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-49_-58delTAAGAAAGCA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110991 </td>
<td>A </td>
<td>AATAT </td>
<td>rs864598505_1 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-56_202-55insATAT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111033 </td>
<td>T </td>
<td>A </td>
<td>rs126829817 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-15T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111042 </td>
<td>T </td>
<td>C </td>
<td>rs126829818 </td>
<td>0.0697 </td>
<td>0.21 </td>
<td>0.0419 </td>
<td>Splice_region_ variant </td>
<td>c.202T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111349 </td>
<td>A </td>
<td>G </td>
<td>rs126829819 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.304 + 72A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114633 </td>
<td>G </td>
<td>A </td>
<td>Chr16_31114633 </td>
<td>5.46E-06 </td>
<td>4.14E-06 </td>
<td>3.05E-07 </td>
<td>Missense_ variant </td>
<td>p.Gly169Asp/c.506G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114658 </td>
<td>AGA </td>
<td>AA </td>
<td>Chr16_31114658_2 </td>
<td>0.0765 </td>
<td>0.0538 </td>
<td>0.043 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*1delG </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114930 </td>
<td>G </td>
<td>T </td>
<td>rs126829846 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*272G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em>-</em> </td>
<td>Chr16 </td>
<td>36179909 </td>
<td>T </td>
<td>G </td>
<td>rs126888526 </td>
<td>1.13E-07 </td>
<td>3.18E-06 </td>
<td>3.35E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180002 </td>
<td>G </td>
<td>A </td>
<td>rs126888527 </td>
<td>1.09E-07 </td>
<td>1.63E-06 </td>
<td>1.98E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180087 </td>
<td>T </td>
<td>TGA </td>
<td>Chr16_36180087 </td>
<td>8.8E-08 </td>
<td>2.16E-06 </td>
<td>2.07E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*39_*40insTC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180122 </td>
<td>A </td>
<td>C </td>
<td>rs126888528 </td>
<td>4.19E-07 </td>
<td>9.01E-06 </td>
<td>1.03E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*5T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180183 </td>
<td>T </td>
<td>A </td>
<td>rs389022394 </td>
<td>4.96E-07 </td>
<td>1.03E-05 </td>
<td>1.32E-06 </td>
<td>Missense_variant </td>
<td>p.Thr219Ser/c.655A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182854 </td>
<td>T </td>
<td>TA </td>
<td>Chr16_36182854 </td>
<td>6.15E-06 </td>
<td>4.11E-05 </td>
<td>3.79E-06 </td>
<td>Intron_variant </td>
<td>c.511-49_511-50insT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182857 </td>
<td>T </td>
<td>A </td>
<td>rs126888574 </td>
<td>8.41E-07 </td>
<td>7.3E-06 </td>
<td>5.2E-07 </td>
<td>Intron_variant </td>
<td>c.511-52A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183423 </td>
<td>T </td>
<td>G </td>
<td>rs126888580_2 </td>
<td>6.9E-06 </td>
<td>5.27E-06 </td>
<td>2.61E-06 </td>
<td>Intron_variant </td>
<td>c.427-19A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183426 </td>
<td>G </td>
<td>C </td>
<td>rs126888581_1 </td>
<td>0.00286 </td>
<td>0.00159 </td>
<td>0.0018 </td>
<td>Intron_variant </td>
<td>c.427-22C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183435 </td>
<td>G </td>
<td>C </td>
<td>rs126888582 </td>
<td>6.8E-06 </td>
<td>5.07E-06 </td>
<td>2.66E-06 </td>
<td>Intron_variant </td>
<td>c.427-31C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183450 </td>
<td>G </td>
<td>A </td>
<td>rs126888583 </td>
<td>0.0104 </td>
<td>0.00721 </td>
<td>0.00804 </td>
<td>Intron_variant </td>
<td>c.427-46C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183510 </td>
<td>C </td>
<td>T </td>
<td>rs126888584 </td>
<td>0.000219 </td>
<td>0.00275 </td>
<td>0.000461 </td>
<td>Intron_variant </td>
<td>c.427-106G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183518 </td>
<td>A </td>
<td>G </td>
<td>rs126888585 </td>
<td>3.22E-10 </td>
<td>9.24E-09 </td>
<td>3.46E-10 </td>
<td>Intron_variant </td>
<td>c.427-114T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183541 </td>
<td>C </td>
<td>A </td>
<td>rs126888586 </td>
<td>2.98E-10 </td>
<td>8.83E-09 </td>
<td>3.27E-10 </td>
<td>Intron_variant </td>
<td>c.427-137G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183556 </td>
<td>T </td>
<td>C </td>
<td>Chr16_36183556 </td>
<td>0.0507 </td>
<td>0.0363 </td>
<td>0.048 </td>
<td>Intron_variant </td>
<td>c.427-152A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183568 </td>
<td>T </td>
<td>A </td>
<td>rs126888588 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-164A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183573 </td>
<td>T </td>
<td>A </td>
<td>rs126888589 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-169A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184165 </td>
<td>T </td>
<td>C </td>
<td>rs126888605 </td>
<td>4.71E-05 </td>
<td>3.54E-05 </td>
<td>0.000019 </td>
<td>Intron_variant </td>
<td>c.327-41A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184729 </td>
<td>T </td>
<td>C </td>
<td>rs126888609 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Missense_variant </td>
<td>p.Thr79Ala/c.235A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184733 </td>
<td>C </td>
<td>T </td>
<td>rs126888610 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser77Ser/c.231G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184819 </td>
<td>A </td>
<td>T </td>
<td>rs126888611 </td>
<td>0.0341 </td>
<td>0.0817 </td>
<td>0.0244 </td>
<td>Intron_variant </td>
<td>c.183-38T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187211 </td>
<td>A </td>
<td>C </td>
<td>rs744233319 </td>
<td>0.000287 </td>
<td>0.000744 </td>
<td>0.00118 </td>
<td>Synonymous_ variant </td>
<td>p.Ser36Ser/c.108T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187552 </td>
<td>T </td>
<td>G </td>
<td>rs126888636 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187600 </td>
<td>A </td>
<td>G </td>
<td>rs126888637 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Chr17 </td>
<td>3955280 </td>
<td>A </td>
<td>AT </td>
<td>rs126942305_1 </td>
<td>0.00214 </td>
<td>0.00191 </td>
<td>0.00197 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955411 </td>
<td>A </td>
<td>G </td>
<td>rs126942313 </td>
<td>2.56E-10 </td>
<td>3.12E-10 </td>
<td>1.07E-10 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955475 </td>
<td>TC </td>
<td>AA </td>
<td>rs126942314_1 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955476 </td>
<td>C </td>
<td>A </td>
<td>rs126942315 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955546 </td>
<td>T </td>
<td>A </td>
<td>rs126942316 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955554 </td>
<td>T </td>
<td>C </td>
<td>rs126942317 </td>
<td>0.031 </td>
<td>0.0299 </td>
<td>0.0358 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*117A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955716 </td>
<td>A </td>
<td>G </td>
<td>rs388258139 </td>
<td>0.00418 </td>
<td>0.00761 </td>
<td>0.00338 </td>
<td>Synonymous_ variant </td>
<td>p.Ala307Ala/c.921T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955763 </td>
<td>C </td>
<td>CG </td>
<td>Chr17_3955763 </td>
<td>0.00444 </td>
<td>0.00194 </td>
<td>0.00318 </td>
<td>Frameshift_ variant </td>
<td>p.Val291_Ala292fs/c.873_874insC </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955764 </td>
<td>A </td>
<td>G </td>
<td>rs126942318 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val291Val/c.873T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955884 </td>
<td>G </td>
<td>C </td>
<td>rs126942319 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val251Val/c.753C&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956142 </td>
<td>T </td>
<td>C </td>
<td>rs126942321 </td>
<td>6.36E-08 </td>
<td>3.72E-08 </td>
<td>1.25E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala165Ala/c.495A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956163 </td>
<td>T </td>
<td>C </td>
<td>rs126942322 </td>
<td>0.00644 </td>
<td>0.00295 </td>
<td>0.00466 </td>
<td>Synonymous_ variant </td>
<td>p.Lys158Lys/c.474A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958319 </td>
<td>C </td>
<td>G </td>
<td>rs126942340 </td>
<td>9.03E-10 </td>
<td>4.86E-10 </td>
<td>1.98E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser16Ser/c.48G&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958374 </td>
<td>T </td>
<td>G </td>
<td>rs126942341 </td>
<td>2.32E-10 </td>
<td>9.96E-11 </td>
<td>5.12E-11 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-8A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>T </td>
<td>A </td>
<td>rs127928573 (<em>dt1</em>) </td>
<td>0.00122 </td>
<td>0.00587 </td>
<td>0.000401 </td>
<td>Missense_variant </td>
<td>p.Arg166Trp/c.496A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183808 </td>
<td>C </td>
<td>T </td>
<td>rs745009806 </td>
<td>0.0319 </td>
<td>0.0184 </td>
<td>0.0153 </td>
<td>Missense_variant </td>
<td>p.Arg130Lys/c.389G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183859 </td>
<td>G </td>
<td>A </td>
<td>rs127928574 </td>
<td>4.29E-11 </td>
<td>3.71E-11 </td>
<td>6.02E-10 </td>
<td>Missense_variant </td>
<td>p.Pro113Leu/c.338C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45184581 </td>
<td>G </td>
<td>T </td>
<td>Chr19_45184581 </td>
<td>0.014 </td>
<td>0.153 </td>
<td>0.0428 </td>
<td>Splice_region_ variant </td>
<td>c.202C&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45185131 </td>
<td>C </td>
<td>T </td>
<td>Chr19_45185131 </td>
<td>0.00805 </td>
<td>0.00351 </td>
<td>0.00764 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-142G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Chr19 </td>
<td>47633086 </td>
<td>T </td>
<td>TA </td>
<td>Chr19_47633086 </td>
<td>0.000763 </td>
<td>0.000353 </td>
<td>0.00085 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-693_-694insA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47634596 </td>
<td>A </td>
<td>G </td>
<td>rs127944661 </td>
<td>0.0052 </td>
<td>0.00108 </td>
<td>0.00429 </td>
<td>Synonymous_ variant </td>
<td>p.Ser43Ser/c.129A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635025 </td>
<td>C </td>
<td>T </td>
<td>rs388644281 </td>
<td>3.18E-05 </td>
<td>3.43E-05 </td>
<td>3.69E-05 </td>
<td>Synonymous_ variant </td>
<td>p.Ile186Ile/c.558C&gt;T </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635737 </td>
<td>C </td>
<td>A </td>
<td>rs389001110 </td>
<td>0.0534 </td>
<td>0.052 </td>
<td>0.048 </td>
<td>Missense_variant </td>
<td>p.Leu424Ile/c.1270C&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636564 </td>
<td>G </td>
<td>A </td>
<td>rs127944664 </td>
<td>0.0098 </td>
<td>0.00232 </td>
<td>0.00888 </td>
<td>Intron_variant </td>
<td>c.2074 + 23G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636607 </td>
<td>G </td>
<td>T </td>
<td>rs127944665 </td>
<td>0.0127 </td>
<td>0.0031 </td>
<td>0.0113 </td>
<td>Intron_variant </td>
<td>c.2074 + 66G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47637258 </td>
<td>A </td>
<td>G </td>
<td>rs393405985_1 </td>
<td>0.044 </td>
<td>0.0447 </td>
<td>0.0427 </td>
<td>Missense_variant </td>
<td>p.Thr832Ala/c.2494A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638302 </td>
<td>G </td>
<td>A </td>
<td>Chr19_47638302 (<em>e3-Mo</em>) </td>
<td>0.0433 </td>
<td>0.0306 </td>
<td>0.0485 </td>
<td>Missense_variant </td>
<td>p.Gly1050Arg/c.3148G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638344 </td>
<td>A </td>
<td>T </td>
<td>rs389636522 </td>
<td>0.00508 </td>
<td>0.00113 </td>
<td>0.00433 </td>
<td>Splice_region_ variant </td>
<td>c.3183A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>  </td>
<td>15 kb deletion </td>
<td>
<em>e3-tr</em> </td>
<td>1.9E-13 </td>
<td>1.61E-13 </td>
<td>5.15E-14 </td>
<td>Loss of exon 4 </td>
<td>  </td>
<td>HIGH </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Scaffold_32 </td>
<td>197169 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_197169 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*958G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197421 </td>
<td>T </td>
<td>C </td>
<td>Scaffold_32_197421 </td>
<td>0.000034 </td>
<td>0.00007 </td>
<td>8.54E-05 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*706A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197459 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_197459 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*668T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198053 </td>
<td>T </td>
<td>A </td>
<td>Scaffold_32_198053 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.*133 + 29A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198773 </td>
<td>A </td>
<td>AT </td>
<td>Scaffold_32_198773 </td>
<td>1.85E-06 </td>
<td>3.75E-06 </td>
<td>2.85E-06 </td>
<td>Frameshift_variant </td>
<td>p.Met737_Ala738fs/c.2209_2210insA </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199551 </td>
<td>A </td>
<td>C </td>
<td>Scaffold_32_199551 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala529Ala/c.1587T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199605 </td>
<td>T </td>
<td>G </td>
<td>Scaffold_32_199605 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.1575-42A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199656 </td>
<td>TA </td>
<td>T </td>
<td>Scaffold_32_199656 </td>
<td>0.0096 </td>
<td>0.0104 </td>
<td>0.011 </td>
<td>Intron_variant </td>
<td>c.1574 + 2delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199722 </td>
<td>C </td>
<td>G </td>
<td>Scaffold_32_199722 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>Missense_variant </td>
<td>p.Gly504Ala/c.1511G&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>202754 </td>
<td>G </td>
<td>A </td>
<td>Scaffold_32_202754 </td>
<td>0.0142 </td>
<td>0.01 </td>
<td>0.0105 </td>
<td>Stop_gained </td>
<td>p.Arg308*/c.922C&gt;T </td>
<td> </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>206717 </td>
<td>A </td>
<td>G </td>
<td>Scaffold_32_206717 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>Intron_variant </td>
<td>c.792 + 27T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>216494 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_216494 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Val168Val/c.504G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>218380 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_218380 </td>
<td>3.29E-09 </td>
<td>4.29E-08 </td>
<td>8.4E-09 </td>
<td>Synonymous_ variant </td>
<td>p.Pro73Pro/c.219T&gt;A </td>
<td>SILENT </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T3" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Gene description </th>
<th rowspan="2">Gene symbol<a reveal-id="tblfn11" data-open="tblfn11" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th>
<th rowspan="2">Chromosome </th>
<th rowspan="2">Position (bp) </th>
<th rowspan="2">Reference sequence </th>
<th rowspan="2">Alternative sequence </th>
<th rowspan="2">SNP name </th>
<th colspan="3">
<em>P</em> value of association tests<hr> </th>
<th rowspan="2">Functional effect </th>
<th rowspan="2">Amino acid change </th>
<th rowspan="2">Functional class </th>
</tr>
<tr>
<th>2011 </th>
<th>2012 </th>
<th>2013 </th>
</tr>
</thead>
<tbody>
<tr>
<td>Glyma.06G207800 </td>
<td>B3 domain- containing protein </td>
<td>
<em>E1</em> </td>
<td>Chr06 </td>
<td>20207322 </td>
<td>C </td>
<td>G </td>
<td>rs123612969 (<em>e1-as</em>) </td>
<td>0.0226 </td>
<td>0.0153 </td>
<td>0.0333 </td>
<td>Missense_variant </td>
<td>p.Thr75Arg/c.224C&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>Glyma.10G221500 </td>
<td>GIGANTEA </td>
<td>
<em>E2</em> </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>A </td>
<td>T </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>2E-16 </td>
<td>Stop-gained </td>
<td>p.Lys527*/c.1582A&gt;T </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Chr12 </td>
<td>5508242 </td>
<td>T </td>
<td>G </td>
<td>rs125308101 </td>
<td>0.00258 </td>
<td>0.0082 </td>
<td>0.0019 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508672 </td>
<td>CT </td>
<td>C </td>
<td>rs745192414 </td>
<td>0.00155 </td>
<td>0.00261 </td>
<td>0.00503 </td>
<td>Intron_variant </td>
<td>c.-120 + 219delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5508702 </td>
<td>T </td>
<td>C </td>
<td>rs743387935 </td>
<td>0.0211 </td>
<td>0.0228 </td>
<td>0.0109 </td>
<td>Intron_variant </td>
<td>c.-119-198T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509310 </td>
<td>G </td>
<td>A </td>
<td>rs388619068 </td>
<td>0.0146 </td>
<td>0.0119 </td>
<td>0.00907 </td>
<td>Missense_variant </td>
<td>p.Asp98Asn/c.292G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5509317 </td>
<td>C </td>
<td>T </td>
<td>rs125308103 </td>
<td>0.00107 </td>
<td>0.00148 </td>
<td>0.00319 </td>
<td>Missense_variant </td>
<td>p.Ser100Leu/c.299C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5519728 </td>
<td>A </td>
<td>C </td>
<td>rs125308115 </td>
<td>0.00229 </td>
<td>0.00749 </td>
<td>0.00169 </td>
<td>Missense_variant </td>
<td>p.Lys378Gln/c.1132A&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520578 </td>
<td>A </td>
<td>G </td>
<td>rs389122657 </td>
<td>0.014 </td>
<td>0.0112 </td>
<td>0.00865 </td>
<td>Synonymous_ variant </td>
<td>p.Ala504Ala/c.1512A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>T </td>
<td>C </td>
<td>rs125308117 </td>
<td>8.96E-10 </td>
<td>1.76E-08 </td>
<td>7.74E-11 </td>
<td>Stop_lost </td>
<td>p.Ter627Glnext*?/c.1879T&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr12 </td>
<td>5521029 </td>
<td>G </td>
<td>T </td>
<td>rs743179814 </td>
<td>0.0244 </td>
<td>0.0203 </td>
<td>0.0089 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*82G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.15G140000 </td>
<td>Phytochrome B </td>
<td>
<em>-</em> </td>
<td>Chr15 </td>
<td>11436193 </td>
<td>G </td>
<td>A </td>
<td>rs388435741 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11441207 </td>
<td>C </td>
<td>T </td>
<td>rs126279495 </td>
<td>5.17E-06 </td>
<td>7.93E-06 </td>
<td>8.47E-06 </td>
<td>Missense_variant </td>
<td>p.Val394Ile/c.1180G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr15 </td>
<td>11442400 </td>
<td>A </td>
<td>C </td>
<td>rs126279502 </td>
<td>0.0753 </td>
<td>0.0769 </td>
<td>0.0431 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-14T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G150700 </td>
<td>FLOWERING LOCUS T </td>
<td>
<em>GmFT2a/GmFTL3</em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGC </td>
<td>T </td>
<td>rs392064733_1 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-50_-59delATAAGAAAGC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110004 </td>
<td>TATAAGAAAGCA </td>
<td>TA </td>
<td>rs392064733_2 </td>
<td>0.0397 </td>
<td>0.117 </td>
<td>0.0266 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-49_-58delTAAGAAAGCA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31110991 </td>
<td>A </td>
<td>AATAT </td>
<td>rs864598505_1 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-56_202-55insATAT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111033 </td>
<td>T </td>
<td>A </td>
<td>rs126829817 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.202-15T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111042 </td>
<td>T </td>
<td>C </td>
<td>rs126829818 </td>
<td>0.0697 </td>
<td>0.21 </td>
<td>0.0419 </td>
<td>Splice_region_ variant </td>
<td>c.202T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31111349 </td>
<td>A </td>
<td>G </td>
<td>rs126829819 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>Intron_variant </td>
<td>c.304 + 72A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114633 </td>
<td>G </td>
<td>A </td>
<td>Chr16_31114633 </td>
<td>5.46E-06 </td>
<td>4.14E-06 </td>
<td>3.05E-07 </td>
<td>Missense_ variant </td>
<td>p.Gly169Asp/c.506G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114658 </td>
<td>AGA </td>
<td>AA </td>
<td>Chr16_31114658_2 </td>
<td>0.0765 </td>
<td>0.0538 </td>
<td>0.043 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*1delG </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>31114930 </td>
<td>G </td>
<td>T </td>
<td>rs126829846 </td>
<td>6.24E-05 </td>
<td>0.000274 </td>
<td>0.000111 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*272G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.16G200700 </td>
<td>MADS box protein </td>
<td>
<em>-</em> </td>
<td>Chr16 </td>
<td>36179909 </td>
<td>T </td>
<td>G </td>
<td>rs126888526 </td>
<td>1.13E-07 </td>
<td>3.18E-06 </td>
<td>3.35E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*218A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180002 </td>
<td>G </td>
<td>A </td>
<td>rs126888527 </td>
<td>1.09E-07 </td>
<td>1.63E-06 </td>
<td>1.98E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180087 </td>
<td>T </td>
<td>TGA </td>
<td>Chr16_36180087 </td>
<td>8.8E-08 </td>
<td>2.16E-06 </td>
<td>2.07E-07 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*39_*40insTC </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180122 </td>
<td>A </td>
<td>C </td>
<td>rs126888528 </td>
<td>4.19E-07 </td>
<td>9.01E-06 </td>
<td>1.03E-06 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*5T&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36180183 </td>
<td>T </td>
<td>A </td>
<td>rs389022394 </td>
<td>4.96E-07 </td>
<td>1.03E-05 </td>
<td>1.32E-06 </td>
<td>Missense_variant </td>
<td>p.Thr219Ser/c.655A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182854 </td>
<td>T </td>
<td>TA </td>
<td>Chr16_36182854 </td>
<td>6.15E-06 </td>
<td>4.11E-05 </td>
<td>3.79E-06 </td>
<td>Intron_variant </td>
<td>c.511-49_511-50insT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36182857 </td>
<td>T </td>
<td>A </td>
<td>rs126888574 </td>
<td>8.41E-07 </td>
<td>7.3E-06 </td>
<td>5.2E-07 </td>
<td>Intron_variant </td>
<td>c.511-52A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183423 </td>
<td>T </td>
<td>G </td>
<td>rs126888580_2 </td>
<td>6.9E-06 </td>
<td>5.27E-06 </td>
<td>2.61E-06 </td>
<td>Intron_variant </td>
<td>c.427-19A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183426 </td>
<td>G </td>
<td>C </td>
<td>rs126888581_1 </td>
<td>0.00286 </td>
<td>0.00159 </td>
<td>0.0018 </td>
<td>Intron_variant </td>
<td>c.427-22C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183435 </td>
<td>G </td>
<td>C </td>
<td>rs126888582 </td>
<td>6.8E-06 </td>
<td>5.07E-06 </td>
<td>2.66E-06 </td>
<td>Intron_variant </td>
<td>c.427-31C&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183450 </td>
<td>G </td>
<td>A </td>
<td>rs126888583 </td>
<td>0.0104 </td>
<td>0.00721 </td>
<td>0.00804 </td>
<td>Intron_variant </td>
<td>c.427-46C&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183510 </td>
<td>C </td>
<td>T </td>
<td>rs126888584 </td>
<td>0.000219 </td>
<td>0.00275 </td>
<td>0.000461 </td>
<td>Intron_variant </td>
<td>c.427-106G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183518 </td>
<td>A </td>
<td>G </td>
<td>rs126888585 </td>
<td>3.22E-10 </td>
<td>9.24E-09 </td>
<td>3.46E-10 </td>
<td>Intron_variant </td>
<td>c.427-114T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183541 </td>
<td>C </td>
<td>A </td>
<td>rs126888586 </td>
<td>2.98E-10 </td>
<td>8.83E-09 </td>
<td>3.27E-10 </td>
<td>Intron_variant </td>
<td>c.427-137G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183556 </td>
<td>T </td>
<td>C </td>
<td>Chr16_36183556 </td>
<td>0.0507 </td>
<td>0.0363 </td>
<td>0.048 </td>
<td>Intron_variant </td>
<td>c.427-152A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183568 </td>
<td>T </td>
<td>A </td>
<td>rs126888588 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-164A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36183573 </td>
<td>T </td>
<td>A </td>
<td>rs126888589 </td>
<td>0.00193 </td>
<td>0.0172 </td>
<td>0.00328 </td>
<td>Intron_variant </td>
<td>c.427-169A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184165 </td>
<td>T </td>
<td>C </td>
<td>rs126888605 </td>
<td>4.71E-05 </td>
<td>3.54E-05 </td>
<td>0.000019 </td>
<td>Intron_variant </td>
<td>c.327-41A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184729 </td>
<td>T </td>
<td>C </td>
<td>rs126888609 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Missense_variant </td>
<td>p.Thr79Ala/c.235A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184733 </td>
<td>C </td>
<td>T </td>
<td>rs126888610 </td>
<td>2.18E-09 </td>
<td>2.34E-08 </td>
<td>6.84E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser77Ser/c.231G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36184819 </td>
<td>A </td>
<td>T </td>
<td>rs126888611 </td>
<td>0.0341 </td>
<td>0.0817 </td>
<td>0.0244 </td>
<td>Intron_variant </td>
<td>c.183-38T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187211 </td>
<td>A </td>
<td>C </td>
<td>rs744233319 </td>
<td>0.000287 </td>
<td>0.000744 </td>
<td>0.00118 </td>
<td>Synonymous_ variant </td>
<td>p.Ser36Ser/c.108T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187552 </td>
<td>T </td>
<td>G </td>
<td>rs126888636 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr16 </td>
<td>36187600 </td>
<td>A </td>
<td>G </td>
<td>rs126888637 </td>
<td>0.0383 </td>
<td>0.0315 </td>
<td>0.0302 </td>
<td>Upstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.17G052100 </td>
<td>WD repeat-containing protein 61 </td>
<td>
<em>WDR61</em> </td>
<td>Chr17 </td>
<td>3955280 </td>
<td>A </td>
<td>AT </td>
<td>rs126942305_1 </td>
<td>0.00214 </td>
<td>0.00191 </td>
<td>0.00197 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955411 </td>
<td>A </td>
<td>G </td>
<td>rs126942313 </td>
<td>2.56E-10 </td>
<td>3.12E-10 </td>
<td>1.07E-10 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955475 </td>
<td>TC </td>
<td>AA </td>
<td>rs126942314_1 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955476 </td>
<td>C </td>
<td>A </td>
<td>rs126942315 </td>
<td>0.00359 </td>
<td>0.00645 </td>
<td>0.00279 </td>
<td>Downstream_gene_ variant </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955546 </td>
<td>T </td>
<td>A </td>
<td>rs126942316 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*125A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955554 </td>
<td>T </td>
<td>C </td>
<td>rs126942317 </td>
<td>0.031 </td>
<td>0.0299 </td>
<td>0.0358 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*117A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955716 </td>
<td>A </td>
<td>G </td>
<td>rs388258139 </td>
<td>0.00418 </td>
<td>0.00761 </td>
<td>0.00338 </td>
<td>Synonymous_ variant </td>
<td>p.Ala307Ala/c.921T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955763 </td>
<td>C </td>
<td>CG </td>
<td>Chr17_3955763 </td>
<td>0.00444 </td>
<td>0.00194 </td>
<td>0.00318 </td>
<td>Frameshift_ variant </td>
<td>p.Val291_Ala292fs/c.873_874insC </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955764 </td>
<td>A </td>
<td>G </td>
<td>rs126942318 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val291Val/c.873T&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3955884 </td>
<td>G </td>
<td>C </td>
<td>rs126942319 </td>
<td>0.00374 </td>
<td>0.00157 </td>
<td>0.00264 </td>
<td>Synonymous_ variant </td>
<td>p.Val251Val/c.753C&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956142 </td>
<td>T </td>
<td>C </td>
<td>rs126942321 </td>
<td>6.36E-08 </td>
<td>3.72E-08 </td>
<td>1.25E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala165Ala/c.495A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3956163 </td>
<td>T </td>
<td>C </td>
<td>rs126942322 </td>
<td>0.00644 </td>
<td>0.00295 </td>
<td>0.00466 </td>
<td>Synonymous_ variant </td>
<td>p.Lys158Lys/c.474A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958319 </td>
<td>C </td>
<td>G </td>
<td>rs126942340 </td>
<td>9.03E-10 </td>
<td>4.86E-10 </td>
<td>1.98E-10 </td>
<td>Synonymous_ variant </td>
<td>p.Ser16Ser/c.48G&gt;C </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr17 </td>
<td>3958374 </td>
<td>T </td>
<td>G </td>
<td>rs126942341 </td>
<td>2.32E-10 </td>
<td>9.96E-11 </td>
<td>5.12E-11 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-8A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>
<em>Dt1/GmTFL1</em> </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>T </td>
<td>A </td>
<td>rs127928573 (<em>dt1</em>) </td>
<td>0.00122 </td>
<td>0.00587 </td>
<td>0.000401 </td>
<td>Missense_variant </td>
<td>p.Arg166Trp/c.496A&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183808 </td>
<td>C </td>
<td>T </td>
<td>rs745009806 </td>
<td>0.0319 </td>
<td>0.0184 </td>
<td>0.0153 </td>
<td>Missense_variant </td>
<td>p.Arg130Lys/c.389G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45183859 </td>
<td>G </td>
<td>A </td>
<td>rs127928574 </td>
<td>4.29E-11 </td>
<td>3.71E-11 </td>
<td>6.02E-10 </td>
<td>Missense_variant </td>
<td>p.Pro113Leu/c.338C&gt;T </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45184581 </td>
<td>G </td>
<td>T </td>
<td>Chr19_45184581 </td>
<td>0.014 </td>
<td>0.153 </td>
<td>0.0428 </td>
<td>Splice_region_ variant </td>
<td>c.202C&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>45185131 </td>
<td>C </td>
<td>T </td>
<td>Chr19_45185131 </td>
<td>0.00805 </td>
<td>0.00351 </td>
<td>0.00764 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-142G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>Glyma.19G224200 </td>
<td>Phytochrome A </td>
<td>
<em>E3</em> </td>
<td>Chr19 </td>
<td>47633086 </td>
<td>T </td>
<td>TA </td>
<td>Chr19_47633086 </td>
<td>0.000763 </td>
<td>0.000353 </td>
<td>0.00085 </td>
<td>5_Prime_UTR_ variant </td>
<td>c.-693_-694insA </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47634596 </td>
<td>A </td>
<td>G </td>
<td>rs127944661 </td>
<td>0.0052 </td>
<td>0.00108 </td>
<td>0.00429 </td>
<td>Synonymous_ variant </td>
<td>p.Ser43Ser/c.129A&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635025 </td>
<td>C </td>
<td>T </td>
<td>rs388644281 </td>
<td>3.18E-05 </td>
<td>3.43E-05 </td>
<td>3.69E-05 </td>
<td>Synonymous_ variant </td>
<td>p.Ile186Ile/c.558C&gt;T </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47635737 </td>
<td>C </td>
<td>A </td>
<td>rs389001110 </td>
<td>0.0534 </td>
<td>0.052 </td>
<td>0.048 </td>
<td>Missense_variant </td>
<td>p.Leu424Ile/c.1270C&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636564 </td>
<td>G </td>
<td>A </td>
<td>rs127944664 </td>
<td>0.0098 </td>
<td>0.00232 </td>
<td>0.00888 </td>
<td>Intron_variant </td>
<td>c.2074 + 23G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47636607 </td>
<td>G </td>
<td>T </td>
<td>rs127944665 </td>
<td>0.0127 </td>
<td>0.0031 </td>
<td>0.0113 </td>
<td>Intron_variant </td>
<td>c.2074 + 66G&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47637258 </td>
<td>A </td>
<td>G </td>
<td>rs393405985_1 </td>
<td>0.044 </td>
<td>0.0447 </td>
<td>0.0427 </td>
<td>Missense_variant </td>
<td>p.Thr832Ala/c.2494A&gt;G </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638302 </td>
<td>G </td>
<td>A </td>
<td>Chr19_47638302 (<em>e3-Mo</em>) </td>
<td>0.0433 </td>
<td>0.0306 </td>
<td>0.0485 </td>
<td>Missense_variant </td>
<td>p.Gly1050Arg/c.3148G&gt;A </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47638344 </td>
<td>A </td>
<td>T </td>
<td>rs389636522 </td>
<td>0.00508 </td>
<td>0.00113 </td>
<td>0.00433 </td>
<td>Splice_region_ variant </td>
<td>c.3183A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>  </td>
<td>15 kb deletion </td>
<td>
<em>e3-tr</em> </td>
<td>1.9E-13 </td>
<td>1.61E-13 </td>
<td>5.15E-14 </td>
<td>Loss of exon 4 </td>
<td>  </td>
<td>HIGH </td>
</tr>
<tr>
<td>Glyma.U034500 </td>
<td>Two-component response regulator-like </td>
<td>
<em>-</em> </td>
<td>Scaffold_32 </td>
<td>197169 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_197169 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*958G&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197421 </td>
<td>T </td>
<td>C </td>
<td>Scaffold_32_197421 </td>
<td>0.000034 </td>
<td>0.00007 </td>
<td>8.54E-05 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*706A&gt;G </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>197459 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_197459 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>3_Prime_UTR_ variant </td>
<td>c.*668T&gt;A </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198053 </td>
<td>T </td>
<td>A </td>
<td>Scaffold_32_198053 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.*133 + 29A&gt;T </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>198773 </td>
<td>A </td>
<td>AT </td>
<td>Scaffold_32_198773 </td>
<td>1.85E-06 </td>
<td>3.75E-06 </td>
<td>2.85E-06 </td>
<td>Frameshift_variant </td>
<td>p.Met737_Ala738fs/c.2209_2210insA </td>
<td>HIGH </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199551 </td>
<td>A </td>
<td>C </td>
<td>Scaffold_32_199551 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Ala529Ala/c.1587T&gt;G </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199605 </td>
<td>T </td>
<td>G </td>
<td>Scaffold_32_199605 </td>
<td>2.16E-05 </td>
<td>4.99E-05 </td>
<td>0.000048 </td>
<td>Intron_variant </td>
<td>c.1575-42A&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199656 </td>
<td>TA </td>
<td>T </td>
<td>Scaffold_32_199656 </td>
<td>0.0096 </td>
<td>0.0104 </td>
<td>0.011 </td>
<td>Intron_variant </td>
<td>c.1574 + 2delT </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>199722 </td>
<td>C </td>
<td>G </td>
<td>Scaffold_32_199722 </td>
<td>0.000318 </td>
<td>0.000771 </td>
<td>0.000612 </td>
<td>Missense_variant </td>
<td>p.Gly504Ala/c.1511G&gt;C </td>
<td>MISSENSE </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>202754 </td>
<td>G </td>
<td>A </td>
<td>Scaffold_32_202754 </td>
<td>0.0142 </td>
<td>0.01 </td>
<td>0.0105 </td>
<td>Stop_gained </td>
<td>p.Arg308*/c.922C&gt;T </td>
<td> </td>
</tr>
<tr>
<td>HIGH  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>206717 </td>
<td>A </td>
<td>G </td>
<td>Scaffold_32_206717 </td>
<td>1.23E-09 </td>
<td>1.94E-08 </td>
<td>2.71E-09 </td>
<td>Intron_variant </td>
<td>c.792 + 27T&gt;C </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>216494 </td>
<td>C </td>
<td>T </td>
<td>Scaffold_32_216494 </td>
<td>4.88E-09 </td>
<td>6.54E-08 </td>
<td>1.23E-08 </td>
<td>Synonymous_ variant </td>
<td>p.Val168Val/c.504G&gt;A </td>
<td>SILENT </td>
</tr>
<tr>
<td>  </td>
<td>  </td>
<td>
<em> </em> </td>
<td>Scaffold_32 </td>
<td>218380 </td>
<td>A </td>
<td>T </td>
<td>Scaffold_32_218380 </td>
<td>3.29E-09 </td>
<td>4.29E-08 </td>
<td>8.4E-09 </td>
<td>Synonymous_ variant </td>
<td>p.Pro73Pro/c.219T&gt;A </td>
<td>SILENT </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn11"><div class="fn">
<div class="label">a</div>
<p>Gene names refer to those in Kong et al.,<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Fan et al.,<a reveal-id="dsz005-B52" data-open="dsz005-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and Cao et al.<a reveal-id="dsz005-B53" data-open="dsz005-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p>
</div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025737" target="_blank">Open in new tab</a></div>
</div></div>                    <h4 scrollto-destination="137025738" id="137025738" class="section-title">
<em>E1</em> and <em>E1</em>-like genes</h4>
<p>Five alleles, <em>E1</em>, <em>e1-as</em>, <em>e1-nl</em>, <em>e1-fs</em>, and one novel missense (Chr06_20207355) were identified at <em>E1</em> (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S8 and S9</a></span>). In the other two B3 domain containing <em>E1</em>-like genes, only one synonymous variant (rs123097808) was detected in <em>Glyma.04G156400/E1La</em>, whereas no variant was detected in the coding region of <em>Glyma.04G143300/E1Lb</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span> and C). Among the five <em>E1</em> alleles, the frequency of <em>e1-as</em> allele (Chr06:20207322 C: Williams 82 type) was 0.09 and Chr06:20207322 C to G nucleotide change (rs123612969) was 0.91 among the soybean mini-core collection. In contrast, <em>e1-nl</em>, which lacks the entire <em>E1</em> gene,<a reveal-id="dsz005-B10" data-open="dsz005-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> was only found in Swedish cultivar FiskebyV (PGC001) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). This allele was determined by the read coverage at the <em>E1</em> genomic region. The average normalized read coverage of all six amplicons (AMPL1037682–AMPL1037687) was three in PGC001, whereas that of these amplicons in the other accessions was 181 (ranging from 36 to 430). Additional experiments to confirm the deletion in the <em>E1</em> genomic region by PCR amplification revealed that only PGC001 lacks the <em>E1</em> genomic region among all accessions and possesses the <em>e1-nl</em> allele. Another allele, <em>e1-fs</em>, which had 1-bp deletion variant (Chr06_20207323) was also found in one accession PGC002 (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). The deletion (Chr06_20207323) causes a frameshift and introduces a premature stop codon at Lys76. These loss-of-function alleles were not included in the association analysis due to very low allele frequency (only one accession each), but might explain very early flowering of PGC001 and PGC002 under long-day-length field condition. PGC002 (Wase kuro daizu) is originated from the southern part of Japan and classified as the summer-type soybean, early-maturity group in low-latitude regions of Japan. The summer-type soybean has low photoperiod sensitivity, and <em>e1-fs</em> can explain this characteristic. In contrast, the novel missense (Chr06_20207355) from PGC139 and PGC147 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>) did not show large effect on flowering time and might not significantly affect the E1 function. </p>                        <a id="137025740" scrollto-destination="137025740"></a>
<div id="dsz005-F4" data-id="dsz005-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f4.png?Expires=1576535926&amp;Signature=APbb9r7R3~U4U5Hi9ZIHDi3Bnpti8bz3PIM42in17taVSO~j2EhUtXXcPEwuZtZimJqr8i845HxkJa7v43OPo5Nf93aNSRKJjiVW8JFn6Hz7MYxf2uZoX4yZ15syeNZlR6JVgNGRPSNvX1IHIev2dY9eey-vtMcd1wSzrjITquw8gcIbfNA99NthNSXQXmT2mnWDSwb0uuLS02IGSuFKQXsb1gPdO1VnXTJ3SqACUGiA0T~-p59n8exskp-AwRUaNz0Kr-RTMYjZi8ZAKVB3ArEOX3VOkZFO-c3T2DUqpL1pCrxHeWXp0Se6HTET0kvnlCXoCTvfpXd9PrxyrFCqOQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detected variants in E1, E3, E4, FT5a, and FT2a from 192 mini-core collection. The grey and white boxes indicate UTRs and exons. The solid lines indicate 5′-uptream and intron regions. The black and grey circles indicate loss-of-function (described as ‘HIGH’ impact on the gene function in Supplementary Table S8) and missense variants. Braces indicate known large InDels. These InDels can be detected by read depth." path-from-xml="dsz005f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025740/dsz005f4.tif" path-from-xml="dsz005f4.tif" target="_blank">Open in new tab</a><a section="137025740" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f4.png?Expires=1576535926&amp;Signature=Gy-UT918QrKsS6k8sfoUx-VDNqOVapdgt7WHKZCBaAVa7E9pwQe8tcfovk2gqfzDhhEPUDm6cjj7m~WOfK7cTK31tRyFzovx9G4Vo9sdos91DCCy-oyELIdyyDI20TTwucsgwiBRpNsuz~V~4ZFA5w7mJULuPi4Qje5r6QCn~47kr3g8mG5i3GcnH5ObHixsT29CU7Li6DNVj0JomJMTjh4DvcNqxf56ZT7AWaWQd1swbMpWdBNWKIIIoYUYb0yhr-LSGUcFkd1MGHgSEdElk6NttIHxyJM5hLMnoyVG6i0hugX9YeDqsiPJZ5Ts~g0mxrPWPNkAWvCujIKjQIbaGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025740&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detected variants in <em>E1</em>, <em>E3</em>, <em>E4</em>, <em>FT5a</em>, and <em>FT2a</em> from 192 mini-core collection. The grey and white boxes indicate UTRs and exons. The solid lines indicate 5′-uptream and intron regions. The black and grey circles indicate loss-of-function (described as ‘HIGH’ impact on the gene function in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>) and missense variants. Braces indicate known large InDels. These InDels can be detected by read depth.</p></div>
</div>
<div content-id="dsz005-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/m_dsz005f4.png?Expires=1576535926&amp;Signature=APbb9r7R3~U4U5Hi9ZIHDi3Bnpti8bz3PIM42in17taVSO~j2EhUtXXcPEwuZtZimJqr8i845HxkJa7v43OPo5Nf93aNSRKJjiVW8JFn6Hz7MYxf2uZoX4yZ15syeNZlR6JVgNGRPSNvX1IHIev2dY9eey-vtMcd1wSzrjITquw8gcIbfNA99NthNSXQXmT2mnWDSwb0uuLS02IGSuFKQXsb1gPdO1VnXTJ3SqACUGiA0T~-p59n8exskp-AwRUaNz0Kr-RTMYjZi8ZAKVB3ArEOX3VOkZFO-c3T2DUqpL1pCrxHeWXp0Se6HTET0kvnlCXoCTvfpXd9PrxyrFCqOQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detected variants in E1, E3, E4, FT5a, and FT2a from 192 mini-core collection. The grey and white boxes indicate UTRs and exons. The solid lines indicate 5′-uptream and intron regions. The black and grey circles indicate loss-of-function (described as ‘HIGH’ impact on the gene function in Supplementary Table S8) and missense variants. Braces indicate known large InDels. These InDels can be detected by read depth." path-from-xml="dsz005f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025740/dsz005f4.tif" path-from-xml="dsz005f4.tif" target="_blank">Open in new tab</a><a section="137025740" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005f4.png?Expires=1576535926&amp;Signature=Gy-UT918QrKsS6k8sfoUx-VDNqOVapdgt7WHKZCBaAVa7E9pwQe8tcfovk2gqfzDhhEPUDm6cjj7m~WOfK7cTK31tRyFzovx9G4Vo9sdos91DCCy-oyELIdyyDI20TTwucsgwiBRpNsuz~V~4ZFA5w7mJULuPi4Qje5r6QCn~47kr3g8mG5i3GcnH5ObHixsT29CU7Li6DNVj0JomJMTjh4DvcNqxf56ZT7AWaWQd1swbMpWdBNWKIIIoYUYb0yhr-LSGUcFkd1MGHgSEdElk6NttIHxyJM5hLMnoyVG6i0hugX9YeDqsiPJZ5Ts~g0mxrPWPNkAWvCujIKjQIbaGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025740&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz005f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Detected variants in <em>E1</em>, <em>E3</em>, <em>E4</em>, <em>FT5a</em>, and <em>FT2a</em> from 192 mini-core collection. The grey and white boxes indicate UTRs and exons. The solid lines indicate 5′-uptream and intron regions. The black and grey circles indicate loss-of-function (described as ‘HIGH’ impact on the gene function in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>) and missense variants. Braces indicate known large InDels. These InDels can be detected by read depth.</p></div>
</div>                    <h4 scrollto-destination="137025741" id="137025741" class="section-title">E2</h4>
<p>Two alleles, <em>E2</em> and <em>e2</em>, and one novel SNP variant (Chr10_45310686) were detected at <em>E2</em> (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5D</a></span>, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Among them, functional defective <em>e2</em> allele had A to T nucleotide change (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a>, K528*, rs124971350) and the allele frequency among the soybean mini-core collection was 0.42 (Supplementary <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S8</a></span>). A novel SNP (Chr10_45310686), which causes missense variant of Ile490Met, was detected only in PGC086 with the <em>e2</em> allele (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5D</a></span>).</p>                    <h4 scrollto-destination="137025743" id="137025743" class="section-title">E3</h4>
<p>Among two alleles, <em>e3-tr</em> and <em>e3-Mo</em>, detected at <em>E3</em>, the frequency of <em>e3-tr</em> allele, which has a large deletion in the fourth exon, was 0.19 (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5E</a></span>, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7 and S8</a></span>). The missense variant of <em>e3-Mo</em> (Chr19_47638302: G to A, Gly1050Arg) in the third exon of <em>E3</em> was detected in PGC019 and PGC042 (Moshidou Gong 503)<a reveal-id="dsz005-B3" data-open="dsz005-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> derived from Korean Peninsula and China, respectively. The <em>e3-Mo</em> variant is not registered in dbSNP, but we found in one Chinese landrace Ni Ding Hua Mei Dou from 302 soybean re-sequence data<a reveal-id="dsz005-B28" data-open="dsz005-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> (SRR1533240 in NCBI SRA).</p>                    <h4 scrollto-destination="137025745" id="137025745" class="section-title">E4</h4>
<p>The <em>e4-SORE-1</em> allele has a 6.2-kb insertion in the first exon. It was difficult to estimate this insertion from the read coverage of amplicon in the region. However, the presence or absence of a large insertion could be estimated from the read coverage of amplicon (AMPL1037734) at the break point of a large insertion (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5F</a></span>). The average APKM of break point was 325 in the reference type sequence, whereas it was zero in the insertion type sequence of PGC001 and PGC021 derived from Sweden and Japan, respectively. This insertion was also confirmed by the PCR.</p>
<p>Most SNPs (11 of 13 sites) in <em>E4</em> were found from PGC123 and PGC134 derived from Nepal and China (Supplementary <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S9</a></span>), but there was only one missense variant (rs390866037: Leu151Ser), which likely affects gene function. As these variants were detected as homozygous, they are considered to be real variants, not detected by the miss-mapped reads. A frameshift variant in the second exon was only found in PGC005 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). This accession flowered earlier than Williams 82 under field conditions in spite of the same gene combination for all other flowering-related genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p>                    <h4 scrollto-destination="137025748" id="137025748" class="section-title">Other <em>Phytochrome A</em> genes</h4>
<p>Five variants, three frameshifts and two splice site variants, were identified to be high-impact variant to another <em>PhyA</em> gene, <em>Glyma.03G227300/GmPHYA4</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5G</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). This <em>PhyA</em> gene consisted of two main haplotypes, namely, reference type (Hap1–Hap5) and pseudogene type (Hap6–Hap11), which had various loss-of-function sites. The other <em>PhyA</em> gene, <em>Glyma.10G141400</em>, had only one novel frameshift variant (Chr10_37491867) from two accessions, PGC045 and PGC189 derived from Korea and East Timor (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S6 and S9</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5H</a></span>).</p>                    <h4 scrollto-destination="137025750" id="137025750" class="section-title">
<em>Phytochrome B</em> genes</h4>
<p>There has been no report of natural variation in the <em>PhyB</em> genes affecting flowering, but the overexpression of <em>GmPHYB1</em> accelerates flowering under short-day conditions in <em>Arabidopsis.</em><a reveal-id="dsz005-B29" data-open="dsz005-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Only one missense variant (rs124458274) was found from <em>GmPHYB1</em> (<em>Glyma.09G035500</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5I</a></span>). rs124458274 was a common variation in the mini-core collection (allele frequency = 0.69). Two novel frameshifts and six missense variants (one was novel) were found in the other <em>PhyB</em> gene <em>Glyma.15G140000</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5J</a></span>). Although the frameshift variant (Chr15_11442094) was identified in the 19 accessions (Hap8, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), no association with flowering time under the examined field conditions was observed.</p>                    <h4 scrollto-destination="137025752" id="137025752" class="section-title">FLOWERING LOCUS T</h4>
<p>Two florigen genes <em>FT5a</em> and <em>FT2a</em> in the soybean genome play a major role in the induction of flowering.<a reveal-id="dsz005-B17" data-open="dsz005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsz005-B18" data-open="dsz005-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsz005-B30" data-open="dsz005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> As no variant was detected in the exon of <em>FT5a</em>, it appears that <em>FT5a</em> is highly conserved under the evolutionary constraint (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5K</a></span>). Nine and five variants were detected in the intron and 3′-UTR, respectively. Of these, four variants were associated with flowering time determined by simple linear regression analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S8</a></span>). Two variants (rs126630615 in 3′-UTR and rs126639618 in the third intron) were reported by Takeshima et al.<a reveal-id="dsz005-B18" data-open="dsz005-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> This <em>FT5a</em> region has been reported to be one of the flowering time quantitative trait loci (QTLs) in the chromosomal segment substitution lines (CSSLs) derived from a cross between Peking and Enrei.<a reveal-id="dsz005-B31" data-open="dsz005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> In this study, the nucleotide differences between Enrei and Peking were identified as rs126639616_1 (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>) in the intron and rs388994144_1 (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>) in the 3′-UTR region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5K</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). As natural variants in 5′- and 3′-UTR of the <em>FT</em>-like gene affect gene expression and flowering time in rice,<a reveal-id="dsz005-B32" data-open="dsz005-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsz005-B33" data-open="dsz005-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> the variant rs388994144_1 in 3′-UTR region might be involved in gene expression of <em>FT5a</em> and regulation of flowering time in soybean.</p>
<p>The <em>FT2a</em> is a paralogue of <em>FT5a</em> and has been named as <em>E9.</em><a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><em>E9</em> is a leaky allele that is caused by allele-specific transcriptional repression due to the insertion of <em>SORE-1</em> into the first intron. The presence of <em>SORE-1</em> (<em>FT2a-TO</em> allele) delays flowering for 10 days under natural day-length conditions at Harbin, China (45°43′N, 126°45′E).<a reveal-id="dsz005-B8" data-open="dsz005-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Zhao et al.<a reveal-id="dsz005-B17" data-open="dsz005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> also reported a difference of 10 days or more in flowering time between <em>E9</em> and <em>e9</em> in Sapporo, Japan (43°07′N, 141°35′E). As we did not design the primers on the intron of <em>FT2a</em>, the presence or absence of <em>SORE-1</em> is unknown. In this study, one frameshift and two missense variants were found in <em>FT2a</em> (<em>Glyma.16G150700</em>) (<a reveal-id="dsz005-F4" data-open="dsz005-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5L</a></span>). In the first exon, missense SNP rs388788554 (Glu23Asp) was detected in PGC066 (Hap13, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). In the fourth exon, missense novel SNP (Chr16_31114633) was detected in seven accessions (Hap10, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Another novel frameshift variant (Chr16_31111088) in the fourth exon was detected in PGC166 (Hap14, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Enrei had four known and two novel variants in the intron and 3′-UTR of <em>FT2a</em>, whereas there was no variant in Peking. As QTL is not reported in the <em>FT2a</em> region of the CSSL between Peking and Enrei,<a reveal-id="dsz005-B31" data-open="dsz005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> these variants might be not involved in the regulation of flowering time under the evaluation conditions of CSSLs.</p>                    <h4 scrollto-destination="137025755" id="137025755" class="section-title">
<em>FT</em>-like genes</h4>
<p>Four missense variants were detected in the other three <em>FT</em> homologues, rs126830445 in <em>Glyma.16G151000</em> (<em>GmFT2b/GmFTL5</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5M</a></span>), rs127848197 in <em>Glyma.19G108200</em> (<em>GmFT5b/GmFTL6</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5O</a></span>), a novel SNP (Chr16_35778390) in <em>Glyma.16G196300</em> (<em>GmTFL3</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5N</a></span>), and Chr16_4162554 in <em>Glyma.16G044200</em> (<em>FT3a/GmFTL1</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5P</a></span>). The frequency of rs126830445 in <em>GmFT2b/GmFTL5</em> was 0.45, whereas that of rs127848197 in <em>GmFT5b/GmFTL6</em> was 0.94. A novel missense (Val98Ile) SNP (Chr16_35778390) in <em>GmTFL3</em> was only found in PGC037. This accession (YAKUMO MEAKA) is a landrace from Hokkaido, northern part of Japan. In contrast, a novel missense variant (Chr16_4162554) in <em>FT3a/GmFTL1</em> was only found in PGC134 (Hap7, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), which is a medium-maturing accession. No functional defect or missense variant in the other <em>FT</em>-like genes, <em>Glyma.02G069500</em> (<em>GmFTL7</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5Q</a></span>), <em>Glyma.08G363100</em> (<em>GmFT4</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5R</a></span>), and <em>Glyma.08G363200</em> (<em>GmFTL6</em>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5S</a></span>), was found. No variants were detected in <em>Glyma.19G108100</em> (<em>GmFT3b/GmFTL2</em>). The information of alleles identified in these <em>FT</em>-like genes will be useful to clarify the influence of these variants on flowering regulation.</p>                    <h4 scrollto-destination="137025757" id="137025757" class="section-title">
<em>TFL1</em>-like genes</h4>
<p>Two <em>TFL1</em>-like genes, <em>GmTFL2</em> (<em>Glyma.03G194700</em>) and <em>GmTFL1</em> (<em>Glyma.19G194300</em>), exist in the soybean genome. No loss-of-function or missense variant was found in <em>GmTFL2</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5T</a></span>), whereas five missense variants were found in <em>GmTFL1</em>, which determine the growth habit of soybean, classically named as <em>Dt1</em> locus<a reveal-id="dsz005-B34" data-open="dsz005-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5U</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). These sites are located where amino acids are highly conserved across <em>TFL1</em> orthologues: GmTFL2, GmTFL1/Dt1, <em>Lotus japonicas</em> CEN/TFL1, pea TFL1a, <em>Arabidopsis</em> TFL1, <em>Arabidopsis</em> ATC, and <em>Antirrhinum majus</em> CEN.<a reveal-id="dsz005-B35 dsz005-B36 dsz005-B37 dsz005-B38 dsz005-B39" data-open="dsz005-B35 dsz005-B36 dsz005-B37 dsz005-B38 dsz005-B39" class="link link-ref link-reveal xref-bibr"><sup>35–39</sup></a> The variant site of rs745009806 (Arg130Lys) was conserved in <em>TFL</em>, but not in ATC and CEN. Another four variant sites, rs127928577 (Arg62Ser), rs392653457 (Leu67Gln), rs127928574 (Pro113Leu), and rs127928573 (Arg166Trp), exist at a highly conserved amino acid site. Of these, rs127928573 (Arg166Trp) is known as <em>dt1</em> allele in soybean.<a reveal-id="dsz005-B34" data-open="dsz005-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> As the loss-of-function <em>Sidt1</em> allele has been reported at S79N in <em>Sesamum indium</em> L.,<a reveal-id="dsz005-B40" data-open="dsz005-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> the other three missense variants should be examined to verify whether they are new defective <em>dt1</em> alleles or not.</p>                    <h4 scrollto-destination="137025759" id="137025759" class="section-title">
<em>Two-component response regulator</em>-like genes</h4>
<p>Among three <em>two-component response regulator</em>-like genes screened, the stop-lost variant (rs125308117) and five missense variants were found in <em>Glyma.12G073900</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5V</a></span>). The allele frequency of the stop-lost variant (rs125308117) was 0.31. In <em>Glyma.19G260400</em>, only seven missense variants were found (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5W</a></span>). Among seven variants with high impact on gene function in <em>Glyma.U034500</em> on scaffold 32 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), four were frameshift variants due to InDels. Frameshift variant (A &gt; AT, M737I, Scaffold32: 198773) was the major allele, and 82% of the mini-core collection possesses this allele. In contrast, other frameshift variants of insertion (C &gt; CT, Q759M, scaffold_32: 198706) from PGC044, deletion (TTGCC &gt; -, G409D, scaffold_32: 200001) from 16 accessions, and deletion (AC &gt; A, V355L, scaffold_32: 200258) from three accessions, PGC005, PGC094, and PGC174, were rare alleles in the mini-core collection (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). The remaining three variants (scaffold_32_199043, scaffold_32_202754, and scaffold_32_218486) with high impact on gene function were stop-gained variant. These results indicate that <em>Glyma.U034500</em> of most soybean accessions, except for Hap1, Hap2, Hap3, and Hap4 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), losses its function. Among these three genes, <em>Glyma.12G073900</em> and <em>Glyma.U034500</em> showed high similarity (91%) at the amino acid sequence level. The fact that length of the amino acid sequence of <em>Glyma.12G073900</em> of Williams 82 is shorter (92 aa) than that of <em>Glyma.U034500</em> (765 aa) at the C terminal indicates that <em>Glyma.12G073900</em> encodes truncated protein. Although flowering control by two-component response regulator-like genes has been reported in various species,<a reveal-id="dsz005-B41 dsz005-B42 dsz005-B43" data-open="dsz005-B41 dsz005-B42 dsz005-B43" class="link link-ref link-reveal xref-bibr"><sup>41–43</sup></a> the role of this gene and its variant in soybean flowering are unknown. Among the <em>two-component response regulator</em>-like genes, only variants in <em>Glyma.12G073900</em> and <em>Glyma.U034500</em> were associated with flowering time, determined by simple linear regression analysis (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). <em>Glyma.U034500</em> (Chr11 11.23-11.26Mb on Gmax189) is located near a previously reported QTL as <em>qFT-B1</em> (nearest marker: Satt519 74.7cM, Chr11 13.98Mb on Gmax189) in the 96 from the cross between Tokei 780 and the soja accession Hidaka 4.<a reveal-id="dsz005-B44" data-open="dsz005-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Although they reported the effect of <em>qFT-B1</em> is 3.4–10.8 days, the genotype of this frameshift site (Scaffold 32:198773) in the parent of recombinant inbred lines (RILs) is unknown. It can be confirmed using the detected variants as a DNA marker whether detected stop-lost and stop-gained variants in <em>two-component response regulator</em>-like genes are responsible genes for the flowering time.</p>                    <h4 scrollto-destination="137025761" id="137025761" class="section-title">Other genes</h4>
<p>Seven missense variants were identified in <em>Glyma.16G200700</em> encoding MADS box protein, whereas functional defect variant was not found (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5Y</a></span>). MADS-domain transcription factor of the <em>AGL6</em> gene is known to be a factor responsible for the regulation of lateral organ development, flowering time, and circadian clock in <em>Arabidopsis.</em><a reveal-id="dsz005-B45" data-open="dsz005-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsz005-B46" data-open="dsz005-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><em>AGL6</em> regulates flowering through the <em>FLC</em> family genes and <em>FT.</em><a reveal-id="dsz005-B46" data-open="dsz005-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Two missense variants, rs389022394 and rs126888609, on <em>Glyma.16G200700</em> are significantly associated with the flowering time, determined by the simple linear regression analysis (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). As there is no report for <em>FLC</em>-like genes in soybean, it will be important to examine whether the detected two variants from <em>Glyma.16G200700</em> have an effect on the flowering time.</p>
<p>One novel frameshift variant (Chr17_3955763) of WD repeat-containing protein 61 (<em>Glyma.17g52100</em>) was significantly associated with flowering time, determined by the simple linear regression analysis (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Although the allele frequency of the frameshift variant was 0.52 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>), no QTL has been reported in this region. In <em>Arabidopsis</em>, WD repeat-containing protein VIP3 regulates flowering time via the vernalization pathway;<a reveal-id="dsz005-B47" data-open="dsz005-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> however, the vernalization pathway is not known in soybean and it is difficult to infer the role. As a large proportion of the mini-core collection has the novel frameshift variant for <em>Glyma.17G052100</em> and missense variant for <em>Glyma.16G200700</em>, it is necessary to confirm genetically whether these novel variants really affect the flowering time. The transcription factor gene, <em>Glyma.17G090500,</em> was sequenced as the control for variant detection. All known variants were detected correctly (data not shown).</p>                    <h3 scrollto-destination="137025764" id="137025764" class="section-title">3.4. Gene-based association test for flowering time</h3>
<p>To refine responsible variants associated with variation in flowering time in the mini-core collection, we performed multiple linear regression analysis using variants significantly associated with flowering time in the simple linear regression analysis. The variants of <em>e2</em>, <em>e3-tr</em> and stop-lost variant (rs125308117) of <em>two-component response regulator</em>-like gene on Chr12 were significant in 3 yrs, and rs127928573 in <em>Dt1</em> was significant only in 2013 (<a reveal-id="dsz005-T4" data-open="dsz005-T4" class="link link-reveal link-table xref-fig">Table 4</a>). These genes could explain 51.82%, 51.13%, and 52.83% of the phenotypic variation of flowering time among the mini-core collection in 3 yrs, respectively. In this study, the variants of <em>E1</em> and <em>E4</em> could not be incorporated into the association analysis due to the low frequency of <em>e1-nl</em> (0.5%) and <em>e4-SORE1</em> (1%) alleles in the mini-core collection. The extent of variation explained in this study was ∼10% lower than 62–66% reported by Zhai et al.<a reveal-id="dsz005-B48" data-open="dsz005-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Even though the allele frequency of <em>e1-as</em> was relatively high (9%), <em>e1-as</em> was not significant in the multiple linear regression analysis. This is probably because the genetic effect of <em>e1-as</em> is smaller than that of <em>E2, E3,</em> and <em>two-component response regulator</em>-like gene. In the simple linear regression analysis, the <em>P</em>-value of <em>E2</em> (2.0e<sup>−16</sup>)<em>, E3</em> (5.2e<sup>−14</sup>–1.9e<sup>−13</sup>), and <em>two-component response regulator</em>-like gene (7.7e<sup>−11</sup>–1.8e<sup>−8</sup>) was considerably lower than that of <em>e1-as</em> (0.015–0.033) (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Further experiment using a larger population size is required to examine the remaining variation that could not be explained by the three genes with <em>e1-as</em>.</p>                        <a id="137025766" scrollto-destination="137025766"></a>
<div content-id="dsz005-T4" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T4" class="table-wrap-title" data-id="dsz005-T4">
<div class="label">Table 4</div>
<div class="caption"><p>DNA polymorphisms responsible for flowering time variation in the mini-core collection as revealed by multiple regression analysis</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Data set </th>
<th rowspan="2">SNP No. </th>
<th rowspan="2">SNP name </th>
<th colspan="2">Physical position<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr> </th>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Description<a reveal-id="tblfn14" data-open="tblfn14" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Alleles </th>
<th rowspan="2">Effect<a reveal-id="tblfn15" data-open="tblfn15" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">MAF </th>
<th colspan="3">Parameters estimated by linear regression<hr> </th>
<th rowspan="2">Contribution rate (%) </th>
</tr>
<tr>
<th>Chromosome </th>
<th>bp </th>
<th>β<a reveal-id="tblfn16" data-open="tblfn16" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th>
<span class="small-caps">s</span>.<span class="small-caps">e</span>. </th>
<th>
<em>P</em>-value<sup>e</sup> </th>
</tr>
</thead>
<tbody>
<tr>
<td>2011 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td> </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.21 </td>
<td>1.51 </td>
<td>2.5E-03** </td>
<td>21 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−9.88 </td>
<td>3.05 </td>
<td>3.9E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>11.24 </td>
<td>3.53 </td>
<td>4.5E-03** </td>
<td>12 </td>
</tr>
<tr>
<td>2012 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.44 </td>
<td>1.47 </td>
<td>1.3E-03** </td>
<td>22 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−8.50 </td>
<td>2.95 </td>
<td>9.0E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>9.65 </td>
<td>3.42 </td>
<td>1.0E-02* </td>
<td>10 </td>
</tr>
<tr>
<td>2013 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.36 </td>
<td>1.66 </td>
<td>4.0E-03** </td>
<td>20 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−12.18 </td>
<td>3.34 </td>
<td>1.5E-03** </td>
<td>13 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>12.69 </td>
<td>3.86 </td>
<td>3.5E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP349 </td>
<td>rs127928573 </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>T/A </td>
<td>Missense_variant </td>
<td>0.09 </td>
<td>11.23 </td>
<td>5.11 </td>
<td>3.9E-02* </td>
<td>0.5 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T4" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Data set </th>
<th rowspan="2">SNP No. </th>
<th rowspan="2">SNP name </th>
<th colspan="2">Physical position<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr> </th>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Description<a reveal-id="tblfn14" data-open="tblfn14" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Alleles </th>
<th rowspan="2">Effect<a reveal-id="tblfn15" data-open="tblfn15" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">MAF </th>
<th colspan="3">Parameters estimated by linear regression<hr> </th>
<th rowspan="2">Contribution rate (%) </th>
</tr>
<tr>
<th>Chromosome </th>
<th>bp </th>
<th>β<a reveal-id="tblfn16" data-open="tblfn16" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th>
<span class="small-caps">s</span>.<span class="small-caps">e</span>. </th>
<th>
<em>P</em>-value<sup>e</sup> </th>
</tr>
</thead>
<tbody>
<tr>
<td>2011 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td> </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.21 </td>
<td>1.51 </td>
<td>2.5E-03** </td>
<td>21 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−9.88 </td>
<td>3.05 </td>
<td>3.9E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>11.24 </td>
<td>3.53 </td>
<td>4.5E-03** </td>
<td>12 </td>
</tr>
<tr>
<td>2012 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.44 </td>
<td>1.47 </td>
<td>1.3E-03** </td>
<td>22 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−8.50 </td>
<td>2.95 </td>
<td>9.0E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>9.65 </td>
<td>3.42 </td>
<td>1.0E-02* </td>
<td>10 </td>
</tr>
<tr>
<td>2013 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.36 </td>
<td>1.66 </td>
<td>4.0E-03** </td>
<td>20 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−12.18 </td>
<td>3.34 </td>
<td>1.5E-03** </td>
<td>13 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>12.69 </td>
<td>3.86 </td>
<td>3.5E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP349 </td>
<td>rs127928573 </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>T/A </td>
<td>Missense_variant </td>
<td>0.09 </td>
<td>11.23 </td>
<td>5.11 </td>
<td>3.9E-02* </td>
<td>0.5 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn12"><div class="fn"><p>MAF: minor allele frequency; <span class="small-caps">s</span>.<span class="small-caps">e</span>.: standard error.</p></div></div>
<div content-id="tblfn13"><div class="fn">
<div class="label">a</div>
<p>Physical position on Gmax275.</p>
</div></div>
<div content-id="tblfn14"><div class="fn">
<div class="label">b</div>
<p>Gene description was obtained from Phytozome 12.</p>
</div></div>
<div content-id="tblfn15"><div class="fn">
<div class="label">c</div>
<p>Effect to gene function annotated by snpEff. Effect of AMPL1040314 was defined by manually.</p>
</div></div>
<div content-id="tblfn16"><div class="fn">
<div class="label">d</div>
<p>Standardized regression coefficients.</p>
</div></div>
<div content-id="tblfn17"><div class="fn">
<div class="label">e</div>
<p>Adjusted <em>P</em>-value was obtained from multivariate models days to flowering and genotype as covariates. Signification codes: ‘**’ 0.01 ‘*’ 0.05.</p>
</div></div>
<div content-id="tblfn18"><div class="fn">
<div class="label">f</div>
<p>Large deletion on E3 estimated by coverage of four amplicon on 4th exon.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025766" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz005-T4" class="table-wrap-title" data-id="dsz005-T4">
<div class="label">Table 4</div>
<div class="caption"><p>DNA polymorphisms responsible for flowering time variation in the mini-core collection as revealed by multiple regression analysis</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Data set </th>
<th rowspan="2">SNP No. </th>
<th rowspan="2">SNP name </th>
<th colspan="2">Physical position<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr> </th>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Description<a reveal-id="tblfn14" data-open="tblfn14" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Alleles </th>
<th rowspan="2">Effect<a reveal-id="tblfn15" data-open="tblfn15" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">MAF </th>
<th colspan="3">Parameters estimated by linear regression<hr> </th>
<th rowspan="2">Contribution rate (%) </th>
</tr>
<tr>
<th>Chromosome </th>
<th>bp </th>
<th>β<a reveal-id="tblfn16" data-open="tblfn16" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th>
<span class="small-caps">s</span>.<span class="small-caps">e</span>. </th>
<th>
<em>P</em>-value<sup>e</sup> </th>
</tr>
</thead>
<tbody>
<tr>
<td>2011 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td> </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.21 </td>
<td>1.51 </td>
<td>2.5E-03** </td>
<td>21 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−9.88 </td>
<td>3.05 </td>
<td>3.9E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>11.24 </td>
<td>3.53 </td>
<td>4.5E-03** </td>
<td>12 </td>
</tr>
<tr>
<td>2012 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.44 </td>
<td>1.47 </td>
<td>1.3E-03** </td>
<td>22 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−8.50 </td>
<td>2.95 </td>
<td>9.0E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>9.65 </td>
<td>3.42 </td>
<td>1.0E-02* </td>
<td>10 </td>
</tr>
<tr>
<td>2013 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.36 </td>
<td>1.66 </td>
<td>4.0E-03** </td>
<td>20 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−12.18 </td>
<td>3.34 </td>
<td>1.5E-03** </td>
<td>13 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>12.69 </td>
<td>3.86 </td>
<td>3.5E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP349 </td>
<td>rs127928573 </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>T/A </td>
<td>Missense_variant </td>
<td>0.09 </td>
<td>11.23 </td>
<td>5.11 </td>
<td>3.9E-02* </td>
<td>0.5 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz005-T4" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="2">Data set </th>
<th rowspan="2">SNP No. </th>
<th rowspan="2">SNP name </th>
<th colspan="2">Physical position<a reveal-id="tblfn13" data-open="tblfn13" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr> </th>
<th rowspan="2">Glyma ID </th>
<th rowspan="2">Description<a reveal-id="tblfn14" data-open="tblfn14" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th>
<th rowspan="2">Alleles </th>
<th rowspan="2">Effect<a reveal-id="tblfn15" data-open="tblfn15" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th>
<th rowspan="2">MAF </th>
<th colspan="3">Parameters estimated by linear regression<hr> </th>
<th rowspan="2">Contribution rate (%) </th>
</tr>
<tr>
<th>Chromosome </th>
<th>bp </th>
<th>β<a reveal-id="tblfn16" data-open="tblfn16" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th>
<th>
<span class="small-caps">s</span>.<span class="small-caps">e</span>. </th>
<th>
<em>P</em>-value<sup>e</sup> </th>
</tr>
</thead>
<tbody>
<tr>
<td>2011 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td> </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.21 </td>
<td>1.51 </td>
<td>2.5E-03** </td>
<td>21 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−9.88 </td>
<td>3.05 </td>
<td>3.9E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>11.24 </td>
<td>3.53 </td>
<td>4.5E-03** </td>
<td>12 </td>
</tr>
<tr>
<td>2012 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.44 </td>
<td>1.47 </td>
<td>1.3E-03** </td>
<td>22 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−8.50 </td>
<td>2.95 </td>
<td>9.0E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>9.65 </td>
<td>3.42 </td>
<td>1.0E-02* </td>
<td>10 </td>
</tr>
<tr>
<td>2013 </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
<td>  </td>
</tr>
<tr>
<td>  </td>
<td>E2 </td>
<td>rs124971350 (<em>e2</em>) </td>
<td>Chr10 </td>
<td>45310798 </td>
<td>Glyma.10G221500 </td>
<td>GIGANTEA (E2) </td>
<td>A/T </td>
<td>Stop_gained </td>
<td>0.42 </td>
<td>−5.36 </td>
<td>1.66 </td>
<td>4.0E-03** </td>
<td>20 </td>
</tr>
<tr>
<td>  </td>
<td>SNP379 </td>
<td>
<em>e3tr</em><a reveal-id="tblfn18" data-open="tblfn18" class="link link-ref link-reveal xref-fn"><sup>f</sup></a> </td>
<td>Chr19 </td>
<td>47641562 </td>
<td>Glyma.19G224200 </td>
<td>Phytochrome A (E3) </td>
<td>Large deletion </td>
<td>Loss of exon 4 </td>
<td>0.17 </td>
<td>−12.18 </td>
<td>3.34 </td>
<td>1.5E-03** </td>
<td>13 </td>
</tr>
<tr>
<td>  </td>
<td>SNP156 </td>
<td>rs125308117 </td>
<td>Chr12 </td>
<td>5520945 </td>
<td>Glyma.12G073900 </td>
<td>Two-component response regulator-like </td>
<td>T/C </td>
<td>Stop_lost </td>
<td>0.31 </td>
<td>12.69 </td>
<td>3.86 </td>
<td>3.5E-03** </td>
<td>19 </td>
</tr>
<tr>
<td>  </td>
<td>SNP349 </td>
<td>rs127928573 </td>
<td>Chr19 </td>
<td>45183701 </td>
<td>Glyma.19G194300 </td>
<td>TERMINAL FLOWER 1 </td>
<td>T/A </td>
<td>Missense_variant </td>
<td>0.09 </td>
<td>11.23 </td>
<td>5.11 </td>
<td>3.9E-02* </td>
<td>0.5 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn12"><div class="fn"><p>MAF: minor allele frequency; <span class="small-caps">s</span>.<span class="small-caps">e</span>.: standard error.</p></div></div>
<div content-id="tblfn13"><div class="fn">
<div class="label">a</div>
<p>Physical position on Gmax275.</p>
</div></div>
<div content-id="tblfn14"><div class="fn">
<div class="label">b</div>
<p>Gene description was obtained from Phytozome 12.</p>
</div></div>
<div content-id="tblfn15"><div class="fn">
<div class="label">c</div>
<p>Effect to gene function annotated by snpEff. Effect of AMPL1040314 was defined by manually.</p>
</div></div>
<div content-id="tblfn16"><div class="fn">
<div class="label">d</div>
<p>Standardized regression coefficients.</p>
</div></div>
<div content-id="tblfn17"><div class="fn">
<div class="label">e</div>
<p>Adjusted <em>P</em>-value was obtained from multivariate models days to flowering and genotype as covariates. Signification codes: ‘**’ 0.01 ‘*’ 0.05.</p>
</div></div>
<div content-id="tblfn18"><div class="fn">
<div class="label">f</div>
<p>Large deletion on E3 estimated by coverage of four amplicon on 4th exon.</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025766" target="_blank">Open in new tab</a></div>
</div></div>
<p>The other five genes, namely, WD repeat-containing protein 61 (<em>Glyma.17G052100</em>), MADS-box protein (<em>Glyma.16G200700</em>), PhyB (<em>Glyma.15G140000</em>), two-component response regulator-like gene (<em>Glyma.U034500</em>), and <em>FT2a/GmFTL3</em> (<em>Glyma.16G150700</em>), were significant in the simple linear regression analysis (<em>P </em>&lt;<em> </em>0.0001) but not significant in the multiple linear regression analysis (<a reveal-id="dsz005-T3" data-open="dsz005-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Variants that differ between Enrei and Peking can be used to confirm allele effect on flowering time using the phenotypic data of CSSLs.<a reveal-id="dsz005-B31" data-open="dsz005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Peking had a novel frameshift variant (Chr17_3955763) in WD repeat-containing protein 61 (<em>Glyma.17G052100</em>), two missense variants (rs389022394 and rs126888609) in MADS-box protein (<em>Glyma.16G200700</em>), and one frameshift variant (Chr15_11442094) in PhyB (<em>Glyma.15G140000</em>), and no variant in Enrei (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). However, no flowering time QTL has been reported to Chr17, Chr16, and Chr15; these genes may not be involved in flowering time regulation under the evaluation conditions of CSSLs.<a reveal-id="dsz005-B31" data-open="dsz005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>
<p>It was the stop-gain allele <em>E2</em> that showed the highest association with flowering time. The effect of this variant promotes flowering about 5 days (<a reveal-id="dsz005-T4" data-open="dsz005-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Watanabe et al.<a reveal-id="dsz005-B11" data-open="dsz005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> reported that the difference in days to flowering between <em>E2/E2</em> and <em>e2/e2</em> was ∼9 days, which is consistent with the result of this study. The next strong association with flowering time was observed at <em>E3</em>. The <em>e3-tr</em> allele (Horosy-<em>e3</em>) has been reported to promote flowering for ∼17 days,<a reveal-id="dsz005-B13" data-open="dsz005-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> but it was estimated as 9–13 days in this study. The smaller estimation at <em>E3</em> can be explained by the absence of <em>e3-Mo</em> allele. As there are only two accessions, PGC019 and PGC042 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576535926&amp;Signature=Q1T5Dy0YJkFnvgjzvllIuao2076~-85H~iCBICO~umCX5xmpnwWZpg8wqY9d6t1zMEoZTcqDf4LQzDRcESqczxeQ8yh3JOAAa1TUbVI4jtdwQ2v67Lz7YGNbOH3hd9BHajL0OTP0mIR-gn8F5XM-EYXW~a2n-lE0XcTN9gCrEhcfsAsSuaXQ-akKLhnX6FM442W1kVLDx0T3jhVo0AhPbGlo2Xwhy8sqOw7WT~PmGKGLSJymX0JA5rrRSqnfTq3HdWJEptdYNRHJvJtMZTvdsMhhh23rgaN1C6Cr5MXYLSAnFNL4W7K~CE6T9T7X9eqQ7J5S8yjDBXl~KvjrPMT0Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), the <em>e3-Mo</em> allele could not be included in the association analysis.</p>
<p>The effect of the missense variant (rs127928573, Arg166Trp) of <em>Dt1</em> was detected only for 2013 data set; it delayed flowering by ∼11 days compared with that by the <em>Dt1</em> allele (<a reveal-id="dsz005-T4" data-open="dsz005-T4" class="link link-reveal link-table xref-fig">Table 4</a>). <em>Dt1</em> is reported as the locus strongly associated with days to maturity and plant height.<a reveal-id="dsz005-B49" data-open="dsz005-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Zhang et al.<a reveal-id="dsz005-B49" data-open="dsz005-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> identified the <em>Dt1</em> gene at 18.6-kb upstream of the peak SNP, which was associated with days to maturity and plant height. <em>Dt1</em> plays a primary role in not only stem termination but also floral transition.<a reveal-id="dsz005-B50" data-open="dsz005-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><a reveal-id="dsz005-B51" data-open="dsz005-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> As no visible influence on the flowering time has been reported with <em>dt1</em> VIGS-induced suppression,<a reveal-id="dsz005-B34" data-open="dsz005-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> the detected SNP on <em>Dt1</em> in this study suggests the presence of other gene in the surrounding region related to the flowering time.</p>
<p>The effect of stop-lost variant (rs125308117) in the <em>two-component response regulator</em>-like gene (<em>Glyma.12G073900</em>) was significant (<em>P </em>=<em> </em>4.5 e<sup>−3</sup> in 2011, <em>P </em>=<em> </em>1.0 e<sup>−2</sup> in 2012, <em>P </em>=<em> </em>3.5 e<sup>−3</sup> in 2013), and the plant flowers ∼10–13 days later. Involvement of the <em>two-component response regulator</em>-like gene in flowering time has been reported in <em>Arabidopsis</em> and rice; it may be functionally preserved as a flowering time-related gene in soybean. Williams 82 (reference genome) has C-terminal truncated protein as described above, whereas the rs125308117 variant has longer amino acid sequence and allele effect of delayed flowering for 4.7 days (<a reveal-id="dsz005-T4" data-open="dsz005-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Although <em>Glyma.12G073900</em> is located near a previously reported QTL as <em>qFT-H</em> (nearest marker: Satt442 on Chr12: 6,390,806–6,391,062) in RILs with the <em>E1</em> allele from the cross between Tokei 780 and the <em>soja</em> accession Hidaka 4,<a reveal-id="dsz005-B44" data-open="dsz005-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> the allele type of <em>Glyma.12G073900</em> in both accessions is unknown. The genomic region surrounding <em>Glyma.12G073900</em> has been reported to include flowering time QTL <em>qDFF-Gm12</em> in CSSLs.<a reveal-id="dsz005-B31" data-open="dsz005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><em>Glyma.12G073900</em> of Peking (PGC084) is the stop-lost type (longer protein), whereas that of Enrei (PGC025) is reference type (truncated protein). Similar to the present study, Peking allele delayed flowering by ∼3.7 days (LOD score is 36.3, flanking markers: C12-BARC- 015603-02006 and s024200450).<a reveal-id="dsz005-B31" data-open="dsz005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> These data suggest that <em>Glyma.12G073900</em> is one of the candidate gene for <em>qDFF-Gm12</em>.</p>                    <h2 scrollto-destination="137025771" id="137025771" class="section-title">4. Conclusions</h2>
<p>Flowering time and maturity are the most important factors affecting adaptability and yield. To increase the yield of soybean, it is necessary to control flowering time at an appropriate time using a combination of flowering time-related genes or alleles. Preparing a catalogue of flowering time-related genes makes it possible to freely combine alleles with various effects using the DNA markers. Our results indicate that novel alleles and accessions with such novel alleles can be rapidly detected using the AmpliSeq technology. Although multiple defective alleles were identified, we could not include all of them in the association study of flowering time due to low allele frequency. Nevertheless, the variants detected in this study could explain 51.1–52.3% of the flowering time variation in the soybean mini-core collection. These variants consisted of a novel two-component response regulator gene besides known flowering time-related genes. Therefore, the AmpliSeq technology is useful for discovering novel variants in the target genes.</p>                    <h2 scrollto-destination="137025773" id="137025773" class="section-title">Data availability</h2>
<p>All sequences analysed in this study have been deposited in the DDBJ database under the BioProject Accession number: PRJDB7633.</p>                    <h2 scrollto-destination="137025775" id="137025775" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>We thank Naoki Kamei and Osamu Takahashi (Thermo Fisher Scientific, Life Technologies, Japan Ltd.) for sequencing using Ion S5 XL and offering technical support in using the Ion S5 sequencer. We also thank Yuko Yokota (NICS, NARO) and Takashi Sayama (NICS, NARO) for providing the materials necessary for the study. This study was supported in part by the Special Scheme Project on Advanced Research and Development for Next-Generation Technology from the Ministry of Agriculture, Forestry and Fisheries of Japan. </p>                    <h3 scrollto-destination="137025777" id="137025777" class="section-title">Accession number</h3>
<p>All sequences analyzed in the present study have been deposited in the DDBJ database under the BioProject Accession number: PRJDB7633.</p>                    <h3 scrollto-destination="137025779" id="137025779" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="137025781" id="137025781" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz005-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cober</div> <div class="given-names">E.R.</div>
</div>, <div class="name string-name">
<div class="surname">Morrison</div> <div class="given-names">M.J.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Regulation of seed yield and agronomic characters by photoperiod sensitivity and growth habit genes in soybean</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">120</div>, <div class="fpage">1005</div>–<div class="lpage">12</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20seed%20yield%20and%20agronomic%20characters%20by%20photoperiod%20sensitivity%20and%20growth%20habit%20genes%20in%20soybean&amp;author=E.R.%20Cober&amp;author=M.J.%20Morrison&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet&amp;volume=120&amp;pages=1005-12" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00122-009-1228-6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1228-6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00122-009-1228-6"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20012856" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Regulation%20of%20seed%20yield%20and%20agronomic%20characters%20by%20photoperiod%20sensitivity%20and%20growth%20habit%20genes%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">W.X.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">M.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Van</div> <div class="given-names">K.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">QTL identification of yield-related traits and their association with flowering and maturity in soybean</div>, <div class="source">J. Crop Sci. Biotechnol</div> ., <div class="volume">14</div>, <div class="fpage">65</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=QTL%20identification%20of%20yield-related%20traits%20and%20their%20association%20with%20flowering%20and%20maturity%20in%20soybean&amp;author=W.X.%20Liu&amp;author=M.Y.%20Kim&amp;author=K.%20Van&amp;publication_year=2011&amp;journal=J.%20Crop%20Sci.%20Biotechnol&amp;volume=14&amp;pages=65-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s12892-010-0115-7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12892-010-0115-7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs12892-010-0115-7"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:QTL%20identification%20of%20yield-related%20traits%20and%20their%20association%20with%20flowering%20and%20maturity%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tsubokura</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Xia</div> <div class="given-names">Z.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Natural variation in the genes responsible for maturity loci <em>E1</em>, <em>E2</em>, <em>E3</em> and <em>E4</em> in soybean</div>, <div class="source">Ann. Bot</div> ., <div class="volume">113</div>, <div class="fpage">429</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20in%20the%20genes%20responsible%20for%20maturity%20loci%20E1%2C%20E2%2C%20E3%20and%20E4%20in%20soybean&amp;author=Y.%20Tsubokura&amp;author=S.%20Watanabe&amp;author=Z.%20Xia&amp;publication_year=2014&amp;journal=Ann.%20Bot&amp;volume=113&amp;pages=429-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/aob/mct269" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmct269" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Faob%2Fmct269"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24284817" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Natural%20variation%20in%20the%20genes%20responsible%20for%20maturity%20loci%20E1%2C%20E2%2C%20E3%20and%20E4%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Bernard</div> <div class="given-names">R.L.</div>
</div></div> <div class="year">1971</div>, 
<div class="article-title">Two major genes for time of flowering and maturity in soybean</div>, <div class="source">Crop Sci</div> ., <div class="volume">11</div>, <div class="fpage">242</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Two%20major%20genes%20for%20time%20of%20flowering%20and%20maturity%20in%20soybean&amp;author=R.L.%20Bernard&amp;publication_year=1971&amp;journal=Crop%20Sci&amp;volume=11&amp;pages=242-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2135/cropsci1971.0011183X001100020022x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1971.0011183X001100020022x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2135%2Fcropsci1971.0011183X001100020022x"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Two%20major%20genes%20for%20time%20of%20flowering%20and%20maturity%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Buzzell</div> <div class="given-names">R.I.</div>
</div></div> <div class="year">1971</div>, 
<div class="article-title">Inheritance of a soybean flowering response to fluorescent daylength conditions</div>, <div class="source">Can. J. Genet. Cytol</div> ., <div class="volume">13</div>, <div class="fpage">703</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Inheritance%20of%20a%20soybean%20flowering%20response%20to%20fluorescent%20daylength%20conditions&amp;author=R.I.%20Buzzell&amp;publication_year=1971&amp;journal=Can.%20J.%20Genet.%20Cytol&amp;volume=13&amp;pages=703-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1139/g71-100" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg71-100" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1139%2Fg71-100"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inheritance%20of%20a%20soybean%20flowering%20response%20to%20fluorescent%20daylength%20conditions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Buzzell</div> <div class="given-names">R.I.</div>
</div>, <div class="name string-name">
<div class="surname">Voldeng</div> <div class="given-names">H.D.</div>
</div>
</div> <div class="year">1980</div>, 
<div class="article-title">Inheritance of insensitivity to long daylength</div>, <div class="source">Soyb. Genet. Newsl</div> ., <div class="volume">7</div>, <div class="fpage">26</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Inheritance%20of%20insensitivity%20to%20long%20daylength&amp;aulast=Buzzell&amp;date=1980&amp;spage=26&amp;epage=9&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inheritance%20of%20insensitivity%20to%20long%20daylength&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cober</div> <div class="given-names">E.R.</div>
</div>, <div class="name string-name">
<div class="surname">Voldeng</div> <div class="given-names">H.D.</div>
</div>
</div> <div class="year">2001</div>, 
<div class="article-title">A new soybean maturity and photoperiod-sensitivity locus linked to E1 and T</div>, <div class="source">Crop Sci</div> ., <div class="volume">41</div>, <div class="fpage">698</div>–<div class="lpage">701</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20new%20soybean%20maturity%20and%20photoperiod-sensitivity%20locus%20linked%20to%20E1%20and%20T&amp;author=E.R.%20Cober&amp;author=H.D.%20Voldeng&amp;publication_year=2001&amp;journal=Crop%20Sci&amp;volume=41&amp;pages=698-701" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2135/cropsci2001.413698x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2001.413698x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2135%2Fcropsci2001.413698x"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20new%20soybean%20maturity%20and%20photoperiod-sensitivity%20locus%20linked%20to%20E1%20and%20T&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kong</div> <div class="given-names">F.J.</div>
</div>, <div class="name string-name">
<div class="surname">Nan</div> <div class="given-names">H.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Cao</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">A new dominant gene E9 conditions early flowering and maturity in soybean</div>, <div class="source">Crop Sci</div> ., <div class="volume">54</div>, <div class="fpage">2529</div>–<div class="lpage">35</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20new%20dominant%20gene%20E9%20conditions%20early%20flowering%20and%20maturity%20in%20soybean&amp;author=F.J.%20Kong&amp;author=H.Y.%20Nan&amp;author=D.%20Cao&amp;publication_year=2014&amp;journal=Crop%20Sci&amp;volume=54&amp;pages=2529-35" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2135/cropsci2014.03.0228" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2014.03.0228" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2135%2Fcropsci2014.03.0228"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20new%20dominant%20gene%20E9%20conditions%20early%20flowering%20and%20maturity%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Samanfar</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Molnar</div> <div class="given-names">S.J.</div>
</div>, <div class="name string-name">
<div class="surname">Charette</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Mapping and identification of a potential candidate gene for a novel maturity locus, E10, in soybean</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">130</div>, <div class="fpage">377</div>–<div class="lpage">90</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20identification%20of%20a%20potential%20candidate%20gene%20for%20a%20novel%20maturity%20locus%2C%20E10%2C%20in%20soybean&amp;author=B.%20Samanfar&amp;author=S.J.%20Molnar&amp;author=M.%20Charette&amp;publication_year=2017&amp;journal=Theor.%20Appl.%20Genet&amp;volume=130&amp;pages=377-90" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00122-016-2819-7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-016-2819-7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00122-016-2819-7"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27832313" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mapping%20and%20identification%20of%20a%20potential%20candidate%20gene%20for%20a%20novel%20maturity%20locus%2C%20E10%2C%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xia</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Yamada</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Positional cloning and characterization reveal the molecular basis for soybean maturity locus <em>E1</em> that regulates photoperiodic flowering,</div> <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">E2155</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Positional%20cloning%20and%20characterization%20reveal%20the%20molecular%20basis%20for%20soybean%20maturity%20locus%20E1%20that%20regulates%20photoperiodic%20flowering%2C&amp;author=Z.%20Xia&amp;author=S.%20Watanabe&amp;author=T.%20Yamada&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=E2155-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1117982109" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1117982109" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1117982109"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Positional%20cloning%20and%20characterization%20reveal%20the%20molecular%20basis%20for%20soybean%20maturity%20locus%20E1%20that%20regulates%20photoperiodic%20flowering%2C&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Xia</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Hideshima</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">A map-based cloning strategy employing a residual heterozygous line reveals that the <em>GIGANTEA</em> gene is involved in soybean maturity and flowering</div>, <div class="source">Genetics</div> , <div class="volume">188</div>, <div class="fpage">395</div>–<div class="lpage">407</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20map-based%20cloning%20strategy%20employing%20a%20residual%20heterozygous%20line%20reveals%20that%20the%20GIGANTEA%20gene%20is%20involved%20in%20soybean%20maturity%20and%20flowering&amp;author=S.%20Watanabe&amp;author=Z.%20Xia&amp;author=R.%20Hideshima&amp;publication_year=2011&amp;journal=Genetics&amp;volume=188&amp;pages=395-407" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.110.125062" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.125062" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.110.125062"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21406680" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20map-based%20cloning%20strategy%20employing%20a%20residual%20heterozygous%20line%20reveals%20that%20the%20GIGANTEA%20gene%20is%20involved%20in%20soybean%20maturity%20and%20flowering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Hideshima</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Xia</div> <div class="given-names">Z.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Map-based cloning of the gene associated with the soybean maturity locus <em>E3</em>
</div>, <div class="source">Genetics</div> , <div class="volume">182</div>, <div class="fpage">1251</div>–<div class="lpage">62</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Map-based%20cloning%20of%20the%20gene%20associated%20with%20the%20soybean%20maturity%20locus%20E3&amp;author=S.%20Watanabe&amp;author=R.%20Hideshima&amp;author=Z.%20Xia&amp;publication_year=2009&amp;journal=Genetics&amp;volume=182&amp;pages=1251-62" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.108.098772" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.098772" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.108.098772"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19474204" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Map-based%20cloning%20of%20the%20gene%20associated%20with%20the%20soybean%20maturity%20locus%20E3&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Kanazawa</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Matsumura</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Takahashi</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Harada</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Abe</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Genetic redundancy in soybean photoresponses associated with duplication of the phytochrome A gene</div>, <div class="source">Genetics</div> , <div class="volume">180</div>, <div class="fpage">995</div>–<div class="lpage">1007</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetic%20redundancy%20in%20soybean%20photoresponses%20associated%20with%20duplication%20of%20the%20phytochrome%20A%20gene&amp;author=B.%20Liu&amp;author=A.%20Kanazawa&amp;author=H.%20Matsumura&amp;author=R.%20Takahashi&amp;author=K.%20Harada&amp;author=J.%20Abe&amp;publication_year=2008&amp;journal=Genetics&amp;volume=180&amp;pages=995-1007" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.108.092742" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.092742" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.108.092742"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18780733" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetic%20redundancy%20in%20soybean%20photoresponses%20associated%20with%20duplication%20of%20the%20phytochrome%20A%20gene&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genetic variation in four maturity genes affects photoperiod insensitivity and PHYA-regulated post-flowering responses of soybean</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">13</div>, <div class="fpage">91</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20four%20maturity%20genes%20affects%20photoperiod%20insensitivity%20and%20PHYA-regulated%20post-flowering%20responses%20of%20soybean&amp;author=M.%20Xu&amp;author=Z.%20Xu&amp;author=B.%20Liu&amp;publication_year=2013&amp;journal=BMC%20Plant%20Biol&amp;volume=13&amp;pages=91" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2229-13-91" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-13-91" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2229-13-91"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23799885" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetic%20variation%20in%20four%20maturity%20genes%20affects%20photoperiod%20insensitivity%20and%20PHYA-regulated%20post-flowering%20responses%20of%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tsubokura</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Matsumura</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genetic variation in soybean at the maturity locus <em>E4</em> is involved in adaptation to long days at high latitudes</div>, <div class="source">Agronomy</div> , <div class="volume">3</div>, <div class="fpage">117</div>–<div class="lpage">34</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20soybean%20at%20the%20maturity%20locus%20E4%20is%20involved%20in%20adaptation%20to%20long%20days%20at%20high%20latitudes&amp;author=Y.%20Tsubokura&amp;author=H.%20Matsumura&amp;author=M.%20Xu&amp;publication_year=2013&amp;journal=Agronomy&amp;volume=3&amp;pages=117-34" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3390/agronomy3010117" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fagronomy3010117" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3390%2Fagronomy3010117"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetic%20variation%20in%20soybean%20at%20the%20maturity%20locus%20E4%20is%20involved%20in%20adaptation%20to%20long%20days%20at%20high%20latitudes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kong</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Xia</div> <div class="given-names">Z.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Two coordinately regulated homologs of <em>FLOWERING LOCUS T</em> are involved in the control of photoperiodic flowering in soybean</div>, <div class="source">Plant Physiol</div> ., <div class="volume">154</div>, <div class="fpage">1220</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Two%20coordinately%20regulated%20homologs%20of%20FLOWERING%20LOCUS%20T%20are%20involved%20in%20the%20control%20of%20photoperiodic%20flowering%20in%20soybean&amp;author=F.%20Kong&amp;author=B.%20Liu&amp;author=Z.%20Xia&amp;publication_year=2010&amp;journal=Plant%20Physiol&amp;volume=154&amp;pages=1220-31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.110.160796" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.160796" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.110.160796"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20864544" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Two%20coordinately%20regulated%20homologs%20of%20FLOWERING%20LOCUS%20T%20are%20involved%20in%20the%20control%20of%20photoperiodic%20flowering%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Takeshima</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">A recessive allele for delayed flowering at the soybean maturity locus E9 is a leaky allele of FT2a, a FLOWERING LOCUS T ortholog</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">16</div>, <div class="fpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20recessive%20allele%20for%20delayed%20flowering%20at%20the%20soybean%20maturity%20locus%20E9%20is%20a%20leaky%20allele%20of%20FT2a%2C%20a%20FLOWERING%20LOCUS%20T%20ortholog&amp;author=C.%20Zhao&amp;author=R.%20Takeshima&amp;author=J.%20Zhu&amp;publication_year=2016&amp;journal=BMC%20Plant%20Biol&amp;volume=16&amp;pages=20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12870-016-0704-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-016-0704-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12870-016-0704-9"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26786479" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20recessive%20allele%20for%20delayed%20flowering%20at%20the%20soybean%20maturity%20locus%20E9%20is%20a%20leaky%20allele%20of%20FT2a%2C%20a%20FLOWERING%20LOCUS%20T%20ortholog&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Takeshima</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Hayashi</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">A soybean quantitative trait locus that promotes flowering under long days is identified as FT5a, a FLOWERING LOCUS T ortholog</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">67</div>, <div class="fpage">5247</div>–<div class="lpage">58</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20soybean%20quantitative%20trait%20locus%20that%20promotes%20flowering%20under%20long%20days%20is%20identified%20as%20FT5a%2C%20a%20FLOWERING%20LOCUS%20T%20ortholog&amp;author=R.%20Takeshima&amp;author=T.%20Hayashi&amp;author=J.%20Zhu&amp;publication_year=2016&amp;journal=J.%20Exp.%20Bot&amp;volume=67&amp;pages=5247-58" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/erw283" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferw283" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Ferw283"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27422993" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20soybean%20quantitative%20trait%20locus%20that%20promotes%20flowering%20under%20long%20days%20is%20identified%20as%20FT5a%2C%20a%20FLOWERING%20LOCUS%20T%20ortholog&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation"> Ion AmpliSeq Designer. https://ampliseq.com/help/startDesign.action (25 March 2019, date last accessed) </div>
</div></div></div>
<div content-id="dsz005-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">AmpliSeq protocol. <a class="link link-uri" href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0006735_AmpliSeq_DNA_RNA_LibPrep_UG.pdf" target="_blank">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0006735_AmpliSeq_DNA_RNA_LibPrep_UG.pdf</a>.</div>
</div></div></div>
<div content-id="dsz005-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kaga</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Shimizu</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Evaluation of soybean germplasm conserved in NIAS genebank and development of mini core collections</div>, <div class="source">Breed. Sci</div> ., <div class="volume">61</div>, <div class="fpage">566</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20soybean%20germplasm%20conserved%20in%20NIAS%20genebank%20and%20development%20of%20mini%20core%20collections&amp;author=A.%20Kaga&amp;author=T.%20Shimizu&amp;author=S.%20Watanabe&amp;publication_year=2012&amp;journal=Breed.%20Sci&amp;volume=61&amp;pages=566-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1270/jsbbs.61.566" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.61.566" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1270%2Fjsbbs.61.566"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136496" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluation%20of%20soybean%20germplasm%20conserved%20in%20NIAS%20genebank%20and%20development%20of%20mini%20core%20collections&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chankaew</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Isemura</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Naito</div> <div class="given-names">K.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">QTL mapping for salt tolerance and domestication-related traits in <em>Vigna marina</em> subsp. <em>oblonga</em>, a halophytic species</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">127</div>, <div class="fpage">691</div>–<div class="lpage">702</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20for%20salt%20tolerance%20and%20domestication-related%20traits%20in%20Vigna%20marina%20subsp.%20oblonga%2C%20a%20halophytic%20species&amp;author=S.%20Chankaew&amp;author=T.%20Isemura&amp;author=K.%20Naito&amp;publication_year=2014&amp;journal=Theor.%20Appl.%20Genet&amp;volume=127&amp;pages=691-702" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00122-013-2251-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2251-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00122-013-2251-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24370961" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:QTL%20mapping%20for%20salt%20tolerance%20and%20domestication-related%20traits%20in%20Vigna%20marina%20subsp.%20oblonga%2C%20a%20halophytic%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schmutz</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Cannon</div> <div class="given-names">S.B.</div>
</div>, <div class="name string-name">
<div class="surname">Schlueter</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="volume">463</div>, <div class="fpage">178</div>–<div class="lpage">83</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature08670"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cingolani</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Platts</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">L.L.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3</div>, <div class="source">Fly (Austin)</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.L.%20Wang&amp;publication_year=2012&amp;journal=Fly%20(Austin)&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.4161%2Ffly.19695"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sherry</div> <div class="given-names">S.T.</div>
</div>, <div class="name string-name">
<div class="surname">Ward</div> <div class="given-names">M.H.</div>
</div>, <div class="name string-name">
<div class="surname">Kholodov</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2001</div>, 
<div class="article-title">dbSNP: the NCBI database of genetic variation</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">29</div>, <div class="fpage">308</div>–<div class="lpage">11</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=dbSNP%3A%20the%20NCBI%20database%20of%20genetic%20variation&amp;author=S.T.%20Sherry&amp;author=M.H.%20Ward&amp;author=M.%20Kholodov&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=308-11" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/29.1.308" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.308" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F29.1.308"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125122" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:dbSNP%3A%20the%20NCBI%20database%20of%20genetic%20variation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">R Core Team. <div class="year">2017</div>, <div class="source">R: A Language and Environment for Statistical Computing</div> . 
<div class="publisher-name">R Foundation for Statistical Computing</div>: 
<div class="publisher-loc">Vienna, Austria</div>. <a class="link link-uri" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=R%3A%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&amp;date=2017" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:R%3A%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=R%3A%20A%20Language%20and%20Environment%20for%20Statistical%20Computing">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<em>Troubleshooting Guides</em>. <a class="link link-uri" href="https://www.ampliseq.com/help/troubleshooting.action" target="_blank">https://www.ampliseq.com/help/troubleshooting.action</a>.</div>
</div></div></div>
<div content-id="dsz005-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhou</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Z.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Resequencing 302 wild and cultivated accessions identifies genes related to domestication and improvement in soybean</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">408</div>–<div class="lpage">14</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Resequencing%20302%20wild%20and%20cultivated%20accessions%20identifies%20genes%20related%20to%20domestication%20and%20improvement%20in%20soybean&amp;author=Z.%20Zhou&amp;author=Y.%20Jiang&amp;author=Z.%20Wang&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=408-14" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.3096" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3096" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.3096"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25643055" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Resequencing%20302%20wild%20and%20cultivated%20accessions%20identifies%20genes%20related%20to%20domestication%20and%20improvement%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Fu</div> <div class="given-names">Y.F.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Ectopic expression reveals a conserved PHYB homolog in soybean</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e27737</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ectopic%20expression%20reveals%20a%20conserved%20PHYB%20homolog%20in%20soybean&amp;author=F.%20Wu&amp;author=X.%20Zhang&amp;author=D.%20Li&amp;author=Y.F.%20Fu&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e27737" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0027737" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0027737" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0027737"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110748" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ectopic%20expression%20reveals%20a%20conserved%20PHYB%20homolog%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sun</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Jia</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Cao</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">GmFT2a, a soybean homolog of FLOWERING LOCUS T, is involved in flowering transition and maintenance</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e29238</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GmFT2a%2C%20a%20soybean%20homolog%20of%20FLOWERING%20LOCUS%20T%2C%20is%20involved%20in%20flowering%20transition%20and%20maintenance&amp;author=H.%20Sun&amp;author=Z.%20Jia&amp;author=D.%20Cao&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e29238" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0029238" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029238" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0029238"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22195028" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GmFT2a%2C%20a%20soybean%20homolog%20of%20FLOWERING%20LOCUS%20T%2C%20is%20involved%20in%20flowering%20transition%20and%20maintenance&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Shimizu</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Machita</div> <div class="given-names">K.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Development of a high-density linkage map and chromosome segment substitution lines for Japanese soybean cultivar Enrei</div>, <div class="source">DNA Res</div> ., <div class="volume">25</div>, <div class="fpage">123</div>–<div class="lpage">36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20high-density%20linkage%20map%20and%20chromosome%20segment%20substitution%20lines%20for%20Japanese%20soybean%20cultivar%20Enrei&amp;author=S.%20Watanabe&amp;author=T.%20Shimizu&amp;author=K.%20Machita&amp;publication_year=2018&amp;journal=DNA%20Res&amp;volume=25&amp;pages=123-36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsx043" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsx043" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsx043"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29186379" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Development%20of%20a%20high-density%20linkage%20map%20and%20chromosome%20segment%20substitution%20lines%20for%20Japanese%20soybean%20cultivar%20Enrei&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kojima</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Takahashi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Kobayashi</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2002</div>, 
<div class="article-title">
<em>Hd3a</em>, a rice ortholog of the <em>Arabidopsis FT</em> gene, promotes transition to flowering downstream of Hd1 under short-day conditions</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">43</div>, <div class="fpage">1096</div>–<div class="lpage">105</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hd3a%2C%20a%20rice%20ortholog%20of%20the%20Arabidopsis%20FT%20gene%2C%20promotes%20transition%20to%20flowering%20downstream%20of%20Hd1%20under%20short-day%20conditions&amp;author=S.%20Kojima&amp;author=Y.%20Takahashi&amp;author=Y.%20Kobayashi&amp;publication_year=2002&amp;journal=Plant%20Cell%20Physiol&amp;volume=43&amp;pages=1096-105" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/pcp/pcf156" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcf156" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fpcp%2Fpcf156"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12407188" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hd3a%2C%20a%20rice%20ortholog%20of%20the%20Arabidopsis%20FT%20gene%2C%20promotes%20transition%20to%20flowering%20downstream%20of%20Hd1%20under%20short-day%20conditions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ogiso-Tanaka</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Matsubara</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Yamamoto</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Natural variation of the RICE FLOWERING LOCUS T 1 contributes to flowering time divergence in rice</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e75959</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20of%20the%20RICE%20FLOWERING%20LOCUS%20T%201%20contributes%20to%20flowering%20time%20divergence%20in%20rice&amp;author=E.%20Ogiso-Tanaka&amp;author=K.%20Matsubara&amp;author=S.%20Yamamoto&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e75959" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0075959" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0075959" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0075959"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24098411" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Natural%20variation%20of%20the%20RICE%20FLOWERING%20LOCUS%20T%201%20contributes%20to%20flowering%20time%20divergence%20in%20rice&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Watanabe</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Uchiyama</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">The soybean stem growth habit gene <em>Dt1</em> is an ortholog of <em>Arabidopsis terminal flower1</em>
</div>, <div class="source">Plant Physiol</div> ., <div class="volume">153</div>, <div class="fpage">198</div>–<div class="lpage">210</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20soybean%20stem%20growth%20habit%20gene%20Dt1%20is%20an%20ortholog%20of%20Arabidopsis%20terminal%20flower1&amp;author=B.%20Liu&amp;author=S.%20Watanabe&amp;author=T.%20Uchiyama&amp;publication_year=2010&amp;journal=Plant%20Physiol&amp;volume=153&amp;pages=198-210" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.109.150607" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.150607" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.109.150607"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20219831" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20soybean%20stem%20growth%20habit%20gene%20Dt1%20is%20an%20ortholog%20of%20Arabidopsis%20terminal%20flower1&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Guo</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">A putative CENTRORADIALIS/TERMINAL FLOWER 1-like gene, Ljcen1, plays a role in phase transition in <em>Lotus japonicus</em>
</div>, <div class="source">J. Plant Physiol</div> ., <div class="volume">163</div>, <div class="fpage">436</div>–<div class="lpage">44</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20putative%20CENTRORADIALIS%2FTERMINAL%20FLOWER%201-like%20gene%2C%20Ljcen1%2C%20plays%20a%20role%20in%20phase%20transition%20in%20Lotus%20japonicus&amp;author=X.%20Guo&amp;author=Z.%20Zhao&amp;author=J.%20Chen&amp;publication_year=2006&amp;journal=J.%20Plant%20Physiol&amp;volume=163&amp;pages=436-44" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.jplph.2005.04.037" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2005.04.037" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jplph.2005.04.037"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16455357" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20putative%20CENTRORADIALIS%2FTERMINAL%20FLOWER%201-like%20gene%2C%20Ljcen1%2C%20plays%20a%20role%20in%20phase%20transition%20in%20Lotus%20japonicus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Foucher</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Morin</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Courtiade</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2003</div>, 
<div class="article-title">DETERMINATE and LATE FLOWERING are two TERMINAL FLOWER1/CENTRORADIALIS homologs that control two distinct phases of flowering initiation and development in pea</div>, <div class="source">Plant Cell</div> , <div class="volume">15</div>, <div class="fpage">2742</div>–<div class="lpage">54</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DETERMINATE%20and%20LATE%20FLOWERING%20are%20two%20TERMINAL%20FLOWER1%2FCENTRORADIALIS%20homologs%20that%20control%20two%20distinct%20phases%20of%20flowering%20initiation%20and%20development%20in%20pea&amp;author=F.%20Foucher&amp;author=J.%20Morin&amp;author=J.%20Courtiade&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=2742-54" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.015701" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.015701" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.015701"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14563931" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DETERMINATE%20and%20LATE%20FLOWERING%20are%20two%20TERMINAL%20FLOWER1%2FCENTRORADIALIS%20homologs%20that%20control%20two%20distinct%20phases%20of%20flowering%20initiation%20and%20development%20in%20pea&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bradley</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Ratcliffe</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Vincent</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Carpenter</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Coen</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">1997</div>, 
<div class="article-title">Inflorescence commitment and architecture in <em>Arabidopsis</em>
</div>, <div class="source">Science</div> , <div class="volume">275</div>, <div class="fpage">80</div>–<div class="lpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Inflorescence%20commitment%20and%20architecture%20in%20Arabidopsis&amp;author=D.%20Bradley&amp;author=O.%20Ratcliffe&amp;author=C.%20Vincent&amp;author=R.%20Carpenter&amp;author=E.%20Coen&amp;publication_year=1997&amp;journal=Science&amp;volume=275&amp;pages=80-3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.275.5296.80" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.275.5296.80" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.275.5296.80"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8974397" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inflorescence%20commitment%20and%20architecture%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mimida</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Goto</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Kobayashi</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2001</div>, 
<div class="article-title">Functional divergence of the TFL1-like gene family in <em>Arabidopsis</em> revealed by characterization of a novel homologue</div>, <div class="source">Genes Cells</div> , <div class="volume">6</div>, <div class="fpage">327</div>–<div class="lpage">36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Functional%20divergence%20of%20the%20TFL1-like%20gene%20family%20in%20Arabidopsis%20revealed%20by%20characterization%20of%20a%20novel%20homologue&amp;author=N.%20Mimida&amp;author=K.%20Goto&amp;author=Y.%20Kobayashi&amp;publication_year=2001&amp;journal=Genes%20Cells&amp;volume=6&amp;pages=327-36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1365-2443.2001.00425.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2443.2001.00425.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1365-2443.2001.00425.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11318875" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Functional%20divergence%20of%20the%20TFL1-like%20gene%20family%20in%20Arabidopsis%20revealed%20by%20characterization%20of%20a%20novel%20homologue&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bradley</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Carpenter</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Copsey</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Vincent</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Rothstein</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Coen</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">1996</div>, 
<div class="article-title">Control of inflorescence architecture in <em>Antirrhinum</em>
</div>, <div class="source">Nature</div> , <div class="volume">379</div>, <div class="fpage">791</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Control%20of%20inflorescence%20architecture%20in%20Antirrhinum&amp;author=D.%20Bradley&amp;author=R.%20Carpenter&amp;author=L.%20Copsey&amp;author=C.%20Vincent&amp;author=S.%20Rothstein&amp;author=E.%20Coen&amp;publication_year=1996&amp;journal=Nature&amp;volume=379&amp;pages=791-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/379791a0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F379791a0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F379791a0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8587601" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Control%20of%20inflorescence%20architecture%20in%20Antirrhinum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Miao</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Ultra-dense SNP genetic map construction and identification of SiDt gene controlling the determinate growth habit in <em>Sesamum indicum</em> L</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">31556</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ultra-dense%20SNP%20genetic%20map%20construction%20and%20identification%20of%20SiDt%20gene%20controlling%20the%20determinate%20growth%20habit%20in%20Sesamum%20indicum%20L&amp;author=H.%20Zhang&amp;author=H.%20Miao&amp;author=C.%20Li&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=31556" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep31556" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep31556" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep31556"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27527492" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ultra-dense%20SNP%20genetic%20map%20construction%20and%20identification%20of%20SiDt%20gene%20controlling%20the%20determinate%20growth%20habit%20in%20Sesamum%20indicum%20L&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nakamichi</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Kita</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Niinuma</div> <div class="given-names">K.</div>
</div>
</div>, et al.  <div class="year">2007</div>, 
<div class="article-title">
<em>Arabidopsis</em> clock-associated pseudo-response regulators PRR9, PRR7 and PRR5 coordinately and positively regulate flowering time through the canonical CONSTANS-dependent photoperiodic pathway</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">48</div>, <div class="fpage">822</div>–<div class="lpage">32</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20clock-associated%20pseudo-response%20regulators%20PRR9%2C%20PRR7%20and%20PRR5%20coordinately%20and%20positively%20regulate%20flowering%20time%20through%20the%20canonical%20CONSTANS-dependent%20photoperiodic%20pathway&amp;author=N.%20Nakamichi&amp;author=M.%20Kita&amp;author=K.%20Niinuma&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol&amp;volume=48&amp;pages=822-32" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/pcp/pcm056" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm056" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fpcp%2Fpcm056"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17504813" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Arabidopsis%20clock-associated%20pseudo-response%20regulators%20PRR9%2C%20PRR7%20and%20PRR5%20coordinately%20and%20positively%20regulate%20flowering%20time%20through%20the%20canonical%20CONSTANS-dependent%20photoperiodic%20pathway&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kwon</div> <div class="given-names">C.T.</div>
</div>, <div class="name string-name">
<div class="surname">Koo</div> <div class="given-names">B.H.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Yoo</div> <div class="given-names">S.C.</div>
</div>, <div class="name string-name">
<div class="surname">Paek</div> <div class="given-names">N.C.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Casein kinases I and 2α phosphorylate <em>Oryza sativa</em> pseudo-response regulator 37 (OsPRR37) in photoperiodic flowering in rice</div>, <div class="source">Mol. Cells</div> , <div class="volume">38</div>, <div class="fpage">81</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Casein%20kinases%20I%20and%202%CE%B1%20phosphorylate%20Oryza%20sativa%20pseudo-response%20regulator%2037%20(OsPRR37)%20in%20photoperiodic%20flowering%20in%20rice&amp;author=C.T.%20Kwon&amp;author=B.H.%20Koo&amp;author=D.%20Kim&amp;author=S.C.%20Yoo&amp;author=N.C.%20Paek&amp;publication_year=2015&amp;journal=Mol.%20Cells&amp;volume=38&amp;pages=81-8" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25431424" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Casein%20kinases%20I%20and%202%CE%B1%20phosphorylate%20Oryza%20sativa%20pseudo-response%20regulator%2037%20(OsPRR37)%20in%20photoperiodic%20flowering%20in%20rice&amp;aulast=Kwon&amp;date=2015&amp;spage=81&amp;epage=8&amp;volume=38" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Casein%20kinases%20I%20and%202%CE%B1%20phosphorylate%20Oryza%20sativa%20pseudo-response%20regulator%2037%20(OsPRR37)%20in%20photoperiodic%20flowering%20in%20rice&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Turner</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Beales</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Faure</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Dunford</div> <div class="given-names">R.P.</div>
</div>, <div class="name string-name">
<div class="surname">Laurie</div> <div class="given-names">D.A.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">The pseudo-response regulator Ppd-H1 provides adaptation to photoperiod in 799 barley</div>, <div class="source">Science</div> , <div class="volume">310</div>, <div class="fpage">1031</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20pseudo-response%20regulator%20Ppd-H1%20provides%20adaptation%20to%20photoperiod%20in%20799%20barley&amp;author=A.%20Turner&amp;author=J.%20Beales&amp;author=S.%20Faure&amp;author=R.P.%20Dunford&amp;author=D.A.%20Laurie&amp;publication_year=2005&amp;journal=Science&amp;volume=310&amp;pages=1031-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1117619" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1117619" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1117619"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16284181" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20pseudo-response%20regulator%20Ppd-H1%20provides%20adaptation%20to%20photoperiod%20in%20799%20barley&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">S.J.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.I.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Identification of additional QTLs for flowering time by removing the effect of the maturity gene <em>E1</em> in soybean</div>, <div class="source">J. Integr. Agric</div> ., <div class="volume">15</div>, <div class="fpage">42</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20additional%20QTLs%20for%20flowering%20time%20by%20removing%20the%20effect%20of%20the%20maturity%20gene%20E1%20in%20soybean&amp;author=S.J.%20Lu&amp;author=Y.%20Li&amp;author=J.I.%20Wang&amp;publication_year=2016&amp;journal=J.%20Integr.%20Agric&amp;volume=15&amp;pages=42-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S2095-3119(15)61046-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS2095-3119(15)61046-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS2095-3119(15)61046-2"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20additional%20QTLs%20for%20flowering%20time%20by%20removing%20the%20effect%20of%20the%20maturity%20gene%20E1%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Koo</div> <div class="given-names">S.C.</div>
</div>, <div class="name string-name">
<div class="surname">Bracko</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Park</div> <div class="given-names">M.S.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Control of lateral organ development and flowering time by the <em>Arabidopsis thaliana</em> MADS-box gene AGAMOUS-LIKE6</div>, <div class="source">Plant J</div> ., <div class="volume">62</div>, <div class="fpage">807</div>–<div class="lpage">16</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Control%20of%20lateral%20organ%20development%20and%20flowering%20time%20by%20the%20Arabidopsis%20thaliana%20MADS-box%20gene%20AGAMOUS-LIKE6&amp;author=S.C.%20Koo&amp;author=O.%20Bracko&amp;author=M.S.%20Park&amp;publication_year=2010&amp;journal=Plant%20J&amp;volume=62&amp;pages=807-16" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/tpj.2010.62.issue-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.62.issue-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ftpj.2010.62.issue-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20230491" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Control%20of%20lateral%20organ%20development%20and%20flowering%20time%20by%20the%20Arabidopsis%20thaliana%20MADS-box%20gene%20AGAMOUS-LIKE6&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yoo</div> <div class="given-names">S.K.</div>
</div>, <div class="name string-name">
<div class="surname">Wu</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">J.S.</div>
</div>, <div class="name string-name">
<div class="surname">Ahn</div> <div class="given-names">J.H.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">AGAMOUS-LIKE 6 is a floral promoter that negatively regulates the FLC/MAF clade genes and positively regulates FT in <em>Arabidopsis</em>
</div>, <div class="source">Plant J</div> ., <div class="volume">65</div>, <div class="fpage">62</div>–<div class="lpage">76</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=AGAMOUS-LIKE%206%20is%20a%20floral%20promoter%20that%20negatively%20regulates%20the%20FLC%2FMAF%20clade%20genes%20and%20positively%20regulates%20FT%20in%20Arabidopsis&amp;author=S.K.%20Yoo&amp;author=X.%20Wu&amp;author=J.S.%20Lee&amp;author=J.H.%20Ahn&amp;publication_year=2011&amp;journal=Plant%20J&amp;volume=65&amp;pages=62-76" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1365-313X.2010.04402.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2010.04402.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1365-313X.2010.04402.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21175890" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:AGAMOUS-LIKE%206%20is%20a%20floral%20promoter%20that%20negatively%20regulates%20the%20FLC%2FMAF%20clade%20genes%20and%20positively%20regulates%20FT%20in%20Arabidopsis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">van Nocker</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Ludwig</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">The WD-repeat protein superfamily in <em>Arabidopsis</em>: conservation and divergence in structure and function</div>, <div class="source">BMC Genomics</div> , <div class="volume">4</div>, <div class="fpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20WD-repeat%20protein%20superfamily%20in%20Arabidopsis%3A%20conservation%20and%20divergence%20in%20structure%20and%20function&amp;author=S.%20van%20Nocker&amp;author=P.%20Ludwig&amp;publication_year=2003&amp;journal=BMC%20Genomics&amp;volume=4&amp;pages=50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-4-50" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-4-50" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-4-50"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14672542" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20WD-repeat%20protein%20superfamily%20in%20Arabidopsis%3A%20conservation%20and%20divergence%20in%20structure%20and%20function&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhai</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Lü</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Allelic variations at four major maturity <em>E</em> genes and transcriptional abundance of the <em>E1</em> gene are associated with flowering time and maturity of soybean cultivars</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e97636</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Allelic%20variations%20at%20four%20major%20maturity%20E%20genes%20and%20transcriptional%20abundance%20of%20the%20E1%20gene%20are%20associated%20with%20flowering%20time%20and%20maturity%20of%20soybean%20cultivars&amp;author=H.%20Zhai&amp;author=S.%20L%C3%BC&amp;author=Y.%20Wang&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e97636" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0097636" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0097636" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0097636"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24830458" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Allelic%20variations%20at%20four%20major%20maturity%20E%20genes%20and%20transcriptional%20abundance%20of%20the%20E1%20gene%20are%20associated%20with%20flowering%20time%20and%20maturity%20of%20soybean%20cultivars&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">J.P.</div>
</div>, <div class="name string-name">
<div class="surname">Song</div> <div class="given-names">Q.J.</div>
</div>, <div class="name string-name">
<div class="surname">Cregan</div> <div class="given-names">P.B.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Genome-wide association study for flowering time, maturity dates and plant height in early maturing soybean (<em>Glycine max</em>) germplasm</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">217</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20for%20flowering%20time%2C%20maturity%20dates%20and%20plant%20height%20in%20early%20maturing%20soybean%20(Glycine%20max)%20germplasm&amp;author=J.P.%20Zhang&amp;author=Q.J.%20Song&amp;author=P.B.%20Cregan&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=217" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-015-1441-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1441-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-015-1441-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25887991" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20association%20study%20for%20flowering%20time%2C%20maturity%20dates%20and%20plant%20height%20in%20early%20maturing%20soybean%20(Glycine%20max)%20germplasm&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Bernard</div> <div class="given-names">R.</div>
</div></div> <div class="year">1972</div>, 
<div class="article-title">Two genes affecting stem termination in soybeans</div>, <div class="source">Crop Sci</div> ., <div class="volume">12</div>, <div class="fpage">235</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Two%20genes%20affecting%20stem%20termination%20in%20soybeans&amp;author=R.%20Bernard&amp;publication_year=1972&amp;journal=Crop%20Sci&amp;volume=12&amp;pages=235-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2135/cropsci1972.0011183X001200020028x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1972.0011183X001200020028x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2135%2Fcropsci1972.0011183X001200020028x"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Two%20genes%20affecting%20stem%20termination%20in%20soybeans&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tian</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">Artificial selection for determinate growth habit in soybean</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">8563</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Artificial%20selection%20for%20determinate%20growth%20habit%20in%20soybean&amp;author=Z.%20Tian&amp;author=X.%20Wang&amp;author=R.%20Lee&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=8563-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1000088107" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1000088107" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1000088107"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Artificial%20selection%20for%20determinate%20growth%20habit%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fan</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Hu</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Conserved CO-FT regulons contribute to the photoperiod flowering control in soybean</div>, <div class="source">BMC Plant Biol</div> ., <div class="volume">14</div>, <div class="fpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Conserved%20CO-FT%20regulons%20contribute%20to%20the%20photoperiod%20flowering%20control%20in%20soybean&amp;author=C.%20Fan&amp;author=R.%20Hu&amp;author=X.%20Zhang&amp;publication_year=2014&amp;journal=BMC%20Plant%20Biol&amp;volume=14&amp;pages=9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2229-14-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-14-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2229-14-9"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24397545" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Conserved%20CO-FT%20regulons%20contribute%20to%20the%20photoperiod%20flowering%20control%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz005-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz005-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cao</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Takeshima</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Molecular mechanisms of flowering under long days and stem growth habit in soybean</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">68</div>, <div class="fpage">1873</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20flowering%20under%20long%20days%20and%20stem%20growth%20habit%20in%20soybean&amp;author=D.%20Cao&amp;author=R.%20Takeshima&amp;author=C.%20Zhao&amp;publication_year=2017&amp;journal=J.%20Exp.%20Bot&amp;volume=68&amp;pages=1873-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/erx382" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferx382" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Ferx382"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28338712" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20mechanisms%20of%20flowering%20under%20long%20days%20and%20stem%20growth%20habit%20in%20soybean&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5474979&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsz005/1/dsz005_supplementary_data.zip?Expires=1576706115&amp;Signature=xKjr5RPahtYaBmgz2yWrU4PlaryxqRgHmw8CvFNQlPDNFda~ICMyxVUx8ID-hiGNB9D9kk28gz~K29u-PLa2H0oYGnJQIY-mobMTLPTv-lxKqAf3m35odzF2eK2byTpAXgbLC8yNUOaVZcxqMDqC9oqsgLO~YDLDdgLYtfRaaSbtWq6xOgTeXNFDzleOVisY4~SZzHACFJn3tb9jG4GUwgp0llDWerFVf8YyK14ejeE-SvLQcrHGl0jYFPwlBL2jEqtcT6h7qHhOQ6IPUIHLlgl6-xqMZtcAtAOoquZeCBH6vDN1W4Szl5bkKuH4PY~RTG9ukKKy4vKErMowiAniHg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz005_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,059</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz005"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5474979"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493010500005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f3%2f243%2f5474979">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493010500005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F3%2F243%2F5474979" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5474979">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Highly multiplexed AmpliSeq technology identifies novel variation of flowering time-related genes in soybean (Glycine max) | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz005");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f243%2f5474979">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
