<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>De novo whole-genome assembly in Chrysanthemum seticuspe, a model species of Chrysanthemums, and its application to genetic and gene discovery analysis | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/3/195/5303208","@type":"ScholarlyArticle","name":" De novo  whole-genome assembly in  Chrysanthemum seticuspe , a model species of Chrysanthemums, and its application to genetic and gene discovery analysis","datePublished":"2019-01-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2019-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy048","keywords":["<italic>Chrysanthemum seticuspe</italic>","genome sequence assembly","linkage map","flowering genes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/3/195/5303208","author":[{"name":"Hirakawa, Hideki","affiliation":"Kazusa DNA Research Institute, Kisarazu, Chiba, Japan","@type":"Person"},{"name":"Sumitomo, Katsuhiko","affiliation":"Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Hisamatsu, Tamotsu","affiliation":"Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Nagano, Soichiro","affiliation":"Kazusa DNA Research Institute, Kisarazu, Chiba, Japan  Forest Tree Breeding Center, Forestry and Forest Products Research Institute, Juo, Hitachi, Ibaraki, Japan","@type":"Person"},{"name":"Shirasawa, Kenta","affiliation":"Kazusa DNA Research Institute, Kisarazu, Chiba, Japan","@type":"Person","sameAs":"http://orcid.org/0000-0001-7880-6221"},{"name":"Higuchi, Yohei","affiliation":"Graduate School of Agricultural and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo, Japan","@type":"Person"},{"name":"Kusaba, Makoto","affiliation":"Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan","@type":"Person"},{"name":"Koshioka, Masaji","affiliation":"College of Bioresource Sciences, Nihon University, Kameino, Fujisawa, Kanagawa, Japan","@type":"Person"},{"name":"Nakano, Yoshihiro","affiliation":"Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Yagi, Masafumi","affiliation":"Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Yamaguchi, Hiroyasu","affiliation":"Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan","@type":"Person"},{"name":"Taniguchi, Kenji","affiliation":"Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan","@type":"Person"},{"name":"Nakano, Michiharu","affiliation":"Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan","@type":"Person"},{"name":"Isobe, Sachiko N","affiliation":"Kazusa DNA Research Institute, Kisarazu, Chiba, Japan","@type":"Person"}],"description":"Abstract.  Cultivated chrysanthemum (Chrysanthemum morifolium Ramat.) is one of the most economically important ornamental crops grown worldwide. It has a compl","pageStart":"195","pageEnd":"203","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636238294&Signature=5CZUcLr2kzv-g~hTtTSZRiKnbGJYeMsGiunUcTgIVTs5~5bz6erocQsEqgtMI2Hb-4~23aq4LEgMK7eeWV4yWnvOzG~mevgG5x3vOCrcoIDMjCfyCxNepuEffTfgRAEAkZGeFwnIitZ~T~hFWjxHvjUj~Ix7b~LNP~rAnEQELOBtSoysP8~g4JCpYrLVnGehs2jmJ8YKqdyDuigcA7ZLwnJE9vJNMnpEmPMJzbHqV0GkLj3CBpBorfLx6ILJqmrCR5fiLBNNjEe6CSZL1EwWMaBIxLge1OtnwCQ0uGD63-itOgL-wcDJcneW7Z0QtLdFrHhVhd6s67oGM1vIzLuEoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" De novo  whole-genome assembly in  Chrysanthemum seticuspe , a model species of Chrysanthemums, and its application to genetic and gene discovery analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636238294&Signature=5CZUcLr2kzv-g~hTtTSZRiKnbGJYeMsGiunUcTgIVTs5~5bz6erocQsEqgtMI2Hb-4~23aq4LEgMK7eeWV4yWnvOzG~mevgG5x3vOCrcoIDMjCfyCxNepuEffTfgRAEAkZGeFwnIitZ~T~hFWjxHvjUj~Ix7b~LNP~rAnEQELOBtSoysP8~g4JCpYrLVnGehs2jmJ8YKqdyDuigcA7ZLwnJE9vJNMnpEmPMJzbHqV0GkLj3CBpBorfLx6ILJqmrCR5fiLBNNjEe6CSZL1EwWMaBIxLge1OtnwCQ0uGD63-itOgL-wcDJcneW7Z0QtLdFrHhVhd6s67oGM1vIzLuEoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content=" De novo  whole-genome assembly in  Chrysanthemum seticuspe , a model species of Chrysanthemums, and its application to genetic and gene discovery analysis">
<meta property="og:description" content="Abstract.  Cultivated chrysanthemum (Chrysanthemum morifolium Ramat.) is one of the most economically important ornamental crops grown worldwide. It has a compl">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/3/195/5303208">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1636238294&amp;Signature=5CZUcLr2kzv-g~hTtTSZRiKnbGJYeMsGiunUcTgIVTs5~5bz6erocQsEqgtMI2Hb-4~23aq4LEgMK7eeWV4yWnvOzG~mevgG5x3vOCrcoIDMjCfyCxNepuEffTfgRAEAkZGeFwnIitZ~T~hFWjxHvjUj~Ix7b~LNP~rAnEQELOBtSoysP8~g4JCpYrLVnGehs2jmJ8YKqdyDuigcA7ZLwnJE9vJNMnpEmPMJzbHqV0GkLj3CBpBorfLx6ILJqmrCR5fiLBNNjEe6CSZL1EwWMaBIxLge1OtnwCQ0uGD63-itOgL-wcDJcneW7Z0QtLdFrHhVhd6s67oGM1vIzLuEoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-01-27">
<meta name="citation_author" content="Hirakawa, Hideki">
<meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan">
<meta name="citation_author" content="Sumitomo, Katsuhiko">
<meta name="citation_author_institution" content="Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Hisamatsu, Tamotsu">
<meta name="citation_author_institution" content="Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Nagano, Soichiro">
<meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan">
<meta name="citation_author_institution" content="Forest Tree Breeding Center, Forestry and Forest Products Research Institute, Juo, Hitachi, Ibaraki, Japan">
<meta name="citation_author" content="Shirasawa, Kenta">
<meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan">
<meta name="citation_author" content="Higuchi, Yohei">
<meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo, Japan">
<meta name="citation_author" content="Kusaba, Makoto">
<meta name="citation_author_institution" content="Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan">
<meta name="citation_author" content="Koshioka, Masaji">
<meta name="citation_author_institution" content="College of Bioresource Sciences, Nihon University, Kameino, Fujisawa, Kanagawa, Japan">
<meta name="citation_author" content="Nakano, Yoshihiro">
<meta name="citation_author_institution" content="Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Yagi, Masafumi">
<meta name="citation_author_institution" content="Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Yamaguchi, Hiroyasu">
<meta name="citation_author_institution" content="Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Taniguchi, Kenji">
<meta name="citation_author_institution" content="Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan">
<meta name="citation_author" content="Nakano, Michiharu">
<meta name="citation_author_institution" content="Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan">
<meta name="citation_author" content="Isobe, Sachiko N">
<meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan">
<meta name="citation_title" content="De novo whole-genome assembly in Chrysanthemum seticuspe, a model species of Chrysanthemums, and its application to genetic and gene discovery analysis">
<meta name="citation_firstpage" content="195">
<meta name="citation_lastpage" content="203">
<meta name="citation_doi" content="10.1093/dnares/dsy048">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="3">
<meta name="citation_publication_date" content="2019/06/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Effect of the relative length of day and night and other factors of the environment on growth and reproduction in plants;  citation_journal_title=J. Agric. Res;  citation_year=1920;  citation_volume=18;  citation_pages=553-606; ">
<meta name="citation_reference" content="citation_title=New facts in support of the hormonal theory of plant development, Dokl;  citation_journal_title=Akad. Nauk. SSSR;  citation_year=1936;  citation_volume=13;  citation_pages=79-83; ">
<meta name="citation_reference" content="citation_title=Studies on the regulation of Chrysanthemum flowering with special reference to plant regulators, I. The inhibiting action of non-induced leaves on floral stimulus;  citation_journal_title=J. Japan. Soc. Hort. Sci;  citation_year=1967;  citation_volume=36;  citation_pages=77-85; ">
<meta name="citation_reference" content="citation_title=Move on up, it’s time for change—mobile signals controlling photoperiod-dependent flowering;  citation_journal_title=Genes Dev;  citation_year=2007;  citation_volume=21;  citation_pages=2371-84; ">
<meta name="citation_reference" content="citation_title=Leaf-produced floral signals;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2008;  citation_volume=11;  citation_pages=541-7; ">
<meta name="citation_reference" content="citation_title=Long-distance regulation of flowering time;  citation_journal_title=J. Exp. Bot;  citation_year=2011;  citation_volume=62;  citation_pages=4399-413; ">
<meta name="citation_reference" content="citation_title=Manipulating plant architecture with members of the CETS gene family;  citation_journal_title=Plant Sci;  citation_year=2012;  citation_volume=188–189;  citation_pages=71-81; ">
<meta name="citation_reference" content="citation_title=CsFTL3, a chrysanthemum FLOWERING LOCUS T-like gene, is a key regulator of photoperiodic flowering in chrysanthemums;  citation_journal_title=J. Exp. Bot;  citation_year=2012;  citation_volume=63;  citation_pages=1461-77; ">
<meta name="citation_reference" content="citation_title=The gated induction of a systemic floral inhibitor, antiflorigen, determines obligate short-day flowering in chrysanthemums;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=17137-42; ">
<meta name="citation_reference" content="citation_title=Conclusive evidence for hexasomic inheritance in chrysanthemum based on analysis of a 183 k SNP array;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=585">
<meta name="citation_reference" content="citation_title=An ultra-dense integrated linkage map for hexaploid chrysanthemum enables multi-allelic QTL analysis;  citation_journal_title=Theor. Appl. Genet;  citation_year=2017;  citation_volume=130;  citation_pages=2527-41; ">
<meta name="citation_reference" content="citation_title=Genetic studies of self-incompatibility in the garden chrysanthemum, Chrysanthemum morifolium Ramat;  citation_journal_title=Theor. Appl. Genet;  citation_year=1973;  citation_volume=43;  citation_pages=1-5; ">
<meta name="citation_reference" content=" citation_title=J. Japan. Bot;  citation_year=2004;  citation_volume=79;  citation_pages=186-95; ">
<meta name="citation_reference" content="citation_title=Application of RAPD analysis in the study on the origin of Chinese cultivated chrysanthemum;  citation_journal_title=Acta Bot. Sin;  citation_year=1998;  citation_volume=40;  citation_pages=1053-9; ">
<meta name="citation_reference" content="citation_title=Molecular evidence for multiple polyploidization and lineage recombination in the Chrysanthemum indicum polyploid complex (Asteraceae);  citation_journal_title=New Phytol;  citation_year=2006;  citation_volume=171;  citation_pages=875-86; ">
<meta name="citation_reference" content="citation_title=Sequence analysis of the genome of carnation (Dianthus caryophyllus L.);  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=231-41; ">
<meta name="citation_reference" content="citation_title=The Sol Genomics Network (solgenomics.net): growing tomatoes using Perl;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=D1149-55; ">
<meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray);  citation_journal_title=Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; ">
<meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; ">
<meta name="citation_reference" content="citation_title=Genome sequence and analysis of the tuber crop potato;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=189-95; ">
<meta name="citation_reference" content="citation_title=Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=145-53; ">
<meta name="citation_reference" content="citation_title=Construction of an integrated high density simple sequence repeat linkage map in cultivated strawberry (Fragaria × ananassa) and its applicability;  citation_journal_title=DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=79-92; ">
<meta name="citation_reference" content="citation_title=Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce;  citation_journal_title=Nat. Commun;  citation_year=2017;  citation_volume=8;  citation_pages=14953">
<meta name="citation_reference" content="citation_title=The sunflower genome provides insights into oil metabolism, flowering and Asterid evolution;  citation_journal_title=Nature;  citation_year=2017;  citation_volume=546;  citation_pages=148-52; ">
<meta name="citation_reference" content="citation_title=Day light quality affects the night-break response in the short-day plant chrysanthemum, suggesting differential phytochrome-mediated regulation of flowering;  citation_journal_title=J. Plant Physiol;  citation_year=2012;  citation_volume=169;  citation_pages=1789-96; ">
<meta name="citation_reference" content="citation_title=Flowering retardation by high temperature in chrysanthemums: involvement of FLOWERING LOCUS T-like 3 gene repression;  citation_journal_title=J. Exp. Bot;  citation_year=2013;  citation_volume=64;  citation_pages=909-20; ">
<meta name="citation_reference" content="citation_title=CsTFL1, a constitutive local repressor of flowering, modulates floral initiation by antagonising florigen complex activity in chrysanthemum;  citation_journal_title=Plant Sci;  citation_year=2015;  citation_volume=237;  citation_pages=1-7; ">
<meta name="citation_reference" content="citation_title=Structural features determining flower-promoting activity of Arabidopsis FLOWERING LOCUS T;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=552-64; ">
<meta name="citation_reference" content="citation_title=Seasonal variability in dormancy and flowering competence in Chrysanthemum: chilling impacts on shoot extension growth and flowering capacity;  citation_journal_title=JARQ;  citation_year=2017;  citation_volume=51;  citation_pages=343-50; ">
<meta name="citation_reference" content="citation_title=The value of sampling anomalous taxa in phylogenetic studies: major clades of the Asteraceae revealed;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2008;  citation_volume=47;  citation_pages=757-82; ">
<meta name="citation_reference" content="citation_title=A high-quality genome sequence of Rosa chinensis to elucidate ornamental traits;  citation_journal_title=Nat. Plants;  citation_year=2018;  citation_volume=4;  citation_pages=473-84; ">
<meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; ">
<meta name="citation_reference" content="citation_title=A high-density SNP genetic map consisting of a complete set of homologous groups in autohexaploid sweetpotato (Ipomoea batatas);  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=44207">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/3/195/28855561/dsy048.pdf">
<meta name="description" content="Abstract.  Cultivated chrysanthemum (Chrysanthemum morifolium Ramat.) is one of the most economically important ornamental crops grown worldwide. It has a compl">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/3/195/5303208">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/3/195/5303208">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/3/195/5303208");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy048';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f195%2f5303208" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f195%2f5303208" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115841"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/3/1/m_dnares_26_3cover.png?Expires=1576706106&amp;Signature=fgMnyvVYW-N0hiRIUrbAiFxkaF5udsL5J6YtQeQB3EuivI38Vwd4-T8v6cweN~2RZPIskmCsYI9YfLbYWoMEqAqW20kj1Bww4Z5k1TQApn6jsedqUpYQxmH4lEfokLp5ukgsBB8QCx1Pwpr-ZJbWgn-6uPaCn6WMBxV8lSF2CD2J3bOeGiiHFzTB8VisJ4VJlloJMdf~mSk8wudKGZ129q9Vu4xrVcY0g9zFFt0Q-8IQRu6xLJxi2bfxMR0wKCPzjYO-IKNIWkx7Kl0QswaF3VwdZaged2vkiIHAuHFPkimCYXo091ahTGbPp5gCdaidjGS5OIY-s0SPT1ct7dxeVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/3">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="137025361">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025361">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025363">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025363">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025367">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025367">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025386">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025386">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025417">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025417">4. Conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025419">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025419">5. Data availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="137025421">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025421">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="137025425">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#137025425">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="empty arrow"> </li>
            <li class="next arrow">
            <a href="/dnaresearch/article/26/3/205/5320315">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    <em>De novo</em> whole-genome assembly in <em>Chrysanthemum seticuspe</em>, a model species of Chrysanthemums, and its application to genetic and gene discovery analysis
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Kazusa DNA Research Institute, Kisarazu, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hirakawa%20Hideki%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuhiko Sumitomo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuhiko+Sumitomo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sumitomo%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sumitomo%20Katsuhiko%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Katsuhiko Sumitomo</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tamotsu Hisamatsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tamotsu+Hisamatsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hisamatsu%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Hisamatsu%20Tamotsu%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Tamotsu Hisamatsu</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Soichiro Nagano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Kazusa DNA Research Institute, Kisarazu, Chiba, Japan</div>
<div class="aff">Forest Tree Breeding Center, Forestry and Forest Products Research Institute, Juo, Hitachi, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Soichiro+Nagano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagano%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nagano%20Soichiro%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Soichiro Nagano</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Kazusa DNA Research Institute, Kisarazu, Chiba, Japan</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0001-7880-6221">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0001-7880-6221
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shirasawa%20Kenta%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Kenta Shirasawa</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yohei Higuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Agricultural and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yohei+Higuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Higuchi%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Higuchi%20Yohei%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Yohei Higuchi</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Kusaba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Kusaba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kusaba%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kusaba%20Makoto%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Makoto Kusaba</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaji Koshioka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">College of Bioresource Sciences, Nihon University, Kameino, Fujisawa, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaji+Koshioka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koshioka%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Koshioka%20Masaji%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Masaji Koshioka</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Nakano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Nakano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakano%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nakano%20Yoshihiro%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Yoshihiro Nakano</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masafumi Yagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masafumi+Yagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yagi%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yagi%20Masafumi%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Masafumi Yagi</a><span class="delimiter-hidden">, </span>
                                </span>
                                    <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                        <span>...</span> Show more
                                    </a>
                                    <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyasu Yamaguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Vegetable and Floriculture Sciences, NARO, Tsukuba, Ibaraki, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyasu+Yamaguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamaguchi%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yamaguchi%20Hiroyasu%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Hiroyasu Yamaguchi</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Taniguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Taniguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Taniguchi%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Taniguchi%20Kenji%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Kenji Taniguchi</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michiharu Nakano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Science, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michiharu+Nakano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakano%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nakano%20Michiharu%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Michiharu Nakano</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko N Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Kazusa DNA Research Institute, Kisarazu, Chiba, Japan</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 438 52 3928. Fax. +81 438 52 3934. Email: <a href="mailto:sisobe@kazusa.or.jp" target="_blank">sisobe@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+N+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Isobe%20Sachiko%20N%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Sachiko N Isobe</a><span class="delimiter"></span>
                                        </span>
                                    </div>
                            </div>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy048-FM2"><div class="fn"><div class="footnote-content">Hideki Hirakawa and Katsuhiko Sumitomo authors contributed equally to this work.</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 195–203, <a href="https://doi.org/10.1093/dnares/dsy048">https://doi.org/10.1093/dnares/dsy048</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">27 January 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">23 July 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">02 January 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">27 January 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5303208" href="/dnaresearch/article-pdf/26/3/195/28855561/dsy048.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideki Hirakawa, Katsuhiko Sumitomo, Tamotsu Hisamatsu, Soichiro Nagano, Kenta Shirasawa, Yohei Higuchi, Makoto Kusaba, Masaji Koshioka, Yoshihiro Nakano, Masafumi Yagi, Hiroyasu Yamaguchi, Kenji Taniguchi, Michiharu Nakano, Sachiko N Isobe,  <em>De novo</em> whole-genome assembly in <em>Chrysanthemum seticuspe</em>, a model species of Chrysanthemums, and its application to genetic and gene discovery analysis, <em>DNA Research</em>, Volume 26, Issue 3, June 2019, Pages 195–203, <a href="https://doi.org/10.1093/dnares/dsy048">https://doi.org/10.1093/dnares/dsy048</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5303208&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BDe%20novo%26lt%3B%2Fem%26gt%3B%20whole-genome%20assembly%20in%20%26lt%3Bem%26gt%3BChrysanthemum%20seticuspe%26lt%3B%2Fem%26gt%3B%2C%20a%20model%20species%20of%20Chrysanthemums%2C%20and%20its%20application%20to%20genetic%20and%20gene%20discovery%20analysis&amp;publicationDate=2019-01-27&amp;volumeNum=26&amp;issueNum=3&amp;author=Hirakawa%2C%20Hideki%3B%20Sumitomo%2C%20Katsuhiko&amp;startPage=195&amp;endPage=203&amp;contentId=10.1093%2Fdnares%2Fdsy048&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f195%2f5303208" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="137025361" id="137025361" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Cultivated chrysanthemum (<em>Chrysanthemum morifolium</em> Ramat.) is one of the most economically important ornamental crops grown worldwide. It has a complex hexaploid genome (2<em>n</em> = 6<em>x</em> = 54) and large genome size. The diploid <em>Chrysanthemum seticuspe</em> is often used as a model of cultivated chrysanthemum, since the two species are closely related. To expand our knowledge of the cultivated chrysanthemum, we here performed <em>de novo</em> whole-genome assembly in <em>C. seticuspe</em> using the Illumina sequencing platform. XMRS10, a <em>C. seticuspe</em> accession developed by five generations of self-crossing from a self-compatible strain, AEV2, was used for genome sequencing. The 2.72 Gb of assembled sequences (CSE_r1.0), consisting of 354,212 scaffolds, covered 89.0% of the 3.06 Gb <em>C. seticuspe</em> genome estimated by k-mer analysis. The N50 length of scaffolds was 44,741 bp. For protein-encoding genes, 71,057 annotated genes were deduced (CSE_r1.1_cds). Next, based on the assembled genome sequences, we performed linkage map construction, gene discovery and comparative analyses for <em>C. seticuspe</em> and cultivated chrysanthemum. The generated <em>C. seticuspe</em> linkage map revealed skewed regions in segregation on the AEV2 genome. In gene discovery analysis, candidate flowering-related genes were newly found in CSE_r1.1_cds. Moreover, single nucleotide polymorphism identification and annotation on the <em>C</em>. × <em>morifolium</em> genome showed that the <em>C. seticuspe</em> genome was applicable to genetic analysis in cultivated chrysanthemums. The genome sequences assembled herein are expected to contribute to future chrysanthemum studies. In addition, our approach demonstrated the usefulness of short-read genome assembly and the importance of choosing an appropriate next genome sequencing technology based on the purpose of the post-genome analysis.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"Chrysanthemum seticuspe"'><em>Chrysanthemum seticuspe</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genome sequence assembly"'>genome sequence assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"linkage map"'>linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"flowering genes"'>flowering genes</a>
</div>                    <h2 scrollto-destination="137025363" id="137025363" class="section-title">1. Introduction</h2>
<p>Cultivated chrysanthemum (<em>Chrysanthemum morifolium</em> Ramat.), which is produced either as cut flowers or potted and garden plants, is an herbaceous perennial in the family Asteraceae (Compositae). Chrysanthemum was first cultivated in China and developed for horticultural purposes in East Asia. Soon after the discovery of the response of plants to day length, i.e. ‘photoperiodism’,<a reveal-id="dsy048-B1" data-open="dsy048-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> it was determined that the flowering time of this short-day (SD) plant could be controlled for year-round commercial production by manipulating the day length using blackouts or artificial lighting. Chrysanthemum has also been used in classical physiological studies of photoperiodism, leading to the proposal that floral stimuli (florigens) and floral repressors (antiflorigens) are synthesized in the leaves under inductive/non-inductive photoperiods.<a reveal-id="dsy048-B2" data-open="dsy048-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy048-B3" data-open="dsy048-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Recent studies have demonstrated that FLOWERING LOCUS T (FT) and its orthologues act as florigens in several species, including chrysanthemum.<a reveal-id="dsy048-B4 dsy048-B5 dsy048-B6 dsy048-B7 dsy048-B8" data-open="dsy048-B4 dsy048-B5 dsy048-B6 dsy048-B7 dsy048-B8" class="link link-ref link-reveal xref-bibr"><sup>4–8</sup></a> In 2013, an antiflorigen (CsAFT) was first discovered in a wild chrysanthemum by using a reverse-genetic approach.<a reveal-id="dsy048-B9" data-open="dsy048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> An ultra-dense linkage map was recently constructed in cultivated chrysanthemum,<a reveal-id="dsy048-B10" data-open="dsy048-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy048-B11" data-open="dsy048-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> but the complex genome structure of the cultivated chrysanthemum, such as hexaploidy (2<em>n</em> = 6<em>x</em> = 54), along with the large genome size and self-incompatibility,<a reveal-id="dsy048-B12" data-open="dsy048-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> have obstructed genetic studies on horticulturally and physiologically important characteristics.</p>
<p>The genus <em>Chrysanthemum</em> found in Japan includes 32 species ranging from diploid (2<em>n</em> = 2<em>x</em> = 18) to decaploid (2<em>n</em> = 10<em>x</em> = 90).<a reveal-id="dsy048-B13" data-open="dsy048-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Diploid <em>C. seticuspe</em> (Maxim.) Hand.-Mazz., a wild relative of chrysanthemum, is closely related to cultivated chrysanthemum, with both plants being herbaceous perennial, SD responsive and self-incompatible. Although <em>C. seticuspe</em> is generally not thought to be a direct progenitor of the cultivated chrysanthemum, since it is suggested that cultivated chrysanthemum is derived from hybridization between other chrysanthemum species,<a reveal-id="dsy048-B14" data-open="dsy048-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy048-B15" data-open="dsy048-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> it is considered a model species of cultivated chrysanthemum and is thus used for molecular-genetic and physiological analysis.</p>
<p>The recent advances in next genome sequencing (NGS) technology have brought <em>de novo</em> whole-genome sequencing to various organisms. The sequencing cost has been dramatically decreasing while the quality of the assembled sequences has been increasing along with the growth of long-read sequencing technologies. However, <em>de novo</em> whole-genome sequencing is still costly in species with large genomes, and thus many of these species have not benefitted from the new NGS technologies. In this study, we performed <em>de novo</em> whole-genome assembly in <em>C. seticuspe</em> by using only the Illumina sequencing platform to achieve low cost assembly. Based on the assembled genome, gene discovery analysis was conducted for genes related to flowering, which is the most important trait in chrysanthemum. Genetic analysis was also performed such as linkage map construction, comparative phylogenetic investigation between <em>C. seticuspe</em> and other species, and identification of single nucleotide polymorphisms (SNPs) of cultivated chrysanthemum. Our approach suggests a potential strategy for advancing genetic and genomic studies in species that are not candidates for high quality <em>de novo</em> whole-genome assembly due to biological or other difficulties.</p>                    <h2 scrollto-destination="137025367" id="137025367" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="137025368" id="137025368" class="section-title">2.1. Plant materials</h3>
<p>Three <em>C. seticuspe</em> accessions, AEV2, NIFS-0 and NIFS-3, were used in this study. AEV2 is a natural self-compatible mutant that was selfed for five generations to reduce genomic heterogeneity. The resultant line, XMRS10, was used for the whole-genome DNA sequencing. Linkage maps were constructed based on an F<sub>1</sub> mapping population generated from a cross between NIFS-0 and AEV2. NIFS-3 was used for genome size estimation by flow cytometry and for transcript sequencing for simple sequence repeat (SSR) marker development for linkage map construction.</p>
<p>The genome size of NIFS-3 was estimated in comparison with those of <em>Arabidopsis thaliana</em> ‘Columbia’, <em>Oryza sativa</em> ‘Nipponbare’, <em>Dianthus caryophyllus</em> ‘Francesco’, <em>Ipomoea nil</em> ‘Murasaki’, <em>Eustoma grandiflorum</em> ‘Piccorosa snow’ and <em>Zea mays</em> ‘Honey Bantam Peter 610’ by flow cytometry. Nuclei suspensions of young leaves were prepared using a solution of CyStain PI Absolute P (Partec GmbH, Münster, Germany) added to 2% PVP (polyvinylpyrrolidone)and 15 mM DTT (dithiothreitol) according to the manufacturer’s instructions and analysed with a PAS flow cytometer with 532 nm laser excitation.</p>                    <h3 scrollto-destination="137025371" id="137025371" class="section-title">2.2 Whole-genome sequencing and assembly</h3>
<p>Total cellular DNA of XMRS10 was extracted from young leaves by using a DNeasy Plant Mini Kit (Qiagen, Hilden, Germany) and used for construction of paired-end (PE) and mate-pair (MP) libraries. The expected insert size of the PE libraries was 500 bp, and sequences were generated by Illumina HiSeq 2000 and MiSeq (Illumina, San Diego, CA, USA) with read lengths of 101 and 301 nt, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242087&amp;Signature=JkqPpDjT5zSX4aAHHbXOFsVE6ICVwZds4vsyl428WRF7f5BA4rdzIVb-BA73zSYOV5fjMMMeG62e6RXWTSPr5e0d9Nyb4E-8ikbQEIPggjnyQPzLrLtLmxvC0e-h1Zb1woI51RbB-rjTCl5VjqwSu~vSA3jnxXWYHBS3mXqkwkA8qyoT7C6suXH~9OnyPR4RpuSdOfSEPOtw1LY05Mbtm5N7QrXiRUby1VxI6Iv0uu9Nsg-EacojY4hM~2Zo4-A5hDfHqM-SCfez0v8zS9SuMeTJig1MIb~Z4hIyC2z5X7M0RJPhp5k2GiaupqqUuKIZxz1q6YEaTxmFThUr94MKUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1a</a></span>). Four MP libraries were constructed with expected insert sizes of 2, 5, 10 and 15 Kb and sequenced by HiSeq 2000 with a read length of 101 nt. Sequenced bases with quality scores less than 10 were filtered by PRINSEQ 0.20.4 (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242087&amp;Signature=JkqPpDjT5zSX4aAHHbXOFsVE6ICVwZds4vsyl428WRF7f5BA4rdzIVb-BA73zSYOV5fjMMMeG62e6RXWTSPr5e0d9Nyb4E-8ikbQEIPggjnyQPzLrLtLmxvC0e-h1Zb1woI51RbB-rjTCl5VjqwSu~vSA3jnxXWYHBS3mXqkwkA8qyoT7C6suXH~9OnyPR4RpuSdOfSEPOtw1LY05Mbtm5N7QrXiRUby1VxI6Iv0uu9Nsg-EacojY4hM~2Zo4-A5hDfHqM-SCfez0v8zS9SuMeTJig1MIb~Z4hIyC2z5X7M0RJPhp5k2GiaupqqUuKIZxz1q6YEaTxmFThUr94MKUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> for information on the bioinformatic tools used in this study), and adaptor sequences in the reads were trimmed using fastx_clipper from the FASTX-Toolkit 0.0.13. The genome size of XMRS10 was estimated by using Jellyfish ver. 1.1.6 based on a k-mer = 17 frequency.</p>
<p>The MiSeq PE reads were assembled by SOAPdenovo2 rev 240 with k-mer = 127, and the gaps were closed by GapCloser 1.10 (<em>P</em> = 31, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242087&amp;Signature=JkqPpDjT5zSX4aAHHbXOFsVE6ICVwZds4vsyl428WRF7f5BA4rdzIVb-BA73zSYOV5fjMMMeG62e6RXWTSPr5e0d9Nyb4E-8ikbQEIPggjnyQPzLrLtLmxvC0e-h1Zb1woI51RbB-rjTCl5VjqwSu~vSA3jnxXWYHBS3mXqkwkA8qyoT7C6suXH~9OnyPR4RpuSdOfSEPOtw1LY05Mbtm5N7QrXiRUby1VxI6Iv0uu9Nsg-EacojY4hM~2Zo4-A5hDfHqM-SCfez0v8zS9SuMeTJig1MIb~Z4hIyC2z5X7M0RJPhp5k2GiaupqqUuKIZxz1q6YEaTxmFThUr94MKUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The resultant sequences were subjected to scaffolding with the MP reads using SSPACE2.0 with the parameters of −k 3 and −x 0. The gaps of the scaffolds were closed by GapFiller with the HiSeq PE reads. Potential contaminated sequences on the assembled scaffolds were identified and removed using BLASTN searches against the chloroplast and mitochondrial genome sequence of <em>A. thaliana</em> (accession numbers: NC_000932.1 and NC_001284.2), against the human (hg19, <a class="link link-uri" href="https://genome.ucsc.edu/" target="_blank">https://genome.ucsc.edu/</a>), fungal and bacterial genome sequences registered in NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov" target="_blank">http://www.ncbi.nlm.nih.gov</a>), against vector sequences from UniVec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/" target="_blank">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>) and against PhiX (NC_001422.1) sequences with E-value cutoffs of 1E-10 and length coverage &gt; 10%. The resultant draft genome sequences were designated CSE_r1.0. Assembly quality was assessed by benchmarking universal single-copy orthologs (BUSCOs) using BUSCO v3.0 software. Known repetitive sequences registered in Repbase and <em>de novo</em> repetitive sequences defined by RepeatModeler 1.0.11 were identified by RepeatMasker 4.0.7.</p>                    <h3 scrollto-destination="137025374" id="137025374" class="section-title">2.3. Transcript sequencing, gene prediction and annotation</h3>
<p>Total RNAs were extracted from a total of 35 samples of XMRS10 grown under various conditions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Total RNA was extracted from shoot tips, leaves, young buds, flowers and roots using RNAiso Plus (Takara Bio, Shiga, Japan) followed by an RNeasy Mini Kit (Qiagen) for purification according to the manufacturer’s instructions and LiCl precipitation for the polysaccharide removal if necessary. Libraries were constructed by using a TruSeq standard mRNA HT sample prep kit (Illumina), and sequences were obtained by HiSeq 2000.</p>
<p>Evidence-based gene prediction was first performed by BRAKER1 with the obtained transcript reads and the assembled genome sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The predicted genes were used as a training set for ab initio gene prediction by Augustus in the MAKER-P pipeline, along with the additional training sets derived from the following amino acid sequences: <em>A. thaliana</em> (TAIR10, <a class="link link-uri" href="https://www.arabidopsis.org/" target="_blank">https://www.arabidopsis.org/</a>); <em>D. caryophyllus</em> (DCA_r1.0)<a reveal-id="dsy048-B16" data-open="dsy048-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a>; <em>Petunia axillaris</em> (v1.6.2)<a reveal-id="dsy048-B17" data-open="dsy048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>; <em>Solanum lycopersicum</em> (ITAG3.2)<a reveal-id="dsy048-B17" data-open="dsy048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>; <em>Populus trichocarpa</em> (v3.0)<a reveal-id="dsy048-B18" data-open="dsy048-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a>; <em>O. sativa</em> (IRGSP-1.0)<a reveal-id="dsy048-B19" data-open="dsy048-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a>; <em>Solanum tuberosum</em> (PGSC DM v3.4)<a reveal-id="dsy048-B20" data-open="dsy048-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a>; and <em>Nelumbo nucifera</em> (<a class="link link-uri" href="http://lotus-db.wbgcas.cn" target="_blank">http://lotus-db.wbgcas.cn</a>, downloaded June 2016). In parallel, evidence-based gene prediction was performed by mapping the <em>C. seticuspe</em> transcript reads onto the assembled genome sequences using the Tophat-Cufflinks pipeline and merged with the results of ab initio gene prediction by the MAKER-P pipeline. The resultant sequences were filtered by sequence length and annotation edit distance (AED) score, and then subjected to homology searches against the NCBI NR database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov" target="_blank">http://www.ncbi.nlm.nih.gov</a>) and <em>A. thaliana</em> in TAIR11 using BLASTP with an E-value cut-off of 1E-20. Domain search was conducted by InterProScan against the InterPro database with an E-value cutoff of 1.0. Transposable elements (TEs) were identified by BLAST searches against the NCBI NR database and domain searches against InterPro and GyDB 2.0 with an E-value cutoff of 1.0. The sets of sequences showed significant similarity against the databases, with the exception that TEs were designated as CSE_r1.1_cds. Functional analysis of coding DNA sequences (CDSs) in CSE_r1.1_cds was performed by classifying the sequences into the plant gene ontology (GO) slim categories and the ‘euKaryotic clusters of Orthologous Groups’ (KOG) categories, and mapping onto the Kyoto Encyclopaedia of Genes and Genomes (KEGGs) reference pathways. Transfer RNA genes were predicted using tRNAscan-SE ver. 1.23 with default parameters. rRNA genes were predicted by BLAST searches with an E-value cutoff of 1E-10. The <em>A. thaliana</em> 5.8S and 25S rRNAs (accession number: X52320.1) and 18S rRNA (accession number: X16077.1) were used as query sequences.</p>                    <h3 scrollto-destination="137025377" id="137025377" class="section-title">2.4. Gene discovery analysis related to flowering</h3>
<p>The genes related to flowering of <em>C. seticuspe</em> were identified based on the BLAST searches and OrthoMCL analysis (see below). The amino acid sequences of flowering pathway genes of Arabidopsis were used as query for tblastn search against CSE_r1.1_cds and CSE_r1.1_maker_cds. The phylogenetic tree of the phosphatidylethanolamine-binding proteins (PEBPs) was constructed using the Neighbour-Joining method in MEGA7 software based on the amino acid sequences of PEBP from the plant species, Al: <em>Arabidopsis lyrate</em>, Am: <em>Antirrhinum majus</em>, At: <em>A. thaliana</em>, Bv: <em>Beta vulgaris</em>, Ci: <em>Citrus unshiu</em>, Cs: <em>Chrysanthemum seticuspe</em>; Ej: <em>Eriobotrya japonica,</em> Fv: <em>Fragaria vesca,</em> Gm: <em>Glycine max,</em> In: <em>Ipomoea nil,</em> Le: <em>Lycopersicon esculentum,</em> Lp: <em>Lolium perenne,</em> Md: <em>Malus</em> × <em>domestica,</em> Nt: <em>Nicotiana tabacum,</em> Os: <em>O. sativa,</em> Pc: <em>Pyrus communis,</em> Pn: <em>Populus nigra,</em> Pp: <em>Physcomitrella patens,</em> Ps: <em>Pisum sativum</em>, Ro: <em>Rosa chinensis,</em> Sm: <em>Selaginella moellendorfii,</em> Ta: <em>Triticum aestivum</em> and Vv: <em>Vitis vinifera.</em> Bootstrap values for 1,000 resampling are shown on each branch.</p>
<p>Of the investigated candidate genes, three genes—<em>CEN-like</em> (Cse_sc005398.1_g060.1), <em>CsAFL1</em> and <em>CsFL</em>—were subjected to gene expression analysis in rosette-forming <em>C</em>. <em>seticuspe</em> line NIFS-3. The rosette-forming plants were divided into two groups, a non-chilled group and a chilled (5°C, 4 weeks) group. After growing the plants under the respective conditions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), shoot tips from each group were sampled for RNA extraction. Total RNA were extracted with an RNA extraction plant mini kit (Qiagen). The expression levels of the three genes were investigated using quantitative real-time PCR. The primers used were as follows. For the <em>CEN</em>-like gene: qRT-f, AGGATTCATCAATCCATGTG; and qRT-r, CCTCCGTGAATATCAACCCG; for <em>CsAFL1</em> (the <em>AP1/FUL</em>-like gene): <em>CsAFL1</em>-QF, TGATCCCTACATCAGTGCTCCA; and <em>CsAFL1</em>-QR: CAAGGGGGCAAGACTATTGATG; and for <em>CsFL</em> (a homologue of <em>LEAFY</em>): <em>CsFL</em>-QF, CTTTGTCATGCTGAACGGAGTG; and <em>CsFL</em>-QR, GAGCATATGACCAACACCACCA. <em>CsACTIN</em> (AB770470) was used as an internal standard to normalize raw data.</p>                    <h3 scrollto-destination="137025380" id="137025380" class="section-title">2.5. Linkage-map construction</h3>
<p>Linkage maps were constructed based on a F<sub>1</sub> mapping population generated as a cross between NIFS-0 and AEV2. SNP genotyping by ddRAD-Seq analysis was performed for 84F<sub>1</sub> individuals and the parental lines as described by Shirasawa et al.<a reveal-id="dsy048-B21" data-open="dsy048-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The ddRAD-Seq libraries were constructed with a combination of restriction enzymes, <em>Pst</em>I and <em>Msp</em>I and sequenced by MiSeq with a read length of 251 nt. The reads were mapped onto CSE_r1.0 by Bowtie2, and variant calls were performed by SAMtools 0.1.19 and VarScan 2.3 as described by Shirasawa et al.<a reveal-id="dsy048-B21" data-open="dsy048-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The candidate SNPs were filtered under the following conditions: exclude in/dels, minimum depth = 5, minimum quality = 10, max missing = 0.5, minor allele frequency = 0.2, polymorphic between the parental lines, and exclude AEV2 = Alt (alternative) homozygous.</p>
<p>Segregation analysis was also performed using the SSR markers developed in this study. A total of 4,056,757 transcript sequences of leaves and shoot apices of NIFS-3 were obtained by Roche 454 FLX (Roche Diagnostics, Indianapolis, IN, USA) and assembled by Newbler 2.6 (Roche Diagnostics). SSR motifs on the assembled sequences were identified using SciRoKo software and the Fuzznuc tool from EMBOSS. Primers were designed by Primer3 for the SSRs that had 8, 5, 4, 3 and 3 repeats in the di-, tri-, tetra-, penta- and hexa-motifs, respectively. Polymorphic analysis of the SSR markers was performed using an ABI 3730xl fluorescent fragment analyser (Applied Biosystems, Foster City, CA, USA) or 10% polyacrylamide gel electrophoresis according to Isobe et al.<a reveal-id="dsy048-B22" data-open="dsy048-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a></p>
<p>For linkage analysis, the segregating SNPs were grouped using MSTmap with the following parameters: cut-off <em>P</em>-value = 1, no map size = 0; the SSR loci were separately grouped by JoinMap 4. Then, correspondent SNP and SSR linkage groups (LGs) were identified and integrated based on segregation patterns in the mapping population. The segregation data was categorized into two parent-specific datasets and ordered by JoinMap 4 under the following conditions: Kosambi’s mapping function, LOD (logarithm of the odds) &gt;1.0, REC frequency &lt; 0.4, goodness of fit jump threshold for removal of loci = 5.0, number of added loci after which a ripple is performed = 1, and third round = yes.</p>                    <h3 scrollto-destination="137025384" id="137025384" class="section-title">2.6. Comparative and phylogenetic analysis</h3>
<p>The translated protein sequences of CSE_r1.1_cds were clustered by OrthoMCL with those of <em>A. thaliana</em> (Araport11, <a class="link link-uri" href="https://www.araport.org/data/araport11/" target="_blank">https://www.araport.org/data/araport11/</a>), <em>S. lycopersicum</em> (tomato, ITAG3.2),<a reveal-id="dsy048-B17" data-open="dsy048-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><em>Lactuca sativa</em> (lettuce, Lsativa_467_v5, <a class="link link-uri" href="http://lgr.genomecenter.ucdavis.edu/" target="_blank">http://lgr.genomecenter.ucdavis.edu/</a>)<a reveal-id="dsy048-B23" data-open="dsy048-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and <em>Helianthus annuus</em> (sunflower, Ha412v1r1_port_v1.0)<a reveal-id="dsy048-B24" data-open="dsy048-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> with the parameters c = 0.6 and aL = 0.9. The phylogenetic analysis was performed by MEGA 7.0.9 beta and TIMETREE based on the single copy genes conserved in <em>C. seticusp</em>e and the four species. The divergence time of 104 MYA between <em>A. thaliana</em> and <em>S. lycopersicum</em> was used for calibration. The published sequence read archive (SRA) transcripts for six cultivated chrysanthemum (<em>C. × morifolium</em>) varieties were mapped onto CSE_r1.0 by TopHat v2.1.1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1b</a></span>) in order to investigate the mapped ratios and identify variants. Variants were called based on the mapping result using SAMtools 0.1.19 and subsequently filtered using VarScan 2.3. SNP effects on gene function were predicted by using SnpEff ver.4.0.</p>                    <h2 scrollto-destination="137025386" id="137025386" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="137025387" id="137025387" class="section-title">3.1. Whole-genome sequencing and assembly of <em>C. seticuspe</em>
</h3>
<p>A total of 94.2 Gb of MiSeq PE and 137.5 Gb of HiSeq PE reads were subjected to k-mer frequency analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Two peaks were observed in the distribution of the number of distinct k-mers at a given multiplicity distribution. Based on the multiplicity value of the higher peak (multiplicity = 67), the genome size of XMRS10 was estimated as 3.06 Gb. This estimated genome size was close to 2.90 ± 0.03 Gb, which was the estimated genome size of NIFS-3 by flow-cytometry analysis. The large difference in height between the two peaks (multiplicity = 67 and 134) suggests that most of the XMRS10 genome is homozygous; however, heterozygous regions still remain after selfing five times from AEV2.</p>
<p>The 94.2 Gb MiSeq PE reads were assembled by SOAPdenovo2 (k-mer = 127) and GapCloser, and 6,480,313 scaffolds were generated with a total length of 3,189, 146,773 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Then, the sequences were further scaffolded by SSPACE2.0 with MP reads, and gaps were filled with HiSeq PE reads by GapFiller. After excluding contaminated sequences and those shorter than 500 bp, the remaining 354,212 sequences were designated as a draft genome sequence, CSE_r1.0, and subjected to analysis (<a reveal-id="dsy048-T1" data-open="dsy048-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The total and N50 lengths of CSE_r1.0 were 2,721, 839,164 and 44,741 bp, respectively. The assembled sequences covered 89.0% of the XMRS10 genome when the genome size was estimated as 3.06 Gb. The CSE_r1.0 sequences were mapped onto the 1, 440 BUSCOs to assess the sequence quality. The number of complete BUSCOs was 1, 279 (88.8%), including 1,153 (80.1%) single copy genes and 125 (8.7%) duplicated genes. The numbers of fragmented and missing BUSCOs were 40 and 121, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Although the assembled genome sequences were fragmented, the BUSCO results suggested that the CSE_r1.0 had an acceptable quality for gene prediction and annotation.</p>                        <a id="137025390" scrollto-destination="137025390"></a>
<div content-id="dsy048-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy048-T1" class="table-wrap-title" data-id="dsy048-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Statistics of the <em>C. seticuspe</em> genome assembly and CDS</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th>CSE_r1.0 </th>
<th>CSE_r1.1_maker_cds </th>
<th>CSE_r1.1_cds </th>
</tr>
<tr>
<th> </th>
<th>Genome </th>
<th>CDS </th>
<th>CDS (after filtering) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Number of sequences </td>
<td>354,212 </td>
<td>158,375 </td>
<td>71,057 </td>
</tr>
<tr>
<td>Total length (bp) </td>
<td>2,721,839,164 </td>
<td>188,275,935 </td>
<td>89,043,396 </td>
</tr>
<tr>
<td>Average length (bp) </td>
<td>7,684 </td>
<td>1,189 </td>
<td>1,253 </td>
</tr>
<tr>
<td>Maximum length (bp) </td>
<td>447,582 </td>
<td>16,482 </td>
<td>16,482 </td>
</tr>
<tr>
<td>Minimum length (bp) </td>
<td>500 </td>
<td>300 </td>
<td>300 </td>
</tr>
<tr>
<td>N50 length (bp) </td>
<td>44,741 </td>
<td>1,560 </td>
<td>1,584 </td>
</tr>
<tr>
<td>GC% </td>
<td>36.0 </td>
<td>41.1 </td>
<td>40.8 </td>
</tr>
<tr>
<td>Repeat % </td>
<td>72.5 </td>
<td>— </td>
<td>— </td>
</tr>
<tr>
<td>Number of complete genes </td>
<td>— </td>
<td>15,796 </td>
<td>15,304 </td>
</tr>
<tr>
<td>Number of partial genes </td>
<td>— </td>
<td>142,579 </td>
<td>55,753 </td>
</tr>
<tr>
<td>TEs </td>
<td>— </td>
<td>55,741 </td>
<td>— </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy048-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th>CSE_r1.0 </th>
<th>CSE_r1.1_maker_cds </th>
<th>CSE_r1.1_cds </th>
</tr>
<tr>
<th> </th>
<th>Genome </th>
<th>CDS </th>
<th>CDS (after filtering) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Number of sequences </td>
<td>354,212 </td>
<td>158,375 </td>
<td>71,057 </td>
</tr>
<tr>
<td>Total length (bp) </td>
<td>2,721,839,164 </td>
<td>188,275,935 </td>
<td>89,043,396 </td>
</tr>
<tr>
<td>Average length (bp) </td>
<td>7,684 </td>
<td>1,189 </td>
<td>1,253 </td>
</tr>
<tr>
<td>Maximum length (bp) </td>
<td>447,582 </td>
<td>16,482 </td>
<td>16,482 </td>
</tr>
<tr>
<td>Minimum length (bp) </td>
<td>500 </td>
<td>300 </td>
<td>300 </td>
</tr>
<tr>
<td>N50 length (bp) </td>
<td>44,741 </td>
<td>1,560 </td>
<td>1,584 </td>
</tr>
<tr>
<td>GC% </td>
<td>36.0 </td>
<td>41.1 </td>
<td>40.8 </td>
</tr>
<tr>
<td>Repeat % </td>
<td>72.5 </td>
<td>— </td>
<td>— </td>
</tr>
<tr>
<td>Number of complete genes </td>
<td>— </td>
<td>15,796 </td>
<td>15,304 </td>
</tr>
<tr>
<td>Number of partial genes </td>
<td>— </td>
<td>142,579 </td>
<td>55,753 </td>
</tr>
<tr>
<td>TEs </td>
<td>— </td>
<td>55,741 </td>
<td>— </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025390" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy048-T1" class="table-wrap-title" data-id="dsy048-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Statistics of the <em>C. seticuspe</em> genome assembly and CDS</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th>CSE_r1.0 </th>
<th>CSE_r1.1_maker_cds </th>
<th>CSE_r1.1_cds </th>
</tr>
<tr>
<th> </th>
<th>Genome </th>
<th>CDS </th>
<th>CDS (after filtering) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Number of sequences </td>
<td>354,212 </td>
<td>158,375 </td>
<td>71,057 </td>
</tr>
<tr>
<td>Total length (bp) </td>
<td>2,721,839,164 </td>
<td>188,275,935 </td>
<td>89,043,396 </td>
</tr>
<tr>
<td>Average length (bp) </td>
<td>7,684 </td>
<td>1,189 </td>
<td>1,253 </td>
</tr>
<tr>
<td>Maximum length (bp) </td>
<td>447,582 </td>
<td>16,482 </td>
<td>16,482 </td>
</tr>
<tr>
<td>Minimum length (bp) </td>
<td>500 </td>
<td>300 </td>
<td>300 </td>
</tr>
<tr>
<td>N50 length (bp) </td>
<td>44,741 </td>
<td>1,560 </td>
<td>1,584 </td>
</tr>
<tr>
<td>GC% </td>
<td>36.0 </td>
<td>41.1 </td>
<td>40.8 </td>
</tr>
<tr>
<td>Repeat % </td>
<td>72.5 </td>
<td>— </td>
<td>— </td>
</tr>
<tr>
<td>Number of complete genes </td>
<td>— </td>
<td>15,796 </td>
<td>15,304 </td>
</tr>
<tr>
<td>Number of partial genes </td>
<td>— </td>
<td>142,579 </td>
<td>55,753 </td>
</tr>
<tr>
<td>TEs </td>
<td>— </td>
<td>55,741 </td>
<td>— </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy048-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th> </th>
<th>CSE_r1.0 </th>
<th>CSE_r1.1_maker_cds </th>
<th>CSE_r1.1_cds </th>
</tr>
<tr>
<th> </th>
<th>Genome </th>
<th>CDS </th>
<th>CDS (after filtering) </th>
</tr>
</thead>
<tbody>
<tr>
<td>Number of sequences </td>
<td>354,212 </td>
<td>158,375 </td>
<td>71,057 </td>
</tr>
<tr>
<td>Total length (bp) </td>
<td>2,721,839,164 </td>
<td>188,275,935 </td>
<td>89,043,396 </td>
</tr>
<tr>
<td>Average length (bp) </td>
<td>7,684 </td>
<td>1,189 </td>
<td>1,253 </td>
</tr>
<tr>
<td>Maximum length (bp) </td>
<td>447,582 </td>
<td>16,482 </td>
<td>16,482 </td>
</tr>
<tr>
<td>Minimum length (bp) </td>
<td>500 </td>
<td>300 </td>
<td>300 </td>
</tr>
<tr>
<td>N50 length (bp) </td>
<td>44,741 </td>
<td>1,560 </td>
<td>1,584 </td>
</tr>
<tr>
<td>GC% </td>
<td>36.0 </td>
<td>41.1 </td>
<td>40.8 </td>
</tr>
<tr>
<td>Repeat % </td>
<td>72.5 </td>
<td>— </td>
<td>— </td>
</tr>
<tr>
<td>Number of complete genes </td>
<td>— </td>
<td>15,796 </td>
<td>15,304 </td>
</tr>
<tr>
<td>Number of partial genes </td>
<td>— </td>
<td>142,579 </td>
<td>55,753 </td>
</tr>
<tr>
<td>TEs </td>
<td>— </td>
<td>55,741 </td>
<td>— </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/137025390" target="_blank">Open in new tab</a></div>
</div></div>
<p>The total length of repetitive sequences in CSE_r1.0 was 1, 774 Mb, and these sequences occupied 72.5% of the <em>C. seticuspe</em> genome (CSE_r1.0, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The ratio of repeat sequences was similar to that for lettuce (77.5%), which has an estimated genome size of 2.5 Gb.<a reveal-id="dsy048-B23" data-open="dsy048-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The ratio of unknown repeat sequences in CSE_r1.0 was higher than those in the sunflower and lettuce genomes. The total length of known repeats was 928 Mb, and Class I (long terminal repeat) elements were most frequently observed.</p>                    <h3 scrollto-destination="137025392" id="137025392" class="section-title">3.2. Gene prediction and functional analysis</h3>
<p>A total of 717,973 genes were identified by the MAKER-P pipeline. Because the number of predicted genes was large, it was considered that a large number of suspicious gene sequences were included in the results. The AED score was therefore calculated for each gene, and filtering of the gene sequences was performed by AED score (&lt;0.5) and sequence length (≥300 bp). The resultant number of predicted genes was 158,374 and designated as CSE_r1.0_maker_cds. The gene sequences with more accurate estimations were further selected by means of a BLASTP search against the NCBI NR database and InterProScan. The remaining 71,052 putative genes were designated as CSE_r1.0_cds. The number of identified tRNA- and rRNA-encoding genes was 2,738 and 100, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).</p>
<p>In order to investigate the quality of the predicted genes, a BLASTN search (E-value cut-off of 1E-10 and length coverage &gt; 10%) was performed for 47 previously identified genes involved in flowering-time regulation in <em>C. seticuspe</em> NIFS-3.<a reveal-id="dsy048-B8" data-open="dsy048-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy048-B9" data-open="dsy048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy048-B25" data-open="dsy048-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsy048-B26" data-open="dsy048-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> A total of 44 genes were found in CSE_r1.0_maker_cds, and three were missing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Of the 44 identified genes, four were tagged as TE, and not included in CSE_r1.0_cds. Thirteen sequences predicted by BRAKER1 showed higher similarity than those in CSE_r1.0_maker_cds. CsFTL2, one of the three missing genes, was found adjacent to CsFTL1 in one scaffold. PCR and Sanger sequencing showed that the physical distance between these genes was 2.5 kbp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The CDS identity of these genes was 96.4%.<a reveal-id="dsy048-B8" data-open="dsy048-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> By using GeneScan, CsFTL2 was successfully predicted. The sequences or tags were replaced or added, and the revised sets of gene sequences were designated CSE_r1.1_maker_cds and CSE_r1.1_cds (<a reveal-id="dsy048-T1" data-open="dsy048-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p>
<p>The results of gene prediction often differ according to the methods used, and there is no single best solution. Therefore, we combined multiple methods for gene prediction by MAKER-P and filtered the suspicious sequences. Although some genes were miss-classified as TE and the results of BRAKER1 were sometimes better than those of MAKER-P, the number of missing genes was only three of 47. In addition, there was a possibility that the two genes that were not identified by GeneScan do not exist in the XMRS10 genome. Hence, we concluded that CSE_r1.1_maker_cds covered most of the genes in XMRS10, but some of the genes might have been missed in CSE_r1.1_cds.</p>
<p>The numbers of complete and partial genes in CSE_r1.1_cds were 15,304 and 55,753, respectively, while those in CSE_r1.1_maker_cds were 15,796 and 142,579 (<a reveal-id="dsy048-T1" data-open="dsy048-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The putative genes (CDSs) having similarity to TE made up 35.2% of CSE_r1.1_maker_cds (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). The CDSs were tagged with ‘f’, ‘p’ and ‘d’ according to their level of similarity against the NR database (f: E-values ≤ 1E-20 and identity ≥ 70%; p: E-values ≤ E-20 and identity &lt;70%) and InterPro (d: <em>E</em>-values ≤ 1.0). Of the 71,057 CSE_r1.1_cds, 55,197 (77.7%) CDSs were tagged ‘f’ (15,304 CDSs) or ‘p’ (39,893 CDSs). The number of CDSs tagged ‘d’ was 65,317.</p>
<p>Functional analysis was performed for CSE_r1.1_cds by classifying the putative genes into the GO, KOG and KEGG databases. A total of 33,372 (47.0%) genes were annotated with GO categories, including 19,116 genes involved in biological processes, 6,780 genes coding for cellular components and 29,988 genes associated with molecular function (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The ratio of annotated genes was less than in the other four compared species, i.e. sunflower, lettuce, tomato and <em>A. thaliana.</em> The genes in these four species were predicted based on pseudomolecules, while the <em>C. seticuspe</em> genes were predicted based on 354,212 scaffolds. Hence, it was considered that the <em>C. seticuspe</em> genes predicted in this study were more fragmented than the genes of the four other species used for comparison, and the fragmentation caused the lower ratio of annotated genes. A total of 54,735 genes showed significant similarity to genes in the KOG database, and the distributions of the KOG functional categories were similar to those of the compared species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). The total number of mapped genes on the KEGG pathway was 4,941 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>).</p>                    <h3 scrollto-destination="137025398" id="137025398" class="section-title">3.3. Gene discovery analysis related to flowering</h3>
<p>Photoperiodic flowering is an important characteristic controlling the flowering time in the commercial production of chrysanthemums. In contrast to the model plant species such as <em>Arabidopsis</em> and rice, photoperiodic flowering of chrysanthemums is dependent on the absolute duration of darkness and is sensitive to light exposure at midnight.<a reveal-id="dsy048-B9" data-open="dsy048-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> The majority of the photoperiod-dependent flowering pathway genes found in Arabidopsis, such as photoreceptors, circadian clock components or clock-controlled transcription factor genes, have been identified in the <em>C. seticuspe</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). This indicates that the basic components in the photoperiodic flowering pathway are well conserved even among species with different photoperiodic responses, but that the regulation of each gene might be different or that a unique, alternative pathway might also exist in chrysanthemums.</p>
<p>Photoperiodic floral initiation of chrysanthemums is regulated by the antagonistic action of two homologous FT/TERMINAL FLOWER 1 (TFL1)-like proteins. Three <em>FT</em>-like genes (<em>CsFTL1</em>, <em>CsFTL2</em> and <em>CsFTL3</em>) and two <em>TFL1/BFT</em>-like genes (<em>CsAFT</em> and <em>CsTFL1</em>) were identified in <em>C. seticuspe</em> in previous reports.<a reveal-id="dsy048-B8" data-open="dsy048-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy048-B9" data-open="dsy048-B9" class="link link-ref link-reveal xref-bibr"><sup>9,</sup></a><a reveal-id="dsy048-B27" data-open="dsy048-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Two additional <em>TFL1</em>-like genes and one <em>MFT</em>-like gene were newly identified in this study, but no additional <em>FT</em>-like genes were found in the <em>C. seticuspe</em> genome (<a reveal-id="dsy048-F1" data-open="dsy048-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Phylogenetic analysis and amino acid alignment using the PEBPs from other plant species suggested that only three <em>FT</em>-like genes (<em>CsFTL1</em>, <em>2</em> and <em>3</em>) have floral promoter activity, while four <em>TFL1/CEN/BFT</em>-like genes have floral repressor activity (<a reveal-id="dsy048-F1" data-open="dsy048-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>).<a reveal-id="dsy048-B28" data-open="dsy048-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p>                        <a id="137025401" scrollto-destination="137025401"></a>
<div id="dsy048-F1" data-id="dsy048-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f1.png?Expires=1576242088&amp;Signature=f63APDLHH7X9sBXNOwsYauq4s3wdQmLU1~v7z-FFVN2hHJgxMAcSIatuhFmenxSy796OHEyZrnQQ25EJvxWMGtn1T5C7tvSVnOzQTBoqj3lE5FLYdrbEbu9d2hPqMFbWuZK7ArRzmgrFrTnkOIUjHSWkPRCi0etlHrzDDLvYQ-w-ZLbxmuVZpeJxPv2lr1oMKbsLVlHYU4BQvf8pW~Amp7Pbtwqr642uNQo-d28jTWU9x4-qfecQOTNC~05YoflHBrdjqbW5KXTGEI9GbYxMZMrzZy2QDPjf6kcjczGRl2QixFNFtwxkODHdXzkXC1XyJWyfrFkHdlPtJG6MG9MmuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene discovery analysis related to flowering. (A) Phylogenetic tree of PEBP family proteins from the plant species including C. seticuspe homologues. The tree was constructed using the Neighbor-Joining method in MEGA7 software. Bootstrap values for 1,000 re-samplings are shown on each branch. Black and red dots represent C. seticuspe homologues identified in previously and the present studies, respectively. (B) Amino acid alignment of FT/TFL1 family proteins at the conserved positions that determine inducer or repressor activities. (C) Effects of chilling on expression of the newly identified CEN-like gene (Cse_sc005398.1_g060.1) and a floral meristem identity gene, CsFL (a homologue of LEAFY) in the rosette-forming plants. Data are means ± SE (n = 5). The value of a non-chilled plant at the start of the experiment was set to one." path-from-xml="dsy048f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025401/dsy048f1.tif" path-from-xml="dsy048f1.tif" target="_blank">Open in new tab</a><a section="137025401" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f1.png?Expires=1576242088&amp;Signature=eAbmcZM5k~pPoAp5s1aBrU5EdXW1xOzi8UFtivqm6klwaJhS17ejOjpKOt5Fk3pgnhEYGG3mfX4uoA5RQfhiE5m6I-WLPm4qYET1qsIRxaUnT-cSF85QPPvWFVd73DSjZARlmPWCZAS3JgN7JkeNwymSW8JfPwljBjQ3lfhhPdVMIFAV9bOJoq~cb43BmRhBTaO9iazSd8AQ3F4tSAsOJhZFjNkANoW2y1G-5n7TbEheXgxTp3-GQLA77CfPbrq-TfYsRxQXie76WlUPRZd6HcyVyrW3ZWOPvaTmGAkcxRiUGRskHZmDCpyZQEAXi3vCevj5NT2i6cZyYMbUsQNzCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025401&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Gene discovery analysis related to flowering. (A) Phylogenetic tree of PEBP family proteins from the plant species including <em>C. seticuspe</em> homologues. The tree was constructed using the Neighbor-Joining method in MEGA7 software. Bootstrap values for 1,000 re-samplings are shown on each branch. Black and red dots represent <em>C. seticuspe</em> homologues identified in previously and the present studies, respectively. (B) Amino acid alignment of FT/TFL1 family proteins at the conserved positions that determine inducer or repressor activities. (C) Effects of chilling on expression of the newly identified <em>CEN</em>-like gene (Cse_sc005398.1_g060.1) and a floral meristem identity gene, <em>CsFL</em> (a homologue of LEAFY) in the rosette-forming plants. Data are means ± SE (<em>n</em> = 5). The value of a non-chilled plant at the start of the experiment was set to one.</p></div>
</div>
<div content-id="dsy048-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f1.png?Expires=1576242088&amp;Signature=f63APDLHH7X9sBXNOwsYauq4s3wdQmLU1~v7z-FFVN2hHJgxMAcSIatuhFmenxSy796OHEyZrnQQ25EJvxWMGtn1T5C7tvSVnOzQTBoqj3lE5FLYdrbEbu9d2hPqMFbWuZK7ArRzmgrFrTnkOIUjHSWkPRCi0etlHrzDDLvYQ-w-ZLbxmuVZpeJxPv2lr1oMKbsLVlHYU4BQvf8pW~Amp7Pbtwqr642uNQo-d28jTWU9x4-qfecQOTNC~05YoflHBrdjqbW5KXTGEI9GbYxMZMrzZy2QDPjf6kcjczGRl2QixFNFtwxkODHdXzkXC1XyJWyfrFkHdlPtJG6MG9MmuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene discovery analysis related to flowering. (A) Phylogenetic tree of PEBP family proteins from the plant species including C. seticuspe homologues. The tree was constructed using the Neighbor-Joining method in MEGA7 software. Bootstrap values for 1,000 re-samplings are shown on each branch. Black and red dots represent C. seticuspe homologues identified in previously and the present studies, respectively. (B) Amino acid alignment of FT/TFL1 family proteins at the conserved positions that determine inducer or repressor activities. (C) Effects of chilling on expression of the newly identified CEN-like gene (Cse_sc005398.1_g060.1) and a floral meristem identity gene, CsFL (a homologue of LEAFY) in the rosette-forming plants. Data are means ± SE (n = 5). The value of a non-chilled plant at the start of the experiment was set to one." path-from-xml="dsy048f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025401/dsy048f1.tif" path-from-xml="dsy048f1.tif" target="_blank">Open in new tab</a><a section="137025401" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f1.png?Expires=1576242088&amp;Signature=eAbmcZM5k~pPoAp5s1aBrU5EdXW1xOzi8UFtivqm6klwaJhS17ejOjpKOt5Fk3pgnhEYGG3mfX4uoA5RQfhiE5m6I-WLPm4qYET1qsIRxaUnT-cSF85QPPvWFVd73DSjZARlmPWCZAS3JgN7JkeNwymSW8JfPwljBjQ3lfhhPdVMIFAV9bOJoq~cb43BmRhBTaO9iazSd8AQ3F4tSAsOJhZFjNkANoW2y1G-5n7TbEheXgxTp3-GQLA77CfPbrq-TfYsRxQXie76WlUPRZd6HcyVyrW3ZWOPvaTmGAkcxRiUGRskHZmDCpyZQEAXi3vCevj5NT2i6cZyYMbUsQNzCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025401&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Gene discovery analysis related to flowering. (A) Phylogenetic tree of PEBP family proteins from the plant species including <em>C. seticuspe</em> homologues. The tree was constructed using the Neighbor-Joining method in MEGA7 software. Bootstrap values for 1,000 re-samplings are shown on each branch. Black and red dots represent <em>C. seticuspe</em> homologues identified in previously and the present studies, respectively. (B) Amino acid alignment of FT/TFL1 family proteins at the conserved positions that determine inducer or repressor activities. (C) Effects of chilling on expression of the newly identified <em>CEN</em>-like gene (Cse_sc005398.1_g060.1) and a floral meristem identity gene, <em>CsFL</em> (a homologue of LEAFY) in the rosette-forming plants. Data are means ± SE (<em>n</em> = 5). The value of a non-chilled plant at the start of the experiment was set to one.</p></div>
</div>
<p>Chrysanthemum plants form rosettes by cessation of shoot elongation in autumn. This habit of chrysanthemum is considered to be a type of dormancy adaptation in the annual cycle between growth and dormancy of herbaceous perennials.<a reveal-id="dsy048-B29" data-open="dsy048-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Flowering is blocked in dormant rosette-forming chrysanthemums even under floral inductive day-length conditions. This flowering incompetence is reversed by chilling, somewhat as in a vernalization response.<a reveal-id="dsy048-B29" data-open="dsy048-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> It is noteworthy that the newly identified <em>CEN</em>-like gene (Cse_sc005398.1_g060.1) is highly expressed in the shoot tips of rosette-forming dormant plants that are incompetent to initiate flowering (<a reveal-id="dsy048-F1" data-open="dsy048-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). In the chilled plants where expression of the <em>CEN</em>-like gene was suppressed to a low level, the expression of two floral meristem identity genes (<em>CsAFL1</em> and <em>CsFL</em>) of <em>C. seticuspe</em> were up-regulated within three weeks after the transfer from the non-inductive NB to inductive SD conditions (<a reveal-id="dsy048-F1" data-open="dsy048-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The expression pattern of the <em>CEN</em>-like gene indicates that this gene might be involved in the seasonal change of flowering competence in chrysanthemum.</p>                    <h3 scrollto-destination="137025403" id="137025403" class="section-title">3.4. Linkage map construction</h3>
<p>In order to demonstrate an application of the assembled <em>C. seticuspe</em> genome to genetic analysis, <em>C. seticuspe</em> linkage maps were constructed based on an F<sub>1</sub> mapping population derived from a cross between AEV2 and NIFS-0 based on ddRAD-Seq and SSR markers. The total bases of the ddRAD-Seq reads in each F<sub>1</sub> individual were 43.8 Mb (203, 321 reads) on average. The reads were mapped onto CSE_r1.0 with a mean mapped ratio of 83.4%, and a total of 5,206 SNPs were identified. Meanwhile, a total of 66,438 assembled sequences were generated by Newbler2.6 from 2,626,286 454/FLX transcript sequences. Based on the 66,438 sequences, 9,605 primer pairs were designed on the flanking regions of SSRs identified by Fuzznuc (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). The existence of SSRs between the primer sequences was further investigated by SciRoKo, and the confirmed 3,140 SSR markers were subjected to the subsequent polymorphic analysis.</p>
<p>Parent-specific linkage maps were generated for AEV2 and NIFS-0 with nine LGs in each. The number of mapped loci was 2,124 in the AEV2 map, while that in the NIFS-0 map was 2,239 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S13 and S14</a></span>). The number of bins (non-identical locus positions) in AEV2 and NIFS-0 was 581 and 615, respectively. A higher ratio of segregation distortion was observed in the AeG5 and AeG6 LGs in the AEV2 map (<a reveal-id="dsy048-F2" data-open="dsy048-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The reference alleles were more frequently observed at the distorted loci, particularly on AeG5, suggesting the existence of a biological bias in meiosis or chromosome pairing in AEV2.</p>                        <a id="137025406" scrollto-destination="137025406"></a>
<div id="dsy048-F2" data-id="dsy048-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f2.png?Expires=1576242088&amp;Signature=BDrGVzAK4bqV61njYajizZ3FES1bsTp-VsbsbIX9wKcuZ3e5O4d1cJ3mkDYiBdd6lpuaFdHyiGrZDnzHnwjpvjcLu67dZdYwdC5VXEnih1obaqPAKFXpc7qyp1a~3rcfuIpHyYNk8707qKOPsCrT5Iq~o64jxVO6yf88NqVTYnF5hsIAs0yLP8tj0cpVTugba8Jg3k5qz~~l9rZOi6vqIE3kiUZRHb6zaDWqZJKiGqVxpgDiyPndEILgkgAwV0oQ8hgVlbJjHRA5m5II8B1zBuyl96LnW-ImOigyTpzgsuN1E1a9222dm8Q2r8eZ7ccA6ofyhoEaYVJnTBg1anKLkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AEV2 (left) and NIFS-0 (right) bin linkage maps. Blue, red and green bars show locus positions mapped by SNPs, SSR markers and both types of markers. Grey bars represent regions where mapped loci showed significant segregation distortion (P ≤ 0.05)." path-from-xml="dsy048f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025406/dsy048f2.tif" path-from-xml="dsy048f2.tif" target="_blank">Open in new tab</a><a section="137025406" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f2.png?Expires=1576242088&amp;Signature=mgUsOONaub0j-LcZdgv-zQ5dmCLxQixxvnBBzHdt-Gsn5atdqLzMCECXBUeG-GgJZzDRftRAcd5KkbodKpFWKzBJcpfXbyVieY1vbvyPQCzG6XELPBjfxTLMcycsbfx66EqGS95ZuG2khFlrumpsCpu65X2m6c5czmFQBqVerXReQusBpPz7jdBY~bICIarFUYi-Rwh2CQ8SMnLBVR7AojTvv-kP8AniHRIInLuBeb9jlbKqfvDOJmyP~9B~G56oM2ij7oX2TzLqx5zdpiBg~uhTwK9jBXsAvDP3L~-yYT7GOd0smC6w75d0XvrPF46eHtPGxDvajD2v~9uxS049sQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025406&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>AEV2 (left) and NIFS-0 (right) bin linkage maps. Blue, red and green bars show locus positions mapped by SNPs, SSR markers and both types of markers. Grey bars represent regions where mapped loci showed significant segregation distortion (<em>P</em> ≤ 0.05).</p></div>
</div>
<div content-id="dsy048-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f2.png?Expires=1576242088&amp;Signature=BDrGVzAK4bqV61njYajizZ3FES1bsTp-VsbsbIX9wKcuZ3e5O4d1cJ3mkDYiBdd6lpuaFdHyiGrZDnzHnwjpvjcLu67dZdYwdC5VXEnih1obaqPAKFXpc7qyp1a~3rcfuIpHyYNk8707qKOPsCrT5Iq~o64jxVO6yf88NqVTYnF5hsIAs0yLP8tj0cpVTugba8Jg3k5qz~~l9rZOi6vqIE3kiUZRHb6zaDWqZJKiGqVxpgDiyPndEILgkgAwV0oQ8hgVlbJjHRA5m5II8B1zBuyl96LnW-ImOigyTpzgsuN1E1a9222dm8Q2r8eZ7ccA6ofyhoEaYVJnTBg1anKLkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AEV2 (left) and NIFS-0 (right) bin linkage maps. Blue, red and green bars show locus positions mapped by SNPs, SSR markers and both types of markers. Grey bars represent regions where mapped loci showed significant segregation distortion (P ≤ 0.05)." path-from-xml="dsy048f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025406/dsy048f2.tif" path-from-xml="dsy048f2.tif" target="_blank">Open in new tab</a><a section="137025406" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f2.png?Expires=1576242088&amp;Signature=mgUsOONaub0j-LcZdgv-zQ5dmCLxQixxvnBBzHdt-Gsn5atdqLzMCECXBUeG-GgJZzDRftRAcd5KkbodKpFWKzBJcpfXbyVieY1vbvyPQCzG6XELPBjfxTLMcycsbfx66EqGS95ZuG2khFlrumpsCpu65X2m6c5czmFQBqVerXReQusBpPz7jdBY~bICIarFUYi-Rwh2CQ8SMnLBVR7AojTvv-kP8AniHRIInLuBeb9jlbKqfvDOJmyP~9B~G56oM2ij7oX2TzLqx5zdpiBg~uhTwK9jBXsAvDP3L~-yYT7GOd0smC6w75d0XvrPF46eHtPGxDvajD2v~9uxS049sQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025406&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>AEV2 (left) and NIFS-0 (right) bin linkage maps. Blue, red and green bars show locus positions mapped by SNPs, SSR markers and both types of markers. Grey bars represent regions where mapped loci showed significant segregation distortion (<em>P</em> ≤ 0.05).</p></div>
</div>                    <h3 scrollto-destination="137025407" id="137025407" class="section-title">3.5. Comparison with other plant species at the gene level</h3>
<p>The translated protein sequences in CSE_r1.1_cds were clustered and compared with the protein sequences in other plant species (sunflower, lettuce, tomato and <em>A. thaliana</em>) at the amino acid level by OrthoMCL. Of the 71,057 genes in CSE_r1.1_cds, 59,927 genes were classified into 16,098 clusters and 2,405 were classified into <em>C. seticuspe</em>-specific clusters. A total of 19,979 (33.3%) of the genes were clustered with all four of the plant species compared (<a reveal-id="dsy048-F3" data-open="dsy048-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). The finding that a larger percentage of <em>C. seticuspe</em> genes was clustered exclusively with sunflower genes (10.0%) compared to the percentage clustered exclusively with lettuce genes (3.5%) suggested that, between these two species in the family <em>Asteraceae</em>, sunflower showed the greater similarity to <em>C. seticuspe</em>. The family Asteraceae has been classified into 12 subfamilies,<a reveal-id="dsy048-B30" data-open="dsy048-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and the genera Chrysanthemum and Helianthus belong to the subfamily <em>Asyeroideae</em>, while the genus <em>Lactuca</em> belongs to the subfamily Cichorioideae. These results agree with the previous taxonomy studies.</p>                        <a id="137025409" scrollto-destination="137025409"></a>
<div id="dsy048-F3" data-id="dsy048-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f3.png?Expires=1576242088&amp;Signature=dOy~gAvW~rFB8niFNGFGQx7iv1lGxYaPQNF5syK9weYhamppnqTWJ8wa2pFdzpzGXCYQpDdvZkmC90TktxRGZdMurUtNKgxovlQD80sCrYMhJlOj1~ZvSmtaGsI6rOYHCFHwOKweq6L4iNlmvI7TAUHQ4YwkyviUlN561yTDZAgm54G92ZqTxJTwOqyOdEKfhol2jvMeDxtZ86QZ832SAG0zZIe53ROVqK8vpR2gqDHnBzJ-YzaNI60Eu6plyTz6O2gBSLQNuez30BpGwPC4OHDxKKK5YR9o6sYJz0SJqueLOrzV3-ik-qdr1u~xLyOjUIZQiL1z2lt1PfuaZ1WqbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing numbers of gene clusters in C. seticuspe, sunflower (H. annuus), lettuce (L. sativa), tomato (S. lycopersicum) and A. thaliana." path-from-xml="dsy048f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025409/dsy048f3.tif" path-from-xml="dsy048f3.tif" target="_blank">Open in new tab</a><a section="137025409" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f3.png?Expires=1576242088&amp;Signature=HRQGTq542e7LsHQrrRL8NaNvJDdwfZahUrguBFI~x~5K7C1gyDg8~3fX93a4zQDVhW7r6jCVKKi4vRC34NrlQmGIneGTGgwjGPAgFDJAp5Nvj6dHFp5q3UtoBG2MPdSWqHn~bsso7J9~Mbovh31bX3rw5cXHI-n2YLROSLh3f9UgFHC78i0HQMm4x03QDPBmLvOnwVeJNd54nwkIMCRtu5ouX-~1QSrfjoKRvup1p1MOOZbL2T~rSj-3M1Ha6c8NplBnCmaX-bf6qZn0CiTE1ECXvz2PHxbOzn1rRfL7UCE-dLJQ0OnE1UsN~YAM4bEY9Q~PvfARfxEZvXVwZVbgzw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025409&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Venn diagram showing numbers of gene clusters in <em>C. seticuspe</em>, sunflower (<em>H. annuus</em>), lettuce (<em>L. sativa</em>), tomato (<em>S. lycopersicum</em>) and <em>A. thaliana</em>.</p></div>
</div>
<div content-id="dsy048-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f3.png?Expires=1576242088&amp;Signature=dOy~gAvW~rFB8niFNGFGQx7iv1lGxYaPQNF5syK9weYhamppnqTWJ8wa2pFdzpzGXCYQpDdvZkmC90TktxRGZdMurUtNKgxovlQD80sCrYMhJlOj1~ZvSmtaGsI6rOYHCFHwOKweq6L4iNlmvI7TAUHQ4YwkyviUlN561yTDZAgm54G92ZqTxJTwOqyOdEKfhol2jvMeDxtZ86QZ832SAG0zZIe53ROVqK8vpR2gqDHnBzJ-YzaNI60Eu6plyTz6O2gBSLQNuez30BpGwPC4OHDxKKK5YR9o6sYJz0SJqueLOrzV3-ik-qdr1u~xLyOjUIZQiL1z2lt1PfuaZ1WqbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing numbers of gene clusters in C. seticuspe, sunflower (H. annuus), lettuce (L. sativa), tomato (S. lycopersicum) and A. thaliana." path-from-xml="dsy048f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025409/dsy048f3.tif" path-from-xml="dsy048f3.tif" target="_blank">Open in new tab</a><a section="137025409" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f3.png?Expires=1576242088&amp;Signature=HRQGTq542e7LsHQrrRL8NaNvJDdwfZahUrguBFI~x~5K7C1gyDg8~3fX93a4zQDVhW7r6jCVKKi4vRC34NrlQmGIneGTGgwjGPAgFDJAp5Nvj6dHFp5q3UtoBG2MPdSWqHn~bsso7J9~Mbovh31bX3rw5cXHI-n2YLROSLh3f9UgFHC78i0HQMm4x03QDPBmLvOnwVeJNd54nwkIMCRtu5ouX-~1QSrfjoKRvup1p1MOOZbL2T~rSj-3M1Ha6c8NplBnCmaX-bf6qZn0CiTE1ECXvz2PHxbOzn1rRfL7UCE-dLJQ0OnE1UsN~YAM4bEY9Q~PvfARfxEZvXVwZVbgzw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025409&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Venn diagram showing numbers of gene clusters in <em>C. seticuspe</em>, sunflower (<em>H. annuus</em>), lettuce (<em>L. sativa</em>), tomato (<em>S. lycopersicum</em>) and <em>A. thaliana</em>.</p></div>
</div>
<p>Phylogenetic analysis was further performed with the 2,280 single copy genes observed in the four Asterids species (<em>C. seticuspe, H. annuus, L. sativa</em> and <em>S. lycopersicum</em>) and <em>A. thaliana,</em> as <em>A. thaliana</em> was used for the outgroup (<a reveal-id="dsy048-F4" data-open="dsy048-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Our results showed that the subfamilies Asterodeae and Cichorioideae diverged ∼48.67 MYA, and this estimate agreed with the previous study.<a reveal-id="dsy048-B24" data-open="dsy048-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The divergence time between super tribes Asterodeae and Helianthodae was estimated as ∼45.99 MYA, 2.68 MYA after the divergence of the subfamilies Asterodeae and Cichorioideae.</p>                        <a id="137025411" scrollto-destination="137025411"></a>
<div id="dsy048-F4" data-id="dsy048-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f4.png?Expires=1576242088&amp;Signature=Nu2O0HDxIry7el2Eia9GZZ0YAEDVVSpNY2VKPONXaRpEGAyiAl50GeO0BZSBEy3fwD5aNDMvyGmO6D-6xOCIjaHMHkEuOye9NJBlQqUazMAtjgqOydtd7pS6luwFc2xiH-5DIap8Uv6FhBDBV07HNlcZu-Alc7s5JG5wzDChh4firCs7LLBjIy3HXIEuTu7k3VlK2CewmdZ-Pr9nrFInZpBdLexeESpK2IZ2m3CSl12akHWoAItPma8TcGCjpWkceRtGrnxfQ0pMGM3l1ibBmcnS8XVYokBdW-0AH6IRgTVYEI5paIZ1Lq264XT4mZBOjIglAptb76KdAMEymYpWVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A phylogenetic tree based on the 2,280 common single-copy genes of the four Asterids species and A. thaliana." path-from-xml="dsy048f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025411/dsy048f4.tif" path-from-xml="dsy048f4.tif" target="_blank">Open in new tab</a><a section="137025411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f4.png?Expires=1576242088&amp;Signature=UJMTYYqPktbJbjDov0sWRE8hydIqv0jk65Io1LyuNOdxxQrERpOxFMswjCb7MqwWu5-1gtyGhVQ8vOA36Xq4xFsUlVbufeDRAsgo4B03dROpqDUwucG2SySpdyL1PmAVzlqjR8mUo2cpK5-XBYqAB18CXtt-8ESQMeP3d8xyMMNAvLS0x9s1wpQPgk5jIT1RvMoNFCgHCylpzNo85sBhXMnIdhysWMBbZ6NiXg5LiGl7QiTYqy4Js8EMS83l-DV9g4GTmiHDMlnmEC-AStXc9x41mtbb19CLmM6AwJ2gqjH-5PO7wSpFH7FRKtLVelnwLmr89mOpLJdfvTOKCgy8Xw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025411&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A phylogenetic tree based on the 2,280 common single-copy genes of the four <em>Asterids</em> species and <em>A. thaliana</em>.</p></div>
</div>
<div content-id="dsy048-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/m_dsy048f4.png?Expires=1576242088&amp;Signature=Nu2O0HDxIry7el2Eia9GZZ0YAEDVVSpNY2VKPONXaRpEGAyiAl50GeO0BZSBEy3fwD5aNDMvyGmO6D-6xOCIjaHMHkEuOye9NJBlQqUazMAtjgqOydtd7pS6luwFc2xiH-5DIap8Uv6FhBDBV07HNlcZu-Alc7s5JG5wzDChh4firCs7LLBjIy3HXIEuTu7k3VlK2CewmdZ-Pr9nrFInZpBdLexeESpK2IZ2m3CSl12akHWoAItPma8TcGCjpWkceRtGrnxfQ0pMGM3l1ibBmcnS8XVYokBdW-0AH6IRgTVYEI5paIZ1Lq264XT4mZBOjIglAptb76KdAMEymYpWVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A phylogenetic tree based on the 2,280 common single-copy genes of the four Asterids species and A. thaliana." path-from-xml="dsy048f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/137025411/dsy048f4.tif" path-from-xml="dsy048f4.tif" target="_blank">Open in new tab</a><a section="137025411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048f4.png?Expires=1576242088&amp;Signature=UJMTYYqPktbJbjDov0sWRE8hydIqv0jk65Io1LyuNOdxxQrERpOxFMswjCb7MqwWu5-1gtyGhVQ8vOA36Xq4xFsUlVbufeDRAsgo4B03dROpqDUwucG2SySpdyL1PmAVzlqjR8mUo2cpK5-XBYqAB18CXtt-8ESQMeP3d8xyMMNAvLS0x9s1wpQPgk5jIT1RvMoNFCgHCylpzNo85sBhXMnIdhysWMBbZ6NiXg5LiGl7QiTYqy4Js8EMS83l-DV9g4GTmiHDMlnmEC-AStXc9x41mtbb19CLmM6AwJ2gqjH-5PO7wSpFH7FRKtLVelnwLmr89mOpLJdfvTOKCgy8Xw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=137025411&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy048f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A phylogenetic tree based on the 2,280 common single-copy genes of the four <em>Asterids</em> species and <em>A. thaliana</em>.</p></div>
</div>                    <h3 scrollto-destination="137025412" id="137025412" class="section-title">3.6. SNP mining and identification of variants in hexaploid-cultivated chrysanthemums</h3>
<p>Despite their importance as ornamental plants, cultivated chrysanthemums (<em>C. × morifolium</em>) are rather difficult to study at a genetic level due to their autohexaploidy, large genome size and self-incompatibility. Diploid genome sequences are often used as references of genetic and genomic analysis in polyploid species, such as rose (<em>Rosa hybrida</em>), strawberry (<em>Fragaria</em> × <em>ananassa</em>) and sweetpotato (<em>Ipomoea batatas</em>).<a reveal-id="dsy048-B31 dsy048-B32 dsy048-B33" data-open="dsy048-B31 dsy048-B32 dsy048-B33" class="link link-ref link-reveal xref-bibr"><sup>31–33</sup></a> The genome sequence of <em>C. seticuspe</em>, a diploid wild relative, can shed light on the complex nature of the cultivated chrysanthemum genome as a reference plant for chrysanthemum species.</p>
<p>The transcript sequences of the six cultivated chrysanthemums (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1b</a></span>) were mapped onto the genome sequences of <em>C. seticuspe</em> and variants were identified in order to investigate the usefulness of the <em>C. seticuspe</em> genome for genetic studies in cultivated chrysanthemums. A total of 20.4–156.9 M reads of each variety were mapped onto CSE_r1.0. The mapped ratio of the reads ranged from 56.7% (Jinba) to 82.5% (Nannongxunzhang) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15a</a></span>). Two factors were considered as the causes of the large difference in the mapped ratios: read length differences in the six cultivars (49∼100 nt; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1b</a></span>) and different sequence similarities against <em>C. seticuspe</em>. The total number of SNPs identified among the six varieties was 954,706, and 294,601 SNPs were identified in all six varieties (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15b</a></span>). The remaining 660,105 SNPs (69%) were missing in one to five of the varieties, implying the existence of large gene sequence variation among cultivated chrysanthemums.</p>
<p>The SNP genotype call was performed for the 294,601 SNPs that were identified in all six varieties with consideration for the percentages of reads having reference (Ref) and alternative (Alt) bases. SNPs with Alt read ratios of 8.35–91.65 were designated as hetero SNPs. A total of 27, 855 SNPs exhibited a uniform genotype among the six varieties (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15c</a></span>). These SNPs were considered to represent the variants between <em>C. seticuspe,</em> XMRS10, and cultivated chrysanthemum. The functions of the SNPs were annotated by SnpEff based on their positions in the mapped gene sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S16</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576242088&amp;Signature=S-fF97hhcH-DJdqdFU-2VfrRzTyRq9QbgbjqzD1jbRlj~MTMOOLmx~kpcVgGSSDNc~ATiBg308kJUP8VbMZZt658KlsS8tU1pXqCVXpYBsSUdmM4A80jKP6IVLBcS~WPAIAg0Zyug~hg4xwJvjguTDu-08Ezd2c6mcnm7zVALYEGAFBcra5aiEqobHYrwthydAJQdH8HAf0OTq0yMIB~aPyFQ-BhkizU1sy0cW5BdR4h38FdL458q7Vic0QROjDLOnjt7YknF2-RfkN5JtLFnTiuuFm0PS6~z8heDHGQZHhRsGce6goXjl32iW~-NYe9js9rZP0aTAAjXk8Z16nu6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S9</a></span>). Of the total 22,470 genes, 14,741 were identified as SNPs with multiple impacts and 533 were identified as high-impact SNPs. As a result of breeding, cultivated chrysanthemums possess a wider variety of characteristics than <em>C. seticuspe</em>—including with respect to flowering time, flower shape and colour. Causative variants for the characteristics of cultivated chrysanthemums might be identified by further analysis of the functions of the SNPs.</p>
<p>The results described above showed that the <em>C. seticuspe</em> genome is applicable to genetic and gene functional analyses in cultivated autohexaploid chrysanthemums. In autohexaploid sweetpotatoes, single-dose SNP markers were detected by using the genome sequence of diploid <em>I. trifida</em> and selected according to the allelic dose for each SNP locus, and a high density genetic map consisting of single-dose SNP markers was constructed.<a reveal-id="dsy048-B33" data-open="dsy048-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The technique is also applicable to high-density mapping for genetic improvement in cultivated chrysanthemums. A large number of genome-wide SNP markers are expected to be produced by mapping RAD-seq or other sequence data of cultivated chrysanthemum onto the assembled genome sequences, CSE_r1.0.</p>                    <h2 scrollto-destination="137025417" id="137025417" class="section-title">4. Conclusions</h2>
<p>In this study, <em>de novo</em> whole-genome assembly was performed in the genus <em>Chrysanthemum</em> with Illumina reads only. Various NGS technologies have recently become available at reasonable cost, although some of these technologies are still expensive for organisms with large genomes. Illumina reads are the lowest-cost NGS technology, and unlike long-read technologies, Illumina does not require large computer resources for assembly. Although in this study the numbers of assembled sequences and predicted genes indicated that the sequences were still fragmented, the generated draft genome and linkage maps revealed the features of the <em>C. seticuspe</em> genome. In addition, a candidate flowering-related gene, the CEN-like gene (Cse_sc005398.1_g060.1), was newly discovered based on the assembled genome. Moreover, our results demonstrated the applicability of the <em>C. seticuspe</em> genome for genetic and gene functional analysis in cultivated autohexaploid chrysanthemums (<em>C.</em> × <em>morifolium</em>). The genome and gene sequences generated in this study are thus expected to contribute to future chrysanthemum investigations. In addition, our approach demonstrated the usefulness of short reads genome assembly and the importance of choice of NGS technologies depending on the purpose of post-genome analysis.</p>                    <h2 scrollto-destination="137025419" id="137025419" class="section-title">5. Data availability</h2>
<p>The genome assembly data, annotations, gene models and SSRs on the linkage map are available at the MumGARDEN (<a class="link link-uri" href="http://mum-garden.kazusa.or.jp/" target="_blank">http://mum-garden.kazusa.or.jp/</a>). The obtained genome sequence reads are available from the DDBJ Sequence Read Archive (DRA) under the accession number DRA005542. The BioProject accession number of the study is PRJDB5536.</p>                    <h2 scrollto-destination="137025421" id="137025421" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>This work was supported by a grant from the Ministry of Agriculture, Forestry, and Fisheries of Japan (Genomics-based Technology for Agricultural Improvement, DHR5), a KAKENHI grant for Scientific Research (B) (No. 16H04877) from the Japan Society for the Promotion of Science and funds from the Kazusa DNA Research Institute Foundation. We thank A. Watanabe, S. Nakayama, K. Nani, H. Tsuruoka, C. Minami, Y. Kishida, M. Kohara, T. Fujishiro and S. Sasamoto (KDRI) and S. Kamei, Y. Hirakawa, T. Hashimoto and Y. Watase (NARO) for their technical assistance. The <em>C. seticuspe</em> lines AEV2 and XMRS10 were obtained from the National BioResource Project Chrysanthemum (<a class="link link-uri" href="http://shigen.nig.ac.jp/chrysanthemum/" target="_blank">http://shigen.nig.ac.jp/chrysanthemum/</a>). We wish to acknowledge Prof S. Fukai (Kagawa University) for the donation of the NIFS-0 plant.</p>                    <h3 scrollto-destination="137025423" id="137025423" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="137025425" id="137025425" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsy048-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Garner</div> <div class="given-names">W.W.</div>
</div>, <div class="name string-name">
<div class="surname">Allard</div> <div class="given-names">H.A.</div>
</div>
</div> <div class="year">1920</div>, 
<div class="article-title">Effect of the relative length of day and night and other factors of the environment on growth and reproduction in plants</div>, <div class="source">J. Agric. Res</div> ., <div class="volume">18</div>, <div class="fpage">553</div>–<div class="lpage">606</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Effect%20of%20the%20relative%20length%20of%20day%20and%20night%20and%20other%20factors%20of%20the%20environment%20on%20growth%20and%20reproduction%20in%20plants&amp;aulast=Garner&amp;date=1920&amp;spage=553&amp;epage=606&amp;volume=18" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Effect%20of%20the%20relative%20length%20of%20day%20and%20night%20and%20other%20factors%20of%20the%20environment%20on%20growth%20and%20reproduction%20in%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Chailakhyan</div> <div class="given-names">M.K.</div>
</div></div> <div class="year">1936</div>, 
<div class="article-title">New facts in support of the hormonal theory of plant development, <em>Dokl</em>
</div>, <div class="source">Akad. Nauk. SSSR</div> , <div class="volume">13</div>, <div class="fpage">79</div>–<div class="lpage">83</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=New%20facts%20in%20support%20of%20the%20hormonal%20theory%20of%20plant%20development%2C%20Dokl&amp;aulast=Chailakhyan&amp;date=1936&amp;spage=79&amp;epage=83&amp;volume=13" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20facts%20in%20support%20of%20the%20hormonal%20theory%20of%20plant%20development%2C%20Dokl&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Tanaka</div> <div class="given-names">T.</div>
</div></div> <div class="year">1967</div>, 
<div class="article-title">Studies on the regulation of Chrysanthemum flowering with special reference to plant regulators, I. The inhibiting action of non-induced leaves on floral stimulus</div>, <div class="source">J. Japan. Soc. Hort. Sci</div> ., <div class="volume">36</div>, <div class="fpage">77</div>–<div class="lpage">85</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20the%20regulation%20of%20Chrysanthemum%20flowering%20with%20special%20reference%20to%20plant%20regulators%2C%20I.%20The%20inhibiting%20action%20of%20non-induced%20leaves%20on%20floral%20stimulus&amp;author=T.%20Tanaka&amp;publication_year=1967&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci&amp;volume=36&amp;pages=77-85" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2503/jjshs.36.339" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs.36.339" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2503%2Fjjshs.36.339"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Studies%20on%20the%20regulation%20of%20Chrysanthemum%20flowering%20with%20special%20reference%20to%20plant%20regulators%2C%20I.%20The%20inhibiting%20action%20of%20non-induced%20leaves%20on%20floral%20stimulus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kobayashi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Weigel</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Move on up, it’s time for change—mobile signals controlling photoperiod-dependent flowering</div>, <div class="source">Genes Dev</div> ., <div class="volume">21</div>, <div class="fpage">2371</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Move%20on%20up%2C%20it%E2%80%99s%20time%20for%20change%E2%80%94mobile%20signals%20controlling%20photoperiod-dependent%20flowering&amp;author=Y.%20Kobayashi&amp;author=D.%20Weigel&amp;publication_year=2007&amp;journal=Genes%20Dev&amp;volume=21&amp;pages=2371-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gad.1589007" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1589007" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgad.1589007"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17908925" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Move%20on%20up%2C%20it%E2%80%99s%20time%20for%20change%E2%80%94mobile%20signals%20controlling%20photoperiod-dependent%20flowering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Zeevaart</div> <div class="given-names">J.A.</div>
</div></div> <div class="year">2008</div>, 
<div class="article-title">Leaf-produced floral signals</div>, <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">11</div>, <div class="fpage">541</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Leaf-produced%20floral%20signals&amp;author=J.A.%20Zeevaart&amp;publication_year=2008&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=11&amp;pages=541-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.pbi.2008.06.009" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2008.06.009" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.pbi.2008.06.009"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18691931" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Leaf-produced%20floral%20signals&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Turnbull</div> <div class="given-names">C.</div>
</div></div> <div class="year">2011</div>, 
<div class="article-title">Long-distance regulation of flowering time</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">62</div>, <div class="fpage">4399</div>–<div class="lpage">413</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Long-distance%20regulation%20of%20flowering%20time&amp;author=C.%20Turnbull&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot&amp;volume=62&amp;pages=4399-413" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/err191" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr191" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Ferr191"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21778182" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Long-distance%20regulation%20of%20flowering%20time&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">McGarry</div> <div class="given-names">R.C.</div>
</div>, <div class="name string-name">
<div class="surname">Ayre</div> <div class="given-names">B.G.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Manipulating plant architecture with members of the CETS gene family</div>, <div class="source">Plant Sci</div> ., <div class="volume">188–189</div>, <div class="fpage">71</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Manipulating%20plant%20architecture%20with%20members%20of%20the%20CETS%20gene%20family&amp;author=R.C.%20McGarry&amp;author=B.G.%20Ayre&amp;publication_year=2012&amp;journal=Plant%20Sci&amp;volume=188%E2%80%93189&amp;pages=71-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.plantsci.2012.03.002" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2012.03.002" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.plantsci.2012.03.002"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22525246" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Manipulating%20plant%20architecture%20with%20members%20of%20the%20CETS%20gene%20family&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oda</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Narumi</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">
<em>CsFTL3</em>, a chrysanthemum <em>FLOWERING LOCUS T-like</em> gene, is a key regulator of photoperiodic flowering in chrysanthemums</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">63</div>, <div class="fpage">1461</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CsFTL3%2C%20a%20chrysanthemum%20FLOWERING%20LOCUS%20T-like%20gene%2C%20is%20a%20key%20regulator%20of%20photoperiodic%20flowering%20in%20chrysanthemums&amp;author=A.%20Oda&amp;author=T.%20Narumi&amp;author=T.%20Li&amp;publication_year=2012&amp;journal=J.%20Exp.%20Bot&amp;volume=63&amp;pages=1461-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/err387" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr387" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Ferr387"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140240" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CsFTL3%2C%20a%20chrysanthemum%20FLOWERING%20LOCUS%20T-like%20gene%2C%20is%20a%20key%20regulator%20of%20photoperiodic%20flowering%20in%20chrysanthemums&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Higuchi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Narumi</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Oda</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">The gated induction of a systemic floral inhibitor, antiflorigen, determines obligate short-day flowering in chrysanthemums</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">17137</div>–<div class="lpage">42</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20gated%20induction%20of%20a%20systemic%20floral%20inhibitor%2C%20antiflorigen%2C%20determines%20obligate%20short-day%20flowering%20in%20chrysanthemums&amp;author=Y.%20Higuchi&amp;author=T.%20Narumi&amp;author=A.%20Oda&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=17137-42" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1307617110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1307617110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1307617110"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20gated%20induction%20of%20a%20systemic%20floral%20inhibitor%2C%20antiflorigen%2C%20determines%20obligate%20short-day%20flowering%20in%20chrysanthemums&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Van Geest</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Voorrips</div> <div class="given-names">R.E.</div>
</div>, <div class="name string-name">
<div class="surname">Esselink</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Post</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Visser</div> <div class="given-names">R.G.F.</div>
</div>, <div class="name string-name">
<div class="surname">Arens</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Conclusive evidence for hexasomic inheritance in chrysanthemum based on analysis of a 183 k SNP array</div>, <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">585</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Conclusive%20evidence%20for%20hexasomic%20inheritance%20in%20chrysanthemum%20based%20on%20analysis%20of%20a%20183%20k%20SNP%20array&amp;author=G.%20Van%20Geest&amp;author=R.E.%20Voorrips&amp;author=D.%20Esselink&amp;author=A.%20Post&amp;author=R.G.F.%20Visser&amp;author=P.%20Arens&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=585" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-017-4003-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-4003-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-017-4003-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28784083" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Conclusive%20evidence%20for%20hexasomic%20inheritance%20in%20chrysanthemum%20based%20on%20analysis%20of%20a%20183%20k%20SNP%20array&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Van Geest</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Bourke</div> <div class="given-names">P.M.</div>
</div>, <div class="name string-name">
<div class="surname">Voorrips</div> <div class="given-names">R.E.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">An ultra-dense integrated linkage map for hexaploid chrysanthemum enables multi-allelic QTL analysis</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">130</div>, <div class="fpage">2527</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20ultra-dense%20integrated%20linkage%20map%20for%20hexaploid%20chrysanthemum%20enables%20multi-allelic%20QTL%20analysis&amp;author=G.%20Van%20Geest&amp;author=P.M.%20Bourke&amp;author=R.E.%20Voorrips&amp;publication_year=2017&amp;journal=Theor.%20Appl.%20Genet&amp;volume=130&amp;pages=2527-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00122-017-2974-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-017-2974-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00122-017-2974-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28852802" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20ultra-dense%20integrated%20linkage%20map%20for%20hexaploid%20chrysanthemum%20enables%20multi-allelic%20QTL%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Drewlow</div> <div class="given-names">L.W.</div>
</div>, <div class="name string-name">
<div class="surname">Ascher</div> <div class="given-names">P.D.</div>
</div>, <div class="name string-name">
<div class="surname">Widmer</div> <div class="given-names">R.E.</div>
</div>
</div> <div class="year">1973</div>, 
<div class="article-title">Genetic studies of self-incompatibility in the garden chrysanthemum, <em>Chrysanthemum morifolium</em> Ramat</div>, <div class="source">Theor. Appl. Genet</div> ., <div class="volume">43</div>, <div class="fpage">1</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetic%20studies%20of%20self-incompatibility%20in%20the%20garden%20chrysanthemum%2C%20Chrysanthemum%20morifolium%20Ramat&amp;author=L.W.%20Drewlow&amp;author=P.D.%20Ascher&amp;author=R.E.%20Widmer&amp;publication_year=1973&amp;journal=Theor.%20Appl.%20Genet&amp;volume=43&amp;pages=1-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF00277824" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00277824" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00277824"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24424841" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetic%20studies%20of%20self-incompatibility%20in%20the%20garden%20chrysanthemum%2C%20Chrysanthemum%20morifolium%20Ramat&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ohashi</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Yonekura</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2004</div>, New combinations in <em>Chrysanthemum</em> (Compositae-Anthemideae) of Asia with a list of Japanese species, <div class="source">J. Japan. Bot</div> ., <div class="volume">79</div>, <div class="fpage">186</div>–<div class="lpage">95</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=J.%20Japan.%20Bot&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=J.%20Japan.%20Bot&amp;aulast=Ohashi&amp;date=2004&amp;spage=186&amp;epage=95&amp;volume=79" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:J.%20Japan.%20Bot&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=J.%20Japan.%20Bot">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dai</div> <div class="given-names">S.L.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">J.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">W.B.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Application of RAPD analysis in the study on the origin of Chinese cultivated chrysanthemum</div>, <div class="source">Acta Bot. Sin</div> ., <div class="volume">40</div>, <div class="fpage">1053</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Application%20of%20RAPD%20analysis%20in%20the%20study%20on%20the%20origin%20of%20Chinese%20cultivated%20chrysanthemum&amp;aulast=Dai&amp;date=1998&amp;spage=1053&amp;epage=9&amp;volume=40" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Application%20of%20RAPD%20analysis%20in%20the%20study%20on%20the%20origin%20of%20Chinese%20cultivated%20chrysanthemum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Glover</div> <div class="given-names">B.J.</div>
</div>, <div class="name string-name">
<div class="surname">Rao</div> <div class="given-names">G.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Molecular evidence for multiple polyploidization and lineage recombination in the <em>Chrysanthemum indicum</em> polyploid complex (Asteraceae)</div>, <div class="source">New Phytol</div> ., <div class="volume">171</div>, <div class="fpage">875</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20evidence%20for%20multiple%20polyploidization%20and%20lineage%20recombination%20in%20the%20Chrysanthemum%20indicum%20polyploid%20complex%20(Asteraceae)&amp;author=W.%20Yang&amp;author=B.J.%20Glover&amp;author=G.Y.%20Rao&amp;author=J.%20Yang&amp;publication_year=2006&amp;journal=New%20Phytol&amp;volume=171&amp;pages=875-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.2006.171.issue-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2006.171.issue-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.2006.171.issue-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16918557" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20evidence%20for%20multiple%20polyploidization%20and%20lineage%20recombination%20in%20the%20Chrysanthemum%20indicum%20polyploid%20complex%20(Asteraceae)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yagi</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Kosugi</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Hirakawa</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Sequence analysis of the genome of carnation (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">231</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;author=M.%20Yagi&amp;author=S.%20Kosugi&amp;author=H.%20Hirakawa&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=231-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dst053" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst053" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdst053"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24344172" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequence%20analysis%20of%20the%20genome%20of%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bombarely</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Menda</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Tecle</div> <div class="given-names">I.Y.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">The Sol Genomics Network (solgenomics.net): growing tomatoes using Perl</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">D1149</div>–<div class="lpage">55</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Sol%20Genomics%20Network%20(solgenomics.net)%3A%20growing%20tomatoes%20using%20Perl&amp;author=A.%20Bombarely&amp;author=N.%20Menda&amp;author=I.Y.%20Tecle&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=D1149-55" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkq866" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq866" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkq866"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20935049" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Sol%20Genomics%20Network%20(solgenomics.net)%3A%20growing%20tomatoes%20using%20Perl&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tuskan</div> <div class="given-names">G.A.</div>
</div>, <div class="name string-name">
<div class="surname">Difazio</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Jansson</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="volume">313</div>, <div class="fpage">1596</div>–<div class="lpage">604</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1128691"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">International Rice Genome Sequencing Project. <div class="year">2005</div>, 
<div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="volume">436</div>, <div class="fpage">793</div>–<div class="lpage">800</div>.<div class="citation-links">
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature03895"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20map-based%20sequence%20of%20the%20rice%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">The Potato Genome Sequencing Consortium. <div class="year">2011</div>, 
<div class="article-title">Genome sequence and analysis of the tuber crop potato</div>, <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">189</div>–<div class="lpage">95</div>.<div class="citation-links">
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature10158" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10158" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature10158"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743474" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20and%20analysis%20of%20the%20tuber%20crop%20potato&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shirasawa</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Hirakawa</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Isobe</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato</div>, <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">145</div>–<div class="lpage">53</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analytical%20workflow%20of%20double-digest%20restriction%20site-associated%20DNA%20sequencing%20based%20on%20empirical%20and%20in%20silico%20optimization%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;author=S.%20Isobe&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=145-53" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsw004" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw004" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsw004"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26932983" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analytical%20workflow%20of%20double-digest%20restriction%20site-associated%20DNA%20sequencing%20based%20on%20empirical%20and%20in%20silico%20optimization%20in%20tomato&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Isobe</div> <div class="given-names">S.N.</div>
</div>, <div class="name string-name">
<div class="surname">Hirakawa</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Sato</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Construction of an integrated high density simple sequence repeat linkage map in cultivated strawberry (<em>Fragaria</em> × <em>ananassa</em>) and its applicability</div>, <div class="source">DNA Res</div> ., <div class="volume">20</div>, <div class="fpage">79</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20integrated%20high%20density%20simple%20sequence%20repeat%20linkage%20map%20in%20cultivated%20strawberry%20(Fragaria%20%C3%97%20ananassa)%20and%20its%20applicability&amp;author=S.N.%20Isobe&amp;author=H.%20Hirakawa&amp;author=S.%20Sato&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=79-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dss035" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss035" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdss035"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23248204" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Construction%20of%20an%20integrated%20high%20density%20simple%20sequence%20repeat%20linkage%20map%20in%20cultivated%20strawberry%20(Fragaria%20%C3%97%20ananassa)%20and%20its%20applicability&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Reyes-Chin-Wo</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">X.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Genome assembly with in vitro proximity ligation data and whole-genome triplication in lettuce</div>, <div class="source">Nat. Commun</div> ., <div class="volume">8</div>, <div class="fpage">14953</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20assembly%20with%20in%20vitro%20proximity%20ligation%20data%20and%20whole-genome%20triplication%20in%20lettuce&amp;author=S.%20Reyes-Chin-Wo&amp;author=Z.%20Wang&amp;author=X.%20Yang&amp;publication_year=2017&amp;journal=Nat.%20Commun&amp;volume=8&amp;pages=14953" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ncomms14953" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms14953" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fncomms14953"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28401891" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20assembly%20with%20in%20vitro%20proximity%20ligation%20data%20and%20whole-genome%20triplication%20in%20lettuce&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Badouin</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Gouzy</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Grassa</div> <div class="given-names">C.J.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">The sunflower genome provides insights into oil metabolism, flowering and Asterid evolution</div>, <div class="source">Nature</div> , <div class="volume">546</div>, <div class="fpage">148</div>–<div class="lpage">52</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20sunflower%20genome%20provides%20insights%20into%20oil%20metabolism%2C%20flowering%20and%20Asterid%20evolution&amp;author=H.%20Badouin&amp;author=J.%20Gouzy&amp;author=C.J.%20Grassa&amp;publication_year=2017&amp;journal=Nature&amp;volume=546&amp;pages=148-52" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature22380" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature22380" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature22380"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28538728" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20sunflower%20genome%20provides%20insights%20into%20oil%20metabolism%2C%20flowering%20and%20Asterid%20evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Higuchi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Sumitomo</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Oda</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Shimizu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Hisamatsu</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Day light quality affects the night-break response in the short-day plant chrysanthemum, suggesting differential phytochrome-mediated regulation of flowering</div>, <div class="source">J. Plant Physiol</div> ., <div class="volume">169</div>, <div class="fpage">1789</div>–<div class="lpage">96</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Day%20light%20quality%20affects%20the%20night-break%20response%20in%20the%20short-day%20plant%20chrysanthemum%2C%20suggesting%20differential%20phytochrome-mediated%20regulation%20of%20flowering&amp;author=Y.%20Higuchi&amp;author=K.%20Sumitomo&amp;author=A.%20Oda&amp;author=H.%20Shimizu&amp;author=T.%20Hisamatsu&amp;publication_year=2012&amp;journal=J.%20Plant%20Physiol&amp;volume=169&amp;pages=1789-96" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.jplph.2012.07.003" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2012.07.003" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jplph.2012.07.003"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22840324" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Day%20light%20quality%20affects%20the%20night-break%20response%20in%20the%20short-day%20plant%20chrysanthemum%2C%20suggesting%20differential%20phytochrome-mediated%20regulation%20of%20flowering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nakano</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Higuchi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Sumitomo</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Hisamatsu</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Flowering retardation by high temperature in chrysanthemums: involvement of FLOWERING LOCUS T-like 3 gene repression</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">64</div>, <div class="fpage">909</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Flowering%20retardation%20by%20high%20temperature%20in%20chrysanthemums%3A%20involvement%20of%20FLOWERING%20LOCUS%20T-like%203%20gene%20repression&amp;author=Y.%20Nakano&amp;author=Y.%20Higuchi&amp;author=K.%20Sumitomo&amp;author=T.%20Hisamatsu&amp;publication_year=2013&amp;journal=J.%20Exp.%20Bot&amp;volume=64&amp;pages=909-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jxb/ers370" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fers370" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjxb%2Fers370"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23314814" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Flowering%20retardation%20by%20high%20temperature%20in%20chrysanthemums%3A%20involvement%20of%20FLOWERING%20LOCUS%20T-like%203%20gene%20repression&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Higuchi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Hisamatsu</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">CsTFL1, a constitutive local repressor of flowering, modulates floral initiation by antagonising florigen complex activity in chrysanthemum</div>, <div class="source">Plant Sci</div> ., <div class="volume">237</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CsTFL1%2C%20a%20constitutive%20local%20repressor%20of%20flowering%2C%20modulates%20floral%20initiation%20by%20antagonising%20florigen%20complex%20activity%20in%20chrysanthemum&amp;author=Y.%20Higuchi&amp;author=T.%20Hisamatsu&amp;publication_year=2015&amp;journal=Plant%20Sci&amp;volume=237&amp;pages=1-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.plantsci.2015.04.011" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2015.04.011" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.plantsci.2015.04.011"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26089146" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CsTFL1%2C%20a%20constitutive%20local%20repressor%20of%20flowering%2C%20modulates%20floral%20initiation%20by%20antagonising%20florigen%20complex%20activity%20in%20chrysanthemum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ho</div> <div class="given-names">W.W.</div>
</div>, <div class="name string-name">
<div class="surname">Weigel</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Structural features determining flower-promoting activity of <em>Arabidopsis</em> FLOWERING LOCUS T</div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">552</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Structural%20features%20determining%20flower-promoting%20activity%20of%20Arabidopsis%20FLOWERING%20LOCUS%20T&amp;author=W.W.%20Ho&amp;author=D.%20Weigel&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=552-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.113.115220" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.115220" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.113.115220"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24532592" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Structural%20features%20determining%20flower-promoting%20activity%20of%20Arabidopsis%20FLOWERING%20LOCUS%20T&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hisamatsu</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Sumitomo</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Shibata</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Koshioka</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Seasonal variability in dormancy and flowering competence in Chrysanthemum: chilling impacts on shoot extension growth and flowering capacity</div>, <div class="source">JARQ</div> , <div class="volume">51</div>, <div class="fpage">343</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Seasonal%20variability%20in%20dormancy%20and%20flowering%20competence%20in%20Chrysanthemum%3A%20chilling%20impacts%20on%20shoot%20extension%20growth%20and%20flowering%20capacity&amp;author=T.%20Hisamatsu&amp;author=K.%20Sumitomo&amp;author=M.%20Shibata&amp;author=M.%20Koshioka&amp;publication_year=2017&amp;journal=JARQ&amp;volume=51&amp;pages=343-50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.6090/jarq.51.343" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.6090%2Fjarq.51.343" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.6090%2Fjarq.51.343"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Seasonal%20variability%20in%20dormancy%20and%20flowering%20competence%20in%20Chrysanthemum%3A%20chilling%20impacts%20on%20shoot%20extension%20growth%20and%20flowering%20capacity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Panero</div> <div class="given-names">J.L.</div>
</div>, <div class="name string-name">
<div class="surname">Funk</div> <div class="given-names">V.A.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">The value of sampling anomalous taxa in phylogenetic studies: major clades of the Asteraceae revealed</div>, <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">47</div>, <div class="fpage">757</div>–<div class="lpage">82</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20value%20of%20sampling%20anomalous%20taxa%20in%20phylogenetic%20studies%3A%20major%20clades%20of%20the%20Asteraceae%20revealed&amp;author=J.L.%20Panero&amp;author=V.A.%20Funk&amp;publication_year=2008&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=47&amp;pages=757-82" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ympev.2008.02.011" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.02.011" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2008.02.011"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375151" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20value%20of%20sampling%20anomalous%20taxa%20in%20phylogenetic%20studies%3A%20major%20clades%20of%20the%20Asteraceae%20revealed&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hibrand-Saint Oyant</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Ruttink</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Hamama</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">A high-quality genome sequence of <em>Rosa chinensis</em> to elucidate ornamental traits</div>, <div class="source">Nat. Plants</div> , <div class="volume">4</div>, <div class="fpage">473</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20high-quality%20genome%20sequence%20of%20Rosa%20chinensis%20to%20elucidate%20ornamental%20traits&amp;author=L.%20Hibrand-Saint%20Oyant&amp;author=T.%20Ruttink&amp;author=L.%20Hamama&amp;publication_year=2018&amp;journal=Nat.%20Plants&amp;volume=4&amp;pages=473-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41477-018-0166-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41477-018-0166-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41477-018-0166-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29892093" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20high-quality%20genome%20sequence%20of%20Rosa%20chinensis%20to%20elucidate%20ornamental%20traits&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hirakawa</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Shirasawa</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Kosugi</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of <em>Fragaria</em> species</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdst049"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy048-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy048-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shirasawa</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Tanaka</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Takahata</div> <div class="given-names">Y.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">A high-density SNP genetic map consisting of a complete set of homologous groups in autohexaploid sweetpotato (<em>Ipomoea batatas</em>)</div>, <div class="source">Sci. Rep</div> <em>.,</em> <div class="volume">7</div>, <div class="fpage">44207</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20SNP%20genetic%20map%20consisting%20of%20a%20complete%20set%20of%20homologous%20groups%20in%20autohexaploid%20sweetpotato%20(Ipomoea%20batatas)&amp;author=K.%20Shirasawa&amp;author=M.%20Tanaka&amp;author=Y.%20Takahata&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=44207" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep44207" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep44207" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep44207"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28281636" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20high-density%20SNP%20genetic%20map%20consisting%20of%20a%20complete%20set%20of%20homologous%20groups%20in%20autohexaploid%20sweetpotato%20(Ipomoea%20batatas)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="dsy048-FM2"><div class="fn"><div class="footnote-content">Hideki Hirakawa and Katsuhiko Sumitomo authors contributed equally to this work.</div></div></div>
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5303208&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/3/10.1093_dnares_dsy048/1/dsy048_supplementary_data.zip?Expires=1576701829&amp;Signature=yGR9776CFX9nKlGz-3uKAs5WkRvHf9ELB8CCoUk6Vh4Gn22wErTyhSDKDKwaU4rfUllb-QSwpZ3W8i8qQhBNSmW3uUYDHhGu7zUJdt60ketksUILKU3JnhyIW~3aGnmzgEMO0w1BOtkH6kRyXlj2DX9V4myA7gKJMTjBAxwRj5TwoP0kfh78EWWf2Gtbn~3OOP8LWY9pTPuBAchKE6UebZCH9Fg~vuEd7xzTPutHJ6lkm~vvWHEEKSPcEQCVmhWSzoJIvdswC~aFbjWq6skQtkLrtXjs-nqQFDthYKwlReVHX3PWK9mKP9zyyd-09~lRKg-ZihW3m~DGQzQsuebgtQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsy048_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">3,889</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy048"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5303208"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493010500001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f3%2f195%2f5303208">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493010500001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F3%2F195%2F5303208" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5303208">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="De novo whole-genome assembly in Chrysanthemum seticuspe, a model species of Chrysanthemums, and its application to genetic and gene discovery analysis | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy048");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f3%2f195%2f5303208">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
