<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>genome of Populus alba x Populus tremula var. glandulosa clone 84K | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/5/423/5580662","@type":"ScholarlyArticle","name":"The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K","datePublished":"2019-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2019-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz020","keywords":["poplar 84K","genome sequencing","subgenome assignment","<italic>P. alba</italic>","<italic>P. tremula</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/5/423/5580662","author":[{"name":"Qiu, Deyou","affiliation":"State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The  Research Institute of Forestry, Chinese Academy of Forestry , Beijing, China","@type":"Person"},{"name":"Bai, Shenglong","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China","@type":"Person"},{"name":"Ma, Jianchao","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China","@type":"Person"},{"name":"Zhang, Lisha","affiliation":"State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The  Research Institute of Forestry, Chinese Academy of Forestry , Beijing, China","@type":"Person"},{"name":"Shao, Fenjuan","affiliation":"State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The  Research Institute of Forestry, Chinese Academy of Forestry , Beijing, China","@type":"Person"},{"name":"Zhang, Kaikai","affiliation":"State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The  Research Institute of Forestry, Chinese Academy of Forestry , Beijing, China   College of Horticulture, Agricultural University of Hebei , Baoding, China","@type":"Person"},{"name":"Yang, Yanfang","affiliation":"State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The  Research Institute of Forestry, Chinese Academy of Forestry , Beijing, China","@type":"Person"},{"name":"Sun, Ting","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China","@type":"Person"},{"name":"Huang, Jinling","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China","@type":"Person"},{"name":"Zhou, Yun","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China","@type":"Person"},{"name":"Galbraith, David W","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China   School of Plant Sciences and Bio5 Institute, The University of Arizona , Tucson, AZ, USA","@type":"Person"},{"name":"Wang, Zhaoshan","affiliation":"State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The  Research Institute of Forestry, Chinese Academy of Forestry , Beijing, China","@type":"Person"},{"name":"Sun, Guiling","affiliation":"Key Laboratory of Plant Stress Biology,  State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University , Kaifeng, China","@type":"Person"}],"description":"Abstract.  Poplar 84K (Populus alba x P. tremula var. glandulosa) is a fast-growing poplar hybrid. Originated in South Korea, this hybrid has been extensively c","pageStart":"423","pageEnd":"431","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1636242816&Signature=P3VDy0NXQeRrHrxEAq9yFnWEhiMeIJDhYsVL1S0N-hTvXE5LerYHtgAfyV5LcG8Bjhj4W958T8YDfzL4xsZ-Yi~oQ0nsuaRe4GYL3mHPu-~AwVgWR4LDZ0FAZZIZ~VaUPpIWSsJOli5AlEGUlgaKzO5YcRir0T~aFHwDMjAgPm-vGtg-4gSPcgCWlFizBU~YmOvZ7tlcrqnbmsVxKmt3pxNlM0m2XpEt5e96ABsD-Gp0uQ2e41uJooSYFbMVGMCnWbgibffFfqct4Gi~GHwJaOx8OYDFcLmTimQmKYl3fFkWxIiJCtlBT8VISDqaQoH5X-gopqjtH8ntCnn-afy6PA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1636242816&Signature=P3VDy0NXQeRrHrxEAq9yFnWEhiMeIJDhYsVL1S0N-hTvXE5LerYHtgAfyV5LcG8Bjhj4W958T8YDfzL4xsZ-Yi~oQ0nsuaRe4GYL3mHPu-~AwVgWR4LDZ0FAZZIZ~VaUPpIWSsJOli5AlEGUlgaKzO5YcRir0T~aFHwDMjAgPm-vGtg-4gSPcgCWlFizBU~YmOvZ7tlcrqnbmsVxKmt3pxNlM0m2XpEt5e96ABsD-Gp0uQ2e41uJooSYFbMVGMCnWbgibffFfqct4Gi~GHwJaOx8OYDFcLmTimQmKYl3fFkWxIiJCtlBT8VISDqaQoH5X-gopqjtH8ntCnn-afy6PA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K">
<meta property="og:description" content="Abstract.  Poplar 84K (Populus alba x P. tremula var. glandulosa) is a fast-growing poplar hybrid. Originated in South Korea, this hybrid has been extensively c">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/5/423/5580662">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1636242816&amp;Signature=P3VDy0NXQeRrHrxEAq9yFnWEhiMeIJDhYsVL1S0N-hTvXE5LerYHtgAfyV5LcG8Bjhj4W958T8YDfzL4xsZ-Yi~oQ0nsuaRe4GYL3mHPu-~AwVgWR4LDZ0FAZZIZ~VaUPpIWSsJOli5AlEGUlgaKzO5YcRir0T~aFHwDMjAgPm-vGtg-4gSPcgCWlFizBU~YmOvZ7tlcrqnbmsVxKmt3pxNlM0m2XpEt5e96ABsD-Gp0uQ2e41uJooSYFbMVGMCnWbgibffFfqct4Gi~GHwJaOx8OYDFcLmTimQmKYl3fFkWxIiJCtlBT8VISDqaQoH5X-gopqjtH8ntCnn-afy6PA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-10-03">
<meta name="citation_author" content="Qiu, Deyou">
<meta name="citation_author_institution" content="State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China">
<meta name="citation_author" content="Bai, Shenglong">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_author" content="Ma, Jianchao">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_author" content="Zhang, Lisha">
<meta name="citation_author_institution" content="State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China">
<meta name="citation_author" content="Shao, Fenjuan">
<meta name="citation_author_institution" content="State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China">
<meta name="citation_author" content="Zhang, Kaikai">
<meta name="citation_author_institution" content="State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China">
<meta name="citation_author_institution" content="College of Horticulture, Agricultural University of Hebei, Baoding, China">
<meta name="citation_author" content="Yang, Yanfang">
<meta name="citation_author_institution" content="State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China">
<meta name="citation_author" content="Sun, Ting">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_author" content="Huang, Jinling">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_author" content="Zhou, Yun">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_author" content="Galbraith, David W">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_author_institution" content="School of Plant Sciences and Bio5 Institute, The University of Arizona, Tucson, AZ, USA">
<meta name="citation_author" content="Wang, Zhaoshan">
<meta name="citation_author_institution" content="State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The Research Institute of Forestry, Chinese Academy of Forestry, Beijing, China">
<meta name="citation_author" content="Sun, Guiling">
<meta name="citation_author_institution" content="Key Laboratory of Plant Stress Biology, State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University, Kaifeng, China">
<meta name="citation_title" content="The genome of Populus alba x Populus tremula var. glandulosa clone 84K">
<meta name="citation_firstpage" content="423">
<meta name="citation_lastpage" content="431">
<meta name="citation_doi" content="10.1093/dnares/dsz020">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="5">
<meta name="citation_publication_date" content="2019/10/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_publisher=NRC Research Press, Ottawa, ;  citation_title=Biology of Populus and Its Implications for Management and Conservation;  citation_year=1996; ">
<meta name="citation_reference" content="citation_title=Poplar genome sequence: functional genomics in an ecologically dominant plant species;  citation_journal_title=Trends Plant Sci;  citation_year=2004;  citation_volume=9;  citation_pages=49-56; ">
<meta name="citation_reference" content="citation_title=Genomics and forest biology: populus emerges as the perennial favorite;  citation_journal_title=Plant Cell;  citation_year=2002;  citation_volume=14;  citation_pages=2651-5; ">
<meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray);  citation_journal_title=Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; ">
<meta name="citation_reference" content="citation_title=Genomic insights into salt adaptation in a desert poplar;  citation_journal_title=Nat. Commun;  citation_year=2013;  citation_volume=4; ">
<meta name="citation_reference" content="citation_title=The draft genome sequence of a desert tree Populus pruinosa;  citation_journal_title=Gigascience;  citation_year=2017;  citation_volume=6;  citation_pages=1-7; ">
<meta name="citation_reference" content="citation_title=Genome sequence and genetic transformation of a widely distributed and cultivated poplar;  citation_journal_title=Plant Biotechnol. J;  citation_year=2019;  citation_volume=17;  citation_pages=451-60; ">
<meta name="citation_reference" content="citation_title=Genetic transformation of Populus trichocarpa genotype Nisqually-1: a functional genomic tool for woody plants;  citation_journal_title=Plant Cell Physiol;  citation_year=2006;  citation_volume=47;  citation_pages=1582-9; ">
<meta name="citation_reference" content="citation_title=Salt tolerance conferred by over-expression of OsNHX1 gene in Poplar 84K;  citation_journal_title=Chinese Sci. Bull;  citation_year=2005;  citation_volume=50;  citation_pages=224-8; ">
<meta name="citation_reference" content="citation_title=Enhanced expression of glutamine synthetase (GS1a) confers altered fibre and wood chemistry in field grown hybrid poplar (Populus tremula X alba) (717-1B4);  citation_journal_title=Plant Biotechnol J;  citation_year=2012;  citation_volume=10;  citation_pages=883-9; ">
<meta name="citation_reference" content="citation_title=Characterization and varied expression of a membrane-bound endo-beta-1,4-glucanase in hybrid poplar;  citation_journal_title=Plant Biotechnol J;  citation_year=2010;  citation_volume=8;  citation_pages=294-307; ">
<meta name="citation_reference" content="citation_title=Overexpression of PtrMYB119, a R2R3-MYB transcription factor from Populus trichocarpa, promotes anthocyanin production in hybrid poplar;  citation_journal_title=Tree Physiol;  citation_year=2016;  citation_volume=36;  citation_pages=1162-76; ">
<meta name="citation_reference" content="citation_title=Whole-genome draft assembly of Populus tremula x P. alba clone INRA 717-1B4;  citation_journal_title=Silvae Genet;  citation_year=2016;  citation_volume=65;  citation_pages=74-9; ">
<meta name="citation_reference" content="citation_title=Transgenic poplar trees expressing yeast cadmium factor 1 exhibit the characteristics necessary for the phytoremediation of mine tailing soil;  citation_journal_title=Chemosphere;  citation_year=2013;  citation_volume=90;  citation_pages=1478-86; ">
<meta name="citation_reference" content="citation_title=Isolation of plant DNA from fresh tissue;  citation_journal_title=Focus;  citation_year=1990;  citation_volume=12;  citation_pages=13-5; ">
<meta name="citation_reference" content="citation_title=Rapid flow cytometric analysis of the cell cycle in intact plant tissues;  citation_journal_title=Science;  citation_year=1983;  citation_volume=220;  citation_pages=1049-51; ">
<meta name="citation_reference" content="Liu B., Shi Y., Yuan J.  2013 , Estimation of genomic characteristics by analyzing k-mer frequency in de novo genome projects. arXiv , 1308.2012. ">
<meta name="citation_reference" content="citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation;  citation_journal_title=Genome Res;  citation_year=2017;  citation_volume=27;  citation_pages=722-36; ">
<meta name="citation_reference" content="citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=563-9; ">
<meta name="citation_reference" content="citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e112963">
<meta name="citation_reference" content="citation_title=BLAST+: architecture and applications;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=421">
<meta name="citation_reference" content="citation_title=Genome sequences of Populus tremula chloroplast and mitochondrion: implications for holistic poplar breeding;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0147209">
<meta name="citation_reference" content="citation_title=GeSeq - versatile and accurate annotation of organelle genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=W6-W11; ">
<meta name="citation_reference" content="citation_title=Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=1119-25; ">
<meta name="citation_reference" content="citation_title=HiCPlotter integrates genomic data with interaction matrices;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=198">
<meta name="citation_reference" content="citation_title=HISAT: a fast spliced aligner with low memory requirements;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=357-60; ">
<meta name="citation_reference" content="Li H.  2013 , Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint arXiv: 1303.3997. ">
<meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; ">
<meta name="citation_reference" content="citation_title=Assessing genome assembly quality using the LTR Assembly Index (LAI);  citation_journal_title=Nucleic Acids Res;  citation_year=2018;  citation_volume=46;  citation_pages=e126">
<meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; ">
<meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; ">
<meta name="citation_reference" content="citation_title=Using RepeatMasker to identify repetitive elements in genomic sequences;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2004; ">
<meta name="citation_reference" content="citation_title=Repbase update, a database of repetitive elements in eukaryotic genomes;  citation_journal_title=Mob. DNA;  citation_year=2015;  citation_volume=6;  citation_pages=11">
<meta name="citation_reference" content="citation_title=The Dfam database of repetitive DNA families;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D81-89; ">
<meta name="citation_reference" content="citation_title=Non-coding RNA analysis using the Rfam database;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2018;  citation_volume=62;  citation_pages=e51">
<meta name="citation_reference" content="citation_title=Infernal 1.1: 100-fold faster RNA homology searches;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=2933-5; ">
<meta name="citation_reference" content="citation_title=tRNAscan-SE on-line: integrating search and context for analysis of transfer RNA genes;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=W54-57; ">
<meta name="citation_reference" content="citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=3100-8; ">
<meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49">
<meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes;  citation_journal_title=Genome Biol;  citation_year=2004;  citation_volume=5;  citation_pages=R12">
<meta name="citation_reference" content="citation_title=Functional and evolutionary genomic inferences in Populus through genome and population sequencing of American and European aspen;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2018;  citation_volume=115;  citation_pages=E10970-E10978; ">
<meta name="citation_reference" content="citation_title=Correlation between nuclear plastid DNA abundance and plastid number supports the limited transfer window hypothesis;  citation_journal_title=Genome Biol. Evol;  citation_year=2011;  citation_volume=3;  citation_pages=365-71; ">
<meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; ">
<meta name="citation_reference" content="citation_title=Assemblytics: a web analytics tool for the detection of variants from an assembly;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=3021-3; ">
<meta name="citation_reference" content="citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=290-5; ">
<meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=139-40; ">
<meta name="citation_reference" content="citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=157">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/5/423/30159666/dsz020.pdf">
<meta name="description" content="Abstract.  Poplar 84K (Populus alba x P. tremula var. glandulosa) is a fast-growing poplar hybrid. Originated in South Korea, this hybrid has been extensively c">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/5/423/5580662">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/5/423/5580662">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/5/423/5580662");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2779951,2796125&authnips=128.235.85.32&doi=10.1093/dnares/dsz020';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f423%2f5580662" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f423%2f5580662" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=116936"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/5/1/m_dnares_26_5cover.png?Expires=1576718527&amp;Signature=QYjleWFKrWyKBT8i7t2S1cnOzLHDDWKXmCRlzOv~U8IsrHnTo2Bxp55GE73blFlKEzQUFcUE-zHaQOFkjaxmG-SFxKhkWVFP2jKOe~MLfjblrRNOL0hNGZzUqTOxkAG1x9lxg5uMHpNJuyWE2fUA-FMlCECA51faZAhZDzCfdpx~OI1~NlMHyDU4rc7AMgZhU8LqkYAVXpMD9Osg3pF-cyqgTJtLxBILPYNZXl9bVGzG1lYif2xhSOvChBDtKy98AvB1cZSy0b7zd5J07bZPte7Yu9ykBd-6uSc08FT~ETUISR~O9p5btaA~LgjLbXwrBrpaJ8hOeAwm05fO1YxMdQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/5">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="164508446">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508446">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508448">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508448">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508452">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508452">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508477">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508477">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508504">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508504">4. Summary and conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508506">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508506">Data availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="164508508">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508508">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="164508512">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#164508512">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/5/411/5556422">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/5/433/5588861">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    The genome of <em>Populus alba</em> x <em>Populus tremula</em> var. <em>glandulosa</em> clone 84K
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deyou Qiu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The <div class="insititution">Research Institute of Forestry, Chinese Academy of Forestry</div>, Beijing, China</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed: Tel. +86 010 62889641. Fax. +81 010 22872015. Email: <a href="mailto:qiudy@caf.ac.cn" target="_blank">qiudy@caf.ac.cn</a> (D.Q.); Tel. +86 0371 23881387. Fax. +81 0371 23881387. Email: <a href="mailto:sung@vip.henu.edu.cn" target="_blank">sung@vip.henu.edu.cn</a> (G.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deyou+Qiu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qiu%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Qiu%20Deyou%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Deyou Qiu</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shenglong Bai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shenglong+Bai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bai%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Bai%20Shenglong%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Shenglong Bai</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jianchao Ma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jianchao+Ma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ma%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ma%20Jianchao%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Jianchao Ma</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lisha Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The <div class="insititution">Research Institute of Forestry, Chinese Academy of Forestry</div>, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lisha+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20Lisha%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Lisha Zhang</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fenjuan Shao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The <div class="insititution">Research Institute of Forestry, Chinese Academy of Forestry</div>, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fenjuan+Shao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shao%20F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Shao%20Fenjuan%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Fenjuan Shao</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kaikai Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The <div class="insititution">Research Institute of Forestry, Chinese Academy of Forestry</div>, Beijing, China</div>
<div class="aff">
<div class="insititution">College of Horticulture, Agricultural University of Hebei</div>, Baoding, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kaikai+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20Kaikai%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Kaikai Zhang</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanfang Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The <div class="insititution">Research Institute of Forestry, Chinese Academy of Forestry</div>, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanfang+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yang%20Yanfang%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Yanfang Yang</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ting Sun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ting+Sun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sun%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sun%20Ting%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Ting Sun</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jinling Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jinling+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Huang%20Jinling%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Jinling Huang</a><span class="delimiter">, </span>
                                </span>
                                <span class="al-author-name">



                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yun Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yun+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhou%20Yun%22">Google Scholar</a>
            </div>

</div>
                                    </span>



                                    <a class="linked-name" href="javascript:;">Yun Zhou</a><span class="delimiter-hidden">, </span>
                                </span>
                                    <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                        <span>...</span> Show more
                                    </a>
                                    <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David W Galbraith</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
<div class="aff">
<div class="insititution">School of Plant Sciences and Bio5 Institute, The University of Arizona</div>, Tucson, AZ, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+W+Galbraith">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Galbraith%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Galbraith%20David%20W%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">David W Galbraith</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhaoshan Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Tree Genetics and Breeding, Key Laboratory of Tree Breeding and Cultivation of National Forestry and Grassland Administration, The <div class="insititution">Research Institute of Forestry, Chinese Academy of Forestry</div>, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhaoshan+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang%20Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wang%20Zhaoshan%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Zhaoshan Wang</a><span class="delimiter">, </span>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guiling Sun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Plant Stress Biology, <div class="insititution">State Key Laboratory of Cotton Biology, School of Life Sciences, Henan University</div>, Kaifeng, China</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed: Tel. +86 010 62889641. Fax. +81 010 22872015. Email: <a href="mailto:qiudy@caf.ac.cn" target="_blank">qiudy@caf.ac.cn</a> (D.Q.); Tel. +86 0371 23881387. Fax. +81 0371 23881387. Email: <a href="mailto:sung@vip.henu.edu.cn" target="_blank">sung@vip.henu.edu.cn</a> (G.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guiling+Sun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sun%20G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sun%20Guiling%22">Google Scholar</a>
            </div>

</div>
                                            </span>
  
                                            
                                            <a class="linked-name" href="javascript:;">Guiling Sun</a><span class="delimiter"></span>
                                        </span>
                                    </div>
                            </div>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsz020-FM1"><div class="fn"><div class="footnote-content">Deyou Qiu, Shenglong Bai and Jianchao Ma authors contributed equally to this work.</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 5, October 2019, Pages 423–431, <a href="https://doi.org/10.1093/dnares/dsz020">https://doi.org/10.1093/dnares/dsz020</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">03 October 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">28 April 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">09 September 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">03 October 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5580662" href="/dnaresearch/article-pdf/26/5/423/30159666/dsz020.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Deyou Qiu, Shenglong Bai, Jianchao Ma, Lisha Zhang, Fenjuan Shao, Kaikai Zhang, Yanfang Yang, Ting Sun, Jinling Huang, Yun Zhou, David W Galbraith, Zhaoshan Wang, Guiling Sun,  The genome of <em>Populus alba</em> x <em>Populus tremula</em> var. <em>glandulosa</em> clone 84K, <em>DNA Research</em>, Volume 26, Issue 5, October 2019, Pages 423–431, <a href="https://doi.org/10.1093/dnares/dsz020">https://doi.org/10.1093/dnares/dsz020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5580662&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20genome%20of%20%26lt%3Bem%26gt%3BPopulus%20alba%26lt%3B%2Fem%26gt%3B%20x%20%26lt%3Bem%26gt%3BPopulus%20tremula%26lt%3B%2Fem%26gt%3B%20var.%20%26lt%3Bem%26gt%3Bglandulosa%26lt%3B%2Fem%26gt%3B%20clone%2084K&amp;publicationDate=2019-10-03&amp;volumeNum=26&amp;issueNum=5&amp;author=Qiu%2C%20Deyou%3B%20Bai%2C%20Shenglong&amp;startPage=423&amp;endPage=431&amp;contentId=10.1093%2Fdnares%2Fdsz020&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f423%2f5580662" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="164508446" id="164508446" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Poplar 84K (<em>Populus alba</em> x <em>P. tremula</em> var. <em>glandulosa</em>) is a fast-growing poplar hybrid. Originated in South Korea, this hybrid has been extensively cultivated in northern China. Due to the economic and ecological importance of this hybrid and high transformability, we now report the <em>de novo</em> sequencing and assembly of a male individual of poplar 84K using PacBio and Hi-C technologies. The final reference nuclear genome (747.5 Mb) has a contig N50 size of 1.99 Mb and a scaffold N50 size of 19.6 Mb. Complete chloroplast and mitochondrial genomes were also assembled from the sequencing data. Based on similarities to the genomes of <em>P. alba</em> var. <em>pyramidalis</em> and <em>P. tremula</em>, we were able to identify two subgenomes, representing 356 Mb from <em>P. alba</em> (subgenome A) and 354 Mb from <em>P. tremula</em> var. <em>glandulosa</em> (subgenome G). The phased assembly allowed us to detect the transcriptional bias between the two subgenomes, and we found that the subgenome from <em>P. tremula</em> displayed dominant expression in both 84K and another widely used hybrid, <em>P. tremula</em> x <em>P. alba</em>. This high-quality poplar 84K genome will be a valuable resource for poplar breeding and for molecular biology studies.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"poplar 84K"'>poplar 84K</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genome sequencing"'>genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"subgenome assignment"'>subgenome assignment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"P. alba"'><em>P. alba</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"P. tremula"'><em>P. tremula</em></a>
</div>                    <h2 scrollto-destination="164508448" id="164508448" class="section-title">1. Introduction</h2>
<p>The genus <em>Populus</em> comprises around 30 economically and ecologically important species.<a reveal-id="dsz020-B1" data-open="dsz020-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Besides wood products, these species provide a range of services including bioenergy production, carbon sequestration, bioremediation, nutrient cycling, biofiltration, and habitat diversification.<a reveal-id="dsz020-B2" data-open="dsz020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Due to its modest genome size, rapid growth rate, simple vegetative propagation, and short rotation cycle, coupled to high levels of genetic diversity as well as facile genetic manipulation, this genus has become a model for the study of tree molecular biology.<a reveal-id="dsz020-B3" data-open="dsz020-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Up to now, several high-quality poplar genomes have become available, including <em>P. trichocarpa</em> (Torr. and Gray),<a reveal-id="dsz020-B4" data-open="dsz020-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>P. euphratica</em> Oliv,<a reveal-id="dsz020-B5" data-open="dsz020-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><em>P. pruinosa</em>,<a reveal-id="dsz020-B6" data-open="dsz020-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and <em>P. alba</em> var. <em>pyramidalis.</em><a reveal-id="dsz020-B7" data-open="dsz020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Unfortunately, these species are not easily transformed by <em>Agrobacterium tumefaciens.</em><a reveal-id="dsz020-B8" data-open="dsz020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Interestingly, it seems that only hybrid poplar species, which have much higher transformation rates, have been widely used for genetic transformation, examples including <em>P. alba</em> x <em>P. tremula var. glandulosa</em> clone 84K,<a reveal-id="dsz020-B9" data-open="dsz020-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><em>P. tremula</em> x <em>P. alba</em>,<a reveal-id="dsz020-B10" data-open="dsz020-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><em>P. alba</em> x <em>P. grandidentata</em>,<a reveal-id="dsz020-B11" data-open="dsz020-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and <em>P. alba</em> x <em>P. tremula</em>.<a reveal-id="dsz020-B12" data-open="dsz020-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p>
<p>White poplar <em>P. alba</em>, commonly called silver poplar, is widely distributed in central Europe and central Asia. It requires abundant light and ample moisture, and stands up well to flooding. Due to its attractive green-and-white leaves, it is often planted as an ornamental tree. Its extensive root system and good tolerance of salt make it an effective tree on windy coasts. This species has also been widely used in short rotation plantation for timber and pulp production. <em>P. tremula</em>, commonly called aspen, is widely distributed in cool temperate regions of Europe and Asia. It is usually found in mountains at high altitudes. <em>P. tremula</em> is a very hardy species and can tolerate long, cold winters. It has been widely planted for timber, firewood, and veneer production. Natural populations of <em>P. alba</em> and <em>P. tremula</em> or its varieties hybridize frequently and have been selected as the parental species for artificial hybrid breeding. Two main hybrids, <em>P. tremula</em> x <em>P. alba</em> and <em>P. alba</em> x <em>P. tremula var. glandulosa</em>, have been frequently used in molecular biological studies due to their high rates of transformation. In 2016, Mader et al. reported a 900 Mb draft genome of a female interspecific hybrid <em>P. tremula</em> x <em>P. alba</em> clone INRA 717-1B4, with a N50 contig length of 3,850 bp.<a reveal-id="dsz020-B13" data-open="dsz020-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> However, the genome sequence of this hybrid has not been assembled to the level of chromosomes. Evidently, further development of high-quality genome sequences in hybrid poplar species having high transformation efficiencies will be critical to advances in molecular biology and genetic engineering of this woody genus.</p>
<p>Here we focus on the clone 84K, a male <em>P. alba</em> x <em>P. tremula var. glandulosa</em> interspecific hybrid. This hybrid resulted from a breeding program led by Professor SinKyu Hyun (Seoul National University, Korea), and was first introduced into China in 1984 by Professor Qiwen Zhang (The Research Institute of Forestry, Chinese Academy of Forestry, Beijing). Attractive characteristics of this interspecific hybrid include a high growth rate and excellent adaptation to diverse environments. This superior clone has been commonly used in short rotation plantation for timber, firewood, and pulp production. More importantly, it is easily accessible for genetic transformation using <em>A. tumefaciens</em>, and this clone is widely used by scientists in transgenic experiments as a model for woody species.<a reveal-id="dsz020-B14" data-open="dsz020-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Many transgenic poplar lines based on this clone have been created for commercial applications in China. Regrettably, no genome sequence is currently available for this clone. In this study, we describe a <em>de novo</em> assembly of the genome sequence of the hybrid poplar (<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em>) clone 84K (poplar 84K, hereafter), and identify two subgenomes via comparison to the genomes of <em>P. alba</em> var. <em>pyramidalis</em> and <em>P. tremula</em>. The genome resources of this hybrid will facilitate further gene functional analyses, optimization of genetic transformation experiments, and poplar breeding practices, as well as comparative genomic analyses across different poplars.</p>                    <h2 scrollto-destination="164508452" id="164508452" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="164508453" id="164508453" class="section-title">2.1. Plant materials</h3>
<p>Poplar 84K (<em>P. alba</em> x <em>P. tremula var. glandulosa</em>) was grown in the greenhouse at 20 °C, under a 12-h light/12-h dark illumination cycle. Fresh young leaves were collected from 1-month-old plants and immediately frozen in liquid nitrogen. Genomic DNA was isolated for library construction following the steps of the CTAB method described by Doyle et al.<a reveal-id="dsz020-B15" data-open="dsz020-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Total RNA of the leaves and shoots from 1-month-old plants was extracted with TRI Reagent (Sigma, St. Louis, MO) for transcriptome sequencing (RNA-seq).</p>                    <h3 scrollto-destination="164508455" id="164508455" class="section-title">2.2. Genome sequencing and RNA-seq</h3>
<p>Two DNA libraries with 270 and 500 bp insertions, and one DNA library with 20 kb insertions were constructed and separately sequenced on Illumina NovaSeq 6000 and PacBio Sequel platforms. Detailed methods for DNA library construction can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span>. For transcriptome sequencing (RNA-seq), four RNA libraries for leaves and one library for shoots were constructed according to the TrueSeq<sup>®</sup> RNA Sample Preparation protocol, and were sequenced on an Illumina NovaSeq sequencing system.</p>                    <h3 scrollto-destination="164508457" id="164508457" class="section-title">2.3. Estimation of genome size</h3>
<p>The 1C value of poplar 84K was measured using flow cytometry with propidium iodide (PI) as the DNA stain and <em>Arabidopsis thaliana</em> (col-0) as the standard plant as described previously.<a reveal-id="dsz020-B16" data-open="dsz020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The genome size of poplar 84K was calculated using the equation provided by the regression analysis and the known <em>A. thaliana</em> 2C DNA content (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span> for details). We also used 28.44 Gb of Illumina NovaSeq short reads to estimate the genome size and other features using the GCE software based on <em>k-mer</em> depth-frequency distribution.<a reveal-id="dsz020-B17" data-open="dsz020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The versions and main parameters of GCE and other software packages used in this study are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                    <h3 scrollto-destination="164508459" id="164508459" class="section-title">2.4. Assembly of the poplar 84K genome sequences</h3>
<p>SMRT subreads were corrected, trimmed, and assembled using CANU<a reveal-id="dsz020-B18" data-open="dsz020-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span> for details), and then polished the draft assembly using Arrow.<a reveal-id="dsz020-B19" data-open="dsz020-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Lastly, Pilon<a reveal-id="dsz020-B20" data-open="dsz020-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> was used to perform two rounds of error correction using Illumina NovaSeq reads from the 270 and 500 bp insert libraries.</p>                    <h3 scrollto-destination="164508461" id="164508461" class="section-title">2.5. Identification of potential contamination and organelle genomes</h3>
<p>For contigs less than 1 Mb, we performed a BLASTN<a reveal-id="dsz020-B21" data-open="dsz020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> search against the non-redundant nucleotide (NT, downloaded on 4 March 2018) database in GenBank with an <em>E</em>-value of 1E-5. Contigs having the most matches to non-plant species were designated as environmental sequence contamination. For the identification of the chloroplast genome, the complete chloroplast genome sequence of <em>P. tremula</em> x <em>P. alba</em> (accession number: <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_028504.1">NC_028504.1</a>)<a reveal-id="dsz020-B22" data-open="dsz020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was used as a query to search against all the remaining PacBio contigs with an <em>E</em>-value of 1E-5. The mitochondrial genome of <em>P. tremula</em> x <em>P. alba</em> (accession number: <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_028329.1">NC_028329.1</a>)<a reveal-id="dsz020-B22" data-open="dsz020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> was chosen to search against all remaining PacBio contigs with an <em>E</em>-value of 1E-5 for mitochondrial fragments. Gene structure annotations and figures of the organelle genomes were produced using GESEQ.<a reveal-id="dsz020-B23" data-open="dsz020-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p>                    <h3 scrollto-destination="164508463" id="164508463" class="section-title">2.6. Pseudomolecule construction</h3>
<p>Hi-C mapping was employed to facilitate pseudomolecule construction and the details were described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span>. The PacBio contigs were divided into fragments having a length of 300 kb, and were then error corrected, clustered, ordered, and orientated by the LACHESIS software<a reveal-id="dsz020-B24" data-open="dsz020-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> operating on the valid interaction read pair dataset. Finally, contact maps were plotted using the HiCPlotter software.<a reveal-id="dsz020-B25" data-open="dsz020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>                    <h3 scrollto-destination="164508465" id="164508465" class="section-title">2.7. Evaluation of genome completeness, continuity, and accuracy</h3>
<p>To evaluate the completeness of genome assembly, we checked the mapping rates by aligning RNA-seq reads from five libraries and DNA short reads from four libraries to the final assembly using HISAT2<a reveal-id="dsz020-B26" data-open="dsz020-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and BWA-MEM,<a reveal-id="dsz020-B27" data-open="dsz020-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> respectively, and performed BUSCO analysis.<a reveal-id="dsz020-B28" data-open="dsz020-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> We assessed the continuity of the final assembly of poplar 84K along with two other poplar genomes, <em>P. trichocarpa</em><a reveal-id="dsz020-B4" data-open="dsz020-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and <em>P. deltoides</em> (assembly version 2.1, produced by the US Department of Energy Joint Genome Institute in collaboration with the user community), using LTR_retriever<a reveal-id="dsz020-B29" data-open="dsz020-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> by calculating the LTR Assembly Index (LAI) score that evaluates the <em>de novo</em> assembly quality of intergenic and repetitive regions. Raw LAI is defined as the ratio of intact LTR retrotransposon length to total LTR sequence length.<a reveal-id="dsz020-B29" data-open="dsz020-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> After standardization, the corrected LAI can be used as a reference-free genome metric regardless of genome size, LTR retrotransposon content, and gene content completeness.<a reveal-id="dsz020-B29" data-open="dsz020-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The accuracy of the final assembly was estimated by aligning Illumina short reads using BWA-MEM<a reveal-id="dsz020-B27" data-open="dsz020-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and GATK<a reveal-id="dsz020-B30" data-open="dsz020-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> to call variants.</p>                    <h3 scrollto-destination="164508467" id="164508467" class="section-title">2.8. Gene prediction and annotation</h3>
<p>Tandem repeats within the poplar 84K genome were identified with Tandem Repeat Finder.<a reveal-id="dsz020-B31" data-open="dsz020-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> To identify known transposable elements (TEs), we used RepeatMasker,<a reveal-id="dsz020-B32" data-open="dsz020-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> loaded with the Repbase<a reveal-id="dsz020-B33" data-open="dsz020-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and the Dfam<a reveal-id="dsz020-B34" data-open="dsz020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> library databases, Homology-based ncRNA annotation was performed by mapping plant miRNA and snRNA genes from the Rfam database (release 14.0)<a reveal-id="dsz020-B35" data-open="dsz020-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> to the poplar 84K genome using infernal.<a reveal-id="dsz020-B36" data-open="dsz020-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> tRNAscan-SE<a reveal-id="dsz020-B37" data-open="dsz020-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> was used for tRNA annotation. RNAmmer<a reveal-id="dsz020-B38" data-open="dsz020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> was used to predict rRNAs and their subunits. Protein-coding genes were predicted based on transcriptomic, homologous, and <em>de novo</em> methods, the details of which along with functional annotation of the predicted gene models were described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span>.</p>                    <h3 scrollto-destination="164508469" id="164508469" class="section-title">2.9. Subgenome assignment</h3>
<p>The chromosome pairs were firstly identified with MCScanX,<a reveal-id="dsz020-B39" data-open="dsz020-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> based on protein collinearity between all the 38 Hi-C linkage groups in poplar 84K and all the 19 chromosomes of <em>P. trichocarpa</em>. The pairwise relationship of the chromosomes was further confirmed by their DNA-level collinearity identified using nucmer.<a reveal-id="dsz020-B40" data-open="dsz020-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> For each chromosome pair, the chromosomes derived from the parental species were distinguished by similarity to the genomes of related species <em>P. alba</em> var. <em>pyramidalis</em><a reveal-id="dsz020-B7" data-open="dsz020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and <em>P. tremula</em>,<a reveal-id="dsz020-B41" data-open="dsz020-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and were assigned to subgenome A and G, respectively.</p>                    <h3 scrollto-destination="164508471" id="164508471" class="section-title">2.10. Evolutionary analysis of chloroplast and mitochondrial genome</h3>
<p>Phylogenetic tree reconstruction of chloroplast and mitochondrial protein coding regions was described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span>. The organelle-DNA similar fragments in nuclear genome were detected using BLASTN with the chloroplast and mitochondrial contigs as query to search against poplar 84K genome. The parameters of BLASTN and the filtration setting were as described previously.<a reveal-id="dsz020-B42" data-open="dsz020-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                    <h3 scrollto-destination="164508473" id="164508473" class="section-title">2.11. Analysis of genome collinearity and variation between subgenome A and G</h3>
<p>Collinear blocks between subgenome A and subgenome G were determined using MCscanX<a reveal-id="dsz020-B39" data-open="dsz020-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and plotted with Circos,<a reveal-id="dsz020-B43" data-open="dsz020-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> treating at least eight genes as a collinear block. We further aligned the DNA sequences of the two subgenomes using nucmer,<a reveal-id="dsz020-B40" data-open="dsz020-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and only retained uniquely anchored sequences larger than 10 kb for variance calling by the Assemblytics software.<a reveal-id="dsz020-B44" data-open="dsz020-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a></p>                    <h3 scrollto-destination="164508475" id="164508475" class="section-title">2.12. Transcriptional profiles of the allelic genes in subgenomes</h3>
<p>Being that the RNA-seq experiments performed in this study did not contain replicates and were from few tissues, we downloaded the RNA-seq datasets of four tissues in poplar 84K and of four tissues in <em>P. tremula</em> x <em>P. alba</em> from GenBank SRA database (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) and mapped them to the genome of poplar 84K by HISAT2.<a reveal-id="dsz020-B26" data-open="dsz020-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Gene expression values were calculated with FPKM and TMP values using the Stringtie software.<a reveal-id="dsz020-B45" data-open="dsz020-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Allelic genes determined by MCSCANX, and those showing transcriptional bias between subgenome A and G were identified using edgeR<a reveal-id="dsz020-B46" data-open="dsz020-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> with a 4-fold change and a <em>P-</em>value of &lt;0.05. The qRT-PCR method was described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span>.</p>                    <h2 scrollto-destination="164508477" id="164508477" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="164508478" id="164508478" class="section-title">3.1. Genome assembly and identification of potential contamination and organelle genomes</h3>
<p>Two DNA libraries carrying 270 and 500 bp insertions, and one DNA library carrying 20 kb insertions were constructed and separately sequenced on Illumina NovaSeq 6000 and PacBio Sequel platforms, producing 56.3 Gb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3a</a></span>) and 48.4 Gb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3b</a></span>) data. The mean and N50 of PacBio subread lengths were 8.1 kb and 13.7 kb, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3b</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Fig. S1</a></span>). The haploid genome size of poplar 84K was estimated to be 470.155 ± 5.94 Mb by flow cytometry (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S4</a></span>) and 427.2 Mb based on <em>k-mer</em> depth distribution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Heterozygosity was estimated to be 2.16% using GCE software.<a reveal-id="dsz020-B17" data-open="dsz020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> A 753.8-Mb genome assembly of poplar 84K was constructed (<a reveal-id="dsz020-T1" data-open="dsz020-T1" class="link link-reveal link-table xref-fig">Table 1</a>), comprising 1,384 contigs and a contig N50 of 2.24 Mb. The difference between assembly size and estimated genome size may be due to the heterozygous character of poplar 84K and was investigated further in the following analysis. To remove fungal and bacterial DNA contamination, for 1,142 contigs less than 1 Mb (total 157,986,688 bp), we performed a BLASTN<a reveal-id="dsz020-B21" data-open="dsz020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> search against the non-redundant nucleotide (NT) database in GenBank with an <em>E</em>-value of 1E-5. A total of 5,264,546 bp from 114 contigs were identified as bacterial or fungal contamination, and was excluded from further analysis. Three contigs with length of 40,693, 115,255, and 64,703 bp were found with high identities to the chloroplast sequence of <em>P. tremula</em> x <em>P. alba</em> (<a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_028504.1">NC_028504.1</a>).<a reveal-id="dsz020-B22" data-open="dsz020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The chloroplast genome (156,462 bp) was then assembled from these contigs based on a collinear relationship with the chloroplast sequence of <em>P. tremula</em> x <em>P. alba</em> (<a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_028504.1">NC_028504.1</a>).<a reveal-id="dsz020-B22" data-open="dsz020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> One contig of 874,696 bp was found to encode the mitochondrial genome. Gene structure annotations and figures of the organelle genomes were produced using GESEQ (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4 and S5</a></span>).<a reveal-id="dsz020-B23" data-open="dsz020-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> After assignment of microbial contamination and organelles, 747.5 Mb from poplar 84K nuclear genome were obtained. With the 95.2 Gb clean reads generated by Hi-C sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3d</a></span>), a total of 1,042 contigs were clustered into 38 groups for pseudomolecule construction. Of these, 544 contigs, with total length of 710 Mb, were ordered and oriented in all linkage groups (<a reveal-id="dsz020-F1" data-open="dsz020-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>, <a reveal-id="dsz020-T1" data-open="dsz020-T1" class="link link-reveal link-table xref-fig">Table 1</a>). </p>                        <a id="164508480" scrollto-destination="164508480"></a>
<div id="dsz020-F1" data-id="dsz020-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/m_dsz020f1.png?Expires=1576204279&amp;Signature=P6KS-vqWJZkPFaHcmr9hUOCpSvIl8N5NLq9s~MF9g0lwny8Ra6LCeJAaMceodsD6jIk4~bEGBdWequVMkR2eooAiFJImOke4td~7sQ5T44m~FQvo6F0bRt~0H4C91SlOECi0qzHRsTnqPkduWhpGZIgAdEXVb2KHZd9UxeNIPVttzfz2JdkrvL1gTtGATI1Gb2rRJ0kxHf359ezxCUcbFT8NWagTERbcWTZzmo9IOIbqce-SUotIDVGjXoyJQKkZy6JYsw6m3hGKqTTw7TCEh-Yf47rCIhCHS1SMQB2rZDB4DaswwIRvoibiMQQozqzpcHgiGb62cxz9wtQipNDciQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Assessment of the assembled 84K genome. (a) Interaction frequency distributions of Hi-C linkage groups. The log2 of the valid interaction link number of Hi-C data between any pair of 500 kb non-overlapping bins were calculated and is displayed as a heatmap by HiCPlotter.25 The black/white bar of the heatmap indicates the interaction frequency of the Hi-C links. The square indicates the abnormal high frequency of interaction between linkage group 35 (chromosome 11 of subgenome G) and linkage group 10 (chromosome 12 in subgenome A). (b) BUSCO analysis of the genomes from poplar 84K, P. trichocarpa, P. alba var. pyramidalis, and P. tremula. (C) complete; (S) single-copy; (D) duplicated; (F) Fragmented; (M) Missing." path-from-xml="dsz020f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508480/dsz020f1.tif" path-from-xml="dsz020f1.tif" target="_blank">Open in new tab</a><a section="164508480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020f1.png?Expires=1576204279&amp;Signature=QR3Ohyw0gRxhyLGV7fVOmTg9PIBnrPWtV19EfgF3VPcl6UvUlN1sDmmNUF3WeSbgKnuwcfWJsvJcMGeecap1IBH2g2UBSrvAnjEp~4mEa6YvQF25oc8QOC4JbiPLZuGrWU6wegq~CHPFalQYm-vNn8xWRFGkOZJqdoTDIXBOUgbFeMzbtzLFKlIhp1YM2T~BCZ3XVS0xiqTPIol~6mxmOAGE5q3c0wzI9EJWnFHQG5a6knMuIuwHWst84pzz~-jWmSQwx6IrNF9~wdVGmkiWaiIhRzjNjZtQqXwUWDH~36SUygacB49b~YB4NpMWCs8kuUp3fiRnWvHRrzqXM5PkJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508480&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz020f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Assessment of the assembled 84K genome. (a) Interaction frequency distributions of Hi-C linkage groups. The log2 of the valid interaction link number of Hi-C data between any pair of 500 kb non-overlapping bins were calculated and is displayed as a heatmap by HiCPlotter.<a reveal-id="dsz020-B25" data-open="dsz020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> The black/white bar of the heatmap indicates the interaction frequency of the Hi-C links. The square indicates the abnormal high frequency of interaction between linkage group 35 (chromosome 11 of subgenome G) and linkage group 10 (chromosome 12 in subgenome A). (b) BUSCO analysis of the genomes from poplar 84K, P. trichocarpa, P. alba var. pyramidalis, and P. tremula. (C) complete; (S) single-copy; (D) duplicated; (F) Fragmented; (M) Missing.</p></div>
</div>
<div content-id="dsz020-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/m_dsz020f1.png?Expires=1576204279&amp;Signature=P6KS-vqWJZkPFaHcmr9hUOCpSvIl8N5NLq9s~MF9g0lwny8Ra6LCeJAaMceodsD6jIk4~bEGBdWequVMkR2eooAiFJImOke4td~7sQ5T44m~FQvo6F0bRt~0H4C91SlOECi0qzHRsTnqPkduWhpGZIgAdEXVb2KHZd9UxeNIPVttzfz2JdkrvL1gTtGATI1Gb2rRJ0kxHf359ezxCUcbFT8NWagTERbcWTZzmo9IOIbqce-SUotIDVGjXoyJQKkZy6JYsw6m3hGKqTTw7TCEh-Yf47rCIhCHS1SMQB2rZDB4DaswwIRvoibiMQQozqzpcHgiGb62cxz9wtQipNDciQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Assessment of the assembled 84K genome. (a) Interaction frequency distributions of Hi-C linkage groups. The log2 of the valid interaction link number of Hi-C data between any pair of 500 kb non-overlapping bins were calculated and is displayed as a heatmap by HiCPlotter.25 The black/white bar of the heatmap indicates the interaction frequency of the Hi-C links. The square indicates the abnormal high frequency of interaction between linkage group 35 (chromosome 11 of subgenome G) and linkage group 10 (chromosome 12 in subgenome A). (b) BUSCO analysis of the genomes from poplar 84K, P. trichocarpa, P. alba var. pyramidalis, and P. tremula. (C) complete; (S) single-copy; (D) duplicated; (F) Fragmented; (M) Missing." path-from-xml="dsz020f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508480/dsz020f1.tif" path-from-xml="dsz020f1.tif" target="_blank">Open in new tab</a><a section="164508480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020f1.png?Expires=1576204279&amp;Signature=QR3Ohyw0gRxhyLGV7fVOmTg9PIBnrPWtV19EfgF3VPcl6UvUlN1sDmmNUF3WeSbgKnuwcfWJsvJcMGeecap1IBH2g2UBSrvAnjEp~4mEa6YvQF25oc8QOC4JbiPLZuGrWU6wegq~CHPFalQYm-vNn8xWRFGkOZJqdoTDIXBOUgbFeMzbtzLFKlIhp1YM2T~BCZ3XVS0xiqTPIol~6mxmOAGE5q3c0wzI9EJWnFHQG5a6knMuIuwHWst84pzz~-jWmSQwx6IrNF9~wdVGmkiWaiIhRzjNjZtQqXwUWDH~36SUygacB49b~YB4NpMWCs8kuUp3fiRnWvHRrzqXM5PkJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508480&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz020f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Assessment of the assembled 84K genome. (a) Interaction frequency distributions of Hi-C linkage groups. The log2 of the valid interaction link number of Hi-C data between any pair of 500 kb non-overlapping bins were calculated and is displayed as a heatmap by HiCPlotter.<a reveal-id="dsz020-B25" data-open="dsz020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> The black/white bar of the heatmap indicates the interaction frequency of the Hi-C links. The square indicates the abnormal high frequency of interaction between linkage group 35 (chromosome 11 of subgenome G) and linkage group 10 (chromosome 12 in subgenome A). (b) BUSCO analysis of the genomes from poplar 84K, P. trichocarpa, P. alba var. pyramidalis, and P. tremula. (C) complete; (S) single-copy; (D) duplicated; (F) Fragmented; (M) Missing.</p></div>
</div>                        <a id="164508481" scrollto-destination="164508481"></a>
<div content-id="dsz020-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz020-T1" class="table-wrap-title" data-id="dsz020-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Statistics of genome and subgenome assembly of poplar 84K using different sequencing data</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Length (bp) /number </th>
<th>PacBio assembly </th>
<th>Hi-C assembly </th>
<th>Pseudomolecules </th>
<th>Subgenome A </th>
<th>Subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>N90 </td>
<td>429,473 </td>
<td>12,722,015 </td>
<td>13,367,995 </td>
<td>13,017,457 </td>
<td>13,367,995 </td>
</tr>
<tr>
<td>N80 </td>
<td>959,980 </td>
<td>13,867,023 </td>
<td>14,133,090 </td>
<td>13,907,236 </td>
<td>14,133,090 </td>
</tr>
<tr>
<td>N70 </td>
<td>1,381,280 </td>
<td>14,269,014 </td>
<td>16,373,422 </td>
<td>16,345,784 </td>
<td>16,373,422 </td>
</tr>
<tr>
<td>N60 </td>
<td>1,818,021 </td>
<td>17,170,365 </td>
<td>17,428,093 </td>
<td>17,170,365 </td>
<td>17,629,401 </td>
</tr>
<tr>
<td>N50 </td>
<td>2,239,559 </td>
<td>19,641,611 </td>
<td>19,971,399 </td>
<td>18,769,030 </td>
<td>20,657,227 </td>
</tr>
<tr>
<td>N40 </td>
<td>2,832,264 </td>
<td>20,657,227 </td>
<td>20,657,227 </td>
<td>20,018,725 </td>
<td>21,475,114 </td>
</tr>
<tr>
<td>N30 </td>
<td>3,174,752 </td>
<td>21,475,114 </td>
<td>21,827,254 </td>
<td>21,194,248 </td>
<td>22,685,518 </td>
</tr>
<tr>
<td>N20 </td>
<td>4,522,243 </td>
<td>23,093,407 </td>
<td>24,417,576 </td>
<td>24,417,576 </td>
<td>25,197,140 </td>
</tr>
<tr>
<td>N10 </td>
<td>6,295,541 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Maximum length </td>
<td>12,124,769 </td>
<td>49,243,677 </td>
<td>49,243,677 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Minimum length </td>
<td>1,245 </td>
<td>1,245 </td>
<td>6,540,224 </td>
<td>11,888,367 </td>
<td>6,540,224 </td>
</tr>
<tr>
<td>Total length </td>
<td>753,822,854<sup>a</sup> </td>
<td>747,538,837<sup>b</sup> </td>
<td>710,053,368 </td>
<td>356,027,211 </td>
<td>354,026,157 </td>
</tr>
<tr>
<td>Total sequences </td>
<td>1,384 </td>
<td>841 </td>
<td>38 </td>
<td>19 </td>
<td>19 </td>
</tr>
<tr>
<td>Gap numbers </td>
<td>— </td>
<td>505 </td>
<td>505 </td>
<td>227 </td>
<td>278 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz020-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Length (bp) /number </th>
<th>PacBio assembly </th>
<th>Hi-C assembly </th>
<th>Pseudomolecules </th>
<th>Subgenome A </th>
<th>Subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>N90 </td>
<td>429,473 </td>
<td>12,722,015 </td>
<td>13,367,995 </td>
<td>13,017,457 </td>
<td>13,367,995 </td>
</tr>
<tr>
<td>N80 </td>
<td>959,980 </td>
<td>13,867,023 </td>
<td>14,133,090 </td>
<td>13,907,236 </td>
<td>14,133,090 </td>
</tr>
<tr>
<td>N70 </td>
<td>1,381,280 </td>
<td>14,269,014 </td>
<td>16,373,422 </td>
<td>16,345,784 </td>
<td>16,373,422 </td>
</tr>
<tr>
<td>N60 </td>
<td>1,818,021 </td>
<td>17,170,365 </td>
<td>17,428,093 </td>
<td>17,170,365 </td>
<td>17,629,401 </td>
</tr>
<tr>
<td>N50 </td>
<td>2,239,559 </td>
<td>19,641,611 </td>
<td>19,971,399 </td>
<td>18,769,030 </td>
<td>20,657,227 </td>
</tr>
<tr>
<td>N40 </td>
<td>2,832,264 </td>
<td>20,657,227 </td>
<td>20,657,227 </td>
<td>20,018,725 </td>
<td>21,475,114 </td>
</tr>
<tr>
<td>N30 </td>
<td>3,174,752 </td>
<td>21,475,114 </td>
<td>21,827,254 </td>
<td>21,194,248 </td>
<td>22,685,518 </td>
</tr>
<tr>
<td>N20 </td>
<td>4,522,243 </td>
<td>23,093,407 </td>
<td>24,417,576 </td>
<td>24,417,576 </td>
<td>25,197,140 </td>
</tr>
<tr>
<td>N10 </td>
<td>6,295,541 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Maximum length </td>
<td>12,124,769 </td>
<td>49,243,677 </td>
<td>49,243,677 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Minimum length </td>
<td>1,245 </td>
<td>1,245 </td>
<td>6,540,224 </td>
<td>11,888,367 </td>
<td>6,540,224 </td>
</tr>
<tr>
<td>Total length </td>
<td>753,822,854<sup>a</sup> </td>
<td>747,538,837<sup>b</sup> </td>
<td>710,053,368 </td>
<td>356,027,211 </td>
<td>354,026,157 </td>
</tr>
<tr>
<td>Total sequences </td>
<td>1,384 </td>
<td>841 </td>
<td>38 </td>
<td>19 </td>
<td>19 </td>
</tr>
<tr>
<td>Gap numbers </td>
<td>— </td>
<td>505 </td>
<td>505 </td>
<td>227 </td>
<td>278 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn"><p><sup>a</sup> The original assembly obtained using CANU.</p></div></div>
<div content-id="tblfn2"><div class="fn"><p><sup>b</sup> Popar 84K nuclear genome assembly after removing organellar sequences and microorganism contamination.</p></div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/164508481" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz020-T1" class="table-wrap-title" data-id="dsz020-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Statistics of genome and subgenome assembly of poplar 84K using different sequencing data</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Length (bp) /number </th>
<th>PacBio assembly </th>
<th>Hi-C assembly </th>
<th>Pseudomolecules </th>
<th>Subgenome A </th>
<th>Subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>N90 </td>
<td>429,473 </td>
<td>12,722,015 </td>
<td>13,367,995 </td>
<td>13,017,457 </td>
<td>13,367,995 </td>
</tr>
<tr>
<td>N80 </td>
<td>959,980 </td>
<td>13,867,023 </td>
<td>14,133,090 </td>
<td>13,907,236 </td>
<td>14,133,090 </td>
</tr>
<tr>
<td>N70 </td>
<td>1,381,280 </td>
<td>14,269,014 </td>
<td>16,373,422 </td>
<td>16,345,784 </td>
<td>16,373,422 </td>
</tr>
<tr>
<td>N60 </td>
<td>1,818,021 </td>
<td>17,170,365 </td>
<td>17,428,093 </td>
<td>17,170,365 </td>
<td>17,629,401 </td>
</tr>
<tr>
<td>N50 </td>
<td>2,239,559 </td>
<td>19,641,611 </td>
<td>19,971,399 </td>
<td>18,769,030 </td>
<td>20,657,227 </td>
</tr>
<tr>
<td>N40 </td>
<td>2,832,264 </td>
<td>20,657,227 </td>
<td>20,657,227 </td>
<td>20,018,725 </td>
<td>21,475,114 </td>
</tr>
<tr>
<td>N30 </td>
<td>3,174,752 </td>
<td>21,475,114 </td>
<td>21,827,254 </td>
<td>21,194,248 </td>
<td>22,685,518 </td>
</tr>
<tr>
<td>N20 </td>
<td>4,522,243 </td>
<td>23,093,407 </td>
<td>24,417,576 </td>
<td>24,417,576 </td>
<td>25,197,140 </td>
</tr>
<tr>
<td>N10 </td>
<td>6,295,541 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Maximum length </td>
<td>12,124,769 </td>
<td>49,243,677 </td>
<td>49,243,677 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Minimum length </td>
<td>1,245 </td>
<td>1,245 </td>
<td>6,540,224 </td>
<td>11,888,367 </td>
<td>6,540,224 </td>
</tr>
<tr>
<td>Total length </td>
<td>753,822,854<sup>a</sup> </td>
<td>747,538,837<sup>b</sup> </td>
<td>710,053,368 </td>
<td>356,027,211 </td>
<td>354,026,157 </td>
</tr>
<tr>
<td>Total sequences </td>
<td>1,384 </td>
<td>841 </td>
<td>38 </td>
<td>19 </td>
<td>19 </td>
</tr>
<tr>
<td>Gap numbers </td>
<td>— </td>
<td>505 </td>
<td>505 </td>
<td>227 </td>
<td>278 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz020-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Length (bp) /number </th>
<th>PacBio assembly </th>
<th>Hi-C assembly </th>
<th>Pseudomolecules </th>
<th>Subgenome A </th>
<th>Subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>N90 </td>
<td>429,473 </td>
<td>12,722,015 </td>
<td>13,367,995 </td>
<td>13,017,457 </td>
<td>13,367,995 </td>
</tr>
<tr>
<td>N80 </td>
<td>959,980 </td>
<td>13,867,023 </td>
<td>14,133,090 </td>
<td>13,907,236 </td>
<td>14,133,090 </td>
</tr>
<tr>
<td>N70 </td>
<td>1,381,280 </td>
<td>14,269,014 </td>
<td>16,373,422 </td>
<td>16,345,784 </td>
<td>16,373,422 </td>
</tr>
<tr>
<td>N60 </td>
<td>1,818,021 </td>
<td>17,170,365 </td>
<td>17,428,093 </td>
<td>17,170,365 </td>
<td>17,629,401 </td>
</tr>
<tr>
<td>N50 </td>
<td>2,239,559 </td>
<td>19,641,611 </td>
<td>19,971,399 </td>
<td>18,769,030 </td>
<td>20,657,227 </td>
</tr>
<tr>
<td>N40 </td>
<td>2,832,264 </td>
<td>20,657,227 </td>
<td>20,657,227 </td>
<td>20,018,725 </td>
<td>21,475,114 </td>
</tr>
<tr>
<td>N30 </td>
<td>3,174,752 </td>
<td>21,475,114 </td>
<td>21,827,254 </td>
<td>21,194,248 </td>
<td>22,685,518 </td>
</tr>
<tr>
<td>N20 </td>
<td>4,522,243 </td>
<td>23,093,407 </td>
<td>24,417,576 </td>
<td>24,417,576 </td>
<td>25,197,140 </td>
</tr>
<tr>
<td>N10 </td>
<td>6,295,541 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Maximum length </td>
<td>12,124,769 </td>
<td>49,243,677 </td>
<td>49,243,677 </td>
<td>49,053,667 </td>
<td>49,243,677 </td>
</tr>
<tr>
<td>Minimum length </td>
<td>1,245 </td>
<td>1,245 </td>
<td>6,540,224 </td>
<td>11,888,367 </td>
<td>6,540,224 </td>
</tr>
<tr>
<td>Total length </td>
<td>753,822,854<sup>a</sup> </td>
<td>747,538,837<sup>b</sup> </td>
<td>710,053,368 </td>
<td>356,027,211 </td>
<td>354,026,157 </td>
</tr>
<tr>
<td>Total sequences </td>
<td>1,384 </td>
<td>841 </td>
<td>38 </td>
<td>19 </td>
<td>19 </td>
</tr>
<tr>
<td>Gap numbers </td>
<td>— </td>
<td>505 </td>
<td>505 </td>
<td>227 </td>
<td>278 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn"><p><sup>a</sup> The original assembly obtained using CANU.</p></div></div>
<div content-id="tblfn2"><div class="fn"><p><sup>b</sup> Popar 84K nuclear genome assembly after removing organellar sequences and microorganism contamination.</p></div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/164508481" target="_blank">Open in new tab</a></div>
</div></div>                    <h3 scrollto-destination="164508482" id="164508482" class="section-title">3.2.Evaluation of completeness, continuity, and accuracy of final assembly</h3>
<p>We found 95.64–97.88% of the RNA-seq reads and 95.74–97.74% of the DNA short reads could be aligned to the final assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). BUSCO analysis<a reveal-id="dsz020-B27" data-open="dsz020-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> showed that 1,386 (96.3%) of 1,440 plant single-copy orthologues were complete, but 1,291 (93.1%) of them presented as duplicated copies. Similar percentages of the 1,440 plant single-copy orthologues were detected in <em>P. trichocarpa</em>,<a reveal-id="dsz020-B4" data-open="dsz020-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>P. alba</em> var. <em>Pyramidalis</em>,<a reveal-id="dsz020-B7" data-open="dsz020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and <em>P. tremula</em>,<a reveal-id="dsz020-B41" data-open="dsz020-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and at least 85.1% of them were single copy in these species (<a reveal-id="dsz020-F1" data-open="dsz020-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). This suggested that the heterozygous regions were obtained, based on which these heterozygous regions could be separated using their parental genome information.</p>
<p>We assessed the continuity of the final assembly of poplar 84K in comparison to two poplar reference genomes, <em>P. trichocarpa</em><a reveal-id="dsz020-B4" data-open="dsz020-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and <em>P. deltoides</em> (assembly version 2.1, produced by the US Department of Energy Joint Genome Institute in collaboration with the user community). The standardized LAI scores are 9.34 in <em>P. trichocarpa</em>, 7.95 in <em>P. deltoides</em>, and 14.79 in poplar 84K. This implies the poplar 84K assembly has achieved reference genome quality.</p>
<p>The accuracy of the final assembly was estimated by aligning Illumina short reads using BWA-MEM<a reveal-id="dsz020-B27" data-open="dsz020-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and GATK<a reveal-id="dsz020-B30" data-open="dsz020-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> to call variants. A total of 318 homozygous SNPs and 6,398 homozygous INDELs were considered as errors, and 35,735 heterozygous SNPs and 24,860 heterozygous INDELs were found.</p>                    <h3 scrollto-destination="164508486" id="164508486" class="section-title">3.3. Genome annotation and subgenome assignment</h3>
<p>We identified 184.0 Mb of TE sequence (24.4% of the assembly) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The largest class of TEs comprised retrotransposons, accounting for 18.2% of the assembly, and consisted mostly of Gypsy and Copia retrotransposon families. DNA transposons accounted for 3.8% of the assembly. These analyses also identified 1,983 miRNAs, 1,312 tRNAs, 1,140 rRNAs, and 1,126 snRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).</p>
<p>Four RNA libraries for leaves and one library for shoots were constructed, and sequenced using the Illumina NovaSeq platform, generating 145.8 million pair reads for genome annotation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3c</a></span>). Finally, 85,755 consensus protein-coding genes were predicted, the average gene length, average transcript length, average CDS length, and exon number per gene being 2,948 bp, 2,937 bp, 1,075 bp and 4.48, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Functional annotation of the predicted protein-coding genes revealed that 72,574 of the total of 85,755 genes (84.6%) could be assigned putative functions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>).</p>
<p>Chromosome pairs were first identified based on protein collinearity between all the 38 Hi-C linkage groups in poplar 84K and all the 19 chromosomes of <em>P. trichocarpa</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The pairwise relationship of the chromosomes was further confirmed by their DNA-level collinearity (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). For each chromosome pair, the chromosomes derived from the parental species were distinguished by similarity to the genomes of related species <em>P. alba</em> var. <em>pyramidalis</em> and <em>P. tremula</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>).</p>
<p>The chromosome numbering and orientation were determined by comparison to <em>P. trichocarpa</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S10</a></span>). Ultimately, 356 Mb from female parental species <em>P. alba</em> and 354 Mb from male parental species <em>P. tremula</em> var. <em>glandulosa</em> were obtained, and were designated as subgenome A and subgenome G. BUSCO analysis indicated each subgenome has characteristics similar to those of <em>P. trichocarpa</em>, <em>P. alba</em> var. <em>pyramidalis</em> and <em>P. tremula</em> (<a reveal-id="dsz020-F1" data-open="dsz020-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>), which implies high completeness of the two subgenomes. Specifically, the numbers of duplicated genes dropped from 1,291 (89.7%) to 187 in subgenome A and to 186 in subgenome G. This supports our previous speculation that combination of two haploid genomes causes the assembly size to increase by 30%.</p>                    <h3 scrollto-destination="164508491" id="164508491" class="section-title">3.4. Characteristics of the poplar 84K genome</h3>
<p>The characteristics of the two subgenomes, along with sequencing depths, are shown in <a reveal-id="dsz020-F2" data-open="dsz020-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, including the chromosome length, gene density, TE content, GC content, and location of collinear regions. Chromosome 11 of subgenome G (6.54 Mb) was much shorter than that of subgenome A (17.42 Mb) (<a reveal-id="dsz020-F2" data-open="dsz020-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). This may be due to large fragment loss during inter-species hybridization and/or subsequent chromosome stabilization, or defects of the current assembly algorithm in dealing with the regions with high identities. Another anomalous region is a 6.92 Mb fragment at the 5′ end of chromosome 12 of subgenome A, which showed a doubled sequencing depth as compared with other regions (<a reveal-id="dsz020-F2" data-open="dsz020-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). This 6.92 Mb region also showed a high frequency of interaction with the short chromosome 11 of subgenome G (<a reveal-id="dsz020-F1" data-open="dsz020-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). We speculate that one additional 6.92 Mb region derived from recent large DNA fragment duplication may be responsible for the connection to chromosome 11 of subgenome G. </p>                        <a id="164508493" scrollto-destination="164508493"></a>
<div id="dsz020-F2" data-id="dsz020-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/m_dsz020f2.png?Expires=1576204279&amp;Signature=tL3TCW059l8LxKIAGOCsCCXKjO2CwCo~mZuNwBp3Zelm~U3ReiSxJ1oYPkOMUqav-LT26c28Tzp~CmdS6WbeNaWyRfC9F5N~9vfHLTTkmI5pECxEvucoC5ifHkP0MypcIkaa5UNipXhoQQb3JCLAPxv~WAi2UYqOQUxUjm7ASI5FnjrbOShXa2FTEg46WS1VjkNdQpv3o~iZE-Zw4DYuJ0I1uLGjLl7p1iCxJUJE6~1L10enOewdOUs2qCThohX7~~o01aBVQZyK2WzKzdQPdu4TrF~abAy7EvzcY2oVA0y54jSAQ00z9gZT53XMpFX5J1YeYkUz1xx-KdtxSkaHvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics of the poplar 84K genome. From the outer edge inward: (a) circles represent the subgenomes A (right) and G (left); (b) gene density on each chromosome; (c) repeat density at 10 kb intervals; (d) GC content at 10 kb intervals; (e) the sequencing depth of Illumina short reads at 10 kb intervals, and collinear blocks linked by grey lines." path-from-xml="dsz020f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508493/dsz020f2.tif" path-from-xml="dsz020f2.tif" target="_blank">Open in new tab</a><a section="164508493" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020f2.png?Expires=1576204279&amp;Signature=HMKF4kL0q7bapCm8FOg1Sxt6ByeFJskGJjlzyFbxzuYggprdFvuv3TwmjrJ9wpkn~ZutkKGvPc~4fjMw1dBkPiBOXXrCKYriWv6RHEw4qmYeiBx5rgMh1TnX5yurK0bx7nsO2deZBCaCsKQ5F4TEky6YzePoWBQF9XqKYgqOrJMEwJrE1SVlNupAhzze8RwzZdX6sHnDSUcftBBPswajaJ3MgyjK3frsdrIzijWrUP6A7-WWtBF2UlepHvwi~UDYFvBC6vNVLjUfVxnABu-tn5j2~06OXb9PSYny3wIL949g6M5wioceecX1Mjif1WApRXGe4wWqHuHahwm1FMmvHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508493&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz020f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characteristics of the poplar 84K genome. From the outer edge inward: (a) circles represent the subgenomes A (right) and G (left); (b) gene density on each chromosome; (c) repeat density at 10 kb intervals; (d) GC content at 10 kb intervals; (e) the sequencing depth of Illumina short reads at 10 kb intervals, and collinear blocks linked by grey lines.</p></div>
</div>
<div content-id="dsz020-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/m_dsz020f2.png?Expires=1576204279&amp;Signature=tL3TCW059l8LxKIAGOCsCCXKjO2CwCo~mZuNwBp3Zelm~U3ReiSxJ1oYPkOMUqav-LT26c28Tzp~CmdS6WbeNaWyRfC9F5N~9vfHLTTkmI5pECxEvucoC5ifHkP0MypcIkaa5UNipXhoQQb3JCLAPxv~WAi2UYqOQUxUjm7ASI5FnjrbOShXa2FTEg46WS1VjkNdQpv3o~iZE-Zw4DYuJ0I1uLGjLl7p1iCxJUJE6~1L10enOewdOUs2qCThohX7~~o01aBVQZyK2WzKzdQPdu4TrF~abAy7EvzcY2oVA0y54jSAQ00z9gZT53XMpFX5J1YeYkUz1xx-KdtxSkaHvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics of the poplar 84K genome. From the outer edge inward: (a) circles represent the subgenomes A (right) and G (left); (b) gene density on each chromosome; (c) repeat density at 10 kb intervals; (d) GC content at 10 kb intervals; (e) the sequencing depth of Illumina short reads at 10 kb intervals, and collinear blocks linked by grey lines." path-from-xml="dsz020f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508493/dsz020f2.tif" path-from-xml="dsz020f2.tif" target="_blank">Open in new tab</a><a section="164508493" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020f2.png?Expires=1576204279&amp;Signature=HMKF4kL0q7bapCm8FOg1Sxt6ByeFJskGJjlzyFbxzuYggprdFvuv3TwmjrJ9wpkn~ZutkKGvPc~4fjMw1dBkPiBOXXrCKYriWv6RHEw4qmYeiBx5rgMh1TnX5yurK0bx7nsO2deZBCaCsKQ5F4TEky6YzePoWBQF9XqKYgqOrJMEwJrE1SVlNupAhzze8RwzZdX6sHnDSUcftBBPswajaJ3MgyjK3frsdrIzijWrUP6A7-WWtBF2UlepHvwi~UDYFvBC6vNVLjUfVxnABu-tn5j2~06OXb9PSYny3wIL949g6M5wioceecX1Mjif1WApRXGe4wWqHuHahwm1FMmvHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508493&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz020f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Characteristics of the poplar 84K genome. From the outer edge inward: (a) circles represent the subgenomes A (right) and G (left); (b) gene density on each chromosome; (c) repeat density at 10 kb intervals; (d) GC content at 10 kb intervals; (e) the sequencing depth of Illumina short reads at 10 kb intervals, and collinear blocks linked by grey lines.</p></div>
</div>
<p>Further phylogenetic analyses of chloroplast proteins were conducted. The tree topology indicated that <em>P. trichocarpa</em> is closely related to <em>P. balsamifera</em> and <em>P. fremontii</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>), consistent with previous studies.<a reveal-id="dsz020-B19" data-open="dsz020-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><em>P. yunnanensis</em> formed a clade with <em>P. alba</em>, <em>P. tremula</em> and <em>P. tremula</em> x <em>P. alba</em>, within which the poplar 84K genome formed a sister branch with the genome of <em>P. alba</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). The mitochondrial genome in poplar 84K also showed maternal inheritance in that they were mostly related to those from <em>P. alba</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>).</p>
<p>The lateral transfer of organelle fragments into the nuclear genome has been widely reported in plants. To get an insight into the scale of such transfer and to exclude the possibility of misidentification of organelle genome in poplar 84K, we analysed the insertion times, sequence identities, and maximum and total lengths of organelle-DNA similar fragments in the nuclear genome. About 2,234 and 2,093 insertions with total length of 533 kb and 530 kb of chloroplast sequences, and 3,916 and 3,844 insertions with total length of 597.2 kb and 625.6 kb of mitochondrial sequences were found in all the chromosomes of subgenome A and G, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). The mean insertion lengths were 236 bp for chloroplasts and 155 bp for mitochondria (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). The longest chloroplast and mitochondrial insertion lengths were 15.2 kb (98.9% identity) and 19.1 kb (98.1% identity), respectively. The statistics of organelle-DNA similar fragments in the nuclear genome of poplar 84K were similar to the previous report of <em>P. trichocarpa.</em><a reveal-id="dsz020-B42" data-open="dsz020-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>
<p>We performed orthologous assignment and gene family comparison in the two poplar 84K subgenomes and four other <em>Populus</em> species using OrthoFinder.<a reveal-id="dsz020-B47" data-open="dsz020-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Total of 30,342 clusters were obtained in them, among which 17,690 (58.30%) were shared by the four genomes and two poplar 84K subgenomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>).</p>                    <h3 scrollto-destination="164508497" id="164508497" class="section-title">3.5. Genome collinearity and variation between subgenome A and G</h3>
<p>About 323 Mb of regions of collinearity, including 28,974 allelic gene pairs, were identified between the two subgenomes (90.7% of subgenome A and 91.2% of subgenome G). We then aligned the DNA sequences of the two subgenomes and only retained uniquely anchored sequences larger than 10 kb for variant calling. We found 5,398,437 SNPs and 1,108,357 indels (1 bp–10 kb) in these collinear regions. These variations within allelic genes would influence their gene structures, expression patterns, functions, and regulation, which together contribute the morphologic and physiologic characteristics of hybrid 84K.</p>                    <h3 scrollto-destination="164508499" id="164508499" class="section-title">3.6. Transcriptional bias of the allelic genes in the two subgenomes</h3>
<p>We further explored the expressional bias of allelic genes in the two subgenomes by mapping RNA-seq data of four tissues from 84K and four tissues from backcross hybrid <em>P. tremula</em> x <em>P. alba</em> to the poplar 84 genome assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Of all the 28,974 allelic gene pairs, 6,780 gene pairs were found with transcriptional bias in poplar 84K, and 5,349 in <em>P. tremula</em> x <em>P. alba</em> (<a reveal-id="dsz020-F3" data-open="dsz020-F3" class="link link-reveal link-table xref-fig">Fig. 3a and b</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). Among these allelic genes with transcriptional bias, 2,121 gene pairs showed bias in all four tissue in poplar 84K, the corresponding number in <em>P. tremula</em> x <em>P. alba</em> being 1,166 (<a reveal-id="dsz020-F3" data-open="dsz020-F3" class="link link-reveal link-table xref-fig">Fig. 3a and b</a>). We further calculated the number of genes that showed transcriptional bias to subgenomes A and G in two hybrids. Interestingly, we found more than 54.4% of allelic genes showed transcriptional bias to subgenome G, and less than 45.6% of allelic genes to subgenome A in all the tissues that were used (<a reveal-id="dsz020-T2" data-open="dsz020-T2" class="link link-reveal link-table xref-fig">Table 2</a>), which implies subgenome G plays a more important role during the growth process in both hybrids. We selected four allelic gene pairs and performed qRT-PCR analysis in shoots and leaves, which showed expression patterns that were consistent with the RNA-seq data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S13</a></span>). The similarities of promoters were additionally analysed between the allelic pairs showing expression bias. The result indicated that a higher divergence in promoter regions was present in the allelic pairs having expression bias than those without detectable expression bias (<a reveal-id="dsz020-F3" data-open="dsz020-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>). </p>                        <a id="164508501" scrollto-destination="164508501"></a>
<div id="dsz020-F3" data-id="dsz020-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/m_dsz020f3.png?Expires=1576204279&amp;Signature=zHKeJ2BmL6j4Iq4UbgzNa89zxAm5EhlWqUhbZmndb1FYwkNqX-uJsibRYeqm8ztjy8jPEaWnFsJ7PziZKKCKcnNFJG4tErMCA6y1Tvq0lGbkThABMXQe8y4SeeHJkHy2Ayp~NJdTj-8qhIzAMnvy3AULGQ9ZsmhEFX2lOO2UHPpKKF18o1L6iasCwcFjeU4D5e-GgoA343aVkyEDC7lilwvu0TiHxHrn6n-7KY6nFL6HP~TXqlJp7JTsAng19RYBaghUBF9o8Dh~uFJw9Ry7wQowNG11nYNBxP0EqlYqMUHoDO2s6UNZ4k56el2499S8ObrqTVbnPB349-0tJ9V5Mg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Allelic genes with transcriptional bias and their promoter region identity in poplar 84K and P. tremula x P. alba. (a) Identified allelic genes with transcriptional bias in poplar 84K. (b) Identified allelic genes with transcriptional bias in P. tremula x P. alba. (c) Boxplot of the identity in allelic gene promoter region. Significant differences of identity between allelic genes with transcriptional bias and no-bias were supported by the Kolmogorov–Smirnov test (P-value &lt; 0.001) in poplar 84K and P. tremula x P. alba." path-from-xml="dsz020f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508501/dsz020f3.tif" path-from-xml="dsz020f3.tif" target="_blank">Open in new tab</a><a section="164508501" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020f3.png?Expires=1576204279&amp;Signature=5P2f9F3OJxhAuQpD7gG9IzkUE~04N~IGqcDechN7l8O12~lZIIIxMje~IHeC0ekBNLLUvMN83ImP1nCgMiP4LhUQAllEh7MmKUlyFA-iVv67TeFOI1AYJSmrMmmZJGf4Ci1Ght8SEZiOPozwTLaUZUTUtANnxoIdowhQdBbEm9PimnmLWXljStH7rYzd6n9Afzmte5370BIs8Ro17NSWrruz5ME3sAYuFjnhvtAp3Nf15opY4bEvq5KEoitZ026Zbspq~fc3w9USNP~kEQRQ9-Dhb0aQ7QOcJT44LcqRMzuIoP~ri~HdEI8I4GZuYkMsF5n48Y0gjYtrUBAQRYR3lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508501&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz020f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Allelic genes with transcriptional bias and their promoter region identity in poplar 84K and <em>P. tremula</em> x <em>P. alba</em>. (a) Identified allelic genes with transcriptional bias in poplar 84K. (b) Identified allelic genes with transcriptional bias in <em>P. tremula</em> x <em>P. alba</em>. (c) Boxplot of the identity in allelic gene promoter region. Significant differences of identity between allelic genes with transcriptional bias and no-bias were supported by the Kolmogorov–Smirnov test (<em>P-</em>value &lt; 0.001) in poplar 84K and <em>P. tremula</em> x <em>P. alba</em>.</p></div>
</div>
<div content-id="dsz020-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/m_dsz020f3.png?Expires=1576204279&amp;Signature=zHKeJ2BmL6j4Iq4UbgzNa89zxAm5EhlWqUhbZmndb1FYwkNqX-uJsibRYeqm8ztjy8jPEaWnFsJ7PziZKKCKcnNFJG4tErMCA6y1Tvq0lGbkThABMXQe8y4SeeHJkHy2Ayp~NJdTj-8qhIzAMnvy3AULGQ9ZsmhEFX2lOO2UHPpKKF18o1L6iasCwcFjeU4D5e-GgoA343aVkyEDC7lilwvu0TiHxHrn6n-7KY6nFL6HP~TXqlJp7JTsAng19RYBaghUBF9o8Dh~uFJw9Ry7wQowNG11nYNBxP0EqlYqMUHoDO2s6UNZ4k56el2499S8ObrqTVbnPB349-0tJ9V5Mg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Allelic genes with transcriptional bias and their promoter region identity in poplar 84K and P. tremula x P. alba. (a) Identified allelic genes with transcriptional bias in poplar 84K. (b) Identified allelic genes with transcriptional bias in P. tremula x P. alba. (c) Boxplot of the identity in allelic gene promoter region. Significant differences of identity between allelic genes with transcriptional bias and no-bias were supported by the Kolmogorov–Smirnov test (P-value &lt; 0.001) in poplar 84K and P. tremula x P. alba." path-from-xml="dsz020f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/164508501/dsz020f3.tif" path-from-xml="dsz020f3.tif" target="_blank">Open in new tab</a><a section="164508501" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020f3.png?Expires=1576204279&amp;Signature=5P2f9F3OJxhAuQpD7gG9IzkUE~04N~IGqcDechN7l8O12~lZIIIxMje~IHeC0ekBNLLUvMN83ImP1nCgMiP4LhUQAllEh7MmKUlyFA-iVv67TeFOI1AYJSmrMmmZJGf4Ci1Ght8SEZiOPozwTLaUZUTUtANnxoIdowhQdBbEm9PimnmLWXljStH7rYzd6n9Afzmte5370BIs8Ro17NSWrruz5ME3sAYuFjnhvtAp3Nf15opY4bEvq5KEoitZ026Zbspq~fc3w9USNP~kEQRQ9-Dhb0aQ7QOcJT44LcqRMzuIoP~ri~HdEI8I4GZuYkMsF5n48Y0gjYtrUBAQRYR3lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=164508501&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz020f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Allelic genes with transcriptional bias and their promoter region identity in poplar 84K and <em>P. tremula</em> x <em>P. alba</em>. (a) Identified allelic genes with transcriptional bias in poplar 84K. (b) Identified allelic genes with transcriptional bias in <em>P. tremula</em> x <em>P. alba</em>. (c) Boxplot of the identity in allelic gene promoter region. Significant differences of identity between allelic genes with transcriptional bias and no-bias were supported by the Kolmogorov–Smirnov test (<em>P-</em>value &lt; 0.001) in poplar 84K and <em>P. tremula</em> x <em>P. alba</em>.</p></div>
</div>                        <a id="164508502" scrollto-destination="164508502"></a>
<div content-id="dsz020-T2" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsz020-T2" class="table-wrap-title" data-id="dsz020-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Genes with transcriptional bias within different tissues in two poplar hybrids</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Species </th>
<th>Tissues </th>
<th>DEG numbers </th>
<th>Dominant expression in subgenome A </th>
<th>Dominant expression in subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Shoot </td>
<td>4124 </td>
<td>1,856 (45.00%) </td>
<td>2,268 (55.00%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Rooting </td>
<td>4109 </td>
<td>1,826 (44.44%) </td>
<td>2,283 (55.56%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Callus </td>
<td>3965 </td>
<td>1,802 (45.45%) </td>
<td>2,163 (54.55%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Leaf </td>
<td>4253 </td>
<td>1,936 (45.52%) </td>
<td>2,317 (54.48%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Leaf </td>
<td>2792 </td>
<td>1,065 (38.14%) </td>
<td>1,727 (61.86%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Bark </td>
<td>3197 </td>
<td>1,314 (41.10%) </td>
<td>1,883 (58.90%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Xylem </td>
<td>2808 </td>
<td>1,150 (40.95%) </td>
<td>1,658 (59.05%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Root </td>
<td>3121 </td>
<td>1,288 (41.27%) </td>
<td>1,833 (58.73%) </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz020-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Species </th>
<th>Tissues </th>
<th>DEG numbers </th>
<th>Dominant expression in subgenome A </th>
<th>Dominant expression in subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Shoot </td>
<td>4124 </td>
<td>1,856 (45.00%) </td>
<td>2,268 (55.00%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Rooting </td>
<td>4109 </td>
<td>1,826 (44.44%) </td>
<td>2,283 (55.56%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Callus </td>
<td>3965 </td>
<td>1,802 (45.45%) </td>
<td>2,163 (54.55%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Leaf </td>
<td>4253 </td>
<td>1,936 (45.52%) </td>
<td>2,317 (54.48%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Leaf </td>
<td>2792 </td>
<td>1,065 (38.14%) </td>
<td>1,727 (61.86%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Bark </td>
<td>3197 </td>
<td>1,314 (41.10%) </td>
<td>1,883 (58.90%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Xylem </td>
<td>2808 </td>
<td>1,150 (40.95%) </td>
<td>1,658 (59.05%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Root </td>
<td>3121 </td>
<td>1,288 (41.27%) </td>
<td>1,833 (58.73%) </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa </em>=<em> </em>84K.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/164508502" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsz020-T2" class="table-wrap-title" data-id="dsz020-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Genes with transcriptional bias within different tissues in two poplar hybrids</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Species </th>
<th>Tissues </th>
<th>DEG numbers </th>
<th>Dominant expression in subgenome A </th>
<th>Dominant expression in subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Shoot </td>
<td>4124 </td>
<td>1,856 (45.00%) </td>
<td>2,268 (55.00%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Rooting </td>
<td>4109 </td>
<td>1,826 (44.44%) </td>
<td>2,283 (55.56%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Callus </td>
<td>3965 </td>
<td>1,802 (45.45%) </td>
<td>2,163 (54.55%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Leaf </td>
<td>4253 </td>
<td>1,936 (45.52%) </td>
<td>2,317 (54.48%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Leaf </td>
<td>2792 </td>
<td>1,065 (38.14%) </td>
<td>1,727 (61.86%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Bark </td>
<td>3197 </td>
<td>1,314 (41.10%) </td>
<td>1,883 (58.90%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Xylem </td>
<td>2808 </td>
<td>1,150 (40.95%) </td>
<td>1,658 (59.05%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Root </td>
<td>3121 </td>
<td>1,288 (41.27%) </td>
<td>1,833 (58.73%) </td>
</tr>
</tbody>
</table></div>
<div content-id="dsz020-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Species </th>
<th>Tissues </th>
<th>DEG numbers </th>
<th>Dominant expression in subgenome A </th>
<th>Dominant expression in subgenome G </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Shoot </td>
<td>4124 </td>
<td>1,856 (45.00%) </td>
<td>2,268 (55.00%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Rooting </td>
<td>4109 </td>
<td>1,826 (44.44%) </td>
<td>2,283 (55.56%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Callus </td>
<td>3965 </td>
<td>1,802 (45.45%) </td>
<td>2,163 (54.55%) </td>
</tr>
<tr>
<td>
<em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa</em> </td>
<td>Leaf </td>
<td>4253 </td>
<td>1,936 (45.52%) </td>
<td>2,317 (54.48%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Leaf </td>
<td>2792 </td>
<td>1,065 (38.14%) </td>
<td>1,727 (61.86%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Bark </td>
<td>3197 </td>
<td>1,314 (41.10%) </td>
<td>1,883 (58.90%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Xylem </td>
<td>2808 </td>
<td>1,150 (40.95%) </td>
<td>1,658 (59.05%) </td>
</tr>
<tr>
<td>
<em> P. tremula</em> x <em>P. alba</em> </td>
<td>Root </td>
<td>3121 </td>
<td>1,288 (41.27%) </td>
<td>1,833 (58.73%) </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><em>P. alba</em> x <em>P. tremula</em> var. <em>glandulosa </em>=<em> </em>84K.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/164508502" target="_blank">Open in new tab</a></div>
</div></div>
<p>Further examination uncovered 285 allelic gene pairs with bias to subgenome A (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14a</a></span>) and 339 with bias toward subgenome G, for all the tissues and in both hybrids (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14b</a></span>). These genes might be essential, playing important functions in both hybrids. Gene enrichment analysis of these two datasets revealed that protein metabolic process and nitrogen compound metabolic process were represented in both subgenomes. Subgenome A has more allelic gene pairs in the categories of monovalent inorganic cation transport and chromatin organization, whereas subgenome G has more in the categories of mRNA processing, ncRNA metabolic process, and sulphur compound metabolic process. As for their cellular component classifications, allelic gene pairs showing bias to subgenome A function in mitochondria and the nucleoplasm, and those in subgenome G play roles in plastids, endosomes, vesicles, and ribosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). This implies these genes function in different cellular compartments and different biological processes. More transcriptome data will be needed to obtain further insights into the transcriptional bias of each subgenome in different developmental stages under different stresses.</p>                    <h2 scrollto-destination="164508504" id="164508504" class="section-title">4. Summary and conclusions</h2>
<p>We employed PacBio single-molecular real-time sequencing and Hi-C technology to generate a reference sequence of the poplar 84K (<em>Populus alba</em> x <em>Populus tremula</em> var. <em>glandulosa</em>) genome. The genome sequences were assembled into the chromosome levels, with a contig N50 size of 1.99 Mb and a scaffold N50 size of 19.6 Mb. About 356 Mb from the female parental species (<em>P. alba</em>) and 354 Mb from the male parental species (<em>P. tremula</em> var. <em>glandulosa</em>) were assigned. The two subgenomes showed high collinearity over 323 Mb. The allelic gene pairs with transcriptional bias toward subgenome A or G function in different cellular compartments and different biological processes. More allelic gene pairs showed transcriptional bias toward subgenome G in poplar 84K and another widely used hybrid <em>P. tremula</em> x <em>P. alba</em>. The dominant expression of subgenome G indicates that it plays a more important role than subgenome A during hybrid growth. The high-quality genome of poplar 84K provides an important gene resource for poplar breeding and molecular biology research.</p>                    <h2 scrollto-destination="164508506" id="164508506" class="section-title">Data availability</h2>
<p>The sequencing reads from each sequencing library have been deposited at NCBI with the Project ID PRJNA556338 and CNGB Nucleotide Sequence Archive (CNSA) with the Project ID CNP0000339. Software versions and main parameters are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576204279&amp;Signature=UX3RxPjsXkk~y6tvOvSEL3KG3Y6mLjE-2Fi3jCtlKu2Itvs8eUWKRNAZLw0Z~j8bSK7D8PqIQUwME-aUPWYWfyvKUvjLD4z6N0GcyEcrBFcVbEahqcQdrCQKRuIZM2irRdvKx3ifZHMiYLRFhuUaq09buez8YX-cYb5za-HzqZrLP-ODD~v9gPjQD760NJIAMAyk0mYyfpNqNJRkit~KPGgEDbKU9efvU-1oYqT2-4xmACyYnVMwJthpHe9K8Fiy6Iw3sjpfoT17xzTtF65rzBsrZlJXVAqqBv0NFrIDuNpJ3ggWXdj8t-DB5~AUwkBFnctL08EAtnrAdNvA5DQrdw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span>.</p>                    <h2 scrollto-destination="164508508" id="164508508" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>This project was supported by the special fund from Key Laboratory of Tree Breeding and Cultivation of State Forestry Administration (ZDRIF201705), the National Natural Science Foundation of China (31771414), and the Program for Science and Technology Innovation Talents in Universities of Henan Province (18HASTIT041).</p>                    <h3 scrollto-destination="164508510" id="164508510" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="164508512" id="164508512" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz020-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Stettler</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Bradshaw</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Heilman</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Hinckley</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">1996</div>, <div class="source">Biology of Populus and Its Implications for Management and Conservation</div> . 
<div class="publisher-name">NRC Research Press, Ottawa</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Biology%20of%20Populus%20and%20Its%20Implications%20for%20Management%20and%20Conservation&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Biology%20of%20Populus%20and%20Its%20Implications%20for%20Management%20and%20Conservation&amp;aulast=Stettler&amp;date=1996" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biology%20of%20Populus%20and%20Its%20Implications%20for%20Management%20and%20Conservation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Biology%20of%20Populus%20and%20Its%20Implications%20for%20Management%20and%20Conservation">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Brunner</div> <div class="given-names">A.M.</div>
</div>, <div class="name string-name">
<div class="surname">Busov</div> <div class="given-names">V.B.</div>
</div>, <div class="name string-name">
<div class="surname">Strauss</div> <div class="given-names">S.H.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Poplar genome sequence: functional genomics in an ecologically dominant plant species</div>, <div class="source">Trends Plant Sci</div> ., <div class="volume">9</div>, <div class="fpage">49</div>–<div class="lpage">56</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Poplar%20genome%20sequence%3A%20functional%20genomics%20in%20an%20ecologically%20dominant%20plant%20species&amp;author=A.M.%20Brunner&amp;author=V.B.%20Busov&amp;author=S.H.%20Strauss&amp;publication_year=2004&amp;journal=Trends%20Plant%20Sci&amp;volume=9&amp;pages=49-56" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tplants.2003.11.006" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2003.11.006" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tplants.2003.11.006"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14729219" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Poplar%20genome%20sequence%3A%20functional%20genomics%20in%20an%20ecologically%20dominant%20plant%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wullschleger</div> <div class="given-names">S.D.</div>
</div>, <div class="name string-name">
<div class="surname">Jansson</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Taylor</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">Genomics and forest biology: populus emerges as the perennial favorite</div>, <div class="source">Plant Cell</div> , <div class="volume">14</div>, <div class="fpage">2651</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomics%20and%20forest%20biology%3A%20populus%20emerges%20as%20the%20perennial%20favorite&amp;author=S.D.%20Wullschleger&amp;author=S.%20Jansson&amp;author=G.%20Taylor&amp;publication_year=2002&amp;journal=Plant%20Cell&amp;volume=14&amp;pages=2651-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1105/tpc.141120" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.141120" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1105%2Ftpc.141120"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12417692" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomics%20and%20forest%20biology%3A%20populus%20emerges%20as%20the%20perennial%20favorite&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tuskan</div> <div class="given-names">G.A.</div>
</div>, <div class="name string-name">
<div class="surname">Difazio</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Jansson</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="volume">313</div>, <div class="fpage">1596</div>–<div class="lpage">604</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1128691"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ma</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Zhou</div> <div class="given-names">G.K.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genomic insights into salt adaptation in a desert poplar</div>, <div class="source">Nat. Commun</div> ., <div class="volume">4</div>, 2797–805.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Genomic%20insights%20into%20salt%20adaptation%20in%20a%20desert%20poplar&amp;aulast=Ma&amp;date=2013&amp;volume=4" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomic%20insights%20into%20salt%20adaptation%20in%20a%20desert%20poplar&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Ma</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Ma</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">The draft genome sequence of a desert tree <em>Populus pruinosa</em>
</div>, <div class="source">Gigascience</div> , <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20draft%20genome%20sequence%20of%20a%20desert%20tree%20Populus%20pruinosa&amp;aulast=Yang&amp;date=2017&amp;spage=1&amp;epage=7&amp;volume=6" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20draft%20genome%20sequence%20of%20a%20desert%20tree%20Populus%20pruinosa&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ma</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Wan</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Duan</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2019</div>, 
<div class="article-title">Genome sequence and genetic transformation of a widely distributed and cultivated poplar</div>, <div class="source">Plant Biotechnol. J</div> ., <div class="volume">17</div>, <div class="fpage">451</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20genetic%20transformation%20of%20a%20widely%20distributed%20and%20cultivated%20poplar&amp;author=J.%20Ma&amp;author=D.%20Wan&amp;author=B.%20Duan&amp;publication_year=2019&amp;journal=Plant%20Biotechnol.%20J&amp;volume=17&amp;pages=451-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/pbi.2019.17.issue-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2019.17.issue-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fpbi.2019.17.issue-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30044051" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20and%20genetic%20transformation%20of%20a%20widely%20distributed%20and%20cultivated%20poplar&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Song</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">Z.Z.</div>
</div>, <div class="name string-name">
<div class="surname">Lourenco</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Chiang</div> <div class="given-names">V.L.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Genetic transformation of <em>Populus trichocarpa</em> genotype Nisqually-1: a functional genomic tool for woody plants</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">47</div>, <div class="fpage">1582</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genetic%20transformation%20of%20Populus%20trichocarpa%20genotype%20Nisqually-1%3A%20a%20functional%20genomic%20tool%20for%20woody%20plants&amp;author=J.%20Song&amp;author=S.%20Lu&amp;author=Z.Z.%20Chen&amp;author=R.%20Lourenco&amp;author=V.L.%20Chiang&amp;publication_year=2006&amp;journal=Plant%20Cell%20Physiol&amp;volume=47&amp;pages=1582-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/pcp/pcl018" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcl018" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fpcp%2Fpcl018"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17018558" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genetic%20transformation%20of%20Populus%20trichocarpa%20genotype%20Nisqually-1%3A%20a%20functional%20genomic%20tool%20for%20woody%20plants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">S.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">Q.J.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">W.L.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">X.C.</div>
</div>, <div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">M.Z.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Salt tolerance conferred by over-expression of OsNHX1 gene in Poplar 84K</div>, <div class="source">Chinese Sci. Bull</div> ., <div class="volume">50</div>, <div class="fpage">224</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Salt%20tolerance%20conferred%20by%20over-expression%20of%20OsNHX1%20gene%20in%20Poplar%2084K&amp;author=S.Y.%20Wang&amp;author=Q.J.%20Chen&amp;author=W.L.%20Wang&amp;author=X.C.%20Wang&amp;author=M.Z.%20Lu&amp;publication_year=2005&amp;journal=Chinese%20Sci.%20Bull&amp;volume=50&amp;pages=224-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1360/982004-236" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1360%2F982004-236" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1360%2F982004-236"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Salt%20tolerance%20conferred%20by%20over-expression%20of%20OsNHX1%20gene%20in%20Poplar%2084K&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Coleman</div> <div class="given-names">H.D.</div>
</div>, <div class="name string-name">
<div class="surname">Canovas</div> <div class="given-names">F.M.</div>
</div>, <div class="name string-name">
<div class="surname">Man</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Kirby</div> <div class="given-names">E.G.</div>
</div>, <div class="name string-name">
<div class="surname">Mansfield</div> <div class="given-names">S.D.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Enhanced expression of glutamine synthetase (GS1a) confers altered fibre and wood chemistry in field grown hybrid poplar (<em>Populus tremula</em> X <em>alba</em>) (717-1B4)</div>, <div class="source">Plant Biotechnol J</div> ., <div class="volume">10</div>, <div class="fpage">883</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Enhanced%20expression%20of%20glutamine%20synthetase%20(GS1a)%20confers%20altered%20fibre%20and%20wood%20chemistry%20in%20field%20grown%20hybrid%20poplar%20(Populus%20tremula%20X%20alba)%20(717-1B4)&amp;author=H.D.%20Coleman&amp;author=F.M.%20Canovas&amp;author=H.%20Man&amp;author=E.G.%20Kirby&amp;author=S.D.%20Mansfield&amp;publication_year=2012&amp;journal=Plant%20Biotechnol%20J&amp;volume=10&amp;pages=883-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1467-7652.2012.00714.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00714.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1467-7652.2012.00714.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22672155" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Enhanced%20expression%20of%20glutamine%20synthetase%20(GS1a)%20confers%20altered%20fibre%20and%20wood%20chemistry%20in%20field%20grown%20hybrid%20poplar%20(Populus%20tremula%20X%20alba)%20(717-1B4)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Maloney</div> <div class="given-names">V.J.</div>
</div>, <div class="name string-name">
<div class="surname">Mansfield</div> <div class="given-names">S.D.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Characterization and varied expression of a membrane-bound endo-beta-1,4-glucanase in hybrid poplar</div>, <div class="source">Plant Biotechnol J</div> ., <div class="volume">8</div>, <div class="fpage">294</div>–<div class="lpage">307</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20varied%20expression%20of%20a%20membrane-bound%20endo-beta-1%2C4-glucanase%20in%20hybrid%20poplar&amp;author=V.J.%20Maloney&amp;author=S.D.%20Mansfield&amp;publication_year=2010&amp;journal=Plant%20Biotechnol%20J&amp;volume=8&amp;pages=294-307" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/pbi.2010.8.issue-3" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2010.8.issue-3" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fpbi.2010.8.issue-3"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20070872" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterization%20and%20varied%20expression%20of%20a%20membrane-bound%20endo-beta-1%2C4-glucanase%20in%20hybrid%20poplar&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cho</div> <div class="given-names">J.S.</div>
</div>, <div class="name string-name">
<div class="surname">Nguyen</div> <div class="given-names">V.P.</div>
</div>, <div class="name string-name">
<div class="surname">Jeon</div> <div class="given-names">H.W.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Overexpression of PtrMYB119, a R2R3-MYB transcription factor from <em>Populus trichocarpa</em>, promotes anthocyanin production in hybrid poplar</div>, <div class="source">Tree Physiol</div> ., <div class="volume">36</div>, <div class="fpage">1162</div>–<div class="lpage">76</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Overexpression%20of%20PtrMYB119%2C%20a%20R2R3-MYB%20transcription%20factor%20from%20Populus%20trichocarpa%2C%20promotes%20anthocyanin%20production%20in%20hybrid%20poplar&amp;author=J.S.%20Cho&amp;author=V.P.%20Nguyen&amp;author=H.W.%20Jeon&amp;publication_year=2016&amp;journal=Tree%20Physiol&amp;volume=36&amp;pages=1162-76" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/treephys/tpw046" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ftreephys%2Ftpw046" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Ftreephys%2Ftpw046"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27259636" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Overexpression%20of%20PtrMYB119%2C%20a%20R2R3-MYB%20transcription%20factor%20from%20Populus%20trichocarpa%2C%20promotes%20anthocyanin%20production%20in%20hybrid%20poplar&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mader</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Le Paslier</div> <div class="given-names">M.C.</div>
</div>, <div class="name string-name">
<div class="surname">Bounon</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Whole-genome draft assembly of <em>Populus tremula</em> x <em>P. alba</em> clone INRA 717-1B4</div>, <div class="source">Silvae Genet</div> ., <div class="volume">65</div>, <div class="fpage">74</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20draft%20assembly%20of%20Populus%20tremula%20x%20P.%20alba%20clone%20INRA%20717-1B4&amp;author=M.%20Mader&amp;author=M.C.%20Le%20Paslier&amp;author=R.%20Bounon&amp;publication_year=2016&amp;journal=Silvae%20Genet&amp;volume=65&amp;pages=74-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1515/sg-2016-0019" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1515%2Fsg-2016-0019" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1515%2Fsg-2016-0019"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Whole-genome%20draft%20assembly%20of%20Populus%20tremula%20x%20P.%20alba%20clone%20INRA%20717-1B4&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Shim</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Choi</div> <div class="given-names">Y.I.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Transgenic poplar trees expressing yeast cadmium factor 1 exhibit the characteristics necessary for the phytoremediation of mine tailing soil</div>, <div class="source">Chemosphere</div> , <div class="volume">90</div>, <div class="fpage">1478</div>–<div class="lpage">86</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transgenic%20poplar%20trees%20expressing%20yeast%20cadmium%20factor%201%20exhibit%20the%20characteristics%20necessary%20for%20the%20phytoremediation%20of%20mine%20tailing%20soil&amp;author=D.%20Shim&amp;author=S.%20Kim&amp;author=Y.I.%20Choi&amp;publication_year=2013&amp;journal=Chemosphere&amp;volume=90&amp;pages=1478-86" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.chemosphere.2012.09.044" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chemosphere.2012.09.044" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.chemosphere.2012.09.044"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23062827" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transgenic%20poplar%20trees%20expressing%20yeast%20cadmium%20factor%201%20exhibit%20the%20characteristics%20necessary%20for%20the%20phytoremediation%20of%20mine%20tailing%20soil&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Doyle</div> <div class="given-names">J.J.</div>
</div>, <div class="name string-name">
<div class="surname">Doyle</div> <div class="given-names">J.L.</div>
</div>
</div> <div class="year">1990</div>, 
<div class="article-title">Isolation of plant DNA from fresh tissue</div>, <div class="source">Focus</div> , <div class="volume">12</div>, <div class="fpage">13</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Isolation%20of%20plant%20DNA%20from%20fresh%20tissue&amp;aulast=Doyle&amp;date=1990&amp;spage=13&amp;epage=5&amp;volume=12" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Isolation%20of%20plant%20DNA%20from%20fresh%20tissue&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Galbraith</div> <div class="given-names">D.W.</div>
</div>, <div class="name string-name">
<div class="surname">Harkins</div> <div class="given-names">K.R.</div>
</div>, <div class="name string-name">
<div class="surname">Maddox</div> <div class="given-names">J.M.</div>
</div>, <div class="name string-name">
<div class="surname">Ayres</div> <div class="given-names">N.M.</div>
</div>, <div class="name string-name">
<div class="surname">Sharma</div> <div class="given-names">D.P.</div>
</div>, <div class="name string-name">
<div class="surname">Firoozabady</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">1983</div>, 
<div class="article-title">Rapid flow cytometric analysis of the cell cycle in intact plant tissues</div>, <div class="source">Science</div> , <div class="volume">220</div>, <div class="fpage">1049</div>–<div class="lpage">51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rapid%20flow%20cytometric%20analysis%20of%20the%20cell%20cycle%20in%20intact%20plant%20tissues&amp;author=D.W.%20Galbraith&amp;author=K.R.%20Harkins&amp;author=J.M.%20Maddox&amp;author=N.M.%20Ayres&amp;author=D.P.%20Sharma&amp;author=E.%20Firoozabady&amp;publication_year=1983&amp;journal=Science&amp;volume=220&amp;pages=1049-51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.220.4601.1049" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.220.4601.1049" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.220.4601.1049"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17754551" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rapid%20flow%20cytometric%20analysis%20of%20the%20cell%20cycle%20in%20intact%20plant%20tissues&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Shi</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Yuan</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2013</div>, Estimation of genomic characteristics by analyzing k-mer frequency in de novo genome projects. <em>arXiv</em>, 1308.2012.</div>
</div></div></div>
<div content-id="dsz020-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Koren</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Walenz</div> <div class="given-names">B.P.</div>
</div>, <div class="name string-name">
<div class="surname">Berlin</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Miller</div> <div class="given-names">J.R.</div>
</div>, <div class="name string-name">
<div class="surname">Bergman</div> <div class="given-names">N.H.</div>
</div>, <div class="name string-name">
<div class="surname">Phillippy</div> <div class="given-names">A.M.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation</div>, <div class="source">Genome Res</div> ., <div class="volume">27</div>, <div class="fpage">722</div>–<div class="lpage">36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&amp;author=S.%20Koren&amp;author=B.P.%20Walenz&amp;author=K.%20Berlin&amp;author=J.R.%20Miller&amp;author=N.H.%20Bergman&amp;author=A.M.%20Phillippy&amp;publication_year=2017&amp;journal=Genome%20Res&amp;volume=27&amp;pages=722-36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.215087.116" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.215087.116" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.215087.116"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28298431" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chin</div> <div class="given-names">C.S.</div>
</div>, <div class="name string-name">
<div class="surname">Alexander</div> <div class="given-names">D.H.</div>
</div>, <div class="name string-name">
<div class="surname">Marks</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">563</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nonhybrid%2C%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data&amp;author=C.S.%20Chin&amp;author=D.H.%20Alexander&amp;author=P.%20Marks&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=563-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.2474" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2474" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.2474"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644548" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nonhybrid%2C%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Walker</div> <div class="given-names">B.J.</div>
</div>, <div class="name string-name">
<div class="surname">Abeel</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Shea</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e112963</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;author=B.J.%20Walker&amp;author=T.%20Abeel&amp;author=T.%20Shea&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e112963" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0112963" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112963" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0112963"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Camacho</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Coulouris</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Avagyan</div> <div class="given-names">V.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">BLAST+: architecture and applications</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">10</div>, <div class="fpage">421</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BLAST%2B%3A%20architecture%20and%20applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=421" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-10-421" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-421" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-10-421"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BLAST%2B%3A%20architecture%20and%20applications&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kersten</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Faivre Rampant</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Mader</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Genome sequences of <em>Populus tremula</em> chloroplast and mitochondrion: implications for holistic poplar breeding</div>, <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0147209</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequences%20of%20Populus%20tremula%20chloroplast%20and%20mitochondrion%3A%20implications%20for%20holistic%20poplar%20breeding&amp;author=B.%20Kersten&amp;author=P.%20Faivre%20Rampant&amp;author=M.%20Mader&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0147209" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0147209" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0147209" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0147209"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26800039" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequences%20of%20Populus%20tremula%20chloroplast%20and%20mitochondrion%3A%20implications%20for%20holistic%20poplar%20breeding&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tillich</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Lehwark</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Pellizzer</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">GeSeq - versatile and accurate annotation of organelle genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">W6</div>–<div class="lpage">W11</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=GeSeq%20-%20versatile%20and%20accurate%20annotation%20of%20organelle%20genomes&amp;author=M.%20Tillich&amp;author=P.%20Lehwark&amp;author=T.%20Pellizzer&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=W6-W11" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkx391" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkx391" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkx391"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28486635" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:GeSeq%20-%20versatile%20and%20accurate%20annotation%20of%20organelle%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Burton</div> <div class="given-names">J.N.</div>
</div>, <div class="name string-name">
<div class="surname">Adey</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Patwardhan</div> <div class="given-names">R.P.</div>
</div>, <div class="name string-name">
<div class="surname">Qiu</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Kitzman</div> <div class="given-names">J.O.</div>
</div>, <div class="name string-name">
<div class="surname">Shendure</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">1119</div>–<div class="lpage">25</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Chromosome-scale%20scaffolding%20of%20de%20novo%20genome%20assemblies%20based%20on%20chromatin%20interactions&amp;author=J.N.%20Burton&amp;author=A.%20Adey&amp;author=R.P.%20Patwardhan&amp;author=R.%20Qiu&amp;author=J.O.%20Kitzman&amp;author=J.%20Shendure&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=1119-25" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.2727" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2727" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.2727"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185095" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Chromosome-scale%20scaffolding%20of%20de%20novo%20genome%20assemblies%20based%20on%20chromatin%20interactions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Akdemir</div> <div class="given-names">K.C.</div>
</div>, <div class="name string-name">
<div class="surname">Chin</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">HiCPlotter integrates genomic data with interaction matrices</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">198</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=HiCPlotter%20integrates%20genomic%20data%20with%20interaction%20matrices&amp;author=K.C.%20Akdemir&amp;author=L.%20Chin&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=198" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-015-0767-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0767-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0767-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26392354" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:HiCPlotter%20integrates%20genomic%20data%20with%20interaction%20matrices&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Langmead</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">HISAT: a fast spliced aligner with low memory requirements</div>, <div class="source">Nat. Methods</div> ., <div class="volume">12</div>, <div class="fpage">357</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=HISAT%3A%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&amp;author=D.%20Kim&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=357-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.3317" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3317" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.3317"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25751142" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:HISAT%3A%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div></div> <div class="year">2013</div>, Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <em>arXiv preprint arXiv: 1303.3997</em>.</div>
</div></div></div>
<div content-id="dsz020-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Simao</div> <div class="given-names">F.A.</div>
</div>, <div class="name string-name">
<div class="surname">Waterhouse</div> <div class="given-names">R.M.</div>
</div>, <div class="name string-name">
<div class="surname">Ioannidis</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Kriventseva</div> <div class="given-names">E.V.</div>
</div>, <div class="name string-name">
<div class="surname">Zdobnov</div> <div class="given-names">E.M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.A.%20Simao&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtv351"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ou</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">N.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">Assessing genome assembly quality using the LTR Assembly Index (LAI)</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">46</div>, <div class="fpage">e126</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assessing%20genome%20assembly%20quality%20using%20the%20LTR%20Assembly%20Index%20(LAI)&amp;author=S.%20Ou&amp;author=J.%20Chen&amp;author=N.%20Jiang&amp;publication_year=2018&amp;journal=Nucleic%20Acids%20Res&amp;volume=46&amp;pages=e126" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30107434" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Assessing%20genome%20assembly%20quality%20using%20the%20LTR%20Assembly%20Index%20(LAI)&amp;aulast=Ou&amp;date=2018&amp;spage=e126&amp;volume=46" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assessing%20genome%20assembly%20quality%20using%20the%20LTR%20Assembly%20Index%20(LAI)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">McKenna</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Hanna</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Banks</div> <div class="given-names">E.</div>
</div>
</div>, et al.  <div class="year">2010</div>, 
<div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">303</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.107524.110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Benson</div> <div class="given-names">G.</div>
</div></div> <div class="year">1999</div>, 
<div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">573</div>–<div class="lpage">80</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F27.2.573"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">N.</div>
</div></div> <div class="year">2004</div>, 
<div class="article-title">Using RepeatMasker to identify repetitive elements in genomic sequences</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , Chapter 4, Unit 4.10.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Using%20RepeatMasker%20to%20identify%20repetitive%20elements%20in%20genomic%20sequences&amp;aulast=Chen&amp;date=2004" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20RepeatMasker%20to%20identify%20repetitive%20elements%20in%20genomic%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bao</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Kojima</div> <div class="given-names">K.K.</div>
</div>, <div class="name string-name">
<div class="surname">Kohany</div> <div class="given-names">O.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Repbase update, a database of repetitive elements in eukaryotic genomes</div>, <div class="source">Mob. DNA</div> , <div class="volume">6</div>, <div class="fpage">11</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Repbase%20update%2C%20a%20database%20of%20repetitive%20elements%20in%20eukaryotic%20genomes&amp;author=W.%20Bao&amp;author=K.K.%20Kojima&amp;author=O.%20Kohany&amp;publication_year=2015&amp;journal=Mob.%20DNA&amp;volume=6&amp;pages=11" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13100-015-0041-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13100-015-0041-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13100-015-0041-9"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26045719" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Repbase%20update%2C%20a%20database%20of%20repetitive%20elements%20in%20eukaryotic%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hubley</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Finn</div> <div class="given-names">R.D.</div>
</div>, <div class="name string-name">
<div class="surname">Clements</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">The Dfam database of repetitive DNA families</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D81</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Dfam%20database%20of%20repetitive%20DNA%20families&amp;author=R.%20Hubley&amp;author=R.D.%20Finn&amp;author=J.%20Clements&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D81-89" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkv1272" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1272" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkv1272"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26612867" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Dfam%20database%20of%20repetitive%20DNA%20families&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kalvari</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Nawrocki</div> <div class="given-names">E.P.</div>
</div>, <div class="name string-name">
<div class="surname">Argasinska</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Non-coding RNA analysis using the Rfam database</div>, <div class="source">Curr. Protoc. Bioinformatics</div> ., <div class="volume">62</div>, <div class="fpage">e51</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Non-coding%20RNA%20analysis%20using%20the%20Rfam%20database&amp;author=I.%20Kalvari&amp;author=E.P.%20Nawrocki&amp;author=J.%20Argasinska&amp;publication_year=2018&amp;journal=Curr.%20Protoc.%20Bioinformatics&amp;volume=62&amp;pages=e51" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/cpbi.v62.1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fcpbi.v62.1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fcpbi.v62.1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29927072" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Non-coding%20RNA%20analysis%20using%20the%20Rfam%20database&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nawrocki</div> <div class="given-names">E.P.</div>
</div>, <div class="name string-name">
<div class="surname">Eddy</div> <div class="given-names">S.R.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Infernal 1.1: 100-fold faster RNA homology searches</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">2933</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Infernal%201.1%3A%20100-fold%20faster%20RNA%20homology%20searches&amp;author=E.P.%20Nawrocki&amp;author=S.R.%20Eddy&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=2933-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt509" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt509" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt509"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24008419" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Infernal%201.1%3A%20100-fold%20faster%20RNA%20homology%20searches&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lowe</div> <div class="given-names">T.M.</div>
</div>, <div class="name string-name">
<div class="surname">Chan</div> <div class="given-names">P.P.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">tRNAscan-SE on-line: integrating search and context for analysis of transfer RNA genes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">W54</div>–<div class="lpage">57</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%20on-line%3A%20integrating%20search%20and%20context%20for%20analysis%20of%20transfer%20RNA%20genes&amp;author=T.M.%20Lowe&amp;author=P.P.%20Chan&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=W54-57" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkw413" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw413" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkw413"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27174935" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:tRNAscan-SE%20on-line%3A%20integrating%20search%20and%20context%20for%20analysis%20of%20transfer%20RNA%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lagesen</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Hallin</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Rodland</div> <div class="given-names">E.A.</div>
</div>, <div class="name string-name">
<div class="surname">Staerfeldt</div> <div class="given-names">H.H.</div>
</div>, <div class="name string-name">
<div class="surname">Rognes</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Ussery</div> <div class="given-names">D.W.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">3100</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RNAmmer%3A%20consistent%20and%20rapid%20annotation%20of%20ribosomal%20RNA%20genes&amp;author=K.%20Lagesen&amp;author=P.%20Hallin&amp;author=E.A.%20Rodland&amp;author=H.H.%20Staerfeldt&amp;author=T.%20Rognes&amp;author=D.W.%20Ussery&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=3100-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkm160" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm160" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkm160"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452365" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RNAmmer%3A%20consistent%20and%20rapid%20annotation%20of%20ribosomal%20RNA%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Debarry</div> <div class="given-names">J.D.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.D.%20Debarry&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1293"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kurtz</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Phillippy</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Delcher</div> <div class="given-names">A.L.</div>
</div>
</div>, et al.  <div class="year">2004</div>, 
<div class="article-title">Versatile and open software for comparing large genomes</div>, <div class="source">Genome Biol</div> ., <div class="volume">5</div>, <div class="fpage">R12</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2004-5-2-r12"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lin</div> <div class="given-names">Y.C.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Delhomme</div> <div class="given-names">N.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Functional and evolutionary genomic inferences in Populus through genome and population sequencing of American and European aspen</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">115</div>, <div class="fpage">E10970</div>–<div class="lpage">E10978</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20evolutionary%20genomic%20inferences%20in%20Populus%20through%20genome%20and%20population%20sequencing%20of%20American%20and%20European%20aspen&amp;author=Y.C.%20Lin&amp;author=J.%20Wang&amp;author=N.%20Delhomme&amp;publication_year=2018&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=115&amp;pages=E10970-E10978" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1801437115" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1801437115" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1801437115"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30373829" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Functional%20and%20evolutionary%20genomic%20inferences%20in%20Populus%20through%20genome%20and%20population%20sequencing%20of%20American%20and%20European%20aspen&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">D.R.</div>
</div>, <div class="name string-name">
<div class="surname">Crosby</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">R.W.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Correlation between nuclear plastid DNA abundance and plastid number supports the limited transfer window hypothesis</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">3</div>, <div class="fpage">365</div>–<div class="lpage">71</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20nuclear%20plastid%20DNA%20abundance%20and%20plastid%20number%20supports%20the%20limited%20transfer%20window%20hypothesis&amp;author=D.R.%20Smith&amp;author=K.%20Crosby&amp;author=R.W.%20Lee&amp;publication_year=2011&amp;journal=Genome%20Biol.%20Evol&amp;volume=3&amp;pages=365-71" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/gbe/evr001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevr001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgbe%2Fevr001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21292629" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Correlation%20between%20nuclear%20plastid%20DNA%20abundance%20and%20plastid%20number%20supports%20the%20limited%20transfer%20window%20hypothesis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Krzywinski</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Schein</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Birol</div> <div class="given-names">I.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Circos: an information aesthetic for comparative genomics</div>, <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1639</div>–<div class="lpage">45</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.092759.109"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nattestad</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Schatz</div> <div class="given-names">M.C.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Assemblytics: a web analytics tool for the detection of variants from an assembly</div>, <div class="source">Bioinformatics</div> , <div class="volume">32</div>, <div class="fpage">3021</div>–<div class="lpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assemblytics%3A%20a%20web%20analytics%20tool%20for%20the%20detection%20of%20variants%20from%20an%20assembly&amp;author=M.%20Nattestad&amp;author=M.C.%20Schatz&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=3021-3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btw369" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtw369" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtw369"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27318204" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assemblytics%3A%20a%20web%20analytics%20tool%20for%20the%20detection%20of%20variants%20from%20an%20assembly&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pertea</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Pertea</div> <div class="given-names">G.M.</div>
</div>, <div class="name string-name">
<div class="surname">Antonescu</div> <div class="given-names">C.M.</div>
</div>, <div class="name string-name">
<div class="surname">Chang</div> <div class="given-names">T.C.</div>
</div>, <div class="name string-name">
<div class="surname">Mendell</div> <div class="given-names">J.T.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">290</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&amp;author=M.%20Pertea&amp;author=G.M.%20Pertea&amp;author=C.M.%20Antonescu&amp;author=T.C.%20Chang&amp;author=J.T.%20Mendell&amp;author=S.L.%20Salzberg&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=290-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.3122" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3122" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.3122"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25690850" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Robinson</div> <div class="given-names">M.D.</div>
</div>, <div class="name string-name">
<div class="surname">McCarthy</div> <div class="given-names">D.J.</div>
</div>, <div class="name string-name">
<div class="surname">Smyth</div> <div class="given-names">G.K.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">139</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;author=M.D.%20Robinson&amp;author=D.J.%20McCarthy&amp;author=G.K.%20Smyth&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp616" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp616" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp616"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz020-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz020-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Emms</div> <div class="given-names">D.M.</div>
</div>, <div class="name string-name">
<div class="surname">Kelly</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">157</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;author=D.M.%20Emms&amp;author=S.%20Kelly&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=157" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s13059-015-0721-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0721-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0721-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26243257" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="dsz020-FM1"><div class="fn"><div class="footnote-content">Deyou Qiu, Shenglong Bai and Jianchao Ma authors contributed equally to this work.</div></div></div>
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5580662&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/5/10.1093_dnares_dsz020/1/dsz020_supplementary_data.zip?Expires=1576701882&amp;Signature=3b4MijlxrcQk~hhek2rvGCcJ2KA9G2aeHN2p3Zvo4oWU4eUzuyWibooQvSDI~DEyZXbALMbmxhOGAKOySBkt~lAWwQ-kIUELhMQxCIDuQr6CRizW60D-X6g83qnpc0mCtNXPbwwA4tsVSbfQrYMQwxD9D9hOl58z-ofD2p6wDBe4a0k9L0vc4unGFF--K8ftKu1qnq-CiefymGGhxdREvsV5dmiTdwJhn5Wf~PR84XEzAhrgJvRpI3M-c0jKLvPhRDLiiADrO5MahqIfmxGC8S7Kb8CLpC9~E6Vae6VJJ16g36NpUltve7nT-~Zif7WIM3K7Dm65whsyJZsdRFnGag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz020_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">727</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz020"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5580662"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f5%2f423%2f5580662">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F5%2F423%2F5580662" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5580662">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="genome of Populus alba x Populus tremula var. glandulosa clone 84K | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz020");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f5%2f423%2f5580662">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
