<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>Genome sequence of the non-conventional wine yeast Hanseniaspora guilliermondii UTAD222 unveils relevant traits of this species and of the Hanseniaspora genus in the context of wine fermentation | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/1/67/5193709","@type":"ScholarlyArticle","name":"Genome sequence of the non-conventional wine yeast  Hanseniaspora guilliermondii  UTAD222 unveils relevant traits of this species and of the  Hanseniaspora  genus in the context of wine fermentation","datePublished":"2018-11-20","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2019-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy039","keywords":["<italic>Hanseniaspora guilliermondii</italic>","non-Saccharomyces wine yeasts","genome sequencing and annotation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/1/67/5193709","author":[{"name":"Seixas, Isabel","affiliation":"WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal  BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal","@type":"Person"},{"name":"Barbosa, Catarina","affiliation":"WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal  BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal","@type":"Person"},{"name":"Mendes-Faia, Arlete","affiliation":"WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal  BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal","@type":"Person"},{"name":"Güldener, Ulrich","affiliation":"Department of Bioinformatics, Wissenschaftszentrum Weihenstephan, Technische Universität München, Maximus von-Imhof-Forum 3, Freising, Germany","@type":"Person"},{"name":"Tenreiro, Rogério","affiliation":"BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal","@type":"Person"},{"name":"Mendes-Ferreira, Ana","affiliation":"WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal  BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal","@type":"Person"},{"name":"Mira, Nuno P","affiliation":"Department of Bioengineering, iBB - Institute for Bioengineering and Biosciences, Instituto Superior Técnico, Universidade de Lisboa, Avenida Rovisco Pais, Lisbon, Portugal","@type":"Person"}],"description":"Abstract.  Hanseanispora species, including H. guilliermondii, are long known to be abundant in wine grape-musts and to play a critical role in vinification by ","pageStart":"67","pageEnd":"83","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636236965&Signature=LIYH8aaD~aye4OMqhDylzex8PCWPG~hKvsgh5Iv0pE~gMpi3Wb5UQxy6rcf5SYlvDV-ihjyPJmJ-l-N88o-0vGsgkY3r-LJIQm2s1VWhd2R7-BUmimUByBcnojGRp0s~UgPZM6f3DreyetthH3xZdzOCNjjf5ap09BdVCyVBkzDMpR4V4MwIPFwDJ2dOeYPAGqwzFsjqwrEyQE~IXrk9rFvFS1arMj3AJLs8ffQIA3Ts4Xe-QmtTmym4jUAA4PXaxPQd34051cI4Y~AfoctuAhJws4axRof14MmnMCEU34SYNTpu7A~4tiid9NzCTJ-Zi7id3wxgcut0IAnktojwFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome sequence of the non-conventional wine yeast  Hanseniaspora guilliermondii  UTAD222 unveils relevant traits of this species and of the  Hanseniaspora  genus in the context of wine fermentation","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636236965&Signature=LIYH8aaD~aye4OMqhDylzex8PCWPG~hKvsgh5Iv0pE~gMpi3Wb5UQxy6rcf5SYlvDV-ihjyPJmJ-l-N88o-0vGsgkY3r-LJIQm2s1VWhd2R7-BUmimUByBcnojGRp0s~UgPZM6f3DreyetthH3xZdzOCNjjf5ap09BdVCyVBkzDMpR4V4MwIPFwDJ2dOeYPAGqwzFsjqwrEyQE~IXrk9rFvFS1arMj3AJLs8ffQIA3Ts4Xe-QmtTmym4jUAA4PXaxPQd34051cI4Y~AfoctuAhJws4axRof14MmnMCEU34SYNTpu7A~4tiid9NzCTJ-Zi7id3wxgcut0IAnktojwFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="Genome sequence of the non-conventional wine yeast  Hanseniaspora guilliermondii  UTAD222 unveils relevant traits of this species and of the  Hanseniaspora  genus in the context of wine fermentation">
<meta property="og:description" content="Abstract.  Hanseanispora species, including H. guilliermondii, are long known to be abundant in wine grape-musts and to play a critical role in vinification by ">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/1/67/5193709">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1636236965&amp;Signature=LIYH8aaD~aye4OMqhDylzex8PCWPG~hKvsgh5Iv0pE~gMpi3Wb5UQxy6rcf5SYlvDV-ihjyPJmJ-l-N88o-0vGsgkY3r-LJIQm2s1VWhd2R7-BUmimUByBcnojGRp0s~UgPZM6f3DreyetthH3xZdzOCNjjf5ap09BdVCyVBkzDMpR4V4MwIPFwDJ2dOeYPAGqwzFsjqwrEyQE~IXrk9rFvFS1arMj3AJLs8ffQIA3Ts4Xe-QmtTmym4jUAA4PXaxPQd34051cI4Y~AfoctuAhJws4axRof14MmnMCEU34SYNTpu7A~4tiid9NzCTJ-Zi7id3wxgcut0IAnktojwFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2018-11-20">
<meta name="citation_author" content="Seixas, Isabel">
<meta name="citation_author_institution" content="WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal">
<meta name="citation_author_institution" content="BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal">
<meta name="citation_author" content="Barbosa, Catarina">
<meta name="citation_author_institution" content="WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal">
<meta name="citation_author_institution" content="BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal">
<meta name="citation_author" content="Mendes-Faia, Arlete">
<meta name="citation_author_institution" content="WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal">
<meta name="citation_author_institution" content="BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal">
<meta name="citation_author" content="Güldener, Ulrich">
<meta name="citation_author_institution" content="Department of Bioinformatics, Wissenschaftszentrum Weihenstephan, Technische Universität München, Maximus von-Imhof-Forum 3, Freising, Germany">
<meta name="citation_author" content="Tenreiro, Rogério">
<meta name="citation_author_institution" content="BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal">
<meta name="citation_author" content="Mendes-Ferreira, Ana">
<meta name="citation_author_institution" content="WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal">
<meta name="citation_author_institution" content="BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal">
<meta name="citation_author" content="Mira, Nuno P">
<meta name="citation_author_institution" content="Department of Bioengineering, iBB - Institute for Bioengineering and Biosciences, Instituto Superior Técnico, Universidade de Lisboa, Avenida Rovisco Pais, Lisbon, Portugal">
<meta name="citation_title" content="Genome sequence of the non-conventional wine yeast Hanseniaspora guilliermondii UTAD222 unveils relevant traits of this species and of the Hanseniaspora genus in the context of wine fermentation">
<meta name="citation_firstpage" content="67">
<meta name="citation_lastpage" content="83">
<meta name="citation_doi" content="10.1093/dnares/dsy039">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="1">
<meta name="citation_publication_date" content="2019/02/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Associations among wine grape microbiome, metabolome, and fermentation behavior suggest microbial contribution to regional wine characteristics;  citation_journal_title=MBio;  citation_year=2016;  citation_volume=7;  citation_pages=e00631-16; ">
<meta name="citation_reference" content="citation_title=Microbial biogeography of wine grapes is conditioned by cultivar, vintage, and climate;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2014;  citation_volume=111;  citation_pages=E139-48; ">
<meta name="citation_reference" content="citation_title=Microbial terroir and food innovation: the case of yeast biodiversity in wine;  citation_journal_title=Microbiol. Res;  citation_year=2015;  citation_volume=181;  citation_pages=75-83; ">
<meta name="citation_reference" content="citation_title=Not your ordinary yeast: non-Saccharomyces yeasts in wine production uncovered;  citation_journal_title=FEMS Yeast Res;  citation_year=2014;  citation_volume=14;  citation_pages=215-37; ">
<meta name="citation_reference" content="citation_title=Microbial contribution to wine aroma and its intended use for wine quality improvement;  citation_journal_title=Molecules;  citation_year=2017;  citation_volume=22;  citation_pages=189">
<meta name="citation_reference" content="citation_title=Fungal diversity in grape must and wine fermentation assessed by massive sequencing, quantitative PCR and DGGE;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=1156">
<meta name="citation_reference" content="citation_title=Dynamics and diversity of non-Saccharomyces yeasts during the early stages in winemaking;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2008;  citation_volume=125;  citation_pages=197-203; ">
<meta name="citation_reference" content="citation_title=Diversity of yeast strains of the genus Hanseniaspora in the winery environment: what is their involvement in grape must fermentation?;  citation_journal_title=Food Microbiol;  citation_year=2015;  citation_volume=50;  citation_pages=70-7; ">
<meta name="citation_reference" content="citation_title=A combined meta-barcoding and shotgun metagenomic analysis of spontaneous wine fermentation;  citation_journal_title=Gigascience;  citation_year=2017;  citation_volume=6;  citation_pages=1-10; ">
<meta name="citation_reference" content="citation_title=Persistence of two non-saccharomyces yeasts (Hanseniaspora and Starmerella) in the cellar;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=268">
<meta name="citation_reference" content="citation_title=Yeast biodiversity from DOQ priorat uninoculated fermentations;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=930">
<meta name="citation_reference" content="citation_title=Diversity of Saccharomyces and non-Saccharomyces yeasts in three red grape varieties cultured in the Serrania de Ronda (Spain) vine-growing region;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2010;  citation_volume=143;  citation_pages=241-5; ">
<meta name="citation_reference" content="citation_title=Indigenous Georgian wine-associated yeasts and grape cultivars to edit the wine quality in a precision oenology perspective;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=352">
<meta name="citation_reference" content="citation_title=Yeast species associated with spontaneous fermentation of taberna, a traditional palm wine from the southeast of Mexico;  citation_journal_title=Ann. Microbiol;  citation_year=2015;  citation_volume=65;  citation_pages=287-96; ">
<meta name="citation_reference" content="citation_title=Analysis of non-Saccharomyces yeast populations isolated from grape musts from Sicily (Italy);  citation_journal_title=J. Appl. Microbiol;  citation_year=2008;  citation_volume=105;  citation_pages=2248-54; ">
<meta name="citation_reference" content="citation_title=Heavy sulphur compounds, higher alcohols and esters production profile of Hanseniaspora uvarum and Hanseniaspora guilliermondii grown as pure and mixed cultures in grape must;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2008;  citation_volume=124;  citation_pages=231-8; ">
<meta name="citation_reference" content="citation_title=H. guilliermondii impacts growth kinetics and metabolic activity of S. cerevisiae: the role of initial nitrogen concentration;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2014;  citation_volume=172;  citation_pages=62-9; ">
<meta name="citation_reference" content="citation_title=Volatile compounds contribution of Hanseniaspora guilliermondii and Hanseniaspora uvarum during red wine vinifications;  citation_journal_title=Food Control;  citation_year=2011;  citation_volume=22;  citation_pages=662-7; ">
<meta name="citation_reference" content="citation_title=Acetate ester formation in wine by mixed cultures in laboratory fermentations;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2003;  citation_volume=86;  citation_pages=181-8; ">
<meta name="citation_reference" content="citation_title=Influence of fermentation with Hanseniaspora sp. yeast on the volatile profile of fermented apple;  citation_journal_title=J. Agric. Food Chem;  citation_year=2012;  citation_volume=60;  citation_pages=9815-21; ">
<meta name="citation_reference" content="citation_title=The impact of yeast starter cultures on the microbial communities and volatile compounds in cocoa fermentation and the resulting sensory attributes of chocolate;  citation_journal_title=J. Food Sci. Technol;  citation_year=2016;  citation_volume=53;  citation_pages=1101-10; ">
<meta name="citation_reference" content="citation_title=Outlining the influence of non-conventional yeasts in wine ageing over lees;  citation_journal_title=Yeast;  citation_year=2016;  citation_volume=33;  citation_pages=329-38; ">
<meta name="citation_reference" content="citation_title=Use of non-Saccharomyces wine yeasts as novel sources of mannoproteins in wine;  citation_journal_title=Food Microbiol;  citation_year=2014;  citation_volume=43;  citation_pages=5-15; ">
<meta name="citation_reference" content="citation_title=Evaluation of non-Saccharomyces yeasts for the reduction of alcohol content in wine;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2014;  citation_volume=80;  citation_pages=1670-8; ">
<meta name="citation_reference" content="citation_title=Non-conventional yeast species for lowering ethanol content of wines;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=642">
<meta name="citation_reference" content="citation_title=The oenological potential of Hanseniaspora uvarum in simultaneous and sequential co-fermentation with Saccharomyces cerevisiae for industrial wine production;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=670">
<meta name="citation_reference" content="citation_title=Genomic expression program of Saccharomyces cerevisiae along a mixed-culture wine fermentation with Hanseniaspora guilliermondii;  citation_journal_title=Microbial. Cell Factories;  citation_year=2015;  citation_volume=14;  citation_pages=124">
<meta name="citation_reference" content="citation_title=Survival rate of wine-related yeasts during alcoholic fermentation assessed by direct live/dead staining combined with fluorescence in situ hybridization;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2012;  citation_volume=158;  citation_pages=49-57; ">
<meta name="citation_reference" content="citation_title=Cellular death of two non-Saccharomyces wine-related yeasts during mixed fermentations with Saccharomyces cerevisiae;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2006;  citation_volume=108;  citation_pages=336-45; ">
<meta name="citation_reference" content="citation_title=Ethanol tolerance of five non-Saccharomyces wine yeasts in comparison with a strain of Saccharomyces cerevisiae—influence of different culture conditions;  citation_journal_title=Food Microbiol;  citation_year=2004;  citation_volume=21;  citation_pages=439-47; ">
<meta name="citation_reference" content="citation_title=Dominance of Saccharomyces cerevisiae in alcoholic fermentation processes: role of physiological fitness and microbial interactions;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2016;  citation_volume=100;  citation_pages=2035-46; ">
<meta name="citation_reference" content="citation_title=Saccharomyces cerevisiae accumulates GAPDH-derived peptides on its cell surface that induce death of non-Saccharomyces yeasts by cell-to-cell contact;  citation_journal_title=FEMS Microbiol. Ecol;  citation_year=2017;  citation_volume=93; ">
<meta name="citation_reference" content="citation_title=Genome sequence of the nonconventional wine yeast Hanseniaspora guilliermondii UTAD222;  citation_journal_title=Genome Announc;  citation_year=2017;  citation_volume=5;  citation_pages=e01515-6; ">
<meta name="citation_reference" content="citation_title=Genome sequences of three species of Hanseniaspora isolated from spontaneous wine fermentations;  citation_journal_title=Genome Announc;  citation_year=2016;  citation_volume=4;  citation_pages=e01287-16; ">
<meta name="citation_reference" content="citation_title=Glycolytic functions are conserved in the genome of the wine yeast Hanseniaspora uvarum, and pyruvate kinase limits its capacity for alcoholic fermentation;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2017;  citation_volume=83;  citation_pages=e01580-17; ">
<meta name="citation_reference" content="citation_title=Yeast populations residing on healthy or botrytis-infected grapes from a vineyard in Attica, Greece;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2007;  citation_volume=73;  citation_pages=2765-8; ">
<meta name="citation_reference" content="citation_title=The role of non-Saccharomyces species in releasing glycosidic bound fraction of grape aroma components – a preliminary study;  citation_journal_title=J. Appl. Microbiol;  citation_year=2001;  citation_volume=91;  citation_pages=67-71; ">
<meta name="citation_reference" content="citation_title=Determination of enzymatic activities in ecotypic Saccharomyces and non-Saccharomyces yeast;  citation_journal_title=Electron. J. Environ. Agric. Food Chem;  citation_year=2004;  citation_volume=3;  citation_pages=743-50; ">
<meta name="citation_reference" content="citation_title=Biochemical and molecular characterization of Saccharomyces cerevisiae strains obtained from sugar-cane juice fermentations and their impact in cachaca production;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2008;  citation_volume=74;  citation_pages=693-701; ">
<meta name="citation_reference" content="citation_title=Reordering contigs of draft genomes using the Mauve aligner;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2071-3; ">
<meta name="citation_reference" content="citation_title=Ab initio gene finding in Drosophila genomic DNA;  citation_journal_title=Genome Res;  citation_year=2000;  citation_volume=10;  citation_pages=516-22; ">
<meta name="citation_reference" content="citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=1979-90; ">
<meta name="citation_reference" content="citation_title=AUGUSTUS: ab initio prediction of alternative transcripts;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W435-9; ">
<meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; ">
<meta name="citation_reference" content="citation_title=PEDANT covers all complete RefSeq genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=D408-11; ">
<meta name="citation_reference" content="citation_title=BlastKOALA and GhostKOALA: kEGG tools for functional characterization of genome and metagenome sequences;  citation_journal_title=J. Mol. Biol;  citation_year=2016;  citation_volume=428;  citation_pages=726-31; ">
<meta name="citation_reference" content="citation_title=Wine fermentation microbiome: a landscape from different Portuguese wine appellations;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=905">
<meta name="citation_reference" content="citation_title=Association between grape yeast communities and the vineyard ecosystems;  citation_journal_title=PLoS One;  citation_year=2017;  citation_volume=12;  citation_pages=e0169883">
<meta name="citation_reference" content="citation_title=Hanseniaspora meyeri sp. nov., Hanseniaspora clermontiae sp. nov., Hanseniaspora lachancei sp. nov. and Hanseniaspora opuntiae sp. nov., novel apiculate yeast species;  citation_journal_title=Int. J. Syst. Evol. Microbiol;  citation_year=2003;  citation_volume=53;  citation_pages=1671-80; ">
<meta name="citation_reference" content="citation_title=Hanseniaspora uvarum from winemaking environments show spatial and temporal genetic clustering;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=1569">
<meta name="citation_reference" content="citation_title=Molecular characterisation of Hanseniaspora species;  citation_journal_title=Antonie Van Leeuwenhoek;  citation_year=2001;  citation_volume=80;  citation_pages=85-92; ">
<meta name="citation_reference" content="citation_title=Molecular identification and genetic diversity among Photorhabdus and Xenorhabdus isolates;  citation_journal_title=3 Biotech;  citation_year=2017;  citation_volume=7;  citation_pages=6">
<meta name="citation_reference" content="citation_title=The mitochondrial genome of the wine yeast Hanseniaspora uvarum: a unique genome organization among yeast/fungal counterparts;  citation_journal_title=FEMS Yeast Res;  citation_year=2006;  citation_volume=6;  citation_pages=77-90; ">
<meta name="citation_reference" content="citation_title=Comparative genomics of protoploid Saccharomycetaceae;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1696-709; ">
<meta name="citation_reference" content="citation_title=Fermentative lifestyle in yeasts belonging to the Saccharomyces complex;  citation_journal_title=FEBS J;  citation_year=2007;  citation_volume=274;  citation_pages=976-89; ">
<meta name="citation_reference" content="citation_title=Occurrence of FFZ genes in yeasts and correlation with fructophilic behaviour;  citation_journal_title=Microbiology;  citation_year=2015;  citation_volume=161;  citation_pages=2008-18; ">
<meta name="citation_reference" content="citation_title=Spermidine but not spermine is essential for hypusine biosynthesis and growth in Saccharomyces cerevisiae: spermine is converted to spermidine in vivo by the FMS1-amine oxidase;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2003;  citation_volume=100;  citation_pages=13869-74; ">
<meta name="citation_reference" content="citation_title=Biogenic amines in wine: understanding the headache;  citation_journal_title=S. Afr. J. Enol. Vitic;  citation_year=2008;  citation_volume=29;  citation_pages=109-27; ">
<meta name="citation_reference" content="Barnett J. A., Payne R. W., Yarrow D.  1990 , Yeasts: characteristics and identification. ">
<meta name="citation_reference" content="citation_title=Early thiamin assimilation by yeasts under enological conditions: impact on alcoholic fermentation kinetics;  citation_journal_title=J. Ferment. Bioeng;  citation_year=1996;  citation_volume=82;  citation_pages=145-50; ">
<meta name="citation_reference" content="citation_title=Deciphering regulatory variation of THI genes in alcoholic fermentation indicate an impact of Thi3p on PDC1 expression;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1085">
<meta name="citation_reference" content="citation_title=Characterisation of glucose transport in Saccharomyces cerevisiae with plasma membrane vesicles (countertransport) and intact cells (initial uptake) with single Hxt1, Hxt2, Hxt3, Hxt4, Hxt6, Hxt7 or Gal2 transporters;  citation_journal_title=FEMS Yeast Res;  citation_year=2002;  citation_volume=2;  citation_pages=539-50; ">
<meta name="citation_reference" content="citation_title=Controlled mixed culture fermentation: a new perspective on the use of non-Saccharomyces yeasts in winemaking;  citation_journal_title=FEMS Yeast Res;  citation_year=2010;  citation_volume=10;  citation_pages=123-33; ">
<meta name="citation_reference" content="citation_title=Selective sugar consumption by apiculate yeasts;  citation_journal_title=Lett. Appl. Microbiol;  citation_year=1999;  citation_volume=28;  citation_pages=203-6; ">
<meta name="citation_reference" content="citation_title=The high-capacity specific fructose facilitator ZrFfz1 is essential for the fructophilic behavior of Zygosaccharomyces rouxii CBS 732T;  citation_journal_title=Eukaryot. Cell;  citation_year=2014;  citation_volume=13;  citation_pages=1371-9; ">
<meta name="citation_reference" content="citation_title=ZrFsy1, a high-affinity fructose/H+ symporter from fructophilic yeast Zygosaccharomyces rouxii;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e68165">
<meta name="citation_reference" content="citation_title=An optimized procedure for the enological selection of non-Saccharomyces starter cultures;  citation_journal_title=Antonie Van Leeuwenhoek;  citation_year=2011;  citation_volume=99;  citation_pages=189-200; ">
<meta name="citation_reference" content="citation_title=Stepwise functional evolution in a fungal sugar transporter family;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=352-66; ">
<meta name="citation_reference" content="citation_title=Allelic variants of hexose transporter Hxt3p and hexokinases Hxk1p/Hxk2p in strains of Saccharomyces cerevisiae and interspecies hybrids;  citation_journal_title=Yeast;  citation_year=2015;  citation_volume=32;  citation_pages=657-69; ">
<meta name="citation_reference" content="citation_title=Correlation between glucose/fructose discrepancy and hexokinase kinetic properties in different Saccharomyces cerevisiae wine yeast strains;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2008;  citation_volume=77;  citation_pages=1083-91; ">
<meta name="citation_reference" content="Sutterlin K. A.  2010 , Fructophilic yeasts to cure stuck fermentations in alcoholic beverages. PhD thesis, Viticulture and Oenology, University of Stellenbosch. ">
<meta name="citation_reference" content="citation_title=Rational selection of non-Saccharomyces wine yeasts for mixed starters based on ester formation and enological traits;  citation_journal_title=Food Microbiol;  citation_year=2008;  citation_volume=25;  citation_pages=778-85; ">
<meta name="citation_reference" content="citation_title=Yeast killer toxin K28: biology and unique strategy of host cell intoxication and killing;  citation_journal_title=Toxins;  citation_year=2017;  citation_volume=9;  citation_pages=73">
<meta name="citation_reference" content="citation_title=Eukaryote-to-eukaryote gene transfer events revealed by the genome sequence of the wine yeast Saccharomyces cerevisiae EC1118;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2009;  citation_volume=106;  citation_pages=16333-8; ">
<meta name="citation_reference" content="citation_title=Screening of beta-glucosidase and beta-xylosidase activities in four non-Saccharomyces yeast isolates;  citation_journal_title=J. Food Sci;  citation_year=2015;  citation_volume=80;  citation_pages=C1696-704; ">
<meta name="citation_reference" content="citation_title=Past and future of non-saccharomyces yeasts: from spoilage microorganisms to biotechnological tools for improving wine aroma complexity;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=411">
<meta name="citation_reference" content="citation_title=Melatonin is formed during winemaking at safe levels of biogenic amines;  citation_journal_title=Food Chem. Toxicol;  citation_year=2013;  citation_volume=57;  citation_pages=140-6; ">
<meta name="citation_reference" content="citation_title=Production of biogenic amines by wine microorganisms;  citation_journal_title=Bull. O.I.V;  citation_year=2005;  citation_volume=78;  citation_pages=595-609; ">
<meta name="citation_reference" content="citation_title=Genomic expression programs in the response of yeast cells to environmental changes;  citation_journal_title=Mol. Biol. Cell;  citation_year=2000;  citation_volume=11;  citation_pages=4241-57; ">
<meta name="citation_reference" content="citation_title=The genome sequence of the highly acetic acid-tolerant Zygosaccharomyces bailii-derived interspecies hybrid strain ISA1307, isolated from a sparkling wine plant;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=299-313; ">
<meta name="citation_reference" content="citation_title=Comparative genomics of the environmental stress response in ascomycete fungi;  citation_journal_title=Yeast;  citation_year=2007;  citation_volume=24;  citation_pages=961-76; ">
<meta name="citation_reference" content="citation_title=Global transcriptional responses of fission yeast to environmental stress;  citation_journal_title=. Mol. Biol. Cell;  citation_year=2003;  citation_volume=14;  citation_pages=214-29; ">
<meta name="citation_reference" content="citation_title=Differences in environmental stress response among yeasts is consistent with species-specific lifestyles;  citation_journal_title=Mol. Biol. Cell;  citation_year=2016;  citation_volume=27;  citation_pages=1694-705; ">
<meta name="citation_reference" content="citation_title=Genome-wide identification of Saccharomyces cerevisiae genes required for maximal tolerance to ethanol;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2009;  citation_volume=75;  citation_pages=5761-72; ">
<meta name="citation_reference" content="citation_title=Comprehensive phenotypic analysis for identification of genes affecting growth under ethanol stress in Saccharomyces cerevisiae;  citation_journal_title=FEMS Yeast Res;  citation_year=2009;  citation_volume=9;  citation_pages=32-44; ">
<meta name="citation_reference" content="citation_title=Identification of genes required for maximal tolerance to high-glucose concentrations, as those present in industrial alcoholic fermentation media, through a chemogenomics approach;  citation_journal_title=OMICS;  citation_year=2010;  citation_volume=14;  citation_pages=201-10; ">
<meta name="citation_reference" content="citation_title=New genes involved in osmotic stress tolerance in Saccharomyces cerevisiae;  citation_journal_title=Front. Microbiol;  citation_year=2016;  citation_volume=7;  citation_pages=1545">
<meta name="citation_reference" content="citation_title=Genome-wide identification of the Fermentome; genes required for successful and timely completion of wine-like fermentation by Saccharomyces cerevisiae;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=552">
<meta name="citation_reference" content="citation_title=Genome-wide study of the adaptation of Saccharomyces cerevisiae to the early stages of wine fermentation;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e74086">
<meta name="citation_reference" content="citation_title=Biometric study of acetoin production in Hanseniaspora guilliermondii and Kloeckera apiculata;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1993;  citation_volume=59;  citation_pages=1838-41; ">
<meta name="citation_reference" content="citation_title=Enhancement of bio-ethanol production from date molasses by non-conventional yeasts;  citation_journal_title=Res. J. Microbiol;  citation_year=2015;  citation_volume=10;  citation_pages=114-25; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/1/67/28033456/dsy039.pdf">
<meta name="description" content="Abstract.  Hanseanispora species, including H. guilliermondii, are long known to be abundant in wine grape-musts and to play a critical role in vinification by ">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/1/67/5193709">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article-abstract/26/1/67/5193709">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/1/67/5193709");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2779951,2796125&authnips=128.235.85.32&doi=10.1093/dnares/dsy039';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f67%2f5193709" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f67%2f5193709" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=114871"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/1/2/m_dnares_26_1cover.png?Expires=1576706068&amp;Signature=Pjqj~e-5DL17K2keaeJre5FczKGcv1ABV3sU-ROyeNa3kyne-rVRqNEqBZLUREfL3XlyZCTlCFdEvT-JEFCKMBKQYhHPILqmHM7taK2gUH1zdY7TBDrnN3svxupQsujyk405-GEkapNKHb9EBp~IFHwu~jF~ojNTjB-RzZElsXsaG9RJoSeeeGFSvvNyoYUvrF8GA32KFnh4e4-s2n3TA2Yx~9HmhgJ2j3oSoU97aI6iLsRD87BE2cKROqTbKqcsi8qnNVmqUA7rycZllvMwV5re4kJrxb-zaLWDPPTA5CYmRS4VYfPZLX92ZfFYpv384o8TyzL1PXOWP2YQtqIW4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/1">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="131979834">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979834">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979836">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979836">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979841">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979841">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979852">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979852">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="131979885">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979885">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="131979891">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#131979891">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/1/55/5194314">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/1/85/5233427">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Genome sequence of the non-conventional wine yeast <em>Hanseniaspora guilliermondii</em> UTAD222 unveils relevant traits of this species and of the <em>Hanseniaspora</em> genus in the context of wine fermentation
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Isabel Seixas</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal</div>
<div class="aff">BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Isabel+Seixas">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seixas%20I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Seixas%20Isabel%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Isabel Seixas</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Catarina Barbosa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal</div>
<div class="aff">BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Catarina+Barbosa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Barbosa%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Barbosa%20Catarina%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Catarina Barbosa</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arlete Mendes-Faia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal</div>
<div class="aff">BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arlete+Mendes-Faia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mendes-Faia%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mendes-Faia%20Arlete%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Arlete Mendes-Faia</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ulrich Güldener</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bioinformatics, Wissenschaftszentrum Weihenstephan, Technische Universität München, Maximus von-Imhof-Forum 3, Freising, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ulrich+G%c3%bcldener">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=G%C3%BCldener%20U">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22G%C3%BCldener%20Ulrich%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Ulrich Güldener</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rogério Tenreiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rog%c3%a9rio+Tenreiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tenreiro%20R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Tenreiro%20Rog%C3%A9rio%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Rogério Tenreiro</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ana Mendes-Ferreira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">WM&amp;B—Laboratory of Wine Microbiology &amp; Biotechnology, Department of Biology and Environment, University of Trás-os-Montes and Alto Douro, Vila Real, Portugal</div>
<div class="aff">BioISI-Biosystems and Integrative Sciences Institute, Faculdade de Ciências, Universidade de Lisboa Campo Grande, Lisbon, Portugal</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +351218419181. Email: <a href="mailto:nuno.mira@tecnico.ulisboa.pt" target="_blank">nuno.mira@tecnico.ulisboa.pt</a> (N.P.M.); Tel. +351 259 350 550. Email: <a href="mailto:anamf@utad.pt" target="_blank">anamf@utad.pt</a> (A.M.-F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ana+Mendes-Ferreira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mendes-Ferreira%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mendes-Ferreira%20Ana%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Ana Mendes-Ferreira</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nuno P Mira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bioengineering, iBB - Institute for Bioengineering and Biosciences, Instituto Superior Técnico, Universidade de Lisboa, Avenida Rovisco Pais, Lisbon, Portugal</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +351218419181. Email: <a href="mailto:nuno.mira@tecnico.ulisboa.pt" target="_blank">nuno.mira@tecnico.ulisboa.pt</a> (N.P.M.); Tel. +351 259 350 550. Email: <a href="mailto:anamf@utad.pt" target="_blank">anamf@utad.pt</a> (A.M.-F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nuno+P+Mira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mira%20N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mira%20Nuno%20P%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Nuno P Mira</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 67–83, <a href="https://doi.org/10.1093/dnares/dsy039">https://doi.org/10.1093/dnares/dsy039</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">20 November 2018</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">23 May 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">16 October 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">20 November 2018</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5193709" href="/dnaresearch/article-pdf/26/1/67/28033456/dsy039.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Isabel Seixas, Catarina Barbosa, Arlete Mendes-Faia, Ulrich Güldener, Rogério Tenreiro, Ana Mendes-Ferreira, Nuno P Mira,  Genome sequence of the non-conventional wine yeast <em>Hanseniaspora guilliermondii</em> UTAD222 unveils relevant traits of this species and of the <em>Hanseniaspora</em> genus in the context of wine fermentation, <em>DNA Research</em>, Volume 26, Issue 1, February 2019, Pages 67–83, <a href="https://doi.org/10.1093/dnares/dsy039">https://doi.org/10.1093/dnares/dsy039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5193709&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20sequence%20of%20the%20non-conventional%20wine%20yeast%20%26lt%3Bem%26gt%3BHanseniaspora%20guilliermondii%26lt%3B%2Fem%26gt%3B%20UTAD222%20unveils%20relevant%20traits%20of%20this%20species%20and%20of%20the%20%26lt%3Bem%26gt%3BHanseniaspora%26lt%3B%2Fem%26gt%3B%20genus%20in%20the%20context%20of%20wine%20fermentation&amp;publicationDate=2018-11-20&amp;volumeNum=26&amp;issueNum=1&amp;author=Seixas%2C%20Isabel%3B%20Barbosa%2C%20Catarina&amp;startPage=67&amp;endPage=83&amp;contentId=10.1093%2Fdnares%2Fdsy039&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f67%2f5193709" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="131979834" id="131979834" class="abstract-title">Abstract</h2>
<section class="abstract"><p><em>Hanseanispora</em> species, including <em>H. guilliermondii</em>, are long known to be abundant in wine grape-musts and to play a critical role in vinification by modulating, among other aspects, the wine sensory profile. Despite this, the genetics and physiology of <em>Hanseniaspora</em> species remains poorly understood. The first genomic sequence of a <em>H. guilliermondii</em> strain (UTAD222) and the discussion of its potential significance are presented in this work. Metabolic reconstruction revealed that <em>H. guilliermondii</em> is not equipped with a functional gluconeogenesis or glyoxylate cycle, nor does it harbours key enzymes for glycerol or galactose catabolism or for biosynthesis of biotin and thiamine. Also, no fructose-specific transporter could also be predicted from the analysis of <em>H. guilliermondii</em> genome leaving open the mechanisms underlying the fructophilic character of this yeast. Comparative analysis involving <em>H. guilliermondii</em>, <em>H. uvarum</em>, <em>H. opuntiae</em> and <em>S. cerevisiae</em> revealed 14 <em>H. guilliermondii-</em>specific genes (including five viral proteins and one β-glucosidase). Furthermore, 870 proteins were only found within the <em>Hanseniaspora</em> proteomes including several β-glucosidases and decarboxylases required for catabolism of biogenic amines. The release of <em>H. guilliermondii</em> genomic sequence and the comparative genomics/proteomics analyses performed, is expected to accelerate research focused on <em>Hanseniaspora</em> species and to broaden their application in the wine industry and in other bio-industries in which they could be explored as cell factories.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"Hanseniaspora guilliermondii"'><em>Hanseniaspora guilliermondii</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"non-Saccharomyces wine yeasts"'>non-Saccharomyces wine yeasts</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genome sequencing and annotation"'>genome sequencing and annotation</a>
</div>                    <h2 scrollto-destination="131979836" id="131979836" class="section-title" data-legacy-id="SEC1">1. Introduction</h2>
<p>Wine fermentation is a complex biochemical process in which yeasts are the pivotal players. Although <em>Saccharomyces cerevisiae</em> is the leading microorganism in conducting alcoholic fermentation of the grape must, it is being increasingly acknowledged the critical role of the remaining microbiota in shaping the properties of the produced wines giving rise to the concept of ‘microbial terroir’.<a reveal-id="dsy039-B1 dsy039-B2 dsy039-B3" data-open="dsy039-B1 dsy039-B2 dsy039-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> In particular, the activity of many other yeast species belonging to various non-<em>Saccharomyces</em> genera has been found to greatly influence the course of the fermentation and also the sensory characteristics of the final wine.<a reveal-id="dsy039-B4" data-open="dsy039-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy039-B5" data-open="dsy039-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Apiculate yeasts of the <em>Hanseniaspora</em> genus are among the non-<em>Saccharomyces</em> species more abundant in grape-musts, particularly in the early stages of the fermentation.<a reveal-id="dsy039-B3" data-open="dsy039-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy039-B6 dsy039-B7 dsy039-B8 dsy039-B9 dsy039-B10" data-open="dsy039-B6 dsy039-B7 dsy039-B8 dsy039-B9 dsy039-B10" class="link link-ref link-reveal xref-bibr"><sup>6–10</sup></a> Although <em>H. uvarum</em> is usually the more abundant species,<a reveal-id="dsy039-B6" data-open="dsy039-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy039-B9" data-open="dsy039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy039-B11" data-open="dsy039-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> several studies also report a high abundance of <em>H. guilliermondii</em>.<a reveal-id="dsy039-B8" data-open="dsy039-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy039-B9" data-open="dsy039-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy039-B12 dsy039-B13 dsy039-B14 dsy039-B15" data-open="dsy039-B12 dsy039-B13 dsy039-B14 dsy039-B15" class="link link-ref link-reveal xref-bibr"><sup>12–15</sup></a> As observed with other <em>Hanseniaspora</em> species, the presence of <em>H. guilliermondii</em> has a positive effect in the aroma profile of wines<a reveal-id="dsy039-B16 dsy039-B17 dsy039-B18 dsy039-B19" data-open="dsy039-B16 dsy039-B17 dsy039-B18 dsy039-B19" class="link link-ref link-reveal xref-bibr"><sup>16–19</sup></a> as well as of other fermented beverages.<a reveal-id="dsy039-B20" data-open="dsy039-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsy039-B21" data-open="dsy039-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> In general, the positive contribution of non-<em>Saccharomyces</em> species (and of <em>H. guilliermondii</em> in particular) to the aroma profile of the wine is believed to result from their ability to produce esters, higher alcohols or glycerol, but also through the synthesis of enzymes (such as β-glucosidases) that release flavor and aroma compounds (usually terpenols) present in the grapes as flavourless glycoconjugate precursors.<a reveal-id="dsy039-B4" data-open="dsy039-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy039-B5" data-open="dsy039-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><em>Hanseniaspora</em> species have also been used to improve the content in glycerol or mannoproteins<a reveal-id="dsy039-B22" data-open="dsy039-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsy039-B23" data-open="dsy039-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> or to reduce acidity or alcohol level.<a reveal-id="dsy039-B24 dsy039-B25 dsy039-B26" data-open="dsy039-B24 dsy039-B25 dsy039-B26" class="link link-ref link-reveal xref-bibr"><sup>24–26</sup></a></p>
<p>When co-inoculated with <em>S. cerevisae</em> in grape-must <em>H. guilliermondii</em> predominates in an initial stage, however, as the fermentation proceeds its abundance decreases prominently.<a reveal-id="dsy039-B8" data-open="dsy039-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy039-B27 dsy039-B28 dsy039-B29" data-open="dsy039-B27 dsy039-B28 dsy039-B29" class="link link-ref link-reveal xref-bibr"><sup>27–29</sup></a> The low tolerance to ethanol has been suggested to account for the reduced fitness of <em>H. guilliermondii</em> in the later stages of the fermentation (in analogy to what was proposed for other non-Saccharomyces yeasts); however, the relatively high tolerance observed in some strains<a reveal-id="dsy039-B30" data-open="dsy039-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> along with the demonstration that death of <em>H. guilliermondii</em> cells occurred in the presence of <em>S. cerevisiae</em> regardless of the amount of ethanol present in the fermentation broth,<a reveal-id="dsy039-B29" data-open="dsy039-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> puts in question this simple model by which <em>S. cerevisae</em> and <em>H. guilliermondii</em> interact.<a reveal-id="dsy039-B31" data-open="dsy039-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Indeed, it was recently demonstrated that <em>H. guilliermondii</em> death in the presence of <em>S. cerevisiae</em> results from this latter species producing antimicrobial peptides that induce loss of cell viability in the <em>H. guilliermondii cells</em> upon cell–cell contacts.<a reveal-id="dsy039-B32" data-open="dsy039-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p>
<p>Although it is clear that the presence of <em>H. guilliermondii</em> affects the performance of alcoholic fermentation, the exact molecular mechanisms underlying its interaction with <em>S. cerevisiae</em> cells are still elusive questions. In a first attempt to address this issue, Barbosa et al. have studied the effect exerted by <em>H. guilliermondii</em> on the genomic expression of <em>S. cerevisiae</em> during a mixed-culture fermentation of a natural grape-must.<a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The expression of about 300 <em>S. cerevisiae</em> genes was altered by the presence of <em>H. guilliermondii</em>, in comparison with the corresponding transcript levels registered in single-cultures.<a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><em>Saccharomyces cerevisiae</em> genes involved in the biosynthesis of vitamins were enriched among those found to be more up-regulated in the mixed <em>S. cerevisiae</em>–<em>H. guilliermondii</em> fermentation, while genes related with the uptake and biosynthesis of amino acids were enriched among those more expressed in the single-culture.<a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Notably, the differences in the aroma profiles of the wines obtained in mixed-culture fermentations, appeared to be tightly correlated with the changes observed in the transcript profile of <em>S. cerevisae</em> genes related with formation of the aroma compounds observed in mixed-culture fermentations and attributable to the presence of <em>H. guilliermondii</em>.<a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p>
<p>The first annotated genomic sequence of a <em>H. guilliermondii</em> strain (the UTAD222 strain, also focused in this work) was recently released<a reveal-id="dsy039-B33" data-open="dsy039-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> accompanying the releases of genomic sequences for <em>H. uvarum</em> (strains AWRI3580 and DSM2768)<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and <em>H. opuntiae</em> (strain AWRI3578).<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> However, in those short studies, the information coming from the release of these genomic sequences was not explored, this being an essential aspect to better understand the biology and physiology of the <em>Hanseniaspora</em> species, specially in the context of wine fermentation. In this work, a thorough analysis of the <em>H. guilliermondii</em> genomic sequence was performed, involving not only extensive functional analysis of the <em>H. guilliermondii</em> genome but also a comparative analysis with <em>H. uvarum</em>, <em>H. opuntiae</em> and also <em>S. cerevisiae</em>. Exploring those comparative genomic results a chromosomal map for the <em>H. guilliermondii</em> UTAD222 strain is proposed, this being the first time that this information is given for a strain of this species.</p>                    <h2 scrollto-destination="131979841" id="131979841" class="section-title" data-legacy-id="SEC2">2. Materials and methods</h2>
                    <h3 scrollto-destination="131979842" id="131979842" class="section-title" data-legacy-id="SEC2.1">2.1. Strains</h3>
<p>In this work, 33 indigenous strains from the UTAD culture collection having a presumptive identification of belonging to the <em>Hanseniaspora</em> genus were used, these being listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s1.xlsx?Expires=1576481646&amp;Signature=fBfpEbHBYIBgOniVH-ikuA3iWY5~wkOWQqdHsXGE~4dQnLj0V7IISMjh0oBUDoaAEGJgthJfce5NrWyC8Fn~BTwvQlLt6~tDpPzGKvPNH2ZcrnCKHfUSo511PndE4b6eR5AercCdSwLQAeY2cK9jS86olop3Un-xA1icFwCmA9YA7x68z430LcnjCkyfoU8bYqBM3Sodj4i7XvTWMEKd-CqTSl0h3hBVj5UjxMvXIe3QcE8yCP56vOspCXhHhmnyJsLVpOrMnK2HJHF0l7aBwNNBWBEwlu-kSOMUCueuJqZ3-8syINuWKGeeMEvg33AZksu-I4Be7nMMC6mWj92z1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. The strains were recovered from 10 different wineries of the Douro Demarcated Region in Portugal and were isolated from grapes, grape-must or wine. Pre-selection of these autochthons was based on the ability to grow on <span class="small-caps">l</span>-lysine agar selective medium followed by microscopic examination of the size, morphology and reproduction mode of the cells. Although two non-<em>Saccharomyces</em> genera associated with winemaking environment are characterized by bipolar budding, <em>Hanseniaspora</em> and <em>Saccharomycodes</em>, the first can be easily distinguished from the second by the smaller size of the cells. Seven reference strains were also used: <em>H. guilliermondii</em> CECT11027<sup>T</sup>, <em>H. occidentalis</em> CECT11341<sup>T</sup>, <em>H. osmophila</em> CECT<sup>T</sup> 11206, <em>H. uvarum</em> CECT1444<sup>T</sup> and <em>H. vineae</em> CECT1471T, from the Spanish Type Culture Collection (CECT); and <em>H. guilliermondii</em> CBS465<sup>T</sup> and <em>H. opuntiae</em> CBS8733<sup>T</sup> obtained from CBS-KNAW fungal collection (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s1.xlsx?Expires=1576481646&amp;Signature=fBfpEbHBYIBgOniVH-ikuA3iWY5~wkOWQqdHsXGE~4dQnLj0V7IISMjh0oBUDoaAEGJgthJfce5NrWyC8Fn~BTwvQlLt6~tDpPzGKvPNH2ZcrnCKHfUSo511PndE4b6eR5AercCdSwLQAeY2cK9jS86olop3Un-xA1icFwCmA9YA7x68z430LcnjCkyfoU8bYqBM3Sodj4i7XvTWMEKd-CqTSl0h3hBVj5UjxMvXIe3QcE8yCP56vOspCXhHhmnyJsLVpOrMnK2HJHF0l7aBwNNBWBEwlu-kSOMUCueuJqZ3-8syINuWKGeeMEvg33AZksu-I4Be7nMMC6mWj92z1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p>                    <h3 scrollto-destination="131979844" id="131979844" class="section-title" data-legacy-id="SEC2.2">2.2. Genotypic characterization of a cohort of indigenous <em>Hanseniaspora</em> strains</h3>
<p>Thirty-two autochthonous <em>Hanseniaspora</em> strains along with the above-mentioned reference strains were subjected to (GTG)<sub>5-</sub>PCR fingerprinting for species identification. To extract the DNA from the isolates, two loopfulls of freshly grown cultures were resuspended in 500 µl of lysis buffer (50 mM Tris–HCl, 250 mM. NaCl, 50 mM EDTA, 0.3% SDS, pH 8.0). After adding one volume of phenol (pH 8.0), cells were disrupted by bead-beating for 30 s at speed 4.0 using a Fastprep cell disrupter (Bio101, Inc), followed by standard phenol: chloroform extraction. Total DNA was precipitated from the aqueous phase by the addition of 2 vol of ice-cold ethanol and of 1/10 volume of 3 M Na-acetate for 60 min at −70°C. The precipitated DNA was recovered by centrifugation at 15,000 <em>g</em> for 10 min at 4°C; the pellet was washed with 70% ethanol and resuspended in water. All PCR reactions were performed in 25 µl reaction volumes containing ∼10 ng of genomic DNA, 1× PCR buffer, 0.4 mM of each of the four dNTPs, 1 µM primer (GTG)<sub>5</sub> 5′-GTGGTGGTGGTGGTG-3′ and 1 U of Taq DNA polymerase (Invitrogen). The amplification conditions used were: initial template denaturation at 94°C for 5 min, 40 cycles of denaturation for 1 min at 94°C, annealing at 31°C for 1 min and extension at 72°C for 2 min. A final elongation at 72°C for 7 min was performed in the end. PCR products were separated on a 1.2% TBE agarose gel and were visualized after staining with ethidium bromide (0.5 μg/ml) and the DNA banding patterns analysed using the BioNumerics software (Applied Maths). Similarities among isolates were estimated using the Pearson coefficient and clustering was based on the UPGMA method. The identification of the autochthonous isolates was achieved by comparing their genotype profiles with those of the reference strains. To further support the results obtained, representatives of each cluster identified in the PCR fingerprinting were selected for 26S rDNA D1/D2 sequencing using primers NL-1 (5′-GCA TAT CAA TAA GCG GAG GAA AAG-3′) and NL-4 (5′-GGT CCG TGT TTC AAG ACG G-3′). The obtained 26S rDNA D1/D2 sequences were deposited at NCBI under accession numbers (MG832576 to MG832583, MG877743 and MG87744). The sequences obtained were compared with other sequences reported in the GenBank using the BLAST algorithm and a phylogenetic tree was obtained using a neighbor-joining algorithm. Species identification was ascribed to an isolate if its 26S rDNA D1/D2 sequence differed by no more than 2–3 base substitutions (i.e.  ≥ 99.0% sequence identity) to that of a taxonomically accepted species type strain. Differentiation of <em>H. opuntiae</em> and <em>H. guilliermondii</em> was achieved by 5.8S-ITS-RFLP analysis, in which ITS1/ITS4 amplification products were digested with restriction enzyme DraI, and separated on 2% agarose gel on 1× TBE buffer as previously described.<a reveal-id="dsy039-B36" data-open="dsy039-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a></p>                    <h3 scrollto-destination="131979846" id="131979846" class="section-title" data-legacy-id="SEC2.3">2.3. Phenotypic screening of the cohort of indigenous <em>Hanseniaspora</em> strains</h3>
<p>The autochthonous <em>Hanseniaspora</em> strains were phenotypically profiled for relevant oenological traits, including their ability to excrete β-glucosidases, proteases and pectinases, potential to produce H<sub>2</sub>S and resistance to ethanol, sulphur dioxide (SO2), cerulenine and 5,5,5-trifluoro-dl-leucine (TFL). All assays were performed using 3 μl of a cell suspension (at an OD<sub>600nm</sub> of 0.1) prepared from an overnight pre-culture in YPD growth medium (containing, per liter, 20 g of glucose, 10 g yeast extract and 20 g bactopeptone). β-glycosidase activity was tested as previously described.<a reveal-id="dsy039-B37" data-open="dsy039-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Briefly, yeast cells were grown in a medium containing 0.5% of arbutine as the sole carbon source, 0.67% yeast nitrogen base and 2% agar (adjusted at pH 5.0 prior sterilization) and supplemented with 2 ml (per 100 ml of medium) ferric ammonium citrate solution (1% w/v). The plates were incubated at 30°C and β-glycosidase activity was observed after 2, 4, 6 and 8 h by the appearance of a dark brown color in the colonies. A non-inoculated plate of arbutin agar was used as a negative control. Proteolytic activity was determined assessing growth on skim milk agar plates, after incubation for 3–8 days at 30°C<a reveal-id="dsy039-B38" data-open="dsy039-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> while the potential ability to produce H2S was evaluated by growing yeast cells in Biggy agar.<a reveal-id="dsy039-B37" data-open="dsy039-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> The strains were classified in different categories according with the appearance of a brown-black coloration of the colonies, which is directly correlated with sulfite-reductase activity. Resistance to SO<sub>2</sub> and ethanol were evaluated on YPD agar medium (buffered at pH 3.5), supplemented with increasing concentrations of SO2 (0.25, 0.5, 0.75, 1.0 and 1.5 mM) or ethanol (5.0, 6.0, 7.5, 9.0 and 10% v/v). Susceptibility phenotypes were registered after incubation at 30°C during 3 days, after visual inspection of colonies growth and comparison with growth observed in non-supplemented YPD agar. Resistance to TFL and cerulenin was assessed by determining growth on agar plates containing 0.67% of yeast nitrogen base (Difco), 2% of agar supplemented with glucose (2% w/v) in combination with TFL (1 mM) or cerulenin (25 or 100 µM). The plates were maintained at 30°C for 5 days and colonies that developed on the plates containing the inhibitors were considered resistant to TFL or cerulenin.<a reveal-id="dsy039-B39" data-open="dsy039-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Based on all the phenotypic data gathered, cluster analysis was carried out using SAS JMP 11.0 (JMP, 2011, Cary, NC) and a phenogram of yeast strains was obtained with Euclidean distance as an association measure and complete linkage as the agglomeration method.</p>                    <h3 scrollto-destination="131979848" id="131979848" class="section-title" data-legacy-id="SEC2.4">2.4. Genome sequencing, assembly and comparison of <em>H. guilliermondii</em> UTAD222 genome with those available for other <em>Hanseniaspora</em> species</h3>
<p>The genome of the UTAD222 strain was sequenced at StabVida (Portugal) using a whole-genome shotgun approach that explored paired-end Illumina MiSeq platform, as briefly described before.<a reveal-id="dsy039-B33" data-open="dsy039-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> In specific, the genomic DNA of <em>H. guilliermondii</em> UTAD222 was extracted and used to construct a DNA library having inserts in the range of 250–350 bp. Library amplification was performed on the cluster generation station of the GAIIx and using the Illumina cluster generation kit. To obtain the paired-end reads, primers were designed to hybridized with Illumina specific adaptors resulting in reading of each end as a separate run. The sequencing reaction was run for 100 cycles (tagging, imaging and cleavage of one terminal base at a time), and four images of each tile on the chip were taken in different wavelengths for exciting each base-specific fluorophore. The obtained reads were afterwards assembled into contigs using the <em>de novo</em> assembler available on the CLC Genomics Workbench (version 9; Quiagen) set at default parameters. The obtained 208 contigs were organized into seven chromosomes, based on the identification of seven chromosomal bands in PFGE analysis of <em>H. guilliermondii</em> UTAD222 genomic DNA and using as a reference the proposed organization for the genome of <em>H. uvarum</em> DSM2768.<a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> To perform this organization, an iterative process was used: at first, whole-genome alignments of the UTAD222 contigs and the contigs available described for <em>H. uvarum</em> AWRI3580 and <em>H. opuntiae</em> AWRI3578<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> were performed using Mauve Contig Mover.<a reveal-id="dsy039-B40" data-open="dsy039-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> These whole-genome alignments were performed using the default parameters (match seed weight 15, min LBC weight 200, scoring matrix HOXD). Whenever possible the whole-genome alignments were also performed using the contigs available for the <em>H. uvarum</em> DSM2768 strain.<a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> To reinforce the organization proposed by the whole-genome alignments, it was investigated colinearity between the genomes of <em>H. guilliermondii</em>, <em>H. uvarum AWRI3580</em> and <em>H. opuntiae</em> AWRI3578. For this, pairwise alignments between the proteins predicted for the three species were performed and the best hits selected. These best hit pairs were then mapped in the contig organization proposed by the whole-genome alignments giving rise to the figures shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>. As shown, in the vast majority of the cases it was possible to confirm colinearity with the best gene orthologues between the three species being located contiguously. To get further confirmation that the proposed organization for UTAD222 contigs was correct a number of junctions (chosen in all chromosomes) were experimentally validated by PCR. While some these PCR reactions were randomly chosen, others aimed at solving divergences identified in the whole-genome alignments. This is, for example, the case of contigs 4 and 5 which were mapped to chromosomes G and F, respectively, in <em>H. guilliermondii</em>, but that in <em>H. uvarum</em> and <em>H. opuntiae</em> are located in other genomic regions (as highlighted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). All contig junctions experimentally verified are highlighted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>.</p>                    <h3 scrollto-destination="131979850" id="131979850" class="section-title" data-legacy-id="SEC2.5">2.5. Annotation of <em>H. guilliermondii</em> UTAD222 genome and subsequent comparison with the proteome described for other <em>Hanseniaspora</em> species and for <em>S. cerevisiae</em>
</h3>
<p>Primary structural annotation of the <em>H. guilliermondii</em> UTAD222 genome was achieved by applying three <em>de novo</em> prediction programs: (i) Fgenesh with different matrices (trained on <em>Aspergillus nidulans</em>, <em>Neurospora crassa</em> and a mixed matrix based on different species);<a reveal-id="dsy039-B41" data-open="dsy039-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> (ii) GeneMark-ES<a reveal-id="dsy039-B42" data-open="dsy039-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and (iii) Augustus.<a reveal-id="dsy039-B43" data-open="dsy039-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The different gene structures and evidences were displayed in GBrowse allowing manual validation and, whenever needed, correction of the predicted coding sequences. In specific, gene models differently predicted by the algorithms were manually curated based on the structure obtained for homologues found in related annotated genomes including <em>H. uvarum</em>, <em>H. opuntiae</em> and <em>S. cerevisiae</em>. The final call set comprises 4,070 protein coding genes. tRNA-encoding genes were predicted using tRNAscan-SE.<a reveal-id="dsy039-B44" data-open="dsy039-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The protein coding genes were analysed and functionally annotated using the Pedant system.<a reveal-id="dsy039-B45" data-open="dsy039-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> The genome and annotation were submitted to the European Nucleotide Archive, ENA at <a class="link link-uri" href="http://www.ebi.ac.uk/ena/data/view/" target="_blank">http://www.ebi.ac.uk/ena/data/view/</a>FQNF01000001–FQNF01000208. Metabolic reconstruction of the <em>H. guilliermondii</em> UTAD222 metabolic network was performed using KEGG Koala reconstruction tool,<a reveal-id="dsy039-B46" data-open="dsy039-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> choosing Fungi as the taxonomy group of the genome and allowing the KEEG database to be searched for both eukaryotic and prokaryotic sequences. The comparison of the proteomes of <em>H. guilliermondii</em> UTAD222 with the proteomes of <em>H. uvarum</em>, <em>H. opuntiae</em> or <em>S. cerevisae</em> EC1118 was based on BLASTP using the sets of proteins available for each one of these species. <em>Hanseniaspora guilliermondii</em> UTAD222 proteins were consider similar to those present in other <em>Hanseniaspora</em> spp (or in <em>S. cerevisiae</em> EC1118) when the associated alignment had an associated e-value below e-20 and a minimum of identity of 30%. Pairwise alignments between <em>H. uvarum</em>/<em>H. guilliermondii</em> (or <em>H. opuntiae</em>/<em>H. guilliermondii</em>) having an identity between 30 and 50% were considered similar, while those having an associated identity &gt;50% were considered highly similar. Under these conditions, we could identify 3,958 highly similar and 65 similar <em>H. guilliermondii</em>/<em>H. opuntiae</em> protein pairs; 3,790 highly similar and 169 similar <em>H. guilliermondii</em>/<em>H. uvarum</em> protein pairs; and 1,906 highly similar and 771 similar <em>H. guilliermondii</em>/<em>S. cerevisiae</em> protein pairs.</p>                    <h2 scrollto-destination="131979852" id="131979852" class="section-title" data-legacy-id="SEC3">3. Results and discussion</h2>
                    <h3 scrollto-destination="131979853" id="131979853" class="section-title" data-legacy-id="SEC3.1">3.1. Genotypic screening of a cohort of autochthonous <em>Hanseniaspora</em> strains</h3>
<p>Prior studies have addressed the genetic diversity of <em>Hanseniaspora</em> strains in French and Italian wine producing regions, for example,<a reveal-id="dsy039-B8" data-open="dsy039-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy039-B10" data-open="dsy039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy039-B15" data-open="dsy039-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> however, no equivalent information was available for Portuguese wine producing areas. Recent reports describing the microbiomes of Portuguese wine regions have unveiled a high frequency for members of the <em>Hanseniaspora</em> genus,<a reveal-id="dsy039-B47" data-open="dsy039-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsy039-B48" data-open="dsy039-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> but the data gathered did not allowed discrimination at the strain level. To get insights into this, we have examined the distribution of species in a cohort of 30 strains presumptively identified as belonging to the <em>Hanseaniaspora</em> genus, and that also included the UTAD222 strain focused in this work. These presumed <em>Hanseniaspora</em> strains were isolated from grapes, grape-must or wine obtained in different wineries across the Demarcated Douro region (details in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) and that also included the herein focused UTAD222 strain. To characterize the genetic diversity of the strains, PCR fingerprinting was used. As controls, five reference strains from the Spanish Culture Collection representative of <em>Hanseniaspora</em> species already described to be present in grape-musts were included in this screening: <em>H. guilliermondii</em> CECT11027<sup>T</sup>, <em>H. occidentalis</em> CECT11341<sup>T</sup>, <em>H. osmophila</em> CECT11206<sup>T</sup>, <em>H. uvarum</em> CECT1444<sup>T</sup> and <em>H. vineae</em> CECT1471<sup>T</sup>. The results obtained clearly showed that the strains under examination could be divided in six groups (at a cut-off value of 50% similarity, <a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). The method used had a good discriminating power at the species level since the five reference strains were clustered separately (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). To obtain further information concerning the identification of the strains, the conserved ITS region of a randomly selected set of strains representative of each one of the groups (UTAD53, UTAD222, UTAD617, UTAD618 e UTAD621) was obtained and compared with sequences attributed to <em>Hanseniaspora</em> strains deposited at NCBI generating the phylogenetic tree that is shown in <a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>. Strains UTAD53 and UTAD618 were identified as belonging to the <em>H. uvarum</em> species, while strain UTAD222 was identified as <em>H. guilliermondii</em> (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>), this being in agreement with the results of the PCR fingerprinting (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Strikingly, sequencing of the D1/D2 region of strain UTAD617 that clustered with the <em>H. guilliermondii</em> reference strain in the PCR fingerprinting, identified this strain as belonging to the <em>H. opuntiae</em> species (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). Previous reports showed a close genetic relatedness between <em>H. guilliermondii</em> and <em>H. opuntiae</em>,<a reveal-id="dsy039-B36" data-open="dsy039-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsy039-B49" data-open="dsy039-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> which could account for a difficulty in discriminating these two species with the PCR fingerprinting used. Thus, we have compared the profile of bands obtained upon digestion with DraI of the ITS region of strains UTAD616, UTAD617, UTAD620, UTAD621, UTAD222 since this method allows a clear distinction between <em>H. opuntiae</em> (three bands with expected sizes of 420, 300 and 30 bp) and <em>H. guilliermondii</em> (four bands with expected sizes of 420, 150, 130 and 30 bp).<a reveal-id="dsy039-B49" data-open="dsy039-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The reference strains <em>H. guilliermondii</em> CECT11027<sup>T</sup>, <em>H. opuntiae</em> CBS8733<sup>T</sup> and <em>H. guilliermondii</em> CBS465<sup>T</sup> were also used in this assay as controls (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). The pattern of bands obtained for strain UTAD222 is similar to the one of CBS465<sup>T</sup>, confirming the identification of the isolate as <em>H. guilliermondii</em> (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). UTAD616 and UTAD621 isolates, whose species identification could not be achieved in the PCR fingerprinting, were also identified as <em>H. guilliermondii</em>. UTAD617 and UTAD620 were identified as <em>H. opuntiae</em> (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>), as well as the reference strain CECT11027<sup>T</sup>, which is in accordance with the close similarity of these three strains observed by PCR fingerprinting (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Surprisingly, the reference strain CECT11027<sup>T</sup> was identified as <em>H. opuntiae</em>. The genetic analysis performed rendered clear the higher predominance of <em>H. uvarum</em> strains in our small cohort of strains, this predominance being in line with what is reported in Iberian wine producing regions.<a reveal-id="dsy039-B6" data-open="dsy039-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy039-B12" data-open="dsy039-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsy039-B48" data-open="dsy039-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> It was also clear the absence of a correlation between the genotype affiliation and the geographical origin of the <em>Hanseniaspora</em> isolates recovered, indicating that grape-musts from a particular grape variety or winery do not appear to be preferentially colonized by a specific strain (<a reveal-id="dsy039-F1" data-open="dsy039-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Consistently, the absence of a specific genetic signature in a given winery or in grape varieties was also reported in other studies focusing the genetic diversity of <em>H. uvarum</em> and <em>H. guilliermondii</em> isolates.<a reveal-id="dsy039-B10" data-open="dsy039-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy039-B50" data-open="dsy039-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a></p>                        <a id="131979855" scrollto-destination="131979855"></a>
<div id="dsy039-F1" data-id="dsy039-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f1.png?Expires=1576481646&amp;Signature=2H-dqcVaN0KtA8GpiyQIwWmZpU0AqmsQdobUHbbsJHNO0us1nUrFu4MZ8cEcx7x0kj3pMrLfW67kBZyY0duEHDR4BI91G6JO87cI5VvKvUl868vmMMW0SoJ1TnAAuME0V~Rl1DObId0~YJYF-JKgI8BDfMJ36KFC0di5uB5ZuECp6d8KTIaErLn3rBfildW3YpWktP4knJhcO3aWssZw4vKiYRXfx8bR6zP05Ul~8QJ0nzPkHnaSK4Zs6wZlvz0sfibFYZAu3iYE16K7V6d2jWPt1P41NhaydKjp9VOu5PoHh0hJAYaytGq2cgil8P99zDhFiCndrsZttn5BbFw9kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic characterization, based on PCR-fingerprinting, of the 33 UTAD Hanseniaspora strains recovered from the Douro demarcated region and of the selected reference strains H. guilliermondii CECT11027T, H. occidentalis CECT11341T, H. osmophila CECT 11206T, H. uvarum CECT1444T and H. vineae CECT1471T; (A) Dendrogram obtained by hierarchical analysis of PCR (GTG)5 patterns using Pearson’s correlation coefficient and UPGMA clustering method; (B) Neighbour-joining phylogenetic tree showing the relationships of selected Hanseniaspora strains (marked with an asterisk in A), inferred from the sequences of the D1/D2 domain of the LSU RNA gene. Bootstrap percentages &gt;50% from 1,000 bootstrap replicates are shown. The outgroup species was Saccharomyces cerevisiae. Bar, 1% sequence divergence; (C) Restriction patterns obtained upon restriction with DraI of the ITS region of Hanseniaspora isolates or references strain: M, molecular marker; (a) isolate UTAD616, (b) isolate UTAD617, (c) isolate UTAD620, (d) isolate UTAD621, (e) isolate UTAD222, (f) reference strain H. guilliermondii CECT11027, (g) type strain H. opuntiae CBS8733T and (h) type strain H. guilliermondii CBS465T. The squares designate the different wineries from where the isolates were retrieved." path-from-xml="dsy039f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979855/dsy039f1.tif" path-from-xml="dsy039f1.tif" target="_blank">Open in new tab</a><a section="131979855" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f1.png?Expires=1576481646&amp;Signature=M99ec-lBEA4kEs0hqxUnSw0DiXkSb2NArnt1nYVRQAhR72n2cAmYPA0SKfCB1kIQFYbbcKRJRra~LYu22Bm-NGL2qJ4wcdh5qCMf0GfJ~KDXcoCIcsPf97~JQpRr0~sMnpI0Be7Va1U5FiAOIFnSwvU~wtfGNdqr7coHBNmT-UZKLK28GoTivwRNdpB8zhuNdpBExWOOnuphLfW~qIVsJXtJqWHmNVxYkmn~tEe26h74cNZ-sgD-D4plD30q8gL9DijSa29ZsTj~Q7x4a6QhQiqqrPv8RD2u~5Jm2tWQCNKiJgL3aCz4Eqmbh0u3GjfxoI0oM2XxygguuwKyRvuGRg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979855&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Genetic characterization, based on PCR-fingerprinting, of the 33 UTAD <em>Hanseniaspora</em> strains recovered from the Douro demarcated region and of the selected reference strains <em>H. guilliermondii</em> CECT11027<sup>T</sup>, <em>H. occidentalis</em> CECT11341<sup>T</sup>, <em>H. osmophila</em> CECT 11206<sup>T</sup>, <em>H. uvarum</em> CECT1444<sup>T</sup> and <em>H. vineae</em> CECT1471<sup>T</sup>; (A) Dendrogram obtained by hierarchical analysis of PCR (GTG)<sub>5</sub> patterns using Pearson’s correlation coefficient and UPGMA clustering method; (B) Neighbour-joining phylogenetic tree showing the relationships of selected <em>Hanseniaspora</em> strains (marked with an asterisk in A), inferred from the sequences of the D1/D2 domain of the LSU RNA gene. Bootstrap percentages &gt;50% from 1,000 bootstrap replicates are shown. The outgroup species was <em>Saccharomyces cerevisiae</em>. Bar, 1% sequence divergence; (C) Restriction patterns obtained upon restriction with DraI of the ITS region of <em>Hanseniaspora</em> isolates or references strain: M, molecular marker; (a) isolate UTAD616, (b) isolate UTAD617, (c) isolate UTAD620, (d) isolate UTAD621, (e) isolate UTAD222, (f) reference strain <em>H. guilliermondii</em> CECT11027, (g) type strain <em>H. opuntiae</em> CBS8733<sup>T</sup> and (h) type strain <em>H. guilliermondii</em> CBS465<sup>T</sup>. The squares designate the different wineries from where the isolates were retrieved.</p></div>
</div>
<div content-id="dsy039-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f1.png?Expires=1576481646&amp;Signature=2H-dqcVaN0KtA8GpiyQIwWmZpU0AqmsQdobUHbbsJHNO0us1nUrFu4MZ8cEcx7x0kj3pMrLfW67kBZyY0duEHDR4BI91G6JO87cI5VvKvUl868vmMMW0SoJ1TnAAuME0V~Rl1DObId0~YJYF-JKgI8BDfMJ36KFC0di5uB5ZuECp6d8KTIaErLn3rBfildW3YpWktP4knJhcO3aWssZw4vKiYRXfx8bR6zP05Ul~8QJ0nzPkHnaSK4Zs6wZlvz0sfibFYZAu3iYE16K7V6d2jWPt1P41NhaydKjp9VOu5PoHh0hJAYaytGq2cgil8P99zDhFiCndrsZttn5BbFw9kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic characterization, based on PCR-fingerprinting, of the 33 UTAD Hanseniaspora strains recovered from the Douro demarcated region and of the selected reference strains H. guilliermondii CECT11027T, H. occidentalis CECT11341T, H. osmophila CECT 11206T, H. uvarum CECT1444T and H. vineae CECT1471T; (A) Dendrogram obtained by hierarchical analysis of PCR (GTG)5 patterns using Pearson’s correlation coefficient and UPGMA clustering method; (B) Neighbour-joining phylogenetic tree showing the relationships of selected Hanseniaspora strains (marked with an asterisk in A), inferred from the sequences of the D1/D2 domain of the LSU RNA gene. Bootstrap percentages &gt;50% from 1,000 bootstrap replicates are shown. The outgroup species was Saccharomyces cerevisiae. Bar, 1% sequence divergence; (C) Restriction patterns obtained upon restriction with DraI of the ITS region of Hanseniaspora isolates or references strain: M, molecular marker; (a) isolate UTAD616, (b) isolate UTAD617, (c) isolate UTAD620, (d) isolate UTAD621, (e) isolate UTAD222, (f) reference strain H. guilliermondii CECT11027, (g) type strain H. opuntiae CBS8733T and (h) type strain H. guilliermondii CBS465T. The squares designate the different wineries from where the isolates were retrieved." path-from-xml="dsy039f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979855/dsy039f1.tif" path-from-xml="dsy039f1.tif" target="_blank">Open in new tab</a><a section="131979855" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f1.png?Expires=1576481646&amp;Signature=M99ec-lBEA4kEs0hqxUnSw0DiXkSb2NArnt1nYVRQAhR72n2cAmYPA0SKfCB1kIQFYbbcKRJRra~LYu22Bm-NGL2qJ4wcdh5qCMf0GfJ~KDXcoCIcsPf97~JQpRr0~sMnpI0Be7Va1U5FiAOIFnSwvU~wtfGNdqr7coHBNmT-UZKLK28GoTivwRNdpB8zhuNdpBExWOOnuphLfW~qIVsJXtJqWHmNVxYkmn~tEe26h74cNZ-sgD-D4plD30q8gL9DijSa29ZsTj~Q7x4a6QhQiqqrPv8RD2u~5Jm2tWQCNKiJgL3aCz4Eqmbh0u3GjfxoI0oM2XxygguuwKyRvuGRg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979855&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Genetic characterization, based on PCR-fingerprinting, of the 33 UTAD <em>Hanseniaspora</em> strains recovered from the Douro demarcated region and of the selected reference strains <em>H. guilliermondii</em> CECT11027<sup>T</sup>, <em>H. occidentalis</em> CECT11341<sup>T</sup>, <em>H. osmophila</em> CECT 11206<sup>T</sup>, <em>H. uvarum</em> CECT1444<sup>T</sup> and <em>H. vineae</em> CECT1471<sup>T</sup>; (A) Dendrogram obtained by hierarchical analysis of PCR (GTG)<sub>5</sub> patterns using Pearson’s correlation coefficient and UPGMA clustering method; (B) Neighbour-joining phylogenetic tree showing the relationships of selected <em>Hanseniaspora</em> strains (marked with an asterisk in A), inferred from the sequences of the D1/D2 domain of the LSU RNA gene. Bootstrap percentages &gt;50% from 1,000 bootstrap replicates are shown. The outgroup species was <em>Saccharomyces cerevisiae</em>. Bar, 1% sequence divergence; (C) Restriction patterns obtained upon restriction with DraI of the ITS region of <em>Hanseniaspora</em> isolates or references strain: M, molecular marker; (a) isolate UTAD616, (b) isolate UTAD617, (c) isolate UTAD620, (d) isolate UTAD621, (e) isolate UTAD222, (f) reference strain <em>H. guilliermondii</em> CECT11027, (g) type strain <em>H. opuntiae</em> CBS8733<sup>T</sup> and (h) type strain <em>H. guilliermondii</em> CBS465<sup>T</sup>. The squares designate the different wineries from where the isolates were retrieved.</p></div>
</div>                    <h3 scrollto-destination="131979856" id="131979856" class="section-title" data-legacy-id="SEC3.2">3.2. Phenotypic screening of the autochthonous <em>Hanseniaspora</em> strains for interesting oenological properties</h3>
<p>The set of strains previously used in the genotypic screening was subjected to a phenotypic characterization focused on relevant oenological properties such as tolerance to sulphur dioxide and ethanol, ability to grow in presence of 5,5,5-trifluoro-<span class="small-caps">dl</span>-leucine (TFL) or cerulenin and activity of sulphite reductase, proteases or β-glucosidases (<a reveal-id="dsy039-F2" data-open="dsy039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The results obtained showed a significant variation in the phenotypic profile of the <em>H. uvarum</em> isolates tested (e.g. strains showed a marked difference in tolerance to ethanol or to SO<sub>2</sub>), while the three <em>H. guilliermondii</em> isolates (UTAD616, UTAD621, UTAD222) exhibited, in general, a more similar phenotypic profile (<a reveal-id="dsy039-F2" data-open="dsy039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The <em>H. opuntiae</em> isolates exhibited similar phenotypic traits to those exhibited by the <em>H. guilliermondii</em> isolates, with the exception of the reference strain CECT11027<sup>T</sup> that differed by having lower sulphite reductase activity and higher resistance to ethanol (<a reveal-id="dsy039-F2" data-open="dsy039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Another aspect that emerged from this phenotypic analysis was the considerably low protease activity and low tolerance to ethanol of most <em>H. uvarum</em> isolates, in comparison with the isolates gathered from the other <em>Hanseniaspora</em> species and, in particular, with the <em>H. guilliermondii</em> isolates (<a reveal-id="dsy039-F2" data-open="dsy039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). This observation is in line with previous studies reporting a higher resilience of <em>H. guilliermondii</em> to ethanol stress, specially when compared with other non-<em>Saccharomyces</em> species.<a reveal-id="dsy039-B30" data-open="dsy039-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The interesting oenological properties herein observed for the <em>H. guilliermondii</em> UTAD222 strain, namely the low activity of sulphite reductase, the high protease activity and the high tolerance to ethanol, reinforce the interest in deepening the study of this strain specially in what concerns to the analysis of its genome sequence. </p>                        <a id="131979858" scrollto-destination="131979858"></a>
<div id="dsy039-F2" data-id="dsy039-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f2.png?Expires=1576481646&amp;Signature=SHX0H3-X5zob0lLyA4wCmZNSYDPhGXqdmv4MaTM8DakeRb5mkxQnOH7TBQRhVovMuWGax1pm8tof0E648BUnabYRHnnWECske6VrbZpBjmRHbJou4IZuKB2OsO5joYTupVmza8oBvsv-lwiCWtD5tmIvTSasL8Ot1ejcrj4UVyWBkXWm~T~kKGTqAAxNf5xzqLxnOzr5rdXJJlc4nxo-SDZtIrlpYGoeSCch6RcsIRs26qanw1WNDyHe9VCpPEhKq3EpGO-epiB9gkjrLEHSI~xOX13chOQKurnZxViu9lIcOcjrkzGt5SZRLW4CbZhukKEhl5FZ9yRlpZbGv2w6gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenomic characterization of the Hanseniaspora indigenous strains concerning relevant oenological properties. Besides the isolates, the type strains H. opuntiae CECT11027T (formerly designated as H. guilliermondii but herein demonstrated to be H. opuntiae), H. occidentalis CECT11341T, H. osmophila CECT 11206T, H. uvarum CECT1444T and H. vineae CECT1471T. After conducting the phenotypic tests (in triplicate), the results were used to build the heat map shown in the figure and used for the subsequent clustering analysis shown. Strains classification was based on the results of the PCR fingerprinting and, in some cases, also based on results of the D1/D2 sequencing and subsequent analysis of the DraI restriction profile (to distinguish between the H. opuntiae and H. guilliermondii strains). Hanseniaspora uvarum strains are identified in light blue, H. opuntiae strains in green and H. guilliermondii in red." path-from-xml="dsy039f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979858/dsy039f2.tif" path-from-xml="dsy039f2.tif" target="_blank">Open in new tab</a><a section="131979858" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f2.png?Expires=1576481646&amp;Signature=oRcJGYCvDLjal8284OwOLZ6XJkBVVpLOwfH~w9OgsRY~EUIdYWWAoeZdT-tix56rHe85oA641vM1snP2VfIfcvDo7xhDMx0cjyJwtg~MXnOWkCTyhqlH4wsrXm3yPbCmTtrOOZZChojKOiukQSkhjswDlJ0SyS43qcCCv0l6v4EaYkRA64CfQnf8S1nkWGllo~k1pIE5V10PDDXcHDjTTJPMQmYenQvKPk~bgjXtsfbTwQwypMK48sf5kqyBOGL9~Y04Ghhe2-yZePIUbiofmuSXQ3qIHu-JCeDRkOuA2Fku26maqTvW2nmBo53ym3JIs6gSWe7nz3CNqB3CWNyAXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979858&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Phenomic characterization of the <em>Hanseniaspora</em> indigenous strains concerning relevant oenological properties. Besides the isolates, the type strains <em>H. opuntiae</em> CECT11027<sup>T</sup> (formerly designated as <em>H. guilliermondii</em> but herein demonstrated to be <em>H. opuntiae</em>), <em>H. occidentalis</em> CECT11341<sup>T</sup>, <em>H. osmophila</em> CECT 11206<sup>T</sup>, <em>H. uvarum</em> CECT1444<sup>T</sup> and <em>H. vineae</em> CECT1471<sup>T</sup>. After conducting the phenotypic tests (in triplicate), the results were used to build the heat map shown in the figure and used for the subsequent clustering analysis shown. Strains classification was based on the results of the PCR fingerprinting and, in some cases, also based on results of the D1/D2 sequencing and subsequent analysis of the DraI restriction profile (to distinguish between the <em>H. opuntiae</em> and <em>H. guilliermondii</em> strains). <em>Hanseniaspora uvarum</em> strains are identified in light blue, <em>H. opuntiae</em> strains in green and <em>H. guilliermondii</em> in red.</p></div>
</div>
<div content-id="dsy039-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f2.png?Expires=1576481646&amp;Signature=SHX0H3-X5zob0lLyA4wCmZNSYDPhGXqdmv4MaTM8DakeRb5mkxQnOH7TBQRhVovMuWGax1pm8tof0E648BUnabYRHnnWECske6VrbZpBjmRHbJou4IZuKB2OsO5joYTupVmza8oBvsv-lwiCWtD5tmIvTSasL8Ot1ejcrj4UVyWBkXWm~T~kKGTqAAxNf5xzqLxnOzr5rdXJJlc4nxo-SDZtIrlpYGoeSCch6RcsIRs26qanw1WNDyHe9VCpPEhKq3EpGO-epiB9gkjrLEHSI~xOX13chOQKurnZxViu9lIcOcjrkzGt5SZRLW4CbZhukKEhl5FZ9yRlpZbGv2w6gw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phenomic characterization of the Hanseniaspora indigenous strains concerning relevant oenological properties. Besides the isolates, the type strains H. opuntiae CECT11027T (formerly designated as H. guilliermondii but herein demonstrated to be H. opuntiae), H. occidentalis CECT11341T, H. osmophila CECT 11206T, H. uvarum CECT1444T and H. vineae CECT1471T. After conducting the phenotypic tests (in triplicate), the results were used to build the heat map shown in the figure and used for the subsequent clustering analysis shown. Strains classification was based on the results of the PCR fingerprinting and, in some cases, also based on results of the D1/D2 sequencing and subsequent analysis of the DraI restriction profile (to distinguish between the H. opuntiae and H. guilliermondii strains). Hanseniaspora uvarum strains are identified in light blue, H. opuntiae strains in green and H. guilliermondii in red." path-from-xml="dsy039f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979858/dsy039f2.tif" path-from-xml="dsy039f2.tif" target="_blank">Open in new tab</a><a section="131979858" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f2.png?Expires=1576481646&amp;Signature=oRcJGYCvDLjal8284OwOLZ6XJkBVVpLOwfH~w9OgsRY~EUIdYWWAoeZdT-tix56rHe85oA641vM1snP2VfIfcvDo7xhDMx0cjyJwtg~MXnOWkCTyhqlH4wsrXm3yPbCmTtrOOZZChojKOiukQSkhjswDlJ0SyS43qcCCv0l6v4EaYkRA64CfQnf8S1nkWGllo~k1pIE5V10PDDXcHDjTTJPMQmYenQvKPk~bgjXtsfbTwQwypMK48sf5kqyBOGL9~Y04Ghhe2-yZePIUbiofmuSXQ3qIHu-JCeDRkOuA2Fku26maqTvW2nmBo53ym3JIs6gSWe7nz3CNqB3CWNyAXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979858&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Phenomic characterization of the <em>Hanseniaspora</em> indigenous strains concerning relevant oenological properties. Besides the isolates, the type strains <em>H. opuntiae</em> CECT11027<sup>T</sup> (formerly designated as <em>H. guilliermondii</em> but herein demonstrated to be <em>H. opuntiae</em>), <em>H. occidentalis</em> CECT11341<sup>T</sup>, <em>H. osmophila</em> CECT 11206<sup>T</sup>, <em>H. uvarum</em> CECT1444<sup>T</sup> and <em>H. vineae</em> CECT1471<sup>T</sup>. After conducting the phenotypic tests (in triplicate), the results were used to build the heat map shown in the figure and used for the subsequent clustering analysis shown. Strains classification was based on the results of the PCR fingerprinting and, in some cases, also based on results of the D1/D2 sequencing and subsequent analysis of the DraI restriction profile (to distinguish between the <em>H. opuntiae</em> and <em>H. guilliermondii</em> strains). <em>Hanseniaspora uvarum</em> strains are identified in light blue, <em>H. opuntiae</em> strains in green and <em>H. guilliermondii</em> in red.</p></div>
</div>                    <h4 scrollto-destination="131979859" id="131979859" class="section-title" data-legacy-id="SEC3.2.1">3.2.1. Genome structure of <em>H. guilliermondii</em> UTAD222 and comparative analysis with the genomes of other <em>Hanseniaspora</em> species</h4>
<p>To obtain the genome sequence of the <em>H. guilliermondii</em> UTAD222 strain, two independent rounds of paired-end MiSeq Illumina-sequencing were used.<a reveal-id="dsy039-B33" data-open="dsy039-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The assembly of the ca. 54 million reads obtained resulted in 208 contigs, yielding a total of 9,037,850 assembled bases (<a reveal-id="dsy039-T1" data-open="dsy039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This size predicted for the genome of the <em>H. guilliermondii</em> UTAD222 strain is in line with the genome size described for <em>H. uvarum</em> (ranging between 8.8. and 9.7 Mb<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a>) and <em>H. opuntiae</em> (8.8 Mb).<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The predicted GC content of <em>H. guilliermondii</em> UTAD222 is ∼31%, also in line with the percentages reported for other <em>Hanseniaspora</em> species (34.7% for <em>H. opuntiae</em> and 31.6% for <em>H. uvarum</em>) and for <em>S. cerevisiae</em> strains (38%). Prior karyotyping of <em>H. guilliermondii</em> wine strains from different geographical origins suggested that that this species is equipped with seven chromosomes, the same number that was attributed to most <em>H. uvarum</em> strains.<a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsy039-B51" data-open="dsy039-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsy039-B52" data-open="dsy039-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> In line with these results, karyotyping of the UTAD222 strain also revealed seven chromosomal bands (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Using this information and also taking advantage of the recently proposed chromosomal map for <em>H. uvarum</em> DSM2768 strain<a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and the close similarity between <em>H. guilliermondii</em>, <em>H. opuntiae</em> and <em>H. uvarum</em> species, we have built an organized chromosomal map of the 208 contigs obtained for the UTAD222 strain (<a reveal-id="dsy039-F3" data-open="dsy039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and further information provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). With this approach 192 out of the 208 contigs obtained for the <em>H. guilliermondii</em> UTAD222 strain were assigned to the 7 <em>H. guilliermondii</em> chromosomes (<a reveal-id="dsy039-F3" data-open="dsy039-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Among the non-assigned contigs was contig 141 (with an approximate size of 12 kb) which is predicted to be the mitochondrial chromosome, based on the high similarity with the mitochondrial sequence described for <em>H. uvarum.</em><a reveal-id="dsy039-B53" data-open="dsy039-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Comparison with the genomes <em>H. uvarum</em> AWRI3580 and <em>H. opuntiae</em> AWRI3578 proteins evidenced a significant colinearity since in the large majority of the cases orthologues between these species were found to be located contiguously. Despite the large genomic similarity, two relevant genomic alterations were observed in the genome of <em>H. guilliermondii</em> UTAD222 corresponding to presumable translocations in chromosomes C, G and F, as detailed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>. Another aspect that was also rendered clear by our comparative analysis was a higher divergence of the <em>H. uvarum</em> DSM2768 strain, comparing to <em>H. opuntiae</em> AWRI3578, <em>H. guilliermondii</em> UTAD222 and even to <em>H. uvarum</em> AWRI3580 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). With the data available, it is not possible to conclude if this divergence corresponds to a different genomic architectures of this strain or if it result from misleading attribution of contigs in the DSM2768 chromosomes since this assignment was very much based on adjusting the size of the chromosomes to the size of the contigs, thereby differing from the herein presented comparative genome analysis. </p>
<div content-id="dsy039-T1" class="table-modal"><div class="table-wrap" content-id="dsy039-T1">
<div id="dsy039-T1" data-id="dsy039-T1" class="table-wrap-title">
<div class="label">Table 1</div>
<div class="caption"><p>General features of the <em>H. guilliermondi</em>i UTAD222 genome obtained after the sequencing and the subsequent manually curated annotation</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr><th colspan="2">
<em>H. guilliermondii</em> genomic features </th></tr></thead>
<tbody>
<tr><td colspan="2">
<strong>Genome assembly statistics</strong> </td></tr>
<tr>
<td>
<em>Total number of reads</em> </td>
<td>53,913,308 </td>
</tr>
<tr>
<td>
<em>Nr of contigs</em> </td>
<td>208 </td>
</tr>
<tr>
<td>
<em>Coverage</em> </td>
<td>x819 </td>
</tr>
<tr>
<td>
<em>N50 (bp)</em> </td>
<td>91.417 </td>
</tr>
<tr>
<td>
<em>Maximum contig length (bp)</em> </td>
<td>247,000 </td>
</tr>
<tr>
<td>
<em>Minimum contig length (bp)</em> </td>
<td>1.023 </td>
</tr>
<tr>
<td>
<em>Average contig length (bp)</em> </td>
<td>43.451 </td>
</tr>
<tr>
<td>
<em>Assembly size (bp)</em> </td>
<td>9,037,850 </td>
</tr>
<tr>
<td>
<em>Average GC content (%)</em> </td>
<td>30.9 </td>
</tr>
<tr><td colspan="2">
<strong>Annotation</strong> </td></tr>
<tr>
<td>
<strong>Total nr of CDS</strong> </td>
<td>4,070 </td>
</tr>
<tr>
<td>
<em>tRNAs</em> </td>
<td>79 </td>
</tr>
<tr>
<td>
<em>rRNAs</em> </td>
<td>3 </td>
</tr>
<tr>
<td>
<em>Genes with 1 exon</em> </td>
<td>3,998 </td>
</tr>
<tr>
<td>
<em>Genes with 2 exons</em> </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Genes with more than 2 exons</em> </td>
<td>3 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy039-T1" class="table-modal"><table>
<thead name="thead" align="left"><tr><th colspan="2">
<em>H. guilliermondii</em> genomic features </th></tr></thead>
<tbody>
<tr><td colspan="2">
<strong>Genome assembly statistics</strong> </td></tr>
<tr>
<td>
<em>Total number of reads</em> </td>
<td>53,913,308 </td>
</tr>
<tr>
<td>
<em>Nr of contigs</em> </td>
<td>208 </td>
</tr>
<tr>
<td>
<em>Coverage</em> </td>
<td>x819 </td>
</tr>
<tr>
<td>
<em>N50 (bp)</em> </td>
<td>91.417 </td>
</tr>
<tr>
<td>
<em>Maximum contig length (bp)</em> </td>
<td>247,000 </td>
</tr>
<tr>
<td>
<em>Minimum contig length (bp)</em> </td>
<td>1.023 </td>
</tr>
<tr>
<td>
<em>Average contig length (bp)</em> </td>
<td>43.451 </td>
</tr>
<tr>
<td>
<em>Assembly size (bp)</em> </td>
<td>9,037,850 </td>
</tr>
<tr>
<td>
<em>Average GC content (%)</em> </td>
<td>30.9 </td>
</tr>
<tr><td colspan="2">
<strong>Annotation</strong> </td></tr>
<tr>
<td>
<strong>Total nr of CDS</strong> </td>
<td>4,070 </td>
</tr>
<tr>
<td>
<em>tRNAs</em> </td>
<td>79 </td>
</tr>
<tr>
<td>
<em>rRNAs</em> </td>
<td>3 </td>
</tr>
<tr>
<td>
<em>Genes with 1 exon</em> </td>
<td>3,998 </td>
</tr>
<tr>
<td>
<em>Genes with 2 exons</em> </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Genes with more than 2 exons</em> </td>
<td>3 </td>
</tr>
</tbody>
</table></div>
</div></div>
<div class="table-wrap" content-id="dsy039-T1">
<div id="dsy039-T1" data-id="dsy039-T1" class="table-wrap-title">
<div class="label">Table 1</div>
<div class="caption"><p>General features of the <em>H. guilliermondi</em>i UTAD222 genome obtained after the sequencing and the subsequent manually curated annotation</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="left"><tr><th colspan="2">
<em>H. guilliermondii</em> genomic features </th></tr></thead>
<tbody>
<tr><td colspan="2">
<strong>Genome assembly statistics</strong> </td></tr>
<tr>
<td>
<em>Total number of reads</em> </td>
<td>53,913,308 </td>
</tr>
<tr>
<td>
<em>Nr of contigs</em> </td>
<td>208 </td>
</tr>
<tr>
<td>
<em>Coverage</em> </td>
<td>x819 </td>
</tr>
<tr>
<td>
<em>N50 (bp)</em> </td>
<td>91.417 </td>
</tr>
<tr>
<td>
<em>Maximum contig length (bp)</em> </td>
<td>247,000 </td>
</tr>
<tr>
<td>
<em>Minimum contig length (bp)</em> </td>
<td>1.023 </td>
</tr>
<tr>
<td>
<em>Average contig length (bp)</em> </td>
<td>43.451 </td>
</tr>
<tr>
<td>
<em>Assembly size (bp)</em> </td>
<td>9,037,850 </td>
</tr>
<tr>
<td>
<em>Average GC content (%)</em> </td>
<td>30.9 </td>
</tr>
<tr><td colspan="2">
<strong>Annotation</strong> </td></tr>
<tr>
<td>
<strong>Total nr of CDS</strong> </td>
<td>4,070 </td>
</tr>
<tr>
<td>
<em>tRNAs</em> </td>
<td>79 </td>
</tr>
<tr>
<td>
<em>rRNAs</em> </td>
<td>3 </td>
</tr>
<tr>
<td>
<em>Genes with 1 exon</em> </td>
<td>3,998 </td>
</tr>
<tr>
<td>
<em>Genes with 2 exons</em> </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Genes with more than 2 exons</em> </td>
<td>3 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy039-T1" class="table-modal"><table>
<thead name="thead" align="left"><tr><th colspan="2">
<em>H. guilliermondii</em> genomic features </th></tr></thead>
<tbody>
<tr><td colspan="2">
<strong>Genome assembly statistics</strong> </td></tr>
<tr>
<td>
<em>Total number of reads</em> </td>
<td>53,913,308 </td>
</tr>
<tr>
<td>
<em>Nr of contigs</em> </td>
<td>208 </td>
</tr>
<tr>
<td>
<em>Coverage</em> </td>
<td>x819 </td>
</tr>
<tr>
<td>
<em>N50 (bp)</em> </td>
<td>91.417 </td>
</tr>
<tr>
<td>
<em>Maximum contig length (bp)</em> </td>
<td>247,000 </td>
</tr>
<tr>
<td>
<em>Minimum contig length (bp)</em> </td>
<td>1.023 </td>
</tr>
<tr>
<td>
<em>Average contig length (bp)</em> </td>
<td>43.451 </td>
</tr>
<tr>
<td>
<em>Assembly size (bp)</em> </td>
<td>9,037,850 </td>
</tr>
<tr>
<td>
<em>Average GC content (%)</em> </td>
<td>30.9 </td>
</tr>
<tr><td colspan="2">
<strong>Annotation</strong> </td></tr>
<tr>
<td>
<strong>Total nr of CDS</strong> </td>
<td>4,070 </td>
</tr>
<tr>
<td>
<em>tRNAs</em> </td>
<td>79 </td>
</tr>
<tr>
<td>
<em>rRNAs</em> </td>
<td>3 </td>
</tr>
<tr>
<td>
<em>Genes with 1 exon</em> </td>
<td>3,998 </td>
</tr>
<tr>
<td>
<em>Genes with 2 exons</em> </td>
<td>69 </td>
</tr>
<tr>
<td>
<em>Genes with more than 2 exons</em> </td>
<td>3 </td>
</tr>
</tbody>
</table></div>
</div>                        <a id="131979861" scrollto-destination="131979861"></a>
<div id="dsy039-F3" data-id="dsy039-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f3.png?Expires=1576481646&amp;Signature=RsqUkxtS3jVDRRYE7tPRJh9PzPYboiWGl2m16OnNSAqUBY5PuMGBAizbcSUThoYX2MkkHthRWM5ZqjwHAeKfCxhyT7KQfAd1oo-P644PJMYR6aYM75adCtbwD5FPocdqSRAw0co-z6l8p9yMdzS8XYFpTpvg7l91qBDyZ4BRYfuuavLzciDF9nKyQDyhtiG8n0K3kBWjkye8IoVb4bo5SrVA0QO4AejOoFE1jiDA8OAJ9Xb0Srex5IRJ1O-lLFM6QBRXbNQSfFM6QtT757rX1ScaEWs~lC6tPEli~tEW-66l0FQl4-~wkCn56LH0LuTlUNYsQbqMXDXkLDXOUdh28w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proposed chromosomal map for the H. guilliermondii UTAD222, based on data gathered from comparative genomic analysis herein gathered with those reported for H. uvarum (strains AWRI3580 and DSM7230)34,35 and H. opuntiae AWRI3578.34 Further details on how this map was obtained are provided in Supplementary Fig. S3 and in Materials and methods. In yellow, the contig including information predicted to harbour viral DNA is highlighted, while in green, the mitochondrial chromosome is highlighted." path-from-xml="dsy039f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979861/dsy039f3.tif" path-from-xml="dsy039f3.tif" target="_blank">Open in new tab</a><a section="131979861" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f3.png?Expires=1576481646&amp;Signature=1CLpjGTIcOL4jITgqNsMysDY7lEY-gnse0mqFOnxd7nlCT05MitaJ1XQ1-Yn~tywVcyWC9xST3fLLTD05yP79o7dMqxRfYtNgIDENOI5HkPVS9rJ4mUd16Uft9~4dnaKzOfZ1rIvFvFGUfrVH65LgbVJksir3GYNqr3O2WvB19~Gjgq8vbW1FRhb~iVbEvzpab8eQL4B1RHch8KbKRpO2tMWOkxB4yGDfxhFY66jIx-21a8hKRYUR2-yQ2E5xlY1WbuUyeM4yImv5AdEgbQTfDBZHpvQvHDsXxkq9FIkvNAuEA3Pv3jVNTYdIu~umdofLyHLV-SeoZOwtMXXmnU0Mw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979861&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Proposed chromosomal map for the <em>H. guilliermondii</em> UTAD222, based on data gathered from comparative genomic analysis herein gathered with those reported for <em>H. uvarum</em> (strains AWRI3580 and DSM7230)<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and <em>H. opuntiae</em> AWRI3578.<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Further details on how this map was obtained are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> and in Materials and methods. In yellow, the contig including information predicted to harbour viral DNA is highlighted, while in green, the mitochondrial chromosome is highlighted.</p></div>
</div>
<div content-id="dsy039-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f3.png?Expires=1576481646&amp;Signature=RsqUkxtS3jVDRRYE7tPRJh9PzPYboiWGl2m16OnNSAqUBY5PuMGBAizbcSUThoYX2MkkHthRWM5ZqjwHAeKfCxhyT7KQfAd1oo-P644PJMYR6aYM75adCtbwD5FPocdqSRAw0co-z6l8p9yMdzS8XYFpTpvg7l91qBDyZ4BRYfuuavLzciDF9nKyQDyhtiG8n0K3kBWjkye8IoVb4bo5SrVA0QO4AejOoFE1jiDA8OAJ9Xb0Srex5IRJ1O-lLFM6QBRXbNQSfFM6QtT757rX1ScaEWs~lC6tPEli~tEW-66l0FQl4-~wkCn56LH0LuTlUNYsQbqMXDXkLDXOUdh28w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proposed chromosomal map for the H. guilliermondii UTAD222, based on data gathered from comparative genomic analysis herein gathered with those reported for H. uvarum (strains AWRI3580 and DSM7230)34,35 and H. opuntiae AWRI3578.34 Further details on how this map was obtained are provided in Supplementary Fig. S3 and in Materials and methods. In yellow, the contig including information predicted to harbour viral DNA is highlighted, while in green, the mitochondrial chromosome is highlighted." path-from-xml="dsy039f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979861/dsy039f3.tif" path-from-xml="dsy039f3.tif" target="_blank">Open in new tab</a><a section="131979861" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f3.png?Expires=1576481646&amp;Signature=1CLpjGTIcOL4jITgqNsMysDY7lEY-gnse0mqFOnxd7nlCT05MitaJ1XQ1-Yn~tywVcyWC9xST3fLLTD05yP79o7dMqxRfYtNgIDENOI5HkPVS9rJ4mUd16Uft9~4dnaKzOfZ1rIvFvFGUfrVH65LgbVJksir3GYNqr3O2WvB19~Gjgq8vbW1FRhb~iVbEvzpab8eQL4B1RHch8KbKRpO2tMWOkxB4yGDfxhFY66jIx-21a8hKRYUR2-yQ2E5xlY1WbuUyeM4yImv5AdEgbQTfDBZHpvQvHDsXxkq9FIkvNAuEA3Pv3jVNTYdIu~umdofLyHLV-SeoZOwtMXXmnU0Mw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979861&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Proposed chromosomal map for the <em>H. guilliermondii</em> UTAD222, based on data gathered from comparative genomic analysis herein gathered with those reported for <em>H. uvarum</em> (strains AWRI3580 and DSM7230)<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and <em>H. opuntiae</em> AWRI3578.<a reveal-id="dsy039-B34" data-open="dsy039-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Further details on how this map was obtained are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> and in Materials and methods. In yellow, the contig including information predicted to harbour viral DNA is highlighted, while in green, the mitochondrial chromosome is highlighted.</p></div>
</div>                    <h4 scrollto-destination="131979862" id="131979862" class="section-title" data-legacy-id="SEC3.2.2">3.2.2. Annotation of the <em>H. guilliermondii</em> UTAD222 genomic sequence</h4>
<p>Automatic annotation of the genome sequence of <em>H. guilliermondii</em> UTAD222 was firstly performed using <em>ab initio</em> gene detection, afterwards manually curated to refine the results obtained by the informatics analysis. The predicted ORFeome of <em>H. guilliermondii</em> UTAD222 is estimated in 4,070 protein-encoding genes and 79 tRNAs (<a reveal-id="dsy039-T1" data-open="dsy039-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The incidence of introns is ∼1.8%, in line with the low percentages described for other yeast species.<a reveal-id="dsy039-B54" data-open="dsy039-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> In specific, 98% (corresponding to 3,998 genes) of the genes were predicted to be intron-free, 69 genes predicted to have two exons and three genes predicted to have three exons. The 4,070 genes that compose the predicted UTAD222 ORFeome were clustered according to their physiological function using MIPS-based Functional Catalogue. As depicted in <a reveal-id="dsy039-F4" data-open="dsy039-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, the number of <em>H. guilliermondii</em> genes (as well as of <em>H. opuntiae</em> and <em>H. uvarum)</em> included in each functional category was, on an average 70% the number of genes considered for <em>S. cerevisiae</em>, used herein as reference considering the extensive information available on gene functional analysis for this yeast species. This observation along with the fact that many of the <em>H. guilliermondii</em> proteins are similar to more than one <em>S. cerevisiae</em> protein (as detailed in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), strongly indicates that <em>H. guilliermondii</em> is a pre-genome duplication species, a trait that has been proposed for this species<a reveal-id="dsy039-B55" data-open="dsy039-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> and recently demonstrated for <em>H. uvarum.</em><a reveal-id="dsy039-B35" data-open="dsy039-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                        <a id="131979864" scrollto-destination="131979864"></a>
<div id="dsy039-F4" data-id="dsy039-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f4.png?Expires=1576481646&amp;Signature=lNyIKhFiTHNeOwVb0KieTdCczCGitxP1hQXr917Xu8i5YuJWaNT2T45RMbH5Cdg~C80Udsqi6nlb-SyZ8sjD57XMfcz1cEKivp--FgONqVuGJpeqXwiHEPM9hgxMGtv2eyY3ld09p0MZF7uaoqlZ~EcWwrjjGEsO3vZNDPiIGTiN2F8CQoKdz0bIAdqfMV3knvYmiGafEvYEtplFgU2Wh5l0ps-Vqstp2WgeilXS99XKH-LX2MVpNtXia3DqYKZ7Wb1xOIKHufNAfaFUMVApTeooWhEpThCBnRVttrb4vo9LA3aZfrw2TASjQG1AA8aEJoq~XmoJvRieqhEFjn3k3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional clustering of the proteins predicted to be encoded by H. guilliermondii UTAD222. Using the annotated and manually validated H. guilliermondii gene models (Hg), functional clustering was performed using MIPS functional catalogue. For the sake of comparison, a similar analysis was also performed for H. uvarum AWRI3578 (Hu), H. opuntiae AWRI3580 (Ho) and S. cerevisiae S288c (Sc) proteomes. For this, the different sets of proteins predicted to be encoded by the genomes of these three species were imported into Pedant database and then clustered, based on physiological function, using MIPS functional catalogue." path-from-xml="dsy039f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979864/dsy039f4.tif" path-from-xml="dsy039f4.tif" target="_blank">Open in new tab</a><a section="131979864" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f4.png?Expires=1576481646&amp;Signature=iImZkkYjncHslq1rswvKGSmH4oUbgiNi7wQd2N7sVtBzEalECn2-2~d0oAoNmzUnOfWt6O2nnuLrhhEeU7PPgcJakkIDiePxP4VFv2qSDvRgNFwH-Kkns8ZRb6DNMNqY658lU1ZXB17eCDDC3RdXBOX8MdUI5O7-zTqMwikEa9mchW2tLcvT2Ep7Fpq6PavOnI1Q5AApJw3g1bBha8FZqrPP3KALG1gr8veM1gTfaiMRaDCRQaG0PARMZfKPIysKhskvMzPK4WlYc5Pto7aEHzWof~7URAIGWBSMPt2H5ud-P13UwR5Cjcl5lpRatWp9Oh~c02ZPL1Pd6fd2JXJw6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979864&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Functional clustering of the proteins predicted to be encoded by <em>H. guilliermondii</em> UTAD222. Using the annotated and manually validated <em>H. guilliermondii</em> gene models (Hg), functional clustering was performed using MIPS functional catalogue. For the sake of comparison, a similar analysis was also performed for <em>H. uvarum</em> AWRI3578 (Hu), <em>H. opuntiae</em> AWRI3580 (Ho) and <em>S. cerevisiae S288c</em> (Sc) proteomes. For this, the different sets of proteins predicted to be encoded by the genomes of these three species were imported into Pedant database and then clustered, based on physiological function, using MIPS functional catalogue.</p></div>
</div>
<div content-id="dsy039-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f4.png?Expires=1576481646&amp;Signature=lNyIKhFiTHNeOwVb0KieTdCczCGitxP1hQXr917Xu8i5YuJWaNT2T45RMbH5Cdg~C80Udsqi6nlb-SyZ8sjD57XMfcz1cEKivp--FgONqVuGJpeqXwiHEPM9hgxMGtv2eyY3ld09p0MZF7uaoqlZ~EcWwrjjGEsO3vZNDPiIGTiN2F8CQoKdz0bIAdqfMV3knvYmiGafEvYEtplFgU2Wh5l0ps-Vqstp2WgeilXS99XKH-LX2MVpNtXia3DqYKZ7Wb1xOIKHufNAfaFUMVApTeooWhEpThCBnRVttrb4vo9LA3aZfrw2TASjQG1AA8aEJoq~XmoJvRieqhEFjn3k3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional clustering of the proteins predicted to be encoded by H. guilliermondii UTAD222. Using the annotated and manually validated H. guilliermondii gene models (Hg), functional clustering was performed using MIPS functional catalogue. For the sake of comparison, a similar analysis was also performed for H. uvarum AWRI3578 (Hu), H. opuntiae AWRI3580 (Ho) and S. cerevisiae S288c (Sc) proteomes. For this, the different sets of proteins predicted to be encoded by the genomes of these three species were imported into Pedant database and then clustered, based on physiological function, using MIPS functional catalogue." path-from-xml="dsy039f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979864/dsy039f4.tif" path-from-xml="dsy039f4.tif" target="_blank">Open in new tab</a><a section="131979864" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f4.png?Expires=1576481646&amp;Signature=iImZkkYjncHslq1rswvKGSmH4oUbgiNi7wQd2N7sVtBzEalECn2-2~d0oAoNmzUnOfWt6O2nnuLrhhEeU7PPgcJakkIDiePxP4VFv2qSDvRgNFwH-Kkns8ZRb6DNMNqY658lU1ZXB17eCDDC3RdXBOX8MdUI5O7-zTqMwikEa9mchW2tLcvT2Ep7Fpq6PavOnI1Q5AApJw3g1bBha8FZqrPP3KALG1gr8veM1gTfaiMRaDCRQaG0PARMZfKPIysKhskvMzPK4WlYc5Pto7aEHzWof~7URAIGWBSMPt2H5ud-P13UwR5Cjcl5lpRatWp9Oh~c02ZPL1Pd6fd2JXJw6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979864&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Functional clustering of the proteins predicted to be encoded by <em>H. guilliermondii</em> UTAD222. Using the annotated and manually validated <em>H. guilliermondii</em> gene models (Hg), functional clustering was performed using MIPS functional catalogue. For the sake of comparison, a similar analysis was also performed for <em>H. uvarum</em> AWRI3578 (Hu), <em>H. opuntiae</em> AWRI3580 (Ho) and <em>S. cerevisiae S288c</em> (Sc) proteomes. For this, the different sets of proteins predicted to be encoded by the genomes of these three species were imported into Pedant database and then clustered, based on physiological function, using MIPS functional catalogue.</p></div>
</div>                    <h4 scrollto-destination="131979865" id="131979865" class="section-title" data-legacy-id="SEC3.2.3">3.2.3. Reconstruction of <em>H. guilliermondii</em> UTAD222 metabolic network</h4>
<p>Using the KEGG Koala tool, the metabolic network of <em>H. guilliermondii</em> UTAD222 was reconstructed (available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4 and S5</a></span> and briefly summarized in <a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The results show that the <em>H. guilliermondii</em> UTAD222 strain is equipped with genes encoding enzymes involved in all major pathways of central carbon metabolism including glycolysis, TCA cycle and pentose phosphate pathway as well as the anaplerotic enzymes, pyruvate carboxylase and malic enzyme (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Notably, no genes encoding the key neoglucogenic enzymes fructose-1,6-bisphosphatase and phosphoenolpyruvate carboxykinase could be found among <em>H. guilliermondii</em> predicted proteins (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Homologues of these proteins have also not been identified in the predicted proteomes of <em>H. uvarum</em>, <em>H. opuntiae, H. vineae</em>, <em>H. osmophila</em> and <em>H. valbensys</em> (results not shown) suggesting that the absence of a functional neoglucogenesis is a feature of the <em>Hanseniaspora</em> genus. This is a particular interesting trait since robust orthologues for these enzymes are present in members of all the other families that compose the <em>Saccharomycetales</em> order including <em>Saccharomycetaceae</em>, <em>Metschnikowiaceae</em> and <em>Pichiaceae</em> that, like <em>Hanseniaspora</em>, also harbour wine species (results not shown). Consistent with the described ability of <em>H. guilliermondii</em> to use fructose,<a reveal-id="dsy039-B18" data-open="dsy039-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsy039-B56" data-open="dsy039-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> genes involved in catabolism of this hexose were predicted in the metabolic network of the UTAD222 strain, as well as genes involved in utilization of mannose and starch (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Conversely, genes required for the catabolism of xylose, galactose, lactate, acetate or glycerol were absent from <em>H. guilliermondii</em> genetic repertoire (and also from <em>H. uvarum</em> and <em>H. opuntiae</em>) including isocitrate lyase, galactokinase and glycerol or lactate dehydrogenase (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The lack of these genes is consistent with the inability of UTAD222 cells to grow using these carbohydrates as the sole sources of carbon and energy (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Although up to now the preferences of <em>Hanseniaspora</em> species concerning carbon sources have essentially focused their ability to consume glucose and/or fructose, it was recently reported the inability of <em>H. uvarum</em> wine isolates to use glycerol as the sole carbon source.<a reveal-id="dsy039-B50" data-open="dsy039-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Overall, the reconstruction of <em>H. guilliermondii</em> UTAD222 complemented with further analysis on the genome of other species having the genome sequenced shows that <em>Hanseniaspora</em> species appear to have a narrow range of carbon sources that can be used among those available in grape-musts, this being a factor that may contribute to reduce their competitiveness in the grape-must environment in which a strong competition for carbon sources is expected to occur. </p>                        <a id="131979867" scrollto-destination="131979867"></a>
<div id="dsy039-F5" data-id="dsy039-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f5.png?Expires=1576481646&amp;Signature=NUTP0PihiyYpjpMDBDVv34QEXZ9FcNrvH1MjiNy6LS1WRNPqUaKy0i2VjuSoh4AjSMTDoAuEA8qiNK82kAy6Rit7Wmnb0QYpzQaZTOuJ8m4bl5L4wnAd-LJl-MWwlH3ywNpSya1dmebmOkD~7S~-jyE6KkkH1wdOSsLxTrtr5pYH37feBEPgEZ-e9jPY-R-2WEJ6Cjy1fhX140z2rHT6m1EUSAQY0s-UE9tpiFRtXhipoaFNPVQfoZ4e3lYSn-QKhOdvGxKLjYAkoeOxiZ28lCKK8tbdnGuZypONX53yHAkljHMc190X4LTrhyjbYjA0A1pTQgy1YYVuGTBnAQv7nQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted metabolic network of H. guilliermondii focused on carbon (A) or on nitrogen/sulphur metabolism (B), as suggested by in silico metabolic reconstruction of the gathered genomics data. The gene models predicted for the H. guilliermondii UTAD222 were clustered according with the biochemical pathways they are predicted to be involved in using KEGG reconstruction tool. These results (shown in Supplementary Figs S3 and S4) along with the in silico comparative proteome analyses performed with H. opuntiae, H. uvarum and S. cerevisiae (described below) were used to draw the schematic representation herein presented." path-from-xml="dsy039f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979867/dsy039f5.tif" path-from-xml="dsy039f5.tif" target="_blank">Open in new tab</a><a section="131979867" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f5.png?Expires=1576481646&amp;Signature=y4LzZhY9GV2CrM30bT6QhWpV2CI8hMS4vEB4yK2fET8lRRbTgjJov9RIDtvOheIo1uXaTVQZB4b40I-b1jKDrWa4eev8HHJjsJjuA8~41ZWngznieSeDYqUd6Nm-FWdBgij39XHIdUZsmp3NauPX~Gi2gthL2h-pa24-Z17Yd6teDIUXGBhybXveGspGdyGgVaZA1fC10vfMFVCbKBZejrFKlY8K-QkuQ5VlrhqpijvCmMjJzp3xT63DJ2jpOXQeespHmoDi3rImmjIeOVGdp5AftOuhybhlxoUUv~o7~KCIWp47L2RbE95bcD2cmtw6f83Xah9nUAx14ATcN4vwGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979867&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Predicted metabolic network of <em>H. guilliermondii</em> focused on carbon (A) or on nitrogen/sulphur metabolism (B), as suggested by <em>in silico</em> metabolic reconstruction of the gathered genomics data. The gene models predicted for the <em>H. guilliermondii</em> UTAD222 were clustered according with the biochemical pathways they are predicted to be involved in using KEGG reconstruction tool. These results (shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>) along with the <em>in silico</em> comparative proteome analyses performed with <em>H. opuntiae</em>, <em>H. uvarum</em> and <em>S. cerevisiae</em> (described below) were used to draw the schematic representation herein presented.</p></div>
</div>
<div content-id="dsy039-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f5.png?Expires=1576481646&amp;Signature=NUTP0PihiyYpjpMDBDVv34QEXZ9FcNrvH1MjiNy6LS1WRNPqUaKy0i2VjuSoh4AjSMTDoAuEA8qiNK82kAy6Rit7Wmnb0QYpzQaZTOuJ8m4bl5L4wnAd-LJl-MWwlH3ywNpSya1dmebmOkD~7S~-jyE6KkkH1wdOSsLxTrtr5pYH37feBEPgEZ-e9jPY-R-2WEJ6Cjy1fhX140z2rHT6m1EUSAQY0s-UE9tpiFRtXhipoaFNPVQfoZ4e3lYSn-QKhOdvGxKLjYAkoeOxiZ28lCKK8tbdnGuZypONX53yHAkljHMc190X4LTrhyjbYjA0A1pTQgy1YYVuGTBnAQv7nQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted metabolic network of H. guilliermondii focused on carbon (A) or on nitrogen/sulphur metabolism (B), as suggested by in silico metabolic reconstruction of the gathered genomics data. The gene models predicted for the H. guilliermondii UTAD222 were clustered according with the biochemical pathways they are predicted to be involved in using KEGG reconstruction tool. These results (shown in Supplementary Figs S3 and S4) along with the in silico comparative proteome analyses performed with H. opuntiae, H. uvarum and S. cerevisiae (described below) were used to draw the schematic representation herein presented." path-from-xml="dsy039f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979867/dsy039f5.tif" path-from-xml="dsy039f5.tif" target="_blank">Open in new tab</a><a section="131979867" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f5.png?Expires=1576481646&amp;Signature=y4LzZhY9GV2CrM30bT6QhWpV2CI8hMS4vEB4yK2fET8lRRbTgjJov9RIDtvOheIo1uXaTVQZB4b40I-b1jKDrWa4eev8HHJjsJjuA8~41ZWngznieSeDYqUd6Nm-FWdBgij39XHIdUZsmp3NauPX~Gi2gthL2h-pa24-Z17Yd6teDIUXGBhybXveGspGdyGgVaZA1fC10vfMFVCbKBZejrFKlY8K-QkuQ5VlrhqpijvCmMjJzp3xT63DJ2jpOXQeespHmoDi3rImmjIeOVGdp5AftOuhybhlxoUUv~o7~KCIWp47L2RbE95bcD2cmtw6f83Xah9nUAx14ATcN4vwGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979867&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Predicted metabolic network of <em>H. guilliermondii</em> focused on carbon (A) or on nitrogen/sulphur metabolism (B), as suggested by <em>in silico</em> metabolic reconstruction of the gathered genomics data. The gene models predicted for the <em>H. guilliermondii</em> UTAD222 were clustered according with the biochemical pathways they are predicted to be involved in using KEGG reconstruction tool. These results (shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>) along with the <em>in silico</em> comparative proteome analyses performed with <em>H. opuntiae</em>, <em>H. uvarum</em> and <em>S. cerevisiae</em> (described below) were used to draw the schematic representation herein presented.</p></div>
</div>
<p>Genes encoding enzymes required for the biosynthesis of all proteinogenic amino acids could be predicted from the proteome of <em>H. guilliermondii</em> UTAD222, as well as those genes involved in the incorporation of ammonia to glutamine via a NAD<sup>+</sup>-dependent glutamate dehydrogenase similar to ScGdh2 (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). <em>Hanseniaspora guilliermondii</em> UTAD222 is also equipped with a functional transsulfuration pathway for sulphate assimilation, although no genes could be detected for the methionine salvage pathway (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). This observation is consistent with the lack of genes predicted to be involved in synthesis of polyamines (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>) considering that one of the functions of the methionine salvage pathway is to supply precursors for the synthesis of spermine and spermidine (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). The fact that <em>H. guilliermondii</em> (and also <em>H. opuntiae</em> and <em>H. uvarum</em>) do not encode genes for synthesis of spermidine is interesting considering that this is an essential nutrient for <em>S. cerevisiae</em> growth.<a reveal-id="dsy039-B57" data-open="dsy039-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Spermine and spermidine are among the biogenic amines known to be present in grape-musts and they can also be produced along the alcoholic fermentation,<a reveal-id="dsy039-B58" data-open="dsy039-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> which thereby turns possible for <em>Hanseniaspora</em> spp to eventually scavenge these nutrients from the environment rather than prompting their synthesis.</p>
<p>Another relevant observation that emerged from the reconstruction of the metabolic network of the UTAD222 strain was the absence of genes involved in biosynthesis of thiamine, pyridoxal, pyridoxine, biotin and for the <em>de novo</em> synthesis of nicotinic acid (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). The absence of these genes is consistent with the previously described auxotrophy of <em>H. guilliermondii</em> for thiamin, niacin, biotin and pyridoxine.<a reveal-id="dsy039-B59" data-open="dsy039-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> The inability of <em>Hanseniaspora</em> species to synthesize these vitamins is likely to be a relevant factor in the described low fermentative capacity of these species, particularly in the case of thiamine whose abundance was found to determine the glycolytic flux and, consequently, the fermentation rate of <em>S. cerevisiae</em> cells.<a reveal-id="dsy039-B60" data-open="dsy039-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsy039-B61" data-open="dsy039-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> To respond to the depletion of thiamine, <em>S. cerevisiae</em> genes involved in biosynthesis of this vitamin are strongly up-regulated in stationary phase and wine yeast strains were found to evolve adaptive responses to improve the expression of these genes and, consequently, to improve the fermentation rate in grape-musts.<a reveal-id="dsy039-B61" data-open="dsy039-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Consistent with the idea that <em>H. guilliermondii</em> scavenges vitamins from the environment, <em>S. cerevisiae</em> genes involved in synthesis of thiamine (<em>ScTHI20</em> and Sc<em>THI21</em>), biotin (<em>ScBIO3</em>) or pyridoxine (e.g. <em>ScSNO1</em>), were among those found to be more strongly up-regulated during cultivation in the presence of <em>H. guilliermondii</em> UTAD222.<a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p>                    <h3 scrollto-destination="131979870" id="131979870" class="section-title" data-legacy-id="SEC3.3">3.3. The ‘transportome’ of <em>H. guilliermondii</em> UTAD222</h3>
<p>Around 600 genes are predicted to have a transport-related function in <em>H. guilliermondii</em> UTAD222, similar to the number of <em>H. uvarum</em> and <em>H. opuntiae</em> proteins predicted to have this function (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s2.xlsx?Expires=1576481646&amp;Signature=rqx4haQAQDF4gvmsOZqy3ut2Bts7KikhWiPA3XVEe6tmASDCK635-HQXr4gUKgcbHTr1Xag9pmMpRLQ3yFphTO9lbwCjLviersPZrvWzJORGIsmOVOagmUQdheXnZzq8wya6wRiz8jYgObtaMRl1gfShBq9uxAq~4A5RXRCpPyQ36RPa0mIu~JbRP-NVBxMxbOlBhkZbsRkjhHDu6Dv2e8933c7zw5inBgM48H4zNLDKqd4SQy4fNFS79MRA6KlA-B8Unobvea8kcKNRdUEstfSKEbZONG~Q4PeiAua8VM5RAEyxmCuYUGoV09V5QKPer-bq5q6byYOf5YRFW6DpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). A closer analysis into this ‘transportome’ of <em>H. guilliermondii</em> UTAD222, revealed that it is equipped with 22 predicted sugar transporters, 10 of these being predicted to be hexose transporters (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s2.xlsx?Expires=1576481646&amp;Signature=rqx4haQAQDF4gvmsOZqy3ut2Bts7KikhWiPA3XVEe6tmASDCK635-HQXr4gUKgcbHTr1Xag9pmMpRLQ3yFphTO9lbwCjLviersPZrvWzJORGIsmOVOagmUQdheXnZzq8wya6wRiz8jYgObtaMRl1gfShBq9uxAq~4A5RXRCpPyQ36RPa0mIu~JbRP-NVBxMxbOlBhkZbsRkjhHDu6Dv2e8933c7zw5inBgM48H4zNLDKqd4SQy4fNFS79MRA6KlA-B8Unobvea8kcKNRdUEstfSKEbZONG~Q4PeiAua8VM5RAEyxmCuYUGoV09V5QKPer-bq5q6byYOf5YRFW6DpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> and <a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). All these putative <em>H. guilliermondii</em> hexose transporters were found to harbour a very high similarity with the high-affinity glucose transporter ScHxt2<a reveal-id="dsy039-B62" data-open="dsy039-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> or with ScGal2, the galactose permease that has also been shown to transport glucose with a moderate affinity.<a reveal-id="dsy039-B62" data-open="dsy039-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Although <em>H. guilliermondii</em> strains (including UTAD222, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>) are fructophilic<a reveal-id="dsy039-B56" data-open="dsy039-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><a reveal-id="dsy039-B63" data-open="dsy039-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsy039-B64" data-open="dsy039-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> there was no orthologue of the fructose specific transporters Ffz1 or Fsy1.<a reveal-id="dsy039-B65" data-open="dsy039-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsy039-B66" data-open="dsy039-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> This observation is consistent with results of a previous study that pinpointed the <em>H. guilliermondii</em> NCYC2380 strain as exhibiting fructophilic behaviour but not harbouring Ffz-like genes.<a reveal-id="dsy039-B56" data-open="dsy039-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> A large survey for Ffz-like genes in Fungi has also failed to identify robust orthologues for these transporters in the genome of <em>H. uvarum,</em> another reported fructophilic yeast,<a reveal-id="dsy039-B64" data-open="dsy039-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsy039-B67" data-open="dsy039-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> as well as in <em>H. vineae</em> and <em>H. valbensyis</em>.<a reveal-id="dsy039-B68" data-open="dsy039-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Interestingly, the <em>H. guilliermondii</em> UTAD222 protein showing a higher similarity with <em>Z. bailii</em> Ffz1 (e-value of the associated alignment of −56, 27% identity) is HGUI01014, predicted to be a polyamine exporter based on its high degree of similarity with the MFS multi-drug resistance transport ScTpo1 (e-value of the associated alignment of −156, 57% identity) (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s2.xlsx?Expires=1576481646&amp;Signature=rqx4haQAQDF4gvmsOZqy3ut2Bts7KikhWiPA3XVEe6tmASDCK635-HQXr4gUKgcbHTr1Xag9pmMpRLQ3yFphTO9lbwCjLviersPZrvWzJORGIsmOVOagmUQdheXnZzq8wya6wRiz8jYgObtaMRl1gfShBq9uxAq~4A5RXRCpPyQ36RPa0mIu~JbRP-NVBxMxbOlBhkZbsRkjhHDu6Dv2e8933c7zw5inBgM48H4zNLDKqd4SQy4fNFS79MRA6KlA-B8Unobvea8kcKNRdUEstfSKEbZONG~Q4PeiAua8VM5RAEyxmCuYUGoV09V5QKPer-bq5q6byYOf5YRFW6DpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Further studies are required to better understand what could be the function of predicted hexose transporters in <em>H. guilliermondii</em> and, in specific, if they are somehow able to contribute for the fructophilic behaviour of the species by mediating fructose transport. Phosphorylation of fructose mediated by hexokinases is another critical parameter that was shown to determine the capacity of wine <em>S. cerevisiae</em> strains to use fructose in detriment of glucose.<a reveal-id="dsy039-B69" data-open="dsy039-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a><sup>,</sup><a reveal-id="dsy039-B70" data-open="dsy039-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Thus, it can also be hypothesized that the two hexokinases predicted to be encoded by <em>H. guilliermondii</em> UTAD222 (HGUI_01771, 47% identity with glucokinase ScGlk1 and HGUI_03922, 72% identity) could have a higher affinity for fructose than for glucose. Indeed, preliminary results have shown that <em>Z. bailii</em> encodes a hexokinase having a preference for fructose (over glucose) phosphorylation,<a reveal-id="dsy039-B71" data-open="dsy039-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> the hexokinases encoded by <em>H. guilliermondii</em> showing a very high similarity (99% identity at the amino acid level) with those encoded by <em>Z. bailii</em> strains having their genome sequenced (results not shown).</p>
<p>Thirty-two <em>H. guilliermondii</em> UTAD222 genes are predicted to encode transporters of nitrogenous compounds (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s3.xlsx?Expires=1576481646&amp;Signature=gOGzhwkfzYhZhqwxGLarymUfQDqvQNKl~cDwFencFF5QT6yTGJLYfMPD8OEL5MqbF5TYWFxJJ0q0fWOzq7P66UKmGv7~v-aU6MY47yAC1DEEpH7eBzNSihZ~v1e8FWb~AEDb9H0nr~flWVBcxYvbC788yeyHwmKrOONRSCXBBAcRNlcURilczLbLd5IRm8kCjiz5-k8Gurlxhp-YffN6WHb1BBOwOlWSEYxfFu0NLgh3J9svjTOzbHVQQMrwXfz-izYLF1IKFltc5K~Xh3W3nRGTGzXZDTkIMcb0K65iQcDKs~ayzIJkpFgP6igCR1SPIRYuhmcvNx0rz8FZdOKKwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> and <a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>) including permeases for amino acids, ammonium, allantoin, urea and GABA (<a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Remarkably, 13 of the predicted <em>H. guilliermondii</em> amino acid permeases bear the highest similarity with the general amino acid permeases Sc<em>GAP1</em> and Sc<em>AGP1</em>, only being possible to identify specific permeases for lysine and methionine (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s2.xlsx?Expires=1576481646&amp;Signature=rqx4haQAQDF4gvmsOZqy3ut2Bts7KikhWiPA3XVEe6tmASDCK635-HQXr4gUKgcbHTr1Xag9pmMpRLQ3yFphTO9lbwCjLviersPZrvWzJORGIsmOVOagmUQdheXnZzq8wya6wRiz8jYgObtaMRl1gfShBq9uxAq~4A5RXRCpPyQ36RPa0mIu~JbRP-NVBxMxbOlBhkZbsRkjhHDu6Dv2e8933c7zw5inBgM48H4zNLDKqd4SQy4fNFS79MRA6KlA-B8Unobvea8kcKNRdUEstfSKEbZONG~Q4PeiAua8VM5RAEyxmCuYUGoV09V5QKPer-bq5q6byYOf5YRFW6DpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> and <a reveal-id="dsy039-F5" data-open="dsy039-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Indeed, almost all <em>S. cerevisiae</em> amino acid-specific permeases were found to be absent in <em>H. guilliermondii</em> and also in <em>H. opuntiae</em> and in <em>H. uvarum</em> (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> and Table 4). This observation suggests that <em>Hanseniaspora</em> species favor the utilization of permeases with a broader range of substrates in detriment of using specific permeases, which is a reasonable adaptive mechanism since the amount of nitrogen present in wine musts is largely variable and composed by different amino acids.</p>                    <h3 scrollto-destination="131979873" id="131979873" class="section-title" data-legacy-id="SEC3.3.1">3.3.1. The <em>H. guilliermondii</em> UTAD222 ‘flavorome’</h3>
<p>It is known that <em>H. guilliermondii</em> affects the production of volatile compounds of wines when in consortium with <em>S. cerevisiae</em>.<a reveal-id="dsy039-B16 dsy039-B17 dsy039-B18" data-open="dsy039-B16 dsy039-B17 dsy039-B18" class="link link-ref link-reveal xref-bibr"><sup>16–18</sup></a><sup>,</sup><a reveal-id="dsy039-B27" data-open="dsy039-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Other studies have also reported the ability of <em>H. guilliermondii</em> itself to produce compounds having an impact in wine aroma even in single-culture including 2-phenyl ethyl acetate, ethyl acetate, isobutyric acid, ethyl esters and higher alcohols, albeit in most of these cases the production titers are below those reported for <em>S. cerevisiae.</em><a reveal-id="dsy039-B16 dsy039-B17 dsy039-B18 dsy039-B19" data-open="dsy039-B16 dsy039-B17 dsy039-B18 dsy039-B19" class="link link-ref link-reveal xref-bibr"><sup>16–19</sup></a><sup>,</sup><a reveal-id="dsy039-B72" data-open="dsy039-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Consistent with these observations, reconstruction of <em>H. guilliermondii</em> UTAD222 metabolic network shows that this strain is equipped with four genes encoding β-glucosidases, as well as genes required for synthesis of acetaldehyde, ethyl esters and higher alcohols (<a reveal-id="dsy039-F6" data-open="dsy039-F6" class="link link-reveal link-table xref-fig">Fig. 6</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s3.xlsx?Expires=1576481646&amp;Signature=gOGzhwkfzYhZhqwxGLarymUfQDqvQNKl~cDwFencFF5QT6yTGJLYfMPD8OEL5MqbF5TYWFxJJ0q0fWOzq7P66UKmGv7~v-aU6MY47yAC1DEEpH7eBzNSihZ~v1e8FWb~AEDb9H0nr~flWVBcxYvbC788yeyHwmKrOONRSCXBBAcRNlcURilczLbLd5IRm8kCjiz5-k8Gurlxhp-YffN6WHb1BBOwOlWSEYxfFu0NLgh3J9svjTOzbHVQQMrwXfz-izYLF1IKFltc5K~Xh3W3nRGTGzXZDTkIMcb0K65iQcDKs~ayzIJkpFgP6igCR1SPIRYuhmcvNx0rz8FZdOKKwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Notably, it was not found within the UTAD222 ORFeome proteins similar to the <em>S. cerevisae</em> acetyl transferases involved in synthesis of acetate esters Sc<em>ATF1</em>, Sc<em>ATF2</em> and Sc<em>AYT1</em>, although it is known that these are precisely the aroma compounds whose production is more strongly impacted by the presence of <em>H. guilliermondii.</em><a reveal-id="dsy039-B17" data-open="dsy039-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy039-B19" data-open="dsy039-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy039-B72" data-open="dsy039-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Four proteins (HGUI_00697, HGUI_00952, HGUI_01907 and HGUI_01910) harboring motifs conserved within the alcohol acetyltransferase enzyme family were identified in the predicted proteome of <em>H. guilliermondii</em> UTAD222, these proteins only having orthologues in other species of the <em>Hanseniaspora</em> genus (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s3.xlsx?Expires=1576481646&amp;Signature=gOGzhwkfzYhZhqwxGLarymUfQDqvQNKl~cDwFencFF5QT6yTGJLYfMPD8OEL5MqbF5TYWFxJJ0q0fWOzq7P66UKmGv7~v-aU6MY47yAC1DEEpH7eBzNSihZ~v1e8FWb~AEDb9H0nr~flWVBcxYvbC788yeyHwmKrOONRSCXBBAcRNlcURilczLbLd5IRm8kCjiz5-k8Gurlxhp-YffN6WHb1BBOwOlWSEYxfFu0NLgh3J9svjTOzbHVQQMrwXfz-izYLF1IKFltc5K~Xh3W3nRGTGzXZDTkIMcb0K65iQcDKs~ayzIJkpFgP6igCR1SPIRYuhmcvNx0rz8FZdOKKwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Further studies are required to confirm if these proteins do represent a novel set of acetyl transferases involved in synthesis of aroma compounds. Another observation of remark was the absence of aryl-alcohol dehydrogenases required for synthesis of higher alcohols from corresponding aldehydes in <em>H. guilliermondii</em>, which could contribute for the reported lower ability of this species to produce these compounds, specially when compared with <em>S. cerevisiae</em>. </p>                        <a id="131979875" scrollto-destination="131979875"></a>
<div id="dsy039-F6" data-id="dsy039-F6" class="fig fig-section">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f6.png?Expires=1576481646&amp;Signature=zmgZZc0Pjg3F4GPlTyiQmbZAJuDZ1JBt6rK~Q4o7qbnn2GmIpQp7aQzBO4RwZ6Nz~GoMyrJPTtNWk7aaFA79MT4ggHRx7ej6BUnbjqr19gkRpHdXa7HV~6sTAs-W6kkgUnr4tjdARfM-l5hWcKQLDZubo9sEED4GATVslMzE~xPdJvMcNdQpZIhVTwrCKd18mCAg-EU1qmGLUKEjBLfayZnuAbJs-2n5884PbrJXAOAHWlDTqjxAAUXnAv-jGgqrRk4WV6KlVmwv57mNsyQs98p3P~LJGmljWns2WlyrzABf92cA83L-sg8xJDMkqA6rNMNIo2ChjqKVPIaxlSkk~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hanseniaspora guilliermondii UTAD222 genes predicted to be involved in formation of aroma compounds. Based on results of metabolic reconstruction and also on orthology with S. cerevisiae genes, the set of H. guilliermondii UTAD222 genes predicted to be involved in pathways leading to the production of aroma compounds was identified. Further details on these genes involved in the ‘flavorome’ of H. guilliermondii are provided in Supplementary Table S4. No genes encoding acetyl transferases involved in synthesis of acetate esters could be predicted in the H. guilliermondii UTAD222 ORFeome and comparative analysis with the proteins having such a function in S. cerevisiae also did not produce results. Nonetheless, four proteins (HGUI_00697, HGUI_00952, HGUI_01907 and HGUI_01910) with motifs found in acetyl transferases were identified in H. guilliermondii suggesting that these could represent a new class of these enzymes." path-from-xml="dsy039f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979875/dsy039f6.tif" path-from-xml="dsy039f6.tif" target="_blank">Open in new tab</a><a section="131979875" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f6.png?Expires=1576481646&amp;Signature=QtX9yO39sdCGfK9gEtJbOX6MuNAOZGKPIpr~WKW3jUo7zAfQRI37EHyv~mmIpjWRjSCb1~4SdpBfGcCuV9Qkq-ifrwycQiAcfuRv8DZrSZJjRHGuiGohK7-QI-wKV2lhsDW07Z1t9aZS2FoFw1shGfpFiQZszekUdgb-c3zXpkZb8jaLKun~LqDkim1VmIVH0f~zP4jnZxr-kaNT78FUpwdkO0hac6IQiamGRDZ7yHbD-vyW8XOaPk7I0UYCMwbeOgxGYZjfb5MIskhgj5DVe47It4gsmRoXsxz8pqS-kW7oUOGIRzBbuMC-9iaszSfxytqdZDFhVx0no~bUxLNZBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979875&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p><em>Hanseniaspora guilliermondii</em> UTAD222 genes predicted to be involved in formation of aroma compounds. Based on results of metabolic reconstruction and also on orthology with <em>S. cerevisiae</em> genes, the set of <em>H. guilliermondii</em> UTAD222 genes predicted to be involved in pathways leading to the production of aroma compounds was identified. Further details on these genes involved in the ‘flavorome’ of <em>H. guilliermondii</em> are provided in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s4.pdf?Expires=1576481646&amp;Signature=xmD7KbN6w9V1Nck6RqPJFJsYCM8W81TRckr8GzdsJS5-aE-5TVI2mCHpeTol6m0JLmCd4~hgpF-zT3P-KxHC2g7EIsas-f4DNr90r5-JaBRkT-uakvVJQnCEKheLdbZ0U698g~gJfQHGoB~EOAf2p3-NrcAIcb7x75esWKBXl5c3LlX~iNfuToQgqS85JEGkvZgv8roFKrVD7qSvrtaSkSjml6w9rQHV5Bwwb5qjea5c-vu0zNxikMxZD~yDT386CKDRykXORhYzH~17oXC3grGXq2gGnq-Qr8mp4SmEMM5ZyK8stSPdwhdMIZLPpNBHP3W~ojNm-ELOXLKi1ZxwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. No genes encoding acetyl transferases involved in synthesis of acetate esters could be predicted in the <em>H. guilliermondii</em> UTAD222 ORFeome and comparative analysis with the proteins having such a function in <em>S. cerevisiae</em> also did not produce results. Nonetheless, four proteins (<em>HGUI_00697, HGUI_00952, HGUI_01907</em> and <em>HGUI_01910)</em> with motifs found in acetyl transferases were identified in <em>H. guilliermondii</em> suggesting that these could represent a new class of these enzymes.</p></div>
</div>
<div content-id="dsy039-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f6.png?Expires=1576481646&amp;Signature=zmgZZc0Pjg3F4GPlTyiQmbZAJuDZ1JBt6rK~Q4o7qbnn2GmIpQp7aQzBO4RwZ6Nz~GoMyrJPTtNWk7aaFA79MT4ggHRx7ej6BUnbjqr19gkRpHdXa7HV~6sTAs-W6kkgUnr4tjdARfM-l5hWcKQLDZubo9sEED4GATVslMzE~xPdJvMcNdQpZIhVTwrCKd18mCAg-EU1qmGLUKEjBLfayZnuAbJs-2n5884PbrJXAOAHWlDTqjxAAUXnAv-jGgqrRk4WV6KlVmwv57mNsyQs98p3P~LJGmljWns2WlyrzABf92cA83L-sg8xJDMkqA6rNMNIo2ChjqKVPIaxlSkk~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hanseniaspora guilliermondii UTAD222 genes predicted to be involved in formation of aroma compounds. Based on results of metabolic reconstruction and also on orthology with S. cerevisiae genes, the set of H. guilliermondii UTAD222 genes predicted to be involved in pathways leading to the production of aroma compounds was identified. Further details on these genes involved in the ‘flavorome’ of H. guilliermondii are provided in Supplementary Table S4. No genes encoding acetyl transferases involved in synthesis of acetate esters could be predicted in the H. guilliermondii UTAD222 ORFeome and comparative analysis with the proteins having such a function in S. cerevisiae also did not produce results. Nonetheless, four proteins (HGUI_00697, HGUI_00952, HGUI_01907 and HGUI_01910) with motifs found in acetyl transferases were identified in H. guilliermondii suggesting that these could represent a new class of these enzymes." path-from-xml="dsy039f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979875/dsy039f6.tif" path-from-xml="dsy039f6.tif" target="_blank">Open in new tab</a><a section="131979875" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f6.png?Expires=1576481646&amp;Signature=QtX9yO39sdCGfK9gEtJbOX6MuNAOZGKPIpr~WKW3jUo7zAfQRI37EHyv~mmIpjWRjSCb1~4SdpBfGcCuV9Qkq-ifrwycQiAcfuRv8DZrSZJjRHGuiGohK7-QI-wKV2lhsDW07Z1t9aZS2FoFw1shGfpFiQZszekUdgb-c3zXpkZb8jaLKun~LqDkim1VmIVH0f~zP4jnZxr-kaNT78FUpwdkO0hac6IQiamGRDZ7yHbD-vyW8XOaPk7I0UYCMwbeOgxGYZjfb5MIskhgj5DVe47It4gsmRoXsxz8pqS-kW7oUOGIRzBbuMC-9iaszSfxytqdZDFhVx0no~bUxLNZBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979875&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p><em>Hanseniaspora guilliermondii</em> UTAD222 genes predicted to be involved in formation of aroma compounds. Based on results of metabolic reconstruction and also on orthology with <em>S. cerevisiae</em> genes, the set of <em>H. guilliermondii</em> UTAD222 genes predicted to be involved in pathways leading to the production of aroma compounds was identified. Further details on these genes involved in the ‘flavorome’ of <em>H. guilliermondii</em> are provided in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s4.pdf?Expires=1576481646&amp;Signature=xmD7KbN6w9V1Nck6RqPJFJsYCM8W81TRckr8GzdsJS5-aE-5TVI2mCHpeTol6m0JLmCd4~hgpF-zT3P-KxHC2g7EIsas-f4DNr90r5-JaBRkT-uakvVJQnCEKheLdbZ0U698g~gJfQHGoB~EOAf2p3-NrcAIcb7x75esWKBXl5c3LlX~iNfuToQgqS85JEGkvZgv8roFKrVD7qSvrtaSkSjml6w9rQHV5Bwwb5qjea5c-vu0zNxikMxZD~yDT386CKDRykXORhYzH~17oXC3grGXq2gGnq-Qr8mp4SmEMM5ZyK8stSPdwhdMIZLPpNBHP3W~ojNm-ELOXLKi1ZxwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. No genes encoding acetyl transferases involved in synthesis of acetate esters could be predicted in the <em>H. guilliermondii</em> UTAD222 ORFeome and comparative analysis with the proteins having such a function in <em>S. cerevisiae</em> also did not produce results. Nonetheless, four proteins (<em>HGUI_00697, HGUI_00952, HGUI_01907</em> and <em>HGUI_01910)</em> with motifs found in acetyl transferases were identified in <em>H. guilliermondii</em> suggesting that these could represent a new class of these enzymes.</p></div>
</div>                    <h3 scrollto-destination="131979876" id="131979876" class="section-title" data-legacy-id="SEC3.3.2">3.3.2. Comparative analysis of the predicted proteomes of <em>H. guilliermondii</em>, <em>H. uvarum</em> and <em>H. opuntiae</em>
</h3>
<p>To contribute for a better understanding of the differences and similarities existing among <em>H. guilliermondii</em>, <em>H. uvarum</em> and <em>H. opuntiae</em>, the UTAD222 predicted proteins were compared with those predicted for <em>H. opuntiae</em> AWRI3578 and for <em>H. uvarum</em> (AWRI3580 and DSM2768 strains). This comparative analysis rendered clear a very high similarity between the predicted ORFeomes of these three <em>Hanseniaspora</em> species, more evident for <em>H. guilliermondii</em> and <em>H. opuntiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s4.pdf?Expires=1576481646&amp;Signature=xmD7KbN6w9V1Nck6RqPJFJsYCM8W81TRckr8GzdsJS5-aE-5TVI2mCHpeTol6m0JLmCd4~hgpF-zT3P-KxHC2g7EIsas-f4DNr90r5-JaBRkT-uakvVJQnCEKheLdbZ0U698g~gJfQHGoB~EOAf2p3-NrcAIcb7x75esWKBXl5c3LlX~iNfuToQgqS85JEGkvZgv8roFKrVD7qSvrtaSkSjml6w9rQHV5Bwwb5qjea5c-vu0zNxikMxZD~yDT386CKDRykXORhYzH~17oXC3grGXq2gGnq-Qr8mp4SmEMM5ZyK8stSPdwhdMIZLPpNBHP3W~ojNm-ELOXLKi1ZxwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Fourteen <em>H. guilliermondii</em> proteins had no orthologue in <em>H. opuntiae</em> AWRI3578 or <em>H. uvarum</em> (AWRI3580 and DSM2768 strains), these being listed in <a reveal-id="dsy039-T2" data-open="dsy039-T2" class="link link-reveal link-table xref-fig">Table 2</a>. Nine of these proteins had a robust homologue in <em>H. valbyensis</em> indicating that their presence is not specific of the <em>H. guilliermondii</em> species (<a reveal-id="dsy039-T2" data-open="dsy039-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The remaining five proteins could not be identified in the genome of any other species of the <em>Hanseniaspora</em> genus, although homologues could be identified in the genome of other fungal species such as <em>Rhizopus microspores</em>, <em>Zygosaccharomyces bailii</em> or <em>Saitoella complicate</em> (<a reveal-id="dsy039-T2" data-open="dsy039-T2" class="link link-reveal link-table xref-fig">Table 2</a>). These five proteins are all of viral origin including components necessary for the formation of capsid and spike and also a protein required for replication of viral dsDNA (<a reveal-id="dsy039-T2" data-open="dsy039-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The fact that the genes encoding these proteins are contiguous and located in the same genomic region (contig 168) and also the observation that this genomic region exhibits a very strong homology (&gt;95% identity at the nucleotide level) with the genomic sequences of bacteriophages, suggest that <em>H. guilliermondii</em> UTAD222 harbors a viral genome. The existence of viral DNA has been well documented in other wine yeast species including <em>S. cerevisiae</em>, <em>H. uvarum</em> and <em>Z. bailii</em>,<a reveal-id="dsy039-B73" data-open="dsy039-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> this being responsible for the killer phenotype that allows cells to secrete toxin(s) to the growth medium that will cause death of sensitive species. In the cases of the three above referred yeast species, the killer phenotype is mediated by non-infectious dsRNA viruses,<a reveal-id="dsy039-B73" data-open="dsy039-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> but this phenotype can also be chromosomally encoded or associated with dsDNA plasmids.<a reveal-id="dsy039-B73" data-open="dsy039-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> Up to now the ability of <em>H. guilliermondii</em> to induce the killer phenotype has not been studied and the viral genomic sequence herein uncovered does not provide enough information to understand if a toxin can indeed be produced by UTAD222 cells. Nonetheless, this is a topic deserving further investigation considering the role that the killer phenotype has in shaping competitiveness of wine yeast species. </p>
<div content-id="dsy039-T2" class="table-modal"><div class="table-wrap" content-id="dsy039-T2">
<div id="dsy039-T2" data-id="dsy039-T2" class="table-wrap-title">
<div class="label">Table 2</div>
<div class="caption"><p>Results obtained from the comparative analysis of the predicted <em>H. guilliermondii</em> UTAD222 ORFeome with the set of proteins predicted for <em>H. uvarum</em> (AWRI3580 and DSM7210 strains) or <em>H. opuntiae</em></p></div>
</div>
<div class="table-overflow"><table><tbody>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins not having an orthologue in both <em>H. opuntiae</em> and <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>ORF name </td>
<td>Probable function </td>
<td>Best ortdologue </td>
</tr>
<tr>
<td>HGUI_01127 </td>
<td>BolA-like protein </td>
<td>HANVADRAFT_51108 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01165 </td>
<td>Ribosomal protein </td>
<td>HANVADRAFT_5002 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01445 </td>
<td>Predicted small nuclear ribonucleo protein </td>
<td>HANVADRAFT_11820 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01959 </td>
<td>Predicted ribonucleo protein complex subunit 3 </td>
<td>HANVADRAFT_53037 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02479 </td>
<td>Unknown </td>
<td>HANVADRAFT_53473 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02541 </td>
<td>Unknown </td>
<td>HANVADRAFT_53361 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03386 </td>
<td>Unknown </td>
<td>HANVADRAFT_53900 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03446 </td>
<td>Unknown </td>
<td>HANVADRAFT_23140 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03596 </td>
<td>Predicted vacuolar ATPase assembly integral membrane protein </td>
<td>HANVADRAFT_22909 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_04019 </td>
<td>Minor spike protein H </td>
<td>CP02DC15_1109 from <em>Chlamydia psittaci</em> 02DC15 </td>
</tr>
<tr>
<td>HGUI_04020 </td>
<td>Replication-associated protein A </td>
<td>HMPREF1334_01983 from <em>Enterococcus faecalis</em> ERV41 </td>
</tr>
<tr>
<td>HGUI_04021 </td>
<td>External scaffolding protein D </td>
<td>M771_11055 from <em>Neisseria gonorrhoeae</em> MU_NG1 </td>
</tr>
<tr>
<td>HGUI_04022 </td>
<td>Capsid protein </td>
<td>M743_11545 from <em>Neisseria gonorrhoeae</em> NYC_2011_05_13 </td>
</tr>
<tr>
<td>HGUI_04023 </td>
<td>Major spike protein G </td>
<td>MPLA_2010001 from <em>Mesorhizobium sp</em>. ORS335 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. uvarum</em> but not in <em>H. opuntiae</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. uvarum</em> orthologue (in AWRI3580/in DSM2768)</strong> </td>
</tr>
<tr>
<td>HGUI_03532 </td>
<td>Predicted to be involved in endosome-to-Golgi intracellular trafficking </td>
<td>−/HuPEP8 </td>
</tr>
<tr>
<td>HGUI_00706 </td>
<td>Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3580_g2240/HuALG6 </td>
</tr>
<tr>
<td>HGUI_03286 </td>
<td>Dolichyl-diphosphooligosaccharide–protein glycosyltransferase </td>
<td>AWRI3580_g1764/HuOST2 </td>
</tr>
<tr>
<td>HGUI_03882 </td>
<td>Dol-P-Glc: Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase </td>
<td>AWRI3580_g1578/- </td>
</tr>
<tr>
<td>HGUI_01192 </td>
<td>Eukaryotic translation initiation factor eIF-1 </td>
<td>AWRI3580_g477/- </td>
</tr>
<tr>
<td>HGUI_00394 </td>
<td>Predicted hydroxyacyl-thioester dehydratase involved in fatty acid biosynthesis </td>
<td>AWRI3580_g894/HuHTD2 </td>
</tr>
<tr>
<td>HGUI_03531 </td>
<td>Predicted mitochondrial import inner membrane translocase subunit </td>
<td>AWRI3580_g2710/HuTIM54 </td>
</tr>
<tr>
<td>HGUI_02284 </td>
<td>Predicted to be involved in intracellular protein trafficking, similar to ScSss1 </td>
<td>AWRI3580_g3696/- </td>
</tr>
<tr>
<td>HGUI_03514 </td>
<td>Predicted UDP-N-acetylglucosamine transferase </td>
<td>AWRI3580_g742/HuALG14 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. opuntiae</em> but not in <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. opuntiae</em> orthologue</strong> </td>
</tr>
<tr>
<td>HGUI_02309 </td>
<td>Predicted chitin synthase export chaperone </td>
<td>AWRI3578_g694 </td>
</tr>
<tr>
<td>HGUI_03602 </td>
<td>Predicted vacuolar membrane Cu transporter </td>
<td>AWRI3578_g678 </td>
</tr>
<tr>
<td>HGUI_02076 </td>
<td>Cytochrome c oxidase assembly protein Cox19 </td>
<td>AWRI3578_g1039 </td>
</tr>
<tr>
<td>HGUI_01733 </td>
<td>Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3578_g4036 </td>
</tr>
<tr>
<td>HGUI_01849 </td>
<td>Predicted to be involved in meiosis, similar to ScSpo11 </td>
<td>AWRI3578_g1481 </td>
</tr>
<tr>
<td>HGUI_02598 </td>
<td>Mitochondrial import inner membrane translocase subunit TIM12 </td>
<td>AWRI3578_g3410 </td>
</tr>
<tr>
<td>HGUI_01086 </td>
<td>Mitochondrial protein required for assembly of cytochrome c oxidase, mitochondrial </td>
<td>AWRI3578_g916 </td>
</tr>
<tr>
<td>HGUI_02427 </td>
<td>Predicted to be involved in protein palmitoylation, similar to ScSwf1 </td>
<td>AWRI3578_g3718 </td>
</tr>
<tr>
<td>HGUI_00194 </td>
<td>Predicted transcriptional regulator of the Zn-finger family, shows some similarity with ScRim101 </td>
<td>AWRI3578_g3297 </td>
</tr>
<tr>
<td>HGUI_02287 </td>
<td>Putative 5-hydroxyisourate hydrolase </td>
<td>AWRI3578_g2657 </td>
</tr>
<tr>
<td>HGUI_00707 </td>
<td>Putative glucan endo-1,3-beta-glucosidase btgC </td>
<td>AWRI3578_g1650 </td>
</tr>
<tr>
<td>HGUI_01349 </td>
<td>Predicted GPI-anchored protein of unknown function </td>
<td>AWRI3578_g2558 </td>
</tr>
</tbody></table></div>
<div content-id="dsy039-T2" class="table-modal"><table><tbody>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins not having an orthologue in both <em>H. opuntiae</em> and <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>ORF name </td>
<td>Probable function </td>
<td>Best ortdologue </td>
</tr>
<tr>
<td>HGUI_01127 </td>
<td>BolA-like protein </td>
<td>HANVADRAFT_51108 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01165 </td>
<td>Ribosomal protein </td>
<td>HANVADRAFT_5002 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01445 </td>
<td>Predicted small nuclear ribonucleo protein </td>
<td>HANVADRAFT_11820 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01959 </td>
<td>Predicted ribonucleo protein complex subunit 3 </td>
<td>HANVADRAFT_53037 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02479 </td>
<td>Unknown </td>
<td>HANVADRAFT_53473 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02541 </td>
<td>Unknown </td>
<td>HANVADRAFT_53361 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03386 </td>
<td>Unknown </td>
<td>HANVADRAFT_53900 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03446 </td>
<td>Unknown </td>
<td>HANVADRAFT_23140 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03596 </td>
<td>Predicted vacuolar ATPase assembly integral membrane protein </td>
<td>HANVADRAFT_22909 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_04019 </td>
<td>Minor spike protein H </td>
<td>CP02DC15_1109 from <em>Chlamydia psittaci</em> 02DC15 </td>
</tr>
<tr>
<td>HGUI_04020 </td>
<td>Replication-associated protein A </td>
<td>HMPREF1334_01983 from <em>Enterococcus faecalis</em> ERV41 </td>
</tr>
<tr>
<td>HGUI_04021 </td>
<td>External scaffolding protein D </td>
<td>M771_11055 from <em>Neisseria gonorrhoeae</em> MU_NG1 </td>
</tr>
<tr>
<td>HGUI_04022 </td>
<td>Capsid protein </td>
<td>M743_11545 from <em>Neisseria gonorrhoeae</em> NYC_2011_05_13 </td>
</tr>
<tr>
<td>HGUI_04023 </td>
<td>Major spike protein G </td>
<td>MPLA_2010001 from <em>Mesorhizobium sp</em>. ORS335 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. uvarum</em> but not in <em>H. opuntiae</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. uvarum</em> orthologue (in AWRI3580/in DSM2768)</strong> </td>
</tr>
<tr>
<td>HGUI_03532 </td>
<td>Predicted to be involved in endosome-to-Golgi intracellular trafficking </td>
<td>−/HuPEP8 </td>
</tr>
<tr>
<td>HGUI_00706 </td>
<td>Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3580_g2240/HuALG6 </td>
</tr>
<tr>
<td>HGUI_03286 </td>
<td>Dolichyl-diphosphooligosaccharide–protein glycosyltransferase </td>
<td>AWRI3580_g1764/HuOST2 </td>
</tr>
<tr>
<td>HGUI_03882 </td>
<td>Dol-P-Glc: Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase </td>
<td>AWRI3580_g1578/- </td>
</tr>
<tr>
<td>HGUI_01192 </td>
<td>Eukaryotic translation initiation factor eIF-1 </td>
<td>AWRI3580_g477/- </td>
</tr>
<tr>
<td>HGUI_00394 </td>
<td>Predicted hydroxyacyl-thioester dehydratase involved in fatty acid biosynthesis </td>
<td>AWRI3580_g894/HuHTD2 </td>
</tr>
<tr>
<td>HGUI_03531 </td>
<td>Predicted mitochondrial import inner membrane translocase subunit </td>
<td>AWRI3580_g2710/HuTIM54 </td>
</tr>
<tr>
<td>HGUI_02284 </td>
<td>Predicted to be involved in intracellular protein trafficking, similar to ScSss1 </td>
<td>AWRI3580_g3696/- </td>
</tr>
<tr>
<td>HGUI_03514 </td>
<td>Predicted UDP-N-acetylglucosamine transferase </td>
<td>AWRI3580_g742/HuALG14 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. opuntiae</em> but not in <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. opuntiae</em> orthologue</strong> </td>
</tr>
<tr>
<td>HGUI_02309 </td>
<td>Predicted chitin synthase export chaperone </td>
<td>AWRI3578_g694 </td>
</tr>
<tr>
<td>HGUI_03602 </td>
<td>Predicted vacuolar membrane Cu transporter </td>
<td>AWRI3578_g678 </td>
</tr>
<tr>
<td>HGUI_02076 </td>
<td>Cytochrome c oxidase assembly protein Cox19 </td>
<td>AWRI3578_g1039 </td>
</tr>
<tr>
<td>HGUI_01733 </td>
<td>Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3578_g4036 </td>
</tr>
<tr>
<td>HGUI_01849 </td>
<td>Predicted to be involved in meiosis, similar to ScSpo11 </td>
<td>AWRI3578_g1481 </td>
</tr>
<tr>
<td>HGUI_02598 </td>
<td>Mitochondrial import inner membrane translocase subunit TIM12 </td>
<td>AWRI3578_g3410 </td>
</tr>
<tr>
<td>HGUI_01086 </td>
<td>Mitochondrial protein required for assembly of cytochrome c oxidase, mitochondrial </td>
<td>AWRI3578_g916 </td>
</tr>
<tr>
<td>HGUI_02427 </td>
<td>Predicted to be involved in protein palmitoylation, similar to ScSwf1 </td>
<td>AWRI3578_g3718 </td>
</tr>
<tr>
<td>HGUI_00194 </td>
<td>Predicted transcriptional regulator of the Zn-finger family, shows some similarity with ScRim101 </td>
<td>AWRI3578_g3297 </td>
</tr>
<tr>
<td>HGUI_02287 </td>
<td>Putative 5-hydroxyisourate hydrolase </td>
<td>AWRI3578_g2657 </td>
</tr>
<tr>
<td>HGUI_00707 </td>
<td>Putative glucan endo-1,3-beta-glucosidase btgC </td>
<td>AWRI3578_g1650 </td>
</tr>
<tr>
<td>HGUI_01349 </td>
<td>Predicted GPI-anchored protein of unknown function </td>
<td>AWRI3578_g2558 </td>
</tr>
</tbody></table></div>
<div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>In this table, a selected set of proteins was chosen to illustrate those only found in <em>H. guilliermondii</em>, those found in <em>H. guilliermondii</em> and in <em>H. opuntiae</em> and those found in <em>H. guilliermondii</em> and <em>H. uvarum</em>. A more comprehensive list including all the results is provided in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p></div></div></div>
</div></div>
<div class="table-wrap" content-id="dsy039-T2">
<div id="dsy039-T2" data-id="dsy039-T2" class="table-wrap-title">
<div class="label">Table 2</div>
<div class="caption"><p>Results obtained from the comparative analysis of the predicted <em>H. guilliermondii</em> UTAD222 ORFeome with the set of proteins predicted for <em>H. uvarum</em> (AWRI3580 and DSM7210 strains) or <em>H. opuntiae</em></p></div>
</div>
<div class="table-overflow"><table><tbody>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins not having an orthologue in both <em>H. opuntiae</em> and <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>ORF name </td>
<td>Probable function </td>
<td>Best ortdologue </td>
</tr>
<tr>
<td>HGUI_01127 </td>
<td>BolA-like protein </td>
<td>HANVADRAFT_51108 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01165 </td>
<td>Ribosomal protein </td>
<td>HANVADRAFT_5002 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01445 </td>
<td>Predicted small nuclear ribonucleo protein </td>
<td>HANVADRAFT_11820 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01959 </td>
<td>Predicted ribonucleo protein complex subunit 3 </td>
<td>HANVADRAFT_53037 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02479 </td>
<td>Unknown </td>
<td>HANVADRAFT_53473 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02541 </td>
<td>Unknown </td>
<td>HANVADRAFT_53361 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03386 </td>
<td>Unknown </td>
<td>HANVADRAFT_53900 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03446 </td>
<td>Unknown </td>
<td>HANVADRAFT_23140 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03596 </td>
<td>Predicted vacuolar ATPase assembly integral membrane protein </td>
<td>HANVADRAFT_22909 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_04019 </td>
<td>Minor spike protein H </td>
<td>CP02DC15_1109 from <em>Chlamydia psittaci</em> 02DC15 </td>
</tr>
<tr>
<td>HGUI_04020 </td>
<td>Replication-associated protein A </td>
<td>HMPREF1334_01983 from <em>Enterococcus faecalis</em> ERV41 </td>
</tr>
<tr>
<td>HGUI_04021 </td>
<td>External scaffolding protein D </td>
<td>M771_11055 from <em>Neisseria gonorrhoeae</em> MU_NG1 </td>
</tr>
<tr>
<td>HGUI_04022 </td>
<td>Capsid protein </td>
<td>M743_11545 from <em>Neisseria gonorrhoeae</em> NYC_2011_05_13 </td>
</tr>
<tr>
<td>HGUI_04023 </td>
<td>Major spike protein G </td>
<td>MPLA_2010001 from <em>Mesorhizobium sp</em>. ORS335 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. uvarum</em> but not in <em>H. opuntiae</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. uvarum</em> orthologue (in AWRI3580/in DSM2768)</strong> </td>
</tr>
<tr>
<td>HGUI_03532 </td>
<td>Predicted to be involved in endosome-to-Golgi intracellular trafficking </td>
<td>−/HuPEP8 </td>
</tr>
<tr>
<td>HGUI_00706 </td>
<td>Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3580_g2240/HuALG6 </td>
</tr>
<tr>
<td>HGUI_03286 </td>
<td>Dolichyl-diphosphooligosaccharide–protein glycosyltransferase </td>
<td>AWRI3580_g1764/HuOST2 </td>
</tr>
<tr>
<td>HGUI_03882 </td>
<td>Dol-P-Glc: Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase </td>
<td>AWRI3580_g1578/- </td>
</tr>
<tr>
<td>HGUI_01192 </td>
<td>Eukaryotic translation initiation factor eIF-1 </td>
<td>AWRI3580_g477/- </td>
</tr>
<tr>
<td>HGUI_00394 </td>
<td>Predicted hydroxyacyl-thioester dehydratase involved in fatty acid biosynthesis </td>
<td>AWRI3580_g894/HuHTD2 </td>
</tr>
<tr>
<td>HGUI_03531 </td>
<td>Predicted mitochondrial import inner membrane translocase subunit </td>
<td>AWRI3580_g2710/HuTIM54 </td>
</tr>
<tr>
<td>HGUI_02284 </td>
<td>Predicted to be involved in intracellular protein trafficking, similar to ScSss1 </td>
<td>AWRI3580_g3696/- </td>
</tr>
<tr>
<td>HGUI_03514 </td>
<td>Predicted UDP-N-acetylglucosamine transferase </td>
<td>AWRI3580_g742/HuALG14 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. opuntiae</em> but not in <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. opuntiae</em> orthologue</strong> </td>
</tr>
<tr>
<td>HGUI_02309 </td>
<td>Predicted chitin synthase export chaperone </td>
<td>AWRI3578_g694 </td>
</tr>
<tr>
<td>HGUI_03602 </td>
<td>Predicted vacuolar membrane Cu transporter </td>
<td>AWRI3578_g678 </td>
</tr>
<tr>
<td>HGUI_02076 </td>
<td>Cytochrome c oxidase assembly protein Cox19 </td>
<td>AWRI3578_g1039 </td>
</tr>
<tr>
<td>HGUI_01733 </td>
<td>Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3578_g4036 </td>
</tr>
<tr>
<td>HGUI_01849 </td>
<td>Predicted to be involved in meiosis, similar to ScSpo11 </td>
<td>AWRI3578_g1481 </td>
</tr>
<tr>
<td>HGUI_02598 </td>
<td>Mitochondrial import inner membrane translocase subunit TIM12 </td>
<td>AWRI3578_g3410 </td>
</tr>
<tr>
<td>HGUI_01086 </td>
<td>Mitochondrial protein required for assembly of cytochrome c oxidase, mitochondrial </td>
<td>AWRI3578_g916 </td>
</tr>
<tr>
<td>HGUI_02427 </td>
<td>Predicted to be involved in protein palmitoylation, similar to ScSwf1 </td>
<td>AWRI3578_g3718 </td>
</tr>
<tr>
<td>HGUI_00194 </td>
<td>Predicted transcriptional regulator of the Zn-finger family, shows some similarity with ScRim101 </td>
<td>AWRI3578_g3297 </td>
</tr>
<tr>
<td>HGUI_02287 </td>
<td>Putative 5-hydroxyisourate hydrolase </td>
<td>AWRI3578_g2657 </td>
</tr>
<tr>
<td>HGUI_00707 </td>
<td>Putative glucan endo-1,3-beta-glucosidase btgC </td>
<td>AWRI3578_g1650 </td>
</tr>
<tr>
<td>HGUI_01349 </td>
<td>Predicted GPI-anchored protein of unknown function </td>
<td>AWRI3578_g2558 </td>
</tr>
</tbody></table></div>
<div content-id="dsy039-T2" class="table-modal"><table><tbody>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins not having an orthologue in both <em>H. opuntiae</em> and <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>ORF name </td>
<td>Probable function </td>
<td>Best ortdologue </td>
</tr>
<tr>
<td>HGUI_01127 </td>
<td>BolA-like protein </td>
<td>HANVADRAFT_51108 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01165 </td>
<td>Ribosomal protein </td>
<td>HANVADRAFT_5002 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01445 </td>
<td>Predicted small nuclear ribonucleo protein </td>
<td>HANVADRAFT_11820 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_01959 </td>
<td>Predicted ribonucleo protein complex subunit 3 </td>
<td>HANVADRAFT_53037 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02479 </td>
<td>Unknown </td>
<td>HANVADRAFT_53473 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_02541 </td>
<td>Unknown </td>
<td>HANVADRAFT_53361 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03386 </td>
<td>Unknown </td>
<td>HANVADRAFT_53900 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03446 </td>
<td>Unknown </td>
<td>HANVADRAFT_23140 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_03596 </td>
<td>Predicted vacuolar ATPase assembly integral membrane protein </td>
<td>HANVADRAFT_22909 from <em>H. valbyensis</em> NRRL Y-1626 </td>
</tr>
<tr>
<td>HGUI_04019 </td>
<td>Minor spike protein H </td>
<td>CP02DC15_1109 from <em>Chlamydia psittaci</em> 02DC15 </td>
</tr>
<tr>
<td>HGUI_04020 </td>
<td>Replication-associated protein A </td>
<td>HMPREF1334_01983 from <em>Enterococcus faecalis</em> ERV41 </td>
</tr>
<tr>
<td>HGUI_04021 </td>
<td>External scaffolding protein D </td>
<td>M771_11055 from <em>Neisseria gonorrhoeae</em> MU_NG1 </td>
</tr>
<tr>
<td>HGUI_04022 </td>
<td>Capsid protein </td>
<td>M743_11545 from <em>Neisseria gonorrhoeae</em> NYC_2011_05_13 </td>
</tr>
<tr>
<td>HGUI_04023 </td>
<td>Major spike protein G </td>
<td>MPLA_2010001 from <em>Mesorhizobium sp</em>. ORS335 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. uvarum</em> but not in <em>H. opuntiae</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. uvarum</em> orthologue (in AWRI3580/in DSM2768)</strong> </td>
</tr>
<tr>
<td>HGUI_03532 </td>
<td>Predicted to be involved in endosome-to-Golgi intracellular trafficking </td>
<td>−/HuPEP8 </td>
</tr>
<tr>
<td>HGUI_00706 </td>
<td>Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3580_g2240/HuALG6 </td>
</tr>
<tr>
<td>HGUI_03286 </td>
<td>Dolichyl-diphosphooligosaccharide–protein glycosyltransferase </td>
<td>AWRI3580_g1764/HuOST2 </td>
</tr>
<tr>
<td>HGUI_03882 </td>
<td>Dol-P-Glc: Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase </td>
<td>AWRI3580_g1578/- </td>
</tr>
<tr>
<td>HGUI_01192 </td>
<td>Eukaryotic translation initiation factor eIF-1 </td>
<td>AWRI3580_g477/- </td>
</tr>
<tr>
<td>HGUI_00394 </td>
<td>Predicted hydroxyacyl-thioester dehydratase involved in fatty acid biosynthesis </td>
<td>AWRI3580_g894/HuHTD2 </td>
</tr>
<tr>
<td>HGUI_03531 </td>
<td>Predicted mitochondrial import inner membrane translocase subunit </td>
<td>AWRI3580_g2710/HuTIM54 </td>
</tr>
<tr>
<td>HGUI_02284 </td>
<td>Predicted to be involved in intracellular protein trafficking, similar to ScSss1 </td>
<td>AWRI3580_g3696/- </td>
</tr>
<tr>
<td>HGUI_03514 </td>
<td>Predicted UDP-N-acetylglucosamine transferase </td>
<td>AWRI3580_g742/HuALG14 </td>
</tr>
<tr>
<td colspan="2">
<strong><em>H. guilliermondii</em> UTAD222 proteins having an orthologue in <em>H. opuntiae</em> but not in <em>H. uvarum</em></strong> </td>
<td> </td>
</tr>
<tr>
<td>
<strong>ORF name</strong> </td>
<td>
<strong>Probable function</strong> </td>
<td>
<strong><em>H. opuntiae</em> orthologue</strong> </td>
</tr>
<tr>
<td>HGUI_02309 </td>
<td>Predicted chitin synthase export chaperone </td>
<td>AWRI3578_g694 </td>
</tr>
<tr>
<td>HGUI_03602 </td>
<td>Predicted vacuolar membrane Cu transporter </td>
<td>AWRI3578_g678 </td>
</tr>
<tr>
<td>HGUI_02076 </td>
<td>Cytochrome c oxidase assembly protein Cox19 </td>
<td>AWRI3578_g1039 </td>
</tr>
<tr>
<td>HGUI_01733 </td>
<td>Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase </td>
<td>AWRI3578_g4036 </td>
</tr>
<tr>
<td>HGUI_01849 </td>
<td>Predicted to be involved in meiosis, similar to ScSpo11 </td>
<td>AWRI3578_g1481 </td>
</tr>
<tr>
<td>HGUI_02598 </td>
<td>Mitochondrial import inner membrane translocase subunit TIM12 </td>
<td>AWRI3578_g3410 </td>
</tr>
<tr>
<td>HGUI_01086 </td>
<td>Mitochondrial protein required for assembly of cytochrome c oxidase, mitochondrial </td>
<td>AWRI3578_g916 </td>
</tr>
<tr>
<td>HGUI_02427 </td>
<td>Predicted to be involved in protein palmitoylation, similar to ScSwf1 </td>
<td>AWRI3578_g3718 </td>
</tr>
<tr>
<td>HGUI_00194 </td>
<td>Predicted transcriptional regulator of the Zn-finger family, shows some similarity with ScRim101 </td>
<td>AWRI3578_g3297 </td>
</tr>
<tr>
<td>HGUI_02287 </td>
<td>Putative 5-hydroxyisourate hydrolase </td>
<td>AWRI3578_g2657 </td>
</tr>
<tr>
<td>HGUI_00707 </td>
<td>Putative glucan endo-1,3-beta-glucosidase btgC </td>
<td>AWRI3578_g1650 </td>
</tr>
<tr>
<td>HGUI_01349 </td>
<td>Predicted GPI-anchored protein of unknown function </td>
<td>AWRI3578_g2558 </td>
</tr>
</tbody></table></div>
<div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>In this table, a selected set of proteins was chosen to illustrate those only found in <em>H. guilliermondii</em>, those found in <em>H. guilliermondii</em> and in <em>H. opuntiae</em> and those found in <em>H. guilliermondii</em> and <em>H. uvarum</em>. A more comprehensive list including all the results is provided in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p></div></div></div>
</div>
<p>Another aspect of relevance that emerged from the comparative analysis between the <em>H. guilliermondii</em>, <em>H. opuntiae</em> and <em>H. uvarum</em> proteomes was the identification of 26 proteins only shared by <em>H. guilliermondii</em> and <em>H. uvarum</em> and of 87 proteins only shared by <em>H. guilliermondii</em> and <em>H. opuntiae</em>. A full list of these proteins is provided in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s4.pdf?Expires=1576481646&amp;Signature=xmD7KbN6w9V1Nck6RqPJFJsYCM8W81TRckr8GzdsJS5-aE-5TVI2mCHpeTol6m0JLmCd4~hgpF-zT3P-KxHC2g7EIsas-f4DNr90r5-JaBRkT-uakvVJQnCEKheLdbZ0U698g~gJfQHGoB~EOAf2p3-NrcAIcb7x75esWKBXl5c3LlX~iNfuToQgqS85JEGkvZgv8roFKrVD7qSvrtaSkSjml6w9rQHV5Bwwb5qjea5c-vu0zNxikMxZD~yDT386CKDRykXORhYzH~17oXC3grGXq2gGnq-Qr8mp4SmEMM5ZyK8stSPdwhdMIZLPpNBHP3W~ojNm-ELOXLKi1ZxwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> and a selected set in <a reveal-id="dsy039-T2" data-open="dsy039-T2" class="link link-reveal link-table xref-fig">Table 2</a>. From the functional point of view, the proteins apparently absent in <em>H. opuntiae</em> include an enzyme predicted to be involved in fatty acid biosynthesis (HGUI_00394), several proteins involved in post-translational modifications (HGUI_03286, HGUI_03882, HGUI_03882, HGUI_03514) and in intracellular protein trafficking (HGUI_03532, HGUI_03531, HGUI_02284) (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s3.xlsx?Expires=1576481646&amp;Signature=gOGzhwkfzYhZhqwxGLarymUfQDqvQNKl~cDwFencFF5QT6yTGJLYfMPD8OEL5MqbF5TYWFxJJ0q0fWOzq7P66UKmGv7~v-aU6MY47yAC1DEEpH7eBzNSihZ~v1e8FWb~AEDb9H0nr~flWVBcxYvbC788yeyHwmKrOONRSCXBBAcRNlcURilczLbLd5IRm8kCjiz5-k8Gurlxhp-YffN6WHb1BBOwOlWSEYxfFu0NLgh3J9svjTOzbHVQQMrwXfz-izYLF1IKFltc5K~Xh3W3nRGTGzXZDTkIMcb0K65iQcDKs~ayzIJkpFgP6igCR1SPIRYuhmcvNx0rz8FZdOKKwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> and <a reveal-id="dsy039-T2" data-open="dsy039-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Within the data set of proteins apparently absent in <em>H. uvarum</em>, it was possible to identify an homologue of a putative vacuolar membrane copper transporter (HGUI_03602), two proteins involved in the assembly of cytochrome c oxidase (HGUI_02076, HGUI_01086) and a putative β-glucosidase (HGUI_00707). Considering the lack of information concerning the functionality of these proteins, it is hard to link the presence/absence of these proteins with physiological traits of the different <em>Hanseniaspora</em> species, although this is certainly a topic deserving further attention.</p>                    <h3 scrollto-destination="131979879" id="131979879" class="section-title" data-legacy-id="SEC3.3.3">3.3.3. Comparative analysis of the predicted proteomes of <em>Hanseniaspora</em> spp. and <em>S. cerevisiae</em>: emphasis on genes associated with tolerance to oenological relevant stresses</h3>
<p><em>Hanseniaspora</em> spp. are known to have multiple phenotypic traits different from those observed in <em>S. cerevisiae</em> wine strains (e.g. different fermentation rates and ability to use terpene-like molecules for the production of aroma compounds)<a reveal-id="dsy039-B4" data-open="dsy039-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and thus, we have hypothesized whether this could result from divergences in the proteomes of the different species. To scrutinize this, the set of <em>H. guilliermondii</em> UTAD222 predicted proteins was compared with those described for the <em>S. cerevisiae</em> wine strain EC1118, widely used for wine-making and having a well annotated genomic sequence.<a reveal-id="dsy039-B74" data-open="dsy039-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> The results of this <em>in silico</em> proteomic analysis are detailed in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S6</a></span>. Approximately 1,294 <em>H. guilliermondii</em> proteins had no robust homologue in the <em>S. cerevisiae</em> strains examined (listed in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). This could occur either because the <em>S. cerevisiae</em> strains do not encode these proteins or because the proteins with a corresponding function are highly divergent from those one found in <em>H. guilliermondii</em>. These <em>H. guilliermondii</em> proteins that had no orthologue in <em>S. cerevisiae</em> were present in <em>H. uvarum</em> and in <em>H. opuntiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576481646&amp;Signature=IZIM9DIuUXi-cfm6LsoAakVe~sVnmJ78b-2IK-0rycS6h9TLHzYyX6ICSg3WMCU3CkexlEwu~BNtCWVnDmI8dlD3amX7px8JyeOYSDcrenWH3vfWJi8iZg2Lc6xeMXS~yuRrTmstNMmIfcoh81I0kxNgsOqyeag-uOs9dGMuo-6tn~ZVn6Nss1d6T-xOhIfca3-I0d2ec8Ux5KXTgGj6tXygZ~kxHBD~WJZjiNUJibu7NGh-c67298Lz8IWnUQLAyByiTAZjJqVq-glLEpga~v74IzmBEfdLwQHakN-8R~1QCWP0gFxkVJLbFqMWbeWaZsvF1KmHUPk9z9h7GMJmnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), suggesting that they comprise specific proteins of the <em>Hanseniaspora</em> genus. The majority of these ‘<em>Hanseniaspora</em>-specific’ proteins (879) have a poorly characterized function but we could identify β-glucosidases (e.g. HGUI_02084, HGUI_02647 and HGUI_02781), flocculins (e.g. HGUI_01978, HGUI_03264, HGUI_04053, HGUI_01804, HGUI_03963 and HGUI_03989), proteins involved in transcriptional regulation and in translation, as well as proteins involved in respiration and in metabolism of various carbohydrate and nitrogen compounds (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The identification of β-glucosidase-encoding genes within the set of the putative ‘<em>Hanseniaspora</em>-specific’ genes was expected considering the demonstration of the influence of members of this species, including <em>H. guilliermondii</em>, in increasing the amount of terpenes present in wines;<a reveal-id="dsy039-B75" data-open="dsy039-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> a phenotypic trait that is absent in <em>S. cerevisiae</em> strains.<a reveal-id="dsy039-B76" data-open="dsy039-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> It was also interesting the emergence in this comparative analysis of several enzymes predicted to be involved in metabolism of tyramine, histamine, histidine or tyrosine (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576481646&amp;Signature=oKHtD47JslPTxPKTqVYjv1Qr7UmjkF8wzphO7682EuF3FfYdmtGQ1iINNGRXgGfCRL7tQ15SLkbQDi9EgAR4NycS2~inclKp99EOfh7h3pyj5I~fNxVDqpaV1JrRP5mZaMZ8auavbB66dCF3PFqdLDSR~k0GG3JTEPPnvkG~nY5CR4J-pLZTyeZL5hcP11FP8FvErAOZI2ynuVAgiMcNd2Fdr0gfJ4l0DPx8Z9Oo5BXyTta9ovUFInOmdv1WsfQKLLH3mynNZ1I2RHHLwrvq-OS0ZZthDj9~kiq~PCmWexoiZUU6JqO-0WyIirnZHjXcYuQkjAoZlKxus1AxmZqwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), these being biogenic amines that are known to be present in grape-musts and whose content increases along alcoholic fermentation.<a reveal-id="dsy039-B58" data-open="dsy039-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsy039-B77" data-open="dsy039-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> This observation suggests that the presence of <em>H. guilliermondii</em> (and also <em>H. uvarum</em> and <em>H. opuntiae</em>) may play a role in controlling the levels of tyramine and histamine obtained in wines counter-acting, at least in part, the production prompted by <em>S. cerevisiae</em> and other microbiota present in musts.<a reveal-id="dsy039-B58" data-open="dsy039-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Consistent with this idea, previous studies have reported a decrease in the level of biogenic amines produced during the early stages of wine fermentations conducted in the presence of <em>Kloeckera apiculata</em>, the anamorph of <em>H. uvarum</em>, this effect being alleviated as the fermentation proceeds and the abundance of this non-<em>Saccharomyces</em> yeast is reduced.<a reveal-id="dsy039-B78" data-open="dsy039-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> This is a particular relevant aspect since the level of tyramine and histamine is a critical parameter determining wine quality and consumers’ health and recommended levels for the amount of these compounds that can be present in wine has been established in a variety of countries.<a reveal-id="dsy039-B58" data-open="dsy039-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a></p>
<p>The comparative analysis performed also led to the identification of about 2,797 <em>S. cerevisiae</em> proteins for which we could not identify an orthologue in the predicted ORFeomes of <em>H. guilliermondii</em>, <em>H. uvarum</em> or <em>H. opuntiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). This data set of <em>S. cerevisiae</em> proteins absent in <em>Hanseniaspora</em> is enriched in proteins involved in carbohydrate metabolism, in nitrogen metabolism, in cell cycle, in DNA processing, in RNA synthesis, in transcriptional regulation, in protein synthesis and in intracellular trafficking (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Consistent with the results of the metabolic reconstruction described earlier, among the <em>S. cerevisae</em> genes missing in the <em>Hanseniaspora</em> species were <em>ICL1</em> encoding isocitrate lyase, <em>PCK1</em>, encoding phosphoenolpyruvate carboxykinase; <em>FBP1</em>, encoding fructose-1,6-biphosphatase, as well as the genes involved in biosynthesis of thiamine, niacin, biotin or polyamines (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Among the large set of <em>S. cerevisiae</em> proteins involved in transcriptional regulation that were not found in <em>H. guilliermondii</em> were Msn2 and Msn4 transcription factors, essential for control of environmental stress response in the budding yeast<a reveal-id="dsy039-B79" data-open="dsy039-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> (<a reveal-id="dsy039-F7" data-open="dsy039-F7" class="link link-reveal link-table xref-fig">Fig. 7</a> and <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). No protein showing similarity to ScMsn2 or ScMsn4 could also be identified in the genome of the other <em>Hanseniaspora</em> species having a genome disclosed indicating that this is a feature of the genus and not specific of the <em>H. guilliermondii</em> species. This observation is intriguing as it leaves open the elucidation of the mechanisms by which <em>Hanseniaspora</em> spp respond to environmental stress, specially taking into account that robust homologues of Msn2 and of its target genes had been identified in the genome of other wine Non-Saccharomyces species such as <em>Z. bailii.</em><a reveal-id="dsy039-B80" data-open="dsy039-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> The existence of a general response to stress showing similarities to the one described in the budding yeast has been characterized in distant fungal species including <em>Lachancea kluyveri</em>, <em>Candida albicans</em> or <em>Schizosaccharomyces pombe</em>, albeit this transcriptional response was found to be mediated by regulators having little homology with ScMsn2.<a reveal-id="dsy039-B81 dsy039-B82 dsy039-B83" data-open="dsy039-B81 dsy039-B82 dsy039-B83" class="link link-ref link-reveal xref-bibr"><sup>81–83</sup></a> Thus, the absence of Msn2/Msn4 orthologues within the <em>Hanseniaspora</em> genus does not <em>per se</em> means that these yeasts will not be equipped with a general stress response. </p>                        <a id="131979882" scrollto-destination="131979882"></a>
<div id="dsy039-F7" data-id="dsy039-F7" class="fig fig-section">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f7.png?Expires=1576481646&amp;Signature=WxSR3mK7WIoZ0ZcMH5N99EQD9Xkn6~xgxtcJ3KHfOnsH9BvxDeBoSlyYxTHh94af88~GlXiMKSHexIafUhYPsccGSzHENiWhBlwxbDM66xi-VVRJmUk3Ak~pKmp83-wpcT3T3o~TWbDvJ0NZi0kGMBpUYEWpPsFGaUZ8ge37gV4GLaVgZ5zG1fspr06-Jrsx~sTzkS4hM6qZaToVd1LtV7-aeT6iEc7KyUeTvLRtQ2qDkbh1LvN42tZnpKm5Z9aOytQi-UVZc~9nl9e7gk~2fuRamNMUiOJEx4qguJiyy4t1hXa~jRoHy1n1XyUSq0nEtKKXwa0EJDp3mLPhlQl7VA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conservation in the ORFeome of H. guilliermondii UTAD222 of S. cerevisiae genes identified to be required for tolerance of this species against relevant oenological stresses. Based on results from large-scale phenotypic screenings the S. cerevisiae genes required for tolerance against high concentrations of glucose or ethanol were identified, as well, as the genes required for maximal fermentation in grape-juice medium (also called the ‘fermentome’). The predicted ORFeome of H. guilliermondii UTAD222 was searched for orthologues of these determinants of stress tolerance being the number of conserved genes identified in each horizontal bar. A selected set of the conserved and non-conserved genes is also shown while the full result is available in Supplementary Table S6. Since Hanseniaspora are highly tolerant to high-glucose concentrations it was not considered of interest to specify the set of non-conserved genes. Saccharomyces cerevisiae genes mediating multidrug resistance are underlined. *Although Ssu1 was not identified within the set of genes belonging to the fermentome, the absence of this sulphite efflux pump in the Hanseniaspora species is also highlighted in the figure since tolerance to SO2 is generally accepted as a highly relevant trait for oenological strains." path-from-xml="dsy039f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979882/dsy039f7.tif" path-from-xml="dsy039f7.tif" target="_blank">Open in new tab</a><a section="131979882" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f7.png?Expires=1576481646&amp;Signature=gmXd4IV22QofS0r-J8SzHz6aeBp9RWnruDT3Xdbu0ZG~xREnYsnvtUtFEAb-8dokheeJscNQN-r42frnbZnLCk-sXT9T~R6xRNk7hXUn53vGisFHjp4zPv~lIb2fskQmsUleH-09yZJ-5papuX5bSgWHvp2ccL3OCfFiucamwSKvSITm8dDbzdmiN9-Ibnzk40iQJFFx~bFd3MmL3bSl9OvEH1X62NijL2~Z3FXd7DmkE7s2tFVWPZvIszqqoyiYxyCKLUM1youJE90i3GQxR2kLQ2V9PoCtG0K7djJgCiT-k~zWVElRw87~HW~6e8SXgyY1B97s9SmYVR2uLCfn0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979882&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Conservation in the ORFeome of <em>H. guilliermondii</em> UTAD222 of <em>S. cerevisiae</em> genes identified to be required for tolerance of this species against relevant oenological stresses. Based on results from large-scale phenotypic screenings the <em>S. cerevisiae</em> genes required for tolerance against high concentrations of glucose or ethanol were identified, as well, as the genes required for maximal fermentation in grape-juice medium (also called the ‘fermentome’). The predicted ORFeome of <em>H. guilliermondii</em> UTAD222 was searched for orthologues of these determinants of stress tolerance being the number of conserved genes identified in each horizontal bar. A selected set of the conserved and non-conserved genes is also shown while the full result is available in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>. Since <em>Hanseniaspora</em> are highly tolerant to high-glucose concentrations it was not considered of interest to specify the set of non-conserved genes. <em>Saccharomyces cerevisiae</em> genes mediating multidrug resistance are underlined. *Although Ssu1 was not identified within the set of genes belonging to the fermentome, the absence of this sulphite efflux pump in the <em>Hanseniaspora</em> species is also highlighted in the figure since tolerance to SO<sub>2</sub> is generally accepted as a highly relevant trait for oenological strains.</p></div>
</div>
<div content-id="dsy039-F7" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/m_dsy039f7.png?Expires=1576481646&amp;Signature=WxSR3mK7WIoZ0ZcMH5N99EQD9Xkn6~xgxtcJ3KHfOnsH9BvxDeBoSlyYxTHh94af88~GlXiMKSHexIafUhYPsccGSzHENiWhBlwxbDM66xi-VVRJmUk3Ak~pKmp83-wpcT3T3o~TWbDvJ0NZi0kGMBpUYEWpPsFGaUZ8ge37gV4GLaVgZ5zG1fspr06-Jrsx~sTzkS4hM6qZaToVd1LtV7-aeT6iEc7KyUeTvLRtQ2qDkbh1LvN42tZnpKm5Z9aOytQi-UVZc~9nl9e7gk~2fuRamNMUiOJEx4qguJiyy4t1hXa~jRoHy1n1XyUSq0nEtKKXwa0EJDp3mLPhlQl7VA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conservation in the ORFeome of H. guilliermondii UTAD222 of S. cerevisiae genes identified to be required for tolerance of this species against relevant oenological stresses. Based on results from large-scale phenotypic screenings the S. cerevisiae genes required for tolerance against high concentrations of glucose or ethanol were identified, as well, as the genes required for maximal fermentation in grape-juice medium (also called the ‘fermentome’). The predicted ORFeome of H. guilliermondii UTAD222 was searched for orthologues of these determinants of stress tolerance being the number of conserved genes identified in each horizontal bar. A selected set of the conserved and non-conserved genes is also shown while the full result is available in Supplementary Table S6. Since Hanseniaspora are highly tolerant to high-glucose concentrations it was not considered of interest to specify the set of non-conserved genes. Saccharomyces cerevisiae genes mediating multidrug resistance are underlined. *Although Ssu1 was not identified within the set of genes belonging to the fermentome, the absence of this sulphite efflux pump in the Hanseniaspora species is also highlighted in the figure since tolerance to SO2 is generally accepted as a highly relevant trait for oenological strains." path-from-xml="dsy039f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/131979882/dsy039f7.tif" path-from-xml="dsy039f7.tif" target="_blank">Open in new tab</a><a section="131979882" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039f7.png?Expires=1576481646&amp;Signature=gmXd4IV22QofS0r-J8SzHz6aeBp9RWnruDT3Xdbu0ZG~xREnYsnvtUtFEAb-8dokheeJscNQN-r42frnbZnLCk-sXT9T~R6xRNk7hXUn53vGisFHjp4zPv~lIb2fskQmsUleH-09yZJ-5papuX5bSgWHvp2ccL3OCfFiucamwSKvSITm8dDbzdmiN9-Ibnzk40iQJFFx~bFd3MmL3bSl9OvEH1X62NijL2~Z3FXd7DmkE7s2tFVWPZvIszqqoyiYxyCKLUM1youJE90i3GQxR2kLQ2V9PoCtG0K7djJgCiT-k~zWVElRw87~HW~6e8SXgyY1B97s9SmYVR2uLCfn0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=131979882&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy039f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Conservation in the ORFeome of <em>H. guilliermondii</em> UTAD222 of <em>S. cerevisiae</em> genes identified to be required for tolerance of this species against relevant oenological stresses. Based on results from large-scale phenotypic screenings the <em>S. cerevisiae</em> genes required for tolerance against high concentrations of glucose or ethanol were identified, as well, as the genes required for maximal fermentation in grape-juice medium (also called the ‘fermentome’). The predicted ORFeome of <em>H. guilliermondii</em> UTAD222 was searched for orthologues of these determinants of stress tolerance being the number of conserved genes identified in each horizontal bar. A selected set of the conserved and non-conserved genes is also shown while the full result is available in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>. Since <em>Hanseniaspora</em> are highly tolerant to high-glucose concentrations it was not considered of interest to specify the set of non-conserved genes. <em>Saccharomyces cerevisiae</em> genes mediating multidrug resistance are underlined. *Although Ssu1 was not identified within the set of genes belonging to the fermentome, the absence of this sulphite efflux pump in the <em>Hanseniaspora</em> species is also highlighted in the figure since tolerance to SO<sub>2</sub> is generally accepted as a highly relevant trait for oenological strains.</p></div>
</div>
<p>Taking advantage of the wide panoply of genetic tools available for <em>S. cerevisiae</em>, large-scale phenotypic screenings have been conducted to identify key players involved in tolerance to oenological relevant stresses including tolerance to ethanol<a reveal-id="dsy039-B84" data-open="dsy039-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a><sup>,</sup><a reveal-id="dsy039-B85" data-open="dsy039-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> or to high glucose concentrations.<a reveal-id="dsy039-B86" data-open="dsy039-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a><sup>,</sup><a reveal-id="dsy039-B87" data-open="dsy039-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> More recently, the <em>S. cerevisiae</em> genes required for fermentation in grape juice simulated medium, generally known as the ‘fermentome’, were also identified using similar chemogenomics screenings.<a reveal-id="dsy039-B88" data-open="dsy039-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a><sup>,</sup><a reveal-id="dsy039-B89" data-open="dsy039-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a> Only half of the <em>S. cerevisae</em> genes required for fermentation in grape-simulated medium (48 in total of 90 genes) were found to have robust homologues in <em>H. guilliermondii</em> and in <em>H. uvarum</em> or <em>H. opuntiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s7.xlsx?Expires=1576481646&amp;Signature=u2Y5OCp3sIxNlaVkyy-Z~K2sUYzXgM9V80QXQL9Nb~5ucSKUvPSZLyp5loeLPE28hVHOPD~ucdGNjH5tc6~n9~t7CirV5v6bXNta-x~WZBwkq8h5mkN-gt-URjBnbNyaZI0dz~I5ef3wAl~Ukl-6skjyApWnRSm473vdqgTi9VAcokHt8H~eCenWA2VRaj-XwMCH~m-DbsGaEdl0m5IkL6DrrezvLisQmLPEFIFdiRbuUssFL6hPedgVhacNJJa93o-O-3OLZ9Y9KyLsopwH5wjMx4NoUUnJVrPB303rIJfZdPmiP8OCRhT24oLoL6oXX8ZJDjBTPWpqFYIGKz~4hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>), while 300 ethanol-resistance and 155 ‘high-glucose’ resistance genes were identified in <em>H. guilliermondii</em> UTAD222. Similar numbers were also obtained for <em>H. uvarum</em> or <em>H. opuntiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Among the genes found to be conserved in the <em>Hanseniaspora</em> species is the protein kinase Hog1 (<a reveal-id="dsy039-F7" data-open="dsy039-F7" class="link link-reveal link-table xref-fig">Fig. 7</a> and <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>) as well as most members of the HOG-pathway indicating that this signalling pathway, essential for <em>S. cerevisiae</em> response to stress induced by vinification,<a reveal-id="dsy039-B89" data-open="dsy039-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a> is conserved in the <em>Hanseniaspora</em> genus. Despite this, we could not identify orthologues within the <em>Hanseniaspora</em> species similar to ScMsn2/ScMsn4, ScHot1 or ScSko1, the known mediators of Hog1-dependent response (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576481646&amp;Signature=azb-BIw5NCeAOHuwjzZUNmEGNEsssE-pAm5RvAY-XMasOw0UhQkmqttVjt-brIwLk02Q0sjETT5zgipWT6~2FYsaApPgLjqv30js788NYG2ks7SMNWDB6N8~YSVkzD92OwtEU72uUc3AU~jUTUH85brvuNnv7yXnHEyy6H1~jJT3jSh~Ncjn4g1~z77njX8SoFLpTWbzwHDwXD40OHxObhxjGwLkXwoEd85BCqMNU0mD7GOvUNXn4oK158M1vo27YbqCnhigC9vrWppi4l0dM80r17xFX~0GiyPrJxKRCZlmdqMrZ~Vf7EQGNWa5eyqbmblOE1sW2UrpfD86sPqa8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Proteins involved in the assembly and function of the vacuolar ATPase, also identified for <em>S. cerevisae</em> tolerance to vinification-stress, comprised another group of proteins for which we could not identify orthologues in <em>H. guilliermondii</em> (<a reveal-id="dsy039-F7" data-open="dsy039-F7" class="link link-reveal link-table xref-fig">Fig. 7</a> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s7.xlsx?Expires=1576481646&amp;Signature=u2Y5OCp3sIxNlaVkyy-Z~K2sUYzXgM9V80QXQL9Nb~5ucSKUvPSZLyp5loeLPE28hVHOPD~ucdGNjH5tc6~n9~t7CirV5v6bXNta-x~WZBwkq8h5mkN-gt-URjBnbNyaZI0dz~I5ef3wAl~Ukl-6skjyApWnRSm473vdqgTi9VAcokHt8H~eCenWA2VRaj-XwMCH~m-DbsGaEdl0m5IkL6DrrezvLisQmLPEFIFdiRbuUssFL6hPedgVhacNJJa93o-O-3OLZ9Y9KyLsopwH5wjMx4NoUUnJVrPB303rIJfZdPmiP8OCRhT24oLoL6oXX8ZJDjBTPWpqFYIGKz~4hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Among the <em>S. cerevisiae</em> ethanol-resistance genes that had no orthologue in <em>H. guilliermondii</em> are several proteins with peroxisomal function, as well as proteins of the respiratory chain and involved in intracellular protein trafficking (<a reveal-id="dsy039-F7" data-open="dsy039-F7" class="link link-reveal link-table xref-fig">Fig. 7</a> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s7.xlsx?Expires=1576481646&amp;Signature=u2Y5OCp3sIxNlaVkyy-Z~K2sUYzXgM9V80QXQL9Nb~5ucSKUvPSZLyp5loeLPE28hVHOPD~ucdGNjH5tc6~n9~t7CirV5v6bXNta-x~WZBwkq8h5mkN-gt-URjBnbNyaZI0dz~I5ef3wAl~Ukl-6skjyApWnRSm473vdqgTi9VAcokHt8H~eCenWA2VRaj-XwMCH~m-DbsGaEdl0m5IkL6DrrezvLisQmLPEFIFdiRbuUssFL6hPedgVhacNJJa93o-O-3OLZ9Y9KyLsopwH5wjMx4NoUUnJVrPB303rIJfZdPmiP8OCRhT24oLoL6oXX8ZJDjBTPWpqFYIGKz~4hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). It remains to be established if the absence of these genes and eventually of others mediating tolerance to ethanol in <em>S. cerevisae</em> contributes for the reduced tolerance of <em>Hanseniaspora</em> species to stressful concentrations of ethanol. An interesting observation that emerged from this comparative analysis was the absence within the three <em>Hanseniaspora</em> species examined of an orthologue for the sulphite efflux pump Ssu1, an essential determinant of <em>S. cerevisiae</em> tolerance to SO<sub>2</sub>, which is in line with the observed low resilience of these species to SO<sub>2</sub>-stress in the phenotypic screening shown in <a reveal-id="dsy039-F2" data-open="dsy039-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and also reported before.<a reveal-id="dsy039-B8" data-open="dsy039-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p>
<p>Overall, the comparative genomic analysis herein performed shows significant differences in the way by which <em>S. cerevisiae</em> and <em>Hanseniaspora</em> spp respond to environmental stress and, in particular, in the response to relevant stresses in the context of wine fermentation with emphasis on the toxicity imposed by high concentrations of ethanol. The release of the genomic sequence of <em>H. guilliermondii</em> as well as the annotation and the comparative analysis herein presented also focusing <em>H. opuntiae</em> and <em>H. uvarum</em> is expected to contribute for a better elucidation of the physiology and biology of these species, essentially by boosting gene and genomic functional analyses. Necessarily, the results of this research will foster the design (or re-design) of <em>Hanseniaspora</em> strains to be used not only in wine-making but also in other relevant biotechnological industrial processes, such as production of aroma compounds, where these species have been identified as potentially interesting cell-factories.<a reveal-id="dsy039-B90" data-open="dsy039-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a><sup>,</sup><a reveal-id="dsy039-B91" data-open="dsy039-B91" class="link link-ref link-reveal xref-bibr"><sup>91</sup></a></p>                    <h2 scrollto-destination="131979885" id="131979885" class="backacknowledgements-title" data-legacy-id="Ack1">Acknowledgements</h2>
<p>This work was financed by FEDER through POCI-COMPETE 2020 and by Fundação para a Ciência e Tecnologia (BI/PTDC/AGR-TEC/3315/2014_2016, Project SMARTWINE-‘Smarter win fermentations: integrating omic-tools for development of novel mixed-starter cultures for tailoring wine production’), and supported by FCT to Biosystems and Integrative Sciences Institute (BioISI; FCT/UID/Multi/04046/2013) and to iBB-Institute for Bioengineering and Biosciences (through contract UID/BIO/04565/2013). Programa Operacional Regional de Lisboa 2020 is also acknowledged for its financial support to iBB (project no. 007317). I.S. is recipient of a PhD grant funded by FCT with the reference SFRH/BD/122200/2016. The authors also thank the technical support of Filipa Antunes Silva in PFGE analysis of <em>H. guilliermondii</em> strains.</p>                    <h3 scrollto-destination="131979887" id="131979887" class="section-title">Accession numbers</h3>
<p>26S DNA sequences of Hanseaniaspora isoaltes used in this study were submitted to GenBank under the accession numbers <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MG832576">MG832576</a> to <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MG832583">MG832583</a>, <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MG877743">MG877743</a> and <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/MG877743">MG87744</a>. The genome sequence of the <em>H. guilliermondii</em><a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/UTAD222">UTAD222</a> sequence has been deposited in ENA (accession numbers <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/FQNF01000001">FQNF01000001</a> to <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/FQNF01000208">FQNF01000208</a>).</p>                    <h3 scrollto-destination="131979889" id="131979889" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="131979891" id="131979891" class="backreferences-title" data-legacy-id="Ref1">References</h2>
<div class="ref-list">
<div content-id="dsy039-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bokulich</div> <div class="given-names">N. A.</div>
</div>, <div class="name string-name">
<div class="surname">Collins</div> <div class="given-names">T. S.</div>
</div>, <div class="name string-name">
<div class="surname">Masarweh</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Associations among wine grape microbiome, metabolome, and fermentation behavior suggest microbial contribution to regional wine characteristics</div>, <div class="source">MBio</div> , <div class="volume">7</div>, <div class="fpage">e00631</div>–<div class="lpage">16</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Associations%20among%20wine%20grape%20microbiome%2C%20metabolome%2C%20and%20fermentation%20behavior%20suggest%20microbial%20contribution%20to%20regional%20wine%20characteristics&amp;author=N.%20A.%20Bokulich&amp;author=T.%20S.%20Collins&amp;author=C.%20Masarweh&amp;publication_year=2016&amp;journal=MBio&amp;volume=7&amp;pages=e00631-16" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/mBio.00631-16" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FmBio.00631-16" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FmBio.00631-16"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27302757" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Associations%20among%20wine%20grape%20microbiome%2C%20metabolome%2C%20and%20fermentation%20behavior%20suggest%20microbial%20contribution%20to%20regional%20wine%20characteristics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bokulich</div> <div class="given-names">N. A.</div>
</div>, <div class="name string-name">
<div class="surname">Thorngate</div> <div class="given-names">J. H.</div>
</div>, <div class="name string-name">
<div class="surname">Richardson</div> <div class="given-names">P. M.</div>
</div>, <div class="name string-name">
<div class="surname">Mills</div> <div class="given-names">D. A.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Microbial biogeography of wine grapes is conditioned by cultivar, vintage, and climate</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> , <div class="volume">111</div>, <div class="fpage">E139</div>–<div class="lpage">48</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Microbial%20biogeography%20of%20wine%20grapes%20is%20conditioned%20by%20cultivar%2C%20vintage%2C%20and%20climate&amp;author=N.%20A.%20Bokulich&amp;author=J.%20H.%20Thorngate&amp;author=P.%20M.%20Richardson&amp;author=D.%20A.%20Mills&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=111&amp;pages=E139-48" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1317377110" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1317377110" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1317377110"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24277822" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Microbial%20biogeography%20of%20wine%20grapes%20is%20conditioned%20by%20cultivar%2C%20vintage%2C%20and%20climate&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Capozzi</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Garofalo</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Chiriatti</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Grieco</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Spano</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Microbial terroir and food innovation: the case of yeast biodiversity in wine</div>, <div class="source">Microbiol. Res</div> ., <div class="volume">181</div>, <div class="fpage">75</div>–<div class="lpage">83</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Microbial%20terroir%20and%20food%20innovation%3A%20the%20case%20of%20yeast%20biodiversity%20in%20wine&amp;author=V.%20Capozzi&amp;author=C.%20Garofalo&amp;author=M.%20A.%20Chiriatti&amp;author=F.%20Grieco&amp;author=G.%20Spano&amp;publication_year=2015&amp;journal=Microbiol.%20Res&amp;volume=181&amp;pages=75-83" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.micres.2015.10.005" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.micres.2015.10.005" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.micres.2015.10.005"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26521127" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Microbial%20terroir%20and%20food%20innovation%3A%20the%20case%20of%20yeast%20biodiversity%20in%20wine&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jolly</div> <div class="given-names">N. P.</div>
</div>, <div class="name string-name">
<div class="surname">Varela</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Pretorius</div> <div class="given-names">I. S.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Not your ordinary yeast: non-<em>Saccharomyces</em> yeasts in wine production uncovered</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">14</div>, <div class="fpage">215</div>–<div class="lpage">37</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Not%20your%20ordinary%20yeast%3A%20non-Saccharomyces%20yeasts%20in%20wine%20production%20uncovered&amp;author=N.%20P.%20Jolly&amp;author=C.%20Varela&amp;author=I.%20S.%20Pretorius&amp;publication_year=2014&amp;journal=FEMS%20Yeast%20Res&amp;volume=14&amp;pages=215-37" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/fyr.2014.14.issue-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2014.14.issue-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffyr.2014.14.issue-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24164726" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Not%20your%20ordinary%20yeast%3A%20non-Saccharomyces%20yeasts%20in%20wine%20production%20uncovered&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Belda</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Ruiz</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Esteban-Fernandez</div> <div class="given-names">A.</div>
</div>
</div>, et al. <div class="year">2017</div>, 
<div class="article-title">Microbial contribution to wine aroma and its intended use for wine quality improvement</div>, <div class="source">Molecules</div> , <div class="volume">22</div>, <div class="fpage">189</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Microbial%20contribution%20to%20wine%20aroma%20and%20its%20intended%20use%20for%20wine%20quality%20improvement&amp;author=I.%20Belda&amp;author=J.%20Ruiz&amp;author=A.%20Esteban-Fernandez&amp;publication_year=2017&amp;journal=Molecules&amp;volume=22&amp;pages=189" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3390/molecules22020189" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fmolecules22020189" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3390%2Fmolecules22020189"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Microbial%20contribution%20to%20wine%20aroma%20and%20its%20intended%20use%20for%20wine%20quality%20improvement&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Garcia-Fernandez</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Mas</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Esteve-Zarzoso</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Fungal diversity in grape must and wine fermentation assessed by massive sequencing, quantitative PCR and DGGE</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">1156</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fungal%20diversity%20in%20grape%20must%20and%20wine%20fermentation%20assessed%20by%20massive%20sequencing%2C%20quantitative%20PCR%20and%20DGGE&amp;author=C.%20Wang&amp;author=D.%20Garcia-Fernandez&amp;author=A.%20Mas&amp;author=B.%20Esteve-Zarzoso&amp;publication_year=2015&amp;journal=Front.%20Microbiol&amp;volume=6&amp;pages=1156" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26557110" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Fungal%20diversity%20in%20grape%20must%20and%20wine%20fermentation%20assessed%20by%20massive%20sequencing%2C%20quantitative%20PCR%20and%20DGGE&amp;aulast=Wang&amp;date=2015&amp;spage=1156&amp;volume=6" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fungal%20diversity%20in%20grape%20must%20and%20wine%20fermentation%20assessed%20by%20massive%20sequencing%2C%20quantitative%20PCR%20and%20DGGE&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zott</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Miot-Sertier</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Claisse</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Lonvaud-Funel</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Masneuf-Pomarede</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Dynamics and diversity of non-Saccharomyces yeasts during the early stages in winemaking</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">125</div>, <div class="fpage">197</div>–<div class="lpage">203</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dynamics%20and%20diversity%20of%20non-Saccharomyces%20yeasts%20during%20the%20early%20stages%20in%20winemaking&amp;author=K.%20Zott&amp;author=C.%20Miot-Sertier&amp;author=O.%20Claisse&amp;author=A.%20Lonvaud-Funel&amp;author=I.%20Masneuf-Pomarede&amp;publication_year=2008&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=125&amp;pages=197-203" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijfoodmicro.2008.04.001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2008.04.001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijfoodmicro.2008.04.001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18495281" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dynamics%20and%20diversity%20of%20non-Saccharomyces%20yeasts%20during%20the%20early%20stages%20in%20winemaking&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grangeteau</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Gerhards</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Rousseaux</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">von Wallbrunn</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Alexandre</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Guilloux-Benatier</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Diversity of yeast strains of the genus Hanseniaspora in the winery environment: what is their involvement in grape must fermentation?</div> <div class="source">Food Microbiol</div> ., <div class="volume">50</div>, <div class="fpage">70</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20yeast%20strains%20of%20the%20genus%20Hanseniaspora%20in%20the%20winery%20environment%3A%20what%20is%20their%20involvement%20in%20grape%20must%20fermentation%3F&amp;author=C.%20Grangeteau&amp;author=D.%20Gerhards&amp;author=S.%20Rousseaux&amp;author=C.%20von%20Wallbrunn&amp;author=H.%20Alexandre&amp;author=M.%20Guilloux-Benatier&amp;publication_year=2015&amp;journal=Food%20Microbiol&amp;volume=50&amp;pages=70-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.fm.2015.03.009" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fm.2015.03.009" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.fm.2015.03.009"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25998817" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Diversity%20of%20yeast%20strains%20of%20the%20genus%20Hanseniaspora%20in%20the%20winery%20environment%3A%20what%20is%20their%20involvement%20in%20grape%20must%20fermentation%3F&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sternes</div> <div class="given-names">P. R.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Kutyna</div> <div class="given-names">D. R.</div>
</div>, <div class="name string-name">
<div class="surname">Borneman</div> <div class="given-names">A. R.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">A combined meta-barcoding and shotgun metagenomic analysis of spontaneous wine fermentation</div>, <div class="source">Gigascience</div> , <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20combined%20meta-barcoding%20and%20shotgun%20metagenomic%20analysis%20of%20spontaneous%20wine%20fermentation&amp;author=P.%20R.%20Sternes&amp;author=D.%20Lee&amp;author=D.%20R.%20Kutyna&amp;author=A.%20R.%20Borneman&amp;publication_year=2017&amp;journal=Gigascience&amp;volume=6&amp;pages=1-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/gigascience/gix040" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgigascience%2Fgix040" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgigascience%2Fgix040"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28595314" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20combined%20meta-barcoding%20and%20shotgun%20metagenomic%20analysis%20of%20spontaneous%20wine%20fermentation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Grangeteau</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Gerhards</div> <div class="given-names">D., V.</div>
</div>, <div class="name string-name">
<div class="surname">Wallbrunn</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Alexandre</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Rousseaux</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Guilloux-Benatier</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Persistence of two non-saccharomyces yeasts (Hanseniaspora and Starmerella) in the cellar</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">268</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Persistence%20of%20two%20non-saccharomyces%20yeasts%20(Hanseniaspora%20and%20Starmerella)%20in%20the%20cellar&amp;author=C.%20Grangeteau&amp;author=D.%2C%20V.%20Gerhards&amp;author=C.%20Wallbrunn&amp;author=H.%20Alexandre&amp;author=S.%20Rousseaux&amp;author=M.%20Guilloux-Benatier&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=268" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fmicb.2016.00268" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2016.00268" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffmicb.2016.00268"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27014199" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Persistence%20of%20two%20non-saccharomyces%20yeasts%20(Hanseniaspora%20and%20Starmerella)%20in%20the%20cellar&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Padilla</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Garcia-Fernandez</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Gonzalez</div> <div class="given-names">B.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Yeast biodiversity from DOQ priorat uninoculated fermentations</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">930</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Yeast%20biodiversity%20from%20DOQ%20priorat%20uninoculated%20fermentations&amp;author=B.%20Padilla&amp;author=D.%20Garcia-Fernandez&amp;author=B.%20Gonzalez&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=930" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27379060" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Yeast%20biodiversity%20from%20DOQ%20priorat%20uninoculated%20fermentations&amp;aulast=Padilla&amp;date=2016&amp;spage=930&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Yeast%20biodiversity%20from%20DOQ%20priorat%20uninoculated%20fermentations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Clavijo</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Calderon</div> <div class="given-names">I. L.</div>
</div>, <div class="name string-name">
<div class="surname">Paneque</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Diversity of Saccharomyces and non-Saccharomyces yeasts in three red grape varieties cultured in the Serrania de Ronda (Spain) vine-growing region</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">143</div>, <div class="fpage">241</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20Saccharomyces%20and%20non-Saccharomyces%20yeasts%20in%20three%20red%20grape%20varieties%20cultured%20in%20the%20Serrania%20de%20Ronda%20(Spain)%20vine-growing%20region&amp;author=A.%20Clavijo&amp;author=I.%20L.%20Calderon&amp;author=P.%20Paneque&amp;publication_year=2010&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=143&amp;pages=241-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijfoodmicro.2010.08.010" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2010.08.010" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijfoodmicro.2010.08.010"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20851489" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Diversity%20of%20Saccharomyces%20and%20non-Saccharomyces%20yeasts%20in%20three%20red%20grape%20varieties%20cultured%20in%20the%20Serrania%20de%20Ronda%20(Spain)%20vine-growing%20region&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Vigentini</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Maghradze</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Petrozziello</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Indigenous Georgian wine-associated yeasts and grape cultivars to edit the wine quality in a precision oenology perspective</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">352</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Indigenous%20Georgian%20wine-associated%20yeasts%20and%20grape%20cultivars%20to%20edit%20the%20wine%20quality%20in%20a%20precision%20oenology%20perspective&amp;author=I.%20Vigentini&amp;author=D.%20Maghradze&amp;author=M.%20Petrozziello&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=352" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fmicb.2016.00352" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2016.00352" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffmicb.2016.00352"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27047468" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Indigenous%20Georgian%20wine-associated%20yeasts%20and%20grape%20cultivars%20to%20edit%20the%20wine%20quality%20in%20a%20precision%20oenology%20perspective&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Santiago-Urbina</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Arias-García</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Ruiz-Terán</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Yeast species associated with spontaneous fermentation of taberna, a traditional palm wine from the southeast of Mexico</div>, <div class="source">Ann. Microbiol</div> ., <div class="volume">65</div>, <div class="fpage">287</div>–<div class="lpage">96</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Yeast%20species%20associated%20with%20spontaneous%20fermentation%20of%20taberna%2C%20a%20traditional%20palm%20wine%20from%20the%20southeast%20of%20Mexico&amp;author=J.%20A.%20Santiago-Urbina&amp;author=J.%20A.%20Arias-Garc%C3%ADa&amp;author=F.%20Ruiz-Ter%C3%A1n&amp;publication_year=2015&amp;journal=Ann.%20Microbiol&amp;volume=65&amp;pages=287-96" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s13213-014-0861-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13213-014-0861-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs13213-014-0861-8"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Yeast%20species%20associated%20with%20spontaneous%20fermentation%20of%20taberna%2C%20a%20traditional%20palm%20wine%20from%20the%20southeast%20of%20Mexico&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Romancino</div> <div class="given-names">D. P.</div>
</div>, <div class="name string-name">
<div class="surname">Di Maio</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Muriella</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Oliva</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Analysis of non-Saccharomyces yeast populations isolated from grape musts from Sicily (Italy)</div>, <div class="source">J. Appl. Microbiol</div> ., <div class="volume">105</div>, <div class="fpage">2248</div>–<div class="lpage">54</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20non-Saccharomyces%20yeast%20populations%20isolated%20from%20grape%20musts%20from%20Sicily%20(Italy)&amp;author=D.%20P.%20Romancino&amp;author=S.%20Di%20Maio&amp;author=R.%20Muriella&amp;author=D.%20Oliva&amp;publication_year=2008&amp;journal=J.%20Appl.%20Microbiol&amp;volume=105&amp;pages=2248-54" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/jam.2008.105.issue-6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjam.2008.105.issue-6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fjam.2008.105.issue-6"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19120668" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analysis%20of%20non-Saccharomyces%20yeast%20populations%20isolated%20from%20grape%20musts%20from%20Sicily%20(Italy)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Moreira</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Guedes de Pinho</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Hogg</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Vasconcelos</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Heavy sulphur compounds, higher alcohols and esters production profile of <em>Hanseniaspora uvarum</em> and <em>Hanseniaspora guilliermondii</em> grown as pure and mixed cultures in grape must</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">124</div>, <div class="fpage">231</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Heavy%20sulphur%20compounds%2C%20higher%20alcohols%20and%20esters%20production%20profile%20of%20Hanseniaspora%20uvarum%20and%20Hanseniaspora%20guilliermondii%20grown%20as%20pure%20and%20mixed%20cultures%20in%20grape%20must&amp;author=N.%20Moreira&amp;author=F.%20Mendes&amp;author=P.%20Guedes%20de%20Pinho&amp;author=T.%20Hogg&amp;author=I.%20Vasconcelos&amp;publication_year=2008&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=124&amp;pages=231-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijfoodmicro.2008.03.025" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2008.03.025" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijfoodmicro.2008.03.025"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18457893" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Heavy%20sulphur%20compounds%2C%20higher%20alcohols%20and%20esters%20production%20profile%20of%20Hanseniaspora%20uvarum%20and%20Hanseniaspora%20guilliermondii%20grown%20as%20pure%20and%20mixed%20cultures%20in%20grape%20must&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lage</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Barbosa</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Mateus</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Vasconcelos</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes-Faia</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes-Ferreira</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">
<em>H. guilliermondii</em> impacts growth kinetics and metabolic activity of <em>S. cerevisiae</em>: the role of initial nitrogen concentration</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">172</div>, <div class="fpage">62</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=H.%20guilliermondii%20impacts%20growth%20kinetics%20and%20metabolic%20activity%20of%20S.%20cerevisiae%3A%20the%20role%20of%20initial%20nitrogen%20concentration&amp;author=P.%20Lage&amp;author=C.%20Barbosa&amp;author=B.%20Mateus&amp;author=I.%20Vasconcelos&amp;author=A.%20Mendes-Faia&amp;author=A.%20Mendes-Ferreira&amp;publication_year=2014&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=172&amp;pages=62-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijfoodmicro.2013.11.031" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2013.11.031" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijfoodmicro.2013.11.031"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24361834" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:H.%20guilliermondii%20impacts%20growth%20kinetics%20and%20metabolic%20activity%20of%20S.%20cerevisiae%3A%20the%20role%20of%20initial%20nitrogen%20concentration&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Moreira</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Pina</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Couto</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Hogg</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Vasconcelos</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Volatile compounds contribution of <em>Hanseniaspora guilliermondii</em> and <em>Hanseniaspora uvarum</em> during red wine vinifications</div>, <div class="source">Food Control</div> , <div class="volume">22</div>, <div class="fpage">662</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Volatile%20compounds%20contribution%20of%20Hanseniaspora%20guilliermondii%20and%20Hanseniaspora%20uvarum%20during%20red%20wine%20vinifications&amp;author=N.%20Moreira&amp;author=C.%20Pina&amp;author=F.%20Mendes&amp;author=J.%20A.%20Couto&amp;author=T.%20Hogg&amp;author=I.%20Vasconcelos&amp;publication_year=2011&amp;journal=Food%20Control&amp;volume=22&amp;pages=662-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.foodcont.2010.07.025" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodcont.2010.07.025" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.foodcont.2010.07.025"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Volatile%20compounds%20contribution%20of%20Hanseniaspora%20guilliermondii%20and%20Hanseniaspora%20uvarum%20during%20red%20wine%20vinifications&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rojas</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Gil</div> <div class="given-names">J. V.</div>
</div>, <div class="name string-name">
<div class="surname">Pinaga</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Manzanares</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">Acetate ester formation in wine by mixed cultures in laboratory fermentations</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">86</div>, <div class="fpage">181</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Acetate%20ester%20formation%20in%20wine%20by%20mixed%20cultures%20in%20laboratory%20fermentations&amp;author=V.%20Rojas&amp;author=J.%20V.%20Gil&amp;author=F.%20Pinaga&amp;author=P.%20Manzanares&amp;publication_year=2003&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=86&amp;pages=181-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0168-1605(03)00255-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-1605(03)00255-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0168-1605(03)00255-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12892933" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Acetate%20ester%20formation%20in%20wine%20by%20mixed%20cultures%20in%20laboratory%20fermentations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">de Arruda Moura Pietrowski</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">dos Santos</div> <div class="given-names">C. M.</div>
</div>, <div class="name string-name">
<div class="surname">Sauer</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Wosiacki</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Nogueira</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Influence of fermentation with <em>Hanseniaspora</em> sp. yeast on the volatile profile of fermented apple</div>, <div class="source">J. Agric. Food Chem</div> ., <div class="volume">60</div>, <div class="fpage">9815</div>–<div class="lpage">21</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Influence%20of%20fermentation%20with%20Hanseniaspora%20sp.%20yeast%20on%20the%20volatile%20profile%20of%20fermented%20apple&amp;author=G.%20de%20Arruda%20Moura%20Pietrowski&amp;author=C.%20M.%20dos%20Santos&amp;author=E.%20Sauer&amp;author=G.%20Wosiacki&amp;author=A.%20Nogueira&amp;publication_year=2012&amp;journal=J.%20Agric.%20Food%20Chem&amp;volume=60&amp;pages=9815-21" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1021/jf302290k" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf302290k" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1021%2Fjf302290k"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22958203" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Influence%20of%20fermentation%20with%20Hanseniaspora%20sp.%20yeast%20on%20the%20volatile%20profile%20of%20fermented%20apple&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Batista</div> <div class="given-names">N. N.</div>
</div>, <div class="name string-name">
<div class="surname">Ramos</div> <div class="given-names">C. L.</div>
</div>, <div class="name string-name">
<div class="surname">Dias</div> <div class="given-names">D. R.</div>
</div>, <div class="name string-name">
<div class="surname">Pinheiro</div> <div class="given-names">A. C.</div>
</div>, <div class="name string-name">
<div class="surname">Schwan</div> <div class="given-names">R. F.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">The impact of yeast starter cultures on the microbial communities and volatile compounds in cocoa fermentation and the resulting sensory attributes of chocolate</div>, <div class="source">J. Food Sci. Technol</div> ., <div class="volume">53</div>, <div class="fpage">1101</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20yeast%20starter%20cultures%20on%20the%20microbial%20communities%20and%20volatile%20compounds%20in%20cocoa%20fermentation%20and%20the%20resulting%20sensory%20attributes%20of%20chocolate&amp;author=N.%20N.%20Batista&amp;author=C.%20L.%20Ramos&amp;author=D.%20R.%20Dias&amp;author=A.%20C.%20Pinheiro&amp;author=R.%20F.%20Schwan&amp;publication_year=2016&amp;journal=J.%20Food%20Sci.%20Technol&amp;volume=53&amp;pages=1101-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s13197-015-2132-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13197-015-2132-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs13197-015-2132-5"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27162390" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20impact%20of%20yeast%20starter%20cultures%20on%20the%20microbial%20communities%20and%20volatile%20compounds%20in%20cocoa%20fermentation%20and%20the%20resulting%20sensory%20attributes%20of%20chocolate&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Belda</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Navascues</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Marquina</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Santos</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Calderon</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Benito</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Outlining the influence of non-conventional yeasts in wine ageing over lees</div>, <div class="source">Yeast</div> , <div class="volume">33</div>, <div class="fpage">329</div>–<div class="lpage">38</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Outlining%20the%20influence%20of%20non-conventional%20yeasts%20in%20wine%20ageing%20over%20lees&amp;author=I.%20Belda&amp;author=E.%20Navascues&amp;author=D.%20Marquina&amp;author=A.%20Santos&amp;author=F.%20Calderon&amp;author=S.%20Benito&amp;publication_year=2016&amp;journal=Yeast&amp;volume=33&amp;pages=329-38" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/yea.v33.7" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fyea.v33.7" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fyea.v33.7"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27017923" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Outlining%20the%20influence%20of%20non-conventional%20yeasts%20in%20wine%20ageing%20over%20lees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Domizio</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Bisson</div> <div class="given-names">L. F.</div>
</div>, <div class="name string-name">
<div class="surname">Barile</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Use of non-Saccharomyces wine yeasts as novel sources of mannoproteins in wine</div>, <div class="source">Food Microbiol</div> ., <div class="volume">43</div>, <div class="fpage">5</div>–<div class="lpage">15</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Use%20of%20non-Saccharomyces%20wine%20yeasts%20as%20novel%20sources%20of%20mannoproteins%20in%20wine&amp;author=P.%20Domizio&amp;author=Y.%20Liu&amp;author=L.%20F.%20Bisson&amp;author=D.%20Barile&amp;publication_year=2014&amp;journal=Food%20Microbiol&amp;volume=43&amp;pages=5-15" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.fm.2014.04.005" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fm.2014.04.005" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.fm.2014.04.005"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929876" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Use%20of%20non-Saccharomyces%20wine%20yeasts%20as%20novel%20sources%20of%20mannoproteins%20in%20wine&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Contreras</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Hidalgo</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Henschke</div> <div class="given-names">P. A.</div>
</div>, <div class="name string-name">
<div class="surname">Chambers</div> <div class="given-names">P. J.</div>
</div>, <div class="name string-name">
<div class="surname">Curtin</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Varela</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Evaluation of non-Saccharomyces yeasts for the reduction of alcohol content in wine</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">80</div>, <div class="fpage">1670</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20non-Saccharomyces%20yeasts%20for%20the%20reduction%20of%20alcohol%20content%20in%20wine&amp;author=A.%20Contreras&amp;author=C.%20Hidalgo&amp;author=P.%20A.%20Henschke&amp;author=P.%20J.%20Chambers&amp;author=C.%20Curtin&amp;author=C.%20Varela&amp;publication_year=2014&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=80&amp;pages=1670-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.03780-13" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03780-13" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.03780-13"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24375129" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluation%20of%20non-Saccharomyces%20yeasts%20for%20the%20reduction%20of%20alcohol%20content%20in%20wine&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ciani</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Morales</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Comitini</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Non-conventional yeast species for lowering ethanol content of wines</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">642</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Non-conventional%20yeast%20species%20for%20lowering%20ethanol%20content%20of%20wines&amp;author=M.%20Ciani&amp;author=P.%20Morales&amp;author=F.%20Comitini&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=642" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27199967" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Non-conventional%20yeast%20species%20for%20lowering%20ethanol%20content%20of%20wines&amp;aulast=Ciani&amp;date=2016&amp;spage=642&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Non-conventional%20yeast%20species%20for%20lowering%20ethanol%20content%20of%20wines&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tristezza</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Tufariello</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Capozzi</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Spano</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Mita</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Grieco</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">The oenological potential of <em>Hanseniaspora uvarum</em> in simultaneous and sequential co-fermentation with <em>Saccharomyces cerevisiae</em> for industrial wine production</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">670</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20oenological%20potential%20of%20Hanseniaspora%20uvarum%20in%20simultaneous%20and%20sequential%20co-fermentation%20with%20Saccharomyces%20cerevisiae%20for%20industrial%20wine%20production&amp;author=M.%20Tristezza&amp;author=M.%20Tufariello&amp;author=V.%20Capozzi&amp;author=G.%20Spano&amp;author=G.%20Mita&amp;author=F.%20Grieco&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=670" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fmicb.2016.00670" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2016.00670" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffmicb.2016.00670"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27242698" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20oenological%20potential%20of%20Hanseniaspora%20uvarum%20in%20simultaneous%20and%20sequential%20co-fermentation%20with%20Saccharomyces%20cerevisiae%20for%20industrial%20wine%20production&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Barbosa</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes-Faia</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Lage</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Mira</div> <div class="given-names">N. P.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes-Ferreira</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Genomic expression program of <em>Saccharomyces cerevisiae</em> along a mixed-culture wine fermentation with <em>Hanseniaspora guilliermondii</em>
</div>, <div class="source">Microbial. Cell Factories</div> , <div class="volume">14</div>, <div class="fpage">124</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomic%20expression%20program%20of%20Saccharomyces%20cerevisiae%20along%20a%20mixed-culture%20wine%20fermentation%20with%20Hanseniaspora%20guilliermondii&amp;author=C.%20Barbosa&amp;author=A.%20Mendes-Faia&amp;author=P.%20Lage&amp;author=N.%20P.%20Mira&amp;author=A.%20Mendes-Ferreira&amp;publication_year=2015&amp;journal=Microbial.%20Cell%20Factories&amp;volume=14&amp;pages=124" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12934-015-0318-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12934-015-0318-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12934-015-0318-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26314747" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomic%20expression%20program%20of%20Saccharomyces%20cerevisiae%20along%20a%20mixed-culture%20wine%20fermentation%20with%20Hanseniaspora%20guilliermondii&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Branco</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Monteiro</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Moura</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Albergaria</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Survival rate of wine-related yeasts during alcoholic fermentation assessed by direct live/dead staining combined with fluorescence in situ hybridization</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">158</div>, <div class="fpage">49</div>–<div class="lpage">57</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Survival%20rate%20of%20wine-related%20yeasts%20during%20alcoholic%20fermentation%20assessed%20by%20direct%20live%2Fdead%20staining%20combined%20with%20fluorescence%20in%20situ%20hybridization&amp;author=P.%20Branco&amp;author=M.%20Monteiro&amp;author=P.%20Moura&amp;author=H.%20Albergaria&amp;publication_year=2012&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=158&amp;pages=49-57" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijfoodmicro.2012.06.020" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2012.06.020" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijfoodmicro.2012.06.020"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22819715" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Survival%20rate%20of%20wine-related%20yeasts%20during%20alcoholic%20fermentation%20assessed%20by%20direct%20live%2Fdead%20staining%20combined%20with%20fluorescence%20in%20situ%20hybridization&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Perez-Nevado</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Albergaria</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Hogg</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Girio</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Cellular death of two non-Saccharomyces wine-related yeasts during mixed fermentations with <em>Saccharomyces cerevisiae</em>
</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">108</div>, <div class="fpage">336</div>–<div class="lpage">45</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Cellular%20death%20of%20two%20non-Saccharomyces%20wine-related%20yeasts%20during%20mixed%20fermentations%20with%20Saccharomyces%20cerevisiae&amp;author=F.%20Perez-Nevado&amp;author=H.%20Albergaria&amp;author=T.%20Hogg&amp;author=F.%20Girio&amp;publication_year=2006&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=108&amp;pages=336-45" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16564103" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Cellular%20death%20of%20two%20non-Saccharomyces%20wine-related%20yeasts%20during%20mixed%20fermentations%20with%20Saccharomyces%20cerevisiae&amp;aulast=Perez-Nevado&amp;date=2006&amp;spage=336&amp;epage=45&amp;volume=108" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cellular%20death%20of%20two%20non-Saccharomyces%20wine-related%20yeasts%20during%20mixed%20fermentations%20with%20Saccharomyces%20cerevisiae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pina</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Santos</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Couto</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Hogg</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Ethanol tolerance of five non-Saccharomyces wine yeasts in comparison with a strain of <em>Saccharomyces cerevisiae</em>—influence of different culture conditions</div>, <div class="source">Food Microbiol</div> ., <div class="volume">21</div>, <div class="fpage">439</div>–<div class="lpage">47</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ethanol%20tolerance%20of%20five%20non-Saccharomyces%20wine%20yeasts%20in%20comparison%20with%20a%20strain%20of%20Saccharomyces%20cerevisiae%E2%80%94influence%20of%20different%20culture%20conditions&amp;author=C.%20Pina&amp;author=C.%20Santos&amp;author=J.%20A.%20Couto&amp;author=T.%20Hogg&amp;publication_year=2004&amp;journal=Food%20Microbiol&amp;volume=21&amp;pages=439-47" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.fm.2003.10.009" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fm.2003.10.009" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.fm.2003.10.009"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ethanol%20tolerance%20of%20five%20non-Saccharomyces%20wine%20yeasts%20in%20comparison%20with%20a%20strain%20of%20Saccharomyces%20cerevisiae%E2%80%94influence%20of%20different%20culture%20conditions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Albergaria</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Arneborg</div> <div class="given-names">N.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Dominance of <em>Saccharomyces cerevisiae</em> in alcoholic fermentation processes: role of physiological fitness and microbial interactions</div>, <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">100</div>, <div class="fpage">2035</div>–<div class="lpage">46</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dominance%20of%20Saccharomyces%20cerevisiae%20in%20alcoholic%20fermentation%20processes%3A%20role%20of%20physiological%20fitness%20and%20microbial%20interactions&amp;author=H.%20Albergaria&amp;author=N.%20Arneborg&amp;publication_year=2016&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=100&amp;pages=2035-46" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00253-015-7255-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-015-7255-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00253-015-7255-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26728020" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dominance%20of%20Saccharomyces%20cerevisiae%20in%20alcoholic%20fermentation%20processes%3A%20role%20of%20physiological%20fitness%20and%20microbial%20interactions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Branco</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Kemsawasd</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Santos</div> <div class="given-names">L.</div>
</div>
</div>, et al. <div class="year">2017</div>, 
<div class="article-title">
<em>Saccharomyces cerevisiae</em> accumulates GAPDH-derived peptides on its cell surface that induce death of non-Saccharomyces yeasts by cell-to-cell contact</div>, <div class="source">FEMS Microbiol. Ecol</div> ., <div class="volume">93</div>, doi: 10.1093/femsec/fix055.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Saccharomyces%20cerevisiae%20accumulates%20GAPDH-derived%20peptides%20on%20its%20cell%20surface%20that%20induce%20death%20of%20non-Saccharomyces%20yeasts%20by%20cell-to-cell%20contact&amp;aulast=Branco&amp;date=2017&amp;volume=93" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Saccharomyces%20cerevisiae%20accumulates%20GAPDH-derived%20peptides%20on%20its%20cell%20surface%20that%20induce%20death%20of%20non-Saccharomyces%20yeasts%20by%20cell-to-cell%20contact&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Seixas</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Barbosa</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Salazar</div> <div class="given-names">S. B.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Genome sequence of the nonconventional wine yeast <em>Hanseniaspora guilliermondii</em> UTAD222</div>, <div class="source">Genome Announc</div> ., <div class="volume">5</div>, <div class="fpage">e01515</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20nonconventional%20wine%20yeast%20Hanseniaspora%20guilliermondii%20UTAD222&amp;author=I.%20Seixas&amp;author=C.%20Barbosa&amp;author=S.%20B.%20Salazar&amp;publication_year=2017&amp;journal=Genome%20Announc&amp;volume=5&amp;pages=e01515-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/genomeA.01515-16" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FgenomeA.01515-16" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FgenomeA.01515-16"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28153887" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20of%20the%20nonconventional%20wine%20yeast%20Hanseniaspora%20guilliermondii%20UTAD222&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sternes</div> <div class="given-names">P. R.</div>
</div>, <div class="name string-name">
<div class="surname">Lee</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Kutyna</div> <div class="given-names">D. R.</div>
</div>, <div class="name string-name">
<div class="surname">Borneman</div> <div class="given-names">A. R.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Genome sequences of three species of <em>Hanseniaspora</em> isolated from spontaneous wine fermentations</div>, <div class="source">Genome Announc</div> ., <div class="volume">4</div>, <div class="fpage">e01287</div>–<div class="lpage">16</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequences%20of%20three%20species%20of%20Hanseniaspora%20isolated%20from%20spontaneous%20wine%20fermentations&amp;author=P.%20R.%20Sternes&amp;author=D.%20Lee&amp;author=D.%20R.%20Kutyna&amp;author=A.%20R.%20Borneman&amp;publication_year=2016&amp;journal=Genome%20Announc&amp;volume=4&amp;pages=e01287-16" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/genomeA.01287-16" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FgenomeA.01287-16" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FgenomeA.01287-16"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27856586" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequences%20of%20three%20species%20of%20Hanseniaspora%20isolated%20from%20spontaneous%20wine%20fermentations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Langenberg</div> <div class="given-names">A. K.</div>
</div>, <div class="name string-name">
<div class="surname">Bink</div> <div class="given-names">F. J.</div>
</div>, <div class="name string-name">
<div class="surname">Wolff</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Glycolytic functions are conserved in the genome of the wine yeast <em>Hanseniaspora uvarum</em>, and pyruvate kinase limits its capacity for alcoholic fermentation</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">83</div>, <div class="fpage">e01580</div>–<div class="lpage">17</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Glycolytic%20functions%20are%20conserved%20in%20the%20genome%20of%20the%20wine%20yeast%20Hanseniaspora%20uvarum%2C%20and%20pyruvate%20kinase%20limits%20its%20capacity%20for%20alcoholic%20fermentation&amp;author=A.%20K.%20Langenberg&amp;author=F.%20J.%20Bink&amp;author=L.%20Wolff&amp;publication_year=2017&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=83&amp;pages=e01580-17" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.01580-17" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01580-17" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.01580-17"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28887422" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Glycolytic%20functions%20are%20conserved%20in%20the%20genome%20of%20the%20wine%20yeast%20Hanseniaspora%20uvarum%2C%20and%20pyruvate%20kinase%20limits%20its%20capacity%20for%20alcoholic%20fermentation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nisiotou</div> <div class="given-names">A. A.</div>
</div>, <div class="name string-name">
<div class="surname">Nychas</div> <div class="given-names">G. J.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Yeast populations residing on healthy or botrytis-infected grapes from a vineyard in Attica, Greece</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">73</div>, <div class="fpage">2765</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Yeast%20populations%20residing%20on%20healthy%20or%20botrytis-infected%20grapes%20from%20a%20vineyard%20in%20Attica%2C%20Greece&amp;author=A.%20A.%20Nisiotou&amp;author=G.%20J.%20Nychas&amp;publication_year=2007&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=73&amp;pages=2765-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.01864-06" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01864-06" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.01864-06"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17293525" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Yeast%20populations%20residing%20on%20healthy%20or%20botrytis-infected%20grapes%20from%20a%20vineyard%20in%20Attica%2C%20Greece&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mendes Ferreira</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Climaco</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes Faia</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2001</div>, 
<div class="article-title">The role of non-<em>Saccharomyces</em> species in releasing glycosidic bound fraction of grape aroma components – a preliminary study</div>, <div class="source">J. Appl. Microbiol</div> ., <div class="volume">91</div>, <div class="fpage">67</div>–<div class="lpage">71</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20non-Saccharomyces%20species%20in%20releasing%20glycosidic%20bound%20fraction%20of%20grape%20aroma%20components%20%E2%80%93%20a%20preliminary%20study&amp;author=A.%20Mendes%20Ferreira&amp;author=M.%20C.%20Climaco&amp;author=A.%20Mendes%20Faia&amp;publication_year=2001&amp;journal=J.%20Appl.%20Microbiol&amp;volume=91&amp;pages=67-71" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1365-2672.2001.01348.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2672.2001.01348.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1365-2672.2001.01348.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11442715" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20role%20of%20non-Saccharomyces%20species%20in%20releasing%20glycosidic%20bound%20fraction%20of%20grape%20aroma%20components%20%E2%80%93%20a%20preliminary%20study&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">González</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Gallardo</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Pombar</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rego</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Rodríguez</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Determination of enzymatic activities in ecotypic Saccharomyces and non-Saccharomyces yeast</div>, <div class="source">Electron. J. Environ. Agric. Food Chem</div> ., <div class="volume">3</div>, <div class="fpage">743</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Determination%20of%20enzymatic%20activities%20in%20ecotypic%20Saccharomyces%20and%20non-Saccharomyces%20yeast&amp;aulast=Gonz%C3%A1lez&amp;date=2004&amp;spage=743&amp;epage=50&amp;volume=3" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Determination%20of%20enzymatic%20activities%20in%20ecotypic%20Saccharomyces%20and%20non-Saccharomyces%20yeast&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oliveira</div> <div class="given-names">V. A.</div>
</div>, <div class="name string-name">
<div class="surname">Vicente</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Fietto</div> <div class="given-names">L. G.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Biochemical and molecular characterization of <em>Saccharomyces cerevisiae</em> strains obtained from sugar-cane juice fermentations and their impact in cachaca production</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">74</div>, <div class="fpage">693</div>–<div class="lpage">701</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Biochemical%20and%20molecular%20characterization%20of%20Saccharomyces%20cerevisiae%20strains%20obtained%20from%20sugar-cane%20juice%20fermentations%20and%20their%20impact%20in%20cachaca%20production&amp;author=V.%20A.%20Oliveira&amp;author=M.%20A.%20Vicente&amp;author=L.%20G.%20Fietto&amp;publication_year=2008&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=74&amp;pages=693-701" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.01729-07" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01729-07" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.01729-07"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18065624" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biochemical%20and%20molecular%20characterization%20of%20Saccharomyces%20cerevisiae%20strains%20obtained%20from%20sugar-cane%20juice%20fermentations%20and%20their%20impact%20in%20cachaca%20production&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rissman</div> <div class="given-names">A. I.</div>
</div>, <div class="name string-name">
<div class="surname">Mau</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Biehl</div> <div class="given-names">B. S.</div>
</div>, <div class="name string-name">
<div class="surname">Darling</div> <div class="given-names">A. E.</div>
</div>, <div class="name string-name">
<div class="surname">Glasner</div> <div class="given-names">J. D.</div>
</div>, <div class="name string-name">
<div class="surname">Perna</div> <div class="given-names">N. T.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Reordering contigs of draft genomes using the Mauve aligner</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2071</div>–<div class="lpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Reordering%20contigs%20of%20draft%20genomes%20using%20the%20Mauve%20aligner&amp;author=A.%20I.%20Rissman&amp;author=B.%20Mau&amp;author=B.%20S.%20Biehl&amp;author=A.%20E.%20Darling&amp;author=J.%20D.%20Glasner&amp;author=N.%20T.%20Perna&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2071-3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp356" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp356" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp356"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19515959" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Reordering%20contigs%20of%20draft%20genomes%20using%20the%20Mauve%20aligner&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Salamov</div> <div class="given-names">A. A.</div>
</div>, <div class="name string-name">
<div class="surname">Solovyev</div> <div class="given-names">V. V.</div>
</div>
</div> <div class="year">2000</div>, 
<div class="article-title">Ab initio gene finding in Drosophila genomic DNA</div>, <div class="source">Genome Res</div> ., <div class="volume">10</div>, <div class="fpage">516</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ab%20initio%20gene%20finding%20in%20Drosophila%20genomic%20DNA&amp;author=A.%20A.%20Salamov&amp;author=V.%20V.%20Solovyev&amp;publication_year=2000&amp;journal=Genome%20Res&amp;volume=10&amp;pages=516-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.10.4.516" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.4.516" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.10.4.516"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10779491" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ab%20initio%20gene%20finding%20in%20Drosophila%20genomic%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ter-Hovhannisyan</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Lomsadze</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Chernoff</div> <div class="given-names">Y. O.</div>
</div>, <div class="name string-name">
<div class="surname">Borodovsky</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training</div>, <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">1979</div>–<div class="lpage">90</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20in%20novel%20fungal%20genomes%20using%20an%20ab%20initio%20algorithm%20with%20unsupervised%20training&amp;author=V.%20Ter-Hovhannisyan&amp;author=A.%20Lomsadze&amp;author=Y.%20O.%20Chernoff&amp;author=M.%20Borodovsky&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=1979-90" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.081612.108" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.081612.108" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.081612.108"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18757608" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20prediction%20in%20novel%20fungal%20genomes%20using%20an%20ab%20initio%20algorithm%20with%20unsupervised%20training&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Stanke</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Keller</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Gunduz</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Hayes</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Waack</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Morgenstern</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">AUGUSTUS: ab initio prediction of alternative transcripts</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W435</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;author=M.%20Stanke&amp;author=O.%20Keller&amp;author=I.%20Gunduz&amp;author=A.%20Hayes&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W435-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkl200" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl200" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl200"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845043" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lowe</div> <div class="given-names">T. M.</div>
</div>, <div class="name string-name">
<div class="surname">Eddy</div> <div class="given-names">S. R.</div>
</div>
</div> <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/25.5.955" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.955" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F25.5.955"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Walter</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Rattei</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Arnold</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">PEDANT covers all complete RefSeq genomes</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">D408</div>–<div class="lpage">11</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PEDANT%20covers%20all%20complete%20RefSeq%20genomes&amp;author=M.%20C.%20Walter&amp;author=T.%20Rattei&amp;author=R.%20Arnold&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=D408-11" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkn749" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn749" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn749"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940859" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PEDANT%20covers%20all%20complete%20RefSeq%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kanehisa</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Sato</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Morishima</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">BlastKOALA and GhostKOALA: kEGG tools for functional characterization of genome and metagenome sequences</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">428</div>, <div class="fpage">726</div>–<div class="lpage">31</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BlastKOALA%20and%20GhostKOALA%3A%20kEGG%20tools%20for%20functional%20characterization%20of%20genome%20and%20metagenome%20sequences&amp;author=M.%20Kanehisa&amp;author=Y.%20Sato&amp;author=K.%20Morishima&amp;publication_year=2016&amp;journal=J.%20Mol.%20Biol&amp;volume=428&amp;pages=726-31" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.jmb.2015.11.006" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2015.11.006" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jmb.2015.11.006"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26585406" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BlastKOALA%20and%20GhostKOALA%3A%20kEGG%20tools%20for%20functional%20characterization%20of%20genome%20and%20metagenome%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pinto</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Pinho</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Cardoso</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Wine fermentation microbiome: a landscape from different Portuguese wine appellations</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">905</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Wine%20fermentation%20microbiome%3A%20a%20landscape%20from%20different%20Portuguese%20wine%20appellations&amp;author=C.%20Pinto&amp;author=D.%20Pinho&amp;author=R.%20Cardoso&amp;publication_year=2015&amp;journal=Front.%20Microbiol&amp;volume=6&amp;pages=905" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fmicb.2015.00905" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2015.00905" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffmicb.2015.00905"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26388852" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Wine%20fermentation%20microbiome%3A%20a%20landscape%20from%20different%20Portuguese%20wine%20appellations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Drumonde-Neves</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Franco-Duarte</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Lima</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Schuller</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Pais</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Association between grape yeast communities and the vineyard ecosystems</div>, <div class="source">PLoS One</div> , <div class="volume">12</div>, <div class="fpage">e0169883</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Association%20between%20grape%20yeast%20communities%20and%20the%20vineyard%20ecosystems&amp;author=J.%20Drumonde-Neves&amp;author=R.%20Franco-Duarte&amp;author=T.%20Lima&amp;author=D.%20Schuller&amp;author=C.%20Pais&amp;publication_year=2017&amp;journal=PLoS%20One&amp;volume=12&amp;pages=e0169883" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0169883" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0169883" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0169883"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28085916" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Association%20between%20grape%20yeast%20communities%20and%20the%20vineyard%20ecosystems&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cadez</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Poot</div> <div class="given-names">G. A.</div>
</div>, <div class="name string-name">
<div class="surname">Raspor</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">M. T.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">
<em>Hanseniaspora meyeri</em> sp. nov., <em>Hanseniaspora clermontiae</em> sp. nov., <em>Hanseniaspora lachancei</em> sp. nov. and <em>Hanseniaspora opuntiae</em> sp. nov., novel apiculate yeast species</div>, <div class="source">Int. J. Syst. Evol. Microbiol</div> ., <div class="volume">53</div>, <div class="fpage">1671</div>–<div class="lpage">80</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hanseniaspora%20meyeri%20sp.%20nov.%2C%20Hanseniaspora%20clermontiae%20sp.%20nov.%2C%20Hanseniaspora%20lachancei%20sp.%20nov.%20and%20Hanseniaspora%20opuntiae%20sp.%20nov.%2C%20novel%20apiculate%20yeast%20species&amp;author=N.%20Cadez&amp;author=G.%20A.%20Poot&amp;author=P.%20Raspor&amp;author=M.%20T.%20Smith&amp;publication_year=2003&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol&amp;volume=53&amp;pages=1671-80" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1099/ijs.0.02618-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.02618-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1099%2Fijs.0.02618-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13130067" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hanseniaspora%20meyeri%20sp.%20nov.%2C%20Hanseniaspora%20clermontiae%20sp.%20nov.%2C%20Hanseniaspora%20lachancei%20sp.%20nov.%20and%20Hanseniaspora%20opuntiae%20sp.%20nov.%2C%20novel%20apiculate%20yeast%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Albertin</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Setati</div> <div class="given-names">M. E.</div>
</div>, <div class="name string-name">
<div class="surname">Miot-Sertier</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">
<em>Hanseniaspora uvarum</em> from winemaking environments show spatial and temporal genetic clustering</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">1569</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hanseniaspora%20uvarum%20from%20winemaking%20environments%20show%20spatial%20and%20temporal%20genetic%20clustering&amp;author=W.%20Albertin&amp;author=M.%20E.%20Setati&amp;author=C.%20Miot-Sertier&amp;publication_year=2015&amp;journal=Front.%20Microbiol&amp;volume=6&amp;pages=1569" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26834719" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Hanseniaspora%20uvarum%20from%20winemaking%20environments%20show%20spatial%20and%20temporal%20genetic%20clustering&amp;aulast=Albertin&amp;date=2015&amp;spage=1569&amp;volume=6" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hanseniaspora%20uvarum%20from%20winemaking%20environments%20show%20spatial%20and%20temporal%20genetic%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Esteve-Zarzoso</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Peris-Toran</div> <div class="given-names">M. J.</div>
</div>, <div class="name string-name">
<div class="surname">Ramon</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Quero</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2001</div>, 
<div class="article-title">Molecular characterisation of <em>Hanseniaspora</em> species</div>, <div class="source">Antonie Van Leeuwenhoek</div> , <div class="volume">80</div>, <div class="fpage">85</div>–<div class="lpage">92</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterisation%20of%20Hanseniaspora%20species&amp;author=B.%20Esteve-Zarzoso&amp;author=M.%20J.%20Peris-Toran&amp;author=D.%20Ramon&amp;author=A.%20Quero&amp;publication_year=2001&amp;journal=Antonie%20Van%20Leeuwenhoek&amp;volume=80&amp;pages=85-92" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1023/A:1012268931569" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1012268931569" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1023%2FA:1012268931569"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11761370" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20characterisation%20of%20Hanseniaspora%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Moghaieb</div> <div class="given-names">R. E. A.</div>
</div>, <div class="name string-name">
<div class="surname">Abdelhadi</div> <div class="given-names">A. A.</div>
</div>, <div class="name string-name">
<div class="surname">El-Sadawy</div> <div class="given-names">H. A.</div>
</div>, <div class="name string-name">
<div class="surname">Allam</div> <div class="given-names">N. A. T.</div>
</div>, <div class="name string-name">
<div class="surname">Baiome</div> <div class="given-names">B. A.</div>
</div>, <div class="name string-name">
<div class="surname">Soliman</div> <div class="given-names">M. H.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Molecular identification and genetic diversity among Photorhabdus and Xenorhabdus isolates</div>, <div class="source">3 Biotech</div> , <div class="volume">7</div>, <div class="fpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20identification%20and%20genetic%20diversity%20among%20Photorhabdus%20and%20Xenorhabdus%20isolates&amp;author=R.%20E.%20A.%20Moghaieb&amp;author=A.%20A.%20Abdelhadi&amp;author=H.%20A.%20El-Sadawy&amp;author=N.%20A.%20T.%20Allam&amp;author=B.%20A.%20Baiome&amp;author=M.%20H.%20Soliman&amp;publication_year=2017&amp;journal=3%20Biotech&amp;volume=7&amp;pages=6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s13205-016-0594-4" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13205-016-0594-4" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs13205-016-0594-4"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28391470" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20identification%20and%20genetic%20diversity%20among%20Photorhabdus%20and%20Xenorhabdus%20isolates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pramateftaki</div> <div class="given-names">P. V.</div>
</div>, <div class="name string-name">
<div class="surname">Kouvelis</div> <div class="given-names">V. N.</div>
</div>, <div class="name string-name">
<div class="surname">Lanaridis</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Typas</div> <div class="given-names">M. A.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">The mitochondrial genome of the wine yeast <em>Hanseniaspora uvarum</em>: a unique genome organization among yeast/fungal counterparts</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">6</div>, <div class="fpage">77</div>–<div class="lpage">90</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20mitochondrial%20genome%20of%20the%20wine%20yeast%20Hanseniaspora%20uvarum%3A%20a%20unique%20genome%20organization%20among%20yeast%2Ffungal%20counterparts&amp;author=P.%20V.%20Pramateftaki&amp;author=V.%20N.%20Kouvelis&amp;author=P.%20Lanaridis&amp;author=M.%20A.%20Typas&amp;publication_year=2006&amp;journal=FEMS%20Yeast%20Res&amp;volume=6&amp;pages=77-90" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/fyr.2006.6.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2006.6.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffyr.2006.6.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16423073" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20mitochondrial%20genome%20of%20the%20wine%20yeast%20Hanseniaspora%20uvarum%3A%20a%20unique%20genome%20organization%20among%20yeast%2Ffungal%20counterparts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Souciet</div> <div class="given-names">J.-L.</div>
</div>, <div class="name string-name">
<div class="surname">Dujon</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Gaillardin</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Comparative genomics of protoploid Saccharomycetaceae</div>, <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1696</div>–<div class="lpage">709</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20protoploid%20Saccharomycetaceae&amp;author=J.-L.%20Souciet&amp;author=B.%20Dujon&amp;author=C.%20Gaillardin&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1696-709" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.091546.109" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.091546.109" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.091546.109"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19525356" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparative%20genomics%20of%20protoploid%20Saccharomycetaceae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Merico</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Sulo</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Piskur</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Compagno</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Fermentative lifestyle in yeasts belonging to the Saccharomyces complex</div>, <div class="source">FEBS J</div> ., <div class="volume">274</div>, <div class="fpage">976</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fermentative%20lifestyle%20in%20yeasts%20belonging%20to%20the%20Saccharomyces%20complex&amp;author=A.%20Merico&amp;author=P.%20Sulo&amp;author=J.%20Piskur&amp;author=C.%20Compagno&amp;publication_year=2007&amp;journal=FEBS%20J&amp;volume=274&amp;pages=976-89" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1742-4658.2007.05645.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2007.05645.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1742-4658.2007.05645.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17239085" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fermentative%20lifestyle%20in%20yeasts%20belonging%20to%20the%20Saccharomyces%20complex&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B56" class="ref-content ">
<div class="label">56</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cabral</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Prista</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Loureiro-Dias</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Leandro</div> <div class="given-names">M. J.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Occurrence of FFZ genes in yeasts and correlation with fructophilic behaviour</div>, <div class="source">Microbiology</div> , <div class="volume">161</div>, <div class="fpage">2008</div>–<div class="lpage">18</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Occurrence%20of%20FFZ%20genes%20in%20yeasts%20and%20correlation%20with%20fructophilic%20behaviour&amp;author=S.%20Cabral&amp;author=C.%20Prista&amp;author=M.%20C.%20Loureiro-Dias&amp;author=M.%20J.%20Leandro&amp;publication_year=2015&amp;journal=Microbiology&amp;volume=161&amp;pages=2008-18" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1099/mic.0.000154" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.000154" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1099%2Fmic.0.000154"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26253443" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Occurrence%20of%20FFZ%20genes%20in%20yeasts%20and%20correlation%20with%20fructophilic%20behaviour&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B57" class="ref-content ">
<div class="label">57</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chattopadhyay</div> <div class="given-names">M. K.</div>
</div>, <div class="name string-name">
<div class="surname">Tabor</div> <div class="given-names">C. W.</div>
</div>, <div class="name string-name">
<div class="surname">Tabor</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">Spermidine but not spermine is essential for hypusine biosynthesis and growth in <em>Saccharomyces cerevisiae</em>: spermine is converted to spermidine in vivo by the FMS1-amine oxidase</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">100</div>, <div class="fpage">13869</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Spermidine%20but%20not%20spermine%20is%20essential%20for%20hypusine%20biosynthesis%20and%20growth%20in%20Saccharomyces%20cerevisiae%3A%20spermine%20is%20converted%20to%20spermidine%20in%20vivo%20by%20the%20FMS1-amine%20oxidase&amp;author=M.%20K.%20Chattopadhyay&amp;author=C.%20W.%20Tabor&amp;author=H.%20Tabor&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=100&amp;pages=13869-74" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1835918100" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1835918100" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1835918100"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14617780" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Spermidine%20but%20not%20spermine%20is%20essential%20for%20hypusine%20biosynthesis%20and%20growth%20in%20Saccharomyces%20cerevisiae%3A%20spermine%20is%20converted%20to%20spermidine%20in%20vivo%20by%20the%20FMS1-amine%20oxidase&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B58" class="ref-content ">
<div class="label">58</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Smit</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Toit</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Toit</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Biogenic amines in wine: understanding the headache</div>, <div class="source">S. Afr. J. Enol. Vitic</div> ., <div class="volume">29</div>, <div class="fpage">109</div>–<div class="lpage">27</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Biogenic%20amines%20in%20wine%3A%20understanding%20the%20headache&amp;aulast=Smit&amp;date=2008&amp;spage=109&amp;epage=27&amp;volume=29" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biogenic%20amines%20in%20wine%3A%20understanding%20the%20headache&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B59" class="ref-content ">
<div class="label">59</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Barnett</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Payne</div> <div class="given-names">R. W.</div>
</div>, <div class="name string-name">
<div class="surname">Yarrow</div> <div class="given-names">D.</div>
</div>
</div> <div class="year">1990</div>, Yeasts: characteristics and identification.</div>
</div></div></div>
<div content-id="dsy039-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B60" class="ref-content ">
<div class="label">60</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bataillon</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Rico</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Sablayrolles</div> <div class="given-names">J.-M.</div>
</div>, <div class="name string-name">
<div class="surname">Salmon</div> <div class="given-names">J.-M.</div>
</div>, <div class="name string-name">
<div class="surname">Barre</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">1996</div>, 
<div class="article-title">Early thiamin assimilation by yeasts under enological conditions: impact on alcoholic fermentation kinetics</div>, <div class="source">J. Ferment. Bioeng</div> ., <div class="volume">82</div>, <div class="fpage">145</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Early%20thiamin%20assimilation%20by%20yeasts%20under%20enological%20conditions%3A%20impact%20on%20alcoholic%20fermentation%20kinetics&amp;author=M.%20Bataillon&amp;author=A.%20Rico&amp;author=J.-M.%20Sablayrolles&amp;author=J.-M.%20Salmon&amp;author=P.%20Barre&amp;publication_year=1996&amp;journal=J.%20Ferment.%20Bioeng&amp;volume=82&amp;pages=145-50" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/0922-338X(96)85037-9" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0922-338X(96)85037-9" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0922-338X(96)85037-9"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Early%20thiamin%20assimilation%20by%20yeasts%20under%20enological%20conditions%3A%20impact%20on%20alcoholic%20fermentation%20kinetics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B61" class="ref-content ">
<div class="label">61</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Brion</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Ambroset</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Delobel</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Sanchez</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Blondin</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Deciphering regulatory variation of THI genes in alcoholic fermentation indicate an impact of Thi3p on PDC1 expression</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1085</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Deciphering%20regulatory%20variation%20of%20THI%20genes%20in%20alcoholic%20fermentation%20indicate%20an%20impact%20of%20Thi3p%20on%20PDC1%20expression&amp;author=C.%20Brion&amp;author=C.%20Ambroset&amp;author=P.%20Delobel&amp;author=I.%20Sanchez&amp;author=B.%20Blondin&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1085" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-15-1085" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-1085" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-15-1085"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25494835" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Deciphering%20regulatory%20variation%20of%20THI%20genes%20in%20alcoholic%20fermentation%20indicate%20an%20impact%20of%20Thi3p%20on%20PDC1%20expression&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B62" class="ref-content ">
<div class="label">62</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Maier</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Volker</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Boles</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Fuhrmann</div> <div class="given-names">G. F.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">Characterisation of glucose transport in <em>Saccharomyces cerevisiae</em> with plasma membrane vesicles (countertransport) and intact cells (initial uptake) with single Hxt1, Hxt2, Hxt3, Hxt4, Hxt6, Hxt7 or Gal2 transporters</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">2</div>, <div class="fpage">539</div>–<div class="lpage">50</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterisation%20of%20glucose%20transport%20in%20Saccharomyces%20cerevisiae%20with%20plasma%20membrane%20vesicles%20(countertransport)%20and%20intact%20cells%20(initial%20uptake)%20with%20single%20Hxt1%2C%20Hxt2%2C%20Hxt3%2C%20Hxt4%2C%20Hxt6%2C%20Hxt7%20or%20Gal2%20transporters&amp;author=A.%20Maier&amp;author=B.%20Volker&amp;author=E.%20Boles&amp;author=G.%20F.%20Fuhrmann&amp;publication_year=2002&amp;journal=FEMS%20Yeast%20Res&amp;volume=2&amp;pages=539-50" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12702270" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Characterisation%20of%20glucose%20transport%20in%20Saccharomyces%20cerevisiae%20with%20plasma%20membrane%20vesicles%20(countertransport)%20and%20intact%20cells%20(initial%20uptake)%20with%20single%20Hxt1%2C%20Hxt2%2C%20Hxt3%2C%20Hxt4%2C%20Hxt6%2C%20Hxt7%20or%20Gal2%20transporters&amp;aulast=Maier&amp;date=2002&amp;spage=539&amp;epage=50&amp;volume=2" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterisation%20of%20glucose%20transport%20in%20Saccharomyces%20cerevisiae%20with%20plasma%20membrane%20vesicles%20(countertransport)%20and%20intact%20cells%20(initial%20uptake)%20with%20single%20Hxt1%2C%20Hxt2%2C%20Hxt3%2C%20Hxt4%2C%20Hxt6%2C%20Hxt7%20or%20Gal2%20transporters&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B63" class="ref-content ">
<div class="label">63</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ciani</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Comitini</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Mannazzu</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Domizio</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Controlled mixed culture fermentation: a new perspective on the use of non-Saccharomyces yeasts in winemaking</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">10</div>, <div class="fpage">123</div>–<div class="lpage">33</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Controlled%20mixed%20culture%20fermentation%3A%20a%20new%20perspective%20on%20the%20use%20of%20non-Saccharomyces%20yeasts%20in%20winemaking&amp;author=M.%20Ciani&amp;author=F.%20Comitini&amp;author=I.%20Mannazzu&amp;author=P.%20Domizio&amp;publication_year=2010&amp;journal=FEMS%20Yeast%20Res&amp;volume=10&amp;pages=123-33" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/fyr.2010.10.issue-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2010.10.issue-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffyr.2010.10.issue-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19807789" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Controlled%20mixed%20culture%20fermentation%3A%20a%20new%20perspective%20on%20the%20use%20of%20non-Saccharomyces%20yeasts%20in%20winemaking&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B64" class="ref-content ">
<div class="label">64</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ciani</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Fatichenti</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">1999</div>, 
<div class="article-title">Selective sugar consumption by apiculate yeasts</div>, <div class="source">Lett. Appl. Microbiol</div> ., <div class="volume">28</div>, <div class="fpage">203</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Selective%20sugar%20consumption%20by%20apiculate%20yeasts&amp;author=M.%20Ciani&amp;author=F.%20Fatichenti&amp;publication_year=1999&amp;journal=Lett.%20Appl.%20Microbiol&amp;volume=28&amp;pages=203-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1365-2672.1999.00505.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2672.1999.00505.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1365-2672.1999.00505.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23962066" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Selective%20sugar%20consumption%20by%20apiculate%20yeasts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B65" class="ref-content ">
<div class="label">65</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leandro</div> <div class="given-names">M. J.</div>
</div>, <div class="name string-name">
<div class="surname">Cabral</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Prista</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Loureiro-Dias</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Sychrova</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">The high-capacity specific fructose facilitator ZrFfz1 is essential for the fructophilic behavior of <em>Zygosaccharomyces rouxii</em> CBS 732T</div>, <div class="source">Eukaryot. Cell</div> , <div class="volume">13</div>, <div class="fpage">1371</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20high-capacity%20specific%20fructose%20facilitator%20ZrFfz1%20is%20essential%20for%20the%20fructophilic%20behavior%20of%20Zygosaccharomyces%20rouxii%20CBS%20732T&amp;author=M.%20J.%20Leandro&amp;author=S.%20Cabral&amp;author=C.%20Prista&amp;author=M.%20C.%20Loureiro-Dias&amp;author=H.%20Sychrova&amp;publication_year=2014&amp;journal=Eukaryot.%20Cell&amp;volume=13&amp;pages=1371-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/EC.00137-14" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00137-14" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FEC.00137-14"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25172765" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20high-capacity%20specific%20fructose%20facilitator%20ZrFfz1%20is%20essential%20for%20the%20fructophilic%20behavior%20of%20Zygosaccharomyces%20rouxii%20CBS%20732T&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B66" class="ref-content ">
<div class="label">66</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Leandro</div> <div class="given-names">M. J.</div>
</div>, <div class="name string-name">
<div class="surname">Sychrova</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Prista</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Loureiro-Dias</div> <div class="given-names">M. C.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">ZrFsy1, a high-affinity fructose/H+ symporter from fructophilic yeast <em>Zygosaccharomyces rouxii</em>
</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e68165</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ZrFsy1%2C%20a%20high-affinity%20fructose%2FH%2B%20symporter%20from%20fructophilic%20yeast%20Zygosaccharomyces%20rouxii&amp;author=M.%20J.%20Leandro&amp;author=H.%20Sychrova&amp;author=C.%20Prista&amp;author=M.%20C.%20Loureiro-Dias&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e68165" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0068165" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0068165" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0068165"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23844167" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ZrFsy1%2C%20a%20high-affinity%20fructose%2FH%2B%20symporter%20from%20fructophilic%20yeast%20Zygosaccharomyces%20rouxii&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B67" class="ref-content ">
<div class="label">67</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">De Benedictis</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Bleve</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Grieco</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Tristezza</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Tufariello</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Grieco</div> <div class="given-names">F.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">An optimized procedure for the enological selection of non-Saccharomyces starter cultures</div>, <div class="source">Antonie Van Leeuwenhoek</div> , <div class="volume">99</div>, <div class="fpage">189</div>–<div class="lpage">200</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20optimized%20procedure%20for%20the%20enological%20selection%20of%20non-Saccharomyces%20starter%20cultures&amp;author=M.%20De%20Benedictis&amp;author=G.%20Bleve&amp;author=F.%20Grieco&amp;author=M.%20Tristezza&amp;author=M.%20Tufariello&amp;author=F.%20Grieco&amp;publication_year=2011&amp;journal=Antonie%20Van%20Leeuwenhoek&amp;volume=99&amp;pages=189-200" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s10482-010-9475-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10482-010-9475-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs10482-010-9475-8"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20571862" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20optimized%20procedure%20for%20the%20enological%20selection%20of%20non-Saccharomyces%20starter%20cultures&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B68" class="ref-content ">
<div class="label">68</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goncalves</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Coelho</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Salema-Oom</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Goncalves</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Stepwise functional evolution in a fungal sugar transporter family</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">352</div>–<div class="lpage">66</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Stepwise%20functional%20evolution%20in%20a%20fungal%20sugar%20transporter%20family&amp;author=C.%20Goncalves&amp;author=M.%20A.%20Coelho&amp;author=M.%20Salema-Oom&amp;author=P.%20Goncalves&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=352-66" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msv220" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv220" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsv220"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26474848" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Stepwise%20functional%20evolution%20in%20a%20fungal%20sugar%20transporter%20family&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B69" class="ref-content ">
<div class="label">69</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zuchowska</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Jaenicke</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Konig</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Claus</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Allelic variants of hexose transporter Hxt3p and hexokinases Hxk1p/Hxk2p in strains of <em>Saccharomyces cerevisiae</em> and interspecies hybrids</div>, <div class="source">Yeast</div> , <div class="volume">32</div>, <div class="fpage">657</div>–<div class="lpage">69</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Allelic%20variants%20of%20hexose%20transporter%20Hxt3p%20and%20hexokinases%20Hxk1p%2FHxk2p%20in%20strains%20of%20Saccharomyces%20cerevisiae%20and%20interspecies%20hybrids&amp;author=M.%20Zuchowska&amp;author=E.%20Jaenicke&amp;author=H.%20Konig&amp;author=H.%20Claus&amp;publication_year=2015&amp;journal=Yeast&amp;volume=32&amp;pages=657-69" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/yea.v32.11" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fyea.v32.11" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fyea.v32.11"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26202678" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Allelic%20variants%20of%20hexose%20transporter%20Hxt3p%20and%20hexokinases%20Hxk1p%2FHxk2p%20in%20strains%20of%20Saccharomyces%20cerevisiae%20and%20interspecies%20hybrids&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B70" class="ref-content ">
<div class="label">70</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Berthels</div> <div class="given-names">N. J.</div>
</div>, <div class="name string-name">
<div class="surname">Cordero Otero</div> <div class="given-names">R. R.</div>
</div>, <div class="name string-name">
<div class="surname">Bauer</div> <div class="given-names">F. F.</div>
</div>, <div class="name string-name">
<div class="surname">Pretorius</div> <div class="given-names">I. S.</div>
</div>, <div class="name string-name">
<div class="surname">Thevelein</div> <div class="given-names">J. M.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Correlation between glucose/fructose discrepancy and hexokinase kinetic properties in different <em>Saccharomyces cerevisiae</em> wine yeast strains</div>, <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">77</div>, <div class="fpage">1083</div>–<div class="lpage">91</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20glucose%2Ffructose%20discrepancy%20and%20hexokinase%20kinetic%20properties%20in%20different%20Saccharomyces%20cerevisiae%20wine%20yeast%20strains&amp;author=N.%20J.%20Berthels&amp;author=R.%20R.%20Cordero%20Otero&amp;author=F.%20F.%20Bauer&amp;author=I.%20S.%20Pretorius&amp;author=J.%20M.%20Thevelein&amp;publication_year=2008&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=77&amp;pages=1083-91" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00253-007-1231-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-007-1231-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00253-007-1231-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17955190" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Correlation%20between%20glucose%2Ffructose%20discrepancy%20and%20hexokinase%20kinetic%20properties%20in%20different%20Saccharomyces%20cerevisiae%20wine%20yeast%20strains&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B71" class="ref-content ">
<div class="label">71</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Sutterlin</div> <div class="given-names">K. A.</div>
</div></div> <div class="year">2010</div>, Fructophilic yeasts to cure stuck fermentations in alcoholic beverages. PhD thesis, Viticulture and Oenology, University of Stellenbosch.</div>
</div></div></div>
<div content-id="dsy039-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B72" class="ref-content ">
<div class="label">72</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Viana</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Gil</div> <div class="given-names">J. V.</div>
</div>, <div class="name string-name">
<div class="surname">Genoves</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Valles</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Manzanares</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Rational selection of non-<em>Saccharomyces</em> wine yeasts for mixed starters based on ester formation and enological traits</div>, <div class="source">Food Microbiol</div> ., <div class="volume">25</div>, <div class="fpage">778</div>–<div class="lpage">85</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rational%20selection%20of%20non-Saccharomyces%20wine%20yeasts%20for%20mixed%20starters%20based%20on%20ester%20formation%20and%20enological%20traits&amp;author=F.%20Viana&amp;author=J.%20V.%20Gil&amp;author=S.%20Genoves&amp;author=S.%20Valles&amp;author=P.%20Manzanares&amp;publication_year=2008&amp;journal=Food%20Microbiol&amp;volume=25&amp;pages=778-85" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.fm.2008.04.015" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fm.2008.04.015" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.fm.2008.04.015"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18620969" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rational%20selection%20of%20non-Saccharomyces%20wine%20yeasts%20for%20mixed%20starters%20based%20on%20ester%20formation%20and%20enological%20traits&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B73" class="ref-content ">
<div class="label">73</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Becker</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Schmitt</div> <div class="given-names">M. J.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Yeast killer toxin K28: biology and unique strategy of host cell intoxication and killing</div>, <div class="source">Toxins</div> , <div class="volume">9</div>, <div class="fpage">73</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Yeast%20killer%20toxin%20K28%3A%20biology%20and%20unique%20strategy%20of%20host%20cell%20intoxication%20and%20killing&amp;author=B.%20Becker&amp;author=M.%20J.%20Schmitt&amp;publication_year=2017&amp;journal=Toxins&amp;volume=9&amp;pages=73" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3390/toxins9100333" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Ftoxins9100333" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3390%2Ftoxins9100333"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Yeast%20killer%20toxin%20K28%3A%20biology%20and%20unique%20strategy%20of%20host%20cell%20intoxication%20and%20killing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B74" class="ref-content ">
<div class="label">74</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Novo</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Bigey</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Beyne</div> <div class="given-names">E.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Eukaryote-to-eukaryote gene transfer events revealed by the genome sequence of the wine yeast <em>Saccharomyces cerevisiae</em> EC1118</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">106</div>, <div class="fpage">16333</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Eukaryote-to-eukaryote%20gene%20transfer%20events%20revealed%20by%20the%20genome%20sequence%20of%20the%20wine%20yeast%20Saccharomyces%20cerevisiae%20EC1118&amp;author=M.%20Novo&amp;author=F.%20Bigey&amp;author=E.%20Beyne&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=106&amp;pages=16333-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0904673106" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0904673106" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0904673106"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19805302" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Eukaryote-to-eukaryote%20gene%20transfer%20events%20revealed%20by%20the%20genome%20sequence%20of%20the%20wine%20yeast%20Saccharomyces%20cerevisiae%20EC1118&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B75" class="ref-content ">
<div class="label">75</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lopez</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Mateo</div> <div class="given-names">J. J.</div>
</div>, <div class="name string-name">
<div class="surname">Maicas</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Screening of beta-glucosidase and beta-xylosidase activities in four non-Saccharomyces yeast isolates</div>, <div class="source">J. Food Sci</div> ., <div class="volume">80</div>, <div class="fpage">C1696</div>–<div class="lpage">704</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Screening%20of%20beta-glucosidase%20and%20beta-xylosidase%20activities%20in%20four%20non-Saccharomyces%20yeast%20isolates&amp;author=M.%20C.%20Lopez&amp;author=J.%20J.%20Mateo&amp;author=S.%20Maicas&amp;publication_year=2015&amp;journal=J.%20Food%20Sci&amp;volume=80&amp;pages=C1696-704" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/1750-3841.12954" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1750-3841.12954" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2F1750-3841.12954"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26126488" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Screening%20of%20beta-glucosidase%20and%20beta-xylosidase%20activities%20in%20four%20non-Saccharomyces%20yeast%20isolates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B76" class="ref-content ">
<div class="label">76</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Padilla</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Gil</div> <div class="given-names">J. V.</div>
</div>, <div class="name string-name">
<div class="surname">Manzanares</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Past and future of non-saccharomyces yeasts: from spoilage microorganisms to biotechnological tools for improving wine aroma complexity</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">411</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Past%20and%20future%20of%20non-saccharomyces%20yeasts%3A%20from%20spoilage%20microorganisms%20to%20biotechnological%20tools%20for%20improving%20wine%20aroma%20complexity&amp;author=B.%20Padilla&amp;author=J.%20V.%20Gil&amp;author=P.%20Manzanares&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=411" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27065975" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Past%20and%20future%20of%20non-saccharomyces%20yeasts%3A%20from%20spoilage%20microorganisms%20to%20biotechnological%20tools%20for%20improving%20wine%20aroma%20complexity&amp;aulast=Padilla&amp;date=2016&amp;spage=411&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Past%20and%20future%20of%20non-saccharomyces%20yeasts%3A%20from%20spoilage%20microorganisms%20to%20biotechnological%20tools%20for%20improving%20wine%20aroma%20complexity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B77" class="ref-content ">
<div class="label">77</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rodriguez-Naranjo</div> <div class="given-names">M. I.</div>
</div>, <div class="name string-name">
<div class="surname">Ordonez</div> <div class="given-names">J. L.</div>
</div>, <div class="name string-name">
<div class="surname">Callejon</div> <div class="given-names">R. M.</div>
</div>, <div class="name string-name">
<div class="surname">Cantos-Villar</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Garcia-Parrilla</div> <div class="given-names">M. C.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Melatonin is formed during winemaking at safe levels of biogenic amines</div>, <div class="source">Food Chem. Toxicol</div> ., <div class="volume">57</div>, <div class="fpage">140</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Melatonin%20is%20formed%20during%20winemaking%20at%20safe%20levels%20of%20biogenic%20amines&amp;author=M.%20I.%20Rodriguez-Naranjo&amp;author=J.%20L.%20Ordonez&amp;author=R.%20M.%20Callejon&amp;author=E.%20Cantos-Villar&amp;author=M.%20C.%20Garcia-Parrilla&amp;publication_year=2013&amp;journal=Food%20Chem.%20Toxicol&amp;volume=57&amp;pages=140-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.fct.2013.03.014" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fct.2013.03.014" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.fct.2013.03.014"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23531627" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Melatonin%20is%20formed%20during%20winemaking%20at%20safe%20levels%20of%20biogenic%20amines&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B78" class="ref-content ">
<div class="label">78</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Granchi</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Romano</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Mangani</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Production of biogenic amines by wine microorganisms</div>, <div class="source">Bull. O.I.V</div> ., <div class="volume">78</div>, <div class="fpage">595</div>–<div class="lpage">609</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Production%20of%20biogenic%20amines%20by%20wine%20microorganisms&amp;aulast=Granchi&amp;date=2005&amp;spage=595&amp;epage=609&amp;volume=78" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Production%20of%20biogenic%20amines%20by%20wine%20microorganisms&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B79" class="ref-content ">
<div class="label">79</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gasch</div> <div class="given-names">A. P.</div>
</div>, <div class="name string-name">
<div class="surname">Spellman</div> <div class="given-names">P. T.</div>
</div>, <div class="name string-name">
<div class="surname">Kao</div> <div class="given-names">C. M.</div>
</div>
</div>, et al.  <div class="year">2000</div>, 
<div class="article-title">Genomic expression programs in the response of yeast cells to environmental changes</div>, <div class="source">Mol. Biol. Cell</div> , <div class="volume">11</div>, <div class="fpage">4241</div>–<div class="lpage">57</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomic%20expression%20programs%20in%20the%20response%20of%20yeast%20cells%20to%20environmental%20changes&amp;author=A.%20P.%20Gasch&amp;author=P.%20T.%20Spellman&amp;author=C.%20M.%20Kao&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Cell&amp;volume=11&amp;pages=4241-57" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1091/mbc.11.12.4241" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.11.12.4241" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1091%2Fmbc.11.12.4241"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11102521" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomic%20expression%20programs%20in%20the%20response%20of%20yeast%20cells%20to%20environmental%20changes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B80" class="ref-content ">
<div class="label">80</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mira</div> <div class="given-names">N. P.</div>
</div>, <div class="name string-name">
<div class="surname">Munsterkotter</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Dias-Valada</div> <div class="given-names">F.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">The genome sequence of the highly acetic acid-tolerant <em>Zygosaccharomyces bailii</em>-derived interspecies hybrid strain ISA1307, isolated from a sparkling wine plant</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">299</div>–<div class="lpage">313</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20highly%20acetic%20acid-tolerant%20Zygosaccharomyces%20bailii-derived%20interspecies%20hybrid%20strain%20ISA1307%2C%20isolated%20from%20a%20sparkling%20wine%20plant&amp;author=N.%20P.%20Mira&amp;author=M.%20Munsterkotter&amp;author=F.%20Dias-Valada&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=299-313" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dst058" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst058" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdst058"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24453040" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20genome%20sequence%20of%20the%20highly%20acetic%20acid-tolerant%20Zygosaccharomyces%20bailii-derived%20interspecies%20hybrid%20strain%20ISA1307%2C%20isolated%20from%20a%20sparkling%20wine%20plant&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B81" class="ref-content ">
<div class="label">81</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Gasch</div> <div class="given-names">A. P.</div>
</div></div> <div class="year">2007</div>, 
<div class="article-title">Comparative genomics of the environmental stress response in ascomycete fungi</div>, <div class="source">Yeast</div> , <div class="volume">24</div>, <div class="fpage">961</div>–<div class="lpage">76</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20the%20environmental%20stress%20response%20in%20ascomycete%20fungi&amp;author=A.%20P.%20Gasch&amp;publication_year=2007&amp;journal=Yeast&amp;volume=24&amp;pages=961-76" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2F(ISSN)1097-0061"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17605132" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparative%20genomics%20of%20the%20environmental%20stress%20response%20in%20ascomycete%20fungi&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B82" class="ref-content ">
<div class="label">82</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Toone</div> <div class="given-names">W. M.</div>
</div>, <div class="name string-name">
<div class="surname">Mata</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2003</div>, 
<div class="article-title">Global transcriptional responses of fission yeast to environmental stress</div>,<div class="source">. Mol. Biol. Cell</div> , <div class="volume">14</div>, <div class="fpage">214</div>–<div class="lpage">29</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptional%20responses%20of%20fission%20yeast%20to%20environmental%20stress&amp;author=D.%20Chen&amp;author=W.%20M.%20Toone&amp;author=J.%20Mata&amp;publication_year=2003&amp;journal=.%20Mol.%20Biol.%20Cell&amp;volume=14&amp;pages=214-29" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1091/mbc.e02-08-0499" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.e02-08-0499" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1091%2Fmbc.e02-08-0499"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12529438" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Global%20transcriptional%20responses%20of%20fission%20yeast%20to%20environmental%20stress&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B83" class="ref-content ">
<div class="label">83</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Brion</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Pflieger</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Souali-Crespo</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Friedrich</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Schacherer</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">Differences in environmental stress response among yeasts is consistent with species-specific lifestyles</div>, <div class="source">Mol. Biol. Cell</div> , <div class="volume">27</div>, <div class="fpage">1694</div>–<div class="lpage">705</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20environmental%20stress%20response%20among%20yeasts%20is%20consistent%20with%20species-specific%20lifestyles&amp;author=C.%20Brion&amp;author=D.%20Pflieger&amp;author=S.%20Souali-Crespo&amp;author=A.%20Friedrich&amp;author=J.%20Schacherer&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Cell&amp;volume=27&amp;pages=1694-705" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1091/mbc.E15-12-0816" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.E15-12-0816" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1091%2Fmbc.E15-12-0816"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27009200" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Differences%20in%20environmental%20stress%20response%20among%20yeasts%20is%20consistent%20with%20species-specific%20lifestyles&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B84" class="ref-content ">
<div class="label">84</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Teixeira</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Raposo</div> <div class="given-names">L. R.</div>
</div>, <div class="name string-name">
<div class="surname">Mira</div> <div class="given-names">N. P.</div>
</div>, <div class="name string-name">
<div class="surname">Lourenco</div> <div class="given-names">A. B.</div>
</div>, <div class="name string-name">
<div class="surname">Sa-Correia</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Genome-wide identification of <em>Saccharomyces cerevisiae</em> genes required for maximal tolerance to ethanol</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">75</div>, <div class="fpage">5761</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20Saccharomyces%20cerevisiae%20genes%20required%20for%20maximal%20tolerance%20to%20ethanol&amp;author=M.%20C.%20Teixeira&amp;author=L.%20R.%20Raposo&amp;author=N.%20P.%20Mira&amp;author=A.%20B.%20Lourenco&amp;author=I.%20Sa-Correia&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=75&amp;pages=5761-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.00845-09" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00845-09" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.00845-09"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19633105" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20identification%20of%20Saccharomyces%20cerevisiae%20genes%20required%20for%20maximal%20tolerance%20to%20ethanol&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B85" class="ref-content ">
<div class="label">85</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yoshikawa</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Tanaka</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Furusawa</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Nagahisa</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Hirasawa</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Shimizu</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Comprehensive phenotypic analysis for identification of genes affecting growth under ethanol stress in <em>Saccharomyces cerevisiae</em>
</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">9</div>, <div class="fpage">32</div>–<div class="lpage">44</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20phenotypic%20analysis%20for%20identification%20of%20genes%20affecting%20growth%20under%20ethanol%20stress%20in%20Saccharomyces%20cerevisiae&amp;author=K.%20Yoshikawa&amp;author=T.%20Tanaka&amp;author=C.%20Furusawa&amp;author=K.%20Nagahisa&amp;author=T.%20Hirasawa&amp;author=H.%20Shimizu&amp;publication_year=2009&amp;journal=FEMS%20Yeast%20Res&amp;volume=9&amp;pages=32-44" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/fyr.2009.9.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2009.9.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffyr.2009.9.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19054128" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comprehensive%20phenotypic%20analysis%20for%20identification%20of%20genes%20affecting%20growth%20under%20ethanol%20stress%20in%20Saccharomyces%20cerevisiae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B86" class="ref-content ">
<div class="label">86</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Teixeira</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Raposo</div> <div class="given-names">L. R.</div>
</div>, <div class="name string-name">
<div class="surname">Palma</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Sa-Correia</div> <div class="given-names">I.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Identification of genes required for maximal tolerance to high-glucose concentrations, as those present in industrial alcoholic fermentation media, through a chemogenomics approach</div>, <div class="source">OMICS</div> , <div class="volume">14</div>, <div class="fpage">201</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20required%20for%20maximal%20tolerance%20to%20high-glucose%20concentrations%2C%20as%20those%20present%20in%20industrial%20alcoholic%20fermentation%20media%2C%20through%20a%20chemogenomics%20approach&amp;author=M.%20C.%20Teixeira&amp;author=L.%20R.%20Raposo&amp;author=M.%20Palma&amp;author=I.%20Sa-Correia&amp;publication_year=2010&amp;journal=OMICS&amp;volume=14&amp;pages=201-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1089/omi.2009.0149" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fomi.2009.0149" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1089%2Fomi.2009.0149"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20210661" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20genes%20required%20for%20maximal%20tolerance%20to%20high-glucose%20concentrations%2C%20as%20those%20present%20in%20industrial%20alcoholic%20fermentation%20media%2C%20through%20a%20chemogenomics%20approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B87" class="ref-content ">
<div class="label">87</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gonzalez</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Morales</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Tronchoni</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">New genes involved in osmotic stress tolerance in <em>Saccharomyces cerevisiae</em>
</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">7</div>, <div class="fpage">1545</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=New%20genes%20involved%20in%20osmotic%20stress%20tolerance%20in%20Saccharomyces%20cerevisiae&amp;author=R.%20Gonzalez&amp;author=P.%20Morales&amp;author=J.%20Tronchoni&amp;publication_year=2016&amp;journal=Front.%20Microbiol&amp;volume=7&amp;pages=1545" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27733850" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=New%20genes%20involved%20in%20osmotic%20stress%20tolerance%20in%20Saccharomyces%20cerevisiae&amp;aulast=Gonzalez&amp;date=2016&amp;spage=1545&amp;volume=7" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20genes%20involved%20in%20osmotic%20stress%20tolerance%20in%20Saccharomyces%20cerevisiae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B88" class="ref-content ">
<div class="label">88</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Walker</div> <div class="given-names">M. E.</div>
</div>, <div class="name string-name">
<div class="surname">Nguyen</div> <div class="given-names">T. D.</div>
</div>, <div class="name string-name">
<div class="surname">Liccioli</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide identification of the Fermentome; genes required for successful and timely completion of wine-like fermentation by <em>Saccharomyces cerevisiae</em>
</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">552</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20the%20Fermentome%3B%20genes%20required%20for%20successful%20and%20timely%20completion%20of%20wine-like%20fermentation%20by%20Saccharomyces%20cerevisiae&amp;author=M.%20E.%20Walker&amp;author=T.%20D.%20Nguyen&amp;author=T.%20Liccioli&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=552" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-15-552" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-552" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-15-552"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24993029" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20identification%20of%20the%20Fermentome%3B%20genes%20required%20for%20successful%20and%20timely%20completion%20of%20wine-like%20fermentation%20by%20Saccharomyces%20cerevisiae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B89" class="ref-content ">
<div class="label">89</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Novo</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Mangado</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Quiros</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Morales</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Salvado</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Gonzalez</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Genome-wide study of the adaptation of <em>Saccharomyces cerevisiae</em> to the early stages of wine fermentation</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e74086</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20study%20of%20the%20adaptation%20of%20Saccharomyces%20cerevisiae%20to%20the%20early%20stages%20of%20wine%20fermentation&amp;author=M.%20Novo&amp;author=A.%20Mangado&amp;author=M.%20Quiros&amp;author=P.%20Morales&amp;author=Z.%20Salvado&amp;author=R.%20Gonzalez&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e74086" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0074086" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0074086" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0074086"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24040173" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20study%20of%20the%20adaptation%20of%20Saccharomyces%20cerevisiae%20to%20the%20early%20stages%20of%20wine%20fermentation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B90" class="ref-content ">
<div class="label">90</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Romano</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Suzzi</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Zironi</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Comi</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">1993</div>, 
<div class="article-title">Biometric study of acetoin production in <em>Hanseniaspora guilliermondii</em> and <em>Kloeckera apiculata</em>
</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">59</div>, <div class="fpage">1838</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Biometric%20study%20of%20acetoin%20production%20in%20Hanseniaspora%20guilliermondii%20and%20Kloeckera%20apiculata&amp;author=P.%20Romano&amp;author=G.%20Suzzi&amp;author=R.%20Zironi&amp;author=G.%20Comi&amp;publication_year=1993&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=59&amp;pages=1838-41" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16348961" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Biometric%20study%20of%20acetoin%20production%20in%20Hanseniaspora%20guilliermondii%20and%20Kloeckera%20apiculata&amp;aulast=Romano&amp;date=1993&amp;spage=1838&amp;epage=41&amp;volume=59" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biometric%20study%20of%20acetoin%20production%20in%20Hanseniaspora%20guilliermondii%20and%20Kloeckera%20apiculata&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy039-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy039-B91" class="ref-content ">
<div class="label">91</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Alamri</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Hashem</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Alrumman</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Al-Qahtani</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Enhancement of bio-ethanol production from date molasses by non-conventional yeasts</div>, <div class="source">Res. J. Microbiol</div> ., <div class="volume">10</div>, <div class="fpage">114</div>–<div class="lpage">25</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Enhancement%20of%20bio-ethanol%20production%20from%20date%20molasses%20by%20non-conventional%20yeasts&amp;author=S.%20Alamri&amp;author=M.%20Hashem&amp;author=S.%20Alrumman&amp;author=M.%20Al-Qahtani&amp;publication_year=2015&amp;journal=Res.%20J.%20Microbiol&amp;volume=10&amp;pages=114-25" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3923/jm.2015.114.125" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3923%2Fjm.2015.114.125" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3923%2Fjm.2015.114.125"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Enhancement%20of%20bio-ethanol%20production%20from%20date%20molasses%20by%20non-conventional%20yeasts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5193709&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_figures.pdf?Expires=1576700340&amp;Signature=DHDPQSFCX2ZqkEyxOGkMUYDfPhyL4p3RnHg~unGThl3jDTpVdkNrBbWUS9GfdQv-dmTvAkvF3-EZQUePNKPnCJKYSdeJk8nz5TEbO5gQ5Lo7~JFzg3hSm~xnW1z7DyMMssN33nx1fKb7Z70Xl0tPqdqPQnHau5GkgDIjls73z2xpqcW7g6pFhu2RfhcyvheIruz9S3ZRdu7J5h8g9rzA~kEor3E4QVY3yLwXzS2OyNt4b-TP8o9TtoTmblPqdt3i7aSrCnYLjGdGVbEdGdfi8dN6woBs5Ti6Dij5aJeP7V4TlC16aCXVFk5lIoDEQhdsgza2nt4l36Dd85mOZhe3ug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Figures</div></a> - pdf file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s1.xlsx?Expires=1576700340&amp;Signature=JCBOlzOEHNhHi5IOsPkQzMhJgR7PuvR5JeYWk88fOS7frfTBQ2DzCR7x1XeuY3y8nVD8v7RJOwlsvXo8cDlitqF1R7GMkph9fX5546IvgUxSiRUELrHZuuKc0lF1zHDEgLVsu0e~s7NFYeUrSK4loPnx2EXvfXwlL5Vh~27vfPZPWtr44DNC~aBQxcebcoAFyEi4DvI0k15S1LGCiQoPpduZYsAElHZNR2gBsitSyRR9KyLy-KnEqGIqzQTji34oJs-Qs9mPHbbAkejUapt7cHHGJoxAIy1y-TffrKlX7mu9WtG6m8czS99YqKk8XA4mtR9GEzMniLWHMzMz6agzeQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S1</div></a> - xlsx file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s2.xlsx?Expires=1576700340&amp;Signature=q1i0IJjuTNbfRfPbqaNzeYP77KBlOo1hKbs~b2q1iKrmqPXDgtVwd13NvY3j0m-oLk4xSRVyPac~xxSb0Q-XSvR6gM9lVURCJNRgr2CW65w8DZPFNmyhDuX~zCdX-D0kpjHlMQQwng87ZyhTlKsb-wCeMzP-DT-UfB7iRepmKjDF6roRIUeVqTtIDwTsaRWwfu-TjmNHdGxzyegXETk6YWdUs8kXv~1wp2Wgl5aR0YvyQlYYi3hR94HOwEwlZ9cOyiypWckbl66uYLWAu7apR-sTDwKrP4bgo3UTBcQyodBqf1g8esReJar645LOkv-c4RZjaEW70KsHTDmJ5ZzDqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S2</div></a> - xlsx file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s3.xlsx?Expires=1576700340&amp;Signature=KhFsIBazfnYxL3byJOgHY55KylVa~~mDam4WPGNkoNNI2JXSwsEqCfTHSokld0Zi5EdEHWkxfd2px0Gdo8mSc65y7gxzRI0NmYBAW80O8Qcvoh08VaFaFvXkKIJm0R~pdjf1lc5iii3tDcz3-jdx~tV0wREHZz8Gl-hSVwwlNB69fg-Y8bVQGp6DIyGROdJf0ptExJEYmUEgfJsWQ6mw1VM4zhWo-RbS2WTRl15umRNPV7csJl7SRGM3i0NBlxg5-i8g~4Ooa~9vNjkrsu0Id9lAqZ61qvVZDnduU~rY7j2Rb3p-JABZhx8w70MJb0d5leQ9-1BAqR5jJQQnjk3clw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S3</div></a> - xlsx file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s4.pdf?Expires=1576700340&amp;Signature=5L3vkIXVoL0uiPdaod4jP4jet8BzKhHcuBHTkH1FW-D4hZkX6MgnCJ0DvvuZHu6DWNECtzfohXRu3p37ANYfznH~hclEvx7PkpTQKedWh05qR1NLrQfce2tV9E63gxAhvtwoK2rA4M0Rlwff2Ldy9XF9MSlgN3~vS1T~VJYMy~9SnvqonghoqYhSJTWcjn2rX~vkureigvq0P5a9~iogs6UMgc1YJPMUiirPJWg2d3b1Rdatl1w5j7Dlh2xbbwZc1dZhP0waG8VdUjG2y7VGiq9mA9pz2zLQmOMO-jP5UL6b1QXfwi~1SvlKy4l9UdWZsg-GG1vY5nhr7fQucnYOiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S4</div></a> - pdf file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s5.xlsx?Expires=1576700340&amp;Signature=iuMiIhLcFlA~ENCg9YW8bTFH-ZWE0bIx2x2a9okPtweRkemg29DXB0V7var0ChNVJ5TEVUEU4DBNR531ayL68135-jJDviOVf65JdNp28M1JImelVq1b0O8YTzwa~PbyxLvfw0r~4m4E8BTvd7stEcsRPd70Lv8KB84JXirRv3-qg8EH4mxCfA07QNne845iOELB81v6nbKxyRpSR42UK1kfY0p12UPDPG~SEWmXFUVLbgKQUwx82vLO3~8lG9USI4DZN3HucJw4yPDp04iItq5gqXJAbzIotuRuQ-aSBNVOu2dPN5VRxo1xa0svTFGOR9KKqWUDCTG9GKqPR2Q1ww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S5</div></a> - xlsx file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s6.xlsx?Expires=1576700340&amp;Signature=h27afdh2X~0ylDBcf7da~kYCmxberb4uhtJHb96kPYLSqCuv~0JAXlISxxGXIL2rY59LckuWY0ZRybczkV9Ijc1Tw1N2mVrAkGoaI9sY11v9MNoKLKg3Mwu94GrZfBq3~2rKSxhqZ8Yqx769gjh4KzwPAEasPOH6m8oSuwtdHkCNGmZvWhVBnWoQ0~glan9ArFaiHbLTUE4j8kF7OFBQJtuGF~hdnkWfTrzxJ7wy3xmPgv9xrWcE05Ym96F~MDG4tRI3scz0lBIw~Xqsby12IE8h2XVGaDPtQsGaZvTt5A-uGvGXnFP8w3It44tG1zgi3ftY9wNbNoDb4vLfgQKGuQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S6</div></a> - xlsx file</div>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/1/10.1093_dnares_dsy039/2/dsy039_supplementary_table_s7.xlsx?Expires=1576700340&amp;Signature=bF~azqpUdzh317Omg2z85PLe3XOgfH5FwddK21l-0z4f1YnMaFfutOu4jWLNG8wzx8RUBLWPNjB-f-Xomq27CXU9GyIyeAkc-RpljLlcaV5aB5INXg5pfRAumlqzpg158MwnkYOyWsrKu5rUBl296VVSQzAm2wOWQPz3KUkyuraN2y-8-ev8m1sTPFru4Ylv-JSUAWSVp3SSbT~q~WnC1dFUP8K9JAl4KRR7Ar6aJY04IbUm067vrNBx1TAYZye5YDccdDFMksvzacVZw04mFPEkhtofoO7Yd4qPvmWQ7wTYcBmadikDqteBGBFbImLOSxAlY4eNQKvcq2QpH802BA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Table S7</div></a> - xlsx file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,513</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy039"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5193709"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f1%2f67%2f5193709">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000460615800007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F1%2F67%2F5193709" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5193709">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="Genome sequence of the non-conventional wine yeast Hanseniaspora guilliermondii UTAD222 unveils relevant traits of this species and of the Hanseniaspora genus in the context of wine fermentation | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy039");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f1%2f67%2f5193709">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
