<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/4/327/5525352","@type":"ScholarlyArticle","name":"SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes","datePublished":"2019-06-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2019-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsz012","keywords":["Mycoplasma","transposon","regulatory region","transformation efficiency","essentiality"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/4/327/5525352","author":[{"name":"Montero-Blay, Ariadna","affiliation":"Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain","@type":"Person"},{"name":"Miravet-Verde, Samuel","affiliation":"Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain","@type":"Person"},{"name":"Lluch-Senar, Maria","affiliation":"Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain","@type":"Person"},{"name":"Piñero-Lambea, Carlos","affiliation":"Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain","@type":"Person","sameAs":"http://orcid.org/0000-0002-5263-3702"},{"name":"Serrano, Luis","affiliation":"Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain  Universitat Pompeu Fabra (UPF), Barcelona, Spain  ICREA, Pg. Lluis Companys 23, Barcelona, Spain","@type":"Person"}],"description":"Abstract.  Mycoplasmas are important model organisms for Systems and Synthetic Biology, and are pathogenic to a wide variety of species. Despite their relevance","pageStart":"327","pageEnd":"339","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636238942&Signature=Dj8ysXoNI2BWFImfNW~h91vvbXM6g4uhpztZd4kw9Fs3K7yrM97QvbQ7fj5ZJScKc1nrm0dztDsL1cblx80Rtd4sIBY~BDhLs1S2AJ5A4up37itiDdJsE3DEkvfgJcB0F6hSI4i3ZdUqtxogourKzixmJ3bT2JjLkX1a5t32x~bjCcU4ykhIPl5qbqW8cA9ydQAH095Y95xqr5afqV2tVy8ZpAUUzjnLd3LaiuWkjW198qXAoWDDEwhL32cEZAaYW4Xf4w-9X~oTQN2QfJ77zzbi7wqJjdatIIDitAWX3tGzAOeGPENbB~8F4vxMQZNHxbdVUIh~VIZkmnhUBPvN1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636238942&Signature=Dj8ysXoNI2BWFImfNW~h91vvbXM6g4uhpztZd4kw9Fs3K7yrM97QvbQ7fj5ZJScKc1nrm0dztDsL1cblx80Rtd4sIBY~BDhLs1S2AJ5A4up37itiDdJsE3DEkvfgJcB0F6hSI4i3ZdUqtxogourKzixmJ3bT2JjLkX1a5t32x~bjCcU4ykhIPl5qbqW8cA9ydQAH095Y95xqr5afqV2tVy8ZpAUUzjnLd3LaiuWkjW198qXAoWDDEwhL32cEZAaYW4Xf4w-9X~oTQN2QfJ77zzbi7wqJjdatIIDitAWX3tGzAOeGPENbB~8F4vxMQZNHxbdVUIh~VIZkmnhUBPvN1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes">
<meta property="og:description" content="Abstract.  Mycoplasmas are important model organisms for Systems and Synthetic Biology, and are pathogenic to a wide variety of species. Despite their relevance">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/4/327/5525352">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1636238942&amp;Signature=Dj8ysXoNI2BWFImfNW~h91vvbXM6g4uhpztZd4kw9Fs3K7yrM97QvbQ7fj5ZJScKc1nrm0dztDsL1cblx80Rtd4sIBY~BDhLs1S2AJ5A4up37itiDdJsE3DEkvfgJcB0F6hSI4i3ZdUqtxogourKzixmJ3bT2JjLkX1a5t32x~bjCcU4ykhIPl5qbqW8cA9ydQAH095Y95xqr5afqV2tVy8ZpAUUzjnLd3LaiuWkjW198qXAoWDDEwhL32cEZAaYW4Xf4w-9X~oTQN2QfJ77zzbi7wqJjdatIIDitAWX3tGzAOeGPENbB~8F4vxMQZNHxbdVUIh~VIZkmnhUBPvN1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-06-30">
<meta name="citation_author" content="Montero-Blay, Ariadna">
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain">
<meta name="citation_author" content="Miravet-Verde, Samuel">
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain">
<meta name="citation_author" content="Lluch-Senar, Maria">
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain">
<meta name="citation_author" content="Piñero-Lambea, Carlos">
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain">
<meta name="citation_author" content="Serrano, Luis">
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain">
<meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF), Barcelona, Spain">
<meta name="citation_author_institution" content="ICREA, Pg. Lluis Companys 23, Barcelona, Spain ">
<meta name="citation_title" content="SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes">
<meta name="citation_firstpage" content="327">
<meta name="citation_lastpage" content="339">
<meta name="citation_doi" content="10.1093/dnares/dsz012">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="4">
<meta name="citation_publication_date" content="2019/08/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Phylogenetic analysis of the mycoplasmas;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1980;  citation_volume=77;  citation_pages=494-8; ">
<meta name="citation_reference" content="citation_title=Complete sequence analysis of the genome of the bacterium Mycoplasma pneumoniae;  citation_journal_title=Nucleic Acids Res;  citation_year=1996;  citation_volume=24;  citation_pages=4420-49; ">
<meta name="citation_reference" content="citation_title=Transcriptome complexity in a genome-reduced bacterium;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1268-71; ">
<meta name="citation_reference" content="citation_title=Quantification of mRNA and protein and integration with protein turnover in a bacterium;  citation_journal_title=Mol. Syst. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=511">
<meta name="citation_reference" content="citation_title=Comprehensive methylome characterization of Mycoplasma genitalium and Mycoplasma pneumoniae at single-base resolution;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003191">
<meta name="citation_reference" content="citation_title=Defining a minimal cell: essentiality of small ORFs and ncRNAs in a genome-reduced bacterium;  citation_journal_title=Mol. Syst. Biol;  citation_year=2015;  citation_volume=11;  citation_pages=780">
<meta name="citation_reference" content="citation_title=A whole-cell computational model predicts phenotype from genotype;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=150;  citation_pages=389-401; ">
<meta name="citation_reference" content="citation_title=Essential genes of a minimal bacterium;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=425-30; ">
<meta name="citation_reference" content="citation_title=Large-scale transposon mutagenesis of Mycoplasma pulmonis;  citation_journal_title=Mol. Microbiol;  citation_year=2008;  citation_volume=69;  citation_pages=67-76; ">
<meta name="citation_reference" content="citation_title=Genes found essential in other mycoplasmas are dispensable in Mycoplasma bovis;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e97100">
<meta name="citation_reference" content="citation_title=Design and synthesis of a minimal bacterial genome;  citation_journal_title=Science;  citation_year=2016;  citation_volume=351;  citation_pages=aad6253">
<meta name="citation_reference" content="citation_title=Mycoplasma pneumoniae and its role as a human pathogen;  citation_journal_title=Clin. Microbiol. Rev;  citation_year=2004;  citation_volume=17;  citation_pages=697-728; ">
<meta name="citation_reference" content="citation_title=Avian mycoplasmosis (Mycoplasma gallisepticum);  citation_journal_title=Rev. Sci. Technol;  citation_year=2000;  citation_volume=19;  citation_pages=425-42; ">
<meta name="citation_reference" content="citation_title=Mycoplasma agalactiae, an etiological agent of contagious agalactia in small ruminants: a review;  citation_journal_title=Vet. Med. Int;  citation_year=2014;  citation_volume=2014;  citation_pages=1">
<meta name="citation_reference" content="citation_title=Comprehensive essentiality analysis of the Mycobacterium tuberculosis genome via saturating transposon mutagenesis;  citation_journal_title=MBio;  citation_year=2017;  citation_volume=8; ">
<meta name="citation_reference" content="citation_title=Regulatory protein phosphorylation in Mycoplasma pneumoniae. A PP2C-type phosphatase serves to dephosphorylate HPr (Ser-P);  citation_journal_title=J. Biol. Chem;  citation_year=2006;  citation_volume=281;  citation_pages=26253-9; ">
<meta name="citation_reference" content="citation_title=Tools for the genetic analysis of Mycoplasma;  citation_journal_title=Int. J. Med. Microbiol;  citation_year=2007;  citation_volume=297;  citation_pages=37-44; ">
<meta name="citation_reference" content="citation_title=Tn4001: a gentamicin and kanamycin resistance transposon in Staphylococcus aureus;  citation_journal_title=Mol. Gen. Genet;  citation_year=1984;  citation_volume=193;  citation_pages=554-6; ">
<meta name="citation_reference" content="citation_title=Construction and use of derivatives of transposon Tn4001 that function in Mycoplasma pulmonis and Mycoplasma arthritidis;  citation_journal_title=J. Bacteriol;  citation_year=2000;  citation_volume=182;  citation_pages=4343-7; ">
<meta name="citation_reference" content="citation_title=Construction of mini-Tn4001tet and its use in Mycoplasma gallisepticum;  citation_journal_title=Plasmid;  citation_year=2002;  citation_volume=47;  citation_pages=129-37; ">
<meta name="citation_reference" content="citation_title=Construction and analysis of a modified Tn4001 conferring chloramphenicol resistance in Mycoplasma pneumoniae;  citation_journal_title=Plasmid;  citation_year=1999;  citation_volume=41;  citation_pages=120-4; ">
<meta name="citation_reference" content="citation_title=New selectable marker for manipulating the simple genomes of Mycoplasma species;  citation_journal_title=Antimicrob. Agents Chemother;  citation_year=2009;  citation_volume=53;  citation_pages=4429-32; ">
<meta name="citation_reference" content="citation_title=Comparative analysis of antibiotic resistance gene markers in Mycoplasma genitalium: application to studies of the minimal gene complement;  citation_journal_title=Microbiology;  citation_year=2006;  citation_volume=152;  citation_pages=519-27; ">
<meta name="citation_reference" content="citation_title=First steps towards the genetic manipulation of Mycoplasma agalactiae and Mycoplasma bovis using the transposon Tn4001mod;  citation_journal_title=Int. J. Med. Microbiol;  citation_year=2005;  citation_volume=294;  citation_pages=447-53; ">
<meta name="citation_reference" content="citation_title=Synthesis of adenylate nucleotides by Mollicutes (mycoplasmas);  citation_journal_title=J. Gen. Microbiol;  citation_year=1983;  citation_volume=129;  citation_pages=3103-10; ">
<meta name="citation_reference" content="citation_title=Mycoplasma feriruminatoris sp. nov., a fast growing Mycoplasma species isolated from wild Caprinae;  citation_journal_title=Syst. Appl. Microbiol;  citation_year=2013;  citation_volume=36;  citation_pages=533-8; ">
<meta name="citation_reference" content="citation_title=Enzymatic assembly of DNA molecules up to several hundred kilobases;  citation_journal_title=Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=343-5; ">
<meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;  citation_journal_title=Mol. Biol. Evol;  citation_year=Saitou, N. and Nei, M. 1987; ">
<meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using the bootstrap;  citation_journal_title=Evolution;  citation_year=1985;  citation_volume=39;  citation_pages=783">
<meta name="citation_reference" content="citation_title=Prospects for inferring very large phylogenies by using the neighbor-joining method;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=11030-5; ">
<meta name="citation_reference" content="citation_title=MEGA X: molecular evolutionary genetics analysis across computing platforms;  citation_journal_title=Mol. Biol. Evol;  citation_year=2018;  citation_volume=35;  citation_pages=1547-9; ">
<meta name="citation_reference" content="citation_title=Distribution and diversity of ribosome binding sites in prokaryotic genomes;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=604">
<meta name="citation_reference" content="citation_title=Synthesis of a small, cysteine-rich, 29 amino acids long peptide in Mycoplasma pneumoniae;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2005;  citation_volume=253;  citation_pages=315-21; ">
<meta name="citation_reference" content="citation_title=Accurate whole human genome sequencing using reversible terminator chemistry;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=53-9; ">
<meta name="citation_reference" content="citation_title=ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter;  citation_journal_title=Genome Res;  citation_year=2017;  citation_volume=27;  citation_pages=768-77; ">
<meta name="citation_reference" content="citation_title=FastUniq: a fast de novo duplicates removal tool for paired short reads;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e52249">
<meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; ">
<meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; ">
<meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAM tools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; ">
<meta name="citation_reference" content="citation_title=Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper;  citation_journal_title=Mol. Biol. Evol;  citation_year=2017;  citation_volume=34;  citation_pages=2115-22; ">
<meta name="citation_reference" content="citation_title=Insights into the mechanisms of basal coordination of transcription using a genome-reduced bacterium;  citation_journal_title=Cell Syst;  citation_year=2018;  citation_volume=7;  citation_pages=227-9; ">
<meta name="citation_reference" content="citation_title=Is the sequence-specific binding of aminoacyl-tRNAs by EF-Tu universal among bacteria?;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=9746-58; ">
<meta name="citation_reference" content="citation_title=Phylogenetic conservation of antigenic determinants in archaebacterial elongation factors (Tu proteins);  citation_journal_title=Can. J. Microbiol;  citation_year=1989;  citation_volume=35;  citation_pages=2-10; ">
<meta name="citation_reference" content="citation_title=A reporter system coupled with high-throughput sequencing unveils key bacterial transcription and translation determinants;  citation_journal_title=Nat. Commun;  citation_year=2017;  citation_volume=8; ">
<meta name="citation_reference" content="citation_title=Distinguishing between productive and abortive promoters using a random forest classifier in Mycoplasma pneumoniae;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=3442-53; ">
<meta name="citation_reference" content="citation_title=All-in-one construct for genome engineering using Cre-lox technology;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=263-70; ">
<meta name="citation_reference" content="citation_title=Fewer essential genes in mycoplasmas than previous studies suggest;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2010;  citation_volume=311;  citation_pages=51-5; ">
<meta name="citation_reference" content="citation_title=Inferring the minimal genome of Mesoplasma florum by comparative genomics and transposon mutagenesis;  citation_journal_title=mSystems;  citation_year=2018;  citation_volume=3; ">
<meta name="citation_reference" content="citation_title=FSPP: a Tool for genome-wide prediction of smORF-encoded peptides and their functions;  citation_journal_title=Front. Genet;  citation_year=2018;  citation_volume=9;  citation_pages=96">
<meta name="citation_reference" content="citation_title=Unraveling the hidden universe of small proteins in bacterial genomes;  citation_journal_title=Mol. Syst. Biol;  citation_year=2019;  citation_volume=15;  citation_pages=e8290">
<meta name="citation_reference" content="citation_title=Tn5 in vitro transposition;  citation_journal_title=J. Biol. Chem;  citation_year=1998;  citation_volume=273;  citation_pages=7367-74; ">
<meta name="citation_reference" content="citation_title=Magic pools: parallel assessment of transposon delivery vectors in bacteria;  citation_journal_title=mSystems;  citation_year=2018;  citation_volume=3; ">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/4/327/29197181/dsz012.pdf">
<meta name="description" content="Abstract.  Mycoplasmas are important model organisms for Systems and Synthetic Biology, and are pathogenic to a wide variety of species. Despite their relevance">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/4/327/5525352">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article-abstract/26/4/327/5525352">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/4/327/5525352");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2779951,2796125&authnips=128.235.85.32&doi=10.1093/dnares/dsz012';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f327%2f5525352" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f327%2f5525352" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=116309"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/4/2/m_dnares_26_4cover.png?Expires=1576706132&amp;Signature=F2WzqY5Rvw~5fnpnOvrqWmeUMWqA6TuHlcXGX5cKMZHC~icn0X1kJeAYo3hIjul1IYqABU5PFelPNYB7icUBfC2Utuqe0Kanc6xUPdQTVSqmHs8XGD0SD1jSofLoGYkyX4fvyBnDT6DegKNRAlK0p3CMB3IF5m7bw4ZfJ0hDTWLv8tmuUuQL6s7NPysvupS6F7Bi2JshmCvK0~2D16GAFc8Z0ZzUtHzETSwnvLON6PshV~D1U5sZReZew8O7QtN01WiuIN3o83L12vAOkYBx79sQQQquD3T4JorhpCEJNC83N3L0L-vg1O7y4o3hyzSvVe4q2sJ6SO7fYQ0CEMfTEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/4">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="140465190">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465190">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465192">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465192">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465197">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465197">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465226">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465226">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="140465263">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465263">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="140465271">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#140465271">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/4/313/5512299">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/4/341/5522131">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ariadna Montero-Blay</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ariadna+Montero-Blay">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Montero-Blay%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Montero-Blay%20Ariadna%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Ariadna Montero-Blay</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Samuel Miravet-Verde</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Samuel+Miravet-Verde">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miravet-Verde%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Miravet-Verde%20Samuel%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Samuel Miravet-Verde</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Lluch-Senar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Lluch-Senar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lluch-Senar%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Lluch-Senar%20Maria%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Maria Lluch-Senar</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Carlos Piñero-Lambea</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +34 933160198; +34 933160259. Fax. +34 93 316 00 99. Email: <a href="mailto:carlos.pinero@crg.eu" target="_blank">carlos.pinero@crg.eu</a> (C.P.-L.); <a href="mailto:luis.serrano@crg.eu" target="_blank">luis.serrano@crg.eu</a> (L.S.) 
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0002-5263-3702">
                <img class="orchid-icon" alt="" src="//oup.silverchair-cdn.com/UI/app/img/mini-icon.png">  http://orcid.org/0000-0002-5263-3702
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Carlos+Pi%c3%b1ero-Lambea">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pi%C3%B1ero-Lambea%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Pi%C3%B1ero-Lambea%20Carlos%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Carlos Piñero-Lambea</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Luis Serrano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Centre for Genomic Regulation (CRG), The Barcelona Institute of Science and Technology, Dr. Aiguader 88, Barcelona, Spain</div>
<div class="aff">Universitat Pompeu Fabra (UPF), Barcelona, Spain</div>
<div class="aff">ICREA, Pg. Lluis Companys 23, Barcelona, Spain </div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +34 933160198; +34 933160259. Fax. +34 93 316 00 99. Email: <a href="mailto:carlos.pinero@crg.eu" target="_blank">carlos.pinero@crg.eu</a> (C.P.-L.); <a href="mailto:luis.serrano@crg.eu" target="_blank">luis.serrano@crg.eu</a> (L.S.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Luis+Serrano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Serrano%20L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Serrano%20Luis%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Luis Serrano</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 4, August 2019, Pages 327–339, <a href="https://doi.org/10.1093/dnares/dsz012">https://doi.org/10.1093/dnares/dsz012</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">30 June 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">12 November 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">16 May 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">30 June 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5525352" href="/dnaresearch/article-pdf/26/4/327/29197181/dsz012.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ariadna Montero-Blay, Samuel Miravet-Verde, Maria Lluch-Senar, Carlos Piñero-Lambea, Luis Serrano,  SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes, <em>DNA Research</em>, Volume 26, Issue 4, August 2019, Pages 327–339, <a href="https://doi.org/10.1093/dnares/dsz012">https://doi.org/10.1093/dnares/dsz012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5525352&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=SynMyco%20transposon%3A%20engineering%20transposon%20vectors%20for%20efficient%20transformation%20of%20minimal%20genomes&amp;publicationDate=2019-06-30&amp;volumeNum=26&amp;issueNum=4&amp;author=Montero-Blay%2C%20Ariadna%3B%20Miravet-Verde%2C%20Samuel&amp;startPage=327&amp;endPage=339&amp;contentId=10.1093%2Fdnares%2Fdsz012&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f327%2f5525352" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="140465190" id="140465190" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Mycoplasmas are important model organisms for Systems and Synthetic Biology, and are pathogenic to a wide variety of species. Despite their relevance, many of the tools established for genome editing in other microorganisms are not available for Mycoplasmas. The Tn4001 transposon is the reference tool to work with these bacteria, but the transformation efficiencies (TEs) reported for the different species vary substantially. Here, we explore the mechanisms underlying these differences in four Mycoplasma species, <em>Mycoplasma agalactiae</em>, <em>Mycoplasma feriruminatoris</em>, <em>Mycoplasma gallisepticum</em> and <em>Mycoplasma pneumoniae</em>, selected for being representative members of each cluster of the Mycoplasma genus. We found that regulatory regions (RRs) driving the expression of the transposase and the antibiotic resistance marker have a major impact on the TEs. We then designed a synthetic RR termed SynMyco RR to control the expression of the key transposon vector elements. Using this synthetic RR, we were able to increase the TE for <em>M. gallisepticum</em>, <em>M. feriruminatoris</em> and <em>M. agalactiae</em> by 30-, 980- and 1036-fold, respectively. Finally, to illustrate the potential of this new transposon, we performed the first essentiality study in <em>M. agalactiae</em>, basing our study on more than 199,000 genome insertions.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword="Mycoplasma">Mycoplasma</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transposon">transposon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"regulatory region"'>regulatory region</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"transformation efficiency"'>transformation efficiency</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="essentiality">essentiality</a>
</div>                    <h2 scrollto-destination="140465192" id="140465192" class="section-title">1. Introduction</h2>
<p>The <em>Mollicutes</em> class represents a taxonomic group of bacteria that has undergone an extreme genome downsizing process termed degenerative evolution.<a reveal-id="dsz012-B1" data-open="dsz012-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> As a consequence of this evolutionary process, these bacteria are characterized by the lack of a cell wall, streamlined genomes and very limited biosynthetic pathways. All these features have turned <em>Mollicutes</em>, and particularly some species encompassed within the Mycoplasma genus, into appealing models to study basic principles of complex cellular processes, such as transcription and translation. For instance, <em>Mycoplasma pneumoniae</em> is an important model organism for Systems Biology as highlighted by the comprehensive knowledge acquired about its complete genome,<a reveal-id="dsz012-B2" data-open="dsz012-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> transcriptome,<a reveal-id="dsz012-B3" data-open="dsz012-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> proteome,<a reveal-id="dsz012-B4" data-open="dsz012-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> DNA methylome<a reveal-id="dsz012-B5" data-open="dsz012-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and gene essentiality.<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> In addition, the first whole-cell computational model developed was for <em>Mycoplasma genitalium.</em><a reveal-id="dsz012-B7" data-open="dsz012-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> At the same time, Mycoplasmas are also interesting organisms for the emerging field of Synthetic Biology and to study the minimal set of genes required to sustain life.<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz012-B8 dsz012-B9 dsz012-B10" data-open="dsz012-B8 dsz012-B9 dsz012-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> Indeed, the genome of the first synthetic bacterium, JCVI Syn3.0, was inferred from the genome of <em>Mycoplasma mycoides</em> following a top-down approach in a design-build and test cycle.<a reveal-id="dsz012-B11" data-open="dsz012-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Aside from their relevance as model organisms, Mycoplasmas are also a serious concern for the medical and veterinary fields given their pathogenic effects on a wide variety of species. For instance, <em>M. pneumoniae</em> is a human pathogen that causes atypical pneumonia,<a reveal-id="dsz012-B7" data-open="dsz012-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsz012-B12" data-open="dsz012-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><em>Mycoplasma gallisepticum</em> causes pneumonia in poultry<a reveal-id="dsz012-B13" data-open="dsz012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and <em>Mycoplasma agalactiae</em> causes contagious agalactia, a common disease in small ruminants with a tremendous economic impact.<a reveal-id="dsz012-B14" data-open="dsz012-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p>
<p>Identifying genes involved in pathogenicity is critical for generating new vaccines and therapies. Genes that are dispensable for <em>in vitro</em> growth (i.e. non-essential, NE) but essential (E) for the infection process could be target genes for vaccine design. Construction of essentiality maps is a fast way to decipher the essential or non-essential character of every single gene found in the genome of a given bacterium. Furthermore, these maps can also help in the identification of those genes that are not essential but whose disruption affects the fitness of the bacteria in a particular environment (i.e. fitness, F). The generation of these maps relies on the construction of saturating libraries of transposon mutants, followed by high-throughput insertion tracking by ultra sequencing (HITS) at different passages.<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz012-B15" data-open="dsz012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Furthermore, construction of saturating libraries is also relevant for Haystack mutagenesis,<a reveal-id="dsz012-B16" data-open="dsz012-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> a technique useful to establish gene–function relationships that is widely employed in Mycoplasmas, as a consequence of the paucity of other genome editing tools for these bacteria.<a reveal-id="dsz012-B17" data-open="dsz012-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p>
<p>Tn4001 is a gentamicin, tobramycin and kanamycin resistance-conferring transposon that was originally found in <em>Staphylococcus aureus</em>,<a reveal-id="dsz012-B18" data-open="dsz012-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> but has been widely employed in Mycoplasmas.<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz012-B10" data-open="dsz012-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsz012-B19" data-open="dsz012-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Few modifications have been made to this transposon to adapt it for use in Mycoplasmas aside from (i), placing the transposase coding gene outside the inverted repeats (in plasmids termed mini-transposons) to prevent re-excision from the transposon after the first transposition event,<a reveal-id="dsz012-B20" data-open="dsz012-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and (ii) replacing the original gentamicin resistance marker by tetracycline,<a reveal-id="dsz012-B18" data-open="dsz012-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> chloramphenicol<a reveal-id="dsz012-B21" data-open="dsz012-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> or puromycin<a reveal-id="dsz012-B22" data-open="dsz012-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> resistance genes. Such a poor adaptation of the Tn4001 transposon to Mycoplasmas might partially explain the dramatic differences observed among transformation efficiencies (TEs) in species from this genus. For example, TE is consistently higher in those species belonging to the pneumoniae cluster (i.e. species closely related to <em>M. pneumoniae</em>)<a reveal-id="dsz012-B23" data-open="dsz012-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> than in those species encompassed in the spiroplasma<a reveal-id="dsz012-B24" data-open="dsz012-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> or hominis clusters.<a reveal-id="dsz012-B25" data-open="dsz012-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>
<p>Here, we explored and identified the reasons underlying the lower transposon TE in a set of different Mycoplasma species that differ in their phylogenetic distance to <em>M. pneumoniae</em>. We hypothesized that poor recognition of certain gene regulatory regions (RRs) in the transposon vector might limit TE in some species. Therefore, we rationally engineered a vector variant that significantly increased the TE in all species tested except for <em>M. pneumoniae</em>, which was already transformed at high efficiencies with the unmodified vector<em>.</em> These species were selected to encompass the three different clusters described in the Mycoplasma phylogenetic tree, thereby opening up the door to more global studies of Mycoplasma species. Furthermore, a reporter assay allowed us to identify which of the RRs determining expression of the antibiotic resistance and transposase protein coding genes found in the native transposon vector were inefficiently recognized in each of the strains selected for this work. Finally, to show the potential of our transposon vector, we performed an essentiality study of <em>M. agalactiae</em>, obtaining an insertional coverage similar to the one reported for <em>M. pneumoniae.</em><a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p>                    <h2 scrollto-destination="140465197" id="140465197" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="140465198" id="140465198" class="section-title">2.1. Bacteria strains and culture conditions</h3>
<p>For wild-type <em>M. pneumoniae</em>, the strain M129 (ATTC 29342, subtype 1, broth passage no. 35) was used. Wild-type <em>M. agalactiae</em> 7784 was kindly provided by Christine Citti. Wild-type <em>M. gallisepticum</em> R high was kindly provided by Michael Szostak.<a reveal-id="dsz012-B25" data-open="dsz012-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Wild-type <em>Mycoplasma feriruminatoris</em> G5847 was kindly provided by Carole Lartigue.<a reveal-id="dsz012-B26" data-open="dsz012-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> All strains were grown at 37°C in standard Hayflick medium supplemented with 100 µg/ml ampicillin and phenol red (0.005% w/v). <em>M. agalactiae</em> standard Hayflick culture was supplemented with sodium pyruvate (0.5%, pH 7.6, Sigma-Aldrich). <em>M. feriruminatoris</em>, <em>M. agalactiae and M. gallisepticum</em> were grown in suspension (180 rpm, 37°C). <em>M. pneumoniae</em> was grown without shaking at 37°C and with 5% CO<sub>2</sub>.</p>                    <h3 scrollto-destination="140465200" id="140465200" class="section-title">2.2. Plasmids</h3>
<p>The plasmids were generated following the Gibson assembly method.<a reveal-id="dsz012-B27" data-open="dsz012-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The list of plasmids as well as a detailed description of the procedure followed to build all the constructs are described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. For plasmid generation, DNA was isolated from NEB<sup>®</sup> 5-alpha High Efficiency (C2987P). The clones were isolated using LB agar + ampicillin (100 µg/ml) plates and confirmed by sequencing (GATC biotech). The list of all the primers used in this study for the generation of the plasmids can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                    <h3 scrollto-destination="140465202" id="140465202" class="section-title">2.3. Transformation protocol</h3>
<p>Transformation procedures were initially based on methods previously described but later slightly modified.<a reveal-id="dsz012-B24" data-open="dsz012-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> For <em>M. feriruminatoris</em>, <em>M. agalactiae</em> and <em>M. gallisepticum</em> cultures, 10-ml log-phase cultures were harvested at 10,000 g for 10 min at 4°C. The medium was removed and 10 ml of fresh Hayflick was added. After 3 h, the culture was centrifuged at 10,000 g at 4°C and then washed three times with chilled electroporation buffer (EB; 272 mM sucrose, 8 mM HEPES, pH 7.4) before final resuspension in 300 µl chilled EB. After mixing 50 µl of cells with 1.5 µg of DNA and incubating for 20 min on ice, the mix was transferred into 0.1-cm electro cuvettes and electroporated in a BIO-RAD Gene Pulser Xcell apparatus. For <em>M. pneumoniae</em> (i.e. adherent strain) cells were grown in a 75-cm<sup>2</sup> tissue flask containing 20 ml of fresh Hayflick and incubated at 37°C under 5% CO<sub>2</sub> until late exponential phase. Cells were washed twice, resuspended in precooled EB, scraped off and passed through a 25-gauge (G25) syringe needle 10 times. Aliquots of 50 µl of cells in 0.1-cm cuvettes with 1.5 µg of the corresponding plasmid were kept on ice during 20 min. The electroporation settings were common for all strains: 1250 V/25 µF/100 Ω. Immediately after the pulse, 420 µl of fresh Hayflick was added to the cells. Subsequently, the cells were incubated at 37°C before seeding on agar plates. The incubation time was 30 min for <em>M. feriruminatoris</em>, 90 min for <em>M. agalactiae</em> and <em>M. gallisepticum</em> and 120 min for <em>M. pneumoniae</em>.</p>                    <h3 scrollto-destination="140465204" id="140465204" class="section-title">2.4. Determination of transformation efficiency in Mycoplasma species</h3>
<p>After incubating the transformations at 37°C during the above-mentioned time depending on the strain, 10-fold serial dilutions of the cultures were performed (from −1 to −8). The dilutions were made in a total volume of 100 µl and 10 µl of each dilution was plated onto Hayflick 0.8% agar plates. Transformations done with pMTnGm or pMTnGm-SynMyco vectors were counted on agar plates supplemented with 100 µg/ml gentamicin. Transformations done with pMTnCm vector were selected in agar plates supplemented with 20 µg/ml chloramphenicol. The mutant counts refer to mutants per transformation, where the final volume is 500 µl. TE is defined as the ratio of colony forming units (CFUs) counted on antibiotic supplemented plates (i.e. transformed cells carrying a transposon insertion) to non-supplemented plates (i.e. viable cells after transformation). To assess differences in TE between pMTnGm and pMTnCm, one-tailed paired <em>t</em>-tests were applied to three different experimental replicates for each strain employed in the study. As the mutants obtained for <em>M. agalactiae</em>, <em>M. gallisepticum</em> and <em>M. feriruminatoris</em> using pMTnCm transposon were below the limit of detection, for the statistical analysis, the number of mutants obtained was set to 49 CFU per batch, a value that corresponds to the maximum number of CFU under the limit of detection for each species. The <em>P</em>-values obtained for TE differences between pMTnGm and pMTnCm were 0.470, 0.186, 0.026 and 0.067 for <em>M. feriruminatoris</em>, <em>M. agalactiae</em>, <em>M. gallisepticum</em> and <em>M. pneumoniae</em>, respectively (for the raw data of these transformations as well as the statistical analysis, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The same statistical approach was applied to compare TE obtained with pMTnGm versus pMTnGm-SynMyco in three different experimental replicates for each species employed in the study (<em>P</em>-values of 1.49 × 10<sup>−4</sup>, 4.28 × 10<sup>−2</sup>, 4.97 × 10<sup>−2</sup> and 0.325 for <em>M. feriruminatoris</em>, <em>M. agalactiae</em>, <em>M. gallisepticum</em> and <em>M. pneumoniae</em>, respectively). Data of TEs obtained with pMTnGm and pMTnGm-SynMyco as well as the statistical analysis are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>.</p>                    <h3 scrollto-destination="140465206" id="140465206" class="section-title">2.5. Phylogenetic analysis of selected Mycoplasma species</h3>
<p>For the generation of the phylogenetic tree, 21 Mycoplasma species were selected. DNA sequences encoding the 16S rRNA of each species were aligned using multiple sequence alignment by ClustalW. For those species containing more than a single copy coding for the 16S rRNA, we selected one of the copies arbitrarily. The evolutionary history of these microorganisms was inferred using the neighbour-joining method.<a reveal-id="dsz012-B28" data-open="dsz012-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> The optimal tree with the sum of branch length = 0.91 is shown. The percentages of replicate trees in which the associated taxa clustered together in the bootstrap test (500 iterations) are shown next to the branches. The evolutionary distances were computed using the maximum composite likelihood method<a reveal-id="dsz012-B29" data-open="dsz012-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsz012-B30" data-open="dsz012-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and are in the units of the number of base substitutions per site. Codon positions included were 1st + 2nd + 3rd + non-coding. All positions containing gaps and missing data were eliminated. There were a total of 1,464 positions in the final dataset. Evolutionary analyses were conducted in MEGA X.<a reveal-id="dsz012-B31" data-open="dsz012-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The 21 different DNA sequences coding for 16S rRNA and the accession number of the species to which they belong are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>, except for <em>M. feriruminatoris</em> whose assembled genome sequence was kindly provided by Dr Carole Lartigue.</p>                    <h3 scrollto-destination="140465208" id="140465208" class="section-title">2.6. Ribosome-binding site inclusion along different Mycoplasma species</h3>
<p>As a reference we used the same set of 21 Mycoplasma species that were used to build up the phylogenetic tree. Specifically, for each species we extracted the 15 bases before the start codon of all their annotated genes. Within these sequences, we then checked for the presence of any of the subsequences reported to be RBS.<a reveal-id="dsz012-B32" data-open="dsz012-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> This list included: GGA, GAG, AGG, AGGA, GGAG, GAGG, AGGAG, GGAGG, AGAAGG, AGCAGG, AGGAGG, AGTAGG, AGGCGG, AGGGGG and AGGTGG. Inclusion was represented as the percentage of genes in a bacterial species that included one of these RBS motifs (for the percentage of RBS inclusion of the species of the work, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p>                    <h3 scrollto-destination="140465210" id="140465210" class="section-title">2.7. Strength evaluation of native regulatory regions driving the expression of the transposase and gentamicin resistance gene in pMTnGm, and comparison with SynMyco regulatory region in different Mycoplasma species</h3>
<p>Three different reporter plasmids containing the mCherry coding sequence under the control of the three different RRs (i.e. pMTnGm-SynMyco+SynMyco RR-Venus+SynMyco RR-mCherry, pMTnGm-SynMyco+SynMyco RR-Venus+GmR-mCherry and pMTnGm-SynMyco+SynMyco RR-Venus+Tnp RR-mCherry) were generated. These plasmids contain genes coding for two fluorescent proteins (i.e. Venus and mCherry). These proteins have fused in its N-terminal part the mp-200 sequence, a 29 amino acid signal from <em>M. pneumoniae</em> MPN391a gene that has been previously fused to Venus and mCherry sequences to improve protein stability.<a reveal-id="dsz012-B33" data-open="dsz012-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Whereas the gene coding for the Venus fluorescent protein was included for normalization purposes in all constructs under the control of SynMyco RR, the gene coding for mCherry has been placed under the control of three different RRs depending on the construct: (i) the SynMyco RR, (ii) the 150 bp upstream region of the transposase coding gene (Tnp RR) and (iii) the 150 upstream region of the gentamicin resistance coding gene (Gm RR).</p>
<p>All constructs were transformed in <em>M. agalactiae</em>, <em>M. gallisepticum</em>, <em>M. feriruminatoris</em> and <em>M. pneumoniae</em> following the protocol described above. To generate a primary stock of the transformations, from each 500 µl batch 100 µl was inoculated in a flask containing 5 ml of Hayflick medium supplemented with 100 µg/ml gentamicin for <em>M. pneumoniae</em>, while for the other non-adherent strains the 100 µl was inoculated in 50 ml Falcon tubes (Fisher Scientific, 14-432-22) filled with 10 ml of Hayflick medium supplemented with 100 µg/ml gentamicin. When cultures were grown, the total biomass was resuspended in 1 ml of Hayflick medium and stored at −80°C.</p>
<p>To determine the fluorescence levels of each construct, cultures of the four different Mycoplasma strains carrying the three different reporter constructs plus a negative control for each strain not carrying any construct were used. The cultures were grown as described above using 10 µl of their respective primary stocks as inoculum (i.e. around 12 h for <em>M. feriruminatoris</em>, 48 h for <em>M. gallisepticum</em> and <em>M. agalactiae</em> and 72 h for <em>M. pneumoniae</em>) and cells were harvested and washed twice with chilled PBS buffer until final resuspension of the cultures in 500 µl of PBS. Five-fold serial dilutions of the final cell suspensions were done, and 100 µl of all dilutions were loaded in 96-Well Optical Btm Plt Polymerbase Black Lid plates (165305, Thermo Scientific).</p>
<p>The absorbance and fluorescence values were measured using Tecan I-control 1.9.17.0 Infinite 200. The settings were determined for optimal gain, 25 flashes and 20 µs of integration time. The fluorescence settings were ƛ<sub>ex</sub> = 514 nm and ƛ<sub>em</sub> = 574 nm for Venus and ƛ<sub>ex</sub> = 550 nm and ƛ<sub>em</sub> = 630 nm for mCherry. For each strain, the absorbance at ƛ = 600 nm was measured. For the fluorescence analysis we took the data of those wells (i.e. dilutions) in which OD<sub>600nm</sub> absorbance values were between 0.075–0.2; 0.2–0.35; 0.075–0.4 and 0.22–0.66 for <em>M. feriruminatoris</em>, <em>M. agalactiae</em>, <em>M. pneumoniae</em> and <em>M. gallisepticum</em>, respectively. These dilutions were selected so that fluorescent signals were clearly different from negative controls of each strain (i.e. not carrying fluorescent constructs) and proportional to the absorbance (i.e. not saturating signals). Fluorescence arbitrary units (AU) measured for Venus and mCherry were normalized to OD<sub>600nm</sub> for each condition (Venus AU/OD<sub>600</sub> and mCherry AU/OD<sub>600</sub>) to obtain normalized fluorescence AU. For each of the four strains of the work, the mCherry and Venus fluorescence levels of WT cells (i.e. not carrying any fluorescent construct) were determined and subtracted from the fluorescence values obtained for each condition, to obtain subtracted AU. Finally, to compare the strength of all RRs analysed we calculated for each strain the ratio between normalized and subtracted mCherry UA/normalized and subtracted Venus UA. Statistical paired <em>t</em>-test analysis with one-tailed distribution was performed for each strain comparing the ratio of mCherry AU/Venus AU for (i) SynMyco-mCherry versus Gm RR-mCherry and (ii) SynMyco-mCherry versus Tnp RR-mCherry. For the data of the strength evaluation of SynMyco RR and the native RR driving the expression of the gentamicin and the transposase coding genes in the native pMTnGm as well as the statistical analysis, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>.</p>                    <h3 scrollto-destination="140465215" id="140465215" class="section-title">2.8. <em>M. agalactiae</em> pMTnGm-SynMyco transformation for essentiality study</h3>
<p><em>M. agalactiae</em> was transformed using the pMTnGm-SynMyco transposon following the protocol described above. Two hours after the transformation, ⅘ parts of the total 500 µl batch were inoculated into 10 ml Hayflick + 0.5% sodium pyruvate (Sigma-Aldrich P8574-5G) + 100 µg/ml gentamicin. After 24 h at 37°C, the culture was centrifuged for 10 min at 10,000 g. The supernatant was discarded and the cells were collected in 500 µl of Hayflick constituting passage 1 of the transformation. The procedure described above was repeated two more times until passage 3, using a volume of 8 µl of the immediately preceding passage, to grow all the passages (1/62.5 dilution). Taking into account the doubling time of <em>M. agalactiae</em> (4–5 h), and the passages performed the expected number of cell divisions in passage 3 is 20. From passage 3, 350 µl out of the total 500 µl were taken to extract genomic DNA using the MasterPure Complete DNA and RNA Purification Kit (MC85200, Lucigen) following the protocol described by the manufacturer.</p>                    <h3 scrollto-destination="140465217" id="140465217" class="section-title">2.9. Genome assembly and <em>de novo</em> annotation for <em>M. agalactiae</em> 7784</h3>
<p><em>M. agalactiae</em> was grown 48 h in 25 ml of Hayflick medium supplemented with sodium pyruvate at 37°C as described previously. After, the genomic DNA was isolated using the MasterPure Complete DNA and RNA Purification Kit (MC85200, Lucigen) following the protocol described by the manufacturer. The genomic DNA was sheared to 200–300 bp fragments using a Covaris S2 device. Then, paired-end Illumina libraries were created following previously described protocols<a reveal-id="dsz012-B34" data-open="dsz012-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and the size selected was 125 bp. The resulting libraries were quantified on an Agilent Bioanalyzer chip (Agilent Technologies). Double-stranded templates were amplified and sequenced on an Illumina GAII. Raw reads were analysed using the FastQC tool (website: <a class="link link-uri" href="http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc" target="_blank">http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc</a>) for assessing the quality and the presence of adapters. Contigs were <em>de novo</em> assembled using ABySS<a reveal-id="dsz012-B35" data-open="dsz012-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and <em>M. agalactiae</em> 5632 genome as annotation reference resulting in 159 contigs with average length of 5,511 bp. In total, 503 genes present in <em>M. agalactiae</em> 5632 displayed homologs within contigs of <em>M. agalactiae</em> 7784. We increased this value up to 689 with a specific BlastN search where we restrictively selected hits with an alignment length greater than 95% and an <em>e</em>-value less than 1 × 10<sup>−5</sup>. This allowed us to provide a more accurate list of genes presented by the strain of interest to assign putative functions by homology as well as to detect gene duplications in the studied genome. We observed that 12 contigs were informative enough to capture a set of 689 genes. This set of contigs averaged 71,163 bp each in length and 853,960 bp in total. The latter value was considered to be the genome size for <em>M. agalactiae</em> 7784. The raw data of DNAseq, genome assembly and <em>de novo</em> annotation have been submitted as BioProject under accession PRJNA528179.</p>                    <h3 scrollto-destination="140465219" id="140465219" class="section-title">2.10. Sequencing of <em>M. agalactiae</em> 7784 transformed with pMTnGm-SynMyco</h3>
<p>Genomic DNA sequencing was performed in the Genomics facility at Centre for Genomic Regulation in a HiSeq Sequencing v4 Chemistry controlled by Software HiSeq Control Software 2.2.58. Sequencing settings were fixed at 125 nucleotides in paired-end format. In the HiSeq sequencing technology from Illumina Genome Analyzer, the protocol starts with DNA fragmentation. Then, the fragmented DNA is amplified using oligos that add adapters allowing the subsequent binding of PCR products to the glass flow cell. Later, the sequencing is performed by synthesis cycles, in which a single complementary base for each deoxynucleotide (dNTP) is incorporated using a fluorescently labeled dNTP. Finally, lasers excite the fluorophores while a camera captures images of the flow cell.</p>                    <h3 scrollto-destination="140465221" id="140465221" class="section-title">2.11. Transposon mapping</h3>
<p>For <em>M. agalactiae</em> 7784, paired-end sequencing raw reads were filtered to remove PCR duplicates using Fastuniq.<a reveal-id="dsz012-B36" data-open="dsz012-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Then, a specific inverted repeat (IR) associated to the transposon insertion process <strong>(</strong>TTTTACACAATTATACGGACTTTATC, length = 26) was trimmed by Trimmomatic<a reveal-id="dsz012-B37" data-open="dsz012-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and then mapped to the reference using Bowtie2 allowing 1 mismatch.<a reveal-id="dsz012-B38" data-open="dsz012-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Later, we selected paired reads mapped unambiguously with a minimum alignment quality of 30 using SAMtools.<a reveal-id="dsz012-B39" data-open="dsz012-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The last step relied on shell text processing tools (awk/grep/sed) to identify those pairs where one of the reads presented a shorter length than the original read length minus 26 (expected length if the IR was removed). Position reported represents the first mapped position in the chromosome contiguous to the IR.</p>                    <h3 scrollto-destination="140465223" id="140465223" class="section-title">2.12. Essentiality definition and training set generation for <em>M. agalactiae</em> and <em>M. pneumoniae</em>
</h3>
<div class="block-child-p">The essentiality studies were developed using as reference the study previously done in <em>M. pneumoniae.</em><a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Specifically, we reanalysed the T4 dataset of <em>M. pneumoniae.</em><a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Transposition events are considered to behave as random events resembling a Poisson process over large portions of the chromosome with a uniform density. The analysis starts with two different linear insertion densities (<em>r</em>) for essential (<em>r</em><sub>E</sub>) and non-essential genes (<em>r</em><sub>NE</sub>). These values correspond to the total number of insertions mapped normalized by gene length for a training set of genes which, based on prior knowledge, we assume are either E or NE and can be used as reference to classify the rest of genes. For <em>M. pneumoniae</em>, we used as a training set the same group of genes that were defined in its essentiality study.<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> In the case of <em>M. agalactiae</em>, a new E training set containing 32 genes was generated using the same criteria as for <em>M. pneumoniae</em>. For NE genes, we extracted 84 genes with no homolog in four closely related species to <em>M. agalactiae: Mycoplasma hyosynoviae</em>, <em>Mycoplasma arthritidis</em>, <em>Mycoplasma pulmonis</em> and <em>Mycoplasma synoviae</em>. Out of this group, we selected 17 genes that had been confirmed as NE in the essentiality study done in <em>M. bovis</em> PG45.<a reveal-id="dsz012-B10" data-open="dsz012-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The probability of a specific gene to be essential or not is evaluated predicting two values: <em>P</em><sub>E</sub> and <em>P</em><sub>NE</sub> using formula (1). In this formula, <em>N</em> represents the number of insertions mapped to a gene and <em>L</em> the length of that gene. <em>L</em> corresponds to the inner 90% part of each gene (removing the 5% in each terminus) since it has been observed that these regions allow a higher number of non-disruptive mutations as they would not be located in the core regions of the encoded protein. The value of <em>r</em>, inferred from the training sets, varies between essential (<em>r</em><sub>E</sub>) and non-essential genes (<em>r</em><sub>NE</sub>) and allows us to determine the probability of a particular gene to be essential or not with its specific <em>N</em> and <em>L</em>. If <em>P</em><sub>E</sub> &gt; 0.0 and <em>P</em><sub>NE</sub> = 0.0, the gene is classified as E (essential), <em>P</em><sub>E</sub> = 0.0 and <em>P</em><sub>NE</sub> &gt; 0.0 will correspond to NE (non-essential), and if both probabilities are non-zero we assume that the gene is F (fitness). <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap">
<div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>P</mi></mrow><mi>N</mi></msub><mo stretchy="false">(</mo><mi>L</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><msup><mrow><mrow><mo stretchy="false">(</mo><mi mathvariant="italic">rL</mi><mo stretchy="false">)</mo></mrow></mrow><mi>N</mi></msup></mrow><mrow><mi>N</mi><mo>!</mo></mrow></mfrac><msup><mrow><mi mathvariant="normal">e</mi></mrow><mrow><mo>−</mo><mi mathvariant="italic">rL</mi></mrow></msup></mrow></math></div></div>
<div class="label">(1)</div>
</div>
</div>
<p>After Clusters of Orthologous Groups (COGs) were associated to the <em>M. agalactiae</em> genome using eggNOG-mapper<a reveal-id="dsz012-B40" data-open="dsz012-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> utilizing the DIAMOND mapping mode, taxonomic scope set on Firmicutes and prioritizing quality over coverage. A subsequent step of manual curation of COG categories was performed. For the set of 689 genes found for <em>M. agalactiae</em> 7784, the genes included in the training list, the COG category assignment, their essentiality assigned class and the statistical analysis, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>.</p>                    <h2 scrollto-destination="140465226" id="140465226" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="140465227" id="140465227" class="section-title">3.1. Transformation efficiencies in Mycoplasmas show dramatic differences depending on the strain</h3>
<p>As a starting point of the project, we aimed to quantify the transposon TE in representative members of each cluster of the Mycoplasma genus (<a reveal-id="dsz012-F1" data-open="dsz012-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). To this end, cultures of <em>M. feriruminatoris</em> (spiroplasma cluster), <em>M. agalactiae</em> (hominis cluster), <em>M. pneumoniae</em> and <em>M. gallisepticum</em> (pneumoniae cluster) were transformed with the pMTnGm vector,<a reveal-id="dsz012-B23" data-open="dsz012-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> a mini-transposon plasmid derived from the original Tn4001 that confers resistance to gentamicin and is broadly employed in the Mycoplasma field. As expected from previous reports, <em>M. pneumoniae</em> showed the highest TE, with approximately one transformant for every 10<sup>3</sup> cells. <em>M. pneumoniae</em> was followed by <em>M. gallisepticum</em> and <em>M. agalactiae</em>, with three transformants for every 10<sup>6</sup> cells. Far behind these TEs was that of <em>M. feriruminatoris</em>, for which we found one transformant for every 10<sup>9</sup> cells (<a reveal-id="dsz012-F1" data-open="dsz012-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). </p>                        <a id="140465229" scrollto-destination="140465229"></a>
<div id="dsz012-F1" data-id="dsz012-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f1.png?Expires=1576602286&amp;Signature=pumBErXajOmtmLwp6aetMGNO~BhyPfdUX0JVwiOikewIew61SrkAITzoPmYhjhstxmlvRv6E0BH6xtWeX1mjjRnrtmigeES0LJOkhek2qP2Anl5q26vDqLjbq74i52Sx5sqVi5TBibiP2Y4HYVbLy~omVmSm3FzXA-SwQdmW3tLKEnk2mo5DmGmw2kjzVKhm1nIvHkJ21VsCaK~RcJTssD6BFW6tbaRQsPv1XUlrUVFHZLQHi80GeB9adg27VsXIwwJ1GnleXcOCnAPNkR-oxp-~6Puu1ZwV~tGqOyf5vNVvWcRAbc90wLBZJp8DHa9wbYmlcT2bjQzk7vVY9wpbSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screening of transposon TEs across the mycoplasmal landscape. (A) Phylogenetic tree of 21 selected Mycoplasma species in which three main clusters (pneumoniae, spiroplasma and hominis) can be identified using the maximum composite likelihood method. The tree is drawn to scale with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree.29 (B) Bar plot showing the average of gentamicin resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnGm vector (n = 3). (C) Bar plot showing the average of chloramphenicol resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnCm vector. For the statistical analysis, for those species in which no mutants were detected the number of CFU was set to 49, the maximum value below the limit of detection. One-tailed t-test P-values are indicated with one asterisk (*) when P &lt; 0.05 for TE obtained with pMTnGm vector compared to the TE obtained with pMTnCm vector. " path-from-xml="dsz012f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465229/dsz012f1.tif" path-from-xml="dsz012f1.tif" target="_blank">Open in new tab</a><a section="140465229" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f1.png?Expires=1576602286&amp;Signature=HZ3IEP1~b2Lpc8GogvvlqZMU8jpEWrhpjeUZ6HILaWrTf7R1-4nkNZVpysZxrzcuYd8OjuPBZ~SQoD9XjqJA4HxHftuByZISVPksDtIyhTVcauA8P5yct3d810dhxd7wRz8w8mXm8IHVAievajjpZ3Vw7rf0OYvPiIfDOqkVCj7oqBnZDdpegxTO8LHRIrYUCtHa2Pcq-iyKh4QpUako5s0jznzEGOR8kxwkRACVS2qEL2k~rkhStAiRdiFBYwqyzrdSEvvAPNIhCN1bOvyBtUzc7pq3gTZN3Kz-X75SgN2wHtShJQq0jhjH-tLXxyL8jaGXO22huhWuwLqyU677Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465229&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Screening of transposon TEs across the mycoplasmal landscape. (A) Phylogenetic tree of 21 selected Mycoplasma species in which three main clusters (pneumoniae, spiroplasma and hominis) can be identified using the maximum composite likelihood method. The tree is drawn to scale with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree.<a reveal-id="dsz012-B29" data-open="dsz012-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (B) Bar plot showing the average of gentamicin resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnGm vector (<em>n</em> = 3). (C) Bar plot showing the average of chloramphenicol resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnCm vector. For the statistical analysis, for those species in which no mutants were detected the number of CFU was set to 49, the maximum value below the limit of detection. One-tailed <em>t</em>-test <em>P</em>-values are indicated with one asterisk (*) when <em>P</em> &lt; 0.05 for TE obtained with pMTnGm vector compared to the TE obtained with pMTnCm vector. </p></div>
</div>
<div content-id="dsz012-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f1.png?Expires=1576602286&amp;Signature=pumBErXajOmtmLwp6aetMGNO~BhyPfdUX0JVwiOikewIew61SrkAITzoPmYhjhstxmlvRv6E0BH6xtWeX1mjjRnrtmigeES0LJOkhek2qP2Anl5q26vDqLjbq74i52Sx5sqVi5TBibiP2Y4HYVbLy~omVmSm3FzXA-SwQdmW3tLKEnk2mo5DmGmw2kjzVKhm1nIvHkJ21VsCaK~RcJTssD6BFW6tbaRQsPv1XUlrUVFHZLQHi80GeB9adg27VsXIwwJ1GnleXcOCnAPNkR-oxp-~6Puu1ZwV~tGqOyf5vNVvWcRAbc90wLBZJp8DHa9wbYmlcT2bjQzk7vVY9wpbSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screening of transposon TEs across the mycoplasmal landscape. (A) Phylogenetic tree of 21 selected Mycoplasma species in which three main clusters (pneumoniae, spiroplasma and hominis) can be identified using the maximum composite likelihood method. The tree is drawn to scale with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree.29 (B) Bar plot showing the average of gentamicin resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnGm vector (n = 3). (C) Bar plot showing the average of chloramphenicol resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnCm vector. For the statistical analysis, for those species in which no mutants were detected the number of CFU was set to 49, the maximum value below the limit of detection. One-tailed t-test P-values are indicated with one asterisk (*) when P &lt; 0.05 for TE obtained with pMTnGm vector compared to the TE obtained with pMTnCm vector. " path-from-xml="dsz012f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465229/dsz012f1.tif" path-from-xml="dsz012f1.tif" target="_blank">Open in new tab</a><a section="140465229" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f1.png?Expires=1576602286&amp;Signature=HZ3IEP1~b2Lpc8GogvvlqZMU8jpEWrhpjeUZ6HILaWrTf7R1-4nkNZVpysZxrzcuYd8OjuPBZ~SQoD9XjqJA4HxHftuByZISVPksDtIyhTVcauA8P5yct3d810dhxd7wRz8w8mXm8IHVAievajjpZ3Vw7rf0OYvPiIfDOqkVCj7oqBnZDdpegxTO8LHRIrYUCtHa2Pcq-iyKh4QpUako5s0jznzEGOR8kxwkRACVS2qEL2k~rkhStAiRdiFBYwqyzrdSEvvAPNIhCN1bOvyBtUzc7pq3gTZN3Kz-X75SgN2wHtShJQq0jhjH-tLXxyL8jaGXO22huhWuwLqyU677Wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465229&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Screening of transposon TEs across the mycoplasmal landscape. (A) Phylogenetic tree of 21 selected Mycoplasma species in which three main clusters (pneumoniae, spiroplasma and hominis) can be identified using the maximum composite likelihood method. The tree is drawn to scale with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree.<a reveal-id="dsz012-B29" data-open="dsz012-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (B) Bar plot showing the average of gentamicin resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnGm vector (<em>n</em> = 3). (C) Bar plot showing the average of chloramphenicol resistant CFUs (in logarithmic scale) obtained for each of the indicated strains when using the pMTnCm vector. For the statistical analysis, for those species in which no mutants were detected the number of CFU was set to 49, the maximum value below the limit of detection. One-tailed <em>t</em>-test <em>P</em>-values are indicated with one asterisk (*) when <em>P</em> &lt; 0.05 for TE obtained with pMTnGm vector compared to the TE obtained with pMTnCm vector. </p></div>
</div>
<p>Since the expression of the gentamicin resistance gene has been found to exert a detrimental effect on growth in <em>M. genitalium</em> even in the absence of gentamicin itself,<a reveal-id="dsz012-B23" data-open="dsz012-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> we wanted to determine whether this effect was exacerbated in <em>M. gallisepticum</em>, <em>M. agalactiae and M. feriruminatoris.</em> This could be responsible for the low TE observed in these species with the pMTnGm vector. To this end, we repeated the same set of transformations, but this time with a modified version of pMTnGm vector termed pMTnCm. In this vector, the native gentamicin resistance gene as well as its RR were replaced by a chloramphenicol resistance under the control of the p438 RR, a minimal 22-bp sequence that controls the expression of a putative restriction enzyme in <em>M. genitalium.</em></p>
<p>For the sake of clarity, from this point on, we will refer to the sequence immediately upstream of the translational start codon of each gene as the RR, comprising the promoter region involved in transcription as well as the 5′ UTR involved in translation.</p>
<p>We found that the number of <em>M. pneumoniae</em> transformants obtained with the pMTnCm vector was more than three times higher than the number obtained with the pMTnGm vector, thus falling in line with the previous reports of toxicity associated with expression of the gentamicin resistance gene.<a reveal-id="dsz012-B23" data-open="dsz012-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> On the other hand, the TE was dramatically lower in all the other species tested, with the number of total transformants falling under our detection limit of 50 CFU per batch for <em>M. gallisepticum</em>, <em>M. feriruminatoris and M. agalactiae</em> (<a reveal-id="dsz012-F1" data-open="dsz012-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). The most plausible explanation for these results is that only <em>M. pneumoniae</em> is able to efficiently recognize both the RR found upstream of the gentamicin resistance gene and the <em>M. genitalium</em>-derived p438 sequence driving the expression of the chloramphenicol resistance gene. In contrast, it seems that <em>M. gallisepticum</em>, <em>M. feriruminatoris</em> and <em>M. agalactiae</em> can recognize the RR controlling the gentamicin resistance gene but not the one controlling the chloramphenicol resistance gene.</p>
<p>These results suggest that TE might be directly related to the ability of the transcriptional/translational machinery of each strain to efficiently recognize the RRs controlling not only the antibiotic resistance gene, but also the transposase coding gene.</p>                    <h3 scrollto-destination="140465234" id="140465234" class="section-title">3.2. Design of an efficient regulatory region for a broad range of Mycoplasma species</h3>
<p>Several aspects might influence the TE of a transposon vector and can be classified into two different categories. In the first category, we can include those factors that affect the TE of any vector, such as degradation of the plasmid by bacterial restriction machinery, or the optimality of the transformation protocol itself. Leaving these aside, there is a second group of factors that are specific for transposon vectors. Thus, once the vector has entered the cell, proper transcription/translation of the transposase coding gene is required for the correct insertion of the antibiotic resistance gene into the chromosome. Second, the insertion mutant would only be able to grow if the protein levels of the antibiotic resistance gene reach a certain threshold that is sufficient to promote growth on selective medium. Therefore, in an attempt to maximize the protein levels of both the transposase coding gene and the antibiotic resistance gene in all Mycoplasma species, we decided to design a RR termed SynMyco Regulatory Region (SynMyco RR) that would allow efficient transcription and translation in different mycoplasma species.</p>
<p>As a reference for the design, we chose the RR of MPN665, a gene whose transcript is in the top 5% of most transcribed genes in the transcriptome of <em>M. pneumoniae.</em><a reveal-id="dsz012-B41" data-open="dsz012-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Also, its protein levels (2,646 copies per cell) are among the highest in the proteome of <em>M. pneumoniae.</em><a reveal-id="dsz012-B4" data-open="dsz012-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> MPN665 encodes for the elongation factor thermo-unstable (EF-Tu) protein. As a main component of the ribosome, the protein product of this gene has an essential role in translation and is universally conserved in the prokaryotic world.<a reveal-id="dsz012-B42" data-open="dsz012-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsz012-B43" data-open="dsz012-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Thus, with the aim of identifying important elements, we aligned the RRs of MPN665 orthologues found in a representative set of Mycoplasma species (<a reveal-id="dsz012-F2" data-open="dsz012-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). For the design of the SynMyco RR, we mainly kept those bases found in the MPN665 RR that are not changed to another base in other species. However, given that native RRs did not evolve to be the most productive transcriptional drivers, we also favoured those bases that were found to be optimal as transcription/translation determinants in a recent screen of synthetic sequences.<a reveal-id="dsz012-B44" data-open="dsz012-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> This is exemplified by including an extended Pribnow box motif (TGN-Pribnow) within the SynMyco RR, a feature that is not found in any of the native EF-Tu RRs analysed but promotes higher transcription rates. Based on the screen mentioned above, we also changed the Pribnow box sequence from TAAAAT to the canonical TATAAT motif. Lastly, to ensure the functionality of the SynMyco RR in a broad range of Mycoplasma species, we paid special attention to other features found in the RRs such as the -35 box or the Shine-Dalgarno region (also known as, Ribosome Binding Site, RBS). In particular, the -35 box seems to have lost its prevalent role as a transcriptional driver during Mycoplasma evolution, as indicated by the absence of any consensus sequence among the most productive RRs found in the screen of synthetic sequences.<a reveal-id="dsz012-B44" data-open="dsz012-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> In fact, only a degenerated -35 box with the sequence TTGANN can be found in the RRs of just 20% of the genes of <em>M. pneumoniae.</em><a reveal-id="dsz012-B45" data-open="dsz012-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> However, as the most prevalent sequence found at the -35 area of the EF-Tu RRs analysed was TGC, we included this in the SynMyco RR. On the other hand, the Shine-Dalgarno region, which is responsible for ribosome docking onto the mRNA, is present in only 26% of <em>M. pneumoniae</em> genes. In contrast, in other species such as <em>M. agalactiae</em>, it is found in as much as 73% of the coding sequences (<a reveal-id="dsz012-F2" data-open="dsz012-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). For this reason, we also included a Shine-Dalgarno region in the design of the SynMyco RR, using the sequence that is most frequently found in the EF-Tu RRs analysed (i.e. 5′-AAAGGA-3′). The complete sequence of the SynMyco RR with its main sequence determinants indicated is shown in <a reveal-id="dsz012-F2" data-open="dsz012-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>. </p>                        <a id="140465237" scrollto-destination="140465237"></a>
<div id="dsz012-F2" data-id="dsz012-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f2.png?Expires=1576602286&amp;Signature=VDQro08g3Rh1nA3Duj9R4xzSWy3ShZowmmMWNoZNLxBLzPxlx1zADDYDG6bpYL0BEN8zmG50zAgqL6J2rDrobkxlZ2GxNEFIDHVL~D3lBCskVcxX4IbI48yKj08wYk-K~VFGtHZUe3EVNA0CTmGsGEQLxrrsB7aXhI5dNUCJdwdDIHSxc~hEcs5q5OFP8OE79AdxRUTdsOa~U-Aheb0~GBGa59qiQ5CNMajo13occ~jPANQKGmFQV6s~cnASfUX129s7f5GnObf~CesbrsOmuEgF3-onUWqtouJwz1zeFXfO-bK2yABHi8dTnX2RkVxgGxA~TOvqiOsMzVvitDhC2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of RRs found in Mycoplasma species. (A) Sequence alignment of the EF-Tu RRs of 10 selected Mycoplasma species. Four main domains are highlighted in boxes: the -35 box, Pribnow box, the RBS sequence and the Translation Starting Codon. In addition, the experimentally determined transcriptional start site (TSS) is also shown for the M. pneumoniae RR. (B) Bar plot representing the fraction of RBS-positive genes normalized by the total number of genes per genome in 21 different Mycoplasma species. (C) Sequence of the SynMyco RR. The boxes highlight the same domains shown in panel A, plus the extended Pribnow domain, and the expected TSS inferred from the one experimentally determined in M. pneumoniae." path-from-xml="dsz012f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465237/dsz012f2.tif" path-from-xml="dsz012f2.tif" target="_blank">Open in new tab</a><a section="140465237" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f2.png?Expires=1576602286&amp;Signature=hGQOjiBqftwaW9gTeN5NH6WRGNU7DZKtRYRCSPu9VlZ3rdb7KPfS01Ks0-v9u0ouoI04GJ~NueVOcXGj4xSKZ2S5~YoRoVzDtLF1X18nLW8ksSx6EcjERDav0O-YQdugwM06FVKhv0KZwGUIhvuuoYxV9Ya0Z-5sV-g6pbyLeoIl06DUePUkYvbZ75qUUWfeIaIJv8tiyL6C-uTa9qo4W3Hw5TpQzwdz-e6k9bW-QQnE77o8cbaPBxGxXRAByFAwvA3sy402U4jNyuDDAlsIke1RRgcobpxUm1Ie78nllCJAdzKiSPC~XuPfq1ljkPpXz6kiCddxNpTSnTfxGTuRmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465237&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Analysis of RRs found in Mycoplasma species. (A) Sequence alignment of the EF-Tu RRs of 10 selected Mycoplasma species. Four main domains are highlighted in boxes: the -35 box, Pribnow box, the RBS sequence and the Translation Starting Codon. In addition, the experimentally determined transcriptional start site (TSS) is also shown for the <em>M. pneumoniae</em> RR. (B) Bar plot representing the fraction of RBS-positive genes normalized by the total number of genes per genome in 21 different Mycoplasma species. (C) Sequence of the SynMyco RR. The boxes highlight the same domains shown in panel A, plus the extended Pribnow domain, and the expected TSS inferred from the one experimentally determined in <em>M. pneumoniae</em>.</p></div>
</div>
<div content-id="dsz012-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f2.png?Expires=1576602286&amp;Signature=VDQro08g3Rh1nA3Duj9R4xzSWy3ShZowmmMWNoZNLxBLzPxlx1zADDYDG6bpYL0BEN8zmG50zAgqL6J2rDrobkxlZ2GxNEFIDHVL~D3lBCskVcxX4IbI48yKj08wYk-K~VFGtHZUe3EVNA0CTmGsGEQLxrrsB7aXhI5dNUCJdwdDIHSxc~hEcs5q5OFP8OE79AdxRUTdsOa~U-Aheb0~GBGa59qiQ5CNMajo13occ~jPANQKGmFQV6s~cnASfUX129s7f5GnObf~CesbrsOmuEgF3-onUWqtouJwz1zeFXfO-bK2yABHi8dTnX2RkVxgGxA~TOvqiOsMzVvitDhC2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of RRs found in Mycoplasma species. (A) Sequence alignment of the EF-Tu RRs of 10 selected Mycoplasma species. Four main domains are highlighted in boxes: the -35 box, Pribnow box, the RBS sequence and the Translation Starting Codon. In addition, the experimentally determined transcriptional start site (TSS) is also shown for the M. pneumoniae RR. (B) Bar plot representing the fraction of RBS-positive genes normalized by the total number of genes per genome in 21 different Mycoplasma species. (C) Sequence of the SynMyco RR. The boxes highlight the same domains shown in panel A, plus the extended Pribnow domain, and the expected TSS inferred from the one experimentally determined in M. pneumoniae." path-from-xml="dsz012f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465237/dsz012f2.tif" path-from-xml="dsz012f2.tif" target="_blank">Open in new tab</a><a section="140465237" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f2.png?Expires=1576602286&amp;Signature=hGQOjiBqftwaW9gTeN5NH6WRGNU7DZKtRYRCSPu9VlZ3rdb7KPfS01Ks0-v9u0ouoI04GJ~NueVOcXGj4xSKZ2S5~YoRoVzDtLF1X18nLW8ksSx6EcjERDav0O-YQdugwM06FVKhv0KZwGUIhvuuoYxV9Ya0Z-5sV-g6pbyLeoIl06DUePUkYvbZ75qUUWfeIaIJv8tiyL6C-uTa9qo4W3Hw5TpQzwdz-e6k9bW-QQnE77o8cbaPBxGxXRAByFAwvA3sy402U4jNyuDDAlsIke1RRgcobpxUm1Ie78nllCJAdzKiSPC~XuPfq1ljkPpXz6kiCddxNpTSnTfxGTuRmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465237&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Analysis of RRs found in Mycoplasma species. (A) Sequence alignment of the EF-Tu RRs of 10 selected Mycoplasma species. Four main domains are highlighted in boxes: the -35 box, Pribnow box, the RBS sequence and the Translation Starting Codon. In addition, the experimentally determined transcriptional start site (TSS) is also shown for the <em>M. pneumoniae</em> RR. (B) Bar plot representing the fraction of RBS-positive genes normalized by the total number of genes per genome in 21 different Mycoplasma species. (C) Sequence of the SynMyco RR. The boxes highlight the same domains shown in panel A, plus the extended Pribnow domain, and the expected TSS inferred from the one experimentally determined in <em>M. pneumoniae</em>.</p></div>
</div>                    <h3 scrollto-destination="140465238" id="140465238" class="section-title">3.3. A transposon vector carrying the SynMyco regulatory region dramatically increases transformation efficiency</h3>
<p>Taking the widely employed pMTnGm vector as reference (<a reveal-id="dsz012-F3" data-open="dsz012-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), we constructed a new transposon vector termed pMTnGm-SynMyco in which both the transposase and the gentamicin resistance coding genes were placed under the control of the SynMyco RR (<a reveal-id="dsz012-F3" data-open="dsz012-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). Subsequently, we used this vector to determine whether TE is higher with pMTnGm-SynMyco than with pMTnGm. To this end, cultures of <em>M. pneumoniae</em>, <em>M. agalactiae</em>, <em>M. gallisepticum</em>, and <em>M. feriruminatoris</em> were transformed in parallel with both vectors. </p>                        <a id="140465240" scrollto-destination="140465240"></a>
<div id="dsz012-F3" data-id="dsz012-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f3.png?Expires=1576602286&amp;Signature=3alfjS-oHrm~T3yQggNGRJ6iVlgakQtlzKPja838a4d94dDSTiIA2W6vIMi0pQJmyfcJoghbGMdDZH9s1DZLlSDOcnriUjEOcNWJD-sB20IrMuYHnTJh-9tD1d7wdP0V1Uu6XZPUFt6YEds83zMp5Ej5U4c601Hm99A7Kj8lhZHqTFMD~e4cpNFxTnfPwIpRiZCaKJ00W8ewDiYgCVCkSejkQ1Ms5SSw55b0pMrM0px0wKBPJ22okQHvSUEweXFlTDwDOMcxrG9rfou2ZTksMTf-aKf7kq5g8caeLbrxs8elTme7xHLHZv45-E979ZnPWOWUXyo7e-lTJTAuq-eXWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between the transposon TE obtained with pMTnGm and pMTnGm-SynMyco in four different Mycoplasma species. (A) Scheme of the key modules of the pMTnGm transposon. (B) Scheme of the key modules of the pMTnGm-SynMyco transposon. For both (A) and (B) the abbreviations that appears in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication and GmR for gentamicin resistance coding gene. (C) Bar plot representing the average CFUs of M. pneumoniae resistant to gentamicin (in logarithmic scale) obtained for three independent transformation replicates carried out with either pMTnGm (left side of each panel) or pMTnGm-SynMyco (right side of each panel). For each group of bars, the average of TE (CFU resistant to gentamicin / total CFU viable after transformation) is displayed on top. The fold change in TE is indicated in arrows connecting both sides of each panel. One-tailed t-test p-values are indicated with one asterisk (*) when P &lt; 0.05 for TE obtained with pMTnGm-SynMyco vector compared to the TE obtained with pMTnGm vector. Similar bar plots are shown in (D) for M. gallisepticum, (E) for M. feriruminatoris and (F) for M. agalactiae." path-from-xml="dsz012f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465240/dsz012f3.tif" path-from-xml="dsz012f3.tif" target="_blank">Open in new tab</a><a section="140465240" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f3.png?Expires=1576602286&amp;Signature=OxHe1VVTvkbO68h9yc1PLD1Uk-cp8aK6W9gxMPiRue05sNc7vFwhpysAmJ~I2wAEXAny6lqBMT6t9pwRStqcxk4bPynmfvszxYg29A6ZO-cogCDX028C343KqWkthLWGyv1xr70J0UyeuTDk1ViRqBWboC0hvx8v9Qt4-D0tb6h-zSUduF3Sc4XNlDwb7vv8c9Us0SJwhl2HJDAycUWC70hVeZEaf5EG39BWIiO9jXw9dvNyNQDhqVi~3lildPNuArBIyajip39LVJ~xSHSepgR-8UZPRsN9t93q8Wn34cjAv5koo2uSNkgxbPoZYw28-RYxacHXnkxurLzQ8AaNdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465240&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison between the transposon TE obtained with pMTnGm and pMTnGm-SynMyco in four different Mycoplasma species. (A) Scheme of the key modules of the pMTnGm transposon. (B) Scheme of the key modules of the pMTnGm-SynMyco transposon. For both (A) and (B) the abbreviations that appears in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication and GmR for gentamicin resistance coding gene. (C) Bar plot representing the average CFUs of <em>M. pneumoniae</em> resistant to gentamicin (in logarithmic scale) obtained for three independent transformation replicates carried out with either pMTnGm (left side of each panel) or pMTnGm-SynMyco (right side of each panel). For each group of bars, the average of TE (CFU resistant to gentamicin / total CFU viable after transformation) is displayed on top. The fold change in TE is indicated in arrows connecting both sides of each panel. One-tailed t-test p-values are indicated with one asterisk (*) when <em>P</em> &lt; 0.05 for TE obtained with pMTnGm-SynMyco vector compared to the TE obtained with pMTnGm vector. Similar bar plots are shown in (D) for <em>M. gallisepticum</em>, (E) for <em>M. feriruminatoris</em> and (F) for <em>M. agalactiae</em>.</p></div>
</div>
<div content-id="dsz012-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f3.png?Expires=1576602286&amp;Signature=3alfjS-oHrm~T3yQggNGRJ6iVlgakQtlzKPja838a4d94dDSTiIA2W6vIMi0pQJmyfcJoghbGMdDZH9s1DZLlSDOcnriUjEOcNWJD-sB20IrMuYHnTJh-9tD1d7wdP0V1Uu6XZPUFt6YEds83zMp5Ej5U4c601Hm99A7Kj8lhZHqTFMD~e4cpNFxTnfPwIpRiZCaKJ00W8ewDiYgCVCkSejkQ1Ms5SSw55b0pMrM0px0wKBPJ22okQHvSUEweXFlTDwDOMcxrG9rfou2ZTksMTf-aKf7kq5g8caeLbrxs8elTme7xHLHZv45-E979ZnPWOWUXyo7e-lTJTAuq-eXWg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between the transposon TE obtained with pMTnGm and pMTnGm-SynMyco in four different Mycoplasma species. (A) Scheme of the key modules of the pMTnGm transposon. (B) Scheme of the key modules of the pMTnGm-SynMyco transposon. For both (A) and (B) the abbreviations that appears in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication and GmR for gentamicin resistance coding gene. (C) Bar plot representing the average CFUs of M. pneumoniae resistant to gentamicin (in logarithmic scale) obtained for three independent transformation replicates carried out with either pMTnGm (left side of each panel) or pMTnGm-SynMyco (right side of each panel). For each group of bars, the average of TE (CFU resistant to gentamicin / total CFU viable after transformation) is displayed on top. The fold change in TE is indicated in arrows connecting both sides of each panel. One-tailed t-test p-values are indicated with one asterisk (*) when P &lt; 0.05 for TE obtained with pMTnGm-SynMyco vector compared to the TE obtained with pMTnGm vector. Similar bar plots are shown in (D) for M. gallisepticum, (E) for M. feriruminatoris and (F) for M. agalactiae." path-from-xml="dsz012f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465240/dsz012f3.tif" path-from-xml="dsz012f3.tif" target="_blank">Open in new tab</a><a section="140465240" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f3.png?Expires=1576602286&amp;Signature=OxHe1VVTvkbO68h9yc1PLD1Uk-cp8aK6W9gxMPiRue05sNc7vFwhpysAmJ~I2wAEXAny6lqBMT6t9pwRStqcxk4bPynmfvszxYg29A6ZO-cogCDX028C343KqWkthLWGyv1xr70J0UyeuTDk1ViRqBWboC0hvx8v9Qt4-D0tb6h-zSUduF3Sc4XNlDwb7vv8c9Us0SJwhl2HJDAycUWC70hVeZEaf5EG39BWIiO9jXw9dvNyNQDhqVi~3lildPNuArBIyajip39LVJ~xSHSepgR-8UZPRsN9t93q8Wn34cjAv5koo2uSNkgxbPoZYw28-RYxacHXnkxurLzQ8AaNdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465240&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison between the transposon TE obtained with pMTnGm and pMTnGm-SynMyco in four different Mycoplasma species. (A) Scheme of the key modules of the pMTnGm transposon. (B) Scheme of the key modules of the pMTnGm-SynMyco transposon. For both (A) and (B) the abbreviations that appears in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication and GmR for gentamicin resistance coding gene. (C) Bar plot representing the average CFUs of <em>M. pneumoniae</em> resistant to gentamicin (in logarithmic scale) obtained for three independent transformation replicates carried out with either pMTnGm (left side of each panel) or pMTnGm-SynMyco (right side of each panel). For each group of bars, the average of TE (CFU resistant to gentamicin / total CFU viable after transformation) is displayed on top. The fold change in TE is indicated in arrows connecting both sides of each panel. One-tailed t-test p-values are indicated with one asterisk (*) when <em>P</em> &lt; 0.05 for TE obtained with pMTnGm-SynMyco vector compared to the TE obtained with pMTnGm vector. Similar bar plots are shown in (D) for <em>M. gallisepticum</em>, (E) for <em>M. feriruminatoris</em> and (F) for <em>M. agalactiae</em>.</p></div>
</div>
<p>We did not find significant differences in TE for <em>M. pneumoniae</em> when transformed with either of the two vectors (<a reveal-id="dsz012-F3" data-open="dsz012-F3" class="link link-reveal link-table xref-fig">Fig 3C</a>), which is not surprising taking into consideration that <em>M. pneumoniae</em> already showed a high TE with the pMTnGm vector. Therefore, these data suggest that the expression levels obtained with the RRs of the pMTnGm vector were already enough to saturate the system in this strain.</p>
<p>On the other hand, the TEs obtained with the pMTnGm-SynMyco vector were significantly higher (<em>P</em> &lt; 0.05) in all other species when compared with the TEs obtained with the pMTnGm vector. In particular, for <em>M. gallisepticum</em>, we observed a 30-fold increase in TE, obtaining more than 10<sup>6</sup> mutants when transformed with pMTnGm-SynMyco (<a reveal-id="dsz012-F3" data-open="dsz012-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>).</p>
<p>For <em>M. feriruminatoris</em>, the strain showing the worst TE with pMTnGm (less than 100 total transformed cells per replicate), we observed a 980-fold increase in TE when transformed with pMTnGm-SynMyco, resulting in more than 90,000 individual clones carrying a transposon insertion (<a reveal-id="dsz012-F3" data-open="dsz012-F3" class="link link-reveal link-table xref-fig">Fig. 3E</a>).</p>
<p>Lastly, for <em>M. agalactiae</em>, we found that whereas with the pMTnGm vector we usually observed slightly more than 200 transformed cells per replicate, we obtained more than 3 × 10<sup>5</sup> insertion mutants when transformed with pMTnGm-SynMyco, a value that represents an increase in TE of more than 1036 times (<a reveal-id="dsz012-F3" data-open="dsz012-F3" class="link link-reveal link-table xref-fig">Fig. 3F</a>).</p>
<p>Thus, when transforming these Mycoplasmas strains with the pMTnGm vector, we can classify them into three different groups according to their TEs. The first group is the high efficiency group and contains <em>M. pneumoniae</em>, which showed a TE of 10<sup>−3</sup>. This is followed by the intermediate TE group (i.e. 10<sup>−5</sup>–10<sup>−6</sup>) composed of <em>M. agalactiae</em> and <em>M. gallisepticum</em>, and finally by the low TE group (i.e. 10<sup>−9</sup>) with <em>M. feriruminatoris</em> as the representative. In contrast, when transforming with the pMTnGm-SynMyco vector, we are only able to distinguish a high TE group (composed of <em>M. pneumoniae</em>, <em>M. agalactiae</em> and <em>M. gallisepticum</em>) and an intermediate TE group, (<em>M. feriruminatoris</em> alone).</p>
<p>It should be noted that although TE is consistently increased in <em>M. gallisepticum</em>, <em>M. feriruminatoris</em> and <em>M. agalactiae</em> when transformed with pMTnGm-SynMyco vector, the actual numbers of total mutants obtained might be further increased carefully by optimizing the transformation protocol for each species, something that we have not addressed in this study. In addition, while we have generated only a gentamicin version of the SynMyco transposon, we hypothesized that the other resistance markers available for Mycoplasmas (i.e. chloramphenicol, puromycin and tetracycline)<a reveal-id="dsz012-B18" data-open="dsz012-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and already implemented in native Tn4001 transposon-derived vectors, might in the future be included in pMTnGm-SynMyco vector in substitution of the gentamicin resistance gene. This would allow the generation of a set of four different transposon vectors highly efficient in a broad range of Mycoplasmas. Furthermore, as was recently shown for <em>M. genitalium</em>, resistance genes can also be flanked by lox sites in vectors carrying the SynMyco RR thereby allowing the recycling of antibiotic markers when employing protocols that require the iterative use of transposon insertion mutagenesis.<a reveal-id="dsz012-B46" data-open="dsz012-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a></p>                    <h3 scrollto-destination="140465247" id="140465247" class="section-title">3.4. Strength comparison of SynMyco regulatory region and those in the native pMTnGm vector</h3>
<p>In all species tested, the number of transformed cells was higher when transformed with pMTnGm-SynMyco vector than with unmodified pMTnGm vector. These results suggested that the protein yields of the transposase and/or the protein conferring resistance to gentamicin were major determinants of the TE. In order to find out which of the two gene products, or both, were responsible for the increase in TE, we developed a reporter assay. This reporter assay allows quantification of the relative strength of SynMyco RR compared with the RRs driving the expression of the transposase (i.e. Tnp RR) and the gentamicin resistance gene (i.e. Gm RR) in the native pMTnGm vector. To this end, we created three different vectors derived from the pMTnGm-SynMyco in which the genes coding for the mCherry and Venus fluorescent proteins were introduced for quantification and normalization purposes, respectively. The region controlling the expression levels of the mCherry protein varies between the three different vectors, being either SynMyco RR, Tnp RR or Gm RR, whereas the RR controlling the expression of Venus reporter is constant in all the constructs, being always SynMyco RR (<a reveal-id="dsz012-F4" data-open="dsz012-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). The three different constructs were transformed in all of the species studied in this work. Subsequently, the ratio between mCherry fluorescence and Venus fluorescence allowed us to determine the relative strength of each RR in <em>M. pneumoniae</em> (<a reveal-id="dsz012-F4" data-open="dsz012-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>), <em>M. gallisepticum</em> (<a reveal-id="dsz012-F4" data-open="dsz012-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>), <em>M. feriruminatoris</em> (<a reveal-id="dsz012-F4" data-open="dsz012-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>) <em>and M. agalactiae</em> (<a reveal-id="dsz012-F4" data-open="dsz012-F4" class="link link-reveal link-table xref-fig">Fig. 4E</a>). </p>                        <a id="140465249" scrollto-destination="140465249"></a>
<div id="dsz012-F4" data-id="dsz012-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f4.png?Expires=1576602286&amp;Signature=dJQotQyncaLn7xYiW5pO2tD7VIwUAbt~Zsq2kLuO8A~Vc-tOwZZwVu0KEZ6lZm4udHPx8lKFZYzGum4vT74AzgfEOmsade~qBTaZeersWiRKOfpDmDGp-gfOGlllZAJFSMErmx8OHyAU0uz1oNUahxmooOinGTSD-cx6DVLkwE50G8O1y7z6Om6xpodR-KiaxZLW866i80KqQJnZ4bMUjhci~KHodazr-eCsTa9AN4BxZX2n690rsVs2tly5IvRDGOM6hucS82rAtzBIRdqn1odO3PXFX5AQSkkLcXk8AgNff51JkV1wtnQHVCbKxbcX-k7HbVZUDx1yV~Lorcykjg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the SynMyco RR efficiency versus the native regulatory regions of pMTnGm transposon. (A) Scheme of the key modules of (i) pMTnGm-SynMyco+SynMyco RR-Venus+SynMycoRR-mCherry transposon, (ii) pMTnGm-SynMyco+SynMycoRR-Venus+Gm RR-mCherry transposon and (iii) pMTnGm-SynMyco+SynMyco-Venus+TnpRR-mCherry transposon. The abbreviations that appear in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication, GmR for gentamicin resistance coding gene, Venus for Venus protein coding gene and mCherry as mCherry protein coding gene. (B) Bar plot representing the ratio obtained in M. pneumoniae  for mCherry/Venus fluorescence using transposons represented in panel A. While Venus reporter is always under control of SynMyco RR, the mCherry gene is controlled by SynMyco RR in the left bar, Gm RR in the central bar and Tnp RR in the right bar. On top of each bar is represented the average ratio of mCherry/Venus fluorescence obtained for each of the constructs in three replicates. One-tailed t-test p-values are indicated with one asterisk (*) when P &lt; 0.05 for the ratio of mCherry/Venus under the control of SynMyco RR versus either Gm RR or Tnp RR. Similar bar plots are shown in (C) for M. gallisepticum, (D) for M. feriruminatoris and (E) for M. agalactiae." path-from-xml="dsz012f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465249/dsz012f4.tif" path-from-xml="dsz012f4.tif" target="_blank">Open in new tab</a><a section="140465249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f4.png?Expires=1576602286&amp;Signature=Hef3reKK4BKWYQMGdaRvVpoJOf39fsIn5A8anNPfDVao23GJox-SmgjKxLt9AnL04jf3IgaQuN4Q1tY6e90nHjCKT1VWx1FcnVSdCLBWj4xhT-0ds8UYcqtWS1jWsJyPqI77dSSbbYRZLpg0wXaouVtYkSpI5QGdx0~EpEN4ghA1qun0mI3YnSWT4QIY5SJ4B~0Whi5AwU0~9fal8xaVCif1JewyoLkf~DaeHYMmBqFpytUH28yDxHMBQ78nt77qd0t82UAyylNk4LUGRg6pYpu5jz5JnlGbCFop61G3mr3ug41JIL-1YK2TJTGNPwp1dBPJ5BsRfNOddjjyE4Wp5Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465249&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of the SynMyco RR efficiency versus the native regulatory regions of pMTnGm transposon. (A) Scheme of the key modules of (i) pMTnGm-SynMyco+SynMyco RR-Venus+SynMycoRR-mCherry transposon, (ii) pMTnGm-SynMyco+SynMycoRR-Venus+Gm RR-mCherry transposon and (iii) pMTnGm-SynMyco+SynMyco-Venus+TnpRR-mCherry transposon. The abbreviations that appear in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication, GmR for gentamicin resistance coding gene, Venus for Venus protein coding gene and mCherry as mCherry protein coding gene. (B) Bar plot representing the ratio obtained in <em>M. pneumoniae </em> for mCherry/Venus fluorescence using transposons represented in panel A. While Venus reporter is always under control of SynMyco RR, the mCherry gene is controlled by SynMyco RR in the left bar, Gm RR in the central bar and Tnp RR in the right bar. On top of each bar is represented the average ratio of mCherry/Venus fluorescence obtained for each of the constructs in three replicates. One-tailed t-test p-values are indicated with one asterisk (*) when <em>P</em> &lt; 0.05 for the ratio of mCherry/Venus under the control of SynMyco RR versus either Gm RR or Tnp RR. Similar bar plots are shown in (C) for <em>M. gallisepticum</em>, (D) for <em>M. feriruminatoris</em> and (E) for <em>M. agalactiae</em>.</p></div>
</div>
<div content-id="dsz012-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f4.png?Expires=1576602286&amp;Signature=dJQotQyncaLn7xYiW5pO2tD7VIwUAbt~Zsq2kLuO8A~Vc-tOwZZwVu0KEZ6lZm4udHPx8lKFZYzGum4vT74AzgfEOmsade~qBTaZeersWiRKOfpDmDGp-gfOGlllZAJFSMErmx8OHyAU0uz1oNUahxmooOinGTSD-cx6DVLkwE50G8O1y7z6Om6xpodR-KiaxZLW866i80KqQJnZ4bMUjhci~KHodazr-eCsTa9AN4BxZX2n690rsVs2tly5IvRDGOM6hucS82rAtzBIRdqn1odO3PXFX5AQSkkLcXk8AgNff51JkV1wtnQHVCbKxbcX-k7HbVZUDx1yV~Lorcykjg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the SynMyco RR efficiency versus the native regulatory regions of pMTnGm transposon. (A) Scheme of the key modules of (i) pMTnGm-SynMyco+SynMyco RR-Venus+SynMycoRR-mCherry transposon, (ii) pMTnGm-SynMyco+SynMycoRR-Venus+Gm RR-mCherry transposon and (iii) pMTnGm-SynMyco+SynMyco-Venus+TnpRR-mCherry transposon. The abbreviations that appear in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication, GmR for gentamicin resistance coding gene, Venus for Venus protein coding gene and mCherry as mCherry protein coding gene. (B) Bar plot representing the ratio obtained in M. pneumoniae  for mCherry/Venus fluorescence using transposons represented in panel A. While Venus reporter is always under control of SynMyco RR, the mCherry gene is controlled by SynMyco RR in the left bar, Gm RR in the central bar and Tnp RR in the right bar. On top of each bar is represented the average ratio of mCherry/Venus fluorescence obtained for each of the constructs in three replicates. One-tailed t-test p-values are indicated with one asterisk (*) when P &lt; 0.05 for the ratio of mCherry/Venus under the control of SynMyco RR versus either Gm RR or Tnp RR. Similar bar plots are shown in (C) for M. gallisepticum, (D) for M. feriruminatoris and (E) for M. agalactiae." path-from-xml="dsz012f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465249/dsz012f4.tif" path-from-xml="dsz012f4.tif" target="_blank">Open in new tab</a><a section="140465249" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f4.png?Expires=1576602286&amp;Signature=Hef3reKK4BKWYQMGdaRvVpoJOf39fsIn5A8anNPfDVao23GJox-SmgjKxLt9AnL04jf3IgaQuN4Q1tY6e90nHjCKT1VWx1FcnVSdCLBWj4xhT-0ds8UYcqtWS1jWsJyPqI77dSSbbYRZLpg0wXaouVtYkSpI5QGdx0~EpEN4ghA1qun0mI3YnSWT4QIY5SJ4B~0Whi5AwU0~9fal8xaVCif1JewyoLkf~DaeHYMmBqFpytUH28yDxHMBQ78nt77qd0t82UAyylNk4LUGRg6pYpu5jz5JnlGbCFop61G3mr3ug41JIL-1YK2TJTGNPwp1dBPJ5BsRfNOddjjyE4Wp5Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465249&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of the SynMyco RR efficiency versus the native regulatory regions of pMTnGm transposon. (A) Scheme of the key modules of (i) pMTnGm-SynMyco+SynMyco RR-Venus+SynMycoRR-mCherry transposon, (ii) pMTnGm-SynMyco+SynMycoRR-Venus+Gm RR-mCherry transposon and (iii) pMTnGm-SynMyco+SynMyco-Venus+TnpRR-mCherry transposon. The abbreviations that appear in the figure are: Tnp for transposase coding gene, Amp for ampicillin resistance coding gene, IR for inverted repeats, ColE1 ori for ColE1 origin of replication, GmR for gentamicin resistance coding gene, Venus for Venus protein coding gene and mCherry as mCherry protein coding gene. (B) Bar plot representing the ratio obtained in <em>M. pneumoniae </em> for mCherry/Venus fluorescence using transposons represented in panel A. While Venus reporter is always under control of SynMyco RR, the mCherry gene is controlled by SynMyco RR in the left bar, Gm RR in the central bar and Tnp RR in the right bar. On top of each bar is represented the average ratio of mCherry/Venus fluorescence obtained for each of the constructs in three replicates. One-tailed t-test p-values are indicated with one asterisk (*) when <em>P</em> &lt; 0.05 for the ratio of mCherry/Venus under the control of SynMyco RR versus either Gm RR or Tnp RR. Similar bar plots are shown in (C) for <em>M. gallisepticum</em>, (D) for <em>M. feriruminatoris</em> and (E) for <em>M. agalactiae</em>.</p></div>
</div>
<p>In all species, the ratio between mCherry and Venus fluorescence is close to 1 (i.e. 0.9 for <em>M. pneumoniae</em>, 1.1 for <em>M. gallisepticum</em>, 1.1 for <em>M. feriruminatoris</em> and 1.3 for <em>M. agalactiae</em>) when both reporters were under control of the SynMyco RR. Interestingly, in <em>M. gallisepticum</em> and <em>M. feriruminatoris</em>, the ratio is also close to 1 when mCherry is under control of Gm RR (i.e. 1 for <em>M. gallisepticum</em> and 1.1 for <em>M. feriruminatoris</em>) but drastically drops when the expression of mCherry is driven by Tnp RR (i.e. 0.2 for <em>M. gallisepticum</em> and 0.03 for <em>M. feriruminatoris).</em> Altogether, these data suggest that in these two species, when transforming with pMTnGm or pMTnGm-SynMyco vectors, the difference in TE would be due to the expression level of the transposase gene and not the antibiotic resistance gene.</p>
<p>Poor performance of Tnp RR was also observed in <em>M. pneumoniae</em> and <em>M. agalactiae</em> as indicated by the low fluorescence ratio observed when mCherry is under control of Tnp RR (i.e. 0.1 for <em>M. pneumoniae</em> and 0.1 for <em>M. agalactiae</em>). Moreover, in contrast to <em>M. gallisepticum</em> and <em>M. feriruminatoris</em> where Gm RR and SynMyco RR provide similar protein yields, in <em>M. pneumoniae</em> and <em>M. agalactiae</em> the fluorescence ratio is also reduced when the expression of mCherry is driven by Gm RR (i.e. 0.1 for <em>M. pneumoniae</em> and 0.2 for <em>M. agalactiae</em>). Thus, in <em>M. pneumoniae</em> and <em>M. agalactiae</em>, SynMyco RR outperforms not only Tnp RR but also GmRR in terms of protein yields.</p>
<p>It has not escaped our notice that the reporter assay shows the same expression profile for <em>M. agalactiae</em> and <em>M. pneumoniae</em>, whereas TE upon using pMTnGm-SynMyco is only drastically increased in <em>M. agalactiae</em> (i.e. 1036-fold change) but not in <em>M. pneumoniae</em> (i.e. 1.1-fold change)<em>.</em> We hypothesized that this observation might be related with the different doubling times of the species. Slow dividing species, such as <em>M. pneumoniae</em>, would have more time to deliver the transposon cargo into the chromosome before cell duplication starts to dilute the plasmid within the growing population. In contrast, in species dividing faster, such as <em>M. agalactiae</em>, a quick transposition into the chromosome mediated by high expression levels of the transposase coding gene would represent an advantage to avoid the dilution effect associated with cell division and thus would lead to an increased TE. An alternative or complementary hypothesis is that <em>M. pneumoniae</em> might be somehow more permissive than the other species for the presence of extrachromosomal elements inside the cell. In this scenario, a fast transposition into the chromosome mediated by high expression levels of the transposase coding gene would represent a greater advantage for the other mycoplasma strains than for <em>M. pneumoniae.</em></p>
<p>In summary, our reporter assay shows that SynMyco RR is efficiently recognized in all the species tested, and suggests that the factor limiting the TE with native pMTnGm vector in most Mycoplasma species is the expression of the transposase coding gene.</p>                    <h3 scrollto-destination="140465254" id="140465254" class="section-title">3.5. Unblocking the global study of essential and dispensable genes in <em>M. agalactiae</em>
</h3>
<p>To the best of our knowledge, studies regarding essentiality in Mycoplasmas have only been published so far for <em>M. genitalium</em>,<a reveal-id="dsz012-B8" data-open="dsz012-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><em>M. pneumoniae</em>,<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><em>Mycoplasma pulmonis</em>,<a reveal-id="dsz012-B47" data-open="dsz012-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><em>Mycoplasma bovis</em><a reveal-id="dsz012-B10" data-open="dsz012-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> or <em>Mesoplasma florum.</em><a reveal-id="dsz012-B48" data-open="dsz012-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> However, while for <em>M. pneumoniae</em> almost 350,000 unique transposon insertion mutants have been tracked, the other studies analysed a substantially lower number of clones. For instance, the <em>M. bovis</em> study only obtained 319 mutants, representing one insertion every 3,145 bp of the total genome,<a reveal-id="dsz012-B10" data-open="dsz012-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> and the study in <em>M. genitalium</em> analysed around 3,300 mutants, showing an insertional coverage of one disruption over every 175 bp.<a reveal-id="dsz012-B8" data-open="dsz012-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Obviously, the accuracy of the assignment of genes to each one of the three categories of essentiality [i.e. essential (E), non-essential (NE) and fitness (F)] is directly related to the insertional coverage of the transposon mutagenesis. Moreover, the existence in bacteria of small open reading frame-encoded polypeptides (i.e. short open reading frame-encoded proteins, also known as SEPs) as short as 11 amino acids in length is becoming widely accepted.<a reveal-id="dsz012-B49" data-open="dsz012-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Thus, high transposon TE is necessary to study the essentiality of these SEPs.<a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsz012-B50" data-open="dsz012-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a></p>
<p>To illustrate the potential that our pMTnGm-SynMyco vector could have in the study of a broad range of Mycoplasma species, we decided to perform an essentiality study for <em>M. agalactiae</em> 7784, a strain for which no essentiality map is currently available. First, we sequenced, assembled and annotated for the first time the genome of this strain. Then, after transforming the strain with pMTnGm-SynMyco vector we were able to map 199,723 transposon insertions to the <em>M. agalactiae</em> genome, with an estimated genome size of 853,960. This represents ∼23.3 insertions over every 100 bp, which is around half of the coverage observed in <em>M. pneumoniae</em><a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> (354,447 transposon insertions mapped, representing ∼43 insertions over every 100 bp) but still substantially higher than the ones reported in other essentiality studies (<a reveal-id="dsz012-F5" data-open="dsz012-F5" class="link link-reveal link-table xref-fig">Fig. 5A and B</a>). </p>                        <a id="140465257" scrollto-destination="140465257"></a>
<div id="dsz012-F5" data-id="dsz012-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f5.png?Expires=1576602286&amp;Signature=CQiQzog5E8CQILntCVxAqMiqjURQMszUg~HEVCbmyMz-BURODw~MylXMqguo9I~li5gNfGDAW6BIhTvL4m7KbIN-xVDmIuqhWdVxa~89jJ~jlGI4efOfZkkUEXAEYQdlNAxLp~oRGxPImJ28fjR4o9mLnWS-u2cfCWN8KEcZTr2aLiV-m94zl18~H7VOfmTjJ1epkpyzNAuXLvn2Xa1tGDIUDqAdaC206MMlCQTxURyynu4GS47Y8ktJjrLnZriIMbSaZh7Oa9H6yeskTbyKD7Yrk6Hnzi30GTdVr7A3chrD7xZdEe47eWhnQSwuXRB6qVUIB-md3ZooDxXxbYMTiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Essentiality study in M. agalactiae using the pMTnGm-SynMyco transposon and a comparison with previous studies in M. pneumoniae. (A) Genome disruption profile for M. pneumoniae. The y-axis represents the logarithmic average of total reads covering a window of 1,000 bp (x-axis). (B) Genome disruption profile for M. agalactiae representing the same information as in the previous panel. (C) Insertion density by gene distribution in M. pneumoniae and M. agalactiae as indicated. The x-axis represents the percentage of bp in a gene that is disrupted and the y-axis the frequency of densities in the distribution. To better compare M. pneumoniae and M. agalactiae transposon insertion distributions, we standardized both distributions using min-max scaling. (D) Box-plot representing the statistical comparison of specific subsets of genes expected to be essential (E) and non-essential (NE) in M. pneumoniae and M. agalactiae as indicated. The asterisk represents P-value &lt; 0.05 (3.62 e−41 and 1.20 e−20) when comparing density of insertions of E and NE coding genes in M. pneumoniae and M. agalactiae respectively." path-from-xml="dsz012f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465257/dsz012f5.tif" path-from-xml="dsz012f5.tif" target="_blank">Open in new tab</a><a section="140465257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f5.png?Expires=1576602286&amp;Signature=4ueo-H7zHoZ~roqKAFa3DEzgum6YqMAhV83yFF0IHwnSVYVltgLidxKdhvDkdHix0NoJtRY1hZ0HRPOW11WQwgp7lVZnWBjev3L8zGtdZqauQ0xECwEREa4kkzLz-ipyizgD3pwfRrbflHqEemDqFLf96zabnmFVjz7lMETAFjdMyZGo9Y70VTdEahWjVB17nv3wcc8a3WL-IYrL-5MUAgqv1UtCOX99esjr4hfSGEaRE1WU-drLa8sI95J2kljEuTJtS13E2dBTRK0OPE4G7ZkSYtoFsYraU-ffftqyu3m8H4RuJuw2sJQtmRhxeVGfySRKW4PKsVBpAhAeDpQQ6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465257&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Essentiality study in <em>M. agalactiae </em>using the pMTnGm-SynMyco transposon and a comparison with previous studies in <em>M. pneumoniae</em>. (A) Genome disruption profile for <em>M. pneumoniae</em>. The y-axis represents the logarithmic average of total reads covering a window of 1,000 bp (x-axis). (B) Genome disruption profile for <em>M. agalactiae</em> representing the same information as in the previous panel. (C) Insertion density by gene distribution in <em>M. pneumoniae</em> and <em>M. agalactiae</em> as indicated. The x-axis represents the percentage of bp in a gene that is disrupted and the y-axis the frequency of densities in the distribution. To better compare <em>M. pneumoniae</em> and <em>M. agalactiae</em> transposon insertion distributions, we standardized both distributions using min-max scaling. (D) Box-plot representing the statistical comparison of specific subsets of genes expected to be essential (E) and non-essential (NE) in <em>M. pneumoniae</em> and <em>M. agalactiae</em> as indicated. The asterisk represents <em>P</em>-value &lt; 0.05 (3.62 e<sup>−41</sup> and 1.20 e<sup>−20</sup>) when comparing density of insertions of E and NE coding genes in <em>M. pneumoniae</em> and <em>M. agalactiae</em> respectively.</p></div>
</div>
<div content-id="dsz012-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/m_dsz012f5.png?Expires=1576602286&amp;Signature=CQiQzog5E8CQILntCVxAqMiqjURQMszUg~HEVCbmyMz-BURODw~MylXMqguo9I~li5gNfGDAW6BIhTvL4m7KbIN-xVDmIuqhWdVxa~89jJ~jlGI4efOfZkkUEXAEYQdlNAxLp~oRGxPImJ28fjR4o9mLnWS-u2cfCWN8KEcZTr2aLiV-m94zl18~H7VOfmTjJ1epkpyzNAuXLvn2Xa1tGDIUDqAdaC206MMlCQTxURyynu4GS47Y8ktJjrLnZriIMbSaZh7Oa9H6yeskTbyKD7Yrk6Hnzi30GTdVr7A3chrD7xZdEe47eWhnQSwuXRB6qVUIB-md3ZooDxXxbYMTiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Essentiality study in M. agalactiae using the pMTnGm-SynMyco transposon and a comparison with previous studies in M. pneumoniae. (A) Genome disruption profile for M. pneumoniae. The y-axis represents the logarithmic average of total reads covering a window of 1,000 bp (x-axis). (B) Genome disruption profile for M. agalactiae representing the same information as in the previous panel. (C) Insertion density by gene distribution in M. pneumoniae and M. agalactiae as indicated. The x-axis represents the percentage of bp in a gene that is disrupted and the y-axis the frequency of densities in the distribution. To better compare M. pneumoniae and M. agalactiae transposon insertion distributions, we standardized both distributions using min-max scaling. (D) Box-plot representing the statistical comparison of specific subsets of genes expected to be essential (E) and non-essential (NE) in M. pneumoniae and M. agalactiae as indicated. The asterisk represents P-value &lt; 0.05 (3.62 e−41 and 1.20 e−20) when comparing density of insertions of E and NE coding genes in M. pneumoniae and M. agalactiae respectively." path-from-xml="dsz012f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/140465257/dsz012f5.tif" path-from-xml="dsz012f5.tif" target="_blank">Open in new tab</a><a section="140465257" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012f5.png?Expires=1576602286&amp;Signature=4ueo-H7zHoZ~roqKAFa3DEzgum6YqMAhV83yFF0IHwnSVYVltgLidxKdhvDkdHix0NoJtRY1hZ0HRPOW11WQwgp7lVZnWBjev3L8zGtdZqauQ0xECwEREa4kkzLz-ipyizgD3pwfRrbflHqEemDqFLf96zabnmFVjz7lMETAFjdMyZGo9Y70VTdEahWjVB17nv3wcc8a3WL-IYrL-5MUAgqv1UtCOX99esjr4hfSGEaRE1WU-drLa8sI95J2kljEuTJtS13E2dBTRK0OPE4G7ZkSYtoFsYraU-ffftqyu3m8H4RuJuw2sJQtmRhxeVGfySRKW4PKsVBpAhAeDpQQ6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=140465257&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsz012f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Essentiality study in <em>M. agalactiae </em>using the pMTnGm-SynMyco transposon and a comparison with previous studies in <em>M. pneumoniae</em>. (A) Genome disruption profile for <em>M. pneumoniae</em>. The y-axis represents the logarithmic average of total reads covering a window of 1,000 bp (x-axis). (B) Genome disruption profile for <em>M. agalactiae</em> representing the same information as in the previous panel. (C) Insertion density by gene distribution in <em>M. pneumoniae</em> and <em>M. agalactiae</em> as indicated. The x-axis represents the percentage of bp in a gene that is disrupted and the y-axis the frequency of densities in the distribution. To better compare <em>M. pneumoniae</em> and <em>M. agalactiae</em> transposon insertion distributions, we standardized both distributions using min-max scaling. (D) Box-plot representing the statistical comparison of specific subsets of genes expected to be essential (E) and non-essential (NE) in <em>M. pneumoniae</em> and <em>M. agalactiae</em> as indicated. The asterisk represents <em>P</em>-value &lt; 0.05 (3.62 e<sup>−41</sup> and 1.20 e<sup>−20</sup>) when comparing density of insertions of E and NE coding genes in <em>M. pneumoniae</em> and <em>M. agalactiae</em> respectively.</p></div>
</div>
<p>When considering the frequency of insertions per gene, we observed a bimodal distribution separating essential from non-essential genes in both strains (<a reveal-id="dsz012-F5" data-open="dsz012-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). Next, we explored the insertional profile at the gene level using sets of known E and NE genes as inferred from the reference essentiality study in <em>M. pneumoniae.</em><a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Whereas for <em>M. pneumoniae</em> the training sets for E and NE genes comprise 27 and 28 genes, respectively, for <em>M. agalactiae</em> we generated dedicated training sets based on bibliography containing 32 E genes and 17 NE genes. When comparing the average density of insertions between E and NE genes in <em>M. pneumoniae</em> and <em>M. agalactiae</em>, we observed that the two groups were significantly different within each species (two-tailed <em>t</em>-test with equal variances, <em>P</em>-values equal to 1.42e<sup>−47</sup> and 1.05e<sup>−20</sup>, respectively; <a reveal-id="dsz012-F5" data-open="dsz012-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). Taken altogether, we were able to assign one category for 689 genes found in the <em>M. agalactiae 7784</em> genome: E (43.98% of genes), F (25.25% of genes) or NE (30.77% of genes) (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>), in line with the percentage of E (49.28%), F (13.40%) and NE (37.32%) genes obtained for <em>M. pneumoniae.</em><a reveal-id="dsz012-B6" data-open="dsz012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p>
<p>When <em>M. agalactiae</em> essential genes were classified according to the COGs functional categories annotation system, we found that the categories that were significantly enriched in essential genes were: (i) protein coding genes involved in translation, ribosomal structure and biogenesis, (ii) functional RNAs, (iii) protein coding genes without assigned COG category and (iv) protein coding genes with unknown function (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576602286&amp;Signature=R2By-umywrjNP6gwJDANrti3ssCcTiHHwrpJyfXd8kAmMwR-oBWJvJOaRwFbZO94bw-iuajqeZLAj2Iul6vSRozsC7b0g6hST1e2bHb0n~MdyBviQo3B6CPpy7BO3zN0eKwYA4npsR8gSshweEZGmJW-samZn5Ba22uyjpqzeIUI4Hk1gFtyVnW0eKLFFuNidI77oEe1-9Ku-zRe12GJ9V8mTDHyZQmR2BZbE8Ydr8Fc5z5jE39lws~dfRiWPYAxrEDFlliNYZZwK-GNfZmzRImytgXZOqV4L41iX5iWmlQE4WGNkvsSmR66dslcu5~GFn1ce0-nvMon9Hk0m7YpuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>).</p>
<p>However, for the purpose of this work it is more relevant to compare the essentiality map of <em>M. agalactiae</em> at high density of insertions with one obtained with low density to illustrate the importance of high coverage<em>.</em> Unfortunately, there is no essentiality study done in <em>M. agalactiae</em>, but there is a low coverage analysis in its closely related specie <em>M. bovis</em> PG45<sup>10</sup> (319 transposon insertional mutants individually sequenced, versus more than 199,000 transposon insertions analysed by deep sequencing in our study). Sequencing of individual insertion clones is useful to classify the genes within the F-NE categories (i.e. genes disrupted by a transposon insertion) but cannot distinguish between both. Examples of this limitation are the genes coding for a tRNA modification GTPase and for the deoxyribonuclease IV. In the <em>M. bovis</em> essentiality study, both genes (i.e. MBOVPG45_0060 and MBOVPG45_0301) were classified as NE given the isolation of individual clones carrying insertions within their coding region. However, our essentiality study based on the analysis of at least 600 times more mutants than the one of <em>M. bovis</em> and mapped by ultra sequencing classified these two genes (i.e. MAGA7784_RS00280 and MAGA7784_RS02715) as F in <em>M. agalactiae</em>. This implies that although these genes are non-essential, their disruption cause a growth impairment of these particular clones at least for <em>M. agalactiae</em>, something that cannot be directly measured by genomic sequencing of transposon insertion in a limited number of clones.</p>
<p>Moreover, aside from its lack of accuracy in distinguishing between NE and F genes, the main limitation of small transposon libraries is related with E genes. Specifically, the low coverage of these libraries makes it difficult to ascertain whether a gene is free of insertions because of its essential character or as a result of the randomness of the integration and/or low sampling of the mutants. Indeed, only two genes were suggested by the authors as highly probable essential genes in the <em>M. bovis</em> study, MBOVPG45_0337 and MBOVPG45_0710, coding for an ATP-binding protein and SGNH/GDSL hydrolase, respectively. This assumption was based on the lack of clones carrying a transposon insertion within the coding regions of these genes in spite of their large size (i.e. 3420 bp for MBOVPG45_0337 and 8013 bp for MBOVPG45_0710). Nonetheless, it should be noted that the insertional coverage of the mutant library of this study, with one insertion every 3145 bp of the genome on average, is in the same range of these genes in terms of size. In our <em>M. agalactiae</em> essentiality study, the orthologue of the SGNH/GDSL hydrolase coding gene (i.e. MAGA7784_RS03410) was found E, confirming the hypothesis of the <em>M. bovis</em> report. In contrast, the gene coding for the ATP-binding protein (i.e. MAGA7784_RS02615) was determined as NE as indicated by the presence of 891 transposon insertions within its sequence. This suggests that the absence of clones carrying an insertion within MBOVPG45_0337 gene is most likely a consequence of the low coverage of the insertional library, rather than the gene being truly essential in <em>M. bovis.</em> Furthermore, the limitation of low coverage libraries to assign a truly E character for a given gene is even more evident with small coding sequences. For instance, in <em>M. bovis</em> there were no clones found to carry a transposon insertion within the coding sequences of genes MBOVPG45_0043 (582 bp) or MBOVPG45_0596 (963 bp), coding for sigma-70 RNA polymerase factor and Holliday junction branch migration DNA helicase ruvB, respectively. This might indicate that these genes are E. However, homologues of these genes are fully covered with transposon insertions not only in <em>M. agalactiae</em> (MAGA7784_RS00215 and MAGA7784_RS01205), but also in <em>M. pneumoniae</em> (MPN626 and MPN536), suggesting the lack of insertions in the <em>M. bovis</em> study is more likely related with the small size of the genes rather than with their essential character. All of these examples illustrate the importance of high coverage transposon insertion libraries for the appropriate category assignment of a given gene, something that can be only achieved with efficient transposon vectors such as pMTnGm-SynMyco.</p>
<p>In conclusion, we demonstrate that the RRs driving the expression of the transposase and the resistance genes have a tremendous impact on the TE achieved after vector transformation. Although problems derived from poor recognition of vector RRs can be avoided with transformation procedures based on purified transposases,<a reveal-id="dsz012-B51" data-open="dsz012-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> it should be noted that not all transposases are commercially available and their cost might limit use in many research groups. Indeed, screening libraries of RRs for the key elements of transposon vectors has already been shown to be a useful strategy to produce a moderate increase (i.e. around one order of magnitude) in the TE of different bacteria.<a reveal-id="dsz012-B52" data-open="dsz012-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> However, an approach such as the one followed in our study involving rational design of RRs rather than the screen of a limited number of variants might be more effective in increasing TE. In fact, we have seen that the pMTnGm-SynMyco vector significantly increases the TE in three phylogenetically distant Mycoplasma species. For this reason, we hypothesize that this vector might improve the reported TE in all species belonging to the Mycoplasma genus. Even though we have shown that the SynMyco RR is efficiently recognized in different species, it cannot be excluded that other designs of RRs could also lead to an increase in TE. As exemplified in this work with <em>M. agalactiae</em>, the higher TE obtained using this vector may unblock the development of new essentiality maps for other Mycoplasma species, and thereby promote global knowledge of these interesting microorganisms. In addition, the higher insertional coverage obtained with the pMTnGm-SynMyco vector should facilitate the isolation of clones of interest from libraries of insertional mutants, an advancement which may boost gene-function assignments in Mycoplasma species that have been poorly studied so far.</p>                    <h2 scrollto-destination="140465263" id="140465263" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>We thank Dr Sarah A. Head for critical manuscript revision. We also thank Tony Ferrar for language editing (<a class="link link-uri" href="http://www.theeditorsite.com" target="_blank">http://www.theeditorsite.com</a>). We acknowledge the staff of the CRG Genomics Unit and the CRG Bioinformatics Unit, especially Jochen Hecht and Luca Cozzuto for their assistance and fruitful discussions.</p>                    <h3 scrollto-destination="140465265" id="140465265" class="section-title">Funding</h3>
<p>This project has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement 634942 (MycoSynVac) and was also ﬁnanced by the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme, under grant agreement 670216 (MYCOCHASSIS). We also acknowledge support of the Spanish Ministry of Economy, Industry and Competitiveness (MEIC) to the EMBL partnership, the Centro de Excelencia Severo Ochoa and the CERCA Programme/Generalitat de Catalunya.</p>                    <h3 scrollto-destination="140465267" id="140465267" class="section-title">Accession numbers</h3>
<p>The raw data of DNAseq, genome assembly and <em>de novo </em>annotation was submitted as BioProject under accession PRJNA528179. The transposon sequencing in <em>Mycoplasma agalactiae </em>dataset can be found in ArrayExpress with accession number: E-MTAB-7425.</p>                    <h3 scrollto-destination="140465269" id="140465269" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="140465271" id="140465271" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsz012-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Woese</div> <div class="given-names">C.R.</div>
</div>, <div class="name string-name">
<div class="surname">Maniloff</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Zablen</div> <div class="given-names">L.B.</div>
</div>
</div> <div class="year">1980</div>, 
<div class="article-title">Phylogenetic analysis of the mycoplasmas</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">77</div>, <div class="fpage">494</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20the%20mycoplasmas&amp;author=C.R.%20Woese&amp;author=J.%20Maniloff&amp;author=L.B.%20Zablen&amp;publication_year=1980&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=77&amp;pages=494-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.77.1.494" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.77.1.494" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.77.1.494"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenetic%20analysis%20of%20the%20mycoplasmas&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Himmelreich</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Hilbert</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Plagens</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Pirkl</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">B.C.</div>
</div>, <div class="name string-name">
<div class="surname">Herrmann</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">1996</div>, 
<div class="article-title">Complete sequence analysis of the genome of the bacterium Mycoplasma pneumoniae</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">24</div>, <div class="fpage">4420</div>–<div class="lpage">49</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Complete%20sequence%20analysis%20of%20the%20genome%20of%20the%20bacterium%20Mycoplasma%20pneumoniae&amp;author=R.%20Himmelreich&amp;author=H.%20Hilbert&amp;author=H.%20Plagens&amp;author=E.%20Pirkl&amp;author=B.C.%20Li&amp;author=R.%20Herrmann&amp;publication_year=1996&amp;journal=Nucleic%20Acids%20Res&amp;volume=24&amp;pages=4420-49" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/24.22.4420" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.22.4420" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F24.22.4420"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8948633" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Complete%20sequence%20analysis%20of%20the%20genome%20of%20the%20bacterium%20Mycoplasma%20pneumoniae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Guell</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">van Noort</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Yus</div> <div class="given-names">E.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Transcriptome complexity in a genome-reduced bacterium</div>, <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">1268</div>–<div class="lpage">71</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20complexity%20in%20a%20genome-reduced%20bacterium&amp;author=M.%20Guell&amp;author=V.%20van%20Noort&amp;author=E.%20Yus&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1268-71" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1176951" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1176951" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1176951"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965477" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcriptome%20complexity%20in%20a%20genome-reduced%20bacterium&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Maier</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Schmidt</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Guell</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Quantification of mRNA and protein and integration with protein turnover in a bacterium</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">7</div>, <div class="fpage">511</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Quantification%20of%20mRNA%20and%20protein%20and%20integration%20with%20protein%20turnover%20in%20a%20bacterium&amp;author=T.%20Maier&amp;author=A.%20Schmidt&amp;author=M.%20Guell&amp;publication_year=2011&amp;journal=Mol.%20Syst.%20Biol&amp;volume=7&amp;pages=511" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/msb.2011.38" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.38" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fmsb.2011.38"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21772259" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Quantification%20of%20mRNA%20and%20protein%20and%20integration%20with%20protein%20turnover%20in%20a%20bacterium&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lluch-Senar</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Luong</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Lloréns-Rico</div> <div class="given-names">V.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Comprehensive methylome characterization of Mycoplasma genitalium and Mycoplasma pneumoniae at single-base resolution</div>, <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003191</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20methylome%20characterization%20of%20Mycoplasma%20genitalium%20and%20Mycoplasma%20pneumoniae%20at%20single-base%20resolution&amp;author=M.%20Lluch-Senar&amp;author=K.%20Luong&amp;author=V.%20Llor%C3%A9ns-Rico&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003191" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1003191" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003191" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1003191"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300489" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comprehensive%20methylome%20characterization%20of%20Mycoplasma%20genitalium%20and%20Mycoplasma%20pneumoniae%20at%20single-base%20resolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lluch-Senar</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Delgado</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">W.-H.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Defining a minimal cell: essentiality of small ORFs and ncRNAs in a genome-reduced bacterium</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">11</div>, <div class="fpage">780</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Defining%20a%20minimal%20cell%3A%20essentiality%20of%20small%20ORFs%20and%20ncRNAs%20in%20a%20genome-reduced%20bacterium&amp;author=M.%20Lluch-Senar&amp;author=J.%20Delgado&amp;author=W.-H.%20Chen&amp;publication_year=2015&amp;journal=Mol.%20Syst.%20Biol&amp;volume=11&amp;pages=780" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.15252/msb.20145558" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.15252%2Fmsb.20145558" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.15252%2Fmsb.20145558"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25609650" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Defining%20a%20minimal%20cell%3A%20essentiality%20of%20small%20ORFs%20and%20ncRNAs%20in%20a%20genome-reduced%20bacterium&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Karr</div> <div class="given-names">J.R.</div>
</div>, <div class="name string-name">
<div class="surname">Sanghvi</div> <div class="given-names">J.C.</div>
</div>, <div class="name string-name">
<div class="surname">Macklin</div> <div class="given-names">D.N.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">A whole-cell computational model predicts phenotype from genotype</div>, <div class="source">Cell</div> , <div class="volume">150</div>, <div class="fpage">389</div>–<div class="lpage">401</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20whole-cell%20computational%20model%20predicts%20phenotype%20from%20genotype&amp;author=J.R.%20Karr&amp;author=J.C.%20Sanghvi&amp;author=D.N.%20Macklin&amp;publication_year=2012&amp;journal=Cell&amp;volume=150&amp;pages=389-401" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.cell.2012.05.044" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.05.044" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.cell.2012.05.044"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22817898" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20whole-cell%20computational%20model%20predicts%20phenotype%20from%20genotype&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Glass</div> <div class="given-names">J.I.</div>
</div>, <div class="name string-name">
<div class="surname">Assad-Garcia</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Alperovich</div> <div class="given-names">N.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">Essential genes of a minimal bacterium</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">425</div>–<div class="lpage">30</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Essential%20genes%20of%20a%20minimal%20bacterium&amp;author=J.I.%20Glass&amp;author=N.%20Assad-Garcia&amp;author=N.%20Alperovich&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=425-30" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0510013103" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0510013103" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0510013103"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Essential%20genes%20of%20a%20minimal%20bacterium&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">French</div> <div class="given-names">C.T.</div>
</div>, <div class="name string-name">
<div class="surname">Lao</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Loraine</div> <div class="given-names">A.E.</div>
</div>, <div class="name string-name">
<div class="surname">Matthews</div> <div class="given-names">B.T.</div>
</div>, <div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Dybvig</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Large-scale transposon mutagenesis of Mycoplasma pulmonis</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">69</div>, <div class="fpage">67</div>–<div class="lpage">76</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Large-scale%20transposon%20mutagenesis%20of%20Mycoplasma%20pulmonis&amp;author=C.T.%20French&amp;author=P.%20Lao&amp;author=A.E.%20Loraine&amp;author=B.T.%20Matthews&amp;author=H.%20Yu&amp;author=K.%20Dybvig&amp;publication_year=2008&amp;journal=Mol.%20Microbiol&amp;volume=69&amp;pages=67-76" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/mmi.2008.69.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2008.69.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmmi.2008.69.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18452587" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Large-scale%20transposon%20mutagenesis%20of%20Mycoplasma%20pulmonis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sharma</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Markham</div> <div class="given-names">P.F.</div>
</div>, <div class="name string-name">
<div class="surname">Browning</div> <div class="given-names">G.F.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Genes found essential in other mycoplasmas are dispensable in Mycoplasma bovis</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e97100</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genes%20found%20essential%20in%20other%20mycoplasmas%20are%20dispensable%20in%20Mycoplasma%20bovis&amp;author=S.%20Sharma&amp;author=P.F.%20Markham&amp;author=G.F.%20Browning&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e97100" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0097100" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0097100" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0097100"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24897538" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genes%20found%20essential%20in%20other%20mycoplasmas%20are%20dispensable%20in%20Mycoplasma%20bovis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hutchison</div> <div class="given-names">C.A.</div>3rd</div>, <div class="name string-name">
<div class="surname">Chuang</div> <div class="given-names">R.-Y.</div>
</div>, <div class="name string-name">
<div class="surname">Noskov</div> <div class="given-names">V.N.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Design and synthesis of a minimal bacterial genome</div>, <div class="source">Science</div> , <div class="volume">351</div>, <div class="fpage">aad6253</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Design%20and%20synthesis%20of%20a%20minimal%20bacterial%20genome&amp;author=C.A.%20Hutchison&amp;author=R.-Y.%20Chuang&amp;author=V.N.%20Noskov&amp;publication_year=2016&amp;journal=Science&amp;volume=351&amp;pages=aad6253" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.aad6253" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad6253" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.aad6253"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27013737" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Design%20and%20synthesis%20of%20a%20minimal%20bacterial%20genome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Waites</div> <div class="given-names">K.B.</div>
</div>, <div class="name string-name">
<div class="surname">Talkington</div> <div class="given-names">D.F.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Mycoplasma pneumoniae and its role as a human pathogen</div>, <div class="source">Clin. Microbiol. Rev</div> ., <div class="volume">17</div>, <div class="fpage">697</div>–<div class="lpage">728</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mycoplasma%20pneumoniae%20and%20its%20role%20as%20a%20human%20pathogen&amp;author=K.B.%20Waites&amp;author=D.F.%20Talkington&amp;publication_year=2004&amp;journal=Clin.%20Microbiol.%20Rev&amp;volume=17&amp;pages=697-728" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/CMR.17.4.697-728.2004" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FCMR.17.4.697-728.2004" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FCMR.17.4.697-728.2004"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489344" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mycoplasma%20pneumoniae%20and%20its%20role%20as%20a%20human%20pathogen&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Levisohn</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Kleven</div> <div class="given-names">S.H.</div>
</div>
</div> <div class="year">2000</div>, 
<div class="article-title">Avian mycoplasmosis (<em>Mycoplasma gallisepticum</em>)</div>, <div class="source">Rev. Sci. Technol</div> ., <div class="volume">19</div>, <div class="fpage">425</div>–<div class="lpage">42</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Avian%20mycoplasmosis%20(Mycoplasma%20gallisepticum)&amp;author=S.%20Levisohn&amp;author=S.H.%20Kleven&amp;publication_year=2000&amp;journal=Rev.%20Sci.%20Technol&amp;volume=19&amp;pages=425-42" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.20506/rst.issue.19.2.14" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.20506%2Frst.issue.19.2.14" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.20506%2Frst.issue.19.2.14"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Avian%20mycoplasmosis%20(Mycoplasma%20gallisepticum)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kumar</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rahal</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Chakraborty</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Verma</div> <div class="given-names">A.K.</div>
</div>, <div class="name string-name">
<div class="surname">Dhama</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Mycoplasma agalactiae, an etiological agent of contagious agalactia in small ruminants: a review</div>, <div class="source">Vet. Med. Int</div> ., <div class="volume">2014</div>, <div class="fpage">1</div>.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Mycoplasma%20agalactiae%2C%20an%20etiological%20agent%20of%20contagious%20agalactia%20in%20small%20ruminants%3A%20a%20review&amp;aulast=Kumar&amp;date=2014&amp;spage=1&amp;volume=2014" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mycoplasma%20agalactiae%2C%20an%20etiological%20agent%20of%20contagious%20agalactia%20in%20small%20ruminants%3A%20a%20review&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">De Jesus</div> <div class="given-names">M.A.</div>
</div>, <div class="name string-name">
<div class="surname">Gerrick</div> <div class="given-names">E.R.</div>
</div>, <div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">W.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Comprehensive essentiality analysis of the Mycobacterium tuberculosis genome via saturating transposon mutagenesis</div>, <div class="source">MBio</div> , <div class="volume">8</div>, e02133-16.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Comprehensive%20essentiality%20analysis%20of%20the%20Mycobacterium%20tuberculosis%20genome%20via%20saturating%20transposon%20mutagenesis&amp;aulast=De%20Jesus&amp;date=2017&amp;volume=8" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comprehensive%20essentiality%20analysis%20of%20the%20Mycobacterium%20tuberculosis%20genome%20via%20saturating%20transposon%20mutagenesis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Halbedel</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Busse</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Schmidl</div> <div class="given-names">S.R.</div>
</div>, <div class="name string-name">
<div class="surname">Stülke</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Regulatory protein phosphorylation in Mycoplasma pneumoniae. A PP2C-type phosphatase serves to dephosphorylate HPr (Ser-P)</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">281</div>, <div class="fpage">26253</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Regulatory%20protein%20phosphorylation%20in%20Mycoplasma%20pneumoniae.%20A%20PP2C-type%20phosphatase%20serves%20to%20dephosphorylate%20HPr%20(Ser-P)&amp;author=S.%20Halbedel&amp;author=J.%20Busse&amp;author=S.R.%20Schmidl&amp;author=J.%20St%C3%BClke&amp;publication_year=2006&amp;journal=J.%20Biol.%20Chem&amp;volume=281&amp;pages=26253-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1074/jbc.M605010200" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M605010200" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1074%2Fjbc.M605010200"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16857667" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Regulatory%20protein%20phosphorylation%20in%20Mycoplasma%20pneumoniae.%20A%20PP2C-type%20phosphatase%20serves%20to%20dephosphorylate%20HPr%20(Ser-P)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Halbedel</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Stülke</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Tools for the genetic analysis of Mycoplasma</div>, <div class="source">Int. J. Med. Microbiol</div> ., <div class="volume">297</div>, <div class="fpage">37</div>–<div class="lpage">44</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tools%20for%20the%20genetic%20analysis%20of%20Mycoplasma&amp;author=S.%20Halbedel&amp;author=J.%20St%C3%BClke&amp;publication_year=2007&amp;journal=Int.%20J.%20Med.%20Microbiol&amp;volume=297&amp;pages=37-44" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijmm.2006.11.001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijmm.2006.11.001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijmm.2006.11.001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17223385" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tools%20for%20the%20genetic%20analysis%20of%20Mycoplasma&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lyon</div> <div class="given-names">B.R.</div>
</div>, <div class="name string-name">
<div class="surname">May</div> <div class="given-names">J.W.</div>
</div>, <div class="name string-name">
<div class="surname">Skurray</div> <div class="given-names">R.A.</div>
</div>
</div> <div class="year">1984</div>, 
<div class="article-title">Tn4001: a gentamicin and kanamycin resistance transposon in <em>Staphylococcus aureus</em>
</div>, <div class="source">Mol. Gen. Genet</div> ., <div class="volume">193</div>, <div class="fpage">554</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tn4001%3A%20a%20gentamicin%20and%20kanamycin%20resistance%20transposon%20in%20Staphylococcus%20aureus&amp;author=B.R.%20Lyon&amp;author=J.W.%20May&amp;author=R.A.%20Skurray&amp;publication_year=1984&amp;journal=Mol.%20Gen.%20Genet&amp;volume=193&amp;pages=554-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF00382099" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00382099" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00382099"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6323927" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tn4001%3A%20a%20gentamicin%20and%20kanamycin%20resistance%20transposon%20in%20Staphylococcus%20aureus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dybvig</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">French</div> <div class="given-names">C.T.</div>
</div>, <div class="name string-name">
<div class="surname">Voelker</div> <div class="given-names">L.L.</div>
</div>
</div> <div class="year">2000</div>, 
<div class="article-title">Construction and use of derivatives of transposon Tn4001 that function in <em>Mycoplasma pulmonis</em> and <em>Mycoplasma arthritidis</em>
</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">182</div>, <div class="fpage">4343</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20use%20of%20derivatives%20of%20transposon%20Tn4001%20that%20function%20in%20Mycoplasma%20pulmonis%20and%20Mycoplasma%20arthritidis&amp;author=K.%20Dybvig&amp;author=C.T.%20French&amp;author=L.L.%20Voelker&amp;publication_year=2000&amp;journal=J.%20Bacteriol&amp;volume=182&amp;pages=4343-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/JB.182.15.4343-4347.2000" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.15.4343-4347.2000" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FJB.182.15.4343-4347.2000"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10894746" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Construction%20and%20use%20of%20derivatives%20of%20transposon%20Tn4001%20that%20function%20in%20Mycoplasma%20pulmonis%20and%20Mycoplasma%20arthritidis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pour-El</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Adams</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Minion</div> <div class="given-names">F.C.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">Construction of mini-Tn4001tet and its use in Mycoplasma gallisepticum</div>, <div class="source">Plasmid</div> ., <div class="volume">47</div>, <div class="fpage">129</div>–<div class="lpage">37</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20mini-Tn4001tet%20and%20its%20use%20in%20Mycoplasma%20gallisepticum&amp;author=I.%20Pour-El&amp;author=C.%20Adams&amp;author=F.C.%20Minion&amp;publication_year=2002&amp;journal=Plasmid&amp;volume=47&amp;pages=129-37" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1006/plas.2001.1558" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fplas.2001.1558" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1006%2Fplas.2001.1558"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11982334" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Construction%20of%20mini-Tn4001tet%20and%20its%20use%20in%20Mycoplasma%20gallisepticum&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hahn</div> <div class="given-names">T.W.</div>
</div>, <div class="name string-name">
<div class="surname">Mothershed</div> <div class="given-names">E.A.</div>
</div>, <div class="name string-name">
<div class="surname">Waldo</div> <div class="given-names">R.H.</div>3rd</div>, <div class="name string-name">
<div class="surname">Krause</div> <div class="given-names">D.C.</div>
</div>
</div> <div class="year">1999</div>, 
<div class="article-title">Construction and analysis of a modified Tn4001 conferring chloramphenicol resistance in Mycoplasma pneumoniae</div>, <div class="source">Plasmid</div> , <div class="volume">41</div>, <div class="fpage">120</div>–<div class="lpage">4</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20analysis%20of%20a%20modified%20Tn4001%20conferring%20chloramphenicol%20resistance%20in%20Mycoplasma%20pneumoniae&amp;author=T.W.%20Hahn&amp;author=E.A.%20Mothershed&amp;author=R.H.%20Waldo&amp;author=D.C.%20Krause&amp;publication_year=1999&amp;journal=Plasmid&amp;volume=41&amp;pages=120-4" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1006/plas.1998.1387" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fplas.1998.1387" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1006%2Fplas.1998.1387"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10087215" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Construction%20and%20analysis%20of%20a%20modified%20Tn4001%20conferring%20chloramphenicol%20resistance%20in%20Mycoplasma%20pneumoniae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Algire</div> <div class="given-names">M.A.</div>
</div>, <div class="name string-name">
<div class="surname">Lartigue</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Thomas</div> <div class="given-names">D.W.</div>
</div>, <div class="name string-name">
<div class="surname">Assad-Garcia</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Glass</div> <div class="given-names">J.I.</div>
</div>, <div class="name string-name">
<div class="surname">Merryman</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">New selectable marker for manipulating the simple genomes of Mycoplasma species</div>, <div class="source">Antimicrob. Agents Chemother</div> ., <div class="volume">53</div>, <div class="fpage">4429</div>–<div class="lpage">32</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=New%20selectable%20marker%20for%20manipulating%20the%20simple%20genomes%20of%20Mycoplasma%20species&amp;author=M.A.%20Algire&amp;author=C.%20Lartigue&amp;author=D.W.%20Thomas&amp;author=N.%20Assad-Garcia&amp;author=J.I.%20Glass&amp;author=C.%20Merryman&amp;publication_year=2009&amp;journal=Antimicrob.%20Agents%20Chemother&amp;volume=53&amp;pages=4429-32" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AAC.00388-09" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.00388-09" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAAC.00388-09"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19687239" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20selectable%20marker%20for%20manipulating%20the%20simple%20genomes%20of%20Mycoplasma%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pich</div> <div class="given-names">O.Q.</div>
</div>, <div class="name string-name">
<div class="surname">Burgos</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Planell</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Querol</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Piñol</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Comparative analysis of antibiotic resistance gene markers in Mycoplasma genitalium: application to studies of the minimal gene complement</div>, <div class="source">Microbiology</div> , <div class="volume">152</div>, <div class="fpage">519</div>–<div class="lpage">27</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20antibiotic%20resistance%20gene%20markers%20in%20Mycoplasma%20genitalium%3A%20application%20to%20studies%20of%20the%20minimal%20gene%20complement&amp;author=O.Q.%20Pich&amp;author=R.%20Burgos&amp;author=R.%20Planell&amp;author=E.%20Querol&amp;author=J.%20Pi%C3%B1ol&amp;publication_year=2006&amp;journal=Microbiology&amp;volume=152&amp;pages=519-27" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1099/mic.0.28287-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.28287-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1099%2Fmic.0.28287-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16436439" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparative%20analysis%20of%20antibiotic%20resistance%20gene%20markers%20in%20Mycoplasma%20genitalium%3A%20application%20to%20studies%20of%20the%20minimal%20gene%20complement&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chopra-Dewasthaly</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Zimmermann</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Rosengarten</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Citti</div> <div class="given-names">C.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">First steps towards the genetic manipulation of <em>Mycoplasma agalactiae</em> and <em>Mycoplasma bovis</em> using the transposon Tn4001mod</div>, <div class="source">Int. J. Med. Microbiol</div> ., <div class="volume">294</div>, <div class="fpage">447</div>–<div class="lpage">53</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=First%20steps%20towards%20the%20genetic%20manipulation%20of%20Mycoplasma%20agalactiae%20and%20Mycoplasma%20bovis%20using%20the%20transposon%20Tn4001mod&amp;author=R.%20Chopra-Dewasthaly&amp;author=M.%20Zimmermann&amp;author=R.%20Rosengarten&amp;author=C.%20Citti&amp;publication_year=2005&amp;journal=Int.%20J.%20Med.%20Microbiol&amp;volume=294&amp;pages=447-53" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ijmm.2004.09.010" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijmm.2004.09.010" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ijmm.2004.09.010"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15715173" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:First%20steps%20towards%20the%20genetic%20manipulation%20of%20Mycoplasma%20agalactiae%20and%20Mycoplasma%20bovis%20using%20the%20transposon%20Tn4001mod&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Beaman</div> <div class="given-names">K.D.</div>
</div>, <div class="name string-name">
<div class="surname">Pollack</div> <div class="given-names">J.D.</div>
</div>
</div> <div class="year">1983</div>, 
<div class="article-title">Synthesis of adenylate nucleotides by Mollicutes (mycoplasmas)</div>, <div class="source">J. Gen. Microbiol</div> ., <div class="volume">129</div>, <div class="fpage">3103</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Synthesis%20of%20adenylate%20nucleotides%20by%20Mollicutes%20(mycoplasmas)&amp;author=K.D.%20Beaman&amp;author=J.D.%20Pollack&amp;publication_year=1983&amp;journal=J.%20Gen.%20Microbiol&amp;volume=129&amp;pages=3103-10" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6655457" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Synthesis%20of%20adenylate%20nucleotides%20by%20Mollicutes%20(mycoplasmas)&amp;aulast=Beaman&amp;date=1983&amp;spage=3103&amp;epage=10&amp;volume=129" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Synthesis%20of%20adenylate%20nucleotides%20by%20Mollicutes%20(mycoplasmas)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jores</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Fischer</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Sirand-Pugnet</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Mycoplasma feriruminatoris sp. nov., a fast growing Mycoplasma species isolated from wild Caprinae</div>, <div class="source">Syst. Appl. Microbiol</div> ., <div class="volume">36</div>, <div class="fpage">533</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mycoplasma%20feriruminatoris%20sp.%20nov.%2C%20a%20fast%20growing%20Mycoplasma%20species%20isolated%20from%20wild%20Caprinae&amp;author=J.%20Jores&amp;author=A.%20Fischer&amp;author=P.%20Sirand-Pugnet&amp;publication_year=2013&amp;journal=Syst.%20Appl.%20Microbiol&amp;volume=36&amp;pages=533-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.syapm.2013.07.005" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.syapm.2013.07.005" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.syapm.2013.07.005"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24016869" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mycoplasma%20feriruminatoris%20sp.%20nov.%2C%20a%20fast%20growing%20Mycoplasma%20species%20isolated%20from%20wild%20Caprinae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gibson</div> <div class="given-names">D.G.</div>
</div>, <div class="name string-name">
<div class="surname">Young</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Chuang</div> <div class="given-names">R.-Y.</div>
</div>, <div class="name string-name">
<div class="surname">Venter</div> <div class="given-names">J.C.</div>
</div>, <div class="name string-name">
<div class="surname">Hutchison</div> <div class="given-names">C.A.</div>3rd</div>, <div class="name string-name">
<div class="surname">Smith</div> <div class="given-names">H.O.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Enzymatic assembly of DNA molecules up to several hundred kilobases</div>, <div class="source">Nat. Methods</div> , <div class="volume">6</div>, <div class="fpage">343</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Enzymatic%20assembly%20of%20DNA%20molecules%20up%20to%20several%20hundred%20kilobases&amp;author=D.G.%20Gibson&amp;author=L.%20Young&amp;author=R.-Y.%20Chuang&amp;author=J.C.%20Venter&amp;author=C.A.%20Hutchison&amp;author=H.O.%20Smith&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=343-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1318" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1318" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1318"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Enzymatic%20assembly%20of%20DNA%20molecules%20up%20to%20several%20hundred%20kilobases&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="year">Saitou, N. and Nei, M. 1987</div>, 
<div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>. <div class="source">Mol. Biol. Evol</div> ., 4, 406–425.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;date=Saitou, N. and Nei, M. 1987" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Felsenstein</div> <div class="given-names">J.</div>
</div></div> <div class="year">1985</div>, 
<div class="article-title">Confidence limits on phylogenies: an approach using the bootstrap</div>, <div class="source">Evolution</div> , <div class="volume">39</div>, <div class="fpage">783</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;author=J.%20Felsenstein&amp;publication_year=1985&amp;journal=Evolution&amp;volume=39&amp;pages=783" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1558-5646.1985.tb00420.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1558-5646.1985.tb00420.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1558-5646.1985.tb00420.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28561359" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Tamura</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Nei</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Kumar</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Prospects for inferring very large phylogenies by using the neighbor-joining method</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">101</div>, <div class="fpage">11030</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Prospects%20for%20inferring%20very%20large%20phylogenies%20by%20using%20the%20neighbor-joining%20method&amp;author=K.%20Tamura&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=11030-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0404206101" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404206101" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0404206101"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Prospects%20for%20inferring%20very%20large%20phylogenies%20by%20using%20the%20neighbor-joining%20method&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kumar</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Stecher</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Li</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Knyaz</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Tamura</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">MEGA X: molecular evolutionary genetics analysis across computing platforms</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">35</div>, <div class="fpage">1547</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MEGA%20X%3A%20molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms&amp;author=S.%20Kumar&amp;author=G.%20Stecher&amp;author=M.%20Li&amp;author=C.%20Knyaz&amp;author=K.%20Tamura&amp;publication_year=2018&amp;journal=Mol.%20Biol.%20Evol&amp;volume=35&amp;pages=1547-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msy096" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsy096" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsy096"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29722887" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MEGA%20X%3A%20molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Omotajo</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Tate</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Cho</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Choudhary</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Distribution and diversity of ribosome binding sites in prokaryotic genomes</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">604</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Distribution%20and%20diversity%20of%20ribosome%20binding%20sites%20in%20prokaryotic%20genomes&amp;author=D.%20Omotajo&amp;author=T.%20Tate&amp;author=H.%20Cho&amp;author=M.%20Choudhary&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=604" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-015-1808-6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1808-6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-015-1808-6"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26268350" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Distribution%20and%20diversity%20of%20ribosome%20binding%20sites%20in%20prokaryotic%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zimmerman</div> <div class="given-names">C.-U.</div>
</div>, <div class="name string-name">
<div class="surname">U. Zimmerman</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Herrmann</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Synthesis of a small, cysteine-rich, 29 amino acids long peptide in Mycoplasma pneumoniae</div>, <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">253</div>, <div class="fpage">315</div>–<div class="lpage">21</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Synthesis%20of%20a%20small%2C%20cysteine-rich%2C%2029%20amino%20acids%20long%20peptide%20in%20Mycoplasma%20pneumoniae&amp;author=C.-U.%20Zimmerman&amp;author=C.%20U.%20Zimmerman&amp;author=R.%20Herrmann&amp;publication_year=2005&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=253&amp;pages=315-21" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.femsle.2005.09.054" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.femsle.2005.09.054" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.femsle.2005.09.054"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16260096" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Synthesis%20of%20a%20small%2C%20cysteine-rich%2C%2029%20amino%20acids%20long%20peptide%20in%20Mycoplasma%20pneumoniae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bentley</div> <div class="given-names">D.R.</div>
</div>, <div class="name string-name">
<div class="surname">Balasubramanian</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Swerdlow</div> <div class="given-names">H.P.</div>
</div>
</div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Accurate whole human genome sequencing using reversible terminator chemistry</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">53</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accurate%20whole%20human%20genome%20sequencing%20using%20reversible%20terminator%20chemistry&amp;author=D.R.%20Bentley&amp;author=S.%20Balasubramanian&amp;author=H.P.%20Swerdlow&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=53-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature07517" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07517" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature07517"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18987734" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accurate%20whole%20human%20genome%20sequencing%20using%20reversible%20terminator%20chemistry&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jackman</div> <div class="given-names">S.D.</div>
</div>, <div class="name string-name">
<div class="surname">Vandervalk</div> <div class="given-names">B.P.</div>
</div>, <div class="name string-name">
<div class="surname">Mohamadi</div> <div class="given-names">H.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">ABySS 2.0: resource-efficient assembly of large genomes using a Bloom filter</div>, <div class="source">Genome Res</div> ., <div class="volume">27</div>, <div class="fpage">768</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ABySS%202.0%3A%20resource-efficient%20assembly%20of%20large%20genomes%20using%20a%20Bloom%20filter&amp;author=S.D.%20Jackman&amp;author=B.P.%20Vandervalk&amp;author=H.%20Mohamadi&amp;publication_year=2017&amp;journal=Genome%20Res&amp;volume=27&amp;pages=768-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.214346.116" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.214346.116" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.214346.116"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28232478" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ABySS%202.0%3A%20resource-efficient%20assembly%20of%20large%20genomes%20using%20a%20Bloom%20filter&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Luo</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Qian</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">FastUniq: a fast de novo duplicates removal tool for paired short reads</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e52249</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=FastUniq%3A%20a%20fast%20de%20novo%20duplicates%20removal%20tool%20for%20paired%20short%20reads&amp;author=H.%20Xu&amp;author=X.%20Luo&amp;author=J.%20Qian&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e52249" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0052249" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052249" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0052249"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23284954" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FastUniq%3A%20a%20fast%20de%20novo%20duplicates%20removal%20tool%20for%20paired%20short%20reads&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bolger</div> <div class="given-names">A.M.</div>
</div>, <div class="name string-name">
<div class="surname">Lohse</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Usadel</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu170"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Langmead</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S.L.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1923"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Handsaker</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Wysoker</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAM tools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAM%20tools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp352"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20sequence%20alignment%2Fmap%20format%20and%20SAM%20tools&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Huerta-Cepas</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Forslund</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Coelho</div> <div class="given-names">L.P.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">34</div>, <div class="fpage">2115</div>–<div class="lpage">22</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&amp;author=J.%20Huerta-Cepas&amp;author=K.%20Forslund&amp;author=L.P.%20Coelho&amp;publication_year=2017&amp;journal=Mol.%20Biol.%20Evol&amp;volume=34&amp;pages=2115-22" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msx148" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsx148" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsx148"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28460117" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Junier</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Unal</div> <div class="given-names">E.B.</div>
</div>, <div class="name string-name">
<div class="surname">Yus</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Lloréns-Rico</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Serrano</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2018</div>, 
<div class="article-title">Insights into the mechanisms of basal coordination of transcription using a genome-reduced bacterium</div>, <div class="source">Cell Syst</div> ., <div class="volume">7</div>, <div class="fpage">227</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20mechanisms%20of%20basal%20coordination%20of%20transcription%20using%20a%20genome-reduced%20bacterium&amp;author=I.%20Junier&amp;author=E.B.%20Unal&amp;author=E.%20Yus&amp;author=V.%20Llor%C3%A9ns-Rico&amp;author=L.%20Serrano&amp;publication_year=2018&amp;journal=Cell%20Syst&amp;volume=7&amp;pages=227-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.cels.2018.08.006" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cels.2018.08.006" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.cels.2018.08.006"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30138582" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insights%20into%20the%20mechanisms%20of%20basal%20coordination%20of%20transcription%20using%20a%20genome-reduced%20bacterium&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schrader</div> <div class="given-names">J.M.</div>
</div>, <div class="name string-name">
<div class="surname">Uhlenbeck</div> <div class="given-names">O.C.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Is the sequence-specific binding of aminoacyl-tRNAs by EF-Tu universal among bacteria?</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">9746</div>–<div class="lpage">58</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Is%20the%20sequence-specific%20binding%20of%20aminoacyl-tRNAs%20by%20EF-Tu%20universal%20among%20bacteria%3F&amp;author=J.M.%20Schrader&amp;author=O.C.%20Uhlenbeck&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=9746-58" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr641" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr641" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr641"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21893586" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Is%20the%20sequence-specific%20binding%20of%20aminoacyl-tRNAs%20by%20EF-Tu%20universal%20among%20bacteria%3F&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cammarano</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Tiboni</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Sanangelantoni</div> <div class="given-names">A.M.</div>
</div>
</div> <div class="year">1989</div>, 
<div class="article-title">Phylogenetic conservation of antigenic determinants in archaebacterial elongation factors (Tu proteins)</div>, <div class="source">Can. J. Microbiol</div> ., <div class="volume">35</div>, <div class="fpage">2</div>–<div class="lpage">10</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20conservation%20of%20antigenic%20determinants%20in%20archaebacterial%20elongation%20factors%20(Tu%20proteins)&amp;author=P.%20Cammarano&amp;author=O.%20Tiboni&amp;author=A.M.%20Sanangelantoni&amp;publication_year=1989&amp;journal=Can.%20J.%20Microbiol&amp;volume=35&amp;pages=2-10" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1139/m89-002" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fm89-002" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1139%2Fm89-002"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2470483" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenetic%20conservation%20of%20antigenic%20determinants%20in%20archaebacterial%20elongation%20factors%20(Tu%20proteins)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yus</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Yang</div> <div class="given-names">J.-S.</div>
</div>, <div class="name string-name">
<div class="surname">Sogues</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Serrano</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">A reporter system coupled with high-throughput sequencing unveils key bacterial transcription and translation determinants</div>, <div class="source">Nat. Commun</div> ., <div class="volume">8</div>, 368.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20reporter%20system%20coupled%20with%20high-throughput%20sequencing%20unveils%20key%20bacterial%20transcription%20and%20translation%20determinants&amp;aulast=Yus&amp;date=2017&amp;volume=8" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20reporter%20system%20coupled%20with%20high-throughput%20sequencing%20unveils%20key%20bacterial%20transcription%20and%20translation%20determinants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lloréns-Rico</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Lluch-Senar</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Serrano</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Distinguishing between productive and abortive promoters using a random forest classifier in Mycoplasma pneumoniae</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">3442</div>–<div class="lpage">53</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Distinguishing%20between%20productive%20and%20abortive%20promoters%20using%20a%20random%20forest%20classifier%20in%20Mycoplasma%20pneumoniae&amp;author=V.%20Llor%C3%A9ns-Rico&amp;author=M.%20Lluch-Senar&amp;author=L.%20Serrano&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=3442-53" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkv170" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv170" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkv170"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25779052" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Distinguishing%20between%20productive%20and%20abortive%20promoters%20using%20a%20random%20forest%20classifier%20in%20Mycoplasma%20pneumoniae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mariscal</div> <div class="given-names">A.M.</div>
</div>, <div class="name string-name">
<div class="surname">González-González</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Querol</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Piñol</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">All-in-one construct for genome engineering using Cre-lox technology</div>, <div class="source">DNA Res</div> ., <div class="volume">23</div>, <div class="fpage">263</div>–<div class="lpage">70</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=All-in-one%20construct%20for%20genome%20engineering%20using%20Cre-lox%20technology&amp;author=A.M.%20Mariscal&amp;author=L.%20Gonz%C3%A1lez-Gonz%C3%A1lez&amp;author=E.%20Querol&amp;author=J.%20Pi%C3%B1ol&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=263-70" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/dnares/dsw015" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw015" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdnares%2Fdsw015"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27084897" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:All-in-one%20construct%20for%20genome%20engineering%20using%20Cre-lox%20technology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Dybvig</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Lao</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Jordan</div> <div class="given-names">D.S.</div>
</div>, <div class="name string-name">
<div class="surname">Simmons</div> <div class="given-names">W.L.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">Fewer essential genes in mycoplasmas than previous studies suggest</div>, <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">311</div>, <div class="fpage">51</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Fewer%20essential%20genes%20in%20mycoplasmas%20than%20previous%20studies%20suggest&amp;author=K.%20Dybvig&amp;author=P.%20Lao&amp;author=D.S.%20Jordan&amp;author=W.L.%20Simmons&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=311&amp;pages=51-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/fml.2010.311.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2010.311.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffml.2010.311.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20722737" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Fewer%20essential%20genes%20in%20mycoplasmas%20than%20previous%20studies%20suggest&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Baby</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Lachance</div> <div class="given-names">J.-C.</div>
</div>, <div class="name string-name">
<div class="surname">Gagnon</div> <div class="given-names">J.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Inferring the minimal genome of Mesoplasma florum by comparative genomics and transposon mutagenesis</div>, <div class="source">mSystems</div> , <div class="volume">3</div>, e00198–17.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Inferring%20the%20minimal%20genome%20of%20Mesoplasma%20florum%20by%20comparative%20genomics%20and%20transposon%20mutagenesis&amp;aulast=Baby&amp;date=2018&amp;volume=3" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inferring%20the%20minimal%20genome%20of%20Mesoplasma%20florum%20by%20comparative%20genomics%20and%20transposon%20mutagenesis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Xiao</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">L.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">FSPP: a Tool for genome-wide prediction of smORF-encoded peptides and their functions</div>, <div class="source">Front. Genet</div> ., <div class="volume">9</div>, <div class="fpage">96</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=FSPP%3A%20a%20Tool%20for%20genome-wide%20prediction%20of%20smORF-encoded%20peptides%20and%20their%20functions&amp;author=H.%20Li&amp;author=L.%20Xiao&amp;author=L.%20Zhang&amp;publication_year=2018&amp;journal=Front.%20Genet&amp;volume=9&amp;pages=96" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fgene.2018.00096" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2018.00096" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffgene.2018.00096"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29675032" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FSPP%3A%20a%20Tool%20for%20genome-wide%20prediction%20of%20smORF-encoded%20peptides%20and%20their%20functions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Miravet-Verde</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Ferrar</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Espadas-García</div> <div class="given-names">G.</div>
</div>
</div>, et al.  <div class="year">2019</div>, 
<div class="article-title">Unraveling the hidden universe of small proteins in bacterial genomes</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">15</div>, <div class="fpage">e8290</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Unraveling%20the%20hidden%20universe%20of%20small%20proteins%20in%20bacterial%20genomes&amp;author=S.%20Miravet-Verde&amp;author=T.%20Ferrar&amp;author=G.%20Espadas-Garc%C3%ADa&amp;publication_year=2019&amp;journal=Mol.%20Syst.%20Biol&amp;volume=15&amp;pages=e8290" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.15252/msb.20188290" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.15252%2Fmsb.20188290" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.15252%2Fmsb.20188290"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30796087" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Unraveling%20the%20hidden%20universe%20of%20small%20proteins%20in%20bacterial%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goryshin</div> <div class="given-names">I.Y.</div>
</div>, <div class="name string-name">
<div class="surname">Reznikoff</div> <div class="given-names">W.S.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Tn5 in vitro transposition</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">273</div>, <div class="fpage">7367</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tn5%20in%20vitro%20transposition&amp;author=I.Y.%20Goryshin&amp;author=W.S.%20Reznikoff&amp;publication_year=1998&amp;journal=J.%20Biol.%20Chem&amp;volume=273&amp;pages=7367-74" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1074/jbc.273.13.7367" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.273.13.7367" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1074%2Fjbc.273.13.7367"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9516433" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tn5%20in%20vitro%20transposition&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsz012-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsz012-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Price</div> <div class="given-names">M.N.</div>
</div>, <div class="name string-name">
<div class="surname">Waters</div> <div class="given-names">R.J.</div>
</div>
</div>, et al.  <div class="year">2018</div>, 
<div class="article-title">Magic pools: parallel assessment of transposon delivery vectors in bacteria</div>, <div class="source">mSystems</div> , <div class="volume">3</div>, e00143–17.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Magic%20pools%3A%20parallel%20assessment%20of%20transposon%20delivery%20vectors%20in%20bacteria&amp;aulast=Liu&amp;date=2018&amp;volume=3" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Magic%20pools%3A%20parallel%20assessment%20of%20transposon%20delivery%20vectors%20in%20bacteria&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5525352&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/4/10.1093_dnares_dsz012/2/dsz012_supplementary_data.zip?Expires=1576706136&amp;Signature=0XmCebAiJjEVSt9fDdQTcd-OxqewgsC076nUuSCBijgDMhQCnlT~iW583x6LU0OtcNuSXJWsYm09wivGG2QEZ1qJxgnaUg2pmNlDdMhKuvQJn6h40mLpJuy4IclQXlbuKTgkF7yORcOeBIC8TIyhhufILEhUSjbcmHzKAbxmx3cKY9PC8sGmysV2txU7WGkPX3GtYcddCLf3-0gkXABwQHH5wQ4mQlo7Yes6RdzvHJQYJ2jHoXfbhRigarMsC3hq21F~jUJKYzjfdazpdmRcWetk6PI33ymJamH30~HGbcKRiEzxVXT60vvvdf~EBS~Bks-c~ZC9gYj7PK9sWNepWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">dsz012_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">451</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsz012"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5525352"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f4%2f327%2f5525352">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F4%2F327%2F5525352" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5525352">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="SynMyco transposon: engineering transposon vectors for efficient transformation of minimal genomes | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsz012");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f4%2f327%2f5525352">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
