<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>unique life-strategy of an endophytic yeast Rhodotorula mucilaginosa JGTA-S1—a comparative genomics viewpoint | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/2/131/5277053","@type":"ScholarlyArticle","name":"A unique life-strategy of an endophytic yeast  Rhodotorula mucilaginosa  JGTA-S1 — a comparative genomics viewpoint","datePublished":"2019-01-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2019-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy044","keywords":["genome assembly","oxford nanopore","endophyte","<italic>Rhodotorula</italic>","endosymbiont"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/2/131/5277053","author":[{"name":"Sen, Diya","affiliation":"Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India","@type":"Person"},{"name":"Paul, Karnelia","affiliation":"Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India","@type":"Person"},{"name":"Saha, Chinmay","affiliation":"Department of Endocrinology &amp; Metabolism, Institute of Post Graduate Medical Education &amp; Research and SSKM Hospital, Kolkata, West Bengal, India","@type":"Person"},{"name":"Mukherjee, Gairik","affiliation":"Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India","@type":"Person"},{"name":"Nag, Mayurakshi","affiliation":"Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India","@type":"Person"},{"name":"Ghosh, Samrat","affiliation":"Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India  Academy of Scientific and Innovative Research, New Delhi, India","@type":"Person"},{"name":"Das, Abhishek","affiliation":"Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India  Academy of Scientific and Innovative Research, New Delhi, India","@type":"Person"},{"name":"Seal, Anindita","affiliation":"Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India","@type":"Person"},{"name":"Tripathy, Sucheta","affiliation":"Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India  Academy of Scientific and Innovative Research, New Delhi, India","@type":"Person"}],"description":"Abstract.  Endophytic yeasts of genus Rhodotorula are gaining importance for their ability to improve plant growth. The nature of their interaction with plants,","pageStart":"131","pageEnd":"146","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636231856&Signature=cT51LD-zDpvoKFIsKhaU9pfGQBHyP1Qi-yhG7EZu0mAZ8yGMmxGxvI2FRi0KyrfKUI2s34kobpfywucds2wYeFotbBS~yGGRlsQ5~dI482re6xAGE3~OdDpKO9m7E1eRflwb1Z7jH4e9uSObgpQaAlvZv9GD3TVMV5Ym-IWhG7UFldpqG-qm8ali-xV6c6p4dDSuW~yzCPjjaGYyJvKC1~egyVSi5Pzuk5gnbQG7PcV2017obNpxJDNWehwtIo3y0gvM05pMLMNVWa8~tRJ9Pz-eSjnEbOsiI5ok6W3Jcv2S1SrrX290hg-BHMTf73URjs60xVqCPl5HTTAL4rkdRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A unique life-strategy of an endophytic yeast  Rhodotorula mucilaginosa  JGTA-S1 — a comparative genomics viewpoint","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636231856&Signature=cT51LD-zDpvoKFIsKhaU9pfGQBHyP1Qi-yhG7EZu0mAZ8yGMmxGxvI2FRi0KyrfKUI2s34kobpfywucds2wYeFotbBS~yGGRlsQ5~dI482re6xAGE3~OdDpKO9m7E1eRflwb1Z7jH4e9uSObgpQaAlvZv9GD3TVMV5Ym-IWhG7UFldpqG-qm8ali-xV6c6p4dDSuW~yzCPjjaGYyJvKC1~egyVSi5Pzuk5gnbQG7PcV2017obNpxJDNWehwtIo3y0gvM05pMLMNVWa8~tRJ9Pz-eSjnEbOsiI5ok6W3Jcv2S1SrrX290hg-BHMTf73URjs60xVqCPl5HTTAL4rkdRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="unique life-strategy of an endophytic yeast  Rhodotorula mucilaginosa  JGTA-S1 — a comparative genomics viewpoint">
<meta property="og:description" content="Abstract.  Endophytic yeasts of genus Rhodotorula are gaining importance for their ability to improve plant growth. The nature of their interaction with plants,">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/2/131/5277053">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636231856&amp;Signature=cT51LD-zDpvoKFIsKhaU9pfGQBHyP1Qi-yhG7EZu0mAZ8yGMmxGxvI2FRi0KyrfKUI2s34kobpfywucds2wYeFotbBS~yGGRlsQ5~dI482re6xAGE3~OdDpKO9m7E1eRflwb1Z7jH4e9uSObgpQaAlvZv9GD3TVMV5Ym-IWhG7UFldpqG-qm8ali-xV6c6p4dDSuW~yzCPjjaGYyJvKC1~egyVSi5Pzuk5gnbQG7PcV2017obNpxJDNWehwtIo3y0gvM05pMLMNVWa8~tRJ9Pz-eSjnEbOsiI5ok6W3Jcv2S1SrrX290hg-BHMTf73URjs60xVqCPl5HTTAL4rkdRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-01-07">
<meta name="citation_author" content="Sen, Diya">
<meta name="citation_author_institution" content="Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India">
<meta name="citation_author" content="Paul, Karnelia">
<meta name="citation_author_institution" content="Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India">
<meta name="citation_author" content="Saha, Chinmay">
<meta name="citation_author_institution" content="Department of Endocrinology &amp; Metabolism, Institute of Post Graduate Medical Education &amp; Research and SSKM Hospital, Kolkata, West Bengal, India">
<meta name="citation_author" content="Mukherjee, Gairik">
<meta name="citation_author_institution" content="Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India">
<meta name="citation_author" content="Nag, Mayurakshi">
<meta name="citation_author_institution" content="Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India">
<meta name="citation_author" content="Ghosh, Samrat">
<meta name="citation_author_institution" content="Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India">
<meta name="citation_author_institution" content="Academy of Scientific and Innovative Research, New Delhi, India">
<meta name="citation_author" content="Das, Abhishek">
<meta name="citation_author_institution" content="Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India">
<meta name="citation_author_institution" content="Academy of Scientific and Innovative Research, New Delhi, India">
<meta name="citation_author" content="Seal, Anindita">
<meta name="citation_author_institution" content="Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India">
<meta name="citation_author" content="Tripathy, Sucheta">
<meta name="citation_author_institution" content="Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India">
<meta name="citation_author_institution" content="Academy of Scientific and Innovative Research, New Delhi, India">
<meta name="citation_title" content="A unique life-strategy of an endophytic yeast Rhodotorula mucilaginosa JGTA-S1—a comparative genomics viewpoint">
<meta name="citation_firstpage" content="131">
<meta name="citation_lastpage" content="146">
<meta name="citation_doi" content="10.1093/dnares/dsy044">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="2">
<meta name="citation_publication_date" content="2019/04/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=Rhodotorula psychrophila sp. nov., Rhodotorula psychrophenolica sp. nov. and Rhodotorula glacialis sp. nov., novel psychrophilic basidiomycetous yeast species isolated from alpine environments;  citation_journal_title=Int. J. Syst. Evol. Microbiol;  citation_year=2007;  citation_volume=57;  citation_pages=2179-84; ">
<meta name="citation_reference" content="citation_publisher=Springer Berlin Heidelberg, Berlin, Heidelberg;  citation_title=Cold-adapted Yeasts: Biodiversity, Adaptation Strategies and Biotechnological Significance;  citation_year=2014;  citation_pages=49-74; ">
<meta name="citation_reference" content="citation_title=Epidemiology of Rhodotorula: an emerging pathogen;  citation_journal_title=Interdiscip. Perspect. Infect. Dis;  citation_year=2012;  citation_volume=2012;  citation_pages=1">
<meta name="citation_reference" content="citation_title=Characterization of three endophytic, indole-3-acetic acid-producing yeasts occurring in Populus trees;  citation_journal_title=Mycol. Res;  citation_year=2009;  citation_volume=113;  citation_pages=973-80; ">
<meta name="citation_reference" content="citation_title=Biological nitrogen fixation and biomass accumulation within poplar clones as a result of inoculations with diazotrophic endophyte consortia;  citation_journal_title=New Phytol;  citation_year=2014;  citation_volume=201;  citation_pages=599-609; ">
<meta name="citation_reference" content="citation_title=Endophytic yeast Rhodotorula rubra strain TG-1: antagonistic and plant protection activities;  citation_journal_title=Biochem. Physiol;  citation_year=2013;  citation_volume=2; ">
<meta name="citation_reference" content="citation_title=Taxonomic studies in the Microbotryomycetidae: leucosporidium golubevii sp. nov., Leucosporidiella gen. nov. and the new orders Leucosporidiales and Sporidiobolales;  citation_journal_title=Mycol. Prog;  citation_year=2003;  citation_volume=2;  citation_pages=53-68; ">
<meta name="citation_reference" content="citation_publisher=Springer Berlin Heidelberg, Berlin, Heidelberg;  citation_title=Systematics and Evolution;  citation_year=2001;  citation_pages=3-35; ">
<meta name="citation_reference" content="citation_title=Sexual reproduction and dimorphism in the pathogenic basidiomycetes;  citation_journal_title=FEMS Yeast Res;  citation_year=2009;  citation_volume=9;  citation_pages=161-77; ">
<meta name="citation_reference" content="citation_title=A multi-omic map of the lipid-producing yeast Rhodosporidium toruloides;  citation_journal_title=Nat. Commun;  citation_year=2012;  citation_volume=3;  citation_pages=1112">
<meta name="citation_reference" content="citation_title=Genome sequence of the plant growth promoting endophytic yeast Rhodotorula graminis WP1;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=978">
<meta name="citation_reference" content="citation_title=Improved high-quality draft genome sequence of the eurypsychrophile Rhodotorula sp. JG1b, isolated from permafrost in the hyperarid upper-elevation mcmurdo dry valleys, Antarctica;  citation_journal_title=Genome Announce;  citation_year=2016;  citation_volume=4; ">
<meta name="citation_reference" content="citation_title=Draft genome sequence of Rhodotorula mucilaginosa, an emergent opportunistic pathogen;  citation_journal_title=Genome Announce;  citation_year=2015;  citation_volume=3; ">
<meta name="citation_reference" content="citation_title=Early changes in shoot transcriptome of rice in response to Rhodotorula mucilaginosa JGTA-S1;  citation_journal_title=Genome Data;  citation_year=2015;  citation_volume=6;  citation_pages=237-40; ">
<meta name="citation_reference" content="citation_title=Poretools: a toolkit for analyzing nanopore sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3399-401; ">
<meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; ">
<meta name="citation_reference" content="citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation;  citation_journal_title=Genome Res;  citation_year=2017;  citation_volume=27;  citation_pages=722-36; ">
<meta name="citation_reference" content="citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing;  citation_journal_title=J. Comput. Biol;  citation_year=2012;  citation_volume=19;  citation_pages=455-77; ">
<meta name="citation_reference" content="citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e112963">
<meta name="citation_reference" content="citation_title=QUAST: quality assessment tool for genome assemblies;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=1072-5; ">
<meta name="citation_reference" content="citation_title=Using MUMmer to identify similar regions in large sequence sets;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2003; ">
<meta name="citation_reference" content="citation_title=AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W465-7; ">
<meta name="citation_reference" content="citation_title=Blast2GO: a comprehensive suite for functional analysis in plant genomics;  citation_journal_title=Int. J. Plant Genomics;  citation_year=2008;  citation_volume=2008;  citation_pages=1">
<meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; ">
<meta name="citation_reference" content="Gregory R. W., Bolker B., Bonebakker L. , et al. 2016 , gplots: Various R Programming Tools for Plotting Data. R package version 3.0.1. Available at: https://CRAN.R-project.org/package=gplots (accessed June 2018). ">
<meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; ">
<meta name="citation_reference" content="citation_title=clusterProfiler: an R package for comparing biological themes among gene clusters;  citation_journal_title=OMICS;  citation_year=2012;  citation_volume=16;  citation_pages=284-7; ">
<meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49">
<meta name="citation_reference" content="citation_title=DAGchainer: a tool for mining segmental genome duplications and synteny;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=3643-6; ">
<meta name="citation_reference" content="citation_title=Multiple sequence alignment using ClustalW and ClustalX;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2002; ">
<meta name="citation_reference" content="citation_title=ProML–the protein markup language for specification of protein sequences, structures and families;  citation_journal_title=In Silico Biol;  citation_year=2002;  citation_volume=2;  citation_pages=313-24; ">
<meta name="citation_reference" content="citation_title=Improved prediction of signal peptides: signalP 3.0;  citation_journal_title=J. Mol. Biol;  citation_year=2004;  citation_volume=340;  citation_pages=783-95; ">
<meta name="citation_reference" content="citation_title=Phosphate availability alters architecture and causes changes in hormone sensitivity in the Arabidopsis root system;  citation_journal_title=Plant Physiol;  citation_year=2002;  citation_volume=129;  citation_pages=244-56; ">
<meta name="citation_reference" content="citation_title=Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction;  citation_journal_title=BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=134">
<meta name="citation_reference" content="citation_title=Genome sequencing provides insight into the reproductive biology, nutritional mode and ploidy of the fern pathogen Mixia osmundae;  citation_journal_title=New Phytol;  citation_year=2014;  citation_volume=202;  citation_pages=554-64; ">
<meta name="citation_reference" content="citation_title=Nearing the cold-arid limits of microbial life in permafrost of an upper dry valley, Antarctica;  citation_journal_title=Isme J;  citation_year=2016;  citation_volume=10;  citation_pages=1613">
<meta name="citation_reference" content="Banno I.  1967 ,  Studies on the sexuality of Rhodotorula. The Journal of General and Applied Microbiology , 13 , 167 – 196. ">
<meta name="citation_reference" content="citation_title=LTR Retrotransposons in Fungi;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e29425">
<meta name="citation_reference" content="citation_title=An endophytic Basidiomycete, Grammothele lineata, isolated from Corchorus olitorius, produces paclitaxel that shows cytotoxicity;  citation_journal_title=PLoS One;  citation_year=2017;  citation_volume=12;  citation_pages=e0178612">
<meta name="citation_reference" content="citation_title=Piriformospora indica, gen. et sp. nov., a new root-colonizing fungus;  citation_journal_title=Mycologia;  citation_year=1998;  citation_volume=90;  citation_pages=896-903; ">
<meta name="citation_reference" content="citation_title=Genome sequence of the plant growth promoting endophytic yeast Rhodotorula graminis WP1;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6; ">
<meta name="citation_reference" content="citation_title=Insights from the genome of the biotrophic fungal plant pathogen Ustilago maydis;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=97-101; ">
<meta name="citation_reference" content="citation_title=Next Generation sequencing provides rapid access to the genome of Puccinia striiformis f. sp. tritici, the causal agent of wheat stripe rust;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e24230">
<meta name="citation_reference" content="citation_title=Patterns of genomic variation in the poplar rust fungus Melampsora larici-populina identify pathogenesis-related factors;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5; ">
<meta name="citation_reference" content="citation_title=Sex and parasites: genomic and transcriptomic analysis of Microbotryum lychnidis-dioicae, the biotrophic and plant-castrating anther smut fungus;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=461">
<meta name="citation_reference" content="citation_title=Characterization of the Aspergillus nidulans nmrA gene involved in nitrogen metabolite repression;  citation_journal_title=J. Bacteriol;  citation_year=1998;  citation_volume=180;  citation_pages=1973-7; ">
<meta name="citation_reference" content="citation_title=Characterization of the largest effector gene cluster of Ustilago maydis;  citation_journal_title=PLoS Pathog;  citation_year=2014;  citation_volume=10;  citation_pages=e1003866">
<meta name="citation_reference" content="citation_title=Proteophosphoglycan, a major secreted product of intracellular leishmania mexicana amastigotes, is a poor b-cell antigen and does not elicit a specific conventional CD4(+) T-cell response;  citation_journal_title=Infect. Immun;  citation_year=1999;  citation_volume=67;  citation_pages=5379-85; ">
<meta name="citation_reference" content="citation_title=Role of Glycopeptides and peptides in inhibition of crystallization of water in polar fishes [and discussion];  citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci;  citation_year=1984;  citation_volume=304;  citation_pages=575-88; ">
<meta name="citation_reference" content="citation_title=The unique architecture and function of cellulose-interacting proteins in oomycetes revealed by genomic and structural analyses;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=605">
<meta name="citation_reference" content="citation_title=The root endophytic fungus Piriformospora indica requires host cell death for proliferation during mutualistic symbiosis with barley;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=18450-7; ">
<meta name="citation_reference" content="citation_title=Diversity of cell wall related proteins in human pathogenic fungi;  citation_journal_title=J. Fungi (Basel;  citation_year=2017;  citation_volume=4; ">
<meta name="citation_reference" content="citation_title=Action of an endo-β-1, 3(4)-glucanase on cellobiosyl unit structure in barley β-1, 3: 1, 4-glucan;  citation_journal_title=Biosci. Biotechnol. Biochem;  citation_year=2015;  citation_volume=79;  citation_pages=1810-7; ">
<meta name="citation_reference" content="citation_title=A chitin deacetylase from the endophytic fungus Pestalotiopsis sp. efficiently inactivates the elicitor activity of chitin oligomers in rice cells;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=38018">
<meta name="citation_reference" content="citation_title=LysM effectors: secreted proteins supporting fungal life;  citation_journal_title=PLOS Pathog;  citation_year=2013;  citation_volume=9;  citation_pages=e1003769">
<meta name="citation_reference" content="citation_title=Dissection of salicylic acid-mediated defense signaling networks;  citation_journal_title=Plant Signal. Behav;  citation_year=2009;  citation_volume=4;  citation_pages=713-7; ">
<meta name="citation_reference" content="citation_title=Degradation of the plant defence hormone salicylic acid by the biotrophic fungus Ustilago maydis;  citation_journal_title=Mol. Microbiol;  citation_year=2013;  citation_volume=89;  citation_pages=179-88; ">
<meta name="citation_reference" content="citation_title=Antagonistic effect of salicylic acid and jasmonic acid on the expression of pathogenesis-related (PR) protein genes in wounded mature tobacco leaves;  citation_journal_title=Plant Cell Physiol;  citation_year=1998;  citation_volume=39;  citation_pages=500-7; ">
<meta name="citation_reference" content="citation_title=Transcriptomic analysis of paulownia infected by Paulownia Witches’-broom phytoplasma;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e77217">
<meta name="citation_reference" content="citation_title=Plant growth promotion induced by phosphate solubilizing endophytic Pseudomonas isolates;  citation_journal_title=Front. Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=745">
<meta name="citation_reference" content="citation_title=Effector candidates in the secretome of Piriformospora indica, a ubiquitous plant-associated fungus;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=4; ">
<meta name="citation_reference" content="citation_title=Structure analysis uncovers a highly diverse but structurally conserved effector family in phytopathogenic fungi;  citation_journal_title=PLoS Pathog;  citation_year=2015;  citation_volume=11;  citation_pages=e1005228">
<meta name="citation_reference" content="citation_title=A Deviation from the bipolar-tetrapolar mating paradigm in an early diverged basidiomycete;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001052">
<meta name="citation_reference" content="citation_title=Obligate biotrophy features unraveled by the genomic analysis of rust fungi;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=9166">
<meta name="citation_reference" content="citation_title=Physical and genetic interaction between ammonium transporters and the signaling protein rho1 in the plant pathogen Ustilago maydis;  citation_journal_title=Eukaryotic Cell;  citation_year=2014;  citation_volume=13;  citation_pages=1328-36; ">
<meta name="citation_reference" content="citation_title=A novel intracellular nitrogen-fixing symbiosis made by Ustilago maydis and Bacillus spp;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=207;  citation_pages=769-77; ">
<meta name="citation_reference" content="citation_title=Plants, mycorrhizal fungi, and bacteria: a network of interactions;  citation_journal_title=Annu. Rev. Microbiol;  citation_year=2009;  citation_volume=63;  citation_pages=363-83; ">
<meta name="citation_reference" content="citation_title=Mosaic genome of endobacteria in arbuscular mycorrhizal fungi: transkingdom gene transfer in an ancient mycoplasma-fungus association;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=7785">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/2/131/28495630/dsy044.pdf">
<meta name="description" content="Abstract.  Endophytic yeasts of genus Rhodotorula are gaining importance for their ability to improve plant growth. The nature of their interaction with plants,">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/2/131/5277053">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article-abstract/26/2/131/5277053">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/2/131/5277053");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy044';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f131%2f5277053" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f131%2f5277053" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115366"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1576706086&amp;Signature=uRFPhQhoz14xF2LQW1K85gbxEc~e87vaRRUpr06lCH7LKFKTxk7q2JP~3qkrtlGe4FO5hnA-CaTVaZxNBnUm8ILt8d71NrOSMA1lpWgLWGtpfatmMTaOuKJDTkorHnaIrZpIKU7zoxsLE5HpI7lwbwAVatuz~~6Iq5pQZQcY1uc9e5aFXiqWfrcW37iO0Iz-8J~lrpL9jV5i7GEbrV~kE0VlMvT35Uk7BxSe~JYvPUTK27pGPi5RfvJmwFJLjIZcwo8pYt41bX0NmccTBFUhD0osHIo6H4UpllfoJBRsn0qblJEvUs06-q0PaHAdajnMs-C9Rzc1H9mNAxG90VX4Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/2">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="134010249">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010249">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010251">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010251">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010255">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010255">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010288">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010288">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010329">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010329">4. Conclusion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134010331">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010331">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="134010337">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134010337">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/2/119/5259342">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/2/147/5280828">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    A unique life-strategy of an endophytic yeast <em>Rhodotorula mucilaginosa</em> JGTA-S1<em>—</em>a comparative genomics viewpoint
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Diya Sen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Diya+Sen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sen%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sen%20Diya%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Diya Sen</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karnelia Paul</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karnelia+Paul">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paul%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Paul%20Karnelia%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Karnelia Paul</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chinmay Saha</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Endocrinology &amp; Metabolism, Institute of Post Graduate Medical Education &amp; Research and SSKM Hospital, Kolkata, West Bengal, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chinmay+Saha">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saha%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Saha%20Chinmay%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Chinmay Saha</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gairik Mukherjee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gairik+Mukherjee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mukherjee%20G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mukherjee%20Gairik%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Gairik Mukherjee</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mayurakshi Nag</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mayurakshi+Nag">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nag%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nag%20Mayurakshi%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Mayurakshi Nag</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Samrat Ghosh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India</div>
<div class="aff">Academy of Scientific and Innovative Research, New Delhi, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Samrat+Ghosh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ghosh%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ghosh%20Samrat%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Samrat Ghosh</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Abhishek Das</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India</div>
<div class="aff">Academy of Scientific and Innovative Research, New Delhi, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Abhishek+Das">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Das%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Das%20Abhishek%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Abhishek Das</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anindita Seal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology, Dr. B. C. Guha Centre for Genetic Engineering and Biotechnology, University of Calcutta, Kolkata, India</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +91 33 2461 4959. Email: <a href="mailto:aninditaseal@gmail.com" target="_blank">aninditaseal@gmail.com</a>/<a href="mailto:asbcg@caluniv.ac.in" target="_blank">asbcg@caluniv.ac.in</a> (A.S.); Tel. +91 33 2499 5894. Email: <a href="mailto:tsucheta@gmail.com" target="_blank">tsucheta@gmail.com</a>/<a href="mailto:tsucheta@iicb.res.in" target="_blank">tsucheta@iicb.res.in</a> (S.T.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anindita+Seal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seal%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Seal%20Anindita%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Anindita Seal</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sucheta Tripathy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Structural Biology and Bioinformatics Division, CSIR Indian Institute of Chemical Biology, Kolkata, India</div>
<div class="aff">Academy of Scientific and Innovative Research, New Delhi, India</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +91 33 2461 4959. Email: <a href="mailto:aninditaseal@gmail.com" target="_blank">aninditaseal@gmail.com</a>/<a href="mailto:asbcg@caluniv.ac.in" target="_blank">asbcg@caluniv.ac.in</a> (A.S.); Tel. +91 33 2499 5894. Email: <a href="mailto:tsucheta@gmail.com" target="_blank">tsucheta@gmail.com</a>/<a href="mailto:tsucheta@iicb.res.in" target="_blank">tsucheta@iicb.res.in</a> (S.T.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sucheta+Tripathy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tripathy%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Tripathy%20Sucheta%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Sucheta Tripathy</a><span class="delimiter"></span>
                        </span>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy044-FM2"><div class="fn"><div class="footnote-content">Diya Sen and Karnelia Paul authors contributed equally to this study.</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 131–146, <a href="https://doi.org/10.1093/dnares/dsy044">https://doi.org/10.1093/dnares/dsy044</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">07 January 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">28 December 2017</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">05 December 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">07 January 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5277053" href="/dnaresearch/article-pdf/26/2/131/28495630/dsy044.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Diya Sen, Karnelia Paul, Chinmay Saha, Gairik Mukherjee, Mayurakshi Nag, Samrat Ghosh, Abhishek Das, Anindita Seal, Sucheta Tripathy,  A unique life-strategy of an endophytic yeast <em>Rhodotorula mucilaginosa</em> JGTA-S1<em>—</em>a comparative genomics viewpoint, <em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 131–146, <a href="https://doi.org/10.1093/dnares/dsy044">https://doi.org/10.1093/dnares/dsy044</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5277053&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20unique%20life-strategy%20of%20an%20endophytic%20yeast%20%26lt%3Bem%26gt%3BRhodotorula%20mucilaginosa%26lt%3B%2Fem%26gt%3B%20JGTA-S1%26lt%3Bem%26gt%3B%E2%80%94%26lt%3B%2Fem%26gt%3Ba%20comparative%20genomics%20viewpoint&amp;publicationDate=2019-01-07&amp;volumeNum=26&amp;issueNum=2&amp;author=Sen%2C%20Diya%3B%20Paul%2C%20Karnelia&amp;startPage=131&amp;endPage=146&amp;contentId=10.1093%2Fdnares%2Fdsy044&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f131%2f5277053" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="134010249" id="134010249" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Endophytic yeasts of genus <em>Rhodotorula</em> are gaining importance for their ability to improve plant growth. The nature of their interaction with plants, however, remains unknown. <em>Rhodotorula mucilaginosa</em> JGTA-S1 was isolated as an endophyte of <em>Typha angustifolia</em> and promoted growth in the host. To investigate the life-strategy of the yeast from a genomics perspective, we used Illumina and Oxford Nanopore reads to generate a high-quality annotated draft assembly of JGTA-S1 and compared its genome to three other <em>Rhodotorula</em> yeasts and the close relative <em>Rhodosporidium toruloides</em>. JGTA-S1 is a haploid yeast possessing several genes potentially facilitating its endophytic lifestyle such as those responsible for solubilizing phosphate and producing phytohormones. An intact mating-locus in JGTA-S1 raised the possibility of a yet unknown sexual reproductive cycle in <em>Rhodotorula</em> yeasts. Additionally, JGTA-S1 had functional anti-freezing genes and was also unique in lacking a functional nitrate-assimilation pathway—a feature that is associated with obligate biotrophs. Nitrogen-fixing endobacteria were found within JGTA-S1 that may circumvent this defective N-metabolism. JGTA-S1 genome data coupled with experimental evidence give us an insight into the nature of its beneficial interaction with plants.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword='"genome assembly"'>genome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"oxford nanopore"'>oxford nanopore</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="endophyte">endophyte</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Rhodotorula"><em>Rhodotorula</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="endosymbiont">endosymbiont</a>
</div>                    <h2 scrollto-destination="134010251" id="134010251" class="section-title">1. Introduction</h2>
<p><em>Rhodotorula</em> is a group of yeasts belonging to the phylum Basidiomycota, family Sporidiobolaceae, and class Microbotryomycetes. These are found from widely varying environmental sources ranging from extreme climates of deep-sea vents to arctic cold deserts<a reveal-id="dsy044-B1" data-open="dsy044-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy044-B2" data-open="dsy044-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> from the soil, air, or as organisms associated with plants. Clinically <em>Rhodotorula</em> sp. is described as an opportunistic pathogen that causes pathogenesis in immune-suppressed patients with most cases reporting the causal organism to be <em>R. mucilaginosa.</em><a reveal-id="dsy044-B3" data-open="dsy044-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, plant-associated <em>Rhodotorula</em> sp. appears to have several beneficial effects on the host rather than any deleterious effects. <em>Rhodotorula graminis</em> strain WP1 v1.1 was isolated from poplar as an auxin producing yeast<a reveal-id="dsy044-B4" data-open="dsy044-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> having significant plant growth promoting (PGP) effects.<a reveal-id="dsy044-B5" data-open="dsy044-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><em>Rhodotorula mucilaginosa</em> (previously <em>Rhodotorula rubra</em>) TG-1 was reported to exhibit strong biocontrol activity against fungal and bacterial pathogens upon stem inoculation of rice.<a reveal-id="dsy044-B6" data-open="dsy044-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Although several <em>Rhodotorula</em> yeasts have started being described from plants, their interaction with plants and their life-strategies have not yet been studied.</p>
<p><em>Rhodosporidium</em> is a sexual, dimorphic red yeast belonging to the family Sporidiobolaceae<a reveal-id="dsy044-B7" data-open="dsy044-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> that also includes <em>Rhodotorula</em>. Haploid yeast cells of <em>Rhodosporidium</em> belonging to opposite mating types can conjugate leading to the formation of dikaryotic mycelia.<a reveal-id="dsy044-B8" data-open="dsy044-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> However, unlike <em>Rhodosporidium</em>, <em>Rhodotorula</em> is considered to be monomorphic i.e. consisting of the asexual haploid yeast-form with no transition between yeast and filament-forms.<a reveal-id="dsy044-B9" data-open="dsy044-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> This is the major distinction between the genus <em>Rhodosporidium</em> and <em>Rhodotorula</em>. The genome sequence of the haploid <em>Rhodotosporidium toruloides</em> NP11 has become available.<a reveal-id="dsy044-B10" data-open="dsy044-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Several other <em>Rhodotorula</em> genomes have been sequenced, such as <em>R. graminis</em> strain WP1,<a reveal-id="dsy044-B11" data-open="dsy044-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> a eurypsychrophilic strain, <em>R</em>. sp. JG1b isolated from permafrost in Antarctica,<a reveal-id="dsy044-B12" data-open="dsy044-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> and <em>R. mucilaginosa</em> strain C2.5t1 isolated from cacao seeds.<a reveal-id="dsy044-B13" data-open="dsy044-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p>
<p><em>Rhodotorula mucilaginosa</em> strain JGTA-S1 was isolated as an endophyte of <em>Typha angustifolia</em>, a wetland macrophyte growing in the metal-contaminated Uranium-mine-associated wetlands in Jaduguda, India.<a reveal-id="dsy044-B14" data-open="dsy044-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> JGTA-S1 promoted the growth of its natural host, <em>T. angustifolia</em>. We undertook a comparative genomics approach to understand the of the yeast <em>in planta</em> contributing to its endophytic life-strategy lifestyle and validated at least some of the attributes through experimental approaches that make JGTA-S1 unique. We sequenced, assembled, annotated the genome of JGTA-S1 and compared it to the sequenced genomes of some <em>Rhodotorula</em> yeasts and <em>R. toruloides</em>. The genome of JGTA-S1 was found to be haploid, with a low repeat content. Comparisons showed all five genomes under study to encode for genes with the potential for benefitting plant growth through phosphate solubilization and phytohormone production. In addition, the genomes appeared to have conserved strategies for adaptation to cold temperatures. The yeasts encoded genes suitable for bypassing plant defence using multiple tactics to facilitate establishment within the plant endosphere. JGTA-S1 was however different from <em>Rhodosporidium</em> and <em>R. graminis</em> in its nitrogen (N)-metabolism because of a truncated nitrate assimilation pathway which probably led to its association and maintenance of diazotrophic endobacteria leading to complex interaction. JGTA-S1 had an intact mating locus showing that it may undergo a filamentous intermediate in its lifecycle like <em>Rhodosporidium</em>.</p>                    <h2 scrollto-destination="134010255" id="134010255" class="section-title">2. Materials and methods</h2>
                    <h3 scrollto-destination="134010256" id="134010256" class="section-title">2.1. Endophytic colonization of JGTA-S1</h3>
<p><em>Typha angustifolia</em> seeds were surface sterilized and grown on artificial soil (Soilrite, Keltech Energies, India) supplemented with 2× Murashige Skoog salt mixture (MS, 15 ml media per 30 g soil) with or without <em>R. mucilaginosa</em> culture (10<sup>7</sup> cfu of JGTA-S1 suspended in 10 ml 1× MS). The yeast was added dropwise around the root of the plants and the plants were grown for 2 months. Shoot lengths were measured for control (<em>n</em> = 24) and JGTA-S1-treated (<em>n</em> = 24) plants. The significance of the increase in shoot length was tested by Mann-Whitney test and plotted using GraphPad Prism 5. The <em>Typha</em> shoots of treated plants 3-weeks post-infection (wpi) were hand sectioned and stained with 0.01% Acridine orange. The sections were observed under a fluorescence microscope (Olympus BX40).</p>                    <h3 scrollto-destination="134010258" id="134010258" class="section-title">2.2. Culture conditions and isolation of genomic DNA</h3>
<p>Genomic DNA was isolated from a saturated culture of <em>R. mucilaginosa</em> JGTA-S1. Yeast cells were collected by centrifugation from 10 ml saturated culture and was resuspended in 0.5 ml of distilled water. The tube was spun for additional 5 min and the supernatant was discarded. The tube was vortexed briefly to resuspend the pellet in the residual liquid. Two hundred microlitre of a solution containing 2% Triton X100, 1% sodium dodecyl sulphate (SDS), 100 mM NaCl, 100 mM Tris (pH: 8) and 1 mM ethylenediaminetetraacetic acid (EDTA) was added along with 0.2 ml phenol-chloroform and 0.3 g acid-washed glass beads. The tubes were vortexed vigorously for 10 min and 0.2 ml of Tris–EDTA (TE) buffer was added. The tubes were spun for 15 min at 13,000 rpm. The aqueous layer was collected and ethanol precipitated. The pellet was resuspended and treated with 30 µg RNaseA (Himedia) and re-precipitated with 4 M ammonium acetate and 2.5 volumes 100% ethanol at −80°C for 1 h. The tubes were spun at 13,000 rpm for 15 min, washed with 70% ethanol and resuspended in 50 µl TE.</p>                    <h3 scrollto-destination="134010260" id="134010260" class="section-title">2.3. Library preparation and whole genome sequencing</h3>
<p>A paired-end library was constructed using the NEXTFlex DNA-sequencing kit (BIOO Scientific, Austin, TX, USA) and sequencing was carried out on Illumina Miseq. The paired-end library had a read length of ∼250 bp with an insert size of 130–580 bp generating 10,499,886 paired-end reads at 262× coverage. Library for MinION was constructed with the 2D library preparation kit, SQK-NSK007, and sequencing was carried out on MinION Mk1b using R9 flowcell (Oxford Nanopore Technologies, Oxford, UK) in a 48 h sequencing protocol on MinKNOW 1.1.20. Reads generated from the run were basecalled using Metrichor V.2.45.1 and converted to fasta format using Poretools.<a reveal-id="dsy044-B15" data-open="dsy044-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The long-read library consisted of 74,892 reads containing 850,015,301 bp of 1D and 2D data at 42× coverage.</p>                    <h3 scrollto-destination="134010262" id="134010262" class="section-title">2.4. Genome assembly</h3>
<p>Adapters and low-quality bases were trimmed from the raw Illumina reads with Trimmomatic.<a reveal-id="dsy044-B16" data-open="dsy044-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Long reads from MinION were error-corrected with Canu.<a reveal-id="dsy044-B17" data-open="dsy044-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Whole genome assembly was carried out with SPAdes<a reveal-id="dsy044-B18" data-open="dsy044-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> using the trimmed and error-corrected reads. Polishing of the final assembly was done with Pilon.<a reveal-id="dsy044-B19" data-open="dsy044-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> In order to identify the mitochondrial genome, the complete mitochondrial genome of <em>Rhodotorula taiwanenis</em> RS1 (Accession no. HF558455.1) was aligned against A2 assembly with NUCmer. To determine the effect of the MinION reads on the assembly, a MinION-free assembly was carried out with SPAdes using only the trimmed Illumina reads. The quality of all assemblies was evaluated with QUAST<a reveal-id="dsy044-B20" data-open="dsy044-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and MUMmer.<a reveal-id="dsy044-B21" data-open="dsy044-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a></p>                    <h3 scrollto-destination="134010264" id="134010264" class="section-title">2.5. Ploidy calculation</h3>
<p>KmerCountExact programme from BBmap package (<a class="link link-uri" href="https://sourceforge.net/projects/bbmap/" target="_blank">https://sourceforge.net/projects/bbmap/</a>) was used to determine the ploidy and size of the genome.</p>                    <h3 scrollto-destination="134010266" id="134010266" class="section-title">2.6. Gene prediction and annotation</h3>
<p>Gene prediction was carried out using AUGUSTUS<a reveal-id="dsy044-B22" data-open="dsy044-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> with protein sequences of strain <em>Rhodotorula</em> sp. JG1b as the training dataset. Annotation was carried out in two ways. First, proteins were annotated using Blast2GO.<a reveal-id="dsy044-B23" data-open="dsy044-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Second, proteins were queried against NCBI’s nr database using BLASTP searches (E-value &lt; 1e-5) and the annotation of the top hit was fetched from the UniProt database. For Blast2GO, InterProScan was also carried out.<a reveal-id="dsy044-B24" data-open="dsy044-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The KEGG-KAAS server (<a class="link link-uri" href="http://www.genome.jp/tools/kaas/" target="_blank">http://www.genome.jp/tools/kaas/</a>) was used for pathway annotation. For prediction of carbohydrate-active-enzymes, protein sequences were submitted to the dbCAN server (<a class="link link-uri" href="http://csbl.bmb.uga.edu/dbCAN/" target="_blank">http://csbl.bmb.uga.edu/dbCAN/</a>). Annotation of the mitochondrial genome was carried out with MITOS WebServer (<a class="link link-uri" href="http://mitos.bioinf.uni-leipzig.de/index.py" target="_blank">http://mitos.bioinf.uni-leipzig.de/index.py</a>). Cluster analysis was carried out with the heatmap.2 function of the gplots package<a reveal-id="dsy044-B25" data-open="dsy044-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> in R.</p>                    <h3 scrollto-destination="134010268" id="134010268" class="section-title">2.7. Orthologue prediction</h3>
<p>For comparison among the <em>Rhodotorula</em> and <em>Rhodosporidium</em> genomes, protein sequences were extracted from <em>R</em>. <em>mucilaginosa</em> JGTA-S1, <em>R. mucilaginosa</em> C2.5t1, <em>R</em>. sp. JG1b, <em>R. graminis</em> WP1, and <em>R. toruloides</em> NP11. For comparison between the yeasts, endophytes, and pathogens from phylum Basidiomycota, protein sequences were extracted from <em>Grammothele lineata</em>, <em>Serendipita indica, R. graminis</em> WP1, <em>Ustilago maydis</em>, <em>Puccinia graminis, Melampsora larici-populina, Microbotryum intermedium,</em> and <em>Microbotryum lychnidis-dioicae</em>. OrthoMCL<a reveal-id="dsy044-B26" data-open="dsy044-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was run on these sets and analysed using BLASTP (E-value &lt; 1e-5) and an inflation value of 1.5. Enrichment analysis was carried out with clusterProfiler package<a reveal-id="dsy044-B27" data-open="dsy044-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> in R.</p>                    <h3 scrollto-destination="134010270" id="134010270" class="section-title">2.8. Synteny analysis</h3>
<p>Syntenic blocks between <em>R. mucilaginosa</em> JGTA-S1 and <em>R. mucilaginosa</em> C2.5t1 were identified by MCScanX<a reveal-id="dsy044-B28" data-open="dsy044-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and DAGchainer<a reveal-id="dsy044-B29" data-open="dsy044-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> using a cut-off of five orthologous genes to define a syntenic block.</p>                    <h3 scrollto-destination="134010272" id="134010272" class="section-title">2.9. Phylogeny prediction</h3>
<p>A phylogeny based on 3,660 single-copy orthologous proteins was inferred from the OrthoMCL-generated orthologous groups. Each orthologous group of proteins was aligned with ClustalW2<a reveal-id="dsy044-B30" data-open="dsy044-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and concatenated into a superalignment. Proml<a reveal-id="dsy044-B31" data-open="dsy044-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> was used with default parameters to infer a maximum likelihood tree. Similarity among the genomes was inferred from a binary matrix comprising of presence or absence of each orthologous protein. Jaccard distance was used to convert the binary matrix into a distance matrix which was subjected to hierarchical clustering.</p>                    <h3 scrollto-destination="134010274" id="134010274" class="section-title">2.10. Effector prediction</h3>
<p>Signal peptide-containing proteins were predicted using SignalP V3.0<a reveal-id="dsy044-B32" data-open="dsy044-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> (Prediction = ‘Signal peptide’, Y<sub>max</sub> ≥ 0.5, D score ≥ 0.5, <em>P</em> ≥ 0.9). Presence of transmembrane domains and mitochondrial signal peptides in these proteins was predicted using TMHMM V2.0c (<a class="link link-uri" href="http://www.cbs.dtu.dk/services/TMHMM/" target="_blank">http://www.cbs.dtu.dk/services/TMHMM/</a>) and only those proteins with no transmembrane domains were retained in the analysis. TargetP (<a class="link link-uri" href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank">http://www.cbs.dtu.dk/services/TargetP/</a>) was used to predict subcellular localization. Only those proteins predicted to be secreted were retained (Location = ‘Secreted’). Effectors were predicted from secretory proteins with EffectorP (<a class="link link-uri" href="http://effectorp.csiro.au/" target="_blank">http://effectorp.csiro.au/</a>).</p>                    <h3 scrollto-destination="134010276" id="134010276" class="section-title">2.11. Nucleotide sequence accession numbers</h3>
<p><em>Rhodotorula mucilaginosa</em> JGTA-S1 genome sequencing project has been deposited in NCBI under BioProject PRJNA393004 and BioSample SAMN07313544. The genome assembly has been deposited at DDBJ/ENA/GenBank under the accession <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/PEFX00000000">PEFX00000000</a>. The Illumina and MinION reads were deposited to Sequence Read Archive under accession numbers <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/SRR5821556">SRR5821556</a> and <a class="link link-gen" target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/SRR5821557">SRR5821557</a>.</p>                    <h3 scrollto-destination="134010278" id="134010278" class="section-title">2.12. Assay of phosphate solubilization</h3>
<p><em>Rhodotorula mucilaginosa</em> JGTA-S1 culture grown for 48 h was spotted on Pikovskaya’s agar plates for qualitative phosphate solubilization assay and halo formation was scored.</p>                    <h3 scrollto-destination="134010280" id="134010280" class="section-title">2.13. Effect of <em>R. mucilaginosa</em> on rice plants in presence of low phosphate</h3>
<p>Rice seeds (<em>Oryza sativa</em> cv. Swarna MTU7029) were surface-sterilized and germinated on filter papers. Four- to five-day-old seedlings of equal lengths were transferred to Soilrite<sup>TM</sup> containing artificial soil supplied with low-phosphate media [1 µm NaH<sub>2</sub>PO<sub>4</sub> (as only phosphate source), 2.0 mM NH<sub>4</sub>NO<sub>3</sub>, 1.9 mM KNO<sub>3</sub>, 0.3 mM CaCl<sub>2</sub>.2H<sub>2</sub>0, 0.15 mM MgSO<sub>4</sub>.7H<sub>2</sub>0, 5 µm KI, 25 µm H<sub>3</sub>BO<sub>3</sub>, 0.1 mM MnSO<sub>4</sub>.H<sub>2</sub>O, 0.3 mM ZnSO<sub>4</sub>.7H<sub>2</sub>0, 1 µM Na<sub>2</sub>MoO<sub>4</sub>.2H<sub>2</sub>0, 0.1 µM CuSO<sub>4</sub>.5H<sub>2</sub>0, 0.1 µM CoCl<sub>2</sub>.6H<sub>2</sub>O, 0.1 mM FeSO<sub>4</sub>.7H<sub>2</sub>0, 0.1 mM Na<sub>2</sub>EDTA.2H<sub>2</sub>0, inositol (10 mg l<sup>−</sup><a reveal-id="dsy044-B1" data-open="dsy044-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>), and Glycine (0.2 mg l<sup>−</sup><a reveal-id="dsy044-B1" data-open="dsy044-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>)] as described before.<a reveal-id="dsy044-B33" data-open="dsy044-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> For treated plants, 10<sup>7</sup> cfu of JGTA-S1 cells suspended in low-phosphate media was added to the soil around a plant root. In control plants, an equal volume of low-phosphate media was supplied. Pictures were taken 10 days post-infection.</p>                    <h3 scrollto-destination="134010282" id="134010282" class="section-title">2.14. Assay of cold tolerance of <em>R. mucilaginosa</em> JGTA-S1</h3>
<p><em>Rhodotorula mucilaginosa</em> JGTA-S1 and <em>Saccharomyces cerevisiae</em> ThyAP4 cells were grown in YPD media to saturation. The cells were spotted on YPD agar at OD<sub>600</sub>=1.0. The cells were incubated at 30 and 5°C and growth was observed after 4 days.</p>                    <h3 scrollto-destination="134010284" id="134010284" class="section-title">2.15. Real-time PCR of anti-freeze genes</h3>
<p>JGTA-S1 cells were spotted on tryptic soy broth (TSB) media and grown at 30°C for 4 days. Control cells were frozen in liquid nitrogen after 4 days. For cold treatment, the second batch of cells was shifted from 30 to 5°C and incubated at 5°C for 4 additional days. Primers for anti-freezing genes g5846.t1 (Forward 5’ GGTCTTAAGCCGGTCGTCAA3’ and Reverse 5’AAGGTGGGAGGGGGATAGTC3’) and g5876.t1 (Forward 5’TCGTGTTCCACAGGCTACTG3’ and Reverse 5’TACGGGTCGTTTTCGCAGTT3’) were designed by Primer-BLAST software<a reveal-id="dsy044-B34" data-open="dsy044-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> using <em>Rhodotorula</em> sp. JG1b as an organism. For RNA isolation, cell pellets were suspended in urea buffer (urea 8 M, NaCl 0.5 M, Tris 20 mM, EDTA 20 mM, SDS 2%, pH 8). Glass beads and an equal volume of Trizol (Invitrogen) was added to the suspended cells and subjected to vigorous vortexing. Cells were kept in ice every 2 min. Chloroform was added (one-fifth volume) and vortexed again. The solution was kept on ice for 10 min and centrifuged. An equal amount of Trizol was again added to the supernatant and kept in ice for another 5 min. The chloroform step was repeated. The aqueous layer was transferred and precipitated with 50% (v/v) isopropanol and glycogen (1 µg/ml). The RNA pellet was washed with 85% ethanol and dissolved in RNase-free water. RNAs were subjected to DNase treatment and converted to cDNA with M-MuLV reverse transcriptase and random hexamers (Thermo Scientific) as per the manufacturer’s protocol. An equal amount of cDNA was used for Real-time PCR. A 10 µl of qRT-PCR reaction contained 5 µl DyNAmoColorFlash SYBR Green I master mix (Thermo Scientific), 0.5µM of each primer and 50 ng (Nanodrop 2000, Thermo scientific) of cDNA. Real-time PCR was performed using Step one plus real-time PCR machine (Applied Biosystems). The thermal step-up was 10 min at 95°C, followed by 40 cycles of 15 s at 95°C, 1 min at 60°C. Data analysis was performed using ExpressionSuite Software v1.0.4 (Applied Biosystems). The fold change was calculated by 2<sup>−ΔΔCt</sup> method using JGTA-S1 glyceraldehyde 3-phosphate dehydrogenase (GAPDH) gene (Fwd 5’ GAGTTGATGCGCGAGCTGGAGAACG 3’ and Rev 5’ GTGGCCGAGAAGGGTCGGCCGAGG 3’) as housekeeping gene. The primer efficiencies were checked and found to be &gt;90 for every primer pairs using LinReg PCR 2017.1.0.0 software. The result corresponds to two independent experiments each in triplicate.</p>                    <h3 scrollto-destination="134010286" id="134010286" class="section-title">2.16. Effect of removal of endosymbiotic bacteria from JGTA-S1 and real-time PCR of 16SrDNA specific DNA</h3>
<p>JGTA-S1 was streaked five times in presence of Timentin (250 µg/ml) and Tetracycline (10 µg/ml) on N-free media (5 g Glucose; 0.1 g yeast extract; 0.15 g CaCl<sub>2</sub>; 0.005 g Na<sub>2</sub>MoO<sub>4</sub>; 0.2 g MgSO<sub>4</sub>.7H<sub>2</sub>0; 0.04 g FeSO<sub>4</sub> 7H<sub>2</sub>0; 0.5 g K<sub>2</sub>HPO<sub>4</sub>; 1.0 g CaCO<sub>3</sub>; 15 g Agar; distilled water to 1 l; pH 7.0) with 5 g/l (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>. Every time, isolated single colonies were used for streaking. Almost 30 colonies were cultured in TSB and genomic DNA was isolated as described earlier. The genomic DNA was used as a template to screen for colonies where no amplification was observed for <em>Pseudomonas stutzeri</em> 16SrDNA (Fwd 5’AGGCGACGATCCGTAACTGG3’ and Rev 5’GTCATCCCCACCTTCCTCCG3’). The PCR cycle was 5 min at 98°C followed by 35 cycles of 30 s at 98°C, 15 s at 62°C, 27 s at 72°C with a final extension of 7 min at 72°C. A strain (Strain 11 henceforth) lacking <em>P. stutzeri</em> 16SrDNA amplification in two independent PCR reactions was used for further studies. Strain 11 or JGTA-S1 control cells were grown in TSB or N-free media containing (NH<sub>4</sub>)<sub>2</sub>SO<sub>4.</sub> The cell pellets were washed twice with water and suspended in water. The OD<sub>600</sub> was normalized and spotted on TSB and N-free media at different dilutions [undiluted (0.5 OD), 1:100 and 1:1,000]. For qPCR genomic DNA was isolated from JGTA-S1 (control) and Strain 11. <em>Pseudomonas stutzeri</em> 16SrDNA gene specific primers (Fwd 5’ TGAGTAATGCCTAGGAATCTGCC 3’ and Rev 5’ ATCCGACCTAGGCTCATCTGATA 3’) were designed using Primer-BLAST software using <em>Pseudomonas stutzeri</em> DSM 4166 as an organism for qPCR. An equal amount of genomic DNA was used for Real-time PCR. A 10 µl of qRT-PCR reaction contained 5 µl DyNAmoColorFlash SYBR Green I master mix (Thermo Scientific), 0.5 µM of each primer and 1.5 ng (measured in Qubit, Thermo Fisher) of genomic DNA. Real-time PCR was performed using a 7500 real-time PCR machine (Applied Biosystems). The thermal step-up was 10 min at 95°C, followed by 40 cycles of 15 s at 95°C, 30 s at 60°C. The fold change was calculated by 2<sup>−ΔΔCt</sup> method using JGTA-S1 GAPDH gene as housekeeping gene. The result corresponds to three independent experiments each performed in triplicate. The significance of the increase in relative <em>P. stutzeri</em> levels/JGTA-S1 cell was tested by Mann-Whitney test and plotted using GraphPad Prism 5. The primer efficiencies were checked using LinReg PCR 2017.1.0.0 software as described before. A Strain 11.2 was generated from Strain 11. Strain 11 was subjected to one passage in presence of Carbenicillin (100 µg/ml), Gentamycin (30 µg/ml), and Kanamycin (50 µg/ml) in N-free media + (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>. The colonies were revived once by growing them on TSB plate w/o antibiotics followed by another round of streaking in N-free media + (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> containing Carbenicillin, Gentamycin, and Kanamycin. A primer pair corresponding to the V3 region of 16SrDNA of bacteria (Ba341Fwd: CCTACGGGAGGCAGCAG and Ba518Rev: ATTACCGCGGCTGCTGG) was used for amplification of total endobacteria. Genomic DNA (1 µg, measured by Nanodrop, Thermo scientific) from control, Strains 11 and 11.2 was used as a template. For no template control (NTC) no DNA was added. The PCR cycle was 5 min at 94°C, followed by 11 cycles of 1 min at 94°C, 1 min at 65°C, 3 min at 72°C and 20 cycles of 1 min at 94°C, 1 min at 55°C, 3 min at 72°C and a final extension at 72°C for 10 min. Spotting assay (on N-free media) and qPCR from Strain 11.2 were done as described for Strain 11.</p>                    <h2 scrollto-destination="134010288" id="134010288" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="134010289" id="134010289" class="section-title">3.1. <em>Typha</em> endophyte <em>R. mucilaginosa</em> JGTA-S1 has a compact repeat-poor haploid genome</h3>
<p><em>Rhodotorula mucilaginosa</em> JGTA-S1 was isolated as an endophyte of <em>T. angustifolia</em>. JGTA-S1 was isolated from <em>Typha</em> shoot. An independent isolate of <em>R. mucilaginosa</em> (JGTA-R1, KU051691.1) was isolated from <em>Typha</em> root demonstrating that the yeast was probably able to colonize both shoot and root of <em>Typha</em>. The yeast showed strong growth promoting effects on <em>Typha</em> (<a reveal-id="dsy044-F1" data-open="dsy044-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). The increase in shoot lengths of JGTA-S1-treated plants was tested with Mann-Whitney analysis and the median value was found to be significantly increased upon yeast treatment (<a reveal-id="dsy044-F1" data-open="dsy044-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>, right panel). The endophytic colonization of JGTA-S1 was confirmed by staining shoot sections of <em>Typha</em> infected with JGTA-S1 with acridine orange 3 wpi. JGTA cells were found in <em>Typha</em> sections validating its endophytic colonization (<a reveal-id="dsy044-F1" data-open="dsy044-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>). To get a better insight into its life-strategy, JGTA-S1 was sequenced on Illumina Miseq at 262× coverage and Oxford Nanopore MinION at 42× coverage. Self-corrected MionION reads were combined with adapter-trimmed and quality-checked Illumina reads to generate a hybrid assembly. Use of Nanopore reads in addition to the Illumina reads resulted in a better assembly comprising of fewer but longer scaffolds and almost 3-fold increased N50 value (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) resulting in Assembly A1 (<a reveal-id="dsy044-T1" data-open="dsy044-T1" class="link link-reveal link-table xref-fig">Table 1</a>). This was further improved by removing scaffolds that were &lt;2,000 bp to generate assembly A2 that consisted of only 46 scaffolds, N50 of 685,833, length of 20,108,097 bp where the largest scaffold was 2,159,349 bp (<a reveal-id="dsy044-T1" data-open="dsy044-T1" class="link link-reveal link-table xref-fig">Table 1</a>). KmerCountExact was also able to determine that JGTA-S1 was a haploid genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) of size 21,822,513 bp. JGTA-S1 assembly size was close to the estimated size. Moreover, it had only 46 scaffolds in comparison to the 1, 034 scaffolds of <em>R</em>. <em>mucilaginosa</em> strain C2.5t1. MUMmerplots of NUCmer and PROmer alignments showed colinearity between JGTA-S1 and C2.5t1 suggesting a good agreement between the genomes (<a reveal-id="dsy044-F2" data-open="dsy044-F2" class="link link-reveal link-table xref-fig">Fig. 2a and b</a>). The mitochondrial genome of JGTA-S1 was integrated into Scaffold 2 of the assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The annotated mitochondrial genome consisted of tRNA, rRNA, genes encoding ATP synthase subunits and cytochrome oxidase subunits (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p>                        <a id="134010291" scrollto-destination="134010291"></a>
<div id="dsy044-F1" data-id="dsy044-F1" class="fig fig-section">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f1.png?Expires=1576462815&amp;Signature=YzLQS8m4DXDb~QKX~xIpYyNsuLW-X6eHWKKkZrg-Gd~niLm16VO7dDQqBJifQlzZdXf1QNMlh4AgBDqB7Eny2JKyZvs2lRnuGUZc5R~AqRFX5NVhxla1nvTs~A6ePTizYblDIy8eT60puJRN9ZOBNT25LlvSmU1djtOTFqTPu-BcIRUpi668Qe3nnwAnZi6zArgp2RWfRAZWspL4mQPQki~QutRoS67ZdVjM-TDpKxzuFhL28UAzaRgcSEdnqe-SyYgvX6IcvpxOUfOEaIvgQ6wYXGwF5QS0Cyo~s9AKrT5MJ7jfrYn0x8ccSQUgVNG-X0dJR6YW-6HI7uOdmUPvGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rhodotorula mucilaginosa JGTA-S1 colonizes in T. angustifolia endophytically and promotes growth. Typha angustifolia seeds were surface sterilized and germinated on artificial soil. JGTA-S1 culture was added around the root. For control plants, 1× MS was added. Plant length was measured after 2 months. Plant lengths were subjected to the Mann-Whitney test. For determination of colonization, Typha shoots were hand sectioned and stained with Acridine orange 3wpi. (a) Digital image of control and JGTA-S1-treated Typha (left), Mann-Whitney test for shoot length (right). (b) Rhodotorula cells in shoot sections of Typha. Color figures are available at DNARES online." path-from-xml="dsy044f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010291/dsy044f1.tif" path-from-xml="dsy044f1.tif" target="_blank">Open in new tab</a><a section="134010291" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f1.png?Expires=1576462815&amp;Signature=0vBAKMk42OFuHbnH2LktchtHagRuFso-xjTgzJODKKmhT9r43tDG8ou-wy7puUOFMrNwsABd45tu1NZJw7Dne66hZ8GPNyMAQfgns96Z5vzmeaWZ23pJ6fyBfHzoUqEhOMc~7PTKPvN08JdtDQ2b8hR5gBKtnv16eDdL7Cdni2cbM9hBJxS1~Xq4EGvEyxyAQOhppP47SVGrVHq7KjjfEYFjo2ToRyFS9kCkSOCb3PEHrGNnQ2EK~NEtA9vdV8KMzaSl43q74hQC8b2HrqWfKBTOC~G9uAf1lNcw1LO8HF2~~MK4n-lYS9cPSE-nOKHi4pqsocg4Eg9TvNCbLu5w5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010291&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p><em>Rhodotorula mucilaginosa</em> JGTA-S1 colonizes in <em>T. angustifolia</em> endophytically and promotes growth. <em>Typha angustifolia</em> seeds were surface sterilized and germinated on artificial soil. JGTA-S1 culture was added around the root. For control plants, 1× MS was added. Plant length was measured after 2 months. Plant lengths were subjected to the Mann-Whitney test. For determination of colonization, <em>Typha</em> shoots were hand sectioned and stained with Acridine orange 3wpi. (a) Digital image of control and JGTA-S1-treated <em>Typha</em> (left), Mann-Whitney test for shoot length (right). (b) <em>Rhodotorula</em> cells in shoot sections of <em>Typha</em>. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f1.png?Expires=1576462815&amp;Signature=YzLQS8m4DXDb~QKX~xIpYyNsuLW-X6eHWKKkZrg-Gd~niLm16VO7dDQqBJifQlzZdXf1QNMlh4AgBDqB7Eny2JKyZvs2lRnuGUZc5R~AqRFX5NVhxla1nvTs~A6ePTizYblDIy8eT60puJRN9ZOBNT25LlvSmU1djtOTFqTPu-BcIRUpi668Qe3nnwAnZi6zArgp2RWfRAZWspL4mQPQki~QutRoS67ZdVjM-TDpKxzuFhL28UAzaRgcSEdnqe-SyYgvX6IcvpxOUfOEaIvgQ6wYXGwF5QS0Cyo~s9AKrT5MJ7jfrYn0x8ccSQUgVNG-X0dJR6YW-6HI7uOdmUPvGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rhodotorula mucilaginosa JGTA-S1 colonizes in T. angustifolia endophytically and promotes growth. Typha angustifolia seeds were surface sterilized and germinated on artificial soil. JGTA-S1 culture was added around the root. For control plants, 1× MS was added. Plant length was measured after 2 months. Plant lengths were subjected to the Mann-Whitney test. For determination of colonization, Typha shoots were hand sectioned and stained with Acridine orange 3wpi. (a) Digital image of control and JGTA-S1-treated Typha (left), Mann-Whitney test for shoot length (right). (b) Rhodotorula cells in shoot sections of Typha. Color figures are available at DNARES online." path-from-xml="dsy044f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010291/dsy044f1.tif" path-from-xml="dsy044f1.tif" target="_blank">Open in new tab</a><a section="134010291" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f1.png?Expires=1576462815&amp;Signature=0vBAKMk42OFuHbnH2LktchtHagRuFso-xjTgzJODKKmhT9r43tDG8ou-wy7puUOFMrNwsABd45tu1NZJw7Dne66hZ8GPNyMAQfgns96Z5vzmeaWZ23pJ6fyBfHzoUqEhOMc~7PTKPvN08JdtDQ2b8hR5gBKtnv16eDdL7Cdni2cbM9hBJxS1~Xq4EGvEyxyAQOhppP47SVGrVHq7KjjfEYFjo2ToRyFS9kCkSOCb3PEHrGNnQ2EK~NEtA9vdV8KMzaSl43q74hQC8b2HrqWfKBTOC~G9uAf1lNcw1LO8HF2~~MK4n-lYS9cPSE-nOKHi4pqsocg4Eg9TvNCbLu5w5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010291&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p><em>Rhodotorula mucilaginosa</em> JGTA-S1 colonizes in <em>T. angustifolia</em> endophytically and promotes growth. <em>Typha angustifolia</em> seeds were surface sterilized and germinated on artificial soil. JGTA-S1 culture was added around the root. For control plants, 1× MS was added. Plant length was measured after 2 months. Plant lengths were subjected to the Mann-Whitney test. For determination of colonization, <em>Typha</em> shoots were hand sectioned and stained with Acridine orange 3wpi. (a) Digital image of control and JGTA-S1-treated <em>Typha</em> (left), Mann-Whitney test for shoot length (right). (b) <em>Rhodotorula</em> cells in shoot sections of <em>Typha</em>. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                        <a id="134010292" scrollto-destination="134010292"></a>
<div id="dsy044-F2" data-id="dsy044-F2" class="fig fig-section">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f2.png?Expires=1576462815&amp;Signature=zXPa-PADegcKJrQeSf9lx0XCydJnxWPcWJIfZO33F87oJbJPha8jZWvus2FAAhyPsbybpX~ltXvbxnuKzNMaeLX1b9pDw2HYZVNQAaUTsrj~yUARuEU-ug1b03qrffdHcOxzJX94TnG9FAOAuOB6E~ajHkyP4i36ZJP5dKo-2OiRjzDsQfn688FPmodOZE9aWA9~AshmWw6scXwpqpgd5OHzJ4WS9Z9pXVE3RCkJQSr9sqX-SL58C0MlUrQNIjn2a42dbFYMszuHzkuAwZuRF8cYgzB-3MPe91pEqLHWRE2lT~PEqLVhUI0EfrVwsJKA6vKXV6CGuqN0dsMahaAofg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome synteny between R. mucilaginosa JGTA-S1 and R. mucilaginosa C2.5t1. (a) Dot plot showing nucleotide vs nucleotide alignment of JGTA-S1 genome assembly (Y-axis) compared with C2.5t1 genome assembly (X-axis) was generated by the NUCmer programme from the MUMmer 3.23 package.20 (b) Dot plot showing alignment after six-frame translations of JGTA-S1 genome assembly (Y-axis) and C2.5t1 genome assembly (X-axis) was generated by the PROmer programme from the MUMmer 3.23 package.20 Red dots indicate matches from the same strand and blue dots indicate matches from opposite strands. The straight red line indicates good alignment between the X and Y axis. (c) Collinear blocks between JGTA-S1 (labelled as ‘or’ in the top half of semi-circle) and C2.5t1 (labelled as ‘nr’ in the bottom half of semi-circle) were identified by MCScanX28 from all-against-all BLASTP searches between JGTA-S1 and C2.5t1. The circle plotter script in MCScanX28 was used to generate the plot. Each rectangle depicts a scaffold in the genome and the synteny between scaffolds is depicted as ribbons. Color figures are available at DNARES online." path-from-xml="dsy044f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010292/dsy044f2.tif" path-from-xml="dsy044f2.tif" target="_blank">Open in new tab</a><a section="134010292" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f2.png?Expires=1576462815&amp;Signature=STPWqRavCWuQj7AHZjp4kZW89BzD6kZ~DEr~cQ8XuB4-lzyk8kqZV0RnusyrLOuKAahVJIGuoP2VoepkLX3XKzw5vdwUOoEtmB2xHxm6NyJvhSurrN2puzu4cBSCJDZYs2~vfSnVFKIQIVhQ5KI2J2v7oVSQejUXzh2c0n232YgwJYNgscwk6s-ikEH2cwr0hTe5BqC2ggND0h3we1GCRtAxIfcZaNd9IkYXXMf67Y6W7LUQOn8alpTcGQ-wzYYgtjAYbPqFuSUFZIDEmxloq-SUZ-bg97KTUeZH8xr3L2NkutQz1gC5QLWGaYbLiFnby-8Z3TeTWqD4ASfcEaC8bQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010292&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Genome synteny between <em>R. mucilaginosa</em> JGTA-S1 and <em>R. mucilaginosa</em> C2.5t1. (a) Dot plot showing nucleotide vs nucleotide alignment of JGTA-S1 genome assembly (Y-axis) compared with C2.5t1 genome assembly (X-axis) was generated by the NUCmer programme from the MUMmer 3.23 package.<a reveal-id="dsy044-B20" data-open="dsy044-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (b) Dot plot showing alignment after six-frame translations of JGTA-S1 genome assembly (Y-axis) and C2.5t1 genome assembly (X-axis) was generated by the PROmer programme from the MUMmer 3.23 package.<a reveal-id="dsy044-B20" data-open="dsy044-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Red dots indicate matches from the same strand and blue dots indicate matches from opposite strands. The straight red line indicates good alignment between the X and Y axis. (c) Collinear blocks between JGTA-S1 (labelled as ‘or’ in the top half of semi-circle) and C2.5t1 (labelled as ‘nr’ in the bottom half of semi-circle) were identified by MCScanX<a reveal-id="dsy044-B28" data-open="dsy044-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> from all-against-all BLASTP searches between JGTA-S1 and C2.5t1. The circle plotter script in MCScanX<a reveal-id="dsy044-B28" data-open="dsy044-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> was used to generate the plot. Each rectangle depicts a scaffold in the genome and the synteny between scaffolds is depicted as ribbons. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f2.png?Expires=1576462815&amp;Signature=zXPa-PADegcKJrQeSf9lx0XCydJnxWPcWJIfZO33F87oJbJPha8jZWvus2FAAhyPsbybpX~ltXvbxnuKzNMaeLX1b9pDw2HYZVNQAaUTsrj~yUARuEU-ug1b03qrffdHcOxzJX94TnG9FAOAuOB6E~ajHkyP4i36ZJP5dKo-2OiRjzDsQfn688FPmodOZE9aWA9~AshmWw6scXwpqpgd5OHzJ4WS9Z9pXVE3RCkJQSr9sqX-SL58C0MlUrQNIjn2a42dbFYMszuHzkuAwZuRF8cYgzB-3MPe91pEqLHWRE2lT~PEqLVhUI0EfrVwsJKA6vKXV6CGuqN0dsMahaAofg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome synteny between R. mucilaginosa JGTA-S1 and R. mucilaginosa C2.5t1. (a) Dot plot showing nucleotide vs nucleotide alignment of JGTA-S1 genome assembly (Y-axis) compared with C2.5t1 genome assembly (X-axis) was generated by the NUCmer programme from the MUMmer 3.23 package.20 (b) Dot plot showing alignment after six-frame translations of JGTA-S1 genome assembly (Y-axis) and C2.5t1 genome assembly (X-axis) was generated by the PROmer programme from the MUMmer 3.23 package.20 Red dots indicate matches from the same strand and blue dots indicate matches from opposite strands. The straight red line indicates good alignment between the X and Y axis. (c) Collinear blocks between JGTA-S1 (labelled as ‘or’ in the top half of semi-circle) and C2.5t1 (labelled as ‘nr’ in the bottom half of semi-circle) were identified by MCScanX28 from all-against-all BLASTP searches between JGTA-S1 and C2.5t1. The circle plotter script in MCScanX28 was used to generate the plot. Each rectangle depicts a scaffold in the genome and the synteny between scaffolds is depicted as ribbons. Color figures are available at DNARES online." path-from-xml="dsy044f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010292/dsy044f2.tif" path-from-xml="dsy044f2.tif" target="_blank">Open in new tab</a><a section="134010292" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f2.png?Expires=1576462815&amp;Signature=STPWqRavCWuQj7AHZjp4kZW89BzD6kZ~DEr~cQ8XuB4-lzyk8kqZV0RnusyrLOuKAahVJIGuoP2VoepkLX3XKzw5vdwUOoEtmB2xHxm6NyJvhSurrN2puzu4cBSCJDZYs2~vfSnVFKIQIVhQ5KI2J2v7oVSQejUXzh2c0n232YgwJYNgscwk6s-ikEH2cwr0hTe5BqC2ggND0h3we1GCRtAxIfcZaNd9IkYXXMf67Y6W7LUQOn8alpTcGQ-wzYYgtjAYbPqFuSUFZIDEmxloq-SUZ-bg97KTUeZH8xr3L2NkutQz1gC5QLWGaYbLiFnby-8Z3TeTWqD4ASfcEaC8bQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010292&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Genome synteny between <em>R. mucilaginosa</em> JGTA-S1 and <em>R. mucilaginosa</em> C2.5t1. (a) Dot plot showing nucleotide vs nucleotide alignment of JGTA-S1 genome assembly (Y-axis) compared with C2.5t1 genome assembly (X-axis) was generated by the NUCmer programme from the MUMmer 3.23 package.<a reveal-id="dsy044-B20" data-open="dsy044-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (b) Dot plot showing alignment after six-frame translations of JGTA-S1 genome assembly (Y-axis) and C2.5t1 genome assembly (X-axis) was generated by the PROmer programme from the MUMmer 3.23 package.<a reveal-id="dsy044-B20" data-open="dsy044-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Red dots indicate matches from the same strand and blue dots indicate matches from opposite strands. The straight red line indicates good alignment between the X and Y axis. (c) Collinear blocks between JGTA-S1 (labelled as ‘or’ in the top half of semi-circle) and C2.5t1 (labelled as ‘nr’ in the bottom half of semi-circle) were identified by MCScanX<a reveal-id="dsy044-B28" data-open="dsy044-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> from all-against-all BLASTP searches between JGTA-S1 and C2.5t1. The circle plotter script in MCScanX<a reveal-id="dsy044-B28" data-open="dsy044-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> was used to generate the plot. Each rectangle depicts a scaffold in the genome and the synteny between scaffolds is depicted as ribbons. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                        <a id="134010293" scrollto-destination="134010293"></a>
<div content-id="dsy044-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy044-T1" class="table-wrap-title" data-id="dsy044-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Assembly statistics of <em>R. mucilaginosa</em> JGTA-S1</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>A1 assembly </th>
<th>A2 assembly </th>
</tr></thead>
<tbody>
<tr>
<td>No of scaffolds </td>
<td>341 </td>
<td>46 </td>
</tr>
<tr>
<td>N50 </td>
<td>685,833 </td>
<td>685,833 </td>
</tr>
<tr>
<td>Length of assembly </td>
<td>20,358,951 bp </td>
<td>20,108,097 bp </td>
</tr>
<tr>
<td>Largest scaffold </td>
<td>2,159,349 bp </td>
<td>2,159,349 bp </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy044-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>A1 assembly </th>
<th>A2 assembly </th>
</tr></thead>
<tbody>
<tr>
<td>No of scaffolds </td>
<td>341 </td>
<td>46 </td>
</tr>
<tr>
<td>N50 </td>
<td>685,833 </td>
<td>685,833 </td>
</tr>
<tr>
<td>Length of assembly </td>
<td>20,358,951 bp </td>
<td>20,108,097 bp </td>
</tr>
<tr>
<td>Largest scaffold </td>
<td>2,159,349 bp </td>
<td>2,159,349 bp </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010293" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy044-T1" class="table-wrap-title" data-id="dsy044-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Assembly statistics of <em>R. mucilaginosa</em> JGTA-S1</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>A1 assembly </th>
<th>A2 assembly </th>
</tr></thead>
<tbody>
<tr>
<td>No of scaffolds </td>
<td>341 </td>
<td>46 </td>
</tr>
<tr>
<td>N50 </td>
<td>685,833 </td>
<td>685,833 </td>
</tr>
<tr>
<td>Length of assembly </td>
<td>20,358,951 bp </td>
<td>20,108,097 bp </td>
</tr>
<tr>
<td>Largest scaffold </td>
<td>2,159,349 bp </td>
<td>2,159,349 bp </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy044-T1" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>A1 assembly </th>
<th>A2 assembly </th>
</tr></thead>
<tbody>
<tr>
<td>No of scaffolds </td>
<td>341 </td>
<td>46 </td>
</tr>
<tr>
<td>N50 </td>
<td>685,833 </td>
<td>685,833 </td>
</tr>
<tr>
<td>Length of assembly </td>
<td>20,358,951 bp </td>
<td>20,108,097 bp </td>
</tr>
<tr>
<td>Largest scaffold </td>
<td>2,159,349 bp </td>
<td>2,159,349 bp </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010293" target="_blank">Open in new tab</a></div>
</div></div>
<p>In the absence of annotations from C2.5t1 the only sequenced <em>mucilaginosa</em> genome, we used the genome and proteome of <em>R.</em> sp. JG1b for training AUGUSTUS.<a reveal-id="dsy044-B22" data-open="dsy044-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> A total of 5,922 protein-coding genes were predicted from the genome of <em>R. mucilaginosa</em> JGTA-S1 (<a reveal-id="dsy044-T2" data-open="dsy044-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The average gene lengths, mean exon lengths and mean intron lengths were determined to be 2,778, 275, and 134 bp, respectively (<a reveal-id="dsy044-T2" data-open="dsy044-T2" class="link link-reveal link-table xref-fig">Table 2</a>). It was found to average 200 genes Mb<sup>−1</sup> with the densest region having ∼350 genes Mb<sup>−1</sup> which is within the range reported for Puccinomycotina genomes.<a reveal-id="dsy044-B35" data-open="dsy044-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> The same gene prediction protocol resulted in 6,463 predicted genes in <em>R. mucilaginosa</em> C2.5t1 (<a reveal-id="dsy044-T2" data-open="dsy044-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The UniProt and Blast2GO annotations of JGTA-S1 are presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. Among the top 5 GO terms in ‘Biological Process’ were those involved in the N-metabolic process (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p>                        <a id="134010295" scrollto-destination="134010295"></a>
<div content-id="dsy044-T2" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy044-T2" class="table-wrap-title" data-id="dsy044-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Gene prediction statistics for <em>R. mucilaginosa</em> JGTA-S1 and C2.5t1</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>
<em>R. mucilaginosa</em> JGTA-S1 </th>
<th>
<em>R. mucilaginosa</em> C2.5t1 </th>
</tr></thead>
<tbody>
<tr>
<td>No. of protein-coding genes </td>
<td>5,922 </td>
<td>6,463 </td>
</tr>
<tr>
<td>No. of predicted CDSs </td>
<td>42,105 </td>
<td>43,928 </td>
</tr>
<tr>
<td>No. of predicted introns </td>
<td>36,215 </td>
<td>37,981 </td>
</tr>
<tr>
<td>Average gene length </td>
<td>2,778 bp </td>
<td>2,453 bp </td>
</tr>
<tr>
<td>Average CDS length </td>
<td>275 bp </td>
<td>263 bp </td>
</tr>
<tr>
<td>Average intron length </td>
<td>134 bp </td>
<td>112 bp </td>
</tr>
<tr>
<td>Max gene length </td>
<td>25,521 bp </td>
<td>18,172 bp </td>
</tr>
<tr>
<td>Min gene length </td>
<td>208 bp </td>
<td>95 bp </td>
</tr>
<tr>
<td>Max intron length </td>
<td>7,824 bp </td>
<td>3,480 bp </td>
</tr>
<tr>
<td>Min intron length </td>
<td>42 bp </td>
<td>6 bp </td>
</tr>
<tr>
<td>Average number of intron/ protein-coding gene </td>
<td>6.12 </td>
<td>5.88 </td>
</tr>
<tr>
<td>Max CDS length </td>
<td>4,641 bp </td>
<td>4,509 bp </td>
</tr>
<tr>
<td>Min CDS length </td>
<td>3 bp </td>
<td>3 bp </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy044-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>
<em>R. mucilaginosa</em> JGTA-S1 </th>
<th>
<em>R. mucilaginosa</em> C2.5t1 </th>
</tr></thead>
<tbody>
<tr>
<td>No. of protein-coding genes </td>
<td>5,922 </td>
<td>6,463 </td>
</tr>
<tr>
<td>No. of predicted CDSs </td>
<td>42,105 </td>
<td>43,928 </td>
</tr>
<tr>
<td>No. of predicted introns </td>
<td>36,215 </td>
<td>37,981 </td>
</tr>
<tr>
<td>Average gene length </td>
<td>2,778 bp </td>
<td>2,453 bp </td>
</tr>
<tr>
<td>Average CDS length </td>
<td>275 bp </td>
<td>263 bp </td>
</tr>
<tr>
<td>Average intron length </td>
<td>134 bp </td>
<td>112 bp </td>
</tr>
<tr>
<td>Max gene length </td>
<td>25,521 bp </td>
<td>18,172 bp </td>
</tr>
<tr>
<td>Min gene length </td>
<td>208 bp </td>
<td>95 bp </td>
</tr>
<tr>
<td>Max intron length </td>
<td>7,824 bp </td>
<td>3,480 bp </td>
</tr>
<tr>
<td>Min intron length </td>
<td>42 bp </td>
<td>6 bp </td>
</tr>
<tr>
<td>Average number of intron/ protein-coding gene </td>
<td>6.12 </td>
<td>5.88 </td>
</tr>
<tr>
<td>Max CDS length </td>
<td>4,641 bp </td>
<td>4,509 bp </td>
</tr>
<tr>
<td>Min CDS length </td>
<td>3 bp </td>
<td>3 bp </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tf1"><div class="fn"><p>CDS: Coding sequence.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010295" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy044-T2" class="table-wrap-title" data-id="dsy044-T2">
<div class="label">Table 2</div>
<div class="caption"><p>Gene prediction statistics for <em>R. mucilaginosa</em> JGTA-S1 and C2.5t1</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>
<em>R. mucilaginosa</em> JGTA-S1 </th>
<th>
<em>R. mucilaginosa</em> C2.5t1 </th>
</tr></thead>
<tbody>
<tr>
<td>No. of protein-coding genes </td>
<td>5,922 </td>
<td>6,463 </td>
</tr>
<tr>
<td>No. of predicted CDSs </td>
<td>42,105 </td>
<td>43,928 </td>
</tr>
<tr>
<td>No. of predicted introns </td>
<td>36,215 </td>
<td>37,981 </td>
</tr>
<tr>
<td>Average gene length </td>
<td>2,778 bp </td>
<td>2,453 bp </td>
</tr>
<tr>
<td>Average CDS length </td>
<td>275 bp </td>
<td>263 bp </td>
</tr>
<tr>
<td>Average intron length </td>
<td>134 bp </td>
<td>112 bp </td>
</tr>
<tr>
<td>Max gene length </td>
<td>25,521 bp </td>
<td>18,172 bp </td>
</tr>
<tr>
<td>Min gene length </td>
<td>208 bp </td>
<td>95 bp </td>
</tr>
<tr>
<td>Max intron length </td>
<td>7,824 bp </td>
<td>3,480 bp </td>
</tr>
<tr>
<td>Min intron length </td>
<td>42 bp </td>
<td>6 bp </td>
</tr>
<tr>
<td>Average number of intron/ protein-coding gene </td>
<td>6.12 </td>
<td>5.88 </td>
</tr>
<tr>
<td>Max CDS length </td>
<td>4,641 bp </td>
<td>4,509 bp </td>
</tr>
<tr>
<td>Min CDS length </td>
<td>3 bp </td>
<td>3 bp </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy044-T2" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Features </th>
<th>
<em>R. mucilaginosa</em> JGTA-S1 </th>
<th>
<em>R. mucilaginosa</em> C2.5t1 </th>
</tr></thead>
<tbody>
<tr>
<td>No. of protein-coding genes </td>
<td>5,922 </td>
<td>6,463 </td>
</tr>
<tr>
<td>No. of predicted CDSs </td>
<td>42,105 </td>
<td>43,928 </td>
</tr>
<tr>
<td>No. of predicted introns </td>
<td>36,215 </td>
<td>37,981 </td>
</tr>
<tr>
<td>Average gene length </td>
<td>2,778 bp </td>
<td>2,453 bp </td>
</tr>
<tr>
<td>Average CDS length </td>
<td>275 bp </td>
<td>263 bp </td>
</tr>
<tr>
<td>Average intron length </td>
<td>134 bp </td>
<td>112 bp </td>
</tr>
<tr>
<td>Max gene length </td>
<td>25,521 bp </td>
<td>18,172 bp </td>
</tr>
<tr>
<td>Min gene length </td>
<td>208 bp </td>
<td>95 bp </td>
</tr>
<tr>
<td>Max intron length </td>
<td>7,824 bp </td>
<td>3,480 bp </td>
</tr>
<tr>
<td>Min intron length </td>
<td>42 bp </td>
<td>6 bp </td>
</tr>
<tr>
<td>Average number of intron/ protein-coding gene </td>
<td>6.12 </td>
<td>5.88 </td>
</tr>
<tr>
<td>Max CDS length </td>
<td>4,641 bp </td>
<td>4,509 bp </td>
</tr>
<tr>
<td>Min CDS length </td>
<td>3 bp </td>
<td>3 bp </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot"><div content-id="tf1"><div class="fn"><p>CDS: Coding sequence.</p></div></div></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010295" target="_blank">Open in new tab</a></div>
</div></div>
<p>The previously reported C2.5t1 genome had numerous scaffolds, so the C2.5t1 genome was scaffolded using an in-house assembler with the JGTA-S1 genome as a reference assembly (<a class="link link-uri" href="https://github.com/madhubioinfo/STLab-assembler" target="_blank">https://github.com/madhubioinfo/STLab-assembler</a>). This assembler aligns scaffolds to a reference genome using length cut-off of 1,500 bp (–maxgap = 500 –mincluster = 100) and &gt;97% identity to eliminate the poorly aligned region. The scaffolds were then ordered based on the reference assembly and the gaps were filled with Ns. Subsequently, the scaffolded assembly of C2.5t1 also rendered to 46 scaffolds making it easier to compare synteny (<a reveal-id="dsy044-F2" data-open="dsy044-F2" class="link link-reveal link-table xref-fig">Fig. 2c</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Forty-four pairs of scaffolds containing 5,359 gene pairs were syntenic between JGTA-S1 and C2.5t1. The two pairs of scaffolds that did not contain significant synteny, i.e. Scaffold 42 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4a</a></span>, Scaffold 42 on Y axis and corresponding scaffold of C2.5t1 on X axis) and Scaffold 46 of JGTA-S1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4b</a></span>, Scaffold 46 on Y axis and corresponding scaffold of C2.5t1 on X axis) were small scaffolds (40,000 and 8,000 bp in length) and were most likely caused by a small inversion and a deletion. Overall a one-to-one relationship was observed between pairs of scaffolds of the two genomes again confirming that JGTA-S1 and C2.5t1 were closely related and represented haploid genomes.</p>
<p>The genome of JGTA-S1 was compared with the whole genome sequences of four other <em>Rhodotorula</em> genomes and <em>R. toruloides</em> (<a reveal-id="dsy044-T3" data-open="dsy044-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <em>Rhodotorula graminis</em> WP1 was found to endophytically colonize poplar.<a reveal-id="dsy044-B5" data-open="dsy044-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><em>Rhodotorula mucilaginosa</em> strain C2.5t1 was isolated from cacao seeds.<a reveal-id="dsy044-B13" data-open="dsy044-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><em>R</em>. sp JG1b was isolated from permafrost in Antarctica.<a reveal-id="dsy044-B36" data-open="dsy044-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><em>Rhodosporidium toruloides</em> (previously <em>Rhodotorula glutinis</em>) was also found in association with the woody structure of plants.<a reveal-id="dsy044-B37" data-open="dsy044-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Thus <em>Rhodosporidium</em> and <em>Rhodotorula</em> yeasts appeared to have a tendency for plant-association although except for <em>R. graminis</em> strain WP1 none of the other strains have been experimentally demonstrated as endophytes neither have their life strategies been demonstrated in plants. The completeness of the assembly of JGTA-S1 was assessed with BUSCO and compared with the other genomes (<a reveal-id="dsy044-T3" data-open="dsy044-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Thus, the assembly of JGTA-S1 was comparable to the assemblies of the other genomes. Overall the <em>Rhodotorula</em> genomes and the <em>Rhodosporidium</em> were ∼20 MB in size. The genome size of this group was small when compared with other classes in order Pucciniomycotina (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). This reduction in genome size appears to be a feature common to class Microbotryomycetes. The relationship among the five <em>Rhodotorula</em> genomes was assessed with phylogeny and cluster analysis. A total of 34,483 proteins from the five organisms were clustered into 6,774 groups by OrthoMCL. Phylogeny was inferred from 3,660 single-copy orthologues and showed that JGTA-S1 and C2.5t1 were most closely related to the eurypsychrophilic organism <em>R</em>. sp. JG1b (<a reveal-id="dsy044-F3" data-open="dsy044-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). This was compared with a cluster analysis on the presence–absence of all orthologues (<a reveal-id="dsy044-F3" data-open="dsy044-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>). The two trees had the same topography indicating that the gene content of these organisms reflected their evolutionary history that was largely unaffected by factors such as horizontal gene transfer. Core orthologues defined as orthologues that were present in all five organisms consisted of 4,315 proteins (<a reveal-id="dsy044-F3" data-open="dsy044-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>) enriched in 88 GO IDs. These core proteins belonged to various biosynthetic pathways, DNA replication machinery, signalling pathways (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5a</a></span>) and were located in the mitochondria (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5b</a></span>) and functioned as helicases, transferases, and GTPases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5c</a></span>). Thus, the core proteome mostly had housekeeping functions. Also included in the core proteome were several plant hormones synthesis pathways, such as cytokinin, auxin, and lipid biosynthesis genes such as isocitrate dehydrogenase and malic enzyme (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>) (discussed below). Thus, all genomes appeared to have host-beneficial and lipid-producing capabilities. The <em>R. mucilaginosas</em>, JGTA-S1 and C2.5t1, and JG1b had smaller genomes and encoded mostly core proteins having very few unique orthologues, i.e. 72 in JGTA-S1, 380 and 406 in C2.5t1 and JG1b respectively. Most of these 72 JGTA-S1 proteins had unknown functions or shared no similarity with any known proteins. Among the proteins having annotations, gag-pol polyproteins that are part of retro elements were a major class (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).</p>                        <a id="134010298" scrollto-destination="134010298"></a>
<div id="dsy044-F3" data-id="dsy044-F3" class="fig fig-section">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f3.png?Expires=1576462815&amp;Signature=HCgS9QYdFyIr~7W1z5LynnjfMpjC4rl6aNoFLoGu-m9RmtH2DgCye6w2Ci3A1nHPxu6Kur31MIrzbm0j~wpghk17nf5NJz95yY08oBB7oKFs42LGrrUCXDgzEFWAS5fx49EYsFnVtcVI337xvMOiV2UrDchsZnbQT16Z5oQOroX25VXGCXakU0GGmdA4KQGxPI-QDnQkblDO3ySDhNmqZZTcSpVxa63bldiLWKXS6DQgjlN0mzXqRUYVcCOF7yeh62OiXyr~wrqiehoZ~DHLxOB1LGa7iPrx9qRYoffE9-RoMTNzwQwB4nCYltKwtnopo6rDtin6LvNyOLdH2aiw1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relatedness of R. mucilaginosa with other closely related red yeasts. Rhodotorula mucilaginosa JGTA-S1 (JGTA-S1) genome was compared with C2.5t1 (R. mucilaginosa C2.5t1), WP1 (R. graminis WP1), JG1b (R. sp JG1b) and NP11 (R. toruloides NP11). (a) Maximum likelihood tree based on the concatenated alignment of 3,660 single-copy orthologous groups. OrthoMCL26 was used to infer orthologues among the Rhodotorula genomes. Proteins of each orthologous group were aligned with ClustalW230 and concatenated into a superalignment and used for phylogeny prediction. (b) Hierarchical clustering based on the presence or absence of orthologous gene families. A binary matrix consisting of the presence or absence of orthologous gene families was converted into a distance matrix and subjected to hierarchical clustering. The heatmap shows presence (pink) and absence (blue) of orthologues. (c) Shared and unique orthologue gene families among the five genomes are shown in the Venn diagram. Color figures are available at DNARES online." path-from-xml="dsy044f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010298/dsy044f3.tif" path-from-xml="dsy044f3.tif" target="_blank">Open in new tab</a><a section="134010298" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f3.png?Expires=1576462815&amp;Signature=nBZZ4Mv3QCjY7ywctKK6ObbhGMYq5s7tweURrx5eThXRTwh0og~tJ0taHC~FwZ3moUSqAxGJEZYzEcWzfUSwdb~Ls9Yh00zsmS84ta43IqbgZx~3n2uJ1bxeJG7OJxz-jri0dWBZ71sk4uDgCP75qYsodjQfBGVeW0smjFHhiWEjU8Dn6oYGxAfwksce6ie9A~Lc3SdZ3J30BAqKZMaiKy76jb2H5q6~110i4akR1GmQZcXf1w91bN1-pOfgRCDm7hIlfejUXL5erpW1eI6Plw82kqN-5SpsPan6Sm5GYlNB19thqqDlOr7qoqgwLwIm5E1hrRq4e3C4akZuzDeCqg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010298&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Relatedness of <em>R. mucilaginosa</em> with other closely related red yeasts. <em>Rhodotorula mucilaginosa</em> JGTA-S1 (JGTA-S1) genome was compared with C2.5t1 (<em>R. mucilaginosa</em> C2.5t1), WP1 (<em>R. graminis</em> WP1), JG1b (<em>R.</em> sp JG1b) and NP11 (<em>R. toruloides</em> NP11). (a) Maximum likelihood tree based on the concatenated alignment of 3,660 single-copy orthologous groups. OrthoMCL<a reveal-id="dsy044-B26" data-open="dsy044-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was used to infer orthologues among the <em>Rhodotorula</em> genomes. Proteins of each orthologous group were aligned with ClustalW2<a reveal-id="dsy044-B30" data-open="dsy044-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and concatenated into a superalignment and used for phylogeny prediction. (b) Hierarchical clustering based on the presence or absence of orthologous gene families. A binary matrix consisting of the presence or absence of orthologous gene families was converted into a distance matrix and subjected to hierarchical clustering. The heatmap shows presence (pink) and absence (blue) of orthologues. (c) Shared and unique orthologue gene families among the five genomes are shown in the Venn diagram. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f3.png?Expires=1576462815&amp;Signature=HCgS9QYdFyIr~7W1z5LynnjfMpjC4rl6aNoFLoGu-m9RmtH2DgCye6w2Ci3A1nHPxu6Kur31MIrzbm0j~wpghk17nf5NJz95yY08oBB7oKFs42LGrrUCXDgzEFWAS5fx49EYsFnVtcVI337xvMOiV2UrDchsZnbQT16Z5oQOroX25VXGCXakU0GGmdA4KQGxPI-QDnQkblDO3ySDhNmqZZTcSpVxa63bldiLWKXS6DQgjlN0mzXqRUYVcCOF7yeh62OiXyr~wrqiehoZ~DHLxOB1LGa7iPrx9qRYoffE9-RoMTNzwQwB4nCYltKwtnopo6rDtin6LvNyOLdH2aiw1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relatedness of R. mucilaginosa with other closely related red yeasts. Rhodotorula mucilaginosa JGTA-S1 (JGTA-S1) genome was compared with C2.5t1 (R. mucilaginosa C2.5t1), WP1 (R. graminis WP1), JG1b (R. sp JG1b) and NP11 (R. toruloides NP11). (a) Maximum likelihood tree based on the concatenated alignment of 3,660 single-copy orthologous groups. OrthoMCL26 was used to infer orthologues among the Rhodotorula genomes. Proteins of each orthologous group were aligned with ClustalW230 and concatenated into a superalignment and used for phylogeny prediction. (b) Hierarchical clustering based on the presence or absence of orthologous gene families. A binary matrix consisting of the presence or absence of orthologous gene families was converted into a distance matrix and subjected to hierarchical clustering. The heatmap shows presence (pink) and absence (blue) of orthologues. (c) Shared and unique orthologue gene families among the five genomes are shown in the Venn diagram. Color figures are available at DNARES online." path-from-xml="dsy044f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010298/dsy044f3.tif" path-from-xml="dsy044f3.tif" target="_blank">Open in new tab</a><a section="134010298" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f3.png?Expires=1576462815&amp;Signature=nBZZ4Mv3QCjY7ywctKK6ObbhGMYq5s7tweURrx5eThXRTwh0og~tJ0taHC~FwZ3moUSqAxGJEZYzEcWzfUSwdb~Ls9Yh00zsmS84ta43IqbgZx~3n2uJ1bxeJG7OJxz-jri0dWBZ71sk4uDgCP75qYsodjQfBGVeW0smjFHhiWEjU8Dn6oYGxAfwksce6ie9A~Lc3SdZ3J30BAqKZMaiKy76jb2H5q6~110i4akR1GmQZcXf1w91bN1-pOfgRCDm7hIlfejUXL5erpW1eI6Plw82kqN-5SpsPan6Sm5GYlNB19thqqDlOr7qoqgwLwIm5E1hrRq4e3C4akZuzDeCqg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010298&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Relatedness of <em>R. mucilaginosa</em> with other closely related red yeasts. <em>Rhodotorula mucilaginosa</em> JGTA-S1 (JGTA-S1) genome was compared with C2.5t1 (<em>R. mucilaginosa</em> C2.5t1), WP1 (<em>R. graminis</em> WP1), JG1b (<em>R.</em> sp JG1b) and NP11 (<em>R. toruloides</em> NP11). (a) Maximum likelihood tree based on the concatenated alignment of 3,660 single-copy orthologous groups. OrthoMCL<a reveal-id="dsy044-B26" data-open="dsy044-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was used to infer orthologues among the <em>Rhodotorula</em> genomes. Proteins of each orthologous group were aligned with ClustalW2<a reveal-id="dsy044-B30" data-open="dsy044-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and concatenated into a superalignment and used for phylogeny prediction. (b) Hierarchical clustering based on the presence or absence of orthologous gene families. A binary matrix consisting of the presence or absence of orthologous gene families was converted into a distance matrix and subjected to hierarchical clustering. The heatmap shows presence (pink) and absence (blue) of orthologues. (c) Shared and unique orthologue gene families among the five genomes are shown in the Venn diagram. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                        <a id="134010299" scrollto-destination="134010299"></a>
<div content-id="dsy044-T3" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy044-T3" class="table-wrap-title" data-id="dsy044-T3">
<div class="label">Table 3</div>
<div class="caption"><p>Genomes used in the comparative study and their genome statistics</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Genomes </th>
<th>Source </th>
<th>Size of genome (MB) </th>
<th>N50 value of the assembly </th>
<th>Complete BUSCOs </th>
<th>Fragmented BUSCOs </th>
<th>Missing BUSCOs </th>
<th>Number of protein- coding genes </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>R. mucilaginosa</em> JGTA- S1 </td>
<td>Stem of <em>T. angustifolia</em>, India </td>
<td>20.11 </td>
<td>685,833 </td>
<td>269 </td>
<td>10 </td>
<td>11 </td>
<td>5,922 </td>
</tr>
<tr>
<td>
<em>R. mucilaginosa</em> C2.5t1 </td>
<td>Seeds of <em>Theobroma cacao</em>, Cameroon </td>
<td>19.98 </td>
<td>45,031 </td>
<td>255 </td>
<td>25 </td>
<td>10 </td>
<td>6,463 </td>
</tr>
<tr>
<td>
<em>R. sp</em> JGIb </td>
<td>Permafrost, Antarctica </td>
<td>19.39 </td>
<td>301,937 </td>
<td>274 </td>
<td>4 </td>
<td>12 </td>
<td>6,681 </td>
</tr>
<tr>
<td>
<em>R. graminis</em> WP1 </td>
<td>Stem of <em>Populus trichocarpa U.S.A</em> </td>
<td>21.01 </td>
<td>1,420,730 </td>
<td>271 </td>
<td>4 </td>
<td>15 </td>
<td>7,278 </td>
</tr>
<tr>
<td>
<em>R. toruloides</em> NP11 </td>
<td>Unknown </td>
<td>20.22 </td>
<td>574,942 </td>
<td>275 </td>
<td>4 </td>
<td>11 </td>
<td>8,140 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy044-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Genomes </th>
<th>Source </th>
<th>Size of genome (MB) </th>
<th>N50 value of the assembly </th>
<th>Complete BUSCOs </th>
<th>Fragmented BUSCOs </th>
<th>Missing BUSCOs </th>
<th>Number of protein- coding genes </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>R. mucilaginosa</em> JGTA- S1 </td>
<td>Stem of <em>T. angustifolia</em>, India </td>
<td>20.11 </td>
<td>685,833 </td>
<td>269 </td>
<td>10 </td>
<td>11 </td>
<td>5,922 </td>
</tr>
<tr>
<td>
<em>R. mucilaginosa</em> C2.5t1 </td>
<td>Seeds of <em>Theobroma cacao</em>, Cameroon </td>
<td>19.98 </td>
<td>45,031 </td>
<td>255 </td>
<td>25 </td>
<td>10 </td>
<td>6,463 </td>
</tr>
<tr>
<td>
<em>R. sp</em> JGIb </td>
<td>Permafrost, Antarctica </td>
<td>19.39 </td>
<td>301,937 </td>
<td>274 </td>
<td>4 </td>
<td>12 </td>
<td>6,681 </td>
</tr>
<tr>
<td>
<em>R. graminis</em> WP1 </td>
<td>Stem of <em>Populus trichocarpa U.S.A</em> </td>
<td>21.01 </td>
<td>1,420,730 </td>
<td>271 </td>
<td>4 </td>
<td>15 </td>
<td>7,278 </td>
</tr>
<tr>
<td>
<em>R. toruloides</em> NP11 </td>
<td>Unknown </td>
<td>20.22 </td>
<td>574,942 </td>
<td>275 </td>
<td>4 </td>
<td>11 </td>
<td>8,140 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010299" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy044-T3" class="table-wrap-title" data-id="dsy044-T3">
<div class="label">Table 3</div>
<div class="caption"><p>Genomes used in the comparative study and their genome statistics</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align=""><tr>
<th>Genomes </th>
<th>Source </th>
<th>Size of genome (MB) </th>
<th>N50 value of the assembly </th>
<th>Complete BUSCOs </th>
<th>Fragmented BUSCOs </th>
<th>Missing BUSCOs </th>
<th>Number of protein- coding genes </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>R. mucilaginosa</em> JGTA- S1 </td>
<td>Stem of <em>T. angustifolia</em>, India </td>
<td>20.11 </td>
<td>685,833 </td>
<td>269 </td>
<td>10 </td>
<td>11 </td>
<td>5,922 </td>
</tr>
<tr>
<td>
<em>R. mucilaginosa</em> C2.5t1 </td>
<td>Seeds of <em>Theobroma cacao</em>, Cameroon </td>
<td>19.98 </td>
<td>45,031 </td>
<td>255 </td>
<td>25 </td>
<td>10 </td>
<td>6,463 </td>
</tr>
<tr>
<td>
<em>R. sp</em> JGIb </td>
<td>Permafrost, Antarctica </td>
<td>19.39 </td>
<td>301,937 </td>
<td>274 </td>
<td>4 </td>
<td>12 </td>
<td>6,681 </td>
</tr>
<tr>
<td>
<em>R. graminis</em> WP1 </td>
<td>Stem of <em>Populus trichocarpa U.S.A</em> </td>
<td>21.01 </td>
<td>1,420,730 </td>
<td>271 </td>
<td>4 </td>
<td>15 </td>
<td>7,278 </td>
</tr>
<tr>
<td>
<em>R. toruloides</em> NP11 </td>
<td>Unknown </td>
<td>20.22 </td>
<td>574,942 </td>
<td>275 </td>
<td>4 </td>
<td>11 </td>
<td>8,140 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy044-T3" class="table-modal"><table>
<thead name="thead" align=""><tr>
<th>Genomes </th>
<th>Source </th>
<th>Size of genome (MB) </th>
<th>N50 value of the assembly </th>
<th>Complete BUSCOs </th>
<th>Fragmented BUSCOs </th>
<th>Missing BUSCOs </th>
<th>Number of protein- coding genes </th>
</tr></thead>
<tbody>
<tr>
<td>
<em>R. mucilaginosa</em> JGTA- S1 </td>
<td>Stem of <em>T. angustifolia</em>, India </td>
<td>20.11 </td>
<td>685,833 </td>
<td>269 </td>
<td>10 </td>
<td>11 </td>
<td>5,922 </td>
</tr>
<tr>
<td>
<em>R. mucilaginosa</em> C2.5t1 </td>
<td>Seeds of <em>Theobroma cacao</em>, Cameroon </td>
<td>19.98 </td>
<td>45,031 </td>
<td>255 </td>
<td>25 </td>
<td>10 </td>
<td>6,463 </td>
</tr>
<tr>
<td>
<em>R. sp</em> JGIb </td>
<td>Permafrost, Antarctica </td>
<td>19.39 </td>
<td>301,937 </td>
<td>274 </td>
<td>4 </td>
<td>12 </td>
<td>6,681 </td>
</tr>
<tr>
<td>
<em>R. graminis</em> WP1 </td>
<td>Stem of <em>Populus trichocarpa U.S.A</em> </td>
<td>21.01 </td>
<td>1,420,730 </td>
<td>271 </td>
<td>4 </td>
<td>15 </td>
<td>7,278 </td>
</tr>
<tr>
<td>
<em>R. toruloides</em> NP11 </td>
<td>Unknown </td>
<td>20.22 </td>
<td>574,942 </td>
<td>275 </td>
<td>4 </td>
<td>11 </td>
<td>8,140 </td>
</tr>
</tbody>
</table></div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134010299" target="_blank">Open in new tab</a></div>
</div></div>
<p>Only 2% of these genomes were occupied by repeat elements (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). A higher number of long terminal repeat (LTR) elements (Copia, Gypsy) were found in the genomes of JGTA-S1 (57), C2.5t1 (97), and JG1b (73) in comparison to NP11 (34) and WP1 (26) while a large number of LINES were found in NP11. Overall the LTR repeat content of these genomes was low like that of the corn smut pathogen, <em>U. maydis.</em><a reveal-id="dsy044-B38" data-open="dsy044-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p>                    <h3 scrollto-destination="134010301" id="134010301" class="section-title">3.2. Comparison with other endophytes and pathogens</h3>
<p>To determine what genes were common between JGTA-S1 and other endophytes an orthologue analysis was carried out with known endophytes from Basidiomycota. <em>Grammothele lineata</em>—an endophyte that colonizes jute,<a reveal-id="dsy044-B39" data-open="dsy044-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><em>S. indica</em> (previously <em>Piriformospora indica</em>)—a root-colonizing endophyte<a reveal-id="dsy044-B40" data-open="dsy044-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and <em>R. graminis</em> WP1<a reveal-id="dsy044-B41" data-open="dsy044-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a>—a poplar endophyte were used in the study. In contrast, pathogens such as the corn smut biotroph, <em>U. maydis</em>,<a reveal-id="dsy044-B42" data-open="dsy044-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> wheat stem rust biotroph, <em>P. graminis</em>,<a reveal-id="dsy044-B43" data-open="dsy044-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> the poplar rust biotroph<em>, M. larici-populina</em>,<a reveal-id="dsy044-B44" data-open="dsy044-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> and two closely related biotrophs, <em>M. intermedium,</em> and <em>M. lychnidis-dioicae</em><a reveal-id="dsy044-B45" data-open="dsy044-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> were included. The Venn diagram shows that 2,996 orthologue groups were shared by all 9 species but only 25 groups were shared exclusively by <em>Rhodotorula</em> (JGTA-S1 and WP1) and the endophytes (<em>G. lineata</em> and <em>S. indica</em>) (red highlight in <a reveal-id="dsy044-F4" data-open="dsy044-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). These 25 endophyte-specific groups included proteins such as NmrA-domain-containing proteins (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). JGTA-S1 encoded 6 such NmrA proteins. NmrA is a transcriptional regulator involved in regulating N-metabolism when different N-sources are available.<a reveal-id="dsy044-B46" data-open="dsy044-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> In addition, <em>Rhodotorula</em> and other members of Pucciniomycotina lacked a gene cluster homologous to 19 A coding for secreted effectors in <em>U. maydis</em><a reveal-id="dsy044-B47" data-open="dsy044-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> and some other Ustilaginomycotina. The deletion of the 19 A cluster genes in <em>U. maydis</em> led the mutant to behave like an endophyte. However, in absence of sufficient homology in the region flanking the 19 A cluster of <em>U. maydis</em> with <em>Rhodotorula</em> sp. it is difficult to correlate the lack of 19 A cluster with the endophytic lifestyle of <em>Rhodotorula</em>. On the other hand, <em>Rhodotorula</em> shared a high number of orthologues, i.e. 528 with the pathogens <em>M. intermedium</em> and <em>M. lychnidis-dioicae</em> both of which belong to the same class as <em>Rhodotorula</em>, i.e. Microbotryomycetes. A predominant class of genes common to the Microbotryomycetes was the proteophosphoglycans. These are glycoproteins that in the intercellular human pathogen, <em>Leishmania donovani</em> have been shown to help in attaching to host cell during infection.<a reveal-id="dsy044-B48" data-open="dsy044-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> This group of proteins appeared to have undergone expansion in <em>Rhodotorula</em>. JGTA-S1 had 85 copies of proteophosphoglycan ppg4 and 18 copies of proteophosphoglycan ppg5. A search for ‘proteophosphoglycan AND basidiomycetes’ ‘[porgn:__txid5204]’ in the NR database resulted in 1887 hits (search done on 10.05.18). Of those, 81% were hits to <em>Rhodotorula</em>. The role of proteophosphoglycan in fungi is unclear but they may be involved in host cell invasion. </p>                        <a id="134010303" scrollto-destination="134010303"></a>
<div id="dsy044-F4" data-id="dsy044-F4" class="fig fig-section">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f4.png?Expires=1576462815&amp;Signature=W0Dnf7nMHgmUVIrdIBPUvQBeyhtq82O49e6sDX-tr~Zlwu30ItNbxSFLhU4sJojgwq1oK59DVHsWPgEdv4jY3QambONiNKUYcsX0PlWKIriqqPb11TjeR5EeUOUPIzIbteynaFCEExNuIrig~OgZP9Npfb3RJ3mQMyQ-oLYAoCmcp9YpmwMuNSHacy5FaLKMPKJkWw9N2TVA6ovvc7983Rtq592EZQH-AcHETm2eD2J4YJHPYYmMcKeFH4B58uCIvnSx8R5xFpglQahzbF07pM3vyDhZbg7tUQTbkNE2Owh0OUas5HF-4lPJ5gKx18CTJRca0nLgjnnMnVtdlvep5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of orthologues of R. mucilaginosa JGTA-S1 with other Basidiomycetous fungi. OrthoMCL26 was used to infer orthologues for R. mucilaginosa JGTA-S1, Umay (U. maydis), Min (M. intermedium), Mld (M. lychnidis-dioicae), Gram (G. lineata), JGTA-S1 (R. mucilaginosa JGTA-S1), WP1 (R. graminis WP1), Sin (S. indica), Mela (M. larici-populina), and Pucc (P. graminis). Shared and unique orthologue families are shown in the Venn diagram. To make visualization simple, closely-related genomes have been grouped together. Color figures are available at DNARES online." path-from-xml="dsy044f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010303/dsy044f4.tif" path-from-xml="dsy044f4.tif" target="_blank">Open in new tab</a><a section="134010303" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f4.png?Expires=1576462815&amp;Signature=HTzIAk8En6Pm9dNCHS1V5cMu13uZaVwzvhIBPf5UTrJoCPkBzQcg8LcYgHdS0oMTImPFvXjFVtqBaCTCVb-iPj8Lp7Ym88xs2JU1qNXo2YYHswRVIb2TDYEPCQach636NjPhnFlNqQGeZFTnSP9spz5Fxm3iPtFzzVr4Be99MHOiaTNVNr7qK4U49rcW7qKFKagCYQfWcdaPjTEOBRh0hu2RYkySJaBmiGTNGvFNXmOko4daLbgEFV1ISxvDwz3SXg~uC4ZLelhE-fNrwBkq1xSXtkcjPc1p0faGwcruaH819uJFkSXNIgGGL7p05~saUjgM66t6~QOG74pY8JdjGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010303&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of orthologues of <em>R. mucilaginosa</em> JGTA-S1 with other Basidiomycetous fungi. OrthoMCL<a reveal-id="dsy044-B26" data-open="dsy044-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was used to infer orthologues for <em>R. mucilaginosa</em> JGTA-S1, Umay (<em>U. maydis</em>), Min (<em>M. intermedium</em>), Mld (<em>M. lychnidis-dioicae</em>), Gram (<em>G. lineata</em>), JGTA-S1 (<em>R. mucilaginosa</em> JGTA-S1), WP1 (<em>R. graminis</em> WP1), Sin (<em>S. indica</em>), Mela (<em>M. larici-populina),</em> and Pucc (<em>P. graminis</em>). Shared and unique orthologue families are shown in the Venn diagram. To make visualization simple, closely-related genomes have been grouped together. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F4" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 4</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f4.png?Expires=1576462815&amp;Signature=W0Dnf7nMHgmUVIrdIBPUvQBeyhtq82O49e6sDX-tr~Zlwu30ItNbxSFLhU4sJojgwq1oK59DVHsWPgEdv4jY3QambONiNKUYcsX0PlWKIriqqPb11TjeR5EeUOUPIzIbteynaFCEExNuIrig~OgZP9Npfb3RJ3mQMyQ-oLYAoCmcp9YpmwMuNSHacy5FaLKMPKJkWw9N2TVA6ovvc7983Rtq592EZQH-AcHETm2eD2J4YJHPYYmMcKeFH4B58uCIvnSx8R5xFpglQahzbF07pM3vyDhZbg7tUQTbkNE2Owh0OUas5HF-4lPJ5gKx18CTJRca0nLgjnnMnVtdlvep5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of orthologues of R. mucilaginosa JGTA-S1 with other Basidiomycetous fungi. OrthoMCL26 was used to infer orthologues for R. mucilaginosa JGTA-S1, Umay (U. maydis), Min (M. intermedium), Mld (M. lychnidis-dioicae), Gram (G. lineata), JGTA-S1 (R. mucilaginosa JGTA-S1), WP1 (R. graminis WP1), Sin (S. indica), Mela (M. larici-populina), and Pucc (P. graminis). Shared and unique orthologue families are shown in the Venn diagram. To make visualization simple, closely-related genomes have been grouped together. Color figures are available at DNARES online." path-from-xml="dsy044f4.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010303/dsy044f4.tif" path-from-xml="dsy044f4.tif" target="_blank">Open in new tab</a><a section="134010303" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f4.png?Expires=1576462815&amp;Signature=HTzIAk8En6Pm9dNCHS1V5cMu13uZaVwzvhIBPf5UTrJoCPkBzQcg8LcYgHdS0oMTImPFvXjFVtqBaCTCVb-iPj8Lp7Ym88xs2JU1qNXo2YYHswRVIb2TDYEPCQach636NjPhnFlNqQGeZFTnSP9spz5Fxm3iPtFzzVr4Be99MHOiaTNVNr7qK4U49rcW7qKFKagCYQfWcdaPjTEOBRh0hu2RYkySJaBmiGTNGvFNXmOko4daLbgEFV1ISxvDwz3SXg~uC4ZLelhE-fNrwBkq1xSXtkcjPc1p0faGwcruaH819uJFkSXNIgGGL7p05~saUjgM66t6~QOG74pY8JdjGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010303&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f4.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Comparison of orthologues of <em>R. mucilaginosa</em> JGTA-S1 with other Basidiomycetous fungi. OrthoMCL<a reveal-id="dsy044-B26" data-open="dsy044-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was used to infer orthologues for <em>R. mucilaginosa</em> JGTA-S1, Umay (<em>U. maydis</em>), Min (<em>M. intermedium</em>), Mld (<em>M. lychnidis-dioicae</em>), Gram (<em>G. lineata</em>), JGTA-S1 (<em>R. mucilaginosa</em> JGTA-S1), WP1 (<em>R. graminis</em> WP1), Sin (<em>S. indica</em>), Mela (<em>M. larici-populina),</em> and Pucc (<em>P. graminis</em>). Shared and unique orthologue families are shown in the Venn diagram. To make visualization simple, closely-related genomes have been grouped together. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                    <h3 scrollto-destination="134010304" id="134010304" class="section-title">3.3. <em>Rhodotorula</em> yeasts have genes for cold adaptation</h3>
<p>Many genera within the class Microbotryomycetes can tolerate cold. These include <em>Rhodotorula</em>, <em>Rhodosporidium</em>, <em>Leucosporidium</em>, <em>Microbotryum,</em> and <em>Sporobolomyces.</em><a reveal-id="dsy044-B2" data-open="dsy044-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Since JGTA-S1 genome showed strong relatedness to the eurypsychrophilic <em>Rhodotorula</em> sp. JG1b that was isolated from the McMurdo Dry Valleys of Antarctica,<a reveal-id="dsy044-B34" data-open="dsy044-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> our genomes were searched for the presence of cold tolerance genes. All five <em>Rhodotorula</em> genomes were found to contain multiple copies of genes encoding putative anti-freeze glycoproteins. These are g574.t1, g736.t1, g5846.t1, g5847.t1, g5875.t1, and g5876.t1 in JGTA-S1 and their orthologues in other genomes are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>. These proteins found in cold-adapted organisms are known to prevent ice crystal formation enabling growth at low temperatures.<a reveal-id="dsy044-B49" data-open="dsy044-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The ability of <em>R. mucilaginosa</em> JGTA-S1 to withstand low temperature was tested by growing it at on YPD plates at 5°C. JGTA-S1 showed much better growth at 5°C in comparison to <em>S. cerevisiae</em> strain ThyAP4 4-days post-spotting (<a reveal-id="dsy044-F5" data-open="dsy044-F5" class="link link-reveal link-table xref-fig">Fig. 5a</a>). In accordance with this data, two randomly chosen anti-freezing genes g5846.t1 and g5876.t1 were found to be upregulated 6.6- and 1.9-fold, respectively in a Real-time PCR experiment when JGTA-S1 was grown at 5°C compared with control cells grown at 30°C (<a reveal-id="dsy044-F5" data-open="dsy044-F5" class="link link-reveal link-table xref-fig">Fig. 5b</a>). This result showed that JGTA-S1 was cold-tolerant and the two anti-freezing genes studied were functional. Since the cold-tolerant genes were found both in JGTA-S1 as well as in JG1b, the genes could have similar functions in the other red yeasts as well. </p>                        <a id="134010306" scrollto-destination="134010306"></a>
<div id="dsy044-F5" data-id="dsy044-F5" class="fig fig-section">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f5.png?Expires=1576462815&amp;Signature=a7ZrYwH7kStuyGUnA-FCiFixnch7DnLuKfFLfpelyaicpAzYD61PG-GCXsZ7AZAKJHSj-DIeLKcHWkSGppeulUiyZJho~XXi4iQlYIq8DXetpUI4dHJ6UhmvDVj6UCfSheJJcCykCFz8F9sOsfnZieflP4qad-iiOqv~YxJjmGka1g2mPM4Mov9ZzNy4n8~0xwbCQQ7Ty~pvUBIzQJxPA2aIFCgJp6vX8Mr95amWbbrOnmxMeAtWfX-OnO3~MEPtaLKts1XvszwkDVqtCfj1rCn53EM5DnIGGxRUF-OZ2eq9SqKMKCFYn8~Lur8mrqj9nQ81DO6cbSwEeNxxOGID-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cold tolerance of R. mucilaginosa JGTA-S1. JGTA-S1 and S. cerevisiae ThyAP4 cells were grown in YPD medium till saturation and adjusted to an O.D. of 1. The cells were serial diluted and spotted on YPD plates and incubated at 30 and 5°C for 4days. For RNA isolation JGTA-S1 cells were grown at 30°C for 4 days followed by 4 days at 5°C. Control cells were frozen after 4 days of growth at 30°C. Real-time PCR was done with two putative anti-freezing genes. Results are from two independent experiments done in triplicate. (a) JGTA-S1 and S. cerevisiae ThyAP4 cells spotted on YPD (b) Expression of g5846 and g5876 compared with JGTA-S1 GAPDH as housekeeping gene." path-from-xml="dsy044f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010306/dsy044f5.tif" path-from-xml="dsy044f5.tif" target="_blank">Open in new tab</a><a section="134010306" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f5.png?Expires=1576462815&amp;Signature=4J-6cJIFLCGChdcRsL4Jex68UdkxJAWL0ZgcmNB6jI1D3UW8U7FXoL6n3NLjJwkpyV0EA2oE0cxeGSwuz8HEHt0Yn1P-JqjiWwN7RhQ1vsqNhv5k6NvGkV3pZQvSSL8Fdi2okeF8LnXSNiz~cXOR-OLxFeQ45eCfJLkvLRpD3GHzIm-bzPG5BOo2oPsEXlg8Saa0~RedTHI~hVagEq1wQjeSWq-IANsBPHjTWlI5la88Wz4N6ZHSQ2pkNJNB9FaCJvYh~IcVW1s1zD-ptcN6hv-XbsfjdgeeMI2JuIUqqnGao9Nv-~8K5K1hATt1xl3mjX7fSmMwu4sM0~vMSa3nqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010306&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Cold tolerance of <em>R. mucilaginosa</em> JGTA-S1. JGTA-S1 and <em>S. cerevisiae</em> ThyAP4 cells were grown in YPD medium till saturation and adjusted to an O.D. of 1. The cells were serial diluted and spotted on YPD plates and incubated at 30 and 5°C for 4days. For RNA isolation JGTA-S1 cells were grown at 30°C for 4 days followed by 4 days at 5°C. Control cells were frozen after 4 days of growth at 30°C. Real-time PCR was done with two putative anti-freezing genes. Results are from two independent experiments done in triplicate. (a) JGTA-S1 and <em>S. cerevisiae</em> ThyAP4 cells spotted on YPD (b) Expression of g5846 and g5876 compared with JGTA-S1 GAPDH as housekeeping gene.</p></div>
</div>
<div content-id="dsy044-F5" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 5</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f5.png?Expires=1576462815&amp;Signature=a7ZrYwH7kStuyGUnA-FCiFixnch7DnLuKfFLfpelyaicpAzYD61PG-GCXsZ7AZAKJHSj-DIeLKcHWkSGppeulUiyZJho~XXi4iQlYIq8DXetpUI4dHJ6UhmvDVj6UCfSheJJcCykCFz8F9sOsfnZieflP4qad-iiOqv~YxJjmGka1g2mPM4Mov9ZzNy4n8~0xwbCQQ7Ty~pvUBIzQJxPA2aIFCgJp6vX8Mr95amWbbrOnmxMeAtWfX-OnO3~MEPtaLKts1XvszwkDVqtCfj1rCn53EM5DnIGGxRUF-OZ2eq9SqKMKCFYn8~Lur8mrqj9nQ81DO6cbSwEeNxxOGID-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cold tolerance of R. mucilaginosa JGTA-S1. JGTA-S1 and S. cerevisiae ThyAP4 cells were grown in YPD medium till saturation and adjusted to an O.D. of 1. The cells were serial diluted and spotted on YPD plates and incubated at 30 and 5°C for 4days. For RNA isolation JGTA-S1 cells were grown at 30°C for 4 days followed by 4 days at 5°C. Control cells were frozen after 4 days of growth at 30°C. Real-time PCR was done with two putative anti-freezing genes. Results are from two independent experiments done in triplicate. (a) JGTA-S1 and S. cerevisiae ThyAP4 cells spotted on YPD (b) Expression of g5846 and g5876 compared with JGTA-S1 GAPDH as housekeeping gene." path-from-xml="dsy044f5.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010306/dsy044f5.tif" path-from-xml="dsy044f5.tif" target="_blank">Open in new tab</a><a section="134010306" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f5.png?Expires=1576462815&amp;Signature=4J-6cJIFLCGChdcRsL4Jex68UdkxJAWL0ZgcmNB6jI1D3UW8U7FXoL6n3NLjJwkpyV0EA2oE0cxeGSwuz8HEHt0Yn1P-JqjiWwN7RhQ1vsqNhv5k6NvGkV3pZQvSSL8Fdi2okeF8LnXSNiz~cXOR-OLxFeQ45eCfJLkvLRpD3GHzIm-bzPG5BOo2oPsEXlg8Saa0~RedTHI~hVagEq1wQjeSWq-IANsBPHjTWlI5la88Wz4N6ZHSQ2pkNJNB9FaCJvYh~IcVW1s1zD-ptcN6hv-XbsfjdgeeMI2JuIUqqnGao9Nv-~8K5K1hATt1xl3mjX7fSmMwu4sM0~vMSa3nqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010306&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f5.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Cold tolerance of <em>R. mucilaginosa</em> JGTA-S1. JGTA-S1 and <em>S. cerevisiae</em> ThyAP4 cells were grown in YPD medium till saturation and adjusted to an O.D. of 1. The cells were serial diluted and spotted on YPD plates and incubated at 30 and 5°C for 4days. For RNA isolation JGTA-S1 cells were grown at 30°C for 4 days followed by 4 days at 5°C. Control cells were frozen after 4 days of growth at 30°C. Real-time PCR was done with two putative anti-freezing genes. Results are from two independent experiments done in triplicate. (a) JGTA-S1 and <em>S. cerevisiae</em> ThyAP4 cells spotted on YPD (b) Expression of g5846 and g5876 compared with JGTA-S1 GAPDH as housekeeping gene.</p></div>
</div>                    <h3 scrollto-destination="134010307" id="134010307" class="section-title">3.4. <em>Rhodotorula</em> has a reduced set of CAZY enzymes</h3>
<p>Overall, JGTA-S1 had only 271 CAZY enzymes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). In order to look for CAZY enzymes that are specific for endophytic function, we compared CAZY families in JGTA-S1 with those of <em>G. lineata</em>, <em>S. indica</em> and <em>R. graminis</em> WP1, <em>U. maydis</em>, <em>P. graminis, M. larici-populina, M. intermedium,</em> and <em>M. lychnidis-dioicae</em>. Interestingly, clustering based on copy numbers of CAZY genes showed that the <em>Rhodotorulas</em> (JGTA-S1 and WP1) were more similar in CAZY profile to <em>M. intermedium</em> and <em>M. lychnidis-dioicae</em> than to endophytes <em>S. indica</em> and <em>G. lineata</em> (<a reveal-id="dsy044-F6" data-open="dsy044-F6" class="link link-reveal link-table xref-fig">Fig. 6b</a>). Only two CAZY families were unique to the endophytes, carbohydrate-binding module 1 (CBM1) and glycosyl hydrolase 88 (GH88) (<a reveal-id="dsy044-F6" data-open="dsy044-F6" class="link link-reveal link-table xref-fig">Fig. 6b</a>). These profiles were present only in JGTA-S1, WP1, <em>S. indica,</em> and <em>G. lineata</em>. CBM1 is a protein domain that binds to polysaccharides and is found in fungal hydrolytic enzymes that degrade plant cell wall. Larroque <em>et al</em>.<a reveal-id="dsy044-B50" data-open="dsy044-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> found many CBM1-domain-containing proteins in necrotrophic and hemibiotrophic fungi but no CBM1-domains was found in biotrophic fungi. Our study showed that endophytes also have CBM1 domains and may indeed be more similar to necrotrophs than to biotrophs. It is relevant to mention here that <em>S. indica</em> despite being a PGP fungus was reported to cause host cell death during mutualistic symbiosis.<a reveal-id="dsy044-B51" data-open="dsy044-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Some CAZY families were present in higher numbers in JGTA-S1 such as GT90, GH16 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). The GT90 proteins were associated with capsule formation in encapsulated yeasts.<a reveal-id="dsy044-B52" data-open="dsy044-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> GH16 enzymes catalyse the breakdown of β-1, 3-1, 4-glucans<a reveal-id="dsy044-B53" data-open="dsy044-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> commonly found in the cell walls of grasses. JGTA-S1 encodes GH16 enzymes that possibly facilitate penetration and establishment in grasses such as <em>Typha</em> and rice. Several cellulases such as cello biohydrolases (GH6 and GH7), endoglucanases (GH45, GH74) were missing in JGTA-S1. Only glucosidase (GH3) could be detected. Also missing from JGTA-S1 were hemicellulases such as xylanases (GH10 and GH11), arabino uranosidases (GH51, GH54, and GH62), endoarabinase (GH43 and GH93), and xyloglucanase (GH74 and GH12). </p>                        <a id="134010309" scrollto-destination="134010309"></a>
<div id="dsy044-F6" data-id="dsy044-F6" class="fig fig-section">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f6.png?Expires=1576462815&amp;Signature=cymkKymJLVxLOo5CtmWj8U8yQYkhQ0-c6obakTiZmrN4PewzocayZp5zYwpufXVy7b-UkKrrhVIDClGQ1BB0MYo5HNhO3d0CIKL1q14~r-XNVO3USLFg2PG9mKG2I2aJ0AUPp-F7V48dSIBgXNBTQhvZAVttv1s695VRgaPSXRMO~v2~Gwu565s63Xkia-9j7gQPKX9Oc--w8iOtUc1SNHbfBqASDads-JUGBZq-~XiCXFDgAIGi~Txt3FaroPmBHB~wM~n0ZjZhO6eXzYF0ady6OdxpdW--o~FU9vaxvBbAQioU1vR89WprdwJxETs0hBC9RrxAtt3OQLTmrOPRMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAZY enzymes of R. mucilaginosa JGTA-S1 in comparison with other Basidiomycetous fungi. CAZY enzymes were inferred from the proteome of JGTA-S1, Umay (U. maydis), Min (M. intermedium), Mld (M. lychnidis-dioicae), Gram (G. lineata), WP1 (R. graminis WP1), Sin (S. indica), Mela, (M. larici-populina), and Pucc (P. graminis). (a) Shared and unique CAZY enzymes are shown in the Venn diagram. To make visualization simple, closely related genomes have been grouped together. (b) Comparison of CAZY profile of JGTA-S1 to other Basidiomycetous fungi by cluster analysis based on of number of CAZY genes per genome. Heatmap shows gene counts per CAZY family. Color figures are available at DNARES online." path-from-xml="dsy044f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010309/dsy044f6.tif" path-from-xml="dsy044f6.tif" target="_blank">Open in new tab</a><a section="134010309" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f6.png?Expires=1576462815&amp;Signature=jhpQJ1RcsfmOhV4zlSWqNc-mAFI22kPCGDYoo9G-VVrq90n3~62yLTjLGwGC9yRA29StCXdayQ6Wld9vNZwNbOMS1uCGVSEEVi~TA7b6Pe5ed3aaImEQK3gyfxJ9dsfnc4lfW9FeoIDIbocM7OfaTkbvSRLwJJfS9oiBvZ~cE3~2ljh9qGQVBnoiDXUyekTZY-GPVIgbuD-mfDyD-RCoY0g1I8cuqh6UMY6vXlQL~DuutQsS4kI4RMw54J~NE3kbKK3xhT~N1I6GtQahTCD7SNASbRMyw5lUIDm42rivExABVVGXReyD-9JM6VR7FuuFIl9vNgbVPi15hchDBvU7kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010309&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>CAZY enzymes of <em>R. mucilaginosa</em> JGTA-S1 in comparison with other Basidiomycetous fungi. CAZY enzymes were inferred from the proteome of JGTA-S1<strong>,</strong> Umay (<em>U. maydis</em>), Min (<em>M. intermedium</em>), Mld (<em>M. lychnidis-dioicae</em>), Gram (<em>G. lineata</em>), WP1 (<em>R. graminis</em> WP1), Sin (<em>S. indica</em>), Mela, (<em>M. larici-populina</em>), and Pucc (<em>P. graminis</em>). (a) Shared and unique CAZY enzymes are shown in the Venn diagram. To make visualization simple, closely related genomes have been grouped together. (b) Comparison of CAZY profile of JGTA-S1 to other Basidiomycetous fungi by cluster analysis based on of number of CAZY genes per genome. Heatmap shows gene counts per CAZY family. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F6" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 6</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f6.png?Expires=1576462815&amp;Signature=cymkKymJLVxLOo5CtmWj8U8yQYkhQ0-c6obakTiZmrN4PewzocayZp5zYwpufXVy7b-UkKrrhVIDClGQ1BB0MYo5HNhO3d0CIKL1q14~r-XNVO3USLFg2PG9mKG2I2aJ0AUPp-F7V48dSIBgXNBTQhvZAVttv1s695VRgaPSXRMO~v2~Gwu565s63Xkia-9j7gQPKX9Oc--w8iOtUc1SNHbfBqASDads-JUGBZq-~XiCXFDgAIGi~Txt3FaroPmBHB~wM~n0ZjZhO6eXzYF0ady6OdxpdW--o~FU9vaxvBbAQioU1vR89WprdwJxETs0hBC9RrxAtt3OQLTmrOPRMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAZY enzymes of R. mucilaginosa JGTA-S1 in comparison with other Basidiomycetous fungi. CAZY enzymes were inferred from the proteome of JGTA-S1, Umay (U. maydis), Min (M. intermedium), Mld (M. lychnidis-dioicae), Gram (G. lineata), WP1 (R. graminis WP1), Sin (S. indica), Mela, (M. larici-populina), and Pucc (P. graminis). (a) Shared and unique CAZY enzymes are shown in the Venn diagram. To make visualization simple, closely related genomes have been grouped together. (b) Comparison of CAZY profile of JGTA-S1 to other Basidiomycetous fungi by cluster analysis based on of number of CAZY genes per genome. Heatmap shows gene counts per CAZY family. Color figures are available at DNARES online." path-from-xml="dsy044f6.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010309/dsy044f6.tif" path-from-xml="dsy044f6.tif" target="_blank">Open in new tab</a><a section="134010309" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f6.png?Expires=1576462815&amp;Signature=jhpQJ1RcsfmOhV4zlSWqNc-mAFI22kPCGDYoo9G-VVrq90n3~62yLTjLGwGC9yRA29StCXdayQ6Wld9vNZwNbOMS1uCGVSEEVi~TA7b6Pe5ed3aaImEQK3gyfxJ9dsfnc4lfW9FeoIDIbocM7OfaTkbvSRLwJJfS9oiBvZ~cE3~2ljh9qGQVBnoiDXUyekTZY-GPVIgbuD-mfDyD-RCoY0g1I8cuqh6UMY6vXlQL~DuutQsS4kI4RMw54J~NE3kbKK3xhT~N1I6GtQahTCD7SNASbRMyw5lUIDm42rivExABVVGXReyD-9JM6VR7FuuFIl9vNgbVPi15hchDBvU7kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010309&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f6.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>CAZY enzymes of <em>R. mucilaginosa</em> JGTA-S1 in comparison with other Basidiomycetous fungi. CAZY enzymes were inferred from the proteome of JGTA-S1<strong>,</strong> Umay (<em>U. maydis</em>), Min (<em>M. intermedium</em>), Mld (<em>M. lychnidis-dioicae</em>), Gram (<em>G. lineata</em>), WP1 (<em>R. graminis</em> WP1), Sin (<em>S. indica</em>), Mela, (<em>M. larici-populina</em>), and Pucc (<em>P. graminis</em>). (a) Shared and unique CAZY enzymes are shown in the Venn diagram. To make visualization simple, closely related genomes have been grouped together. (b) Comparison of CAZY profile of JGTA-S1 to other Basidiomycetous fungi by cluster analysis based on of number of CAZY genes per genome. Heatmap shows gene counts per CAZY family. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                    <h3 scrollto-destination="134010310" id="134010310" class="section-title">3.5. <em>Rhodotorula</em> yeasts encode multiple strategies for establishment and evasion of plant defence</h3>
<p>Chitin deacetylases of family CE4 were present in JGTA-S1 and the other <em>Rhodotorula</em> and <em>Rhodosporidium</em> genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). This enzyme converts fungal chitin to chitosan that helps in avoiding detection by the host.<a reveal-id="dsy044-B54" data-open="dsy044-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Thus, CE4 might play a role in the evasion of plant defence in these organisms.</p>
<p>Another way of avoiding chitin-based plant immune response is to scavenge degraded chitin molecules. Fungal LysM domain-containing proteins are chitin-binding proteins that scavenge the degraded chitin molecules that form during fungal penetration preventing activation of plant defence.<a reveal-id="dsy044-B55" data-open="dsy044-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> CBM1 proteins described in the section above are LysM domain-containing enzymes. Ten CBM1 proteins were predicted in JGTA-S1 possibly supporting the ability of the fungus to evade plant defence.</p>
<p>All the <em>Rhodotorula</em> genomes studied here were found to have at least one copy of salicylate hydroxylase, a key enzyme required for breaking down of salicylic acid (SA), a plant defence hormone that is produced both locally as well as systemically upon encounter with microbe-associated molecular patterns<a reveal-id="dsy044-B56" data-open="dsy044-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> triggering plant defence against biotrophs. Salicylate hydroxylase has been reported in the smut fungus <em>U. maydis</em> where it may play a role in biotrophic growth.<a reveal-id="dsy044-B57" data-open="dsy044-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Copies of salicylate hydroxylase were encoded on all genomes such as KWU46987.1 (<em>R.</em> sp JG1b), XP_018270700.1 (<em>R. graminis</em> WP1), g3206.t1, g5580.t1 (C2.5t1), EMS24466.1 (NP11) and g1003.t1, g1030.t1 (JGTA-S1). JGTA-S1 also contained several genes of the Jasmonic acid pathway (g718.t1, g2844.t1, and g4607.t1:<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). Jasmonic acid is an alternate plant defence hormone known to have an antagonistic effect on SA-mediated plant defence.<a reveal-id="dsy044-B58" data-open="dsy044-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a></p>                    <h3 scrollto-destination="134010314" id="134010314" class="section-title">3.6. <em>R. mucilaginosa</em> encodes genes related to PGP</h3>
<p>JGTA-S1 genome contained many genes that could potentially influence plant hormonal response. Genes coding for the auxin pathway were found in JGTA-S1. Three genes of the abscisic acid pathway were also found on JGTA-S1. The gene, g288.t1-encoded isopentenyl-diphosphate delta-isomerase that is required to convert plant hormone cytokinin from inactive to an active form<a reveal-id="dsy044-B59" data-open="dsy044-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and tRNA-dimethylallyl transferase the first gene in the cytokinin biosynthetic pathway (g5608.t1). The yeast also had the potential to influence Brassinosteroid pathway as it contained several genes required for sterol biosynthesis. JGTA-S1 also coded genes for (R)-acetoin and (R, R)-2, 3-butandiol production that are volatiles produced by many PGP rhizobacteria. All genes related to hormone production are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>.</p>                    <h3 scrollto-destination="134010316" id="134010316" class="section-title">3.7. Solubilization and uptake of phosphate</h3>
<p>The release of phosphate from inorganic or organic sources and its uptake is an important way by which endophytes benefit plants.<a reveal-id="dsy044-B60" data-open="dsy044-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> JGTA-S1 contained organic phosphatases (g2241.t1, g4636.t1), inorganic phosphatases (g1099.t1, g3530.t1, g5792.t1, and g2815.t1) and phosphate transporters (g981.t1, g4895.t1, g2031.t1, g660.t1, and g5404.t1) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). We experimentally validated the phosphate solubilization ability of JGTA-S1. Clear zones around JGTA-S1 growth were observed on Pikovskaya’s agar plates (<a reveal-id="dsy044-F7" data-open="dsy044-F7" class="link link-reveal link-table xref-fig">Fig. 7a</a>) which indicated its ability to solubilize insoluble calcium phosphate present in the media. Consistent with this data, JGTA-S1 was able to improve growth in rice plants grown in low-phosphate media (<a reveal-id="dsy044-F7" data-open="dsy044-F7" class="link link-reveal link-table xref-fig">Fig. 7b</a>) in comparison to untreated controls which turned chlorotic 10 days after transfer in low-phosphate media. </p>                        <a id="134010318" scrollto-destination="134010318"></a>
<div id="dsy044-F7" data-id="dsy044-F7" class="fig fig-section">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f7.png?Expires=1576462815&amp;Signature=4iDs~QD3O39c9wM8rvxBN7jY47FpV66~Y1isw3N0hq5pu2v~kdzT~H5MgqthslBqqXb2Z5IwGdMmhBV~fUM7NhIjjRq3HQtPVyF28Jc5n0dtxH05gZ6MgM1B2P7EIY9xBIihgcPMpRMPtxGwY5yTGzECzoXR7oTKw~Li3~~A3RJ859BN24jGElvtjBc-4flAPcs4r~Rt~YEme~3oolzAUG7ucbIZmUD6FjnCrdBOK2Q1A9xZgqX~EFu8csp8a0BKj15yL9s~Wpdk~X4Mw3t6dXjLaySFEU4M3CB5qXiQZkX0YcDOyoLlZTMMG7Mlq9~JLH-LJbl2rsO8qh4DqbcZHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phosphate solubilization by R. mucilaginosa JGTA-S1. JGTA-S1 cells were grown in TSB broth overnight. The O.D. was adjusted to 1 and cells were diluted and spotted on Pikovskaya’s media. For studying the effect of JGTA-S1 on plant phosphate nutrition, rice plants grown on low-phosphate media were supplemented with JGTA-S1. (a) The halo around the JGTA-S1 spots indicates solubilization of phosphate on Pikovskaya’s media. (b) The growth of rice in low-phosphate media in control and Rhodotorula-treated condition 10dpi. Color figures are available at DNARES online." path-from-xml="dsy044f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010318/dsy044f7.tif" path-from-xml="dsy044f7.tif" target="_blank">Open in new tab</a><a section="134010318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f7.png?Expires=1576462815&amp;Signature=ysmKArDFaNlHPcJoaeXvV6Ax10rEM7QNQxXBuiiWeSHHM1LBijur1Hl2SOgU6rJqVJNN3~Ol3zbc-y8oUgLLX0J3geK3STtxIYVus3eKUkJVKUxI9Mlgz1Io5Cai-F6aejP3B43HIclNgwP1REfAugDzo52mlOQAndNa-IorLpAwYfud5uxkF6rNjn~SnR~BojYNhkY7IH9LpA4JV2SDY~JhNBeB5YONbuhFOGHEK8U6gfpos4nuI5QLrxy390Lwn10ClNvR6zwWbHkwE8my164RL07cANm~VvSGJ3eKlLZkIfzsJQOrOQhYYanHV2m~py9I4TSVahg1ZpvNCcFMFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010318&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Phosphate solubilization by <em>R. mucilaginosa</em> JGTA-S1. JGTA-S1 cells were grown in TSB broth overnight. The O.D. was adjusted to 1 and cells were diluted and spotted on Pikovskaya’s media. For studying the effect of JGTA-S1 on plant phosphate nutrition, rice plants grown on low-phosphate media were supplemented with JGTA-S1. (a) The halo around the JGTA-S1 spots indicates solubilization of phosphate on Pikovskaya’s media. (b) The growth of rice in low-phosphate media in control and <em>Rhodotorula</em>-treated condition 10dpi. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F7" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 7</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f7.png?Expires=1576462815&amp;Signature=4iDs~QD3O39c9wM8rvxBN7jY47FpV66~Y1isw3N0hq5pu2v~kdzT~H5MgqthslBqqXb2Z5IwGdMmhBV~fUM7NhIjjRq3HQtPVyF28Jc5n0dtxH05gZ6MgM1B2P7EIY9xBIihgcPMpRMPtxGwY5yTGzECzoXR7oTKw~Li3~~A3RJ859BN24jGElvtjBc-4flAPcs4r~Rt~YEme~3oolzAUG7ucbIZmUD6FjnCrdBOK2Q1A9xZgqX~EFu8csp8a0BKj15yL9s~Wpdk~X4Mw3t6dXjLaySFEU4M3CB5qXiQZkX0YcDOyoLlZTMMG7Mlq9~JLH-LJbl2rsO8qh4DqbcZHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phosphate solubilization by R. mucilaginosa JGTA-S1. JGTA-S1 cells were grown in TSB broth overnight. The O.D. was adjusted to 1 and cells were diluted and spotted on Pikovskaya’s media. For studying the effect of JGTA-S1 on plant phosphate nutrition, rice plants grown on low-phosphate media were supplemented with JGTA-S1. (a) The halo around the JGTA-S1 spots indicates solubilization of phosphate on Pikovskaya’s media. (b) The growth of rice in low-phosphate media in control and Rhodotorula-treated condition 10dpi. Color figures are available at DNARES online." path-from-xml="dsy044f7.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010318/dsy044f7.tif" path-from-xml="dsy044f7.tif" target="_blank">Open in new tab</a><a section="134010318" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f7.png?Expires=1576462815&amp;Signature=ysmKArDFaNlHPcJoaeXvV6Ax10rEM7QNQxXBuiiWeSHHM1LBijur1Hl2SOgU6rJqVJNN3~Ol3zbc-y8oUgLLX0J3geK3STtxIYVus3eKUkJVKUxI9Mlgz1Io5Cai-F6aejP3B43HIclNgwP1REfAugDzo52mlOQAndNa-IorLpAwYfud5uxkF6rNjn~SnR~BojYNhkY7IH9LpA4JV2SDY~JhNBeB5YONbuhFOGHEK8U6gfpos4nuI5QLrxy390Lwn10ClNvR6zwWbHkwE8my164RL07cANm~VvSGJ3eKlLZkIfzsJQOrOQhYYanHV2m~py9I4TSVahg1ZpvNCcFMFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010318&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f7.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Phosphate solubilization by <em>R. mucilaginosa</em> JGTA-S1. JGTA-S1 cells were grown in TSB broth overnight. The O.D. was adjusted to 1 and cells were diluted and spotted on Pikovskaya’s media. For studying the effect of JGTA-S1 on plant phosphate nutrition, rice plants grown on low-phosphate media were supplemented with JGTA-S1. (a) The halo around the JGTA-S1 spots indicates solubilization of phosphate on Pikovskaya’s media. (b) The growth of rice in low-phosphate media in control and <em>Rhodotorula</em>-treated condition 10dpi. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                    <h3 scrollto-destination="134010319" id="134010319" class="section-title">3.8. Effector proteins of JGTA-S1</h3>
<p>EffectorP was used to predict effectors from secretory proteins. Of the 1, 134 candidate effector proteins identified from the five genomes, only 278 proteins (24.5%) could be annotated by BLASTP against nr database (E value &lt; 1e-05) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>) which was not surprising since most effector proteins do not contain domains.<a reveal-id="dsy044-B61" data-open="dsy044-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> The effectors were clustered into 223 groups containing 499 proteins while 635 proteins remained as singletons, which mean that most effectors were unique to each organism. Moreover, no conserved motifs could be identified in the effectors. This observation reiterates the finding that fungal effectors are often species-specific and are evolving at a rapid pace such that orthologues of effectors are not seen in closely related genomes.<a reveal-id="dsy044-B62" data-open="dsy044-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> The annotated proteins included chitin deacetylase that functions in modifying chitin as mentioned previously, a malic enzyme involved in the fatty acid synthesis and a serine carboxypeptidase. Also found among the effectors were some proteophosphoglycan proteins (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>).</p>                    <h3 scrollto-destination="134010321" id="134010321" class="section-title">3.9. <em>Rhodotorula</em> encodes an intact mating locus</h3>
<p>The mating locus of fungi is responsible for sex determination and filament formation. It consisted of homeodomain transcription factors and pheromone-related genes. An intact MAT locus on Scaffold 13 of JGTA-S1 was identified with homologues of a map kinase (<em>STE20</em>), a pheromone receptor gene (<em>STE3</em>), and pheromone precursor genes (<em>RHA1</em>, <em>RHA2</em>). Another MAT locus containing the homeodomain transcription factors <em>HD1</em> and a remnant of <em>HD2</em> were identified on Scaffold 7 of JGTA-S1. It was difficult to conclude if Scaffolds 7 and 13 were part of the same chromosome or were unlinked. These loci were at least 500 kb apart. The large distance between the pheromone-related locus and the HD locus in JGTA-S1 was similar to <em>Mixia osmundae</em><a reveal-id="dsy044-B35" data-open="dsy044-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and <em>Sporidiobolus salmonicolor</em>.<a reveal-id="dsy044-B63" data-open="dsy044-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> This mating system, therefore, could be bipolar, tetrapolar or pseudobipolar. No polymorphisms of <em>STE3</em> and <em>STE20</em> genes were found between the <em>R. mucilaginosa</em> genomes, suggesting either, (i) JGTA-S1 and C2.5t1 belonged to the same mating type and the opposite mating type is yet to be reported, (ii) JGTA-S1 and C2.5t1 were homothallic and were able to mate with single individual, or that (iii) JGTA-S1 and C2.5t1 were clonal members of an asexually reproducing population. Comparison of synteny in the MAT locus showed good agreement among JGTA-S1, C2.5t, and JG1b (<a reveal-id="dsy044-F8" data-open="dsy044-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). Sexual life cycle and transition from yeast to filament form are tightly linked phenomena in many dimorphic fungi. Filament formation occurs after mating of complementary cells of opposite mating types leading to the formation of dikaryotic mycelium in <em>R. toruloides</em>.<a reveal-id="dsy044-B8" data-open="dsy044-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> However, no filamentous growth or sexual lifecycle was ever reported in <em>Rhodotorula</em> yeasts leading to the segregation of the <em>Rhodotorula</em> genus from the sexually reproducing <em>Rhodosporidium</em>. Although could not be demonstrated, a possible filamentous stage was also predicted for <em>R. graminis</em> WP1.<a reveal-id="dsy044-B41" data-open="dsy044-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Presence of a complete set of MAT genes in JGTA-S1 may predict the presence of a filamentous stage similarly, either by the conjugation of yet unidentified complementary mating types or alternatively, from germinating asexual spores as described in Zygomycetes. </p>                        <a id="134010323" scrollto-destination="134010323"></a>
<div id="dsy044-F8" data-id="dsy044-F8" class="fig fig-section">
<div class="label fig-label">Figure 8</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f8.png?Expires=1576462815&amp;Signature=YUCnMjmlBQd5ttnGgoTi1yuZKjyNx8BSxQhLLnEw5bBtqdybXNECu9yhy8g5rv0S-mwbSy8HzSyI1BeFhP-mtHPgAYiMVoWFAmEiJog6LTjipkY~Hjnms08f~pzZgDO-k4FaWBW7CbpFK5H0~Y4A7OgGWGqBuiWv2XU435BkXDlBMKy2bldXewShAKLczxK4ipZFFoqe2FK44rq0R88wGDfylhiUAz9way60EMzUMb2i191KH3YUKn5dpdc0TpkHwxaJEeMOWpIxUpoVo-baS~8V6NewVUk4tj3ZZrIY9sXAEAd2r-3jnmhnTYygpq9RhwAwxAjDK4aOpRa60VLXTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny in the MAT locus of the Rhodotorula genomes. The MAT locus of WP1was annotated. The gene names are indicated and its synteny is compared with other genomes. Each arrow represents a gene. Homologous genes have the same colour and are connected by lines. Color figures are available at DNARES online." path-from-xml="dsy044f8.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010323/dsy044f8.tif" path-from-xml="dsy044f8.tif" target="_blank">Open in new tab</a><a section="134010323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f8.png?Expires=1576462815&amp;Signature=AmkcDOq8-kmyMAMr~gq0xo65m~OcEZpiUja2f77M3jz3KLoc~Io9pxHfx-hoqAu53OyKqV4JeLjrlBWBOR2Ls857k0tLIAk4qXSG94mIBVtIGLyaq7q-PeiYQpo8hH7dnfBFdL-UAbC9MZGqnU9OtagJqY3QJwL-zjtvuIQXjMszQEjNMEPnqlt5-VdK5k3wmjHzhjacV1Bfb8vqtypQXn5x~KOQ~H-oG6AZ8VE1cmfeROTclDr6FqO5x2HGudwKyV1QcnhIQuv328Q25VbyXuuwbLUq7vd0mPj1l2oWpRUa7zWZSAggu5S725r0R~zoDSa-3AQ9ZC~1waphMx-1Gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010323&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f8.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Synteny in the MAT locus of the <em>Rhodotorula</em> genomes. The MAT locus of WP1was annotated. The gene names are indicated and its synteny is compared with other genomes. Each arrow represents a gene. Homologous genes have the same colour and are connected by lines. Color figures are available at <em>DNARES</em> online.</p></div>
</div>
<div content-id="dsy044-F8" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 8</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f8.png?Expires=1576462815&amp;Signature=YUCnMjmlBQd5ttnGgoTi1yuZKjyNx8BSxQhLLnEw5bBtqdybXNECu9yhy8g5rv0S-mwbSy8HzSyI1BeFhP-mtHPgAYiMVoWFAmEiJog6LTjipkY~Hjnms08f~pzZgDO-k4FaWBW7CbpFK5H0~Y4A7OgGWGqBuiWv2XU435BkXDlBMKy2bldXewShAKLczxK4ipZFFoqe2FK44rq0R88wGDfylhiUAz9way60EMzUMb2i191KH3YUKn5dpdc0TpkHwxaJEeMOWpIxUpoVo-baS~8V6NewVUk4tj3ZZrIY9sXAEAd2r-3jnmhnTYygpq9RhwAwxAjDK4aOpRa60VLXTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny in the MAT locus of the Rhodotorula genomes. The MAT locus of WP1was annotated. The gene names are indicated and its synteny is compared with other genomes. Each arrow represents a gene. Homologous genes have the same colour and are connected by lines. Color figures are available at DNARES online." path-from-xml="dsy044f8.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010323/dsy044f8.tif" path-from-xml="dsy044f8.tif" target="_blank">Open in new tab</a><a section="134010323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f8.png?Expires=1576462815&amp;Signature=AmkcDOq8-kmyMAMr~gq0xo65m~OcEZpiUja2f77M3jz3KLoc~Io9pxHfx-hoqAu53OyKqV4JeLjrlBWBOR2Ls857k0tLIAk4qXSG94mIBVtIGLyaq7q-PeiYQpo8hH7dnfBFdL-UAbC9MZGqnU9OtagJqY3QJwL-zjtvuIQXjMszQEjNMEPnqlt5-VdK5k3wmjHzhjacV1Bfb8vqtypQXn5x~KOQ~H-oG6AZ8VE1cmfeROTclDr6FqO5x2HGudwKyV1QcnhIQuv328Q25VbyXuuwbLUq7vd0mPj1l2oWpRUa7zWZSAggu5S725r0R~zoDSa-3AQ9ZC~1waphMx-1Gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010323&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f8.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Synteny in the MAT locus of the <em>Rhodotorula</em> genomes. The MAT locus of WP1was annotated. The gene names are indicated and its synteny is compared with other genomes. Each arrow represents a gene. Homologous genes have the same colour and are connected by lines. Color figures are available at <em>DNARES</em> online.</p></div>
</div>                    <h3 scrollto-destination="134010324" id="134010324" class="section-title">3.10. JGTA-S1 contain a truncated nitrogen metabolic pathway</h3>
<p>Nitrogen is taken up by fungi in its organic form as amino acids and in its inorganic form as nitrate and ammonium. Nitrate is assimilated into ammonium by the nitrate assimilation pathway that consists of a nitrate reductase, nitrite reductase, and nitrate transporters belonging to Major Facilitator Superfamily (MFS). Several rust fungi that are obligate biotrophs such as <em>Melampsora</em> and <em>Puccinia</em> have lost this pathway.<a reveal-id="dsy044-B64" data-open="dsy044-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> We found an incomplete nitrate assimilation pathway in JGTA-S1, C2.5t1, and JG1b that was missing nitrite reductase and the MFS nitrate transporters. On the other hand, NP11 and WP1 had an intact nitrate assimilation pathway. This could partly be circumvented in JGTA-S1 by the uptake of ammonium by the two ammonium transporter proteins (g2114.t1 and g5165.t1). One of these, g5165.t1 was an orthologue of a high-affinity ammonium transporter from <em>U. maydis</em> that can induce filament formation in low ammonium conditions.<a reveal-id="dsy044-B65" data-open="dsy044-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> JGTA-S1 also encoded glutamate synthase (g5248.t1) and glutamine synthetase (g5571.t1) the key enzymes involved in ammonia assimilation. In addition, nine amino acid transporters were encoded by JGTA-S1. <em>Rhodotorula</em> yeasts have the ability to utilize urea as a sole N-source with a potential to improve urea utilization of plants. This property was confirmed by the genome of JGTA-S1 that encoded urea degrading enzymes and urea transporters (g4941.t1 and g5832.t1).</p>                    <h3 scrollto-destination="134010326" id="134010326" class="section-title">3.11. <em>R. mucilaginosa</em> JGTA-S1 may contain endosymbiotic bacteria</h3>
<p>Another way in which JGTA-S1 could circumvent the absence of the complete nitrate assimilation pathway is by association with N-fixing bacteria<em>. Rhodotorula mucilaginosa</em> JGTA-S1 was able to grow in N-free media (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9b</a>). Growth in N-deprived media had earlier been reported in <em>U. maydis</em> where an association with an N-fixing bacterium living as an endosymbiont was assumed to supply N to the host.<a reveal-id="dsy044-B66" data-open="dsy044-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> At least four dinitrogen reductase sequences corresponding to diazotrophs were amplified from JGTA-S1 genomic DNA (MG566093–MG566096). To search for bacteria in the genomic data the Illumina reads were mapped to the final assembly of JGTA-S1 and only the unmapped reads (1.85%) were retained to search for bacterial sequences. Several bacterial sequences could be found within the 15,047 pairs of reads. Out of these, 50 reads having over 98% identity to <em>Pseudomonas stutzeri</em> were found (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). The results corroborated with the finding that two <em>nifH</em> sequences MG566093 and MG566095 corresponding to <em>P. stutzeri</em>—an N-fixing as well as the denitrifying bacterium was amplified from JGTA-S1 genomic DNA. A <em>P. stutzeri</em> strain was experimentally confirmed to be able to colonize endosymbiotically in <em>R. mucilaginosa</em> (Saha <em>et al.,</em> unpublished). Although several reads belonging to other N-fixing bacteria <em>Klebsiella</em> sp., <em>Bradyrhizobium</em> sp., <em>Mesorhizobium loti</em>, <em>Rhizobium</em> sp. and <em>Sinorhizobium</em> sp., etc. were also found (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>), no single bacterium dominated the reads prompting us to speculate (i) low diazotroph load in the genomic DNA of JGTA-S1 or (ii) the presence of several different bacteria. This observation is also supported by the low number of endosymbiotic bacteria/fungal cell (or spore) reported in <em>S. indica</em> and mycorrhizal fungus <em>Gigaspora margarita</em>, N-fixing bacteria comprising even a smaller fraction of this small number.<a reveal-id="dsy044-B67" data-open="dsy044-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> For a recently sequenced genome of an obligatory Mollicutes-related endosymbiont (DhMRE) from Arbuscular mycorrhiza <em>Dentiscutata heterogama</em> the spores of the fungus were enriched before the endobacterial genome was isolated and sequenced.<a reveal-id="dsy044-B68" data-open="dsy044-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> A protocol that enriched the DhMRE sequence and removed mycorrhizal DNA was developed for this purpose. We did not opt for any such enrichment procedures which might explain why the bacterial sequence counts of <em>P. stutzeri</em> (or other diazotrophs) were low. Since the presences of the other diazotrophs were unconfirmed, <em>P. stutzeri</em> was given prime importance for further studies. To investigate the importance of the endosymbionts we streaked JGTA-S1 several times in presence of Tetracycline and Timentin chosen based on the sensitivity of a <em>P. stutzeri</em> strain (JGTA-R3) isolated in the laboratory.<a reveal-id="dsy044-B14" data-open="dsy044-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> After five passages, JGTA-S1 colonies were screened for the absence of <em>P. stutzeri</em> specific 16SrDNA amplification. A strain (Strain 11) showing no amplification of <em>P. stutzeri</em> 16SrDNA from the JGTA-S1 genomic DNA in two independent PCR reactions was used for further experiments (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9a</a>). Spotting assay was performed on TSB (complete media) and N-free media as a measure of growth or fitness of JGTA-S1 cells. In parallel, qPCR was performed where <em>P. stutzeri</em> 16SrDNA level was quantitated and normalized against JGTA-S1 GAPDH gene. This relative abundance reflected the number of <em>P. stutzeri</em> per JGTA-S1 cell. Strain 11 did not show any significant reduction in growth compared with control JGTA-S1 in spotting assay (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9b</a>). The qPCR result indicated that the number of <em>P. stutzeri</em>/JGTA-S1 was lower in Strain 11 compared with control when the Strain 11 was grown on TSB media as expected (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9a and c</a>, left panel). The results were however completely different when ammonium was used as the sole N-source. <em>Pseudomonas stutzeri</em> levels/JGTA-S1 cell in Strain 11 was 1.8-fold higher than control when the cells were grown in N-free media with (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9c</a>, right panel). This result showed that the levels of <em>P. stutzeri</em> alone did not dictate the fitness of JGTA-S1, especially on N-free media. This was not surprising since JGTA-S1 appeared to contain diazotrophs other than <em>P. stutzeri</em> underlining the possibility that the other diazotrophic endobacteria compensated for the loss of <em>P. stutzeri</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576462815&amp;Signature=mz6e8bZ-cTJbfENtvZJAK~wPp4f7jY5ZUCjM3LFUumu0ty8SWhZLcDqnIbk3FTB6J2Cr9prpyx8x8~hfcBljv0-9cntCxA42ExWNb5JYRjp8hftI3qrNZhLTD3o1JvKY4wcp~1xFuet57xOQW2jxahGCKK8~UlDe--X6BQwFYXZMtYdCaWX6VLOKiF-eiepi1nvLY8u5p-294KGC-Oy5KQU9tng9b562w9nogF9sPW~opto39ijuyEEvcTolEdbXi47ehq5~cfiNFB1eOSA4Jb7FJVVbGks7QMwUYV95-eHbOarD~CKKq~CN4QkcqLJv~aS8AMMOBMdwWVUsbl1RTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span> and <a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9d</a>). The relative levels of <em>P. stutzeri/</em>JGTA-S1 in Strain 11 showed an increase compared with JGTA-S1 control cells when ammonium was the sole N-source compared with when it was grown in complete media. This indicated that (i) yeast growth in spotting assay and the number of <em>P. stutzeri</em>/JGTA-S1 cells did not always correlate and there was a more complex interplay (ii) The number of <em>P. stutzeri</em>/JGTA-S1 in Strain 11 depended on whether N was available in organic plus inorganic forms (TSB) or just as inorganic form (ammonium) indicating a possible correlation between <em>P. stutzeri</em> levels/yeast cell and N-nutrition of JGTA-S1. Endobacteria were found to be capable of multiplying within mycorrhiza at different stages of the fungal development.<a reveal-id="dsy044-B67" data-open="dsy044-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Whether this relative increase in <em>P. stutzeri</em>/JGTA-S1 was a consequence of multiplication of <em>P. stutzeri</em> within JGTA-S1 in response to inadequate N-availability is an interesting proposition to study in future. However, a JGTA-S1 strain (Strain 11.2) derived from Strain 11 grown in TSB media contained endobacteria despite treatment with five different antibiotics Timentin, Tetracycline, Carbenicillin, Gentamycin, and Kanamycin. A primer pair targeted to amplify the V3 region of bacterial 16SrDNA gene was used with JGTA-S1 and Strain 11.2 genomic DNA as a template for showing the total endobacteria (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9d</a>, upper panel). Strain 11.2 was grown in presence of ammonium as sole N-source in presence Carbenicillin, Gentamycin, and Kanamycin as it failed to grow in presence of all five antibiotics. Strain 11.2 was then spotted on N-free media or the genomic DNA was used for qPCR to measure <em>P. stutzeri</em> levels/JGTA-S1 cell as before. Strain 11.2, however, had a significant fitness cost unlike Strain 11 (<a reveal-id="dsy044-F9" data-open="dsy044-F9" class="link link-reveal link-table xref-fig">Fig. 9e</a>, left panel) in spotting assay probably due to the removal of additional diazotrophs. Despite that, the <em>P. stutzeri</em> levels/JGTA-S1 cell was increased in Strain 11.2 under the N-inadequate condition, that too, even strongly than Strain 11 further confirming that this increase was a surveillance strategy which was not to be correlated with JGTA-S1 fitness especially under N-inadequate condition. The results can be explained by the fact that endobacteria numbers/fungal cell or spore reported for <em>S. indica</em> (2–20/fungal cell) and <em>Candidatus Glomeribacter gigasporarum</em> (10/spore) are tightly regulated.<a reveal-id="dsy044-B67" data-open="dsy044-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> In <em>Rhodotorula</em>, despite being a survival strategy increase of <em>P. stutzeri</em> levels/JGTA-S1 cell beyond a point may adversely affect the fitness of the fungus. </p>                        <a id="134010328" scrollto-destination="134010328"></a>
<div id="dsy044-F9" data-id="dsy044-F9" class="fig fig-section">
<div class="label fig-label">Figure 9</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f9.png?Expires=1576462815&amp;Signature=3OaMiXOIWXIUaNoxm1l72Puk08T4KFjjV0DlqwaGlEguhn78QFjv~Y6ZnxsOzJXFRFDs7c3QCY3-De9gmjkYZl-XipOLp-hD6jWMu6BZSd~m6AiRapDX7cVZoCpopdiDLRlv1mHt9qiuQSa-uXPbQ~c91plfmNXNHPz6z294mq62WVOxAE7gmKMdW995czgwrR6eBuvcnGKS-vZoLh~kuKaJKFAVPp~x8UrVwHC3ewKggIsoIOYanSBOtj0jMGj38DqJWyYYjAzuUCkkTLSIkAIktu68QOyL356Ls8PRuRs6nfwEGOHVEUJHIBWrlxlqJaGmfLHFHUXTqb1FCUcX0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="JGTA-S1 cells had reduced growth when the endobacteria were removed. JGTA-S1 cells were streaked in presence of Timentin and Tetracycline to kill the endosymbiotic P. stutzeri within JGTA-S1 cells. Single colonies were grown in TSB after five rounds of streaking in presence of antibiotics, genomic DNA was isolated and PCR performed using P. stutzeri specific 16SrDNA primers to isolate strains deficient in P.stutzeri. Strain 11 grown in TSB or N-free media + (NH4)2SO4 was spotted on TSB or N-free media in different dilutions and compared with control JGTA-S1 cells or used for isolation of genomic DNA for qPCR. Strain 11.2 was generated by further streaking Strain 11 in the presence of Carbenicillin, Gentamycin and Kanamycin and used for genomic DNA isolation and PCR using primers for the V3 region of total bacterial 16SrDNA and qPCR with P. stutzeri specific 16SrDNA (a) P. stutzeri specific 16SrDNA amplification from Timentin and Tetracycline treated Strain 11 vs control JGTA-S1 confirming reduced level of P. stutzeri. (b) Spotting assay of control JGTA-S1 and Strain 11 on TSB and N-free media plate. Undiluted (OD600= 0.5) (c) qPCR of Strain 11 and control JGTA-S1 genomic DNA with primers designed to amplify P. stutzeri 16SrDNA relative to yeast GAPDH gene. Results are from three independent experiments each performed in triplicate. The significance is calculated by Mann-Whitney’s test. (d) PCR amplification of the V3 region of bacterial 16SrDNA from JGTA-S1 control and 11.2 cells (upper panel). The lower panel shows amplification of JGTA-S1 GAPDH gene. NTC: No template control (e) Spotting assay of control JGTA-S1 and Strain 11.2 on N-free media plate (Left panel). qPCR of Strain 11.2 genomic DNA with primers designed to amplify P. stutzeri 16SrDNA relative to yeast GAPDH gene (right panel)." path-from-xml="dsy044f9.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010328/dsy044f9.tif" path-from-xml="dsy044f9.tif" target="_blank">Open in new tab</a><a section="134010328" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f9.png?Expires=1576462815&amp;Signature=AF5h8kZ2IlPV7sQi7HMKJ7AG1sNzWZbzeQ3T2DPkmAsuNey2TRTK~u9SDJOEskbQ-uMulfez7Yw04cZyV~tvVtnr~wPOF7q60e0-5eI6Kl0vip~X4WPTx8HXRGxdbVPX9H5a5A8PtInP7W~MZQyOEW-CxqeUoKfoEBH~BKDZM~gMmvU3nP8HZoaVP3QDhNMFBmxm861QB55rsQVmEAw82gvi9EDaLk9IUnb2yqpvLFMhqBzpaRt9rQbo2yiHnPvemjzGN3t5WtIVM-3QOxfsBy0Vb9-2MR7Qwp~30P1rNW~MiUxtHEiuvdii9nEDJDH8DHiOdrPd468AeZzeGdvBOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010328&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f9.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>JGTA-S1 cells had reduced growth when the endobacteria were removed. JGTA-S1 cells were streaked in presence of Timentin and Tetracycline to kill the endosymbiotic <em>P. stutzeri</em> within JGTA-S1 cells. Single colonies were grown in TSB after five rounds of streaking in presence of antibiotics, genomic DNA was isolated and PCR performed using <em>P. stutzeri</em> specific 16SrDNA primers to isolate strains deficient in <em>P.stutzeri</em>. Strain 11 grown in TSB or N-free media + (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> was spotted on TSB or N-free media in different dilutions and compared with control JGTA-S1 cells or used for isolation of genomic DNA for qPCR. Strain 11.2 was generated by further streaking Strain 11 in the presence of Carbenicillin, Gentamycin and Kanamycin and used for genomic DNA isolation and PCR using primers for the V3 region of total bacterial 16SrDNA and qPCR with <em>P. stutzeri</em> specific 16SrDNA (a) <em>P. stutzeri</em> specific 16SrDNA amplification from Timentin and Tetracycline treated Strain 11 vs control JGTA-S1 confirming reduced level of <em>P. stutzeri</em>. (b) Spotting assay of control JGTA-S1 and Strain 11 on TSB and N-free media plate. Undiluted (OD<sub>600</sub>= 0.5) (c) qPCR of Strain 11 and control JGTA-S1 genomic DNA with primers designed to amplify <em>P. stutzeri</em> 16SrDNA relative to yeast GAPDH gene. Results are from three independent experiments each performed in triplicate. The significance is calculated by Mann-Whitney’s test. (d) PCR amplification of the V3 region of bacterial 16SrDNA from JGTA-S1 control and 11.2 cells (upper panel). The lower panel shows amplification of JGTA-S1 GAPDH gene. NTC: No template control (e) Spotting assay of control JGTA-S1 and Strain 11.2 on N-free media plate (Left panel). qPCR of Strain 11.2 genomic DNA with primers designed to amplify <em>P. stutzeri</em> 16SrDNA relative to yeast GAPDH gene (right panel).</p></div>
</div>
<div content-id="dsy044-F9" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 9</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/m_dsy044f9.png?Expires=1576462815&amp;Signature=3OaMiXOIWXIUaNoxm1l72Puk08T4KFjjV0DlqwaGlEguhn78QFjv~Y6ZnxsOzJXFRFDs7c3QCY3-De9gmjkYZl-XipOLp-hD6jWMu6BZSd~m6AiRapDX7cVZoCpopdiDLRlv1mHt9qiuQSa-uXPbQ~c91plfmNXNHPz6z294mq62WVOxAE7gmKMdW995czgwrR6eBuvcnGKS-vZoLh~kuKaJKFAVPp~x8UrVwHC3ewKggIsoIOYanSBOtj0jMGj38DqJWyYYjAzuUCkkTLSIkAIktu68QOyL356Ls8PRuRs6nfwEGOHVEUJHIBWrlxlqJaGmfLHFHUXTqb1FCUcX0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="JGTA-S1 cells had reduced growth when the endobacteria were removed. JGTA-S1 cells were streaked in presence of Timentin and Tetracycline to kill the endosymbiotic P. stutzeri within JGTA-S1 cells. Single colonies were grown in TSB after five rounds of streaking in presence of antibiotics, genomic DNA was isolated and PCR performed using P. stutzeri specific 16SrDNA primers to isolate strains deficient in P.stutzeri. Strain 11 grown in TSB or N-free media + (NH4)2SO4 was spotted on TSB or N-free media in different dilutions and compared with control JGTA-S1 cells or used for isolation of genomic DNA for qPCR. Strain 11.2 was generated by further streaking Strain 11 in the presence of Carbenicillin, Gentamycin and Kanamycin and used for genomic DNA isolation and PCR using primers for the V3 region of total bacterial 16SrDNA and qPCR with P. stutzeri specific 16SrDNA (a) P. stutzeri specific 16SrDNA amplification from Timentin and Tetracycline treated Strain 11 vs control JGTA-S1 confirming reduced level of P. stutzeri. (b) Spotting assay of control JGTA-S1 and Strain 11 on TSB and N-free media plate. Undiluted (OD600= 0.5) (c) qPCR of Strain 11 and control JGTA-S1 genomic DNA with primers designed to amplify P. stutzeri 16SrDNA relative to yeast GAPDH gene. Results are from three independent experiments each performed in triplicate. The significance is calculated by Mann-Whitney’s test. (d) PCR amplification of the V3 region of bacterial 16SrDNA from JGTA-S1 control and 11.2 cells (upper panel). The lower panel shows amplification of JGTA-S1 GAPDH gene. NTC: No template control (e) Spotting assay of control JGTA-S1 and Strain 11.2 on N-free media plate (Left panel). qPCR of Strain 11.2 genomic DNA with primers designed to amplify P. stutzeri 16SrDNA relative to yeast GAPDH gene (right panel)." path-from-xml="dsy044f9.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134010328/dsy044f9.tif" path-from-xml="dsy044f9.tif" target="_blank">Open in new tab</a><a section="134010328" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044f9.png?Expires=1576462815&amp;Signature=AF5h8kZ2IlPV7sQi7HMKJ7AG1sNzWZbzeQ3T2DPkmAsuNey2TRTK~u9SDJOEskbQ-uMulfez7Yw04cZyV~tvVtnr~wPOF7q60e0-5eI6Kl0vip~X4WPTx8HXRGxdbVPX9H5a5A8PtInP7W~MZQyOEW-CxqeUoKfoEBH~BKDZM~gMmvU3nP8HZoaVP3QDhNMFBmxm861QB55rsQVmEAw82gvi9EDaLk9IUnb2yqpvLFMhqBzpaRt9rQbo2yiHnPvemjzGN3t5WtIVM-3QOxfsBy0Vb9-2MR7Qwp~30P1rNW~MiUxtHEiuvdii9nEDJDH8DHiOdrPd468AeZzeGdvBOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134010328&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy044f9.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>JGTA-S1 cells had reduced growth when the endobacteria were removed. JGTA-S1 cells were streaked in presence of Timentin and Tetracycline to kill the endosymbiotic <em>P. stutzeri</em> within JGTA-S1 cells. Single colonies were grown in TSB after five rounds of streaking in presence of antibiotics, genomic DNA was isolated and PCR performed using <em>P. stutzeri</em> specific 16SrDNA primers to isolate strains deficient in <em>P.stutzeri</em>. Strain 11 grown in TSB or N-free media + (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> was spotted on TSB or N-free media in different dilutions and compared with control JGTA-S1 cells or used for isolation of genomic DNA for qPCR. Strain 11.2 was generated by further streaking Strain 11 in the presence of Carbenicillin, Gentamycin and Kanamycin and used for genomic DNA isolation and PCR using primers for the V3 region of total bacterial 16SrDNA and qPCR with <em>P. stutzeri</em> specific 16SrDNA (a) <em>P. stutzeri</em> specific 16SrDNA amplification from Timentin and Tetracycline treated Strain 11 vs control JGTA-S1 confirming reduced level of <em>P. stutzeri</em>. (b) Spotting assay of control JGTA-S1 and Strain 11 on TSB and N-free media plate. Undiluted (OD<sub>600</sub>= 0.5) (c) qPCR of Strain 11 and control JGTA-S1 genomic DNA with primers designed to amplify <em>P. stutzeri</em> 16SrDNA relative to yeast GAPDH gene. Results are from three independent experiments each performed in triplicate. The significance is calculated by Mann-Whitney’s test. (d) PCR amplification of the V3 region of bacterial 16SrDNA from JGTA-S1 control and 11.2 cells (upper panel). The lower panel shows amplification of JGTA-S1 GAPDH gene. NTC: No template control (e) Spotting assay of control JGTA-S1 and Strain 11.2 on N-free media plate (Left panel). qPCR of Strain 11.2 genomic DNA with primers designed to amplify <em>P. stutzeri</em> 16SrDNA relative to yeast GAPDH gene (right panel).</p></div>
</div>                    <h2 scrollto-destination="134010329" id="134010329" class="section-title">4. Conclusion</h2>
<p>Our results, therefore, provide a genomic perspective into the lifecycle of <em>Rhodotorula mucilaginosa</em> JGTA-S1 shedding some light into its ability to invade and improve plant growth and nutrition. The effect of JGTA-S1 on the plant phosphate nutrition, hormonal response, and defence was clear from relevant genes present in the JGTA-S1 genome. JGTA-S1 had a truncated N-assimilation pathway but was still able to grow in absence of nitrogen. This was potentially due to its ability to maintain diazotrophs inside the yeast. These endobacteria could not be cured and their levels/yeast cells were under complex regulation depending on the N-regime under which the yeast was grown. Comparative analyses of various endophytic and pathogens fungi within Basidiomycota showed that although the <em>Rhodotorula</em>s were evolutionarily closer to the hemibiotrophs <em>M. intermedium</em> and <em>M. lychnidis-dioicae</em> they shared some genes required for host invasion exclusively with other endophytes.</p>                    <h2 scrollto-destination="134010331" id="134010331" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>We would like to thank Mathu Malar Chandrababunaidu for helpful discussions. </p>                    <h3 scrollto-destination="134010333" id="134010333" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h3 scrollto-destination="134010335" id="134010335" class="section-title">Funding</h3>
<p>This work was funded by DBT, Ramalingaswamy project, UGC-UPE grant of the University of Calcutta (CU) and DBT project BT/PR15410/BCE/08/861/2011. Diya Sen’s fellowship was funded by Council of Scientific and Industrial Research, Govt. of India. Karnelia Paul’s fellowship was funded by UGC-UPE grant of CU. We also acknowledge DBT-IPLS infrastructural facility at CU.</p>                    <h2 scrollto-destination="134010337" id="134010337" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsy044-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Margesin</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Fonteyne</div> <div class="given-names">P. A.</div>
</div>, <div class="name string-name">
<div class="surname">Schinner</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Sampaio</div> <div class="given-names">J. P.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Rhodotorula psychrophila sp. nov., Rhodotorula psychrophenolica sp. nov. and Rhodotorula glacialis sp. nov., novel psychrophilic basidiomycetous yeast species isolated from alpine environments</div>, <div class="source">Int. J. Syst. Evol. Microbiol</div> ., <div class="volume">57</div>, <div class="fpage">2179</div>–<div class="lpage">84</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rhodotorula%20psychrophila%20sp.%20nov.%2C%20Rhodotorula%20psychrophenolica%20sp.%20nov.%20and%20Rhodotorula%20glacialis%20sp.%20nov.%2C%20novel%20psychrophilic%20basidiomycetous%20yeast%20species%20isolated%20from%20alpine%20environments&amp;author=R.%20Margesin&amp;author=P.%20A.%20Fonteyne&amp;author=F.%20Schinner&amp;author=J.%20P.%20Sampaio&amp;publication_year=2007&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol&amp;volume=57&amp;pages=2179-84" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1099/ijs.0.65111-0" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.65111-0" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1099%2Fijs.0.65111-0"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17766895" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rhodotorula%20psychrophila%20sp.%20nov.%2C%20Rhodotorula%20psychrophenolica%20sp.%20nov.%20and%20Rhodotorula%20glacialis%20sp.%20nov.%2C%20novel%20psychrophilic%20basidiomycetous%20yeast%20species%20isolated%20from%20alpine%20environments&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zalar</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Gunde-Cimerman</div> <div class="given-names">N.</div>
</div>
</div> <div class="year">2014</div>, Cold-adapted yeasts in arctic habitats. In: <div class="person-group">
<div class="name string-name">
<div class="surname">Buzzini</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Margesin</div> <div class="given-names">R.</div>
</div>
</div> (eds), <div class="source">Cold-adapted Yeasts: Biodiversity, Adaptation Strategies and Biotechnological Significance</div> , 
<div class="publisher-name">Springer Berlin Heidelberg</div>: 
<div class="publisher-loc">Berlin, Heidelberg</div>, pp. <div class="fpage">49</div>–<div class="lpage">74</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Cold-adapted%20Yeasts%3A%20Biodiversity%2C%20Adaptation%20Strategies%20and%20Biotechnological%20Significance&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Cold-adapted%20Yeasts%3A%20Biodiversity%2C%20Adaptation%20Strategies%20and%20Biotechnological%20Significance&amp;aulast=Zalar&amp;date=2014&amp;spage=49&amp;epage=74" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cold-adapted%20Yeasts%3A%20Biodiversity%2C%20Adaptation%20Strategies%20and%20Biotechnological%20Significance&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Cold-adapted%20Yeasts%3A%20Biodiversity%2C%20Adaptation%20Strategies%20and%20Biotechnological%20Significance">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wirth</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Goldani</div> <div class="given-names">L. Z.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Epidemiology of Rhodotorula: an emerging pathogen</div>, <div class="source">Interdiscip. Perspect. Infect. Dis</div> ., <div class="volume">2012</div>, <div class="fpage">1</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Epidemiology%20of%20Rhodotorula%3A%20an%20emerging%20pathogen&amp;author=F.%20Wirth&amp;author=L.%20Z.%20Goldani&amp;publication_year=2012&amp;journal=Interdiscip.%20Perspect.%20Infect.%20Dis&amp;volume=2012&amp;pages=1" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1155/2012/465717" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2012%2F465717" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1155%2F2012%2F465717"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Epidemiology%20of%20Rhodotorula%3A%20an%20emerging%20pathogen&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Xin</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Glawe</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Doty</div> <div class="given-names">S. L.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Characterization of three endophytic, indole-3-acetic acid-producing yeasts occurring in Populus trees</div>, <div class="source">Mycol. Res</div> ., <div class="volume">113</div>, <div class="fpage">973</div>–<div class="lpage">80</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20three%20endophytic%2C%20indole-3-acetic%20acid-producing%20yeasts%20occurring%20in%20Populus%20trees&amp;author=G.%20Xin&amp;author=D.%20Glawe&amp;author=S.%20L.%20Doty&amp;publication_year=2009&amp;journal=Mycol.%20Res&amp;volume=113&amp;pages=973-80" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.mycres.2009.06.001" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mycres.2009.06.001" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mycres.2009.06.001"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19539760" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterization%20of%20three%20endophytic%2C%20indole-3-acetic%20acid-producing%20yeasts%20occurring%20in%20Populus%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Knoth</div> <div class="given-names">J. L.</div>
</div>, <div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">S. H.</div>
</div>, <div class="name string-name">
<div class="surname">Ettl</div> <div class="given-names">G. J.</div>
</div>, <div class="name string-name">
<div class="surname">Doty</div> <div class="given-names">S. L.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Biological nitrogen fixation and biomass accumulation within poplar clones as a result of inoculations with diazotrophic endophyte consortia</div>, <div class="source">New Phytol</div> ., <div class="volume">201</div>, <div class="fpage">599</div>–<div class="lpage">609</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Biological%20nitrogen%20fixation%20and%20biomass%20accumulation%20within%20poplar%20clones%20as%20a%20result%20of%20inoculations%20with%20diazotrophic%20endophyte%20consortia&amp;author=J.%20L.%20Knoth&amp;author=S.%20H.%20Kim&amp;author=G.%20J.%20Ettl&amp;author=S.%20L.%20Doty&amp;publication_year=2014&amp;journal=New%20Phytol&amp;volume=201&amp;pages=599-609" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.12536" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12536" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.12536"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24117518" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biological%20nitrogen%20fixation%20and%20biomass%20accumulation%20within%20poplar%20clones%20as%20a%20result%20of%20inoculations%20with%20diazotrophic%20endophyte%20consortia&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Nailya Akhtyamova</div> <div class="given-names">R. K. S.</div>
</div></div> <div class="year">2013</div>, 
<div class="article-title">Endophytic yeast Rhodotorula rubra strain TG-1: antagonistic and plant protection activities</div>, <div class="source">Biochem. Physiol</div> ., <div class="volume">2</div>,<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Endophytic%20yeast%20Rhodotorula%20rubra%20strain%20TG-1%3A%20antagonistic%20and%20plant%20protection%20activities&amp;aulast=Nailya%20Akhtyamova&amp;date=2013&amp;volume=2" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Endophytic%20yeast%20Rhodotorula%20rubra%20strain%20TG-1%3A%20antagonistic%20and%20plant%20protection%20activities&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sampaio</div> <div class="given-names">J. P.</div>
</div>, <div class="name string-name">
<div class="surname">Gadanho</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Bauer</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Weiß</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">Taxonomic studies in the Microbotryomycetidae: leucosporidium golubevii sp. nov., Leucosporidiella gen. nov. and the new orders Leucosporidiales and Sporidiobolales</div>, <div class="source">Mycol. Prog</div> ., <div class="volume">2</div>, <div class="fpage">53</div>–<div class="lpage">68</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Taxonomic%20studies%20in%20the%20Microbotryomycetidae%3A%20leucosporidium%20golubevii%20sp.%20nov.%2C%20Leucosporidiella%20gen.%20nov.%20and%20the%20new%20orders%20Leucosporidiales%20and%20Sporidiobolales&amp;author=J.%20P.%20Sampaio&amp;author=M.%20Gadanho&amp;author=R.%20Bauer&amp;author=M.%20Wei%C3%9F&amp;publication_year=2003&amp;journal=Mycol.%20Prog&amp;volume=2&amp;pages=53-68" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s11557-006-0044-5" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11557-006-0044-5" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs11557-006-0044-5"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Taxonomic%20studies%20in%20the%20Microbotryomycetidae%3A%20leucosporidium%20golubevii%20sp.%20nov.%2C%20Leucosporidiella%20gen.%20nov.%20and%20the%20new%20orders%20Leucosporidiales%20and%20Sporidiobolales&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Fell</div> <div class="given-names">J. W.</div>
</div>, <div class="name string-name">
<div class="surname">Boekhout</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Fonseca</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Sampaio</div> <div class="given-names">J. P.</div>
</div>
</div> <div class="year">2001</div>, Basidiomycetous Yeasts. In: <div class="person-group">
<div class="name string-name">
<div class="surname">McLaughlin</div> <div class="given-names">D. J.</div>
</div>, <div class="name string-name">
<div class="surname">McLaughlin</div> <div class="given-names">E. G.</div>
</div>, <div class="name string-name">
<div class="surname">Lemke</div> <div class="given-names">P. A.</div>
</div>
</div> (eds), <div class="source">Systematics and Evolution</div> , 
<div class="publisher-name">Springer Berlin Heidelberg</div>: 
<div class="publisher-loc">Berlin, Heidelberg</div>, pp. <div class="fpage">3</div>–<div class="lpage">35</div>.<div class="citation-links">
<div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Systematics%20and%20Evolution&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Systematics%20and%20Evolution&amp;aulast=Fell&amp;date=2001&amp;spage=3&amp;epage=35" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Systematics%20and%20Evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>
<div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Systematics%20and%20Evolution">COPAC</a></span></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Morrow</div> <div class="given-names">C. A.</div>
</div>, <div class="name string-name">
<div class="surname">Fraser</div> <div class="given-names">J. A.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Sexual reproduction and dimorphism in the pathogenic basidiomycetes</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">9</div>, <div class="fpage">161</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sexual%20reproduction%20and%20dimorphism%20in%20the%20pathogenic%20basidiomycetes&amp;author=C.%20A.%20Morrow&amp;author=J.%20A.%20Fraser&amp;publication_year=2009&amp;journal=FEMS%20Yeast%20Res&amp;volume=9&amp;pages=161-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/fyr.2009.9.issue-2" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2009.9.issue-2" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffyr.2009.9.issue-2"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19220864" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sexual%20reproduction%20and%20dimorphism%20in%20the%20pathogenic%20basidiomycetes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Liu</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">A multi-omic map of the lipid-producing yeast Rhodosporidium toruloides</div>, <div class="source">Nat. Commun</div> ., <div class="volume">3</div>, <div class="fpage">1112</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20multi-omic%20map%20of%20the%20lipid-producing%20yeast%20Rhodosporidium%20toruloides&amp;author=Z.%20Zhu&amp;author=S.%20Zhang&amp;author=H.%20Liu&amp;publication_year=2012&amp;journal=Nat.%20Commun&amp;volume=3&amp;pages=1112" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ncomms2112" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2112" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fncomms2112"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23047670" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20multi-omic%20map%20of%20the%20lipid-producing%20yeast%20Rhodosporidium%20toruloides&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Firrincieli</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Otillar</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Salamov</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Genome sequence of the plant growth promoting endophytic yeast Rhodotorula graminis WP1</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">978</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20plant%20growth%20promoting%20endophytic%20yeast%20Rhodotorula%20graminis%20WP1&amp;author=A.%20Firrincieli&amp;author=R.%20Otillar&amp;author=A.%20Salamov&amp;publication_year=2015&amp;journal=Front.%20Microbiol&amp;volume=6&amp;pages=978" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fmicb.2015.00978" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2015.00978" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffmicb.2015.00978"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26441909" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20of%20the%20plant%20growth%20promoting%20endophytic%20yeast%20Rhodotorula%20graminis%20WP1&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goordial</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Raymond-Bouchard</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Riley</div> <div class="given-names">R.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Improved high-quality draft genome sequence of the eurypsychrophile Rhodotorula sp. JG1b, isolated from permafrost in the hyperarid upper-elevation mcmurdo dry valleys, Antarctica</div>, <div class="source">Genome Announce</div> ., <div class="volume">4</div>,<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Improved%20high-quality%20draft%20genome%20sequence%20of%20the%20eurypsychrophile%20Rhodotorula%20sp.%20JG1b%2C%20isolated%20from%20permafrost%20in%20the%20hyperarid%20upper-elevation%20mcmurdo%20dry%20valleys%2C%20Antarctica&amp;aulast=Goordial&amp;date=2016&amp;volume=4" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improved%20high-quality%20draft%20genome%20sequence%20of%20the%20eurypsychrophile%20Rhodotorula%20sp.%20JG1b%2C%20isolated%20from%20permafrost%20in%20the%20hyperarid%20upper-elevation%20mcmurdo%20dry%20valleys%2C%20Antarctica&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Deligios</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Fraumene</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Abbondio</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Draft genome sequence of Rhodotorula mucilaginosa, an emergent opportunistic pathogen</div>, <div class="source">Genome Announce</div> ., <div class="volume">3</div>,<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Draft%20genome%20sequence%20of%20Rhodotorula%20mucilaginosa%2C%20an%20emergent%20opportunistic%20pathogen&amp;aulast=Deligios&amp;date=2015&amp;volume=3" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Draft%20genome%20sequence%20of%20Rhodotorula%20mucilaginosa%2C%20an%20emergent%20opportunistic%20pathogen&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Saha</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Seal</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Early changes in shoot transcriptome of rice in response to Rhodotorula mucilaginosa JGTA-S1</div>, <div class="source">Genome Data</div> , <div class="volume">6</div>, <div class="fpage">237</div>–<div class="lpage">40</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Early%20changes%20in%20shoot%20transcriptome%20of%20rice%20in%20response%20to%20Rhodotorula%20mucilaginosa%20JGTA-S1&amp;author=C.%20Saha&amp;author=A.%20Seal&amp;publication_year=2015&amp;journal=Genome%20Data&amp;volume=6&amp;pages=237-40" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gdata.2015.09.023" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gdata.2015.09.023" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gdata.2015.09.023"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Early%20changes%20in%20shoot%20transcriptome%20of%20rice%20in%20response%20to%20Rhodotorula%20mucilaginosa%20JGTA-S1&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Loman</div> <div class="given-names">N. J.</div>
</div>, <div class="name string-name">
<div class="surname">Quinlan</div> <div class="given-names">A. R.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Poretools: a toolkit for analyzing nanopore sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3399</div>–<div class="lpage">401</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Poretools%3A%20a%20toolkit%20for%20analyzing%20nanopore%20sequence%20data&amp;author=N.%20J.%20Loman&amp;author=A.%20R.%20Quinlan&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3399-401" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu555" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu555" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu555"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25143291" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Poretools%3A%20a%20toolkit%20for%20analyzing%20nanopore%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bolger</div> <div class="given-names">A. M.</div>
</div>, <div class="name string-name">
<div class="surname">Lohse</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Usadel</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu170"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Koren</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Walenz</div> <div class="given-names">B. P.</div>
</div>, <div class="name string-name">
<div class="surname">Berlin</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Miller</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">Bergman</div> <div class="given-names">N. H.</div>
</div>, <div class="name string-name">
<div class="surname">Phillippy</div> <div class="given-names">A. M.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation</div>, <div class="source">Genome Res</div> ., <div class="volume">27</div>, <div class="fpage">722</div>–<div class="lpage">36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&amp;author=S.%20Koren&amp;author=B.%20P.%20Walenz&amp;author=K.%20Berlin&amp;author=J.%20R.%20Miller&amp;author=N.%20H.%20Bergman&amp;author=A.%20M.%20Phillippy&amp;publication_year=2017&amp;journal=Genome%20Res&amp;volume=27&amp;pages=722-36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.215087.116" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.215087.116" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.215087.116"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28298431" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bankevich</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Nurk</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Antipov</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</div>, <div class="source">J. Comput. Biol</div> ., <div class="volume">19</div>, <div class="fpage">455</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=SPAdes%3A%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;author=A.%20Bankevich&amp;author=S.%20Nurk&amp;author=D.%20Antipov&amp;publication_year=2012&amp;journal=J.%20Comput.%20Biol&amp;volume=19&amp;pages=455-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1089/cmb.2012.0021" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2012.0021" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1089%2Fcmb.2012.0021"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22506599" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:SPAdes%3A%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Walker</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Abeel</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Shea</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e112963</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;author=B.%20J.%20Walker&amp;author=T.%20Abeel&amp;author=T.%20Shea&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e112963" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0112963" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112963" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0112963"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gurevich</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Saveliev</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Vyahhi</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Tesler</div> <div class="given-names">G.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">QUAST: quality assessment tool for genome assemblies</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">1072</div>–<div class="lpage">5</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=QUAST%3A%20quality%20assessment%20tool%20for%20genome%20assemblies&amp;author=A.%20Gurevich&amp;author=V.%20Saveliev&amp;author=N.%20Vyahhi&amp;author=G.%20Tesler&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=1072-5" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt086" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt086" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt086"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23422339" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:QUAST%3A%20quality%20assessment%20tool%20for%20genome%20assemblies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Delcher</div> <div class="given-names">A. L.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Phillippy</div> <div class="given-names">A. M.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">Using MUMmer to identify similar regions in large sequence sets</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <strong>Chapter 10,</strong> Unit 10.3.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Using%20MUMmer%20to%20identify%20similar%20regions%20in%20large%20sequence%20sets&amp;aulast=Delcher&amp;date=2003" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20MUMmer%20to%20identify%20similar%20regions%20in%20large%20sequence%20sets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Stanke</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Morgenstern</div> <div class="given-names">B.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">W465</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20a%20web%20server%20for%20gene%20prediction%20in%20eukaryotes%20that%20allows%20user-defined%20constraints&amp;author=M.%20Stanke&amp;author=B.%20Morgenstern&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W465-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gki458" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki458" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki458"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980513" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:AUGUSTUS%3A%20a%20web%20server%20for%20gene%20prediction%20in%20eukaryotes%20that%20allows%20user-defined%20constraints&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Conesa</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Gotz</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2008</div>, 
<div class="article-title">Blast2GO: a comprehensive suite for functional analysis in plant genomics</div>, <div class="source">Int. J. Plant Genomics</div> , <div class="volume">2008</div>, <div class="fpage">1</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20comprehensive%20suite%20for%20functional%20analysis%20in%20plant%20genomics&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;publication_year=2008&amp;journal=Int.%20J.%20Plant%20Genomics&amp;volume=2008&amp;pages=1" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1155/2008/619832" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2008%2F619832" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1155%2F2008%2F619832"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Blast2GO%3A%20a%20comprehensive%20suite%20for%20functional%20analysis%20in%20plant%20genomics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Quevillon</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Silventoinen</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Pillai</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2005</div>, 
<div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki442"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:InterProScan%3A%20protein%20domains%20identifier&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Gregory</div> <div class="given-names">R. W.</div>
</div>, <div class="name string-name">
<div class="surname">Bolker</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Bonebakker</div> <div class="given-names">L.</div>
</div>
</div>, et al. <div class="year">2016</div>, gplots: Various R Programming Tools for Plotting Data. R package version 3.0.1. Available at: <a class="link link-uri" href="https://CRAN.R-project.org/package=gplots" target="_blank">https://CRAN.R-project.org/package=gplots</a> (accessed June 2018).</div>
</div></div></div>
<div content-id="dsy044-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Stoeckert</div> <div class="given-names">C. J.</div>Jr</div>, <div class="name string-name">
<div class="surname">Roos</div> <div class="given-names">D. S.</div>
</div>
</div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.%20J.%20Stoeckert&amp;author=D.%20S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.1224503"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">L.-G.</div>
</div>, <div class="name string-name">
<div class="surname">Han</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">He</div> <div class="given-names">Q.-Y.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">clusterProfiler: an R package for comparing biological themes among gene clusters</div>, <div class="source">OMICS</div> , <div class="volume">16</div>, <div class="fpage">284</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=clusterProfiler%3A%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters&amp;author=G.%20Yu&amp;author=L.-G.%20Wang&amp;author=Y.%20Han&amp;author=Q.-Y.%20He&amp;publication_year=2012&amp;journal=OMICS&amp;volume=16&amp;pages=284-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1089/omi.2011.0118" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fomi.2011.0118" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1089%2Fomi.2011.0118"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22455463" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:clusterProfiler%3A%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Tang</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">DeBarry</div> <div class="given-names">J. D.</div>
</div>
</div>, et al.  <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20DeBarry&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1293"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Haas</div> <div class="given-names">B. J.</div>
</div>, <div class="name string-name">
<div class="surname">Delcher</div> <div class="given-names">A. L.</div>
</div>, <div class="name string-name">
<div class="surname">Wortman</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">Salzberg</div> <div class="given-names">S. L.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">DAGchainer: a tool for mining segmental genome duplications and synteny</div>, <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">3643</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DAGchainer%3A%20a%20tool%20for%20mining%20segmental%20genome%20duplications%20and%20synteny&amp;author=B.%20J.%20Haas&amp;author=A.%20L.%20Delcher&amp;author=J.%20R.%20Wortman&amp;author=S.%20L.%20Salzberg&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=3643-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bth397" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth397" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbth397"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247098" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DAGchainer%3A%20a%20tool%20for%20mining%20segmental%20genome%20duplications%20and%20synteny&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Thompson</div> <div class="given-names">J. D.</div>
</div>, <div class="name string-name">
<div class="surname">Gibson</div> <div class="given-names">T. J.</div>
</div>, <div class="name string-name">
<div class="surname">Higgins</div> <div class="given-names">D. G.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">Multiple sequence alignment using ClustalW and ClustalX</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <strong>Chapter 2</strong>, Unit 2.3.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Multiple%20sequence%20alignment%20using%20ClustalW%20and%20ClustalX&amp;aulast=Thompson&amp;date=2002" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Multiple%20sequence%20alignment%20using%20ClustalW%20and%20ClustalX&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hanisch</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Zimmer</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Lengauer</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">ProML–the protein markup language for specification of protein sequences, structures and families</div>, <div class="source">In Silico Biol</div> ., <div class="volume">2</div>, <div class="fpage">313</div>–<div class="lpage">24</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ProML%E2%80%93the%20protein%20markup%20language%20for%20specification%20of%20protein%20sequences%2C%20structures%20and%20families&amp;author=D.%20Hanisch&amp;author=R.%20Zimmer&amp;author=T.%20Lengauer&amp;publication_year=2002&amp;journal=In%20Silico%20Biol&amp;volume=2&amp;pages=313-24" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12542416" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=ProML%E2%80%93the%20protein%20markup%20language%20for%20specification%20of%20protein%20sequences%2C%20structures%20and%20families&amp;aulast=Hanisch&amp;date=2002&amp;spage=313&amp;epage=24&amp;volume=2" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ProML%E2%80%93the%20protein%20markup%20language%20for%20specification%20of%20protein%20sequences%2C%20structures%20and%20families&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bendtsen</div> <div class="given-names">J. D.</div>
</div>, <div class="name string-name">
<div class="surname">Nielsen</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">von Heijne</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Brunak</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2004</div>, 
<div class="article-title">Improved prediction of signal peptides: signalP 3.0</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">340</div>, <div class="fpage">783</div>–<div class="lpage">95</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improved%20prediction%20of%20signal%20peptides%3A%20signalP%203.0&amp;author=J.%20D.%20Bendtsen&amp;author=H.%20Nielsen&amp;author=G.%20von%20Heijne&amp;author=S.%20Brunak&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol&amp;volume=340&amp;pages=783-95" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.jmb.2004.05.028" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2004.05.028" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jmb.2004.05.028"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15223320" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improved%20prediction%20of%20signal%20peptides%3A%20signalP%203.0&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lopez-Bucio</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Hernandez-Abreu</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Sanchez-Calderon</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Nieto-Jacobo</div> <div class="given-names">M. F.</div>
</div>, <div class="name string-name">
<div class="surname">Simpson</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Herrera-Estrella</div> <div class="given-names">L.</div>
</div>
</div> <div class="year">2002</div>, 
<div class="article-title">Phosphate availability alters architecture and causes changes in hormone sensitivity in the Arabidopsis root system</div>, <div class="source">Plant Physiol</div> ., <div class="volume">129</div>, <div class="fpage">244</div>–<div class="lpage">56</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phosphate%20availability%20alters%20architecture%20and%20causes%20changes%20in%20hormone%20sensitivity%20in%20the%20Arabidopsis%20root%20system&amp;author=J.%20Lopez-Bucio&amp;author=E.%20Hernandez-Abreu&amp;author=L.%20Sanchez-Calderon&amp;author=M.%20F.%20Nieto-Jacobo&amp;author=J.%20Simpson&amp;author=L.%20Herrera-Estrella&amp;publication_year=2002&amp;journal=Plant%20Physiol&amp;volume=129&amp;pages=244-56" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1104/pp.010934" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.010934" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1104%2Fpp.010934"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011355" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phosphate%20availability%20alters%20architecture%20and%20causes%20changes%20in%20hormone%20sensitivity%20in%20the%20Arabidopsis%20root%20system&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ye</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Coulouris</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Zaretskaya</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Cutcutache</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Rozen</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Madden</div> <div class="given-names">T. L.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">13</div>, <div class="fpage">134</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Primer-BLAST%3A%20a%20tool%20to%20design%20target-specific%20primers%20for%20polymerase%20chain%20reaction&amp;author=J.%20Ye&amp;author=G.%20Coulouris&amp;author=I.%20Zaretskaya&amp;author=I.%20Cutcutache&amp;author=S.%20Rozen&amp;author=T.%20L.%20Madden&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=134" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-13-134" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-134" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-13-134"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22708584" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Primer-BLAST%3A%20a%20tool%20to%20design%20target-specific%20primers%20for%20polymerase%20chain%20reaction&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Toome</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Ohm</div> <div class="given-names">R. A.</div>
</div>, <div class="name string-name">
<div class="surname">Riley</div> <div class="given-names">R. W.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome sequencing provides insight into the reproductive biology, nutritional mode and ploidy of the fern pathogen Mixia osmundae</div>, <div class="source">New Phytol</div> ., <div class="volume">202</div>, <div class="fpage">554</div>–<div class="lpage">64</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20provides%20insight%20into%20the%20reproductive%20biology%2C%20nutritional%20mode%20and%20ploidy%20of%20the%20fern%20pathogen%20Mixia%20osmundae&amp;author=M.%20Toome&amp;author=R.%20A.%20Ohm&amp;author=R.%20W.%20Riley&amp;publication_year=2014&amp;journal=New%20Phytol&amp;volume=202&amp;pages=554-64" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.12653" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12653" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.12653"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24372469" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequencing%20provides%20insight%20into%20the%20reproductive%20biology%2C%20nutritional%20mode%20and%20ploidy%20of%20the%20fern%20pathogen%20Mixia%20osmundae&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B36" class="ref-content ">
<div class="label">36</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Goordial</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Davila</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Lacelle</div> <div class="given-names">D.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Nearing the cold-arid limits of microbial life in permafrost of an upper dry valley, Antarctica</div>, <div class="source">Isme J</div> ., <div class="volume">10</div>, <div class="fpage">1613</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Nearing%20the%20cold-arid%20limits%20of%20microbial%20life%20in%20permafrost%20of%20an%20upper%20dry%20valley%2C%20Antarctica&amp;author=J.%20Goordial&amp;author=A.%20Davila&amp;author=D.%20Lacelle&amp;publication_year=2016&amp;journal=Isme%20J&amp;volume=10&amp;pages=1613" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ismej.2015.239" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2015.239" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fismej.2015.239"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27323892" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Nearing%20the%20cold-arid%20limits%20of%20microbial%20life%20in%20permafrost%20of%20an%20upper%20dry%20valley%2C%20Antarctica&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B37" class="ref-content ">
<div class="label">37</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Banno</div> <div class="given-names">I.</div>
</div></div> <div class="year">1967</div>, 
<div class="article-title">Studies on the sexuality of Rhodotorula</div>. <div class="source">The Journal of General and Applied Microbiology</div> , <div class="volume">13</div>, <div class="fpage">167</div>–<div class="lpage">196</div>.</div>
</div></div></div>
<div content-id="dsy044-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B38" class="ref-content ">
<div class="label">38</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Muszewska</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Hoffman-Sommer</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Grynberg</div> <div class="given-names">M.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">LTR Retrotransposons in Fungi</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e29425</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=LTR%20Retrotransposons%20in%20Fungi&amp;author=A.%20Muszewska&amp;author=M.%20Hoffman-Sommer&amp;author=M.%20Grynberg&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e29425" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0029425" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029425" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0029425"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22242120" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:LTR%20Retrotransposons%20in%20Fungi&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B39" class="ref-content ">
<div class="label">39</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Das</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rahman</div> <div class="given-names">M. I.</div>
</div>, <div class="name string-name">
<div class="surname">Ferdous</div> <div class="given-names">A. S.</div>
</div>
</div>, et al.  <div class="year">2017</div>, 
<div class="article-title">An endophytic Basidiomycete, Grammothele lineata, isolated from Corchorus olitorius, produces paclitaxel that shows cytotoxicity</div>, <div class="source">PLoS One</div> , <div class="volume">12</div>, <div class="fpage">e0178612</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20endophytic%20Basidiomycete%2C%20Grammothele%20lineata%2C%20isolated%20from%20Corchorus%20olitorius%2C%20produces%20paclitaxel%20that%20shows%20cytotoxicity&amp;author=A.%20Das&amp;author=M.%20I.%20Rahman&amp;author=A.%20S.%20Ferdous&amp;publication_year=2017&amp;journal=PLoS%20One&amp;volume=12&amp;pages=e0178612" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0178612" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0178612" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0178612"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28636663" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20endophytic%20Basidiomycete%2C%20Grammothele%20lineata%2C%20isolated%20from%20Corchorus%20olitorius%2C%20produces%20paclitaxel%20that%20shows%20cytotoxicity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B40" class="ref-content ">
<div class="label">40</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Verma</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Varma</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rexer</div> <div class="given-names">K.-H.</div>
</div>
</div>, et al.  <div class="year">1998</div>, 
<div class="article-title">Piriformospora indica, gen. et sp. nov., a new root-colonizing fungus</div>, <div class="source">Mycologia</div> , <div class="volume">90</div>, <div class="fpage">896</div>–<div class="lpage">903</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Piriformospora%20indica%2C%20gen.%20et%20sp.%20nov.%2C%20a%20new%20root-colonizing%20fungus&amp;author=S.%20Verma&amp;author=A.%20Varma&amp;author=K.-H.%20Rexer&amp;publication_year=1998&amp;journal=Mycologia&amp;volume=90&amp;pages=896-903" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/00275514.1998.12026983" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00275514.1998.12026983" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F00275514.1998.12026983"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Piriformospora%20indica%2C%20gen.%20et%20sp.%20nov.%2C%20a%20new%20root-colonizing%20fungus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B41" class="ref-content ">
<div class="label">41</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Firrincieli</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Otillar</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Salamov</div> <div class="given-names">A.</div>
</div>
</div>, et al. <div class="year">2015</div>, 
<div class="article-title">Genome sequence of the plant growth promoting endophytic yeast Rhodotorula graminis WP1</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, 978. <div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Genome%20sequence%20of%20the%20plant%20growth%20promoting%20endophytic%20yeast%20Rhodotorula%20graminis%20WP1&amp;aulast=Firrincieli&amp;date=2015&amp;volume=6" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome%20sequence%20of%20the%20plant%20growth%20promoting%20endophytic%20yeast%20Rhodotorula%20graminis%20WP1&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B42" class="ref-content ">
<div class="label">42</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kamper</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Kahmann</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Bolker</div> <div class="given-names">M.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">Insights from the genome of the biotrophic fungal plant pathogen Ustilago maydis</div>, <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">97</div>–<div class="lpage">101</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insights%20from%20the%20genome%20of%20the%20biotrophic%20fungal%20plant%20pathogen%20Ustilago%20maydis&amp;author=J.%20Kamper&amp;author=R.%20Kahmann&amp;author=M.%20Bolker&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=97-101" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature05248" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05248" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature05248"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17080091" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insights%20from%20the%20genome%20of%20the%20biotrophic%20fungal%20plant%20pathogen%20Ustilago%20maydis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B43" class="ref-content ">
<div class="label">43</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cantu</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Govindarajulu</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Kozik</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Next Generation sequencing provides rapid access to the genome of Puccinia striiformis f. sp. tritici, the causal agent of wheat stripe rust</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e24230</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Next%20Generation%20sequencing%20provides%20rapid%20access%20to%20the%20genome%20of%20Puccinia%20striiformis%20f.%20sp.%20tritici%2C%20the%20causal%20agent%20of%20wheat%20stripe%20rust&amp;author=D.%20Cantu&amp;author=M.%20Govindarajulu&amp;author=A.%20Kozik&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e24230" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0024230" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0024230" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0024230"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21909385" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Next%20Generation%20sequencing%20provides%20rapid%20access%20to%20the%20genome%20of%20Puccinia%20striiformis%20f.%20sp.%20tritici%2C%20the%20causal%20agent%20of%20wheat%20stripe%20rust&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B44" class="ref-content ">
<div class="label">44</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Persoons</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Morin</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Delaruelle</div> <div class="given-names">C.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Patterns of genomic variation in the poplar rust fungus Melampsora larici-populina identify pathogenesis-related factors</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>,<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Patterns%20of%20genomic%20variation%20in%20the%20poplar%20rust%20fungus%20Melampsora%20larici-populina%20identify%20pathogenesis-related%20factors&amp;aulast=Persoons&amp;date=2014&amp;volume=5" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Patterns%20of%20genomic%20variation%20in%20the%20poplar%20rust%20fungus%20Melampsora%20larici-populina%20identify%20pathogenesis-related%20factors&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B45" class="ref-content ">
<div class="label">45</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Perlin</div> <div class="given-names">M. H.</div>
</div>, <div class="name string-name">
<div class="surname">Amselem</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Fontanillas</div> <div class="given-names">E.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Sex and parasites: genomic and transcriptomic analysis of Microbotryum lychnidis-dioicae, the biotrophic and plant-castrating anther smut fungus</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">461</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sex%20and%20parasites%3A%20genomic%20and%20transcriptomic%20analysis%20of%20Microbotryum%20lychnidis-dioicae%2C%20the%20biotrophic%20and%20plant-castrating%20anther%20smut%20fungus&amp;author=M.%20H.%20Perlin&amp;author=J.%20Amselem&amp;author=E.%20Fontanillas&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=461" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12864-015-1660-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1660-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12864-015-1660-8"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076695" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sex%20and%20parasites%3A%20genomic%20and%20transcriptomic%20analysis%20of%20Microbotryum%20lychnidis-dioicae%2C%20the%20biotrophic%20and%20plant-castrating%20anther%20smut%20fungus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B46" class="ref-content ">
<div class="label">46</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Andrianopoulos</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Kourambas</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Sharp</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Davis</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Hynes</div> <div class="given-names">M. J.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Characterization of the Aspergillus nidulans nmrA gene involved in nitrogen metabolite repression</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">180</div>, <div class="fpage">1973</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20Aspergillus%20nidulans%20nmrA%20gene%20involved%20in%20nitrogen%20metabolite%20repression&amp;author=A.%20Andrianopoulos&amp;author=S.%20Kourambas&amp;author=J.%20A.%20Sharp&amp;author=M.%20A.%20Davis&amp;author=M.%20J.%20Hynes&amp;publication_year=1998&amp;journal=J.%20Bacteriol&amp;volume=180&amp;pages=1973-7" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9537404" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Characterization%20of%20the%20Aspergillus%20nidulans%20nmrA%20gene%20involved%20in%20nitrogen%20metabolite%20repression&amp;aulast=Andrianopoulos&amp;date=1998&amp;spage=1973&amp;epage=7&amp;volume=180" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterization%20of%20the%20Aspergillus%20nidulans%20nmrA%20gene%20involved%20in%20nitrogen%20metabolite%20repression&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B47" class="ref-content ">
<div class="label">47</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Brefort</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Tanaka</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Neidig</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Doehlemann</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Vincon</div> <div class="given-names">V.</div>
</div>, <div class="name string-name">
<div class="surname">Kahmann</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Characterization of the largest effector gene cluster of Ustilago maydis</div>, <div class="source">PLoS Pathog</div> ., <div class="volume">10</div>, <div class="fpage">e1003866</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20largest%20effector%20gene%20cluster%20of%20Ustilago%20maydis&amp;author=T.%20Brefort&amp;author=S.%20Tanaka&amp;author=N.%20Neidig&amp;author=G.%20Doehlemann&amp;author=V.%20Vincon&amp;author=R.%20Kahmann&amp;publication_year=2014&amp;journal=PLoS%20Pathog&amp;volume=10&amp;pages=e1003866" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.ppat.1003866" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1003866" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.ppat.1003866"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24992561" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterization%20of%20the%20largest%20effector%20gene%20cluster%20of%20Ustilago%20maydis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B48" class="ref-content ">
<div class="label">48</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Aebischer</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Harbecke</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Ilg</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">1999</div>, 
<div class="article-title">Proteophosphoglycan, a major secreted product of intracellular leishmania mexicana amastigotes, is a poor b-cell antigen and does not elicit a specific conventional CD4(+) T-cell response</div>, <div class="source">Infect. Immun</div> ., <div class="volume">67</div>, <div class="fpage">5379</div>–<div class="lpage">85</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Proteophosphoglycan%2C%20a%20major%20secreted%20product%20of%20intracellular%20leishmania%20mexicana%20amastigotes%2C%20is%20a%20poor%20b-cell%20antigen%20and%20does%20not%20elicit%20a%20specific%20conventional%20CD4(%2B)%20T-cell%20response&amp;author=T.%20Aebischer&amp;author=D.%20Harbecke&amp;author=T.%20Ilg&amp;publication_year=1999&amp;journal=Infect.%20Immun&amp;volume=67&amp;pages=5379-85" target="_blank">Google Scholar</a></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10496920" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Proteophosphoglycan%2C%20a%20major%20secreted%20product%20of%20intracellular%20leishmania%20mexicana%20amastigotes%2C%20is%20a%20poor%20b-cell%20antigen%20and%20does%20not%20elicit%20a%20specific%20conventional%20CD4(%2B)%20T-cell%20response&amp;aulast=Aebischer&amp;date=1999&amp;spage=5379&amp;epage=85&amp;volume=67" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Proteophosphoglycan%2C%20a%20major%20secreted%20product%20of%20intracellular%20leishmania%20mexicana%20amastigotes%2C%20is%20a%20poor%20b-cell%20antigen%20and%20does%20not%20elicit%20a%20specific%20conventional%20CD4(%2B)%20T-cell%20response&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B49" class="ref-content ">
<div class="label">49</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Vries</div> <div class="given-names">A. L. D.</div>
</div>, <div class="name string-name">
<div class="surname">Price</div> <div class="given-names">T. J.</div>
</div>
</div> <div class="year">1984</div>, 
<div class="article-title">Role of Glycopeptides and peptides in inhibition of crystallization of water in polar fishes [and discussion]</div>, <div class="source">Philos. Trans. R. Soc. Lond. B Biol. Sci</div> ., <div class="volume">304</div>, <div class="fpage">575</div>–<div class="lpage">88</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Role%20of%20Glycopeptides%20and%20peptides%20in%20inhibition%20of%20crystallization%20of%20water%20in%20polar%20fishes%20%5Band%20discussion%5D&amp;author=A.%20L.%20D.%20Vries&amp;author=T.%20J.%20Price&amp;publication_year=1984&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci&amp;volume=304&amp;pages=575-88" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rstb.1984.0048" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1984.0048" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frstb.1984.0048"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Role%20of%20Glycopeptides%20and%20peptides%20in%20inhibition%20of%20crystallization%20of%20water%20in%20polar%20fishes%20%5Band%20discussion%5D&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B50" class="ref-content ">
<div class="label">50</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Larroque</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Barriot</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Bottin</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Barre</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Rougé</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Dumas</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Gaulin</div> <div class="given-names">E.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">The unique architecture and function of cellulose-interacting proteins in oomycetes revealed by genomic and structural analyses</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">605</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20unique%20architecture%20and%20function%20of%20cellulose-interacting%20proteins%20in%20oomycetes%20revealed%20by%20genomic%20and%20structural%20analyses&amp;author=M.%20Larroque&amp;author=R.%20Barriot&amp;author=A.%20Bottin&amp;author=A.%20Barre&amp;author=P.%20Roug%C3%A9&amp;author=B.%20Dumas&amp;author=E.%20Gaulin&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=605" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-13-605" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-605" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-13-605"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23140525" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20unique%20architecture%20and%20function%20of%20cellulose-interacting%20proteins%20in%20oomycetes%20revealed%20by%20genomic%20and%20structural%20analyses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B51" class="ref-content ">
<div class="label">51</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Deshmukh</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Huckelhoven</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Schafer</div> <div class="given-names">P.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">The root endophytic fungus Piriformospora indica requires host cell death for proliferation during mutualistic symbiosis with barley</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">18450</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20root%20endophytic%20fungus%20Piriformospora%20indica%20requires%20host%20cell%20death%20for%20proliferation%20during%20mutualistic%20symbiosis%20with%20barley&amp;author=S.%20Deshmukh&amp;author=R.%20Huckelhoven&amp;author=P.%20Schafer&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=18450-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0605697103" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0605697103" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0605697103"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20root%20endophytic%20fungus%20Piriformospora%20indica%20requires%20host%20cell%20death%20for%20proliferation%20during%20mutualistic%20symbiosis%20with%20barley&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B52" class="ref-content ">
<div class="label">52</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Muszewska</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Pilsyk</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Perlinska-Lenart</div> <div class="given-names">U.</div>
</div>, <div class="name string-name">
<div class="surname">Kruszewska</div> <div class="given-names">J. S.</div>
</div>
</div> <div class="year">2017</div>, 
<div class="article-title">Diversity of cell wall related proteins in human pathogenic fungi</div>, <div class="source">J. Fungi (Basel</div> <em>)</em>, <div class="volume">4</div>,<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Diversity%20of%20cell%20wall%20related%20proteins%20in%20human%20pathogenic%20fungi&amp;aulast=Muszewska&amp;date=2017&amp;volume=4" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Diversity%20of%20cell%20wall%20related%20proteins%20in%20human%20pathogenic%20fungi&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B53" class="ref-content ">
<div class="label">53</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kuge</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Nagoya</div> <div class="given-names">H.</div>
</div>, <div class="name string-name">
<div class="surname">Tryfona</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Action of an endo-β-1, 3(4)-glucanase on cellobiosyl unit structure in barley β-1, 3: 1, 4-glucan</div>, <div class="source">Biosci. Biotechnol. Biochem</div> ., <div class="volume">79</div>, <div class="fpage">1810</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Action%20of%20an%20endo-%CE%B2-1%2C%203(4)-glucanase%20on%20cellobiosyl%20unit%20structure%20in%20barley%20%CE%B2-1%2C%203%3A%201%2C%204-glucan&amp;author=T.%20Kuge&amp;author=H.%20Nagoya&amp;author=T.%20Tryfona&amp;publication_year=2015&amp;journal=Biosci.%20Biotechnol.%20Biochem&amp;volume=79&amp;pages=1810-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/09168451.2015.1046365" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F09168451.2015.1046365" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F09168451.2015.1046365"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26027730" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Action%20of%20an%20endo-%CE%B2-1%2C%203(4)-glucanase%20on%20cellobiosyl%20unit%20structure%20in%20barley%20%CE%B2-1%2C%203%3A%201%2C%204-glucan&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B54" class="ref-content ">
<div class="label">54</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cord-Landwehr</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Melcher</div> <div class="given-names">R. L.</div>
</div>, <div class="name string-name">
<div class="surname">Kolkenbrock</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Moerschbacher</div> <div class="given-names">B. M.</div>
</div>
</div> <div class="year">2016</div>, 
<div class="article-title">A chitin deacetylase from the endophytic fungus Pestalotiopsis sp. efficiently inactivates the elicitor activity of chitin oligomers in rice cells</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">38018</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20chitin%20deacetylase%20from%20the%20endophytic%20fungus%20Pestalotiopsis%20sp.%20efficiently%20inactivates%20the%20elicitor%20activity%20of%20chitin%20oligomers%20in%20rice%20cells&amp;author=S.%20Cord-Landwehr&amp;author=R.%20L.%20Melcher&amp;author=S.%20Kolkenbrock&amp;author=B.%20M.%20Moerschbacher&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=38018" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep38018" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep38018" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep38018"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27901067" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20chitin%20deacetylase%20from%20the%20endophytic%20fungus%20Pestalotiopsis%20sp.%20efficiently%20inactivates%20the%20elicitor%20activity%20of%20chitin%20oligomers%20in%20rice%20cells&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B55" class="ref-content ">
<div class="label">55</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kombrink</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Thomma</div> <div class="given-names">B. P. H. J.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">LysM effectors: secreted proteins supporting fungal life</div>, <div class="source">PLOS Pathog</div> ., <div class="volume">9</div>, <div class="fpage">e1003769</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=LysM%20effectors%3A%20secreted%20proteins%20supporting%20fungal%20life&amp;author=A.%20Kombrink&amp;author=B.%20P.%20H.%20J.%20Thomma&amp;publication_year=2013&amp;journal=PLOS%20Pathog&amp;volume=9&amp;pages=e1003769" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.ppat.1003769" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1003769" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.ppat.1003769"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24348247" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:LysM%20effectors%3A%20secreted%20proteins%20supporting%20fungal%20life&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B56" class="ref-content ">
<div class="label">56</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">H.</div>
</div></div> <div class="year">2009</div>, 
<div class="article-title">Dissection of salicylic acid-mediated defense signaling networks</div>, <div class="source">Plant Signal. Behav</div> ., <div class="volume">4</div>, <div class="fpage">713</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20salicylic%20acid-mediated%20defense%20signaling%20networks&amp;author=H.%20Lu&amp;publication_year=2009&amp;journal=Plant%20Signal.%20Behav&amp;volume=4&amp;pages=713-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.4161/psb.4.8.9173" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.4.8.9173" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.4161%2Fpsb.4.8.9173"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19820324" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dissection%20of%20salicylic%20acid-mediated%20defense%20signaling%20networks&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B57" class="ref-content ">
<div class="label">57</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rabe</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Ajami-Rashidi</div> <div class="given-names">Z.</div>
</div>, <div class="name string-name">
<div class="surname">Doehlemann</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Kahmann</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Djamei</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Degradation of the plant defence hormone salicylic acid by the biotrophic fungus Ustilago maydis</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">89</div>, <div class="fpage">179</div>–<div class="lpage">88</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Degradation%20of%20the%20plant%20defence%20hormone%20salicylic%20acid%20by%20the%20biotrophic%20fungus%20Ustilago%20maydis&amp;author=F.%20Rabe&amp;author=Z.%20Ajami-Rashidi&amp;author=G.%20Doehlemann&amp;author=R.%20Kahmann&amp;author=A.%20Djamei&amp;publication_year=2013&amp;journal=Mol.%20Microbiol&amp;volume=89&amp;pages=179-88" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/mmi.2013.89.issue-1" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2013.89.issue-1" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmmi.2013.89.issue-1"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23692401" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Degradation%20of%20the%20plant%20defence%20hormone%20salicylic%20acid%20by%20the%20biotrophic%20fungus%20Ustilago%20maydis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B58" class="ref-content ">
<div class="label">58</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Niki</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Mitsuhara</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Seo</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Ohtsubo</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Ohashi</div> <div class="given-names">Y.</div>
</div>
</div> <div class="year">1998</div>, 
<div class="article-title">Antagonistic effect of salicylic acid and jasmonic acid on the expression of pathogenesis-related (PR) protein genes in wounded mature tobacco leaves</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">39</div>, <div class="fpage">500</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Antagonistic%20effect%20of%20salicylic%20acid%20and%20jasmonic%20acid%20on%20the%20expression%20of%20pathogenesis-related%20(PR)%20protein%20genes%20in%20wounded%20mature%20tobacco%20leaves&amp;author=T.%20Niki&amp;author=I.%20Mitsuhara&amp;author=S.%20Seo&amp;author=N.%20Ohtsubo&amp;author=Y.%20Ohashi&amp;publication_year=1998&amp;journal=Plant%20Cell%20Physiol&amp;volume=39&amp;pages=500-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.pcp.a029397" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.pcp.a029397" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.pcp.a029397"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Antagonistic%20effect%20of%20salicylic%20acid%20and%20jasmonic%20acid%20on%20the%20expression%20of%20pathogenesis-related%20(PR)%20protein%20genes%20in%20wounded%20mature%20tobacco%20leaves&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B59" class="ref-content ">
<div class="label">59</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Mou</div> <div class="given-names">H.-Q.</div>
</div>, <div class="name string-name">
<div class="surname">Lu</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Zhu</div> <div class="given-names">S.-F.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Transcriptomic analysis of paulownia infected by Paulownia Witches’-broom phytoplasma</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e77217</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20of%20paulownia%20infected%20by%20Paulownia%20Witches%E2%80%99-broom%20phytoplasma&amp;author=H.-Q.%20Mou&amp;author=J.%20Lu&amp;author=S.-F.%20Zhu&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e77217" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0077217" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0077217" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0077217"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24130859" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Transcriptomic%20analysis%20of%20paulownia%20infected%20by%20Paulownia%20Witches%E2%80%99-broom%20phytoplasma&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B60" class="ref-content ">
<div class="label">60</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Oteino</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Lally</div> <div class="given-names">R. D.</div>
</div>, <div class="name string-name">
<div class="surname">Kiwanuka</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Plant growth promotion induced by phosphate solubilizing endophytic Pseudomonas isolates</div>, <div class="source">Front. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">745</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Plant%20growth%20promotion%20induced%20by%20phosphate%20solubilizing%20endophytic%20Pseudomonas%20isolates&amp;author=N.%20Oteino&amp;author=R.%20D.%20Lally&amp;author=S.%20Kiwanuka&amp;publication_year=2015&amp;journal=Front.%20Microbiol&amp;volume=6&amp;pages=745" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.3389/fmicb.2015.00745" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2015.00745" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.3389%2Ffmicb.2015.00745"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26257721" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Plant%20growth%20promotion%20induced%20by%20phosphate%20solubilizing%20endophytic%20Pseudomonas%20isolates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B61" class="ref-content ">
<div class="label">61</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rafiqi</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Jelonek</div> <div class="given-names">L.</div>
</div>, <div class="name string-name">
<div class="surname">Akum</div> <div class="given-names">N.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">F.</div>
</div>, <div class="name string-name">
<div class="surname">Kogel</div> <div class="given-names">K.-H.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">Effector candidates in the secretome of Piriformospora indica, a ubiquitous plant-associated fungus</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">4</div>,<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Effector%20candidates%20in%20the%20secretome%20of%20Piriformospora%20indica%2C%20a%20ubiquitous%20plant-associated%20fungus&amp;aulast=Rafiqi&amp;date=2013&amp;volume=4" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Effector%20candidates%20in%20the%20secretome%20of%20Piriformospora%20indica%2C%20a%20ubiquitous%20plant-associated%20fungus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B62" class="ref-content ">
<div class="label">62</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">de Guillen</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Ortiz-Vallejo</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Gracy</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Fournier</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Kroj</div> <div class="given-names">T.</div>
</div>, <div class="name string-name">
<div class="surname">Padilla</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Structure analysis uncovers a highly diverse but structurally conserved effector family in phytopathogenic fungi</div>, <div class="source">PLoS Pathog</div> ., <div class="volume">11</div>, <div class="fpage">e1005228</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Structure%20analysis%20uncovers%20a%20highly%20diverse%20but%20structurally%20conserved%20effector%20family%20in%20phytopathogenic%20fungi&amp;author=K.%20de%20Guillen&amp;author=D.%20Ortiz-Vallejo&amp;author=J.%20Gracy&amp;author=E.%20Fournier&amp;author=T.%20Kroj&amp;author=A.%20Padilla&amp;publication_year=2015&amp;journal=PLoS%20Pathog&amp;volume=11&amp;pages=e1005228" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.ppat.1005228" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1005228" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.ppat.1005228"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26506000" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Structure%20analysis%20uncovers%20a%20highly%20diverse%20but%20structurally%20conserved%20effector%20family%20in%20phytopathogenic%20fungi&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B63" class="ref-content ">
<div class="label">63</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Coelho</div> <div class="given-names">M. A.</div>
</div>, <div class="name string-name">
<div class="surname">Sampaio</div> <div class="given-names">J. P.</div>
</div>, <div class="name string-name">
<div class="surname">Gonçalves</div> <div class="given-names">P.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">A Deviation from the bipolar-tetrapolar mating paradigm in an early diverged basidiomycete</div>, <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1001052</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20Deviation%20from%20the%20bipolar-tetrapolar%20mating%20paradigm%20in%20an%20early%20diverged%20basidiomycete&amp;author=M.%20A.%20Coelho&amp;author=J.%20P.%20Sampaio&amp;author=P.%20Gon%C3%A7alves&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001052" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1001052" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001052" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1001052"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20700437" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20Deviation%20from%20the%20bipolar-tetrapolar%20mating%20paradigm%20in%20an%20early%20diverged%20basidiomycete&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B64" class="ref-content ">
<div class="label">64</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Duplessis</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Cuomo</div> <div class="given-names">C. A.</div>
</div>, <div class="name string-name">
<div class="surname">Lin</div> <div class="given-names">Y.-C.</div>
</div>
</div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Obligate biotrophy features unraveled by the genomic analysis of rust fungi</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">9166</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Obligate%20biotrophy%20features%20unraveled%20by%20the%20genomic%20analysis%20of%20rust%20fungi&amp;author=S.%20Duplessis&amp;author=C.%20A.%20Cuomo&amp;author=Y.-C.%20Lin&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=9166" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1019315108" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1019315108" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1019315108"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Obligate%20biotrophy%20features%20unraveled%20by%20the%20genomic%20analysis%20of%20rust%20fungi&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B65" class="ref-content ">
<div class="label">65</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Paul</div> <div class="given-names">J. A.</div>
</div>, <div class="name string-name">
<div class="surname">Barati</div> <div class="given-names">M. T.</div>
</div>, <div class="name string-name">
<div class="surname">Cooper</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Perlin</div> <div class="given-names">M. H.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Physical and genetic interaction between ammonium transporters and the signaling protein rho1 in the plant pathogen Ustilago maydis</div>, <div class="source">Eukaryotic Cell</div> , <div class="volume">13</div>, <div class="fpage">1328</div>–<div class="lpage">36</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Physical%20and%20genetic%20interaction%20between%20ammonium%20transporters%20and%20the%20signaling%20protein%20rho1%20in%20the%20plant%20pathogen%20Ustilago%20maydis&amp;author=J.%20A.%20Paul&amp;author=M.%20T.%20Barati&amp;author=M.%20Cooper&amp;author=M.%20H.%20Perlin&amp;publication_year=2014&amp;journal=Eukaryotic%20Cell&amp;volume=13&amp;pages=1328-36" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/EC.00150-14" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00150-14" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FEC.00150-14"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25128189" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Physical%20and%20genetic%20interaction%20between%20ammonium%20transporters%20and%20the%20signaling%20protein%20rho1%20in%20the%20plant%20pathogen%20Ustilago%20maydis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B66" class="ref-content ">
<div class="label">66</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ruiz-Herrera</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Leon-Ramirez</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Vera-Nunez</div> <div class="given-names">A.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">A novel intracellular nitrogen-fixing symbiosis made by Ustilago maydis and Bacillus spp</div>, <div class="source">New Phytol</div> ., <div class="volume">207</div>, <div class="fpage">769</div>–<div class="lpage">77</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20novel%20intracellular%20nitrogen-fixing%20symbiosis%20made%20by%20Ustilago%20maydis%20and%20Bacillus%20spp&amp;author=J.%20Ruiz-Herrera&amp;author=C.%20Leon-Ramirez&amp;author=A.%20Vera-Nunez&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=207&amp;pages=769-77" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/nph.13359" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13359" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fnph.13359"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25754368" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20novel%20intracellular%20nitrogen-fixing%20symbiosis%20made%20by%20Ustilago%20maydis%20and%20Bacillus%20spp&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B67" class="ref-content ">
<div class="label">67</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bonfante</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Anca</div> <div class="given-names">I. A.</div>
</div>
</div> <div class="year">2009</div>, 
<div class="article-title">Plants, mycorrhizal fungi, and bacteria: a network of interactions</div>, <div class="source">Annu. Rev. Microbiol</div> ., <div class="volume">63</div>, <div class="fpage">363</div>–<div class="lpage">83</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Plants%2C%20mycorrhizal%20fungi%2C%20and%20bacteria%3A%20a%20network%20of%20interactions&amp;author=P.%20Bonfante&amp;author=I.%20A.%20Anca&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Microbiol&amp;volume=63&amp;pages=363-83" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev.micro.091208.073504" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.091208.073504" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev.micro.091208.073504"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19514845" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Plants%2C%20mycorrhizal%20fungi%2C%20and%20bacteria%3A%20a%20network%20of%20interactions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy044-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy044-B68" class="ref-content ">
<div class="label">68</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Torres-Cortés</div> <div class="given-names">G.</div>
</div>, <div class="name string-name">
<div class="surname">Ghignone</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Bonfante</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Schüßler</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Mosaic genome of endobacteria in arbuscular mycorrhizal fungi: transkingdom gene transfer in an ancient mycoplasma-fungus association</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">7785</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mosaic%20genome%20of%20endobacteria%20in%20arbuscular%20mycorrhizal%20fungi%3A%20transkingdom%20gene%20transfer%20in%20an%20ancient%20mycoplasma-fungus%20association&amp;author=G.%20Torres-Cort%C3%A9s&amp;author=S.%20Ghignone&amp;author=P.%20Bonfante&amp;author=A.%20Sch%C3%BC%C3%9Fler&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=7785" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1501540112" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1501540112" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1501540112"> </span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mosaic%20genome%20of%20endobacteria%20in%20arbuscular%20mycorrhizal%20fungi%3A%20transkingdom%20gene%20transfer%20in%20an%20ancient%20mycoplasma-fungus%20association&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="dsy044-FM2"><div class="fn"><div class="footnote-content">Diya Sen and Karnelia Paul authors contributed equally to this study.</div></div></div>
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-20127028" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Ito</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Ito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ito%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ito%20Takashi%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Takashi Ito</a>
                            </span>
                        <br>
</div>
            </div>
        </div>
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5277053&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy044/1/dsy044_supp.zip?Expires=1576703349&amp;Signature=kLOInRtmCkGIaah6qQpI0DBE9g5Wx5bFXddVgao1vsKLcEoXfFd9ni1l54lIl3RFJnqn8pR0bDV20unDjoF5ILWbB1KyrKfsBE1u2AbnYmk1nr-tqCNxDO6KuwH8rYzezcQ4EoveaV~3Atc4ft1kyWNGioZJ8RVVJ4pqd4K2WwV8clP7ICIrqJmewxxySUL~FAEhxN2MYD~EgdzYuqx-Q~v5yuCpzH6YJV3LS6HRqlFZEGJSlKdAHc6vnpugqjolzGshmDwNxpICP2xodTTn8jPkKS8tuL7nqUD2PH82eb5iqjt9Ic5R5zWJWG~rbRrGJ6cgmBesNOVuubFKf6Dn8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,804</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy044"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5277053"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f2%2f131%2f5277053">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F2%2F131%2F5277053" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5277053">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="unique life-strategy of an endophytic yeast Rhodotorula mucilaginosa JGTA-S1—a comparative genomics viewpoint | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy044");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f131%2f5277053">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
