<!DOCTYPE html>
<html lang="en" class="no-js">
<head>
<title>OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes | DNA Research | Oxford Academic</title>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js" type="text/javascript"></script><script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
<!-- Bookmark Icons --><link rel="apple-touch-icon" sizes="180x180" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
<link rel="mask-icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
<link rel="manifest" href="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
<meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
<meta name="theme-color" content="#002f65">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/fonts/icons.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/css/v-637087213365645952/style.min.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/oupCookiePolicy/css/v-637087210722688166/jquery.fancybox-1.3.4_1.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jscrollpane/v-637087210722531832/jscrollpane.css">
<link rel="stylesheet" type="text/css" href="//oup.silverchair-cdn.com/UI/app/vendor/jquery.tablesorter/v-637087210722531832/theme.default.css">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic%7CSource+Sans+Pro:400,400italic,700,700italic">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css" rel="stylesheet" type="text/css">
<link href="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site-1470551617.css" rel="stylesheet" type="text/css">
<script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script><script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'dnaresearch';
            App.SiteURL = 'academic.oup.com/dnaresearch';
        </script><link href="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" rel="stylesheet" type="text/css">
<script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/26/2/147/5280828","@type":"ScholarlyArticle","name":"OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes","datePublished":"2019-01-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/26/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2019-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy045","keywords":["database","metagenomics","sequence analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/26/2/147/5280828","author":[{"name":"Yadav, Deepak","affiliation":"Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India","@type":"Person"},{"name":"Dutta, Anirban","affiliation":"Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India","@type":"Person"},{"name":"Mande, Sharmila S","affiliation":"Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India","@type":"Person"}],"description":"Abstract.  Many microbiome studies employ reference-based operational taxonomic unit (OTU)-picking methods, which in general, rely on databases cataloguing refe","pageStart":"147","pageEnd":"156","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636247317&Signature=X7q2iC~vHlT0WddwBvfBEmEmTwIvXLLzgzPU6G-bLK1HkspV-M-sRhEID-l-FXzIViWy2TJwOfXSqMa8DoVewKeBMETVzb9uzCOJREZFcb-Nh9ldomPZTbE9F0TZ4y7R-21OH8NJ4rvVuSNetyUl2IwBU8soqgz9K2gw~GvSNdWs1fuEKjHBGCGpv7MLkrGgxdBLCVypGt2cqUaQFLrVaNBEULc4Pf5gENFKadT~lEM4m0UbZ1ZVkQGiSiUusddEFDeWJ6-x3Wmt~eh4u6IP8pvheqhM3yqyELP-A9rPBdtD6dweOkXZnVkZ4dZU7id3jMDRRCWOw7L6mLPhw8GBZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636247317&Signature=X7q2iC~vHlT0WddwBvfBEmEmTwIvXLLzgzPU6G-bLK1HkspV-M-sRhEID-l-FXzIViWy2TJwOfXSqMa8DoVewKeBMETVzb9uzCOJREZFcb-Nh9ldomPZTbE9F0TZ4y7R-21OH8NJ4rvVuSNetyUl2IwBU8soqgz9K2gw~GvSNdWs1fuEKjHBGCGpv7MLkrGgxdBLCVypGt2cqUaQFLrVaNBEULc4Pf5gENFKadT~lEM4m0UbZ1ZVkQGiSiUusddEFDeWJ6-x3Wmt~eh4u6IP8pvheqhM3yqyELP-A9rPBdtD6dweOkXZnVkZ4dZU7id3jMDRRCWOw7L6mLPhw8GBZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script><meta property="og:site_name" content="OUP Academic">
<meta property="og:type" content="article">
<meta property="og:title" content="OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes">
<meta property="og:description" content="Abstract.  Many microbiome studies employ reference-based operational taxonomic unit (OTU)-picking methods, which in general, rely on databases cataloguing refe">
<meta property="og:url" content="https://academic.oup.com/dnaresearch/article/26/2/147/5280828">
<meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1636247317&amp;Signature=X7q2iC~vHlT0WddwBvfBEmEmTwIvXLLzgzPU6G-bLK1HkspV-M-sRhEID-l-FXzIViWy2TJwOfXSqMa8DoVewKeBMETVzb9uzCOJREZFcb-Nh9ldomPZTbE9F0TZ4y7R-21OH8NJ4rvVuSNetyUl2IwBU8soqgz9K2gw~GvSNdWs1fuEKjHBGCGpv7MLkrGgxdBLCVypGt2cqUaQFLrVaNBEULc4Pf5gENFKadT~lEM4m0UbZ1ZVkQGiSiUusddEFDeWJ6-x3Wmt~eh4u6IP8pvheqhM3yqyELP-A9rPBdtD6dweOkXZnVkZ4dZU7id3jMDRRCWOw7L6mLPhw8GBZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
<meta property="og:updated_time" content="2019-01-08">
<meta name="citation_author" content="Yadav, Deepak">
<meta name="citation_author_institution" content="Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India">
<meta name="citation_author" content="Dutta, Anirban">
<meta name="citation_author_institution" content="Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India">
<meta name="citation_author" content="Mande, Sharmila S">
<meta name="citation_author_institution" content="Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India">
<meta name="citation_title" content="OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes">
<meta name="citation_firstpage" content="147">
<meta name="citation_lastpage" content="156">
<meta name="citation_doi" content="10.1093/dnares/dsy045">
<meta name="citation_journal_title" content="DNA Research">
<meta name="citation_journal_abbrev" content="DNA Res">
<meta name="citation_volume" content="26">
<meta name="citation_issue" content="2">
<meta name="citation_publication_date" content="2019/04/01">
<meta name="citation_issn" content="1340-2838">
<meta name="citation_publisher" content="Narnia">
<meta name="citation_reference" content="citation_title=PyNAST: a flexible tool for aligning sequences to a template alignment;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=266-7; ">
<meta name="citation_reference" content="citation_title=Search and clustering orders of magnitude faster than BLAST;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2460-1; ">
<meta name="citation_reference" content="citation_title=C16S - a Hidden Markov Model based algorithm for taxonomic classification of 16S rRNA gene sequences;  citation_journal_title=Genomics;  citation_year=2012;  citation_volume=99;  citation_pages=195-201; ">
<meta name="citation_reference" content="citation_title=Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2007;  citation_volume=73;  citation_pages=5261-7; ">
<meta name="citation_reference" content="citation_title=Shift in the microbial ecology of a hospital hot water system following the introduction of an on-site monochloramine disinfection system;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e102679">
<meta name="citation_reference" content="citation_title=Phylogenetic stratigraphy in the Guerrero Negro hypersaline microbial mat;  citation_journal_title=Isme J;  citation_year=2013;  citation_volume=7;  citation_pages=50-60; ">
<meta name="citation_reference" content="citation_title=Deriving accurate microbiota profiles from human samples with low bacterial content through post-sequencing processing of Illumina MiSeq data;  citation_journal_title=Microbiome;  citation_year=2015;  citation_volume=3;  citation_pages=19">
<meta name="citation_reference" content="citation_title=The Western English Channel contains a persistent microbial seed bank;  citation_journal_title=Isme J;  citation_year=2012;  citation_volume=6;  citation_pages=1089-93; ">
<meta name="citation_reference" content="citation_title=Data analysis for 16S microbial profiling from different benchtop sequencing platforms;  citation_journal_title=J. Microbiol Methods;  citation_year=2014;  citation_volume=107;  citation_pages=30-7; ">
<meta name="citation_reference" content="citation_title=Subsampled open-reference clustering creates consistent, comprehensive OTU definitions and scales to billions of sequences;  citation_journal_title=PeerJ;  citation_year=2014;  citation_volume=2;  citation_pages=e545">
<meta name="citation_reference" content="citation_title=Insidious effects of sequencing errors on perceived diversity in molecular surveys;  citation_journal_title=New Phytol;  citation_year=2010;  citation_volume=188;  citation_pages=916-8; ">
<meta name="citation_reference" content="citation_title=Strengths and limitations of 16S rRNA gene amplicon sequencing in revealing temporal microbial community dynamics;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9(4); ">
<meta name="citation_reference" content="citation_title=Removing noise from pyrosequenced amplicons;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=38">
<meta name="citation_reference" content="citation_title=Estimation of bacterial diversity using next generation sequencing of 16S rDNA: a comparison of different workflows;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=473">
<meta name="citation_reference" content="citation_title=Comparing clustering and pre-processing in taxonomy analysis;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=2891-7; ">
<meta name="citation_reference" content="citation_title=A comparison of methods for clustering 16S rRNA sequences into OTUs;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8; ">
<meta name="citation_reference" content="citation_title=Assessing and improving methods used in operational taxonomic unit-based approaches for 16S rRNA gene sequence analysis;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2011;  citation_volume=77;  citation_pages=3219-26; ">
<meta name="citation_reference" content="citation_title=Improved OTU-picking using long-read 16S rRNA gene amplicon sequencing and generic hierarchical clustering;  citation_journal_title=Microbiome;  citation_year=2015;  citation_volume=3;  citation_pages=43">
<meta name="citation_reference" content="citation_title=Evaluation of different partial 16S rRNA gene sequence regions for phylogenetic analysis of microbiomes;  citation_journal_title=J. Microbiol. Methods;  citation_year=2011;  citation_volume=84;  citation_pages=81-7; ">
<meta name="citation_reference" content="citation_title=The effects of alignment quality, distance calculation method, sequence filtering, and region on the analysis of 16S rRNA gene-based studies;  citation_journal_title=PLoS Comput. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=e1000844">
<meta name="citation_reference" content="citation_title=Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2006;  citation_volume=72;  citation_pages=5069-72; ">
<meta name="citation_reference" content="citation_title=V-Xtractor: an open-source, high-throughput software tool to identify and extract hypervariable regions of small subunit (16S/18S) ribosomal RNA gene sequences;  citation_journal_title=J. Microbiol. Methods;  citation_year=2010;  citation_volume=83;  citation_pages=250-3; ">
<meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; ">
<meta name="citation_reference" content="citation_title=Gut microbiomes of Indian children of varying nutritional status;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e95547">
<meta name="citation_reference" content="citation_title=Microbial community profiling shows dysbiosis in the lesional skin of Vitiligo subjects;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=18761">
<meta name="citation_reference" content="citation_title=Ocean plankton. Structure and function of the global ocean microbiome;  citation_journal_title=Science;  citation_year=2015;  citation_volume=348;  citation_pages=1261359">
<meta name="citation_reference" content="citation_title=Soil microbiome responses to the short-term effects of Amazonian deforestation;  citation_journal_title=Mol. Ecol;  citation_year=2015;  citation_volume=24;  citation_pages=2433-48; ">
<meta name="citation_reference" content="citation_title=The Ribosomal Database Project (RDP-II): sequences and tools for high-throughput rRNA analysis;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=D294-6; ">
<meta name="citation_reference" content="citation_title=Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2009;  citation_volume=75;  citation_pages=7537-41; ">
<meta name="citation_reference" content="citation_title=Genomic insights that advance the species definition for prokaryotes;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2005;  citation_volume=102;  citation_pages=2567-72; ">
<meta name="citation_reference" content="citation_title=Clustering 16S rRNA for OTU prediction: a method of unsupervised Bayesian clustering;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=611-8; ">
<meta name="citation_reference" content="citation_title=UniFrac: an effective distance metric for microbial community comparison;  citation_journal_title=Isme J;  citation_year=2011;  citation_volume=5;  citation_pages=169-72; ">
<meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach;  citation_journal_title=J. Mol. Evol;  citation_year=1981;  citation_volume=17;  citation_pages=368-76; ">
<meta name="citation_reference" content="citation_title=phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e61217">
<meta name="citation_reference" content="citation_title=Analysis, optimization and verification of illumina-generated 16S rRNA gene amplicon surveys;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e94249">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/26/2/147/28495563/dsy045.pdf">
<meta name="description" content="Abstract.  Many microbiome studies employ reference-based operational taxonomic unit (OTU)-picking methods, which in general, rely on databases cataloguing refe">
</head>
<body>
<div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/26/2/147/5280828">
<link rel="canonical" href="https://academic.oup.com/dnaresearch/article/26/2/147/5280828">
<script async="async" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script><script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/26/2/147/5280828");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script><script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script><noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a>
</noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753,2779951&authnips=128.235.85.32&doi=10.1093/dnares/dsy045';
    </script><noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" type="hidden" value="5275"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" type="hidden" value="5000"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" type="hidden" value="default"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" type="hidden" value="sticky"><div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header"><div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="/">
                    <img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div>

            <ul class="oup-header-menu account-menu">
<li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>
</div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f147%2f5280828" class="register">Register</a></li>
                
            </ul>
</div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f147%2f5280828" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul>
</div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="/dnaresearch" class="journal-logo-container">
                    <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research"></a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute"></a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"><i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410109">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410106">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410110">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410111">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410107">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410112">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410113">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410114">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Open Access</a>
            
        </li>
    </ul>
</li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410115">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410116">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1410108">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
<li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410117">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410118">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410119">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410120">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410121">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410122">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410123">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1410124">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>
</li>
    </ul></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input type="hidden" name="searchScope" id="hfSolrJournalID" value=""><input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals"><input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals"><input type="hidden" name="siteID" id="hfsiteID" value="5275"><input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567"><input type="hidden" id="hfSolrJournalName" value=""><input type="hidden" id="hfSolrMaxAllowSearchChar" value="100"><input type="hidden" id="hfJournalShortName" value=""><input type="hidden" id="hfSearchPlaceholder" value=""><input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value=""><input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals"><input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent"><input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q="><input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=115366"><input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue"><input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q="><input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch"><script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

</section><div id="main" class="content-main js-main ui-base">
        <section class="master-main row"><div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/26/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/26/2/1/m_dnares_26_2cover.png?Expires=1576706089&amp;Signature=Drq6wwqAdVeKbPwhVWu8zD4eEtLUMneAoa7qgyLrFoSYdcjLziC2QEf9qJsl7HDkcKd-TakUMY2eKvx0CyRSuj0JB3tS~144boC8Kb7oDzqRuyc4Z9uVP95S95ybblIpfwy-IuovYxZ3Lt49wu4~T~0C0iqLhCfCDAI0tY8GZ4HIms3p1MqAYE05KUpceWUFFBxxMKbH5PLGSEpFHgkuaEpBJkjdA6Gmmm0llJ9am6EqAKy5Uy0-I~~y8OEznZbfyQwZirWRGjBiJO9J4xN2pmgZ~zEa8WfxHYMKXBTwkHRBa40P3l1Mi89hkaglXuw2OPX5-ICWNkw2xYBfrnEbgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/26/2">
                        <div class="volume trailing-comma">Volume 26</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
<li class="section-jump-link head-1" link-destination="134009549">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009549">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009551">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009551">1. Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009555">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009555">2. Materials and methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009586">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009586">3. Results and discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009599">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009599">4. Conclusion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009601">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009601">Availability</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="134009603">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009603">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="134009607">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#134009607">References</a>
            </div>
        </li>
</ul>
</div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
<li class="prev arrow">
            <a href="/dnaresearch/article/26/2/131/5277053">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/26/2/157/5306610">Next &gt;</a>
        </li>
</ul>
</div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deepak Yadav</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deepak+Yadav">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yadav%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yadav%20Deepak%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Deepak Yadav</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anirban Dutta</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anirban+Dutta">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dutta%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Dutta%20Anirban%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Anirban Dutta</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sharmila S Mande</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Bio-Sciences R&amp;D Division, TCS Research, Tata Consultancy Services Limited, Pune, Maharashtra, India</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +91 20 6608 6432. Fax. +91 20 6608 6399. Email: <a href="mailto:sharmila.mande@tcs.com" target="_blank">sharmila.mande@tcs.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sharmila+S+Mande">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mande%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Mande%20Sharmila%20S%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Sharmila S Mande</a><span class="delimiter"></span>
                        </span>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy045-FM1"><div class="fn"><div class="footnote-content">Deepak Yadav and Anirban Dutta authors contributed equally to this work.</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary">
<em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 147–156, <a href="https://doi.org/10.1093/dnares/dsy045">https://doi.org/10.1093/dnares/dsy045</a>
</div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">08 January 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">08 May 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">13 December 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">08 January 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>
</div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
<li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5280828" href="/dnaresearch/article-pdf/26/2/147/28495563/dsy045.pdf">
            <img src="//oup.silverchair-cdn.com/UI/app/svg/pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="video">
                                    <a href="javascript:;"><span>Video</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="audio">
                                    <a href="javascript:;"><span>Audio</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
</li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Deepak Yadav, Anirban Dutta, Sharmila S Mande,  OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes, <em>DNA Research</em>, Volume 26, Issue 2, April 2019, Pages 147–156, <a href="https://doi.org/10.1093/dnares/dsy045">https://doi.org/10.1093/dnares/dsy045</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
<li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="/Citation/Download?resourceId=5280828&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
<hr>
<p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=OTUX%3A%20V-region%20specific%20OTU%20database%20for%20improved%2016S%20rRNA%20OTU%20picking%20and%20efficient%20cross-study%20taxonomic%20comparison%20of%20microbiomes&amp;publicationDate=2019-01-08&amp;volumeNum=26&amp;issueNum=2&amp;author=Yadav%2C%20Deepak%3B%20Dutta%2C%20Anirban&amp;startPage=147&amp;endPage=156&amp;contentId=10.1093%2Fdnares%2Fdsy045&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
<div class="arrow-up"></div>
                                <li><a class="addthis_button_email"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact"><span>More</span></a></li>
                            </ul>
</li>
                    </ul>
<div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="">All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm"><span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
<li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
<li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
</li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
<li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
</li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f147%2f5280828" class="register nav-link">Register</a>
                </li>
        </ul>
</div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu"></nav><div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter"><optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option>
<option class="header-search-bar-filters-item" value="" selected>All  DNA Research</option>
</optgroup>
<optgroup label="Search beyond this journal"><option class="header-search-bar-filters-item" value="Journals">All  Journals</option></optgroup></select><label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input"><span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" type="hidden" value="False"><input id="routename" name="RouteName" type="hidden" value="dnaresearch">
</div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="134009549" id="134009549" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Many microbiome studies employ reference-based operational taxonomic unit (OTU)-picking methods, which in general, rely on databases cataloguing reference OTUs identified through clustering full-length 16S rRNA genes. Given that the rate of accumulation of mutations are not uniform throughout the length of a 16S rRNA gene across different taxonomic clades, results of OTU identification or taxonomic classification obtained using ‘short-read’ sequence queries (as generated by next-generation sequencing platforms) can be inconsistent and of suboptimal accuracy. <em>De novo</em> OTU clustering results too can significantly vary depending upon the hypervariable region (V-region) targeted for sequencing. As a consequence, comparison of microbiomes profiled in different scientific studies becomes difficult and often poses a challenge in analysing new findings in context of prior knowledge. The OTUX approach of reference-based OTU-picking proposes to overcome these limitations by using ‘customized’ OTU reference databases, which can cater to different sets of short-read sequences corresponding to different 16S V-regions. The results obtained with OTUX-approach (which are in terms of OTUX-OTU identifiers) can also be ‘mapped back’ or represented in terms of other OTU database identifiers/taxonomy, e.g. Greengenes, thus allowing for easy cross-study comparisons. Validation with simulated datasets indicates more efficient, accurate, and consistent taxonomic classifications obtained using OTUX-approach, as compared with conventional methods.</p></section><div class="article-metadata-panel clearfix"></div>
<div class="kwd-group">
<a class="kwd-part kwd-main" href="javascript:;" data-keyword="database">database</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenomics">metagenomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword='"sequence analysis"'>sequence analysis</a>
</div>                    <h2 scrollto-destination="134009551" id="134009551" class="section-title">1. Introduction</h2>
<p>Advances in high-throughput DNA sequencing technologies have enabled culture independent studies of diverse microbial communities (microbiome) inhabiting different environments. Targeted amplicon sequencing of phylogenetic markers genes (e.g. the bacterial 16S rRNA) forms the basis of most of these studies. The downstream steps of taxonomic classification involve a similarity search of the sequenced DNA fragments (reads) against a reference database of pre-classified sequences (with taxonomic annotations).<a reveal-id="dsy045-B1 dsy045-B2 dsy045-B3 dsy045-B4" data-open="dsy045-B1 dsy045-B2 dsy045-B3 dsy045-B4" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> Inspite of extensive sampling of microbiomes from different environments being performed over the past decade, it would be optimistic to claim the completeness of current generation reference databases, with respect to cataloguing (and annotating) the entire diversity of the microbial world. Given this, many of the sequenced reads in a microbiome study may not find close matches in the reference databases, and thereby remain as unclassified. OTU (operational taxonomic unit)-based methods constitute an alternate approach, wherein the sequenced reads are clustered based on some predefined similarity threshold (<em>de novo</em> OTU clustering). Sequences clustered into the same OTU may be inferred to have originated from the same taxonomic group. Given that all sequenced reads can be grouped into OTUs prior to analysis, the problem with unclassified sequences can be traversed using this method. Further, taxonomic affiliations of these OTUs (and sequences clustered therein) can still be ascertained using similarity searches against reference databases. In practice, ‘open-reference OTU-picking’ approaches have been the choice of a number of recent studies.<a reveal-id="dsy045-B5 dsy045-B6 dsy045-B7 dsy045-B8 dsy045-B9 dsy045-B10" data-open="dsy045-B5 dsy045-B6 dsy045-B7 dsy045-B8 dsy045-B9 dsy045-B10" class="link link-ref link-reveal xref-bibr"><sup>5–10</sup></a> Such an approach involves an initial similarity search step against a reference database (of pre-clustered OTUs) for identifying sequences corresponding to previously annotated OTUs, followed by a clustering step with the remaining sequenced reads for identifying <em>de novo</em> OTUs.</p>
<p>Conventional OTU-picking approaches also have certain limitations. Previous studies have indicated how DNA sequencing errors can lead to an increased number of detected OTUs,<a reveal-id="dsy045-B11 dsy045-B12 dsy045-B13" data-open="dsy045-B11 dsy045-B12 dsy045-B13" class="link link-ref link-reveal xref-bibr"><sup>11–13</sup></a> or how the use of different clustering approaches may result in formation of alternate OTU clusters.<a reveal-id="dsy045-B14 dsy045-B15 dsy045-B16 dsy045-B17" data-open="dsy045-B14 dsy045-B15 dsy045-B16 dsy045-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> However, some other important limitations of the OTU-based approaches, especially in the context of high-throughput DNA-sequencing technologies, have received relatively lesser attention. Given the limitations pertaining to the length of contiguous stretch of DNA (read length) that can be sequenced by currently available technologies, the targeted amplicon often consists of a selected region from the phylogenetic marker gene, instead of the entire gene. For example, in case of the bacterial 16S rRNA gene, stretches consisting of one or more specific ‘variable’ regions (spanning around 400–600 bp) are targeted for sequencing. However, since the reference databases catalogue sequences of full length marker genes, querying the same with ‘short’ sequence reads for OTU identification or taxonomic classification can yield suboptimal results.<a reveal-id="dsy045-B18" data-open="dsy045-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Furthermore, it may be noted that rate of evolution (accumulation of mutations) is not always uniform across the length of a chosen marker gene (or in its variable regions) across different taxonomic clades.<a reveal-id="dsy045-B19" data-open="dsy045-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy045-B20" data-open="dsy045-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> It is possible that while a short region may remain more or less identical during the course of evolution, the flanking regions may be more prone to mutations. Alternately, barring a small hyper-variable stretch, a major fraction of the marker gene may remain unchanged through evolution. Given this scenario, OTU clustering results can vary significantly based on the choice of the target region. Although OTUs identified or classified using reference-based methods vs <em>de novo</em> clustering methods can provide different results, any comparison between the results obtained from studies utilizing different variable regions of a given marker gene also loses relevance.</p>
<p>The question whether the above limitations can be addressed using reference OTU databases that are specific to the targeted regions of the marker gene has been explored in this study. We present the OTUX (meta)database, which consists of 19 distinct OTU databases corresponding to the different stretches of variable regions (V-regions) of the bacterial 16S rRNA gene, that are commonly targeted for amplicon sequencing in microbiome studies. Each of the V-region-specific databases consists of OTUs (referred to as OTUX-OTUs) identified by clustering sequence fragments from corresponding stretches of V-regions cropped out from full-length 16S rRNA gene sequences catalogued in reference databases. Also presented is a ‘mapping matrix’ (MAPMAT), which lists the probabilities of association of any of the OTUX-OTUs to the reference OTUs present in the widely used Greengenes OTU database (consisting full-length marker genes). An open-reference-based OTU-picking approach against an appropriately selected OTUX V-region database is expected to provide results closer to those obtained with <em>de novo</em> OTU clustering. Further, the OTU abundance profiles, obtained in terms of OTUX-OTUs, can be ‘mapped back’ and represented in terms of Greengenes OTUs, using the MAPMAT. Mapping back enables comparing OTU-picking or taxonomic annotation results from different microbiome studies, even if the choice of targeted V-regions had been different. The utility of the proposed database and OTU-picking approach has been extensively validated with multiple simulated sequence datasets mimicking microbiome samples collected from diverse environments. The results indicate the benefits of using V-region-specific databases for classifying 16S amplicon data generated using short-read sequencing technologies.</p>                    <h2 scrollto-destination="134009555" id="134009555" class="section-title">2. Materials and methods</h2>
<p>The OTUX workflow has two major components. This includes a one-time preprocessing step to generate customized V-region-specific reference databases (called OTUX databases) and a ‘MAPMAT’ for different stretches of V-regions. Subsequently, any amplicon sequencing dataset targeting 16S V-regions can be subjected to a reference-based OTU-picking cum taxonomic classification step using the OTUX reference database(s). <a reveal-id="dsy045-F1" data-open="dsy045-F1" class="link link-reveal link-table xref-fig">Figure 1</a> gives a schematic representation of the OTUX method in context of conventional approach (CA) of OTU picking and taxonomic classification of 16S rRNA amplicon sequencing reads. </p>                        <a id="134009557" scrollto-destination="134009557"></a>
<div id="dsy045-F1" data-id="dsy045-F1" class="fig fig-section">
<div class="label fig-label">Figure 1.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/m_dsy045f1.png?Expires=1576616109&amp;Signature=T08L280xOsl1aIvYaoy9pGOIafaYm0QtiKSe3H6d4PreA6oBeX5NJBXH0~p5WT3MJR2w5ZKKqgFrLo4x18JMC249vyFCNqzlEwECsOygTF-7Xm-ATGoyc2FtBoPczRlApktA24hoQB468Y2iIyOBUWZKF4EAmwpzEpKA7tygGPIp8VOSeNnJYFR2OprF81ubCIef5lxoEeUAFQEitCURQlLeYHt~CRfdsBSl3hjOlJQPRyA9ZLWqmRceQUOfkQ-kff8TVYPpLLEtnpk7ABRJsYmeLEqVxvhxqn6ZwG7qGU8hU~0SVxzokakXzuXPTrsvlH6MLMTvpX6FIo1-mmltEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic representation of the OTUX method in context of CA of OTU picking and taxonomic classification of 16S rRNA amplicon sequencing reads. The utility of the ‘MAPMAT’ in enabling comparison of taxonomic profiles obtained using different V-regions is also depicted." path-from-xml="dsy045f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009557/dsy045f1.tif" path-from-xml="dsy045f1.tif" target="_blank">Open in new tab</a><a section="134009557" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045f1.png?Expires=1576616109&amp;Signature=ojHsBDKgFGACT12R7v5tArMtunAIymg8V0G~MOp1EwXWEtSiLmhVmeMM72-aSPeHyLrQ7DJ9vRc1tmhDFdGRW2q4ath8PYX9PDW8iZMkrFlSU5SGff3vryT8sDHo7FY~QvmOll1kvRFH4z9a1yTSRkjlWTe6WB3RUNnbC6LxYkg076W27I75z069DGZ69ioFhpQc~8u2eRkqLg0DUVR4ojxijkBipRFWXxb-fBpnPvCH-CloKcVlUCRls9NNoqXLlgMFWhRvWlxAowVKT~Uk8ZvQjXR6zRQ1Q4yggHNq9pKzQX99qY98asuUC52Ruru7r0ZYVgnokkoVFigUdSsShg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009557&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy045f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A schematic representation of the OTUX method in context of CA of OTU picking and taxonomic classification of 16S rRNA amplicon sequencing reads. The utility of the ‘MAPMAT’ in enabling comparison of taxonomic profiles obtained using different V-regions is also depicted.</p></div>
</div>
<div content-id="dsy045-F1" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 1.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/m_dsy045f1.png?Expires=1576616109&amp;Signature=T08L280xOsl1aIvYaoy9pGOIafaYm0QtiKSe3H6d4PreA6oBeX5NJBXH0~p5WT3MJR2w5ZKKqgFrLo4x18JMC249vyFCNqzlEwECsOygTF-7Xm-ATGoyc2FtBoPczRlApktA24hoQB468Y2iIyOBUWZKF4EAmwpzEpKA7tygGPIp8VOSeNnJYFR2OprF81ubCIef5lxoEeUAFQEitCURQlLeYHt~CRfdsBSl3hjOlJQPRyA9ZLWqmRceQUOfkQ-kff8TVYPpLLEtnpk7ABRJsYmeLEqVxvhxqn6ZwG7qGU8hU~0SVxzokakXzuXPTrsvlH6MLMTvpX6FIo1-mmltEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic representation of the OTUX method in context of CA of OTU picking and taxonomic classification of 16S rRNA amplicon sequencing reads. The utility of the ‘MAPMAT’ in enabling comparison of taxonomic profiles obtained using different V-regions is also depicted." path-from-xml="dsy045f1.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009557/dsy045f1.tif" path-from-xml="dsy045f1.tif" target="_blank">Open in new tab</a><a section="134009557" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045f1.png?Expires=1576616109&amp;Signature=ojHsBDKgFGACT12R7v5tArMtunAIymg8V0G~MOp1EwXWEtSiLmhVmeMM72-aSPeHyLrQ7DJ9vRc1tmhDFdGRW2q4ath8PYX9PDW8iZMkrFlSU5SGff3vryT8sDHo7FY~QvmOll1kvRFH4z9a1yTSRkjlWTe6WB3RUNnbC6LxYkg076W27I75z069DGZ69ioFhpQc~8u2eRkqLg0DUVR4ojxijkBipRFWXxb-fBpnPvCH-CloKcVlUCRls9NNoqXLlgMFWhRvWlxAowVKT~Uk8ZvQjXR6zRQ1Q4yggHNq9pKzQX99qY98asuUC52Ruru7r0ZYVgnokkoVFigUdSsShg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009557&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy045f1.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>A schematic representation of the OTUX method in context of CA of OTU picking and taxonomic classification of 16S rRNA amplicon sequencing reads. The utility of the ‘MAPMAT’ in enabling comparison of taxonomic profiles obtained using different V-regions is also depicted.</p></div>
</div>                    <h3 scrollto-destination="134009558" id="134009558" class="section-title">2.1. Building OTUX databases</h3>
<p>The ‘prokMSA’ unaligned sequences from Greengenes database<a reveal-id="dsy045-B21" data-open="dsy045-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> was downloaded through the following links: (i) <a class="link link-uri" href="http://greengenes.lbl.gov/Download/Sequence_Data/Fasta_data_files/current_prokMSA" target="_blank">http://greengenes.lbl.gov/Download/Sequence_Data/Fasta_data_files/current_prokMSA</a> _unaligned.fasta.gz and (ii) <a class="link link-uri" href="http://ftp://greengenes.microbio.me/greengenes_release/gg_13_5/gg_13_5.fasta.gz" target="_blank">ftp://greengenes.microbio.me/greengenes_release/gg_13_5/gg_13_5.fasta.gz</a>. The downloaded files were decompressed to obtain the fasta formatted sequences, which were subsequently combined to create a final corpus of 1,079,252 unaligned sequences. The taxonomic classification of these sequences for different taxonomic hierarchical levels (including phylum, class, order, family, genus, species as well as Greengenes OTU IDs at 99% sequence identity) as annotated in Greengenes database version 13.8, were retrieved from <a class="link link-uri" href="http://ftp://greengenes.microbio.me/greengenes_release/gg_13_8_otus/taxonomy/99_otu_taxonomy.txt" target="_blank">ftp://greengenes.microbio.me/greengenes_release/gg_13_8_otus/taxonomy/99_otu_taxonomy.txt</a>. Individual V-regions as well as stretches encompassing consecutive V-regions were extracted from each of the 16S rRNA gene sequences present in the sequence corpus using the software V-xtractor.<a reveal-id="dsy045-B22" data-open="dsy045-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The extracted sequences were then clustered based on sequence similarity using CD-HIT,<a reveal-id="dsy045-B23" data-open="dsy045-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> wherein each resultant cluster constituted sequences sharing 99% sequence identity. Each of the clusters was assigned a V-region-specific OTU identifier (OTUX<sub>V</sub> ID) and was compiled to constitute an OTUX<sub>V</sub> reference database corresponding to a specific V-region.</p>                    <h3 scrollto-destination="134009560" id="134009560" class="section-title">2.2. Building MAPMAT</h3>
<div class="block-child-p">By definition, any OTU-picking approach, using one of the OTUX reference databases as reference, will result in taxonomic annotations in terms of OTUX OTU IDs corresponding to the specific OTUX database. To enable cross-comparisons between studies targeting different V-regions (thereby mandating use of different V-region-specific OTUX databases), as well as between studies using other CAs, a ‘MAPMAT’ between different OTUX OTU IDs and conventionally used Greengenes OTU IDs was created. The procedure of building mapping back matrices has been exemplified below using the V4 region-specific OTUX database. In this example, after compiling the OTUX<sub>V4</sub> database, the propensity of association of a OTUX<sub>V4</sub> ID (OTUX<sub>V4i</sub>) to a Greengenes OTU ID (GG<sub>j</sub>) is calculated using the following formula, <a id="IE1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="IE1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mrow><mtext>MAPMAT</mtext></mrow></mrow><mrow><mi mathvariant="normal">V</mi><mn>4</mn><mtext>ij</mtext></mrow></msub><mo>=</mo><mfrac><mrow><mtext>No</mtext><mo>.</mo><msub><mrow><mrow><mo> </mo><mtext>of sequences</mtext><mo> </mo><mtext>clusteredinto OTUX</mtext></mrow></mrow><mrow><mi mathvariant="normal">V</mi><mn>4</mn><mi mathvariant="normal">i</mi></mrow></msub><msub><mrow><mrow><mtext> whose full length counterparts are assigned to GG</mtext></mrow></mrow><mi mathvariant="normal">j</mi></msub></mrow><mrow><mtext>Total no</mtext><mo>.</mo><msub><mrow><mrow><mo> </mo><mtext>of sequences clustered into OTUX</mtext></mrow></mrow><mrow><mi mathvariant="normal">V</mi><mn>4</mn><mi mathvariant="normal">i</mi></mrow></msub></mrow></mfrac></mrow></math></div></div></div>this is followed by populating the propensity matrix MAPMAT<sub>V4</sub>, for the OTUX<sub>V4</sub> database by computing all values for MAPMAT<sub>V4ij</sub> wherein, <em>i</em> = 1 → N<sub>OTUX</sub> (i.e. total number of OTUX<sub>V4</sub> OTUs); <em>j</em> = 1 → N<sub>GG</sub> (i.e. total number of Greengenes OTUs); and MAPMAT<sub>V4</sub> is a N<sub>GG</sub> × N<sub>OTUX</sub> matrix.</div>
<p>An example is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>, where an OTUX ID OTUX<sub>Vi</sub> is mapped to corresponding Greengenes OTUX IDs. The percentages mentioned represent the propensities of OTUX ID OTUX<sub>Vi</sub> to be associated to any of the hypothetical Greengenes OTU IDs GG<sub>x</sub>, GG<sub>y</sub> and GG<sub>z.</sub></p>                    <h3 scrollto-destination="134009563" id="134009563" class="section-title">2.3. Creating validation datasets</h3>
<p>Assessment of accuracy of taxonomic annotation of any OTU-picking or taxonomic classification method, on a real metagenomic dataset constituting short reads, poses a challenge since the origin (taxonomic affiliation) of the sequenced reads are ‘unknown’. The annotation results predicted by different available methods may vary from each other, with respect to the assigned lineage as well as depth of assignment, and cannot be used as a benchmark/standard for comparison. To overcome this problem, multiple validation sets of simulated short metagenomic reads were created and were classified into OTUs using our method. The simulated metagenomes pertaining to 4 different environments, viz. gut of healthy children (abbreviated as GUT),<a reveal-id="dsy045-B24" data-open="dsy045-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> healthy human skin (abbreviated as SKIN),<a reveal-id="dsy045-B25" data-open="dsy045-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Mediterranean sea (abbreviated as SEA)<a reveal-id="dsy045-B26" data-open="dsy045-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and soil (SOIL),<a reveal-id="dsy045-B27" data-open="dsy045-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> were generated using the following procedure. Publicly available datasets corresponding to metagenomic samples from the above mentioned environments were retrieved (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) and overall proportions of different genera present in each of the environments were obtained from the provided abundance tables (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S2). To create a simulated metagenome pertaining to an environment, full length 16S rRNA genes were randomly drawn from the RDP database (v10.3),<a reveal-id="dsy045-B28" data-open="dsy045-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> ensuring that the proportions of the picked genera were similar to those present in the chosen environment (only top 50 genera in each environment considered). The sequences were drawn in the following manner. Let it be assumed that there are ‘N’ full-length sequences belonging to genus ‘X’ (which may belong to different species/strains of the same genus) in the RDP database. Further, let it be considered that in a simulated metagenome (of a pre-defined size or total number of sequences) pertaining to a specific environment, the genus ‘X’ needs to be represented with ‘i’ number of sequences. To achieve this, ‘i’ random sequences from the set of ‘N’ sequences (belonging to genus ‘X’) were picked with replacement. The process was repeated for every genus constituting the desired simulated metagenome. 100 simulated metagenomic datasets (each constituting 10,000 sequences) were created for each of the four environments (abbreviated as D<sub>GUT/F</sub>, D<sub>SKIN/F</sub>, D<sub>SEA/F</sub> and D<sub>SOIL/F</sub>). Specific V-regions or selected stretches of V-regions from each of the full-length 16S rRNA gene sequences (constituting these simulated metagenomes) were further cropped out to create corresponding simulated ‘short-read’ metagenomic datasets (abbreviated as D<sub>GUT/Vi</sub>, D<sub>SKIN/Vi</sub>, D<sub>SEA/Vi</sub> and D<sub>SOIL/Vi</sub>), mimicking targeted amplicon sequencing. The selected V-regions (and stretches of V-regions) considered for validation included V4, V1-V2, V1-V3, V2-V3, V3-V4, V3-V5, V3-V6, V4-V5, V4-V6, V5-V6 and V6-V8, which are typically employed in 16S rRNA-based microbiome studies.</p>                    <h3 scrollto-destination="134009565" id="134009565" class="section-title">2.4. Classifying and annotating metagenomic sequences</h3>
<p>Reference-based OTU-picking can be performed for a query set of metagenomic sequences using an OTUX<sub>Vi</sub> reference database (wherein ‘i’ corresponds to the V-region or stretch of consecutive V-regions which were targeted during amplicon sequencing). For the purpose of validation, OTU-picking or taxonomic assignment exercise was performed with multiple simulated metagenomic datasets as well as using different OTUX<sub>Vi</sub> reference databases (as appropriate for the individual datasets). The following section illustrates this process of annotation for a set of simulated query sequences corresponding to an amplicon sequencing experiment targeting the V4 region, using OTUX<sub>V4</sub> as the reference database. The same methods can be extended for classifying sequences generated in sequencing experiments targeting a different V-region, by choosing an appropriate OTUX<sub>Vi</sub> reference database. In this work, a naïve Bayesian classifier (Wang’s algorithm),<a reveal-id="dsy045-B4" data-open="dsy045-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> as implemented in the software ‘mothur’,<a reveal-id="dsy045-B29" data-open="dsy045-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> was used for classification, wherein a bootstrap confidence threshold of 80% was used for making assignments at the OTU level. Subsequently, an OTU abundance profile (T<sub>OTUX</sub>) was generated by cumulating the total number of sequenced reads from the given metagenomic sample that could be attributed to each of the OTUX<sub>V4</sub> OTUs. It may be noted here that the choice of the naïve Bayesian classifier for the validation study was influenced by its wide use in microbiome studies and annotation tools (e.g. RDP classifier).<a reveal-id="dsy045-B4" data-open="dsy045-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Other reference-based OTU-picking and taxonomic classification methods can also be used while using OTUX as a reference database. The classification results (obtained in terms of OTUX<sub>V4</sub> OTUs) were mapped back using the corresponding MAPMAT<sub>V4</sub> to represent the results in terms of Greengenes (v13.8 – OTUs obtained at 99% identity) OTU IDs. This ‘mapping back’ can be performed using either a ‘one-to-one mapping’ (one OTX OTU mapped back to a single Greengenes OTU) approach or a ‘one-to-many mapping’ (one OTX OTU mapped back to multiple Greengenes OTUs) approach. These mapping back approaches are described below.</p>                    <h4 scrollto-destination="134009567" id="134009567" class="section-title">2.4.1. One-to-one mapping</h4>
<p>In this procedure, each query sequence is assigned to one particular Greengenes OTU ID. For example, if a particular query sequence ‘s’ has been assigned to the OTU ‘x’ corresponding to the reference database OTUX<sub>V4</sub> (abbreviated as OTUX<sub>V4x</sub>), the MAPMAT<sub>V4</sub> elements {MAPMAT<sub>V4xj</sub>} are retrieved (wherein ‘j’ = 1 → N<sub>GG</sub> i.e. the total no. of Greengenes OTUs). The maximum value of {MAPMAT<sub>V4xj</sub>} is then computed. The sequence ‘s’ can subsequently be classified to the Greengenes OTU ‘y’ (abbreviated as GG<sub>y</sub>), wherein MAPMAT<sub>V4xy</sub> = max{MAPMAT<sub>V4xj</sub>}. In effect, the OTUX ID OTUX<sub>V4x</sub> is mapped to the Greengenes ID GG<sub>y</sub> since they have the highest propensity of association as recorded in the MAPMAT. The process, once repeated for all query sequences will subsequently lead to a new OTU abundance profile (T<sub>GG</sub>), represented in terms of Greengenes OTU IDs.</p>                    <h4 scrollto-destination="134009569" id="134009569" class="section-title">2.4.2. One-to-many mapping</h4>
<div class="block-child-p">Given that one OTUX ID may be associated with multiple Greengenes IDs (and <em>vice versa</em>), the mapping back matrix can also be utilized for one-to-many mapping to report abundance tables representing relative abundance of the OTUs (such as percentage normalized abundance). To begin with, the abundance profile T<sub>OTUX</sub> needs to be generated as described above, wherein the total number of sequences assigned to each of the OTUX<sub>V4</sub> OTUs is represented. <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>T</mi></mrow><mrow><mi>O</mi><mi>T</mi><mi>U</mi><mi>X</mi></mrow></msub><mo>=</mo><mo> </mo><mo> </mo><mrow><mo stretchy="true">(</mo><mrow><mo> </mo><mo> </mo><mtable><mtr><mtd><mrow><mtable><mtr><mtd><mi>a</mi></mtd></mtr><mtr><mtd><mi>b</mi></mtd></mtr><mtr><mtd><mi>c</mi></mtd></mtr></mtable></mrow></mtd></mtr><mtr><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mi>z</mi></mtd></mtr></mtable><mo> </mo><mo> </mo></mrow><mo stretchy="true">)</mo></mrow><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mo> </mo><mtable><mtr><mtd><mrow><mo>…</mo><mo>…</mo><mi>O</mi><mi>T</mi><mi>U</mi><msub><mrow><mi>X</mi></mrow><mrow><mi>V</mi><mn>4</mn></mrow></msub><mn>1</mn></mrow></mtd></mtr><mtr><mtd><mrow><mo>…</mo><mo>…</mo><mi>O</mi><mi>T</mi><mi>U</mi><msub><mrow><mi>X</mi></mrow><mrow><mi>V</mi><mn>4</mn></mrow></msub><mn>2</mn></mrow></mtd></mtr><mtr><mtd><mrow><mtable><mtr><mtd><mrow><mo>…</mo><mo>…</mo><mi>O</mi><mi>T</mi><mi>U</mi><msub><mrow><mi>X</mi></mrow><mrow><mi>V</mi><mn>4</mn></mrow></msub><mn>3</mn></mrow></mtd></mtr><mtr><mtd><mo>⋮</mo></mtd></mtr><mtr><mtd><mrow><mo>…</mo><mo>…</mo><mi>O</mi><mi>T</mi><mi>U</mi><msub><mrow><mi>X</mi></mrow><mrow><mi>V</mi><mn>4</mn></mrow></msub><mi>i</mi></mrow></mtd></mtr></mtable></mrow></mtd></mtr></mtable><mo>.</mo></mrow></math></div></div></div>
</div>
<div class="block-child-p">For example, T<sub>OTUX</sub> can be represented in form of a column matrix (of size N<sub>OTUX</sub> × 1) as depicted above wherein ‘i’ varies from 1 to the total number of OTUX<sub>V4</sub> OTUs (i.e. N<sub>OTUX</sub>), and wherein ‘a’ is the number of sequences assigned to the OTU OTUX<sub>V4</sub>1, ‘b’ is the number of sequences assigned to OTUX<sub>V4</sub>2, ‘c’ is the number of sequences assigned to OTUX<sub>V4</sub>3 and so on. Next, an OTU abundance profile (T<sub>GGraw</sub>) is obtained for the set of query sequences, in terms of Greeengenes OTU IDs by multiplying the matrix MAPMAT<sub>V4</sub> with the matrix T<sub>OTUX</sub>. It may be noted that given the nature of the MAPMAT matrix, the abundance values for each of the Greengenes OTUs in T<sub>GGraw</sub> may be a fractional value. <a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>T</mi></mrow><mrow><mi>G</mi><mi>G</mi><mi>r</mi><mi>a</mi><mi>w</mi></mrow></msub><mo>=</mo><mi>M</mi><mi>A</mi><mi>P</mi><mi>M</mi><mi>A</mi><msub><mrow><mi>T</mi></mrow><mrow><mi>V</mi><mn>4</mn></mrow></msub><mo>×</mo><msub><mrow><mi>T</mi></mrow><mrow><mi>O</mi><mi>T</mi><mi>U</mi><mi>X</mi></mrow></msub><mo>.</mo></mrow></math></div></div></div>wherein, <em>T<sub>GGraw</sub></em> is a column matrix of size (N<sub>GG</sub> × 1), and N<sub>GG</sub> is the total number of Greengenes OTUs. Finally a percentage normalized OTU abundance profile (<em>T</em><sub>GG%</sub>) is obtained by performing the following transformation on each element (<em>T</em><sub>j</sub>) of T<sub>GGraw</sub><a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mi>T</mi></mrow><mi>j</mi></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>T</mi></mrow><mrow><mi>G</mi><mi>G</mi><mi>r</mi><mi>a</mi><mi>w</mi><mi>j</mi></mrow></msub></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><msub><mrow><mi>T</mi></mrow><mrow><mi>G</mi><mi>G</mi><mi>r</mi><mi>a</mi><mi>w</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo>*</mo><mn>100.</mn></mrow></math></div></div></div>wherein, <em>T</em><sub>GG%</sub> is a column matrix of size (N<sub>GG</sub> × 1), and N<sub>GG</sub> is the total number of Greengenes OTUs. The abundance of taxonomic groups present in the metagenomic sample, as obtained in the form of either of the three column matrices, viz. T<sub>OTUX</sub>, T<sub>GG</sub> and T<sub>GG%</sub>, can further be represented at any desired taxonomic level utilizing the taxonomic hierarchy information associated with the Greengenes OTUs.</div>
<p>As mentioned earlier, the current version of OTUX approach has been modelled around the Greengenes database and designed to pick OTUs at the most stringent threshold suggested by Greengenes (i.e. 99% identity). The choice of threshold was guided by the rationale that OTUs picked based on a more stringent identity threshold (e.g. 99%) can later be merged together to make larger OTU clusters corresponding to a lower threshold (e.g. 97%). Further, it has also been reported that stringent average nucleotide identity cut-off of &gt;99% might be required to delineate a new species.<a reveal-id="dsy045-B30" data-open="dsy045-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Although clustering at 97% identity threshold would generate lesser number of clusters and make the data more amenable for analysis, a higher threshold of 99% is expected to be more suitable for accurate identification of strains and ecotypes. However, given that some studies may also prefer lower identity thresholds (e.g. 97%) for OTU clustering, the implementation of OTUX approach (available at <a class="link link-uri" href="https://web.rniapps.net/otux/" target="_blank">https://web.rniapps.net/otux/</a>) provides option to map back OTUX OTUs (originally picked at 99% identity threshold) to Greengenes OTU IDs corresponding to OTUs clustered at identity thresholds of 99% as well as 97%.</p>                    <h3 scrollto-destination="134009573" id="134009573" class="section-title">2.5. Classifying simulated metagenomic sequences (validation datasets)</h3>
<p>The full length sequences belonging to each of the simulated metagenomic datasets (D<sub>GUT/F</sub>, D<sub>SKIN/F</sub>, D<sub>SEA/F</sub> and D<sub>SOIL/F</sub>) were first subjected to ‘OTU picking’ (taxonomic classification at the OTU level) against the Greengenes database (v13.8—<a class="link link-uri" href="http://ftp://greengenes.microbio.me/greengenes_release/gg_13_8_otus/rep_set/99_otus.fasta" target="_blank">ftp://greengenes.microbio.me/greengenes_release/gg_13_8_otus/rep_set/99_otus.fasta</a>) using the naïve Bayesian classifier (Wang’s algorithm) as implemented in the software ‘mothur’ (with a bootstrap confidence threshold of 80%). Given that full-length 16S sequences were compared against a full-length 16S rRNA sequence database, the results obtained using the above procedure reflects the best achievable OTU-classification using 16S rRNA amplicon sequencing (using the given algorithm) and was considered as a ‘benchmark’ or the ‘gold standard’ (abbreviated as GS) of taxonomic classification for our validation study.</p>
<p>The simulated ‘short-read’ metagenomic sequences in the validation datasets were subsequently subjected to taxonomic classification using the following three methods. For ease of explanation, examples corresponding to the simulated metagenomic datasets pertaining to V4 region (D<sub>GUT/V4</sub>, D<sub>SKIN/V4</sub>, D<sub>SEA/V4</sub> and D<sub>SOIL/V4</sub>) have been used in the following sections. The same methods have been followed for validation using other simulated metagenomic datasets.</p>                    <h4 scrollto-destination="134009576" id="134009576" class="section-title">2.5.1. Conventional approach</h4>
<p>Sequences present in each of the simulated metagenomes belonging to the sets D<sub>GUT/V4</sub>, D<sub>SKIN/V4</sub>, D<sub>SEA/V4</sub> and D<sub>SOIL/V4</sub> were classified using the naïve Bayesian classifier (Wang’s algorithm as implemented in the software ‘mothur’ with a bootstrap confidence threshold of 80%), considering the Greengenes OTU database as a reference (similar approach as GS). These results represent taxonomic classification that can be obtained using the CA of OTU-picking or taxonomic classification wherein short-read sequences (encompassing a certain region of a marker gene) are used as a query against a OTU database constituted of full length marker gene sequences. For ease of comparison, abundance profiles (as per Greengenes taxonomy) representing the proportion of OTUs (and other taxonomic hierarchical levels), both in terms of raw sequence counts as well as percentage normalized abundance, were generated.</p>                    <h4 scrollto-destination="134009578" id="134009578" class="section-title">2.5.2. <em>De novo</em> OTU-picking approach using CROP (CR)</h4>
<p>Metagenomic sequences belonging to each of the validation sets (D<sub>GUT/V4</sub>, D<sub>SKIN/V4</sub>, D<sub>SEA/V4</sub> and D<sub>SOIL/V4</sub>) were clustered <em>de novo</em> at 99% identity using the software CROP,<a reveal-id="dsy045-B31" data-open="dsy045-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> wherein each cluster represents a OTU. Subsequently, in order to obtain taxonomic classification of the respective clusters/OTUs, the representative sequences from each of the clusters were classified using the Greengenes OTU database as a reference. The taxonomic profile T<sub>CR</sub> reports abundance of OTUs generated using CROP, wherein abundance of each OTU is equivalent to its cluster size.</p>                    <h4 scrollto-destination="134009580" id="134009580" class="section-title">2.5.3. OTUX approach (OTUX)</h4>
<p>Metagenomic sequences belonging to each of the validation sets (D<sub>GUT/V4</sub>, D<sub>SKIN/V4</sub>, D<sub>SEA/V4</sub> and D<sub>SOIL/V4</sub>) were also classified using the naïve Bayesian classifier (Wang’s algorithm as implemented in the software mother with a bootstrap confidence threshold of 80%), using the OTUX<sub>V4</sub> database as a reference. These results represent the taxonomic classification that can be obtained using the novel OTUX approach of OTU-picking and taxonomic classification wherein short-read sequences (covering a certain region of a marker gene) are used as query against a pre-computed OTU database corresponding to a specific hypervariable region (V4 in this case). It may be noted that the obtained OTU abundance profile (T<sub>OTUX</sub>) reports the results in terms of OTUX<sub>V4</sub> OTU IDs. For ease of comparison, these results are also mapped back in terms of Greengenes OTU IDs and are provided in form of OTU abundance profile T<sub>GG</sub> (using one-to-one mapping approach), wherein raw counts of sequences assigned to individual Greengenes OTUs are depicted. Furthermore, percentage normalized abundance profile(s) T<sub>GG%</sub> (using one-to-many mapping approach) are also generated, wherein abundance/proportion of OTUs (and/or other taxa) is represented in percent normalized terms.</p>                    <h3 scrollto-destination="134009582" id="134009582" class="section-title">2.6. Validation tests</h3>
<p>The correctness of taxonomic assignments obtained with OTUX and CA approaches were assessed by comparing them against the benchmark/‘GS’, which corresponds to OTU assignments obtained using full-length 16S rRNA gene sequences searched against the Greengenes reference database. The comparison was based on the following parameters. First, the accuracy of taxonomic assignments by CA and OTUX at OTU, Genus, and Family level(s) were assessed in terms of correct number of assignments (with reference to GS results). During this assessment, one-to-one mapping of OTUX IDs to Greengenes IDs had been used. Subsequently, a t-test was performed to assess the performance of OTUX with respect to CA (based on results of 100 simulated metagenomic datasets). The approaches were also compared on the basis of computational time required for classification.</p>
<p>To evaluate the overall accuracy as well as coherence of taxonomic assignments obtained using different V-regions and methods, the widely used Unifrac distance metric was chosen.<a reveal-id="dsy045-B32" data-open="dsy045-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Unifrac is a dissimilarity measure which considers phylogenetic information of constituent microbes while comparing microbiome samples, thereby allowing assessment of overall accuracy considering all taxonomic hierarchical levels. Pairwise weighted Unifrac distances were calculated between percentage normalized abundance tables generated by GS, OTUX, CA and CR. In case of OTUX, CA and CR, taxonomic profiles obtained using all relevant V-regions or stretches of V-regions were considered. For this assessment, percentage normalized OTUX abundance profiles (<em>T</em><sub>GG%</sub>) were generated using one-to-many mapping approach between OTUX IDs and Greengenes IDs. Dendograms were constructed from the pairwise Unifrac distances (averaged for 100 simulated metagenomic datasets) using UPGMA clustering, as implemented in the programme ‘Neighbor’ included in the Phylip package,<a reveal-id="dsy045-B33" data-open="dsy045-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> considering GS as an outgroup. This process was repeated for the simulated metagenomes corresponding to all four selected environments (D<sub>GUT</sub>, D<sub>SKIN</sub>, D<sub>SEA</sub> and D<sub>SOIL</sub>). Different diversity measures for the taxonomic profiles obtained with GS, OTUX, CA and CR approaches, viz. Shannon diversity, Simpson index (evenness), Chao1 index (species richness) and number of observed species, were also computed for the D<sub>GUT</sub> environment, using the R Phyloseq package.<a reveal-id="dsy045-B34" data-open="dsy045-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>
<div class="block-child-p">In addition to checking for phylogenetic accuracy, it was also essential to validate whether OTUX results were closer to <em>de novo</em> OTU-picking results, when compared with the CA. For this purpose, the clusters generated using CROP (wherein each of these clusters represent an OTU) were compared against OTUs identified by OTUX and CA. 100 largest clusters/OTUs (for any selected V-region/stretches of V-regions) obtained from OTUX, CA and CROP were selected. Jaccard similarity was calculated between each cluster <em>i</em> obtained by OTUX against all the clusters obtained using CROP with the following formula, <a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mtext mathvariant="italic">Jaccard Similarity</mtext><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mi>i</mi><mo>∩</mo><mi>j</mi></mrow><mrow><mi>i</mi><mo>∪</mo><mi>j</mi></mrow></mfrac></math></div></div></div>where, <em>j</em> represents each cluster obtained using CROP and <em>j</em> = 1→ 100 (total number of selected clusters). The cluster pair with highest Jaccard similarity was considered. Similarly, a reciprocal search was also performed wherein the closest OTUX cluster to any of the CROP clusters were identified. Consequently, 200 pairs of closest clusters (in terms of Jaccard similarity) were obtained between OTUX and CROP, with a fraction of them being reciprocal best hits. An average Jaccard similarity score was calculated using the formula, <a id="E5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>A</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mo> </mo><mi>J</mi><mi>a</mi><mi>c</mi><mi>c</mi><mi>a</mi><mi>r</mi><mi>d</mi><mo> </mo><mi>S</mi><mi>i</mi><mi>m</mi><mi>i</mi><mi>l</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>t</mi><mi>y</mi><mo> </mo><mo>=</mo><mfrac><mrow><mo>∑</mo><mi>J</mi><mi>a</mi><mi>c</mi><mi>c</mi><mi>a</mi><mi>r</mi><mi>d</mi><mo> </mo><mi>s</mi><mi>i</mi><mi>m</mi><mi>i</mi><mi>l</mi><mi>a</mi><mi>r</mi><mi>i</mi><mi>t</mi><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><mn>200</mn></mrow></mfrac><mo>.</mo></mrow></math></div></div></div>The similarity in OTU clustering between OTUX and CROP was subsequently evaluated in terms of average Jaccard similarity and the number of reciprocal best hits. For comparison, the similarity between CA and CROP approaches was also evaluated using the above mentioned steps. Additional analyses were also performed to evaluate the effect of different sequence identity thresholds for clustering as well as the effects of inclusion of relatively smaller clusters (in addition to the 100 largest clusters) during calculation of average Jaccard similarity. To this end, the above mentioned analyses were extended for the top 300 and 500 (most populous) clusters generated for the simulated gut microbiome (GUT) dataset with <em>de novo</em> clustering (CROP) thresholds set at 99%, 97% as well as 95%.</div>                    <h2 scrollto-destination="134009586" id="134009586" class="section-title">3. Results and discussion</h2>
                    <h3 scrollto-destination="134009587" id="134009587" class="section-title">3.1. OTUX approach provides higher proportion of correct taxonomic classifications</h3>
<p><a reveal-id="dsy045-T1" data-open="dsy045-T1" class="link link-reveal link-table xref-fig">Table 1</a> provides a comparison of taxonomic classification accuracy of the proposed OTUX approach, with respect to CA, while classifying different simulated microbiome datasets. As mentioned earlier, CA involves reference-based OTU picking from a query set of short 16S rRNA gene reads using an OTU database consisting of full length 16S rRNA gene sequences. In this study, Greengenes version 13.8, containing OTUs clustered at 99% sequence identity, was used as the reference database for CA. In contrast, OTUX approach involves reference-based OTU picking of the same short reads using an appropriate OTUX V-region database (consisting sequences clustered with 99% identity). OTUs identified using OTUX approach, initially classified in terms of OTUX database identifiers (OTUX IDs), was ‘mapped back’ to be represented in terms of Greengenes IDs (see Materials and methods), so as to enable comparison against CA. Results obtained with both CA and OTUX approaches were evaluated with reference to a ‘GS’, which corresponded to the reference-based OTU-picking results (in terms of Greengenes IDs) that could be obtained using full length 16S rRNA gene amplicons instead of short-read sequences (see Materials and methods). In terms of number of sequences correctly assigned at the OTU level, OTUX approach significantly (<em>P</em> &lt; 0.001) outperforms CA in almost all cases, irrespective of the type of microbiome sample or the targeted V-region (<a reveal-id="dsy045-T1" data-open="dsy045-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Barring a couple of occasions, OTUX is observed to consistently assign over 50% of the query reads to appropriate OTUs, with accuracies reaching as high as 70% in some cases. More than often, OTUX could correctly assign almost double the number of reads compared with CA. CA provides better result than OTUX only in a single instance, wherein infant gut microbiome (GUT) samples were probed using V1–V3 region. In addition to the number of correct OTU assignments, the results of OTUX were also evaluated in terms of overall correctness of the obtained taxonomic profiles for the simulated microbiome samples considering different levels of taxonomic hierarchy (<a reveal-id="dsy045-T1" data-open="dsy045-T1" class="link link-reveal link-table xref-fig">Table 1</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2A</a></span> and B; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S3). Weighted-UniFrac distances between the taxonomic abundance profiles obtained using OTUX (see Materials and methods) and those obtained using GS (using full-length 16S rRNA gene sequences) were computed for this purpose. For comparison, distances between taxonomic profiles obtained using CA and GS were also calculated. Echoing earlier results, taxonomic abundance profiles obtained with OTUX are found to be more similar to the GS taxonomic profiles as compared with the profiles obtained with CA, in almost all cases. The only exceptions pertain to the profiles generated for the infant gut (GUT) microbiome corresponding to experiments targeting the V1–V2 and V1–V3 regions. In summary, OTUX approach appears to be a more reliable method for reference-based OTU picking, providing greater accuracy in taxonomic classification than existing CAs utilizing full-length 16S rRNA sequence databases.</p>                        <a id="134009589" scrollto-destination="134009589"></a>
<div content-id="dsy045-T1" class="table-modal table-full-width-wrap"><div class="table-wrap">
<div id="dsy045-T1" class="table-wrap-title" data-id="dsy045-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Taxonomic classification results obtained with the proposed OTUX approach and CA</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3">Simulated microbiomes </th>
<th colspan="2" rowspan="2">Classification accuracies<hr> </th>
<th colspan="11">Target V-region<hr> </th>
</tr>
<tr>
<th>V1V2<hr> </th>
<th>V1V3<hr> </th>
<th>V2V3<hr> </th>
<th>V3V4<hr> </th>
<th>V3V5<hr> </th>
<th>V3V6<hr> </th>
<th>V4<hr> </th>
<th>V4V5<hr> </th>
<th>V4V6<hr> </th>
<th>V5V6<hr> </th>
<th>V6V8<hr> </th>
</tr>
<tr><th colspan="13">Comparison of taxonomic assignments<sup>a</sup> at OTU level (out of 10,000 reads; averaged for 100 randomly generated simulated datasets) </th></tr>
</thead>
<tbody>
<tr>
<td rowspan="6">
<strong>GUT</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4386.8</strong> </td>
<td>4355.2 </td>
<td>
<strong>5461.2</strong> </td>
<td>
<strong>6265.4</strong> </td>
<td>
<strong>6600.5</strong> </td>
<td>
<strong>6957.4</strong> </td>
<td>
<strong>5795.3</strong> </td>
<td>
<strong>6869.8</strong> </td>
<td>
<strong>6849.7</strong> </td>
<td>
<strong>5862.2</strong> </td>
<td>
<strong>6771.3</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>47.2 </td>
<td>84.3 </td>
<td>102.5 </td>
<td>151.9 </td>
<td>229.2 </td>
<td>97.4 </td>
<td>175.9 </td>
<td>57.4 </td>
<td>78.5 </td>
<td>110.1 </td>
<td>40.3 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.324 </td>
<td>0.292 </td>
<td>0.234 </td>
<td>0.236 </td>
<td>0.213 </td>
<td>0.202 </td>
<td>0.288 </td>
<td>0.216 </td>
<td>0.207 </td>
<td>0.236 </td>
<td>0.215 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>3618.6 </td>
<td>4464.1 </td>
<td>3808.6 </td>
<td>3084.6 </td>
<td>3759.3 </td>
<td>4748.2 </td>
<td>2318.5 </td>
<td>2872.7 </td>
<td>4162.0 </td>
<td>3084.7 </td>
<td>3429.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>555.2 </td>
<td>244.4 </td>
<td>1020.9 </td>
<td>1582.2 </td>
<td>935.7 </td>
<td>490.4 </td>
<td>2631.3 </td>
<td>814.5 </td>
<td>531.8 </td>
<td>1785.7 </td>
<td>1154.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.318 </td>
<td>0.272 </td>
<td>0.280 </td>
<td>0.310 </td>
<td>0.289 </td>
<td>0.238 </td>
<td>0.332 </td>
<td>0.361 </td>
<td>0.270 </td>
<td>0.300 </td>
<td>0.311 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SKIN</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4034.4</strong> </td>
<td>
<strong>4257.3</strong> </td>
<td>
<strong>5377.9</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5710.8</strong> </td>
<td>
<strong>5759.2</strong> </td>
<td>
<strong>3800.5</strong> </td>
<td>
<strong>5027.3</strong> </td>
<td>
<strong>5800.2</strong> </td>
<td>
<strong>5472.8</strong> </td>
<td>
<strong>5295.9</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>52.7 </td>
<td>44.9 </td>
<td>59.8 </td>
<td>163.7 </td>
<td>47.5 </td>
<td>52.6 </td>
<td>34.0 </td>
<td>38.3 </td>
<td>46.8 </td>
<td>52.9 </td>
<td>63.8 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.218 </td>
<td>0.193 </td>
<td>0.101 </td>
<td>0.123 </td>
<td>0.129 </td>
<td>0.115 </td>
<td>0.237 </td>
<td>0.118 </td>
<td>0.125 </td>
<td>0.104 </td>
<td>0.104 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>1330.5 </td>
<td>1789.3 </td>
<td>1807.7 </td>
<td>1478.2 </td>
<td>2254.9 </td>
<td>3238.6 </td>
<td>320.4 </td>
<td>1172.1 </td>
<td>2164.7 </td>
<td>1491.7 </td>
<td>1816.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1508.7 </td>
<td>625.9 </td>
<td>1477.0 </td>
<td>1965.6 </td>
<td>1476.7 </td>
<td>727.5 </td>
<td>1652.0 </td>
<td>1466.1 </td>
<td>1460.9 </td>
<td>2346.1 </td>
<td>1692.2 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.402 </td>
<td>0.388 </td>
<td>0.345 </td>
<td>0.349 </td>
<td>0.317 </td>
<td>0.266 </td>
<td>0.469 </td>
<td>0.387 </td>
<td>0.319 </td>
<td>0.333 </td>
<td>0.336 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SEA</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5017.8</strong> </td>
<td>
<strong>6085.1</strong> </td>
<td>
<strong>6348.9</strong> </td>
<td>
<strong>6290.1</strong> </td>
<td>
<strong>6297.6</strong> </td>
<td>
<strong>7037.4</strong> </td>
<td>
<strong>4932.7</strong> </td>
<td>
<strong>6495.9</strong> </td>
<td>
<strong>6652.1</strong> </td>
<td>
<strong>6253.7</strong> </td>
<td>
<strong>6695.5</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>18.1 </td>
<td>11.0 </td>
<td>24.0 </td>
<td>77.8 </td>
<td>9.2 </td>
<td>16.7 </td>
<td>1.7 </td>
<td>7.6 </td>
<td>15.9 </td>
<td>13.3 </td>
<td>32.6 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.249 </td>
<td>0.152 </td>
<td>0.121 </td>
<td>0.127 </td>
<td>0.145 </td>
<td>0.145 </td>
<td>0.249 </td>
<td>0.146 </td>
<td>0.116 </td>
<td>0.155 </td>
<td>0.130 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>2579.8 </td>
<td>3471.2 </td>
<td>3180.7 </td>
<td>3287.6 </td>
<td>4562.6 </td>
<td>4753.3 </td>
<td>2183.1 </td>
<td>3197.9 </td>
<td>4240.4 </td>
<td>2866.6 </td>
<td>4047.9 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1450.0 </td>
<td>1638.7 </td>
<td>1415.3 </td>
<td>1174.7 </td>
<td>1108.1 </td>
<td>1049.9 </td>
<td>1531.6 </td>
<td>925.8 </td>
<td>1095.9 </td>
<td>1644.5 </td>
<td>1801.0 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.385 </td>
<td>0.316 </td>
<td>0.333 </td>
<td>0.319 </td>
<td>0.224 </td>
<td>0.218 </td>
<td>0.397 </td>
<td>0.328 </td>
<td>0.256 </td>
<td>0.335 </td>
<td>0.256 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SOIL</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5109.9</strong> </td>
<td>
<strong>5086.0</strong> </td>
<td>
<strong>5540.1</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5948.4</strong> </td>
<td>
<strong>5800.0</strong> </td>
<td>
<strong>4155.5</strong> </td>
<td>
<strong>5257.0</strong> </td>
<td>
<strong>5613.2</strong> </td>
<td>
<strong>5222.1</strong> </td>
<td>
<strong>5814.4</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>46.5 </td>
<td>45.4 </td>
<td>54.6 </td>
<td>225.5 </td>
<td>69.0 </td>
<td>51.4 </td>
<td>68.3 </td>
<td>71.5 </td>
<td>47.7 </td>
<td>45.5 </td>
<td>85.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.145 </td>
<td>0.114 </td>
<td>0.078 </td>
<td>0.123 </td>
<td>0.127 </td>
<td>0.153 </td>
<td>0.225 </td>
<td>0.133 </td>
<td>0.120 </td>
<td>0.162 </td>
<td>0.125 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>4089.1 </td>
<td>4680.5 </td>
<td>4703.2 </td>
<td>1478.2 </td>
<td>5176.7 </td>
<td>5637.2 </td>
<td>2287.0 </td>
<td>3806.7 </td>
<td>5357.6 </td>
<td>3714.7 </td>
<td>4388.2 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>690.0 </td>
<td>248.5 </td>
<td>191.5 </td>
<td>444.4 </td>
<td>301.4 </td>
<td>129.3 </td>
<td>685.9 </td>
<td>806.1 </td>
<td>279.4 </td>
<td>833.5 </td>
<td>390.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.187 </td>
<td>0.158 </td>
<td>0.154 </td>
<td>0.349 </td>
<td>0.104 </td>
<td>0.084 </td>
<td>0.334 </td>
<td>0.184 </td>
<td>0.089 </td>
<td>0.193 </td>
<td>0.156 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy045-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3">Simulated microbiomes </th>
<th colspan="2" rowspan="2">Classification accuracies<hr> </th>
<th colspan="11">Target V-region<hr> </th>
</tr>
<tr>
<th>V1V2<hr> </th>
<th>V1V3<hr> </th>
<th>V2V3<hr> </th>
<th>V3V4<hr> </th>
<th>V3V5<hr> </th>
<th>V3V6<hr> </th>
<th>V4<hr> </th>
<th>V4V5<hr> </th>
<th>V4V6<hr> </th>
<th>V5V6<hr> </th>
<th>V6V8<hr> </th>
</tr>
<tr><th colspan="13">Comparison of taxonomic assignments<sup>a</sup> at OTU level (out of 10,000 reads; averaged for 100 randomly generated simulated datasets) </th></tr>
</thead>
<tbody>
<tr>
<td rowspan="6">
<strong>GUT</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4386.8</strong> </td>
<td>4355.2 </td>
<td>
<strong>5461.2</strong> </td>
<td>
<strong>6265.4</strong> </td>
<td>
<strong>6600.5</strong> </td>
<td>
<strong>6957.4</strong> </td>
<td>
<strong>5795.3</strong> </td>
<td>
<strong>6869.8</strong> </td>
<td>
<strong>6849.7</strong> </td>
<td>
<strong>5862.2</strong> </td>
<td>
<strong>6771.3</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>47.2 </td>
<td>84.3 </td>
<td>102.5 </td>
<td>151.9 </td>
<td>229.2 </td>
<td>97.4 </td>
<td>175.9 </td>
<td>57.4 </td>
<td>78.5 </td>
<td>110.1 </td>
<td>40.3 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.324 </td>
<td>0.292 </td>
<td>0.234 </td>
<td>0.236 </td>
<td>0.213 </td>
<td>0.202 </td>
<td>0.288 </td>
<td>0.216 </td>
<td>0.207 </td>
<td>0.236 </td>
<td>0.215 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>3618.6 </td>
<td>4464.1 </td>
<td>3808.6 </td>
<td>3084.6 </td>
<td>3759.3 </td>
<td>4748.2 </td>
<td>2318.5 </td>
<td>2872.7 </td>
<td>4162.0 </td>
<td>3084.7 </td>
<td>3429.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>555.2 </td>
<td>244.4 </td>
<td>1020.9 </td>
<td>1582.2 </td>
<td>935.7 </td>
<td>490.4 </td>
<td>2631.3 </td>
<td>814.5 </td>
<td>531.8 </td>
<td>1785.7 </td>
<td>1154.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.318 </td>
<td>0.272 </td>
<td>0.280 </td>
<td>0.310 </td>
<td>0.289 </td>
<td>0.238 </td>
<td>0.332 </td>
<td>0.361 </td>
<td>0.270 </td>
<td>0.300 </td>
<td>0.311 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SKIN</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4034.4</strong> </td>
<td>
<strong>4257.3</strong> </td>
<td>
<strong>5377.9</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5710.8</strong> </td>
<td>
<strong>5759.2</strong> </td>
<td>
<strong>3800.5</strong> </td>
<td>
<strong>5027.3</strong> </td>
<td>
<strong>5800.2</strong> </td>
<td>
<strong>5472.8</strong> </td>
<td>
<strong>5295.9</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>52.7 </td>
<td>44.9 </td>
<td>59.8 </td>
<td>163.7 </td>
<td>47.5 </td>
<td>52.6 </td>
<td>34.0 </td>
<td>38.3 </td>
<td>46.8 </td>
<td>52.9 </td>
<td>63.8 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.218 </td>
<td>0.193 </td>
<td>0.101 </td>
<td>0.123 </td>
<td>0.129 </td>
<td>0.115 </td>
<td>0.237 </td>
<td>0.118 </td>
<td>0.125 </td>
<td>0.104 </td>
<td>0.104 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>1330.5 </td>
<td>1789.3 </td>
<td>1807.7 </td>
<td>1478.2 </td>
<td>2254.9 </td>
<td>3238.6 </td>
<td>320.4 </td>
<td>1172.1 </td>
<td>2164.7 </td>
<td>1491.7 </td>
<td>1816.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1508.7 </td>
<td>625.9 </td>
<td>1477.0 </td>
<td>1965.6 </td>
<td>1476.7 </td>
<td>727.5 </td>
<td>1652.0 </td>
<td>1466.1 </td>
<td>1460.9 </td>
<td>2346.1 </td>
<td>1692.2 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.402 </td>
<td>0.388 </td>
<td>0.345 </td>
<td>0.349 </td>
<td>0.317 </td>
<td>0.266 </td>
<td>0.469 </td>
<td>0.387 </td>
<td>0.319 </td>
<td>0.333 </td>
<td>0.336 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SEA</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5017.8</strong> </td>
<td>
<strong>6085.1</strong> </td>
<td>
<strong>6348.9</strong> </td>
<td>
<strong>6290.1</strong> </td>
<td>
<strong>6297.6</strong> </td>
<td>
<strong>7037.4</strong> </td>
<td>
<strong>4932.7</strong> </td>
<td>
<strong>6495.9</strong> </td>
<td>
<strong>6652.1</strong> </td>
<td>
<strong>6253.7</strong> </td>
<td>
<strong>6695.5</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>18.1 </td>
<td>11.0 </td>
<td>24.0 </td>
<td>77.8 </td>
<td>9.2 </td>
<td>16.7 </td>
<td>1.7 </td>
<td>7.6 </td>
<td>15.9 </td>
<td>13.3 </td>
<td>32.6 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.249 </td>
<td>0.152 </td>
<td>0.121 </td>
<td>0.127 </td>
<td>0.145 </td>
<td>0.145 </td>
<td>0.249 </td>
<td>0.146 </td>
<td>0.116 </td>
<td>0.155 </td>
<td>0.130 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>2579.8 </td>
<td>3471.2 </td>
<td>3180.7 </td>
<td>3287.6 </td>
<td>4562.6 </td>
<td>4753.3 </td>
<td>2183.1 </td>
<td>3197.9 </td>
<td>4240.4 </td>
<td>2866.6 </td>
<td>4047.9 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1450.0 </td>
<td>1638.7 </td>
<td>1415.3 </td>
<td>1174.7 </td>
<td>1108.1 </td>
<td>1049.9 </td>
<td>1531.6 </td>
<td>925.8 </td>
<td>1095.9 </td>
<td>1644.5 </td>
<td>1801.0 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.385 </td>
<td>0.316 </td>
<td>0.333 </td>
<td>0.319 </td>
<td>0.224 </td>
<td>0.218 </td>
<td>0.397 </td>
<td>0.328 </td>
<td>0.256 </td>
<td>0.335 </td>
<td>0.256 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SOIL</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5109.9</strong> </td>
<td>
<strong>5086.0</strong> </td>
<td>
<strong>5540.1</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5948.4</strong> </td>
<td>
<strong>5800.0</strong> </td>
<td>
<strong>4155.5</strong> </td>
<td>
<strong>5257.0</strong> </td>
<td>
<strong>5613.2</strong> </td>
<td>
<strong>5222.1</strong> </td>
<td>
<strong>5814.4</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>46.5 </td>
<td>45.4 </td>
<td>54.6 </td>
<td>225.5 </td>
<td>69.0 </td>
<td>51.4 </td>
<td>68.3 </td>
<td>71.5 </td>
<td>47.7 </td>
<td>45.5 </td>
<td>85.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.145 </td>
<td>0.114 </td>
<td>0.078 </td>
<td>0.123 </td>
<td>0.127 </td>
<td>0.153 </td>
<td>0.225 </td>
<td>0.133 </td>
<td>0.120 </td>
<td>0.162 </td>
<td>0.125 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>4089.1 </td>
<td>4680.5 </td>
<td>4703.2 </td>
<td>1478.2 </td>
<td>5176.7 </td>
<td>5637.2 </td>
<td>2287.0 </td>
<td>3806.7 </td>
<td>5357.6 </td>
<td>3714.7 </td>
<td>4388.2 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>690.0 </td>
<td>248.5 </td>
<td>191.5 </td>
<td>444.4 </td>
<td>301.4 </td>
<td>129.3 </td>
<td>685.9 </td>
<td>806.1 </td>
<td>279.4 </td>
<td>833.5 </td>
<td>390.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.187 </td>
<td>0.158 </td>
<td>0.154 </td>
<td>0.349 </td>
<td>0.104 </td>
<td>0.084 </td>
<td>0.334 </td>
<td>0.184 </td>
<td>0.089 </td>
<td>0.193 </td>
<td>0.156 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn"><p>Performance of both the approaches have been evaluated with multiple simulated microbiome datasets mimicking different types of environmental samples, (viz., infant gut, skin, sea and soil), and amplicon sequencing experiments targeting different V-regions of the bacterial 16S rRNA gene. Cases wherein the number of correct assignments by OTUX are significantly higher (<em>t</em>-test; <em>P</em> &lt; 0.001) have been highlighted in bold.</p></div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">a</div>
<p>CA and OTUX approaches have been evaluated considering results obtained with full-length 16S rRNA genes to be ‘correct’ or the ‘GS’. Only number of average correct assignments and wrong assignments has been depicted. The remaining reads (out of 10,000) could not be classified at OTU level.</p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">b</div>
<p>Weighted UNIFRAC distance of taxonomic abundance profile from actual taxonomic diversity (GS: gold-standard).</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134009589" target="_blank">Open in new tab</a></div>
</div></div>
<div class="table-full-width-wrap"><div class="table-wrap">
<div id="dsy045-T1" class="table-wrap-title" data-id="dsy045-T1">
<div class="label">Table 1</div>
<div class="caption"><p>Taxonomic classification results obtained with the proposed OTUX approach and CA</p></div> </div>
<div class="table-overflow"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3">Simulated microbiomes </th>
<th colspan="2" rowspan="2">Classification accuracies<hr> </th>
<th colspan="11">Target V-region<hr> </th>
</tr>
<tr>
<th>V1V2<hr> </th>
<th>V1V3<hr> </th>
<th>V2V3<hr> </th>
<th>V3V4<hr> </th>
<th>V3V5<hr> </th>
<th>V3V6<hr> </th>
<th>V4<hr> </th>
<th>V4V5<hr> </th>
<th>V4V6<hr> </th>
<th>V5V6<hr> </th>
<th>V6V8<hr> </th>
</tr>
<tr><th colspan="13">Comparison of taxonomic assignments<sup>a</sup> at OTU level (out of 10,000 reads; averaged for 100 randomly generated simulated datasets) </th></tr>
</thead>
<tbody>
<tr>
<td rowspan="6">
<strong>GUT</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4386.8</strong> </td>
<td>4355.2 </td>
<td>
<strong>5461.2</strong> </td>
<td>
<strong>6265.4</strong> </td>
<td>
<strong>6600.5</strong> </td>
<td>
<strong>6957.4</strong> </td>
<td>
<strong>5795.3</strong> </td>
<td>
<strong>6869.8</strong> </td>
<td>
<strong>6849.7</strong> </td>
<td>
<strong>5862.2</strong> </td>
<td>
<strong>6771.3</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>47.2 </td>
<td>84.3 </td>
<td>102.5 </td>
<td>151.9 </td>
<td>229.2 </td>
<td>97.4 </td>
<td>175.9 </td>
<td>57.4 </td>
<td>78.5 </td>
<td>110.1 </td>
<td>40.3 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.324 </td>
<td>0.292 </td>
<td>0.234 </td>
<td>0.236 </td>
<td>0.213 </td>
<td>0.202 </td>
<td>0.288 </td>
<td>0.216 </td>
<td>0.207 </td>
<td>0.236 </td>
<td>0.215 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>3618.6 </td>
<td>4464.1 </td>
<td>3808.6 </td>
<td>3084.6 </td>
<td>3759.3 </td>
<td>4748.2 </td>
<td>2318.5 </td>
<td>2872.7 </td>
<td>4162.0 </td>
<td>3084.7 </td>
<td>3429.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>555.2 </td>
<td>244.4 </td>
<td>1020.9 </td>
<td>1582.2 </td>
<td>935.7 </td>
<td>490.4 </td>
<td>2631.3 </td>
<td>814.5 </td>
<td>531.8 </td>
<td>1785.7 </td>
<td>1154.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.318 </td>
<td>0.272 </td>
<td>0.280 </td>
<td>0.310 </td>
<td>0.289 </td>
<td>0.238 </td>
<td>0.332 </td>
<td>0.361 </td>
<td>0.270 </td>
<td>0.300 </td>
<td>0.311 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SKIN</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4034.4</strong> </td>
<td>
<strong>4257.3</strong> </td>
<td>
<strong>5377.9</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5710.8</strong> </td>
<td>
<strong>5759.2</strong> </td>
<td>
<strong>3800.5</strong> </td>
<td>
<strong>5027.3</strong> </td>
<td>
<strong>5800.2</strong> </td>
<td>
<strong>5472.8</strong> </td>
<td>
<strong>5295.9</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>52.7 </td>
<td>44.9 </td>
<td>59.8 </td>
<td>163.7 </td>
<td>47.5 </td>
<td>52.6 </td>
<td>34.0 </td>
<td>38.3 </td>
<td>46.8 </td>
<td>52.9 </td>
<td>63.8 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.218 </td>
<td>0.193 </td>
<td>0.101 </td>
<td>0.123 </td>
<td>0.129 </td>
<td>0.115 </td>
<td>0.237 </td>
<td>0.118 </td>
<td>0.125 </td>
<td>0.104 </td>
<td>0.104 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>1330.5 </td>
<td>1789.3 </td>
<td>1807.7 </td>
<td>1478.2 </td>
<td>2254.9 </td>
<td>3238.6 </td>
<td>320.4 </td>
<td>1172.1 </td>
<td>2164.7 </td>
<td>1491.7 </td>
<td>1816.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1508.7 </td>
<td>625.9 </td>
<td>1477.0 </td>
<td>1965.6 </td>
<td>1476.7 </td>
<td>727.5 </td>
<td>1652.0 </td>
<td>1466.1 </td>
<td>1460.9 </td>
<td>2346.1 </td>
<td>1692.2 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.402 </td>
<td>0.388 </td>
<td>0.345 </td>
<td>0.349 </td>
<td>0.317 </td>
<td>0.266 </td>
<td>0.469 </td>
<td>0.387 </td>
<td>0.319 </td>
<td>0.333 </td>
<td>0.336 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SEA</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5017.8</strong> </td>
<td>
<strong>6085.1</strong> </td>
<td>
<strong>6348.9</strong> </td>
<td>
<strong>6290.1</strong> </td>
<td>
<strong>6297.6</strong> </td>
<td>
<strong>7037.4</strong> </td>
<td>
<strong>4932.7</strong> </td>
<td>
<strong>6495.9</strong> </td>
<td>
<strong>6652.1</strong> </td>
<td>
<strong>6253.7</strong> </td>
<td>
<strong>6695.5</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>18.1 </td>
<td>11.0 </td>
<td>24.0 </td>
<td>77.8 </td>
<td>9.2 </td>
<td>16.7 </td>
<td>1.7 </td>
<td>7.6 </td>
<td>15.9 </td>
<td>13.3 </td>
<td>32.6 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.249 </td>
<td>0.152 </td>
<td>0.121 </td>
<td>0.127 </td>
<td>0.145 </td>
<td>0.145 </td>
<td>0.249 </td>
<td>0.146 </td>
<td>0.116 </td>
<td>0.155 </td>
<td>0.130 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>2579.8 </td>
<td>3471.2 </td>
<td>3180.7 </td>
<td>3287.6 </td>
<td>4562.6 </td>
<td>4753.3 </td>
<td>2183.1 </td>
<td>3197.9 </td>
<td>4240.4 </td>
<td>2866.6 </td>
<td>4047.9 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1450.0 </td>
<td>1638.7 </td>
<td>1415.3 </td>
<td>1174.7 </td>
<td>1108.1 </td>
<td>1049.9 </td>
<td>1531.6 </td>
<td>925.8 </td>
<td>1095.9 </td>
<td>1644.5 </td>
<td>1801.0 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.385 </td>
<td>0.316 </td>
<td>0.333 </td>
<td>0.319 </td>
<td>0.224 </td>
<td>0.218 </td>
<td>0.397 </td>
<td>0.328 </td>
<td>0.256 </td>
<td>0.335 </td>
<td>0.256 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SOIL</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5109.9</strong> </td>
<td>
<strong>5086.0</strong> </td>
<td>
<strong>5540.1</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5948.4</strong> </td>
<td>
<strong>5800.0</strong> </td>
<td>
<strong>4155.5</strong> </td>
<td>
<strong>5257.0</strong> </td>
<td>
<strong>5613.2</strong> </td>
<td>
<strong>5222.1</strong> </td>
<td>
<strong>5814.4</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>46.5 </td>
<td>45.4 </td>
<td>54.6 </td>
<td>225.5 </td>
<td>69.0 </td>
<td>51.4 </td>
<td>68.3 </td>
<td>71.5 </td>
<td>47.7 </td>
<td>45.5 </td>
<td>85.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.145 </td>
<td>0.114 </td>
<td>0.078 </td>
<td>0.123 </td>
<td>0.127 </td>
<td>0.153 </td>
<td>0.225 </td>
<td>0.133 </td>
<td>0.120 </td>
<td>0.162 </td>
<td>0.125 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>4089.1 </td>
<td>4680.5 </td>
<td>4703.2 </td>
<td>1478.2 </td>
<td>5176.7 </td>
<td>5637.2 </td>
<td>2287.0 </td>
<td>3806.7 </td>
<td>5357.6 </td>
<td>3714.7 </td>
<td>4388.2 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>690.0 </td>
<td>248.5 </td>
<td>191.5 </td>
<td>444.4 </td>
<td>301.4 </td>
<td>129.3 </td>
<td>685.9 </td>
<td>806.1 </td>
<td>279.4 </td>
<td>833.5 </td>
<td>390.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.187 </td>
<td>0.158 </td>
<td>0.154 </td>
<td>0.349 </td>
<td>0.104 </td>
<td>0.084 </td>
<td>0.334 </td>
<td>0.184 </td>
<td>0.089 </td>
<td>0.193 </td>
<td>0.156 </td>
</tr>
</tbody>
</table></div>
<div content-id="dsy045-T1" class="table-modal"><table>
<thead name="thead" align="">
<tr>
<th rowspan="3">Simulated microbiomes </th>
<th colspan="2" rowspan="2">Classification accuracies<hr> </th>
<th colspan="11">Target V-region<hr> </th>
</tr>
<tr>
<th>V1V2<hr> </th>
<th>V1V3<hr> </th>
<th>V2V3<hr> </th>
<th>V3V4<hr> </th>
<th>V3V5<hr> </th>
<th>V3V6<hr> </th>
<th>V4<hr> </th>
<th>V4V5<hr> </th>
<th>V4V6<hr> </th>
<th>V5V6<hr> </th>
<th>V6V8<hr> </th>
</tr>
<tr><th colspan="13">Comparison of taxonomic assignments<sup>a</sup> at OTU level (out of 10,000 reads; averaged for 100 randomly generated simulated datasets) </th></tr>
</thead>
<tbody>
<tr>
<td rowspan="6">
<strong>GUT</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4386.8</strong> </td>
<td>4355.2 </td>
<td>
<strong>5461.2</strong> </td>
<td>
<strong>6265.4</strong> </td>
<td>
<strong>6600.5</strong> </td>
<td>
<strong>6957.4</strong> </td>
<td>
<strong>5795.3</strong> </td>
<td>
<strong>6869.8</strong> </td>
<td>
<strong>6849.7</strong> </td>
<td>
<strong>5862.2</strong> </td>
<td>
<strong>6771.3</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>47.2 </td>
<td>84.3 </td>
<td>102.5 </td>
<td>151.9 </td>
<td>229.2 </td>
<td>97.4 </td>
<td>175.9 </td>
<td>57.4 </td>
<td>78.5 </td>
<td>110.1 </td>
<td>40.3 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.324 </td>
<td>0.292 </td>
<td>0.234 </td>
<td>0.236 </td>
<td>0.213 </td>
<td>0.202 </td>
<td>0.288 </td>
<td>0.216 </td>
<td>0.207 </td>
<td>0.236 </td>
<td>0.215 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>3618.6 </td>
<td>4464.1 </td>
<td>3808.6 </td>
<td>3084.6 </td>
<td>3759.3 </td>
<td>4748.2 </td>
<td>2318.5 </td>
<td>2872.7 </td>
<td>4162.0 </td>
<td>3084.7 </td>
<td>3429.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>555.2 </td>
<td>244.4 </td>
<td>1020.9 </td>
<td>1582.2 </td>
<td>935.7 </td>
<td>490.4 </td>
<td>2631.3 </td>
<td>814.5 </td>
<td>531.8 </td>
<td>1785.7 </td>
<td>1154.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.318 </td>
<td>0.272 </td>
<td>0.280 </td>
<td>0.310 </td>
<td>0.289 </td>
<td>0.238 </td>
<td>0.332 </td>
<td>0.361 </td>
<td>0.270 </td>
<td>0.300 </td>
<td>0.311 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SKIN</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>4034.4</strong> </td>
<td>
<strong>4257.3</strong> </td>
<td>
<strong>5377.9</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5710.8</strong> </td>
<td>
<strong>5759.2</strong> </td>
<td>
<strong>3800.5</strong> </td>
<td>
<strong>5027.3</strong> </td>
<td>
<strong>5800.2</strong> </td>
<td>
<strong>5472.8</strong> </td>
<td>
<strong>5295.9</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>52.7 </td>
<td>44.9 </td>
<td>59.8 </td>
<td>163.7 </td>
<td>47.5 </td>
<td>52.6 </td>
<td>34.0 </td>
<td>38.3 </td>
<td>46.8 </td>
<td>52.9 </td>
<td>63.8 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.218 </td>
<td>0.193 </td>
<td>0.101 </td>
<td>0.123 </td>
<td>0.129 </td>
<td>0.115 </td>
<td>0.237 </td>
<td>0.118 </td>
<td>0.125 </td>
<td>0.104 </td>
<td>0.104 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>1330.5 </td>
<td>1789.3 </td>
<td>1807.7 </td>
<td>1478.2 </td>
<td>2254.9 </td>
<td>3238.6 </td>
<td>320.4 </td>
<td>1172.1 </td>
<td>2164.7 </td>
<td>1491.7 </td>
<td>1816.6 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1508.7 </td>
<td>625.9 </td>
<td>1477.0 </td>
<td>1965.6 </td>
<td>1476.7 </td>
<td>727.5 </td>
<td>1652.0 </td>
<td>1466.1 </td>
<td>1460.9 </td>
<td>2346.1 </td>
<td>1692.2 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.402 </td>
<td>0.388 </td>
<td>0.345 </td>
<td>0.349 </td>
<td>0.317 </td>
<td>0.266 </td>
<td>0.469 </td>
<td>0.387 </td>
<td>0.319 </td>
<td>0.333 </td>
<td>0.336 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SEA</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5017.8</strong> </td>
<td>
<strong>6085.1</strong> </td>
<td>
<strong>6348.9</strong> </td>
<td>
<strong>6290.1</strong> </td>
<td>
<strong>6297.6</strong> </td>
<td>
<strong>7037.4</strong> </td>
<td>
<strong>4932.7</strong> </td>
<td>
<strong>6495.9</strong> </td>
<td>
<strong>6652.1</strong> </td>
<td>
<strong>6253.7</strong> </td>
<td>
<strong>6695.5</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>18.1 </td>
<td>11.0 </td>
<td>24.0 </td>
<td>77.8 </td>
<td>9.2 </td>
<td>16.7 </td>
<td>1.7 </td>
<td>7.6 </td>
<td>15.9 </td>
<td>13.3 </td>
<td>32.6 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.249 </td>
<td>0.152 </td>
<td>0.121 </td>
<td>0.127 </td>
<td>0.145 </td>
<td>0.145 </td>
<td>0.249 </td>
<td>0.146 </td>
<td>0.116 </td>
<td>0.155 </td>
<td>0.130 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>2579.8 </td>
<td>3471.2 </td>
<td>3180.7 </td>
<td>3287.6 </td>
<td>4562.6 </td>
<td>4753.3 </td>
<td>2183.1 </td>
<td>3197.9 </td>
<td>4240.4 </td>
<td>2866.6 </td>
<td>4047.9 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>1450.0 </td>
<td>1638.7 </td>
<td>1415.3 </td>
<td>1174.7 </td>
<td>1108.1 </td>
<td>1049.9 </td>
<td>1531.6 </td>
<td>925.8 </td>
<td>1095.9 </td>
<td>1644.5 </td>
<td>1801.0 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.385 </td>
<td>0.316 </td>
<td>0.333 </td>
<td>0.319 </td>
<td>0.224 </td>
<td>0.218 </td>
<td>0.397 </td>
<td>0.328 </td>
<td>0.256 </td>
<td>0.335 </td>
<td>0.256 </td>
</tr>
<tr>
<td rowspan="6">
<strong>SOIL</strong> </td>
<td rowspan="3">OTUX </td>
<td>Correct assignments </td>
<td>
<strong>5109.9</strong> </td>
<td>
<strong>5086.0</strong> </td>
<td>
<strong>5540.1</strong> </td>
<td>
<strong>5431.8</strong> </td>
<td>
<strong>5948.4</strong> </td>
<td>
<strong>5800.0</strong> </td>
<td>
<strong>4155.5</strong> </td>
<td>
<strong>5257.0</strong> </td>
<td>
<strong>5613.2</strong> </td>
<td>
<strong>5222.1</strong> </td>
<td>
<strong>5814.4</strong> </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>46.5 </td>
<td>45.4 </td>
<td>54.6 </td>
<td>225.5 </td>
<td>69.0 </td>
<td>51.4 </td>
<td>68.3 </td>
<td>71.5 </td>
<td>47.7 </td>
<td>45.5 </td>
<td>85.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.145 </td>
<td>0.114 </td>
<td>0.078 </td>
<td>0.123 </td>
<td>0.127 </td>
<td>0.153 </td>
<td>0.225 </td>
<td>0.133 </td>
<td>0.120 </td>
<td>0.162 </td>
<td>0.125 </td>
</tr>
<tr>
<td rowspan="3">CA </td>
<td>Correct assignments </td>
<td>4089.1 </td>
<td>4680.5 </td>
<td>4703.2 </td>
<td>1478.2 </td>
<td>5176.7 </td>
<td>5637.2 </td>
<td>2287.0 </td>
<td>3806.7 </td>
<td>5357.6 </td>
<td>3714.7 </td>
<td>4388.2 </td>
</tr>
<tr>
<td>Wrong assignments </td>
<td>690.0 </td>
<td>248.5 </td>
<td>191.5 </td>
<td>444.4 </td>
<td>301.4 </td>
<td>129.3 </td>
<td>685.9 </td>
<td>806.1 </td>
<td>279.4 </td>
<td>833.5 </td>
<td>390.7 </td>
</tr>
<tr>
<td>W-Unifrac<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> from GS </td>
<td>0.187 </td>
<td>0.158 </td>
<td>0.154 </td>
<td>0.349 </td>
<td>0.104 </td>
<td>0.084 </td>
<td>0.334 </td>
<td>0.184 </td>
<td>0.089 </td>
<td>0.193 </td>
<td>0.156 </td>
</tr>
</tbody>
</table></div>
<div class="table-wrap-foot">
<div content-id="tblfn1"><div class="fn"><p>Performance of both the approaches have been evaluated with multiple simulated microbiome datasets mimicking different types of environmental samples, (viz., infant gut, skin, sea and soil), and amplicon sequencing experiments targeting different V-regions of the bacterial 16S rRNA gene. Cases wherein the number of correct assignments by OTUX are significantly higher (<em>t</em>-test; <em>P</em> &lt; 0.001) have been highlighted in bold.</p></div></div>
<div content-id="tblfn2"><div class="fn">
<div class="label">a</div>
<p>CA and OTUX approaches have been evaluated considering results obtained with full-length 16S rRNA genes to be ‘correct’ or the ‘GS’. Only number of average correct assignments and wrong assignments has been depicted. The remaining reads (out of 10,000) could not be classified at OTU level.</p>
</div></div>
<div content-id="tblfn3"><div class="fn">
<div class="label">b</div>
<p>Weighted UNIFRAC distance of taxonomic abundance profile from actual taxonomic diversity (GS: gold-standard).</p>
</div></div>
</div>
<div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/134009589" target="_blank">Open in new tab</a></div>
</div></div>
<p>As mentioned earlier, one of the major reasons behind incorrect taxonomic assignment of short-read sequences to a different taxonomic lineage pertain to un-even evolutionary rate along the length of the 16S rRNA gene. The ‘mapping matrices’ introduced in the OTUX approach (see Building MAPMAT section) probably address this issue to some extent. Although in case of CA, finding high level of similarity of a very short stretch of sequence against a Greengenes (GG) OTU might result in a direct attribution of the short sequence to the lineage of the given GG OTU, the OTUX approach acknowledges that such a short semi-identical stretch could have alternate origins. The ‘mapping matrices’ presented in this work captures the propensities of association of such semi-identical short stretches of sequences (which have been clustered to form OTUX OTUs), to multiple GG OTUs/taxonomic lineages. It is based on these propensities that the final taxonomic abundance tables (according to GG lineages) are generated in the OTUX approach. Although this method (of propensity-based assignments) may also lead to incorrect assignments in some cases, overall the validation results indicate better performance of the OTUX approach compared with the direct assignments obtained with CA.</p>
<p>To evaluate whether the relatively better performance of OTUX spanned all the taxonomic lineages, or were limited to certain specific clades, the proportions of correct and incorrect assignments by OTUX and CA, cumulated at genus and family levels were analysed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material S4</a></span>). Neither OTUX nor CA could provide unbiased performance for all the taxonomic lineages while using different V-regions/stretches of V-regions. However, it was observed that in most cases the proportions of correct assignments by OTUX (averaged over all families or genera) were higher compared with CA. Further, the standard deviations in proportions of correct assignments across families/genera were also observed to be mostly equivalent or slightly lower for OTUX, reiterating the utility of the newly proposed taxonomic classification approach.</p>                    <h3 scrollto-destination="134009592" id="134009592" class="section-title">3.2. OTU clusters identified by OTUX are similar to those obtained using <em>de novo</em> OTU-picking approach</h3>
<p>Given the observed improvements in reference-based OTU-picking and taxonomic classification achieved using OTUX, it was imperative to check how close are these results to those obtained using any sequence similarity based <em>de novo</em> OTU clustering method. <a reveal-id="dsy045-F2" data-open="dsy045-F2" class="link link-reveal link-table xref-fig">Figure 2</a> depicts a comparison of OTU picking results obtained using CROP (CR), a popular <em>de novo</em> OTU clustering method, with those obtained using OTUX and CA. Since the three compared methods do not generate any common identifier(s) for the different OTU clusters obtained, ‘Jaccard similarity’ was computed to check the similarities between clusters generated by different methods (see Materials and methods). Similar clusters would share a greater proportion of query sequences, thereby resulting in a higher Jaccard similarity. Although comparing between OTUX and CR clusters, 100 largest clusters (in terms of number of query sequences in each cluster) generated by both the methods were selected, and pairwise Jaccard similarities were computed. For each of the 100 OTUX clusters, most similar CR clusters were identified based on Jaccard similarity. Similarly, for each of the 100 CR clusters, most similar OTUX clusters were also identified using reciprocal searches. Average Jaccard similarity values corresponding to these 2 × 100 closest pairs of clusters were considered as a measure of overall similarity in OTU clustering obtained using the two methods. Further, the number of reciprocal best hits between OTUX and CR clusters was also used as an indicator to evaluate the similarity between clustering results. Similarity between OTU clustering by CA and CR was also evaluated in a similar manner. As evident from <a reveal-id="dsy045-F2" data-open="dsy045-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, the results obtained using OTUX are relatively more similar to CR results when compared with results obtained with CA. The results of OTUX and CR are observed to be around 1.2–1.7 times more similar to each other in terms of average Jaccard similarity as compared with the similarity between CA and CR approaches. Even in terms of number of reciprocal best hits identified between the sets of clusters, OTUX and CR are more closely associated. Similar results were observed on extending the comparison for the top 300 and 500 (most populous) clusters generated for the simulated gut microbiome dataset with de novo clustering (CROP) thresholds set at 99, 97 and 95% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). These results assume further significance when viewed in context of execution time. It is expected that both CA and OTUX, being reference-based OTU-picking approaches, would have quicker execution time when compared with <em>de novo</em> OTU clustering approaches.<a reveal-id="dsy045-B10" data-open="dsy045-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Comparison of execution times between OTUX and CA revealed an overall equivalent performance for both approaches, with minor variations depending on the targeted V-region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3A</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">B</a></span>). A reasonable execution time coupled to a clustering solution closer to a <em>de novo</em> approach, as well as simultaneously improving on accuracy of taxonomic classification, certainly provides OTUX approach an edge over the conventional reference-based OTU picking. </p>                        <a id="134009594" scrollto-destination="134009594"></a>
<div id="dsy045-F2" data-id="dsy045-F2" class="fig fig-section">
<div class="label fig-label">Figure 2.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/m_dsy045f2.png?Expires=1576616109&amp;Signature=vWpNPqsJ1fOqJh~GtZC9mL74rUmRomH7zMgLZFoYDGDoZRN5oUD9FrSJI~bCkVfiiS~kXG40fme4sWd~-jQfFke~cTw-sj7F0BsmLr2SCDe2DFXgcKRl0UfFSfDQaHQsYHkK7uM3w5q~~aFqLTctBCv~0Brk6KnvhT2QRzelkjw9i-ddL71pDiCmSCTPGSpgyMyMgYfbkN77jxEpITneD6ugmrXJXLjkyngTZiCIvMjKZCd-ICH088rkvxzTd60EGCB3NEHyyR0p3fPkEW~Dw20yg2EtsGQLRN5NS8LZgBrL3COr6b3jWkE8qw-2fSFAzi8ujAZpXQyS0OvjAV0KXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Similarity of OTU clusters obtained using OTUX and CA when compared with de novo OTU clusters obtained using CROP (CR) for different simulated microbiome datasets pertaining to (A) GUT, (B) SKIN, (C) SEA and (D) SOIL. The bars indicate average Jaccard similarity (in terms of query sequence reads) between 100 largest clusters identified by the compared methods (see Materials and methods). The number mentioned above each bar represents the percentage of reciprocal best hits identified between the 100 largest clusters identified by compared methods (see Materials and methods)." path-from-xml="dsy045f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009594/dsy045f2.tif" path-from-xml="dsy045f2.tif" target="_blank">Open in new tab</a><a section="134009594" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045f2.png?Expires=1576616109&amp;Signature=3tM~HcwOshIaWdA1kYBHp5RmGUknD47XV1zmiXT8p7IdCcQwxD4oW7pnS8RWStOSVhSCbP0KMrSr77ddy1~BE1CiyTKeuPY65aWpvG5x07m2DiXROJVf8DYoBq1s4gLEhXc8Khd2C-huxkXmtBEZ3dSTp~EgeQcrxyiLLPREUSEEaoLVZPhjIE3a4zOaWKpa1RcxYMG8M6kChT-S4Iap69m-uebTdN-~oGI1FuBUTRE9pThyDowC-qONcKRqEGfZY4HqS-NDxwMWtM87JBu-Nejg7r9Zxfxil~v8DEpSCjiCmEFVduEiJR8P6NgZhqb34LZ1S0hzEgIKDAqYF2vb9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009594&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy045f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Similarity of OTU clusters obtained using OTUX and CA when compared with <em>de novo</em> OTU clusters obtained using CROP (CR) for different simulated microbiome datasets pertaining to (A) GUT, (B) SKIN, (C) SEA and (D) SOIL. The bars indicate average Jaccard similarity (in terms of query sequence reads) between 100 largest clusters identified by the compared methods (see Materials and methods). The number mentioned above each bar represents the percentage of reciprocal best hits identified between the 100 largest clusters identified by compared methods (see Materials and methods).</p></div>
</div>
<div content-id="dsy045-F2" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 2.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/m_dsy045f2.png?Expires=1576616109&amp;Signature=vWpNPqsJ1fOqJh~GtZC9mL74rUmRomH7zMgLZFoYDGDoZRN5oUD9FrSJI~bCkVfiiS~kXG40fme4sWd~-jQfFke~cTw-sj7F0BsmLr2SCDe2DFXgcKRl0UfFSfDQaHQsYHkK7uM3w5q~~aFqLTctBCv~0Brk6KnvhT2QRzelkjw9i-ddL71pDiCmSCTPGSpgyMyMgYfbkN77jxEpITneD6ugmrXJXLjkyngTZiCIvMjKZCd-ICH088rkvxzTd60EGCB3NEHyyR0p3fPkEW~Dw20yg2EtsGQLRN5NS8LZgBrL3COr6b3jWkE8qw-2fSFAzi8ujAZpXQyS0OvjAV0KXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Similarity of OTU clusters obtained using OTUX and CA when compared with de novo OTU clusters obtained using CROP (CR) for different simulated microbiome datasets pertaining to (A) GUT, (B) SKIN, (C) SEA and (D) SOIL. The bars indicate average Jaccard similarity (in terms of query sequence reads) between 100 largest clusters identified by the compared methods (see Materials and methods). The number mentioned above each bar represents the percentage of reciprocal best hits identified between the 100 largest clusters identified by compared methods (see Materials and methods)." path-from-xml="dsy045f2.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009594/dsy045f2.tif" path-from-xml="dsy045f2.tif" target="_blank">Open in new tab</a><a section="134009594" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045f2.png?Expires=1576616109&amp;Signature=3tM~HcwOshIaWdA1kYBHp5RmGUknD47XV1zmiXT8p7IdCcQwxD4oW7pnS8RWStOSVhSCbP0KMrSr77ddy1~BE1CiyTKeuPY65aWpvG5x07m2DiXROJVf8DYoBq1s4gLEhXc8Khd2C-huxkXmtBEZ3dSTp~EgeQcrxyiLLPREUSEEaoLVZPhjIE3a4zOaWKpa1RcxYMG8M6kChT-S4Iap69m-uebTdN-~oGI1FuBUTRE9pThyDowC-qONcKRqEGfZY4HqS-NDxwMWtM87JBu-Nejg7r9Zxfxil~v8DEpSCjiCmEFVduEiJR8P6NgZhqb34LZ1S0hzEgIKDAqYF2vb9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009594&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy045f2.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Similarity of OTU clusters obtained using OTUX and CA when compared with <em>de novo</em> OTU clusters obtained using CROP (CR) for different simulated microbiome datasets pertaining to (A) GUT, (B) SKIN, (C) SEA and (D) SOIL. The bars indicate average Jaccard similarity (in terms of query sequence reads) between 100 largest clusters identified by the compared methods (see Materials and methods). The number mentioned above each bar represents the percentage of reciprocal best hits identified between the 100 largest clusters identified by compared methods (see Materials and methods).</p></div>
</div>                    <h3 scrollto-destination="134009595" id="134009595" class="section-title">3.3. OTUX approach results obtained using different V-regions are coherent and better descriptors of taxonomic diversity</h3>
<p>Despite the promising results mentioned above, utilizing OTUX for a cross-study comparison of microbiome datasets (targeting different V-regions) only finds relevance if OTUX results obtained with different V-regions are coherent. <a reveal-id="dsy045-F3" data-open="dsy045-F3" class="link link-reveal link-table xref-fig">Figure 3</a> depicts four dendograms, wherein the taxonomic profiles corresponding to the simulated microbiome datasets obtained with OTUX, CA and CR, have been clustered based on pair wise weighted UniFrac distances (UPGMA method). Nodes in the tree correspond to the results obtained when different V-regions were targeted. Although OTUs identified using OTUX approach were mapped back to be represented in terms of Greengenes IDs and taxonomy, the cluster representatives of each of the OTUs generated using CR were also annotated in terms of Greengenes taxonomy to enable comparison with CA approach (see Materials and methods). The ‘GS’ result was used as an ‘outgroup’ during clustering. The clustering patterns in <a reveal-id="dsy045-F3" data-open="dsy045-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> indicate that results obtained using the same method (either of OTUX, CA or CR) group together irrespective of the targeted V-region. Although this observation assures of a certain level of coherence between results obtained using any given method, the proximity of the OTUX results cluster to the GS result is worth noting. Not only are the results obtained using OTUX approach coherent, but they are also consistently better representatives of the taxonomic diversity of the sampled environment, irrespective of the targeted V-region. In addition to the comparison based on weighted UniFrac distances, different diversity measures, like Shannon diversity, Simpson index (evenness), Chao1 index (species richness) and number of observed OTUs pertaining to the simulated gut microbiome, were computed based on the OTUs identified using CA, CR and OTUX approaches (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). These results were also compared in context of the diversity values corresponding to the GS taxonomic profiles. For most of the regions, diversity values obtained with OTUX were closer to the GS diversity values, when compared with CA and CROP methods. </p>                        <a id="134009597" scrollto-destination="134009597"></a>
<div id="dsy045-F3" data-id="dsy045-F3" class="fig fig-section">
<div class="label fig-label">Figure 3.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/m_dsy045f3.png?Expires=1576616109&amp;Signature=O1gvW5iVh-q5Ghs2TEU0ieJOUY1SQbbRO7Q~Y1pJGKw1JX6a0yU5AkT5G5T3GLY8HtNUEkgbEStcmmWifSFDWHNtZvEmSPjMgZTIqjYKO6G3YGjOnBIN2Nh0TaIoIRmAoQFsNTBbHbPv2gKLgpUJIT-VIt-x7uqq2pV~aClgclQGencqu23QAX9FitxJRVnFTgqtFQtQkbeoIWGmNxL1sjPPmoMbXeyawzVMfkB5KyCrg69rU1NRdYbMejVTe9FXBx~wY8sckxZ7Mn9w1GNQIScoALZIxvoOox2g1jYIIKAzzUgZ6iZxxSREGMico-Y~UIq4jPgUK5ZNqq6iCv66FA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dendograms depicting similarity between taxonomic profiles obtained with different OTU classification methods (viz. GS, CA, CR and OTUX) while using different V-regions. The trees have been constructed using the UPGMA method while using weighted UniFrac as the distance measure and considering the taxonomic profile generated with GS as an outgroup. Dendograms corresponding to simulated microbiomes pertaining to the environments (A) GUT, (B) SKIN, (C) SEA and (D) SOIL, have been plotted." path-from-xml="dsy045f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009597/dsy045f3.tif" path-from-xml="dsy045f3.tif" target="_blank">Open in new tab</a><a section="134009597" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045f3.png?Expires=1576616109&amp;Signature=Wr-ip~3L-66XQGskrZmDDTg1-YiP0EydN-LItydJQUJE8L-5HZzjssnTX8NNS3Qh6lvO7TRsU0Z8JvZZireCcJ7zxQXnWVQdV2fG76cMDLS4YGQEyvb9oSOuB06ZIrRFyhNge7I0hw9wcZsDKefVb9OZvMOuOgFAPmZ66WupDSXfUbx2pv9MLszN5HrZFDPuRhyTQzuuijG3jLN73dPf1tuRhZ59MsVgxB7~cydv3Dq0Z~LFplxKPC2RDfhII26tiISy0hZwjyqtsmnbev9VCoABFzfnqxTbzmeXdiquGC~bG0lgJpq2iFV3-gxDDImAU6lwA79WUqhObRkkZs4Ifw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009597&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy045f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Dendograms depicting similarity between taxonomic profiles obtained with different OTU classification methods (viz. GS, CA, CR and OTUX) while using different V-regions. The trees have been constructed using the UPGMA method while using weighted UniFrac as the distance measure and considering the taxonomic profile generated with GS as an outgroup. Dendograms corresponding to simulated microbiomes pertaining to the environments (A) GUT, (B) SKIN, (C) SEA and (D) SOIL, have been plotted.</p></div>
</div>
<div content-id="dsy045-F3" class="fig fig-modal reveal-modal">
<div class="label fig-label">Figure 3.</div>
<div class="graphic-wrap">
<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/m_dsy045f3.png?Expires=1576616109&amp;Signature=O1gvW5iVh-q5Ghs2TEU0ieJOUY1SQbbRO7Q~Y1pJGKw1JX6a0yU5AkT5G5T3GLY8HtNUEkgbEStcmmWifSFDWHNtZvEmSPjMgZTIqjYKO6G3YGjOnBIN2Nh0TaIoIRmAoQFsNTBbHbPv2gKLgpUJIT-VIt-x7uqq2pV~aClgclQGencqu23QAX9FitxJRVnFTgqtFQtQkbeoIWGmNxL1sjPPmoMbXeyawzVMfkB5KyCrg69rU1NRdYbMejVTe9FXBx~wY8sckxZ7Mn9w1GNQIScoALZIxvoOox2g1jYIIKAzzUgZ6iZxxSREGMico-Y~UIq4jPgUK5ZNqq6iCv66FA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dendograms depicting similarity between taxonomic profiles obtained with different OTU classification methods (viz. GS, CA, CR and OTUX) while using different V-regions. The trees have been constructed using the UPGMA method while using weighted UniFrac as the distance measure and considering the taxonomic profile generated with GS as an outgroup. Dendograms corresponding to simulated microbiomes pertaining to the environments (A) GUT, (B) SKIN, (C) SEA and (D) SOIL, have been plotted." path-from-xml="dsy045f3.tif"><div class="fig-orig original-slide">
<a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/134009597/dsy045f3.tif" path-from-xml="dsy045f3.tif" target="_blank">Open in new tab</a><a section="134009597" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045f3.png?Expires=1576616109&amp;Signature=Wr-ip~3L-66XQGskrZmDDTg1-YiP0EydN-LItydJQUJE8L-5HZzjssnTX8NNS3Qh6lvO7TRsU0Z8JvZZireCcJ7zxQXnWVQdV2fG76cMDLS4YGQEyvb9oSOuB06ZIrRFyhNge7I0hw9wcZsDKefVb9OZvMOuOgFAPmZ66WupDSXfUbx2pv9MLszN5HrZFDPuRhyTQzuuijG3jLN73dPf1tuRhZ59MsVgxB7~cydv3Dq0Z~LFplxKPC2RDfhII26tiISy0hZwjyqtsmnbev9VCoABFzfnqxTbzmeXdiquGC~bG0lgJpq2iFV3-gxDDImAU6lwA79WUqhObRkkZs4Ifw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=134009597&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5275" class="download-slide" path-from-xml="dsy045f3.tif" target="_blank">Download slide</a>
</div>
</div>
<div class="caption fig-caption"><p>Dendograms depicting similarity between taxonomic profiles obtained with different OTU classification methods (viz. GS, CA, CR and OTUX) while using different V-regions. The trees have been constructed using the UPGMA method while using weighted UniFrac as the distance measure and considering the taxonomic profile generated with GS as an outgroup. Dendograms corresponding to simulated microbiomes pertaining to the environments (A) GUT, (B) SKIN, (C) SEA and (D) SOIL, have been plotted.</p></div>
</div>
<p>It is however important to note here that certain other biologically important differences and methodological biases (in addition to those resulting from the choice of V-regions) may be inherent to sequenced data from different studies and protocols. The possible effects of these potential biases, on the robustness of taxonomic assignments obtained with CA and OTUX was tested using real microbiome sampling data from an earlier study.<a reveal-id="dsy045-B35" data-open="dsy045-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Microbiomes pertaining to six different environments had been sampled in the earlier study, wherein the DNA sequencing for each of the samples were done on both Illumina (MiSeq <a reveal-id="dsy045-B2" data-open="dsy045-B2" class="link link-ref link-reveal xref-bibr">2</a> × 250 bp paired end protocol) and Roche 454 (GS FLX using Titanium) platforms. Taxonomic assignments for both the Illumina as well as Roche generated datasets from this study were performed with OTUX and CA, followed by a comparison of the similarity of the taxonomic profiles generated based on weighted Unifrac distances. Taxonomic profiles generated from Illumina and Roche datasets using OTUX method were observed to be more similar to each other than those generated by CA, in case of four (out of six) of the sampled environments (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576616109&amp;Signature=hWxM34XyKcdTo52M--QGnrHFE0bsIyNJ8FgwYsQEheqcV64VbKmvAR9apvJhnX5PfpZ94OTN~-HcY3zh1Gh3gaeQ9vG6YtjE-IPE-JcEUdunATJSctmkND~x9ZGRFR3evcvh0F~qf3UCq9RMSwk48n~i9RHDXrVtHLQxAZUbCRatULQovzqP4D9pA3c3hojZcNW9lQDOC88PRmU1EGhe7wfmU3iZPmweGuxvnJuB6RXmAlBcozBJTw2Cxu8hJgBavLhT64JG7eASjuGdOLKWPgW6Vgxj-TUlY~KbuyKcL01yvoMdqDPR2wXMwRJPUIcGmMq53eNA-CdaF2YJIQWYww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). It may be noted here that the study specific biases pertaining to sequencing chemistry, sample storage/handling and other methodological differences, cannot be expected to be completely overcome by using <em>in silico</em> taxonomic classification methods such as OTUX or other CAs. However, given that the results indicate at OTUX’s ability to arrive at more coherent taxonomic profiles when compared with the CA, OTUX is expected to have better utility while performing cross-study taxonomic comparisons.</p>                    <h2 scrollto-destination="134009599" id="134009599" class="section-title">4. Conclusion</h2>
<p>In summary, adopting OTUX approach of OTU picking can enable easy and accurate cross-study comparison of taxonomic profiles, even when targeted V-regions might vary across different studies. Although performing such comparisons at a large scale (with several microbiome datasets) using <em>de novo</em> OTU clustering approach might be computationally prohibitive, using CA like approaches might connote a compromise in accuracy of the comparison. In contrast, such comparisons using OTUX approach not only provides more accurate taxonomic annotations (even at the OTU level), but also attains the same with lesser computational expenses.</p>                    <h2 scrollto-destination="134009601" id="134009601" class="section-title">Availability</h2>
<p>Freely available for academic use on the web at <a class="link link-uri" href="https://web.rniapps.net/otux/" target="_blank">https://web.rniapps.net/otux/</a></p>                    <h2 scrollto-destination="134009603" id="134009603" class="backacknowledgements-title" data-legacy-id="ACK1">Acknowledgements</h2>
<p>All three authors are employees of Tata Consultancy Services Ltd. (TCSL) and would like to acknowledge TCSL for its support in form of salaries.</p>                    <h3 scrollto-destination="134009605" id="134009605" class="section-title">Conflict of interest</h3>
<p>None declared.</p>                    <h2 scrollto-destination="134009607" id="134009607" class="backreferences-title" data-legacy-id="REF1">References</h2>
<div class="ref-list">
<div content-id="dsy045-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B1" class="ref-content ">
<div class="label">1</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Caporaso</div> <div class="given-names">J. G.</div>
</div>, <div class="name string-name">
<div class="surname">Bittinger</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Bushman</div> <div class="given-names">F. D.</div>
</div>, <div class="name string-name">
<div class="surname">DeSantis</div> <div class="given-names">T. Z.</div>
</div>, <div class="name string-name">
<div class="surname">Andersen</div> <div class="given-names">G. L.</div>
</div>, <div class="name string-name">
<div class="surname">Knight</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">PyNAST: a flexible tool for aligning sequences to a template alignment</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">266</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PyNAST%3A%20a%20flexible%20tool%20for%20aligning%20sequences%20to%20a%20template%20alignment&amp;author=J.%20G.%20Caporaso&amp;author=K.%20Bittinger&amp;author=F.%20D.%20Bushman&amp;author=T.%20Z.%20DeSantis&amp;author=G.%20L.%20Andersen&amp;author=R.%20Knight&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=266-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp636" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp636" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp636"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19914921" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PyNAST%3A%20a%20flexible%20tool%20for%20aligning%20sequences%20to%20a%20template%20alignment&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B2" class="ref-content ">
<div class="label">2</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Edgar</div> <div class="given-names">R.C.</div>
</div></div> <div class="year">2010</div>, 
<div class="article-title">Search and clustering orders of magnitude faster than BLAST</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">2460</div>–<div class="lpage">1</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;author=R.C.%20Edgar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2460-1" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq461" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq461" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq461"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B3" class="ref-content ">
<div class="label">3</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ghosh</div> <div class="given-names">T. S.</div>
</div>, <div class="name string-name">
<div class="surname">Gajjalla</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Mohammed</div> <div class="given-names">M. H.</div>
</div>, <div class="name string-name">
<div class="surname">Mande</div> <div class="given-names">S. S.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">C16S - a Hidden Markov Model based algorithm for taxonomic classification of 16S rRNA gene sequences</div>, <div class="source">Genomics</div> , <div class="volume">99</div>, <div class="fpage">195</div>–<div class="lpage">201</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=C16S%20-%20a%20Hidden%20Markov%20Model%20based%20algorithm%20for%20taxonomic%20classification%20of%2016S%20rRNA%20gene%20sequences&amp;author=T.%20S.%20Ghosh&amp;author=P.%20Gajjalla&amp;author=M.%20H.%20Mohammed&amp;author=S.%20S.%20Mande&amp;publication_year=2012&amp;journal=Genomics&amp;volume=99&amp;pages=195-201" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ygeno.2012.01.008" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2012.01.008" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ygeno.2012.01.008"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22326741" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:C16S%20-%20a%20Hidden%20Markov%20Model%20based%20algorithm%20for%20taxonomic%20classification%20of%2016S%20rRNA%20gene%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B4" class="ref-content ">
<div class="label">4</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Wang</div> <div class="given-names">Q.</div>
</div>, <div class="name string-name">
<div class="surname">Garrity</div> <div class="given-names">G. M.</div>
</div>, <div class="name string-name">
<div class="surname">Tiedje</div> <div class="given-names">J. M.</div>
</div>, <div class="name string-name">
<div class="surname">Cole</div> <div class="given-names">J. R.</div>
</div>
</div> <div class="year">2007</div>, 
<div class="article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">73</div>, <div class="fpage">5261</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Naive%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;author=Q.%20Wang&amp;author=G.%20M.%20Garrity&amp;author=J.%20M.%20Tiedje&amp;author=J.%20R.%20Cole&amp;publication_year=2007&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=73&amp;pages=5261-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.00062-07" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00062-07" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.00062-07"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Naive%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B5" class="ref-content ">
<div class="label">5</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Baron</div> <div class="given-names">J. L.</div>
</div>, <div class="name string-name">
<div class="surname">Vikram</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Duda</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Stout</div> <div class="given-names">J. E.</div>
</div>, <div class="name string-name">
<div class="surname">Bibby</div> <div class="given-names">K.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Shift in the microbial ecology of a hospital hot water system following the introduction of an on-site monochloramine disinfection system</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e102679</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Shift%20in%20the%20microbial%20ecology%20of%20a%20hospital%20hot%20water%20system%20following%20the%20introduction%20of%20an%20on-site%20monochloramine%20disinfection%20system&amp;author=J.%20L.%20Baron&amp;author=A.%20Vikram&amp;author=S.%20Duda&amp;author=J.%20E.%20Stout&amp;author=K.%20Bibby&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e102679" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0102679" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0102679" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0102679"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25033448" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Shift%20in%20the%20microbial%20ecology%20of%20a%20hospital%20hot%20water%20system%20following%20the%20introduction%20of%20an%20on-site%20monochloramine%20disinfection%20system&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B6" class="ref-content ">
<div class="label">6</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Harris</div> <div class="given-names">J. K.</div>
</div>, <div class="name string-name">
<div class="surname">Caporaso</div> <div class="given-names">J. G.</div>
</div>, <div class="name string-name">
<div class="surname">Walker</div> <div class="given-names">J. J.</div>
</div>
</div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Phylogenetic stratigraphy in the Guerrero Negro hypersaline microbial mat</div>, <div class="source">Isme J</div> ., <div class="volume">7</div>, <div class="fpage">50</div>–<div class="lpage">60</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20stratigraphy%20in%20the%20Guerrero%20Negro%20hypersaline%20microbial%20mat&amp;author=J.%20K.%20Harris&amp;author=J.%20G.%20Caporaso&amp;author=J.%20J.%20Walker&amp;publication_year=2013&amp;journal=Isme%20J&amp;volume=7&amp;pages=50-60" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ismej.2012.79" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2012.79" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fismej.2012.79"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22832344" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenetic%20stratigraphy%20in%20the%20Guerrero%20Negro%20hypersaline%20microbial%20mat&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B7" class="ref-content ">
<div class="label">7</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Jervis-Bardy</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Leong</div> <div class="given-names">L. E. X.</div>
</div>, <div class="name string-name">
<div class="surname">Marri</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Deriving accurate microbiota profiles from human samples with low bacterial content through post-sequencing processing of Illumina MiSeq data</div>, <div class="source">Microbiome</div> , <div class="volume">3</div>, <div class="fpage">19</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Deriving%20accurate%20microbiota%20profiles%20from%20human%20samples%20with%20low%20bacterial%20content%20through%20post-sequencing%20processing%20of%20Illumina%20MiSeq%20data&amp;author=J.%20Jervis-Bardy&amp;author=L.%20E.%20X.%20Leong&amp;author=S.%20Marri&amp;publication_year=2015&amp;journal=Microbiome&amp;volume=3&amp;pages=19" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s40168-015-0083-8" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs40168-015-0083-8" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs40168-015-0083-8"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25969736" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Deriving%20accurate%20microbiota%20profiles%20from%20human%20samples%20with%20low%20bacterial%20content%20through%20post-sequencing%20processing%20of%20Illumina%20MiSeq%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B8" class="ref-content ">
<div class="label">8</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Caporaso</div> <div class="given-names">J. G.</div>
</div>, <div class="name string-name">
<div class="surname">Paszkiewicz</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Field</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Knight</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Gilbert</div> <div class="given-names">J. A.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">The Western English Channel contains a persistent microbial seed bank</div>, <div class="source">Isme J</div> ., <div class="volume">6</div>, <div class="fpage">1089</div>–<div class="lpage">93</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Western%20English%20Channel%20contains%20a%20persistent%20microbial%20seed%20bank&amp;author=J.%20G.%20Caporaso&amp;author=K.%20Paszkiewicz&amp;author=D.%20Field&amp;author=R.%20Knight&amp;author=J.%20A.%20Gilbert&amp;publication_year=2012&amp;journal=Isme%20J&amp;volume=6&amp;pages=1089-93" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ismej.2011.162" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2011.162" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fismej.2011.162"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22071345" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Western%20English%20Channel%20contains%20a%20persistent%20microbial%20seed%20bank&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B9" class="ref-content ">
<div class="label">9</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Pylro</div> <div class="given-names">V. S.</div>
</div>, <div class="name string-name">
<div class="surname">Roesch</div> <div class="given-names">L. F. W.</div>
</div>, <div class="name string-name">
<div class="surname">Morais</div> <div class="given-names">D. K.</div>
</div>, <div class="name string-name">
<div class="surname">Clark</div> <div class="given-names">I. M.</div>
</div>, <div class="name string-name">
<div class="surname">Hirsch</div> <div class="given-names">P. R.</div>
</div>, <div class="name string-name">
<div class="surname">Tótola</div> <div class="given-names">M. R.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Data analysis for 16S microbial profiling from different benchtop sequencing platforms</div>, <div class="source">J. Microbiol Methods</div> , <div class="volume">107</div>, <div class="fpage">30</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Data%20analysis%20for%2016S%20microbial%20profiling%20from%20different%20benchtop%20sequencing%20platforms&amp;author=V.%20S.%20Pylro&amp;author=L.%20F.%20W.%20Roesch&amp;author=D.%20K.%20Morais&amp;author=I.%20M.%20Clark&amp;author=P.%20R.%20Hirsch&amp;author=M.%20R.%20T%C3%B3tola&amp;publication_year=2014&amp;journal=J.%20Microbiol%20Methods&amp;volume=107&amp;pages=30-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.mimet.2014.08.018" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2014.08.018" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mimet.2014.08.018"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25193439" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Data%20analysis%20for%2016S%20microbial%20profiling%20from%20different%20benchtop%20sequencing%20platforms&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B10" class="ref-content ">
<div class="label">10</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Rideout</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">He</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Navas-Molina</div> <div class="given-names">J. A.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Subsampled open-reference clustering creates consistent, comprehensive OTU definitions and scales to billions of sequences</div>, <div class="source">PeerJ</div> ., <div class="volume">2</div>, <div class="fpage">e545</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Subsampled%20open-reference%20clustering%20creates%20consistent%2C%20comprehensive%20OTU%20definitions%20and%20scales%20to%20billions%20of%20sequences&amp;author=J.%20R.%20Rideout&amp;author=Y.%20He&amp;author=J.%20A.%20Navas-Molina&amp;publication_year=2014&amp;journal=PeerJ&amp;volume=2&amp;pages=e545" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.7717/peerj.545" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7717%2Fpeerj.545" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.7717%2Fpeerj.545"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25177538" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Subsampled%20open-reference%20clustering%20creates%20consistent%2C%20comprehensive%20OTU%20definitions%20and%20scales%20to%20billions%20of%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B11" class="ref-content ">
<div class="label">11</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Dickie</div> <div class="given-names">I. A.</div>
</div></div> <div class="year">2010</div>, 
<div class="article-title">Insidious effects of sequencing errors on perceived diversity in molecular surveys</div>, <div class="source">New Phytol</div> ., <div class="volume">188</div>, <div class="fpage">916</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Insidious%20effects%20of%20sequencing%20errors%20on%20perceived%20diversity%20in%20molecular%20surveys&amp;author=I.%20A.%20Dickie&amp;publication_year=2010&amp;journal=New%20Phytol&amp;volume=188&amp;pages=916-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1469-8137.2010.03473.x" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03473.x" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1469-8137.2010.03473.x"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20854395" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Insidious%20effects%20of%20sequencing%20errors%20on%20perceived%20diversity%20in%20molecular%20surveys&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B12" class="ref-content ">
<div class="label">12</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Poretsky</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Rodriguez-R</div> <div class="given-names">L. M.</div>
</div>, <div class="name string-name">
<div class="surname">Luo</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Tsementzi</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Konstantinidis</div> <div class="given-names">K. T.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Strengths and limitations of 16S rRNA gene amplicon sequencing in revealing temporal microbial community dynamics</div>, <div class="source">PLoS One</div> , <div class="volume">9(4)</div>: e93827.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Strengths%20and%20limitations%20of%2016S%20rRNA%20gene%20amplicon%20sequencing%20in%20revealing%20temporal%20microbial%20community%20dynamics&amp;aulast=Poretsky&amp;date=2014&amp;volume=9(4)" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Strengths%20and%20limitations%20of%2016S%20rRNA%20gene%20amplicon%20sequencing%20in%20revealing%20temporal%20microbial%20community%20dynamics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B13" class="ref-content ">
<div class="label">13</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Quince</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Lanzen</div> <div class="given-names">A.</div>
</div>, <div class="name string-name">
<div class="surname">Davenport</div> <div class="given-names">R. J.</div>
</div>, <div class="name string-name">
<div class="surname">Turnbaugh</div> <div class="given-names">P. J.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Removing noise from pyrosequenced amplicons</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">38</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Removing%20noise%20from%20pyrosequenced%20amplicons&amp;author=C.%20Quince&amp;author=A.%20Lanzen&amp;author=R.%20J.%20Davenport&amp;author=P.%20J.%20Turnbaugh&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=38" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-12-38" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-38" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-12-38"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21276213" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Removing%20noise%20from%20pyrosequenced%20amplicons&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B14" class="ref-content ">
<div class="label">14</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Barriuso</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Valverde</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">Mellado</div> <div class="given-names">R. P.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Estimation of bacterial diversity using next generation sequencing of 16S rDNA: a comparison of different workflows</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">473</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20bacterial%20diversity%20using%20next%20generation%20sequencing%20of%2016S%20rDNA%3A%20a%20comparison%20of%20different%20workflows&amp;author=J.%20Barriuso&amp;author=J.%20R.%20Valverde&amp;author=R.%20P.%20Mellado&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=473" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-12-473" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-473" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-12-473"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22168258" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Estimation%20of%20bacterial%20diversity%20using%20next%20generation%20sequencing%20of%2016S%20rDNA%3A%20a%20comparison%20of%20different%20workflows&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B15" class="ref-content ">
<div class="label">15</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Bonder</div> <div class="given-names">M. J.</div>
</div>, <div class="name string-name">
<div class="surname">Abeln</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Zaura</div> <div class="given-names">E.</div>
</div>, <div class="name string-name">
<div class="surname">Brandt</div> <div class="given-names">B. W.</div>
</div>
</div> <div class="year">2012</div>, 
<div class="article-title">Comparing clustering and pre-processing in taxonomy analysis</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">2891</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparing%20clustering%20and%20pre-processing%20in%20taxonomy%20analysis&amp;author=M.%20J.%20Bonder&amp;author=S.%20Abeln&amp;author=E.%20Zaura&amp;author=B.%20W.%20Brandt&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2891-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bts552" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts552" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts552"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22962346" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparing%20clustering%20and%20pre-processing%20in%20taxonomy%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B16" class="ref-content ">
<div class="label">16</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">C. K.</div>
</div>, <div class="name string-name">
<div class="surname">Cheng</div> <div class="given-names">Y.</div>
</div>, <div class="name string-name">
<div class="surname">Zhang</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Zhao</div> <div class="given-names">H.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">A comparison of methods for clustering 16S rRNA sequences into OTUs</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>(8): e70837.<div class="citation-links">
<div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20comparison%20of%20methods%20for%20clustering%2016S%20rRNA%20sequences%20into%20OTUs&amp;aulast=Chen&amp;date=2013&amp;volume=8" href="#"></a></span></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20comparison%20of%20methods%20for%20clustering%2016S%20rRNA%20sequences%20into%20OTUs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B17" class="ref-content ">
<div class="label">17</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schloss</div> <div class="given-names">P. D.</div>
</div>, <div class="name string-name">
<div class="surname">Westcott</div> <div class="given-names">S. L.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Assessing and improving methods used in operational taxonomic unit-based approaches for 16S rRNA gene sequence analysis</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">77</div>, <div class="fpage">3219</div>–<div class="lpage">26</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assessing%20and%20improving%20methods%20used%20in%20operational%20taxonomic%20unit-based%20approaches%20for%2016S%20rRNA%20gene%20sequence%20analysis&amp;author=P.%20D.%20Schloss&amp;author=S.%20L.%20Westcott&amp;publication_year=2011&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=77&amp;pages=3219-26" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.02810-10" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02810-10" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.02810-10"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21421784" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assessing%20and%20improving%20methods%20used%20in%20operational%20taxonomic%20unit-based%20approaches%20for%2016S%20rRNA%20gene%20sequence%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B18" class="ref-content ">
<div class="label">18</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Franzén</div> <div class="given-names">O.</div>
</div>, <div class="name string-name">
<div class="surname">Hu</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Bao</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Itzkowitz</div> <div class="given-names">S. H.</div>
</div>, <div class="name string-name">
<div class="surname">Peter</div> <div class="given-names">I.</div>
</div>, <div class="name string-name">
<div class="surname">Bashir</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2015</div>, 
<div class="article-title">Improved OTU-picking using long-read 16S rRNA gene amplicon sequencing and generic hierarchical clustering</div>, <div class="source">Microbiome</div> , <div class="volume">3</div>:<div class="fpage">43</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improved%20OTU-picking%20using%20long-read%2016S%20rRNA%20gene%20amplicon%20sequencing%20and%20generic%20hierarchical%20clustering&amp;author=O.%20Franz%C3%A9n&amp;author=J.%20Hu&amp;author=X.%20Bao&amp;author=S.%20H.%20Itzkowitz&amp;author=I.%20Peter&amp;author=A.%20Bashir&amp;publication_year=2015&amp;journal=Microbiome&amp;volume=3&amp;pages=43" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s40168-015-0105-6" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs40168-015-0105-6" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs40168-015-0105-6"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26434730" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improved%20OTU-picking%20using%20long-read%2016S%20rRNA%20gene%20amplicon%20sequencing%20and%20generic%20hierarchical%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B19" class="ref-content ">
<div class="label">19</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Kim</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Morrison</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Yu</div> <div class="given-names">Z.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Evaluation of different partial 16S rRNA gene sequence regions for phylogenetic analysis of microbiomes</div>, <div class="source">J. Microbiol. Methods</div> , <div class="volume">84</div>, <div class="fpage">81</div>–<div class="lpage">7</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20different%20partial%2016S%20rRNA%20gene%20sequence%20regions%20for%20phylogenetic%20analysis%20of%20microbiomes&amp;author=M.%20Kim&amp;author=M.%20Morrison&amp;author=Z.%20Yu&amp;publication_year=2011&amp;journal=J.%20Microbiol.%20Methods&amp;volume=84&amp;pages=81-7" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.mimet.2010.10.020" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2010.10.020" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mimet.2010.10.020"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21047533" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluation%20of%20different%20partial%2016S%20rRNA%20gene%20sequence%20regions%20for%20phylogenetic%20analysis%20of%20microbiomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B20" class="ref-content ">
<div class="label">20</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Schloss</div> <div class="given-names">P. D.</div>
</div></div> <div class="year">2010</div>, 
<div class="article-title">The effects of alignment quality, distance calculation method, sequence filtering, and region on the analysis of 16S rRNA gene-based studies</div>, <div class="source">PLoS Comput. Biol</div> ., <div class="volume">6</div>, <div class="fpage">e1000844</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20effects%20of%20alignment%20quality%2C%20distance%20calculation%20method%2C%20sequence%20filtering%2C%20and%20region%20on%20the%20analysis%20of%2016S%20rRNA%20gene-based%20studies&amp;author=P.%20D.%20Schloss&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol&amp;volume=6&amp;pages=e1000844" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pcbi.1000844" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000844" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pcbi.1000844"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20628621" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20effects%20of%20alignment%20quality%2C%20distance%20calculation%20method%2C%20sequence%20filtering%2C%20and%20region%20on%20the%20analysis%20of%2016S%20rRNA%20gene-based%20studies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B21" class="ref-content ">
<div class="label">21</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">DeSantis</div> <div class="given-names">T. Z.</div>
</div>, <div class="name string-name">
<div class="surname">Hugenholtz</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Larsen</div> <div class="given-names">N.</div>
</div>
</div>, et al.  <div class="year">2006</div>, 
<div class="article-title">Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">72</div>, <div class="fpage">5069</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Greengenes%2C%20a%20chimera-checked%2016S%20rRNA%20gene%20database%20and%20workbench%20compatible%20with%20ARB&amp;author=T.%20Z.%20DeSantis&amp;author=P.%20Hugenholtz&amp;author=N.%20Larsen&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=72&amp;pages=5069-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.03006-05" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03006-05" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.03006-05"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16820507" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Greengenes%2C%20a%20chimera-checked%2016S%20rRNA%20gene%20database%20and%20workbench%20compatible%20with%20ARB&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B22" class="ref-content ">
<div class="label">22</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hartmann</div> <div class="given-names">M.</div>
</div>, <div class="name string-name">
<div class="surname">Howes</div> <div class="given-names">C. G.</div>
</div>, <div class="name string-name">
<div class="surname">Abarenkov</div> <div class="given-names">K.</div>
</div>, <div class="name string-name">
<div class="surname">Mohn</div> <div class="given-names">W. W.</div>
</div>, <div class="name string-name">
<div class="surname">Nilsson</div> <div class="given-names">R. H.</div>
</div>
</div> <div class="year">2010</div>, 
<div class="article-title">V-Xtractor: an open-source, high-throughput software tool to identify and extract hypervariable regions of small subunit (16S/18S) ribosomal RNA gene sequences</div>, <div class="source">J. Microbiol. Methods</div> , <div class="volume">83</div>, <div class="fpage">250</div>–<div class="lpage">3</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=V-Xtractor%3A%20an%20open-source%2C%20high-throughput%20software%20tool%20to%20identify%20and%20extract%20hypervariable%20regions%20of%20small%20subunit%20(16S%2F18S)%20ribosomal%20RNA%20gene%20sequences&amp;author=M.%20Hartmann&amp;author=C.%20G.%20Howes&amp;author=K.%20Abarenkov&amp;author=W.%20W.%20Mohn&amp;author=R.%20H.%20Nilsson&amp;publication_year=2010&amp;journal=J.%20Microbiol.%20Methods&amp;volume=83&amp;pages=250-3" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.mimet.2010.08.008" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2010.08.008" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mimet.2010.08.008"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20804791" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:V-Xtractor%3A%20an%20open-source%2C%20high-throughput%20software%20tool%20to%20identify%20and%20extract%20hypervariable%20regions%20of%20small%20subunit%20(16S%2F18S)%20ribosomal%20RNA%20gene%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B23" class="ref-content ">
<div class="label">23</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Li</div> <div class="given-names">W.</div>
</div>, <div class="name string-name">
<div class="surname">Godzik</div> <div class="given-names">A.</div>
</div>
</div> <div class="year">2006</div>, 
<div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">22</div>, <div class="fpage">1658</div>–<div class="lpage">9</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl158"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B24" class="ref-content ">
<div class="label">24</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ghosh</div> <div class="given-names">T. S.</div>
</div>, <div class="name string-name">
<div class="surname">Gupta</div> <div class="given-names">S. S.</div>
</div>, <div class="name string-name">
<div class="surname">Bhattacharya</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Gut microbiomes of Indian children of varying nutritional status</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e95547</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gut%20microbiomes%20of%20Indian%20children%20of%20varying%20nutritional%20status&amp;author=T.%20S.%20Ghosh&amp;author=S.%20S.%20Gupta&amp;author=T.%20Bhattacharya&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e95547" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0095547" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0095547" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0095547"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24763225" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gut%20microbiomes%20of%20Indian%20children%20of%20varying%20nutritional%20status&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B25" class="ref-content ">
<div class="label">25</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Ganju</div> <div class="given-names">P.</div>
</div>, <div class="name string-name">
<div class="surname">Nagpal</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Mohammed</div> <div class="given-names">M. H.</div>
</div>
</div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Microbial community profiling shows dysbiosis in the lesional skin of Vitiligo subjects</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">18761</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Microbial%20community%20profiling%20shows%20dysbiosis%20in%20the%20lesional%20skin%20of%20Vitiligo%20subjects&amp;author=P.%20Ganju&amp;author=S.%20Nagpal&amp;author=M.%20H.%20Mohammed&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=18761" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/srep18761" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep18761" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fsrep18761"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26758568" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Microbial%20community%20profiling%20shows%20dysbiosis%20in%20the%20lesional%20skin%20of%20Vitiligo%20subjects&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B26" class="ref-content ">
<div class="label">26</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Sunagawa</div> <div class="given-names">S.</div>
</div>, <div class="name string-name">
<div class="surname">Coelho</div> <div class="given-names">L. P.</div>
</div>, <div class="name string-name">
<div class="surname">Chaffron</div> <div class="given-names">S.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Ocean plankton. Structure and function of the global ocean microbiome</div>, <div class="source">Science</div> , <div class="volume">348</div>, <div class="fpage">1261359</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ocean%20plankton.%20Structure%20and%20function%20of%20the%20global%20ocean%20microbiome&amp;author=S.%20Sunagawa&amp;author=L.%20P.%20Coelho&amp;author=S.%20Chaffron&amp;publication_year=2015&amp;journal=Science&amp;volume=348&amp;pages=1261359" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1261359" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1261359" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1261359"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25999513" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ocean%20plankton.%20Structure%20and%20function%20of%20the%20global%20ocean%20microbiome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B27" class="ref-content ">
<div class="label">27</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Navarrete</div> <div class="given-names">A. A.</div>
</div>, <div class="name string-name">
<div class="surname">Tsai</div> <div class="given-names">S. M.</div>
</div>, <div class="name string-name">
<div class="surname">Mendes</div> <div class="given-names">L. W.</div>
</div>
</div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Soil microbiome responses to the short-term effects of Amazonian deforestation</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">24</div>, <div class="fpage">2433</div>–<div class="lpage">48</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Soil%20microbiome%20responses%20to%20the%20short-term%20effects%20of%20Amazonian%20deforestation&amp;author=A.%20A.%20Navarrete&amp;author=S.%20M.%20Tsai&amp;author=L.%20W.%20Mendes&amp;publication_year=2015&amp;journal=Mol.%20Ecol&amp;volume=24&amp;pages=2433-48" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/mec.13172" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.13172" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmec.13172"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25809788" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Soil%20microbiome%20responses%20to%20the%20short-term%20effects%20of%20Amazonian%20deforestation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B28" class="ref-content ">
<div class="label">28</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Cole</div> <div class="given-names">J. R.</div>
</div>, <div class="name string-name">
<div class="surname">Chai</div> <div class="given-names">B.</div>
</div>, <div class="name string-name">
<div class="surname">Farris</div> <div class="given-names">R. J.</div>
</div>
</div>, et al.  <div class="year">2005</div>, 
<div class="article-title">The Ribosomal Database Project (RDP-II): sequences and tools for high-throughput rRNA analysis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">D294</div>–<div class="lpage">6</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Ribosomal%20Database%20Project%20(RDP-II)%3A%20sequences%20and%20tools%20for%20high-throughput%20rRNA%20analysis&amp;author=J.%20R.%20Cole&amp;author=B.%20Chai&amp;author=R.%20J.%20Farris&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=D294-6" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gki038" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki038" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki038"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608200" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Ribosomal%20Database%20Project%20(RDP-II)%3A%20sequences%20and%20tools%20for%20high-throughput%20rRNA%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B29" class="ref-content ">
<div class="label">29</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Schloss</div> <div class="given-names">P. D.</div>
</div>, <div class="name string-name">
<div class="surname">Westcott</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Ryabin</div> <div class="given-names">T.</div>
</div>
</div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">75</div>, <div class="fpage">7537</div>–<div class="lpage">41</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Introducing%20mothur%3A%20open-source%2C%20platform-independent%2C%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;author=P.%20D.%20Schloss&amp;author=S.%20L.%20Westcott&amp;author=T.%20Ryabin&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=75&amp;pages=7537-41" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.01541-09" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01541-09" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.01541-09"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Introducing%20mothur%3A%20open-source%2C%20platform-independent%2C%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B30" class="ref-content ">
<div class="label">30</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Konstantinidis</div> <div class="given-names">K. T.</div>
</div>, <div class="name string-name">
<div class="surname">Tiedje</div> <div class="given-names">J. M.</div>
</div>
</div> <div class="year">2005</div>, 
<div class="article-title">Genomic insights that advance the species definition for prokaryotes</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">102</div>, <div class="fpage">2567</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomic%20insights%20that%20advance%20the%20species%20definition%20for%20prokaryotes&amp;author=K.%20T.%20Konstantinidis&amp;author=J.%20M.%20Tiedje&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=102&amp;pages=2567-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0409727102" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0409727102" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0409727102"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15701695" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomic%20insights%20that%20advance%20the%20species%20definition%20for%20prokaryotes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B31" class="ref-content ">
<div class="label">31</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Hao</div> <div class="given-names">X.</div>
</div>, <div class="name string-name">
<div class="surname">Jiang</div> <div class="given-names">R.</div>
</div>, <div class="name string-name">
<div class="surname">Chen</div> <div class="given-names">T.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">Clustering 16S rRNA for OTU prediction: a method of unsupervised Bayesian clustering</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">611</div>–<div class="lpage">8</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Clustering%2016S%20rRNA%20for%20OTU%20prediction%3A%20a%20method%20of%20unsupervised%20Bayesian%20clustering&amp;author=X.%20Hao&amp;author=R.%20Jiang&amp;author=T.%20Chen&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=611-8" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq725" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq725" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq725"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21233169" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Clustering%2016S%20rRNA%20for%20OTU%20prediction%3A%20a%20method%20of%20unsupervised%20Bayesian%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B32" class="ref-content ">
<div class="label">32</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Lozupone</div> <div class="given-names">C.</div>
</div>, <div class="name string-name">
<div class="surname">Lladser</div> <div class="given-names">M. E.</div>
</div>, <div class="name string-name">
<div class="surname">Knights</div> <div class="given-names">D.</div>
</div>, <div class="name string-name">
<div class="surname">Stombaugh</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Knight</div> <div class="given-names">R.</div>
</div>
</div> <div class="year">2011</div>, 
<div class="article-title">UniFrac: an effective distance metric for microbial community comparison</div>, <div class="source">Isme J</div> ., <div class="volume">5</div>, <div class="fpage">169</div>–<div class="lpage">72</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=UniFrac%3A%20an%20effective%20distance%20metric%20for%20microbial%20community%20comparison&amp;author=C.%20Lozupone&amp;author=M.%20E.%20Lladser&amp;author=D.%20Knights&amp;author=J.%20Stombaugh&amp;author=R.%20Knight&amp;publication_year=2011&amp;journal=Isme%20J&amp;volume=5&amp;pages=169-72" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ismej.2010.133" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2010.133" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fismej.2010.133"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20827291" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:UniFrac%3A%20an%20effective%20distance%20metric%20for%20microbial%20community%20comparison&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B33" class="ref-content ">
<div class="label">33</div>
<div class="citation mixed-citation">
<div class="person-group"><div class="name string-name">
<div class="surname">Felsenstein</div> <div class="given-names">J.</div>
</div></div> <div class="year">1981</div>, 
<div class="article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach</div>, <div class="source">J. Mol. Evol</div> ., <div class="volume">17</div>, <div class="fpage">368</div>–<div class="lpage">76</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;author=J.%20Felsenstein&amp;publication_year=1981&amp;journal=J.%20Mol.%20Evol&amp;volume=17&amp;pages=368-76" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF01734359" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01734359" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF01734359"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7288891" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B34" class="ref-content ">
<div class="label">34</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">McMurdie</div> <div class="given-names">P. J.</div>
</div>, <div class="name string-name">
<div class="surname">Holmes</div> <div class="given-names">S.</div>
</div>
</div> <div class="year">2013</div>, 
<div class="article-title">phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e61217</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=phyloseq%3A%20an%20R%20package%20for%20reproducible%20interactive%20analysis%20and%20graphics%20of%20microbiome%20census%20data&amp;author=P.%20J.%20McMurdie&amp;author=S.%20Holmes&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e61217" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0061217" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0061217" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0061217"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23630581" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:phyloseq%3A%20an%20R%20package%20for%20reproducible%20interactive%20analysis%20and%20graphics%20of%20microbiome%20census%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
<div content-id="dsy045-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy045-B35" class="ref-content ">
<div class="label">35</div>
<div class="citation mixed-citation">
<div class="person-group">
<div class="name string-name">
<div class="surname">Nelson</div> <div class="given-names">M. C.</div>
</div>, <div class="name string-name">
<div class="surname">Morrison</div> <div class="given-names">H. G.</div>
</div>, <div class="name string-name">
<div class="surname">Benjamino</div> <div class="given-names">J.</div>
</div>, <div class="name string-name">
<div class="surname">Grim</div> <div class="given-names">S. L.</div>
</div>, <div class="name string-name">
<div class="surname">Graf</div> <div class="given-names">J.</div>
</div>
</div> <div class="year">2014</div>, 
<div class="article-title">Analysis, optimization and verification of illumina-generated 16S rRNA gene amplicon surveys</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e94249</div>.<div class="citation-links">
<div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analysis%2C%20optimization%20and%20verification%20of%20illumina-generated%2016S%20rRNA%20gene%20amplicon%20surveys&amp;author=M.%20C.%20Nelson&amp;author=H.%20G.%20Morrison&amp;author=J.%20Benjamino&amp;author=S.%20L.%20Grim&amp;author=J.%20Graf&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e94249" target="_blank">Google Scholar</a></div>
<div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0094249" target="_blank">Crossref</a></div>
<div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0094249" target="_blank">Search ADS</a></div>
<div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0094249"> </span></div>
<div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24722003" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div>
<div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analysis%2C%20optimization%20and%20verification%20of%20illumina-generated%2016S%20rRNA%20gene%20amplicon%20surveys&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div> </div>
</div>
</div></div></div>
</div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="dsy045-FM1"><div class="fn"><div class="footnote-content">Deepak Yadav and Anirban Dutta authors contributed equally to this work.</div></div></div>
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div>
<div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div>
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div>
<!-- /.widget-items -->
</div>
<!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full%20Papers">Full Papers</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-20126971" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Nakai</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Nakai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakai%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nakai%20Kenta%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Kenta Nakai</a>
                            </span>
                        <br>
</div>
            </div>
        </div>
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5280828&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5275">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink">
<a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/26/2/10.1093_dnares_dsy045/1/dsy045_supp.zip?Expires=1576700043&amp;Signature=wx5q99Zc27BjgCAXQMavuTd5J0hLjMFgdHWjQiyeDIMOORK1lE0f6H0uw1VSfytWmbKmDkUU0w012gZpQIbrYYr2ELoQfgEHAIbw8ZYSfNyLwiG-Z-q4yPkn8Y9FWVZbsulxpvTgxaDbaC6firUmB1SxkcC2lgpknyiFj0uW53aKucPs5nCfBEp5Ne5CjYoYWYJHKMpyjjLvIONNt~ZscMeuWEV0sZiIDkp7YoPHLwpaC9AlIL43w-nV0efQr6QRM9peTd0igarusuCsZrMyHmiFKR6htCX3RnHPr-S1aLkpDneHTfJxvE9C9sJd1O~mWhvEg910t1fngzK0WiMqCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,445</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/dnares/dsy045"></div>
         <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
</div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5280828"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
<li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493006800004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f26%2f2%2f147%2f5280828">Google Scholar</a>
                </li>
        </ul>
</div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000493006800004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F26%2F2%2F147%2F5280828" target="_blank">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/dnaresearch/crossref-citedby/5280828">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
<li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
<section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent"><div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz023/5620723?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Sequencing of the black rockfish chromosomal genome provides insight into sperm storage in the female ovary
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/433/5588861?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The histone modification H3 lysine 27 tri-methylation has conserved gene regulatory roles in the triplicated genome of  Brassica rapa  L.
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/423/5580662?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The genome of  Populus alba  x  Populus tremula  var.  glandulosa  clone 84K
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsz022/5572119?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                TASUKE+: a web-based platform for exploring genome-wide association studies results and large-scale resequencing data
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="/dnaresearch/article/26/5/411/5556422?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The quagga mussel genome and the evolution of freshwater tolerance
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div>
<!-- /.widget-layout-[orientation] -->
</div>
        </section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide"></section><section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide"></section>
</div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" type="hidden" value="OTUX: V-region specific OTU database for improved 16S rRNA OTU picking and efficient cross-study taxonomic comparison of microbiomes | DNA Research | Oxford Academic"><input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" type="hidden" value="29">
</div>
<!-- /.center-inner-row no-overflow -->
        </section>
</div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer"><script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&amp;domready=1"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&amp;render=explicit"></script><script type="text/javascript" src="https://cdn.jsdelivr.net/chartist.js/latest/chartist.min.js"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213366114614/article-page.min.js"></script><script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy045");
            googletag.enableServices();
        });
    </script><div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
<li class="link-1">
                                    <a href="/dnaresearch/pages/About">About DNA Research</a>
                                </li>
                                <li class="link-2">
                                    <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul>
<ul>
<li class="link-1">
                                    <a href="https://twitter.com/OxfordJournals">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                                </li>

                    </ul>
</div>
<!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/dh" target="">
                        <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research"></a>
            </div>
<!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
<li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright © 2019 Kazusa DNA Research Institute</li>
                </ul>
</div>
<!-- /.journal-footer-colophon -->


        </div>
<!-- /.center-inner-row -->
    </div>
<!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
<li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a>
</li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
<li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
<li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
<li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
<li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
</div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal>
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal>
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f26%2f2%2f147%2f5280828">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script type="text/javascript" src="//oup.silverchair-cdn.com/UI/app/jsdist/v-637087213368770767/footer.min.js"></script><!-- Copyright 2001-2010, IBM Corporation All rights reserved. --><script src="//ouptag.scholarlyiq.com/ntpagetag.js"></script><noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
</noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input type="hidden" class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1">
</div>

    </div>

</body>
</html>
